commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases with associated phenotypes/en_product4.xml b/Rare diseases with associated phenotypes/en_product4.xml
index a5765b2..4a0a497 100644
--- a/Rare diseases with associated phenotypes/en_product4.xml	
+++ b/Rare diseases with associated phenotypes/en_product4.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:33:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:21:21" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <HPODisorderSetStatusList count="4245">
     <HPODisorderSetStatus id="1">
       <Disorder id="2">
@@ -120997,7 +120990,7 @@
       <Disorder id="1195">
         <OrphaCode>1878</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1878</ExpertLink>
-        <Name lang="en">TRIM32-related  limb-girdle muscular dystrophy R8</Name>
+        <Name lang="en">TRIM32-related limb-girdle muscular dystrophy R8</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -596942,7 +596935,7 @@
       <Disorder id="10338">
         <OrphaCode>34516</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34516</ExpertLink>
-        <Name lang="en">DNAJB6-related  limb-girdle muscular dystrophy D1</Name>
+        <Name lang="en">DNAJB6-related limb-girdle muscular dystrophy D1</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -613047,7 +613040,7 @@
       <Disorder id="10337">
         <OrphaCode>34515</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-        <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+        <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -614569,7 +614562,7 @@
       <Disorder id="22329">
         <OrphaCode>363623</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-        <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+        <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -619407,7 +619400,7 @@
       <Disorder id="870">
         <OrphaCode>267</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=267</ExpertLink>
-        <Name lang="en">Calpain-3-related  limb-girdle muscular dystrophy R1</Name>
+        <Name lang="en">Calpain-3-related limb-girdle muscular dystrophy R1</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -619636,7 +619629,7 @@
       <Disorder id="8715">
         <OrphaCode>119</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=119</ExpertLink>
-        <Name lang="en">Beta-sarcoglycan-related  limb-girdle muscular dystrophy R4</Name>
+        <Name lang="en">Beta-sarcoglycan-related limb-girdle muscular dystrophy R4</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -620774,7 +620767,7 @@
       <Disorder id="11732">
         <OrphaCode>86812</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-        <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+        <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -622655,7 +622648,7 @@
       <Disorder id="678">
         <OrphaCode>62</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=62</ExpertLink>
-        <Name lang="en">Alpha-sarcoglycan-related  limb-girdle muscular dystrophy R3</Name>
+        <Name lang="en">Alpha-sarcoglycan-related limb-girdle muscular dystrophy R3</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -625430,7 +625423,7 @@
       <Disorder id="8717">
         <OrphaCode>219</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=219</ExpertLink>
-        <Name lang="en">Delta-sarcoglycan-related  limb-girdle muscular dystrophy R6</Name>
+        <Name lang="en">Delta-sarcoglycan-related limb-girdle muscular dystrophy R6</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -625817,7 +625810,7 @@
       <Disorder id="22434">
         <OrphaCode>369840</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=369840</ExpertLink>
-        <Name lang="en">TRAPPC11-related  limb-girdle muscular dystrophy R18</Name>
+        <Name lang="en">TRAPPC11-related limb-girdle muscular dystrophy R18</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -636023,7 +636016,7 @@
       <Disorder id="8716">
         <OrphaCode>353</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353</ExpertLink>
-        <Name lang="en">Gamma-sarcoglycan-related  limb-girdle muscular dystrophy R5</Name>
+        <Name lang="en">Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -637350,7 +637343,7 @@
       <Disorder id="18520">
         <OrphaCode>206549</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206549</ExpertLink>
-        <Name lang="en">Anoctamin-5-related  limb-girdle muscular dystrophy R12</Name>
+        <Name lang="en">Anoctamin-5-related limb-girdle muscular dystrophy R12</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -702919,7 +702912,7 @@
       <Disorder id="19742">
         <OrphaCode>254361</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=254361</ExpertLink>
-        <Name lang="en">Plectin-related  limb-girdle muscular dystrophy R17</Name>
+        <Name lang="en">Plectin-related limb-girdle muscular dystrophy R17</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -703567,7 +703560,7 @@
       <Disorder id="22066">
         <OrphaCode>352479</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-        <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+        <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -754954,7 +754947,7 @@
       <Disorder id="8720">
         <OrphaCode>268</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268</ExpertLink>
-        <Name lang="en">Dysferlin-related  limb-girdle muscular dystrophy R2</Name>
+        <Name lang="en">Dysferlin-related limb-girdle muscular dystrophy R2</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -912113,7 +912106,7 @@
       <Disorder id="20455">
         <OrphaCode>280333</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280333</ExpertLink>
-        <Name lang="en">Alpha-dystroglycan-related  limb-girdle muscular dystrophy R16</Name>
+        <Name lang="en">Alpha-dystroglycan-related limb-girdle muscular dystrophy R16</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -914376,7 +914369,7 @@
       <Disorder id="18522">
         <OrphaCode>206559</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-        <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+        <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
