commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases with associated phenotypes/es_product4.xml b/Rare diseases with associated phenotypes/es_product4.xml
index 9cf33e0..cc6b47a 100644
--- a/Rare diseases with associated phenotypes/es_product4.xml	
+++ b/Rare diseases with associated phenotypes/es_product4.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:34:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:22:25" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <HPODisorderSetStatusList count="4245">
     <HPODisorderSetStatus id="1">
       <Disorder id="2">
@@ -86275,7 +86268,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=442</ExpertLink>
         <Name lang="es">Hipotiroidismo congénito</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -303914,7 +303907,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79168</ExpertLink>
         <Name lang="es">Anomalía en la síntesis de ácidos biliares</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -326867,7 +326860,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79357</ExpertLink>
         <Name lang="es">Queratodermia palmoplantar hereditaria</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -326906,7 +326899,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79358</ExpertLink>
         <Name lang="es">Poroqueratosis</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -339786,7 +339779,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90970</ExpertLink>
         <Name lang="es">Lipodistrofias primarias</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -351385,7 +351378,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100100</ExpertLink>
         <Name lang="es">Tumor del timo</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -357367,7 +357360,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98292</ExpertLink>
         <Name lang="es">Mastocitosis</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -403576,7 +403569,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=181393</ExpertLink>
         <Name lang="es">Síndrome de insensibilidad a la hormona de crecimiento</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -414083,7 +414076,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=226292</ExpertLink>
         <Name lang="es">Hipotiroidismo congénito permanente</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -485944,7 +485937,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98028</ExpertLink>
         <Name lang="es">Enfermedad rara del sistema circulatorio</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -597840,7 +597833,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=261920</ExpertLink>
         <Name lang="es">Deleción parcial del brazo corto del cromosoma 8</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -608315,7 +608308,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100076</ExpertLink>
         <Name lang="es">Tumor neuroendocrino del duodeno</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -608684,7 +608677,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100077</ExpertLink>
         <Name lang="es">Tumor neuroendocrino yeyunal</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -721412,7 +721405,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=262767</ExpertLink>
         <Name lang="es">Trisomía/tetrasomía parcial del brazo corto del cromosoma 9</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -762439,7 +762432,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217607</ExpertLink>
         <Name lang="es">Miocardiopatía dilatada familiar</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -764018,7 +764011,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=261911</ExpertLink>
         <Name lang="es">Deleción parcial del brazo corto del cromosoma 7</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -921615,7 +921608,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=352</ExpertLink>
         <Name lang="es">Galactosemia</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
@@ -929966,7 +929959,7 @@
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99</ExpertLink>
         <Name lang="es">Ataxia cerebelosa autosómica dominante</Name>
         <DisorderType id="36561">
-          <Name lang="es">Category</Name>
+          <Name lang="es">Categoría</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="es">Grupo de trastornos</Name>
