commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases with associated phenotypes/es_product4.xml b/Rare diseases with associated phenotypes/es_product4.xml
index 0485f14..e7d9313 100644
--- a/Rare diseases with associated phenotypes/es_product4.xml	
+++ b/Rare diseases with associated phenotypes/es_product4.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:48:48" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:49:04" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -89879,7 +89879,7 @@
           <Name lang="es">Trastorno</Name>
         </DisorderGroup>
         <HPODisorderAssociationList count="36">
-          <HPODisorderAssociation id="371723">
+          <HPODisorderAssociation id="530025">
             <HPO id="203">
               <HPOId>HP:0000093</HPOId>
               <HPOTerm>Proteinuria</HPOTerm>
@@ -89889,7 +89889,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371724">
+          <HPODisorderAssociation id="530026">
             <HPO id="838">
               <HPOId>HP:0000509</HPOId>
               <HPOTerm>Conjunctivitis</HPOTerm>
@@ -89901,7 +89901,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371725">
+          <HPODisorderAssociation id="530027">
             <HPO id="156">
               <HPOId>HP:0000988</HPOId>
               <HPOTerm>Skin rash</HPOTerm>
@@ -89913,7 +89913,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371726">
+          <HPODisorderAssociation id="530028">
             <HPO id="174">
               <HPOId>HP:0002633</HPOId>
               <HPOTerm>Vasculitis</HPOTerm>
@@ -89923,7 +89923,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371727">
+          <HPODisorderAssociation id="530029">
             <HPO id="5677">
               <HPOId>HP:0025289</HPOId>
               <HPOTerm>Cervical lymphadenopathy</HPOTerm>
@@ -89935,7 +89935,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371728">
+          <HPODisorderAssociation id="530030">
             <HPO id="7330">
               <HPOId>HP:0025493</HPOId>
               <HPOTerm>Palmoplantar erythema</HPOTerm>
@@ -89947,7 +89947,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371729">
+          <HPODisorderAssociation id="530031">
             <HPO id="834">
               <HPOId>HP:0100776</HPOId>
               <HPOTerm>Recurrent pharyngitis</HPOTerm>
@@ -89957,7 +89957,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371730">
+          <HPODisorderAssociation id="530032">
             <HPO id="399">
               <HPOId>HP:0100825</HPOId>
               <HPOTerm>Cheilitis</HPOTerm>
@@ -89969,7 +89969,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371731">
+          <HPODisorderAssociation id="530033">
             <HPO id="1562">
               <HPOId>HP:0000206</HPOId>
               <HPOTerm>Glossitis</HPOTerm>
@@ -89981,7 +89981,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371732">
+          <HPODisorderAssociation id="530034">
             <HPO id="277">
               <HPOId>HP:0000969</HPOId>
               <HPOTerm>Edema</HPOTerm>
@@ -89991,7 +89991,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371733">
+          <HPODisorderAssociation id="530035">
             <HPO id="80">
               <HPOId>HP:0001369</HPOId>
               <HPOTerm>Arthritis</HPOTerm>
@@ -90001,7 +90001,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371734">
+          <HPODisorderAssociation id="530036">
             <HPO id="291">
               <HPOId>HP:0001654</HPOId>
               <HPOTerm>Abnormal heart valve morphology</HPOTerm>
@@ -90011,7 +90011,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371735">
+          <HPODisorderAssociation id="530037">
             <HPO id="161">
               <HPOId>HP:0001701</HPOId>
               <HPOTerm>Pericarditis</HPOTerm>
@@ -90021,7 +90021,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371736">
+          <HPODisorderAssociation id="530038">
             <HPO id="166">
               <HPOId>HP:0001945</HPOId>
               <HPOTerm>Fever</HPOTerm>
@@ -90033,7 +90033,7 @@
               <Name lang="es">Criterios de diagnóstico</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371737">
+          <HPODisorderAssociation id="530039">
             <HPO id="1816">
               <HPOId>HP:0001974</HPOId>
               <HPOTerm>Leukocytosis</HPOTerm>
@@ -90043,7 +90043,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371738">
+          <HPODisorderAssociation id="530040">
             <HPO id="413">
               <HPOId>HP:0002014</HPOId>
               <HPOTerm>Diarrhea</HPOTerm>
@@ -90053,7 +90053,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371739">
+          <HPODisorderAssociation id="530041">
             <HPO id="167">
               <HPOId>HP:0002027</HPOId>
               <HPOTerm>Abdominal pain</HPOTerm>
@@ -90063,7 +90063,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371740">
+          <HPODisorderAssociation id="530042">
             <HPO id="185">
               <HPOId>HP:0012378</HPOId>
               <HPOTerm>Fatigue</HPOTerm>
@@ -90073,7 +90073,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371741">
+          <HPODisorderAssociation id="530043">
             <HPO id="1819">
               <HPOId>HP:0100643</HPOId>
               <HPOTerm>Abnormality of nail color</HPOTerm>
@@ -90083,7 +90083,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371742">
+          <HPODisorderAssociation id="530044">
             <HPO id="7">
               <HPOId>HP:0000508</HPOId>
               <HPOTerm>Ptosis</HPOTerm>
@@ -90093,7 +90093,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371743">
+          <HPODisorderAssociation id="530045">
             <HPO id="266">
               <HPOId>HP:0000737</HPOId>
               <HPOTerm>Irritability</HPOTerm>
@@ -90103,7 +90103,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371744">
+          <HPODisorderAssociation id="530046">
             <HPO id="346">
               <HPOId>HP:0000952</HPOId>
               <HPOTerm>Jaundice</HPOTerm>
@@ -90113,7 +90113,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371745">
+          <HPODisorderAssociation id="530047">
             <HPO id="158">
               <HPOId>HP:0001082</HPOId>
               <HPOTerm>Cholecystitis</HPOTerm>
@@ -90123,7 +90123,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371746">
+          <HPODisorderAssociation id="530048">
             <HPO id="753">
               <HPOId>HP:0001287</HPOId>
               <HPOTerm>Meningitis</HPOTerm>
@@ -90133,7 +90133,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371747">
+          <HPODisorderAssociation id="530049">
             <HPO id="318">
               <HPOId>HP:0001635</HPOId>
               <HPOTerm>Congestive heart failure</HPOTerm>
@@ -90143,7 +90143,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371748">
+          <HPODisorderAssociation id="530050">
             <HPO id="35">
               <HPOId>HP:0002017</HPOId>
               <HPOTerm>Nausea and vomiting</HPOTerm>
@@ -90153,7 +90153,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371749">
+          <HPODisorderAssociation id="530051">
             <HPO id="508">
               <HPOId>HP:0002076</HPOId>
               <HPOTerm>Migraine</HPOTerm>
@@ -90163,7 +90163,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371750">
+          <HPODisorderAssociation id="530052">
             <HPO id="175">
               <HPOId>HP:0002829</HPOId>
               <HPOTerm>Arthralgia</HPOTerm>
@@ -90173,7 +90173,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371751">
+          <HPODisorderAssociation id="530053">
             <HPO id="631">
               <HPOId>HP:0005111</HPOId>
               <HPOTerm>Dilatation of the ascending aorta</HPOTerm>
@@ -90183,7 +90183,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371752">
+          <HPODisorderAssociation id="530054">
             <HPO id="767">
               <HPOId>HP:0006530</HPOId>
               <HPOTerm>Interstitial pulmonary abnormality</HPOTerm>
@@ -90193,7 +90193,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371753">
+          <HPODisorderAssociation id="530055">
             <HPO id="220">
               <HPOId>HP:0006824</HPOId>
               <HPOTerm>Cranial nerve paralysis</HPOTerm>
@@ -90203,7 +90203,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371754">
+          <HPODisorderAssociation id="530056">
             <HPO id="1817">
               <HPOId>HP:0011658</HPOId>
               <HPOTerm>Double outlet right ventricle with subpulmonary ventricular septal defect without pulmonary stenosis</HPOTerm>
@@ -90213,7 +90213,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371755">
+          <HPODisorderAssociation id="530057">
             <HPO id="183">
               <HPOId>HP:0011675</HPOId>
               <HPOTerm>Arrhythmia</HPOTerm>
@@ -90223,7 +90223,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371756">
+          <HPODisorderAssociation id="530058">
             <HPO id="184">
               <HPOId>HP:0012115</HPOId>
               <HPOTerm>Hepatitis</HPOTerm>
@@ -90233,7 +90233,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371757">
+          <HPODisorderAssociation id="530059">
             <HPO id="189">
               <HPOId>HP:0012819</HPOId>
               <HPOTerm>Myocarditis</HPOTerm>
@@ -90243,7 +90243,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371758">
+          <HPODisorderAssociation id="530060">
             <HPO id="1818">
               <HPOId>HP:0100586</HPOId>
               <HPOTerm>Sterile pyuria</HPOTerm>
@@ -179162,7 +179162,7 @@
       <Disorder id="1749">
         <OrphaCode>1660</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1660</ExpertLink>
-        <Name lang="es">Dermo-odonto displasia</Name>
+        <Name lang="es">Dermo-odonto-displasia</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -467859,7 +467859,7 @@
       <Disorder id="10706">
         <OrphaCode>52901</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=52901</ExpertLink>
-        <Name lang="es">Deficiencia aislada de hormona estimulante folicular (FSH)</Name>
+        <Name lang="es">Deficiencia aislada de hormona folículo-estimulante</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -559189,7 +559189,7 @@
       <Disorder id="10986">
         <OrphaCode>71211</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=71211</ExpertLink>
-        <Name lang="es">Neuromielitis óptica</Name>
+        <Name lang="es">Trastorno del espectro de la neuromielitis óptica</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -830059,7 +830059,7 @@
       <Disorder id="21947">
         <OrphaCode>329918</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=329918</ExpertLink>
-        <Name lang="es">Glomerulonefritis membranoproliferativa no mediada por inmunoglobulinas</Name>
+        <Name lang="es">Glomerulopatía C3</Name>
         <DisorderType id="21450">
           <Name lang="es">Subtipo clínico</Name>
         </DisorderType>
