commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases with associated phenotypes/es_product4.xml b/Rare diseases with associated phenotypes/es_product4.xml
index 5270e6a..0485f14 100644
--- a/Rare diseases with associated phenotypes/es_product4.xml	
+++ b/Rare diseases with associated phenotypes/es_product4.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:47:08" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:48:48" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <HPODisorderSetStatusList count="4261">
+  <HPODisorderSetStatusList count="4259">
     <HPODisorderSetStatus id="1">
       <Disorder id="2">
         <OrphaCode>58</OrphaCode>
@@ -46005,7 +46005,7 @@
       <Disorder id="373">
         <OrphaCode>2773</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2773</ExpertLink>
-        <Name lang="es">Síndrome de osteogénesis imperfecta-retinopatía-convulsiones-discapacidad intelectual</Name>
+        <Name lang="es">Síndrome de osteogénesis imperfecta-retinopatía-crisis-discapacidad intelectual</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -163080,7 +163080,7 @@
       <Disorder id="1720">
         <OrphaCode>1568</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1568</ExpertLink>
-        <Name lang="es">Síndrome de discapacidad intelectual ligada al cromosoma X-malformación de Dandy-Walker-enfermedad de los ganglios basales-convulsiones</Name>
+        <Name lang="es">Síndrome de discapacidad intelectual ligada al cromosoma X-malformación de Dandy-Walker-enfermedad de los ganglios basales-crisis</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -305205,7 +305205,7 @@
       <Disorder id="11182">
         <OrphaCode>79156</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79156</ExpertLink>
-        <Name lang="es">Síndrome de convulsiones-discapacidad intelectual por hidroxilsinuria</Name>
+        <Name lang="es">Síndrome de crisis-discapacidad intelectual por hidroxilsinuria</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -337616,7 +337616,7 @@
       <Disorder id="12210">
         <OrphaCode>93267</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93267</ExpertLink>
-        <Name lang="es">Síndrome de cráneo en hoja de trébol-anomalías congénitas múltiples</Name>
+        <Name lang="es">Síndrome de cráneo en trébol-anomalías congénitas múltiples</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -456184,7 +456184,7 @@
       <Disorder id="22452">
         <OrphaCode>369950</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=369950</ExpertLink>
-        <Name lang="es">Síndrome de discapacidad intelectual-convulsiones-macrocefalia-obesidad</Name>
+        <Name lang="es">Síndrome de discapacidad intelectual-crisis-macrocefalia-obesidad</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -472319,7 +472319,7 @@
       <Disorder id="22449">
         <OrphaCode>369929</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=369929</ExpertLink>
-        <Name lang="es">Síndrome de hiperaldosteronismo primario-convulsiones-anomalías neurológicas</Name>
+        <Name lang="es">Síndrome de hiperaldosteronismo primario-crisis-anomalías neurológicas</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -483512,125 +483512,6 @@
       <Online>y</Online>
       <ValidationDate>2017-05-01 00:00:00.0</ValidationDate>
     </HPODisorderSetStatus>
-    <HPODisorderSetStatus id="2713">
-      <Disorder id="11041">
-        <OrphaCode>73247</OrphaCode>
-        <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73247</ExpertLink>
-        <Name lang="es">Esofagitis eosinofílica</Name>
-        <DisorderType id="21394">
-          <Name lang="es">Enfermedad</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="es">Trastorno</Name>
-        </DisorderGroup>
-        <HPODisorderAssociationList count="10">
-          <HPODisorderAssociation id="173898">
-            <HPO id="277">
-              <HPOId>HP:0000969</HPOId>
-              <HPOTerm>Edema</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28412">
-              <Name lang="es">Muy frecuente (99-80%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173899">
-            <HPO id="34">
-              <HPOId>HP:0002015</HPOId>
-              <HPOTerm>Dysphagia</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28412">
-              <Name lang="es">Muy frecuente (99-80%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173900">
-            <HPO id="3544">
-              <HPOId>HP:0002031</HPOId>
-              <HPOTerm>Abnormal esophagus morphology</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28412">
-              <Name lang="es">Muy frecuente (99-80%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173901">
-            <HPO id="1478">
-              <HPOId>HP:0002043</HPOId>
-              <HPOTerm>Esophageal stricture</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28412">
-              <Name lang="es">Muy frecuente (99-80%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173902">
-            <HPO id="4723">
-              <HPOId>HP:0005240</HPOId>
-              <HPOTerm>Esophageal obstruction</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28412">
-              <Name lang="es">Muy frecuente (99-80%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173903">
-            <HPO id="145">
-              <HPOId>HP:0002020</HPOId>
-              <HPOTerm>Gastroesophageal reflux</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173904">
-            <HPO id="1543">
-              <HPOId>HP:0010450</HPOId>
-              <HPOTerm>Esophageal stenosis</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173905">
-            <HPO id="397">
-              <HPOId>HP:0100749</HPOId>
-              <HPOTerm>Chest pain</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173906">
-            <HPO id="1975">
-              <HPOId>HP:0005203</HPOId>
-              <HPOTerm>Spontaneous esophageal perforation</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="173907">
-            <HPO id="4853">
-              <HPOId>HP:0025270</HPOId>
-              <HPOTerm>Abnormality of esophagus physiology</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-        </HPODisorderAssociationList>
-      </Disorder>
-      <Source>27158535[PMID]</Source>
-      <ValidationStatus>y</ValidationStatus>
-      <Online>y</Online>
-      <ValidationDate>2017-05-09 00:00:00.0</ValidationDate>
-    </HPODisorderSetStatus>
     <HPODisorderSetStatus id="2714">
       <Disorder id="10373">
         <OrphaCode>35122</OrphaCode>
@@ -701011,7 +700892,7 @@
       <Disorder id="26327">
         <OrphaCode>505237</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=505237</ExpertLink>
-        <Name lang="es">Síndrome de inicio temprano de convulsiones-anomalías de las extremidades distales-dismorfia facial-retraso generalizado del desarrollo</Name>
+        <Name lang="es">Síndrome de crisis de inicio precoz-anomalías de las extremidades distales-dismorfia facial-retraso generalizado del desarrollo</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -731776,7 +731657,7 @@
       <Disorder id="11138">
         <OrphaCode>79096</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79096</ExpertLink>
-        <Name lang="es">Convulsiones sensibles al piridoxal fosfato</Name>
+        <Name lang="es">Epilepsia sensible al piridoxal fosfato</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -765764,7 +765645,7 @@
       <Disorder id="10424">
         <OrphaCode>36387</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=36387</ExpertLink>
-        <Name lang="es">Epilepsia generalizada con convulsiones febriles plus</Name>
+        <Name lang="es">Epilepsia generalizada con crisis febriles plus</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -808107,7 +807988,7 @@
       <Disorder id="27609">
         <OrphaCode>529665</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=529665</ExpertLink>
-        <Name lang="es">Síndrome de retraso del neurodesarrollo-convulsiones-anomalías oftálmicas-osteopenia-atrofia cerebelosa</Name>
+        <Name lang="es">Síndrome de retraso del neurodesarrollo-crisis-anomalías oftálmicas-osteopenia-atrofia cerebelosa</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -836583,7 +836464,7 @@
       <Disorder id="27025">
         <OrphaCode>513456</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=513456</ExpertLink>
-        <Name lang="es">Síndrome de discapacidad intelectual-convulsiones-trastornos de la marcha-dismorfia facial</Name>
+        <Name lang="es">Síndrome de discapacidad intelectual-crisis-trastornos de la marcha-dismorfia facial</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -851015,7 +850896,7 @@
       <Disorder id="24178">
         <OrphaCode>466926</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=466926</ExpertLink>
-        <Name lang="es">Síndrome de convulsiones-escoliosis-macrocefalia</Name>
+        <Name lang="es">Síndrome de crisis-escoliosis-macrocefalia</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -883062,7 +882943,7 @@
       <Disorder id="22433">
         <OrphaCode>369837</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=369837</ExpertLink>
-        <Name lang="es">Síndrome de discapacidad intelectual-convulsiones-hipofosfatasia-anomalías oftalmológicas y esqueléticas</Name>
+        <Name lang="es">Síndrome de discapacidad intelectual-crisis-hipofosfatasia-anomalías oftalmológicas y esqueléticas</Name>
         <DisorderType id="21401">
           <Name lang="es">Síndrome de malformación</Name>
         </DisorderType>
@@ -898817,315 +898698,6 @@
       <Online>y</Online>
       <ValidationDate>2019-10-25 00:00:00.0</ValidationDate>
     </HPODisorderSetStatus>
-    <HPODisorderSetStatus id="3921">
-      <Disorder id="813">
-        <OrphaCode>1549</OrphaCode>
-        <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1549</ExpertLink>
-        <Name lang="es">Criptosporidiosis</Name>
-        <DisorderType id="21394">
-          <Name lang="es">Enfermedad</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="es">Trastorno</Name>
-        </DisorderGroup>
-        <HPODisorderAssociationList count="29">
-          <HPODisorderAssociation id="286205">
-            <HPO id="413">
-              <HPOId>HP:0002014</HPOId>
-              <HPOTerm>Diarrhea</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286206">
-            <HPO id="258">
-              <HPOId>HP:0002721</HPOId>
-              <HPOTerm>Immunodeficiency</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286207">
-            <HPO id="4492">
-              <HPOId>HP:0005407</HPOId>
-              <HPOTerm>Decreased proportion of CD4-positive T cells</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286208">
-            <HPO id="6815">
-              <HPOId>HP:0011839</HPOId>
-              <HPOTerm>Abnormal T cell count</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28419">
-              <Name lang="es">Frecuente (79-30%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286209">
-            <HPO id="30">
-              <HPOId>HP:0001508</HPOId>
-              <HPOTerm>Failure to thrive</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286210">
-            <HPO id="1377">
-              <HPOId>HP:0001510</HPOId>
-              <HPOTerm>Growth delay</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286211">
-            <HPO id="163">
-              <HPOId>HP:0001824</HPOId>
-              <HPOTerm>Weight loss</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286212">
-            <HPO id="214">
-              <HPOId>HP:0002013</HPOId>
-              <HPOTerm>Vomiting</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286213">
-            <HPO id="1306">
-              <HPOId>HP:0002018</HPOId>
-              <HPOTerm>Nausea</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286214">
-            <HPO id="167">
-              <HPOId>HP:0002027</HPOId>
-              <HPOTerm>Abdominal pain</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286215">
-            <HPO id="168">
-              <HPOId>HP:0002028</HPOId>
-              <HPOTerm>Chronic diarrhea</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286216">
-            <HPO id="414">
-              <HPOId>HP:0002039</HPOId>
-              <HPOTerm>Anorexia</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286217">
-            <HPO id="3552">
-              <HPOId>HP:0002254</HPOId>
-              <HPOTerm>Intermittent diarrhea</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286218">
-            <HPO id="1795">
-              <HPOId>HP:0011134</HPOId>
-              <HPOTerm>Low-grade fever</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286219">
-            <HPO id="4724">
-              <HPOId>HP:0011848</HPOId>
-              <HPOTerm>Abdominal colic</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28426">
-              <Name lang="es">Ocasional (29-5%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286220">
-            <HPO id="572">
-              <HPOId>HP:0001080</HPOId>
-              <HPOTerm>Biliary tract abnormality</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286221">
-            <HPO id="878">
-              <HPOId>HP:0001609</HPOId>
-              <HPOTerm>Hoarse voice</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286222">
-            <HPO id="162">
-              <HPOId>HP:0001733</HPOId>
-              <HPOTerm>Pancreatitis</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286223">
-            <HPO id="212">
-              <HPOId>HP:0001944</HPOId>
-              <HPOTerm>Dehydration</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286224">
-            <HPO id="34">
-              <HPOId>HP:0002015</HPOId>
-              <HPOTerm>Dysphagia</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286225">
-            <HPO id="3544">
-              <HPOId>HP:0002031</HPOId>
-              <HPOTerm>Abnormal esophagus morphology</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286226">
-            <HPO id="536">
-              <HPOId>HP:0002098</HPOId>
-              <HPOTerm>Respiratory distress</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286227">
-            <HPO id="2311">
-              <HPOId>HP:0002878</HPOId>
-              <HPOTerm>Respiratory failure</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286228">
-            <HPO id="5113">
-              <HPOId>HP:0004796</HPOId>
-              <HPOTerm>Gastrointestinal obstruction</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286229">
-            <HPO id="4135">
-              <HPOId>HP:0011947</HPOId>
-              <HPOTerm>Respiratory tract infection</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286230">
-            <HPO id="3415">
-              <HPOId>HP:0012418</HPOId>
-              <HPOTerm>Hypoxemia</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286231">
-            <HPO id="391">
-              <HPOId>HP:0012735</HPOId>
-              <HPOTerm>Cough</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286232">
-            <HPO id="6287">
-              <HPOId>HP:0030151</HPOId>
-              <HPOTerm>Cholangitis</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-          <HPODisorderAssociation id="286233">
-            <HPO id="4570">
-              <HPOId>HP:0030828</HPOId>
-              <HPOTerm>Wheezing</HPOTerm>
-            </HPO>
-            <HPOFrequency id="28433">
-              <Name lang="es">Muy poco frecuente (4-1%)</Name>
-            </HPOFrequency>
-            <DiagnosticCriteria/>
-          </HPODisorderAssociation>
-        </HPODisorderAssociationList>
-      </Disorder>
-      <Source>28734513[PMID]_29114483[PMID]</Source>
-      <ValidationStatus>y</ValidationStatus>
-      <Online>y</Online>
-      <ValidationDate>2019-10-25 00:00:00.0</ValidationDate>
-    </HPODisorderSetStatus>
     <HPODisorderSetStatus id="3920">
       <Disorder id="12160">
         <OrphaCode>91547</OrphaCode>
@@ -1059804,7 +1059376,7 @@
       <Disorder id="23369">
         <OrphaCode>438213</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=438213</ExpertLink>
-        <Name lang="es">Síndrome neonatal de hipotonía-convulsiones-encefalopatía grave asociado al gen PURA</Name>
+        <Name lang="es">Síndrome neonatal de hipotonía-crisis-encefalopatía grave asociado al gen PURA</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -1060683,7 +1060255,7 @@
       <Disorder id="9793">
         <OrphaCode>31709</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=31709</ExpertLink>
-        <Name lang="es">Convulsiones del lactante y coreoatetosis</Name>
+        <Name lang="es">Crisis del lactante con coreoatetosis</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -1121050,7 +1120622,7 @@
       <Disorder id="23370">
         <OrphaCode>438216</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=438216</ExpertLink>
-        <Name lang="es">Síndrome neonatal de hipotonía-convulsiones-encefalopatía grave asociado al gen PURA por una mutación puntual</Name>
+        <Name lang="es">Síndrome neonatal de hipotonía-crisis-encefalopatía grave asociado al gen PURA por una mutación puntual</Name>
         <DisorderType id="21443">
           <Name lang="es">Subtipo etiológico</Name>
         </DisorderType>
@@ -1121449,7 +1121021,7 @@
       <Disorder id="21510">
         <OrphaCode>314655</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=314655</ExpertLink>
-        <Name lang="es">Síndrome de hipotonía-convulsiones-encefalopatía neonatal grave por microdeleción 5q31.3</Name>
+        <Name lang="es">Síndrome de hipotonía-crisis-encefalopatía neonatal grave por microdeleción 5q31.3</Name>
         <DisorderType id="21443">
           <Name lang="es">Subtipo etiológico</Name>
         </DisorderType>
@@ -1141353,7 +1140925,7 @@
       <Disorder id="871">
         <OrphaCode>1329</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-        <Name lang="es">Defecto del septo aurículoventricular completo</Name>
+        <Name lang="es">Canal aurículoventricular completo</Name>
         <DisorderType id="21415">
           <Name lang="es">Anomalía morfológica</Name>
         </DisorderType>
@@ -1172915,7 +1172487,7 @@
       <Disorder id="11139">
         <OrphaCode>79097</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79097</ExpertLink>
-        <Name lang="es">Convulsiones sensibles al ácido folínico</Name>
+        <Name lang="es">Epilepsia sensible al ácido folínico</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
