commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases with associated phenotypes/it_product4.xml b/Rare diseases with associated phenotypes/it_product4.xml
index 909d3bc..acfff8c 100644
--- a/Rare diseases with associated phenotypes/it_product4.xml	
+++ b/Rare diseases with associated phenotypes/it_product4.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:36:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:24:34" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <HPODisorderSetStatusList count="4245">
     <HPODisorderSetStatus id="1">
       <Disorder id="2">
@@ -86275,7 +86268,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=442</ExpertLink>
         <Name lang="it">Ipotiroidismo congenito</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -143785,7 +143778,7 @@
       <Disorder id="1399">
         <OrphaCode>1118</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1118</ExpertLink>
-        <Name lang="it">Aplasia del perone - ectrodattilia</Name>
+        <Name lang="it">Sindrome da aplasia del perone-ectrodattilia</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -220459,7 +220452,7 @@
       <Disorder id="2556">
         <OrphaCode>2815</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2815</ExpertLink>
-        <Name lang="it">Paraparesi spastica - sordità</Name>
+        <Name lang="it">Sindrome da paraparesi spastica-sordità</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -227595,7 +227588,7 @@
       <Disorder id="2344">
         <OrphaCode>2536</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2536</ExpertLink>
-        <Name lang="it">Microcornea - glaucoma - assenza del seno frontale</Name>
+        <Name lang="it">Sindrome da microcornea-glaucoma-assenza dei seni frontali</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -303914,7 +303907,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79168</ExpertLink>
         <Name lang="it">Malattia della sintesi degli acidi biliari</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -326867,7 +326860,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79357</ExpertLink>
         <Name lang="it">Cheratoderma palmoplantare ereditario</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -326906,7 +326899,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79358</ExpertLink>
         <Name lang="it">Porocheratosi</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -333232,7 +333225,7 @@
       <Disorder id="12316">
         <OrphaCode>93394</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93394</ExpertLink>
-        <Name lang="it">Brachidattilia, tipo A4</Name>
+        <Name lang="it">Brachidattilia tipo A4</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -339786,7 +339779,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90970</ExpertLink>
         <Name lang="it">Lipodistrofie primitive</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -351385,7 +351378,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100100</ExpertLink>
         <Name lang="it">Tumore timico</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -357367,7 +357360,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98292</ExpertLink>
         <Name lang="it">Mastocitosi</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -403576,7 +403569,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=181393</ExpertLink>
         <Name lang="it">Sindrome da resistenza all'ormone della crescita</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -414083,7 +414076,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=226292</ExpertLink>
         <Name lang="it">Ipotiroidismo congenito permanente</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -485944,7 +485937,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98028</ExpertLink>
         <Name lang="it">Malattia rara del sistema circolatorio</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -597840,7 +597833,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=261920</ExpertLink>
         <Name lang="it">Delezione parziale del braccio corto del cromosoma 8</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -608315,7 +608308,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100076</ExpertLink>
         <Name lang="it">Tumore endocrino duodenale</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -608684,7 +608677,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100077</ExpertLink>
         <Name lang="it">Tumore neuroendocrino del digiuno</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -615415,7 +615408,7 @@
       <Disorder id="21506">
         <OrphaCode>314632</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=314632</ExpertLink>
-        <Name lang="it">Parkinsonimo da deficit di ATP13A2</Name>
+        <Name lang="it">Ceroidolipofuscinosi neuronale correlata a ATP13A2</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -622406,7 +622399,7 @@
       <Disorder id="23717">
         <OrphaCode>457260</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=457260</ExpertLink>
-        <Name lang="it">Sindrome disabilità intellettiva - ipotonia - anomalie del comportamento legata all'X</Name>
+        <Name lang="it">Sindrome da disabilità intellettiva-ipotonia-disturbi del movimento, legata all'X</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -675982,7 +675975,7 @@
       <Disorder id="3521">
         <OrphaCode>3240</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3240</ExpertLink>
-        <Name lang="it">Calcificazione del sistema nervoso centrale - sordità - acidosi tubolare - anemia</Name>
+        <Name lang="it">Sindrome da calcificazione del sistema nervoso centrale-sordità-acidosi tubolare-anemia</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -682904,7 +682897,7 @@
       <Disorder id="2191">
         <OrphaCode>2339</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2339</ExpertLink>
-        <Name lang="it">Cheratosi follicolare - nanismo - atrofia cerebrale</Name>
+        <Name lang="it">Sindrome da cheratosi follicolare-nanismo-atrofia cerebrale</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -721412,7 +721405,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=262767</ExpertLink>
         <Name lang="it">Trisomia parziale del braccio corto del cromosoma 9</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -729899,7 +729892,7 @@
       <Disorder id="11181">
         <OrphaCode>79155</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79155</ExpertLink>
-        <Name lang="it">Encefalopatia da idrossichinureninuria</Name>
+        <Name lang="it">Idrossichinureninuria</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -762439,7 +762432,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=217607</ExpertLink>
         <Name lang="it">Cardiomiopatia dilatativa familiare</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -764018,7 +764011,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=261911</ExpertLink>
         <Name lang="it">Delezione parziale del braccio corto del cromosoma 7</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -793418,7 +793411,7 @@
       <Disorder id="8560">
         <OrphaCode>517</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=517</ExpertLink>
-        <Name lang="it">Leucemia acuta mielomonocitica</Name>
+        <Name lang="it">Leucemia mielomonocitica acuta</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -921615,7 +921608,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=352</ExpertLink>
         <Name lang="it">Galattosemia</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
@@ -929966,7 +929959,7 @@
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99</ExpertLink>
         <Name lang="it">Atassia cerebellare, autosomica dominante</Name>
         <DisorderType id="36561">
-          <Name lang="it">Category</Name>
+          <Name lang="it">Categoria</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="it">Gruppo di malattie</Name>
