commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases with associated phenotypes/nl_product4.xml b/Rare diseases with associated phenotypes/nl_product4.xml
index 57c3608..dad81f2 100644
--- a/Rare diseases with associated phenotypes/nl_product4.xml	
+++ b/Rare diseases with associated phenotypes/nl_product4.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:51:33" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:51:53" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -89879,7 +89879,7 @@
           <Name lang="nl">Aandoening</Name>
         </DisorderGroup>
         <HPODisorderAssociationList count="36">
-          <HPODisorderAssociation id="371723">
+          <HPODisorderAssociation id="530025">
             <HPO id="203">
               <HPOId>HP:0000093</HPOId>
               <HPOTerm>Proteinuria</HPOTerm>
@@ -89889,7 +89889,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371724">
+          <HPODisorderAssociation id="530026">
             <HPO id="838">
               <HPOId>HP:0000509</HPOId>
               <HPOTerm>Conjunctivitis</HPOTerm>
@@ -89901,7 +89901,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371725">
+          <HPODisorderAssociation id="530027">
             <HPO id="156">
               <HPOId>HP:0000988</HPOId>
               <HPOTerm>Skin rash</HPOTerm>
@@ -89913,7 +89913,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371726">
+          <HPODisorderAssociation id="530028">
             <HPO id="174">
               <HPOId>HP:0002633</HPOId>
               <HPOTerm>Vasculitis</HPOTerm>
@@ -89923,7 +89923,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371727">
+          <HPODisorderAssociation id="530029">
             <HPO id="5677">
               <HPOId>HP:0025289</HPOId>
               <HPOTerm>Cervical lymphadenopathy</HPOTerm>
@@ -89935,7 +89935,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371728">
+          <HPODisorderAssociation id="530030">
             <HPO id="7330">
               <HPOId>HP:0025493</HPOId>
               <HPOTerm>Palmoplantar erythema</HPOTerm>
@@ -89947,7 +89947,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371729">
+          <HPODisorderAssociation id="530031">
             <HPO id="834">
               <HPOId>HP:0100776</HPOId>
               <HPOTerm>Recurrent pharyngitis</HPOTerm>
@@ -89957,7 +89957,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371730">
+          <HPODisorderAssociation id="530032">
             <HPO id="399">
               <HPOId>HP:0100825</HPOId>
               <HPOTerm>Cheilitis</HPOTerm>
@@ -89969,7 +89969,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371731">
+          <HPODisorderAssociation id="530033">
             <HPO id="1562">
               <HPOId>HP:0000206</HPOId>
               <HPOTerm>Glossitis</HPOTerm>
@@ -89981,7 +89981,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371732">
+          <HPODisorderAssociation id="530034">
             <HPO id="277">
               <HPOId>HP:0000969</HPOId>
               <HPOTerm>Edema</HPOTerm>
@@ -89991,7 +89991,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371733">
+          <HPODisorderAssociation id="530035">
             <HPO id="80">
               <HPOId>HP:0001369</HPOId>
               <HPOTerm>Arthritis</HPOTerm>
@@ -90001,7 +90001,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371734">
+          <HPODisorderAssociation id="530036">
             <HPO id="291">
               <HPOId>HP:0001654</HPOId>
               <HPOTerm>Abnormal heart valve morphology</HPOTerm>
@@ -90011,7 +90011,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371735">
+          <HPODisorderAssociation id="530037">
             <HPO id="161">
               <HPOId>HP:0001701</HPOId>
               <HPOTerm>Pericarditis</HPOTerm>
@@ -90021,7 +90021,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371736">
+          <HPODisorderAssociation id="530038">
             <HPO id="166">
               <HPOId>HP:0001945</HPOId>
               <HPOTerm>Fever</HPOTerm>
@@ -90033,7 +90033,7 @@
               <Name lang="nl">Diagnostisch criterium</Name>
             </DiagnosticCriteria>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371737">
+          <HPODisorderAssociation id="530039">
             <HPO id="1816">
               <HPOId>HP:0001974</HPOId>
               <HPOTerm>Leukocytosis</HPOTerm>
@@ -90043,7 +90043,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371738">
+          <HPODisorderAssociation id="530040">
             <HPO id="413">
               <HPOId>HP:0002014</HPOId>
               <HPOTerm>Diarrhea</HPOTerm>
@@ -90053,7 +90053,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371739">
+          <HPODisorderAssociation id="530041">
             <HPO id="167">
               <HPOId>HP:0002027</HPOId>
               <HPOTerm>Abdominal pain</HPOTerm>
@@ -90063,7 +90063,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371740">
+          <HPODisorderAssociation id="530042">
             <HPO id="185">
               <HPOId>HP:0012378</HPOId>
               <HPOTerm>Fatigue</HPOTerm>
@@ -90073,7 +90073,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371741">
+          <HPODisorderAssociation id="530043">
             <HPO id="1819">
               <HPOId>HP:0100643</HPOId>
               <HPOTerm>Abnormality of nail color</HPOTerm>
@@ -90083,7 +90083,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371742">
+          <HPODisorderAssociation id="530044">
             <HPO id="7">
               <HPOId>HP:0000508</HPOId>
               <HPOTerm>Ptosis</HPOTerm>
@@ -90093,7 +90093,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371743">
+          <HPODisorderAssociation id="530045">
             <HPO id="266">
               <HPOId>HP:0000737</HPOId>
               <HPOTerm>Irritability</HPOTerm>
@@ -90103,7 +90103,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371744">
+          <HPODisorderAssociation id="530046">
             <HPO id="346">
               <HPOId>HP:0000952</HPOId>
               <HPOTerm>Jaundice</HPOTerm>
@@ -90113,7 +90113,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371745">
+          <HPODisorderAssociation id="530047">
             <HPO id="158">
               <HPOId>HP:0001082</HPOId>
               <HPOTerm>Cholecystitis</HPOTerm>
@@ -90123,7 +90123,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371746">
+          <HPODisorderAssociation id="530048">
             <HPO id="753">
               <HPOId>HP:0001287</HPOId>
               <HPOTerm>Meningitis</HPOTerm>
@@ -90133,7 +90133,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371747">
+          <HPODisorderAssociation id="530049">
             <HPO id="318">
               <HPOId>HP:0001635</HPOId>
               <HPOTerm>Congestive heart failure</HPOTerm>
@@ -90143,7 +90143,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371748">
+          <HPODisorderAssociation id="530050">
             <HPO id="35">
               <HPOId>HP:0002017</HPOId>
               <HPOTerm>Nausea and vomiting</HPOTerm>
@@ -90153,7 +90153,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371749">
+          <HPODisorderAssociation id="530051">
             <HPO id="508">
               <HPOId>HP:0002076</HPOId>
               <HPOTerm>Migraine</HPOTerm>
@@ -90163,7 +90163,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371750">
+          <HPODisorderAssociation id="530052">
             <HPO id="175">
               <HPOId>HP:0002829</HPOId>
               <HPOTerm>Arthralgia</HPOTerm>
@@ -90173,7 +90173,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371751">
+          <HPODisorderAssociation id="530053">
             <HPO id="631">
               <HPOId>HP:0005111</HPOId>
               <HPOTerm>Dilatation of the ascending aorta</HPOTerm>
@@ -90183,7 +90183,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371752">
+          <HPODisorderAssociation id="530054">
             <HPO id="767">
               <HPOId>HP:0006530</HPOId>
               <HPOTerm>Interstitial pulmonary abnormality</HPOTerm>
@@ -90193,7 +90193,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371753">
+          <HPODisorderAssociation id="530055">
             <HPO id="220">
               <HPOId>HP:0006824</HPOId>
               <HPOTerm>Cranial nerve paralysis</HPOTerm>
@@ -90203,7 +90203,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371754">
+          <HPODisorderAssociation id="530056">
             <HPO id="1817">
               <HPOId>HP:0011658</HPOId>
               <HPOTerm>Double outlet right ventricle with subpulmonary ventricular septal defect without pulmonary stenosis</HPOTerm>
@@ -90213,7 +90213,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371755">
+          <HPODisorderAssociation id="530057">
             <HPO id="183">
               <HPOId>HP:0011675</HPOId>
               <HPOTerm>Arrhythmia</HPOTerm>
@@ -90223,7 +90223,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371756">
+          <HPODisorderAssociation id="530058">
             <HPO id="184">
               <HPOId>HP:0012115</HPOId>
               <HPOTerm>Hepatitis</HPOTerm>
@@ -90233,7 +90233,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371757">
+          <HPODisorderAssociation id="530059">
             <HPO id="189">
               <HPOId>HP:0012819</HPOId>
               <HPOTerm>Myocarditis</HPOTerm>
@@ -90243,7 +90243,7 @@
             </HPOFrequency>
             <DiagnosticCriteria/>
           </HPODisorderAssociation>
-          <HPODisorderAssociation id="371758">
+          <HPODisorderAssociation id="530060">
             <HPO id="1818">
               <HPOId>HP:0100586</HPOId>
               <HPOTerm>Sterile pyuria</HPOTerm>
@@ -109312,7 +109312,7 @@
       <Disorder id="1115">
         <OrphaCode>1621</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1621</ExpertLink>
-        <Name lang="nl">3q13 microdeletiesyndroom</Name>
+        <Name lang="nl">3q13-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -109501,7 +109501,7 @@
       <Disorder id="1132">
         <OrphaCode>1643</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1643</ExpertLink>
-        <Name lang="nl">Xp22.3 microdeletiesyndroom</Name>
+        <Name lang="nl">Xp22.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -123029,7 +123029,7 @@
       <Disorder id="1160">
         <OrphaCode>1713</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1713</ExpertLink>
-        <Name lang="nl">17p11.2 microduplicatiesyndroom</Name>
+        <Name lang="nl">17p11.2-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -126630,7 +126630,7 @@
       <Disorder id="1313">
         <OrphaCode>1001</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1001</ExpertLink>
-        <Name lang="nl">2q37 microdeletiesyndroom</Name>
+        <Name lang="nl">2q37-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -166251,7 +166251,7 @@
       <Disorder id="1659">
         <OrphaCode>1484</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1484</ExpertLink>
-        <Name lang="nl">Contracturen - ectodermale dysplasie - gespleten lip/verhemelte</Name>
+        <Name lang="nl">Contracturen - ectodermale dysplasie - gespleten lip/gehemelte-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -167583,7 +167583,7 @@
       <Disorder id="1638">
         <OrphaCode>1435</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1435</ExpertLink>
-        <Name lang="nl">Xq21 microdeletiesyndroom</Name>
+        <Name lang="nl">Xq21-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -177710,7 +177710,7 @@
       <Disorder id="1732">
         <OrphaCode>1574</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1574</ExpertLink>
-        <Name lang="nl">Retinadegeneratie - nanoftalmos - glaucoom-syndroom</Name>
+        <Name lang="nl">Retinadegeneratie - nanoftalmie - glaucoom-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -179738,7 +179738,7 @@
       <Disorder id="1738">
         <OrphaCode>1606</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1606</ExpertLink>
-        <Name lang="nl">1p36 deletiesyndroom</Name>
+        <Name lang="nl">1p36-deletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -181861,7 +181861,7 @@
       <Disorder id="1945">
         <OrphaCode>2008</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2008</ExpertLink>
-        <Name lang="nl">Acro-cardio-faciaal syndroom</Name>
+        <Name lang="nl">Acrocardiofaciaal syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -182339,7 +182339,7 @@
       <Disorder id="1949">
         <OrphaCode>2013</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2013</ExpertLink>
-        <Name lang="nl">Gespleten verhemelte - grote oren - klein hoofd</Name>
+        <Name lang="nl">Gespleten gehemelte - grote oren - klein hoofd-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -187381,7 +187381,7 @@
       <Disorder id="1880">
         <OrphaCode>1917</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1917</ExpertLink>
-        <Name lang="nl">Foetaal methylkwiksyndroom</Name>
+        <Name lang="nl">Foetaal syndroom door methylkwik</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -206189,7 +206189,7 @@
       <Disorder id="2249">
         <OrphaCode>2429</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2429</ExpertLink>
-        <Name lang="nl">Macrocefalie - spastische paraplegie - dysmorfie</Name>
+        <Name lang="nl">Macrocefalie - spastische paraplegie - dysmorfie-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -215689,7 +215689,7 @@
       <Disorder id="2480">
         <OrphaCode>2719</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2719</ExpertLink>
-        <Name lang="nl">Oculocerebrale hypopigmentatiesyndroom, Cross-type</Name>
+        <Name lang="nl">Oculocerebrale hypopigmentatie-syndroom, Cross-type</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -218716,7 +218716,7 @@
       <Disorder id="2524">
         <OrphaCode>2774</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2774</ExpertLink>
-        <Name lang="nl">Multicentrische carpo-tarsale osteolyse met of zonder nefropathie</Name>
+        <Name lang="nl">Multicentrische carpotarsale osteolyse met of zonder nefropathie</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -242566,7 +242566,7 @@
       <Disorder id="3101">
         <OrphaCode>2151</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2151</ExpertLink>
-        <Name lang="nl">Ziekte van Hirschsprung - ganglioneuroblastoom</Name>
+        <Name lang="nl">Ziekte van Hirschsprung - ganglioneuroblastoom-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -247438,7 +247438,7 @@
       <Disorder id="2732">
         <OrphaCode>3038</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3038</ExpertLink>
-        <Name lang="nl">Uitgestelde spraak - faciale asymmetrie - strabisme - oorlelplooien-syndroom</Name>
+        <Name lang="nl">Spraakachterstand - faciale asymmetrie - strabisme - oorlelplooien-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -254535,7 +254535,7 @@
       <Disorder id="2892">
         <OrphaCode>3232</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3232</ExpertLink>
-        <Name lang="nl">Doofheid - oormisvorming - facialisparalyse-syndroom</Name>
+        <Name lang="nl">Doofheid - malformatie van oor - aangezichtsverlamming-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -265870,7 +265870,7 @@
       <Disorder id="8568">
         <OrphaCode>345</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=345</ExpertLink>
-        <Name lang="nl">Dissecting cellulitis van de hoofdhuid</Name>
+        <Name lang="nl">Dissecerende cellulitis van hoofdhuid</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -268966,7 +268966,7 @@
       <Disorder id="3381">
         <OrphaCode>851</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=851</ExpertLink>
-        <Name lang="nl">Paris-Trousseautrombocytopenie</Name>
+        <Name lang="nl">Trombocytopenie, Paris-Trousseau-type</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -331052,7 +331052,7 @@
       <Disorder id="12547">
         <OrphaCode>94066</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94066</ExpertLink>
-        <Name lang="nl">Ernstige intellectuele achterstand - epilepsie - anale anomalieën - distale falangeale hypoplasie</Name>
+        <Name lang="nl">Ernstige intellectuele achterstand - epilepsie - anale anomalieën - distale falangeale hypoplasie-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -332016,7 +332016,7 @@
       <Disorder id="12488">
         <OrphaCode>93928</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93928</ExpertLink>
-        <Name lang="nl">Geïsoleerde epispadieën</Name>
+        <Name lang="nl">Geïsoleerde epispadie</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -351016,7 +351016,7 @@
       <Disorder id="16905">
         <OrphaCode>139474</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139474</ExpertLink>
-        <Name lang="nl">17q11.2 microduplicatiesyndroom</Name>
+        <Name lang="nl">17q11.2-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -351195,7 +351195,7 @@
       <Disorder id="16904">
         <OrphaCode>139471</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139471</ExpertLink>
-        <Name lang="nl">Microftalmie met hersen- en vingeranomalieën</Name>
+        <Name lang="nl">Microftalmie met anomalieën van hersenen en digitus</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -355449,7 +355449,7 @@
       <Disorder id="14540">
         <OrphaCode>99967</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99967</ExpertLink>
-        <Name lang="nl">Myxoïd/ronde cel liposarcoom</Name>
+        <Name lang="nl">Myxoïd/rondcellig liposarcoom</Name>
         <DisorderType id="21457">
           <Name lang="nl">Histopathologisch subtype</Name>
         </DisorderType>
@@ -365893,7 +365893,7 @@
       <Disorder id="19870">
         <OrphaCode>261295</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261295</ExpertLink>
-        <Name lang="nl">20p12.3 microdeletiesyndroom</Name>
+        <Name lang="nl">20p12.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -366142,7 +366142,7 @@
       <Disorder id="19871">
         <OrphaCode>261304</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261304</ExpertLink>
-        <Name lang="nl">Paternaal 20q13.2q13.3-microdeletiesyndroom</Name>
+        <Name lang="nl">Paternale 20q13.2q13.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -367130,7 +367130,7 @@
       <Disorder id="19875">
         <OrphaCode>261330</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261330</ExpertLink>
-        <Name lang="nl">Distale 22q11.2 microdeletiesyndroom</Name>
+        <Name lang="nl">Distale 22q11.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -367749,7 +367749,7 @@
       <Disorder id="19876">
         <OrphaCode>261337</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261337</ExpertLink>
-        <Name lang="nl">Distale 22q11.2 microduplicatiesyndroom</Name>
+        <Name lang="nl">Distale 22q11.2-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -368777,7 +368777,7 @@
       <Disorder id="19878">
         <OrphaCode>261349</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261349</ExpertLink>
-        <Name lang="nl">2p15p16.1 microdeletiesyndroom</Name>
+        <Name lang="nl">2p15p16.1-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -369785,7 +369785,7 @@
       <Disorder id="19859">
         <OrphaCode>261211</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261211</ExpertLink>
-        <Name lang="nl">16p11.2p12.2 microdeletiesyndroom</Name>
+        <Name lang="nl">16p11.2p12.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -370234,7 +370234,7 @@
       <Disorder id="19862">
         <OrphaCode>261236</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261236</ExpertLink>
-        <Name lang="nl">16p13.11 microdeletiesyndroom</Name>
+        <Name lang="nl">16p13.11-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -370653,7 +370653,7 @@
       <Disorder id="19863">
         <OrphaCode>261243</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261243</ExpertLink>
-        <Name lang="nl">16p13.11 microduplicatiesyndroom</Name>
+        <Name lang="nl">16p13.11-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -370862,7 +370862,7 @@
       <Disorder id="19864">
         <OrphaCode>261250</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261250</ExpertLink>
-        <Name lang="nl">16q24.3 microdeletiesyndroom</Name>
+        <Name lang="nl">16q24.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -371351,7 +371351,7 @@
       <Disorder id="19866">
         <OrphaCode>261265</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261265</ExpertLink>
-        <Name lang="nl">17q12 microdeletiesyndroom</Name>
+        <Name lang="nl">17q12-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -371580,7 +371580,7 @@
       <Disorder id="19867">
         <OrphaCode>261272</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261272</ExpertLink>
-        <Name lang="nl">17q12 microduplicatiesyndroom</Name>
+        <Name lang="nl">17q12-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -371769,7 +371769,7 @@
       <Disorder id="19868">
         <OrphaCode>261279</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261279</ExpertLink>
-        <Name lang="nl">17q23.1q23.2 microdeletiesyndroom</Name>
+        <Name lang="nl">17q23.1q23.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -372767,7 +372767,7 @@
       <Disorder id="19849">
         <OrphaCode>261102</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261102</ExpertLink>
-        <Name lang="nl">Distale 7q11.23 microduplicatiesyndroom</Name>
+        <Name lang="nl">Distale 7q11.23-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -373075,7 +373075,7 @@
       <Disorder id="19851">
         <OrphaCode>261120</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261120</ExpertLink>
-        <Name lang="nl">14q11.2 microdeletiesyndroom</Name>
+        <Name lang="nl">14q11.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -373943,7 +373943,7 @@
       <Disorder id="19856">
         <OrphaCode>261190</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261190</ExpertLink>
-        <Name lang="nl">15q14 microdeletiesyndroom</Name>
+        <Name lang="nl">15q14-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -374551,7 +374551,7 @@
       <Disorder id="19858">
         <OrphaCode>261204</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261204</ExpertLink>
-        <Name lang="nl">16p11.2p12.2 microduplicatiesyndroom</Name>
+        <Name lang="nl">16p11.2p12.2-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -374710,7 +374710,7 @@
       <Disorder id="19613">
         <OrphaCode>251066</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251066</ExpertLink>
-        <Name lang="nl">8p11.2 deletiesyndroom</Name>
+        <Name lang="nl">8p11.2-deletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -375089,7 +375089,7 @@
       <Disorder id="19611">
         <OrphaCode>251056</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251056</ExpertLink>
-        <Name lang="nl">6q25 microdeletiesyndroom</Name>
+        <Name lang="nl">6q25-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -375428,7 +375428,7 @@
       <Disorder id="19615">
         <OrphaCode>251076</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251076</ExpertLink>
-        <Name lang="nl">8p23.1 duplicatiesyndroom</Name>
+        <Name lang="nl">8p23.1-duplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -375627,7 +375627,7 @@
       <Disorder id="19614">
         <OrphaCode>251071</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251071</ExpertLink>
-        <Name lang="nl">8p23.1 microdeletiesyndroom</Name>
+        <Name lang="nl">8p23.1-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -376136,7 +376136,7 @@
       <Disorder id="19739">
         <OrphaCode>254346</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254346</ExpertLink>
-        <Name lang="nl">19q13.12 microdeletiesyndroom</Name>
+        <Name lang="nl">19q13.12-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -377303,7 +377303,7 @@
       <Disorder id="19740">
         <OrphaCode>254351</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254351</ExpertLink>
-        <Name lang="nl">Distale 7q11.23 microdeletiesyndroom</Name>
+        <Name lang="nl">Distale 7q11.23-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -380156,7 +380156,7 @@
       <Disorder id="20581">
         <OrphaCode>284160</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284160</ExpertLink>
-        <Name lang="nl">8q21.11 microdeletiesyndroom</Name>
+        <Name lang="nl">8q21.11-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -380604,7 +380604,7 @@
       <Disorder id="20375">
         <OrphaCode>276413</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=276413</ExpertLink>
-        <Name lang="nl">10q22.3q23.3 microdeletiesyndroom</Name>
+        <Name lang="nl">10q22.3q23.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -381401,7 +381401,7 @@
       <Disorder id="20095">
         <OrphaCode>264200</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=264200</ExpertLink>
-        <Name lang="nl">14q22q23 microdeletiesyndroom</Name>
+        <Name lang="nl">14q22q23-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -382187,7 +382187,7 @@
       <Disorder id="20376">
         <OrphaCode>276422</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=276422</ExpertLink>
-        <Name lang="nl">10q22.3q23.3 microduplicatiesyndroom</Name>
+        <Name lang="nl">10q22.3q23.3-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -391310,7 +391310,7 @@
       <Disorder id="20583">
         <OrphaCode>284180</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284180</ExpertLink>
-        <Name lang="nl">Xp22.13p22.2 duplicatiesyndroom</Name>
+        <Name lang="nl">Xp22.13p22.2-duplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -391838,7 +391838,7 @@
       <Disorder id="11095">
         <OrphaCode>75857</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=75857</ExpertLink>
-        <Name lang="nl">6q terminale deletiesyndroom</Name>
+        <Name lang="nl">Terminale 6q-deletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -392347,7 +392347,7 @@
       <Disorder id="19879">
         <OrphaCode>261476</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261476</ExpertLink>
-        <Name lang="nl">Xp21 deletiesyndroom</Name>
+        <Name lang="nl">Xp21-deletiesyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -393605,7 +393605,7 @@
       <Disorder id="12716">
         <OrphaCode>96092</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96092</ExpertLink>
-        <Name lang="nl">8p inversie duplicatie/deletie-syndroom</Name>
+        <Name lang="nl">8p geïnverteerde duplicatie/deletie-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -394862,7 +394862,7 @@
       <Disorder id="19600">
         <OrphaCode>250999</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=250999</ExpertLink>
-        <Name lang="nl">1q41q42 microdeletiesyndroom</Name>
+        <Name lang="nl">1q41q42-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -395769,7 +395769,7 @@
       <Disorder id="21479">
         <OrphaCode>314389</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314389</ExpertLink>
-        <Name lang="nl">Xq12-q13.3 duplicatiesyndroom</Name>
+        <Name lang="nl">Xq12-q13.3-duplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -405870,7 +405870,7 @@
       <Disorder id="18032">
         <OrphaCode>178487</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=178487</ExpertLink>
-        <Name lang="nl">Intestinaal botulisme bij volwassenen</Name>
+        <Name lang="nl">Intestinaal botulisme bij volwassene</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -406236,7 +406236,7 @@
       <Disorder id="18399">
         <OrphaCode>199318</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=199318</ExpertLink>
-        <Name lang="nl">15q13.3 microdeletiesyndroom</Name>
+        <Name lang="nl">15q13.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -407013,7 +407013,7 @@
       <Disorder id="18848">
         <OrphaCode>217346</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217346</ExpertLink>
-        <Name lang="nl">19q13.11 microdeletiesyndroom</Name>
+        <Name lang="nl">19q13.11-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -407442,7 +407442,7 @@
       <Disorder id="18847">
         <OrphaCode>217340</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217340</ExpertLink>
-        <Name lang="nl">17q21.31 microduplicatiesyndroom</Name>
+        <Name lang="nl">17q21.31-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -408879,7 +408879,7 @@
       <Disorder id="17936">
         <OrphaCode>171829</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=171829</ExpertLink>
-        <Name lang="nl">6q16 microdeletiesyndroom</Name>
+        <Name lang="nl">6q16-microdeletiesyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -410662,7 +410662,7 @@
       <Disorder id="18030">
         <OrphaCode>178478</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=178478</ExpertLink>
-        <Name lang="nl">Botulisme bij kinderen</Name>
+        <Name lang="nl">Botulisme bij zuigeling</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -411239,7 +411239,7 @@
       <Disorder id="19119">
         <OrphaCode>228384</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228384</ExpertLink>
-        <Name lang="nl">5q14.3 microdeletiesyndroom</Name>
+        <Name lang="nl">5q14.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -411995,7 +411995,7 @@
       <Disorder id="19128">
         <OrphaCode>228415</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228415</ExpertLink>
-        <Name lang="nl">5q35 microduplicatiesyndroom</Name>
+        <Name lang="nl">5q35-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -412791,7 +412791,7 @@
       <Disorder id="19123">
         <OrphaCode>228396</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228396</ExpertLink>
-        <Name lang="nl">Ptosis - bovenste oculaire bewegingsbeperking - agenesie van punctum lacrimale-syndroom</Name>
+        <Name lang="nl">Ptosis - beperkte opwaartse oogbeweging - agenesie van punctum lacrimale-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -413309,7 +413309,7 @@
       <Disorder id="19125">
         <OrphaCode>228402</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228402</ExpertLink>
-        <Name lang="nl">2q23.1 microdeletiesyndroom</Name>
+        <Name lang="nl">2q23.1-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -413648,7 +413648,7 @@
       <Disorder id="19124">
         <OrphaCode>228399</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228399</ExpertLink>
-        <Name lang="nl">8q12 microduplicatiesyndroom</Name>
+        <Name lang="nl">8q12-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -414873,7 +414873,7 @@
       <Disorder id="18852">
         <OrphaCode>217385</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217385</ExpertLink>
-        <Name lang="nl">17p13.3 microduplicatiesyndroom</Name>
+        <Name lang="nl">17p13.3-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -416367,7 +416367,7 @@
       <Disorder id="19305">
         <OrphaCode>238769</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238769</ExpertLink>
-        <Name lang="nl">1q44 microdeletiesyndroom</Name>
+        <Name lang="nl">1q44-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -416716,7 +416716,7 @@
       <Disorder id="19300">
         <OrphaCode>238750</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238750</ExpertLink>
-        <Name lang="nl">4q21 microdeletiesyndroom</Name>
+        <Name lang="nl">4q21-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -417632,7 +417632,7 @@
       <Disorder id="19263">
         <OrphaCode>238446</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238446</ExpertLink>
-        <Name lang="nl">15q11q13 microduplicatiesyndroom</Name>
+        <Name lang="nl">15q11q13-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -417930,7 +417930,7 @@
       <Disorder id="19609">
         <OrphaCode>251046</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251046</ExpertLink>
-        <Name lang="nl">6p22 microdeletiesyndroom</Name>
+        <Name lang="nl">6p22-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -418159,7 +418159,7 @@
       <Disorder id="19607">
         <OrphaCode>251038</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251038</ExpertLink>
-        <Name lang="nl">3q29 microduplicatiesyndroom</Name>
+        <Name lang="nl">3q29-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -418478,7 +418478,7 @@
       <Disorder id="19604">
         <OrphaCode>251019</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251019</ExpertLink>
-        <Name lang="nl">2q32q33 microdeletiesyndroom</Name>
+        <Name lang="nl">2q32q33-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -418927,7 +418927,7 @@
       <Disorder id="19603">
         <OrphaCode>251014</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251014</ExpertLink>
-        <Name lang="nl">2q31.1 microdeletiesyndroom</Name>
+        <Name lang="nl">2q31.1-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -419576,7 +419576,7 @@
       <Disorder id="19599">
         <OrphaCode>250994</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=250994</ExpertLink>
-        <Name lang="nl">1q21.1 microduplicatiesyndroom</Name>
+        <Name lang="nl">1q21.1-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -419845,7 +419845,7 @@
       <Disorder id="19598">
         <OrphaCode>250989</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=250989</ExpertLink>
-        <Name lang="nl">1q21.1 microdeletiesyndroom</Name>
+        <Name lang="nl">1q21.1-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -421320,7 +421320,7 @@
       <Disorder id="19171">
         <OrphaCode>231214</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231214</ExpertLink>
-        <Name lang="nl">Bèta-thalassemie major</Name>
+        <Name lang="nl">Beta-thalassemia major</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -429860,7 +429860,7 @@
       <Disorder id="23325">
         <OrphaCode>436003</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=436003</ExpertLink>
-        <Name lang="nl">Contracturen - ontwikkelingsachterstand - syndroom van Pierre Robin</Name>
+        <Name lang="nl">Contracturen - ontwikkelingsachterstand - Pierre Robin-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -430807,7 +430807,7 @@
       <Disorder id="23476">
         <OrphaCode>444051</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=444051</ExpertLink>
-        <Name lang="nl">20q11.2 microdeletiesyndroom</Name>
+        <Name lang="nl">20q11.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -431185,7 +431185,7 @@
       <Disorder id="23473">
         <OrphaCode>444002</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=444002</ExpertLink>
-        <Name lang="nl">11q22.2q22.3 microdeletiesyndroom</Name>
+        <Name lang="nl">11q22.2q22.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -434674,7 +434674,7 @@
       <Disorder id="23308">
         <OrphaCode>435638</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=435638</ExpertLink>
-        <Name lang="nl">3p25.3 microdeletiesyndroom</Name>
+        <Name lang="nl">3p25.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -439872,7 +439872,7 @@
       <Disorder id="21472">
         <OrphaCode>314034</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314034</ExpertLink>
-        <Name lang="nl">7p22.1 microduplicatiesyndroom</Name>
+        <Name lang="nl">7p22.1-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -448319,7 +448319,7 @@
       <Disorder id="19860">
         <OrphaCode>261222</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261222</ExpertLink>
-        <Name lang="nl">Distale 16p11.2 microdeletiesyndroom</Name>
+        <Name lang="nl">Distale 16p11.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -456703,7 +456703,7 @@
       <Disorder id="20893">
         <OrphaCode>293948</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=293948</ExpertLink>
-        <Name lang="nl">1p21.3 microdeletiesyndroom</Name>
+        <Name lang="nl">1p21.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -457281,7 +457281,7 @@
       <Disorder id="1739">
         <OrphaCode>1617</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1617</ExpertLink>
-        <Name lang="nl">2q24 microdeletiesyndroom</Name>
+        <Name lang="nl">2q24-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -478404,7 +478404,7 @@
       <Disorder id="3335">
         <OrphaCode>79</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79</ExpertLink>
-        <Name lang="nl">Congenitale alfa2-antiplasminedeficiëntie</Name>
+        <Name lang="nl">Congenitale deficiëntie van alfa2-antiplasmine</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -523276,7 +523276,7 @@
       <Disorder id="12745">
         <OrphaCode>96121</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96121</ExpertLink>
-        <Name lang="nl">7q11.23 microduplicatiesyndroom</Name>
+        <Name lang="nl">7q11.23-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -526078,7 +526078,7 @@
       <Disorder id="19872">
         <OrphaCode>261311</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261311</ExpertLink>
-        <Name lang="nl">20q13.33 microdeletiesyndroom</Name>
+        <Name lang="nl">20q13.33-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -547562,7 +547562,7 @@
       <Disorder id="2561">
         <OrphaCode>2820</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2820</ExpertLink>
-        <Name lang="nl">Spastische paraplegie - nefritis - doofheid</Name>
+        <Name lang="nl">Spastische paraplegie - nefritis - doofheid-syndroom</Name>
         <DisorderType id="21422">
           <Name lang="nl">Klinisch syndroom</Name>
         </DisorderType>
@@ -548708,7 +548708,7 @@
       <Disorder id="2578">
         <OrphaCode>2839</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2839</ExpertLink>
-        <Name lang="nl">Bekken-schouderdysplasie</Name>
+        <Name lang="nl">Dysplasie van bekken-schouder</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -556071,7 +556071,7 @@
       <Disorder id="2227">
         <OrphaCode>2400</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2400</ExpertLink>
-        <Name lang="nl">Perifere motorische neuropathie - dysautonomie</Name>
+        <Name lang="nl">Perifere motorische neuropathie - dysautonomie-syndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -556796,7 +556796,7 @@
       <Disorder id="2986">
         <OrphaCode>3369</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3369</ExpertLink>
-        <Name lang="nl">Trigonocefalie - kleine gestalte - ontwikkelingsachterstand</Name>
+        <Name lang="nl">Trigonocefalie - kleine gestalte - ontwikkelingsachterstand-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -569922,7 +569922,7 @@
       <Disorder id="2050">
         <OrphaCode>2150</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2150</ExpertLink>
-        <Name lang="nl">Ziekte van Hirschsprung - type D-brachydactylie-syndroom</Name>
+        <Name lang="nl">Ziekte van Hirschsprung - brachydactylie type D-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -608397,7 +608397,7 @@
       <Disorder id="3007">
         <OrphaCode>3412</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3412</ExpertLink>
-        <Name lang="nl">VACTERL met hydrocefalus</Name>
+        <Name lang="nl">VACTERL met hydrocefalie</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -674065,7 +674065,7 @@
       <Disorder id="27631">
         <OrphaCode>529962</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=529962</ExpertLink>
-        <Name lang="nl">17q24.2 microdeletiesyndroom</Name>
+        <Name lang="nl">17q24.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -696274,7 +696274,7 @@
       <Disorder id="26569">
         <OrphaCode>508488</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=508488</ExpertLink>
-        <Name lang="nl">8q24.3 microdeletiesyndroom</Name>
+        <Name lang="nl">8q24.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -710781,7 +710781,7 @@
       <Disorder id="2734">
         <OrphaCode>3041</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3041</ExpertLink>
-        <Name lang="nl">Intellectuele achterstand - kaalheid - patellaluxatie - acromicrie-syndroom</Name>
+        <Name lang="nl">Intellectuele achterstand - kaalheid - luxatie van patella - acromicrie-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -719168,7 +719168,7 @@
       <Disorder id="19893">
         <OrphaCode>261584</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261584</ExpertLink>
-        <Name lang="nl">Familiale adenomateuze polyposis door 5q22.2 microdeletie</Name>
+        <Name lang="nl">Familiale adenomateuze polyposis door 5q22.2-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -725590,7 +725590,7 @@
       <Disorder id="27690">
         <OrphaCode>531151</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=531151</ExpertLink>
-        <Name lang="nl">9q21.13 microdeletiesyndroom</Name>
+        <Name lang="nl">9q21.13-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -731657,7 +731657,7 @@
       <Disorder id="11138">
         <OrphaCode>79096</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79096</ExpertLink>
-        <Name lang="nl">Pyridoxaalfosfaatafhankelijke epilepsie</Name>
+        <Name lang="nl">Pyridoxaalfosfaatresponsieve insulten</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -736368,7 +736368,7 @@
       <Disorder id="16711">
         <OrphaCode>137754</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=137754</ExpertLink>
-        <Name lang="nl">Neurologische aandoeningen geassocieerd met aminoacylase 1-deficiëntie</Name>
+        <Name lang="nl">Neurologische aandoeningen geassocieerd met deficiëntie van aminoacylase 1</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -738051,7 +738051,7 @@
       <Disorder id="24025">
         <OrphaCode>464440</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=464440</ExpertLink>
-        <Name lang="nl">Primaire dystonie, DYT27 type</Name>
+        <Name lang="nl">Primaire dystonie, DYT27-type</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -738150,7 +738150,7 @@
       <Disorder id="23109">
         <OrphaCode>420492</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=420492</ExpertLink>
-        <Name lang="nl">Cervicale dystonie met aanvang op volwassen leeftijd, type DYT23</Name>
+        <Name lang="nl">Cervicale dystonie met aanvang op volwassen leeftijd, DYT23-type</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -781450,7 +781450,7 @@
       <Disorder id="22477">
         <OrphaCode>370079</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=370079</ExpertLink>
-        <Name lang="nl">Proximale 16p11.2 microduplicatiesyndroom</Name>
+        <Name lang="nl">Proximale 16p11.2-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -786066,7 +786066,7 @@
       <Disorder id="27320">
         <OrphaCode>521258</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=521258</ExpertLink>
-        <Name lang="nl">Xq25 microduplicatiesyndroom</Name>
+        <Name lang="nl">Xq25-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -803051,7 +803051,7 @@
       <Disorder id="21764">
         <OrphaCode>324313</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=324313</ExpertLink>
-        <Name lang="nl">9p13 microdeletiesyndroom</Name>
+        <Name lang="nl">9p13-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -803390,7 +803390,7 @@
       <Disorder id="22693">
         <OrphaCode>397695</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=397695</ExpertLink>
-        <Name lang="nl">3q27.3 microdeletiesyndroom</Name>
+        <Name lang="nl">3q27.3-microdeletiesyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -829103,7 +829103,7 @@
       <Disorder id="17996">
         <OrphaCode>177926</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=177926</ExpertLink>
-        <Name lang="nl">Symptomatische vorm van hemofilie A bij vrouwelijke dragers</Name>
+        <Name lang="nl">Stollingsstoornis bij dragers van hemofilie A</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -831445,7 +831445,7 @@
       <Disorder id="25993">
         <OrphaCode>500055</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=500055</ExpertLink>
-        <Name lang="nl">16p13.2 microdeletiesyndroom</Name>
+        <Name lang="nl">16p13.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -854509,7 +854509,7 @@
       <Disorder id="23773">
         <OrphaCode>459074</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=459074</ExpertLink>
-        <Name lang="nl">Agenesie van het corpus callosum - macrocefalie - hypertelorisme-syndroom</Name>
+        <Name lang="nl">Agenesie van corpus callosum - macrocefalie - hypertelorisme-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -860157,7 +860157,7 @@
       <Disorder id="25800">
         <OrphaCode>495818</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=495818</ExpertLink>
-        <Name lang="nl">9q33.3q34.11 microdeletiesyndroom</Name>
+        <Name lang="nl">9q33.3q34.11-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -862462,7 +862462,7 @@
       <Disorder id="23039">
         <OrphaCode>412035</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=412035</ExpertLink>
-        <Name lang="nl">13q12.3 microdeletiesyndroom</Name>
+        <Name lang="nl">13q12.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -866094,7 +866094,7 @@
       <Disorder id="22854">
         <OrphaCode>401935</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=401935</ExpertLink>
-        <Name lang="nl">14q24.1q24.3 microdeletiesyndroom</Name>
+        <Name lang="nl">14q24.1q24.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -866353,7 +866353,7 @@
       <Disorder id="22852">
         <OrphaCode>401923</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=401923</ExpertLink>
-        <Name lang="nl">9q31.1q31.3 microdeletiesyndroom</Name>
+        <Name lang="nl">9q31.1q31.3-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -872834,7 +872834,7 @@
       <Disorder id="23558">
         <OrphaCode>447980</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=447980</ExpertLink>
-        <Name lang="nl">19p13.3 microduplicatiesyndroom</Name>
+        <Name lang="nl">19p13.3-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -876717,7 +876717,7 @@
       <Disorder id="22333">
         <OrphaCode>363659</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363659</ExpertLink>
-        <Name lang="nl">20q11.2 microduplicatiesyndroom</Name>
+        <Name lang="nl">20q11.2-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -897734,7 +897734,7 @@
       <Disorder id="3535">
         <OrphaCode>1546</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1546</ExpertLink>
-        <Name lang="nl">Cryptococcosis</Name>
+        <Name lang="nl">Cryptokokkose</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -899310,7 +899310,7 @@
       <Disorder id="22194">
         <OrphaCode>357001</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=357001</ExpertLink>
-        <Name lang="nl">19p13.13 microdeletiesyndroom</Name>
+        <Name lang="nl">19p13.13-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -901732,7 +901732,7 @@
       <Disorder id="21107">
         <OrphaCode>300305</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300305</ExpertLink>
-        <Name lang="nl">11p15.4 microduplicatiesyndroom</Name>
+        <Name lang="nl">11p15.4-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -902250,7 +902250,7 @@
       <Disorder id="21450">
         <OrphaCode>313781</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313781</ExpertLink>
-        <Name lang="nl">20p13 microdeletiesyndroom</Name>
+        <Name lang="nl">20p13-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -902619,7 +902619,7 @@
       <Disorder id="20609">
         <OrphaCode>284417</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284417</ExpertLink>
-        <Name lang="nl">Fosfoserine-aminotransferasedeficiëntie, infantiele/juveniele vorm</Name>
+        <Name lang="nl">Deficiëntie van fosfoserine-aminotransferase, infantiele/juveniele vorm</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -904044,7 +904044,7 @@
       <Disorder id="19874">
         <OrphaCode>261323</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261323</ExpertLink>
-        <Name lang="nl">21q22.11q22.12 microdeletiesyndroom</Name>
+        <Name lang="nl">21q22.11q22.12-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -908553,7 +908553,7 @@
       <Disorder id="21466">
         <OrphaCode>313947</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313947</ExpertLink>
-        <Name lang="nl">2q23.1 microduplicatiesyndroom</Name>
+        <Name lang="nl">2q23.1-microduplicatiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -908942,7 +908942,7 @@
       <Disorder id="21619">
         <OrphaCode>319171</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319171</ExpertLink>
-        <Name lang="nl">Distale 17p13.1 microdeletiesyndroom</Name>
+        <Name lang="nl">Distale 17p13.1-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -981123,7 +981123,7 @@
       <Disorder id="19857">
         <OrphaCode>261197</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261197</ExpertLink>
-        <Name lang="nl">Proximale 16p11.2 microdeletiesyndroom</Name>
+        <Name lang="nl">Proximale 16p11.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -993291,7 +993291,7 @@
       <Disorder id="19855">
         <OrphaCode>261183</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261183</ExpertLink>
-        <Name lang="nl">15q11.2 microdeletiesyndroom</Name>
+        <Name lang="nl">15q11.2-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -997769,7 +997769,7 @@
       <Disorder id="28732">
         <OrphaCode>572333</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=572333</ExpertLink>
-        <Name lang="nl">Blefarofimose - ptose - epicanthus inversus-syndroom plus</Name>
+        <Name lang="nl">Blefarofimose - ptosis - epicanthus inversus-syndroom plus</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -998956,7 +998956,7 @@
       <Disorder id="730">
         <OrphaCode>322</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=322</ExpertLink>
-        <Name lang="nl">Exstrofie - epispadieën-complex</Name>
+        <Name lang="nl">Exstrofie - epispadie-complex</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -1000921,7 +1000921,7 @@
       <Disorder id="1423">
         <OrphaCode>1159</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1159</ExpertLink>
-        <Name lang="nl">Progressieve pseudoreumatoïde artropathie in de kinderjaren</Name>
+        <Name lang="nl">Progressieve pseudoreumatoïde artropathie op kinderleeftijd</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -1003284,7 +1003284,7 @@
       <Disorder id="12100">
         <OrphaCode>90797</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90797</ExpertLink>
-        <Name lang="nl">Partieel androgeenongevoeligheidssyndroom</Name>
+        <Name lang="nl">Partieel androgeeninsensitiviteitssyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -1003563,7 +1003563,7 @@
       <Disorder id="14212">
         <OrphaCode>99429</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99429</ExpertLink>
-        <Name lang="nl">Compleet androgeenongevoeligheidssyndroom</Name>
+        <Name lang="nl">Compleet androgeeninsensitiviteitssyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -1015172,7 +1015172,7 @@
       <Disorder id="14678">
         <OrphaCode>100924</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100924</ExpertLink>
-        <Name lang="nl">Porfyrie door ALA-dehydratasedeficiëntie</Name>
+        <Name lang="nl">Porfyrie door deficiëntie van ALA-dehydratase</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -1042887,7 +1042887,7 @@
       <Disorder id="12546">
         <OrphaCode>94065</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94065</ExpertLink>
-        <Name lang="nl">15q24 microdeletiesyndroom</Name>
+        <Name lang="nl">15q24-microdeletiesyndroom</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1052495,7 +1052495,7 @@
       <Disorder id="22128">
         <OrphaCode>353281</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=353281</ExpertLink>
-        <Name lang="nl">Rubinstein-Taybisyndroom door 16p13.3-microdeletie</Name>
+        <Name lang="nl">Syndroom van Rubinstein-Taybi door 16p13.3-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1084900,7 +1084900,7 @@
       <Disorder id="20582">
         <OrphaCode>284169</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284169</ExpertLink>
-        <Name lang="nl">Faciale dysmorfie - ontwikkelingsachterstand - gedragsstoornissen-syndroom door microdeletie van 10p11.21p12.31</Name>
+        <Name lang="nl">Faciale dysmorfie - ontwikkelingsachterstand - gedragsstoornissen-syndroom door 10p11.21p12.31-microdeletie</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -1089668,7 +1089668,7 @@
       <Disorder id="20173">
         <OrphaCode>268261</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=268261</ExpertLink>
-        <Name lang="nl">DYRK1A-gerelateerd intellectuele achterstandsyndroom door microdeletie van 21q22.13q22.2</Name>
+        <Name lang="nl">DYRK1A-gerelateerd intellectuele achterstand-syndroom door 21q22.13q22.2-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1107562,7 +1107562,7 @@
       <Disorder id="22350">
         <OrphaCode>363958</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363958</ExpertLink>
-        <Name lang="nl">17q21.31 microdeletiesyndroom</Name>
+        <Name lang="nl">17q21.31-microdeletiesyndroom</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1114162,7 +1114162,7 @@
       <Disorder id="19758">
         <OrphaCode>254525</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254525</ExpertLink>
-        <Name lang="nl">Syndroom van Temple door paternale 14q32.2 microdeletie</Name>
+        <Name lang="nl">Syndroom van Temple door paternale 14q32.2-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1119703,7 +1119703,7 @@
       <Disorder id="22096">
         <OrphaCode>352665</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352665</ExpertLink>
-        <Name lang="nl">Neurologische ontwikkelingsstoornis - craniofaciale dysmorfie - hartdefect - skeletanomalieën-syndroom door 9q21.3 microdeletiesyndroom</Name>
+        <Name lang="nl">Neurologische ontwikkelingsstoornis - craniofaciale dysmorfie - hartdefect - skeletanomalieën-syndroom door 9q21.3-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1121021,7 +1121021,7 @@
       <Disorder id="21510">
         <OrphaCode>314655</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314655</ExpertLink>
-        <Name lang="nl">Ernstige neonatale hypotonie - insulten - encefalopathie-syndroom als gevolg van 5q31.3 microdeletie</Name>
+        <Name lang="nl">Ernstige neonatale hypotonie - insulten - encefalopathie-syndroom door 5q31.3-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1123954,7 +1123954,7 @@
       <Disorder id="19759">
         <OrphaCode>254528</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254528</ExpertLink>
-        <Name lang="nl">Syndroom van Kagami-Ogata door maternale 14q32.2 microdeletie</Name>
+        <Name lang="nl">Syndroom van Kagami-Ogata door maternale 14q32.2-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -1133225,7 +1133225,7 @@
       <Disorder id="12945">
         <OrphaCode>97685</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97685</ExpertLink>
-        <Name lang="nl">17q11 microdeletiesyndroom</Name>
+        <Name lang="nl">17q11-microdeletiesyndroom</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -1160600,7 +1160600,7 @@
       <Disorder id="10464">
         <OrphaCode>41751</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=41751</ExpertLink>
-        <Name lang="nl">Bietti kristallijne dystrofie</Name>
+        <Name lang="nl">Kristallijne dystrofie van Bietti</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -1172487,7 +1172487,7 @@
       <Disorder id="11139">
         <OrphaCode>79097</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79097</ExpertLink>
-        <Name lang="nl">Folinezuurresponsieve epileptische aanvallen</Name>
+        <Name lang="nl">Folinezuurresponsieve insulten</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
