commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases with associated phenotypes/nl_product4.xml b/Rare diseases with associated phenotypes/nl_product4.xml
index e30d51f..665f5d1 100644
--- a/Rare diseases with associated phenotypes/nl_product4.xml	
+++ b/Rare diseases with associated phenotypes/nl_product4.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:37:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:25:38" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <HPODisorderSetStatusList count="4245">
     <HPODisorderSetStatus id="1">
       <Disorder id="2">
@@ -86275,7 +86268,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=442</ExpertLink>
         <Name lang="nl">Congenitale hypothyreoïdie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -303914,7 +303907,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79168</ExpertLink>
         <Name lang="nl">Aandoening van galzuursynthese</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -326867,7 +326860,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79357</ExpertLink>
         <Name lang="nl">Erfelijke palmoplantaire keratodermie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -326906,7 +326899,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79358</ExpertLink>
         <Name lang="nl">Porokeratose</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -339786,7 +339779,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90970</ExpertLink>
         <Name lang="nl">Primaire lipodystrofie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -351385,7 +351378,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100100</ExpertLink>
         <Name lang="nl">Thymische tumor</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -357367,7 +357360,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98292</ExpertLink>
         <Name lang="nl">Mastocytose</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -403576,7 +403569,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=181393</ExpertLink>
         <Name lang="nl">Groeihormoonongevoeligheidsyndroom</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -414083,7 +414076,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=226292</ExpertLink>
         <Name lang="nl">Permanente congenitale hypothyreoïdie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -485944,7 +485937,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98028</ExpertLink>
         <Name lang="nl">Zeldzame bloedsomloopziekte</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -597840,7 +597833,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261920</ExpertLink>
         <Name lang="nl">Partiële deletie van de korte arm van chromosoom 8</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -608315,7 +608308,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100076</ExpertLink>
         <Name lang="nl">Neuro-endocriene tumor van het duodenum</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -608684,7 +608677,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100077</ExpertLink>
         <Name lang="nl">Neuro-endocriene tumor van het jejunum</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -721412,7 +721405,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=262767</ExpertLink>
         <Name lang="nl">Partiële trisomie/tetrasomie van de korte arm van chromosoom 9</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -762439,7 +762432,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217607</ExpertLink>
         <Name lang="nl">Familiale gedilateerde cardiomyopathie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -764018,7 +764011,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261911</ExpertLink>
         <Name lang="nl">Partiële deletie van de korte arm van chromosoom 7</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -921615,7 +921608,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352</ExpertLink>
         <Name lang="nl">Galactosemie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
@@ -929966,7 +929959,7 @@
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99</ExpertLink>
         <Name lang="nl">Autosomale dominante cerebellaire ataxie</Name>
         <DisorderType id="36561">
-          <Name lang="nl">Category</Name>
+          <Name lang="nl">Categorie</Name>
         </DisorderType>
         <DisorderGroup id="36540">
           <Name lang="nl">Groep van aandoeningen</Name>
