commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml
index a3d97bc..6febc0c 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:39:08" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2021-12-01 04:42:33" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <ClassificationList count="1">
     <Classification id="156">
       <OrphaNumber>158293</OrphaNumber>
@@ -2925,7 +2918,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="4">
+                                  <ClassificationNodeChildList count="5">
                                     <ClassificationNode>
                                       <Disorder id="1738">
                                         <OrphaCode>1606</OrphaCode>
@@ -2962,6 +2955,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31324">
+                                        <OrphaCode>615986</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615986</ExpertLink>
+                                        <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome due to biallelic deletions in the ATAD3 gene cluster</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="23675">
                                         <OrphaCode>456298</OrphaCode>
@@ -7217,7 +7222,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="7">
+                  <ClassificationNodeChildList count="6">
                     <ClassificationNode>
                       <Disorder id="6023">
                         <OrphaCode>178</OrphaCode>
@@ -7254,18 +7259,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="14450">
-                        <OrphaCode>99877</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99877</ExpertLink>
-                        <Name lang="en">Familial parathyroid adenoma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="14666">
                         <OrphaCode>100094</OrphaCode>
@@ -15479,7 +15472,7 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="6">
+                              <ClassificationNodeChildList count="4">
                                 <ClassificationNode>
                                   <Disorder id="18309">
                                     <OrphaCode>183678</OrphaCode>
@@ -15528,30 +15521,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19199">
-                                    <OrphaCode>231537</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231537</ExpertLink>
-                                    <Name lang="en">Hermansky-Pudlak syndrome type 8</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="20493">
-                                    <OrphaCode>280663</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280663</ExpertLink>
-                                    <Name lang="en">Hermansky-Pudlak syndrome type 9</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -15693,8 +15662,8 @@
                             <OrphaCode>2346</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                             <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="2">
@@ -15703,8 +15672,8 @@
                                 <OrphaCode>90308</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                                 <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -15715,8 +15684,8 @@
                                 <OrphaCode>90307</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                                 <Name lang="en">Parkes Weber syndrome</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -27639,7 +27608,19 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
+                        <ClassificationNode>
+                          <Disorder id="31321">
+                            <OrphaCode>615964</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615964</ExpertLink>
+                            <Name lang="en">Acute reversible leukoencephalopathy with increased urinary alpha-ketoglutarate</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1259">
                             <OrphaCode>31</OrphaCode>
@@ -29329,7 +29310,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="12">
+                          <ClassificationNodeChildList count="13">
                             <ClassificationNode>
                               <Disorder id="26450">
                                 <OrphaCode>506353</OrphaCode>
@@ -29342,6 +29323,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31318">
+                                <OrphaCode>615938</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615938</ExpertLink>
+                                <Name lang="en">Spastic paraparesis-cataracts-speech delay syndrome</Name>
+                                <DisorderType id="21422">
+                                  <Name lang="en">Clinical syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="586">
                                 <OrphaCode>816</OrphaCode>
@@ -29739,7 +29732,7 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="4">
                         <ClassificationNode>
                           <Disorder id="18309">
                             <OrphaCode>183678</OrphaCode>
@@ -29788,30 +29781,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="19199">
-                            <OrphaCode>231537</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231537</ExpertLink>
-                            <Name lang="en">Hermansky-Pudlak syndrome type 8</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="20493">
-                            <OrphaCode>280663</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280663</ExpertLink>
-                            <Name lang="en">Hermansky-Pudlak syndrome type 9</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -31055,7 +31024,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="5">
+                              <ClassificationNodeChildList count="4">
                                 <ClassificationNode>
                                   <Disorder id="11256">
                                     <OrphaCode>79230</OrphaCode>
@@ -31092,18 +31061,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="23545">
-                                    <OrphaCode>447792</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=447792</ExpertLink>
-                                    <Name lang="en">Hemochromatosis type 5</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="24056">
                                     <OrphaCode>465508</OrphaCode>
@@ -33005,7 +32962,19 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="152">
+                  <ClassificationNodeChildList count="154">
+                    <ClassificationNode>
+                      <Disorder id="31321">
+                        <OrphaCode>615964</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615964</ExpertLink>
+                        <Name lang="en">Acute reversible leukoencephalopathy with increased urinary alpha-ketoglutarate</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="19793">
                         <OrphaCode>254875</OrphaCode>
@@ -33066,6 +33035,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31318">
+                        <OrphaCode>615938</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615938</ExpertLink>
+                        <Name lang="en">Spastic paraparesis-cataracts-speech delay syndrome</Name>
+                        <DisorderType id="21422">
+                          <Name lang="en">Clinical syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="3">
                         <OrphaCode>61</OrphaCode>
@@ -39654,7 +39635,7 @@
                               <Disorder id="11818">
                                 <OrphaCode>88642</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=88642</ExpertLink>
-                                <Name lang="en">Channelopathy-associated congenital insensitivity to pain</Name>
+                                <Name lang="en">Congenital insensitivity to pain-anosmia-neuropathic arthropathy</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -39800,7 +39781,7 @@
                           <Disorder id="18855">
                             <OrphaCode>217399</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217399</ExpertLink>
-                            <Name lang="en">Congenital insensitivity to pain with hyperhidrosis</Name>
+                            <Name lang="en">Congenital insensitivity to pain-hyperhidrosis-absence of C-fiber innervation</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -52211,7 +52192,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="11">
+                              <ClassificationNodeChildList count="12">
                                 <ClassificationNode>
                                   <Disorder id="23536">
                                     <OrphaCode>447753</OrphaCode>
@@ -52344,6 +52325,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31318">
+                                    <OrphaCode>615938</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615938</ExpertLink>
+                                    <Name lang="en">Spastic paraparesis-cataracts-speech delay syndrome</Name>
+                                    <DisorderType id="21422">
+                                      <Name lang="en">Clinical syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -58595,7 +58588,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="11">
+                              <ClassificationNodeChildList count="13">
                                 <ClassificationNode>
                                   <Disorder id="2334">
                                     <OrphaCode>2524</OrphaCode>
@@ -58728,6 +58721,30 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="30702">
+                                    <OrphaCode>613267</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                                    <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="30703">
+                                    <OrphaCode>613274</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                                    <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -58739,7 +58756,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="1">
+                              <ClassificationNodeChildList count="2">
                                 <ClassificationNode>
                                   <Disorder id="1022">
                                     <OrphaCode>475</OrphaCode>
@@ -58752,6 +58769,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="1604">
+                                    <OrphaCode>1397</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1397</ExpertLink>
+                                    <Name lang="en">Hydrocephaly-cerebellar agenesis syndrome</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -60848,7 +60877,7 @@
                           <Disorder id="30683">
                             <OrphaCode>610569</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                            <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                            <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -66284,7 +66313,7 @@
                           <Disorder id="11669">
                             <OrphaCode>85279</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85279</ExpertLink>
-                            <Name lang="en">Syndromic X-linked intellectual disability due to JARID1C mutation</Name>
+                            <Name lang="en">KDM5C-related syndromic X-linked intellectual disability</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -68377,7 +68406,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="11">
+                          <ClassificationNodeChildList count="13">
                             <ClassificationNode>
                               <Disorder id="2334">
                                 <OrphaCode>2524</OrphaCode>
@@ -68510,6 +68539,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="30702">
+                                <OrphaCode>613267</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                                <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="30703">
+                                <OrphaCode>613274</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                                <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -73297,7 +73350,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="649">
+                          <ClassificationNodeChildList count="650">
                             <ClassificationNode>
                               <Disorder id="2011">
                                 <OrphaCode>380</OrphaCode>
@@ -73602,7 +73655,7 @@
                               <Disorder id="30683">
                                 <OrphaCode>610569</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                                <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                                <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -76782,7 +76835,7 @@
                               <Disorder id="11669">
                                 <OrphaCode>85279</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85279</ExpertLink>
-                                <Name lang="en">Syndromic X-linked intellectual disability due to JARID1C mutation</Name>
+                                <Name lang="en">KDM5C-related syndromic X-linked intellectual disability</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -78122,18 +78175,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="12110">
-                                <OrphaCode>91133</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=91133</ExpertLink>
-                                <Name lang="en">Osteopenia-myopia-hearing loss-intellectual disability-facial dysmorphism syndrome</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="12492">
                                 <OrphaCode>93932</OrphaCode>
@@ -81710,6 +81751,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="30702">
+                                <OrphaCode>613267</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                                <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="30703">
+                                <OrphaCode>613274</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                                <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="1687">
                                 <OrphaCode>1520</OrphaCode>
@@ -102247,7 +102312,7 @@
                                           <Name lang="en">Category</Name>
                                         </DisorderType>
                                       </Disorder>
-                                      <ClassificationNodeChildList count="24">
+                                      <ClassificationNodeChildList count="25">
                                         <ClassificationNode>
                                           <Disorder id="3">
                                             <OrphaCode>61</OrphaCode>
@@ -102440,6 +102505,18 @@
                                           <ClassificationNodeChildList count="0">
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="31318">
+                                            <OrphaCode>615938</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615938</ExpertLink>
+                                            <Name lang="en">Spastic paraparesis-cataracts-speech delay syndrome</Name>
+                                            <DisorderType id="21422">
+                                              <Name lang="en">Clinical syndrome</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
                                         <ClassificationNode>
                                           <Disorder id="252">
                                             <OrphaCode>14</OrphaCode>
@@ -102927,8 +103004,8 @@
                                         <OrphaCode>2346</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                                         <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
+                                        <DisorderType id="21436">
+                                          <Name lang="en">Clinical group</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="2">
@@ -102937,8 +103014,8 @@
                                             <OrphaCode>90308</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                                             <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -102949,8 +103026,8 @@
                                             <OrphaCode>90307</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                                             <Name lang="en">Parkes Weber syndrome</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -104869,7 +104946,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="3">
                             <ClassificationNode>
                               <Disorder id="2233">
                                 <OrphaCode>2407</OrphaCode>
@@ -104882,18 +104959,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="27361">
-                                <OrphaCode>522544</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=522544</ExpertLink>
-                                <Name lang="en">Rare genetic conjunctivitis</Name>
-                                <DisorderType id="36561">
-                                  <Name lang="en">Category</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="236">
                                 <OrphaCode>774</OrphaCode>
@@ -115365,7 +115430,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="6">
+                                  <ClassificationNodeChildList count="4">
                                     <ClassificationNode>
                                       <Disorder id="18309">
                                         <OrphaCode>183678</OrphaCode>
@@ -115414,30 +115479,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19199">
-                                        <OrphaCode>231537</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231537</ExpertLink>
-                                        <Name lang="en">Hermansky-Pudlak syndrome type 8</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="20493">
-                                        <OrphaCode>280663</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280663</ExpertLink>
-                                        <Name lang="en">Hermansky-Pudlak syndrome type 9</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -118623,7 +118664,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="5">
+                      <ClassificationNodeChildList count="4">
                         <ClassificationNode>
                           <Disorder id="11256">
                             <OrphaCode>79230</OrphaCode>
@@ -118660,18 +118701,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="23545">
-                            <OrphaCode>447792</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=447792</ExpertLink>
-                            <Name lang="en">Hemochromatosis type 5</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="24056">
                             <OrphaCode>465508</OrphaCode>
@@ -128221,31 +128250,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="5">
-                            <ClassificationNode>
-                              <Disorder id="14450">
-                                <OrphaCode>99877</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99877</ExpertLink>
-                                <Name lang="en">Familial parathyroid adenoma</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="14451">
-                                <OrphaCode>99878</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99878</ExpertLink>
-                                <Name lang="en">Primary parathyroid hyperplasia</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="3">
                             <ClassificationNode>
                               <Disorder id="14452">
                                 <OrphaCode>99879</OrphaCode>
@@ -133471,7 +133476,7 @@
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
                                       </Disorder>
-                                      <ClassificationNodeChildList count="6">
+                                      <ClassificationNodeChildList count="4">
                                         <ClassificationNode>
                                           <Disorder id="18309">
                                             <OrphaCode>183678</OrphaCode>
@@ -133520,30 +133525,6 @@
                                           <ClassificationNodeChildList count="0">
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
-                                        <ClassificationNode>
-                                          <Disorder id="19199">
-                                            <OrphaCode>231537</OrphaCode>
-                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231537</ExpertLink>
-                                            <Name lang="en">Hermansky-Pudlak syndrome type 8</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
-                                            </DisorderType>
-                                          </Disorder>
-                                          <ClassificationNodeChildList count="0">
-                                          </ClassificationNodeChildList>
-                                        </ClassificationNode>
-                                        <ClassificationNode>
-                                          <Disorder id="20493">
-                                            <OrphaCode>280663</OrphaCode>
-                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280663</ExpertLink>
-                                            <Name lang="en">Hermansky-Pudlak syndrome type 9</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
-                                            </DisorderType>
-                                          </Disorder>
-                                          <ClassificationNodeChildList count="0">
-                                          </ClassificationNodeChildList>
-                                        </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
                                   </ClassificationNodeChildList>
@@ -136475,8 +136456,8 @@
                         <OrphaCode>2346</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                         <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
@@ -136485,8 +136466,8 @@
                             <OrphaCode>90308</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                             <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -136497,8 +136478,8 @@
                             <OrphaCode>90307</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                             <Name lang="en">Parkes Weber syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -149939,8 +149920,8 @@
                         <OrphaCode>2346</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                         <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
@@ -149949,8 +149930,8 @@
                             <OrphaCode>90308</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                             <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -149961,8 +149942,8 @@
                             <OrphaCode>90307</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                             <Name lang="en">Parkes Weber syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -156731,7 +156712,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="11">
+                              <ClassificationNodeChildList count="13">
                                 <ClassificationNode>
                                   <Disorder id="2334">
                                     <OrphaCode>2524</OrphaCode>
@@ -156864,6 +156845,30 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="30702">
+                                    <OrphaCode>613267</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                                    <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="30703">
+                                    <OrphaCode>613274</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                                    <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -156875,7 +156880,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="1">
+                              <ClassificationNodeChildList count="2">
                                 <ClassificationNode>
                                   <Disorder id="1022">
                                     <OrphaCode>475</OrphaCode>
@@ -156888,6 +156893,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="1604">
+                                    <OrphaCode>1397</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1397</ExpertLink>
+                                    <Name lang="en">Hydrocephaly-cerebellar agenesis syndrome</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -158984,7 +159001,7 @@
                           <Disorder id="30683">
                             <OrphaCode>610569</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                            <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                            <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -161427,7 +161444,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="649">
+                      <ClassificationNodeChildList count="650">
                         <ClassificationNode>
                           <Disorder id="2011">
                             <OrphaCode>380</OrphaCode>
@@ -161732,7 +161749,7 @@
                           <Disorder id="30683">
                             <OrphaCode>610569</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                            <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                            <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -164912,7 +164929,7 @@
                           <Disorder id="11669">
                             <OrphaCode>85279</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85279</ExpertLink>
-                            <Name lang="en">Syndromic X-linked intellectual disability due to JARID1C mutation</Name>
+                            <Name lang="en">KDM5C-related syndromic X-linked intellectual disability</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -166252,18 +166269,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="12110">
-                            <OrphaCode>91133</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=91133</ExpertLink>
-                            <Name lang="en">Osteopenia-myopia-hearing loss-intellectual disability-facial dysmorphism syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="12492">
                             <OrphaCode>93932</OrphaCode>
@@ -169840,6 +169845,30 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="30702">
+                            <OrphaCode>613267</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                            <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="30703">
+                            <OrphaCode>613274</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                            <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1687">
                             <OrphaCode>1520</OrphaCode>
@@ -173812,7 +173841,7 @@
                                   <Disorder id="30683">
                                     <OrphaCode>610569</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                                    <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                                    <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -176926,7 +176955,7 @@
                       <Disorder id="21494">
                         <OrphaCode>314555</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314555</ExpertLink>
-                        <Name lang="en">Craniofacial dysplasia-osteopenia syndrome</Name>
+                        <Name lang="en">Facial dysmorphism-severe myopia-osteopenia-intellectual disability-dental anomalies syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -185373,8 +185402,8 @@
                             <OrphaCode>2346</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                             <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="2">
@@ -185383,8 +185412,8 @@
                                 <OrphaCode>90308</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                                 <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -185395,8 +185424,8 @@
                                 <OrphaCode>90307</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                                 <Name lang="en">Parkes Weber syndrome</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -190821,8 +190850,8 @@
                             <OrphaCode>2346</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                             <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="2">
@@ -190831,8 +190860,8 @@
                                 <OrphaCode>90308</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                                 <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -190843,8 +190872,8 @@
                                 <OrphaCode>90307</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                                 <Name lang="en">Parkes Weber syndrome</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -209061,7 +209090,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="4">
+                                  <ClassificationNodeChildList count="3">
                                     <ClassificationNode>
                                       <Disorder id="249">
                                         <OrphaCode>167</OrphaCode>
@@ -209098,18 +209127,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="20493">
-                                        <OrphaCode>280663</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280663</ExpertLink>
-                                        <Name lang="en">Hermansky-Pudlak syndrome type 9</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
