commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml
old mode 100755
new mode 100644
index 1bb4b95..f2c7bff
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:34:07" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:36:38" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
