commit b18c171d5ba8181794aeffcce3d0c8a41ddab6cb
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 5 14:56:57 2022 +0100

    DEC 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml
old mode 100644
new mode 100755
index bc46a4b..4877430
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml	
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:18:47" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:38:44" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <Availability> 
+    <Licence>
+      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
+      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
+      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
+    </Licence>
+  </Availability>
   <ClassificationList count="1">
     <Classification id="198">
       <OrphaNumber>162961</OrphaNumber>
@@ -1275,7 +1282,7 @@
                       <Disorder id="11787">
                         <OrphaCode>86914</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86914</ExpertLink>
-                        <Name lang="en">Lymphedema-cerebral arteriovenous anomaly syndrome</Name>
+                        <Name lang="en">Lymphedema-cerebral arteriovenous anomaly-primary pulmonary hypertension syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
