commit b18c171d5ba8181794aeffcce3d0c8a41ddab6cb
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 5 14:56:57 2022 +0100

    DEC 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml
old mode 100644
new mode 100755
index 20fac64..7af8bd4
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:18:17" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:38:14" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <Availability> 
+    <Licence>
+      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
+      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
+      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
+    </Licence>
+  </Availability>
   <ClassificationList count="1">
     <Classification id="193">
       <OrphaNumber>162946</OrphaNumber>
@@ -2024,7 +2031,19 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
+                        <ClassificationNode>
+                          <Disorder id="31664">
+                            <OrphaCode>633028</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633028</ExpertLink>
+                            <Name lang="en">CPE-related Prader-Willi-like syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="17936">
                             <OrphaCode>171829</OrphaCode>
@@ -3484,7 +3503,19 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="3">
+                              <ClassificationNodeChildList count="4">
+                                <ClassificationNode>
+                                  <Disorder id="31664">
+                                    <OrphaCode>633028</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633028</ExpertLink>
+                                    <Name lang="en">CPE-related Prader-Willi-like syndrome</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="17936">
                                     <OrphaCode>171829</OrphaCode>
@@ -9926,7 +9957,19 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="3">
+                                  <ClassificationNodeChildList count="4">
+                                    <ClassificationNode>
+                                      <Disorder id="31664">
+                                        <OrphaCode>633028</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633028</ExpertLink>
+                                        <Name lang="en">CPE-related Prader-Willi-like syndrome</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="17936">
                                         <OrphaCode>171829</OrphaCode>
