commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml
index c3218ec..e833cdb 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:22:33" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:06:59" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -21,7 +21,7 @@
               <Name lang="en">Category</Name>
             </DisorderType>
           </Disorder>
-          <ClassificationNodeChildList count="47">
+          <ClassificationNodeChildList count="49">
             <ClassificationNode>
               <Disorder id="30619">
                 <OrphaCode>603699</OrphaCode>
@@ -363,7 +363,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="8">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
                           <Disorder id="889">
                             <OrphaCode>1934</OrphaCode>
@@ -376,35 +376,11 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="890">
-                            <OrphaCode>1935</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1935</ExpertLink>
-                            <Name lang="en">Early myoclonic encephalopathy</Name>
-                            <DisorderType id="21422">
-                              <Name lang="en">Clinical syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="990">
                             <OrphaCode>1949</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1949</ExpertLink>
-                            <Name lang="en">Benign familial neonatal epilepsy</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="10834">
-                            <OrphaCode>64545</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=64545</ExpertLink>
-                            <Name lang="en">Benign idiopathic neonatal seizures</Name>
+                            <Name lang="en">Self-limited neonatal epilepsy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -428,7 +404,7 @@
                           <Disorder id="20841">
                             <OrphaCode>293181</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293181</ExpertLink>
-                            <Name lang="en">Malignant migrating focal seizures of infancy</Name>
+                            <Name lang="en">Epilepsy of infancy with migrating focal seizures</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -488,7 +464,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">Infantile spasms syndrome</Name>
+                            <Name lang="en">Infantile epileptic spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -512,7 +488,7 @@
                           <Disorder id="10424">
                             <OrphaCode>36387</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36387</ExpertLink>
-                            <Name lang="en">Generalized epilepsy with febrile seizures-plus</Name>
+                            <Name lang="en">Genetic epilepsy with febrile seizure plus</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -524,7 +500,7 @@
                           <Disorder id="11783">
                             <OrphaCode>86908</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86908</ExpertLink>
-                            <Name lang="en">Idiopathic hemiconvulsion-hemiplegia syndrome</Name>
+                            <Name lang="en">Hemiconvulsion-hemiplegia-epilepsy syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -565,7 +541,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="7">
+                          <ClassificationNodeChildList count="6">
                             <ClassificationNode>
                               <Disorder id="1031">
                                 <OrphaCode>569</OrphaCode>
@@ -582,7 +558,7 @@
                               <Disorder id="3651">
                                 <OrphaCode>306</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306</ExpertLink>
-                                <Name lang="en">Benign familial infantile epilepsy</Name>
+                                <Name lang="en">Self-limited infantile epilepsy</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -606,7 +582,7 @@
                               <Disorder id="17000">
                                 <OrphaCode>140927</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140927</ExpertLink>
-                                <Name lang="en">Benign familial neonatal-infantile seizures</Name>
+                                <Name lang="en">Self-limited neonatal-infantile epilepsy</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -614,42 +590,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="17631">
-                                <OrphaCode>166295</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166295</ExpertLink>
-                                <Name lang="en">Benign non-familial infantile seizures</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="17632">
-                                    <OrphaCode>166299</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166299</ExpertLink>
-                                    <Name lang="en">Benign partial epilepsy of infancy with complex partial seizures</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="17633">
-                                    <OrphaCode>166302</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166302</ExpertLink>
-                                    <Name lang="en">Benign partial epilepsy with secondarily generalized seizures in infancy</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="17634">
                                 <OrphaCode>166305</OrphaCode>
@@ -810,7 +750,7 @@
                               <Disorder id="3321">
                                 <OrphaCode>1945</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1945</ExpertLink>
-                                <Name lang="en">Rolandic epilepsy</Name>
+                                <Name lang="en">Self-limited epilepsy with centrotemporal spikes</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -822,7 +762,7 @@
                               <Disorder id="10424">
                                 <OrphaCode>36387</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36387</ExpertLink>
-                                <Name lang="en">Generalized epilepsy with febrile seizures-plus</Name>
+                                <Name lang="en">Genetic epilepsy with febrile seizure plus</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -1064,7 +1004,7 @@
                           <Disorder id="8736">
                             <OrphaCode>725</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=725</ExpertLink>
-                            <Name lang="en">Continuous spikes and waves during sleep</Name>
+                            <Name lang="en">Developmental and epileptic encephalopathy with spike-wave activation in sleep</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1076,7 +1016,7 @@
                           <Disorder id="8755">
                             <OrphaCode>25968</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=25968</ExpertLink>
-                            <Name lang="en">Benign occipital epilepsy</Name>
+                            <Name lang="en">Self-limited childhood occipital epilepsy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1086,7 +1026,7 @@
                               <Disorder id="13832">
                                 <OrphaCode>98815</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98815</ExpertLink>
-                                <Name lang="en">Benign childhood occipital epilepsy, Panayiotopoulos type</Name>
+                                <Name lang="en">Self-limited epilepsy with autonomic seizures</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -1098,7 +1038,7 @@
                               <Disorder id="13833">
                                 <OrphaCode>98816</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98816</ExpertLink>
-                                <Name lang="en">Benign childhood occipital epilepsy, Gastaut type</Name>
+                                <Name lang="en">Childhood occipital visual epilepsy</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -1542,7 +1482,7 @@
                               <Disorder id="11783">
                                 <OrphaCode>86908</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86908</ExpertLink>
-                                <Name lang="en">Idiopathic hemiconvulsion-hemiplegia syndrome</Name>
+                                <Name lang="en">Hemiconvulsion-hemiplegia-epilepsy syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -1638,7 +1578,7 @@
                               <Disorder id="3321">
                                 <OrphaCode>1945</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1945</ExpertLink>
-                                <Name lang="en">Rolandic epilepsy</Name>
+                                <Name lang="en">Self-limited epilepsy with centrotemporal spikes</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -1650,7 +1590,7 @@
                               <Disorder id="10424">
                                 <OrphaCode>36387</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36387</ExpertLink>
-                                <Name lang="en">Generalized epilepsy with febrile seizures-plus</Name>
+                                <Name lang="en">Genetic epilepsy with febrile seizure plus</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -2518,7 +2458,7 @@
                       <Disorder id="557">
                         <OrphaCode>884</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=884</ExpertLink>
-                        <Name lang="en">Tetrasomy 12p</Name>
+                        <Name lang="en">Tetrasomy 12p syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -2681,7 +2621,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="38">
+                  <ClassificationNodeChildList count="40">
                     <ClassificationNode>
                       <Disorder id="29430">
                         <OrphaCode>589515</OrphaCode>
@@ -2840,7 +2780,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">Infantile spasms syndrome</Name>
+                            <Name lang="en">Infantile epileptic spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -2970,7 +2910,7 @@
                               <Disorder id="3321">
                                 <OrphaCode>1945</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1945</ExpertLink>
-                                <Name lang="en">Rolandic epilepsy</Name>
+                                <Name lang="en">Self-limited epilepsy with centrotemporal spikes</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -2982,7 +2922,7 @@
                               <Disorder id="10424">
                                 <OrphaCode>36387</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36387</ExpertLink>
-                                <Name lang="en">Generalized epilepsy with febrile seizures-plus</Name>
+                                <Name lang="en">Genetic epilepsy with febrile seizure plus</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -3128,7 +3068,7 @@
                           <Disorder id="20841">
                             <OrphaCode>293181</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293181</ExpertLink>
-                            <Name lang="en">Malignant migrating focal seizures of infancy</Name>
+                            <Name lang="en">Epilepsy of infancy with migrating focal seizures</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3606,6 +3546,30 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32153">
+                        <OrphaCode>684232</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684232</ExpertLink>
+                        <Name lang="en">Intellectual disability-epilepsy-dental anomalies-facial dysmorphism syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32154">
+                        <OrphaCode>684240</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684240</ExpertLink>
+                        <Name lang="en">Neurodevelopmental disorder-spasticity-movement disorder-epileptic syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -3656,7 +3620,7 @@
                           <Disorder id="3321">
                             <OrphaCode>1945</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1945</ExpertLink>
-                            <Name lang="en">Rolandic epilepsy</Name>
+                            <Name lang="en">Self-limited epilepsy with centrotemporal spikes</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3668,7 +3632,7 @@
                           <Disorder id="10424">
                             <OrphaCode>36387</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36387</ExpertLink>
-                            <Name lang="en">Generalized epilepsy with febrile seizures-plus</Name>
+                            <Name lang="en">Genetic epilepsy with febrile seizure plus</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3790,7 +3754,7 @@
                       <Disorder id="3651">
                         <OrphaCode>306</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306</ExpertLink>
-                        <Name lang="en">Benign familial infantile epilepsy</Name>
+                        <Name lang="en">Self-limited infantile epilepsy</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -3802,7 +3766,7 @@
                       <Disorder id="17000">
                         <OrphaCode>140927</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140927</ExpertLink>
-                        <Name lang="en">Benign familial neonatal-infantile seizures</Name>
+                        <Name lang="en">Self-limited neonatal-infantile epilepsy</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -7796,7 +7760,7 @@
                           <Disorder id="11783">
                             <OrphaCode>86908</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86908</ExpertLink>
-                            <Name lang="en">Idiopathic hemiconvulsion-hemiplegia syndrome</Name>
+                            <Name lang="en">Hemiconvulsion-hemiplegia-epilepsy syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -8730,7 +8694,7 @@
                               <Disorder id="13781">
                                 <OrphaCode>98764</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98764</ExpertLink>
-                                <Name lang="en">Spinocerebellar ataxia type 27</Name>
+                                <Name lang="en">Spinocerebellar ataxia type 27A</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -14719,7 +14683,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="60">
+              <ClassificationNodeChildList count="61">
                 <ClassificationNode>
                   <Disorder id="28139">
                     <OrphaCode>556985</OrphaCode>
@@ -14756,6 +14720,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32156">
+                    <OrphaCode>684290</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684290</ExpertLink>
+                    <Name lang="en">Hypertrophic olivary degeneration</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31449">
                     <OrphaCode>621758</OrphaCode>
@@ -15241,7 +15217,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="8">
+                          <ClassificationNodeChildList count="9">
                             <ClassificationNode>
                               <Disorder id="14704">
                                 <OrphaCode>100993</OrphaCode>
@@ -15326,6 +15302,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32207">
+                                <OrphaCode>689231</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689231</ExpertLink>
+                                <Name lang="en">IFH1-related hereditary spastic paraplegia</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="31579">
                                 <OrphaCode>631068</OrphaCode>
@@ -15349,7 +15337,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="6">
+                          <ClassificationNodeChildList count="7">
                             <ClassificationNode>
                               <Disorder id="14715">
                                 <OrphaCode>101004</OrphaCode>
@@ -15422,6 +15410,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32208">
+                                <OrphaCode>689234</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689234</ExpertLink>
+                                <Name lang="en">RNASEH2B-related hereditary spastic paraplegia</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -18326,7 +18326,7 @@
                                       <Disorder id="13781">
                                         <OrphaCode>98764</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98764</ExpertLink>
-                                        <Name lang="en">Spinocerebellar ataxia type 27</Name>
+                                        <Name lang="en">Spinocerebellar ataxia type 27A</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -22016,7 +22016,7 @@
                           <Disorder id="3036">
                             <OrphaCode>3454</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3454</ExpertLink>
-                            <Name lang="en">Intellectual disability-developmental delay-contractures syndrome</Name>
+                            <Name lang="en">Wieacker-Wolff syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -22096,7 +22096,7 @@
                                   <Disorder id="10782">
                                     <OrphaCode>59135</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=59135</ExpertLink>
-                                    <Name lang="en">Laing early-onset distal myopathy</Name>
+                                    <Name lang="en">Laing distal myopathy</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -22235,7 +22235,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="7">
+                              <ClassificationNodeChildList count="8">
                                 <ClassificationNode>
                                   <Disorder id="8729">
                                     <OrphaCode>602</OrphaCode>
@@ -22308,6 +22308,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="32204">
+                                    <OrphaCode>689021</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689021</ExpertLink>
+                                    <Name lang="en">Asymptomatic hyperCKemia-myalgia-rhabdomyolysis syndrome</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="25255">
                                     <OrphaCode>482601</OrphaCode>
@@ -32168,6 +32180,18 @@
                     </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32203">
+                    <OrphaCode>689001</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689001</ExpertLink>
+                    <Name lang="en">Isolated spontaneous vertebral artery dissection</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="25248">
                     <OrphaCode>482092</OrphaCode>
@@ -32252,18 +32276,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="10420">
-                    <OrphaCode>36382</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36382</ExpertLink>
-                    <Name lang="en">Familial cervical artery dissection</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="20587">
                     <OrphaCode>284247</OrphaCode>
@@ -32814,7 +32826,7 @@
                               <Disorder id="14199">
                                 <OrphaCode>99226</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                                <Name lang="en">Monosomy X</Name>
+                                <Name lang="en">Monosomy X syndrome</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -32826,7 +32838,7 @@
                               <Disorder id="14200">
                                 <OrphaCode>99228</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                                <Name lang="en">Mosaic monosomy X</Name>
+                                <Name lang="en">Mosaic monosomy X syndrome</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -33188,7 +33200,7 @@
                           <Disorder id="17620">
                             <OrphaCode>166108</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166108</ExpertLink>
-                            <Name lang="en">Intellectual disability, Birk-Barel type</Name>
+                            <Name lang="en">Birk-Barel syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -34064,7 +34076,7 @@
                           <Disorder id="10424">
                             <OrphaCode>36387</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36387</ExpertLink>
-                            <Name lang="en">Generalized epilepsy with febrile seizures-plus</Name>
+                            <Name lang="en">Genetic epilepsy with febrile seizure plus</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -38432,7 +38444,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">Infantile spasms syndrome</Name>
+                            <Name lang="en">Infantile epileptic spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -39351,7 +39363,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="679">
+                      <ClassificationNodeChildList count="695">
                         <ClassificationNode>
                           <Disorder id="11695">
                             <OrphaCode>85325</OrphaCode>
@@ -40064,7 +40076,7 @@
                           <Disorder id="20734">
                             <OrphaCode>289522</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=289522</ExpertLink>
-                            <Name lang="en">Microtriplication 11q24.1</Name>
+                            <Name lang="en">Microtriplication 11q24.1 syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -40086,7 +40098,7 @@
                               <Disorder id="1114">
                                 <OrphaCode>1620</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1620</ExpertLink>
-                                <Name lang="en">Distal deletion 3p</Name>
+                                <Name lang="en">Distal deletion 3p syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -40254,7 +40266,7 @@
                               <Disorder id="338">
                                 <OrphaCode>1707</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1707</ExpertLink>
-                                <Name lang="en">Distal duplication 15q</Name>
+                                <Name lang="en">Distal duplication 15q syndrome</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -40266,7 +40278,7 @@
                               <Disorder id="21499">
                                 <OrphaCode>314588</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314588</ExpertLink>
-                                <Name lang="en">Distal triplication 15q</Name>
+                                <Name lang="en">Distal triplication 15q syndrome</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -41092,18 +41104,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="23328">
-                            <OrphaCode>436151</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=436151</ExpertLink>
-                            <Name lang="en">Intellectual disability-expressive aphasia-facial dysmorphism syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="23335">
                             <OrphaCode>436245</OrphaCode>
@@ -41478,7 +41478,7 @@
                               <Disorder id="22720">
                                 <OrphaCode>398069</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=398069</ExpertLink>
-                                <Name lang="en">MAGEL2-related Prader-Willi-like syndrome</Name>
+                                <Name lang="en">Schaaf-Yang syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -41960,7 +41960,7 @@
                           <Disorder id="10418">
                             <OrphaCode>36367</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=36367</ExpertLink>
-                            <Name lang="en">Distal deletion 1q</Name>
+                            <Name lang="en">Distal deletion 1q syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -42668,7 +42668,7 @@
                           <Disorder id="12749">
                             <OrphaCode>96125</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96125</ExpertLink>
-                            <Name lang="en">Distal deletion 6p</Name>
+                            <Name lang="en">Distal deletion 6p syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -44912,7 +44912,7 @@
                           <Disorder id="505">
                             <OrphaCode>2135</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2135</ExpertLink>
-                            <Name lang="en">Hennekam-Beemer syndrome</Name>
+                            <Name lang="en">Cutaneous mastocytosis-deafness-microtia syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -47084,7 +47084,7 @@
                           <Disorder id="23705">
                             <OrphaCode>457193</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=457193</ExpertLink>
-                            <Name lang="en">Autosomal dominant intellectual disability-craniofacial anomalies-cardiac defects syndrome</Name>
+                            <Name lang="en">KAT6-related intellectual disability-craniofacial anomalies-cardiac defects syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -47320,6 +47320,30 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32151">
+                            <OrphaCode>684216</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684216</ExpertLink>
+                            <Name lang="en">Intellectual disability-facial dysmorphism-joint hypermobility-hearing loss syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32158">
+                            <OrphaCode>684742</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684742</ExpertLink>
+                            <Name lang="en">2q13 microdeletion syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="30702">
                             <OrphaCode>613267</OrphaCode>
@@ -47344,6 +47368,150 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32152">
+                            <OrphaCode>684226</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684226</ExpertLink>
+                            <Name lang="en">Intellectual disability-hypotonia-facial dysmorphism-macrocephaly syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32184">
+                            <OrphaCode>687695</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=687695</ExpertLink>
+                            <Name lang="en">10p13-p14 deletion syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32183">
+                            <OrphaCode>687424</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=687424</ExpertLink>
+                            <Name lang="en">10p15 microdeletion syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32176">
+                            <OrphaCode>686482</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686482</ExpertLink>
+                            <Name lang="en">BPTF-related intellectual disability-facial dysmorphism-skeletal anomalies syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32177">
+                            <OrphaCode>686488</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686488</ExpertLink>
+                            <Name lang="en">RNU4-2-related neurodevelopmental disorder-facial dysmorphism-white matter abnormalities-short stature syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32179">
+                            <OrphaCode>686495</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686495</ExpertLink>
+                            <Name lang="en">MADD-related developmental delay-endocrine dysfunction-hypohemoglobinemia syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32164">
+                            <OrphaCode>685017</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=685017</ExpertLink>
+                            <Name lang="en">Combined immunodeficiency due to TBX1 deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32195">
+                            <OrphaCode>688642</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=688642</ExpertLink>
+                            <Name lang="en">Turnpenny-Fry syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32214">
+                            <OrphaCode>689422</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689422</ExpertLink>
+                            <Name lang="en">Okur-Chung neurodevelopmental syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32210">
+                            <OrphaCode>689397</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689397</ExpertLink>
+                            <Name lang="en">Poirier-Bienvenue neurodevelopmental syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32219">
+                            <OrphaCode>689829</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689829</ExpertLink>
+                            <Name lang="en">Microphthalmia-motor delay-language delay-brain anomalies-diaphragmatic hernia syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32212">
+                            <OrphaCode>689408</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689408</ExpertLink>
+                            <Name lang="en">Shashi-Pena syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="31405">
                             <OrphaCode>619233</OrphaCode>
@@ -47685,8 +47853,8 @@
                             <OrphaCode>658540</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658540</ExpertLink>
                             <Name lang="en">16q22 deletion syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -47697,8 +47865,8 @@
                             <OrphaCode>658843</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658843</ExpertLink>
                             <Name lang="en">Developmental delay-ataxia-hypotonia-facial dysmorphism syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -47817,8 +47985,8 @@
                             <OrphaCode>659642</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659642</ExpertLink>
                             <Name lang="en">Rauch-Steindl syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -47836,6 +48004,42 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32157">
+                            <OrphaCode>684305</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684305</ExpertLink>
+                            <Name lang="en">NOCGUS syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32153">
+                            <OrphaCode>684232</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684232</ExpertLink>
+                            <Name lang="en">Intellectual disability-epilepsy-dental anomalies-facial dysmorphism syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32154">
+                            <OrphaCode>684240</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684240</ExpertLink>
+                            <Name lang="en">Neurodevelopmental disorder-spasticity-movement disorder-epileptic syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1269">
                             <OrphaCode>950</OrphaCode>
@@ -48248,7 +48452,7 @@
                           <Disorder id="3036">
                             <OrphaCode>3454</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3454</ExpertLink>
-                            <Name lang="en">Intellectual disability-developmental delay-contractures syndrome</Name>
+                            <Name lang="en">Wieacker-Wolff syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -51732,7 +51936,7 @@
                                           <Disorder id="13781">
                                             <OrphaCode>98764</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98764</ExpertLink>
-                                            <Name lang="en">Spinocerebellar ataxia type 27</Name>
+                                            <Name lang="en">Spinocerebellar ataxia type 27A</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -54058,7 +54262,7 @@
                       <Disorder id="22720">
                         <OrphaCode>398069</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=398069</ExpertLink>
-                        <Name lang="en">MAGEL2-related Prader-Willi-like syndrome</Name>
+                        <Name lang="en">Schaaf-Yang syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -58665,8 +58869,8 @@
                             <OrphaCode>659642</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659642</ExpertLink>
                             <Name lang="en">Rauch-Steindl syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -63683,7 +63887,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="6">
+                              <ClassificationNodeChildList count="7">
                                 <ClassificationNode>
                                   <Disorder id="10625">
                                     <OrphaCode>48431</OrphaCode>
@@ -63888,6 +64092,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="32182">
+                                    <OrphaCode>686999</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686999</ExpertLink>
+                                    <Name lang="en">Lipodystrophy-demyelinating peripheral sensory-motor neuropathy syndrome</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="23676">
                                     <OrphaCode>456312</OrphaCode>
@@ -65458,6 +65674,30 @@
                 </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
+            <ClassificationNode>
+              <Disorder id="32174">
+                <OrphaCode>686468</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686468</ExpertLink>
+                <Name lang="en">Post 5-alpha-reductase inhibitors treatment syndrome</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="0">
+              </ClassificationNodeChildList>
+            </ClassificationNode>
+            <ClassificationNode>
+              <Disorder id="32175">
+                <OrphaCode>686475</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686475</ExpertLink>
+                <Name lang="en">Post-selective serotonin reuptake inhibitor sexual dysfunction</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="0">
+              </ClassificationNodeChildList>
+            </ClassificationNode>
           </ClassificationNodeChildList>
         </ClassificationNode>
       </ClassificationNodeRootList>
