commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_196.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_196.xml
index 38a6a6c..3bde4e2 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_196.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_196.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:05:36" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:24:11" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -406,7 +406,7 @@
                       <Disorder id="22938">
                         <OrphaCode>404553</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=404553</ExpertLink>
-                        <Name lang="en">Vasculitis due to ADA2 deficiency</Name>
+                        <Name lang="en">Adenosine deaminase 2 deficiency</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -679,7 +679,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="34">
+              <ClassificationNodeChildList count="33">
                 <ClassificationNode>
                   <Disorder id="19145">
                     <OrphaCode>230857</OrphaCode>
@@ -874,7 +874,7 @@
                       <Disorder id="11725">
                         <OrphaCode>85458</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85458</ExpertLink>
-                        <Name lang="en">Hereditary cerebral hemorrhage with amyloidosis</Name>
+                        <Name lang="en">Cerebral Amyloid Angiopathy</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -971,8 +971,8 @@
                         <OrphaCode>271861</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=271861</ExpertLink>
                         <Name lang="en">Hereditary ATTR amyloidosis</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
@@ -981,8 +981,8 @@
                             <OrphaCode>85447</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85447</ExpertLink>
                             <Name lang="en">ATTRV30M amyloidosis</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -993,8 +993,8 @@
                             <OrphaCode>85451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85451</ExpertLink>
                             <Name lang="en">ATTRV122I amyloidosis</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -1244,18 +1244,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="11724">
-                    <OrphaCode>85453</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85453</ExpertLink>
-                    <Name lang="en">X-linked reticulate pigmentary disorder</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="13266">
                     <OrphaCode>98249</OrphaCode>
@@ -2290,7 +2278,7 @@
                       <Disorder id="23490">
                         <OrphaCode>444463</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=444463</ExpertLink>
-                        <Name lang="en">Autoimmune hemolytic anemia-autoimmune thrombocytopenia-primary immunodeficiency syndrome</Name>
+                        <Name lang="en">Autoimmune hemolytic anemia-autoimmune thrombocytopenia-primary immunodeficiency syndrome due to TPP2 deficiency</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -2875,7 +2863,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="21">
+              <ClassificationNodeChildList count="20">
                 <ClassificationNode>
                   <Disorder id="11651">
                     <OrphaCode>85193</OrphaCode>
@@ -3080,18 +3068,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="12411">
-                    <OrphaCode>93569</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93569</ExpertLink>
-                    <Name lang="en">Polymyalgia rheumatica</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="12458">
                     <OrphaCode>93665</OrphaCode>
@@ -3256,7 +3232,7 @@
                                   <Disorder id="31819">
                                     <OrphaCode>647815</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647815</ExpertLink>
-                                    <Name lang="en">Keratoendotheliitis fugax hereditaria</Name>
+                                    <Name lang="en">Keratitis fugax hereditaria</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -3680,7 +3656,7 @@
                           <Disorder id="31418">
                             <OrphaCode>619948</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619948</ExpertLink>
-                            <Name lang="en">Early-onset autoimmunity-autoinflammation-immunodeficiency syndrome</Name>
+                            <Name lang="en">Early-onset autoimmunity-autoinflammation-immunodeficiency syndrome due to SOCS1 haploinsufficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3692,7 +3668,7 @@
                           <Disorder id="31951">
                             <OrphaCode>658946</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658946</ExpertLink>
-                            <Name lang="en">Early-onset autoimmune disorder due to DOCK11 partial deficiency</Name>
+                            <Name lang="en">Early-onset immune dysregulation with autoimmunity due to DOCK11 partial deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3747,7 +3723,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="7">
+                      <ClassificationNodeChildList count="8">
                         <ClassificationNode>
                           <Disorder id="703">
                             <OrphaCode>117</OrphaCode>
@@ -3785,10 +3761,22 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="25087">
-                            <OrphaCode>476102</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=476102</ExpertLink>
-                            <Name lang="en">Hereditary pediatric Behçet-like disease</Name>
+                          <Disorder id="996">
+                            <OrphaCode>184</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=184</ExpertLink>
+                            <Name lang="en">Cherubism</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32094">
+                            <OrphaCode>674762</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=674762</ExpertLink>
+                            <Name lang="en">Early-onset autoinflammatory syndrome due to A20 haploinsufficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3797,12 +3785,12 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="996">
-                            <OrphaCode>184</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=184</ExpertLink>
-                            <Name lang="en">Cherubism</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
+                          <Disorder id="32142">
+                            <OrphaCode>676125</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=676125</ExpertLink>
+                            <Name lang="en">X-linked immune dysregulation with inflammatory bowel disease due to ELF4 deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -3812,7 +3800,7 @@
                           <Disorder id="31415">
                             <OrphaCode>619363</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619363</ExpertLink>
-                            <Name lang="en">Neonatal-onset severe multisystemic autoinflammatory disease with increased IL18</Name>
+                            <Name lang="en">NOCARH syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3956,7 +3944,7 @@
                           <Disorder id="22938">
                             <OrphaCode>404553</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=404553</ExpertLink>
-                            <Name lang="en">Vasculitis due to ADA2 deficiency</Name>
+                            <Name lang="en">Adenosine deaminase 2 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
