commit cb98a4638b303dbc159c97a636d6d4f62ca39678
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 6 12:14:41 2022 +0100

    DEC 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml
index 6e54986..89ecdd0 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:15:36" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:35:17" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
