commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml
index bd726fe..4072fc9 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:04:51" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:23:26" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2182,7 +2182,7 @@
                       <Disorder id="3619">
                         <OrphaCode>2841</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2841</ExpertLink>
-                        <Name lang="en">Familial benign chronic pemphigus</Name>
+                        <Name lang="en">Hailey-Hailey disease</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -4196,7 +4196,7 @@
                           <Disorder id="18818">
                             <OrphaCode>217059</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217059</ExpertLink>
-                            <Name lang="en">Isolated congenital digital clubbing</Name>
+                            <Name lang="en">Isolated nail clubbing</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -5434,7 +5434,7 @@
                       <Disorder id="2962">
                         <OrphaCode>3339</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3339</ExpertLink>
-                        <Name lang="en">Toriello-Lacassie-Droste syndrome</Name>
+                        <Name lang="en">Oculoectodermal syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -6125,7 +6125,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="36">
+                  <ClassificationNodeChildList count="37">
                     <ClassificationNode>
                       <Disorder id="26573">
                         <OrphaCode>508512</OrphaCode>
@@ -6138,6 +6138,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="11724">
+                        <OrphaCode>85453</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85453</ExpertLink>
+                        <Name lang="en">X-linked reticulate pigmentary disorder</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="185">
                         <OrphaCode>636</OrphaCode>
@@ -7043,7 +7055,31 @@
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="0">
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="32042">
+                                    <OrphaCode>664500</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664500</ExpertLink>
+                                    <Name lang="en">Hermansky-Pudlak syndrome due to AP3B1 deficiency</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="32043">
+                                    <OrphaCode>664511</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664511</ExpertLink>
+                                    <Name lang="en">Early-onset severe Hermansky-Pudlak syndrome with hearing loss, due to AP3D1 deficiency</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -7791,7 +7827,19 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
+                      <ClassificationNodeChildList count="3">
+                        <ClassificationNode>
+                          <Disorder id="32006">
+                            <OrphaCode>662255</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662255</ExpertLink>
+                            <Name lang="en">Grisel syndrome</Name>
+                            <DisorderType id="21422">
+                              <Name lang="en">Clinical syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="19084">
                             <OrphaCode>228221</OrphaCode>
@@ -9567,7 +9615,19 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="5">
+                        <ClassificationNode>
+                          <Disorder id="20949">
+                            <OrphaCode>295000</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295000</ExpertLink>
+                            <Name lang="en">Amniotic band syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2336">
                             <OrphaCode>2526</OrphaCode>
@@ -9627,19 +9687,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="7">
-                        <ClassificationNode>
-                          <Disorder id="2883">
-                            <OrphaCode>3226</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3226</ExpertLink>
-                            <Name lang="en">Deafness-lymphedema-leukemia syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
                           <Disorder id="28610">
                             <OrphaCode>568051</OrphaCode>
@@ -9886,7 +9934,7 @@
                       <Disorder id="25364">
                         <OrphaCode>487796</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=487796</ExpertLink>
-                        <Name lang="en">Macrothrombocytopenia-lymphedema-developmental delay-facial dysmorphism-camptodactyly syndrome</Name>
+                        <Name lang="en">Takenouchi-Kosaki syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -9898,12 +9946,36 @@
                       <Disorder id="10630">
                         <OrphaCode>48652</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=48652</ExpertLink>
-                        <Name lang="en">Monosomy 22q13.3</Name>
+                        <Name lang="en">Phelan-McDermid syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31993">
+                            <OrphaCode>662172</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662172</ExpertLink>
+                            <Name lang="en">Phelan-McDermid syndrome due to SHANK3 mutation</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31992">
+                            <OrphaCode>662169</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662169</ExpertLink>
+                            <Name lang="en">Phelan-McDermid syndrome due to 22q13.3 deletion</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -10183,7 +10255,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="36">
+              <ClassificationNodeChildList count="38">
                 <ClassificationNode>
                   <Disorder id="185">
                     <OrphaCode>636</OrphaCode>
@@ -11456,6 +11528,30 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32086">
+                    <OrphaCode>673568</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673568</ExpertLink>
+                    <Name lang="en">Eccrine angiomatous hamartoma</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32080">
+                    <OrphaCode>673525</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673525</ExpertLink>
+                    <Name lang="en">Intravascular papillary endothelial hyperplasia</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
@@ -11477,46 +11573,22 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="6">
-                    <ClassificationNode>
-                      <Disorder id="11303">
-                        <OrphaCode>79277</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79277</ExpertLink>
-                        <Name lang="en">Congenital erythropoietic porphyria</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="11304">
-                        <OrphaCode>79278</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79278</ExpertLink>
-                        <Name lang="en">Autosomal erythropoietic protoporphyria</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="2">
                     <ClassificationNode>
-                      <Disorder id="12578">
-                        <OrphaCode>95157</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95157</ExpertLink>
-                        <Name lang="en">Acute hepatic porphyria</Name>
+                      <Disorder id="31964">
+                        <OrphaCode>659681</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659681</ExpertLink>
+                        <Name lang="en">Erythropoietic porphyria</Name>
                         <DisorderType id="21436">
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
-                          <Disorder id="11299">
-                            <OrphaCode>79273</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79273</ExpertLink>
-                            <Name lang="en">Hereditary coproporphyria</Name>
+                          <Disorder id="11303">
+                            <OrphaCode>79277</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79277</ExpertLink>
+                            <Name lang="en">Congenital erythropoietic porphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -11525,10 +11597,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="11302">
-                            <OrphaCode>79276</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79276</ExpertLink>
-                            <Name lang="en">Acute intermittent porphyria</Name>
+                          <Disorder id="11304">
+                            <OrphaCode>79278</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79278</ExpertLink>
+                            <Name lang="en">Autosomal erythropoietic protoporphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -11537,10 +11609,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="11499">
-                            <OrphaCode>79473</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79473</ExpertLink>
-                            <Name lang="en">Porphyria variegata</Name>
+                          <Disorder id="23451">
+                            <OrphaCode>443197</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443197</ExpertLink>
+                            <Name lang="en">X-linked erythropoietic protoporphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -11549,10 +11621,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="14678">
-                            <OrphaCode>100924</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100924</ExpertLink>
-                            <Name lang="en">Porphyria due to ALA dehydratase deficiency</Name>
+                          <Disorder id="20463">
+                            <OrphaCode>280379</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280379</ExpertLink>
+                            <Name lang="en">Erythropoietic uroporphyria associated with myeloid malignancy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -11560,18 +11632,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="12581">
-                        <OrphaCode>95161</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95161</ExpertLink>
-                        <Name lang="en">Chronic hepatic porphyria</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
                           <Disorder id="12580">
                             <OrphaCode>95159</OrphaCode>
@@ -11585,34 +11645,82 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="14841">
-                            <OrphaCode>101330</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=101330</ExpertLink>
-                            <Name lang="en">Porphyria cutanea tarda</Name>
+                          <Disorder id="31963">
+                            <OrphaCode>659672</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659672</ExpertLink>
+                            <Name lang="en">Harderoporphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="2">
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31965">
+                        <OrphaCode>659694</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659694</ExpertLink>
+                        <Name lang="en">Hepatic porphyria</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="12578">
+                            <OrphaCode>95157</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95157</ExpertLink>
+                            <Name lang="en">Acute hepatic porphyria</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="4">
                             <ClassificationNode>
-                              <Disorder id="23434">
-                                <OrphaCode>443057</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443057</ExpertLink>
-                                <Name lang="en">Sporadic porphyria cutanea tarda</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                              <Disorder id="11299">
+                                <OrphaCode>79273</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79273</ExpertLink>
+                                <Name lang="en">Hereditary coproporphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
-                              <Disorder id="23435">
-                                <OrphaCode>443062</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443062</ExpertLink>
-                                <Name lang="en">Familial porphyria cutanea tarda</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                              <Disorder id="11302">
+                                <OrphaCode>79276</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79276</ExpertLink>
+                                <Name lang="en">Acute intermittent porphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="11499">
+                                <OrphaCode>79473</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79473</ExpertLink>
+                                <Name lang="en">Variegate porphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="14678">
+                                <OrphaCode>100924</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100924</ExpertLink>
+                                <Name lang="en">Porphyria due to ALA dehydratase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -11620,30 +11728,54 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="20463">
-                        <OrphaCode>280379</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280379</ExpertLink>
-                        <Name lang="en">Erythropoietic uroporphyria associated with myeloid malignancy</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="23451">
-                        <OrphaCode>443197</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443197</ExpertLink>
-                        <Name lang="en">X-linked erythropoietic protoporphyria</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
+                        <ClassificationNode>
+                          <Disorder id="31966">
+                            <OrphaCode>659698</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659698</ExpertLink>
+                            <Name lang="en">Hepatic cutaneous porphyria</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="1">
+                            <ClassificationNode>
+                              <Disorder id="14841">
+                                <OrphaCode>101330</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=101330</ExpertLink>
+                                <Name lang="en">Porphyria cutanea tarda</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="23434">
+                                    <OrphaCode>443057</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443057</ExpertLink>
+                                    <Name lang="en">Sporadic porphyria cutanea tarda</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="23435">
+                                    <OrphaCode>443062</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443062</ExpertLink>
+                                    <Name lang="en">Familial porphyria cutanea tarda</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -12377,46 +12509,22 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="6">
-                    <ClassificationNode>
-                      <Disorder id="11303">
-                        <OrphaCode>79277</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79277</ExpertLink>
-                        <Name lang="en">Congenital erythropoietic porphyria</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="11304">
-                        <OrphaCode>79278</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79278</ExpertLink>
-                        <Name lang="en">Autosomal erythropoietic protoporphyria</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="2">
                     <ClassificationNode>
-                      <Disorder id="12578">
-                        <OrphaCode>95157</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95157</ExpertLink>
-                        <Name lang="en">Acute hepatic porphyria</Name>
+                      <Disorder id="31964">
+                        <OrphaCode>659681</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659681</ExpertLink>
+                        <Name lang="en">Erythropoietic porphyria</Name>
                         <DisorderType id="21436">
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
-                          <Disorder id="11299">
-                            <OrphaCode>79273</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79273</ExpertLink>
-                            <Name lang="en">Hereditary coproporphyria</Name>
+                          <Disorder id="11303">
+                            <OrphaCode>79277</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79277</ExpertLink>
+                            <Name lang="en">Congenital erythropoietic porphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -12425,10 +12533,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="11302">
-                            <OrphaCode>79276</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79276</ExpertLink>
-                            <Name lang="en">Acute intermittent porphyria</Name>
+                          <Disorder id="11304">
+                            <OrphaCode>79278</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79278</ExpertLink>
+                            <Name lang="en">Autosomal erythropoietic protoporphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -12437,10 +12545,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="11499">
-                            <OrphaCode>79473</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79473</ExpertLink>
-                            <Name lang="en">Porphyria variegata</Name>
+                          <Disorder id="23451">
+                            <OrphaCode>443197</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443197</ExpertLink>
+                            <Name lang="en">X-linked erythropoietic protoporphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -12449,10 +12557,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="14678">
-                            <OrphaCode>100924</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100924</ExpertLink>
-                            <Name lang="en">Porphyria due to ALA dehydratase deficiency</Name>
+                          <Disorder id="20463">
+                            <OrphaCode>280379</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280379</ExpertLink>
+                            <Name lang="en">Erythropoietic uroporphyria associated with myeloid malignancy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -12460,18 +12568,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="12581">
-                        <OrphaCode>95161</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95161</ExpertLink>
-                        <Name lang="en">Chronic hepatic porphyria</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
                           <Disorder id="12580">
                             <OrphaCode>95159</OrphaCode>
@@ -12485,34 +12581,82 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="14841">
-                            <OrphaCode>101330</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=101330</ExpertLink>
-                            <Name lang="en">Porphyria cutanea tarda</Name>
+                          <Disorder id="31963">
+                            <OrphaCode>659672</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659672</ExpertLink>
+                            <Name lang="en">Harderoporphyria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="2">
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31965">
+                        <OrphaCode>659694</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659694</ExpertLink>
+                        <Name lang="en">Hepatic porphyria</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="12578">
+                            <OrphaCode>95157</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95157</ExpertLink>
+                            <Name lang="en">Acute hepatic porphyria</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="4">
                             <ClassificationNode>
-                              <Disorder id="23434">
-                                <OrphaCode>443057</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443057</ExpertLink>
-                                <Name lang="en">Sporadic porphyria cutanea tarda</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                              <Disorder id="11299">
+                                <OrphaCode>79273</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79273</ExpertLink>
+                                <Name lang="en">Hereditary coproporphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
-                              <Disorder id="23435">
-                                <OrphaCode>443062</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443062</ExpertLink>
-                                <Name lang="en">Familial porphyria cutanea tarda</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                              <Disorder id="11302">
+                                <OrphaCode>79276</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79276</ExpertLink>
+                                <Name lang="en">Acute intermittent porphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="11499">
+                                <OrphaCode>79473</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79473</ExpertLink>
+                                <Name lang="en">Variegate porphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="14678">
+                                <OrphaCode>100924</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100924</ExpertLink>
+                                <Name lang="en">Porphyria due to ALA dehydratase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -12520,30 +12664,54 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="20463">
-                        <OrphaCode>280379</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280379</ExpertLink>
-                        <Name lang="en">Erythropoietic uroporphyria associated with myeloid malignancy</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="23451">
-                        <OrphaCode>443197</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443197</ExpertLink>
-                        <Name lang="en">X-linked erythropoietic protoporphyria</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
+                        <ClassificationNode>
+                          <Disorder id="31966">
+                            <OrphaCode>659698</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659698</ExpertLink>
+                            <Name lang="en">Hepatic cutaneous porphyria</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="1">
+                            <ClassificationNode>
+                              <Disorder id="14841">
+                                <OrphaCode>101330</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=101330</ExpertLink>
+                                <Name lang="en">Porphyria cutanea tarda</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="23434">
+                                    <OrphaCode>443057</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443057</ExpertLink>
+                                    <Name lang="en">Sporadic porphyria cutanea tarda</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="23435">
+                                    <OrphaCode>443062</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443062</ExpertLink>
+                                    <Name lang="en">Familial porphyria cutanea tarda</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -12684,7 +12852,7 @@
                   <Disorder id="31951">
                     <OrphaCode>658946</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658946</ExpertLink>
-                    <Name lang="en">Early-onset autoimmune disorder due to DOCK11 partial deficiency</Name>
+                    <Name lang="en">Early-onset immune dysregulation with autoimmunity due to DOCK11 partial deficiency</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -12804,7 +12972,7 @@
                   <Disorder id="8531">
                     <OrphaCode>302</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=302</ExpertLink>
-                    <Name lang="en">Epidermodysplasia verruciformis</Name>
+                    <Name lang="en">Inherited epidermodysplasia verruciformis</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -12871,7 +13039,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="26">
+              <ClassificationNodeChildList count="28">
                 <ClassificationNode>
                   <Disorder id="12540">
                     <OrphaCode>94059</OrphaCode>
@@ -12996,7 +13164,7 @@
                   <Disorder id="10749">
                     <OrphaCode>54251</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=54251</ExpertLink>
-                    <Name lang="en">Corticosteroid-sensitive aseptic abscess syndrome</Name>
+                    <Name lang="en">Aseptic abscess syndrome</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -13112,6 +13280,30 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32056">
+                    <OrphaCode>664787</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664787</ExpertLink>
+                    <Name lang="en">Nicolau syndrome</Name>
+                    <DisorderType id="21422">
+                      <Name lang="en">Clinical syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32057">
+                    <OrphaCode>664901</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664901</ExpertLink>
+                    <Name lang="en">Trigeminal trophic syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31319">
                     <OrphaCode>615943</OrphaCode>
@@ -13637,7 +13829,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="24">
+                  <ClassificationNodeChildList count="25">
                     <ClassificationNode>
                       <Disorder id="19644">
                         <OrphaCode>251523</OrphaCode>
@@ -13734,6 +13926,18 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32094">
+                        <OrphaCode>674762</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=674762</ExpertLink>
+                        <Name lang="en">Early-onset autoinflammatory syndrome due to A20 haploinsufficiency</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="3057">
                         <OrphaCode>3243</OrphaCode>
@@ -13904,7 +14108,7 @@
                           <Disorder id="31819">
                             <OrphaCode>647815</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647815</ExpertLink>
-                            <Name lang="en">Keratoendotheliitis fugax hereditaria</Name>
+                            <Name lang="en">Keratitis fugax hereditaria</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -14086,7 +14290,7 @@
                       <Disorder id="31415">
                         <OrphaCode>619363</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619363</ExpertLink>
-                        <Name lang="en">Neonatal-onset severe multisystemic autoinflammatory disease with increased IL18</Name>
+                        <Name lang="en">NOCARH syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
