commit 481d14fea17f62fcae7251f3673242ec271b28fa
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 4 20:10:24 2023 +0100

    DEC 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml
index 06c35da..e72c590 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-06-22 14:01:43" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-12-04 08:06:20" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -984,7 +984,7 @@
                   <Disorder id="30484">
                     <OrphaCode>600832</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600832</ExpertLink>
-                    <Name lang="en">Legionellosis</Name>
+                    <Name lang="en">Legionella infection</Name>
                     <DisorderType id="21436">
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
@@ -1885,11 +1885,47 @@
                     <OrphaCode>1685</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1685</ExpertLink>
                     <Name lang="en">Distomatosis</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
+                    <DisorderType id="21436">
+                      <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="0">
+                  <ClassificationNodeChildList count="3">
+                    <ClassificationNode>
+                      <Disorder id="31948">
+                        <OrphaCode>658909</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658909</ExpertLink>
+                        <Name lang="en">Fasciolopsiasis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31949">
+                        <OrphaCode>658913</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658913</ExpertLink>
+                        <Name lang="en">Paragonimiasis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31950">
+                        <OrphaCode>658917</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658917</ExpertLink>
+                        <Name lang="en">Clonorchiasis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
