commit df4bbec30a36ad0ab0af6d3588a36df3686578ff
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sun Oct 3 10:11:04 2021 +0200

    OCT 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml
old mode 100755
new mode 100644
index f3b7e1e..d2b6e0f
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-09-01 04:39:57" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-10-01 04:40:11" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -1299,7 +1299,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="8">
+                      <ClassificationNodeChildList count="7">
                         <ClassificationNode>
                           <Disorder id="515">
                             <OrphaCode>2300</OrphaCode>
@@ -1348,18 +1348,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="8710">
-                            <OrphaCode>620</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620</ExpertLink>
-                            <Name lang="en">NON RARE IN EUROPE: Common mesentery</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="516">
                             <OrphaCode>2301</OrphaCode>
