commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml
index f26c425..35810d7 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:24:27" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:08:57" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -934,7 +934,7 @@
                       <Disorder id="1423">
                         <OrphaCode>1159</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1159</ExpertLink>
-                        <Name lang="en">Progressive pseudorheumatoid arthropathy of childhood</Name>
+                        <Name lang="en">Progressive pseudorheumatoid dysplasia</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -1642,7 +1642,7 @@
                       <Disorder id="1423">
                         <OrphaCode>1159</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1159</ExpertLink>
-                        <Name lang="en">Progressive pseudorheumatoid arthropathy of childhood</Name>
+                        <Name lang="en">Progressive pseudorheumatoid dysplasia</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -1654,7 +1654,7 @@
                       <Disorder id="1633">
                         <OrphaCode>1427</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1427</ExpertLink>
-                        <Name lang="en">Otospondylomegaepiphyseal dysplasia</Name>
+                        <Name lang="en">Autosomal recessive otospondylomegaepiphyseal dysplasia</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -5333,7 +5333,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="26">
+                  <ClassificationNodeChildList count="25">
                     <ClassificationNode>
                       <Disorder id="313">
                         <OrphaCode>2771</OrphaCode>
@@ -5502,18 +5502,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="2535">
-                        <OrphaCode>2787</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2787</ExpertLink>
-                        <Name lang="en">Osteoporosis-macrocephaly-blindness-joint hyperlaxity syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="2536">
                         <OrphaCode>2788</OrphaCode>
@@ -8909,7 +8897,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="19">
+                  <ClassificationNodeChildList count="18">
                     <ClassificationNode>
                       <Disorder id="1042">
                         <OrphaCode>2311</OrphaCode>
@@ -9090,18 +9078,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="10656">
-                        <OrphaCode>50817</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=50817</ExpertLink>
-                        <Name lang="en">Duane anomaly-myopathy-scoliosis syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="23556">
                         <OrphaCode>447974</OrphaCode>
@@ -11847,19 +11823,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="7">
-                        <ClassificationNode>
-                          <Disorder id="1335">
-                            <OrphaCode>1027</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1027</ExpertLink>
-                            <Name lang="en">Autosomal recessive amelia</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
                           <Disorder id="1399">
                             <OrphaCode>1118</OrphaCode>
