commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml
index c3e7588..4ec3810 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:05:24" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:23:59" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -31,7 +31,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="6">
+              <ClassificationNodeChildList count="5">
                 <ClassificationNode>
                   <Disorder id="430">
                     <OrphaCode>125</OrphaCode>
@@ -44,18 +44,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="2883">
-                    <OrphaCode>3226</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3226</ExpertLink>
-                    <Name lang="en">Deafness-lymphedema-leukemia syndrome</Name>
-                    <DisorderType id="21401">
-                      <Name lang="en">Malformation syndrome</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="13309">
                     <OrphaCode>98292</OrphaCode>
@@ -647,12 +635,12 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="2">
+                              <ClassificationNodeChildList count="3">
                                 <ClassificationNode>
                                   <Disorder id="14460">
                                     <OrphaCode>99887</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99887</ExpertLink>
-                                    <Name lang="en">Acute megakaryoblastic leukemia in Down syndrome</Name>
+                                    <Name lang="en">Acute megakaryoblastic leukemia in children with Down syndrome</Name>
                                     <DisorderType id="21450">
                                       <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
@@ -664,7 +652,19 @@
                                   <Disorder id="21935">
                                     <OrphaCode>329469</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329469</ExpertLink>
-                                    <Name lang="en">Acute megakaryoblastic leukemia without Down syndrome</Name>
+                                    <Name lang="en">Acute megakaryoblastic leukemia in children without Down syndrome</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="32019">
+                                    <OrphaCode>662934</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662934</ExpertLink>
+                                    <Name lang="en">Acute megakaryoblastic leukemia in adult</Name>
                                     <DisorderType id="21450">
                                       <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
@@ -817,12 +817,12 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="3">
                             <ClassificationNode>
                               <Disorder id="13843">
                                 <OrphaCode>98826</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98826</ExpertLink>
-                                <Name lang="en">Refractory anemia</Name>
+                                <Name lang="en">Myelodysplastic neoplasm with low blasts</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -842,18 +842,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="14807">
-                                <OrphaCode>101096</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=101096</ExpertLink>
-                                <Name lang="en">Aregenerative anemia</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="26578">
                                 <OrphaCode>508542</OrphaCode>
@@ -872,7 +860,7 @@
                           <Disorder id="11749">
                             <OrphaCode>86839</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86839</ExpertLink>
-                            <Name lang="en">Refractory anemia with excess blasts</Name>
+                            <Name lang="en">Myelodysplastic neoplasm with increased blasts</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -882,7 +870,7 @@
                               <Disorder id="14592">
                                 <OrphaCode>100019</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100019</ExpertLink>
-                                <Name lang="en">Refractory anemia with excess blasts type 1</Name>
+                                <Name lang="en">Myelodysplastic neoplasm with increased blasts type 1</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -894,7 +882,7 @@
                               <Disorder id="14593">
                                 <OrphaCode>100020</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100020</ExpertLink>
-                                <Name lang="en">Refractory anemia with excess blasts type 2</Name>
+                                <Name lang="en">Myelodysplastic neoplasm with increased blasts type 2</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -951,7 +939,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="11">
+                      <ClassificationNodeChildList count="12">
                         <ClassificationNode>
                           <Disorder id="25397">
                             <OrphaCode>488280</OrphaCode>
@@ -1132,6 +1120,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32128">
+                            <OrphaCode>675775</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675775</ExpertLink>
+                            <Name lang="en">Severe congenital myelofibrosis-pancytopenia-intellectual disability-neurologic and ophthalmic abnormalities syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -1611,7 +1611,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
                         <ClassificationNode>
                           <Disorder id="10952">
                             <OrphaCode>70568</OrphaCode>
@@ -1648,6 +1648,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32119">
+                            <OrphaCode>675628</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675628</ExpertLink>
+                            <Name lang="en">TLR8-related inflammation-severe neutropenia-bone marrow failure-lymphoproliferation syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -2088,7 +2100,7 @@
                                           <Disorder id="20756">
                                             <OrphaCode>289661</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=289661</ExpertLink>
-                                            <Name lang="en">Epstein-Barr virus-positive diffuse large B-cell lymphoma of the elderly</Name>
+                                            <Name lang="en">Epstein-Barr virus-positive diffuse large B-cell lymphoma</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -2243,7 +2255,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="13">
+                              <ClassificationNodeChildList count="14">
                                 <ClassificationNode>
                                   <Disorder id="10952">
                                     <OrphaCode>70568</OrphaCode>
@@ -2604,6 +2616,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="32072">
+                                    <OrphaCode>667662</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=667662</ExpertLink>
+                                    <Name lang="en">Breast implant-associated anaplastic large cell lymphoma</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="31900">
                                     <OrphaCode>652658</OrphaCode>
@@ -4509,7 +4533,19 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="14">
+                                  <ClassificationNodeChildList count="15">
+                                    <ClassificationNode>
+                                      <Disorder id="32092">
+                                        <OrphaCode>674653</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=674653</ExpertLink>
+                                        <Name lang="en">Actinomyopathy-associated syndromic thrombocytopenia</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="31896">
                                         <OrphaCode>652522</OrphaCode>
@@ -4695,7 +4731,31 @@
                                                   <Name lang="en">Clinical subtype</Name>
                                                 </DisorderType>
                                               </Disorder>
-                                              <ClassificationNodeChildList count="0">
+                                              <ClassificationNodeChildList count="2">
+                                                <ClassificationNode>
+                                                  <Disorder id="32042">
+                                                    <OrphaCode>664500</OrphaCode>
+                                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664500</ExpertLink>
+                                                    <Name lang="en">Hermansky-Pudlak syndrome due to AP3B1 deficiency</Name>
+                                                    <DisorderType id="21450">
+                                                      <Name lang="en">Clinical subtype</Name>
+                                                    </DisorderType>
+                                                  </Disorder>
+                                                  <ClassificationNodeChildList count="0">
+                                                  </ClassificationNodeChildList>
+                                                </ClassificationNode>
+                                                <ClassificationNode>
+                                                  <Disorder id="32043">
+                                                    <OrphaCode>664511</OrphaCode>
+                                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664511</ExpertLink>
+                                                    <Name lang="en">Early-onset severe Hermansky-Pudlak syndrome with hearing loss, due to AP3D1 deficiency</Name>
+                                                    <DisorderType id="21450">
+                                                      <Name lang="en">Clinical subtype</Name>
+                                                    </DisorderType>
+                                                  </Disorder>
+                                                  <ClassificationNodeChildList count="0">
+                                                  </ClassificationNodeChildList>
+                                                </ClassificationNode>
                                               </ClassificationNodeChildList>
                                             </ClassificationNode>
                                             <ClassificationNode>
@@ -4742,7 +4802,7 @@
                                       <Disorder id="25364">
                                         <OrphaCode>487796</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=487796</ExpertLink>
-                                        <Name lang="en">Macrothrombocytopenia-lymphedema-developmental delay-facial dysmorphism-camptodactyly syndrome</Name>
+                                        <Name lang="en">Takenouchi-Kosaki syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -5550,7 +5610,7 @@
                               <Disorder id="900">
                                 <OrphaCode>847</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=847</ExpertLink>
-                                <Name lang="en">Alpha-thalassemia-X-linked intellectual disability syndrome</Name>
+                                <Name lang="en">X-linked alpha-thalassemia-intellectual disability syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -6241,7 +6301,19 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="6">
+                            <ClassificationNode>
+                              <Disorder id="31963">
+                                <OrphaCode>659672</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659672</ExpertLink>
+                                <Name lang="en">Harderoporphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="11303">
                                 <OrphaCode>79277</OrphaCode>
@@ -6506,6 +6578,18 @@
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="11304">
+                                <OrphaCode>79278</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79278</ExpertLink>
+                                <Name lang="en">Autosomal erythropoietic protoporphyria</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -7225,7 +7309,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="5">
                             <ClassificationNode>
                               <Disorder id="10673">
                                 <OrphaCode>51208</OrphaCode>
@@ -7266,7 +7350,19 @@
                               <Disorder id="31946">
                                 <OrphaCode>658813</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658813</ExpertLink>
-                                <Name lang="en">Methylenetetrahydrofolate dehydrogenase 1 deficiency</Name>
+                                <Name lang="en">Combined immunodeficiency-megaloblastic anemia due to methylenetetrahydrofolate dehydrogenase 1 deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31990">
+                                <OrphaCode>661412</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=661412</ExpertLink>
+                                <Name lang="en">Megaloblastic anemia-immunodeficiency due to folate transporter 1 deficiency</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
