commit 481d14fea17f62fcae7251f3673242ec271b28fa
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 4 20:10:24 2023 +0100

    DEC 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml
index 522cdef..0f84d59 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-06-22 14:00:12" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-12-04 08:05:06" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -31,7 +31,19 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="11">
+              <ClassificationNodeChildList count="12">
+                <ClassificationNode>
+                  <Disorder id="31939">
+                    <OrphaCode>658590</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658590</ExpertLink>
+                    <Name lang="en">Eyelid sebaceous carcinoma</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="122">
                     <OrphaCode>790</OrphaCode>
@@ -5925,7 +5937,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="14">
+                                  <ClassificationNodeChildList count="15">
                                     <ClassificationNode>
                                       <Disorder id="3243">
                                         <OrphaCode>65</OrphaCode>
@@ -6226,6 +6238,18 @@
                                         </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31907">
+                                        <OrphaCode>653709</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653709</ExpertLink>
+                                        <Name lang="en">Cone rod dystrophy-short stature syndrome</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="659">
                                         <OrphaCode>791</OrphaCode>
@@ -6309,12 +6333,12 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="4">
+                                  <ClassificationNodeChildList count="3">
                                     <ClassificationNode>
                                       <Disorder id="21356">
                                         <OrphaCode>309178</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=309178</ExpertLink>
-                                        <Name lang="en">Tay-Sachs disease, B variant, infantile form</Name>
+                                        <Name lang="en">Tay-Sachs disease, infantile form</Name>
                                         <DisorderType id="21450">
                                           <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
@@ -6326,19 +6350,7 @@
                                       <Disorder id="21358">
                                         <OrphaCode>309192</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=309192</ExpertLink>
-                                        <Name lang="en">Tay-Sachs disease, B variant, adult form</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="21359">
-                                        <OrphaCode>309239</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=309239</ExpertLink>
-                                        <Name lang="en">Tay-Sachs disease, B1 variant</Name>
+                                        <Name lang="en">Tay-Sachs disease, adult form</Name>
                                         <DisorderType id="21450">
                                           <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
@@ -6350,7 +6362,7 @@
                                       <Disorder id="21357">
                                         <OrphaCode>309185</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=309185</ExpertLink>
-                                        <Name lang="en">Tay-Sachs disease, B variant, juvenile form</Name>
+                                        <Name lang="en">Tay-Sachs disease, juvenile form</Name>
                                         <DisorderType id="21450">
                                           <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
@@ -6417,91 +6429,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="7">
-                                    <ClassificationNode>
-                                      <Disorder id="19107">
-                                        <OrphaCode>228340</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228340</ExpertLink>
-                                        <Name lang="en">CLN4A disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19114">
-                                        <OrphaCode>228363</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228363</ExpertLink>
-                                        <Name lang="en">CLN6 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19106">
-                                        <OrphaCode>228337</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228337</ExpertLink>
-                                        <Name lang="en">CLN10 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19108">
-                                        <OrphaCode>228343</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228343</ExpertLink>
-                                        <Name lang="en">CLN4B disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="22104">
-                                        <OrphaCode>352709</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352709</ExpertLink>
-                                        <Name lang="en">CLN13 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19105">
-                                        <OrphaCode>228329</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228329</ExpertLink>
-                                        <Name lang="en">CLN1 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="21505">
-                                        <OrphaCode>314629</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314629</ExpertLink>
-                                        <Name lang="en">CLN11 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -6513,19 +6441,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="2">
-                                    <ClassificationNode>
-                                      <Disorder id="19105">
-                                        <OrphaCode>228329</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228329</ExpertLink>
-                                        <Name lang="en">CLN1 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="1">
                                     <ClassificationNode>
                                       <Disorder id="20066">
                                         <OrphaCode>263516</OrphaCode>
@@ -6549,19 +6465,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="1">
-                                    <ClassificationNode>
-                                      <Disorder id="19106">
-                                        <OrphaCode>228337</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228337</ExpertLink>
-                                        <Name lang="en">CLN10 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -6573,79 +6477,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="6">
-                                    <ClassificationNode>
-                                      <Disorder id="19106">
-                                        <OrphaCode>228337</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228337</ExpertLink>
-                                        <Name lang="en">CLN10 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19110">
-                                        <OrphaCode>228349</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228349</ExpertLink>
-                                        <Name lang="en">CLN2 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19111">
-                                        <OrphaCode>228354</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228354</ExpertLink>
-                                        <Name lang="en">CLN8 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19105">
-                                        <OrphaCode>228329</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228329</ExpertLink>
-                                        <Name lang="en">CLN1 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19109">
-                                        <OrphaCode>228346</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228346</ExpertLink>
-                                        <Name lang="en">CLN3 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19112">
-                                        <OrphaCode>228357</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228357</ExpertLink>
-                                        <Name lang="en">CLN9 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -6657,91 +6489,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="7">
-                                    <ClassificationNode>
-                                      <Disorder id="19105">
-                                        <OrphaCode>228329</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228329</ExpertLink>
-                                        <Name lang="en">CLN1 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19113">
-                                        <OrphaCode>228360</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228360</ExpertLink>
-                                        <Name lang="en">CLN5 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19111">
-                                        <OrphaCode>228354</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228354</ExpertLink>
-                                        <Name lang="en">CLN8 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19115">
-                                        <OrphaCode>228366</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228366</ExpertLink>
-                                        <Name lang="en">CLN7 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19106">
-                                        <OrphaCode>228337</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228337</ExpertLink>
-                                        <Name lang="en">CLN10 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19110">
-                                        <OrphaCode>228349</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228349</ExpertLink>
-                                        <Name lang="en">CLN2 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="19114">
-                                        <OrphaCode>228363</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228363</ExpertLink>
-                                        <Name lang="en">CLN6 disease</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -9304,7 +9052,7 @@
                                   <Disorder id="173">
                                     <OrphaCode>394</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=394</ExpertLink>
-                                    <Name lang="en">Classic homocystinuria</Name>
+                                    <Name lang="en">Homocystinuria due to cystathionine beta-synthase deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -12184,7 +11932,7 @@
                                   <Disorder id="26077">
                                     <OrphaCode>502430</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=502430</ExpertLink>
-                                    <Name lang="en">Metopic ridging-ptosis-facial dysmorphism syndrome</Name>
+                                    <Name lang="en">Weiss-Kruszka Syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -14051,7 +13799,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="121">
+                              <ClassificationNodeChildList count="120">
                                 <ClassificationNode>
                                   <Disorder id="190">
                                     <OrphaCode>649</OrphaCode>
@@ -15146,7 +14894,7 @@
                                       <Disorder id="173">
                                         <OrphaCode>394</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=394</ExpertLink>
-                                        <Name lang="en">Classic homocystinuria</Name>
+                                        <Name lang="en">Homocystinuria due to cystathionine beta-synthase deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -15199,7 +14947,7 @@
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
                                       </Disorder>
-                                      <ClassificationNodeChildList count="4">
+                                      <ClassificationNodeChildList count="5">
                                         <ClassificationNode>
                                           <Disorder id="11849">
                                             <OrphaCode>88917</OrphaCode>
@@ -15248,6 +14996,18 @@
                                           <ClassificationNodeChildList count="0">
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="31909">
+                                            <OrphaCode>653722</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653722</ExpertLink>
+                                            <Name lang="en">Digenic Alport syndrome</Name>
+                                            <DisorderType id="21450">
+                                              <Name lang="en">Clinical subtype</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
                                   </ClassificationNodeChildList>
@@ -15804,42 +15564,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="523">
-                                    <OrphaCode>2346</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
-                                    <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                                    <DisorderType id="21436">
-                                      <Name lang="en">Clinical group</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="2">
-                                    <ClassificationNode>
-                                      <Disorder id="12004">
-                                        <OrphaCode>90307</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
-                                        <Name lang="en">Parkes Weber syndrome</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="12005">
-                                        <OrphaCode>90308</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
-                                        <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="1363">
                                     <OrphaCode>1068</OrphaCode>
@@ -17104,7 +16828,7 @@
                                   <Disorder id="173">
                                     <OrphaCode>394</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=394</ExpertLink>
-                                    <Name lang="en">Classic homocystinuria</Name>
+                                    <Name lang="en">Homocystinuria due to cystathionine beta-synthase deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -17387,7 +17111,7 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="4">
+                              <ClassificationNodeChildList count="5">
                                 <ClassificationNode>
                                   <Disorder id="11849">
                                     <OrphaCode>88917</OrphaCode>
@@ -17436,6 +17160,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31909">
+                                    <OrphaCode>653722</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653722</ExpertLink>
+                                    <Name lang="en">Digenic Alport syndrome</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -17487,7 +17223,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="20">
+                      <ClassificationNodeChildList count="21">
                         <ClassificationNode>
                           <Disorder id="123">
                             <OrphaCode>534</OrphaCode>
@@ -17524,6 +17260,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="22924">
+                            <OrphaCode>404463</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=404463</ExpertLink>
+                            <Name lang="en">Multisystemic smooth muscle dysfunction syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="23657">
                             <OrphaCode>454718</OrphaCode>
