commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml
index 2a4b3b7..4a17be0 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:47:25" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:45:01" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -439,7 +439,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="63">
+              <ClassificationNodeChildList count="64">
                 <ClassificationNode>
                   <Disorder id="2481">
                     <OrphaCode>2720</OrphaCode>
@@ -1244,6 +1244,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="21494">
+                    <OrphaCode>314555</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314555</ExpertLink>
+                    <Name lang="en">Facial dysmorphism-ocular anomalies-osteopenia-intellectual disability-dental anomalies syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="29437">
                     <OrphaCode>589608</OrphaCode>
