commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml
index 6267c82..1133963 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:47:45" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:45:22" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -308,7 +308,7 @@
                           <Disorder id="18763">
                             <OrphaCode>213726</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213726</ExpertLink>
-                            <Name lang="en">Papillary carcinoma of the corpus uteri</Name>
+                            <Name lang="en">Serous carcinoma of the corpus uteri</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
