commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml
index bf006fd..8b84b1e 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:37:49" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:50:10" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -129,7 +129,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="3">
+                                  <ClassificationNodeChildList count="2">
                                     <ClassificationNode>
                                       <Disorder id="20070">
                                         <OrphaCode>263548</OrphaCode>
@@ -154,18 +154,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="20072">
-                                        <OrphaCode>263558</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263558</ExpertLink>
-                                        <Name lang="en">Peeling skin syndrome type C</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
@@ -220,7 +208,7 @@
                                   <Disorder id="11420">
                                     <OrphaCode>79394</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79394</ExpertLink>
-                                    <Name lang="en">Congenital non-bullous ichthyosiform erythroderma</Name>
+                                    <Name lang="en">Congenital ichthyosiform erythroderma</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -612,7 +600,7 @@
                                           <Disorder id="29044">
                                             <OrphaCode>583612</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583612</ExpertLink>
-                                            <Name lang="en">Neu-laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
+                                            <Name lang="en">Neu-Laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -624,7 +612,7 @@
                                           <Disorder id="29042">
                                             <OrphaCode>583602</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583602</ExpertLink>
-                                            <Name lang="en">Neu-laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
+                                            <Name lang="en">Neu-Laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -636,7 +624,7 @@
                                           <Disorder id="29043">
                                             <OrphaCode>583607</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583607</ExpertLink>
-                                            <Name lang="en">Neu-laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
+                                            <Name lang="en">Neu-Laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -744,7 +732,7 @@
                                           <Disorder id="29044">
                                             <OrphaCode>583612</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583612</ExpertLink>
-                                            <Name lang="en">Neu-laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
+                                            <Name lang="en">Neu-Laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -756,7 +744,7 @@
                                           <Disorder id="29042">
                                             <OrphaCode>583602</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583602</ExpertLink>
-                                            <Name lang="en">Neu-laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
+                                            <Name lang="en">Neu-Laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -768,7 +756,7 @@
                                           <Disorder id="29043">
                                             <OrphaCode>583607</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583607</ExpertLink>
-                                            <Name lang="en">Neu-laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
+                                            <Name lang="en">Neu-Laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -5830,7 +5818,7 @@
                       <Disorder id="19552">
                         <OrphaCode>247820</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=247820</ExpertLink>
-                        <Name lang="en">Ectodermal dysplasia-syndactyly syndrome</Name>
+                        <Name lang="en">Ectodermal dysplasia-pili torti-cutaneous syndactyly syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -5842,7 +5830,7 @@
                       <Disorder id="19553">
                         <OrphaCode>247827</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=247827</ExpertLink>
-                        <Name lang="en">Ectodermal dysplasia-cutaneous syndactyly syndrome</Name>
+                        <Name lang="en">Ectodermal dysplasia-hyperhidrosis-cutaneous syndactyly syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -5924,7 +5912,7 @@
                           <Disorder id="11101">
                             <OrphaCode>77258</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77258</ExpertLink>
-                            <Name lang="en">Trichorhinophalangeal syndrome type 1 and 3</Name>
+                            <Name lang="en">Trichorhinophalangeal syndrome type 1</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -6310,7 +6298,7 @@
                       <Disorder id="2446">
                         <OrphaCode>2678</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2678</ExpertLink>
-                        <Name lang="en">Neurofibromatosis type 6</Name>
+                        <Name lang="en">Familial isolated café-au-lait macules</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -7179,7 +7167,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="13">
+                      <ClassificationNodeChildList count="12">
                         <ClassificationNode>
                           <Disorder id="19145">
                             <OrphaCode>230857</OrphaCode>
@@ -7204,18 +7192,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2533">
-                            <OrphaCode>1306</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1306</ExpertLink>
-                            <Name lang="en">Buschke-Ollendorff syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="3569">
                             <OrphaCode>209</OrphaCode>
@@ -7501,7 +7477,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="15">
+                          <ClassificationNodeChildList count="16">
                             <ClassificationNode>
                               <Disorder id="612">
                                 <OrphaCode>287</OrphaCode>
@@ -7742,6 +7718,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31696">
+                                <OrphaCode>636941</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=636941</ExpertLink>
+                                <Name lang="en">Vascular Ehlers-Danlos-polymicrogyria syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -10027,7 +10015,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="4">
+              <ClassificationNodeChildList count="3">
                 <ClassificationNode>
                   <Disorder id="2296">
                     <OrphaCode>2483</OrphaCode>
@@ -10064,18 +10052,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="25555">
-                    <OrphaCode>493348</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=493348</ExpertLink>
-                    <Name lang="en">Vibratory angioedema</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
@@ -11789,7 +11765,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="12">
+                  <ClassificationNodeChildList count="11">
                     <ClassificationNode>
                       <Disorder id="3480">
                         <OrphaCode>2953</OrphaCode>
@@ -11874,18 +11850,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="12290">
-                        <OrphaCode>93359</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93359</ExpertLink>
-                        <Name lang="en">Spondyloepimetaphyseal dysplasia with joint laxity</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="21246">
                         <OrphaCode>306661</OrphaCode>
@@ -12631,7 +12595,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="15">
+              <ClassificationNodeChildList count="16">
                 <ClassificationNode>
                   <Disorder id="31422">
                     <OrphaCode>619972</OrphaCode>
@@ -12644,6 +12608,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31716">
+                    <OrphaCode>641368</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641368</ExpertLink>
+                    <Name lang="en">Autosomal recessive hyper-IgE syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="21115">
                     <OrphaCode>300359</OrphaCode>
@@ -12896,18 +12872,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="9286">
-                    <OrphaCode>31142</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31142</ExpertLink>
-                    <Name lang="en">Oral erosive lichen</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="10317">
                     <OrphaCode>33314</OrphaCode>
@@ -13064,6 +13028,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31795">
+                    <OrphaCode>645849</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645849</ExpertLink>
+                    <Name lang="en">Primary cutaneous tuberculosis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31319">
                     <OrphaCode>615943</OrphaCode>
@@ -13337,7 +13313,43 @@
                       <Name lang="en">Disease</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="0">
+                  <ClassificationNodeChildList count="3">
+                    <ClassificationNode>
+                      <Disorder id="31785">
+                        <OrphaCode>645617</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645617</ExpertLink>
+                        <Name lang="en">Amyopathic dermatomyositis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31786">
+                        <OrphaCode>645626</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645626</ExpertLink>
+                        <Name lang="en">Adermatopathic dermatomyositis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31784">
+                        <OrphaCode>645613</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645613</ExpertLink>
+                        <Name lang="en">Classical dermatomyositis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -13373,7 +13385,19 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="12">
+                  <ClassificationNodeChildList count="13">
+                    <ClassificationNode>
+                      <Disorder id="17180">
+                        <OrphaCode>158029</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158029</ExpertLink>
+                        <Name lang="en">Sea-blue histiocytosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="17176">
                         <OrphaCode>158014</OrphaCode>
@@ -13541,7 +13565,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="25">
+                  <ClassificationNodeChildList count="28">
                     <ClassificationNode>
                       <Disorder id="19644">
                         <OrphaCode>251523</OrphaCode>
@@ -13666,7 +13690,7 @@
                       <Disorder id="10924">
                         <OrphaCode>69126</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=69126</ExpertLink>
-                        <Name lang="en">Pyogenic arthritis-pyoderma gangrenosum-acne syndrome</Name>
+                        <Name lang="en">PAPA syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -13707,7 +13731,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="5">
                         <ClassificationNode>
                           <Disorder id="2375">
                             <OrphaCode>575</OrphaCode>
@@ -13744,6 +13768,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31819">
+                            <OrphaCode>647815</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647815</ExpertLink>
+                            <Name lang="en">Keratoendotheliitis fugax hereditaria</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="31372">
                             <OrphaCode>617919</OrphaCode>
@@ -13798,7 +13834,7 @@
                       <Disorder id="20725">
                         <OrphaCode>289478</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=289478</ExpertLink>
-                        <Name lang="en">Pyoderma gangrenosum-acne-suppurative hidradenitis syndrome</Name>
+                        <Name lang="en">PASH syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -13962,6 +13998,42 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31720">
+                        <OrphaCode>641380</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641380</ExpertLink>
+                        <Name lang="en">PAPASH syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31721">
+                        <OrphaCode>641385</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641385</ExpertLink>
+                        <Name lang="en">PASS syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31722">
+                        <OrphaCode>641390</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641390</ExpertLink>
+                        <Name lang="en">PsAPASH syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
               </ClassificationNodeChildList>
@@ -14247,7 +14319,19 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
+                        <ClassificationNode>
+                          <Disorder id="31699">
+                            <OrphaCode>636955</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=636955</ExpertLink>
+                            <Name lang="en">Endemic pemphigus foliaceus</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="11506">
                             <OrphaCode>79480</OrphaCode>
