commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_201.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_201.xml
index 957f269..8c902fc 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_201.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_201.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:39:04" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:51:41" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -89,10 +89,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="863">
-                            <OrphaCode>3389</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3389</ExpertLink>
-                            <Name lang="en">Tuberculosis</Name>
+                          <Disorder id="31798">
+                            <OrphaCode>645874</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645874</ExpertLink>
+                            <Name lang="en">Primary genito-urinary tuberculosis</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -723,7 +723,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="5">
                         <ClassificationNode>
                           <Disorder id="12096">
                             <OrphaCode>90793</OrphaCode>
@@ -808,59 +808,11 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="14466">
-                            <OrphaCode>99893</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99893</ExpertLink>
-                            <Name lang="en">ACTH-independent Cushing syndrome</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="18376">
-                                <OrphaCode>189439</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=189439</ExpertLink>
-                                <Name lang="en">Primary pigmented nodular adrenocortical disease</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="23454">
-                                <OrphaCode>443287</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443287</ExpertLink>
-                                <Name lang="en">ACTH-independent Cushing syndrome due to rare cortisol-producing adrenal tumor</Name>
-                                <DisorderType id="36561">
-                                  <Name lang="en">Category</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="1">
-                                <ClassificationNode>
-                                  <Disorder id="3731">
-                                    <OrphaCode>1501</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1501</ExpertLink>
-                                    <Name lang="en">Adrenocortical carcinoma</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="18374">
                             <OrphaCode>189427</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=189427</ExpertLink>
-                            <Name lang="en">Cushing syndrome due to macronodular adrenal hyperplasia</Name>
+                            <Name lang="en">Cushing syndrome due to bilateral macronodular adrenocortical disease</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -908,7 +860,7 @@
                           <Disorder id="203">
                             <OrphaCode>752</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=752</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -920,7 +872,7 @@
                           <Disorder id="324">
                             <OrphaCode>753</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=753</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to 5-alpha-reductase 2 deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development due to 5-alpha-reductase 2 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1292,7 +1244,7 @@
                           <Disorder id="5546">
                             <OrphaCode>393</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=393</ExpertLink>
-                            <Name lang="en">46,XX testicular disorder of sex development</Name>
+                            <Name lang="en">46,XX testicular difference of sex development</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -2019,12 +1971,12 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="5">
                         <ClassificationNode>
                           <Disorder id="203">
                             <OrphaCode>752</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=752</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2152,54 +2104,6 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="14466">
-                            <OrphaCode>99893</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99893</ExpertLink>
-                            <Name lang="en">ACTH-independent Cushing syndrome</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="18376">
-                                <OrphaCode>189439</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=189439</ExpertLink>
-                                <Name lang="en">Primary pigmented nodular adrenocortical disease</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="23454">
-                                <OrphaCode>443287</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443287</ExpertLink>
-                                <Name lang="en">ACTH-independent Cushing syndrome due to rare cortisol-producing adrenal tumor</Name>
-                                <DisorderType id="36561">
-                                  <Name lang="en">Category</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="1">
-                                <ClassificationNode>
-                                  <Disorder id="3731">
-                                    <OrphaCode>1501</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1501</ExpertLink>
-                                    <Name lang="en">Adrenocortical carcinoma</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -2410,7 +2314,7 @@
                       <Disorder id="2037">
                         <OrphaCode>2138</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2138</ExpertLink>
-                        <Name lang="en">46,XX ovotesticular disorder of sex development</Name>
+                        <Name lang="en">46,XX ovotesticular difference of sex development</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -2427,7 +2331,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31739">
+                            <OrphaCode>642945</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 1</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31742">
+                            <OrphaCode>642976</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 2</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
