commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml
index 0512b22..3ab79cd 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:39:28" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:52:09" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -159,19 +159,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="11">
-                        <ClassificationNode>
-                          <Disorder id="18751">
-                            <OrphaCode>213574</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213574</ExpertLink>
-                            <Name lang="en">Rare variants of adenocarcinoma of the corpus uteri</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="10">
                         <ClassificationNode>
                           <Disorder id="18752">
                             <OrphaCode>213589</OrphaCode>
@@ -375,7 +363,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="10">
+                      <ClassificationNodeChildList count="9">
                         <ClassificationNode>
                           <Disorder id="18770">
                             <OrphaCode>213767</OrphaCode>
@@ -496,18 +484,6 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="18780">
-                            <OrphaCode>213817</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213817</ExpertLink>
-                            <Name lang="en">Papillary carcinoma of the cervix uteri</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="18781">
                             <OrphaCode>213823</OrphaCode>
@@ -865,19 +841,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="7">
-                            <ClassificationNode>
-                              <Disorder id="18500">
-                                <OrphaCode>206473</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206473</ExpertLink>
-                                <Name lang="en">Borderline epithelial tumor of ovary</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="5">
                             <ClassificationNode>
                               <Disorder id="18742">
                                 <OrphaCode>213504</OrphaCode>
@@ -926,18 +890,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="22743">
-                                <OrphaCode>398980</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=398980</ExpertLink>
-                                <Name lang="en">Primary peritoneal serous/papillary carcinoma</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="23658">
                                 <OrphaCode>454723</OrphaCode>
@@ -961,7 +913,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="6">
+                          <ClassificationNodeChildList count="5">
                             <ClassificationNode>
                               <Disorder id="22495">
                                 <OrphaCode>370396</OrphaCode>
@@ -1094,42 +1046,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="18744">
-                                <OrphaCode>213517</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213517</ExpertLink>
-                                <Name lang="en">Familial ovarian cancer</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="3384">
-                                    <OrphaCode>145</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=145</ExpertLink>
-                                    <Name lang="en">Hereditary breast and ovarian cancer syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="18745">
-                                    <OrphaCode>213524</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213524</ExpertLink>
-                                    <Name lang="en">Hereditary site-specific ovarian cancer syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="22744">
                                 <OrphaCode>398987</OrphaCode>
@@ -1208,7 +1124,7 @@
                           <Disorder id="3384">
                             <OrphaCode>145</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=145</ExpertLink>
-                            <Name lang="en">Hereditary breast and ovarian cancer syndrome</Name>
+                            <Name lang="en">Hereditary breast and/or ovarian cancer syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1303,7 +1219,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="16">
+              <ClassificationNodeChildList count="17">
                 <ClassificationNode>
                   <Disorder id="5517">
                     <OrphaCode>80</OrphaCode>
@@ -1412,6 +1328,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31820">
+                    <OrphaCode>647823</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647823</ExpertLink>
+                    <Name lang="en">Idiopathic pregnancy-associated osteoporosis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31322">
                     <OrphaCode>615970</OrphaCode>
@@ -1906,7 +1834,7 @@
                       <Disorder id="2679">
                         <OrphaCode>2973</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2973</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development-anorectal anomalies syndrome</Name>
+                        <Name lang="en">46,XX difference of sex development-anorectal anomalies syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -2369,7 +2297,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="17">
+                  <ClassificationNodeChildList count="18">
                     <ClassificationNode>
                       <Disorder id="44">
                         <OrphaCode>881</OrphaCode>
@@ -2475,7 +2403,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31739">
+                            <OrphaCode>642945</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 1</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31742">
+                            <OrphaCode>642976</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 2</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -2634,6 +2586,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31734">
+                        <OrphaCode>642691</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642691</ExpertLink>
+                        <Name lang="en">Fragile X-associated primary ovarian insufficiency</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -3168,53 +3132,41 @@
                   <Disorder id="23298">
                     <OrphaCode>435561</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=435561</ExpertLink>
-                    <Name lang="en">Precocious puberty in female</Name>
+                    <Name lang="en">Rare precocious puberty in female</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="3">
-                    <ClassificationNode>
-                      <Disorder id="279">
-                        <OrphaCode>562</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=562</ExpertLink>
-                        <Name lang="en">McCune-Albright syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="2">
                     <ClassificationNode>
-                      <Disorder id="8674">
-                        <OrphaCode>759</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=759</ExpertLink>
-                        <Name lang="en">Central precocious puberty</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                      <Disorder id="31882">
+                        <OrphaCode>650070</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650070</ExpertLink>
+                        <Name lang="en">Rare central precocious puberty in female</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
                         <ClassificationNode>
-                          <Disorder id="17866">
-                            <OrphaCode>169615</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=169615</ExpertLink>
-                            <Name lang="en">Idiopathic central precocious puberty</Name>
-                            <DisorderType id="21443">
-                              <Name lang="en">Etiological subtype</Name>
+                          <Disorder id="31883">
+                            <OrphaCode>650077</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650077</ExpertLink>
+                            <Name lang="en">Genetic central precocious puberty in female</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="17867">
-                            <OrphaCode>169618</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=169618</ExpertLink>
-                            <Name lang="en">Secondary central precocious puberty</Name>
-                            <DisorderType id="21443">
-                              <Name lang="en">Etiological subtype</Name>
+                          <Disorder id="31884">
+                            <OrphaCode>650082</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650082</ExpertLink>
+                            <Name lang="en">Secondary central precocious puberty in female</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -3223,15 +3175,39 @@
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="18018">
-                        <OrphaCode>178345</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=178345</ExpertLink>
-                        <Name lang="en">Aromatase excess syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                      <Disorder id="31890">
+                        <OrphaCode>650187</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650187</ExpertLink>
+                        <Name lang="en">Rare peripheral precocious puberty in female</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="279">
+                            <OrphaCode>562</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=562</ExpertLink>
+                            <Name lang="en">McCune-Albright syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="18018">
+                            <OrphaCode>178345</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=178345</ExpertLink>
+                            <Name lang="en">Aromatase excess syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -3254,7 +3230,7 @@
               <Disorder id="21848">
                 <OrphaCode>325620</OrphaCode>
                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325620</ExpertLink>
-                <Name lang="en">Disorder of sex development of gynecological interest</Name>
+                <Name lang="en">Difference of sex development of gynecological interest</Name>
                 <DisorderType id="36561">
                   <Name lang="en">Category</Name>
                 </DisorderType>
@@ -3310,7 +3286,7 @@
                       <Disorder id="2037">
                         <OrphaCode>2138</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2138</ExpertLink>
-                        <Name lang="en">46,XX ovotesticular disorder of sex development</Name>
+                        <Name lang="en">46,XX ovotesticular difference of sex development</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -3334,7 +3310,7 @@
                       <Disorder id="21838">
                         <OrphaCode>325345</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325345</ExpertLink>
-                        <Name lang="en">46,XY ovotesticular disorder of sex development</Name>
+                        <Name lang="en">46,XY ovotesticular difference of sex development</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -3360,7 +3336,7 @@
                   <Disorder id="13096">
                     <OrphaCode>98078</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98078</ExpertLink>
-                    <Name lang="en">46,XX disorder of sex development induced by androgens excess</Name>
+                    <Name lang="en">46,XX difference of sex development induced by androgens excess</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -3370,7 +3346,7 @@
                       <Disorder id="12087">
                         <OrphaCode>90776</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90776</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development induced by fetal androgens excess</Name>
+                        <Name lang="en">46,XX difference of sex development induced by fetal androgens excess</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -3478,7 +3454,7 @@
                       <Disorder id="12119">
                         <OrphaCode>91144</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=91144</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development induced by maternal-derived androgen</Name>
+                        <Name lang="en">46,XX difference of sex development induced by maternal-derived androgen</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -3488,7 +3464,7 @@
                           <Disorder id="21829">
                             <OrphaCode>325093</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325093</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by endogenous maternal-derived androgen</Name>
+                            <Name lang="en">46,XX difference of sex development induced by endogenous maternal-derived androgen</Name>
                             <DisorderType id="21436">
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
@@ -3500,7 +3476,7 @@
                           <Disorder id="21830">
                             <OrphaCode>325099</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325099</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by exogenous maternal-derived androgen</Name>
+                            <Name lang="en">46,XX difference of sex development induced by exogenous maternal-derived androgen</Name>
                             <DisorderType id="21436">
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
@@ -3514,7 +3490,7 @@
                       <Disorder id="21828">
                         <OrphaCode>325061</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325061</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development induced by fetoplacental androgens excess</Name>
+                        <Name lang="en">46,XX difference of sex development induced by fetoplacental androgens excess</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -3540,7 +3516,7 @@
                   <Disorder id="21849">
                     <OrphaCode>325632</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325632</ExpertLink>
-                    <Name lang="en">46,XY disorder of sex development of gynecological interest</Name>
+                    <Name lang="en">46,XY difference of sex development of gynecological interest</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -3586,7 +3562,7 @@
                       <Disorder id="12090">
                         <OrphaCode>90786</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90786</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development due to adrenal and testicular steroidogenesis defect</Name>
+                        <Name lang="en">46,XY difference of sex development due to adrenal and testicular steroidogenesis defect</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -3668,7 +3644,7 @@
                           <Disorder id="17771">
                             <OrphaCode>168558</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168558</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3694,7 +3670,7 @@
                       <Disorder id="12091">
                         <OrphaCode>90787</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90787</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development due to testicular steroidogenesis defect</Name>
+                        <Name lang="en">46,XY difference of sex development due to testicular steroidogenesis defect</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -3704,7 +3680,7 @@
                           <Disorder id="203">
                             <OrphaCode>752</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=752</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3716,7 +3692,7 @@
                           <Disorder id="12099">
                             <OrphaCode>90796</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90796</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to isolated 17,20-lyase deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development due to isolated 17,20-lyase deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3730,7 +3706,7 @@
                       <Disorder id="13104">
                         <OrphaCode>98086</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98086</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
+                        <Name lang="en">46,XY difference of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -3740,7 +3716,7 @@
                           <Disorder id="324">
                             <OrphaCode>753</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=753</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to 5-alpha-reductase 2 deficiency</Name>
+                            <Name lang="en">46,XY difference of sex development due to 5-alpha-reductase 2 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3756,7 +3732,7 @@
                   <Disorder id="21850">
                     <OrphaCode>325638</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325638</ExpertLink>
-                    <Name lang="en">Syndrome with disorder of sex development of gynecological interest</Name>
+                    <Name lang="en">Syndrome with difference of sex development of gynecological interest</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -3826,7 +3802,7 @@
                       <Disorder id="334">
                         <OrphaCode>1642</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1642</ExpertLink>
-                        <Name lang="en">Distal monosomy 9p</Name>
+                        <Name lang="en">Distal deletion 9p</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -3850,7 +3826,7 @@
                       <Disorder id="570">
                         <OrphaCode>2983</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2983</ExpertLink>
-                        <Name lang="en">Disorder of sex development-intellectual disability syndrome</Name>
+                        <Name lang="en">Difference of sex development-intellectual disability syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -3862,7 +3838,7 @@
                       <Disorder id="606">
                         <OrphaCode>1422</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1422</ExpertLink>
-                        <Name lang="en">Chondrodysplasia-disorder of sex development syndrome</Name>
+                        <Name lang="en">Chondrodysplasia-difference of sex development syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -3910,7 +3886,7 @@
                       <Disorder id="2151">
                         <OrphaCode>2282</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2282</ExpertLink>
-                        <Name lang="en">Dysmorphism-short stature-deafness-disorder of sex development syndrome</Name>
+                        <Name lang="en">Dysmorphism-short stature-deafness-difference of sex development syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -3927,14 +3903,38 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31739">
+                            <OrphaCode>642945</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 1</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31742">
+                            <OrphaCode>642976</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 2</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="2681">
                         <OrphaCode>2975</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2975</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development-skeletal anomalies syndrome</Name>
+                        <Name lang="en">46,XX difference of sex development-skeletal anomalies syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
