commit cb98a4638b303dbc159c97a636d6d4f62ca39678
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 6 12:14:41 2022 +0100

    DEC 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml
index f53dcc5..5190d29 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:19:55" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:39:54" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -94,7 +94,7 @@
                       <Disorder id="28442">
                         <OrphaCode>565612</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=565612</ExpertLink>
-                        <Name lang="en">Triglyceride deposit cardiomyovasculopathy</Name>
+                        <Name lang="en">Primary triglyceride deposit cardiomyovasculopathy</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -4246,7 +4246,7 @@
                       <Disorder id="183">
                         <OrphaCode>637</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637</ExpertLink>
-                        <Name lang="en">Neurofibromatosis type 2</Name>
+                        <Name lang="en">Full NF2-related schwannomatosis</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -6262,7 +6262,7 @@
                       <Disorder id="12482">
                         <OrphaCode>93921</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93921</ExpertLink>
-                        <Name lang="en">Schwannomatosis</Name>
+                        <Name lang="en">Full schwannomatosis</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
