commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml
old mode 100644
new mode 100755
index b488c05..3fb0e4d
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:33:43" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:34:20" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2972,7 +2972,7 @@
                           <Disorder id="12657">
                             <OrphaCode>95702</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95702</ExpertLink>
-                            <Name lang="en">Cytomegalic congenital adrenal hypoplasia</Name>
+                            <Name lang="en">X-linked adrenal hypoplasia congenita due to a NR0B1 point mutation</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
