commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml
index daf9e79..bc99ef9 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:38:03" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:50:28" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -457,7 +457,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="9">
+                          <ClassificationNodeChildList count="10">
                             <ClassificationNode>
                               <Disorder id="27238">
                                 <OrphaCode>519337</OrphaCode>
@@ -506,18 +506,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="16740">
-                                <OrphaCode>137902</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137902</ExpertLink>
-                                <Name lang="en">Isolated optic nerve hypoplasia/aplasia</Name>
-                                <DisorderType id="21415">
-                                  <Name lang="en">Morphological anomaly</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="16741">
                                 <OrphaCode>137905</OrphaCode>
@@ -914,6 +902,30 @@
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31705">
+                                <OrphaCode>637064</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637064</ExpertLink>
+                                <Name lang="en">Isolated optic nerve aplasia</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31704">
+                                <OrphaCode>637061</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637061</ExpertLink>
+                                <Name lang="en">Isolated optic nerve hypoplasia</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="13688">
                                 <OrphaCode>98671</OrphaCode>
@@ -3112,7 +3124,7 @@
                                   <Disorder id="1086">
                                     <OrphaCode>1581</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1581</ExpertLink>
-                                    <Name lang="en">Non-distal monosomy 10q</Name>
+                                    <Name lang="en">Non-distal deletion 10q</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -3913,7 +3925,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="6">
+                          <ClassificationNodeChildList count="7">
                             <ClassificationNode>
                               <Disorder id="12567">
                                 <OrphaCode>94147</OrphaCode>
@@ -3955,59 +3967,11 @@
                                 <OrphaCode>506</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=506</ExpertLink>
                                 <Name lang="en">Leigh syndrome</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="4">
-                                <ClassificationNode>
-                                  <Disorder id="10945">
-                                    <OrphaCode>70472</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=70472</ExpertLink>
-                                    <Name lang="en">Congenital lactic acidosis, Saguenay-Lac-Saint-Jean type</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="10947">
-                                    <OrphaCode>70474</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=70474</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with cardiomyopathy</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19814">
-                                    <OrphaCode>255241</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=255241</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with leukodystrophy</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19815">
-                                    <OrphaCode>255249</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=255249</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with nephrotic syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -4166,6 +4130,18 @@
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="10945">
+                                <OrphaCode>70472</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=70472</ExpertLink>
+                                <Name lang="en">Congenital lactic acidosis, Saguenay-Lac-Saint-Jean type</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -4217,7 +4193,55 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="1">
+                  <ClassificationNodeChildList count="2">
+                    <ClassificationNode>
+                      <Disorder id="31840">
+                        <OrphaCode>648559</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648559</ExpertLink>
+                        <Name lang="en">Rare scleritis</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="3">
+                        <ClassificationNode>
+                          <Disorder id="31847">
+                            <OrphaCode>648681</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648681</ExpertLink>
+                            <Name lang="en">Immune-mediated scleritis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31846">
+                            <OrphaCode>648675</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648675</ExpertLink>
+                            <Name lang="en">Idiopathic scleritis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31845">
+                            <OrphaCode>648665</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648665</ExpertLink>
+                            <Name lang="en">Infectious scleritis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="27218">
                         <OrphaCode>519296</OrphaCode>
@@ -4227,7 +4251,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="20">
+                      <ClassificationNodeChildList count="19">
                         <ClassificationNode>
                           <Disorder id="372">
                             <OrphaCode>2772</OrphaCode>
@@ -4504,18 +4528,6 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="12290">
-                            <OrphaCode>93359</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93359</ExpertLink>
-                            <Name lang="en">Spondyloepimetaphyseal dysplasia with joint laxity</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2288">
                             <OrphaCode>2475</OrphaCode>
@@ -4575,7 +4587,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
                           <Disorder id="20416">
                             <OrphaCode>279922</OrphaCode>
@@ -4588,18 +4600,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="20516">
-                            <OrphaCode>280914</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280914</ExpertLink>
-                            <Name lang="en">Idiopathic anterior uveitis</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="20519">
                             <OrphaCode>280926</OrphaCode>
@@ -5377,7 +5377,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="16">
+                          <ClassificationNodeChildList count="17">
                             <ClassificationNode>
                               <Disorder id="1645">
                                 <OrphaCode>190</OrphaCode>
@@ -5426,6 +5426,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31848">
+                                <OrphaCode>648684</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648684</ExpertLink>
+                                <Name lang="en">Central retinal artery occlusion</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="19537">
                                 <OrphaCode>247691</OrphaCode>
@@ -8032,7 +8044,7 @@
                                   <Disorder id="30618">
                                     <OrphaCode>603694</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=603694</ExpertLink>
-                                    <Name lang="en">KLHL7-related cold-induced sweating-like syndrome</Name>
+                                    <Name lang="en">KLHL7-related Crisponi/cold-induced sweating-like syndrome</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -8044,7 +8056,7 @@
                                   <Disorder id="30616">
                                     <OrphaCode>603684</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=603684</ExpertLink>
-                                    <Name lang="en">KLHL7-related Bohring-Opitz-like/Cold-induced sweating-like overlap syndrome</Name>
+                                    <Name lang="en">KLHL7-related Bohring-Opitz-like and Crisponi/Cold-induced sweating-like overlap syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -8381,7 +8393,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="8">
+                  <ClassificationNodeChildList count="10">
                     <ClassificationNode>
                       <Disorder id="31348">
                         <OrphaCode>617449</OrphaCode>
@@ -9772,7 +9784,7 @@
                                   <Disorder id="12749">
                                     <OrphaCode>96125</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96125</ExpertLink>
-                                    <Name lang="en">Distal monosomy 6p</Name>
+                                    <Name lang="en">Distal deletion 6p</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -9894,6 +9906,30 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31698">
+                        <OrphaCode>636950</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=636950</ExpertLink>
+                        <Name lang="en">Glaucomatocyclitic crisis disease</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31819">
+                        <OrphaCode>647815</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647815</ExpertLink>
+                        <Name lang="en">Keratoendotheliitis fugax hereditaria</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="11971">
                         <OrphaCode>90080</OrphaCode>
@@ -10454,7 +10490,7 @@
                                       <Disorder id="29044">
                                         <OrphaCode>583612</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583612</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
+                                        <Name lang="en">Neu-Laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
@@ -10466,7 +10502,7 @@
                                       <Disorder id="29042">
                                         <OrphaCode>583602</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583602</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
+                                        <Name lang="en">Neu-Laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
@@ -10478,7 +10514,7 @@
                                       <Disorder id="29043">
                                         <OrphaCode>583607</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583607</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
+                                        <Name lang="en">Neu-Laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
@@ -10888,7 +10924,7 @@
                                   <Disorder id="1156">
                                     <OrphaCode>1705</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1705</ExpertLink>
-                                    <Name lang="en">Distal trisomy 14q</Name>
+                                    <Name lang="en">Distal duplication 14q</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -11256,7 +11292,7 @@
                                           <Disorder id="11420">
                                             <OrphaCode>79394</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79394</ExpertLink>
-                                            <Name lang="en">Congenital non-bullous ichthyosiform erythroderma</Name>
+                                            <Name lang="en">Congenital ichthyosiform erythroderma</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -12069,7 +12105,7 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="5">
+                                  <ClassificationNodeChildList count="6">
                                     <ClassificationNode>
                                       <Disorder id="2739">
                                         <OrphaCode>3047</OrphaCode>
@@ -12110,7 +12146,7 @@
                                       <Disorder id="1114">
                                         <OrphaCode>1620</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1620</ExpertLink>
-                                        <Name lang="en">Distal monosomy 3p</Name>
+                                        <Name lang="en">Distal deletion 3p</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -12130,6 +12166,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31702">
+                                        <OrphaCode>637013</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637013</ExpertLink>
+                                        <Name lang="en">SMARCA2-related blepharophimosis-intellectual disability syndrome</Name>
+                                        <DisorderType id="21401">
+                                          <Name lang="en">Malformation syndrome</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -14003,7 +14051,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="122">
+                              <ClassificationNodeChildList count="121">
                                 <ClassificationNode>
                                   <Disorder id="190">
                                     <OrphaCode>649</OrphaCode>
@@ -14342,7 +14390,7 @@
                                       <Disorder id="910">
                                         <OrphaCode>574</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=574</ExpertLink>
-                                        <Name lang="en">Monosomy 21</Name>
+                                        <Name lang="en">21q deletion syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14366,7 +14414,7 @@
                                       <Disorder id="1092">
                                         <OrphaCode>1590</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1590</ExpertLink>
-                                        <Name lang="en">Distal monosomy 13q</Name>
+                                        <Name lang="en">Distal deletion 13q</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14378,7 +14426,7 @@
                                       <Disorder id="1149">
                                         <OrphaCode>1695</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1695</ExpertLink>
-                                        <Name lang="en">Non-distal trisomy 10q</Name>
+                                        <Name lang="en">Non-distal duplication 10q</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14426,7 +14474,7 @@
                                       <Disorder id="12694">
                                         <OrphaCode>96070</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96070</ExpertLink>
-                                        <Name lang="en">Distal trisomy 2p</Name>
+                                        <Name lang="en">Distal duplication 2p</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14450,7 +14498,7 @@
                                       <Disorder id="12726">
                                         <OrphaCode>96102</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96102</ExpertLink>
-                                        <Name lang="en">Distal trisomy 10q</Name>
+                                        <Name lang="en">Distal duplication 10q</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14484,7 +14532,7 @@
                                           <Disorder id="338">
                                             <OrphaCode>1707</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1707</ExpertLink>
-                                            <Name lang="en">Distal trisomy 15q</Name>
+                                            <Name lang="en">Distal duplication 15q</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -14496,7 +14544,7 @@
                                           <Disorder id="21499">
                                             <OrphaCode>314588</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314588</ExpertLink>
-                                            <Name lang="en">Distal tetrasomy 15q</Name>
+                                            <Name lang="en">Distal triplication 15q</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -16068,18 +16116,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="2486">
-                                    <OrphaCode>2725</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2725</ExpertLink>
-                                    <Name lang="en">Eye defects-arachnodactyly-cardiopathy syndrome</Name>
-                                    <DisorderType id="21401">
-                                      <Name lang="en">Malformation syndrome</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="2111">
                                     <OrphaCode>2238</OrphaCode>
@@ -16841,8 +16877,8 @@
                                     <OrphaCode>633035</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633035</ExpertLink>
                                     <Name lang="en">Intellectual disability-early-onset cataract-microcephaly syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -17612,7 +17648,7 @@
                           <Disorder id="2989">
                             <OrphaCode>3374</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3374</ExpertLink>
-                            <Name lang="en">Triopia</Name>
+                            <Name lang="en">Unilateral ocular duplication</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -17756,7 +17792,7 @@
                           <Disorder id="606">
                             <OrphaCode>1422</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1422</ExpertLink>
-                            <Name lang="en">Chondrodysplasia-disorder of sex development syndrome</Name>
+                            <Name lang="en">Chondrodysplasia-difference of sex development syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -17886,7 +17922,7 @@
                               <Disorder id="12749">
                                 <OrphaCode>96125</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96125</ExpertLink>
-                                <Name lang="en">Distal monosomy 6p</Name>
+                                <Name lang="en">Distal deletion 6p</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
