commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml
index f6a167c..2aeafe5 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:41:45" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:45:17" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -11,7 +11,7 @@
     <Classification id="150">
       <OrphaNumber>156449</OrphaNumber>
       <Name lang="en">Orphanet classification of rare inborn errors of metabolism</Name>
-      <ClassificationNodeRootList count="1">
+      <ClassificationNodeRootList count="2">
         <ClassificationNode>
           <Disorder id="10507">
             <OrphaCode>68367</OrphaCode>
@@ -2045,7 +2045,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="12">
+                  <ClassificationNodeChildList count="11">
                     <ClassificationNode>
                       <Disorder id="6">
                         <OrphaCode>585</OrphaCode>
@@ -2262,30 +2262,6 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="11105">
-                        <OrphaCode>77292</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
-                        <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="11106">
-                        <OrphaCode>77293</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
-                        <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="11230">
                         <OrphaCode>79204</OrphaCode>
@@ -2622,6 +2598,54 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31389">
+                        <OrphaCode>618899</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618899</ExpertLink>
+                        <Name lang="en">Acid sphingomyelinase deficiency</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="3">
+                        <ClassificationNode>
+                          <Disorder id="11105">
+                            <OrphaCode>77292</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
+                            <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="11106">
+                            <OrphaCode>77293</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
+                            <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31388">
+                            <OrphaCode>618891</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
+                            <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -8699,7 +8723,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="13">
+                              <ClassificationNodeChildList count="14">
                                 <ClassificationNode>
                                   <Disorder id="63">
                                     <OrphaCode>550</OrphaCode>
@@ -8872,7 +8896,7 @@
                                   <Disorder id="21781">
                                     <OrphaCode>324525</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=324525</ExpertLink>
-                                    <Name lang="en">Hypertrophic cardiomyopathy and renal tubular disease due to mitochondrial DNA mutation</Name>
+                                    <Name lang="en">Hypertrophic cardiomyopathy with kidney anomalies due to mitochondrial DNA mutation</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -8892,6 +8916,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31442">
+                                    <OrphaCode>620371</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620371</ExpertLink>
+                                    <Name lang="en">Gitelman-like kidney tubulopathy due to mitochondrial DNA mutation</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="315">
                                     <OrphaCode>1349</OrphaCode>
@@ -11341,7 +11377,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="6">
                             <ClassificationNode>
                               <Disorder id="18665">
                                 <OrphaCode>209902</OrphaCode>
@@ -11390,6 +11426,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="11106">
+                                <OrphaCode>77293</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
+                                <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31388">
+                                <OrphaCode>618891</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
+                                <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -13539,137 +13599,77 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="1">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
-                          <Disorder id="10344">
-                            <OrphaCode>34526</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34526</ExpertLink>
-                            <Name lang="en">Genetic primary hypomagnesemia</Name>
-                            <DisorderType id="36561">
-                              <Name lang="en">Category</Name>
+                          <Disorder id="9282">
+                            <OrphaCode>30924</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=30924</ExpertLink>
+                            <Name lang="en">Primary hypomagnesemia with secondary hypocalcemia</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
-                            <ClassificationNode>
-                              <Disorder id="28399">
-                                <OrphaCode>564178</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=564178</ExpertLink>
-                                <Name lang="en">Primary hypomagnesemia with refractory seizures and intellectual disability</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21222">
-                                <OrphaCode>306516</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306516</ExpertLink>
-                                <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="2078">
-                                    <OrphaCode>2196</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
-                                    <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="9284">
-                                    <OrphaCode>31043</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31043</ExpertLink>
-                                    <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis without severe ocular involvement</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21223">
-                                <OrphaCode>306519</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306519</ExpertLink>
-                                <Name lang="en">Genetic primary hypomagnesemia with hypocalciuria</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="1">
-                                <ClassificationNode>
-                                  <Disorder id="10346">
-                                    <OrphaCode>34528</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34528</ExpertLink>
-                                    <Name lang="en">Autosomal dominant primary hypomagnesemia with hypocalciuria</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21224">
-                                <OrphaCode>306522</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306522</ExpertLink>
-                                <Name lang="en">Genetic primary hypomagnesemia with normocalciuria</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="3">
-                                <ClassificationNode>
-                                  <Disorder id="9282">
-                                    <OrphaCode>30924</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=30924</ExpertLink>
-                                    <Name lang="en">Primary hypomagnesemia with secondary hypocalcemia</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="10345">
-                                    <OrphaCode>34527</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34527</ExpertLink>
-                                    <Name lang="en">Familial primary hypomagnesemia with normocalciuria and normocalcemia</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="18401">
-                                    <OrphaCode>199326</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=199326</ExpertLink>
-                                    <Name lang="en">Isolated autosomal dominant hypomagnesemia, Glaudemans type</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="10346">
+                            <OrphaCode>34528</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34528</ExpertLink>
+                            <Name lang="en">Autosomal dominant primary hypomagnesemia with hypocalciuria</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="18401">
+                            <OrphaCode>199326</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=199326</ExpertLink>
+                            <Name lang="en">Isolated autosomal dominant hypomagnesemia, Glaudemans type</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="28399">
+                            <OrphaCode>564178</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=564178</ExpertLink>
+                            <Name lang="en">Primary hypomagnesemia-refractory seizures-intellectual disability syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31440">
+                            <OrphaCode>620363</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620363</ExpertLink>
+                            <Name lang="en">Primary hypomagnesemia-generalized seizures-intellectual disability-obesity syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31441">
+                            <OrphaCode>620368</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620368</ExpertLink>
+                            <Name lang="en">EGF-related primary hypomagnesemia with intellectual disability</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -13728,6 +13728,54 @@
             </ClassificationNode>
           </ClassificationNodeChildList>
         </ClassificationNode>
+        <ClassificationNode>
+          <Disorder id="10344">
+            <OrphaCode>34526</OrphaCode>
+            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34526</ExpertLink>
+            <Name lang="en">OBSOLETE: Genetic primary hypomagnesemia</Name>
+            <DisorderType id="36561">
+              <Name lang="en">Category</Name>
+            </DisorderType>
+          </Disorder>
+          <ClassificationNodeChildList count="1">
+            <ClassificationNode>
+              <Disorder id="21222">
+                <OrphaCode>306516</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306516</ExpertLink>
+                <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="2">
+                <ClassificationNode>
+                  <Disorder id="2078">
+                    <OrphaCode>2196</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
+                    <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
+                    <DisorderType id="21450">
+                      <Name lang="en">Clinical subtype</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="9284">
+                    <OrphaCode>31043</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31043</ExpertLink>
+                    <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis without severe ocular involvement</Name>
+                    <DisorderType id="21450">
+                      <Name lang="en">Clinical subtype</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+              </ClassificationNodeChildList>
+            </ClassificationNode>
+          </ClassificationNodeChildList>
+        </ClassificationNode>
       </ClassificationNodeRootList>
     </Classification>
   </ClassificationList>
