commit 7ab24fe976be5c798004246aa079a429e568689b
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Jun 30 11:46:37 2025 +0200

    JULY 2025

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_209.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_209.xml
old mode 100644
new mode 100755
index b4aa653..982c78b
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_209.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_209.xml	
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-12-03 07:09:42" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<?xml version="1.0" encoding="UTF-8"?>
+<JDBOR date="2025-06-24 07:34:29" version="1.3.42 / 4.1.8 [2025-03-03]" copyright="Orphanet (c) 2025" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -967,7 +967,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="29">
+              <ClassificationNodeChildList count="30">
                 <ClassificationNode>
                   <Disorder id="20370">
                     <OrphaCode>276280</OrphaCode>
@@ -1664,7 +1664,7 @@
                           <Disorder id="1889">
                             <OrphaCode>2209</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2209</ExpertLink>
-                            <Name lang="en">Maternal phenylketonuria</Name>
+                            <Name lang="en">Maternal phenylketonuria syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -2948,6 +2948,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32281">
+                    <OrphaCode>694956</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=694956</ExpertLink>
+                    <Name lang="en">Intellectual disability-lymphoid hypertrophy-macrocephaly syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="28850">
                     <OrphaCode>576278</OrphaCode>
