commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml
index 769b07b..2acf92f 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:23:40" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:08:07" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2351,7 +2351,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="69">
+                              <ClassificationNodeChildList count="68">
                                 <ClassificationNode>
                                   <Disorder id="11307">
                                     <OrphaCode>79281</OrphaCode>
@@ -2556,18 +2556,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="10656">
-                                    <OrphaCode>50817</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=50817</ExpertLink>
-                                    <Name lang="en">Duane anomaly-myopathy-scoliosis syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="11103">
                                     <OrphaCode>77260</OrphaCode>
@@ -2764,7 +2752,7 @@
                                   <Disorder id="11113">
                                     <OrphaCode>77301</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77301</ExpertLink>
-                                    <Name lang="en">Monosomy 9q22.3</Name>
+                                    <Name lang="en">Monosomy 9q22.3 syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -3160,7 +3148,7 @@
                                   <Disorder id="1086">
                                     <OrphaCode>1581</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1581</ExpertLink>
-                                    <Name lang="en">Non-distal deletion 10q</Name>
+                                    <Name lang="en">Non-distal deletion 10q syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -4623,7 +4611,19 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
+                        <ClassificationNode>
+                          <Disorder id="20516">
+                            <OrphaCode>280914</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280914</ExpertLink>
+                            <Name lang="en">Isolated idiopathic anterior uveitis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="20416">
                             <OrphaCode>279922</OrphaCode>
@@ -8020,7 +8020,7 @@
                                   <Disorder id="337">
                                     <OrphaCode>3378</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3378</ExpertLink>
-                                    <Name lang="en">Trisomy 13</Name>
+                                    <Name lang="en">Trisomy 13 syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -9592,7 +9592,7 @@
                                   <Disorder id="12749">
                                     <OrphaCode>96125</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96125</ExpertLink>
-                                    <Name lang="en">Distal deletion 6p</Name>
+                                    <Name lang="en">Distal deletion 6p syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -9877,7 +9877,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="23">
+                          <ClassificationNodeChildList count="24">
                             <ClassificationNode>
                               <Disorder id="1388">
                                 <OrphaCode>1106</OrphaCode>
@@ -10094,6 +10094,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32219">
+                                <OrphaCode>689829</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689829</ExpertLink>
+                                <Name lang="en">Microphthalmia-motor delay-language delay-brain anomalies-diaphragmatic hernia syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="30614">
                                 <OrphaCode>603494</OrphaCode>
@@ -10670,7 +10682,7 @@
                                       <Disorder id="14199">
                                         <OrphaCode>99226</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                                        <Name lang="en">Monosomy X</Name>
+                                        <Name lang="en">Monosomy X syndrome</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
@@ -10682,7 +10694,7 @@
                                       <Disorder id="14200">
                                         <OrphaCode>99228</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                                        <Name lang="en">Mosaic monosomy X</Name>
+                                        <Name lang="en">Mosaic monosomy X syndrome</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
@@ -10720,7 +10732,7 @@
                                   <Disorder id="201">
                                     <OrphaCode>281</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=281</ExpertLink>
-                                    <Name lang="en">Monosomy 5p</Name>
+                                    <Name lang="en">Monosomy 5p syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -10732,7 +10744,7 @@
                                   <Disorder id="1090">
                                     <OrphaCode>1587</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1587</ExpertLink>
-                                    <Name lang="en">Monosomy 13q14</Name>
+                                    <Name lang="en">Monosomy 13q14 syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -10744,7 +10756,7 @@
                                   <Disorder id="1156">
                                     <OrphaCode>1705</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1705</ExpertLink>
-                                    <Name lang="en">Distal duplication 14q</Name>
+                                    <Name lang="en">Distal duplication 14q syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -11966,7 +11978,7 @@
                                       <Disorder id="1114">
                                         <OrphaCode>1620</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1620</ExpertLink>
-                                        <Name lang="en">Distal deletion 3p</Name>
+                                        <Name lang="en">Distal deletion 3p syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -12649,7 +12661,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="14">
+                          <ClassificationNodeChildList count="15">
                             <ClassificationNode>
                               <Disorder id="1864">
                                 <OrphaCode>1896</OrphaCode>
@@ -13670,6 +13682,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32157">
+                                <OrphaCode>684305</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684305</ExpertLink>
+                                <Name lang="en">NOCGUS syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -14076,7 +14100,7 @@
                                           <Disorder id="14199">
                                             <OrphaCode>99226</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                                            <Name lang="en">Monosomy X</Name>
+                                            <Name lang="en">Monosomy X syndrome</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -14088,7 +14112,7 @@
                                           <Disorder id="14200">
                                             <OrphaCode>99228</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                                            <Name lang="en">Mosaic monosomy X</Name>
+                                            <Name lang="en">Mosaic monosomy X syndrome</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -14126,7 +14150,7 @@
                                       <Disorder id="201">
                                         <OrphaCode>281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=281</ExpertLink>
-                                        <Name lang="en">Monosomy 5p</Name>
+                                        <Name lang="en">Monosomy 5p syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14138,7 +14162,7 @@
                                       <Disorder id="328">
                                         <OrphaCode>1598</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1598</ExpertLink>
-                                        <Name lang="en">Monosomy 18p</Name>
+                                        <Name lang="en">Monosomy 18p syndrome</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -14150,7 +14174,7 @@
                                       <Disorder id="330">
                                         <OrphaCode>1600</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1600</ExpertLink>
-                                        <Name lang="en">Monosomy 18q</Name>
+                                        <Name lang="en">Monosomy 18q syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14162,7 +14186,7 @@
                                       <Disorder id="337">
                                         <OrphaCode>3378</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3378</ExpertLink>
-                                        <Name lang="en">Trisomy 13</Name>
+                                        <Name lang="en">Trisomy 13 syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14174,7 +14198,7 @@
                                       <Disorder id="339">
                                         <OrphaCode>3380</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3380</ExpertLink>
-                                        <Name lang="en">Trisomy 18</Name>
+                                        <Name lang="en">Trisomy 18 syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14186,7 +14210,7 @@
                                       <Disorder id="346">
                                         <OrphaCode>236</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=236</ExpertLink>
-                                        <Name lang="en">Trisomy 9p</Name>
+                                        <Name lang="en">Trisomy 9p syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14210,7 +14234,7 @@
                                       <Disorder id="1090">
                                         <OrphaCode>1587</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1587</ExpertLink>
-                                        <Name lang="en">Monosomy 13q14</Name>
+                                        <Name lang="en">Monosomy 13q14 syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14222,7 +14246,7 @@
                                       <Disorder id="1092">
                                         <OrphaCode>1590</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1590</ExpertLink>
-                                        <Name lang="en">Distal deletion 13q</Name>
+                                        <Name lang="en">Distal deletion 13q syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14234,7 +14258,7 @@
                                       <Disorder id="1149">
                                         <OrphaCode>1695</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1695</ExpertLink>
-                                        <Name lang="en">Non-distal duplication 10q</Name>
+                                        <Name lang="en">Non-distal duplication 10q syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14246,7 +14270,7 @@
                                       <Disorder id="1178">
                                         <OrphaCode>1742</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1742</ExpertLink>
-                                        <Name lang="en">Trisomy 5p</Name>
+                                        <Name lang="en">Trisomy 5p syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14258,7 +14282,7 @@
                                       <Disorder id="3491">
                                         <OrphaCode>3309</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3309</ExpertLink>
-                                        <Name lang="en">Tetrasomy 5p</Name>
+                                        <Name lang="en">Tetrasomy 5p syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14270,7 +14294,7 @@
                                       <Disorder id="12694">
                                         <OrphaCode>96070</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96070</ExpertLink>
-                                        <Name lang="en">Distal duplication 2p</Name>
+                                        <Name lang="en">Distal duplication 2p syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14294,7 +14318,7 @@
                                       <Disorder id="12726">
                                         <OrphaCode>96102</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96102</ExpertLink>
-                                        <Name lang="en">Distal duplication 10q</Name>
+                                        <Name lang="en">Distal duplication 10q syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14306,7 +14330,7 @@
                                       <Disorder id="12792">
                                         <OrphaCode>96168</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96168</ExpertLink>
-                                        <Name lang="en">Monosomy 13q34</Name>
+                                        <Name lang="en">Monosomy 13q34 syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -14328,7 +14352,7 @@
                                           <Disorder id="338">
                                             <OrphaCode>1707</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1707</ExpertLink>
-                                            <Name lang="en">Distal duplication 15q</Name>
+                                            <Name lang="en">Distal duplication 15q syndrome</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -14340,7 +14364,7 @@
                                           <Disorder id="21499">
                                             <OrphaCode>314588</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314588</ExpertLink>
-                                            <Name lang="en">Distal triplication 15q</Name>
+                                            <Name lang="en">Distal triplication 15q syndrome</Name>
                                             <DisorderType id="21443">
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
@@ -14366,7 +14390,7 @@
                                       <Disorder id="1232">
                                         <OrphaCode>3376</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3376</ExpertLink>
-                                        <Name lang="en">Triploidy</Name>
+                                        <Name lang="en">Triploidy syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -17312,7 +17336,7 @@
                           <Disorder id="346">
                             <OrphaCode>236</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=236</ExpertLink>
-                            <Name lang="en">Trisomy 9p</Name>
+                            <Name lang="en">Trisomy 9p syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -17718,7 +17742,7 @@
                               <Disorder id="12749">
                                 <OrphaCode>96125</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96125</ExpertLink>
-                                <Name lang="en">Distal deletion 6p</Name>
+                                <Name lang="en">Distal deletion 6p syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
