commit a9304bcd06b429331d3fd652192800e94c21e73c
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml
index 5cf9eb8..fd0f122 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_187.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:29:41" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:32:33" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -549,7 +549,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="10">
+                                  <ClassificationNodeChildList count="7">
                                     <ClassificationNode>
                                       <Disorder id="381">
                                         <OrphaCode>773</OrphaCode>
@@ -670,42 +670,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="29042">
-                                        <OrphaCode>583602</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583602</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="29043">
-                                        <OrphaCode>583607</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583607</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="29044">
-                                        <OrphaCode>583612</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583612</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -717,7 +681,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="8">
+                                  <ClassificationNodeChildList count="5">
                                     <ClassificationNode>
                                       <Disorder id="6">
                                         <OrphaCode>585</OrphaCode>
@@ -766,42 +730,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="29042">
-                                        <OrphaCode>583602</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583602</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="29043">
-                                        <OrphaCode>583607</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583607</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="29044">
-                                        <OrphaCode>583612</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583612</ExpertLink>
-                                        <Name lang="en">Neu-laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="2439">
                                         <OrphaCode>2671</OrphaCode>
@@ -4601,7 +4529,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="110">
+                  <ClassificationNodeChildList count="111">
                     <ClassificationNode>
                       <Disorder id="3069">
                         <OrphaCode>3200</OrphaCode>
@@ -6162,6 +6090,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29437">
+                        <OrphaCode>589608</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589608</ExpertLink>
+                        <Name lang="en">Linear hypopigmentation and craniofacial asymmetry with acral, ocular and brain anomalies</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -6701,7 +6641,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="11">
+                  <ClassificationNodeChildList count="12">
                     <ClassificationNode>
                       <Disorder id="303">
                         <OrphaCode>998</OrphaCode>
@@ -7170,6 +7110,18 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29437">
+                        <OrphaCode>589608</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589608</ExpertLink>
+                        <Name lang="en">Linear hypopigmentation and craniofacial asymmetry with acral, ocular and brain anomalies</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -10255,7 +10207,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="38">
+              <ClassificationNodeChildList count="37">
                 <ClassificationNode>
                   <Disorder id="185">
                     <OrphaCode>636</OrphaCode>
@@ -10797,10 +10749,10 @@
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
-                  <Disorder id="10311">
-                    <OrphaCode>33111</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=33111</ExpertLink>
-                    <Name lang="en">Granulomatous slack skin</Name>
+                  <Disorder id="807">
+                    <OrphaCode>2800</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2800</ExpertLink>
+                    <Name lang="en">Extramammary Paget disease</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -10916,18 +10868,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="11772">
-                    <OrphaCode>86884</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86884</ExpertLink>
-                    <Name lang="en">Subcutaneous panniculitis-like T-cell lymphoma</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="12144">
                     <OrphaCode>91414</OrphaCode>
