commit 481d14fea17f62fcae7251f3673242ec271b28fa
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 4 20:10:24 2023 +0100

    DEC 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml
index 0220bc4..2e6141c 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-06-22 13:55:49" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-12-04 08:01:49" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -77,7 +77,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="15">
+                  <ClassificationNodeChildList count="16">
                     <ClassificationNode>
                       <Disorder id="481">
                         <OrphaCode>1851</OrphaCode>
@@ -606,6 +606,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31897">
+                        <OrphaCode>652528</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652528</ExpertLink>
+                        <Name lang="en">Non-syndromic supernumerary kidneys</Name>
+                        <DisorderType id="21415">
+                          <Name lang="en">Morphological anomaly</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -617,7 +629,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="65">
+                  <ClassificationNodeChildList count="66">
                     <ClassificationNode>
                       <Disorder id="44">
                         <OrphaCode>881</OrphaCode>
@@ -1614,6 +1626,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31921">
+                        <OrphaCode>656130</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656130</ExpertLink>
+                        <Name lang="en">PBX1-related congenital anomalies of kidney and urinary tract syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
               </ClassificationNodeChildList>
@@ -2159,7 +2183,19 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="2">
+                              <ClassificationNodeChildList count="3">
+                                <ClassificationNode>
+                                  <Disorder id="31937">
+                                    <OrphaCode>658574</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658574</ExpertLink>
+                                    <Name lang="en">Isolated pulmonary artery sling</Name>
+                                    <DisorderType id="21415">
+                                      <Name lang="en">Morphological anomaly</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="14066">
                                     <OrphaCode>99049</OrphaCode>
@@ -3971,7 +4007,19 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="6">
+                              <ClassificationNodeChildList count="7">
+                                <ClassificationNode>
+                                  <Disorder id="31901">
+                                    <OrphaCode>652668</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652668</ExpertLink>
+                                    <Name lang="en">Primary superior vena cava aneurysm</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="14126">
                                     <OrphaCode>99109</OrphaCode>
@@ -4055,7 +4103,19 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="5">
+                              <ClassificationNodeChildList count="6">
+                                <ClassificationNode>
+                                  <Disorder id="31902">
+                                    <OrphaCode>652678</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652678</ExpertLink>
+                                    <Name lang="en">Primary inferior vena cava aneurysm</Name>
+                                    <DisorderType id="21415">
+                                      <Name lang="en">Morphological anomaly</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="14136">
                                     <OrphaCode>99119</OrphaCode>
@@ -9513,7 +9573,7 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="6">
+                                  <ClassificationNodeChildList count="7">
                                     <ClassificationNode>
                                       <Disorder id="28816">
                                         <OrphaCode>573278</OrphaCode>
@@ -9610,6 +9670,18 @@
                                         </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31920">
+                                        <OrphaCode>656126</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656126</ExpertLink>
+                                        <Name lang="en">Segmental spinal dysgenesis</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="31755">
                                         <OrphaCode>645273</OrphaCode>
@@ -10045,7 +10117,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="13">
+                          <ClassificationNodeChildList count="14">
                             <ClassificationNode>
                               <Disorder id="10817">
                                 <OrphaCode>63260</OrphaCode>
@@ -10274,6 +10346,18 @@
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31920">
+                                <OrphaCode>656126</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656126</ExpertLink>
+                                <Name lang="en">Segmental spinal dysgenesis</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="946">
                                 <OrphaCode>3027</OrphaCode>
@@ -13297,7 +13381,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="11">
+                          <ClassificationNodeChildList count="12">
                             <ClassificationNode>
                               <Disorder id="954">
                                 <OrphaCode>808</OrphaCode>
@@ -13418,6 +13502,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31940">
+                                <OrphaCode>658595</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658595</ExpertLink>
+                                <Name lang="en">DNMT3A-related microcephalic dwarfism</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="28764">
                                 <OrphaCode>572761</OrphaCode>
@@ -16434,7 +16530,7 @@
                               <Disorder id="26077">
                                 <OrphaCode>502430</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=502430</ExpertLink>
-                                <Name lang="en">Metopic ridging-ptosis-facial dysmorphism syndrome</Name>
+                                <Name lang="en">Weiss-Kruszka Syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -18620,7 +18716,7 @@
                           <Disorder id="173">
                             <OrphaCode>394</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=394</ExpertLink>
-                            <Name lang="en">Classic homocystinuria</Name>
+                            <Name lang="en">Homocystinuria due to cystathionine beta-synthase deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -18831,7 +18927,7 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="5">
                         <ClassificationNode>
                           <Disorder id="11849">
                             <OrphaCode>88917</OrphaCode>
@@ -18880,6 +18976,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31909">
+                            <OrphaCode>653722</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653722</ExpertLink>
+                            <Name lang="en">Digenic Alport syndrome</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -19464,7 +19572,7 @@
                   <Disorder id="173">
                     <OrphaCode>394</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=394</ExpertLink>
-                    <Name lang="en">Classic homocystinuria</Name>
+                    <Name lang="en">Homocystinuria due to cystathionine beta-synthase deficiency</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -22971,7 +23079,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="15">
+                      <ClassificationNodeChildList count="14">
                         <ClassificationNode>
                           <Disorder id="24011">
                             <OrphaCode>464288</OrphaCode>
@@ -23056,18 +23164,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="1270">
-                            <OrphaCode>1795</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1795</ExpertLink>
-                            <Name lang="en">Peripheral dysostosis</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1289">
                             <OrphaCode>969</OrphaCode>
@@ -23655,7 +23751,19 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="9">
+                      <ClassificationNodeChildList count="10">
+                        <ClassificationNode>
+                          <Disorder id="31925">
+                            <OrphaCode>656283</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656283</ExpertLink>
+                            <Name lang="en">Autosomal recessive combined immunodeficiency due to complete IL6ST deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="933">
                             <OrphaCode>140</OrphaCode>
@@ -23757,8 +23865,8 @@
                             <OrphaCode>324307</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=324307</ExpertLink>
                             <Name lang="en">Severe lateral tibial bowing-short stature-mild winged scapula-mild facial dysmorphism syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -31909,7 +32017,19 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="44">
+                          <ClassificationNodeChildList count="45">
+                            <ClassificationNode>
+                              <Disorder id="31944">
+                                <OrphaCode>658805</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658805</ExpertLink>
+                                <Name lang="en">Greig cephalopolysyndactyly-contiguous gene syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="28585">
                                 <OrphaCode>567502</OrphaCode>
@@ -32947,7 +33067,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="52">
+              <ClassificationNodeChildList count="56">
                 <ClassificationNode>
                   <Disorder id="145">
                     <OrphaCode>904</OrphaCode>
@@ -33572,6 +33692,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="2184">
+                    <OrphaCode>2332</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2332</ExpertLink>
+                    <Name lang="en">KBG syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="23714">
                     <OrphaCode>457240</OrphaCode>
@@ -33668,6 +33800,30 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="3137">
+                    <OrphaCode>2183</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2183</ExpertLink>
+                    <Name lang="en">Hydrocephalus-obesity-hypogonadism syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="2475">
+                    <OrphaCode>2714</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2714</ExpertLink>
+                    <Name lang="en">Oculo-palato-cerebral syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="26569">
                     <OrphaCode>508488</OrphaCode>
@@ -33788,6 +33944,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="960">
+                    <OrphaCode>902</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=902</ExpertLink>
+                    <Name lang="en">Werner syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="30690">
                     <OrphaCode>611216</OrphaCode>
@@ -33857,7 +34025,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="29">
+                  <ClassificationNodeChildList count="27">
                     <ClassificationNode>
                       <Disorder id="260">
                         <OrphaCode>116</OrphaCode>
@@ -34050,222 +34218,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="1025">
-                        <OrphaCode>657</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=657</ExpertLink>
-                        <Name lang="en">Congenital isolated hyperinsulinism</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="2">
-                        <ClassificationNode>
-                          <Disorder id="17593">
-                            <OrphaCode>165985</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=165985</ExpertLink>
-                            <Name lang="en">Diazoxide-sensitive diffuse hyperinsulinism</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="9">
-                            <ClassificationNode>
-                              <Disorder id="10987">
-                                <OrphaCode>71212</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=71212</ExpertLink>
-                                <Name lang="en">Hyperinsulinism due to short chain 3-hydroxylacyl-CoA dehydrogenase deficiency</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="11325">
-                                <OrphaCode>79299</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79299</ExpertLink>
-                                <Name lang="en">Congenital glucokinase-related hyperinsulinism</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="17595">
-                                <OrphaCode>165991</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=165991</ExpertLink>
-                                <Name lang="en">Exercise-induced hyperinsulinism</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="20056">
-                                <OrphaCode>263455</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263455</ExpertLink>
-                                <Name lang="en">Congenital hyperinsulinism due to HNF4A deficiency</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="20383">
-                                <OrphaCode>276556</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276556</ExpertLink>
-                                <Name lang="en">Hyperinsulinism due to UCP2 deficiency</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="20384">
-                                <OrphaCode>276575</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276575</ExpertLink>
-                                <Name lang="en">Autosomal dominant hyperinsulinism due to SUR1 deficiency</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="10403">
-                                <OrphaCode>35878</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35878</ExpertLink>
-                                <Name lang="en">Hyperinsulinism-hyperammonemia syndrome</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="20385">
-                                <OrphaCode>276580</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276580</ExpertLink>
-                                <Name lang="en">Autosomal dominant hyperinsulinism due to Kir6.2 deficiency</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21787">
-                                <OrphaCode>324575</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=324575</ExpertLink>
-                                <Name lang="en">Hyperinsulinism due to HNF1A deficiency</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="20386">
-                            <OrphaCode>276585</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276585</ExpertLink>
-                            <Name lang="en">Diazoxide-resistant hyperinsulinism</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="11324">
-                                <OrphaCode>79298</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79298</ExpertLink>
-                                <Name lang="en">Diazoxide-resistant focal hyperinsulinism</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="20387">
-                                    <OrphaCode>276598</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276598</ExpertLink>
-                                    <Name lang="en">Diazoxide-resistant focal hyperinsulinism due to SUR1 deficiency</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="20388">
-                                    <OrphaCode>276603</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276603</ExpertLink>
-                                    <Name lang="en">Diazoxide-resistant focal hyperinsulinism due to Kir6.2 deficiency</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="17594">
-                                <OrphaCode>165988</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=165988</ExpertLink>
-                                <Name lang="en">Diazoxide-resistant diffuse hyperinsulinism</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="11541">
-                                    <OrphaCode>79643</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79643</ExpertLink>
-                                    <Name lang="en">Autosomal recessive hyperinsulinism due to SUR1 deficiency</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="11542">
-                                    <OrphaCode>79644</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79644</ExpertLink>
-                                    <Name lang="en">Autosomal recessive hyperinsulinism due to Kir6.2 deficiency</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="1473">
                         <OrphaCode>109</OrphaCode>
@@ -34470,18 +34422,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="17810">
-                        <OrphaCode>168984</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168984</ExpertLink>
-                        <Name lang="en">CLAPO syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="20896">
                         <OrphaCode>293964</OrphaCode>
@@ -34575,7 +34515,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="8">
+                      <ClassificationNodeChildList count="9">
                         <ClassificationNode>
                           <Disorder id="10805">
                             <OrphaCode>60040</OrphaCode>
@@ -34672,6 +34612,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="17810">
+                            <OrphaCode>168984</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168984</ExpertLink>
+                            <Name lang="en">CLAPO syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -34697,7 +34649,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="36">
+                  <ClassificationNodeChildList count="37">
                     <ClassificationNode>
                       <Disorder id="387">
                         <OrphaCode>819</OrphaCode>
@@ -35310,6 +35262,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31892">
+                        <OrphaCode>652487</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652487</ExpertLink>
+                        <Name lang="en">Developmental delay-overweight-facial dysmorphism-behavioral abnormalities syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="20900">
                         <OrphaCode>293987</OrphaCode>
@@ -45417,7 +45381,19 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="1">
+                                  <ClassificationNodeChildList count="2">
+                                    <ClassificationNode>
+                                      <Disorder id="31924">
+                                        <OrphaCode>656279</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656279</ExpertLink>
+                                        <Name lang="en">1p36.33 duplication syndrome</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="12693">
                                         <OrphaCode>96069</OrphaCode>
@@ -48165,7 +48141,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="1">
+                                  <ClassificationNodeChildList count="2">
                                     <ClassificationNode>
                                       <Disorder id="12750">
                                         <OrphaCode>96126</OrphaCode>
@@ -48178,6 +48154,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31944">
+                                        <OrphaCode>658805</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658805</ExpertLink>
+                                        <Name lang="en">Greig cephalopolysyndactyly-contiguous gene syndrome</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -48885,7 +48873,19 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="2">
+                                  <ClassificationNodeChildList count="3">
+                                    <ClassificationNode>
+                                      <Disorder id="31933">
+                                        <OrphaCode>658540</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658540</ExpertLink>
+                                        <Name lang="en">16q22 deletion syndrome</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="19864">
                                         <OrphaCode>261250</OrphaCode>
@@ -49624,9 +49624,9 @@
                                   <Disorder id="19856">
                                     <OrphaCode>261190</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261190</ExpertLink>
-                                    <Name lang="en">15q14 microdeletion syndrome</Name>
-                                    <DisorderType id="21401">
-                                      <Name lang="en">Malformation syndrome</Name>
+                                    <Name lang="en">Cleft palate-congenital heart defect-intellectual disability syndrome due to 15q14 microdeletion</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -50969,7 +50969,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="657">
+                  <ClassificationNodeChildList count="663">
                     <ClassificationNode>
                       <Disorder id="24010">
                         <OrphaCode>464282</OrphaCode>
@@ -51562,7 +51562,7 @@
                       <Disorder id="22612">
                         <OrphaCode>391372</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=391372</ExpertLink>
-                        <Name lang="en">Intellectual disability-severe speech delay-mild dysmorphism syndrome</Name>
+                        <Name lang="en">FOXP1 Syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -53379,7 +53379,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="11">
+                      <ClassificationNodeChildList count="12">
                         <ClassificationNode>
                           <Disorder id="954">
                             <OrphaCode>808</OrphaCode>
@@ -53500,6 +53500,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31940">
+                            <OrphaCode>658595</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658595</ExpertLink>
+                            <Name lang="en">DNMT3A-related microcephalic dwarfism</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="28764">
                             <OrphaCode>572761</OrphaCode>
@@ -56458,7 +56470,7 @@
                       <Disorder id="26077">
                         <OrphaCode>502430</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=502430</ExpertLink>
-                        <Name lang="en">Metopic ridging-ptosis-facial dysmorphism syndrome</Name>
+                        <Name lang="en">Weiss-Kruszka Syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -58746,18 +58758,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="2011">
-                        <OrphaCode>380</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=380</ExpertLink>
-                        <Name lang="en">Greig cephalopolysyndactyly syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="3051">
                         <OrphaCode>3472</OrphaCode>
@@ -60102,6 +60102,114 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31895">
+                        <OrphaCode>652519</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652519</ExpertLink>
+                        <Name lang="en">Cleft palate-congenital heart defect-intellectual disability syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31894">
+                            <OrphaCode>652514</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652514</ExpertLink>
+                            <Name lang="en">Cleft palate-congenital heart defect-intellectual disability syndrome due to MEIS2 mutation</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="19856">
+                            <OrphaCode>261190</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261190</ExpertLink>
+                            <Name lang="en">Cleft palate-congenital heart defect-intellectual disability syndrome due to 15q14 microdeletion</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31908">
+                        <OrphaCode>653712</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653712</ExpertLink>
+                        <Name lang="en">CHD4-related neurodevelopmental disorder</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31913">
+                        <OrphaCode>653767</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653767</ExpertLink>
+                        <Name lang="en">Jansen-de Vries syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31922">
+                        <OrphaCode>656135</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656135</ExpertLink>
+                        <Name lang="en">Intellectual disability-cupped ears syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31921">
+                        <OrphaCode>656130</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656130</ExpertLink>
+                        <Name lang="en">PBX1-related congenital anomalies of kidney and urinary tract syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31933">
+                        <OrphaCode>658540</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658540</ExpertLink>
+                        <Name lang="en">16q22 deletion syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31947">
+                        <OrphaCode>658843</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658843</ExpertLink>
+                        <Name lang="en">Developmental delay-ataxia-hypotonia-facial dysmorphism syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="570">
                         <OrphaCode>2983</OrphaCode>
@@ -60653,7 +60761,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="176">
+                  <ClassificationNodeChildList count="178">
                     <ClassificationNode>
                       <Disorder id="10993">
                         <OrphaCode>71271</OrphaCode>
@@ -61494,6 +61602,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="2011">
+                        <OrphaCode>380</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=380</ExpertLink>
+                        <Name lang="en">Greig cephalopolysyndactyly syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="172">
                         <OrphaCode>508</OrphaCode>
@@ -62802,6 +62922,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31944">
+                        <OrphaCode>658805</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658805</ExpertLink>
+                        <Name lang="en">Greig cephalopolysyndactyly-contiguous gene syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="31818">
                         <OrphaCode>647811</OrphaCode>
@@ -65259,7 +65391,19 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="44">
+                      <ClassificationNodeChildList count="45">
+                        <ClassificationNode>
+                          <Disorder id="31944">
+                            <OrphaCode>658805</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658805</ExpertLink>
+                            <Name lang="en">Greig cephalopolysyndactyly-contiguous gene syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="28585">
                             <OrphaCode>567502</OrphaCode>
@@ -67289,7 +67433,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="4">
+                  <ClassificationNodeChildList count="5">
                     <ClassificationNode>
                       <Disorder id="13741">
                         <OrphaCode>98724</OrphaCode>
@@ -67919,7 +68063,19 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="6">
+                              <ClassificationNodeChildList count="7">
+                                <ClassificationNode>
+                                  <Disorder id="31901">
+                                    <OrphaCode>652668</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652668</ExpertLink>
+                                    <Name lang="en">Primary superior vena cava aneurysm</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="14126">
                                     <OrphaCode>99109</OrphaCode>
@@ -68003,7 +68159,19 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="5">
+                              <ClassificationNodeChildList count="6">
+                                <ClassificationNode>
+                                  <Disorder id="31902">
+                                    <OrphaCode>652678</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652678</ExpertLink>
+                                    <Name lang="en">Primary inferior vena cava aneurysm</Name>
+                                    <DisorderType id="21415">
+                                      <Name lang="en">Morphological anomaly</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="14136">
                                     <OrphaCode>99119</OrphaCode>
@@ -68238,6 +68406,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="12937">
+                        <OrphaCode>97598</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97598</ExpertLink>
+                        <Name lang="en">Congenital renal artery stenosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="25672">
                         <OrphaCode>494424</OrphaCode>
@@ -71587,7 +71767,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="187">
+              <ClassificationNodeChildList count="189">
                 <ClassificationNode>
                   <Disorder id="27595">
                     <OrphaCode>529574</OrphaCode>
@@ -72257,7 +72437,7 @@
                       <Name lang="en">Disease</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="4">
+                  <ClassificationNodeChildList count="5">
                     <ClassificationNode>
                       <Disorder id="11849">
                         <OrphaCode>88917</OrphaCode>
@@ -72306,6 +72486,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31909">
+                        <OrphaCode>653722</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=653722</ExpertLink>
+                        <Name lang="en">Digenic Alport syndrome</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -73584,7 +73776,7 @@
                   <Disorder id="3521">
                     <OrphaCode>3240</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3240</ExpertLink>
-                    <Name lang="en">Central nervous system calcification-deafness-tubular acidosis-anemia syndrome</Name>
+                    <Name lang="en">Early-onset progressive leukoencephalopathy-central nervous system calcification-deafness-visual impairment syndrome</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -74252,6 +74444,30 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31898">
+                    <OrphaCode>652532</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652532</ExpertLink>
+                    <Name lang="en">Adult-onset progressive leukoencephalopathy-early-onset deafness</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31921">
+                    <OrphaCode>656130</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656130</ExpertLink>
+                    <Name lang="en">PBX1-related congenital anomalies of kidney and urinary tract syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
