commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml
index ad9aee1..24cacff 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:41:49" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:43:23" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -3238,18 +3238,6 @@
               <ClassificationNodeChildList count="0">
               </ClassificationNodeChildList>
             </ClassificationNode>
-            <ClassificationNode>
-              <Disorder id="12590">
-                <OrphaCode>95431</OrphaCode>
-                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95431</ExpertLink>
-                <Name lang="en">Twin to twin transfusion syndrome</Name>
-                <DisorderType id="21394">
-                  <Name lang="en">Disease</Name>
-                </DisorderType>
-              </Disorder>
-              <ClassificationNodeChildList count="0">
-              </ClassificationNodeChildList>
-            </ClassificationNode>
             <ClassificationNode>
               <Disorder id="13444">
                 <OrphaCode>98427</OrphaCode>
@@ -3529,7 +3517,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="6">
+                          <ClassificationNodeChildList count="7">
                             <ClassificationNode>
                               <Disorder id="24021">
                                 <OrphaCode>464343</OrphaCode>
@@ -3602,6 +3590,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="5517">
+                                <OrphaCode>80</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=80</ExpertLink>
+                                <Name lang="en">Antiphospholipid syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -3830,7 +3830,7 @@
                                       <Disorder id="17996">
                                         <OrphaCode>177926</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=177926</ExpertLink>
-                                        <Name lang="en">Bleeding disorder in hemophilia A carriers without FVIII deficiency</Name>
+                                        <Name lang="en">Bleeding disorder in hemophilia A carriers</Name>
                                         <DisorderType id="21450">
                                           <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
@@ -3890,7 +3890,7 @@
                                       <Disorder id="17997">
                                         <OrphaCode>177929</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=177929</ExpertLink>
-                                        <Name lang="en">Bleeding disorder in hemophilia B carriers without FIX deficiency</Name>
+                                        <Name lang="en">Bleeding disorder in hemophilia B carriers</Name>
                                         <DisorderType id="21450">
                                           <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
@@ -7414,6 +7414,18 @@
               <ClassificationNodeChildList count="0">
               </ClassificationNodeChildList>
             </ClassificationNode>
+            <ClassificationNode>
+              <Disorder id="31337">
+                <OrphaCode>617294</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617294</ExpertLink>
+                <Name lang="en">Twin anemia-polycythemia sequence</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="0">
+              </ClassificationNodeChildList>
+            </ClassificationNode>
             <ClassificationNode>
               <Disorder id="28676">
                 <OrphaCode>570470</OrphaCode>
