commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml
index 1d24233..ac05d91 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:05:17" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:23:51" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -747,7 +747,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
+                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
                           <Disorder id="14452">
                             <OrphaCode>99879</OrphaCode>
@@ -772,6 +772,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="121">
+                            <OrphaCode>652</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652</ExpertLink>
+                            <Name lang="en">Multiple endocrine neoplasia type 1</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -1155,7 +1167,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
+                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
                           <Disorder id="14452">
                             <OrphaCode>99879</OrphaCode>
@@ -1180,6 +1192,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="121">
+                            <OrphaCode>652</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652</ExpertLink>
+                            <Name lang="en">Multiple endocrine neoplasia type 1</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -1215,7 +1239,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="7">
                         <ClassificationNode>
                           <Disorder id="3719">
                             <OrphaCode>1652</OrphaCode>
@@ -1252,6 +1276,54 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="11">
+                            <OrphaCode>213</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213</ExpertLink>
+                            <Name lang="en">Cystinosis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="3">
+                            <ClassificationNode>
+                              <Disorder id="23024">
+                                <OrphaCode>411634</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=411634</ExpertLink>
+                                <Name lang="en">Juvenile nephropathic cystinosis</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="23025">
+                                <OrphaCode>411641</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=411641</ExpertLink>
+                                <Name lang="en">Ocular cystinosis</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="23023">
+                                <OrphaCode>411629</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=411629</ExpertLink>
+                                <Name lang="en">Infantile nephropathic cystinosis</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="19477">
                             <OrphaCode>244305</OrphaCode>
@@ -3948,7 +4020,7 @@
                   <Disorder id="12664">
                     <OrphaCode>95709</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95709</ExpertLink>
-                    <Name lang="en">Acquired premature ovarian failure</Name>
+                    <Name lang="en">Rare acquired premature ovarian failure</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -3984,7 +4056,7 @@
                   <Disorder id="12665">
                     <OrphaCode>95710</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95710</ExpertLink>
-                    <Name lang="en">Non-acquired premature ovarian failure</Name>
+                    <Name lang="en">Rare non-acquired premature ovarian failure</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -4892,7 +4964,7 @@
                           <Disorder id="900">
                             <OrphaCode>847</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=847</ExpertLink>
-                            <Name lang="en">Alpha-thalassemia-X-linked intellectual disability syndrome</Name>
+                            <Name lang="en">X-linked alpha-thalassemia-intellectual disability syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -6413,7 +6485,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="12">
+                  <ClassificationNodeChildList count="11">
                     <ClassificationNode>
                       <Disorder id="849">
                         <OrphaCode>3198</OrphaCode>
@@ -6534,18 +6606,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="7037">
-                        <OrphaCode>225</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225</ExpertLink>
-                        <Name lang="en">Maternally-inherited diabetes and deafness</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="10643">
                         <OrphaCode>49827</OrphaCode>
@@ -9444,7 +9504,7 @@
                   <Disorder id="22628">
                     <OrphaCode>391487</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=391487</ExpertLink>
-                    <Name lang="en">Autoimmune enteropathy and endocrinopathy-susceptibility to chronic infections syndrome</Name>
+                    <Name lang="en">STAT1-related autoimmune enteropathy and endocrinopathy-susceptibility to chronic infections syndrome</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -10990,7 +11050,7 @@
                       <Disorder id="17999">
                         <OrphaCode>178029</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=178029</ExpertLink>
-                        <Name lang="en">Central diabetes insipidus</Name>
+                        <Name lang="en">Arginine vasopressin deficiency</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -11000,7 +11060,7 @@
                           <Disorder id="9283">
                             <OrphaCode>30925</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=30925</ExpertLink>
-                            <Name lang="en">Hereditary central diabetes insipidus</Name>
+                            <Name lang="en">Hereditary arginine vasopressin deficiency</Name>
                             <DisorderType id="21450">
                               <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
@@ -11012,7 +11072,7 @@
                           <Disorder id="12650">
                             <OrphaCode>95626</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95626</ExpertLink>
-                            <Name lang="en">Acquired central diabetes insipidus</Name>
+                            <Name lang="en">Acquired arginine vasopressin deficiency</Name>
                             <DisorderType id="21450">
                               <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
