commit 7ab24fe976be5c798004246aa079a429e568689b
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Jun 30 11:46:37 2025 +0200

    JULY 2025

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_204.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_204.xml
old mode 100644
new mode 100755
index 05e6d88..88f440e
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_204.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_204.xml	
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-12-03 07:09:32" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<?xml version="1.0" encoding="UTF-8"?>
+<JDBOR date="2025-06-24 07:34:19" version="1.3.42 / 4.1.8 [2025-03-03]" copyright="Orphanet (c) 2025" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -523,7 +523,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="18">
+              <ClassificationNodeChildList count="19">
                 <ClassificationNode>
                   <Disorder id="28676">
                     <OrphaCode>570470</OrphaCode>
@@ -536,6 +536,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="3134">
+                    <OrphaCode>2582</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2582</ExpertLink>
+                    <Name lang="en">Myalgia-eosinophilia syndrome associated with tryptophan</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="9797">
                     <OrphaCode>31825</OrphaCode>
