commit 481d14fea17f62fcae7251f3673242ec271b28fa
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 4 20:10:24 2023 +0100

    DEC 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml
index 1110b74..cc4c98b 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-06-22 13:59:44" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-12-04 08:04:41" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
