commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml
index cd53392..f6a167c 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:41:31" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:41:45" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2266,7 +2266,7 @@
                       <Disorder id="11105">
                         <OrphaCode>77292</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
-                        <Name lang="en">Niemann-Pick disease type A</Name>
+                        <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -2278,7 +2278,7 @@
                       <Disorder id="11106">
                         <OrphaCode>77293</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
-                        <Name lang="en">Niemann-Pick disease type B</Name>
+                        <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
