commit 7ab24fe976be5c798004246aa079a429e568689b
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Jun 30 11:46:37 2025 +0200

    JULY 2025

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml
old mode 100644
new mode 100755
index f6f0e94..da1c7c7
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml	
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-12-03 07:07:32" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<?xml version="1.0" encoding="UTF-8"?>
+<JDBOR date="2025-06-24 07:32:44" version="1.3.42 / 4.1.8 [2025-03-03]" copyright="Orphanet (c) 2025" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -43,7 +43,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="9">
+              <ClassificationNodeChildList count="12">
                 <ClassificationNode>
                   <Disorder id="236">
                     <OrphaCode>774</OrphaCode>
@@ -140,6 +140,42 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32253">
+                    <OrphaCode>693846</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=693846</ExpertLink>
+                    <Name lang="en">Hepatic arteriovenous malformation</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32269">
+                    <OrphaCode>694228</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=694228</ExpertLink>
+                    <Name lang="en">Congenital intrahepatic arterioportal fistula</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32396">
+                    <OrphaCode>699068</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=699068</ExpertLink>
+                    <Name lang="en">Fontan-associated liver disease</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="29788">
                     <OrphaCode>596937</OrphaCode>
@@ -199,7 +235,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="23">
+              <ClassificationNodeChildList count="24">
                 <ClassificationNode>
                   <Disorder id="873">
                     <OrphaCode>2137</OrphaCode>
@@ -524,6 +560,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32396">
+                    <OrphaCode>699068</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=699068</ExpertLink>
+                    <Name lang="en">Fontan-associated liver disease</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="28337">
                     <OrphaCode>562639</OrphaCode>
@@ -547,7 +595,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="39">
+              <ClassificationNodeChildList count="38">
                 <ClassificationNode>
                   <Disorder id="31389">
                     <OrphaCode>618899</OrphaCode>
@@ -1541,7 +1589,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="14">
+                  <ClassificationNodeChildList count="15">
                     <ClassificationNode>
                       <Disorder id="11344">
                         <OrphaCode>79318</OrphaCode>
@@ -1710,6 +1758,54 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32237">
+                        <OrphaCode>692790</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=692790</ExpertLink>
+                        <Name lang="en">ATP6AP1-CDG</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="20326">
+                    <OrphaCode>275761</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=275761</ExpertLink>
+                    <Name lang="en">Lysosomal acid lipase deficiency</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="2">
+                    <ClassificationNode>
+                      <Disorder id="11067">
+                        <OrphaCode>75233</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=75233</ExpertLink>
+                        <Name lang="en">Wolman disease</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="11068">
+                        <OrphaCode>75234</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=75234</ExpertLink>
+                        <Name lang="en">Cholesteryl ester storage disease</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -1868,30 +1964,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="11067">
-                    <OrphaCode>75233</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=75233</ExpertLink>
-                    <Name lang="en">Wolman disease</Name>
-                    <DisorderType id="21450">
-                      <Name lang="en">Clinical subtype</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="11068">
-                    <OrphaCode>75234</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=75234</ExpertLink>
-                    <Name lang="en">Cholesteryl ester storage disease</Name>
-                    <DisorderType id="21450">
-                      <Name lang="en">Clinical subtype</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="19528">
                     <OrphaCode>247585</OrphaCode>
@@ -1932,7 +2004,7 @@
                   <Disorder id="32009">
                     <OrphaCode>662388</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662388</ExpertLink>
-                    <Name lang="en">Gallbladder duplication</Name>
+                    <Name lang="en">Isolated gallbladder duplication</Name>
                     <DisorderType id="21415">
                       <Name lang="en">Morphological anomaly</Name>
                     </DisorderType>
@@ -2372,7 +2444,7 @@
                           <Disorder id="14551">
                             <OrphaCode>99978</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99978</ExpertLink>
-                            <Name lang="en">Klatskin tumor</Name>
+                            <Name lang="en">Perihilar cholangiocarcinoma</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
