commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml
index be5e9e3..71a4bd0 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_183.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:37:28" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:49:49" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -559,7 +559,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="41">
+              <ClassificationNodeChildList count="42">
                 <ClassificationNode>
                   <Disorder id="31389">
                     <OrphaCode>618899</OrphaCode>
@@ -1284,7 +1284,7 @@
                   <Disorder id="16911">
                     <OrphaCode>139507</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139507</ExpertLink>
-                    <Name lang="en">African iron overload</Name>
+                    <Name lang="en">Dietary iron overload disease</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -1313,24 +1313,12 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="4">
-                    <ClassificationNode>
-                      <Disorder id="11256">
-                        <OrphaCode>79230</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79230</ExpertLink>
-                        <Name lang="en">Hemochromatosis type 2</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="3">
                     <ClassificationNode>
-                      <Disorder id="16909">
-                        <OrphaCode>139491</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139491</ExpertLink>
-                        <Name lang="en">Hemochromatosis type 4</Name>
+                      <Disorder id="24056">
+                        <OrphaCode>465508</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=465508</ExpertLink>
+                        <Name lang="en">Symptomatic form of HFE-related hemochromatosis</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -1339,22 +1327,58 @@
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="18976">
-                        <OrphaCode>225123</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225123</ExpertLink>
-                        <Name lang="en">Hemochromatosis type 3</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                      <Disorder id="31842">
+                        <OrphaCode>648569</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648569</ExpertLink>
+                        <Name lang="en">Non-HFE-related hemochromatosis</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="3">
+                        <ClassificationNode>
+                          <Disorder id="18976">
+                            <OrphaCode>225123</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225123</ExpertLink>
+                            <Name lang="en">TFR2-related hemochromatosis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31821">
+                            <OrphaCode>647834</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647834</ExpertLink>
+                            <Name lang="en">SLC40A1-related hemochromatosis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="11256">
+                            <OrphaCode>79230</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79230</ExpertLink>
+                            <Name lang="en">HJV or HAMP-related hemochromatosis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="24056">
-                        <OrphaCode>465508</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=465508</ExpertLink>
-                        <Name lang="en">Symptomatic form of hemochromatosis type 1</Name>
+                      <Disorder id="31843">
+                        <OrphaCode>648581</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648581</ExpertLink>
+                        <Name lang="en">Digenic hemochromatosis</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -1844,6 +1868,18 @@
                     </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31841">
+                    <OrphaCode>648562</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648562</ExpertLink>
+                    <Name lang="en">Ferroportin Disease</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="10737">
                     <OrphaCode>53693</OrphaCode>
