commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml
index 5190d29..acb7e9a 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:39:54" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:52:35" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -21,7 +21,7 @@
               <Name lang="en">Category</Name>
             </DisorderType>
           </Disorder>
-          <ClassificationNodeChildList count="8">
+          <ClassificationNodeChildList count="9">
             <ClassificationNode>
               <Disorder id="11944">
                 <OrphaCode>90053</OrphaCode>
@@ -70,6 +70,18 @@
               <ClassificationNodeChildList count="0">
               </ClassificationNodeChildList>
             </ClassificationNode>
+            <ClassificationNode>
+              <Disorder id="31676">
+                <OrphaCode>633124</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633124</ExpertLink>
+                <Name lang="en">Invasive scopulariopsis infection</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="0">
+              </ClassificationNodeChildList>
+            </ClassificationNode>
             <ClassificationNode>
               <Disorder id="26441">
                 <OrphaCode>506207</OrphaCode>
@@ -387,7 +399,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="11">
+                      <ClassificationNodeChildList count="10">
                         <ClassificationNode>
                           <Disorder id="55">
                             <OrphaCode>262</OrphaCode>
@@ -568,18 +580,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="24171">
-                            <OrphaCode>466801</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=466801</ExpertLink>
-                            <Name lang="en">LIMS2-related limb-girdle muscular dystrophy</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
@@ -2811,24 +2811,12 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
-                        <ClassificationNode>
-                          <Disorder id="11256">
-                            <OrphaCode>79230</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79230</ExpertLink>
-                            <Name lang="en">Hemochromatosis type 2</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
-                          <Disorder id="16909">
-                            <OrphaCode>139491</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139491</ExpertLink>
-                            <Name lang="en">Hemochromatosis type 4</Name>
+                          <Disorder id="24056">
+                            <OrphaCode>465508</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=465508</ExpertLink>
+                            <Name lang="en">Symptomatic form of HFE-related hemochromatosis</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2837,22 +2825,58 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="24056">
-                            <OrphaCode>465508</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=465508</ExpertLink>
-                            <Name lang="en">Symptomatic form of hemochromatosis type 1</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                          <Disorder id="31842">
+                            <OrphaCode>648569</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648569</ExpertLink>
+                            <Name lang="en">Non-HFE-related hemochromatosis</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="3">
+                            <ClassificationNode>
+                              <Disorder id="11256">
+                                <OrphaCode>79230</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79230</ExpertLink>
+                                <Name lang="en">HJV or HAMP-related hemochromatosis</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="18976">
+                                <OrphaCode>225123</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225123</ExpertLink>
+                                <Name lang="en">TFR2-related hemochromatosis</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31821">
+                                <OrphaCode>647834</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647834</ExpertLink>
+                                <Name lang="en">SLC40A1-related hemochromatosis</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="18976">
-                            <OrphaCode>225123</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225123</ExpertLink>
-                            <Name lang="en">Hemochromatosis type 3</Name>
+                          <Disorder id="31843">
+                            <OrphaCode>648581</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648581</ExpertLink>
+                            <Name lang="en">Digenic hemochromatosis</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -4515,7 +4539,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
                         <ClassificationNode>
                           <Disorder id="19618">
                             <OrphaCode>251274</OrphaCode>
@@ -4528,6 +4552,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31732">
+                            <OrphaCode>642671</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642671</ExpertLink>
+                            <Name lang="en">Familial hyperaldosteronism type IV</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="913">
                             <OrphaCode>403</OrphaCode>
@@ -6334,7 +6370,7 @@
                       <Disorder id="2037">
                         <OrphaCode>2138</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2138</ExpertLink>
-                        <Name lang="en">46,XX ovotesticular disorder of sex development</Name>
+                        <Name lang="en">46,XX ovotesticular difference of sex development</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -6346,7 +6382,7 @@
                       <Disorder id="5546">
                         <OrphaCode>393</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=393</ExpertLink>
-                        <Name lang="en">46,XX testicular disorder of sex development</Name>
+                        <Name lang="en">46,XX testicular difference of sex development</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -6370,7 +6406,7 @@
                       <Disorder id="21840">
                         <OrphaCode>325357</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325357</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development due to impaired androgen production</Name>
+                        <Name lang="en">46,XY difference of sex development due to impaired androgen production</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -6428,7 +6464,7 @@
                           <Disorder id="12089">
                             <OrphaCode>90783</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90783</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to a testosterone synthesis defect</Name>
+                            <Name lang="en">46,XY difference of sex development due to a testosterone synthesis defect</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -6438,7 +6474,7 @@
                               <Disorder id="12091">
                                 <OrphaCode>90787</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90787</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to testicular steroidogenesis defect</Name>
+                                <Name lang="en">46,XY difference of sex development due to testicular steroidogenesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -6448,7 +6484,7 @@
                                   <Disorder id="12099">
                                     <OrphaCode>90796</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90796</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to isolated 17,20-lyase deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development due to isolated 17,20-lyase deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -6460,7 +6496,7 @@
                                   <Disorder id="203">
                                     <OrphaCode>752</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=752</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -6474,7 +6510,7 @@
                               <Disorder id="21842">
                                 <OrphaCode>325511</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325511</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to a cholesterol synthesis defect</Name>
+                                <Name lang="en">46,XY difference of sex development due to a cholesterol synthesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -6498,7 +6534,7 @@
                               <Disorder id="12090">
                                 <OrphaCode>90786</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90786</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to adrenal and testicular steroidogenesis defect</Name>
+                                <Name lang="en">46,XY difference of sex development due to adrenal and testicular steroidogenesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -6508,7 +6544,7 @@
                                   <Disorder id="17771">
                                     <OrphaCode>168558</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168558</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -6608,7 +6644,7 @@
                           <Disorder id="13104">
                             <OrphaCode>98086</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98086</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
+                            <Name lang="en">46,XY difference of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -6618,7 +6654,7 @@
                               <Disorder id="324">
                                 <OrphaCode>753</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=753</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to 5-alpha-reductase 2 deficiency</Name>
+                                <Name lang="en">46,XY difference of sex development due to 5-alpha-reductase 2 deficiency</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -6632,7 +6668,7 @@
                           <Disorder id="23441">
                             <OrphaCode>443090</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443090</ExpertLink>
-                            <Name lang="en">46,XY disorder of sexual development due to dihydrotestosterone backdoor pathway biosynthesis defect</Name>
+                            <Name lang="en">46,XY difference of sexual development due to dihydrotestosterone backdoor pathway biosynthesis defect</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -6642,7 +6678,7 @@
                               <Disorder id="23440">
                                 <OrphaCode>443087</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443087</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to testicular 17,20-desmolase deficiency</Name>
+                                <Name lang="en">46,XY difference of sex development due to testicular 17,20-desmolase deficiency</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -6658,7 +6694,7 @@
                       <Disorder id="21845">
                         <OrphaCode>325537</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325537</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development induced by maternal exposure to endocrine disruptors</Name>
+                        <Name lang="en">46,XY difference of sex development induced by maternal exposure to endocrine disruptors</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -6682,7 +6718,7 @@
                       <Disorder id="21838">
                         <OrphaCode>325345</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325345</ExpertLink>
-                        <Name lang="en">46,XY ovotesticular disorder of sex development</Name>
+                        <Name lang="en">46,XY ovotesticular difference of sex development</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -8529,7 +8565,43 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="0">
+                                  <ClassificationNodeChildList count="3">
+                                    <ClassificationNode>
+                                      <Disorder id="31785">
+                                        <OrphaCode>645617</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645617</ExpertLink>
+                                        <Name lang="en">Amyopathic dermatomyositis</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31786">
+                                        <OrphaCode>645626</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645626</ExpertLink>
+                                        <Name lang="en">Adermatopathic dermatomyositis</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31784">
+                                        <OrphaCode>645613</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645613</ExpertLink>
+                                        <Name lang="en">Classical dermatomyositis</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -9265,7 +9337,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="18">
+                          <ClassificationNodeChildList count="17">
                             <ClassificationNode>
                               <Disorder id="392">
                                 <OrphaCode>2614</OrphaCode>
@@ -9422,18 +9494,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="19815">
-                                <OrphaCode>255249</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=255249</ExpertLink>
-                                <Name lang="en">Leigh syndrome with nephrotic syndrome</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="20470">
                                 <OrphaCode>280406</OrphaCode>
@@ -10842,7 +10902,7 @@
                               <Disorder id="722">
                                 <OrphaCode>1201</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1201</ExpertLink>
-                                <Name lang="en">Atresia of small intestine</Name>
+                                <Name lang="en">Small bowel atresia</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -11478,7 +11538,7 @@
                               <Disorder id="1092">
                                 <OrphaCode>1590</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1590</ExpertLink>
-                                <Name lang="en">Distal monosomy 13q</Name>
+                                <Name lang="en">Distal deletion 13q</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -11718,7 +11778,7 @@
                               <Disorder id="2679">
                                 <OrphaCode>2973</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2973</ExpertLink>
-                                <Name lang="en">46,XX disorder of sex development-anorectal anomalies syndrome</Name>
+                                <Name lang="en">46,XX difference of sex development-anorectal anomalies syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -12246,7 +12306,7 @@
                               <Disorder id="722">
                                 <OrphaCode>1201</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1201</ExpertLink>
-                                <Name lang="en">Atresia of small intestine</Name>
+                                <Name lang="en">Small bowel atresia</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -14146,7 +14206,7 @@
                       <Disorder id="18947">
                         <OrphaCode>221139</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=221139</ExpertLink>
-                        <Name lang="en">Combined immunodeficiency with faciooculoskeletal anomalies</Name>
+                        <Name lang="en">Combined immunodeficiency with facio-oculo-skeletal anomalies</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -14511,7 +14571,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="29">
+                      <ClassificationNodeChildList count="30">
                         <ClassificationNode>
                           <Disorder id="14385">
                             <OrphaCode>99812</OrphaCode>
@@ -14668,6 +14728,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31816">
+                            <OrphaCode>647804</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647804</ExpertLink>
+                            <Name lang="en">Combined immunodeficiency due to FCHO1 deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="17816">
                             <OrphaCode>169082</OrphaCode>
@@ -15346,7 +15418,7 @@
                       <Disorder id="18195">
                         <OrphaCode>182040</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=182040</ExpertLink>
-                        <Name lang="en">Aplastic anemia</Name>
+                        <Name lang="en">Rare aplastic anemia</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -15534,7 +15606,7 @@
                               <Disorder id="429">
                                 <OrphaCode>124</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=124</ExpertLink>
-                                <Name lang="en">Blackfan-Diamond anemia</Name>
+                                <Name lang="en">Diamond-Blackfan anemia</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -15941,7 +16013,19 @@
                                               <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
-                                          <ClassificationNodeChildList count="7">
+                                          <ClassificationNodeChildList count="9">
+                                            <ClassificationNode>
+                                              <Disorder id="31718">
+                                                <OrphaCode>641375</OrphaCode>
+                                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641375</ExpertLink>
+                                                <Name lang="en">B-lymphoblastic leukemia/lymphoma with t(17;19)</Name>
+                                                <DisorderType id="21443">
+                                                  <Name lang="en">Etiological subtype</Name>
+                                                </DisorderType>
+                                              </Disorder>
+                                              <ClassificationNodeChildList count="0">
+                                              </ClassificationNodeChildList>
+                                            </ClassificationNode>
                                             <ClassificationNode>
                                               <Disorder id="29211">
                                                 <OrphaCode>585909</OrphaCode>
@@ -16026,6 +16110,18 @@
                                               <ClassificationNodeChildList count="0">
                                               </ClassificationNodeChildList>
                                             </ClassificationNode>
+                                            <ClassificationNode>
+                                              <Disorder id="31717">
+                                                <OrphaCode>641372</OrphaCode>
+                                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641372</ExpertLink>
+                                                <Name lang="en">B-lymphoblastic leukemia/lymphoma with t(7;9)(q11.2;p13.2)</Name>
+                                                <DisorderType id="21443">
+                                                  <Name lang="en">Etiological subtype</Name>
+                                                </DisorderType>
+                                              </Disorder>
+                                              <ClassificationNodeChildList count="0">
+                                              </ClassificationNodeChildList>
+                                            </ClassificationNode>
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
                                       </ClassificationNodeChildList>
@@ -16538,7 +16634,7 @@
                                       <Disorder id="11764">
                                         <OrphaCode>86870</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86870</ExpertLink>
-                                        <Name lang="en">CD4+/CD56+ hematodermic neoplasm</Name>
+                                        <Name lang="en">Blastic plasmacytoid dendritic cell neoplasm</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -18915,7 +19011,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="21">
+                      <ClassificationNodeChildList count="20">
                         <ClassificationNode>
                           <Disorder id="18682">
                             <OrphaCode>210110</OrphaCode>
@@ -19084,18 +19180,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2533">
-                            <OrphaCode>1306</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1306</ExpertLink>
-                            <Name lang="en">Buschke-Ollendorff syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="12544">
                             <OrphaCode>94063</OrphaCode>
@@ -19655,11 +19739,35 @@
                         <OrphaCode>3188</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3188</ExpertLink>
                         <Name lang="en">Congenital pulmonary veins atresia or stenosis</Name>
-                        <DisorderType id="21415">
-                          <Name lang="en">Morphological anomaly</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="14143">
+                            <OrphaCode>99126</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99126</ExpertLink>
+                            <Name lang="en">Congenital pulmonary vein atresia</Name>
+                            <DisorderType id="21415">
+                              <Name lang="en">Morphological anomaly</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31729">
+                            <OrphaCode>642071</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642071</ExpertLink>
+                            <Name lang="en">Primary pulmonary vein stenosis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
