commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml
index 6dd6540..fc35925 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:34:13" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:34:49" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -4169,7 +4169,7 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="19">
+                  <ClassificationNodeChildList count="18">
                     <ClassificationNode>
                       <Disorder id="12291">
                         <OrphaCode>93360</OrphaCode>
@@ -4338,18 +4338,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="20904">
-                        <OrphaCode>294049</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294049</ExpertLink>
-                        <Name lang="en">Reunion Island Larsen-like syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="12290">
                         <OrphaCode>93359</OrphaCode>
@@ -7153,7 +7141,31 @@
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="10819">
+                                <OrphaCode>63269</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63269</ExpertLink>
+                                <Name lang="en">Antley-Bixler syndrome with genital anomaly and disorder of steroidogenesis</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="29762">
+                                <OrphaCode>596008</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596008</ExpertLink>
+                                <Name lang="en">Antley-Bixler syndrome without genital anomaly or disorder of steroidogenesis</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
