commit 7ab24fe976be5c798004246aa079a429e568689b
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Jun 30 11:46:37 2025 +0200

    JULY 2025

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml
old mode 100644
new mode 100755
index e293780..ed92eea
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_148.xml	
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-12-03 07:05:10" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<?xml version="1.0" encoding="UTF-8"?>
+<JDBOR date="2025-06-24 07:32:06" version="1.3.42 / 4.1.8 [2025-03-03]" copyright="Orphanet (c) 2025" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2743,7 +2743,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="48">
+              <ClassificationNodeChildList count="49">
                 <ClassificationNode>
                   <Disorder id="25408">
                     <OrphaCode>488618</OrphaCode>
@@ -3516,7 +3516,7 @@
                   <Disorder id="32157">
                     <OrphaCode>684305</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684305</ExpertLink>
-                    <Name lang="en">NOCGUS syndrome</Name>
+                    <Name lang="en">Neurooculocardiogenitourinary syndrome</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -3560,6 +3560,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31803">
+                    <OrphaCode>646278</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=646278</ExpertLink>
+                    <Name lang="en">CDK13-related developmental delay-intellectual disability-facial dysmorphism-congenital heart defects syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
           </ClassificationNodeChildList>
