commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml
index d57f2d5..f755aa0 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:29:19" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:15:41" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <ClassificationList count="1">
     <Classification id="150">
       <OrphaNumber>156449</OrphaNumber>
@@ -380,7 +373,7 @@
                           <Disorder id="25220">
                             <OrphaCode>480682</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=480682</ExpertLink>
-                            <Name lang="en">POGLUT1-related  limb-girdle muscular dystrophy R21</Name>
+                            <Name lang="en">POGLUT1-related limb-girdle muscular dystrophy R21</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -596,7 +589,7 @@
                           <Disorder id="10337">
                             <OrphaCode>34515</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-                            <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+                            <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -608,7 +601,7 @@
                           <Disorder id="11732">
                             <OrphaCode>86812</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-                            <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+                            <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -620,7 +613,7 @@
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -632,7 +625,7 @@
                           <Disorder id="18522">
                             <OrphaCode>206559</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-                            <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+                            <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -644,7 +637,7 @@
                           <Disorder id="18523">
                             <OrphaCode>206564</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206564</ExpertLink>
-                            <Name lang="en">POMGNT1-related  limb-girdle muscular dystrophy R15</Name>
+                            <Name lang="en">POMGNT1-related limb-girdle muscular dystrophy R15</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -656,7 +649,7 @@
                           <Disorder id="22066">
                             <OrphaCode>352479</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-                            <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+                            <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -668,7 +661,7 @@
                           <Disorder id="22329">
                             <OrphaCode>363623</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-                            <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+                            <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
