commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml
index 7af8bd4..3a50357 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:38:14" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:50:43" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -1517,7 +1517,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="35">
+                  <ClassificationNodeChildList count="36">
                     <ClassificationNode>
                       <Disorder id="387">
                         <OrphaCode>819</OrphaCode>
@@ -2166,6 +2166,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31815">
+                        <OrphaCode>647799</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647799</ExpertLink>
+                        <Name lang="en">MYT1L-related developmental delay-intellectual disability-obesity syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
               </ClassificationNodeChildList>
@@ -3763,37 +3775,109 @@
                   </Disorder>
                   <ClassificationNodeChildList count="2">
                     <ClassificationNode>
-                      <Disorder id="8674">
-                        <OrphaCode>759</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=759</ExpertLink>
-                        <Name lang="en">Central precocious puberty</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                      <Disorder id="31881">
+                        <OrphaCode>650063</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650063</ExpertLink>
+                        <Name lang="en">Rare central precocious puberty</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
                         <ClassificationNode>
-                          <Disorder id="17866">
-                            <OrphaCode>169615</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=169615</ExpertLink>
-                            <Name lang="en">Idiopathic central precocious puberty</Name>
-                            <DisorderType id="21443">
-                              <Name lang="en">Etiological subtype</Name>
+                          <Disorder id="31867">
+                            <OrphaCode>649929</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=649929</ExpertLink>
+                            <Name lang="en">Central precocious puberty in male</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="31885">
+                                <OrphaCode>650087</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650087</ExpertLink>
+                                <Name lang="en">Primary central precocious puberty in male</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="31887">
+                                    <OrphaCode>650097</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650097</ExpertLink>
+                                    <Name lang="en">Genetic central precocious puberty in male</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31888">
+                                    <OrphaCode>650102</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650102</ExpertLink>
+                                    <Name lang="en">Non-genetic central precocious puberty in male</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31886">
+                                <OrphaCode>650092</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650092</ExpertLink>
+                                <Name lang="en">Secondary central precocious puberty in male</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="17867">
-                            <OrphaCode>169618</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=169618</ExpertLink>
-                            <Name lang="en">Secondary central precocious puberty</Name>
-                            <DisorderType id="21443">
-                              <Name lang="en">Etiological subtype</Name>
+                          <Disorder id="31882">
+                            <OrphaCode>650070</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650070</ExpertLink>
+                            <Name lang="en">Rare central precocious puberty in female</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="31883">
+                                <OrphaCode>650077</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650077</ExpertLink>
+                                <Name lang="en">Genetic central precocious puberty in female</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31884">
+                                <OrphaCode>650082</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=650082</ExpertLink>
+                                <Name lang="en">Secondary central precocious puberty in female</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -3824,7 +3908,7 @@
                           <Disorder id="1225">
                             <OrphaCode>3000</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3000</ExpertLink>
-                            <Name lang="en">Familial male-limited precocious puberty</Name>
+                            <Name lang="en">Familial peripheral male-limited precocious puberty</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3893,7 +3977,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="17">
+                  <ClassificationNodeChildList count="18">
                     <ClassificationNode>
                       <Disorder id="44">
                         <OrphaCode>881</OrphaCode>
@@ -3999,7 +4083,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31739">
+                            <OrphaCode>642945</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 1</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31742">
+                            <OrphaCode>642976</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                            <Name lang="en">Perrault syndrome type 2</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -4158,6 +4266,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31734">
+                        <OrphaCode>642691</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642691</ExpertLink>
+                        <Name lang="en">Fragile X-associated primary ovarian insufficiency</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -4274,7 +4394,7 @@
               <Disorder id="12085">
                 <OrphaCode>90771</OrphaCode>
                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90771</ExpertLink>
-                <Name lang="en">Disorder of sex development</Name>
+                <Name lang="en">Difference of sex development</Name>
                 <DisorderType id="36561">
                   <Name lang="en">Category</Name>
                 </DisorderType>
@@ -4284,7 +4404,7 @@
                   <Disorder id="569">
                     <OrphaCode>2982</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2982</ExpertLink>
-                    <Name lang="en">46,XX disorder of sex development</Name>
+                    <Name lang="en">46,XX difference of sex development</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -4294,7 +4414,7 @@
                       <Disorder id="13096">
                         <OrphaCode>98078</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98078</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development induced by androgens excess</Name>
+                        <Name lang="en">46,XX difference of sex development induced by androgens excess</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -4304,7 +4424,7 @@
                           <Disorder id="12087">
                             <OrphaCode>90776</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90776</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by fetal androgens excess</Name>
+                            <Name lang="en">46,XX difference of sex development induced by fetal androgens excess</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -4412,7 +4532,7 @@
                           <Disorder id="12119">
                             <OrphaCode>91144</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=91144</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by maternal-derived androgen</Name>
+                            <Name lang="en">46,XX difference of sex development induced by maternal-derived androgen</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -4422,7 +4542,7 @@
                               <Disorder id="21829">
                                 <OrphaCode>325093</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325093</ExpertLink>
-                                <Name lang="en">46,XX disorder of sex development induced by endogenous maternal-derived androgen</Name>
+                                <Name lang="en">46,XX difference of sex development induced by endogenous maternal-derived androgen</Name>
                                 <DisorderType id="21436">
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
@@ -4434,7 +4554,7 @@
                               <Disorder id="21830">
                                 <OrphaCode>325099</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325099</ExpertLink>
-                                <Name lang="en">46,XX disorder of sex development induced by exogenous maternal-derived androgen</Name>
+                                <Name lang="en">46,XX difference of sex development induced by exogenous maternal-derived androgen</Name>
                                 <DisorderType id="21436">
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
@@ -4448,7 +4568,7 @@
                           <Disorder id="21828">
                             <OrphaCode>325061</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325061</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by fetoplacental androgens excess</Name>
+                            <Name lang="en">46,XX difference of sex development induced by fetoplacental androgens excess</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -4496,7 +4616,7 @@
                           <Disorder id="2037">
                             <OrphaCode>2138</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2138</ExpertLink>
-                            <Name lang="en">46,XX ovotesticular disorder of sex development</Name>
+                            <Name lang="en">46,XX ovotesticular difference of sex development</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4508,7 +4628,7 @@
                           <Disorder id="5546">
                             <OrphaCode>393</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=393</ExpertLink>
-                            <Name lang="en">46,XX testicular disorder of sex development</Name>
+                            <Name lang="en">46,XX testicular difference of sex development</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4534,7 +4654,7 @@
                       <Disorder id="21831">
                         <OrphaCode>325109</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325109</ExpertLink>
-                        <Name lang="en">Syndrome with 46,XX disorder of sex development</Name>
+                        <Name lang="en">Syndrome with 46,XX difference of sex development</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -4561,14 +4681,38 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="31739">
+                                <OrphaCode>642945</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                                <Name lang="en">Perrault syndrome type 1</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31742">
+                                <OrphaCode>642976</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                                <Name lang="en">Perrault syndrome type 2</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2679">
                             <OrphaCode>2973</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2973</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development-anorectal anomalies syndrome</Name>
+                            <Name lang="en">46,XX difference of sex development-anorectal anomalies syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4580,7 +4724,7 @@
                           <Disorder id="2681">
                             <OrphaCode>2975</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2975</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development-skeletal anomalies syndrome</Name>
+                            <Name lang="en">46,XX difference of sex development-skeletal anomalies syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4644,7 +4788,7 @@
                   <Disorder id="13103">
                     <OrphaCode>98085</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98085</ExpertLink>
-                    <Name lang="en">46,XY disorder of sex development</Name>
+                    <Name lang="en">46,XY difference of sex development</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -4666,7 +4810,7 @@
                       <Disorder id="13105">
                         <OrphaCode>98087</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98087</ExpertLink>
-                        <Name lang="en">Syndrome with 46,XY disorder of sex development</Name>
+                        <Name lang="en">Syndrome with 46,XY difference of sex development</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -4688,7 +4832,7 @@
                           <Disorder id="334">
                             <OrphaCode>1642</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1642</ExpertLink>
-                            <Name lang="en">Distal monosomy 9p</Name>
+                            <Name lang="en">Distal deletion 9p</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4712,7 +4856,7 @@
                           <Disorder id="570">
                             <OrphaCode>2983</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2983</ExpertLink>
-                            <Name lang="en">Disorder of sex development-intellectual disability syndrome</Name>
+                            <Name lang="en">Difference of sex development-intellectual disability syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -4724,7 +4868,7 @@
                           <Disorder id="606">
                             <OrphaCode>1422</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1422</ExpertLink>
-                            <Name lang="en">Chondrodysplasia-disorder of sex development syndrome</Name>
+                            <Name lang="en">Chondrodysplasia-difference of sex development syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4772,7 +4916,7 @@
                           <Disorder id="2151">
                             <OrphaCode>2282</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2282</ExpertLink>
-                            <Name lang="en">Dysmorphism-short stature-deafness-disorder of sex development syndrome</Name>
+                            <Name lang="en">Dysmorphism-short stature-deafness-difference of sex development syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -4964,7 +5108,7 @@
                           <Disorder id="21838">
                             <OrphaCode>325345</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325345</ExpertLink>
-                            <Name lang="en">46,XY ovotesticular disorder of sex development</Name>
+                            <Name lang="en">46,XY ovotesticular difference of sex development</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -4978,7 +5122,7 @@
                       <Disorder id="21839">
                         <OrphaCode>325351</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325351</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development of endocrine origin</Name>
+                        <Name lang="en">46,XY difference of sex development of endocrine origin</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -5036,7 +5180,7 @@
                           <Disorder id="21840">
                             <OrphaCode>325357</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325357</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to impaired androgen production</Name>
+                            <Name lang="en">46,XY difference of sex development due to impaired androgen production</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -5094,7 +5238,7 @@
                               <Disorder id="12089">
                                 <OrphaCode>90783</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90783</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to a testosterone synthesis defect</Name>
+                                <Name lang="en">46,XY difference of sex development due to a testosterone synthesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -5104,7 +5248,7 @@
                                   <Disorder id="12090">
                                     <OrphaCode>90786</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90786</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to adrenal and testicular steroidogenesis defect</Name>
+                                    <Name lang="en">46,XY difference of sex development due to adrenal and testicular steroidogenesis defect</Name>
                                     <DisorderType id="36561">
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
@@ -5186,7 +5330,7 @@
                                       <Disorder id="17771">
                                         <OrphaCode>168558</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168558</ExpertLink>
-                                        <Name lang="en">46,XY disorder of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
+                                        <Name lang="en">46,XY difference of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -5212,7 +5356,7 @@
                                   <Disorder id="12091">
                                     <OrphaCode>90787</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90787</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to testicular steroidogenesis defect</Name>
+                                    <Name lang="en">46,XY difference of sex development due to testicular steroidogenesis defect</Name>
                                     <DisorderType id="36561">
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
@@ -5222,7 +5366,7 @@
                                       <Disorder id="203">
                                         <OrphaCode>752</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=752</ExpertLink>
-                                        <Name lang="en">46,XY disorder of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
+                                        <Name lang="en">46,XY difference of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -5234,7 +5378,7 @@
                                       <Disorder id="12099">
                                         <OrphaCode>90796</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90796</ExpertLink>
-                                        <Name lang="en">46,XY disorder of sex development due to isolated 17,20-lyase deficiency</Name>
+                                        <Name lang="en">46,XY difference of sex development due to isolated 17,20-lyase deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -5248,7 +5392,7 @@
                                   <Disorder id="21842">
                                     <OrphaCode>325511</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325511</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to a cholesterol synthesis defect</Name>
+                                    <Name lang="en">46,XY difference of sex development due to a cholesterol synthesis defect</Name>
                                     <DisorderType id="36561">
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
@@ -5274,7 +5418,7 @@
                               <Disorder id="13104">
                                 <OrphaCode>98086</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98086</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
+                                <Name lang="en">46,XY difference of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -5284,7 +5428,7 @@
                                   <Disorder id="324">
                                     <OrphaCode>753</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=753</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to 5-alpha-reductase 2 deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development due to 5-alpha-reductase 2 deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -5298,7 +5442,7 @@
                               <Disorder id="23441">
                                 <OrphaCode>443090</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443090</ExpertLink>
-                                <Name lang="en">46,XY disorder of sexual development due to dihydrotestosterone backdoor pathway biosynthesis defect</Name>
+                                <Name lang="en">46,XY difference of sexual development due to dihydrotestosterone backdoor pathway biosynthesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -5308,7 +5452,7 @@
                                   <Disorder id="23440">
                                     <OrphaCode>443087</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443087</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to testicular 17,20-desmolase deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development due to testicular 17,20-desmolase deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -5324,7 +5468,7 @@
                           <Disorder id="21845">
                             <OrphaCode>325537</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325537</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development induced by maternal exposure to endocrine disruptors</Name>
+                            <Name lang="en">46,XY difference of sex development induced by maternal exposure to endocrine disruptors</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -5340,7 +5484,7 @@
                   <Disorder id="21846">
                     <OrphaCode>325546</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325546</ExpertLink>
-                    <Name lang="en">Sex chromosome disorder of sex development</Name>
+                    <Name lang="en">Sex chromosome difference of sex development</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -6885,83 +7029,95 @@
               </Disorder>
               <ClassificationNodeChildList count="8">
                 <ClassificationNode>
-                  <Disorder id="8667">
-                    <OrphaCode>553</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=553</ExpertLink>
-                    <Name lang="en">Cushing syndrome</Name>
-                    <DisorderType id="21436">
-                      <Name lang="en">Clinical group</Name>
+                  <Disorder id="31726">
+                    <OrphaCode>641613</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641613</ExpertLink>
+                    <Name lang="en">Endogenous Cushing syndrome</Name>
+                    <DisorderType id="36561">
+                      <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="4">
-                    <ClassificationNode>
-                      <Disorder id="14465">
-                        <OrphaCode>99892</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99892</ExpertLink>
-                        <Name lang="en">ACTH-dependent Cushing syndrome</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="2">
-                        <ClassificationNode>
-                          <Disorder id="12827">
-                            <OrphaCode>96253</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96253</ExpertLink>
-                            <Name lang="en">Cushing disease</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="14462">
-                            <OrphaCode>99889</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99889</ExpertLink>
-                            <Name lang="en">Cushing syndrome due to ectopic ACTH secretion</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="2">
                     <ClassificationNode>
-                      <Disorder id="14466">
-                        <OrphaCode>99893</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99893</ExpertLink>
-                        <Name lang="en">ACTH-independent Cushing syndrome</Name>
+                      <Disorder id="31809">
+                        <OrphaCode>647758</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647758</ExpertLink>
+                        <Name lang="en">Adrenal Cushing syndrome</Name>
                         <DisorderType id="21436">
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
-                        <ClassificationNode>
-                          <Disorder id="18376">
-                            <OrphaCode>189439</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=189439</ExpertLink>
-                            <Name lang="en">Primary pigmented nodular adrenocortical disease</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
-                          <Disorder id="23454">
-                            <OrphaCode>443287</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443287</ExpertLink>
-                            <Name lang="en">ACTH-independent Cushing syndrome due to rare cortisol-producing adrenal tumor</Name>
+                          <Disorder id="21523">
+                            <OrphaCode>314749</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314749</ExpertLink>
+                            <Name lang="en">Rare disease with adrenal Cushing syndrome as a major feature</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="1">
+                          <ClassificationNodeChildList count="7">
+                            <ClassificationNode>
+                              <Disorder id="121">
+                                <OrphaCode>652</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652</ExpertLink>
+                                <Name lang="en">Multiple endocrine neoplasia type 1</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="196">
+                                <OrphaCode>524</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=524</ExpertLink>
+                                <Name lang="en">Li-Fraumeni syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="279">
+                                <OrphaCode>562</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=562</ExpertLink>
+                                <Name lang="en">McCune-Albright syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="2389">
+                                <OrphaCode>1359</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1359</ExpertLink>
+                                <Name lang="en">Carney complex</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="2682">
+                                <OrphaCode>2976</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2976</ExpertLink>
+                                <Name lang="en">Pseudoleprechaunism syndrome, Patterson type</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="3731">
                                 <OrphaCode>1501</OrphaCode>
@@ -6974,37 +7130,49 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31810">
+                                <OrphaCode>647768</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647768</ExpertLink>
+                                <Name lang="en">Rare adrenocortical nodular disease with Cushing syndrome as a major feature</Name>
+                                <DisorderType id="21436">
+                                  <Name lang="en">Clinical group</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="31811">
+                                    <OrphaCode>647772</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647772</ExpertLink>
+                                    <Name lang="en">Isolated primary pigmented nodular adrenocortical disease</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31812">
+                                    <OrphaCode>647782</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647782</ExpertLink>
+                                    <Name lang="en">Isolated micronodular adrenocortical disease</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="18374">
-                        <OrphaCode>189427</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=189427</ExpertLink>
-                        <Name lang="en">Cushing syndrome due to macronodular adrenal hyperplasia</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="21523">
-                        <OrphaCode>314749</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314749</ExpertLink>
-                        <Name lang="en">Rare disease with Cushing syndrome as a major feature</Name>
-                        <DisorderType id="36561">
-                          <Name lang="en">Category</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="5">
                         <ClassificationNode>
-                          <Disorder id="121">
-                            <OrphaCode>652</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652</ExpertLink>
-                            <Name lang="en">Multiple endocrine neoplasia type 1</Name>
+                          <Disorder id="31738">
+                            <OrphaCode>642788</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642788</ExpertLink>
+                            <Name lang="en">Cushing syndrome due to cortisol-producing adrenocortical adenoma</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -7013,10 +7181,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="196">
-                            <OrphaCode>524</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=524</ExpertLink>
-                            <Name lang="en">Li-Fraumeni syndrome</Name>
+                          <Disorder id="18374">
+                            <OrphaCode>189427</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=189427</ExpertLink>
+                            <Name lang="en">Cushing syndrome due to bilateral macronodular adrenocortical disease</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -7024,11 +7192,23 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="14465">
+                        <OrphaCode>99892</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99892</ExpertLink>
+                        <Name lang="en">ACTH-dependent Cushing syndrome</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
-                          <Disorder id="279">
-                            <OrphaCode>562</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=562</ExpertLink>
-                            <Name lang="en">McCune-Albright syndrome</Name>
+                          <Disorder id="12827">
+                            <OrphaCode>96253</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96253</ExpertLink>
+                            <Name lang="en">Cushing disease</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -7037,10 +7217,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="2389">
-                            <OrphaCode>1359</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1359</ExpertLink>
-                            <Name lang="en">Carney complex</Name>
+                          <Disorder id="14462">
+                            <OrphaCode>99889</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99889</ExpertLink>
+                            <Name lang="en">Cushing syndrome due to ectopic ACTH secretion</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -7048,18 +7228,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2682">
-                            <OrphaCode>2976</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2976</ExpertLink>
-                            <Name lang="en">Pseudoleprechaunism syndrome, Patterson type</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -7085,7 +7253,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="9">
+                  <ClassificationNodeChildList count="10">
                     <ClassificationNode>
                       <Disorder id="3731">
                         <OrphaCode>1501</OrphaCode>
@@ -7110,6 +7278,42 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31860">
+                        <OrphaCode>649017</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=649017</ExpertLink>
+                        <Name lang="en">Rare adrenocortical nodular disease</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31811">
+                            <OrphaCode>647772</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647772</ExpertLink>
+                            <Name lang="en">Isolated primary pigmented nodular adrenocortical disease</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31812">
+                            <OrphaCode>647782</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647782</ExpertLink>
+                            <Name lang="en">Isolated micronodular adrenocortical disease</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="121">
                         <OrphaCode>652</OrphaCode>
@@ -7386,7 +7590,7 @@
                               <Disorder id="17771">
                                 <OrphaCode>168558</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168558</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
+                                <Name lang="en">46,XY difference of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -7993,7 +8197,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="3">
+                          <ClassificationNodeChildList count="4">
                             <ClassificationNode>
                               <Disorder id="913">
                                 <OrphaCode>403</OrphaCode>
@@ -8030,6 +8234,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31732">
+                                <OrphaCode>642671</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642671</ExpertLink>
+                                <Name lang="en">Familial hyperaldosteronism type IV</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -8232,7 +8448,7 @@
                   <Disorder id="14659">
                     <OrphaCode>100087</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100087</ExpertLink>
-                    <Name lang="en">Thyroid tumor</Name>
+                    <Name lang="en">Rare thyroid tumor</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -8242,7 +8458,7 @@
                       <Disorder id="14660">
                         <OrphaCode>100088</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=100088</ExpertLink>
-                        <Name lang="en">Thyroid carcinoma</Name>
+                        <Name lang="en">Rare thyroid carcinoma</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -8995,7 +9211,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="7">
+              <ClassificationNodeChildList count="6">
                 <ClassificationNode>
                   <Disorder id="1471">
                     <OrphaCode>1227</OrphaCode>
@@ -9152,18 +9368,6 @@
                     </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="20490">
-                    <OrphaCode>280651</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280651</ExpertLink>
-                    <Name lang="en">Acrodysostosis with multiple hormone resistance</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="20557">
                     <OrphaCode>282196</OrphaCode>
@@ -10215,7 +10419,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="9">
+                      <ClassificationNodeChildList count="10">
                         <ClassificationNode>
                           <Disorder id="12627">
                             <OrphaCode>95503</OrphaCode>
@@ -10647,10 +10851,10 @@
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
-                              <Disorder id="863">
-                                <OrphaCode>3389</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3389</ExpertLink>
-                                <Name lang="en">Tuberculosis</Name>
+                              <Disorder id="31796">
+                                <OrphaCode>645854</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645854</ExpertLink>
+                                <Name lang="en">Multifocal tuberculosis</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10756,6 +10960,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31713">
+                            <OrphaCode>641350</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641350</ExpertLink>
+                            <Name lang="en">Immunotherapy induced hypophysitis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -11166,7 +11382,7 @@
                               <Disorder id="11325">
                                 <OrphaCode>79299</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79299</ExpertLink>
-                                <Name lang="en">Hyperinsulinism due to glucokinase deficiency</Name>
+                                <Name lang="en">Congenital glucokinase-related hyperinsulinism</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -11190,7 +11406,7 @@
                               <Disorder id="20056">
                                 <OrphaCode>263455</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263455</ExpertLink>
-                                <Name lang="en">Hyperinsulinism due to HNF4A deficiency</Name>
+                                <Name lang="en">Congenital hyperinsulinism due to HNF4A deficiency</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
