commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml
index 0e316ee..f166f5f 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:39:18" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:51:59" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -31,7 +31,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="41">
+              <ClassificationNodeChildList count="40">
                 <ClassificationNode>
                   <Disorder id="725">
                     <OrphaCode>1304</OrphaCode>
@@ -289,11 +289,107 @@
                     <OrphaCode>3389</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3389</ExpertLink>
                     <Name lang="en">Tuberculosis</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
+                    <DisorderType id="21436">
+                      <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="0">
+                  <ClassificationNodeChildList count="8">
+                    <ClassificationNode>
+                      <Disorder id="31723">
+                        <OrphaCode>641396</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=641396</ExpertLink>
+                        <Name lang="en">Central nervous system tuberculosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31792">
+                        <OrphaCode>645807</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645807</ExpertLink>
+                        <Name lang="en">Primary tuberculous lymphadenitis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31797">
+                        <OrphaCode>645859</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645859</ExpertLink>
+                        <Name lang="en">Primary tuberculosis of the digestive system</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31798">
+                        <OrphaCode>645874</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645874</ExpertLink>
+                        <Name lang="en">Primary genito-urinary tuberculosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31796">
+                        <OrphaCode>645854</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645854</ExpertLink>
+                        <Name lang="en">Multifocal tuberculosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31795">
+                        <OrphaCode>645849</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645849</ExpertLink>
+                        <Name lang="en">Primary cutaneous tuberculosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31793">
+                        <OrphaCode>645814</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645814</ExpertLink>
+                        <Name lang="en">Primary pulmonary tuberculosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31794">
+                        <OrphaCode>645822</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=645822</ExpertLink>
+                        <Name lang="en">Primary bone and joint tuberculosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -860,18 +956,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="25940">
-                    <OrphaCode>499004</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=499004</ExpertLink>
-                    <Name lang="en">Tuberculous meningitis</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="28678">
                     <OrphaCode>570762</OrphaCode>
@@ -1277,7 +1361,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="16">
+                  <ClassificationNodeChildList count="17">
                     <ClassificationNode>
                       <Disorder id="3275">
                         <OrphaCode>1930</OrphaCode>
@@ -1470,6 +1554,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31703">
+                        <OrphaCode>637051</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637051</ExpertLink>
+                        <Name lang="en">Borna virus encephalitis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -1968,7 +2064,7 @@
                   <Disorder id="8542">
                     <OrphaCode>210</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=210</ExpertLink>
-                    <Name lang="en">Cyclosporosis</Name>
+                    <Name lang="en">Cyclosporiasis</Name>
                     <DisorderType id="21394">
                       <Name lang="en">Disease</Name>
                     </DisorderType>
@@ -2203,7 +2299,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="12">
+              <ClassificationNodeChildList count="13">
                 <ClassificationNode>
                   <Disorder id="719">
                     <OrphaCode>1163</OrphaCode>
@@ -2336,6 +2432,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31697">
+                    <OrphaCode>636945</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=636945</ExpertLink>
+                    <Name lang="en">Invasive Candidiasis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31676">
                     <OrphaCode>633124</OrphaCode>
