commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml
index ac05d91..aff4262 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_193.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:23:51" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:08:20" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2144,7 +2144,7 @@
                           <Disorder id="22720">
                             <OrphaCode>398069</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=398069</ExpertLink>
-                            <Name lang="en">MAGEL2-related Prader-Willi-like syndrome</Name>
+                            <Name lang="en">Schaaf-Yang syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2943,7 +2943,7 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="5">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
                           <Disorder id="7024">
                             <OrphaCode>629</OrphaCode>
@@ -3028,6 +3028,18 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32155">
+                            <OrphaCode>684247</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684247</ExpertLink>
+                            <Name lang="en">Isolated growth hormone deficiency type IV</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -3640,7 +3652,7 @@
                                   <Disorder id="22720">
                                     <OrphaCode>398069</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=398069</ExpertLink>
-                                    <Name lang="en">MAGEL2-related Prader-Willi-like syndrome</Name>
+                                    <Name lang="en">Schaaf-Yang syndrome</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -4076,7 +4088,7 @@
                           <Disorder id="14199">
                             <OrphaCode>99226</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                            <Name lang="en">Monosomy X</Name>
+                            <Name lang="en">Monosomy X syndrome</Name>
                             <DisorderType id="21443">
                               <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
@@ -4088,7 +4100,7 @@
                           <Disorder id="14200">
                             <OrphaCode>99228</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                            <Name lang="en">Mosaic monosomy X</Name>
+                            <Name lang="en">Mosaic monosomy X syndrome</Name>
                             <DisorderType id="21443">
                               <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
@@ -4126,7 +4138,7 @@
                       <Disorder id="390">
                         <OrphaCode>9</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=9</ExpertLink>
-                        <Name lang="en">Tetrasomy X</Name>
+                        <Name lang="en">Tetrasomy X syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -4150,7 +4162,7 @@
                       <Disorder id="1231">
                         <OrphaCode>3375</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3375</ExpertLink>
-                        <Name lang="en">Trisomy X</Name>
+                        <Name lang="en">Trisomy X syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -4282,7 +4294,7 @@
                       <Disorder id="12825">
                         <OrphaCode>96201</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96201</ExpertLink>
-                        <Name lang="en">X small rings</Name>
+                        <Name lang="en">X small rings syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -4916,7 +4928,7 @@
                           <Disorder id="334">
                             <OrphaCode>1642</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1642</ExpertLink>
-                            <Name lang="en">Distal deletion 9p</Name>
+                            <Name lang="en">Distal deletion 9p syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -5588,7 +5600,7 @@
                           <Disorder id="14199">
                             <OrphaCode>99226</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                            <Name lang="en">Monosomy X</Name>
+                            <Name lang="en">Monosomy X syndrome</Name>
                             <DisorderType id="21443">
                               <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
@@ -5600,7 +5612,7 @@
                           <Disorder id="14200">
                             <OrphaCode>99228</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                            <Name lang="en">Mosaic monosomy X</Name>
+                            <Name lang="en">Mosaic monosomy X syndrome</Name>
                             <DisorderType id="21443">
                               <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
@@ -5674,7 +5686,7 @@
                       <Disorder id="18396">
                         <OrphaCode>199310</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=199310</ExpertLink>
-                        <Name lang="en">Tetragametic chimerism</Name>
+                        <Name lang="en">Tetragametic chimerism syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -5705,7 +5717,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="12">
+                  <ClassificationNodeChildList count="13">
                     <ClassificationNode>
                       <Disorder id="1007">
                         <OrphaCode>528</OrphaCode>
@@ -5934,6 +5946,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32182">
+                        <OrphaCode>686999</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686999</ExpertLink>
+                        <Name lang="en">Lipodystrophy-demyelinating peripheral sensory-motor neuropathy syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="22331">
                         <OrphaCode>363649</OrphaCode>
@@ -6694,24 +6718,12 @@
                       <Disorder id="18187">
                         <OrphaCode>181428</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=181428</ExpertLink>
-                        <Name lang="en">Hyperalphalipoproteinemia</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
+                        <Name lang="en">Familial Hyperalphalipoproteinemia</Name>
+                        <DisorderType id="21408">
+                          <Name lang="en">Biological anomaly</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
-                        <ClassificationNode>
-                          <Disorder id="11532">
-                            <OrphaCode>79506</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79506</ExpertLink>
-                            <Name lang="en">Cholesterol-ester transfer protein deficiency</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="1">
                         <ClassificationNode>
                           <Disorder id="16994">
                             <OrphaCode>140905</OrphaCode>
@@ -7099,7 +7111,19 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="8">
+              <ClassificationNodeChildList count="9">
+                <ClassificationNode>
+                  <Disorder id="32197">
+                    <OrphaCode>688649</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=688649</ExpertLink>
+                    <Name lang="en">Isolated adrenal medullary hyperplasia</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31726">
                     <OrphaCode>641613</OrphaCode>
@@ -9283,7 +9307,19 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="6">
+              <ClassificationNodeChildList count="7">
+                <ClassificationNode>
+                  <Disorder id="32179">
+                    <OrphaCode>686495</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686495</ExpertLink>
+                    <Name lang="en">MADD-related developmental delay-endocrine dysfunction-hypohemoglobinemia syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="1471">
                     <OrphaCode>1227</OrphaCode>
@@ -9535,7 +9571,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="7">
+              <ClassificationNodeChildList count="8">
                 <ClassificationNode>
                   <Disorder id="470">
                     <OrphaCode>1672</OrphaCode>
@@ -9577,7 +9613,7 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="5">
+                          <ClassificationNodeChildList count="6">
                             <ClassificationNode>
                               <Disorder id="7024">
                                 <OrphaCode>629</OrphaCode>
@@ -9662,6 +9698,18 @@
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32155">
+                                <OrphaCode>684247</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=684247</ExpertLink>
+                                <Name lang="en">Isolated growth hormone deficiency type IV</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -10274,7 +10322,7 @@
                                       <Disorder id="22720">
                                         <OrphaCode>398069</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=398069</ExpertLink>
-                                        <Name lang="en">MAGEL2-related Prader-Willi-like syndrome</Name>
+                                        <Name lang="en">Schaaf-Yang syndrome</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -11144,6 +11192,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32211">
+                    <OrphaCode>689401</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689401</ExpertLink>
+                    <Name lang="en">Acquired hypothalamic obesity</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
