commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml
index 95af51c..0697ff8 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:29:25" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:15:54" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <ClassificationList count="1">
     <Classification id="156">
       <OrphaNumber>158293</OrphaNumber>
@@ -19182,7 +19175,7 @@
                               <Disorder id="25220">
                                 <OrphaCode>480682</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=480682</ExpertLink>
-                                <Name lang="en">POGLUT1-related  limb-girdle muscular dystrophy R21</Name>
+                                <Name lang="en">POGLUT1-related limb-girdle muscular dystrophy R21</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19398,7 +19391,7 @@
                               <Disorder id="10337">
                                 <OrphaCode>34515</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-                                <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+                                <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19410,7 +19403,7 @@
                               <Disorder id="11732">
                                 <OrphaCode>86812</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-                                <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+                                <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19422,7 +19415,7 @@
                               <Disorder id="18521">
                                 <OrphaCode>206554</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                                <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                                <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19434,7 +19427,7 @@
                               <Disorder id="18522">
                                 <OrphaCode>206559</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-                                <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+                                <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19446,7 +19439,7 @@
                               <Disorder id="18523">
                                 <OrphaCode>206564</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206564</ExpertLink>
-                                <Name lang="en">POMGNT1-related  limb-girdle muscular dystrophy R15</Name>
+                                <Name lang="en">POMGNT1-related limb-girdle muscular dystrophy R15</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19458,7 +19451,7 @@
                               <Disorder id="22066">
                                 <OrphaCode>352479</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-                                <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+                                <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -19470,7 +19463,7 @@
                               <Disorder id="22329">
                                 <OrphaCode>363623</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-                                <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+                                <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -37484,7 +37477,7 @@
                           <Disorder id="10337">
                             <OrphaCode>34515</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-                            <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+                            <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -37520,7 +37513,7 @@
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -37532,7 +37525,7 @@
                           <Disorder id="18523">
                             <OrphaCode>206564</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206564</ExpertLink>
-                            <Name lang="en">POMGNT1-related  limb-girdle muscular dystrophy R15</Name>
+                            <Name lang="en">POMGNT1-related limb-girdle muscular dystrophy R15</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -38120,7 +38113,7 @@
                           <Disorder id="11732">
                             <OrphaCode>86812</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-                            <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+                            <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -38132,7 +38125,7 @@
                           <Disorder id="18522">
                             <OrphaCode>206559</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-                            <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+                            <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -38168,7 +38161,7 @@
                           <Disorder id="20455">
                             <OrphaCode>280333</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280333</ExpertLink>
-                            <Name lang="en">Alpha-dystroglycan-related  limb-girdle muscular dystrophy R16</Name>
+                            <Name lang="en">Alpha-dystroglycan-related limb-girdle muscular dystrophy R16</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -38192,7 +38185,7 @@
                           <Disorder id="22066">
                             <OrphaCode>352479</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-                            <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+                            <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -38216,7 +38209,7 @@
                           <Disorder id="22329">
                             <OrphaCode>363623</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-                            <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+                            <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -47592,7 +47585,7 @@
                                           <Disorder id="10338">
                                             <OrphaCode>34516</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34516</ExpertLink>
-                                            <Name lang="en">DNAJB6-related  limb-girdle muscular dystrophy D1</Name>
+                                            <Name lang="en">DNAJB6-related limb-girdle muscular dystrophy D1</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47604,7 +47597,7 @@
                                           <Disorder id="10758">
                                             <OrphaCode>55595</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=55595</ExpertLink>
-                                            <Name lang="en">TNP03-related  limb-girdle muscular dystrophy D2</Name>
+                                            <Name lang="en">TNP03-related limb-girdle muscular dystrophy D2</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47616,7 +47609,7 @@
                                           <Disorder id="10759">
                                             <OrphaCode>55596</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=55596</ExpertLink>
-                                            <Name lang="en">HNRNPDL-related  limb-girdle muscular dystrophy D3</Name>
+                                            <Name lang="en">HNRNPDL-related limb-girdle muscular dystrophy D3</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47664,7 +47657,7 @@
                                           <Disorder id="25220">
                                             <OrphaCode>480682</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=480682</ExpertLink>
-                                            <Name lang="en">POGLUT1-related  limb-girdle muscular dystrophy R21</Name>
+                                            <Name lang="en">POGLUT1-related limb-girdle muscular dystrophy R21</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47676,7 +47669,7 @@
                                           <Disorder id="678">
                                             <OrphaCode>62</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=62</ExpertLink>
-                                            <Name lang="en">Alpha-sarcoglycan-related  limb-girdle muscular dystrophy R3</Name>
+                                            <Name lang="en">Alpha-sarcoglycan-related limb-girdle muscular dystrophy R3</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47688,7 +47681,7 @@
                                           <Disorder id="870">
                                             <OrphaCode>267</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=267</ExpertLink>
-                                            <Name lang="en">Calpain-3-related  limb-girdle muscular dystrophy R1</Name>
+                                            <Name lang="en">Calpain-3-related limb-girdle muscular dystrophy R1</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47700,7 +47693,7 @@
                                           <Disorder id="1195">
                                             <OrphaCode>1878</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1878</ExpertLink>
-                                            <Name lang="en">TRIM32-related  limb-girdle muscular dystrophy R8</Name>
+                                            <Name lang="en">TRIM32-related limb-girdle muscular dystrophy R8</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47712,7 +47705,7 @@
                                           <Disorder id="8715">
                                             <OrphaCode>119</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=119</ExpertLink>
-                                            <Name lang="en">Beta-sarcoglycan-related  limb-girdle muscular dystrophy R4</Name>
+                                            <Name lang="en">Beta-sarcoglycan-related limb-girdle muscular dystrophy R4</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47724,7 +47717,7 @@
                                           <Disorder id="8716">
                                             <OrphaCode>353</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353</ExpertLink>
-                                            <Name lang="en">Gamma-sarcoglycan-related  limb-girdle muscular dystrophy R5</Name>
+                                            <Name lang="en">Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47736,7 +47729,7 @@
                                           <Disorder id="8717">
                                             <OrphaCode>219</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=219</ExpertLink>
-                                            <Name lang="en">Delta-sarcoglycan-related  limb-girdle muscular dystrophy R6</Name>
+                                            <Name lang="en">Delta-sarcoglycan-related limb-girdle muscular dystrophy R6</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47748,7 +47741,7 @@
                                           <Disorder id="8720">
                                             <OrphaCode>268</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268</ExpertLink>
-                                            <Name lang="en">Dysferlin-related  limb-girdle muscular dystrophy R2</Name>
+                                            <Name lang="en">Dysferlin-related limb-girdle muscular dystrophy R2</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47760,7 +47753,7 @@
                                           <Disorder id="10336">
                                             <OrphaCode>34514</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34514</ExpertLink>
-                                            <Name lang="en">Telethonin-related  limb-girdle muscular dystrophy R7</Name>
+                                            <Name lang="en">Telethonin-related limb-girdle muscular dystrophy R7</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47772,7 +47765,7 @@
                                           <Disorder id="10337">
                                             <OrphaCode>34515</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-                                            <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+                                            <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47784,7 +47777,7 @@
                                           <Disorder id="11732">
                                             <OrphaCode>86812</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-                                            <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+                                            <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47796,7 +47789,7 @@
                                           <Disorder id="16999">
                                             <OrphaCode>140922</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140922</ExpertLink>
-                                            <Name lang="en">Titin-related  limb-girdle muscular dystrophy R10</Name>
+                                            <Name lang="en">Titin-related limb-girdle muscular dystrophy R10</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47808,7 +47801,7 @@
                                           <Disorder id="18520">
                                             <OrphaCode>206549</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206549</ExpertLink>
-                                            <Name lang="en">Anoctamin-5-related  limb-girdle muscular dystrophy R12</Name>
+                                            <Name lang="en">Anoctamin-5-related limb-girdle muscular dystrophy R12</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47820,7 +47813,7 @@
                                           <Disorder id="18521">
                                             <OrphaCode>206554</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47832,7 +47825,7 @@
                                           <Disorder id="18522">
                                             <OrphaCode>206559</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-                                            <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+                                            <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47844,7 +47837,7 @@
                                           <Disorder id="18523">
                                             <OrphaCode>206564</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206564</ExpertLink>
-                                            <Name lang="en">POMGNT1-related  limb-girdle muscular dystrophy R15</Name>
+                                            <Name lang="en">POMGNT1-related limb-girdle muscular dystrophy R15</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47856,7 +47849,7 @@
                                           <Disorder id="19742">
                                             <OrphaCode>254361</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=254361</ExpertLink>
-                                            <Name lang="en">Plectin-related  limb-girdle muscular dystrophy R17</Name>
+                                            <Name lang="en">Plectin-related limb-girdle muscular dystrophy R17</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47868,7 +47861,7 @@
                                           <Disorder id="20455">
                                             <OrphaCode>280333</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280333</ExpertLink>
-                                            <Name lang="en">Alpha-dystroglycan-related  limb-girdle muscular dystrophy R16</Name>
+                                            <Name lang="en">Alpha-dystroglycan-related limb-girdle muscular dystrophy R16</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47880,7 +47873,7 @@
                                           <Disorder id="22066">
                                             <OrphaCode>352479</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-                                            <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+                                            <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47892,7 +47885,7 @@
                                           <Disorder id="22329">
                                             <OrphaCode>363623</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-                                            <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+                                            <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -47904,7 +47897,7 @@
                                           <Disorder id="22434">
                                             <OrphaCode>369840</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=369840</ExpertLink>
-                                            <Name lang="en">TRAPPC11-related  limb-girdle muscular dystrophy R18</Name>
+                                            <Name lang="en">TRAPPC11-related limb-girdle muscular dystrophy R18</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -50392,7 +50385,7 @@
                                   <Disorder id="678">
                                     <OrphaCode>62</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=62</ExpertLink>
-                                    <Name lang="en">Alpha-sarcoglycan-related  limb-girdle muscular dystrophy R3</Name>
+                                    <Name lang="en">Alpha-sarcoglycan-related limb-girdle muscular dystrophy R3</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -50416,7 +50409,7 @@
                                   <Disorder id="8715">
                                     <OrphaCode>119</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=119</ExpertLink>
-                                    <Name lang="en">Beta-sarcoglycan-related  limb-girdle muscular dystrophy R4</Name>
+                                    <Name lang="en">Beta-sarcoglycan-related limb-girdle muscular dystrophy R4</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -50440,7 +50433,7 @@
                                   <Disorder id="8716">
                                     <OrphaCode>353</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353</ExpertLink>
-                                    <Name lang="en">Gamma-sarcoglycan-related  limb-girdle muscular dystrophy R5</Name>
+                                    <Name lang="en">Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -50464,7 +50457,7 @@
                                   <Disorder id="8717">
                                     <OrphaCode>219</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=219</ExpertLink>
-                                    <Name lang="en">Delta-sarcoglycan-related  limb-girdle muscular dystrophy R6</Name>
+                                    <Name lang="en">Delta-sarcoglycan-related limb-girdle muscular dystrophy R6</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -50490,7 +50483,7 @@
                               <Disorder id="8720">
                                 <OrphaCode>268</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268</ExpertLink>
-                                <Name lang="en">Dysferlin-related  limb-girdle muscular dystrophy R2</Name>
+                                <Name lang="en">Dysferlin-related limb-girdle muscular dystrophy R2</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -50814,7 +50807,7 @@
                               <Disorder id="870">
                                 <OrphaCode>267</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=267</ExpertLink>
-                                <Name lang="en">Calpain-3-related  limb-girdle muscular dystrophy R1</Name>
+                                <Name lang="en">Calpain-3-related limb-girdle muscular dystrophy R1</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -50838,7 +50831,7 @@
                               <Disorder id="1195">
                                 <OrphaCode>1878</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1878</ExpertLink>
-                                <Name lang="en">TRIM32-related  limb-girdle muscular dystrophy R8</Name>
+                                <Name lang="en">TRIM32-related limb-girdle muscular dystrophy R8</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -51090,7 +51083,7 @@
                               <Disorder id="16999">
                                 <OrphaCode>140922</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140922</ExpertLink>
-                                <Name lang="en">Titin-related  limb-girdle muscular dystrophy R10</Name>
+                                <Name lang="en">Titin-related limb-girdle muscular dystrophy R10</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -51150,7 +51143,7 @@
                               <Disorder id="10336">
                                 <OrphaCode>34514</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34514</ExpertLink>
-                                <Name lang="en">Telethonin-related  limb-girdle muscular dystrophy R7</Name>
+                                <Name lang="en">Telethonin-related limb-girdle muscular dystrophy R7</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -51534,7 +51527,7 @@
                               <Disorder id="19742">
                                 <OrphaCode>254361</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=254361</ExpertLink>
-                                <Name lang="en">Plectin-related  limb-girdle muscular dystrophy R17</Name>
+                                <Name lang="en">Plectin-related limb-girdle muscular dystrophy R17</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -51818,7 +51811,7 @@
                                       <Disorder id="10337">
                                         <OrphaCode>34515</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-                                        <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+                                        <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -51878,7 +51871,7 @@
                                       <Disorder id="18520">
                                         <OrphaCode>206549</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206549</ExpertLink>
-                                        <Name lang="en">Anoctamin-5-related  limb-girdle muscular dystrophy R12</Name>
+                                        <Name lang="en">Anoctamin-5-related limb-girdle muscular dystrophy R12</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -51890,7 +51883,7 @@
                                       <Disorder id="18521">
                                         <OrphaCode>206554</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                                        <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                                        <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -51938,7 +51931,7 @@
                                       <Disorder id="18523">
                                         <OrphaCode>206564</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206564</ExpertLink>
-                                        <Name lang="en">POMGNT1-related  limb-girdle muscular dystrophy R15</Name>
+                                        <Name lang="en">POMGNT1-related limb-girdle muscular dystrophy R15</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -51986,7 +51979,7 @@
                                       <Disorder id="11732">
                                         <OrphaCode>86812</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-                                        <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+                                        <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -52034,7 +52027,7 @@
                                       <Disorder id="18522">
                                         <OrphaCode>206559</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-                                        <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+                                        <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -52048,7 +52041,7 @@
                                   <Disorder id="22066">
                                     <OrphaCode>352479</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-                                    <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+                                    <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -52096,7 +52089,7 @@
                                   <Disorder id="20455">
                                     <OrphaCode>280333</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280333</ExpertLink>
-                                    <Name lang="en">Alpha-dystroglycan-related  limb-girdle muscular dystrophy R16</Name>
+                                    <Name lang="en">Alpha-dystroglycan-related limb-girdle muscular dystrophy R16</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -84354,7 +84347,7 @@
                               <Disorder id="22329">
                                 <OrphaCode>363623</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-                                <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+                                <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -85962,7 +85955,7 @@
                               <Disorder id="11732">
                                 <OrphaCode>86812</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-                                <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+                                <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -85998,7 +85991,7 @@
                               <Disorder id="18522">
                                 <OrphaCode>206559</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-                                <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+                                <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -86082,7 +86075,7 @@
                               <Disorder id="20455">
                                 <OrphaCode>280333</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280333</ExpertLink>
-                                <Name lang="en">Alpha-dystroglycan-related  limb-girdle muscular dystrophy R16</Name>
+                                <Name lang="en">Alpha-dystroglycan-related limb-girdle muscular dystrophy R16</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -86118,7 +86111,7 @@
                               <Disorder id="22066">
                                 <OrphaCode>352479</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-                                <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+                                <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -95420,7 +95413,7 @@
                           <Disorder id="678">
                             <OrphaCode>62</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=62</ExpertLink>
-                            <Name lang="en">Alpha-sarcoglycan-related  limb-girdle muscular dystrophy R3</Name>
+                            <Name lang="en">Alpha-sarcoglycan-related limb-girdle muscular dystrophy R3</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -95432,7 +95425,7 @@
                           <Disorder id="8715">
                             <OrphaCode>119</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=119</ExpertLink>
-                            <Name lang="en">Beta-sarcoglycan-related  limb-girdle muscular dystrophy R4</Name>
+                            <Name lang="en">Beta-sarcoglycan-related limb-girdle muscular dystrophy R4</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -95444,7 +95437,7 @@
                           <Disorder id="8716">
                             <OrphaCode>353</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353</ExpertLink>
-                            <Name lang="en">Gamma-sarcoglycan-related  limb-girdle muscular dystrophy R5</Name>
+                            <Name lang="en">Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -95456,7 +95449,7 @@
                           <Disorder id="8717">
                             <OrphaCode>219</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=219</ExpertLink>
-                            <Name lang="en">Delta-sarcoglycan-related  limb-girdle muscular dystrophy R6</Name>
+                            <Name lang="en">Delta-sarcoglycan-related limb-girdle muscular dystrophy R6</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -95480,7 +95473,7 @@
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -95876,7 +95869,7 @@
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
