commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml
index b8362c7..6bd7e6f 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:41:40" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:43:12" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -892,7 +892,7 @@
                                   <Disorder id="21452">
                                     <OrphaCode>313800</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313800</ExpertLink>
-                                    <Name lang="en">Optic nerve edema-splenomegaly syndrome</Name>
+                                    <Name lang="en">Retinal dystrophy-optic nerve edema-splenomegaly-anhidrosis-migraine headache syndrome</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -8357,7 +8357,19 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="6">
+                  <ClassificationNodeChildList count="7">
+                    <ClassificationNode>
+                      <Disorder id="31348">
+                        <OrphaCode>617449</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617449</ExpertLink>
+                        <Name lang="en">Congenital aphakia-iris hypoplasia-microphthalmia-microcornea syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="17044">
                         <OrphaCode>141132</OrphaCode>
