commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml
index c33b6d3..48365ac 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_194.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:40:45" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2021-12-01 04:44:20" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <ClassificationList count="1">
     <Classification id="194">
       <OrphaNumber>162949</OrphaNumber>
@@ -4613,7 +4606,7 @@
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
                                           </Disorder>
-                                          <ClassificationNodeChildList count="6">
+                                          <ClassificationNodeChildList count="4">
                                             <ClassificationNode>
                                               <Disorder id="18309">
                                                 <OrphaCode>183678</OrphaCode>
@@ -4662,30 +4655,6 @@
                                               <ClassificationNodeChildList count="0">
                                               </ClassificationNodeChildList>
                                             </ClassificationNode>
-                                            <ClassificationNode>
-                                              <Disorder id="19199">
-                                                <OrphaCode>231537</OrphaCode>
-                                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231537</ExpertLink>
-                                                <Name lang="en">Hermansky-Pudlak syndrome type 8</Name>
-                                                <DisorderType id="21450">
-                                                  <Name lang="en">Clinical subtype</Name>
-                                                </DisorderType>
-                                              </Disorder>
-                                              <ClassificationNodeChildList count="0">
-                                              </ClassificationNodeChildList>
-                                            </ClassificationNode>
-                                            <ClassificationNode>
-                                              <Disorder id="20493">
-                                                <OrphaCode>280663</OrphaCode>
-                                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280663</ExpertLink>
-                                                <Name lang="en">Hermansky-Pudlak syndrome type 9</Name>
-                                                <DisorderType id="21450">
-                                                  <Name lang="en">Clinical subtype</Name>
-                                                </DisorderType>
-                                              </Disorder>
-                                              <ClassificationNodeChildList count="0">
-                                              </ClassificationNodeChildList>
-                                            </ClassificationNode>
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
                                       </ClassificationNodeChildList>
