commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml
index f046e27..de50457 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:24:21" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:08:51" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2703,7 +2703,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="7">
                         <ClassificationNode>
                           <Disorder id="16965">
                             <OrphaCode>140436</OrphaCode>
@@ -2716,6 +2716,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32189">
+                            <OrphaCode>688523</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=688523</ExpertLink>
+                            <Name lang="en">Splenic venous malformation</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="18935">
                             <OrphaCode>221061</OrphaCode>
@@ -3052,7 +3064,7 @@
                                   <Disorder id="6520">
                                     <OrphaCode>662</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662</ExpertLink>
-                                    <Name lang="en">Yellow nail syndrome</Name>
+                                    <Name lang="en">Lymphedema with yellow nails</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -3146,7 +3158,7 @@
                                       <Disorder id="14199">
                                         <OrphaCode>99226</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                                        <Name lang="en">Monosomy X</Name>
+                                        <Name lang="en">Monosomy X syndrome</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
@@ -3158,7 +3170,7 @@
                                       <Disorder id="14200">
                                         <OrphaCode>99228</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                                        <Name lang="en">Mosaic monosomy X</Name>
+                                        <Name lang="en">Mosaic monosomy X syndrome</Name>
                                         <DisorderType id="21443">
                                           <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
