commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml
index 393ad62..9a3fd5c 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_205.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:06:30" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:25:07" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -1219,7 +1219,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="17">
+              <ClassificationNodeChildList count="19">
                 <ClassificationNode>
                   <Disorder id="5517">
                     <OrphaCode>80</OrphaCode>
@@ -1340,6 +1340,30 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32015">
+                    <OrphaCode>662721</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662721</ExpertLink>
+                    <Name lang="en">Placenta accreta spectrum disorder</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32017">
+                    <OrphaCode>662786</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662786</ExpertLink>
+                    <Name lang="en">Vasa previa</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31322">
                     <OrphaCode>615970</OrphaCode>
@@ -2268,7 +2292,7 @@
                   <Disorder id="12664">
                     <OrphaCode>95709</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95709</ExpertLink>
-                    <Name lang="en">Acquired premature ovarian failure</Name>
+                    <Name lang="en">Rare acquired premature ovarian failure</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -2304,7 +2328,7 @@
                   <Disorder id="12665">
                     <OrphaCode>95710</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95710</ExpertLink>
-                    <Name lang="en">Non-acquired premature ovarian failure</Name>
+                    <Name lang="en">Rare non-acquired premature ovarian failure</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -3862,7 +3886,7 @@
                       <Disorder id="900">
                         <OrphaCode>847</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=847</ExpertLink>
-                        <Name lang="en">Alpha-thalassemia-X-linked intellectual disability syndrome</Name>
+                        <Name lang="en">X-linked alpha-thalassemia-intellectual disability syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
