commit 7ab24fe976be5c798004246aa079a429e568689b
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Jun 30 11:46:37 2025 +0200

    JULY 2025

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml
old mode 100644
new mode 100755
index 953fe44..dfb8073
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_203.xml	
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-12-03 07:09:27" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<?xml version="1.0" encoding="UTF-8"?>
+<JDBOR date="2025-06-24 07:34:14" version="1.3.42 / 4.1.8 [2025-03-03]" copyright="Orphanet (c) 2025" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -953,7 +953,43 @@
                       <Name lang="en">Disease</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="0">
+                  <ClassificationNodeChildList count="3">
+                    <ClassificationNode>
+                      <Disorder id="32414">
+                        <OrphaCode>699678</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=699678</ExpertLink>
+                        <Name lang="en">Necrotizing cellulitis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32417">
+                        <OrphaCode>699697</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=699697</ExpertLink>
+                        <Name lang="en">Necrotizing fasciitis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32418">
+                        <OrphaCode>699702</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=699702</ExpertLink>
+                        <Name lang="en">Necrotizing myositis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -1409,7 +1445,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="18">
+                  <ClassificationNodeChildList count="17">
                     <ClassificationNode>
                       <Disorder id="21794">
                         <OrphaCode>324625</OrphaCode>
@@ -1566,18 +1602,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="17150">
-                        <OrphaCode>157823</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=157823</ExpertLink>
-                        <Name lang="en">Klüver-Bucy syndrome</Name>
-                        <DisorderType id="21422">
-                          <Name lang="en">Clinical syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="18828">
                         <OrphaCode>217260</OrphaCode>
@@ -1771,7 +1795,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="33">
+              <ClassificationNodeChildList count="34">
                 <ClassificationNode>
                   <Disorder id="32100">
                     <OrphaCode>674947</OrphaCode>
@@ -1796,6 +1820,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32351">
+                    <OrphaCode>697096</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=697096</ExpertLink>
+                    <Name lang="en">Cryptosporidiosis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="210">
                     <OrphaCode>673</OrphaCode>
@@ -2383,7 +2419,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="15">
+              <ClassificationNodeChildList count="18">
                 <ClassificationNode>
                   <Disorder id="719">
                     <OrphaCode>1163</OrphaCode>
@@ -2528,6 +2564,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="28678">
+                    <OrphaCode>570762</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=570762</ExpertLink>
+                    <Name lang="en">Infective endocarditis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="23614">
                     <OrphaCode>449280</OrphaCode>
@@ -2540,6 +2588,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32344">
+                    <OrphaCode>697053</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=697053</ExpertLink>
+                    <Name lang="en">Talaromycosis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31697">
                     <OrphaCode>636945</OrphaCode>
@@ -2564,6 +2624,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32350">
+                    <OrphaCode>697091</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=697091</ExpertLink>
+                    <Name lang="en">Emergomycosis</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
