commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml
old mode 100644
new mode 100755
index c9072ab..48584d2
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:33:11" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:33:49" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -11970,7 +11970,7 @@
                               <Disorder id="24223">
                                 <OrphaCode>468631</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                                <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -34061,7 +34061,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="111">
+                  <ClassificationNodeChildList count="112">
                     <ClassificationNode>
                       <Disorder id="3069">
                         <OrphaCode>3200</OrphaCode>
@@ -35331,7 +35331,7 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
                           <Disorder id="180">
                             <OrphaCode>181</OrphaCode>
@@ -35368,18 +35368,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="13830">
-                            <OrphaCode>98813</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
-                            <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -35634,6 +35622,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="13830">
+                        <OrphaCode>98813</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
+                        <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -46563,8 +46563,8 @@
                                         <OrphaCode>98791</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98791</ExpertLink>
                                         <Name lang="en">Alpha-thalassemia-intellectual disability syndrome linked to chromosome 16</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
+                                        <DisorderType id="21401">
+                                          <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -47248,9 +47248,9 @@
                                   <Disorder id="19853">
                                     <OrphaCode>261144</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
-                                    <Name lang="en">14q12 microdeletion syndrome</Name>
-                                    <DisorderType id="21401">
-                                      <Name lang="en">Malformation syndrome</Name>
+                                    <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -50300,7 +50300,7 @@
                           <Disorder id="24223">
                             <OrphaCode>468631</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                            <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                            <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -53002,7 +53002,7 @@
                       <Disorder id="972">
                         <OrphaCode>776</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=776</ExpertLink>
-                        <Name lang="en">X-linked intellectual disability with marfanoid habitus</Name>
+                        <Name lang="en">Lujan-Fryns syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
