commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml
old mode 100644
new mode 100755
index 82ba8cf..d184caf
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:33:31" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:34:09" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -4311,8 +4311,8 @@
                                         <OrphaCode>98791</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98791</ExpertLink>
                                         <Name lang="en">Alpha-thalassemia-intellectual disability syndrome linked to chromosome 16</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
+                                        <DisorderType id="21401">
+                                          <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -4996,9 +4996,9 @@
                                   <Disorder id="19853">
                                     <OrphaCode>261144</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
-                                    <Name lang="en">14q12 microdeletion syndrome</Name>
-                                    <DisorderType id="21401">
-                                      <Name lang="en">Malformation syndrome</Name>
+                                    <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -7355,19 +7355,7 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="3">
-                                <ClassificationNode>
-                                  <Disorder id="14205">
-                                    <OrphaCode>99361</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99361</ExpertLink>
-                                    <Name lang="en">Familial medullary thyroid carcinoma</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="2">
                                 <ClassificationNode>
                                   <Disorder id="19538">
                                     <OrphaCode>247698</OrphaCode>
@@ -11907,7 +11895,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="110">
+                      <ClassificationNodeChildList count="111">
                         <ClassificationNode>
                           <Disorder id="3069">
                             <OrphaCode>3200</OrphaCode>
@@ -13165,7 +13153,7 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="3">
                             <ClassificationNode>
                               <Disorder id="180">
                                 <OrphaCode>181</OrphaCode>
@@ -13202,18 +13190,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="13830">
-                                <OrphaCode>98813</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
-                                <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -13468,6 +13444,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="13830">
+                            <OrphaCode>98813</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
+                            <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -15044,7 +15032,7 @@
                           <Disorder id="26573">
                             <OrphaCode>508512</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=508512</ExpertLink>
-                            <Name lang="en">Congenital multiple café-au-lait macules-increased sister chromatid exchange syndrome</Name>
+                            <Name lang="en">Intrauterine growth restriction-congenital multiple café-au-lait macules-increased sister chromatid exchange syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -31364,7 +31352,7 @@
                           <Disorder id="23427">
                             <OrphaCode>442835</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                            <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                            <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -31400,7 +31388,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">West syndrome</Name>
+                            <Name lang="en">Infantile spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -31508,7 +31496,7 @@
                           <Disorder id="23427">
                             <OrphaCode>442835</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                            <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                            <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -31676,7 +31664,7 @@
                           <Disorder id="23427">
                             <OrphaCode>442835</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                            <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                            <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -31796,7 +31784,7 @@
                           <Disorder id="23427">
                             <OrphaCode>442835</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                            <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                            <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -44835,7 +44823,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="29875">
+                            <OrphaCode>598164</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="19853">
+                            <OrphaCode>261144</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -59956,7 +59968,7 @@
                                   <Disorder id="24223">
                                     <OrphaCode>468631</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                                    <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                                    <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -64386,7 +64398,7 @@
                               <Disorder id="23427">
                                 <OrphaCode>442835</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                                <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                                <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -64410,7 +64422,7 @@
                               <Disorder id="894">
                                 <OrphaCode>3451</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                                <Name lang="en">West syndrome</Name>
+                                <Name lang="en">Infantile spasms syndrome</Name>
                                 <DisorderType id="21422">
                                   <Name lang="en">Clinical syndrome</Name>
                                 </DisorderType>
@@ -64626,7 +64638,7 @@
                               <Disorder id="23427">
                                 <OrphaCode>442835</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                                <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                                <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -64686,7 +64698,7 @@
                               <Disorder id="891">
                                 <OrphaCode>1942</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1942</ExpertLink>
-                                <Name lang="en">Myoclonic-astastic epilepsy</Name>
+                                <Name lang="en">Myoclonic-astatic epilepsy</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -66534,7 +66546,7 @@
                               <Disorder id="894">
                                 <OrphaCode>3451</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                                <Name lang="en">West syndrome</Name>
+                                <Name lang="en">Infantile spasms syndrome</Name>
                                 <DisorderType id="21422">
                                   <Name lang="en">Clinical syndrome</Name>
                                 </DisorderType>
@@ -67045,7 +67057,31 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="29875">
+                                <OrphaCode>598164</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                                <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="19853">
+                                <OrphaCode>261144</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                                <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -70991,7 +71027,31 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="0">
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="29875">
+                                    <OrphaCode>598164</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                                    <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="19853">
+                                    <OrphaCode>261144</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                                    <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -73940,7 +74000,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">West syndrome</Name>
+                            <Name lang="en">Infantile spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -77526,7 +77586,7 @@
                               <Disorder id="972">
                                 <OrphaCode>776</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=776</ExpertLink>
-                                <Name lang="en">X-linked intellectual disability with marfanoid habitus</Name>
+                                <Name lang="en">Lujan-Fryns syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -79617,8 +79677,8 @@
                             <OrphaCode>98791</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98791</ExpertLink>
                             <Name lang="en">Alpha-thalassemia-intellectual disability syndrome linked to chromosome 16</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -81006,7 +81066,7 @@
                               <Disorder id="24223">
                                 <OrphaCode>468631</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                                <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -81673,7 +81733,31 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="29875">
+                                <OrphaCode>598164</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                                <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="19853">
+                                <OrphaCode>261144</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                                <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -94661,7 +94745,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="34">
+                  <ClassificationNodeChildList count="35">
                     <ClassificationNode>
                       <Disorder id="28092">
                         <OrphaCode>544628</OrphaCode>
@@ -95538,6 +95622,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="11036">
+                        <OrphaCode>73224</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73224</ExpertLink>
+                        <Name lang="en">Tubular renal disease-cardiomyopathy syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="1328">
                         <OrphaCode>64</OrphaCode>
@@ -104759,7 +104855,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="8">
+                              <ClassificationNodeChildList count="9">
                                 <ClassificationNode>
                                   <Disorder id="11735">
                                     <OrphaCode>86815</OrphaCode>
@@ -104817,7 +104913,7 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="4">
+                                  <ClassificationNodeChildList count="3">
                                     <ClassificationNode>
                                       <Disorder id="180">
                                         <OrphaCode>181</OrphaCode>
@@ -104854,18 +104950,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="13830">
-                                        <OrphaCode>98813</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
-                                        <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -104904,6 +104988,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="13830">
+                                    <OrphaCode>98813</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
+                                    <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -113525,7 +113621,19 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="5">
+                  <ClassificationNodeChildList count="6">
+                    <ClassificationNode>
+                      <Disorder id="14205">
+                        <OrphaCode>99361</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99361</ExpertLink>
+                        <Name lang="en">Familial medullary thyroid carcinoma</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="906">
                         <OrphaCode>653</OrphaCode>
@@ -113535,19 +113643,7 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
-                        <ClassificationNode>
-                          <Disorder id="14205">
-                            <OrphaCode>99361</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99361</ExpertLink>
-                            <Name lang="en">Familial medullary thyroid carcinoma</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
                           <Disorder id="19538">
                             <OrphaCode>247698</OrphaCode>
@@ -125807,19 +125903,7 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="3">
-                                <ClassificationNode>
-                                  <Disorder id="14205">
-                                    <OrphaCode>99361</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99361</ExpertLink>
-                                    <Name lang="en">Familial medullary thyroid carcinoma</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="2">
                                 <ClassificationNode>
                                   <Disorder id="19538">
                                     <OrphaCode>247698</OrphaCode>
@@ -127484,7 +127568,7 @@
                           <Disorder id="20323">
                             <OrphaCode>275745</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=275745</ExpertLink>
-                            <Name lang="en">Alpha-thalassemia and related diseases</Name>
+                            <Name lang="en">Alpha-thalassemia and related disorders</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -127530,7 +127614,7 @@
                               <Disorder id="19226">
                                 <OrphaCode>232288</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=232288</ExpertLink>
-                                <Name lang="en">Alpha-thalassemia-related diseases</Name>
+                                <Name lang="en">Syndrome with alpha-thalassemia as a major feature</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -127553,8 +127637,8 @@
                                     <OrphaCode>98791</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98791</ExpertLink>
                                     <Name lang="en">Alpha-thalassemia-intellectual disability syndrome linked to chromosome 16</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -127758,7 +127842,7 @@
                               <Disorder id="19632">
                                 <OrphaCode>251355</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251355</ExpertLink>
-                                <Name lang="en">Sickle cell disease associated with an other hemoglobin anomaly</Name>
+                                <Name lang="en">Sickle cell disease associated with another hemoglobin anomaly</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -149139,7 +149223,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="110">
+                      <ClassificationNodeChildList count="111">
                         <ClassificationNode>
                           <Disorder id="3069">
                             <OrphaCode>3200</OrphaCode>
@@ -150397,7 +150481,7 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
+                          <ClassificationNodeChildList count="3">
                             <ClassificationNode>
                               <Disorder id="180">
                                 <OrphaCode>181</OrphaCode>
@@ -150434,18 +150518,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="13830">
-                                <OrphaCode>98813</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
-                                <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -150700,6 +150772,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="13830">
+                            <OrphaCode>98813</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
+                            <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -154228,7 +154312,7 @@
                                   <Disorder id="24223">
                                     <OrphaCode>468631</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                                    <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                                    <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -157098,7 +157182,7 @@
                               <Disorder id="24223">
                                 <OrphaCode>468631</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                                <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -159860,7 +159944,7 @@
                           <Disorder id="972">
                             <OrphaCode>776</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=776</ExpertLink>
-                            <Name lang="en">X-linked intellectual disability with marfanoid habitus</Name>
+                            <Name lang="en">Lujan-Fryns syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -202987,8 +203071,8 @@
                                 <OrphaCode>98813</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
                                 <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -204192,7 +204276,7 @@
                   <Disorder id="20632">
                     <OrphaCode>284993</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284993</ExpertLink>
-                    <Name lang="en">Marfan and Marfan-related disorders</Name>
+                    <Name lang="en">Marfan syndrome and Marfan-related disorders</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
