commit 7ab24fe976be5c798004246aa079a429e568689b
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Jun 30 11:46:37 2025 +0200

    JULY 2025

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_186.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_186.xml
old mode 100644
new mode 100755
index 210a3b7..2abf3f1
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_186.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_186.xml	
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-12-03 07:07:48" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<?xml version="1.0" encoding="UTF-8"?>
+<JDBOR date="2025-06-24 07:33:00" version="1.3.42 / 4.1.8 [2025-03-03]" copyright="Orphanet (c) 2025" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -79,7 +79,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="15">
+              <ClassificationNodeChildList count="16">
                 <ClassificationNode>
                   <Disorder id="597">
                     <OrphaCode>3346</OrphaCode>
@@ -392,6 +392,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32457">
+                    <OrphaCode>700286</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=700286</ExpertLink>
+                    <Name lang="en">Congenital high airway obstruction syndrome</Name>
+                    <DisorderType id="21422">
+                      <Name lang="en">Clinical syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31859">
                     <OrphaCode>649014</OrphaCode>
@@ -756,7 +768,7 @@
                   <Disorder id="557">
                     <OrphaCode>884</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=884</ExpertLink>
-                    <Name lang="en">Tetrasomy 12p syndrome</Name>
+                    <Name lang="en">Pallister-Killian syndrome</Name>
                     <DisorderType id="21401">
                       <Name lang="en">Malformation syndrome</Name>
                     </DisorderType>
