commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml
index 55fd022..f921ce3 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_197.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:05:41" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:24:16" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -439,7 +439,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="64">
+              <ClassificationNodeChildList count="66">
                 <ClassificationNode>
                   <Disorder id="2481">
                     <OrphaCode>2720</OrphaCode>
@@ -1256,6 +1256,30 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31975">
+                    <OrphaCode>659873</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659873</ExpertLink>
+                    <Name lang="en">Wormian bones-micrognathia-abnormal dentition-progeroid syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32036">
+                    <OrphaCode>664438</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664438</ExpertLink>
+                    <Name lang="en">Gingival fibromatosis-aortic root dilatation-facial dysmorphism-intellectual disability syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="29437">
                     <OrphaCode>589608</OrphaCode>
@@ -1579,7 +1603,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="3">
+              <ClassificationNodeChildList count="4">
                 <ClassificationNode>
                   <Disorder id="21483">
                     <OrphaCode>314419</OrphaCode>
@@ -1616,6 +1640,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32073">
+                    <OrphaCode>667678</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=667678</ExpertLink>
+                    <Name lang="en">Intraoral basal cell carcinoma</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
           </ClassificationNodeChildList>
