commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_146.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_146.xml
index 6ac51a2..ed00c01 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_146.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_146.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:01:44" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:20:13" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -198,7 +198,7 @@
                               <Disorder id="10348">
                                 <OrphaCode>34587</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34587</ExpertLink>
-                                <Name lang="en">Glycogen storage disease due to LAMP-2 deficiency</Name>
+                                <Name lang="en">Danon disease</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -1796,7 +1796,7 @@
                           <Disorder id="10348">
                             <OrphaCode>34587</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34587</ExpertLink>
-                            <Name lang="en">Glycogen storage disease due to LAMP-2 deficiency</Name>
+                            <Name lang="en">Danon disease</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1965,8 +1965,8 @@
                             <OrphaCode>271861</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=271861</ExpertLink>
                             <Name lang="en">Hereditary ATTR amyloidosis</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="2">
@@ -1975,8 +1975,8 @@
                                 <OrphaCode>85451</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85451</ExpertLink>
                                 <Name lang="en">ATTRV122I amyloidosis</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -1987,8 +1987,8 @@
                                 <OrphaCode>85447</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85447</ExpertLink>
                                 <Name lang="en">ATTRV30M amyloidosis</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
