commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml
old mode 100644
new mode 100755
index a4f242e..1bb4b95
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:33:29" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:34:07" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -469,8 +469,8 @@
                     <OrphaCode>103919</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=103919</ExpertLink>
                     <Name lang="en">Autoimmune pancreatitis</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
+                    <DisorderType id="21436">
+                      <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
                   <ClassificationNodeChildList count="2">
@@ -491,8 +491,8 @@
                         <OrphaCode>280315</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280315</ExpertLink>
                         <Name lang="en">Autoimmune pancreatitis type 2</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -702,7 +702,7 @@
                               <Disorder id="26438">
                                 <OrphaCode>506112</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=506112</ExpertLink>
-                                <Name lang="en">Mixed neuroendocrine-nonneuroendocrine neoplasm of pancreas</Name>
+                                <Name lang="en">Mixed neuroendocrine and non-neuroendocrine neoplasm of pancreas</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -3091,7 +3091,19 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="5">
+              <ClassificationNodeChildList count="6">
+                <ClassificationNode>
+                  <Disorder id="14205">
+                    <OrphaCode>99361</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99361</ExpertLink>
+                    <Name lang="en">Familial medullary thyroid carcinoma</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="121">
                     <OrphaCode>652</OrphaCode>
@@ -3113,19 +3125,7 @@
                       <Name lang="en">Disease</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="3">
-                    <ClassificationNode>
-                      <Disorder id="14205">
-                        <OrphaCode>99361</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99361</ExpertLink>
-                        <Name lang="en">Familial medullary thyroid carcinoma</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="2">
                     <ClassificationNode>
                       <Disorder id="19538">
                         <OrphaCode>247698</OrphaCode>
