commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml
old mode 100644
new mode 100755
index e7758ad..a148f4d
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:31:39" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:32:13" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -4371,8 +4371,8 @@
                                         <OrphaCode>353281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -23699,19 +23699,7 @@
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="3">
-                                <ClassificationNode>
-                                  <Disorder id="20606">
-                                    <OrphaCode>284408</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284408</ExpertLink>
-                                    <Name lang="en">Glycerol kinase deficiency, infantile form</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="2">
                                 <ClassificationNode>
                                   <Disorder id="20607">
                                     <OrphaCode>284411</OrphaCode>
@@ -51186,7 +51174,7 @@
                               <Disorder id="421">
                                 <OrphaCode>1146</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-                                <Name lang="en">Digitotalar dysmorphism</Name>
+                                <Name lang="en">Distal arthrogryposis type 1</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -59055,7 +59043,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="30">
+                      <ClassificationNodeChildList count="31">
                         <ClassificationNode>
                           <Disorder id="28138">
                             <OrphaCode>556955</OrphaCode>
@@ -59920,7 +59908,7 @@
                                   <Disorder id="21690">
                                     <OrphaCode>319671</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-                                    <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+                                    <Name lang="en">Alazami syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -61060,6 +61048,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29565">
+                            <OrphaCode>592570</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                            <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -66363,7 +66363,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="38">
+                      <ClassificationNodeChildList count="39">
                         <ClassificationNode>
                           <Disorder id="29430">
                             <OrphaCode>589515</OrphaCode>
@@ -67264,6 +67264,18 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29564">
+                            <OrphaCode>592564</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592564</ExpertLink>
+                            <Name lang="en">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -70851,7 +70863,19 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="7">
+                        <ClassificationNode>
+                          <Disorder id="29564">
+                            <OrphaCode>592564</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592564</ExpertLink>
+                            <Name lang="en">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="21258">
                             <OrphaCode>306719</OrphaCode>
@@ -73383,7 +73407,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="516">
+                      <ClassificationNodeChildList count="519">
                         <ClassificationNode>
                           <Disorder id="24010">
                             <OrphaCode>464282</OrphaCode>
@@ -73603,8 +73627,8 @@
                                 <OrphaCode>353277</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -73615,8 +73639,8 @@
                                 <OrphaCode>353281</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -73627,8 +73651,8 @@
                                 <OrphaCode>353284</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -78911,7 +78935,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="15">
+                              <ClassificationNodeChildList count="14">
                                 <ClassificationNode>
                                   <Disorder id="123">
                                     <OrphaCode>534</OrphaCode>
@@ -79200,18 +79224,6 @@
                                     </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="20606">
-                                    <OrphaCode>284408</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284408</ExpertLink>
-                                    <Name lang="en">Glycerol kinase deficiency, infantile form</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="22517">
                                     <OrphaCode>371054</OrphaCode>
@@ -81114,7 +81126,7 @@
                               <Disorder id="21690">
                                 <OrphaCode>319671</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+                                <Name lang="en">Alazami syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -81368,7 +81380,7 @@
                           <Disorder id="22327">
                             <OrphaCode>363611</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-                            <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+                            <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -82784,7 +82796,7 @@
                           <Disorder id="26002">
                             <OrphaCode>500163</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=500163</ExpertLink>
-                            <Name lang="en">SIN3A-related intellectual disability syndrome</Name>
+                            <Name lang="en">Witteveen-Kolk syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -82964,7 +82976,7 @@
                           <Disorder id="24042">
                             <OrphaCode>464738</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-                            <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+                            <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -84172,6 +84184,42 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29564">
+                            <OrphaCode>592564</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592564</ExpertLink>
+                            <Name lang="en">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29565">
+                            <OrphaCode>592570</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                            <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29566">
+                            <OrphaCode>592574</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                            <Name lang="en">Menke-Hennekam syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -89525,7 +89573,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="41">
+                  <ClassificationNodeChildList count="42">
                     <ClassificationNode>
                       <Disorder id="25408">
                         <OrphaCode>488618</OrphaCode>
@@ -90246,6 +90294,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29565">
+                        <OrphaCode>592570</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                        <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -92727,7 +92787,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="59">
+                      <ClassificationNodeChildList count="60">
                         <ClassificationNode>
                           <Disorder id="110">
                             <OrphaCode>138</OrphaCode>
@@ -92755,8 +92815,8 @@
                                 <OrphaCode>353277</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -92767,8 +92827,8 @@
                                 <OrphaCode>353281</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -92779,8 +92839,8 @@
                                 <OrphaCode>353284</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -93652,6 +93712,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29566">
+                            <OrphaCode>592574</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                            <Name lang="en">Menke-Hennekam syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -97335,8 +97407,8 @@
                                         <OrphaCode>353277</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -97347,8 +97419,8 @@
                                         <OrphaCode>353281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -97359,8 +97431,8 @@
                                         <OrphaCode>353284</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -100022,7 +100094,7 @@
                                       <Disorder id="24042">
                                         <OrphaCode>464738</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-                                        <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+                                        <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
@@ -102789,7 +102861,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="69">
+                                  <ClassificationNodeChildList count="70">
                                     <ClassificationNode>
                                       <Disorder id="29">
                                         <OrphaCode>578</OrphaCode>
@@ -103390,6 +103462,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="29564">
+                                        <OrphaCode>592564</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592564</ExpertLink>
+                                        <Name lang="en">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="27340">
                                         <OrphaCode>522077</OrphaCode>
@@ -110809,8 +110893,8 @@
                                             <OrphaCode>353277</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21443">
+                                              <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -110821,8 +110905,8 @@
                                             <OrphaCode>353281</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21443">
+                                              <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -110833,8 +110917,8 @@
                                             <OrphaCode>353284</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21443">
+                                              <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -112209,8 +112293,8 @@
                             <OrphaCode>353277</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -112221,8 +112305,8 @@
                             <OrphaCode>353281</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -112233,8 +112317,8 @@
                             <OrphaCode>353284</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -118891,8 +118975,8 @@
                                 <OrphaCode>353277</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -118903,8 +118987,8 @@
                                 <OrphaCode>353281</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -118915,8 +118999,8 @@
                                 <OrphaCode>353284</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -142563,8 +142647,8 @@
                                         <OrphaCode>353277</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -142575,8 +142659,8 @@
                                         <OrphaCode>353281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -142587,8 +142671,8 @@
                                         <OrphaCode>353284</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -153339,7 +153423,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="30">
+                      <ClassificationNodeChildList count="31">
                         <ClassificationNode>
                           <Disorder id="28138">
                             <OrphaCode>556955</OrphaCode>
@@ -154204,7 +154288,7 @@
                                   <Disorder id="21690">
                                     <OrphaCode>319671</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-                                    <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+                                    <Name lang="en">Alazami syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -155344,6 +155428,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29565">
+                            <OrphaCode>592570</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                            <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -155571,7 +155667,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="404">
+                      <ClassificationNodeChildList count="406">
                         <ClassificationNode>
                           <Disorder id="24010">
                             <OrphaCode>464282</OrphaCode>
@@ -157170,7 +157266,7 @@
                               <Disorder id="21690">
                                 <OrphaCode>319671</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+                                <Name lang="en">Alazami syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -157364,7 +157460,7 @@
                           <Disorder id="22327">
                             <OrphaCode>363611</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-                            <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+                            <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -159344,7 +159440,7 @@
                           <Disorder id="26002">
                             <OrphaCode>500163</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=500163</ExpertLink>
-                            <Name lang="en">SIN3A-related intellectual disability syndrome</Name>
+                            <Name lang="en">Witteveen-Kolk syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -159668,7 +159764,7 @@
                           <Disorder id="24042">
                             <OrphaCode>464738</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-                            <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+                            <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -159727,8 +159823,8 @@
                                 <OrphaCode>353277</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -159739,8 +159835,8 @@
                                 <OrphaCode>353281</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -159751,8 +159847,8 @@
                                 <OrphaCode>353284</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -161260,6 +161356,30 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29565">
+                            <OrphaCode>592570</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                            <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29566">
+                            <OrphaCode>592574</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                            <Name lang="en">Menke-Hennekam syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -162704,7 +162824,7 @@
                           <Disorder id="421">
                             <OrphaCode>1146</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-                            <Name lang="en">Digitotalar dysmorphism</Name>
+                            <Name lang="en">Distal arthrogryposis type 1</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -165879,7 +165999,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="14">
+                      <ClassificationNodeChildList count="15">
                         <ClassificationNode>
                           <Disorder id="24074">
                             <OrphaCode>465824</OrphaCode>
@@ -166263,8 +166383,8 @@
                                         <OrphaCode>353277</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -166275,8 +166395,8 @@
                                         <OrphaCode>353281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -166287,8 +166407,8 @@
                                         <OrphaCode>353284</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -166900,7 +167020,7 @@
                                   <Disorder id="421">
                                     <OrphaCode>1146</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-                                    <Name lang="en">Digitotalar dysmorphism</Name>
+                                    <Name lang="en">Distal arthrogryposis type 1</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -169168,6 +169288,18 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29565">
+                            <OrphaCode>592570</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                            <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -169191,7 +169323,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="59">
+                      <ClassificationNodeChildList count="60">
                         <ClassificationNode>
                           <Disorder id="110">
                             <OrphaCode>138</OrphaCode>
@@ -169219,8 +169351,8 @@
                                 <OrphaCode>353277</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -169231,8 +169363,8 @@
                                 <OrphaCode>353281</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -169243,8 +169375,8 @@
                                 <OrphaCode>353284</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -170116,6 +170248,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29566">
+                            <OrphaCode>592574</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                            <Name lang="en">Menke-Hennekam syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -176275,8 +176419,8 @@
                                 <OrphaCode>353277</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -176287,8 +176431,8 @@
                                 <OrphaCode>353281</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -176299,8 +176443,8 @@
                                 <OrphaCode>353284</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                 <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -177125,7 +177269,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="46">
+                  <ClassificationNodeChildList count="47">
                     <ClassificationNode>
                       <Disorder id="23771">
                         <OrphaCode>459061</OrphaCode>
@@ -177153,8 +177297,8 @@
                             <OrphaCode>353277</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -177165,8 +177309,8 @@
                             <OrphaCode>353281</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -177177,8 +177321,8 @@
                             <OrphaCode>353284</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -177894,6 +178038,18 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29566">
+                        <OrphaCode>592574</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                        <Name lang="en">Menke-Hennekam syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="29429">
                         <OrphaCode>589442</OrphaCode>
@@ -178217,8 +178373,8 @@
                                     <OrphaCode>353277</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                     <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -178229,8 +178385,8 @@
                                     <OrphaCode>353281</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                     <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -178241,8 +178397,8 @@
                                     <OrphaCode>353284</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                     <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -194545,8 +194701,8 @@
                                             <OrphaCode>353277</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21443">
+                                              <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -194557,8 +194713,8 @@
                                             <OrphaCode>353281</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21443">
+                                              <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
@@ -194569,8 +194725,8 @@
                                             <OrphaCode>353284</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                            <DisorderType id="21450">
-                                              <Name lang="en">Clinical subtype</Name>
+                                            <DisorderType id="21443">
+                                              <Name lang="en">Etiological subtype</Name>
                                             </DisorderType>
                                           </Disorder>
                                           <ClassificationNodeChildList count="0">
