commit b18c171d5ba8181794aeffcce3d0c8a41ddab6cb
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 5 14:56:57 2022 +0100

    DEC 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml
old mode 100644
new mode 100755
index d0c9c68..daf9e79
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:18:06" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:38:03" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <Availability> 
+    <Licence>
+      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
+      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
+      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
+    </Licence>
+  </Availability>
   <ClassificationList count="1">
     <Classification id="189">
       <OrphaNumber>162718</OrphaNumber>
@@ -14537,18 +14544,6 @@
                                     </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="17944">
-                                    <OrphaCode>171860</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=171860</ExpertLink>
-                                    <Name lang="en">Intellectual disability-cataracts-kyphosis syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="19548">
                                     <OrphaCode>247794</OrphaCode>
@@ -16425,7 +16420,7 @@
                                   <Disorder id="183">
                                     <OrphaCode>637</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637</ExpertLink>
-                                    <Name lang="en">Neurofibromatosis type 2</Name>
+                                    <Name lang="en">Full NF2-related schwannomatosis</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -16605,7 +16600,7 @@
                                   <Disorder id="28140">
                                     <OrphaCode>557003</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=557003</ExpertLink>
-                                    <Name lang="en">Oculocerebrodental syndrome</Name>
+                                    <Name lang="en">Oculoskeletodental syndrome</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -16841,6 +16836,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31665">
+                                    <OrphaCode>633035</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633035</ExpertLink>
+                                    <Name lang="en">Intellectual disability-early-onset cataract-microcephaly syndrome</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                           </ClassificationNodeChildList>
