commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml
index ed44b66..1dc38b9 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:43:47" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:47:28" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -6934,94 +6934,214 @@
                       <Disorder id="16882">
                         <OrphaCode>139390</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139390</ExpertLink>
-                        <Name lang="en">Isolated craniosynostosis</Name>
+                        <Name lang="en">Non-syndromic craniosynostosis</Name>
                         <DisorderType id="21436">
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="7">
-                        <ClassificationNode>
-                          <Disorder id="521">
-                            <OrphaCode>2343</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2343</ExpertLink>
-                            <Name lang="en">Isolated cloverleaf skull syndrome</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2918">
-                            <OrphaCode>3267</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3267</ExpertLink>
-                            <Name lang="en">Familial lambdoid synostosis</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2983">
-                            <OrphaCode>3366</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3366</ExpertLink>
-                            <Name lang="en">Isolated trigonocephaly</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="10367">
-                            <OrphaCode>35093</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35093</ExpertLink>
-                            <Name lang="en">Isolated scaphocephaly</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="10368">
-                            <OrphaCode>35098</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35098</ExpertLink>
-                            <Name lang="en">Isolated plagiocephaly</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
-                          <Disorder id="10369">
-                            <OrphaCode>35099</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35099</ExpertLink>
-                            <Name lang="en">Isolated brachycephaly</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
+                          <Disorder id="31425">
+                            <OrphaCode>620096</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620096</ExpertLink>
+                            <Name lang="en">Non-syndromic unisutural craniosynostosis</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="6">
+                            <ClassificationNode>
+                              <Disorder id="31426">
+                                <OrphaCode>620102</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620102</ExpertLink>
+                                <Name lang="en">Non-syndromic unicoronal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31427">
+                                <OrphaCode>620113</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620113</ExpertLink>
+                                <Name lang="en">Non-syndromic unilambdoid craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31428">
+                                <OrphaCode>620139</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620139</ExpertLink>
+                                <Name lang="en">Non-syndromic unifrontosphenoidal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31429">
+                                <OrphaCode>620146</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620146</ExpertLink>
+                                <Name lang="en">Non-syndromic unisquamosal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="2983">
+                                <OrphaCode>3366</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3366</ExpertLink>
+                                <Name lang="en">Non-syndromic metopic craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="10367">
+                                <OrphaCode>35093</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35093</ExpertLink>
+                                <Name lang="en">Non-syndromic sagittal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="10822">
-                            <OrphaCode>63440</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63440</ExpertLink>
-                            <Name lang="en">Isolated oxycephaly</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
+                          <Disorder id="31430">
+                            <OrphaCode>620152</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620152</ExpertLink>
+                            <Name lang="en">Non-syndromic multisutural craniosynostosis</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="9">
+                            <ClassificationNode>
+                              <Disorder id="31431">
+                                <OrphaCode>620158</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620158</ExpertLink>
+                                <Name lang="en">Non-syndromic non-specific multisutural craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31432">
+                                <OrphaCode>620178</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620178</ExpertLink>
+                                <Name lang="en">Non-syndromic bilambdoid craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31433">
+                                <OrphaCode>620186</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620186</ExpertLink>
+                                <Name lang="en">Non-syndromic unicoronal and sagittal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31434">
+                                <OrphaCode>620192</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620192</ExpertLink>
+                                <Name lang="en">Non-syndromic metopic and sagittal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31435">
+                                <OrphaCode>620198</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620198</ExpertLink>
+                                <Name lang="en">Non-syndromic bicoronal and metopic craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31436">
+                                <OrphaCode>620205</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620205</ExpertLink>
+                                <Name lang="en">Non-syndromic bicoronal and sagittal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31437">
+                                <OrphaCode>620212</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620212</ExpertLink>
+                                <Name lang="en">Non-syndromic pansynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="10369">
+                                <OrphaCode>35099</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35099</ExpertLink>
+                                <Name lang="en">Non-syndromic bicoronal craniosynostosis</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="1683">
+                                <OrphaCode>1516</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1516</ExpertLink>
+                                <Name lang="en">Non-syndromic bilambdoid and sagittal craniosynostosis</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                       </ClassificationNodeChildList>
@@ -7035,7 +7155,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="41">
+                      <ClassificationNodeChildList count="40">
                         <ClassificationNode>
                           <Disorder id="244">
                             <OrphaCode>207</OrphaCode>
@@ -7660,18 +7780,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="1683">
-                            <OrphaCode>1516</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1516</ExpertLink>
-                            <Name lang="en">Craniofacial dyssynostosis</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="23705">
                             <OrphaCode>457193</OrphaCode>
@@ -8333,7 +8441,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="23">
+                  <ClassificationNodeChildList count="21">
                     <ClassificationNode>
                       <Disorder id="1042">
                         <OrphaCode>2311</OrphaCode>
@@ -8514,18 +8622,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="12562">
-                        <OrphaCode>94095</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=94095</ExpertLink>
-                        <Name lang="en">Spondylocostal dysostosis-anal atresia-genitourinary malformation syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="20065">
                         <OrphaCode>263508</OrphaCode>
@@ -8550,18 +8646,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="21917">
-                        <OrphaCode>329252</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329252</ExpertLink>
-                        <Name lang="en">Spondylocostal dysostosis-hypospadias-intellectual disability syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="10656">
                         <OrphaCode>50817</OrphaCode>
@@ -8645,7 +8729,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="8">
+                  <ClassificationNodeChildList count="7">
                     <ClassificationNode>
                       <Disorder id="392">
                         <OrphaCode>2614</OrphaCode>
@@ -8658,18 +8742,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="727">
-                        <OrphaCode>1428</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1428</ExpertLink>
-                        <Name lang="en">Familial chondromalacia patellae</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="1676">
                         <OrphaCode>1509</OrphaCode>
@@ -11679,7 +11751,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="47">
+                      <ClassificationNodeChildList count="46">
                         <ClassificationNode>
                           <Disorder id="2956">
                             <OrphaCode>3332</OrphaCode>
@@ -12256,18 +12328,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2804">
-                            <OrphaCode>3133</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3133</ExpertLink>
-                            <Name lang="en">Say-Field-Coldwell syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2831">
                             <OrphaCode>3168</OrphaCode>
