commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml
index c836a0b..f26c425 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_199.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:05:52" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:24:27" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -21,7 +21,7 @@
               <Name lang="en">Category</Name>
             </DisorderType>
           </Disorder>
-          <ClassificationNodeChildList count="10">
+          <ClassificationNodeChildList count="11">
             <ClassificationNode>
               <Disorder id="10552">
                 <OrphaCode>68411</OrphaCode>
@@ -283,7 +283,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="18">
+              <ClassificationNodeChildList count="19">
                 <ClassificationNode>
                   <Disorder id="3">
                     <OrphaCode>61</OrphaCode>
@@ -764,6 +764,30 @@
                     </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32001">
+                    <OrphaCode>662216</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662216</ExpertLink>
+                    <Name lang="en">Mucopolysaccharidosis type 10</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+              </ClassificationNodeChildList>
+            </ClassificationNode>
+            <ClassificationNode>
+              <Disorder id="17787">
+                <OrphaCode>168621</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168621</ExpertLink>
+                <Name lang="en">Dysplasia of head of femur, Meyer type</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="0">
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
@@ -895,19 +919,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="27">
-                <ClassificationNode>
-                  <Disorder id="25395">
-                    <OrphaCode>488265</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=488265</ExpertLink>
-                    <Name lang="en">Osteofibrous dysplasia</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
+              <ClassificationNodeChildList count="26">
                 <ClassificationNode>
                   <Disorder id="25912">
                     <OrphaCode>498445</OrphaCode>
@@ -1062,6 +1074,54 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31953">
+                        <OrphaCode>659387</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659387</ExpertLink>
+                        <Name lang="en">PRC-2 complex-related overgrowth spectrum</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="3">
+                        <ClassificationNode>
+                          <Disorder id="604">
+                            <OrphaCode>3447</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3447</ExpertLink>
+                            <Name lang="en">Weaver syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31955">
+                            <OrphaCode>659396</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659396</ExpertLink>
+                            <Name lang="en">Cohen-Gibson syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31957">
+                            <OrphaCode>659463</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=659463</ExpertLink>
+                            <Name lang="en">Imagawa-Matsumoto syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="109">
                         <OrphaCode>558</OrphaCode>
@@ -1098,18 +1158,6 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="604">
-                        <OrphaCode>3447</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3447</ExpertLink>
-                        <Name lang="en">Weaver syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="588">
                         <OrphaCode>821</OrphaCode>
@@ -1445,7 +1493,19 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="59">
+                  <ClassificationNodeChildList count="61">
+                    <ClassificationNode>
+                      <Disorder id="32028">
+                        <OrphaCode>664377</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=664377</ExpertLink>
+                        <Name lang="en">MGP-related spondyloepiphyseal dysplasia</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="31731">
                         <OrphaCode>642099</OrphaCode>
@@ -1470,6 +1530,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="2024">
+                        <OrphaCode>2114</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2114</ExpertLink>
+                        <Name lang="en">Hip dysplasia, Beukes type</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="215">
                         <OrphaCode>800</OrphaCode>
@@ -2062,7 +2134,7 @@
                       <Disorder id="20061">
                         <OrphaCode>263482</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263482</ExpertLink>
-                        <Name lang="en">Spondyloepiphyseal dysplasia, Maroteaux type</Name>
+                        <Name lang="en">Spondyloepimetaphyseal dysplasia, Maroteaux type</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -2122,7 +2194,7 @@
                       <Disorder id="31730">
                         <OrphaCode>642085</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642085</ExpertLink>
-                        <Name lang="en">Spondyloepimetaphyseal dysplasia with joint laxity, EXOC6B type</Name>
+                        <Name lang="en">EXOC6B-related spondyloepimetaphyseal dysplasia with joint laxity</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -2561,7 +2633,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="6">
+                  <ClassificationNodeChildList count="4">
                     <ClassificationNode>
                       <Disorder id="1809">
                         <OrphaCode>251</OrphaCode>
@@ -2571,7 +2643,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="10">
+                      <ClassificationNodeChildList count="9">
                         <ClassificationNode>
                           <Disorder id="12244">
                             <OrphaCode>93307</OrphaCode>
@@ -2620,18 +2692,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="17598">
-                            <OrphaCode>166011</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166011</ExpertLink>
-                            <Name lang="en">Multiple epiphyseal dysplasia, Beighton type</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="17599">
                             <OrphaCode>166016</OrphaCode>
@@ -2648,7 +2708,7 @@
                           <Disorder id="17601">
                             <OrphaCode>166024</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
-                            <Name lang="en">Multiple epiphyseal dysplasia, Al-Gazali type</Name>
+                            <Name lang="en">Multiple epiphyseal dysplasia-macrocephaly-facial dysmorphism syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2660,7 +2720,7 @@
                           <Disorder id="17602">
                             <OrphaCode>166029</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166029</ExpertLink>
-                            <Name lang="en">Multiple epiphyseal dysplasia, with severe proximal femoral dysplasia</Name>
+                            <Name lang="en">Multiple epiphyseal dysplasia-severe proximal femoral dysplasia syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2672,7 +2732,7 @@
                           <Disorder id="17603">
                             <OrphaCode>166032</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166032</ExpertLink>
-                            <Name lang="en">Multiple epiphyseal dysplasia, with miniepiphyses</Name>
+                            <Name lang="en">Multiple epiphyseal dysplasia-miniepiphyses syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2706,18 +2766,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="2024">
-                        <OrphaCode>2114</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2114</ExpertLink>
-                        <Name lang="en">Hip dysplasia, Beukes type</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="2676">
                         <OrphaCode>750</OrphaCode>
@@ -2730,18 +2778,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="17787">
-                        <OrphaCode>168621</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168621</ExpertLink>
-                        <Name lang="en">Dysplasia of head of femur, Meyer type</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="19597">
                         <OrphaCode>250984</OrphaCode>
@@ -2765,7 +2801,7 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="12">
+                  <ClassificationNodeChildList count="11">
                     <ClassificationNode>
                       <Disorder id="1346">
                         <OrphaCode>1040</OrphaCode>
@@ -2790,18 +2826,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="1486">
-                        <OrphaCode>1240</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1240</ExpertLink>
-                        <Name lang="en">Metaphyseal acroscyphodysplasia</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="1624">
                         <OrphaCode>174</OrphaCode>
@@ -2818,7 +2842,7 @@
                       <Disorder id="1816">
                         <OrphaCode>1837</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1837</ExpertLink>
-                        <Name lang="en">Ulna metaphyseal dysplasia syndrome</Name>
+                        <Name lang="en">Metaphyseal chondrodysplasia, Rosenberg type</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -3509,7 +3533,7 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="24">
+                  <ClassificationNodeChildList count="23">
                     <ClassificationNode>
                       <Disorder id="1043">
                         <OrphaCode>240</OrphaCode>
@@ -3534,18 +3558,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="1776">
-                        <OrphaCode>1765</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1765</ExpertLink>
-                        <Name lang="en">Dyschondrosteosis-nephritis syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="1815">
                         <OrphaCode>1836</OrphaCode>
@@ -3574,7 +3586,7 @@
                       <Disorder id="2309">
                         <OrphaCode>2497</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2497</ExpertLink>
-                        <Name lang="en">Upper limb mesomelic dysplasia</Name>
+                        <Name lang="en">Upper limb mesomelic dysplasia, type Fryns</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -3857,7 +3869,7 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="10">
+                  <ClassificationNodeChildList count="9">
                     <ClassificationNode>
                       <Disorder id="31925">
                         <OrphaCode>656283</OrphaCode>
@@ -3906,18 +3918,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="2421">
-                        <OrphaCode>2646</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2646</ExpertLink>
-                        <Name lang="en">Parastremmatic dwarfism</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="2521">
                         <OrphaCode>2768</OrphaCode>
@@ -4462,7 +4462,7 @@
                       <Disorder id="31730">
                         <OrphaCode>642085</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642085</ExpertLink>
-                        <Name lang="en">Spondyloepimetaphyseal dysplasia with joint laxity, EXOC6B type</Name>
+                        <Name lang="en">EXOC6B-related spondyloepimetaphyseal dysplasia with joint laxity</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -4661,19 +4661,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="30">
-                    <ClassificationNode>
-                      <Disorder id="10698">
-                        <OrphaCode>52430</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=52430</ExpertLink>
-                        <Name lang="en">Inclusion body myopathy with Paget disease of bone and frontotemporal dementia</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="27">
                     <ClassificationNode>
                       <Disorder id="556">
                         <OrphaCode>2801</OrphaCode>
@@ -4722,23 +4710,11 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="1715">
-                        <OrphaCode>1562</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1562</ExpertLink>
-                        <Name lang="en">Dacryocystitis-osteopoikilosis syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="1794">
                         <OrphaCode>1798</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1798</ExpertLink>
-                        <Name lang="en">Dysostosis, Stanescu type</Name>
+                        <Name lang="en">Craniofacial dysostosis-diaphyseal hyperplasia syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -5228,7 +5204,7 @@
                           <Disorder id="2713">
                             <OrphaCode>1832</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1832</ExpertLink>
-                            <Name lang="en">Lethal osteosclerotic bone dysplasia</Name>
+                            <Name lang="en">Osteosclerotic bone dysplasia</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -5346,18 +5322,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="21937">
-                        <OrphaCode>329475</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329475</ExpertLink>
-                        <Name lang="en">Spastic paraplegia-Paget disease of bone syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -6025,7 +5989,55 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="6">
+                          <ClassificationNodeChildList count="7">
+                            <ClassificationNode>
+                              <Disorder id="11">
+                                <OrphaCode>213</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213</ExpertLink>
+                                <Name lang="en">Cystinosis</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="3">
+                                <ClassificationNode>
+                                  <Disorder id="23023">
+                                    <OrphaCode>411629</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=411629</ExpertLink>
+                                    <Name lang="en">Infantile nephropathic cystinosis</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="23024">
+                                    <OrphaCode>411634</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=411634</ExpertLink>
+                                    <Name lang="en">Juvenile nephropathic cystinosis</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="23025">
+                                    <OrphaCode>411641</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=411641</ExpertLink>
+                                    <Name lang="en">Ocular cystinosis</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="19477">
                                 <OrphaCode>244305</OrphaCode>
@@ -6389,7 +6401,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="27">
+                  <ClassificationNodeChildList count="25">
                     <ClassificationNode>
                       <Disorder id="8684">
                         <OrphaCode>73</OrphaCode>
@@ -6666,18 +6678,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="10698">
-                        <OrphaCode>52430</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=52430</ExpertLink>
-                        <Name lang="en">Inclusion body myopathy with Paget disease of bone and frontotemporal dementia</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="10739">
                         <OrphaCode>53697</OrphaCode>
@@ -6750,18 +6750,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="14220">
-                        <OrphaCode>99647</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99647</ExpertLink>
-                        <Name lang="en">Cheirospondyloenchondromatosis</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="17509">
                         <OrphaCode>163634</OrphaCode>
@@ -6775,10 +6763,10 @@
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="21937">
-                        <OrphaCode>329475</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329475</ExpertLink>
-                        <Name lang="en">Spastic paraplegia-Paget disease of bone syndrome</Name>
+                      <Disorder id="25395">
+                        <OrphaCode>488265</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=488265</ExpertLink>
+                        <Name lang="en">Osteofibrous dysplasia</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -7216,18 +7204,6 @@
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="62">
-                        <ClassificationNode>
-                          <Disorder id="10880">
-                            <OrphaCode>65798</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=65798</ExpertLink>
-                            <Name lang="en">Goodman syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="244">
                             <OrphaCode>207</OrphaCode>
@@ -7741,7 +7717,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="7">
+                          <ClassificationNodeChildList count="8">
                             <ClassificationNode>
                               <Disorder id="131">
                                 <OrphaCode>580</OrphaCode>
@@ -7838,6 +7814,18 @@
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32001">
+                                <OrphaCode>662216</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662216</ExpertLink>
+                                <Name lang="en">Mucopolysaccharidosis type 10</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="872">
                                 <OrphaCode>582</OrphaCode>
@@ -8020,18 +8008,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2865">
-                            <OrphaCode>3210</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3210</ExpertLink>
-                            <Name lang="en">Summitt syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="22433">
                             <OrphaCode>369837</OrphaCode>
@@ -8200,6 +8176,30 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32075">
+                            <OrphaCode>672979</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=672979</ExpertLink>
+                            <Name lang="en">Craniosynostosis-facial dysmorphism-brachydactyly syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32076">
+                            <OrphaCode>672985</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=672985</ExpertLink>
+                            <Name lang="en">Craniosynostosis-skeletal and cerebellar anomalies-learning disabilities syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2184">
                             <OrphaCode>2332</OrphaCode>
@@ -8607,7 +8607,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="14">
+                      <ClassificationNodeChildList count="10">
                         <ClassificationNode>
                           <Disorder id="2506">
                             <OrphaCode>2750</OrphaCode>
@@ -8632,18 +8632,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2508">
-                            <OrphaCode>2752</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2752</ExpertLink>
-                            <Name lang="en">Orofaciodigital syndrome type 3</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2509">
                             <OrphaCode>2753</OrphaCode>
@@ -8692,18 +8680,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="3165">
-                            <OrphaCode>2756</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2756</ExpertLink>
-                            <Name lang="en">Orofaciodigital syndrome type 10</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="17017">
                             <OrphaCode>141000</OrphaCode>
@@ -8728,30 +8704,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="17078">
-                            <OrphaCode>141327</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141327</ExpertLink>
-                            <Name lang="en">Orofaciodigital syndrome type 12</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="17079">
-                            <OrphaCode>141330</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141330</ExpertLink>
-                            <Name lang="en">Orofaciodigital syndrome type 13</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="23271">
                             <OrphaCode>434179</OrphaCode>
@@ -8893,7 +8845,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="3">
+                          <ClassificationNodeChildList count="2">
                             <ClassificationNode>
                               <Disorder id="237">
                                 <OrphaCode>107</OrphaCode>
@@ -8906,18 +8858,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="218">
-                                <OrphaCode>857</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=857</ExpertLink>
-                                <Name lang="en">Townes-Brocks syndrome</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="1529">
                                 <OrphaCode>1296</OrphaCode>
@@ -8969,7 +8909,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="21">
+                  <ClassificationNodeChildList count="19">
                     <ClassificationNode>
                       <Disorder id="1042">
                         <OrphaCode>2311</OrphaCode>
@@ -9138,30 +9078,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="11658">
-                        <OrphaCode>85200</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85200</ExpertLink>
-                        <Name lang="en">Ischiovertebral syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="20065">
-                        <OrphaCode>263508</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263508</ExpertLink>
-                        <Name lang="en">COG1-CDG</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="21905">
                         <OrphaCode>329191</OrphaCode>
@@ -9298,7 +9214,7 @@
                       <Disorder id="11728">
                         <OrphaCode>86789</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86789</ExpertLink>
-                        <Name lang="en">Patella aplasia/hypoplasia</Name>
+                        <Name lang="en">Isolated patella aplasia/hypoplasia</Name>
                         <DisorderType id="21415">
                           <Name lang="en">Morphological anomaly</Name>
                         </DisorderType>
@@ -9411,7 +9327,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="13">
+                      <ClassificationNodeChildList count="12">
                         <ClassificationNode>
                           <Disorder id="1864">
                             <OrphaCode>1896</OrphaCode>
@@ -9544,18 +9460,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2609">
-                            <OrphaCode>2878</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2878</ExpertLink>
-                            <Name lang="en">Phocomelia-ectrodactyly-deafness-sinus arrhythmia syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2261">
                             <OrphaCode>2440</OrphaCode>
@@ -9817,7 +9721,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="23">
+                          <ClassificationNodeChildList count="21">
                             <ClassificationNode>
                               <Disorder id="29437">
                                 <OrphaCode>589608</OrphaCode>
@@ -9842,30 +9746,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="22382">
-                                <OrphaCode>364577</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=364577</ExpertLink>
-                                <Name lang="en">Intellectual disability-brachydactyly-Pierre Robin syndrome</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="23479">
-                                <OrphaCode>444077</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=444077</ExpertLink>
-                                <Name lang="en">Cognitive impairment-coarse facies-heart defects-obesity-pulmonary involvement-short stature-skeletal dysplasia syndrome</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="1313">
                                 <OrphaCode>1001</OrphaCode>
@@ -10172,7 +10052,7 @@
                           <Disorder id="25916">
                             <OrphaCode>498457</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=498457</ExpertLink>
-                            <Name lang="en">Longitudinal limb defect</Name>
+                            <Name lang="en">Non-syndromic longitudinal limb defect</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -10182,7 +10062,7 @@
                               <Disorder id="2035">
                                 <OrphaCode>2130</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2130</ExpertLink>
-                                <Name lang="en">Hemimelia</Name>
+                                <Name lang="en">Non-syndromic hemimelia</Name>
                                 <DisorderType id="21436">
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
@@ -10192,7 +10072,7 @@
                                   <Disorder id="12257">
                                     <OrphaCode>93320</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93320</ExpertLink>
-                                    <Name lang="en">Ulnar hemimelia</Name>
+                                    <Name lang="en">Isolated ulnar hemimelia</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10204,7 +10084,7 @@
                                   <Disorder id="12258">
                                     <OrphaCode>93321</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93321</ExpertLink>
-                                    <Name lang="en">Radial hemimelia</Name>
+                                    <Name lang="en">Isolated radial hemimelia</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10216,7 +10096,7 @@
                                   <Disorder id="12259">
                                     <OrphaCode>93322</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93322</ExpertLink>
-                                    <Name lang="en">Tibial hemimelia</Name>
+                                    <Name lang="en">Isolated tibial hemimelia</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10228,7 +10108,7 @@
                                   <Disorder id="12260">
                                     <OrphaCode>93323</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93323</ExpertLink>
-                                    <Name lang="en">Fibular hemimelia</Name>
+                                    <Name lang="en">Isolated fibular hemimelia</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10242,7 +10122,7 @@
                               <Disorder id="20943">
                                 <OrphaCode>294988</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294988</ExpertLink>
-                                <Name lang="en">Congenital hypoplasia of thumb</Name>
+                                <Name lang="en">Isolated hypoplasia of thumb</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10256,7 +10136,7 @@
                           <Disorder id="25917">
                             <OrphaCode>498461</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=498461</ExpertLink>
-                            <Name lang="en">Terminal transverse limb defect</Name>
+                            <Name lang="en">Non-syndromic terminal transverse limb defect</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -10266,7 +10146,7 @@
                               <Disorder id="1255">
                                 <OrphaCode>931</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=931</ExpertLink>
-                                <Name lang="en">Acheiropodia</Name>
+                                <Name lang="en">Isolated acheiropodia</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10278,7 +10158,7 @@
                               <Disorder id="1293">
                                 <OrphaCode>973</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=973</ExpertLink>
-                                <Name lang="en">Congenital absence/hypoplasia of fingers excluding thumb, unilateral</Name>
+                                <Name lang="en">Isolated absence/hypoplasia of fingers excluding thumb, unilateral</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10290,7 +10170,7 @@
                               <Disorder id="20913">
                                 <OrphaCode>294925</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294925</ExpertLink>
-                                <Name lang="en">Amelia</Name>
+                                <Name lang="en">Non-syndromic amelia</Name>
                                 <DisorderType id="21436">
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
@@ -10300,7 +10180,7 @@
                                   <Disorder id="20933">
                                     <OrphaCode>294967</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294967</ExpertLink>
-                                    <Name lang="en">Amelia of upper limb</Name>
+                                    <Name lang="en">Isolated amelia of upper limb</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10312,7 +10192,7 @@
                                   <Disorder id="20934">
                                     <OrphaCode>294969</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294969</ExpertLink>
-                                    <Name lang="en">Amelia of lower limb</Name>
+                                    <Name lang="en">Isolated amelia of lower limb</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10324,7 +10204,7 @@
                                   <Disorder id="20935">
                                     <OrphaCode>294971</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294971</ExpertLink>
-                                    <Name lang="en">Tetra-amelia</Name>
+                                    <Name lang="en">Isolated tetra-amelia</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10338,7 +10218,7 @@
                               <Disorder id="20949">
                                 <OrphaCode>295000</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295000</ExpertLink>
-                                <Name lang="en">Constriction rings syndrome</Name>
+                                <Name lang="en">Amniotic band syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -10350,7 +10230,7 @@
                               <Disorder id="25926">
                                 <OrphaCode>498491</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=498491</ExpertLink>
-                                <Name lang="en">Complete hemimelia</Name>
+                                <Name lang="en">Non-syndromic complete hemimelia</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -10360,7 +10240,7 @@
                                   <Disorder id="20939">
                                     <OrphaCode>294979</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294979</ExpertLink>
-                                    <Name lang="en">Congenital absence of both forearm and hand</Name>
+                                    <Name lang="en">Isolated absence of both forearm and hand</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10372,7 +10252,7 @@
                                   <Disorder id="20940">
                                     <OrphaCode>294981</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294981</ExpertLink>
-                                    <Name lang="en">Congenital absence of both lower leg and foot</Name>
+                                    <Name lang="en">Isolated absence of both lower leg and foot</Name>
                                     <DisorderType id="21415">
                                       <Name lang="en">Morphological anomaly</Name>
                                     </DisorderType>
@@ -10386,7 +10266,7 @@
                               <Disorder id="20941">
                                 <OrphaCode>294983</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294983</ExpertLink>
-                                <Name lang="en">Acheiria</Name>
+                                <Name lang="en">Isolated acheiria</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10398,7 +10278,7 @@
                               <Disorder id="20942">
                                 <OrphaCode>294986</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294986</ExpertLink>
-                                <Name lang="en">Apodia</Name>
+                                <Name lang="en">Isolated apodia</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10412,17 +10292,17 @@
                           <Disorder id="20914">
                             <OrphaCode>294927</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294927</ExpertLink>
-                            <Name lang="en">Intercalary limb defects</Name>
+                            <Name lang="en">Non-syndromic intercalary limb defects</Name>
                             <DisorderType id="21436">
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="5">
+                          <ClassificationNodeChildList count="6">
                             <ClassificationNode>
                               <Disorder id="20937">
                                 <OrphaCode>294975</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294975</ExpertLink>
-                                <Name lang="en">Congenital absence of upper arm and forearm with hand present</Name>
+                                <Name lang="en">Isolated absence of upper arm and forearm with hand present</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10434,7 +10314,7 @@
                               <Disorder id="20938">
                                 <OrphaCode>294977</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294977</ExpertLink>
-                                <Name lang="en">Congenital absence of thigh and lower leg with foot present</Name>
+                                <Name lang="en">Isolated absence of thigh and lower leg with foot present</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10446,9 +10326,9 @@
                               <Disorder id="1927">
                                 <OrphaCode>1987</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1987</ExpertLink>
-                                <Name lang="en">Femoral agenesis/hypoplasia</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
+                                <Name lang="en">Isolated femoral agenesis/hypoplasia</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -10458,7 +10338,7 @@
                               <Disorder id="20936">
                                 <OrphaCode>294973</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294973</ExpertLink>
-                                <Name lang="en">Humeral agenesis/hypoplasia</Name>
+                                <Name lang="en">Isolated humeral agenesis/hypoplasia</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10470,7 +10350,19 @@
                               <Disorder id="31680">
                                 <OrphaCode>633228</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633228</ExpertLink>
-                                <Name lang="en">Proximal femoral focal deficiency</Name>
+                                <Name lang="en">Isolated proximal femoral focal deficiency</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32063">
+                                <OrphaCode>667589</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=667589</ExpertLink>
+                                <Name lang="en">Isolated congenital femoral bifurcation</Name>
                                 <DisorderType id="21415">
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
@@ -10832,7 +10724,7 @@
                           <Disorder id="20950">
                             <OrphaCode>295002</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295002</ExpertLink>
-                            <Name lang="en">Hyperphalangy</Name>
+                            <Name lang="en">Isolated hyperphalangy</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -11014,7 +10906,7 @@
                       <Disorder id="17148">
                         <OrphaCode>157808</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=157808</ExpertLink>
-                        <Name lang="en">Congenital pseudoarthrosis of the limbs</Name>
+                        <Name lang="en">Isolated pseudoarthrosis of the limbs</Name>
                         <DisorderType id="21415">
                           <Name lang="en">Morphological anomaly</Name>
                         </DisorderType>
@@ -11230,17 +11122,17 @@
                       <Disorder id="20924">
                         <OrphaCode>294949</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294949</ExpertLink>
-                        <Name lang="en">Joint formation defects</Name>
+                        <Name lang="en">Non-syndromic joint formation defects</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="7">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
                           <Disorder id="2905">
                             <OrphaCode>3248</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3248</ExpertLink>
-                            <Name lang="en">Distal symphalangism</Name>
+                            <Name lang="en">Isolated distal symphalangism</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -11252,7 +11144,7 @@
                           <Disorder id="2916">
                             <OrphaCode>3265</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3265</ExpertLink>
-                            <Name lang="en">Humero-radial synostosis</Name>
+                            <Name lang="en">Isolated humero-radial synostosis</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -11264,7 +11156,7 @@
                           <Disorder id="2917">
                             <OrphaCode>3266</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3266</ExpertLink>
-                            <Name lang="en">Humero-radio-ulnar synostosis</Name>
+                            <Name lang="en">Isolated humero-radio-ulnar synostosis</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -11276,79 +11168,19 @@
                           <Disorder id="3489">
                             <OrphaCode>3269</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3269</ExpertLink>
-                            <Name lang="en">Congenital radioulnar synostosis</Name>
+                            <Name lang="en">Isolated radio-ulnar synostosis</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="21054">
-                                <OrphaCode>295217</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295217</ExpertLink>
-                                <Name lang="en">Radio-ulnar synostosis, unilateral</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21055">
-                                <OrphaCode>295219</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295219</ExpertLink>
-                                <Name lang="en">Radio-ulnar synostosis, bilateral</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="12538">
                             <OrphaCode>94056</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=94056</ExpertLink>
-                            <Name lang="en">Humero-ulnar synostosis</Name>
-                            <DisorderType id="21415">
-                              <Name lang="en">Morphological anomaly</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="21052">
-                                <OrphaCode>295213</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295213</ExpertLink>
-                                <Name lang="en">Humero-ulnar synostosis, unilateral</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21053">
-                                <OrphaCode>295215</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295215</ExpertLink>
-                                <Name lang="en">Humero-ulnar synostosis, bilateral</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="18818">
-                            <OrphaCode>217059</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217059</ExpertLink>
-                            <Name lang="en">Isolated congenital digital clubbing</Name>
+                            <Name lang="en">Isolated humero-ulnar synostosis</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -11360,7 +11192,7 @@
                           <Disorder id="20963">
                             <OrphaCode>295028</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295028</ExpertLink>
-                            <Name lang="en">Tibio-fibular synostosis</Name>
+                            <Name lang="en">Isolated tibio-fibular synostosis</Name>
                             <DisorderType id="21415">
                               <Name lang="en">Morphological anomaly</Name>
                             </DisorderType>
@@ -11379,7 +11211,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="49">
+                      <ClassificationNodeChildList count="48">
                         <ClassificationNode>
                           <Disorder id="25392">
                             <OrphaCode>488232</OrphaCode>
@@ -11704,18 +11536,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="2609">
-                            <OrphaCode>2878</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2878</ExpertLink>
-                            <Name lang="en">Phocomelia-ectrodactyly-deafness-sinus arrhythmia syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2610">
                             <OrphaCode>2879</OrphaCode>
@@ -12129,8 +11949,8 @@
                             <OrphaCode>658805</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658805</ExpertLink>
                             <Name lang="en">Greig cephalopolysyndactyly-contiguous gene syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -12193,7 +12013,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="14">
+                          <ClassificationNodeChildList count="10">
                             <ClassificationNode>
                               <Disorder id="2506">
                                 <OrphaCode>2750</OrphaCode>
@@ -12218,18 +12038,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="2508">
-                                <OrphaCode>2752</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2752</ExpertLink>
-                                <Name lang="en">Orofaciodigital syndrome type 3</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="2509">
                                 <OrphaCode>2753</OrphaCode>
@@ -12278,18 +12086,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="3165">
-                                <OrphaCode>2756</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2756</ExpertLink>
-                                <Name lang="en">Orofaciodigital syndrome type 10</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="17017">
                                 <OrphaCode>141000</OrphaCode>
@@ -12314,30 +12110,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="17078">
-                                <OrphaCode>141327</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141327</ExpertLink>
-                                <Name lang="en">Orofaciodigital syndrome type 12</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="17079">
-                                <OrphaCode>141330</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141330</ExpertLink>
-                                <Name lang="en">Orofaciodigital syndrome type 13</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="23271">
                                 <OrphaCode>434179</OrphaCode>
