commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_188.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_188.xml
index 69a58dc..9aee5b9 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_188.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_188.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:23:35" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:08:02" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -593,7 +593,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="64">
+                  <ClassificationNodeChildList count="65">
                     <ClassificationNode>
                       <Disorder id="44">
                         <OrphaCode>881</OrphaCode>
@@ -608,7 +608,7 @@
                           <Disorder id="14199">
                             <OrphaCode>99226</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-                            <Name lang="en">Monosomy X</Name>
+                            <Name lang="en">Monosomy X syndrome</Name>
                             <DisorderType id="21443">
                               <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
@@ -620,7 +620,7 @@
                           <Disorder id="14200">
                             <OrphaCode>99228</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-                            <Name lang="en">Mosaic monosomy X</Name>
+                            <Name lang="en">Mosaic monosomy X syndrome</Name>
                             <DisorderType id="21443">
                               <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
@@ -934,7 +934,7 @@
                       <Disorder id="337">
                         <OrphaCode>3378</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3378</ExpertLink>
-                        <Name lang="en">Trisomy 13</Name>
+                        <Name lang="en">Trisomy 13 syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -946,7 +946,7 @@
                       <Disorder id="339">
                         <OrphaCode>3380</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3380</ExpertLink>
-                        <Name lang="en">Trisomy 18</Name>
+                        <Name lang="en">Trisomy 18 syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -1578,6 +1578,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="32218">
+                        <OrphaCode>689822</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=689822</ExpertLink>
+                        <Name lang="en">Structural heart defects-renal anomalies syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
               </ClassificationNodeChildList>
@@ -1637,7 +1649,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="2">
+                  <ClassificationNodeChildList count="3">
                     <ClassificationNode>
                       <Disorder id="28594">
                         <OrphaCode>567556</OrphaCode>
@@ -2382,6 +2394,18 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="11738">
+                        <OrphaCode>86818</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86818</ExpertLink>
+                        <Name lang="en">Alport syndrome-intellectual disability-midface hypoplasia-elliptocytosis syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -2492,7 +2516,7 @@
                           <Disorder id="3551">
                             <OrphaCode>656</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656</ExpertLink>
-                            <Name lang="en">Genetic steroid-resistant nephrotic syndrome</Name>
+                            <Name lang="en">Hereditary steroid-resistant nephrotic syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -4447,7 +4471,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="37">
+              <ClassificationNodeChildList count="38">
                 <ClassificationNode>
                   <Disorder id="1587">
                     <OrphaCode>1380</OrphaCode>
@@ -5444,6 +5468,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="32193">
+                    <OrphaCode>688581</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=688581</ExpertLink>
+                    <Name lang="en">Midface hypoplasia-hearing impairment-elliptocytosis-nephrocalcinosis syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="31442">
                     <OrphaCode>620371</OrphaCode>
