commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml
index ba57025..6da3961 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_152.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:40:06" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:41:35" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -3067,7 +3067,7 @@
                   <Name lang="en">Clinical group</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="4">
+              <ClassificationNodeChildList count="3">
                 <ClassificationNode>
                   <Disorder id="833">
                     <OrphaCode>2070</OrphaCode>
@@ -3080,18 +3080,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="11041">
-                    <OrphaCode>73247</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73247</ExpertLink>
-                    <Name lang="en">Eosinophilic esophagitis</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="22875">
                     <OrphaCode>402035</OrphaCode>
