commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml
index c2251c5..9767c6a 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:44:13" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:47:55" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -11,7 +11,55 @@
     <Classification id="233">
       <OrphaNumber>506203</OrphaNumber>
       <Name lang="en">Orphanet classification of rare transplant-related disorders</Name>
-      <ClassificationNodeRootList count="1">
+      <ClassificationNodeRootList count="2">
+        <ClassificationNode>
+          <Disorder id="10344">
+            <OrphaCode>34526</OrphaCode>
+            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34526</ExpertLink>
+            <Name lang="en">OBSOLETE: Genetic primary hypomagnesemia</Name>
+            <DisorderType id="36561">
+              <Name lang="en">Category</Name>
+            </DisorderType>
+          </Disorder>
+          <ClassificationNodeChildList count="1">
+            <ClassificationNode>
+              <Disorder id="21222">
+                <OrphaCode>306516</OrphaCode>
+                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306516</ExpertLink>
+                <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis</Name>
+                <DisorderType id="21394">
+                  <Name lang="en">Disease</Name>
+                </DisorderType>
+              </Disorder>
+              <ClassificationNodeChildList count="2">
+                <ClassificationNode>
+                  <Disorder id="2078">
+                    <OrphaCode>2196</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
+                    <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
+                    <DisorderType id="21450">
+                      <Name lang="en">Clinical subtype</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="9284">
+                    <OrphaCode>31043</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31043</ExpertLink>
+                    <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis without severe ocular involvement</Name>
+                    <DisorderType id="21450">
+                      <Name lang="en">Clinical subtype</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+              </ClassificationNodeChildList>
+            </ClassificationNode>
+          </ClassificationNodeChildList>
+        </ClassificationNode>
         <ClassificationNode>
           <Disorder id="28445">
             <OrphaCode>565779</OrphaCode>
@@ -668,7 +716,7 @@
                           <Disorder id="11036">
                             <OrphaCode>73224</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73224</ExpertLink>
-                            <Name lang="en">Tubular renal disease-cardiomyopathy syndrome</Name>
+                            <Name lang="en">Kidney tubulopathy-dilated cardiomyopathy syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1264,7 +1312,7 @@
                                   <Disorder id="21781">
                                     <OrphaCode>324525</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=324525</ExpertLink>
-                                    <Name lang="en">Hypertrophic cardiomyopathy and renal tubular disease due to mitochondrial DNA mutation</Name>
+                                    <Name lang="en">Hypertrophic cardiomyopathy with kidney anomalies due to mitochondrial DNA mutation</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -2045,7 +2093,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="39">
+                  <ClassificationNodeChildList count="40">
                     <ClassificationNode>
                       <Disorder id="16">
                         <OrphaCode>367</OrphaCode>
@@ -3846,6 +3894,54 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31389">
+                        <OrphaCode>618899</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618899</ExpertLink>
+                        <Name lang="en">Acid sphingomyelinase deficiency</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="3">
+                        <ClassificationNode>
+                          <Disorder id="11105">
+                            <OrphaCode>77292</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
+                            <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="11106">
+                            <OrphaCode>77293</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
+                            <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31388">
+                            <OrphaCode>618891</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
+                            <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -3857,7 +3953,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="74">
+                  <ClassificationNodeChildList count="80">
                     <ClassificationNode>
                       <Disorder id="3529">
                         <OrphaCode>416</OrphaCode>
@@ -7263,12 +7359,24 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
+                      <ClassificationNodeChildList count="5">
                         <ClassificationNode>
                           <Disorder id="28672">
                             <OrphaCode>570371</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=570371</ExpertLink>
-                            <Name lang="en">Transient antenatal Bartter syndrome</Name>
+                            <Name lang="en">Bartter syndrome type 5</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31438">
+                            <OrphaCode>620217</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620217</ExpertLink>
+                            <Name lang="en">Bartter syndrome type 1</Name>
                             <DisorderType id="21450">
                               <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
@@ -7277,10 +7385,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="12437">
-                            <OrphaCode>93604</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93604</ExpertLink>
-                            <Name lang="en">Antenatal Bartter syndrome</Name>
+                          <Disorder id="31439">
+                            <OrphaCode>620220</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620220</ExpertLink>
+                            <Name lang="en">Bartter syndrome type 2</Name>
                             <DisorderType id="21450">
                               <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
@@ -7292,7 +7400,7 @@
                           <Disorder id="12438">
                             <OrphaCode>93605</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93605</ExpertLink>
-                            <Name lang="en">Classic Bartter syndrome</Name>
+                            <Name lang="en">Bartter syndrome type 3</Name>
                             <DisorderType id="21450">
                               <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
@@ -7304,7 +7412,7 @@
                           <Disorder id="11913">
                             <OrphaCode>89938</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89938</ExpertLink>
-                            <Name lang="en">Infantile Bartter syndrome with sensorineural deafness</Name>
+                            <Name lang="en">Bartter syndrome type 4</Name>
                             <DisorderType id="21450">
                               <Name lang="en">Clinical subtype</Name>
                             </DisorderType>
@@ -7447,135 +7555,15 @@
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="10344">
-                        <OrphaCode>34526</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34526</ExpertLink>
-                        <Name lang="en">Genetic primary hypomagnesemia</Name>
-                        <DisorderType id="36561">
-                          <Name lang="en">Category</Name>
+                      <Disorder id="10346">
+                        <OrphaCode>34528</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34528</ExpertLink>
+                        <Name lang="en">Autosomal dominant primary hypomagnesemia with hypocalciuria</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="4">
-                        <ClassificationNode>
-                          <Disorder id="28399">
-                            <OrphaCode>564178</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=564178</ExpertLink>
-                            <Name lang="en">Primary hypomagnesemia with refractory seizures and intellectual disability</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="21223">
-                            <OrphaCode>306519</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306519</ExpertLink>
-                            <Name lang="en">Genetic primary hypomagnesemia with hypocalciuria</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="1">
-                            <ClassificationNode>
-                              <Disorder id="10346">
-                                <OrphaCode>34528</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34528</ExpertLink>
-                                <Name lang="en">Autosomal dominant primary hypomagnesemia with hypocalciuria</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="21224">
-                            <OrphaCode>306522</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306522</ExpertLink>
-                            <Name lang="en">Genetic primary hypomagnesemia with normocalciuria</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="3">
-                            <ClassificationNode>
-                              <Disorder id="9282">
-                                <OrphaCode>30924</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=30924</ExpertLink>
-                                <Name lang="en">Primary hypomagnesemia with secondary hypocalcemia</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="10345">
-                                <OrphaCode>34527</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34527</ExpertLink>
-                                <Name lang="en">Familial primary hypomagnesemia with normocalciuria and normocalcemia</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="18401">
-                                <OrphaCode>199326</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=199326</ExpertLink>
-                                <Name lang="en">Isolated autosomal dominant hypomagnesemia, Glaudemans type</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="21222">
-                            <OrphaCode>306516</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306516</ExpertLink>
-                            <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis</Name>
-                            <DisorderType id="21436">
-                              <Name lang="en">Clinical group</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="2078">
-                                <OrphaCode>2196</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
-                                <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="9284">
-                                <OrphaCode>31043</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31043</ExpertLink>
-                                <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis without severe ocular involvement</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -8238,6 +8226,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="9282">
+                        <OrphaCode>30924</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=30924</ExpertLink>
+                        <Name lang="en">Primary hypomagnesemia with secondary hypocalcemia</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="28092">
                         <OrphaCode>544628</OrphaCode>
@@ -9570,6 +9570,66 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="18401">
+                        <OrphaCode>199326</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=199326</ExpertLink>
+                        <Name lang="en">Isolated autosomal dominant hypomagnesemia, Glaudemans type</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="28399">
+                        <OrphaCode>564178</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=564178</ExpertLink>
+                        <Name lang="en">Primary hypomagnesemia-refractory seizures-intellectual disability syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31440">
+                        <OrphaCode>620363</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620363</ExpertLink>
+                        <Name lang="en">Primary hypomagnesemia-generalized seizures-intellectual disability-obesity syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31441">
+                        <OrphaCode>620368</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620368</ExpertLink>
+                        <Name lang="en">EGF-related primary hypomagnesemia with intellectual disability</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31442">
+                        <OrphaCode>620371</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620371</ExpertLink>
+                        <Name lang="en">Gitelman-like kidney tubulopathy due to mitochondrial DNA mutation</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -11377,7 +11437,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="44">
+                          <ClassificationNodeChildList count="43">
                             <ClassificationNode>
                               <Disorder id="1814">
                                 <OrphaCode>1834</OrphaCode>
@@ -11486,18 +11546,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="12562">
-                                <OrphaCode>94095</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=94095</ExpertLink>
-                                <Name lang="en">Spondylocostal dysostosis-anal atresia-genitourinary malformation syndrome</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="12809">
                                 <OrphaCode>96185</OrphaCode>
@@ -17775,7 +17823,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="12">
+                      <ClassificationNodeChildList count="11">
                         <ClassificationNode>
                           <Disorder id="22">
                             <OrphaCode>487</OrphaCode>
@@ -18232,18 +18280,6 @@
                             </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="11105">
-                            <OrphaCode>77292</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
-                            <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="94">
                             <OrphaCode>324</OrphaCode>
@@ -18329,15 +18365,51 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="11106">
-                            <OrphaCode>77293</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
-                            <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
+                          <Disorder id="31389">
+                            <OrphaCode>618899</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618899</ExpertLink>
+                            <Name lang="en">Acid sphingomyelinase deficiency</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="3">
+                            <ClassificationNode>
+                              <Disorder id="11105">
+                                <OrphaCode>77292</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
+                                <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="11106">
+                                <OrphaCode>77293</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
+                                <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31388">
+                                <OrphaCode>618891</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
+                                <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -20746,18 +20818,6 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="11106">
-                                        <OrphaCode>77293</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
-                                        <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="11103">
                                         <OrphaCode>77260</OrphaCode>
@@ -20830,6 +20890,54 @@
                                         </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31389">
+                                        <OrphaCode>618899</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618899</ExpertLink>
+                                        <Name lang="en">Acid sphingomyelinase deficiency</Name>
+                                        <DisorderType id="21436">
+                                          <Name lang="en">Clinical group</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="3">
+                                        <ClassificationNode>
+                                          <Disorder id="11105">
+                                            <OrphaCode>77292</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
+                                            <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="11106">
+                                            <OrphaCode>77293</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
+                                            <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="31388">
+                                            <OrphaCode>618891</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
+                                            <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
