commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml
index a0c9b06..26ad111 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_233.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:31:55" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:19:55" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <ClassificationList count="1">
     <Classification id="233">
       <OrphaNumber>506203</OrphaNumber>
@@ -200,7 +193,7 @@
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -440,7 +433,7 @@
                           <Disorder id="678">
                             <OrphaCode>62</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=62</ExpertLink>
-                            <Name lang="en">Alpha-sarcoglycan-related  limb-girdle muscular dystrophy R3</Name>
+                            <Name lang="en">Alpha-sarcoglycan-related limb-girdle muscular dystrophy R3</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -500,7 +493,7 @@
                           <Disorder id="8715">
                             <OrphaCode>119</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=119</ExpertLink>
-                            <Name lang="en">Beta-sarcoglycan-related  limb-girdle muscular dystrophy R4</Name>
+                            <Name lang="en">Beta-sarcoglycan-related limb-girdle muscular dystrophy R4</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -512,7 +505,7 @@
                           <Disorder id="8716">
                             <OrphaCode>353</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353</ExpertLink>
-                            <Name lang="en">Gamma-sarcoglycan-related  limb-girdle muscular dystrophy R5</Name>
+                            <Name lang="en">Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -536,7 +529,7 @@
                           <Disorder id="8717">
                             <OrphaCode>219</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=219</ExpertLink>
-                            <Name lang="en">Delta-sarcoglycan-related  limb-girdle muscular dystrophy R6</Name>
+                            <Name lang="en">Delta-sarcoglycan-related limb-girdle muscular dystrophy R6</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -584,7 +577,7 @@
                           <Disorder id="18521">
                             <OrphaCode>206554</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-                            <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+                            <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
