commit cb98a4638b303dbc159c97a636d6d4f62ca39678
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 6 12:14:41 2022 +0100

    DEC 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml
index 0b8bc91..969d373 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:17:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:36:59" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -9001,7 +9001,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="31">
+                          <ClassificationNodeChildList count="33">
                             <ClassificationNode>
                               <Disorder id="13789">
                                 <OrphaCode>98772</OrphaCode>
@@ -9410,6 +9410,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31590">
+                                <OrphaCode>631103</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631103</ExpertLink>
+                                <Name lang="en">Spinocerebellar ataxia type 48</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31591">
+                                <OrphaCode>631106</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631106</ExpertLink>
+                                <Name lang="en">Spinocerebellar ataxia type 49</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -9421,7 +9445,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="10">
+                          <ClassificationNodeChildList count="11">
                             <ClassificationNode>
                               <Disorder id="29432">
                                 <OrphaCode>589527</OrphaCode>
@@ -9434,6 +9458,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31588">
+                                <OrphaCode>631095</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631095</ExpertLink>
+                                <Name lang="en">Spinocerebellar ataxia type 44</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="13775">
                                 <OrphaCode>98758</OrphaCode>
@@ -16045,7 +16081,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="7">
+                          <ClassificationNodeChildList count="8">
                             <ClassificationNode>
                               <Disorder id="14704">
                                 <OrphaCode>100993</OrphaCode>
@@ -16130,6 +16166,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31579">
+                                <OrphaCode>631068</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631068</ExpertLink>
+                                <Name lang="en">Autosomal dominant spastic paraplegia type 80</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -16141,7 +16189,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="5">
+                          <ClassificationNodeChildList count="6">
                             <ClassificationNode>
                               <Disorder id="14715">
                                 <OrphaCode>101004</OrphaCode>
@@ -16202,6 +16250,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31581">
+                                <OrphaCode>631076</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631076</ExpertLink>
+                                <Name lang="en">Autosomal recessive spastic paraplegia type 83</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -16429,7 +16489,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="39">
+                          <ClassificationNodeChildList count="43">
                             <ClassificationNode>
                               <Disorder id="24010">
                                 <OrphaCode>464282</OrphaCode>
@@ -16934,6 +16994,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31580">
+                                <OrphaCode>631073</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631073</ExpertLink>
+                                <Name lang="en">Autosomal recessive spastic paraplegia type 82</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31582">
+                                <OrphaCode>631079</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631079</ExpertLink>
+                                <Name lang="en">Autosomal recessive spastic paraplegia type 84</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="27024">
                                 <OrphaCode>513436</OrphaCode>
@@ -16946,6 +17030,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31583">
+                                <OrphaCode>631082</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631082</ExpertLink>
+                                <Name lang="en">Autosomal recessive spastic paraplegia type 85</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31584">
+                                <OrphaCode>631085</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631085</ExpertLink>
+                                <Name lang="en">Autosomal recessive spastic paraplegia type 86</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -17125,7 +17233,19 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="11">
+                          <ClassificationNodeChildList count="12">
+                            <ClassificationNode>
+                              <Disorder id="31585">
+                                <OrphaCode>631088</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631088</ExpertLink>
+                                <Name lang="en">Autosomal recessive spastic paraplegia type 87</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="2249">
                                 <OrphaCode>2429</OrphaCode>
@@ -18717,7 +18837,7 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="31">
+                                  <ClassificationNodeChildList count="33">
                                     <ClassificationNode>
                                       <Disorder id="13789">
                                         <OrphaCode>98772</OrphaCode>
@@ -19126,6 +19246,30 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31590">
+                                        <OrphaCode>631103</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631103</ExpertLink>
+                                        <Name lang="en">Spinocerebellar ataxia type 48</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31591">
+                                        <OrphaCode>631106</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631106</ExpertLink>
+                                        <Name lang="en">Spinocerebellar ataxia type 49</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -19137,7 +19281,7 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="10">
+                                  <ClassificationNodeChildList count="11">
                                     <ClassificationNode>
                                       <Disorder id="29432">
                                         <OrphaCode>589527</OrphaCode>
@@ -19150,6 +19294,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31588">
+                                        <OrphaCode>631095</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631095</ExpertLink>
+                                        <Name lang="en">Spinocerebellar ataxia type 44</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="13775">
                                         <OrphaCode>98758</OrphaCode>
@@ -20563,7 +20719,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="15">
+              <ClassificationNodeChildList count="13">
                 <ClassificationNode>
                   <Disorder id="99">
                     <OrphaCode>892</OrphaCode>
@@ -20636,54 +20792,6 @@
                     </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="183">
-                    <OrphaCode>637</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637</ExpertLink>
-                    <Name lang="en">Neurofibromatosis type 2</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="185">
-                    <OrphaCode>636</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=636</ExpertLink>
-                    <Name lang="en">Neurofibromatosis type 1</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="2">
-                    <ClassificationNode>
-                      <Disorder id="12945">
-                        <OrphaCode>97685</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97685</ExpertLink>
-                        <Name lang="en">17q11 microdeletion syndrome</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="22341">
-                        <OrphaCode>363700</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363700</ExpertLink>
-                        <Name lang="en">Neurofibromatosis type 1 due to NF1 mutation or intragenic deletion</Name>
-                        <DisorderType id="21443">
-                          <Name lang="en">Etiological subtype</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="196">
                     <OrphaCode>524</OrphaCode>
@@ -20732,18 +20840,6 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
-                <ClassificationNode>
-                  <Disorder id="12482">
-                    <OrphaCode>93921</OrphaCode>
-                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93921</ExpertLink>
-                    <Name lang="en">Schwannomatosis</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
-                    </DisorderType>
-                  </Disorder>
-                  <ClassificationNodeChildList count="0">
-                  </ClassificationNodeChildList>
-                </ClassificationNode>
                 <ClassificationNode>
                   <Disorder id="14391">
                     <OrphaCode>99818</OrphaCode>
@@ -20804,6 +20900,114 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31695">
+                    <OrphaCode>634518</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=634518</ExpertLink>
+                    <Name lang="en">Neurofibromatosis/schwannomatosis</Name>
+                    <DisorderType id="36561">
+                      <Name lang="en">Category</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="6">
+                    <ClassificationNode>
+                      <Disorder id="12482">
+                        <OrphaCode>93921</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93921</ExpertLink>
+                        <Name lang="en">Full schwannomatosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31691">
+                        <OrphaCode>634461</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=634461</ExpertLink>
+                        <Name lang="en">Mosaic neurofibromatosis type 1</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31692">
+                        <OrphaCode>634475</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=634475</ExpertLink>
+                        <Name lang="en">Mosaic NF2-related schwannomatosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31693">
+                        <OrphaCode>634492</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=634492</ExpertLink>
+                        <Name lang="en">Mosaic schwannomatosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="185">
+                        <OrphaCode>636</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=636</ExpertLink>
+                        <Name lang="en">Neurofibromatosis type 1</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="12945">
+                            <OrphaCode>97685</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97685</ExpertLink>
+                            <Name lang="en">17q11 microdeletion syndrome</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="22341">
+                            <OrphaCode>363700</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363700</ExpertLink>
+                            <Name lang="en">Neurofibromatosis type 1 due to NF1 mutation or intragenic deletion</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="183">
+                        <OrphaCode>637</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=637</ExpertLink>
+                        <Name lang="en">Full NF2-related schwannomatosis</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
@@ -24448,7 +24652,7 @@
                                   <Disorder id="10740">
                                     <OrphaCode>53698</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=53698</ExpertLink>
-                                    <Name lang="en">Hyaline body myopathy</Name>
+                                    <Name lang="en">Myosin storage myopathy</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -25212,7 +25416,7 @@
                                           <Disorder id="28442">
                                             <OrphaCode>565612</OrphaCode>
                                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=565612</ExpertLink>
-                                            <Name lang="en">Triglyceride deposit cardiomyovasculopathy</Name>
+                                            <Name lang="en">Primary triglyceride deposit cardiomyovasculopathy</Name>
                                             <DisorderType id="21394">
                                               <Name lang="en">Disease</Name>
                                             </DisorderType>
@@ -33687,7 +33891,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="352">
+                      <ClassificationNodeChildList count="351">
                         <ClassificationNode>
                           <Disorder id="25681">
                             <OrphaCode>494526</OrphaCode>
@@ -33784,18 +33988,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="17944">
-                            <OrphaCode>171860</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=171860</ExpertLink>
-                            <Name lang="en">Intellectual disability-cataracts-kyphosis syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="18406">
                             <OrphaCode>199343</OrphaCode>
@@ -38511,7 +38703,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="652">
+                      <ClassificationNodeChildList count="656">
                         <ClassificationNode>
                           <Disorder id="11695">
                             <OrphaCode>85325</OrphaCode>
@@ -40597,7 +40789,19 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="3">
+                          <ClassificationNodeChildList count="4">
+                            <ClassificationNode>
+                              <Disorder id="31664">
+                                <OrphaCode>633028</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633028</ExpertLink>
+                                <Name lang="en">CPE-related Prader-Willi-like syndrome</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="17936">
                                 <OrphaCode>171829</OrphaCode>
@@ -42272,7 +42476,7 @@
                           <Disorder id="28140">
                             <OrphaCode>557003</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=557003</ExpertLink>
-                            <Name lang="en">Oculocerebrodental syndrome</Name>
+                            <Name lang="en">Oculoskeletodental syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -46516,6 +46720,78 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31661">
+                            <OrphaCode>633014</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633014</ExpertLink>
+                            <Name lang="en">SLC12A2-related developmental delay-intellectual disability-sensorineural deafness syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="31662">
+                                <OrphaCode>633021</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633021</ExpertLink>
+                                <Name lang="en">SLC12A2-related autosomal recessive neonatal-developmental delay-intellectual disability-feeding difficulty-sensorineural deafness syndrome</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31663">
+                                <OrphaCode>633024</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633024</ExpertLink>
+                                <Name lang="en">SLC12A2-related autosomal dominant infantile-developmental delay-intellectual disability-sensorineural deafness syndrome</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31660">
+                            <OrphaCode>633004</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633004</ExpertLink>
+                            <Name lang="en">KDM3B-related intellectual disability-facial dysmorphism-short stature syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31665">
+                            <OrphaCode>633035</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633035</ExpertLink>
+                            <Name lang="en">Intellectual disability-early-onset cataract-microcephaly syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31658">
+                            <OrphaCode>632603</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=632603</ExpertLink>
+                            <Name lang="en">Horseshoe kidney-Nievergelt/Savarirayan mesomelic dysplasia-seizures-hypertrichosis-intellectual disability-pulmonary involvement syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1360">
                             <OrphaCode>1064</OrphaCode>
@@ -50035,7 +50311,7 @@
                                           <Name lang="en">Clinical group</Name>
                                         </DisorderType>
                                       </Disorder>
-                                      <ClassificationNodeChildList count="31">
+                                      <ClassificationNodeChildList count="33">
                                         <ClassificationNode>
                                           <Disorder id="13789">
                                             <OrphaCode>98772</OrphaCode>
@@ -50444,6 +50720,30 @@
                                           <ClassificationNodeChildList count="0">
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="31590">
+                                            <OrphaCode>631103</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631103</ExpertLink>
+                                            <Name lang="en">Spinocerebellar ataxia type 48</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="31591">
+                                            <OrphaCode>631106</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631106</ExpertLink>
+                                            <Name lang="en">Spinocerebellar ataxia type 49</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
                                     <ClassificationNode>
@@ -50455,7 +50755,7 @@
                                           <Name lang="en">Clinical group</Name>
                                         </DisorderType>
                                       </Disorder>
-                                      <ClassificationNodeChildList count="10">
+                                      <ClassificationNodeChildList count="11">
                                         <ClassificationNode>
                                           <Disorder id="29432">
                                             <OrphaCode>589527</OrphaCode>
@@ -50468,6 +50768,18 @@
                                           <ClassificationNodeChildList count="0">
                                           </ClassificationNodeChildList>
                                         </ClassificationNode>
+                                        <ClassificationNode>
+                                          <Disorder id="31588">
+                                            <OrphaCode>631095</OrphaCode>
+                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631095</ExpertLink>
+                                            <Name lang="en">Spinocerebellar ataxia type 44</Name>
+                                            <DisorderType id="21394">
+                                              <Name lang="en">Disease</Name>
+                                            </DisorderType>
+                                          </Disorder>
+                                          <ClassificationNodeChildList count="0">
+                                          </ClassificationNodeChildList>
+                                        </ClassificationNode>
                                         <ClassificationNode>
                                           <Disorder id="13775">
                                             <OrphaCode>98758</OrphaCode>
@@ -53021,7 +53333,19 @@
                       <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="3">
+                  <ClassificationNodeChildList count="4">
+                    <ClassificationNode>
+                      <Disorder id="31664">
+                        <OrphaCode>633028</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633028</ExpertLink>
+                        <Name lang="en">CPE-related Prader-Willi-like syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="17936">
                         <OrphaCode>171829</OrphaCode>
@@ -53687,7 +54011,7 @@
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="2">
+                              <ClassificationNodeChildList count="3">
                                 <ClassificationNode>
                                   <Disorder id="28815">
                                     <OrphaCode>573253</OrphaCode>
@@ -53712,6 +54036,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31668">
+                                    <OrphaCode>633076</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633076</ExpertLink>
+                                    <Name lang="en">Split cord malformation, composite type</Name>
+                                    <DisorderType id="21450">
+                                      <Name lang="en">Clinical subtype</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -57296,7 +57632,7 @@
                           <Disorder id="18946">
                             <OrphaCode>221126</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=221126</ExpertLink>
-                            <Name lang="en">Fowler vasculopaty</Name>
+                            <Name lang="en">Fowler vasculopathy</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -59468,7 +59804,7 @@
                           <Disorder id="18855">
                             <OrphaCode>217399</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217399</ExpertLink>
-                            <Name lang="en">Congenital insensitivity to pain-hyperhidrosis-absence of C-fiber innervation</Name>
+                            <Name lang="en">Congenital insensitivity to pain-hyperhidrosis-absence of cutaneous sensory innervation</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -61679,7 +62015,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="10">
+                              <ClassificationNodeChildList count="11">
                                 <ClassificationNode>
                                   <Disorder id="192">
                                     <OrphaCode>640</OrphaCode>
@@ -61824,6 +62160,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31592">
+                                    <OrphaCode>631248</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=631248</ExpertLink>
+                                    <Name lang="en">Mitchell Syndrome</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="25100">
                                     <OrphaCode>476394</OrphaCode>
