commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml
index 3570067..7d26f08 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:38:43" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2021-12-01 04:42:06" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <ClassificationList count="1">
     <Classification id="147">
       <OrphaNumber>156268</OrphaNumber>
@@ -5928,7 +5921,7 @@
                   <Disorder id="21494">
                     <OrphaCode>314555</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314555</ExpertLink>
-                    <Name lang="en">Craniofacial dysplasia-osteopenia syndrome</Name>
+                    <Name lang="en">Facial dysmorphism-severe myopia-osteopenia-intellectual disability-dental anomalies syndrome</Name>
                     <DisorderType id="21401">
                       <Name lang="en">Malformation syndrome</Name>
                     </DisorderType>
@@ -9241,7 +9234,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="11">
+                          <ClassificationNodeChildList count="13">
                             <ClassificationNode>
                               <Disorder id="2334">
                                 <OrphaCode>2524</OrphaCode>
@@ -9374,6 +9367,30 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="30702">
+                                <OrphaCode>613267</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                                <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="30703">
+                                <OrphaCode>613274</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                                <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -9515,7 +9532,19 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="1">
+                              <ClassificationNodeChildList count="2">
+                                <ClassificationNode>
+                                  <Disorder id="1604">
+                                    <OrphaCode>1397</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1397</ExpertLink>
+                                    <Name lang="en">Hydrocephaly-cerebellar agenesis syndrome</Name>
+                                    <DisorderType id="21401">
+                                      <Name lang="en">Malformation syndrome</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="1605">
                                     <OrphaCode>1398</OrphaCode>
@@ -11199,7 +11228,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="37">
+                      <ClassificationNodeChildList count="38">
                         <ClassificationNode>
                           <Disorder id="23772">
                             <OrphaCode>459070</OrphaCode>
@@ -11776,6 +11805,42 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31320">
+                            <OrphaCode>615954</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615954</ExpertLink>
+                            <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome</Name>
+                            <DisorderType id="21422">
+                              <Name lang="en">Clinical syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="31323">
+                                <OrphaCode>615983</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615983</ExpertLink>
+                                <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome due to a point mutation</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31324">
+                                <OrphaCode>615986</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615986</ExpertLink>
+                                <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome due to biallelic deletions in the ATAD3 gene cluster</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -12752,7 +12817,7 @@
                           <Disorder id="30683">
                             <OrphaCode>610569</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                            <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                            <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -13441,8 +13506,8 @@
                     <OrphaCode>2346</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                     <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                    <DisorderType id="21394">
-                      <Name lang="en">Disease</Name>
+                    <DisorderType id="21436">
+                      <Name lang="en">Clinical group</Name>
                     </DisorderType>
                   </Disorder>
                   <ClassificationNodeChildList count="2">
@@ -13451,8 +13516,8 @@
                         <OrphaCode>90307</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                         <Name lang="en">Parkes Weber syndrome</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -13463,8 +13528,8 @@
                         <OrphaCode>90308</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                         <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -32207,8 +32272,8 @@
                         <OrphaCode>2346</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                         <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
@@ -32217,8 +32282,8 @@
                             <OrphaCode>90307</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                             <Name lang="en">Parkes Weber syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -32229,8 +32294,8 @@
                             <OrphaCode>90308</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                             <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -45561,7 +45626,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="4">
+                                  <ClassificationNodeChildList count="5">
                                     <ClassificationNode>
                                       <Disorder id="1738">
                                         <OrphaCode>1606</OrphaCode>
@@ -45598,6 +45663,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31324">
+                                        <OrphaCode>615986</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615986</ExpertLink>
+                                        <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome due to biallelic deletions in the ATAD3 gene cluster</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="23675">
                                         <OrphaCode>456298</OrphaCode>
@@ -48809,7 +48886,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="20">
+                  <ClassificationNodeChildList count="21">
                     <ClassificationNode>
                       <Disorder id="2370">
                         <OrphaCode>2570</OrphaCode>
@@ -48834,6 +48911,42 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="31320">
+                        <OrphaCode>615954</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615954</ExpertLink>
+                        <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome</Name>
+                        <DisorderType id="21422">
+                          <Name lang="en">Clinical syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="31323">
+                            <OrphaCode>615983</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615983</ExpertLink>
+                            <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome due to a point mutation</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31324">
+                            <OrphaCode>615986</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=615986</ExpertLink>
+                            <Name lang="en">Lethal pontocerebellar hypoplasia-hypotonia-respiratory insufficiency syndrome due to biallelic deletions in the ATAD3 gene cluster</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="27458">
                         <OrphaCode>527468</OrphaCode>
@@ -49085,7 +49198,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="652">
+                  <ClassificationNodeChildList count="653">
                     <ClassificationNode>
                       <Disorder id="24010">
                         <OrphaCode>464282</OrphaCode>
@@ -50674,7 +50787,7 @@
                       <Disorder id="11669">
                         <OrphaCode>85279</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85279</ExpertLink>
-                        <Name lang="en">Syndromic X-linked intellectual disability due to JARID1C mutation</Name>
+                        <Name lang="en">KDM5C-related syndromic X-linked intellectual disability</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -50910,18 +51023,6 @@
                         </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="12110">
-                        <OrphaCode>91133</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=91133</ExpertLink>
-                        <Name lang="en">Osteopenia-myopia-hearing loss-intellectual disability-facial dysmorphism syndrome</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="12547">
                         <OrphaCode>94066</OrphaCode>
@@ -57562,7 +57663,7 @@
                       <Disorder id="30683">
                         <OrphaCode>610569</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                        <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                        <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -58122,6 +58223,30 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="30702">
+                        <OrphaCode>613267</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613267</ExpertLink>
+                        <Name lang="en">Pontocerebellar hypoplasia type 13</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="30703">
+                        <OrphaCode>613274</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=613274</ExpertLink>
+                        <Name lang="en">Pontocerebellar hypoplasia type 14</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="570">
                         <OrphaCode>2983</OrphaCode>
@@ -62502,7 +62627,7 @@
                               <Disorder id="30683">
                                 <OrphaCode>610569</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=610569</ExpertLink>
-                                <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+                                <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -66491,8 +66616,8 @@
                         <OrphaCode>2346</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2346</ExpertLink>
                         <Name lang="en">Angioosteohypertrophic syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="2">
@@ -66501,8 +66626,8 @@
                             <OrphaCode>90307</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90307</ExpertLink>
                             <Name lang="en">Parkes Weber syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -66513,8 +66638,8 @@
                             <OrphaCode>90308</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90308</ExpertLink>
                             <Name lang="en">Klippel-Trénaunay syndrome</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
