commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml
index a84d999..6b26724 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_185.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:37:39" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:50:00" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -75,7 +75,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="4">
                         <ClassificationNode>
                           <Disorder id="18085">
                             <OrphaCode>180065</OrphaCode>
@@ -424,6 +424,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31814">
+                            <OrphaCode>647794</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647794</ExpertLink>
+                            <Name lang="en">Isolated persistent urogenital sinus</Name>
+                            <DisorderType id="21415">
+                              <Name lang="en">Morphological anomaly</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -1394,7 +1406,7 @@
               <Disorder id="12085">
                 <OrphaCode>90771</OrphaCode>
                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90771</ExpertLink>
-                <Name lang="en">Disorder of sex development</Name>
+                <Name lang="en">Difference of sex development</Name>
                 <DisorderType id="36561">
                   <Name lang="en">Category</Name>
                 </DisorderType>
@@ -1404,7 +1416,7 @@
                   <Disorder id="569">
                     <OrphaCode>2982</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2982</ExpertLink>
-                    <Name lang="en">46,XX disorder of sex development</Name>
+                    <Name lang="en">46,XX difference of sex development</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -1414,7 +1426,7 @@
                       <Disorder id="13096">
                         <OrphaCode>98078</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98078</ExpertLink>
-                        <Name lang="en">46,XX disorder of sex development induced by androgens excess</Name>
+                        <Name lang="en">46,XX difference of sex development induced by androgens excess</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -1424,7 +1436,7 @@
                           <Disorder id="12087">
                             <OrphaCode>90776</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90776</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by fetal androgens excess</Name>
+                            <Name lang="en">46,XX difference of sex development induced by fetal androgens excess</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -1532,7 +1544,7 @@
                           <Disorder id="12119">
                             <OrphaCode>91144</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=91144</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by maternal-derived androgen</Name>
+                            <Name lang="en">46,XX difference of sex development induced by maternal-derived androgen</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -1542,7 +1554,7 @@
                               <Disorder id="21829">
                                 <OrphaCode>325093</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325093</ExpertLink>
-                                <Name lang="en">46,XX disorder of sex development induced by endogenous maternal-derived androgen</Name>
+                                <Name lang="en">46,XX difference of sex development induced by endogenous maternal-derived androgen</Name>
                                 <DisorderType id="21436">
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
@@ -1554,7 +1566,7 @@
                               <Disorder id="21830">
                                 <OrphaCode>325099</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325099</ExpertLink>
-                                <Name lang="en">46,XX disorder of sex development induced by exogenous maternal-derived androgen</Name>
+                                <Name lang="en">46,XX difference of sex development induced by exogenous maternal-derived androgen</Name>
                                 <DisorderType id="21436">
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
@@ -1568,7 +1580,7 @@
                           <Disorder id="21828">
                             <OrphaCode>325061</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325061</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development induced by fetoplacental androgens excess</Name>
+                            <Name lang="en">46,XX difference of sex development induced by fetoplacental androgens excess</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -1616,7 +1628,7 @@
                           <Disorder id="2037">
                             <OrphaCode>2138</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2138</ExpertLink>
-                            <Name lang="en">46,XX ovotesticular disorder of sex development</Name>
+                            <Name lang="en">46,XX ovotesticular difference of sex development</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -1628,7 +1640,7 @@
                           <Disorder id="5546">
                             <OrphaCode>393</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=393</ExpertLink>
-                            <Name lang="en">46,XX testicular disorder of sex development</Name>
+                            <Name lang="en">46,XX testicular difference of sex development</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -1654,7 +1666,7 @@
                       <Disorder id="21831">
                         <OrphaCode>325109</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325109</ExpertLink>
-                        <Name lang="en">Syndrome with 46,XX disorder of sex development</Name>
+                        <Name lang="en">Syndrome with 46,XX difference of sex development</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -1681,14 +1693,38 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="31739">
+                                <OrphaCode>642945</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                                <Name lang="en">Perrault syndrome type 1</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="31742">
+                                <OrphaCode>642976</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                                <Name lang="en">Perrault syndrome type 2</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="2679">
                             <OrphaCode>2973</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2973</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development-anorectal anomalies syndrome</Name>
+                            <Name lang="en">46,XX difference of sex development-anorectal anomalies syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -1700,7 +1736,7 @@
                           <Disorder id="2681">
                             <OrphaCode>2975</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2975</ExpertLink>
-                            <Name lang="en">46,XX disorder of sex development-skeletal anomalies syndrome</Name>
+                            <Name lang="en">46,XX difference of sex development-skeletal anomalies syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -1764,7 +1800,7 @@
                   <Disorder id="13103">
                     <OrphaCode>98085</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98085</ExpertLink>
-                    <Name lang="en">46,XY disorder of sex development</Name>
+                    <Name lang="en">46,XY difference of sex development</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
@@ -1786,7 +1822,7 @@
                       <Disorder id="13105">
                         <OrphaCode>98087</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98087</ExpertLink>
-                        <Name lang="en">Syndrome with 46,XY disorder of sex development</Name>
+                        <Name lang="en">Syndrome with 46,XY difference of sex development</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -1808,7 +1844,7 @@
                           <Disorder id="334">
                             <OrphaCode>1642</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1642</ExpertLink>
-                            <Name lang="en">Distal monosomy 9p</Name>
+                            <Name lang="en">Distal deletion 9p</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -1832,7 +1868,7 @@
                           <Disorder id="570">
                             <OrphaCode>2983</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2983</ExpertLink>
-                            <Name lang="en">Disorder of sex development-intellectual disability syndrome</Name>
+                            <Name lang="en">Difference of sex development-intellectual disability syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -1844,7 +1880,7 @@
                           <Disorder id="606">
                             <OrphaCode>1422</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1422</ExpertLink>
-                            <Name lang="en">Chondrodysplasia-disorder of sex development syndrome</Name>
+                            <Name lang="en">Chondrodysplasia-difference of sex development syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -1892,7 +1928,7 @@
                           <Disorder id="2151">
                             <OrphaCode>2282</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2282</ExpertLink>
-                            <Name lang="en">Dysmorphism-short stature-deafness-disorder of sex development syndrome</Name>
+                            <Name lang="en">Dysmorphism-short stature-deafness-difference of sex development syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -2084,7 +2120,7 @@
                           <Disorder id="21838">
                             <OrphaCode>325345</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325345</ExpertLink>
-                            <Name lang="en">46,XY ovotesticular disorder of sex development</Name>
+                            <Name lang="en">46,XY ovotesticular difference of sex development</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2098,7 +2134,7 @@
                       <Disorder id="21839">
                         <OrphaCode>325351</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325351</ExpertLink>
-                        <Name lang="en">46,XY disorder of sex development of endocrine origin</Name>
+                        <Name lang="en">46,XY difference of sex development of endocrine origin</Name>
                         <DisorderType id="36561">
                           <Name lang="en">Category</Name>
                         </DisorderType>
@@ -2156,7 +2192,7 @@
                           <Disorder id="21840">
                             <OrphaCode>325357</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325357</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development due to impaired androgen production</Name>
+                            <Name lang="en">46,XY difference of sex development due to impaired androgen production</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -2214,7 +2250,7 @@
                               <Disorder id="12089">
                                 <OrphaCode>90783</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90783</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to a testosterone synthesis defect</Name>
+                                <Name lang="en">46,XY difference of sex development due to a testosterone synthesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -2224,7 +2260,7 @@
                                   <Disorder id="12090">
                                     <OrphaCode>90786</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90786</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to adrenal and testicular steroidogenesis defect</Name>
+                                    <Name lang="en">46,XY difference of sex development due to adrenal and testicular steroidogenesis defect</Name>
                                     <DisorderType id="36561">
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
@@ -2306,7 +2342,7 @@
                                       <Disorder id="17771">
                                         <OrphaCode>168558</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168558</ExpertLink>
-                                        <Name lang="en">46,XY disorder of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
+                                        <Name lang="en">46,XY difference of sex development-adrenal insufficiency due to CYP11A1 deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -2332,7 +2368,7 @@
                                   <Disorder id="12091">
                                     <OrphaCode>90787</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90787</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to testicular steroidogenesis defect</Name>
+                                    <Name lang="en">46,XY difference of sex development due to testicular steroidogenesis defect</Name>
                                     <DisorderType id="36561">
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
@@ -2342,7 +2378,7 @@
                                       <Disorder id="203">
                                         <OrphaCode>752</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=752</ExpertLink>
-                                        <Name lang="en">46,XY disorder of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
+                                        <Name lang="en">46,XY difference of sex development due to 17-beta-hydroxysteroid dehydrogenase 3 deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -2354,7 +2390,7 @@
                                       <Disorder id="12099">
                                         <OrphaCode>90796</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90796</ExpertLink>
-                                        <Name lang="en">46,XY disorder of sex development due to isolated 17,20-lyase deficiency</Name>
+                                        <Name lang="en">46,XY difference of sex development due to isolated 17,20-lyase deficiency</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -2368,7 +2404,7 @@
                                   <Disorder id="21842">
                                     <OrphaCode>325511</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325511</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to a cholesterol synthesis defect</Name>
+                                    <Name lang="en">46,XY difference of sex development due to a cholesterol synthesis defect</Name>
                                     <DisorderType id="36561">
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
@@ -2394,7 +2430,7 @@
                               <Disorder id="13104">
                                 <OrphaCode>98086</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98086</ExpertLink>
-                                <Name lang="en">46,XY disorder of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
+                                <Name lang="en">46,XY difference of sex development due to a defect in testosterone metabolism by peripheral tissue</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -2404,7 +2440,7 @@
                                   <Disorder id="324">
                                     <OrphaCode>753</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=753</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to 5-alpha-reductase 2 deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development due to 5-alpha-reductase 2 deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -2418,7 +2454,7 @@
                               <Disorder id="23441">
                                 <OrphaCode>443090</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443090</ExpertLink>
-                                <Name lang="en">46,XY disorder of sexual development due to dihydrotestosterone backdoor pathway biosynthesis defect</Name>
+                                <Name lang="en">46,XY difference of sexual development due to dihydrotestosterone backdoor pathway biosynthesis defect</Name>
                                 <DisorderType id="36561">
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
@@ -2428,7 +2464,7 @@
                                   <Disorder id="23440">
                                     <OrphaCode>443087</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=443087</ExpertLink>
-                                    <Name lang="en">46,XY disorder of sex development due to testicular 17,20-desmolase deficiency</Name>
+                                    <Name lang="en">46,XY difference of sex development due to testicular 17,20-desmolase deficiency</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -2444,7 +2480,7 @@
                           <Disorder id="21845">
                             <OrphaCode>325537</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325537</ExpertLink>
-                            <Name lang="en">46,XY disorder of sex development induced by maternal exposure to endocrine disruptors</Name>
+                            <Name lang="en">46,XY difference of sex development induced by maternal exposure to endocrine disruptors</Name>
                             <DisorderType id="36561">
                               <Name lang="en">Category</Name>
                             </DisorderType>
@@ -2460,7 +2496,7 @@
                   <Disorder id="21846">
                     <OrphaCode>325546</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=325546</ExpertLink>
-                    <Name lang="en">Sex chromosome disorder of sex development</Name>
+                    <Name lang="en">Sex chromosome difference of sex development</Name>
                     <DisorderType id="36561">
                       <Name lang="en">Category</Name>
                     </DisorderType>
