commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml
index 4bd1858..fdb83c2 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:34:18" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:34:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -3929,7 +3929,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="13">
+                  <ClassificationNodeChildList count="14">
                     <ClassificationNode>
                       <Disorder id="25142">
                         <OrphaCode>477742</OrphaCode>
@@ -4482,6 +4482,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29725">
+                        <OrphaCode>595133</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595133</ExpertLink>
+                        <Name lang="en">Perivascular epithelioid cell neoplasm</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
