commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_212.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_212.xml
index 7305586..1c3e8f8 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_212.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_212.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:25:18" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:09:48" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -277,57 +277,33 @@
                     <OrphaCode>31740</OrphaCode>
                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31740</ExpertLink>
                     <Name lang="en">Hypersensitivity pneumonitis</Name>
-                    <DisorderType id="21436">
-                      <Name lang="en">Clinical group</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
                     </DisorderType>
                   </Disorder>
                   <ClassificationNodeChildList count="2">
                     <ClassificationNode>
-                      <Disorder id="14480">
-                        <OrphaCode>99907</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99907</ExpertLink>
-                        <Name lang="en">House allergic alveolitis</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                      <Disorder id="32172">
+                        <OrphaCode>686462</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686462</ExpertLink>
+                        <Name lang="en">Non-fibrotic hypersensitivity pneumonitis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="14482">
-                        <OrphaCode>99909</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99909</ExpertLink>
-                        <Name lang="en">Occupational allergic alveolitis</Name>
-                        <DisorderType id="21436">
-                          <Name lang="en">Clinical group</Name>
+                      <Disorder id="32173">
+                        <OrphaCode>686465</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=686465</ExpertLink>
+                        <Name lang="en">Fibrotic hypersensitivity pneumonitis</Name>
+                        <DisorderType id="21450">
+                          <Name lang="en">Clinical subtype</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
-                        <ClassificationNode>
-                          <Disorder id="14479">
-                            <OrphaCode>99906</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99906</ExpertLink>
-                            <Name lang="en">Farmer's lung disease</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="14481">
-                            <OrphaCode>99908</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99908</ExpertLink>
-                            <Name lang="en">Pigeon-breeder lung disease</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
