commit 481d14fea17f62fcae7251f3673242ec271b28fa
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 4 20:10:24 2023 +0100

    DEC 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml
index de2a763..83969a4 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_202.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-06-22 14:01:30" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-12-04 08:06:11" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -378,7 +378,7 @@
                               <Disorder id="14485">
                                 <OrphaCode>99912</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99912</ExpertLink>
-                                <Name lang="en">Malignant dysgerminomatous germ cell tumor of the ovary</Name>
+                                <Name lang="en">Ovarian dysgerminoma</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -2685,7 +2685,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="11">
+                                  <ClassificationNodeChildList count="13">
                                     <ClassificationNode>
                                       <Disorder id="10952">
                                         <OrphaCode>70568</OrphaCode>
@@ -3046,6 +3046,18 @@
                                       <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31900">
+                                        <OrphaCode>652658</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652658</ExpertLink>
+                                        <Name lang="en">Monomorphic epitheliotropic intestinal T-cell lymphoma</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="26906">
                                         <OrphaCode>512034</OrphaCode>
@@ -3094,6 +3106,18 @@
                                         </ClassificationNode>
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31899">
+                                        <OrphaCode>652650</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652650</ExpertLink>
+                                        <Name lang="en">Nodal T-follicular helper cell lymphoma, follicular type</Name>
+                                        <DisorderType id="21394">
+                                          <Name lang="en">Disease</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                               </ClassificationNodeChildList>
@@ -4517,7 +4541,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="37">
+                  <ClassificationNodeChildList count="36">
                     <ClassificationNode>
                       <Disorder id="185">
                         <OrphaCode>636</OrphaCode>
@@ -4566,18 +4590,6 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="543">
-                        <OrphaCode>587</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=587</ExpertLink>
-                        <Name lang="en">Muir-Torre syndrome</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="660">
                         <OrphaCode>805</OrphaCode>
@@ -6409,7 +6421,19 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="3">
+                          <ClassificationNodeChildList count="4">
+                            <ClassificationNode>
+                              <Disorder id="31900">
+                                <OrphaCode>652658</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=652658</ExpertLink>
+                                <Name lang="en">Monomorphic epitheliotropic intestinal T-cell lymphoma</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="11770">
                                 <OrphaCode>86880</OrphaCode>
@@ -8183,7 +8207,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="5">
+                              <ClassificationNodeChildList count="4">
                                 <ClassificationNode>
                                   <Disorder id="548">
                                     <OrphaCode>635</OrphaCode>
@@ -8221,22 +8245,10 @@
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
-                                  <Disorder id="19686">
-                                    <OrphaCode>251880</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251880</ExpertLink>
-                                    <Name lang="en">Ependymoblastoma</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19687">
-                                    <OrphaCode>251883</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251883</ExpertLink>
-                                    <Name lang="en">Medulloepithelioma of the central nervous system</Name>
+                                  <Disorder id="31929">
+                                    <OrphaCode>656417</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=656417</ExpertLink>
+                                    <Name lang="en">Embryonal tumor with multilayered rosettes</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -8346,7 +8358,7 @@
                               <Disorder id="19696">
                                 <OrphaCode>251919</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251919</ExpertLink>
-                                <Name lang="en">Pineal parenchymal tumor of intermediate differenciation</Name>
+                                <Name lang="en">Pineal parenchymal tumor of intermediate differentiation</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -9842,7 +9854,7 @@
                                       <Disorder id="14485">
                                         <OrphaCode>99912</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99912</ExpertLink>
-                                        <Name lang="en">Malignant dysgerminomatous germ cell tumor of the ovary</Name>
+                                        <Name lang="en">Ovarian dysgerminoma</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -9902,7 +9914,7 @@
                                       <Disorder id="14488">
                                         <OrphaCode>99915</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99915</ExpertLink>
-                                        <Name lang="en">Maligant granulosa cell tumor of the ovary</Name>
+                                        <Name lang="en">Malignant granulosa cell tumor of the ovary</Name>
                                         <DisorderType id="21394">
                                           <Name lang="en">Disease</Name>
                                         </DisorderType>
@@ -10265,7 +10277,19 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="11">
+                  <ClassificationNodeChildList count="12">
+                    <ClassificationNode>
+                      <Disorder id="31939">
+                        <OrphaCode>658590</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=658590</ExpertLink>
+                        <Name lang="en">Eyelid sebaceous carcinoma</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="31370">
                         <OrphaCode>617910</OrphaCode>
@@ -15447,19 +15471,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
-                        <ClassificationNode>
-                          <Disorder id="543">
-                            <OrphaCode>587</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=587</ExpertLink>
-                            <Name lang="en">Muir-Torre syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
                           <Disorder id="3245">
                             <OrphaCode>144</OrphaCode>
@@ -15699,19 +15711,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
-                        <ClassificationNode>
-                          <Disorder id="543">
-                            <OrphaCode>587</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=587</ExpertLink>
-                            <Name lang="en">Muir-Torre syndrome</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
+                      <ClassificationNodeChildList count="2">
                         <ClassificationNode>
                           <Disorder id="3245">
                             <OrphaCode>144</OrphaCode>
