commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml
index 68cfb48..5791a06 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_189.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:43:24" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:47:06" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2853,7 +2853,7 @@
                                       <Name lang="en">Category</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="10">
+                                  <ClassificationNodeChildList count="8">
                                     <ClassificationNode>
                                       <Disorder id="234">
                                         <OrphaCode>710</OrphaCode>
@@ -2942,7 +2942,7 @@
                                       <Disorder id="2983">
                                         <OrphaCode>3366</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3366</ExpertLink>
-                                        <Name lang="en">Isolated trigonocephaly</Name>
+                                        <Name lang="en">Non-syndromic metopic craniosynostosis</Name>
                                         <DisorderType id="21415">
                                           <Name lang="en">Morphological anomaly</Name>
                                         </DisorderType>
@@ -2954,19 +2954,7 @@
                                       <Disorder id="10367">
                                         <OrphaCode>35093</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35093</ExpertLink>
-                                        <Name lang="en">Isolated scaphocephaly</Name>
-                                        <DisorderType id="21415">
-                                          <Name lang="en">Morphological anomaly</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="10368">
-                                        <OrphaCode>35098</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35098</ExpertLink>
-                                        <Name lang="en">Isolated plagiocephaly</Name>
+                                        <Name lang="en">Non-syndromic sagittal craniosynostosis</Name>
                                         <DisorderType id="21415">
                                           <Name lang="en">Morphological anomaly</Name>
                                         </DisorderType>
@@ -2978,19 +2966,7 @@
                                       <Disorder id="10369">
                                         <OrphaCode>35099</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35099</ExpertLink>
-                                        <Name lang="en">Isolated brachycephaly</Name>
-                                        <DisorderType id="21415">
-                                          <Name lang="en">Morphological anomaly</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="10822">
-                                        <OrphaCode>63440</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63440</ExpertLink>
-                                        <Name lang="en">Isolated oxycephaly</Name>
+                                        <Name lang="en">Non-syndromic bicoronal craniosynostosis</Name>
                                         <DisorderType id="21415">
                                           <Name lang="en">Morphological anomaly</Name>
                                         </DisorderType>
@@ -7610,9 +7586,9 @@
                                       <Disorder id="2078">
                                         <OrphaCode>2196</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
-                                        <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
+                                        <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -7970,9 +7946,9 @@
                                       <Disorder id="2078">
                                         <OrphaCode>2196</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
-                                        <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
-                                        <DisorderType id="21394">
-                                          <Name lang="en">Disease</Name>
+                                        <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
