commit a9304bcd06b429331d3fd652192800e94c21e73c
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml
index 48d8896..63ae220 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_182.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:29:32" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:32:24" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -435,7 +435,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="46">
+                      <ClassificationNodeChildList count="45">
                         <ClassificationNode>
                           <Disorder id="126">
                             <OrphaCode>567</OrphaCode>
@@ -1096,18 +1096,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="12508">
-                            <OrphaCode>93950</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93950</ExpertLink>
-                            <Name lang="en">X-linked intellectual disability, Sutherland-Haan type</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1536">
                             <OrphaCode>1305</OrphaCode>
@@ -1781,7 +1769,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="8">
+                  <ClassificationNodeChildList count="9">
                     <ClassificationNode>
                       <Disorder id="20908">
                         <OrphaCode>294415</OrphaCode>
@@ -1806,6 +1794,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="19476">
+                        <OrphaCode>244283</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=244283</ExpertLink>
+                        <Name lang="en">Biliary atresia with splenic malformation syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="253">
                         <OrphaCode>52</OrphaCode>
@@ -2271,7 +2271,7 @@
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="11">
+                      <ClassificationNodeChildList count="12">
                         <ClassificationNode>
                           <Disorder id="1993">
                             <OrphaCode>2078</OrphaCode>
@@ -2476,6 +2476,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="22342">
+                            <OrphaCode>363705</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363705</ExpertLink>
+                            <Name lang="en">Craniofaciofrontodigital syndrome</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
