commit 10888e2451b47a2f6af40fe12c930f1526a24134
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 27 10:28:21 2023 +0200

    JUNE 2023

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml
index 08ec431..5f7eca1 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_150.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-11-28 07:35:22" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-06-19 08:46:54" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -363,7 +363,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="11">
+                      <ClassificationNodeChildList count="10">
                         <ClassificationNode>
                           <Disorder id="24178">
                             <OrphaCode>466926</OrphaCode>
@@ -424,18 +424,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="12290">
-                            <OrphaCode>93359</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93359</ExpertLink>
-                            <Name lang="en">Spondyloepimetaphyseal dysplasia with joint laxity</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="20058">
                             <OrphaCode>263463</OrphaCode>
@@ -785,7 +773,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="10">
+                  <ClassificationNodeChildList count="11">
                     <ClassificationNode>
                       <Disorder id="22835">
                         <OrphaCode>401820</OrphaCode>
@@ -822,6 +810,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="22704">
+                        <OrphaCode>397922</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=397922</ExpertLink>
+                        <Name lang="en">Ferro-cerebro-cutaneous syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="21">
                         <OrphaCode>447</OrphaCode>
@@ -3031,7 +3031,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="4">
+              <ClassificationNodeChildList count="8">
                 <ClassificationNode>
                   <Disorder id="11215">
                     <OrphaCode>79189</OrphaCode>
@@ -3332,6 +3332,54 @@
                     </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31740">
+                    <OrphaCode>642954</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642954</ExpertLink>
+                    <Name lang="en">Autosomal recessive ataxia due to PEX16 deficiency</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="31741">
+                    <OrphaCode>642965</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642965</ExpertLink>
+                    <Name lang="en">Autosomal recessive ataxia due to PEX2 deficiency</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="12592">
+                    <OrphaCode>95433</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95433</ExpertLink>
+                    <Name lang="en">Autosomal recessive spinocerebellar ataxia-blindness-deafness syndrome</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="19551">
+                    <OrphaCode>247815</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=247815</ExpertLink>
+                    <Name lang="en">Autosomal recessive ataxia due to PEX10 deficiency</Name>
+                    <DisorderType id="21394">
+                      <Name lang="en">Disease</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
@@ -3654,7 +3702,7 @@
                               <Disorder id="19526">
                                 <OrphaCode>247573</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=247573</ExpertLink>
-                                <Name lang="en">Adult-onset citrullinemia type I</Name>
+                                <Name lang="en">Late-onset citrullinemia type I</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -3958,7 +4006,7 @@
                       <Disorder id="17780">
                         <OrphaCode>168598</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=168598</ExpertLink>
-                        <Name lang="en">Brain demyelination due to methionine adenosyltransferase deficiency</Name>
+                        <Name lang="en">Methionine adenosyltransferase I/III deficiency</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -4554,7 +4602,7 @@
                               <Disorder id="29042">
                                 <OrphaCode>583602</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583602</ExpertLink>
-                                <Name lang="en">Neu-laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
+                                <Name lang="en">Neu-Laxova syndrome due to phosphoserine aminotransferase deficiency</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -4566,7 +4614,7 @@
                               <Disorder id="29043">
                                 <OrphaCode>583607</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583607</ExpertLink>
-                                <Name lang="en">Neu-laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
+                                <Name lang="en">Neu-Laxova syndrome due to 3-phosphoglycerate dehydrogenase deficiency</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -4578,7 +4626,7 @@
                               <Disorder id="29044">
                                 <OrphaCode>583612</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=583612</ExpertLink>
-                                <Name lang="en">Neu-laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
+                                <Name lang="en">Neu-Laxova syndrome due to 3-phosphoserine phosphatase deficiency</Name>
                                 <DisorderType id="21443">
                                   <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
@@ -6622,7 +6670,7 @@
                       <Disorder id="11325">
                         <OrphaCode>79299</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79299</ExpertLink>
-                        <Name lang="en">Hyperinsulinism due to glucokinase deficiency</Name>
+                        <Name lang="en">Congenital glucokinase-related hyperinsulinism</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -7129,7 +7177,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="14">
+                          <ClassificationNodeChildList count="15">
                             <ClassificationNode>
                               <Disorder id="27441">
                                 <OrphaCode>527276</OrphaCode>
@@ -7183,59 +7231,11 @@
                                 <OrphaCode>506</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=506</ExpertLink>
                                 <Name lang="en">Leigh syndrome</Name>
-                                <DisorderType id="21436">
-                                  <Name lang="en">Clinical group</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="4">
-                                <ClassificationNode>
-                                  <Disorder id="10945">
-                                    <OrphaCode>70472</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=70472</ExpertLink>
-                                    <Name lang="en">Congenital lactic acidosis, Saguenay-Lac-Saint-Jean type</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="10947">
-                                    <OrphaCode>70474</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=70474</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with cardiomyopathy</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19814">
-                                    <OrphaCode>255241</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=255241</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with leukodystrophy</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19815">
-                                    <OrphaCode>255249</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=255249</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with nephrotic syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -7259,7 +7259,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="7">
+                              <ClassificationNodeChildList count="6">
                                 <ClassificationNode>
                                   <Disorder id="1427">
                                     <OrphaCode>1168</OrphaCode>
@@ -7296,18 +7296,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="19815">
-                                    <OrphaCode>255249</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=255249</ExpertLink>
-                                    <Name lang="en">Leigh syndrome with nephrotic syndrome</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="20470">
                                     <OrphaCode>280406</OrphaCode>
@@ -7377,7 +7365,31 @@
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="0">
+                                  <ClassificationNodeChildList count="2">
+                                    <ClassificationNode>
+                                      <Disorder id="31739">
+                                        <OrphaCode>642945</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642945</ExpertLink>
+                                        <Name lang="en">Perrault syndrome type 1</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31742">
+                                        <OrphaCode>642976</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=642976</ExpertLink>
+                                        <Name lang="en">Perrault syndrome type 2</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -8642,6 +8654,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="10945">
+                                <OrphaCode>70472</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=70472</ExpertLink>
+                                <Name lang="en">Congenital lactic acidosis, Saguenay-Lac-Saint-Jean type</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -8800,7 +8824,7 @@
                                   <Disorder id="12050">
                                     <OrphaCode>90641</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90641</ExpertLink>
-                                    <Name lang="en">Mitochondrial non-syndromic sensorineural deafness</Name>
+                                    <Name lang="en">Rare mitochondrial non-syndromic sensorineural deafness</Name>
                                     <DisorderType id="21443">
                                       <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
@@ -9303,7 +9327,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="7">
                         <ClassificationNode>
                           <Disorder id="5520">
                             <OrphaCode>2394</OrphaCode>
@@ -9436,6 +9460,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="23546">
+                            <OrphaCode>447795</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=447795</ExpertLink>
+                            <Name lang="en">Lipoyl transferase 2 deficiency</Name>
+                            <DisorderType id="21408">
+                              <Name lang="en">Biological anomaly</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="26544">
                             <OrphaCode>508093</OrphaCode>
@@ -10414,7 +10450,7 @@
                       <Disorder id="429">
                         <OrphaCode>124</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=124</ExpertLink>
-                        <Name lang="en">Blackfan-Diamond anemia</Name>
+                        <Name lang="en">Diamond-Blackfan anemia</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -10679,8 +10715,8 @@
                         <OrphaCode>633099</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=633099</ExpertLink>
                         <Name lang="en">PAICS deficiency</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -13419,7 +13455,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="8">
+                      <ClassificationNodeChildList count="9">
                         <ClassificationNode>
                           <Disorder id="504">
                             <OrphaCode>446</OrphaCode>
@@ -13472,7 +13508,7 @@
                           <Disorder id="16911">
                             <OrphaCode>139507</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139507</ExpertLink>
-                            <Name lang="en">African iron overload</Name>
+                            <Name lang="en">Dietary iron overload disease</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -13501,24 +13537,12 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="4">
-                            <ClassificationNode>
-                              <Disorder id="11256">
-                                <OrphaCode>79230</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79230</ExpertLink>
-                                <Name lang="en">Hemochromatosis type 2</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="3">
                             <ClassificationNode>
-                              <Disorder id="16909">
-                                <OrphaCode>139491</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139491</ExpertLink>
-                                <Name lang="en">Hemochromatosis type 4</Name>
+                              <Disorder id="24056">
+                                <OrphaCode>465508</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=465508</ExpertLink>
+                                <Name lang="en">Symptomatic form of HFE-related hemochromatosis</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -13527,22 +13551,58 @@
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
-                              <Disorder id="18976">
-                                <OrphaCode>225123</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225123</ExpertLink>
-                                <Name lang="en">Hemochromatosis type 3</Name>
-                                <DisorderType id="21394">
-                                  <Name lang="en">Disease</Name>
+                              <Disorder id="31842">
+                                <OrphaCode>648569</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648569</ExpertLink>
+                                <Name lang="en">Non-HFE-related hemochromatosis</Name>
+                                <DisorderType id="21436">
+                                  <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="0">
+                              <ClassificationNodeChildList count="3">
+                                <ClassificationNode>
+                                  <Disorder id="18976">
+                                    <OrphaCode>225123</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=225123</ExpertLink>
+                                    <Name lang="en">TFR2-related hemochromatosis</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="31821">
+                                    <OrphaCode>647834</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=647834</ExpertLink>
+                                    <Name lang="en">SLC40A1-related hemochromatosis</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="11256">
+                                    <OrphaCode>79230</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79230</ExpertLink>
+                                    <Name lang="en">HJV or HAMP-related hemochromatosis</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
-                              <Disorder id="24056">
-                                <OrphaCode>465508</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=465508</ExpertLink>
-                                <Name lang="en">Symptomatic form of hemochromatosis type 1</Name>
+                              <Disorder id="31843">
+                                <OrphaCode>648581</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648581</ExpertLink>
+                                <Name lang="en">Digenic hemochromatosis</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -13564,6 +13624,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="31841">
+                            <OrphaCode>648562</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=648562</ExpertLink>
+                            <Name lang="en">Ferroportin Disease</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
