commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_200.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_200.xml
index 41f8a3c..f02a08d 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_200.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_200.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:43:51" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:47:32" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2338,7 +2338,7 @@
                       <Disorder id="11913">
                         <OrphaCode>89938</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89938</ExpertLink>
-                        <Name lang="en">Infantile Bartter syndrome with sensorineural deafness</Name>
+                        <Name lang="en">Bartter syndrome type 4</Name>
                         <DisorderType id="21450">
                           <Name lang="en">Clinical subtype</Name>
                         </DisorderType>
