commit 3b410eecdc7bf8d083482edf97dee966c6328b80
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed May 6 16:03:39 2020 +0200

    MAY 2020

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_211.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_211.xml
index 1ab51c0..313bd57 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_211.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_211.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-04-01 05:16:30" version="1.3.1 / 4.1.7 [2019-11-24] (orientdb version)" copyright="Orphanet (c) 2020">
+<JDBOR date="2020-05-04 04:05:30" version="1.3.1 / 4.1.7 [2019-11-24] (orientdb version)" copyright="Orphanet (c) 2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,13 +7,10 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="1">
-    <Disorder id="13051">
-      <OrphaNumber>98033</OrphaNumber>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98033</ExpertLink>
-      <Name lang="en">Rare neurologic disease with psychiatric involvement</Name>
-      <ClassificationNodeList count="0">
-      </ClassificationNodeList>
-    </Disorder>
-  </DisorderList>
+  <Error>
+    <Status>2</Status>
+    <StatusDesc>XML_PARSING</StatusDesc>
+    <Exception>org.xml.sax.SAXParseException</Exception>
+    <Message>Premature end of file.</Message>
+  </Error>
 </JDBOR>
