commit 4a84d9e90bb507ce17a52f782689062056a98175
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jul 2 21:30:06 2024 +0200

    JULY 2024

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml
index 13b4bd2..f046e27 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_198.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-12-04 08:05:46" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-07-01 09:24:21" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -41,7 +41,19 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="11">
+                  <ClassificationNodeChildList count="12">
+                    <ClassificationNode>
+                      <Disorder id="32113">
+                        <OrphaCode>675404</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675404</ExpertLink>
+                        <Name lang="en">May-Thurner syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="31727">
                         <OrphaCode>641829</OrphaCode>
@@ -644,7 +656,7 @@
                           <Disorder id="22938">
                             <OrphaCode>404553</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=404553</ExpertLink>
-                            <Name lang="en">Vasculitis due to ADA2 deficiency</Name>
+                            <Name lang="en">Adenosine deaminase 2 deficiency</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -914,7 +926,7 @@
               <Disorder id="10560">
                 <OrphaCode>68419</OrphaCode>
                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=68419</ExpertLink>
-                <Name lang="en">Vascular anomaly or angioma</Name>
+                <Name lang="en">Rare vascular anomaly</Name>
                 <DisorderType id="36561">
                   <Name lang="en">Category</Name>
                 </DisorderType>
@@ -929,7 +941,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="9">
+                  <ClassificationNodeChildList count="10">
                     <ClassificationNode>
                       <Disorder id="17810">
                         <OrphaCode>168984</OrphaCode>
@@ -1062,6 +1074,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="21927">
+                        <OrphaCode>329324</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329324</ExpertLink>
+                        <Name lang="en">Inverse Klippel-Trénaunay syndrome</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -1273,7 +1297,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="7">
+                          <ClassificationNodeChildList count="8">
                             <ClassificationNode>
                               <Disorder id="2859">
                                 <OrphaCode>3193</OrphaCode>
@@ -1286,6 +1310,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="3434">
+                                <OrphaCode>1456</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1456</ExpertLink>
+                                <Name lang="en">Middle aortic syndrome</Name>
+                                <DisorderType id="21415">
+                                  <Name lang="en">Morphological anomaly</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="28051">
                                 <OrphaCode>542568</OrphaCode>
@@ -1307,31 +1343,7 @@
                                   <Name lang="en">Morphological anomaly</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="444">
-                                    <OrphaCode>1455</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1455</ExpertLink>
-                                    <Name lang="en">Autosomal dominant coarctation of aorta</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="3434">
-                                    <OrphaCode>1456</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1456</ExpertLink>
-                                    <Name lang="en">Atypical coarctation of aorta</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -2093,106 +2105,58 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="17">
-                    <ClassificationNode>
-                      <Disorder id="23750">
-                        <OrphaCode>458758</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458758</ExpertLink>
-                        <Name lang="en">Composite hemangioendothelioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="23751">
-                        <OrphaCode>458763</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458763</ExpertLink>
-                        <Name lang="en">Retiform hemangioendothelioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="1359">
-                        <OrphaCode>1062</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1062</ExpertLink>
-                        <Name lang="en">Hereditary neurocutaneous malformation</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="10316">
-                        <OrphaCode>33276</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=33276</ExpertLink>
-                        <Name lang="en">Kaposi sarcoma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="23752">
-                        <OrphaCode>458768</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458768</ExpertLink>
-                        <Name lang="en">Primary intralymphatic angioendothelioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="2030">
-                        <OrphaCode>2123</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2123</ExpertLink>
-                        <Name lang="en">Diffuse neonatal hemangiomatosis</Name>
-                        <DisorderType id="21401">
-                          <Name lang="en">Malformation syndrome</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
+                  <ClassificationNodeChildList count="3">
                     <ClassificationNode>
-                      <Disorder id="20050">
-                        <OrphaCode>263413</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263413</ExpertLink>
-                        <Name lang="en">Angiosarcoma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                      <Disorder id="32077">
+                        <OrphaCode>673466</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673466</ExpertLink>
+                        <Name lang="en">Malignant vascular tumor</Name>
+                        <DisorderType id="21436">
+                          <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="17144">
+                            <OrphaCode>157791</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=157791</ExpertLink>
+                            <Name lang="en">Epithelioid hemangioendothelioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="20050">
+                            <OrphaCode>263413</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=263413</ExpertLink>
+                            <Name lang="en">Angiosarcoma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="23753">
-                        <OrphaCode>458775</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458775</ExpertLink>
-                        <Name lang="en">Congenital hemangioma</Name>
+                      <Disorder id="32078">
+                        <OrphaCode>673470</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673470</ExpertLink>
+                        <Name lang="en">Benign vascular tumor</Name>
                         <DisorderType id="21436">
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="3">
+                      <ClassificationNodeChildList count="16">
                         <ClassificationNode>
-                          <Disorder id="17056">
-                            <OrphaCode>141184</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141184</ExpertLink>
-                            <Name lang="en">Rapidly involuting congenital hemangioma</Name>
+                          <Disorder id="1359">
+                            <OrphaCode>1062</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1062</ExpertLink>
+                            <Name lang="en">Hereditary neurocutaneous malformation</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2201,10 +2165,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="23754">
-                            <OrphaCode>458785</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458785</ExpertLink>
-                            <Name lang="en">Partially involuting congenital hemangioma</Name>
+                          <Disorder id="3543">
+                            <OrphaCode>1063</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1063</ExpertLink>
+                            <Name lang="en">Tufted angioma</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2213,10 +2177,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="17055">
-                            <OrphaCode>141179</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141179</ExpertLink>
-                            <Name lang="en">Non-involuting congenital hemangioma</Name>
+                          <Disorder id="10988">
+                            <OrphaCode>71213</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=71213</ExpertLink>
+                            <Name lang="en">Retinal capillary malformation</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2224,83 +2188,275 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="18698">
+                            <OrphaCode>210584</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=210584</ExpertLink>
+                            <Name lang="en">Spindle cell hemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="23753">
+                            <OrphaCode>458775</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458775</ExpertLink>
+                            <Name lang="en">Congenital hemangioma</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="3">
+                            <ClassificationNode>
+                              <Disorder id="17056">
+                                <OrphaCode>141184</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141184</ExpertLink>
+                                <Name lang="en">Rapidly involuting congenital hemangioma</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="23754">
+                                <OrphaCode>458785</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458785</ExpertLink>
+                                <Name lang="en">Partially involuting congenital hemangioma</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="17055">
+                                <OrphaCode>141179</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=141179</ExpertLink>
+                                <Name lang="en">Non-involuting congenital hemangioma</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32106">
+                            <OrphaCode>675359</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675359</ExpertLink>
+                            <Name lang="en">Anastomosing haemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32115">
+                            <OrphaCode>675597</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675597</ExpertLink>
+                            <Name lang="en">Acquired elastotic haemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32107">
+                            <OrphaCode>675362</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675362</ExpertLink>
+                            <Name lang="en">Hobnail hemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32108">
+                            <OrphaCode>675369</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675369</ExpertLink>
+                            <Name lang="en">Microvenular haemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32112">
+                            <OrphaCode>675396</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675396</ExpertLink>
+                            <Name lang="en">Epithelioid hemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32082">
+                            <OrphaCode>673543</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673543</ExpertLink>
+                            <Name lang="en">Papillary hemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32087">
+                            <OrphaCode>673574</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673574</ExpertLink>
+                            <Name lang="en">Reactive angioendotheliomatosis</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32086">
+                            <OrphaCode>673568</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673568</ExpertLink>
+                            <Name lang="en">Eccrine angiomatous hamartoma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32081">
+                            <OrphaCode>673538</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673538</ExpertLink>
+                            <Name lang="en">Littoral cell hemangioma of the spleen</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32080">
+                            <OrphaCode>673525</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673525</ExpertLink>
+                            <Name lang="en">Intravascular papillary endothelial hyperplasia</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="18699">
+                            <OrphaCode>210589</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=210589</ExpertLink>
+                            <Name lang="en">Rare infantile hemangioma</Name>
+                            <DisorderType id="21436">
+                              <Name lang="en">Clinical group</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="5">
+                            <ClassificationNode>
+                              <Disorder id="2030">
+                                <OrphaCode>2123</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2123</ExpertLink>
+                                <Name lang="en">Multifocal infantile hemangioma with extracutenous involvement</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="32110">
+                                <OrphaCode>675380</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=675380</ExpertLink>
+                                <Name lang="en">Isolated segmental infantile hemangioma</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="16751">
+                                <OrphaCode>137935</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137935</ExpertLink>
+                                <Name lang="en">Airway infantile hemangioma</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="10577">
+                                <OrphaCode>42775</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=42775</ExpertLink>
+                                <Name lang="en">PHACE syndrome</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="11599">
+                                <OrphaCode>83628</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=83628</ExpertLink>
+                                <Name lang="en">LUMBAR syndrome</Name>
+                                <DisorderType id="21401">
+                                  <Name lang="en">Malformation syndrome</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
-                      <Disorder id="18698">
-                        <OrphaCode>210584</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=210584</ExpertLink>
-                        <Name lang="en">Spindle cell hemangioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="3543">
-                        <OrphaCode>1063</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1063</ExpertLink>
-                        <Name lang="en">Tufted angioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="3544">
-                        <OrphaCode>2122</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2122</ExpertLink>
-                        <Name lang="en">Kaposiform hemangioendothelioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="10988">
-                        <OrphaCode>71213</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=71213</ExpertLink>
-                        <Name lang="en">Retinal capillary malformation</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="17144">
-                        <OrphaCode>157791</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=157791</ExpertLink>
-                        <Name lang="en">Epithelioid hemangioendothelioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="18699">
-                        <OrphaCode>210589</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=210589</ExpertLink>
-                        <Name lang="en">Infantile hemangioma of rare localization</Name>
+                      <Disorder id="32079">
+                        <OrphaCode>673473</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673473</ExpertLink>
+                        <Name lang="en">Borderline vascular tumor</Name>
                         <DisorderType id="21436">
                           <Name lang="en">Clinical group</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="2">
+                      <ClassificationNodeChildList count="6">
                         <ClassificationNode>
-                          <Disorder id="16965">
-                            <OrphaCode>140436</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140436</ExpertLink>
-                            <Name lang="en">Primary intraosseous venous malformation</Name>
+                          <Disorder id="3544">
+                            <OrphaCode>2122</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2122</ExpertLink>
+                            <Name lang="en">Kaposiform hemangioendothelioma</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2309,10 +2465,10 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="16751">
-                            <OrphaCode>137935</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137935</ExpertLink>
-                            <Name lang="en">Laryngotracheal angioma</Name>
+                          <Disorder id="10316">
+                            <OrphaCode>33276</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=33276</ExpertLink>
+                            <Name lang="en">Kaposi sarcoma</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2320,47 +2476,47 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="23758">
-                        <OrphaCode>458827</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458827</ExpertLink>
-                        <Name lang="en">Vascular tumor with associated anomalies</Name>
-                        <DisorderType id="36561">
-                          <Name lang="en">Category</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="3">
                         <ClassificationNode>
-                          <Disorder id="10577">
-                            <OrphaCode>42775</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=42775</ExpertLink>
-                            <Name lang="en">PHACE syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
+                          <Disorder id="23750">
+                            <OrphaCode>458758</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458758</ExpertLink>
+                            <Name lang="en">Composite hemangioendothelioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="11599">
-                            <OrphaCode>83628</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=83628</ExpertLink>
-                            <Name lang="en">LUMBAR syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
+                          <Disorder id="23751">
+                            <OrphaCode>458763</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458763</ExpertLink>
+                            <Name lang="en">Retiform hemangioendothelioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="21927">
-                            <OrphaCode>329324</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329324</ExpertLink>
-                            <Name lang="en">Inverse Klippel-Trénaunay syndrome</Name>
+                          <Disorder id="23752">
+                            <OrphaCode>458768</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=458768</ExpertLink>
+                            <Name lang="en">Papillary intralymphatic angioendothelioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="32085">
+                            <OrphaCode>673556</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=673556</ExpertLink>
+                            <Name lang="en">Pseudomyogenic hemangioendothelioma</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -2368,30 +2524,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="24017">
-                        <OrphaCode>464318</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464318</ExpertLink>
-                        <Name lang="en">Verrucous hemangioma</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
-                      </ClassificationNodeChildList>
-                    </ClassificationNode>
-                    <ClassificationNode>
-                      <Disorder id="24019">
-                        <OrphaCode>464329</OrphaCode>
-                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464329</ExpertLink>
-                        <Name lang="en">Kaposiform lymphangiomatosis</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
-                        </DisorderType>
-                      </Disorder>
-                      <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -2415,7 +2547,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="6">
+                      <ClassificationNodeChildList count="7">
                         <ClassificationNode>
                           <Disorder id="12588">
                             <OrphaCode>95429</OrphaCode>
@@ -2440,6 +2572,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="24017">
+                            <OrphaCode>464318</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464318</ExpertLink>
+                            <Name lang="en">Verrucous hemangioma</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="23759">
                             <OrphaCode>458830</OrphaCode>
@@ -2559,7 +2703,19 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="5">
+                      <ClassificationNodeChildList count="6">
+                        <ClassificationNode>
+                          <Disorder id="16965">
+                            <OrphaCode>140436</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140436</ExpertLink>
+                            <Name lang="en">Primary intraosseous venous malformation</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="18935">
                             <OrphaCode>221061</OrphaCode>
@@ -2661,7 +2817,19 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="4">
+                                  <ClassificationNodeChildList count="5">
+                                    <ClassificationNode>
+                                      <Disorder id="20949">
+                                        <OrphaCode>295000</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=295000</ExpertLink>
+                                        <Name lang="en">Amniotic band syndrome</Name>
+                                        <DisorderType id="21401">
+                                          <Name lang="en">Malformation syndrome</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                     <ClassificationNode>
                                       <Disorder id="11478">
                                         <OrphaCode>79452</OrphaCode>
@@ -2721,19 +2889,7 @@
                                       <Name lang="en">Clinical group</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="7">
-                                    <ClassificationNode>
-                                      <Disorder id="2883">
-                                        <OrphaCode>3226</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3226</ExpertLink>
-                                        <Name lang="en">Deafness-lymphedema-leukemia syndrome</Name>
-                                        <DisorderType id="21401">
-                                          <Name lang="en">Malformation syndrome</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="6">
                                     <ClassificationNode>
                                       <Disorder id="437">
                                         <OrphaCode>1414</OrphaCode>
@@ -2944,7 +3100,7 @@
                                   <Disorder id="25364">
                                     <OrphaCode>487796</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=487796</ExpertLink>
-                                    <Name lang="en">Macrothrombocytopenia-lymphedema-developmental delay-facial dysmorphism-camptodactyly syndrome</Name>
+                                    <Name lang="en">Takenouchi-Kosaki syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
@@ -3064,12 +3220,36 @@
                                   <Disorder id="10630">
                                     <OrphaCode>48652</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=48652</ExpertLink>
-                                    <Name lang="en">Monosomy 22q13.3</Name>
+                                    <Name lang="en">Phelan-McDermid syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="0">
+                                  <ClassificationNodeChildList count="2">
+                                    <ClassificationNode>
+                                      <Disorder id="31993">
+                                        <OrphaCode>662172</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662172</ExpertLink>
+                                        <Name lang="en">Phelan-McDermid syndrome due to SHANK3 mutation</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
+                                    <ClassificationNode>
+                                      <Disorder id="31992">
+                                        <OrphaCode>662169</OrphaCode>
+                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=662169</ExpertLink>
+                                        <Name lang="en">Phelan-McDermid syndrome due to 22q13.3 deletion</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
+                                        </DisorderType>
+                                      </Disorder>
+                                      <ClassificationNodeChildList count="0">
+                                      </ClassificationNodeChildList>
+                                    </ClassificationNode>
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -3145,7 +3325,7 @@
                               <Name lang="en">Category</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="5">
+                          <ClassificationNodeChildList count="6">
                             <ClassificationNode>
                               <Disorder id="14158">
                                 <OrphaCode>99141</OrphaCode>
@@ -3230,6 +3410,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="24019">
+                                <OrphaCode>464329</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464329</ExpertLink>
+                                <Name lang="en">Kaposiform lymphangiomatosis</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="24018">
                                 <OrphaCode>464321</OrphaCode>
