commit c92e9c5f9940abcb2c7950377e1c2245059cad90
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 1 10:54:09 2021 +0100

    MARCH 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml
old mode 100755
new mode 100644
index 741df9a..69442f0
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_156.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-02-01 04:32:49" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-03-01 04:32:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -5189,8 +5189,8 @@
                                     <OrphaCode>268261</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268261</ExpertLink>
                                     <Name lang="en">DYRK1A-related intellectual disability syndrome due to 21q22.13q22.2 microdeletion</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -5343,8 +5343,8 @@
                                         <OrphaCode>313884</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313884</ExpertLink>
                                         <Name lang="en">12p12.1 microdeletion syndrome</Name>
-                                        <DisorderType id="21401">
-                                          <Name lang="en">Malformation syndrome</Name>
+                                        <DisorderType id="21450">
+                                          <Name lang="en">Clinical subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -10170,7 +10170,7 @@
                               <Disorder id="11434">
                                 <OrphaCode>79408</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79408</ExpertLink>
-                                <Name lang="en">Severe generalized recessive dystrophic epidermolysis bullosa</Name>
+                                <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10182,7 +10182,7 @@
                               <Disorder id="11435">
                                 <OrphaCode>79409</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79409</ExpertLink>
-                                <Name lang="en">Recessive dystrophic epidermolysis bullosa inversa</Name>
+                                <Name lang="en">Dystrophic epidermolysis bullosa inversa</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10204,7 +10204,7 @@
                                   <Disorder id="11436">
                                     <OrphaCode>79410</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79410</ExpertLink>
-                                    <Name lang="en">Pretibial dystrophic epidermolysis bullosa</Name>
+                                    <Name lang="en">Localized dystrophic epidermolysis bullosa, pretibial form</Name>
                                     <DisorderType id="21450">
                                       <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
@@ -10216,7 +10216,7 @@
                                   <Disorder id="17233">
                                     <OrphaCode>158673</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158673</ExpertLink>
-                                    <Name lang="en">Acral dystrophic epidermolysis bullosa</Name>
+                                    <Name lang="en">Localized dystrophic epidermolysis bullosa, acral form</Name>
                                     <DisorderType id="21450">
                                       <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
@@ -10228,7 +10228,7 @@
                                   <Disorder id="17234">
                                     <OrphaCode>158676</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158676</ExpertLink>
-                                    <Name lang="en">Dominant dystrophic epidermolysis bullosa, nails only</Name>
+                                    <Name lang="en">Localized dystrophic epidermolysis bullosa, nails only</Name>
                                     <DisorderType id="21450">
                                       <Name lang="en">Clinical subtype</Name>
                                     </DisorderType>
@@ -10242,7 +10242,7 @@
                               <Disorder id="11437">
                                 <OrphaCode>79411</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79411</ExpertLink>
-                                <Name lang="en">Transient bullous dermolysis of the newborn</Name>
+                                <Name lang="en">Self-improving dystrophic epidermolysis bullosa</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10254,7 +10254,7 @@
                               <Disorder id="11907">
                                 <OrphaCode>89842</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89842</ExpertLink>
-                                <Name lang="en">Recessive dystrophic epidermolysis bullosa, generalized intermediate</Name>
+                                <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, intermediate form</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10278,7 +10278,7 @@
                               <Disorder id="19201">
                                 <OrphaCode>231568</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231568</ExpertLink>
-                                <Name lang="en">Generalized dominant dystrophic epidermolysis bullosa</Name>
+                                <Name lang="en">Autosomal dominant generalized dystrophic epidermolysis bullosa</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10307,7 +10307,7 @@
                                   <Name lang="en">Category</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="10">
+                              <ClassificationNodeChildList count="9">
                                 <ClassificationNode>
                                   <Disorder id="23043">
                                     <OrphaCode>412181</OrphaCode>
@@ -10336,7 +10336,7 @@
                                   <Disorder id="11422">
                                     <OrphaCode>79396</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79396</ExpertLink>
-                                    <Name lang="en">Epidermolysis bullosa simplex, generalized severe</Name>
+                                    <Name lang="en">Autosomal dominant generalized epidermolysis bullosa simplex, severe form</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -10360,7 +10360,7 @@
                                   <Disorder id="11425">
                                     <OrphaCode>79399</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79399</ExpertLink>
-                                    <Name lang="en">Epidermolysis bullosa simplex, generalized intermediate</Name>
+                                    <Name lang="en">Autosomal dominant generalized epidermolysis bullosa simplex, intermediate form</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -10384,7 +10384,7 @@
                                   <Disorder id="11427">
                                     <OrphaCode>79401</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79401</ExpertLink>
-                                    <Name lang="en">PLEC-related epidermolysis bullosa simplex without extracutaneous involvement</Name>
+                                    <Name lang="en">PLEC-related intermediate epidermolysis bullosa simplex without extracutaneous involvement</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -10396,19 +10396,7 @@
                                   <Disorder id="11903">
                                     <OrphaCode>89838</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89838</ExpertLink>
-                                    <Name lang="en">Epidermolysis bullosa simplex, autosomal recessive K14</Name>
-                                    <DisorderType id="21394">
-                                      <Name lang="en">Disease</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="11904">
-                                    <OrphaCode>89839</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89839</ExpertLink>
-                                    <Name lang="en">OBSOLETE: Epidermolysis bullosa simplex superficialis</Name>
+                                    <Name lang="en">Autosomal recessive generalized epidermolysis bullosa simplex</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -10492,7 +10480,7 @@
                                   <Disorder id="21112">
                                     <OrphaCode>300333</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=300333</ExpertLink>
-                                    <Name lang="en">Nephrotic syndrome-deafness-pretibial epidermolysis bullosa syndrome</Name>
+                                    <Name lang="en">Nephrotic syndrome-epidermolysis bullosa-sensorineural deafness syndrome</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -10530,7 +10518,7 @@
                               <Disorder id="2233">
                                 <OrphaCode>2407</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-                                <Name lang="en">LOC syndrome</Name>
+                                <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10542,7 +10530,7 @@
                               <Disorder id="11429">
                                 <OrphaCode>79403</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79403</ExpertLink>
-                                <Name lang="en">Junctional epidermolysis bullosa-pyloric atresia syndrome</Name>
+                                <Name lang="en">Junctional epidermolysis bullosa with pyloric atresia</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10554,7 +10542,7 @@
                               <Disorder id="11430">
                                 <OrphaCode>79404</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79404</ExpertLink>
-                                <Name lang="en">Junctional epidermolysis bullosa, generalized severe</Name>
+                                <Name lang="en">Severe generalized junctional epidermolysis bullosa</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10590,7 +10578,7 @@
                               <Disorder id="19640">
                                 <OrphaCode>251393</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251393</ExpertLink>
-                                <Name lang="en">Localized junctional epidermolysis bullosa, non-Herlitz type</Name>
+                                <Name lang="en">Localized junctional epidermolysis bullosa</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10614,7 +10602,7 @@
                               <Disorder id="21219">
                                 <OrphaCode>306504</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306504</ExpertLink>
-                                <Name lang="en">Junctional epidermolysis bullosa with respiratory and renal involvement</Name>
+                                <Name lang="en">Interstitial lung disease-nephrotic syndrome-epidermolysis bullosa syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -10626,7 +10614,7 @@
                               <Disorder id="11428">
                                 <OrphaCode>79402</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79402</ExpertLink>
-                                <Name lang="en">Junctional epidermolysis bullosa, generalized intermediate</Name>
+                                <Name lang="en">Intermediate generalized junctional epidermolysis bullosa</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -53039,7 +53027,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="10">
+                              <ClassificationNodeChildList count="11">
                                 <ClassificationNode>
                                   <Disorder id="14721">
                                     <OrphaCode>101010</OrphaCode>
@@ -53136,6 +53124,18 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
+                                <ClassificationNode>
+                                  <Disorder id="14030">
+                                    <OrphaCode>99013</OrphaCode>
+                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99013</ExpertLink>
+                                    <Name lang="en">Spastic paraplegia type 7</Name>
+                                    <DisorderType id="21394">
+                                      <Name lang="en">Disease</Name>
+                                    </DisorderType>
+                                  </Disorder>
+                                  <ClassificationNodeChildList count="0">
+                                  </ClassificationNodeChildList>
+                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="23537">
                                     <OrphaCode>447757</OrphaCode>
@@ -58823,7 +58823,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="10">
+                              <ClassificationNodeChildList count="9">
                                 <ClassificationNode>
                                   <Disorder id="2334">
                                     <OrphaCode>2524</OrphaCode>
@@ -58872,18 +58872,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="17609">
-                                    <OrphaCode>166068</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166068</ExpertLink>
-                                    <Name lang="en">Pontocerebellar hypoplasia type 5</Name>
-                                    <DisorderType id="21401">
-                                      <Name lang="en">Malformation syndrome</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="17610">
                                     <OrphaCode>166073</OrphaCode>
@@ -68557,7 +68545,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="10">
+                          <ClassificationNodeChildList count="9">
                             <ClassificationNode>
                               <Disorder id="2334">
                                 <OrphaCode>2524</OrphaCode>
@@ -68606,18 +68594,6 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="17609">
-                                <OrphaCode>166068</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166068</ExpertLink>
-                                <Name lang="en">Pontocerebellar hypoplasia type 5</Name>
-                                <DisorderType id="21401">
-                                  <Name lang="en">Malformation syndrome</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="17610">
                                 <OrphaCode>166073</OrphaCode>
@@ -73395,7 +73371,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="519">
+                      <ClassificationNodeChildList count="517">
                         <ClassificationNode>
                           <Disorder id="24010">
                             <OrphaCode>464282</OrphaCode>
@@ -78138,36 +78114,12 @@
                               <Disorder id="3423">
                                 <OrphaCode>2745</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-                                <Name lang="en">Opitz G/BBB syndrome</Name>
+                                <Name lang="en">Opitz GBBB syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="21237">
-                                    <OrphaCode>306588</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-                                    <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-                                    <DisorderType id="21443">
-                                      <Name lang="en">Etiological subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="21238">
-                                    <OrphaCode>306597</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-                                    <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-                                    <DisorderType id="21443">
-                                      <Name lang="en">Etiological subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -80956,18 +80908,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="21461">
-                            <OrphaCode>313892</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313892</ExpertLink>
-                            <Name lang="en">Developmental and speech delay due to SOX5 deficiency</Name>
-                            <DisorderType id="21394">
-                              <Name lang="en">Disease</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="21466">
                             <OrphaCode>313947</OrphaCode>
@@ -81916,18 +81856,6 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
-                        <ClassificationNode>
-                          <Disorder id="21460">
-                            <OrphaCode>313884</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313884</ExpertLink>
-                            <Name lang="en">12p12.1 microdeletion syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
-                            </DisorderType>
-                          </Disorder>
-                          <ClassificationNodeChildList count="0">
-                          </ClassificationNodeChildList>
-                        </ClassificationNode>
                         <ClassificationNode>
                           <Disorder id="1539">
                             <OrphaCode>1313</OrphaCode>
@@ -82357,7 +82285,31 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="21461">
+                                <OrphaCode>313892</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313892</ExpertLink>
+                                <Name lang="en">Developmental and speech delay due to SOX5 deficiency</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="21460">
+                                <OrphaCode>313884</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313884</ExpertLink>
+                                <Name lang="en">12p12.1 microdeletion syndrome</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -82939,8 +82891,8 @@
                                 <OrphaCode>268261</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268261</ExpertLink>
                                 <Name lang="en">DYRK1A-related intellectual disability syndrome due to 21q22.13q22.2 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -82951,8 +82903,8 @@
                                 <OrphaCode>464311</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464311</ExpertLink>
                                 <Name lang="en">Intellectual disability syndrome due to a DYRK1A point mutation</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -85328,7 +85280,7 @@
                           <Disorder id="21112">
                             <OrphaCode>300333</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=300333</ExpertLink>
-                            <Name lang="en">Nephrotic syndrome-deafness-pretibial epidermolysis bullosa syndrome</Name>
+                            <Name lang="en">Nephrotic syndrome-epidermolysis bullosa-sensorineural deafness syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -94424,7 +94376,7 @@
                           <Disorder id="21112">
                             <OrphaCode>300333</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=300333</ExpertLink>
-                            <Name lang="en">Nephrotic syndrome-deafness-pretibial epidermolysis bullosa syndrome</Name>
+                            <Name lang="en">Nephrotic syndrome-epidermolysis bullosa-sensorineural deafness syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -94436,7 +94388,7 @@
                           <Disorder id="21219">
                             <OrphaCode>306504</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306504</ExpertLink>
-                            <Name lang="en">Junctional epidermolysis bullosa with respiratory and renal involvement</Name>
+                            <Name lang="en">Interstitial lung disease-nephrotic syndrome-epidermolysis bullosa syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -96952,7 +96904,7 @@
                                   <Disorder id="11430">
                                     <OrphaCode>79404</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79404</ExpertLink>
-                                    <Name lang="en">Junctional epidermolysis bullosa, generalized severe</Name>
+                                    <Name lang="en">Severe generalized junctional epidermolysis bullosa</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -96964,7 +96916,7 @@
                                   <Disorder id="11434">
                                     <OrphaCode>79408</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79408</ExpertLink>
-                                    <Name lang="en">Severe generalized recessive dystrophic epidermolysis bullosa</Name>
+                                    <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -96976,7 +96928,7 @@
                                   <Disorder id="11907">
                                     <OrphaCode>89842</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89842</ExpertLink>
-                                    <Name lang="en">Recessive dystrophic epidermolysis bullosa, generalized intermediate</Name>
+                                    <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, intermediate form</Name>
                                     <DisorderType id="21394">
                                       <Name lang="en">Disease</Name>
                                     </DisorderType>
@@ -101334,7 +101286,7 @@
                               <Disorder id="2233">
                                 <OrphaCode>2407</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-                                <Name lang="en">LOC syndrome</Name>
+                                <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
                                 <DisorderType id="21394">
                                   <Name lang="en">Disease</Name>
                                 </DisorderType>
@@ -105866,36 +105818,12 @@
                                       <Disorder id="3423">
                                         <OrphaCode>2745</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-                                        <Name lang="en">Opitz G/BBB syndrome</Name>
+                                        <Name lang="en">Opitz GBBB syndrome</Name>
                                         <DisorderType id="21401">
                                           <Name lang="en">Malformation syndrome</Name>
                                         </DisorderType>
                                       </Disorder>
-                                      <ClassificationNodeChildList count="2">
-                                        <ClassificationNode>
-                                          <Disorder id="21237">
-                                            <OrphaCode>306588</OrphaCode>
-                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-                                            <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-                                            <DisorderType id="21443">
-                                              <Name lang="en">Etiological subtype</Name>
-                                            </DisorderType>
-                                          </Disorder>
-                                          <ClassificationNodeChildList count="0">
-                                          </ClassificationNodeChildList>
-                                        </ClassificationNode>
-                                        <ClassificationNode>
-                                          <Disorder id="21238">
-                                            <OrphaCode>306597</OrphaCode>
-                                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-                                            <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-                                            <DisorderType id="21443">
-                                              <Name lang="en">Etiological subtype</Name>
-                                            </DisorderType>
-                                          </Disorder>
-                                          <ClassificationNodeChildList count="0">
-                                          </ClassificationNodeChildList>
-                                        </ClassificationNode>
+                                      <ClassificationNodeChildList count="0">
                                       </ClassificationNodeChildList>
                                     </ClassificationNode>
                                     <ClassificationNode>
@@ -113021,12 +112949,12 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="565">
-                            <OrphaCode>744</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=744</ExpertLink>
-                            <Name lang="en">Proteus syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
+                          <Disorder id="10867">
+                            <OrphaCode>65285</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=65285</ExpertLink>
+                            <Name lang="en">Lhermitte-Duclos disease</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -116182,7 +116110,7 @@
                       <Disorder id="2233">
                         <OrphaCode>2407</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-                        <Name lang="en">LOC syndrome</Name>
+                        <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -116422,7 +116350,7 @@
                       <Disorder id="21219">
                         <OrphaCode>306504</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306504</ExpertLink>
-                        <Name lang="en">Junctional epidermolysis bullosa with respiratory and renal involvement</Name>
+                        <Name lang="en">Interstitial lung disease-nephrotic syndrome-epidermolysis bullosa syndrome</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -117068,36 +116996,12 @@
                           <Disorder id="3423">
                             <OrphaCode>2745</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-                            <Name lang="en">Opitz G/BBB syndrome</Name>
+                            <Name lang="en">Opitz GBBB syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="21237">
-                                <OrphaCode>306588</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-                                <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-                                <DisorderType id="21443">
-                                  <Name lang="en">Etiological subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21238">
-                                <OrphaCode>306597</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-                                <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-                                <DisorderType id="21443">
-                                  <Name lang="en">Etiological subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -120125,7 +120029,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="11">
+                  <ClassificationNodeChildList count="13">
                     <ClassificationNode>
                       <Disorder id="25308">
                         <OrphaCode>485382</OrphaCode>
@@ -122094,6 +121998,30 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="11640">
+                        <OrphaCode>85173</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85173</ExpertLink>
+                        <Name lang="en">IMAGe syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="12657">
+                        <OrphaCode>95702</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95702</ExpertLink>
+                        <Name lang="en">X-linked adrenal hypoplasia congenita due to a NR0B1 point mutation</Name>
+                        <DisorderType id="21394">
+                          <Name lang="en">Disease</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -141306,7 +141234,7 @@
                               <Disorder id="1686">
                                 <OrphaCode>1519</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1519</ExpertLink>
-                                <Name lang="en">Hypertelorism, Teebi type</Name>
+                                <Name lang="en">SPECC1L-related hypertelorism syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -146633,12 +146561,12 @@
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
-                          <Disorder id="565">
-                            <OrphaCode>744</OrphaCode>
-                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=744</ExpertLink>
-                            <Name lang="en">Proteus syndrome</Name>
-                            <DisorderType id="21401">
-                              <Name lang="en">Malformation syndrome</Name>
+                          <Disorder id="10867">
+                            <OrphaCode>65285</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=65285</ExpertLink>
+                            <Name lang="en">Lhermitte-Duclos disease</Name>
+                            <DisorderType id="21394">
+                              <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -148904,7 +148832,7 @@
                           <Disorder id="2233">
                             <OrphaCode>2407</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-                            <Name lang="en">LOC syndrome</Name>
+                            <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -148916,7 +148844,7 @@
                           <Disorder id="11429">
                             <OrphaCode>79403</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79403</ExpertLink>
-                            <Name lang="en">Junctional epidermolysis bullosa-pyloric atresia syndrome</Name>
+                            <Name lang="en">Junctional epidermolysis bullosa with pyloric atresia</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -148928,7 +148856,7 @@
                           <Disorder id="11430">
                             <OrphaCode>79404</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79404</ExpertLink>
-                            <Name lang="en">Junctional epidermolysis bullosa, generalized severe</Name>
+                            <Name lang="en">Severe generalized junctional epidermolysis bullosa</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -148964,7 +148892,7 @@
                           <Disorder id="19640">
                             <OrphaCode>251393</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251393</ExpertLink>
-                            <Name lang="en">Localized junctional epidermolysis bullosa, non-Herlitz type</Name>
+                            <Name lang="en">Localized junctional epidermolysis bullosa</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -148988,7 +148916,7 @@
                           <Disorder id="21219">
                             <OrphaCode>306504</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306504</ExpertLink>
-                            <Name lang="en">Junctional epidermolysis bullosa with respiratory and renal involvement</Name>
+                            <Name lang="en">Interstitial lung disease-nephrotic syndrome-epidermolysis bullosa syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -149000,7 +148928,7 @@
                           <Disorder id="11428">
                             <OrphaCode>79402</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79402</ExpertLink>
-                            <Name lang="en">Junctional epidermolysis bullosa, generalized intermediate</Name>
+                            <Name lang="en">Intermediate generalized junctional epidermolysis bullosa</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -149024,7 +148952,7 @@
                           <Disorder id="11434">
                             <OrphaCode>79408</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79408</ExpertLink>
-                            <Name lang="en">Severe generalized recessive dystrophic epidermolysis bullosa</Name>
+                            <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -149036,7 +148964,7 @@
                           <Disorder id="11435">
                             <OrphaCode>79409</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79409</ExpertLink>
-                            <Name lang="en">Recessive dystrophic epidermolysis bullosa inversa</Name>
+                            <Name lang="en">Dystrophic epidermolysis bullosa inversa</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -149058,7 +148986,7 @@
                               <Disorder id="11436">
                                 <OrphaCode>79410</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79410</ExpertLink>
-                                <Name lang="en">Pretibial dystrophic epidermolysis bullosa</Name>
+                                <Name lang="en">Localized dystrophic epidermolysis bullosa, pretibial form</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -149070,7 +148998,7 @@
                               <Disorder id="17233">
                                 <OrphaCode>158673</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158673</ExpertLink>
-                                <Name lang="en">Acral dystrophic epidermolysis bullosa</Name>
+                                <Name lang="en">Localized dystrophic epidermolysis bullosa, acral form</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -149082,7 +149010,7 @@
                               <Disorder id="17234">
                                 <OrphaCode>158676</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158676</ExpertLink>
-                                <Name lang="en">Dominant dystrophic epidermolysis bullosa, nails only</Name>
+                                <Name lang="en">Localized dystrophic epidermolysis bullosa, nails only</Name>
                                 <DisorderType id="21450">
                                   <Name lang="en">Clinical subtype</Name>
                                 </DisorderType>
@@ -149096,7 +149024,7 @@
                           <Disorder id="11437">
                             <OrphaCode>79411</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79411</ExpertLink>
-                            <Name lang="en">Transient bullous dermolysis of the newborn</Name>
+                            <Name lang="en">Self-improving dystrophic epidermolysis bullosa</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -149108,7 +149036,7 @@
                           <Disorder id="11907">
                             <OrphaCode>89842</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89842</ExpertLink>
-                            <Name lang="en">Recessive dystrophic epidermolysis bullosa, generalized intermediate</Name>
+                            <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, intermediate form</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -149132,7 +149060,7 @@
                           <Disorder id="19201">
                             <OrphaCode>231568</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231568</ExpertLink>
-                            <Name lang="en">Generalized dominant dystrophic epidermolysis bullosa</Name>
+                            <Name lang="en">Autosomal dominant generalized dystrophic epidermolysis bullosa</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -153203,7 +153131,7 @@
                                   <Name lang="en">Clinical group</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="10">
+                              <ClassificationNodeChildList count="9">
                                 <ClassificationNode>
                                   <Disorder id="2334">
                                     <OrphaCode>2524</OrphaCode>
@@ -153252,18 +153180,6 @@
                                   <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="17609">
-                                    <OrphaCode>166068</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166068</ExpertLink>
-                                    <Name lang="en">Pontocerebellar hypoplasia type 5</Name>
-                                    <DisorderType id="21401">
-                                      <Name lang="en">Malformation syndrome</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
                                 <ClassificationNode>
                                   <Disorder id="17610">
                                     <OrphaCode>166073</OrphaCode>
@@ -159176,36 +159092,12 @@
                           <Disorder id="3423">
                             <OrphaCode>2745</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-                            <Name lang="en">Opitz G/BBB syndrome</Name>
+                            <Name lang="en">Opitz GBBB syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="2">
-                            <ClassificationNode>
-                              <Disorder id="21237">
-                                <OrphaCode>306588</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-                                <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-                                <DisorderType id="21443">
-                                  <Name lang="en">Etiological subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
-                            <ClassificationNode>
-                              <Disorder id="21238">
-                                <OrphaCode>306597</OrphaCode>
-                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-                                <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-                                <DisorderType id="21443">
-                                  <Name lang="en">Etiological subtype</Name>
-                                </DisorderType>
-                              </Disorder>
-                              <ClassificationNodeChildList count="0">
-                              </ClassificationNodeChildList>
-                            </ClassificationNode>
+                          <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -159727,8 +159619,8 @@
                                 <OrphaCode>268261</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268261</ExpertLink>
                                 <Name lang="en">DYRK1A-related intellectual disability syndrome due to 21q22.13q22.2 microdeletion</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -159739,8 +159631,8 @@
                                 <OrphaCode>464311</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464311</ExpertLink>
                                 <Name lang="en">Intellectual disability syndrome due to a DYRK1A point mutation</Name>
-                                <DisorderType id="21450">
-                                  <Name lang="en">Clinical subtype</Name>
+                                <DisorderType id="21443">
+                                  <Name lang="en">Etiological subtype</Name>
                                 </DisorderType>
                               </Disorder>
                               <ClassificationNodeChildList count="0">
@@ -163388,7 +163280,7 @@
                           <Disorder id="1686">
                             <OrphaCode>1519</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1519</ExpertLink>
-                            <Name lang="en">Hypertelorism, Teebi type</Name>
+                            <Name lang="en">SPECC1L-related hypertelorism syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -172074,36 +171966,12 @@
                               <Disorder id="3423">
                                 <OrphaCode>2745</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-                                <Name lang="en">Opitz G/BBB syndrome</Name>
+                                <Name lang="en">Opitz GBBB syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
                               </Disorder>
-                              <ClassificationNodeChildList count="2">
-                                <ClassificationNode>
-                                  <Disorder id="21237">
-                                    <OrphaCode>306588</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-                                    <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-                                    <DisorderType id="21443">
-                                      <Name lang="en">Etiological subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
-                                <ClassificationNode>
-                                  <Disorder id="21238">
-                                    <OrphaCode>306597</OrphaCode>
-                                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-                                    <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-                                    <DisorderType id="21443">
-                                      <Name lang="en">Etiological subtype</Name>
-                                    </DisorderType>
-                                  </Disorder>
-                                  <ClassificationNodeChildList count="0">
-                                  </ClassificationNodeChildList>
-                                </ClassificationNode>
+                              <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
                             <ClassificationNode>
@@ -173252,7 +173120,7 @@
                           <Disorder id="2233">
                             <OrphaCode>2407</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-                            <Name lang="en">LOC syndrome</Name>
+                            <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -174736,36 +174604,12 @@
                                   <Disorder id="3423">
                                     <OrphaCode>2745</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-                                    <Name lang="en">Opitz G/BBB syndrome</Name>
+                                    <Name lang="en">Opitz GBBB syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
                                   </Disorder>
-                                  <ClassificationNodeChildList count="2">
-                                    <ClassificationNode>
-                                      <Disorder id="21237">
-                                        <OrphaCode>306588</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-                                        <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
-                                    <ClassificationNode>
-                                      <Disorder id="21238">
-                                        <OrphaCode>306597</OrphaCode>
-                                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-                                        <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-                                        <DisorderType id="21443">
-                                          <Name lang="en">Etiological subtype</Name>
-                                        </DisorderType>
-                                      </Disorder>
-                                      <ClassificationNodeChildList count="0">
-                                      </ClassificationNodeChildList>
-                                    </ClassificationNode>
+                                  <ClassificationNodeChildList count="0">
                                   </ClassificationNodeChildList>
                                 </ClassificationNode>
                                 <ClassificationNode>
@@ -193348,7 +193192,7 @@
                                   <Disorder id="1686">
                                     <OrphaCode>1519</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1519</ExpertLink>
-                                    <Name lang="en">Hypertelorism, Teebi type</Name>
+                                    <Name lang="en">SPECC1L-related hypertelorism syndrome</Name>
                                     <DisorderType id="21401">
                                       <Name lang="en">Malformation syndrome</Name>
                                     </DisorderType>
