commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml
old mode 100644
new mode 100755
index 6aee32f..60554bc
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_181.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:34:09" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:34:49" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -416,7 +416,7 @@
                           <Disorder id="23427">
                             <OrphaCode>442835</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                            <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                            <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -452,7 +452,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">West syndrome</Name>
+                            <Name lang="en">Infantile spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -680,7 +680,7 @@
                           <Disorder id="23427">
                             <OrphaCode>442835</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-                            <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+                            <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -728,7 +728,7 @@
                           <Disorder id="891">
                             <OrphaCode>1942</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1942</ExpertLink>
-                            <Name lang="en">Myoclonic-astastic epilepsy</Name>
+                            <Name lang="en">Myoclonic-astatic epilepsy</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -3164,7 +3164,7 @@
                           <Disorder id="894">
                             <OrphaCode>3451</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                            <Name lang="en">West syndrome</Name>
+                            <Name lang="en">Infantile spasms syndrome</Name>
                             <DisorderType id="21422">
                               <Name lang="en">Clinical syndrome</Name>
                             </DisorderType>
@@ -3675,7 +3675,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="29875">
+                            <OrphaCode>598164</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="19853">
+                            <OrphaCode>261144</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -13969,7 +13993,31 @@
                               <Name lang="en">Disease</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="0">
+                          <ClassificationNodeChildList count="2">
+                            <ClassificationNode>
+                              <Disorder id="29875">
+                                <OrphaCode>598164</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                                <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="19853">
+                                <OrphaCode>261144</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                                <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                                <DisorderType id="21450">
+                                  <Name lang="en">Clinical subtype</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
@@ -14134,7 +14182,7 @@
                       <Disorder id="29581">
                         <OrphaCode>592888</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592888</ExpertLink>
-                        <Name lang="en">Optic neuritis with anti-MOG antibodies</Name>
+                        <Name lang="en">Isolated optic neuritis with anti-MOG antibodies</Name>
                         <DisorderType id="21450">
                           <Name lang="en">Clinical subtype</Name>
                         </DisorderType>
@@ -17518,7 +17566,7 @@
                       <Disorder id="11922">
                         <OrphaCode>90020</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90020</ExpertLink>
-                        <Name lang="en">Amyotrophic lateral sclerosis-parkinsonism-dementia complex</Name>
+                        <Name lang="en">Parkinson-dementia complex of Guam</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -34138,7 +34186,7 @@
                       <Disorder id="894">
                         <OrphaCode>3451</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-                        <Name lang="en">West syndrome</Name>
+                        <Name lang="en">Infantile spasms syndrome</Name>
                         <DisorderType id="21422">
                           <Name lang="en">Clinical syndrome</Name>
                         </DisorderType>
@@ -37496,7 +37544,7 @@
                           <Disorder id="972">
                             <OrphaCode>776</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=776</ExpertLink>
-                            <Name lang="en">X-linked intellectual disability with marfanoid habitus</Name>
+                            <Name lang="en">Lujan-Fryns syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -39575,8 +39623,8 @@
                         <OrphaCode>98791</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98791</ExpertLink>
                         <Name lang="en">Alpha-thalassemia-intellectual disability syndrome linked to chromosome 16</Name>
-                        <DisorderType id="21394">
-                          <Name lang="en">Disease</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -41180,7 +41228,7 @@
                           <Disorder id="24223">
                             <OrphaCode>468631</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                            <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                            <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -42543,7 +42591,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="29875">
+                            <OrphaCode>598164</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="19853">
+                            <OrphaCode>261144</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -45680,7 +45752,7 @@
                           <Disorder id="11922">
                             <OrphaCode>90020</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90020</ExpertLink>
-                            <Name lang="en">Amyotrophic lateral sclerosis-parkinsonism-dementia complex</Name>
+                            <Name lang="en">Parkinson-dementia complex of Guam</Name>
                             <DisorderType id="21394">
                               <Name lang="en">Disease</Name>
                             </DisorderType>
@@ -49611,7 +49683,31 @@
                           <Name lang="en">Disease</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="0">
+                      <ClassificationNodeChildList count="2">
+                        <ClassificationNode>
+                          <Disorder id="29875">
+                            <OrphaCode>598164</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598164</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to intragenic alteration</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="19853">
+                            <OrphaCode>261144</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
+                            <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+                            <DisorderType id="21450">
+                              <Name lang="en">Clinical subtype</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                     <ClassificationNode>
@@ -53322,7 +53418,7 @@
                               <Disorder id="24223">
                                 <OrphaCode>468631</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+                                <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
