commit bbf94d552a3c459862632b82d2e2166d94fb6b4e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_195.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_195.xml
index 5888fa6..7a6ce72 100644
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_195.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_195.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:29:53" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:32:46" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -2749,7 +2749,7 @@
                               <Name lang="en">Clinical group</Name>
                             </DisorderType>
                           </Disorder>
-                          <ClassificationNodeChildList count="9">
+                          <ClassificationNodeChildList count="11">
                             <ClassificationNode>
                               <Disorder id="19269">
                                 <OrphaCode>238505</OrphaCode>
@@ -2858,6 +2858,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="20165">
+                                <OrphaCode>268114</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268114</ExpertLink>
+                                <Name lang="en">RAS-associated autoimmune leukoproliferative disease</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                             <ClassificationNode>
                               <Disorder id="27968">
                                 <OrphaCode>538958</OrphaCode>
@@ -2882,6 +2894,18 @@
                               <ClassificationNodeChildList count="0">
                               </ClassificationNodeChildList>
                             </ClassificationNode>
+                            <ClassificationNode>
+                              <Disorder id="27969">
+                                <OrphaCode>538963</OrphaCode>
+                                <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=538963</ExpertLink>
+                                <Name lang="en">Combined immunodeficiency due to ITK deficiency</Name>
+                                <DisorderType id="21394">
+                                  <Name lang="en">Disease</Name>
+                                </DisorderType>
+                              </Disorder>
+                              <ClassificationNodeChildList count="0">
+                              </ClassificationNodeChildList>
+                            </ClassificationNode>
                           </ClassificationNodeChildList>
                         </ClassificationNode>
                         <ClassificationNode>
