commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml
old mode 100644
new mode 100755
index 138d200..c5bc949
--- a/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
+++ b/Rare diseases and classifications/Classifications of rare diseases/en_product3_147.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:31:19" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:31:53" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -617,7 +617,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="63">
+                  <ClassificationNodeChildList count="64">
                     <ClassificationNode>
                       <Disorder id="44">
                         <OrphaCode>881</OrphaCode>
@@ -705,8 +705,8 @@
                             <OrphaCode>353277</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -717,8 +717,8 @@
                             <OrphaCode>353281</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -729,8 +729,8 @@
                             <OrphaCode>353284</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -1566,6 +1566,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29566">
+                        <OrphaCode>592574</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                        <Name lang="en">Menke-Hennekam syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                     <ClassificationNode>
                       <Disorder id="27331">
                         <OrphaCode>521438</OrphaCode>
@@ -4277,7 +4289,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="41">
+                  <ClassificationNodeChildList count="42">
                     <ClassificationNode>
                       <Disorder id="25408">
                         <OrphaCode>488618</OrphaCode>
@@ -4998,6 +5010,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29565">
+                        <OrphaCode>592570</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                        <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
               </ClassificationNodeChildList>
@@ -11922,7 +11946,7 @@
                               <Disorder id="21690">
                                 <OrphaCode>319671</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-                                <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+                                <Name lang="en">Alazami syndrome</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -12111,7 +12135,7 @@
                           <Name lang="en">Category</Name>
                         </DisorderType>
                       </Disorder>
-                      <ClassificationNodeChildList count="41">
+                      <ClassificationNodeChildList count="42">
                         <ClassificationNode>
                           <Disorder id="27595">
                             <OrphaCode>529574</OrphaCode>
@@ -12604,6 +12628,18 @@
                           <ClassificationNodeChildList count="0">
                           </ClassificationNodeChildList>
                         </ClassificationNode>
+                        <ClassificationNode>
+                          <Disorder id="29565">
+                            <OrphaCode>592570</OrphaCode>
+                            <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                            <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                            <DisorderType id="21401">
+                              <Name lang="en">Malformation syndrome</Name>
+                            </DisorderType>
+                          </Disorder>
+                          <ClassificationNodeChildList count="0">
+                          </ClassificationNodeChildList>
+                        </ClassificationNode>
                       </ClassificationNodeChildList>
                     </ClassificationNode>
                   </ClassificationNodeChildList>
@@ -17193,8 +17229,8 @@
                             <OrphaCode>353277</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -17205,8 +17241,8 @@
                             <OrphaCode>353281</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -17217,8 +17253,8 @@
                             <OrphaCode>353284</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -27603,8 +27639,8 @@
                                         <OrphaCode>353277</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -27615,8 +27651,8 @@
                                         <OrphaCode>353281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -27627,8 +27663,8 @@
                                         <OrphaCode>353284</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -30847,7 +30883,7 @@
                   <Name lang="en">Category</Name>
                 </DisorderType>
               </Disorder>
-              <ClassificationNodeChildList count="48">
+              <ClassificationNodeChildList count="49">
                 <ClassificationNode>
                   <Disorder id="145">
                     <OrphaCode>904</OrphaCode>
@@ -30875,8 +30911,8 @@
                         <OrphaCode>353277</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                         <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
+                        <DisorderType id="21443">
+                          <Name lang="en">Etiological subtype</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -30887,8 +30923,8 @@
                         <OrphaCode>353281</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
+                        <DisorderType id="21443">
+                          <Name lang="en">Etiological subtype</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -30899,8 +30935,8 @@
                         <OrphaCode>353284</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                         <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                        <DisorderType id="21450">
-                          <Name lang="en">Clinical subtype</Name>
+                        <DisorderType id="21443">
+                          <Name lang="en">Etiological subtype</Name>
                         </DisorderType>
                       </Disorder>
                       <ClassificationNodeChildList count="0">
@@ -31688,6 +31724,18 @@
                   <ClassificationNodeChildList count="0">
                   </ClassificationNodeChildList>
                 </ClassificationNode>
+                <ClassificationNode>
+                  <Disorder id="29566">
+                    <OrphaCode>592574</OrphaCode>
+                    <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                    <Name lang="en">Menke-Hennekam syndrome</Name>
+                    <DisorderType id="21401">
+                      <Name lang="en">Malformation syndrome</Name>
+                    </DisorderType>
+                  </Disorder>
+                  <ClassificationNodeChildList count="0">
+                  </ClassificationNodeChildList>
+                </ClassificationNode>
               </ClassificationNodeChildList>
             </ClassificationNode>
             <ClassificationNode>
@@ -32649,8 +32697,8 @@
                             <OrphaCode>353277</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -32661,8 +32709,8 @@
                             <OrphaCode>353281</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -32673,8 +32721,8 @@
                             <OrphaCode>353284</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -46647,8 +46695,8 @@
                                         <OrphaCode>353281</OrphaCode>
                                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                         <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                        <DisorderType id="21450">
-                                          <Name lang="en">Clinical subtype</Name>
+                                        <DisorderType id="21443">
+                                          <Name lang="en">Etiological subtype</Name>
                                         </DisorderType>
                                       </Disorder>
                                       <ClassificationNodeChildList count="0">
@@ -48701,7 +48749,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="405">
+                  <ClassificationNodeChildList count="407">
                     <ClassificationNode>
                       <Disorder id="24010">
                         <OrphaCode>464282</OrphaCode>
@@ -50396,7 +50444,7 @@
                           <Disorder id="21690">
                             <OrphaCode>319671</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-                            <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+                            <Name lang="en">Alazami syndrome</Name>
                             <DisorderType id="21401">
                               <Name lang="en">Malformation syndrome</Name>
                             </DisorderType>
@@ -50590,7 +50638,7 @@
                       <Disorder id="22327">
                         <OrphaCode>363611</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-                        <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+                        <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
                         <DisorderType id="21394">
                           <Name lang="en">Disease</Name>
                         </DisorderType>
@@ -52510,7 +52558,7 @@
                       <Disorder id="26002">
                         <OrphaCode>500163</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=500163</ExpertLink>
-                        <Name lang="en">SIN3A-related intellectual disability syndrome</Name>
+                        <Name lang="en">Witteveen-Kolk syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -52834,7 +52882,7 @@
                       <Disorder id="24042">
                         <OrphaCode>464738</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-                        <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+                        <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -52893,8 +52941,8 @@
                             <OrphaCode>353277</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -52905,8 +52953,8 @@
                             <OrphaCode>353281</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -52917,8 +52965,8 @@
                             <OrphaCode>353284</OrphaCode>
                             <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                             <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                            <DisorderType id="21450">
-                              <Name lang="en">Clinical subtype</Name>
+                            <DisorderType id="21443">
+                              <Name lang="en">Etiological subtype</Name>
                             </DisorderType>
                           </Disorder>
                           <ClassificationNodeChildList count="0">
@@ -54426,6 +54474,30 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29565">
+                        <OrphaCode>592570</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                        <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29566">
+                        <OrphaCode>592574</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+                        <Name lang="en">Menke-Hennekam syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
@@ -56590,7 +56662,7 @@
                       <Disorder id="421">
                         <OrphaCode>1146</OrphaCode>
                         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-                        <Name lang="en">Digitotalar dysmorphism</Name>
+                        <Name lang="en">Distal arthrogryposis type 1</Name>
                         <DisorderType id="21401">
                           <Name lang="en">Malformation syndrome</Name>
                         </DisorderType>
@@ -57821,7 +57893,7 @@
                       <Name lang="en">Category</Name>
                     </DisorderType>
                   </Disorder>
-                  <ClassificationNodeChildList count="14">
+                  <ClassificationNodeChildList count="15">
                     <ClassificationNode>
                       <Disorder id="1548">
                         <OrphaCode>1325</OrphaCode>
@@ -58409,8 +58481,8 @@
                                     <OrphaCode>353277</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
                                     <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -58421,8 +58493,8 @@
                                     <OrphaCode>353281</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
                                     <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -58433,8 +58505,8 @@
                                     <OrphaCode>353284</OrphaCode>
                                     <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
                                     <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-                                    <DisorderType id="21450">
-                                      <Name lang="en">Clinical subtype</Name>
+                                    <DisorderType id="21443">
+                                      <Name lang="en">Etiological subtype</Name>
                                     </DisorderType>
                                   </Disorder>
                                   <ClassificationNodeChildList count="0">
@@ -58842,7 +58914,7 @@
                               <Disorder id="421">
                                 <OrphaCode>1146</OrphaCode>
                                 <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-                                <Name lang="en">Digitotalar dysmorphism</Name>
+                                <Name lang="en">Distal arthrogryposis type 1</Name>
                                 <DisorderType id="21401">
                                   <Name lang="en">Malformation syndrome</Name>
                                 </DisorderType>
@@ -61146,6 +61218,18 @@
                       <ClassificationNodeChildList count="0">
                       </ClassificationNodeChildList>
                     </ClassificationNode>
+                    <ClassificationNode>
+                      <Disorder id="29565">
+                        <OrphaCode>592570</OrphaCode>
+                        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+                        <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+                        <DisorderType id="21401">
+                          <Name lang="en">Malformation syndrome</Name>
+                        </DisorderType>
+                      </Disorder>
+                      <ClassificationNodeChildList count="0">
+                      </ClassificationNodeChildList>
+                    </ClassificationNode>
                   </ClassificationNodeChildList>
                 </ClassificationNode>
                 <ClassificationNode>
