commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and functional consequences/cs_funct_consequences.xml b/Rare diseases and functional consequences/cs_funct_consequences.xml
index ed015a4..ff3c1e3 100644
--- a/Rare diseases and functional consequences/cs_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/cs_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<JDBOR date="2022-04-01 04:11:36" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:11:00" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -338540,7 +338540,7 @@ cs">Sám se umýt</Name>
       <Disorder id="244">
         <OrphaCode>207</OrphaCode>
         <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=207</ExpertLink>
-        <Name lang="en">Crouzon disease</Name>
+        <Name lang="en">Crouzon syndrome</Name>
         <DisorderType id="21401">
           <Name lang="cs">Malformační syndrom</Name>
         </DisorderType>
@@ -473081,7 +473081,7 @@ cs">Plnění odborných úkolů</Name>
       <Disorder id="1544">
         <OrphaCode>1321</OrphaCode>
         <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=1321</ExpertLink>
-        <Name lang="en">Camptodactyly-fibrous tissue hyperplasia-skeletal dysplasia syndrome</Name>
+        <Name lang="en">Camptodactyly-fibrous tissue hyperplasia-skeletal anomalies syndrome</Name>
         <DisorderType id="21401">
           <Name lang="cs">Malformační syndrom</Name>
         </DisorderType>
@@ -584234,7 +584234,7 @@ cs">Vykonávání běžných denních povinností</Name>
       <Disorder id="2497">
         <OrphaCode>661</OrphaCode>
         <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=661</ExpertLink>
-        <Name lang="en">Ondine syndrome</Name>
+        <Name lang="en">Congenital central hypoventilation syndrome</Name>
         <DisorderType id="21394">
           <Name lang="cs">Onemocnění</Name>
         </DisorderType>
