commit bbf94d552a3c459862632b82d2e2166d94fb6b4e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Rare diseases and functional consequences/de_funct_consequences.xml b/Rare diseases and functional consequences/de_funct_consequences.xml
index 4feb1c6..6d688b1 100644
--- a/Rare diseases and functional consequences/de_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/de_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:06:32" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:06:36" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -20435,7 +20435,7 @@
       <Disorder id="3713">
         <OrphaCode>2542</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2542</ExpertLink>
-        <Name lang="de">Anophthalmie/Mikrophthalmie-Syndrom, isoliertes</Name>
+        <Name lang="de">Mikrophthalmie-Anophthalmie-Kolobom, isoliert</Name>
         <DisorderType id="21436">
           <Name lang="de">Phenom-Gruppe</Name>
         </DisorderType>
@@ -40794,7 +40794,7 @@
       <Disorder id="12914">
         <OrphaCode>97363</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=97363</ExpertLink>
-        <Name lang="de">Nierendysplasie, multizystische, unilaterale From</Name>
+        <Name lang="de">Nierendysplasie, multizystische, unilaterale Form</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
@@ -156931,7 +156931,7 @@
       <Disorder id="12466">
         <OrphaCode>93685</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93685</ExpertLink>
-        <Name lang="de">Castleman-Krankheit, lokalisierte</Name>
+        <Name lang="de">Castleman-Krankheit, unizentrische</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
@@ -207213,7 +207213,7 @@
       <Disorder id="18313">
         <OrphaCode>183713</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=183713</ExpertLink>
-        <Name lang="de">Pyogene Bakterieninfektion durch MyD88-Mangel</Name>
+        <Name lang="de">Suszeptibilität für Bakterieninfektion durch TLR Signalweg-Defizienz</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -213135,7 +213135,7 @@
       <Disorder id="10999">
         <OrphaCode>71277</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=71277</ExpertLink>
-        <Name lang="de">Enzephalopathie durch GLUT1-Defekt</Name>
+        <Name lang="de">Klassisches Glukosetransporter-Typ-1-Mangel-Syndrom</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -469380,7 +469380,7 @@
       <Disorder id="1007">
         <OrphaCode>528</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=528</ExpertLink>
-        <Name lang="de">Lipodystrophie Typ Berardinelli</Name>
+        <Name lang="de">Lipodystrophie, kongenitale, Typ Berardinelli</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -529782,7 +529782,7 @@
       <Disorder id="1001">
         <OrphaCode>136</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=136</ExpertLink>
-        <Name lang="de">CADASIL</Name>
+        <Name lang="de">Zerebrale autosomal-dominante Arteriopathie mit subkortikalen Infarkten und Leukenzephalopathie</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
