commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Rare diseases and functional consequences/de_funct_consequences.xml b/Rare diseases and functional consequences/de_funct_consequences.xml
old mode 100644
new mode 100755
index 6d688b1..e0dbbcc
--- a/Rare diseases and functional consequences/de_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/de_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:06:36" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:06:43" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="911">
+  <DisorderDisabilityRelevanceList count="910">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -163656,7 +163656,7 @@
       <Disorder id="10738">
         <OrphaCode>53696</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=53696</ExpertLink>
-        <Name lang="de">Arthrogrypose, letale - Verlust von Vorderhornzellen</Name>
+        <Name lang="de">Arthrogrypose-anteriore Hornzellkrankheit-Syndrom</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -163908,7 +163908,7 @@
       <Disorder id="413">
         <OrphaCode>1046</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1046</ExpertLink>
-        <Name lang="de">Anämie, hämolytische letale - Genitalfehlbildungen</Name>
+        <Name lang="de">Letale hämolytischeAnämie-Genitalfehlbildungen-Syndrom</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -199583,7 +199583,7 @@
       <Disorder id="784">
         <OrphaCode>3002</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3002</ExpertLink>
-        <Name lang="de">Purpura, immune thrombozytopenische</Name>
+        <Name lang="de">Immunthrombozytopenie, immune</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -213107,7 +213107,7 @@
       <Disorder id="21692">
         <OrphaCode>319678</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=319678</ExpertLink>
-        <Name lang="de">Enzephalopathie - hypertrophe Kardiomyopathie - renal-tubuläre Störung</Name>
+        <Name lang="de">Enzephalopathie mit hypertropher Kardiomyopathie und renal-tubulärer Störung</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -255879,7 +255879,7 @@
       <Disorder id="7037">
         <OrphaCode>225</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=225</ExpertLink>
-        <Name lang="de">Diabetes - Schwerhörigkeit, maternal vererbt</Name>
+        <Name lang="de">Maternal vererbter Diabetes mit Schwerhörigkeit</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -273858,7 +273858,7 @@
       <Disorder id="2729">
         <OrphaCode>3035</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3035</ExpertLink>
-        <Name lang="de">Wachstumsretardierung - Hydrozephalus - Lungenhypoplasie</Name>
+        <Name lang="de">Wachstumsretardierung-Hydrozephalus-Lungenhypoplasie-Syndrom</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -289840,34 +289840,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="4145">
-      <Disorder id="20880">
-        <OrphaCode>293848</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=293848</ExpertLink>
-        <Name lang="de">Frontotemporale Demenz, rechtstemporale Variante</Name>
-        <DisorderType id="21394">
-          <Name lang="de">Krankheit</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="de">Störung</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2019-02-04 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="de">Validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="de">Nicht anwendbar</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27306">
-        <Name lang="de">Hypervariable Funktionsweise</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="4097">
       <Disorder id="357">
         <OrphaCode>370</OrphaCode>
@@ -309175,7 +309147,7 @@
       <Disorder id="2459">
         <OrphaCode>2697</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2697</ExpertLink>
-        <Name lang="de">Arthrogrypose - Nierenfunktionsstörung - Cholestase</Name>
+        <Name lang="de">Arthrogrypose-Nierenfunktionsstörung-Cholestase-Syndrom</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -352556,7 +352528,7 @@
       <Disorder id="18946">
         <OrphaCode>221126</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=221126</ExpertLink>
-        <Name lang="de">Fowler-Syndrom</Name>
+        <Name lang="de">Fowler-Vaskulopathie</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -376193,7 +376165,7 @@
       <Disorder id="20508">
         <OrphaCode>280840</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280840</ExpertLink>
-        <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 2</Name>
+        <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 2</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
@@ -376221,7 +376193,7 @@
       <Disorder id="20509">
         <OrphaCode>280847</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280847</ExpertLink>
-        <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 3</Name>
+        <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 3</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
@@ -376249,7 +376221,7 @@
       <Disorder id="20510">
         <OrphaCode>280854</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280854</ExpertLink>
-        <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 4</Name>
+        <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 4</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
@@ -376277,7 +376249,7 @@
       <Disorder id="20506">
         <OrphaCode>280827</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280827</ExpertLink>
-        <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 0</Name>
+        <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 0</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
@@ -436290,7 +436262,7 @@
       <Disorder id="13866">
         <OrphaCode>98849</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=98849</ExpertLink>
-        <Name lang="de">Mastozytose, systemische, assoziiert mit klonaler hämatologischer nicht-Mastzellen-Krankheit</Name>
+        <Name lang="de">Mastozytose, systemische, mit assoziierter hämatologischer Neoplasie</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -445706,7 +445678,7 @@
       <Disorder id="3379">
         <OrphaCode>1561</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1561</ExpertLink>
-        <Name lang="de">Cytochrom-C-Oxidase-Mangel, infantile fatale Form</Name>
+        <Name lang="de">Cytochrom-c-Oxidase-Mangel, infantile fatale Form</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -498071,7 +498043,7 @@
       <Disorder id="17146">
         <OrphaCode>157798</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=157798</ExpertLink>
-        <Name lang="de">Polyposis-Syndrom, hyperplastisches</Name>
+        <Name lang="de">Serratiertes Polyposissyndrom</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -502939,7 +502911,7 @@
       <Disorder id="3759">
         <OrphaCode>319</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=319</ExpertLink>
-        <Name lang="de">Ewing-Sarkom</Name>
+        <Name lang="de">Ewing-Sarkom, skelettales</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -570678,7 +570650,7 @@
       <Disorder id="25934">
         <OrphaCode>498693</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=498693</ExpertLink>
-        <Name lang="de">Arthrogryposis multiplex congenita-Syndrom, autosomal-rezessives, nicht-tödliches, MYBPC1-assoziiertes</Name>
+        <Name lang="de">Arthrogryposis multiplex congenita-Syndrom, autosomal-rezessives, nicht-letales, MYBPC1-assoziiertes</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
