commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Rare diseases and functional consequences/de_funct_consequences.xml b/Rare diseases and functional consequences/de_funct_consequences.xml
old mode 100644
new mode 100755
index e0dbbcc..fe82f83
--- a/Rare diseases and functional consequences/de_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/de_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:06:43" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:06:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="910">
+  <DisorderDisabilityRelevanceList count="911">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -154198,6 +154198,456 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
+    <DisorderDisabilityRelevance id="2121">
+      <Disorder id="101">
+        <OrphaCode>1941</OrphaCode>
+        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1941</ExpertLink>
+        <Name lang="de">Absencen-Epilepsie, juvenile</Name>
+        <DisorderType id="21394">
+          <Name lang="de">Krankheit</Name>
+        </DisorderType>
+        <DisorderGroup id="36547">
+          <Name lang="de">Störung</Name>
+        </DisorderGroup>
+        <DisabilityDisorderAssociationList count="26">
+          <DisabilityDisorderAssociation id="169415">
+            <Disability id="16">
+              <Name lang="de">Aufmerksamkeit fokussieren</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169416">
+            <Disability id="16">
+              <Name lang="de">Aufmerksamkeit fokussieren</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27236">
+              <Name lang="de">Moderate</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169417">
+            <Disability id="34">
+              <Name lang="de">Teilnahme an Konversationen</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169418">
+            <Disability id="61">
+              <Name lang="de">Erhaltung der Gesundheit (Ernährung, Medikation, Prävention, physischer Komfort, physisches und mentales Wohlbefinden)</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169419">
+            <Disability id="68">
+              <Name lang="de">Umgang mit Stress, Verantwortung, Notfallsituationen und Erhaltung der eigenen Sicherheit</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27236">
+              <Name lang="de">Moderate</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169420">
+            <Disability id="69">
+              <Name lang="de">Umgang mit Emotionen und Gemütslage</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169421">
+            <Disability id="77">
+              <Name lang="de">Ein Fahrzeug fahren</Name>
+            </Disability>
+            <FrequenceDisability id="27201">
+              <Name lang="de">Sehr häufig</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27264">
+              <Name lang="de">Komplett</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169422">
+            <Disability id="87">
+              <Name lang="de">Einkaufen</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169423">
+            <Disability id="88">
+              <Name lang="de">Kochen/Mahlzeiten vorbereiten</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169424">
+            <Disability id="89">
+              <Name lang="de">Hausarbeit tätigen</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169425">
+            <Disability id="90">
+              <Name lang="de">Fürsorge/Hilfe für andere</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169426">
+            <Disability id="92">
+              <Name lang="de">Teilnahme Primärstufe</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169427">
+            <Disability id="93">
+              <Name lang="de">Teilnahme Sekundärstufe</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169428">
+            <Disability id="94">
+              <Name lang="de">Teilnahme an einer Schule mit angepassten/besonderen Umfeld</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169429">
+            <Disability id="95">
+              <Name lang="de">Berufliche Ausbildung/Ausbildungsverhältnis in einer Standardumgebung</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169430">
+            <Disability id="96">
+              <Name lang="de">Berufliche Ausbildung/Ausbildungsverhältnis in einem angepassten/besonderen Umfeld</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169431">
+            <Disability id="97">
+              <Name lang="de">Erwerb Höherer Bildung und Ausbildung</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169432">
+            <Disability id="99">
+              <Name lang="de">Bezahlte Tätigkeit in einer Standardumgebung</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169433">
+            <Disability id="101">
+              <Name lang="de">Beschäftigungssuche</Name>
+            </Disability>
+            <FrequenceDisability id="27208">
+              <Name lang="de">Häufig</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27180">
+              <Name lang="de">Permanent limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169434">
+            <Disability id="101">
+              <Name lang="de">Beschäftigungssuche</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169435">
+            <Disability id="102">
+              <Name lang="de">Durchführung fachspezifischer Aufgaben</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169436">
+            <Disability id="108">
+              <Name lang="de">Die tägliche Routine durchführen</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169437">
+            <Disability id="113">
+              <Name lang="de">Ausübung von Sport</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="de">Gelegentlich</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="de">Transient limitation</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="de">Gering</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169438">
+            <Disability id="122">
+              <Name lang="de">Lichtintensität</Name>
+            </Disability>
+            <FrequenceDisability/>
+            <TemporalityDisability/>
+            <SeverityDisability/>
+            <LossOfAbility>y</LossOfAbility>
+            <Type>Environmental factor</Type>
+            <Defined>n</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169439">
+            <Disability id="123">
+              <Name lang="de">Lichtqualität</Name>
+            </Disability>
+            <FrequenceDisability/>
+            <TemporalityDisability/>
+            <SeverityDisability/>
+            <LossOfAbility>y</LossOfAbility>
+            <Type>Environmental factor</Type>
+            <Defined>n</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169440">
+            <Disability id="125">
+              <Name lang="de">Tag/Nachtzyklus</Name>
+            </Disability>
+            <FrequenceDisability/>
+            <TemporalityDisability/>
+            <SeverityDisability/>
+            <LossOfAbility>y</LossOfAbility>
+            <Type>Environmental factor</Type>
+            <Defined>n</Defined>
+          </DisabilityDisorderAssociation>
+        </DisabilityDisorderAssociationList>
+      </Disorder>
+      <SourceOfValidation>Expert</SourceOfValidation>
+      <SpecificManagement>y</SpecificManagement>
+      <Online>y</Online>
+      <AnnotationDate>2021-01-14 00:00:00.0</AnnotationDate>
+      <StatusDisability id="27320">
+        <Name lang="de">Validiert</Name>
+      </StatusDisability>
+      <DisabilityCategory id="27278">
+        <Name lang="de">Aktivitätsbeschränkung/verminderte Teilhabe</Name>
+      </DisabilityCategory>
+      <ReasonForNotApplicable/>
+    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="2162">
       <Disorder id="11713">
         <OrphaCode>85436</OrphaCode>
@@ -163796,7 +164246,7 @@
       <Disorder id="16715">
         <OrphaCode>137776</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=137776</ExpertLink>
-        <Name lang="de">Syndrom der letalen Kontrakturen Typ 2</Name>
+        <Name lang="de">Syndrom der kongenitalen letalen Kontrakturen Typ 2</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -163824,7 +164274,7 @@
       <Disorder id="16716">
         <OrphaCode>137783</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=137783</ExpertLink>
-        <Name lang="de">Syndrom der letalen Kontrakturen Typ 3</Name>
+        <Name lang="de">Syndrom der kongenitalen letalen Kontrakturen Typ 3</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -163852,7 +164302,7 @@
       <Disorder id="20932">
         <OrphaCode>294965</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=294965</ExpertLink>
-        <Name lang="de">Lethal congenital contracture syndrome</Name>
+        <Name lang="de">Syndrom der kongenitalen letalen Kontrakturen</Name>
         <DisorderType id="21436">
           <Name lang="de">Phenom-Gruppe</Name>
         </DisorderType>
@@ -163880,7 +164330,7 @@
       <Disorder id="1660">
         <OrphaCode>1486</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1486</ExpertLink>
-        <Name lang="de">Syndrom der letalen Kontrakturen Typ 1</Name>
+        <Name lang="de">Syndrom der kongenitalen letalen Kontrakturen Typ 1</Name>
         <DisorderType id="21401">
           <Name lang="de">Fehlbildungs-Syndrom</Name>
         </DisorderType>
@@ -340951,7 +341401,7 @@
       <Disorder id="18804">
         <OrphaCode>216981</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=216981</ExpertLink>
-        <Name lang="de">Niemann-Pick-Krankheit Typ C,  juvenile neurologische Form</Name>
+        <Name lang="de">Niemann-Pick-Krankheit Typ C, juvenile neurologische Form</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
