commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and functional consequences/de_funct_consequences.xml b/Rare diseases and functional consequences/de_funct_consequences.xml
index e37cb35..ada497e 100644
--- a/Rare diseases and functional consequences/de_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/de_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:15:35" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 06:56:53" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="1052">
+  <DisorderDisabilityRelevanceList count="1046">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -52007,7 +52007,7 @@
       <Disorder id="370">
         <OrphaCode>626</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=626</ExpertLink>
-        <Name lang="de">Kongenitaler melanozytärer Nävus, großer/riesiger</Name>
+        <Name lang="de">Großer/riesiger kongenitaler melanozytärer Naevus</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -100934,54 +100934,6 @@
         <Name lang="de">Hypervariable Funktionsweise</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="1438">
-      <Disorder id="12030">
-        <OrphaCode>90362</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90362</ExpertLink>
-        <Name lang="de">Lymphangiektasie, intestinale primäre</Name>
-        <DisorderType id="21394">
-          <Name lang="de">Krankheit</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="de">Störung</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="2">
-          <DisabilityDisorderAssociation id="189671">
-            <Disability id="117">
-              <Name lang="de">Ernährung</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="189672">
-            <Disability id="119">
-              <Name lang="de">Temperatur</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Stephane VIGNES[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2023-11-01 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="de">Noch nicht validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="de">Aktivitätsbeschränkung/verminderte Teilhabe</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="1413">
       <Disorder id="1784">
         <OrphaCode>1782</OrphaCode>
@@ -132913,7 +132865,7 @@
       <Disorder id="353">
         <OrphaCode>1947</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1947</ExpertLink>
-        <Name lang="de">Epilepsie, progressive - Intelligenzminderung, Finnischer Typ</Name>
+        <Name lang="de">Progressive Epilepsie-Intelligenzminderung-Syndrom, Finnischer Typ</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -193911,7 +193863,7 @@
       <Disorder id="22301">
         <OrphaCode>363424</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=363424</ExpertLink>
-        <Name lang="de">Mitochondriales Dysfunktions-Syndrom, multiples, Typ 3</Name>
+        <Name lang="de">Syndrom der multiplen mitochondrialen Dysfunktionen Typ 3</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -206876,34 +206828,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3066">
-      <Disorder id="890">
-        <OrphaCode>1935</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1935</ExpertLink>
-        <Name lang="de">Enzephalopathie, frühkindliche myoklonische</Name>
-        <DisorderType id="21422">
-          <Name lang="de">Klinisches Syndrom</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="de">Störung</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2022-01-10 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="de">Validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="de">Nicht anwendbar</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27306">
-        <Name lang="de">Hypervariable Funktionsweise</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3057">
       <Disorder id="12584">
         <OrphaCode>95409</OrphaCode>
@@ -222406,32 +222330,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3125">
-      <Disorder id="18313">
-        <OrphaCode>183713</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=183713</ExpertLink>
-        <Name lang="de">Suszeptibilität für Bakterieninfektion durch TLR Signalweg-Defizienz</Name>
-        <DisorderType id="21394">
-          <Name lang="de">Krankheit</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="de">Störung</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Expert</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2012-11-26 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="de">Noch nicht validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27292">
-        <Name lang="de">Ohne Beeinträchtigung der Körperfunktion</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3118">
       <Disorder id="2701">
         <OrphaCode>3003</OrphaCode>
@@ -231717,7 +231615,7 @@
       <Disorder id="8577">
         <OrphaCode>444</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=444</ExpertLink>
-        <Name lang="de">Hypotrichose, hereditäre kongenitale, Typ Marie Unna</Name>
+        <Name lang="de">Hypotrichose, hereditäre, Typ Marie Unna</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -326082,43 +325980,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="4980">
-      <Disorder id="3631">
-        <OrphaCode>809</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=809</ExpertLink>
-        <Name lang="de">Mischkollagenose</Name>
-        <DisorderType id="21394">
-          <Name lang="de">Krankheit</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="de">Störung</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="1">
-          <DisabilityDisorderAssociation id="189130">
-            <Disability id="119">
-              <Name lang="de">Temperatur</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Ragnar GUNNARSSON[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2023-11-01 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="de">Noch nicht validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="de">Aktivitätsbeschränkung/verminderte Teilhabe</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="5001">
       <Disorder id="225">
         <OrphaCode>912</OrphaCode>
@@ -381186,32 +381047,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="5298">
-      <Disorder id="18899">
-        <OrphaCode>218439</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=218439</ExpertLink>
-        <Name lang="de">Herzrhythmusstörung, erworbene</Name>
-        <DisorderType id="36561">
-          <Name lang="de">Kategorie</Name>
-        </DisorderType>
-        <DisorderGroup id="36540">
-          <Name lang="de">Gruppe von Störungen</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Expert</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2014-02-06 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="de">Noch nicht validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27292">
-        <Name lang="de">Ohne Beeinträchtigung der Körperfunktion</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="5302">
       <Disorder id="8618">
         <OrphaCode>658</OrphaCode>
@@ -482942,7 +482777,7 @@
       <Disorder id="20743">
         <OrphaCode>289573</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=289573</ExpertLink>
-        <Name lang="de">Mitochondriales Dysfunktions-Syndrom, multiples</Name>
+        <Name lang="de">Syndrom der multiplen mitochondrialen Dysfunktionen</Name>
         <DisorderType id="21436">
           <Name lang="de">Phenom-Gruppe</Name>
         </DisorderType>
@@ -482970,7 +482805,7 @@
       <Disorder id="22845">
         <OrphaCode>401869</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=401869</ExpertLink>
-        <Name lang="de">Mitochondriales Dysfunktions-Syndrom, multiples, Typ 1</Name>
+        <Name lang="de">Syndrom der multiplen mitochondrialen Dysfunktionen Typ 1</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -482998,7 +482833,7 @@
       <Disorder id="22846">
         <OrphaCode>401874</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=401874</ExpertLink>
-        <Name lang="de">Mitochondriales Dysfunktions-Syndrom, multiples, Typ 2</Name>
+        <Name lang="de">Syndrom der multiplen mitochondrialen Dysfunktionen Typ 2</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
@@ -483022,43 +482857,6 @@
         <Name lang="de">Krankheit verursacht vorzeitigen Tod</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="8116">
-      <Disorder id="3298">
-        <OrphaCode>832</OrphaCode>
-        <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=832</ExpertLink>
-        <Name lang="de">Succinyl-CoA:3-Ketosäure-CoA-Transferase-Mangel</Name>
-        <DisorderType id="21394">
-          <Name lang="de">Krankheit</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="de">Störung</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="1">
-          <DisabilityDisorderAssociation id="89800">
-            <Disability id="117">
-              <Name lang="de">Ernährung</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Toshiyuki FUKAO[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2018-11-29 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="de">Noch nicht validiert</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="de">Aktivitätsbeschränkung/verminderte Teilhabe</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="8030">
       <Disorder id="3568">
         <OrphaCode>175</OrphaCode>
@@ -572071,7 +571869,7 @@
       <Disorder id="3651">
         <OrphaCode>306</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=306</ExpertLink>
-        <Name lang="de">Epilepsie, benigne familiäre infantile</Name>
+        <Name lang="de">Selbstlimitierende infantile Epilepsie</Name>
         <DisorderType id="21394">
           <Name lang="de">Krankheit</Name>
         </DisorderType>
