commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and functional consequences/de_funct_consequences.xml b/Rare diseases and functional consequences/de_funct_consequences.xml
index 9280125..ceed62a 100644
--- a/Rare diseases and functional consequences/de_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/de_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:09:46" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:09:56" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -309306,7 +309306,7 @@
       <Disorder id="14597">
         <OrphaCode>100024</OrphaCode>
         <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=100024</ExpertLink>
-        <Name lang="de">Mu-Schwerkettenkrankheit</Name>
+        <Name lang="de">My-Schwerkettenkrankheit</Name>
         <DisorderType id="21450">
           <Name lang="de">Klinischer Subtyp</Name>
         </DisorderType>
