commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and functional consequences/en_funct_consequences.xml b/Rare diseases and functional consequences/en_funct_consequences.xml
index 028d47f..fcda0bc 100644
--- a/Rare diseases and functional consequences/en_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/en_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:19:19" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:01:43" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="1052">
+  <DisorderDisabilityRelevanceList count="1046">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -6862,7 +6862,7 @@
       <Disorder id="894">
         <OrphaCode>3451</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-        <Name lang="en">Infantile spasms syndrome</Name>
+        <Name lang="en">Infantile epileptic spasms syndrome</Name>
         <DisorderType id="21422">
           <Name lang="en">Clinical syndrome</Name>
         </DisorderType>
@@ -20464,7 +20464,7 @@
       <Disorder id="339">
         <OrphaCode>3380</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3380</ExpertLink>
-        <Name lang="en">Trisomy 18</Name>
+        <Name lang="en">Trisomy 18 syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -20492,7 +20492,7 @@
       <Disorder id="1232">
         <OrphaCode>3376</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3376</ExpertLink>
-        <Name lang="en">Triploidy</Name>
+        <Name lang="en">Triploidy syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -20520,7 +20520,7 @@
       <Disorder id="337">
         <OrphaCode>3378</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3378</ExpertLink>
-        <Name lang="en">Trisomy 13</Name>
+        <Name lang="en">Trisomy 13 syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -22477,7 +22477,7 @@
       <Disorder id="562">
         <OrphaCode>718</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=718</ExpertLink>
-        <Name lang="en">Isolated Pierre Robin syndrome</Name>
+        <Name lang="en">Isolated Pierre Robin sequence</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -22503,7 +22503,7 @@
       <Disorder id="1231">
         <OrphaCode>3375</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3375</ExpertLink>
-        <Name lang="en">Trisomy X</Name>
+        <Name lang="en">Trisomy X syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -42900,7 +42900,7 @@
       <Disorder id="341">
         <OrphaCode>1716</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1716</ExpertLink>
-        <Name lang="en">Distal duplication 18q</Name>
+        <Name lang="en">Distal duplication 18q syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -52007,7 +52007,7 @@
       <Disorder id="370">
         <OrphaCode>626</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=626</ExpertLink>
-        <Name lang="en">Large congenital melanocytic nevus</Name>
+        <Name lang="en">Large/giant congenital melanocytic nevus</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -100934,54 +100934,6 @@
         <Name lang="en">Hypervariable functioning</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="1438">
-      <Disorder id="12030">
-        <OrphaCode>90362</OrphaCode>
-        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90362</ExpertLink>
-        <Name lang="en">Primary intestinal lymphangiectasia</Name>
-        <DisorderType id="21394">
-          <Name lang="en">Disease</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="en">Disorder</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="2">
-          <DisabilityDisorderAssociation id="189671">
-            <Disability id="117">
-              <Name lang="en">Food</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="189672">
-            <Disability id="119">
-              <Name lang="en">Temperature</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Stephane VIGNES[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2023-11-01 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="en">Not validated</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="en">Activity limitation/participation restriction</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="1413">
       <Disorder id="1784">
         <OrphaCode>1782</OrphaCode>
@@ -206876,34 +206828,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3066">
-      <Disorder id="890">
-        <OrphaCode>1935</OrphaCode>
-        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1935</ExpertLink>
-        <Name lang="en">Early myoclonic encephalopathy</Name>
-        <DisorderType id="21422">
-          <Name lang="en">Clinical syndrome</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="en">Disorder</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2022-01-10 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="en">Validated</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="en">Not applicable</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27306">
-        <Name lang="en">Hypervariable functioning</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3057">
       <Disorder id="12584">
         <OrphaCode>95409</OrphaCode>
@@ -212897,7 +212821,7 @@
       <Disorder id="12839">
         <OrphaCode>96321</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96321</ExpertLink>
-        <Name lang="en">Polyploidy</Name>
+        <Name lang="en">Polyploidy syndrome</Name>
         <DisorderType id="36561">
           <Name lang="en">Category</Name>
         </DisorderType>
@@ -218237,7 +218161,7 @@
       <Disorder id="20841">
         <OrphaCode>293181</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293181</ExpertLink>
-        <Name lang="en">Malignant migrating focal seizures of infancy</Name>
+        <Name lang="en">Epilepsy of infancy with migrating focal seizures</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
@@ -222406,32 +222330,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3125">
-      <Disorder id="18313">
-        <OrphaCode>183713</OrphaCode>
-        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=183713</ExpertLink>
-        <Name lang="en">Bacterial susceptibility due to TLR signaling pathway deficiency</Name>
-        <DisorderType id="21394">
-          <Name lang="en">Disease</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="en">Disorder</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Expert</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2012-11-26 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="en">Not validated</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27292">
-        <Name lang="en">No functional disability</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3118">
       <Disorder id="2701">
         <OrphaCode>3003</OrphaCode>
@@ -304701,7 +304599,7 @@
       <Disorder id="14200">
         <OrphaCode>99228</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99228</ExpertLink>
-        <Name lang="en">Mosaic monosomy X</Name>
+        <Name lang="en">Mosaic monosomy X syndrome</Name>
         <DisorderType id="21443">
           <Name lang="en">Etiological subtype</Name>
         </DisorderType>
@@ -305897,7 +305795,7 @@
       <Disorder id="330">
         <OrphaCode>1600</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1600</ExpertLink>
-        <Name lang="en">Monosomy 18q</Name>
+        <Name lang="en">Monosomy 18q syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -308804,7 +308702,7 @@
       <Disorder id="14199">
         <OrphaCode>99226</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99226</ExpertLink>
-        <Name lang="en">Monosomy X</Name>
+        <Name lang="en">Monosomy X syndrome</Name>
         <DisorderType id="21443">
           <Name lang="en">Etiological subtype</Name>
         </DisorderType>
@@ -326082,43 +325980,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="4980">
-      <Disorder id="3631">
-        <OrphaCode>809</OrphaCode>
-        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=809</ExpertLink>
-        <Name lang="en">Mixed connective tissue disease</Name>
-        <DisorderType id="21394">
-          <Name lang="en">Disease</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="en">Disorder</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="1">
-          <DisabilityDisorderAssociation id="189130">
-            <Disability id="119">
-              <Name lang="en">Temperature</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Ragnar GUNNARSSON[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2023-11-01 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="en">Not validated</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="en">Activity limitation/participation restriction</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="5001">
       <Disorder id="225">
         <OrphaCode>912</OrphaCode>
@@ -381186,32 +381047,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="5298">
-      <Disorder id="18899">
-        <OrphaCode>218439</OrphaCode>
-        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=218439</ExpertLink>
-        <Name lang="en">Non-genetic cardiac rhythm disease</Name>
-        <DisorderType id="36561">
-          <Name lang="en">Category</Name>
-        </DisorderType>
-        <DisorderGroup id="36540">
-          <Name lang="en">Group of disorders</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Expert</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2014-02-06 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="en">Not validated</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27292">
-        <Name lang="en">No functional disability</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="5302">
       <Disorder id="8618">
         <OrphaCode>658</OrphaCode>
@@ -442955,7 +442790,7 @@
       <Disorder id="257">
         <OrphaCode>1646</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1646</ExpertLink>
-        <Name lang="en">Chromosome Y microdeletion</Name>
+        <Name lang="en">Chromosome Y microdeletion syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -443197,7 +443032,7 @@
       <Disorder id="1227">
         <OrphaCode>3305</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3305</ExpertLink>
-        <Name lang="en">Tetraploidy</Name>
+        <Name lang="en">Tetraploidy syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -443225,7 +443060,7 @@
       <Disorder id="557">
         <OrphaCode>884</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=884</ExpertLink>
-        <Name lang="en">Tetrasomy 12p</Name>
+        <Name lang="en">Tetrasomy 12p syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -443253,7 +443088,7 @@
       <Disorder id="361">
         <OrphaCode>3307</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3307</ExpertLink>
-        <Name lang="en">Tetrasomy 18p</Name>
+        <Name lang="en">Tetrasomy 18p syndrome</Name>
         <DisorderType id="21401">
           <Name lang="en">Malformation syndrome</Name>
         </DisorderType>
@@ -483022,43 +482857,6 @@
         <Name lang="en">Early death-causing disease</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="8116">
-      <Disorder id="3298">
-        <OrphaCode>832</OrphaCode>
-        <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=832</ExpertLink>
-        <Name lang="en">Succinyl-CoA:3-oxoacid CoA transferase deficiency</Name>
-        <DisorderType id="21394">
-          <Name lang="en">Disease</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="en">Disorder</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="1">
-          <DisabilityDisorderAssociation id="89800">
-            <Disability id="117">
-              <Name lang="en">Food</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Toshiyuki FUKAO[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2018-11-29 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="en">Not validated</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="en">Activity limitation/participation restriction</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="8030">
       <Disorder id="3568">
         <OrphaCode>175</OrphaCode>
@@ -572071,7 +571869,7 @@
       <Disorder id="3651">
         <OrphaCode>306</OrphaCode>
         <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306</ExpertLink>
-        <Name lang="en">Benign familial infantile epilepsy</Name>
+        <Name lang="en">Self-limited infantile epilepsy</Name>
         <DisorderType id="21394">
           <Name lang="en">Disease</Name>
         </DisorderType>
