commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and functional consequences/es_funct_consequences.xml b/Rare diseases and functional consequences/es_funct_consequences.xml
index 77ba555..b37c8c0 100644
--- a/Rare diseases and functional consequences/es_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/es_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:12:19" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:11:40" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -264656,7 +264656,7 @@
       <Disorder id="17873">
         <OrphaCode>169805</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=169805</ExpertLink>
-        <Name lang="es">Hemofilia A moderamente grave</Name>
+        <Name lang="es">Hemofilia A moderada</Name>
         <DisorderType id="21450">
           <Name lang="es">Subtipo clínico</Name>
         </DisorderType>
@@ -349758,7 +349758,7 @@
       <Disorder id="11106">
         <OrphaCode>77293</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=77293</ExpertLink>
-        <Name lang="es">Enfermedad de Niemann-Pick tipo B</Name>
+        <Name lang="es">Deficiencia de esfingomielinasa ácida visceral crónica</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -350610,7 +350610,7 @@
       <Disorder id="11105">
         <OrphaCode>77292</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=77292</ExpertLink>
-        <Name lang="es">Enfermedad de Niemann-Pick tipo A</Name>
+        <Name lang="es">Deficiencia de esfingomielinasa ácida neurovisceral infantil</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -542924,7 +542924,7 @@
       <Disorder id="10417">
         <OrphaCode>36355</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=36355</ExpertLink>
-        <Name lang="es">Trastorno de la coagulación por defecto en P2Y12</Name>
+        <Name lang="es">Trastorno de sangrado por un defecto de P2Y12</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
