commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Rare diseases and functional consequences/es_funct_consequences.xml b/Rare diseases and functional consequences/es_funct_consequences.xml
old mode 100644
new mode 100755
index 9501256..aee69f9
--- a/Rare diseases and functional consequences/es_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/es_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:07:31" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:07:47" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="910">
+  <DisorderDisabilityRelevanceList count="911">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -154198,6 +154198,456 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
+    <DisorderDisabilityRelevance id="2121">
+      <Disorder id="101">
+        <OrphaCode>1941</OrphaCode>
+        <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1941</ExpertLink>
+        <Name lang="es">Epilepsia de ausencia juvenil</Name>
+        <DisorderType id="21394">
+          <Name lang="es">Enfermedad</Name>
+        </DisorderType>
+        <DisorderGroup id="36547">
+          <Name lang="es">Trastorno</Name>
+        </DisorderGroup>
+        <DisabilityDisorderAssociationList count="26">
+          <DisabilityDisorderAssociation id="169415">
+            <Disability id="16">
+              <Name lang="es">Fijación de la atención</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169416">
+            <Disability id="16">
+              <Name lang="es">Fijación de la atención</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27236">
+              <Name lang="es">Moderada</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169417">
+            <Disability id="34">
+              <Name lang="es">Participación en conversaciones</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169418">
+            <Disability id="61">
+              <Name lang="es">Cuidado de su propia salud (dieta, medicación, prevención, necesidades, ayuda o supervisión)</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169419">
+            <Disability id="68">
+              <Name lang="es">Manejo del estrés, las responsabilidades, las emergencias y garantizar su propia seguridad</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27236">
+              <Name lang="es">Moderada</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169420">
+            <Disability id="69">
+              <Name lang="es">Manejo de las emociones y el estado de ánimo</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169421">
+            <Disability id="77">
+              <Name lang="es">Conducción</Name>
+            </Disability>
+            <FrequenceDisability id="27201">
+              <Name lang="es">Muy frecuente</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27264">
+              <Name lang="es">Completa</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169422">
+            <Disability id="87">
+              <Name lang="es">Hacer la compra</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169423">
+            <Disability id="88">
+              <Name lang="es">Cocinar/preparar la comida</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169424">
+            <Disability id="89">
+              <Name lang="es">Realización de tareas domésticas</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169425">
+            <Disability id="90">
+              <Name lang="es">Cuidado y ayuda a terceros</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169426">
+            <Disability id="92">
+              <Name lang="es">Asistencia a un centro de educación preescolar</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169427">
+            <Disability id="93">
+              <Name lang="es">Asistencia a un centro escolar estándar</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169428">
+            <Disability id="94">
+              <Name lang="es">Asistencia a una escuela adaptada/especializada</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169429">
+            <Disability id="95">
+              <Name lang="es">Aprendizaje de una profesión en un entorno estándar</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169430">
+            <Disability id="96">
+              <Name lang="es">Aprendizaje de una profesión en un entorno adaptado o especializado</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169431">
+            <Disability id="97">
+              <Name lang="es">Acceso a estudios superiores</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169432">
+            <Disability id="99">
+              <Name lang="es">Acceso a un trabajo remunerado en un entorno estándar</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169433">
+            <Disability id="101">
+              <Name lang="es">Búsqueda de empleo</Name>
+            </Disability>
+            <FrequenceDisability id="27208">
+              <Name lang="es">Frecuente</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27180">
+              <Name lang="es">Limitación/restricción permanente</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169434">
+            <Disability id="101">
+              <Name lang="es">Búsqueda de empleo</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169435">
+            <Disability id="102">
+              <Name lang="es">Desempeño de tareas profesionales</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169436">
+            <Disability id="108">
+              <Name lang="es">Desempeño de tareas cotidianas</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169437">
+            <Disability id="113">
+              <Name lang="es">Practica de deporte</Name>
+            </Disability>
+            <FrequenceDisability id="27215">
+              <Name lang="es">Ocasional</Name>
+            </FrequenceDisability>
+            <TemporalityDisability id="27194">
+              <Name lang="es">Limitación/restricción transitoria</Name>
+            </TemporalityDisability>
+            <SeverityDisability id="27222">
+              <Name lang="es">Baja</Name>
+            </SeverityDisability>
+            <LossOfAbility>n</LossOfAbility>
+            <Type>Disability</Type>
+            <Defined>y</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169438">
+            <Disability id="122">
+              <Name lang="es">Intensidad luminosa</Name>
+            </Disability>
+            <FrequenceDisability/>
+            <TemporalityDisability/>
+            <SeverityDisability/>
+            <LossOfAbility>y</LossOfAbility>
+            <Type>Environmental factor</Type>
+            <Defined>n</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169439">
+            <Disability id="123">
+              <Name lang="es">Calidad luminosa</Name>
+            </Disability>
+            <FrequenceDisability/>
+            <TemporalityDisability/>
+            <SeverityDisability/>
+            <LossOfAbility>y</LossOfAbility>
+            <Type>Environmental factor</Type>
+            <Defined>n</Defined>
+          </DisabilityDisorderAssociation>
+          <DisabilityDisorderAssociation id="169440">
+            <Disability id="125">
+              <Name lang="es">Ciclos día/noche</Name>
+            </Disability>
+            <FrequenceDisability/>
+            <TemporalityDisability/>
+            <SeverityDisability/>
+            <LossOfAbility>y</LossOfAbility>
+            <Type>Environmental factor</Type>
+            <Defined>n</Defined>
+          </DisabilityDisorderAssociation>
+        </DisabilityDisorderAssociationList>
+      </Disorder>
+      <SourceOfValidation>Expert</SourceOfValidation>
+      <SpecificManagement>y</SpecificManagement>
+      <Online>y</Online>
+      <AnnotationDate>2021-01-14 00:00:00.0</AnnotationDate>
+      <StatusDisability id="27320">
+        <Name lang="es">Validado</Name>
+      </StatusDisability>
+      <DisabilityCategory id="27278">
+        <Name lang="es">Actividad limitada/participación restringida</Name>
+      </DisabilityCategory>
+      <ReasonForNotApplicable/>
+    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="2162">
       <Disorder id="11713">
         <OrphaCode>85436</OrphaCode>
@@ -164020,7 +164470,7 @@
       <Disorder id="19790">
         <OrphaCode>254857</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=254857</ExpertLink>
-        <Name lang="es">Miopatía mitocondrial letal infantil</Name>
+        <Name lang="es">Miopatía mitocondrial letal del lactante</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -356413,7 +356863,7 @@
       <Disorder id="20766">
         <OrphaCode>289857</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289857</ExpertLink>
-        <Name lang="es">Encefalopatía neonatal por glicina</Name>
+        <Name lang="es">Encefalopatía por glicina neonatal</Name>
         <DisorderType id="21450">
           <Name lang="es">Subtipo clínico</Name>
         </DisorderType>
@@ -413149,7 +413599,7 @@
       <Disorder id="20767">
         <OrphaCode>289860</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289860</ExpertLink>
-        <Name lang="es">Encefalopatía infantil por glicina</Name>
+        <Name lang="es">Encefalopatía por glicina infantil</Name>
         <DisorderType id="21450">
           <Name lang="es">Subtipo clínico</Name>
         </DisorderType>
@@ -445450,7 +445900,7 @@
       <Disorder id="3525">
         <OrphaCode>3286</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3286</ExpertLink>
-        <Name lang="es">Taquicardia ventricular polimórfica catecolaminérgica</Name>
+        <Name lang="es">Taquicardia ventricular catecolaminérgica polimórfica</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -445594,7 +446044,7 @@
       <Disorder id="20472">
         <OrphaCode>280553</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280553</ExpertLink>
-        <Name lang="es">Miopatía miofibrilar hipertónica infantil letal</Name>
+        <Name lang="es">Miopatía miofibrilar hipertónica letal del lactante</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -445622,7 +446072,7 @@
       <Disorder id="776">
         <OrphaCode>17</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=17</ExpertLink>
-        <Name lang="es">Acidosis láctica infantil letal con aciduria metilmalónica</Name>
+        <Name lang="es">Acidosis láctica letal del lactante con aciduria metilmalónica</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -445678,7 +446128,7 @@
       <Disorder id="3379">
         <OrphaCode>1561</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1561</ExpertLink>
-        <Name lang="es">Deficiencia infantil fatal de citocromo C oxidasa</Name>
+        <Name lang="es">Deficiencia de citocromo C oxidasa fatal del lactante</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -536259,7 +536709,7 @@
       <Disorder id="3651">
         <OrphaCode>306</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=306</ExpertLink>
-        <Name lang="es">Epilepsia infantil familiar benigna</Name>
+        <Name lang="es">Epilepsia benigna del lactante familiar</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
