commit 76201feb251c2058792f9cf480d515ad9cf39106
Author: David Lagorce <david.lagorce@inserm.fr>
Date:   Wed Jul 7 12:35:15 2021 +0200

    JULY 2021

diff --git a/Rare diseases and functional consequences/es_funct_consequences.xml b/Rare diseases and functional consequences/es_funct_consequences.xml
old mode 100644
new mode 100755
index 297e302..403dba6
--- a/Rare diseases and functional consequences/es_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/es_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-06-01 04:08:06" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-07-01 04:08:46" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -6862,7 +6862,7 @@
       <Disorder id="894">
         <OrphaCode>3451</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3451</ExpertLink>
-        <Name lang="es">Síndrome de West</Name>
+        <Name lang="es">Síndrome de espasmos infantiles</Name>
         <DisorderType id="21422">
           <Name lang="es">Síndrome clínico</Name>
         </DisorderType>
@@ -164134,7 +164134,7 @@
       <Disorder id="17237">
         <OrphaCode>158687</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158687</ExpertLink>
-        <Name lang="es">Epidermólisis ampollosa acantolítica letal</Name>
+        <Name lang="es">Trastorno erosivo acantolítico letal</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -230655,7 +230655,7 @@
       <Disorder id="19226">
         <OrphaCode>232288</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=232288</ExpertLink>
-        <Name lang="es">Enfermedades asociadas a una alfa-talasemia</Name>
+        <Name lang="es">Síndrome con alfa-talasemia como característica principal</Name>
         <DisorderType id="36561">
           <Name lang="es">Category</Name>
         </DisorderType>
@@ -444807,7 +444807,7 @@
       <Disorder id="3525">
         <OrphaCode>3286</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3286</ExpertLink>
-        <Name lang="es">Taquicardia ventricular catecolaminérgica polimórfica</Name>
+        <Name lang="es">Taquicardia ventricular polimórfica catecolaminérgica</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -445175,7 +445175,7 @@
       <Disorder id="3298">
         <OrphaCode>832</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=832</ExpertLink>
-        <Name lang="es">Deficiencia de succinil-CoA: 3 cetoácido CoA transferasa</Name>
+        <Name lang="es">Deficiencia de succinil-CoA:3-cetoácido-CoA transferasa</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
@@ -447373,7 +447373,7 @@
       <Disorder id="12511">
         <OrphaCode>93953</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93953</ExpertLink>
-        <Name lang="es">Familial thyroglossal duct cyst</Name>
+        <Name lang="es">Quiste del conducto tirogloso familiar</Name>
         <DisorderType id="21415">
           <Name lang="es">Anomalía morfológica</Name>
         </DisorderType>
@@ -468709,7 +468709,7 @@
       <Disorder id="1007">
         <OrphaCode>528</OrphaCode>
         <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=528</ExpertLink>
-        <Name lang="es">Lipodistrofia congénita de Berardinelli-Seip</Name>
+        <Name lang="es">Lipodistrofia congénita generalizada</Name>
         <DisorderType id="21394">
           <Name lang="es">Enfermedad</Name>
         </DisorderType>
