commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Rare diseases and functional consequences/fr_funct_consequences.xml b/Rare diseases and functional consequences/fr_funct_consequences.xml
old mode 100644
new mode 100755
index ed8f35d..2343206
--- a/Rare diseases and functional consequences/fr_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/fr_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:07:47" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:07:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="911">
+  <DisorderDisabilityRelevanceList count="910">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -1744,7 +1744,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="32">
           <DisabilityDisorderAssociation id="40776">
@@ -2284,7 +2284,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -2312,7 +2312,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="32">
           <DisabilityDisorderAssociation id="40893">
@@ -5654,7 +5654,7 @@
           <Name lang="fr">Sous-type histo-pathologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="55">
           <DisabilityDisorderAssociation id="123287">
@@ -6722,7 +6722,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="7">
           <DisabilityDisorderAssociation id="160045">
@@ -11482,7 +11482,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -14466,7 +14466,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="42">
           <DisabilityDisorderAssociation id="116586">
@@ -15964,7 +15964,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -15992,7 +15992,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="82">
           <DisabilityDisorderAssociation id="121083">
@@ -20303,7 +20303,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="5">
           <DisabilityDisorderAssociation id="12686">
@@ -26340,7 +26340,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="42">
           <DisabilityDisorderAssociation id="92439">
@@ -27044,7 +27044,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -27645,7 +27645,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="33">
           <DisabilityDisorderAssociation id="12449">
@@ -28220,7 +28220,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -28376,7 +28376,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -28404,7 +28404,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -28432,7 +28432,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -29809,7 +29809,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -29837,7 +29837,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -33675,7 +33675,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="99">
           <DisabilityDisorderAssociation id="24140">
@@ -36065,7 +36065,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -36093,7 +36093,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -40747,7 +40747,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -40773,7 +40773,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -40799,7 +40799,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -40825,7 +40825,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="143015">
@@ -40938,7 +40938,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="109">
           <DisabilityDisorderAssociation id="108709">
@@ -42817,7 +42817,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="25">
           <DisabilityDisorderAssociation id="166245">
@@ -43256,7 +43256,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="5">
           <DisabilityDisorderAssociation id="25652">
@@ -55378,7 +55378,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="34">
           <DisabilityDisorderAssociation id="97541">
@@ -55928,7 +55928,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="12">
           <DisabilityDisorderAssociation id="97392">
@@ -56140,7 +56140,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="1">
           <DisabilityDisorderAssociation id="13386">
@@ -56177,7 +56177,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="21">
           <DisabilityDisorderAssociation id="97418">
@@ -56494,7 +56494,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="14">
           <DisabilityDisorderAssociation id="99361">
@@ -56952,7 +56952,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -57711,7 +57711,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="38">
           <DisabilityDisorderAssociation id="60365">
@@ -58377,7 +58377,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="59">
           <DisabilityDisorderAssociation id="41554">
@@ -59370,7 +59370,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="85">
           <DisabilityDisorderAssociation id="41677">
@@ -60805,7 +60805,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="37">
           <DisabilityDisorderAssociation id="142294">
@@ -62569,7 +62569,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="108">
           <DisabilityDisorderAssociation id="151899">
@@ -64413,7 +64413,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="17">
           <DisabilityDisorderAssociation id="164527">
@@ -71968,7 +71968,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="60">
           <DisabilityDisorderAssociation id="166184">
@@ -73014,7 +73014,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="60">
           <DisabilityDisorderAssociation id="165980">
@@ -74060,7 +74060,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -75479,7 +75479,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="93">
           <DisabilityDisorderAssociation id="54276">
@@ -77044,7 +77044,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="186">
           <DisabilityDisorderAssociation id="120160">
@@ -80190,7 +80190,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="168">
           <DisabilityDisorderAssociation id="118723">
@@ -86732,7 +86732,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="152">
           <DisabilityDisorderAssociation id="149197">
@@ -89342,7 +89342,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="131">
           <DisabilityDisorderAssociation id="149531">
@@ -101484,7 +101484,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -101512,7 +101512,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="131">
           <DisabilityDisorderAssociation id="149662">
@@ -108169,7 +108169,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="90">
           <DisabilityDisorderAssociation id="162356">
@@ -109725,7 +109725,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="47">
           <DisabilityDisorderAssociation id="162354">
@@ -110550,7 +110550,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="47">
           <DisabilityDisorderAssociation id="162261">
@@ -116609,7 +116609,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="5">
           <DisabilityDisorderAssociation id="12681">
@@ -116720,7 +116720,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="21">
           <DisabilityDisorderAssociation id="165798">
@@ -117360,7 +117360,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -117388,7 +117388,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="57">
           <DisabilityDisorderAssociation id="148269">
@@ -118365,7 +118365,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="120">
           <DisabilityDisorderAssociation id="151203">
@@ -120371,7 +120371,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="45">
           <DisabilityDisorderAssociation id="158086">
@@ -121162,7 +121162,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="109">
           <DisabilityDisorderAssociation id="150096">
@@ -128362,7 +128362,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="39">
           <DisabilityDisorderAssociation id="137272">
@@ -129196,7 +129196,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="65">
           <DisabilityDisorderAssociation id="158308">
@@ -130015,7 +130015,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="50">
           <DisabilityDisorderAssociation id="158415">
@@ -137047,7 +137047,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="2">
           <DisabilityDisorderAssociation id="13242">
@@ -139817,7 +139817,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="2">
           <DisabilityDisorderAssociation id="13244">
@@ -139865,7 +139865,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="11">
           <DisabilityDisorderAssociation id="31179">
@@ -156936,7 +156936,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="118">
           <DisabilityDisorderAssociation id="159107">
@@ -158950,7 +158950,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="112">
           <DisabilityDisorderAssociation id="166378">
@@ -163941,7 +163941,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -164221,7 +164221,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="76">
           <DisabilityDisorderAssociation id="157666">
@@ -165515,7 +165515,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="96">
           <DisabilityDisorderAssociation id="157919">
@@ -166597,7 +166597,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="48">
           <DisabilityDisorderAssociation id="157818">
@@ -167461,7 +167461,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="1465">
@@ -167649,7 +167649,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="1077">
@@ -167743,7 +167743,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="1073">
@@ -169508,7 +169508,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -169564,7 +169564,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="76">
           <DisabilityDisorderAssociation id="147569">
@@ -170882,7 +170882,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="142123">
@@ -171061,7 +171061,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -171089,7 +171089,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="46">
           <DisabilityDisorderAssociation id="34596">
@@ -171867,7 +171867,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="112">
           <DisabilityDisorderAssociation id="104052">
@@ -173773,7 +173773,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="112">
           <DisabilityDisorderAssociation id="99235">
@@ -178393,7 +178393,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -178762,7 +178762,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="114">
           <DisabilityDisorderAssociation id="143513">
@@ -195384,7 +195384,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="80">
           <DisabilityDisorderAssociation id="12871">
@@ -196764,7 +196764,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="84">
           <DisabilityDisorderAssociation id="38149">
@@ -198212,7 +198212,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -203637,7 +203637,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -209049,7 +209049,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="42">
           <DisabilityDisorderAssociation id="121207">
@@ -209771,7 +209771,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="36">
           <DisabilityDisorderAssociation id="73203">
@@ -210391,7 +210391,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="25">
           <DisabilityDisorderAssociation id="153444">
@@ -212362,7 +212362,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -214805,7 +214805,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="51">
           <DisabilityDisorderAssociation id="42187">
@@ -215680,7 +215680,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="41">
           <DisabilityDisorderAssociation id="42206">
@@ -216868,7 +216868,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="45">
           <DisabilityDisorderAssociation id="137379">
@@ -221556,7 +221556,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -226263,7 +226263,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -226291,7 +226291,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -233974,7 +233974,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="99">
           <DisabilityDisorderAssociation id="24137">
@@ -238483,7 +238483,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -238511,7 +238511,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="75">
           <DisabilityDisorderAssociation id="145302">
@@ -242226,7 +242226,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="67">
           <DisabilityDisorderAssociation id="154318">
@@ -245029,7 +245029,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="79">
           <DisabilityDisorderAssociation id="166422">
@@ -250373,7 +250373,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="113">
           <DisabilityDisorderAssociation id="142672">
@@ -252290,7 +252290,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="120">
           <DisabilityDisorderAssociation id="152980">
@@ -254332,7 +254332,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="110003">
@@ -257123,7 +257123,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="23">
           <DisabilityDisorderAssociation id="144157">
@@ -259578,7 +259578,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="84">
           <DisabilityDisorderAssociation id="32077">
@@ -261014,7 +261014,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -262546,7 +262546,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="13">
           <DisabilityDisorderAssociation id="124325">
@@ -262865,7 +262865,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="23">
           <DisabilityDisorderAssociation id="165897">
@@ -263216,7 +263216,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="48">
           <DisabilityDisorderAssociation id="124130">
@@ -272364,7 +272364,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="91">
           <DisabilityDisorderAssociation id="151397">
@@ -273945,7 +273945,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="25">
           <DisabilityDisorderAssociation id="166228">
@@ -277722,7 +277722,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="21">
           <DisabilityDisorderAssociation id="120460">
@@ -278093,7 +278093,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="69">
           <DisabilityDisorderAssociation id="153568">
@@ -285053,7 +285053,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -285081,7 +285081,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="80">
           <DisabilityDisorderAssociation id="117158">
@@ -288679,7 +288679,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="33">
           <DisabilityDisorderAssociation id="12416">
@@ -289840,34 +289840,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="4145">
-      <Disorder id="20880">
-        <OrphaCode>293848</OrphaCode>
-        <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=293848</ExpertLink>
-        <Name lang="fr">Démence fronto-temporale, variante d'atrophie temporale à droite</Name>
-        <DisorderType id="21394">
-          <Name lang="fr">Maladie</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="fr">Pathologie</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2019-02-04 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="fr">Validé</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="fr">Non applicable</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27306">
-        <Name lang="fr">Extrême variabilité des présentations entre les patients</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="4097">
       <Disorder id="357">
         <OrphaCode>370</OrphaCode>
@@ -292810,7 +292782,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="33">
           <DisabilityDisorderAssociation id="12383">
@@ -294536,7 +294508,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="78">
           <DisabilityDisorderAssociation id="167109">
@@ -299845,7 +299817,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -305045,7 +305017,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="6">
           <DisabilityDisorderAssociation id="164504">
@@ -305161,7 +305133,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="164682">
@@ -305778,7 +305750,7 @@
           <Name lang="fr">Sous-type histo-pathologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="7">
           <DisabilityDisorderAssociation id="12361">
@@ -310133,7 +310105,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -314905,7 +314877,7 @@
           <Name lang="fr">Sous-type histo-pathologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="7">
           <DisabilityDisorderAssociation id="12368">
@@ -315050,7 +315022,7 @@
           <Name lang="fr">Sous-type histo-pathologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="7">
           <DisabilityDisorderAssociation id="12375">
@@ -320576,7 +320548,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="108">
           <DisabilityDisorderAssociation id="166707">
@@ -322390,7 +322362,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="14">
           <DisabilityDisorderAssociation id="149795">
@@ -322654,7 +322626,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="63">
           <DisabilityDisorderAssociation id="120397">
@@ -324119,7 +324091,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="175">
           <DisabilityDisorderAssociation id="13041">
@@ -328090,7 +328062,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="56197">
@@ -328245,7 +328217,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="2">
           <DisabilityDisorderAssociation id="56206">
@@ -331605,7 +331577,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="42">
           <DisabilityDisorderAssociation id="116628">
@@ -338292,7 +338264,7 @@
           <Name lang="fr">Sous-type histo-pathologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="7">
           <DisabilityDisorderAssociation id="12354">
@@ -338642,7 +338614,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="64">
           <DisabilityDisorderAssociation id="156672">
@@ -339766,7 +339738,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -340928,7 +340900,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -340956,7 +340928,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -340984,7 +340956,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -341012,7 +340984,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -345005,7 +344977,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -354693,7 +354665,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -354719,7 +354691,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -354771,7 +354743,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -354797,7 +354769,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -356446,7 +356418,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="88">
           <DisabilityDisorderAssociation id="122564">
@@ -370520,7 +370492,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="99">
           <DisabilityDisorderAssociation id="24134">
@@ -375204,7 +375176,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -376170,7 +376142,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -376198,7 +376170,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -376226,7 +376198,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -376254,7 +376226,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -376282,7 +376254,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -379841,7 +379813,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="112">
           <DisabilityDisorderAssociation id="73764">
@@ -381869,7 +381841,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -381996,7 +381968,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="10">
           <DisabilityDisorderAssociation id="166300">
@@ -385976,7 +385948,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13964">
@@ -386149,7 +386121,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13928">
@@ -386322,7 +386294,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13982">
@@ -386495,7 +386467,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13946">
@@ -386668,7 +386640,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13973">
@@ -386841,7 +386813,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13955">
@@ -388205,7 +388177,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="9">
           <DisabilityDisorderAssociation id="13937">
@@ -389605,7 +389577,7 @@
           <Name lang="fr">Sous-type histo-pathologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="55">
           <DisabilityDisorderAssociation id="168269">
@@ -399140,7 +399112,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="74">
           <DisabilityDisorderAssociation id="15287">
@@ -400388,7 +400360,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="85">
           <DisabilityDisorderAssociation id="15689">
@@ -411608,7 +411580,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -411636,7 +411608,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -413182,7 +413154,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="80">
           <DisabilityDisorderAssociation id="122732">
@@ -414562,7 +414534,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="11926">
@@ -414644,7 +414616,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="23">
           <DisabilityDisorderAssociation id="168430">
@@ -414951,7 +414923,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -414979,7 +414951,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="23">
           <DisabilityDisorderAssociation id="168440">
@@ -422331,7 +422303,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -422359,7 +422331,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="54">
           <DisabilityDisorderAssociation id="153705">
@@ -424063,7 +424035,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="6">
           <DisabilityDisorderAssociation id="66640">
@@ -424191,7 +424163,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="6">
           <DisabilityDisorderAssociation id="66646">
@@ -426996,7 +426968,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="80">
           <DisabilityDisorderAssociation id="153908">
@@ -428364,7 +428336,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="48">
           <DisabilityDisorderAssociation id="123990">
@@ -441354,7 +441326,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="31">
           <DisabilityDisorderAssociation id="148419">
@@ -441907,7 +441879,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="31">
           <DisabilityDisorderAssociation id="148451">
@@ -445888,7 +445860,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -445911,7 +445883,7 @@
       <Disorder id="634">
         <OrphaCode>84</OrphaCode>
         <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=84</ExpertLink>
-        <Name lang="fr">Anémie de Fanconi</Name>
+        <Name lang="fr">Maladie de Fanconi</Name>
         <DisorderType id="21401">
           <Name lang="fr">Syndrome  malformatif</Name>
         </DisorderType>
@@ -449541,7 +449513,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="60">
           <DisabilityDisorderAssociation id="142951">
@@ -451211,7 +451183,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -451239,7 +451211,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -455896,7 +455868,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -458671,7 +458643,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="168">
           <DisabilityDisorderAssociation id="124548">
@@ -461511,7 +461483,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125342">
@@ -463180,7 +463152,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125484">
@@ -464849,7 +464821,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125678">
@@ -466518,7 +466490,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125387">
@@ -469214,7 +469186,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="168207">
@@ -469308,7 +469280,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="3">
           <DisabilityDisorderAssociation id="168211">
@@ -474447,7 +474419,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="113">
           <DisabilityDisorderAssociation id="153337">
@@ -476370,7 +476342,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -481215,7 +481187,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="109">
           <DisabilityDisorderAssociation id="123798">
@@ -483094,7 +483066,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="109">
           <DisabilityDisorderAssociation id="108600">
@@ -485001,7 +484973,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="132">
           <DisabilityDisorderAssociation id="144023">
@@ -487265,7 +487237,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="132">
           <DisabilityDisorderAssociation id="144312">
@@ -495253,7 +495225,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -495281,7 +495253,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -495998,7 +495970,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="23">
           <DisabilityDisorderAssociation id="168340">
@@ -498557,7 +498529,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -501257,7 +501229,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="4">
           <DisabilityDisorderAssociation id="12249">
@@ -507998,7 +507970,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="42">
           <DisabilityDisorderAssociation id="116544">
@@ -511816,7 +511788,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="70">
           <DisabilityDisorderAssociation id="58957">
@@ -513002,7 +512974,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -513030,7 +513002,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="21">
           <DisabilityDisorderAssociation id="136720">
@@ -513377,7 +513349,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="142">
           <DisabilityDisorderAssociation id="136781">
@@ -515697,7 +515669,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="79">
           <DisabilityDisorderAssociation id="59171">
@@ -536318,7 +536290,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125274">
@@ -538783,7 +538755,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125009">
@@ -540452,7 +540424,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="97">
           <DisabilityDisorderAssociation id="125104">
@@ -570599,7 +570571,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -570627,7 +570599,7 @@
           <Name lang="fr">Sous-type étiologique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="0">
         </DisabilityDisorderAssociationList>
@@ -570711,7 +570683,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="47">
           <DisabilityDisorderAssociation id="132190">
@@ -571530,7 +571502,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="47">
           <DisabilityDisorderAssociation id="132378">
@@ -572349,7 +572321,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="47">
           <DisabilityDisorderAssociation id="132143">
@@ -573168,7 +573140,7 @@
           <Name lang="fr">Sous-type clinique</Name>
         </DisorderType>
         <DisorderGroup id="36554">
-          <Name lang="fr">Sous-type d'une maladie</Name>
+          <Name lang="fr">Sous-type d'une pathologie</Name>
         </DisorderGroup>
         <DisabilityDisorderAssociationList count="47">
           <DisabilityDisorderAssociation id="132237">
