commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases and functional consequences/fr_funct_consequences.xml b/Rare diseases and functional consequences/fr_funct_consequences.xml
index d0429e9..b4c12e1 100644
--- a/Rare diseases and functional consequences/fr_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/fr_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:11:52" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:11:50" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="946">
+  <DisorderDisabilityRelevanceList count="945">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -263855,111 +263855,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3584">
-      <Disorder id="11041">
-        <OrphaCode>73247</OrphaCode>
-        <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=73247</ExpertLink>
-        <Name lang="fr">NON RARE EN EUROPE : Oesophagite à éosinophiles</Name>
-        <DisorderType id="21394">
-          <Name lang="fr">Maladie</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="fr">Pathologie</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="5">
-          <DisabilityDisorderAssociation id="26430">
-            <Disability id="69">
-              <Name lang="fr">Gérer ses émotions et son humeur</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="fr">Fréquent</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27194">
-              <Name lang="fr">Limitation/restriction transitoire</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27236">
-              <Name lang="fr">Modéré</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26431">
-            <Disability id="69">
-              <Name lang="fr">Gérer ses émotions et son humeur</Name>
-            </Disability>
-            <FrequenceDisability id="27215">
-              <Name lang="fr">Occasionel</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="fr">Limitation/restriction permanente</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="fr">Faible</Name>
-            </SeverityDisability>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26432">
-            <Disability id="110">
-              <Name lang="fr">Participer à la vie en communauté</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="fr">Fréquent</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="fr">Limitation/restriction permanente</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="fr">Faible</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26433">
-            <Disability id="112">
-              <Name lang="fr">Rencontrer ses amis</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="fr">Fréquent</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="fr">Limitation/restriction permanente</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="fr">Faible</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26434">
-            <Disability id="117">
-              <Name lang="fr">Aliments</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Alfredo LUCENDO[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2015-01-10 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="fr">En cours de validation</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="fr">Limitation d’activité/restriction de participation</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3585">
       <Disorder id="755">
         <OrphaCode>3165</OrphaCode>
@@ -350302,7 +350197,7 @@
       <Disorder id="11106">
         <OrphaCode>77293</OrphaCode>
         <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=77293</ExpertLink>
-        <Name lang="fr">Maladie de Niemann-Pick type B</Name>
+        <Name lang="fr">Déficit en sphingomyélinase acide, forme viscérale chronique</Name>
         <DisorderType id="21394">
           <Name lang="fr">Maladie</Name>
         </DisorderType>
@@ -351154,7 +351049,7 @@
       <Disorder id="11105">
         <OrphaCode>77292</OrphaCode>
         <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=77292</ExpertLink>
-        <Name lang="fr">Maladie de Niemann-Pick type A</Name>
+        <Name lang="fr">Déficit en sphingomyélinase acide, forme neuroviscérale infantile</Name>
         <DisorderType id="21394">
           <Name lang="fr">Maladie</Name>
         </DisorderType>
