commit cb98a4638b303dbc159c97a636d6d4f62ca39678
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 6 12:14:41 2022 +0100

    DEC 2022

diff --git a/Rare diseases and functional consequences/fr_funct_consequences.xml b/Rare diseases and functional consequences/fr_funct_consequences.xml
index 52ff2c4..7f150ef 100644
--- a/Rare diseases and functional consequences/fr_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/fr_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:35:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:59:06" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -370505,7 +370505,7 @@
       <Disorder id="183">
         <OrphaCode>637</OrphaCode>
         <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=637</ExpertLink>
-        <Name lang="fr">Neurofibromatose type 2</Name>
+        <Name lang="fr">Schwannomatose liée à NF2 germinale</Name>
         <DisorderType id="21394">
           <Name lang="fr">Maladie</Name>
         </DisorderType>
