commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Rare diseases and functional consequences/fr_funct_consequences.xml b/Rare diseases and functional consequences/fr_funct_consequences.xml
index bac20ea..144fb1b 100644
--- a/Rare diseases and functional consequences/fr_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/fr_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:08:10" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:08:18" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="910">
+  <DisorderDisabilityRelevanceList count="909">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -164354,34 +164354,6 @@
         <Name lang="fr">Maladie conduisant actuellement à un décès précoce</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="2318">
-      <Disorder id="413">
-        <OrphaCode>1046</OrphaCode>
-        <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=1046</ExpertLink>
-        <Name lang="fr">Syndrome d'anémie hémolytique létale-anomalies génitales</Name>
-        <DisorderType id="21401">
-          <Name lang="fr">Syndrome  malformatif</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="fr">Pathologie</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2014-11-21 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="fr">Validé</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="fr">Non applicable</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27299">
-        <Name lang="fr">Maladie conduisant actuellement à un décès précoce</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="2316">
       <Disorder id="21964">
         <OrphaCode>330050</OrphaCode>
@@ -164470,7 +164442,7 @@
       <Disorder id="19790">
         <OrphaCode>254857</OrphaCode>
         <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=254857</ExpertLink>
-        <Name lang="fr">Myopathy mitochondriale létale infantile</Name>
+        <Name lang="fr">Myopathie mitochondriale létale infantile</Name>
         <DisorderType id="21394">
           <Name lang="fr">Maladie</Name>
         </DisorderType>
@@ -226737,8 +226709,8 @@
         <OrphaCode>847</OrphaCode>
         <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=847</ExpertLink>
         <Name lang="fr">Alpha-thalassémie-déficience intellectuelle liée à l'X</Name>
-        <DisorderType id="21394">
-          <Name lang="fr">Maladie</Name>
+        <DisorderType id="21401">
+          <Name lang="fr">Syndrome  malformatif</Name>
         </DisorderType>
         <DisorderGroup id="36547">
           <Name lang="fr">Pathologie</Name>
