commit cb98a4638b303dbc159c97a636d6d4f62ca39678
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 6 12:14:41 2022 +0100

    DEC 2022

diff --git a/Rare diseases and functional consequences/it_funct_consequences.xml b/Rare diseases and functional consequences/it_funct_consequences.xml
index 239bfae..583e182 100644
--- a/Rare diseases and functional consequences/it_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/it_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:35:51" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 07:59:58" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -1549,7 +1549,7 @@
       <Disorder id="3716">
         <OrphaCode>503</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=503</ExpertLink>
-        <Name lang="it">Sindrome di Larsen autosomica dominante</Name>
+        <Name lang="it">Sindrome di Larsen</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -20463,7 +20463,7 @@
       <Disorder id="3713">
         <OrphaCode>2542</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2542</ExpertLink>
-        <Name lang="it">Anoftalmia - microftalmia isolata</Name>
+        <Name lang="it">Microftalmia-anoftalmia-coloboma isoltati</Name>
         <DisorderType id="21436">
           <Name lang="it">Gruppo di fenomi</Name>
         </DisorderType>
@@ -168638,7 +168638,7 @@
       <Disorder id="10308">
         <OrphaCode>33108</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=33108</ExpertLink>
-        <Name lang="it">Sindrome letale degli pterigi multipli</Name>
+        <Name lang="it">Sindrome letale da pterigi multipli</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -223518,7 +223518,7 @@
       <Disorder id="4043">
         <OrphaCode>1900</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1900</ExpertLink>
-        <Name lang="it">Sindrome di Ehlers-Danlos, tipo cifoscoliotico</Name>
+        <Name lang="it">Sindrome di Ehlers-Danlos cifoscoliotica da deficit di lisil idrossilasi 1</Name>
         <DisorderType id="21450">
           <Name lang="it">Sottotipo clinico</Name>
         </DisorderType>
@@ -225386,7 +225386,7 @@
       <Disorder id="3480">
         <OrphaCode>2953</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2953</ExpertLink>
-        <Name lang="it">Sindrome di Ehlers-Danlos, tipo muscolo-contratturale</Name>
+        <Name lang="it">Sindrome di Ehlers-Danlos muscolo-contratturale</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -456619,7 +456619,7 @@
       <Disorder id="3035">
         <OrphaCode>3453</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3453</ExpertLink>
-        <Name lang="it">Poliendocrinopatia autoimmune, tipo 1</Name>
+        <Name lang="it">Poliendocrinopatia autoimmune tipo 1</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -480921,7 +480921,7 @@
       <Disorder id="1447">
         <OrphaCode>1190</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1190</ExpertLink>
-        <Name lang="it">Atelosteogenesi 1</Name>
+        <Name lang="it">Atelosteogenesi tipo 1</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
