commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and functional consequences/it_funct_consequences.xml b/Rare diseases and functional consequences/it_funct_consequences.xml
index 12d7ad4..e919071 100644
--- a/Rare diseases and functional consequences/it_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/it_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:13:43" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:13:00" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -349758,7 +349758,7 @@
       <Disorder id="11106">
         <OrphaCode>77293</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=77293</ExpertLink>
-        <Name lang="it">Malattia di Niemann-Pick, tipo B</Name>
+        <Name lang="it">Deficit cronico di sfingomielinasi acida viscerale</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
@@ -350610,7 +350610,7 @@
       <Disorder id="11105">
         <OrphaCode>77292</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=77292</ExpertLink>
-        <Name lang="it">Malattia di Niemann-Pick, tipo A</Name>
+        <Name lang="it">Deficit di sfingomielinasi acida neuroviscerale infantile</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
