commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases and functional consequences/it_funct_consequences.xml b/Rare diseases and functional consequences/it_funct_consequences.xml
index 1edf75f..eedb1b1 100644
--- a/Rare diseases and functional consequences/it_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/it_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:12:30" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:12:28" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="946">
+  <DisorderDisabilityRelevanceList count="945">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -263855,111 +263855,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3584">
-      <Disorder id="11041">
-        <OrphaCode>73247</OrphaCode>
-        <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=73247</ExpertLink>
-        <Name lang="it">Esofagite eosinofila</Name>
-        <DisorderType id="21394">
-          <Name lang="it">Malattia</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="it">Malattia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="5">
-          <DisabilityDisorderAssociation id="26430">
-            <Disability id="69">
-              <Name lang="it">Gestire le emozioni e l'umore</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="it">Frequente</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27194">
-              <Name lang="it">Limitazione/restrizione transitoria</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27236">
-              <Name lang="it">Moderata</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26431">
-            <Disability id="69">
-              <Name lang="it">Gestire le emozioni e l'umore</Name>
-            </Disability>
-            <FrequenceDisability id="27215">
-              <Name lang="it">Occasionale</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="it">Limitazione/restrizione permanente</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="it">Lieve</Name>
-            </SeverityDisability>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26432">
-            <Disability id="110">
-              <Name lang="it">Prendere parte alla vita nella comunità</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="it">Frequente</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="it">Limitazione/restrizione permanente</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="it">Lieve</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26433">
-            <Disability id="112">
-              <Name lang="it">Socializzare</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="it">Frequente</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="it">Limitazione/restrizione permanente</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="it">Lieve</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26434">
-            <Disability id="117">
-              <Name lang="it">Cibo</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Alfredo LUCENDO[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2015-01-10 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="it">Non ancora confermato</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="it">Limitazione di attività/restrizione alla partecipazione</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3585">
       <Disorder id="755">
         <OrphaCode>3165</OrphaCode>
@@ -320113,7 +320008,7 @@
       <Disorder id="5546">
         <OrphaCode>393</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=393</ExpertLink>
-        <Name lang="it">Sindrome del maschio XX</Name>
+        <Name lang="it">Malattia testicolare dello sviluppo sessuale 46,XX</Name>
         <DisorderType id="21401">
           <Name lang="it">Sindrome malformativa</Name>
         </DisorderType>
@@ -449350,7 +449245,7 @@
       <Disorder id="13866">
         <OrphaCode>98849</OrphaCode>
         <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98849</ExpertLink>
-        <Name lang="it">Mastocitosi sistemica associata a emopatia clonale non mastocitaria</Name>
+        <Name lang="it">Mastocitosi sistemica associata a neoplasia ematologica</Name>
         <DisorderType id="21394">
           <Name lang="it">Malattia</Name>
         </DisorderType>
