commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and functional consequences/nl_funct_consequences.xml b/Rare diseases and functional consequences/nl_funct_consequences.xml
index a695275..518b4cf 100644
--- a/Rare diseases and functional consequences/nl_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/nl_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:14:25" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:13:40" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -28271,7 +28271,7 @@
       <Disorder id="2983">
         <OrphaCode>3366</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3366</ExpertLink>
-        <Name lang="nl">Geïsoleerde trigonocefalie</Name>
+        <Name lang="nl">Niet-syndromale metopische craniosynostose</Name>
         <DisorderType id="21415">
           <Name lang="nl">Morfologische anomalie</Name>
         </DisorderType>
@@ -45165,7 +45165,7 @@
       <Disorder id="12438">
         <OrphaCode>93605</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93605</ExpertLink>
-        <Name lang="nl">Klassiek syndroom van Bartter</Name>
+        <Name lang="nl">Syndroom van Bartter type 3</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -121793,7 +121793,7 @@
       <Disorder id="19334">
         <OrphaCode>240103</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=240103</ExpertLink>
-        <Name lang="nl">Progressieve supranucleaire paralyse - corticobasaal parkinsonisme</Name>
+        <Name lang="nl">Progressieve supranucleaire paralyse - corticobasaal syndroom</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -124590,7 +124590,7 @@
       <Disorder id="19333">
         <OrphaCode>240094</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=240094</ExpertLink>
-        <Name lang="nl">Progressieve supranucleaire paralyse - zuivere akinesie met bevroren gang-syndroom</Name>
+        <Name lang="nl">Progressieve supranucleaire paralyse - zuivere akinesie met gangblokkade-syndroom</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -144112,7 +144112,7 @@
       <Disorder id="3249">
         <OrphaCode>478</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=478</ExpertLink>
-        <Name lang="nl">Kallmannsyndroom</Name>
+        <Name lang="nl">Syndroom van Kallmann</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -433001,7 +433001,7 @@
       <Disorder id="11913">
         <OrphaCode>89938</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89938</ExpertLink>
-        <Name lang="nl">Infantiel syndroom van Bartter met sensorineurale doofheid</Name>
+        <Name lang="nl">Syndroom van Bartter type 4</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -469633,7 +469633,7 @@
       <Disorder id="1544">
         <OrphaCode>1321</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1321</ExpertLink>
-        <Name lang="nl">Camptodactylie - hyperplasie van fibreus weefsel - skeletdysplasie-syndroom</Name>
+        <Name lang="nl">Camptodactylie - hyperplasie van fibreus weefsel - skeletanomalieën-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -579968,7 +579968,7 @@
       <Disorder id="2497">
         <OrphaCode>661</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=661</ExpertLink>
-        <Name lang="nl">Syndroom van Ondine</Name>
+        <Name lang="nl">Congenitale centrale hypoventilatie-syndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
