commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and functional consequences/nl_funct_consequences.xml b/Rare diseases and functional consequences/nl_funct_consequences.xml
index 16cae75..210cbb3 100644
--- a/Rare diseases and functional consequences/nl_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/nl_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:12:55" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:13:08" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -195485,7 +195485,7 @@
       <Disorder id="21338">
         <OrphaCode>309025</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=309025</ExpertLink>
-        <Name lang="nl">Mevalonaatkinasedeficiëntie</Name>
+        <Name lang="nl">Deficiëntie van mevalonaatkinase</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -338825,7 +338825,7 @@
       <Disorder id="1679">
         <OrphaCode>1512</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1512</ExpertLink>
-        <Name lang="nl">Crane-Heisesyndroom</Name>
+        <Name lang="nl">Syndroom van Crane-Heise</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -580512,7 +580512,7 @@
       <Disorder id="2497">
         <OrphaCode>661</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=661</ExpertLink>
-        <Name lang="nl">Ondinesyndroom</Name>
+        <Name lang="nl">Syndroom van Ondine</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
