commit cb98a4638b303dbc159c97a636d6d4f62ca39678
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 6 12:14:41 2022 +0100

    DEC 2022

diff --git a/Rare diseases and functional consequences/nl_funct_consequences.xml b/Rare diseases and functional consequences/nl_funct_consequences.xml
index f030469..4057623 100644
--- a/Rare diseases and functional consequences/nl_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/nl_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-14 16:36:37" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-11-28 08:00:50" version="1.3.18 / 4.1.7 [2022-07-12] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -20326,7 +20326,7 @@
       <Disorder id="19764">
         <OrphaCode>254693</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254693</ExpertLink>
-        <Name lang="nl">Partiële mola hydatidiformis</Name>
+        <Name lang="nl">Partiële mola hydatidosa</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -53849,7 +53849,7 @@
       <Disorder id="10801">
         <OrphaCode>60032</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=60032</ExpertLink>
-        <Name lang="nl">Terugkerende respiratoire papillomatose</Name>
+        <Name lang="nl">Recidiverende respiratoire papillomatose</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -77544,7 +77544,7 @@
       <Disorder id="17766">
         <OrphaCode>168491</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=168491</ExpertLink>
-        <Name lang="nl">Laat infantiele neuronale ceroïdlipofuscinose</Name>
+        <Name lang="nl">Laat-infantiele neuronale ceroïdlipofuscinose</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -120037,7 +120037,7 @@
       <Disorder id="19763">
         <OrphaCode>254688</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254688</ExpertLink>
-        <Name lang="nl">Volledige hydatidiforme mola</Name>
+        <Name lang="nl">Complete mola hydatidosa</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -209219,7 +209219,7 @@
       <Disorder id="14485">
         <OrphaCode>99912</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99912</ExpertLink>
-        <Name lang="nl">Maligne dysgerminomateuze kiemceltumor van het ovarium</Name>
+        <Name lang="nl">Maligne dysgerminomateuze kiemceltumor van ovarium</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -218984,7 +218984,7 @@
       <Disorder id="18908">
         <OrphaCode>220436</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=220436</ExpertLink>
-        <Name lang="nl">Quebec bloedplaatjesstoornis</Name>
+        <Name lang="nl">Quebec-bloedplaatjesstoornis</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -219790,7 +219790,7 @@
       <Disorder id="10999">
         <OrphaCode>71277</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=71277</ExpertLink>
-        <Name lang="nl">Klassiek glucosetransporter type 1-deficiëntiesyndroom</Name>
+        <Name lang="nl">Klassiek glucosetransporter type 1-deficiëntie-syndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -314840,7 +314840,7 @@
       <Disorder id="3631">
         <OrphaCode>809</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=809</ExpertLink>
-        <Name lang="nl">Gemengd bindweefselziekte</Name>
+        <Name lang="nl">Gemengde bindweefselziekte</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -397049,7 +397049,7 @@
       <Disorder id="12700">
         <OrphaCode>96076</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96076</ExpertLink>
-        <Name lang="nl">Beckwith-Wiedemannsyndroom als gevolg van 11p15 microduplicatie</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door 11p15-microduplicatie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -397222,7 +397222,7 @@
       <Disorder id="19282">
         <OrphaCode>238613</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238613</ExpertLink>
-        <Name lang="nl">Syndroom van Beckwith-Wiedemann als gevolg van NSD1-mutatie</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door NSD1-mutatie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -397395,7 +397395,7 @@
       <Disorder id="19155">
         <OrphaCode>231117</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231117</ExpertLink>
-        <Name lang="nl">Beckwith-Wiedemannsyndroom als gevolg van imprintingdefect van 11p15</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door imprintingdefect van 11p15</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -397568,7 +397568,7 @@
       <Disorder id="19159">
         <OrphaCode>231130</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231130</ExpertLink>
-        <Name lang="nl">Beckwith-Wiedemannsyndroom als gevolg van 11p15 translocatie/inversie</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door 11p15-translocatie/inversie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -397741,7 +397741,7 @@
       <Disorder id="19156">
         <OrphaCode>231120</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231120</ExpertLink>
-        <Name lang="nl">Beckwith-Wiedemannsyndroom als gevolg van CDKN1C-mutatie</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door CDKN1C-mutatie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -399105,7 +399105,7 @@
       <Disorder id="12817">
         <OrphaCode>96193</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96193</ExpertLink>
-        <Name lang="nl">Syndroom van Beckwith-Wiedemann als gevolg van paternale uniparentale disomie van chromosoom 11</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door paternale uniparentale disomie van chromosoom 11</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -441705,7 +441705,7 @@
       <Disorder id="806">
         <OrphaCode>3287</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3287</ExpertLink>
-        <Name lang="nl">Takayasu-arteritis</Name>
+        <Name lang="nl">Takayasu-arteriitis</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -442603,7 +442603,7 @@
       <Disorder id="3401">
         <OrphaCode>3162</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3162</ExpertLink>
-        <Name lang="nl">Sezarysyndroom</Name>
+        <Name lang="nl">Syndroom van Sézary</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -459590,7 +459590,7 @@
       <Disorder id="3233">
         <OrphaCode>3206</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3206</ExpertLink>
-        <Name lang="nl">Syndroom van Stuve-Wiedemann</Name>
+        <Name lang="nl">Syndroom van Stüve-Wiedemann</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -463722,7 +463722,7 @@
       <Disorder id="12138">
         <OrphaCode>91387</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91387</ExpertLink>
-        <Name lang="nl">Familiale aneurysma van aorta thoracica en aortadissectie</Name>
+        <Name lang="nl">Familiaal aneurysma van aorta thoracica en aortadissectie</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -510077,7 +510077,7 @@
       <Disorder id="724">
         <OrphaCode>1199</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1199</ExpertLink>
-        <Name lang="nl">Oesofageale atresie</Name>
+        <Name lang="nl">Atresie van oesofagus</Name>
         <DisorderType id="21415">
           <Name lang="nl">Morfologische anomalie</Name>
         </DisorderType>
@@ -542898,7 +542898,7 @@
       <Disorder id="14489">
         <OrphaCode>99916</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99916</ExpertLink>
-        <Name lang="nl">Maligne Sertoli-Leydigceltumor van het ovarium</Name>
+        <Name lang="nl">Maligne Sertoli-Leydigceltumor van ovarium</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
