commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases and functional consequences/nl_funct_consequences.xml b/Rare diseases and functional consequences/nl_funct_consequences.xml
index 210cbb3..fa46b6f 100644
--- a/Rare diseases and functional consequences/nl_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/nl_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:13:08" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:13:06" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="946">
+  <DisorderDisabilityRelevanceList count="945">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -119909,7 +119909,7 @@
       <Disorder id="14212">
         <OrphaCode>99429</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99429</ExpertLink>
-        <Name lang="nl">Compleet androgeenongevoeligheidssyndroom</Name>
+        <Name lang="nl">Compleet androgeeninsensitiviteitssyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -232913,7 +232913,7 @@
       <Disorder id="14540">
         <OrphaCode>99967</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99967</ExpertLink>
-        <Name lang="nl">Myxoïd/ronde cel liposarcoom</Name>
+        <Name lang="nl">Myxoïd/rondcellig liposarcoom</Name>
         <DisorderType id="21457">
           <Name lang="nl">Histopathologisch subtype</Name>
         </DisorderType>
@@ -263855,111 +263855,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3584">
-      <Disorder id="11041">
-        <OrphaCode>73247</OrphaCode>
-        <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=73247</ExpertLink>
-        <Name lang="nl">Eosinofiele oesofagitis</Name>
-        <DisorderType id="21394">
-          <Name lang="nl">Ziekte</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="nl">Aandoening</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="5">
-          <DisabilityDisorderAssociation id="26430">
-            <Disability id="69">
-              <Name lang="nl">Omgaan met emoties en stemming</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="nl">Vaak</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27194">
-              <Name lang="nl">Tijdelijke beperking</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27236">
-              <Name lang="nl">Matig</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26431">
-            <Disability id="69">
-              <Name lang="nl">Omgaan met emoties en stemming</Name>
-            </Disability>
-            <FrequenceDisability id="27215">
-              <Name lang="nl">Soms</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="nl">Permanente beperking</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="nl">Laag</Name>
-            </SeverityDisability>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26432">
-            <Disability id="110">
-              <Name lang="nl">Deelnemen aan het gemeenschapsleven</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="nl">Vaak</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="nl">Permanente beperking</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="nl">Laag</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26433">
-            <Disability id="112">
-              <Name lang="nl">Socialiseren</Name>
-            </Disability>
-            <FrequenceDisability id="27208">
-              <Name lang="nl">Vaak</Name>
-            </FrequenceDisability>
-            <TemporalityDisability id="27180">
-              <Name lang="nl">Permanente beperking</Name>
-            </TemporalityDisability>
-            <SeverityDisability id="27222">
-              <Name lang="nl">Laag</Name>
-            </SeverityDisability>
-            <LossOfAbility>n</LossOfAbility>
-            <Type>Disability</Type>
-            <Defined>y</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="26434">
-            <Disability id="117">
-              <Name lang="nl">Voedsel</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Alfredo LUCENDO[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2015-01-10 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="nl">Nog niet gevalideerd</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="nl">Activiteitsbeperking/deelnamebeperking</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3585">
       <Disorder id="755">
         <OrphaCode>3165</OrphaCode>
@@ -303997,7 +303892,7 @@
       <Disorder id="12546">
         <OrphaCode>94065</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94065</ExpertLink>
-        <Name lang="nl">15q24 microdeletiesyndroom</Name>
+        <Name lang="nl">15q24-microdeletiesyndroom</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -387754,7 +387649,7 @@
       <Disorder id="20581">
         <OrphaCode>284160</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284160</ExpertLink>
-        <Name lang="nl">8q21.11 microdeletiesyndroom</Name>
+        <Name lang="nl">8q21.11-microdeletiesyndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -388893,7 +388788,7 @@
       <Disorder id="12716">
         <OrphaCode>96092</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96092</ExpertLink>
-        <Name lang="nl">8p inversie duplicatie/deletie-syndroom</Name>
+        <Name lang="nl">8p geïnverteerde duplicatie/deletie-syndroom</Name>
         <DisorderType id="21401">
           <Name lang="nl">Malformatiesyndroom</Name>
         </DisorderType>
@@ -397420,7 +397315,7 @@
       <Disorder id="19158">
         <OrphaCode>231127</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231127</ExpertLink>
-        <Name lang="nl">Beckwith-Wiedemannsyndroom als gevolg van 11p15 microdeletie</Name>
+        <Name lang="nl">Syndroom van Beckwith-Wiedemann door 11p15-microdeletie</Name>
         <DisorderType id="21443">
           <Name lang="nl">Etiologisch subtype</Name>
         </DisorderType>
@@ -420104,7 +419999,7 @@
       <Disorder id="12100">
         <OrphaCode>90797</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90797</ExpertLink>
-        <Name lang="nl">Partieel androgeenongevoeligheidssyndroom</Name>
+        <Name lang="nl">Partieel androgeeninsensitiviteitssyndroom</Name>
         <DisorderType id="21394">
           <Name lang="nl">Ziekte</Name>
         </DisorderType>
@@ -494437,7 +494332,7 @@
       <Disorder id="19171">
         <OrphaCode>231214</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231214</ExpertLink>
-        <Name lang="nl">Bèta-thalassemie major</Name>
+        <Name lang="nl">Beta-thalassemia major</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
@@ -502340,7 +502235,7 @@
       <Disorder id="12488">
         <OrphaCode>93928</OrphaCode>
         <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93928</ExpertLink>
-        <Name lang="nl">Geïsoleerde epispadieën</Name>
+        <Name lang="nl">Geïsoleerde epispadie</Name>
         <DisorderType id="21450">
           <Name lang="nl">Klinisch subtype</Name>
         </DisorderType>
