commit 6eb274384f0c4cd30cc83f08ce097956913930ce
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 4 08:42:13 2024 +0100

    DEC 2024

diff --git a/Rare diseases and functional consequences/pt_funct_consequences.xml b/Rare diseases and functional consequences/pt_funct_consequences.xml
index 1e53899..ec42378 100644
--- a/Rare diseases and functional consequences/pt_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/pt_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2024-07-01 09:44:24" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2024-12-03 07:34:51" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2024" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="1052">
+  <DisorderDisabilityRelevanceList count="1046">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -100934,54 +100934,6 @@
         <Name lang="pt">funcionalidade muito variável???</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="1438">
-      <Disorder id="12030">
-        <OrphaCode>90362</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=90362</ExpertLink>
-        <Name lang="pt">Linfangiectasia intestinal primária</Name>
-        <DisorderType id="21394">
-          <Name lang="pt">Doença</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="pt">Patologia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="2">
-          <DisabilityDisorderAssociation id="189671">
-            <Disability id="117">
-              <Name lang="pt">Comida</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-          <DisabilityDisorderAssociation id="189672">
-            <Disability id="119">
-              <Name lang="pt">Temperatura</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Stephane VIGNES[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2023-11-01 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="pt">Ainda não validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="pt">limitação na atividade /restrição na participação</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="1413">
       <Disorder id="1784">
         <OrphaCode>1782</OrphaCode>
@@ -206876,34 +206828,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3066">
-      <Disorder id="890">
-        <OrphaCode>1935</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1935</ExpertLink>
-        <Name lang="pt">Encefalopatia mioclónica precoce</Name>
-        <DisorderType id="21422">
-          <Name lang="pt">Síndrome clínica</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="pt">Patologia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2022-01-10 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="pt">Validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="pt">Não aplicável</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27306">
-        <Name lang="pt">funcionalidade muito variável???</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3057">
       <Disorder id="12584">
         <OrphaCode>95409</OrphaCode>
@@ -222406,32 +222330,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="3125">
-      <Disorder id="18313">
-        <OrphaCode>183713</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=183713</ExpertLink>
-        <Name lang="pt">Infeções bacterianas piogénicas por deficiência de MyD88</Name>
-        <DisorderType id="21394">
-          <Name lang="pt">Doença</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="pt">Patologia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Expert</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2012-11-26 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="pt">Ainda não validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27292">
-        <Name lang="pt">sem incapacidade funcional</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="3118">
       <Disorder id="2701">
         <OrphaCode>3003</OrphaCode>
@@ -326082,43 +325980,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="4980">
-      <Disorder id="3631">
-        <OrphaCode>809</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=809</ExpertLink>
-        <Name lang="pt">Doença mista do tecido conjuntivo</Name>
-        <DisorderType id="21394">
-          <Name lang="pt">Doença</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="pt">Patologia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="1">
-          <DisabilityDisorderAssociation id="189130">
-            <Disability id="119">
-              <Name lang="pt">Temperatura</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Ragnar GUNNARSSON[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2023-11-01 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="pt">Ainda não validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="pt">limitação na atividade /restrição na participação</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="5001">
       <Disorder id="225">
         <OrphaCode>912</OrphaCode>
@@ -381186,32 +381047,6 @@
       </DisabilityCategory>
       <ReasonForNotApplicable/>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="5298">
-      <Disorder id="18899">
-        <OrphaCode>218439</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=218439</ExpertLink>
-        <Name lang="pt">Doença do ritmo cardíaco, adquirida</Name>
-        <DisorderType id="36561">
-          <Name lang="pt">Category</Name>
-        </DisorderType>
-        <DisorderGroup id="36540">
-          <Name lang="pt">Grupo de patologias</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Expert</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2014-02-06 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="pt">Ainda não validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27292">
-        <Name lang="pt">sem incapacidade funcional</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="5302">
       <Disorder id="8618">
         <OrphaCode>658</OrphaCode>
@@ -419868,7 +419703,7 @@
       <Disorder id="10638">
         <OrphaCode>49042</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=49042</ExpertLink>
-        <Name lang="pt">Dentinogénese imperfeita</Name>
+        <Name lang="pt">Dentinogenesis imperfecta</Name>
         <DisorderType id="21394">
           <Name lang="pt">Doença</Name>
         </DisorderType>
@@ -427674,7 +427509,7 @@
       <Disorder id="22639">
         <OrphaCode>391665</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=391665</ExpertLink>
-        <Name lang="pt">Hipercolesterolemia familiar, em homozigotia</Name>
+        <Name lang="pt">Hipercolesterolemia familiar homozigótica</Name>
         <DisorderType id="21394">
           <Name lang="pt">Doença</Name>
         </DisorderType>
@@ -483022,43 +482857,6 @@
         <Name lang="pt">causas de morte precoce</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="8116">
-      <Disorder id="3298">
-        <OrphaCode>832</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=832</ExpertLink>
-        <Name lang="pt">Deficiência de succinil-CoA:3 oxoácida CoA transferase</Name>
-        <DisorderType id="21394">
-          <Name lang="pt">Doença</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="pt">Patologia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="1">
-          <DisabilityDisorderAssociation id="89800">
-            <Disability id="117">
-              <Name lang="pt">Comida</Name>
-            </Disability>
-            <FrequenceDisability/>
-            <TemporalityDisability/>
-            <SeverityDisability/>
-            <LossOfAbility>y</LossOfAbility>
-            <Type>Environmental factor</Type>
-            <Defined>n</Defined>
-          </DisabilityDisorderAssociation>
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Dr Toshiyuki FUKAO[Expert]</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2018-11-29 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27327">
-        <Name lang="pt">Ainda não validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27278">
-        <Name lang="pt">limitação na atividade /restrição na participação</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable/>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="8030">
       <Disorder id="3568">
         <OrphaCode>175</OrphaCode>
