commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Rare diseases and functional consequences/pt_funct_consequences.xml b/Rare diseases and functional consequences/pt_funct_consequences.xml
old mode 100644
new mode 100755
index fe0003d..412151c
--- a/Rare diseases and functional consequences/pt_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/pt_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:09:35" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:09:51" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -230021,8 +230021,8 @@
         <OrphaCode>98791</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=98791</ExpertLink>
         <Name lang="pt">Síndrome de alfa talassemia - défice intelectual ligado ao cromossoma 16</Name>
-        <DisorderType id="21394">
-          <Name lang="pt">Doença</Name>
+        <DisorderType id="21401">
+          <Name lang="pt">Síndrome malformativo</Name>
         </DisorderType>
         <DisorderGroup id="36547">
           <Name lang="pt">Patologia</Name>
