commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Rare diseases and functional consequences/pt_funct_consequences.xml b/Rare diseases and functional consequences/pt_funct_consequences.xml
index 7145d1f..fe0003d 100644
--- a/Rare diseases and functional consequences/pt_funct_consequences.xml	
+++ b/Rare diseases and functional consequences/pt_funct_consequences.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:09:27" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:09:35" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderDisabilityRelevanceList count="910">
+  <DisorderDisabilityRelevanceList count="909">
     <DisorderDisabilityRelevance id="5">
       <Disorder id="230">
         <OrphaCode>893</OrphaCode>
@@ -18552,7 +18552,7 @@
       <Disorder id="10319">
         <OrphaCode>33364</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=33364</ExpertLink>
-        <Name lang="pt">Tricotiodistrofia (termo genérico)</Name>
+        <Name lang="pt">Tricotiodistrofia</Name>
         <DisorderType id="21394">
           <Name lang="pt">Doença</Name>
         </DisorderType>
@@ -98928,7 +98928,7 @@
       <Disorder id="11097">
         <OrphaCode>77240</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=77240</ExpertLink>
-        <Name lang="pt">Linfedema primário (termo genérico)</Name>
+        <Name lang="pt">Linfedema primário</Name>
         <DisorderType id="36561">
           <Name lang="pt">Category</Name>
         </DisorderType>
@@ -164354,34 +164354,6 @@
         <Name lang="pt">causas de morte precoce</Name>
       </ReasonForNotApplicable>
     </DisorderDisabilityRelevance>
-    <DisorderDisabilityRelevance id="2318">
-      <Disorder id="413">
-        <OrphaCode>1046</OrphaCode>
-        <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1046</ExpertLink>
-        <Name lang="pt">Anemia hemolítica letal - anomalias genitais</Name>
-        <DisorderType id="21401">
-          <Name lang="pt">Síndrome malformativo</Name>
-        </DisorderType>
-        <DisorderGroup id="36547">
-          <Name lang="pt">Patologia</Name>
-        </DisorderGroup>
-        <DisabilityDisorderAssociationList count="0">
-        </DisabilityDisorderAssociationList>
-      </Disorder>
-      <SourceOfValidation>Orphanet</SourceOfValidation>
-      <SpecificManagement>n</SpecificManagement>
-      <Online>y</Online>
-      <AnnotationDate>2014-11-21 00:00:00.0</AnnotationDate>
-      <StatusDisability id="27320">
-        <Name lang="pt">Validado</Name>
-      </StatusDisability>
-      <DisabilityCategory id="27285">
-        <Name lang="pt">Não aplicável</Name>
-      </DisabilityCategory>
-      <ReasonForNotApplicable id="27299">
-        <Name lang="pt">causas de morte precoce</Name>
-      </ReasonForNotApplicable>
-    </DisorderDisabilityRelevance>
     <DisorderDisabilityRelevance id="2316">
       <Disorder id="21964">
         <OrphaCode>330050</OrphaCode>
@@ -226737,8 +226709,8 @@
         <OrphaCode>847</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=847</ExpertLink>
         <Name lang="pt">Alfa talassemia - atraso mental, ligada ao X</Name>
-        <DisorderType id="21394">
-          <Name lang="pt">Doença</Name>
+        <DisorderType id="21401">
+          <Name lang="pt">Síndrome malformativo</Name>
         </DisorderType>
         <DisorderGroup id="36547">
           <Name lang="pt">Patologia</Name>
@@ -433424,7 +433396,7 @@
       <Disorder id="124">
         <OrphaCode>536</OrphaCode>
         <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=536</ExpertLink>
-        <Name lang="pt">Lúpus eritematoso</Name>
+        <Name lang="pt">Lúpus eritematoso sistémico</Name>
         <DisorderType id="21394">
           <Name lang="pt">Doença</Name>
         </DisorderType>
