commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/de_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/de_product1.xml
old mode 100755
new mode 100644
index f02d901..d11058b
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/de_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/de_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:28:32" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:31:00" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10542">
+  <DisorderList count="10543">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166024</ExpertLink>
@@ -9763,7 +9763,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="97">
@@ -71852,9 +71859,9 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="121079">
+        <ExternalReference id="188714">
           <Source>ICD-10</Source>
-          <Reference>F01.1</Reference>
+          <Reference>I67.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
@@ -86665,7 +86672,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1018">
@@ -87812,9 +87826,9 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="107737">
+        <ExternalReference id="188713">
           <Source>ICD-10</Source>
-          <Reference>F01.1</Reference>
+          <Reference>I67.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
@@ -283092,7 +283106,7 @@
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="de">Es handelt sich bei diesem Eintrag um eine seltene Kondition, für dessen Behandlung ein Medikament mit europäischer und/oder amerikanischer Orphan-Drug-Designation existiert.Dieser Terminus beschreibt die Indikation für einen pharmakogenetischen Test.</Info>
+            <Info lang="de">Es handelt sich bei diesem Eintrag um eine seltene Kondition, für dessen Behandlung ein Medikament mit europäischer und/oder amerikanischer Orphan-Drug-Designation existiert.</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -300203,7 +300217,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19272">
@@ -300292,7 +300313,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19287">
@@ -306792,7 +306820,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21811">
@@ -342641,7 +342676,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20523">
@@ -342808,7 +342850,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20532">
@@ -343156,7 +343205,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20538">
@@ -343181,7 +343237,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20539">
@@ -343206,7 +343269,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20536">
@@ -343231,7 +343301,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20537">
@@ -354712,7 +354789,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21349">
@@ -362475,14 +362559,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="de">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="de">Dieser Terminus beschreibt die Indikation für einen pharmakogenetischen Test.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21116">
@@ -363791,7 +363868,7 @@
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="de">Es handelt sich bei diesem Eintrag um eine seltene Kondition, für dessen Behandlung ein Medikament mit europäischer und/oder amerikanischer Orphan-Drug-Designation existiert.Dieser Terminus beschreibt die Indikation für einen pharmakogenetischen Test.</Info>
+            <Info lang="de">Es handelt sich bei diesem Eintrag um eine seltene Kondition, für dessen Behandlung ein Medikament mit europäischer und/oder amerikanischer Orphan-Drug-Designation existiert.</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -399298,7 +399375,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23179">
@@ -401537,17 +401621,17 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179672">
+        <ExternalReference id="188684">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>G11.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -407221,7 +407305,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="154602">
+        <ExternalReference id="188685">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -407230,8 +407314,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="158701">
@@ -410411,17 +410495,17 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179670">
+        <ExternalReference id="188682">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>Q07.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -413412,7 +413496,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179671">
+        <ExternalReference id="188683">
           <Source>ICD-10</Source>
           <Reference>G71.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -413421,8 +413505,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="156032">
@@ -428693,19 +428777,6 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="3">
-        <ExternalReference id="114885">
-          <Source>ICD-10</Source>
-          <Reference>G93.4</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="de">Validiert</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
         <ExternalReference id="16869">
           <Source>OMIM</Source>
           <Reference>606777</Reference>
@@ -428728,6 +428799,19 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
+        <ExternalReference id="188712">
+          <Source>ICD-10</Source>
+          <Reference>G40.4</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
@@ -429239,7 +429323,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179634">
+        <ExternalReference id="188705">
           <Source>ICD-10</Source>
           <Reference>K31.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -429248,8 +429332,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -444009,7 +444093,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="178532">
+        <ExternalReference id="188706">
           <Source>ICD-10</Source>
           <Reference>E75.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -444018,8 +444102,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -444157,17 +444241,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179635">
+        <ExternalReference id="188707">
           <Source>ICD-10</Source>
-          <Reference>K59.9</Reference>
+          <Reference>K59.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -446261,7 +446345,20 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188709">
+          <Source>ICD-10</Source>
+          <Reference>E07.8</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
@@ -446362,7 +446459,7 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178095">
+        <ExternalReference id="188708">
           <Source>ICD-10</Source>
           <Reference>E07.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -446371,8 +446468,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="172336">
@@ -449317,7 +449414,7 @@
     <Disorder id="28590">
       <OrphaCode>567548</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567548</ExpertLink>
-      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom</Name>
+      <Name lang="de">Idiopathisches Steroid-resistentes nephrotisches Syndrom</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -465475,7 +465572,7 @@
         <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179602">
+        <ExternalReference id="188721">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -465484,8 +465581,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -465762,7 +465859,7 @@
         <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179603">
+        <ExternalReference id="188717">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -465771,8 +465868,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -465966,7 +466063,7 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179604">
+        <ExternalReference id="188687">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -465975,8 +466072,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -466087,7 +466184,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179605">
+        <ExternalReference id="188688">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -466096,8 +466193,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -467520,17 +467617,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179607">
+        <ExternalReference id="188690">
           <Source>ICD-10</Source>
-          <Reference>D23.2</Reference>
+          <Reference>D23.9</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163484">
@@ -467591,7 +467688,7 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179606">
+        <ExternalReference id="188689">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -467600,8 +467697,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -468149,7 +468246,20 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
+      <ExternalReferenceList count="2">
+        <ExternalReference id="188710">
+          <Source>ICD-10</Source>
+          <Reference>D82.3</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21590">
+            <Name lang="de">Einschlussbegriff (Der ORPHA-Code ist unter einer ICD10-Kategorie eingeschlossen und hat keinen eigenen Code)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
         <ExternalReference id="163478">
           <Source>OMIM</Source>
           <Reference>300635</Reference>
@@ -468198,7 +468308,20 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188711">
+          <Source>ICD-10</Source>
+          <Reference>D89.8</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -468233,7 +468356,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179612">
+        <ExternalReference id="188691">
           <Source>ICD-10</Source>
           <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -468242,8 +468365,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163477">
@@ -469734,17 +469857,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179613">
+        <ExternalReference id="188692">
           <Source>ICD-10</Source>
-          <Reference>D81.1</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163479">
@@ -469851,17 +469974,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179614">
+        <ExternalReference id="188693">
           <Source>ICD-10</Source>
-          <Reference>D81.1</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163481">
@@ -472594,17 +472717,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179615">
+        <ExternalReference id="188694">
           <Source>ICD-10</Source>
-          <Reference>D81.9</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="172005">
@@ -473596,17 +473719,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179618">
+        <ExternalReference id="188695">
           <Source>ICD-10</Source>
-          <Reference>G71.3</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -474130,7 +474253,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179619">
+        <ExternalReference id="188696">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -474139,8 +474262,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -475368,7 +475491,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179622">
+        <ExternalReference id="188699">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -475377,8 +475500,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171455">
@@ -475644,17 +475767,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179621">
+        <ExternalReference id="188698">
           <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
+          <Reference>E72.4</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -475682,17 +475805,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179620">
+        <ExternalReference id="188697">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>Q07.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171424">
@@ -476082,7 +476205,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179623">
+        <ExternalReference id="188700">
           <Source>ICD-10</Source>
           <Reference>G71.2</Reference>
           <DisorderMappingRelation id="21534">
@@ -476091,8 +476214,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171478">
@@ -476366,17 +476489,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179624">
+        <ExternalReference id="188701">
           <Source>ICD-10</Source>
-          <Reference>G31.8</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171468">
@@ -477036,7 +477159,7 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179627">
+        <ExternalReference id="188702">
           <Source>ICD-10</Source>
           <Reference>Q22.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -477045,8 +477168,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -477630,17 +477753,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179632">
+        <ExternalReference id="188704">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="179040">
@@ -477679,17 +477802,17 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179631">
+        <ExternalReference id="188703">
           <Source>ICD-10</Source>
-          <Reference>G11.1</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="de">Noch nicht validiert</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -503250,7 +503373,7 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="117037">
+        <ExternalReference id="188715">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -507198,7 +507321,20 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188719">
+          <Source>ICD-10</Source>
+          <Reference>D84.1</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -507274,7 +507410,20 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188720">
+          <Source>ICD-10</Source>
+          <Reference>D84.1</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="de">NTBT (der im ORPHA-Code verwendete Narrower Term wird Zuordnung auf einen Broader Term abgebildet)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="de">Zugewiesen (Der ICD10-Code wird von Orphanet zugewiesen)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="de">Validiert</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -557689,6 +557838,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29899">
+      <OrphaCode>599082</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599082</ExpertLink>
+      <Name lang="de">CHD3-related developmental delay-speech delay-intellectual disability-abnormalities of vision-facial dysmorphism syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21401">
+        <Name lang="de">Fehlbildungs-Syndrom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12296">
       <OrphaCode>93367</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93367</ExpertLink>
@@ -597980,7 +598154,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="de">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="de">Dieser Terminus bezieht sich auf eine Krankheitsgruppe. Bitte nutzen Sie das Menu Klassifikation, um mehr über die Krankheiten zu erfahren, die sich unter diesem Sammelbegriff zusammenfassen lassen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28746">
@@ -612114,7 +612295,7 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="119528">
+        <ExternalReference id="188716">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -614678,7 +614859,7 @@
             <Name lang="de">Validiert</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="119519">
+        <ExternalReference id="188718">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
