commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml
index 0986113..4e40e29 100644
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:37:21" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:37:33" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10617">
+  <DisorderList count="10619">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -23908,8 +23908,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
+      <SynonymList count="4">
         <Synonym lang="en">Cerebrohepatorenal syndrome</Synonym>
+        <Synonym lang="en">Severe PBD-ZSD</Synonym>
+        <Synonym lang="en">Severe peroxisome biogenesis disorder-Zellweger spectrum disorder</Synonym>
         <Synonym lang="en">ZS</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -25371,8 +25373,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">Mild factor VIII deficiency</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="en">Mild congenital F8 deficiency</Synonym>
+        <Synonym lang="en">Mild congenital factor VIII deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
@@ -25487,8 +25490,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">Severe factor VIII deficiency</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="en">Severe congenital F8 deficiency</Synonym>
+        <Synonym lang="en">Severe congenital factor VIII deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
@@ -25558,8 +25562,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">Moderate factor VIII deficiency</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="en">Moderate congenital F8 deficiency</Synonym>
+        <Synonym lang="en">Moderate congenital factor VIII deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
@@ -28120,8 +28125,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">Mild factor IX deficiency</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="en">Mild congenital F9 deficiency</Synonym>
+        <Synonym lang="en">Mild congenital factor IX deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
@@ -28180,8 +28186,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">Moderate factor IX deficiency</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="en">Moderate congenital F9 deficiency</Synonym>
+        <Synonym lang="en">Moderate congenital factor IX deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
@@ -28240,8 +28247,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">Severe factor IX deficiency</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="en">Severe congenital F9 deficiency</Synonym>
+        <Synonym lang="en">Severe congenital factor IX deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
@@ -37139,7 +37147,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
+        <Synonym lang="en">Intermediate PBD-ZSD</Synonym>
+        <Synonym lang="en">Intermediate peroxisome biogenesis disorder-Zellweger spectrum disorder</Synonym>
         <Synonym lang="en">NALD</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -64593,7 +64603,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Symptomatic hemophilia B in female carriers is a form of hemophilia B (see this term) that manifests in some women with mutations in the &lt;i&gt;F9&lt;/i&gt; gene (Xq28), encoding coagulation factor IX.</Contents>
+              <Contents>Symptomatic hemophilia B in female carriers is a form of hemophilia B (see this term) that manifests in some women with mutations in the &lt;i&gt;F9&lt;/i&gt; gene (Xq27.1), encoding coagulation factor IX.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -64634,14 +64644,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="655">
@@ -65673,7 +65676,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Congenital adrenal hyperplasia (CAH) is an inherited endocrine disorder caused by a steroidogenic enzyme deficiency that is characterized by adrenal insufficiency and variable degrees of hyper or hypo androgeny manifestations, depending of the type and the severity of the disease.</Contents>
+              <Contents>A group of rare inherited endocrine disorders caused by a steroidogenic enzyme deficiency and characterized by adrenal insufficiency and variable degrees of hyper- or hypoandrogenism manifestations, depending on disease type and severity.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -75499,10 +75502,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="4">
+      <SynonymList count="3">
         <Synonym lang="en">Autosomal dominant PHA1</Synonym>
         <Synonym lang="en">Autosomal dominant pseudohypoaldosteronism type 1</Synonym>
-        <Synonym lang="en">PHA1 rénal</Synonym>
         <Synonym lang="en">Renal PHA1</Synonym>
       </SynonymList>
       <DisorderType id="21450">
@@ -85877,7 +85879,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, acquired, endocrine disease characterized by the triad of diffuse skin and mucosa hyperpigmentation, markedly elevated serum adrenocorticotropin (ACTH) levels and an enlarging corticotroph adenoma, which manifest following total bilateral adrenalectomy performed for the treatment of Cushing's disease. Additionally, patients may present with headaches, visual field defects, cranial nerve palsy, pituitary apoplexy, diabetes inspidus, panhypopituitarism, and, occasionally, paraovarian or paratesticular tumors.</Contents>
+              <Contents>A rare, acquired, endocrine disease characterized by the triad of diffuse skin and mucosa hyperpigmentation, markedly elevated serum adrenocorticotropin (ACTH) levels and an enlarging corticotroph adenoma, which manifest following total bilateral adrenalectomy performed for the treatment of Cushing's disease. Additionally, patients may present with headaches, visual field defects, cranial nerve palsy, pituitary apoplexy, diabetes insipidus, panhypopituitarism, and, occasionally, paraovarian or paratesticular tumors.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -122886,7 +122888,7 @@
     <Disorder id="16886">
       <OrphaCode>139402</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139402</ExpertLink>
-      <Name lang="en">Drug rash with eosinophilia and systemic symptoms</Name>
+      <Name lang="en">Drug reaction with eosinophilia and systemic symptoms</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -122895,7 +122897,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="en">DRESS syndrome</Synonym>
-        <Synonym lang="en">Drug reaction eosinophilic systemic syndrome</Synonym>
+        <Synonym lang="en">Drug rash with eosinophilia and systemic symptoms</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -145135,7 +145137,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Camptodactyly - fibrous tissue hyperplasia - skeletal dysplasia syndrome is an extremely rare chondrodysplastic malformation syndrome that is characterized by the combination of arachnodactyly, becoming evident at around the age of 10, camptodactyly (hammertoes) and scoliosis. A mild facial dysmorphism including a broad nose and flaring nostrils, and a mild intellectual disability were also noted. Camptodactyly - fibrous tissue hyperplasia - skeletal dysplasia syndrome has been described once in 3 siblings and is suspected to follow autosomal recessive transmission. There have been no further descriptions in the literature since 1972.</Contents>
+              <Contents>An extremely rare chondrodysplastic malformation syndrome characterized by the combination of arachnodactyly, becoming evident at around the age of 10, camptodactyly, and scoliosis. Additional reported manifestations include a mild intellectual disability and a mild facial dysmorphism including a broad nose and flaring nostrils. There have been no further descriptions in the literature since 1972.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -151977,7 +151979,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare syndrome syndrome characterized by neonatal blisters and milia (small white papules, especially on the face) and congenital absence of dermatoglyphics on the hands and feet. It has been reported in two kindreds (one of which contained 13 affected individuals spanning three generations) and in an unrelated individual. Some affected patients also showed bilateral partial flexion contractures of the fingers and toes, and webbing of the toes. The syndrome is inherited as an autosomal dominant trait.</Contents>
+              <Contents>A rare syndrome characterized by neonatal blisters and milia (small white papules, especially on the face) and congenital absence of dermatoglyphics on the hands and feet. It has been reported in two kindreds (one of which contained 13 affected individuals spanning three generations) and in an unrelated individual. Some affected patients also showed bilateral partial flexion contractures of the fingers and toes, and webbing of the toes. The syndrome is inherited as an autosomal dominant trait.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -156311,7 +156313,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare otorhinolaryngological malformation characterized by the presence of a cyst, sinus or fistula occuring along the anterior border of the sternocleidomastoid muscle. Second branchial cleft fistulae ans sinuses present with skin opening with chronic discharge and recurrent infections, whereas second branchial cleft cysts present as a painless, nontender, stable in size or slowly enlarging lateral neck masses. Cysts occasionally acutely increase in size during upper respiratory tract infection, leading to respiratory compromise, torticollis, and dysphagia.</Contents>
+              <Contents>A rare otorhinolaryngological malformation characterized by the presence of a cyst, sinus or fistula occuring along the anterior border of the sternocleidomastoid muscle. Second branchial cleft fistulae and sinuses present with skin opening with chronic discharge and recurrent infections, whereas second branchial cleft cysts present as a painless, nontender, stable in size or slowly enlarging lateral neck masses. Cysts occasionally acutely increase in size during upper respiratory tract infection, leading to respiratory compromise, torticollis, and dysphagia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -161556,7 +161558,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A teratologic disorder associated with intrauterine exposure of phenorbarbital during the first trimester of pregnancy. Infants are usually asymptomatic but an increased risk of intellectual disability, tetralogy of Fallot, unilateral cleft lip, hypoplasia of the mitral valve and some other mild abnormalities such as hypertelorism, epicanthus, hypoplasia and low insertion of the nose, low insertion of the ears, prognathism, &amp;#64257;nger hypoplasia, brachydactyly and hypospadias have been reported in rare cases.</Contents>
+              <Contents>A teratologic disorder associated with intrauterine exposure of phenobarbital during the first trimester of pregnancy. Infants are usually asymptomatic but an increased risk of intellectual disability, tetralogy of Fallot, unilateral cleft lip, hypoplasia of the mitral valve and some other mild abnormalities such as hypertelorism, epicanthus, hypoplasia and low insertion of the nose, low insertion of the ears, prognathism, &amp;#64257;nger hypoplasia, brachydactyly and hypospadias have been reported in rare cases.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -171564,7 +171566,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare frontonasal dysplasia malformation syndrome characterized by an oxycephalic skull with craniosynostosis, wide nose with anteverted nostrils, hirsutism at base of nose, agenesis of the nasolacrimal ducts, and bilateral, symmertical nasolabial cysts on upper lip. Additional features may include hypertelorism. There have been no further descriptions in the literature since 1991.</Contents>
+              <Contents>A rare frontonasal dysplasia malformation syndrome characterized by an oxycephalic skull with craniosynostosis, wide nose with anteverted nostrils, hirsutism at base of nose, agenesis of the nasolacrimal ducts, and bilateral, symmetrical nasolabial cysts on upper lip. Additional features may include hypertelorism. There have been no further descriptions in the literature since 1991.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -197650,15 +197652,12 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="99293" lang="en">
-          <TextSectionList count="1">
-            <TextSection id="122120" lang="en">
-              <TextSectionType id="16907">
-                <Name lang="en">Definition</Name>
-              </TextSectionType>
-              <Contents>2q33.1 microdeletion syndrome is a rare chromosomal anomaly syndrome, resulting from the partial deletion of the long arm of chromosome 2, with a highly variable phenotype typically characterized by severe intellectual disability, moderate to severe developmental delay (particularly speech), feeding difficulties, failure to thrive, hypotonia, thin, sparse hair, various dental abnormalities and cleft/high-arched palate. Typical dysmorphic features inlcude high, prominent forehead, down-slanting palpebral fissures and prominent nasal bridge with beaked nose. Various behavioral problems (e.g. hyperactivity, chaotic/repetitive behavior, touch avoidance) are also associated.</Contents>
-            </TextSection>
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
           </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This disease is described under  SATB2-associated syndrome</Info>
+          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -209643,7 +209642,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, life-threatening, mitochondrial DNA depletion syndrome disease characterized by severe, progressive sensorimotor neuropathy associated with corneal ulceration, scarring or anesthesia, acral mutilation, metabolic and immunologic derangement, and hepatopathy (which can manifest with fulminant hepatic failure, a Reye-like syndrome or indolent progression to liver cirrhosis, depending on clinical form involved), present in the Navajo Native American population. Clinical presentation includes failure to thrive, distal limb weakness with reduced sensation, limb contractures with loss of funtion, areflexia, recurrent metabolic acidosis with intercurrent illness, immunologic anomalies manifesting with severe systemic infections, and sexual infantilism.</Contents>
+              <Contents>A rare, life-threatening, mitochondrial DNA depletion syndrome disease characterized by severe, progressive sensorimotor neuropathy associated with corneal ulceration, scarring or anesthesia, acral mutilation, metabolic and immunologic derangement, and hepatopathy (which can manifest with fulminant hepatic failure, a Reye-like syndrome or indolent progression to liver cirrhosis, depending on clinical form involved), present in the Navajo Native American population. Clinical presentation includes failure to thrive, distal limb weakness with reduced sensation, limb contractures with loss of function, areflexia, recurrent metabolic acidosis with intercurrent illness, immunologic anomalies manifesting with severe systemic infections, and sexual infantilism.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -241255,7 +241254,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Richieri Costa-da Silva syndrome is a rare, genetic, myotonic syndrome characterized by childhood onset of progressive and severe myotonia (with generalized muscular hypertophy and progressive impairment of gait), short stature, skeletal abnormalities (including pectus carinatum, short, wedge-shaped thoracolumbar vertebrae, kyphoscoliosis, genu valgum, irregular femoral epiphyses), and mild to moderate intellectual deficiency. No facial dysmorphism nor joint limitation is associated. There have been no further descriptions in the literature since 1984.</Contents>
+              <Contents>Richieri Costa-da Silva syndrome is a rare, genetic, myotonic syndrome characterized by childhood onset of progressive and severe myotonia (with generalized muscular hypertrophy and progressive impairment of gait), short stature, skeletal abnormalities (including pectus carinatum, short, wedge-shaped thoracolumbar vertebrae, kyphoscoliosis, genu valgum, irregular femoral epiphyses), and mild to moderate intellectual deficiency. No facial dysmorphism nor joint limitation is associated. There have been no further descriptions in the literature since 1984.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -268241,7 +268240,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, ectodermal dysplasia syndrome characterized by corneal epithelial changes (ranging from roughening to nodular irregularities), diffuse palmoplantar hyperkertosis with thickened, erythematous, scaly lesions affecting the elbows, knees and knuckles, distal onycholysis, brachydactyly accompanied by a single transverse palmar crease, short stature, premature birth, and increased susceptibility to tooth decay. Ocular symptoms include photophobia, reduced night vision, burning and watery eyes, and varying visual acuity. There have been no further descriptions in the literature since 1984.</Contents>
+              <Contents>A rare, genetic, ectodermal dysplasia syndrome characterized by corneal epithelial changes (ranging from roughening to nodular irregularities), diffuse palmoplantar hyperkeratosis with thickened, erythematous, scaly lesions affecting the elbows, knees and knuckles, distal onycholysis, brachydactyly accompanied by a single transverse palmar crease, short stature, premature birth, and increased susceptibility to tooth decay. Ocular symptoms include photophobia, reduced night vision, burning and watery eyes, and varying visual acuity. There have been no further descriptions in the literature since 1984.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -271132,7 +271131,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, immune disease characterized by chronic neutrophilia, increase in the percentage of circulating CD34+ cells in peripheral blood, increase in granulocyte precursors in bone marrow and splenomegaly. Patients are predominantly asymptomatic, but may present with systemic inflammatory response syndrome with fever, dyspena, tachycardia, pleural and pericardial effusion, or myelodysplastic syndrome.</Contents>
+              <Contents>A rare, genetic, immune disease characterized by chronic neutrophilia, increase in the percentage of circulating CD34+ cells in peripheral blood, increase in granulocyte precursors in bone marrow and splenomegaly. Patients are predominantly asymptomatic, but may present with systemic inflammatory response syndrome with fever, dyspnea, tachycardia, pleural and pericardial effusion, or myelodysplastic syndrome.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -278382,9 +278381,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
+      <SynonymList count="3">
         <Synonym lang="en">FMNG</Synonym>
         <Synonym lang="en">Familial MNG</Synonym>
+        <Synonym lang="en">Familial multinodular goiter syndrome</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -289221,7 +289221,7 @@
     <Disorder id="3166">
       <OrphaCode>378</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=378</ExpertLink>
-      <Name lang="en">NON RARE IN EUROPE: Sj÷gren syndrome</Name>
+      <Name lang="en">NON RARE IN EUROPE: Sjögren syndrome</Name>
       <DisorderFlagList count="3">
         <DisorderFlag id="495">
           <Value>8192</Value>
@@ -289250,14 +289250,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is not rare in Europe. It does not belong to the Orphanet nomenclature of rare diseases.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18584">
@@ -312756,13 +312749,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="7">
+      <SynonymList count="3">
         <Synonym lang="en">GSD due to GLUT2 deficiency</Synonym>
-        <Synonym lang="en">GSD type 11</Synonym>
-        <Synonym lang="en">GSD type XI</Synonym>
         <Synonym lang="en">Glycogen storage disease due to GLUT2 deficiency</Synonym>
-        <Synonym lang="en">Glycogen storage disease type 11</Synonym>
-        <Synonym lang="en">Glycogen storage disease type XI</Synonym>
         <Synonym lang="en">Glycogenosis due to GLUT2 deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -318313,11 +318302,15 @@
     <Disorder id="18872">
       <OrphaCode>217601</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217601</ExpertLink>
-      <Name lang="en">Hypertrophic cardiomyopathy due to intensive athletic training</Name>
-      <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+      <Name lang="en">OBSOLETE: Hypertrophic cardiomyopathy due to intensive athletic training</Name>
+      <DisorderFlagList count="2">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -318328,20 +318321,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="121486">
-          <Source>ICD-10</Source>
-          <Reference>I42.2</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -371854,8 +371834,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
         <Synonym lang="en">IRD</Synonym>
+        <Synonym lang="en">Mild PBD-ZSD</Synonym>
+        <Synonym lang="en">Mild peroxisome biogenesis disorder-Zellweger spectrum disorder</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -376977,7 +376959,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, non-severe combined immunodeficiency disorder characterized by variable B- and T-cell defects (including defective B-cell differentiation and impaired T-cell proliferation to mitogens and bacterial antigens) and natural killer cell dysfunction (ranging from impaired cytotoxity to lymphopenia) due to IL21R deficiency, manifesting with recurrent respiratory and/or gastrointestinal tract infections and, in some cases, with severe, chronic, progressive cholangitis and liver cirrhosis associated with cryptosporidial infection.</Contents>
+              <Contents>A rare, genetic, non-severe combined immunodeficiency disorder characterized by variable B- and T-cell defects (including defective B-cell differentiation and impaired T-cell proliferation to mitogens and bacterial antigens) and natural killer cell dysfunction (ranging from impaired cytotoxicity to lymphopenia) due to IL21R deficiency, manifesting with recurrent respiratory and/or gastrointestinal tract infections and, in some cases, with severe, chronic, progressive cholangitis and liver cirrhosis associated with cryptosporidial infection.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -377660,7 +377642,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, syndromic intellectual disability disorder characterized by variable degrees of intellectual disability, behavioral problems (including attention deficit and hyperactivity disorder, autism spectrum disorder, and aggressiveness), an altered sleeping pattern, and delayed speech and language development associated with disruption of ankryin-3 (&lt;i&gt;ANK3&lt;/i&gt; gene). Additional features observed may include muscular hypotonia and spasticity. Epilepsy, chronic hunger, and dysmorphic facial features have been reported.</Contents>
+              <Contents>A rare, genetic, syndromic intellectual disability disorder characterized by variable degrees of intellectual disability, behavioral problems (including attention deficit and hyperactivity disorder, autism spectrum disorder, and aggressiveness), an altered sleeping pattern, and delayed speech and language development associated with disruption of ankyrin-3 (&lt;i&gt;ANK3&lt;/i&gt; gene). Additional features observed may include muscular hypotonia and spasticity. Epilepsy, chronic hunger, and dysmorphic facial features have been reported.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -394091,12 +394073,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="7">
+      <SynonymList count="4">
         <Synonym lang="en">GSD due to lactate dehydrogenase M-subunit deficiency</Synonym>
-        <Synonym lang="en">GSD type 11</Synonym>
-        <Synonym lang="en">Glycogen storage disease type 11</Synonym>
         <Synonym lang="en">Glycogenosis due to lactate dehydrogenase M-subunit deficiency</Synonym>
-        <Synonym lang="en">Glycogenosis type 11</Synonym>
         <Synonym lang="en">LDH-M subunit deficiency</Synonym>
         <Synonym lang="en">Lactate dehydrogenase A deficiency</Synonym>
       </SynonymList>
@@ -395881,7 +395860,7 @@
     <Disorder id="20722">
       <OrphaCode>289395</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=289395</ExpertLink>
-      <Name lang="en">NON RARE IN EUROPE: Secondary Sj÷gren syndrome</Name>
+      <Name lang="en">NON RARE IN EUROPE: Secondary Sjögren syndrome</Name>
       <DisorderFlagList count="3">
         <DisorderFlag id="495">
           <Value>8192</Value>
@@ -395909,14 +395888,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is not rare in Europe. It does not belong to the Orphanet nomenclature of rare diseases.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20721">
@@ -412004,14 +411976,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21355">
@@ -412514,7 +412479,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, complex cerebral cortical malformation characterized by generalized or focal dysgyria (also named polymicrogryia-like cortical dysplasia) or alternatively by microlissencephaly with dysmorphic basal ganglia and dysgenesis of the corpus callosum. Clinical manifestations are variable and include microcephaly, seizures, hypotonia, developmental delay, severe psychomotor delay, ataxia, spastic diplegia or tetraplegia, and ocular abnormalities (strabismus, ptosis or optic atrophy).</Contents>
+              <Contents>A rare, genetic, complex cerebral cortical malformation characterized by generalized or focal dysgyria (also named polymicrogyria-like cortical dysplasia) or alternatively by microlissencephaly with dysmorphic basal ganglia and dysgenesis of the corpus callosum. Clinical manifestations are variable and include microcephaly, seizures, hypotonia, developmental delay, severe psychomotor delay, ataxia, spastic diplegia or tetraplegia, and ocular abnormalities (strabismus, ptosis or optic atrophy).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -422213,7 +422178,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A very rare, chromosomal anomaly characterized by an intrauterine and postanatal growth retardation, short stature, developmental delay, learning difficulties, hearing loss, hypermetropia,and a recognisable facial dysmorphism including prominenet forehead, long, myopathic facies, fine eyebrows, small mouth and micrognathia.</Contents>
+              <Contents>A very rare, chromosomal anomaly characterized by an intrauterine and postnatal growth retardation, short stature, developmental delay, learning difficulties, hearing loss, hypermetropia,and a recognisable facial dysmorphism including prominent forehead, long, myopathic facies, fine eyebrows, small mouth and micrognathia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -426867,7 +426832,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, neurological disorder characterized by childhood-onset severe myoclonic and tonic-clonic seizures and early-onset ataxia leading to severe gait disturbances associated with normal to slightly diminished cognition. Scoliosis, diffuse muscle atrophy and subcutaneous fat loss, as well as developmental delay, may be associated. Brain MRI may reveal complete agenesis of the corpus callosum, venticulomegaly, interhemispheric cysts, and simplified gyration (frontally).</Contents>
+              <Contents>A rare, genetic, neurological disorder characterized by childhood-onset severe myoclonic and tonic-clonic seizures and early-onset ataxia leading to severe gait disturbances associated with normal to slightly diminished cognition. Scoliosis, diffuse muscle atrophy and subcutaneous fat loss, as well as developmental delay, may be associated. Brain MRI may reveal complete agenesis of the corpus callosum, ventriculomegaly, interhemispheric cysts, and simplified gyration (frontally).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -436807,7 +436772,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, autoimflammatory syndrome with immune deficiency disease characterized by recurrent and severe flares of generalized pustular psoriasis associated with high fever, asthenia, and systemic inflammation, due to IL36R antagonist deficiency. Psoriatic nail changes (e.g. pitting and onychomadesis) and ichthyosis may occasionally be associated.</Contents>
+              <Contents>A rare, genetic, autoinflammatory syndrome with immune deficiency disease characterized by recurrent and severe flares of generalized pustular psoriasis associated with high fever, asthenia, and systemic inflammation, due to IL36R antagonist deficiency. Psoriatic nail changes (e.g. pitting and onychomadesis) and ichthyosis may occasionally be associated.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -444012,7 +443977,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, primary immunodeficiency disease characterized by increased susceptibility to recurrent and/or severe bacterial and viral infections (in particular, sinopulmonary bacterial and herpesvirus infections), chronic benign lymphoproliferation (manifesting as lympadenopathy, hepatosplenomegaly and focal nodular lymphoid hyperplasia), and/or autoimmune disease (including immune cytopenias, juvenile arthritis, glomerulonephritis and sclerosing cholangitis). Immunophenotypically, variable degrees of agammaglobulinemia with increased IgM levels, increased circulating transitional B cells, decreased naïve CD4 and CD8 T-cells with increased CD8 effector/memory T cells are observed.</Contents>
+              <Contents>A rare, genetic, primary immunodeficiency disease characterized by increased susceptibility to recurrent and/or severe bacterial and viral infections (in particular, sinopulmonary bacterial and herpesvirus infections), chronic benign lymphoproliferation (manifesting as lymphadenopathy, hepatosplenomegaly and focal nodular lymphoid hyperplasia), and/or autoimmune disease (including immune cytopenias, juvenile arthritis, glomerulonephritis and sclerosing cholangitis). Immunophenotypically, variable degrees of agammaglobulinemia with increased IgM levels, increased circulating transitional B cells, decreased naïve CD4 and CD8 T-cells with increased CD8 effector/memory T cells are observed.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -446263,7 +446228,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, alpha-crystallinopathy disease characterized by adult-onset myofibrillar myopathy, variably associated with cardiomyopathy and/or posterior pole cataracts. Patients typically present progressive proximal and distal muscle weakness and wasting of lower and upper limbs, often with velopharyngeal involvement including dysphagia, dysphonia and ventilatory insufficiency. Electromyography shows myopathic features and muscle biopsy reveals myofibrillar myopthay changes.</Contents>
+              <Contents>A rare, genetic, alpha-crystallinopathy disease characterized by adult-onset myofibrillar myopathy, variably associated with cardiomyopathy and/or posterior pole cataracts. Patients typically present progressive proximal and distal muscle weakness and wasting of lower and upper limbs, often with velopharyngeal involvement including dysphagia, dysphonia and ventilatory insufficiency. Electromyography shows myopathic features and muscle biopsy reveals myofibrillar myopathy changes.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -449934,7 +449899,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic primary immunodeficiency characterized by recurrent respiratory and skin viral infections (Ebstein-Barr virus, herpes simplex virus, human papillomavirus), deficient spontaneous cytotoxicity of natural killer cells, but preserved antibody-dependent cellular cytotoxicity. No other abnormalities are present on immunologic work-up.</Contents>
+              <Contents>A rare, genetic primary immunodeficiency characterized by recurrent respiratory and skin viral infections (Epstein-Barr virus, herpes simplex virus, human papillomavirus), deficient spontaneous cytotoxicity of natural killer cells, but preserved antibody-dependent cellular cytotoxicity. No other abnormalities are present on immunologic work-up.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -452208,7 +452173,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, hereditary disorder of pentose phosphate metabolism characterized by increased urine levels of sedoheptulose and erythtirol, and low-to-normal excretion of sedoheptulose-7P. Clinical presentation of this disorder is currently unclear.</Contents>
+              <Contents>A rare, hereditary disorder of pentose phosphate metabolism characterized by increased urine levels of sedoheptulose and erythritol, and low-to-normal excretion of sedoheptulose-7P. Clinical presentation of this disorder is currently unclear.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -460295,14 +460260,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is not rare in Europe. It does not belong to the Orphanet nomenclature of rare diseases.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23124">
@@ -461731,7 +461689,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Autosomal recessive severe congenital neutropenia due to JAGN1 deficiency is a rare, genetic, primary immunodeficiency disorder characterized by early-onset, recurrent, severe bacterial infections, granulopoiesis maturation arrest at the promyelocyte/myelocyte stage and markedly reduced absolute neutrophil counts, resulting from recessively inherited mutations in the &lt;i&gt;JAGN1&lt;/i&gt; gene. Mild facial dysmorphism (i.e. triangular face), short stature, failure to thrive, hypothyroidism, developmental delay, pancreatic insufficiency and coractation of aorta, as well as bone and urogenital abnormalities, may also be associated.</Contents>
+              <Contents>Autosomal recessive severe congenital neutropenia due to JAGN1 deficiency is a rare, genetic, primary immunodeficiency disorder characterized by early-onset, recurrent, severe bacterial infections, granulopoiesis maturation arrest at the promyelocyte/myelocyte stage and markedly reduced absolute neutrophil counts, resulting from recessively inherited mutations in the &lt;i&gt;JAGN1&lt;/i&gt; gene. Mild facial dysmorphism (i.e. triangular face), short stature, failure to thrive, hypothyroidism, developmental delay, pancreatic insufficiency and coarctation of aorta, as well as bone and urogenital abnormalities, may also be associated.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -474849,7 +474807,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare vulvar carcinoma characterized by an ulcer, nodule, macule, or pedunculated mass which is histologically composed of infiltrating islands of malignant squamous cells. Histological subtypes include keratinizing, non-keratinizing, basaloid, warty, and verrucous carcinomas. Some tumors are associated with human papilloma virus, smoking, high grade squamous intraepithelial lesion, chronic vulvar inflammatory disorders, or differentiated vulvar intraepithelial neoplasia. Patients may present with discharge, bleeding, or pain. Most important predictive features are tumor depth and lymph node status.</Contents>
+              <Contents>A rare vulvar carcinoma characterized by an ulcer, nodule, macule, or pedunculated mass which is histologically composed of infiltrating islands of malignant squamous cells. Histological subtypes include keratinizing, non-keratinizing, basaloid, warty, and verrucous carcinomas. Some tumors are associated with human papilloma virus, smoking, high grade squamous intraepithelial lesion, chronic vulvar inflammatory disorders, or differentiated vulvar intraepithelial neoplasia. Patients may present with discharge, bleeding, or pain. Most important prognostic features are tumor depth and lymph node status.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -508396,7 +508354,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, syndromic, benign, epidemal nevus syndrome characterized by the association of a Becker nevus (i.e. circumscribed, unilateral, irregularly shaped, hyperpigmented macules, with or without hypertrichosis and/or acneiform lesions, occuring predominantly on the anterior upper trunk or scapular region) with ipsilateral breast hypoplasia or other, typically hypoplastic, skeletal, cutaneous, and/or muscular defects, such as pectoralis major hypoplasia, supernumerary nipples, vertebral defects, scoliosis, limb asymmetry, odontomaxillary hypoplasia and lipoatrophy.</Contents>
+              <Contents>A rare, syndromic, benign, epidermal nevus syndrome characterized by the association of a Becker nevus (i.e. circumscribed, unilateral, irregularly shaped, hyperpigmented macules, with or without hypertrichosis and/or acneiform lesions, occuring predominantly on the anterior upper trunk or scapular region) with ipsilateral breast hypoplasia or other, typically hypoplastic, skeletal, cutaneous, and/or muscular defects, such as pectoralis major hypoplasia, supernumerary nipples, vertebral defects, scoliosis, limb asymmetry, odontomaxillary hypoplasia and lipoatrophy.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -517130,8 +517088,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
+        <Synonym lang="en">PBD-ZSD</Synonym>
         <Synonym lang="en">Peroxisome biogenesis disorder spectrum</Synonym>
+        <Synonym lang="en">Peroxisome biogenesis disorder-Zellweger spectrum disorder</Synonym>
       </SynonymList>
       <DisorderType id="21436">
         <Name lang="en">Clinical group</Name>
@@ -522891,14 +522851,17 @@
     <Disorder id="11106">
       <OrphaCode>77293</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
-      <Name lang="en">Niemann-Pick disease type B</Name>
+      <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="3">
+        <Synonym lang="en">Chronic visceral ASMD</Synonym>
+        <Synonym lang="en">NPD-B</Synonym>
+        <Synonym lang="en">Niemann-Pick disease type B</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -523157,14 +523120,17 @@
     <Disorder id="11105">
       <OrphaCode>77292</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
-      <Name lang="en">Niemann-Pick disease type A</Name>
+      <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="3">
+        <Synonym lang="en">Infantile neurovisceral ASMD</Synonym>
+        <Synonym lang="en">NPD-A</Synonym>
+        <Synonym lang="en">Niemann-Pick disease type A</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -528548,11 +528514,19 @@
       <OrphaCode>73247</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73247</ExpertLink>
       <Name lang="en">NON RARE IN EUROPE: Eosinophilic esophagitis</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="3">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
+        <DisorderFlag id="456">
+          <Value>32</Value>
+          <Label>Non-rare disease in Europe</Label>
+        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
         <Synonym lang="en">EoE</Synonym>
@@ -556068,8 +556042,7 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
-        <Synonym lang="en">Dystrophie valvulaire associée à FLNA</Synonym>
+      <SynonymList count="2">
         <Synonym lang="en">FLNA-related valvular dystrophy</Synonym>
         <Synonym lang="en">Filamin A-related X-linked myxomatous valvular dysplasia</Synonym>
       </SynonymList>
@@ -566589,7 +566562,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare axonal hereditary motor and sensoy neuropathy disease characterized by progressive, peripheral, axonal sensorimotor neuropathy (of variable severity), affecting predominantly the distal lower limbs, associated with progressive, variably severe, optic atrophy, which frequently leads to visual loss. Patients typically present distal limb muscle weakness and atrophy, hypo/areflexia, foot deformities, poor visual acuity (often with a central scotoma), nystagmus, and reduced peripheral and nocturnal vision. Additional reported manifestations include sensorineural hearing loss, major joint contractures, anosmia, scoliosis/lumbar hyperlordosis, cognitive impairment and vocal cord paresis.</Contents>
+              <Contents>A rare axonal hereditary motor and sensory neuropathy disease characterized by progressive, peripheral, axonal sensorimotor neuropathy (of variable severity), affecting predominantly the distal lower limbs, associated with progressive, variably severe, optic atrophy, which frequently leads to visual loss. Patients typically present distal limb muscle weakness and atrophy, hypo/areflexia, foot deformities, poor visual acuity (often with a central scotoma), nystagmus, and reduced peripheral and nocturnal vision. Additional reported manifestations include sensorineural hearing loss, major joint contractures, anosmia, scoliosis/lumbar hyperlordosis, cognitive impairment and vocal cord paresis.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -577827,7 +577800,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Bilateral renal dysplasia is a form of renal dysplasia (RD; see this term), a renal tract malformation in which the development of both kidneys is abnormal and incomplete. Bilateral RD can be segmental, and of variable severity, with renal aplasia corresponding to extreme RD.</Contents>
+              <Contents>A form of renal dysplasia (RD), a renal tract malformation, characterized by abnormal or incomplete development of both kidneys. Bilateral RD can be segmental, and of variable severity, with renal aplasia corresponding to extreme RD. Patients may be asymptomatic if the residual kidney function is sufficient. In cases of severe bilateral RD, the risk of renal failure in childhood is high.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -582414,7 +582387,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A disorder that is one of the most severe forms of congenital adrenal hyperplasia (CAH) characterized by severe adrenal insufficiency and sex reversal in males.</Contents>
+              <Contents>A severe form of congenital adrenal hyperplasia (CAH) characterized by severe adrenal insufficiency and sex reversal in males.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -590053,7 +590026,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A very rare form of congenital adrenal hyperplasia (CAH) characterized by glucocorticoid deficiency, hypergonadotrophic hypogonadism and severe hypokalemic hypertension.</Contents>
+              <Contents>A rare form of congenital adrenal hyperplasia due to 17-alpha-hydroxylase (CYP17A1) deficiency and characterized by glucocorticoid deficiency, mineralocorticoid excess leading to hypokalemic hypertension and sex steroid deficiency (hypergonadotrophic hypogonadism). Undervirilization and even female phenotype in 46,XY males, primary amenorrhea in females and lack of pubertal development in both sexes is common. Residual CYP17A1 activity is associated with the severity of this condition with a large spectrum of variability, from presenting in early infancy, to unusually mild courses with isolated sex steroid deficiency but normal ACTH-stimulated cortisol in adult patients.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -590135,7 +590108,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A disorder that is the most common form of congenital adrenal hyperplasia (CAH), characterized by simple virilizing or salt wasting forms that can manifest with genital ambiguity in females and with adrenal insufficiency (in both sexes), and that presents with dehydration, hypoglycemia in the neonatal period (that can be lethal if untreated), and hyperandrogenia.</Contents>
+              <Contents>A form of congenital adrenal hyperplasia (CAH) characterized by simple virilizing or salt wasting forms that can manifest with abnormal genital development with variable levels of virilization in females and with adrenal insufficiency in both sexes, and that presents with dehydration and hypoglycemia (which can be lethal if left untreated) in the neonatal period, as well as hyperandrogenia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -590240,7 +590213,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare form of congenital adrenal hyperplasia (CAH) characterized by glucocorticoid deficiency, hyperandrogenism, hypertension and virilization in females.</Contents>
+              <Contents>A rare form of classic congenital adrenal hyperplasia (CAH) characterized by glucocorticoid deficiency, hyperandrogenism, hypertension and virilization in females.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -590598,7 +590571,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, functioning, pituitary adenoma characterized by the presence of a pituitary mass associated with high levels of circulating, free, thyroid hormones in conjunction with normal to high levels of TSH and unresponsiveness of TSH levels to TRH stimulation and T3 suppression tests, typically manifesting with signs and symtoms of mild to moderate hyperthyroidism (e.g. goiter (most frequently observed), palpitation, excessive sweating, arrhythmia, weight loss, tremor) and/or tumor mass effect (such as headache, visual field defects, hypopituitarism). Occasionally, cosecretion of prolactin and/or growth hormone may cause galactorrhea and/or acromegaly.</Contents>
+              <Contents>A rare, functioning, pituitary adenoma characterized by the presence of a pituitary mass associated with high levels of circulating, free, thyroid hormones in conjunction with normal to high levels of TSH and unresponsiveness of TSH levels to TRH stimulation and T3 suppression tests, typically manifesting with signs and symptoms of mild to moderate hyperthyroidism (e.g. goiter (most frequently observed), palpitation, excessive sweating, arrhythmia, weight loss, tremor) and/or tumor mass effect (such as headache, visual field defects, hypopituitarism). Occasionally, cosecretion of prolactin and/or growth hormone may cause galactorrhea and/or acromegaly.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -591424,14 +591397,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="27542">
@@ -604018,7 +603984,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, non-rhizomelic, chondrodysplasia punctata syndrome characterized, radiologically, by stippled calcifications and disproportionate, short metacarpals and tibiae (with characteristic overshoot of the proximal fibula), clinically manifesting with severe short stature, bilateral shortening of upper and lower limbs, flat midface and nose, in the absence of cataracts and cutaneous anomalies. Neonatal tachnypnea, hydrocephalus and mild developmental delay have been seldomly associated. Additional radiologic features include bowed long bones, platyspondyly and/or vertebral clefts.</Contents>
+              <Contents>A rare, non-rhizomelic, chondrodysplasia punctata syndrome characterized, radiologically, by stippled calcifications and disproportionate, short metacarpals and tibiae (with characteristic overshoot of the proximal fibula), clinically manifesting with severe short stature, bilateral shortening of upper and lower limbs, flat midface and nose, in the absence of cataracts and cutaneous anomalies. Neonatal tachypnea, hydrocephalus and mild developmental delay have been seldomly associated. Additional radiologic features include bowed long bones, platyspondyly and/or vertebral clefts.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -616041,7 +616007,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, acquired, self-limiting, infectious disease due to the mite-borne bacteria &lt;i&gt;Rickettsia akari&lt;/i&gt; characterized by an asymptomatic, 0.5 to 2 cm in diameter papulovesicle that typically ulcerates and forms an eschar, followed by a generalized papulovesicular rash associating variable constitutional symptoms, such as localized lymphadenopathy, fever, malaise, and headaches. Additonal symptoms may include diaphoresis, myalgia and, less frequently, rhinorrhea, pharyngitis, nausea, vomiting, splenomegaly, conjunctival hyperemia, and abdominal pain. Systemic symtoms resolve within 6-10 days.</Contents>
+              <Contents>A rare, acquired, self-limiting, infectious disease due to the mite-borne bacteria &lt;i&gt;Rickettsia akari&lt;/i&gt; characterized by an asymptomatic, 0.5 to 2 cm in diameter papulovesicle that typically ulcerates and forms an eschar, followed by a generalized papulovesicular rash associating variable constitutional symptoms, such as localized lymphadenopathy, fever, malaise, and headaches. Additonal symptoms may include diaphoresis, myalgia and, less frequently, rhinorrhea, pharyngitis, nausea, vomiting, splenomegaly, conjunctival hyperemia, and abdominal pain. Systemic symptoms resolve within 6-10 days.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -620209,8 +620175,7 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="4">
-        <Synonym lang="en">Acquired neuromyotonia</Synonym>
+      <SynonymList count="3">
         <Synonym lang="en">Continuous muscle fiber activity syndrome</Synonym>
         <Synonym lang="en">Isaac-Mertens syndrome</Synonym>
         <Synonym lang="en">Quantal squander syndrome</Synonym>
@@ -637276,7 +637241,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Congenital adrenal hyperplasia due to cytochrome P450 oxidoreductase deficiency is a unique form of congenital adrenal hyperplasia (CAH; see this term) characterized by glucocorticoid deficiency, severe sexual ambiguity in both sexes and skeletal (especially craniofacial) malformations.</Contents>
+              <Contents>A rare form of congenital adrenal hyperplasia due to P450 oxidoreductase deficiency and characterized by glucocorticoid deficiency, virilization of external genitalia in females, and undervirilization in males. Findings range from severely affected infants with 46,XX and 46,XY disorders/differences of sex development (DSD) and cortisol deficiency to mildly affected women who appear to have polycystic ovary syndrome, or mildly affected men with gonadal insufficiency.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -638543,14 +638508,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12618">
@@ -647091,7 +647049,7 @@
     <Disorder id="12408">
       <OrphaCode>93566</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93566</ExpertLink>
-      <Name lang="en">OBSOLETE: Pediatric Sj÷gren syndrome</Name>
+      <Name lang="en">OBSOLETE: Pediatric Sjögren syndrome</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="495">
           <Value>8192</Value>
@@ -649757,8 +649715,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
         <Synonym lang="en">AHA</Synonym>
+        <Synonym lang="en">Acquired F8 deficiency</Synonym>
+        <Synonym lang="en">Acquired factor VIII deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -649783,8 +649743,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
         <Synonym lang="en">AHB</Synonym>
+        <Synonym lang="en">Acquired F9 deficiency</Synonym>
+        <Synonym lang="en">Acquired factor IX deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -671411,9 +671373,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
+      <SynonymList count="3">
         <Synonym lang="en">Christmas disease</Synonym>
-        <Synonym lang="en">Factor IX deficiency</Synonym>
+        <Synonym lang="en">Congenital F9 deficiency</Synonym>
+        <Synonym lang="en">Congenital factor IX deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -671879,9 +671842,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
-        <Synonym lang="en">FVIII deficiency</Synonym>
-        <Synonym lang="en">Factor VIII deficiency</Synonym>
+      <SynonymList count="3">
+        <Synonym lang="en">Congenital F8 deficiency</Synonym>
+        <Synonym lang="en">Congenital FVIII deficiency</Synonym>
+        <Synonym lang="en">Congenital Factor VIII deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -679707,7 +679671,7 @@
         <DisorderDisorderAssociation>
           <TargetDisorder id="11106">
             <OrphaCode>77293</OrphaCode>
-            <Name lang="en">Niemann-Pick disease type B</Name>
+            <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
           </TargetDisorder>
           <RootDisorder id="14039" cycle="true"/>
           <DisorderDisorderAssociationType id="27341">
@@ -696181,7 +696145,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This disease is described under  SATB2-associated syndrome</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28850">
@@ -696207,7 +696178,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131755" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="172827" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic multiple congenital anomalies/dysmorphic syndrome characterized by moderate to severe developmental delay/intellectual disability with absent or limited speech development, various behavioral problems (including autistic features, hyperactivity, or aggressiveness), and craniofacial anomalies such as long face, high and prominent forehead, bulbous nose with low-hanging columella, thin vermillion of the upper lip, palatal (cleft palate, high-arched palate, and bifid uvula) and dental (abnormal upper incisors) abnormalities, and micrognathia. Hypotonia and feeding difficulties are frequent. Other supportive findings may include skeletal anomalies with low bone density and abnormal brain imaging.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28814">
@@ -728995,6 +728976,60 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31389">
+      <OrphaCode>618899</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618899</ExpertLink>
+      <Name lang="en">Acid sphingomyelinase deficiency</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">ASMD</Synonym>
+      </SynonymList>
+      <DisorderType id="21436">
+        <Name lang="en">Clinical group</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="en">Group of disorders</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31388">
+      <OrphaCode>618891</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
+      <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="3">
+        <Synonym lang="en">Chronic neurovisceral ASMD</Synonym>
+        <Synonym lang="en">NPD-A/B</Synonym>
+        <Synonym lang="en">Niemann-Pick disease type A/B</Synonym>
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="31370">
       <OrphaCode>617910</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617910</ExpertLink>
@@ -729049,7 +729084,7 @@
     <Disorder id="31372">
       <OrphaCode>617919</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617919</ExpertLink>
-      <Name lang="en">F12-associated cold auto-inflammatory syndrome</Name>
+      <Name lang="en">F12-associated cold autoinflammatory syndrome</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -729081,7 +729116,8 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="2">
+        <Synonym lang="en">F9 deficiency, Leyden type</Synonym>
         <Synonym lang="en">Factor IX deficiency, Leyden type</Synonym>
       </SynonymList>
       <DisorderType id="21450">
