commit bbf94d552a3c459862632b82d2e2166d94fb6b4e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml
index 43f8ccd..98bd977 100644
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:26:11" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:27:03" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10500">
+  <DisorderList count="10501">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -1131,7 +1131,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Type 1 von Willebrand disease (type 1 VWD) is a form of VWD (see this term) characterized by a bleeding disorder associated with a partial quantitative plasmatic deficiency of an otherwise structurally and functionally normal Willebrand factor (von Willebrand factor; VWF).</Contents>
+              <Contents>A form of von Willebrand disease (VWD) characterized by a bleeding disorder associated with a partial, quantitative plasmatic deficiency of an otherwise structurally and functionally normal von Willebrand factor (VWF).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -4392,8 +4392,7 @@
           <Label>Non-rare disease in Europe</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
-        <Synonym lang="en">UC</Synonym>
+      <SynonymList count="2">
         <Synonym lang="en">Ulcerative proctitis</Synonym>
         <Synonym lang="en">Ulcerative proctosigmoiditis</Synonym>
       </SynonymList>
@@ -7105,7 +7104,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Facioscapulohumeral muscular dystrophy (FSHD) is characterized by progressive muscle weakness with focal involvement of the facial, shoulder and limb muscles.</Contents>
+              <Contents>A rare neuromuscular disease characterized by progressive muscle weakness with focal involvement of the facial, shoulder and limb muscles.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -16901,7 +16900,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A congenital, X-linked, clinical subtype of L1 syndrome, characterized by severe hydrocephalus often of prenatal onset, adducted thumbs, spasticity (mostly evidenced by brisk tendon reflexes and extensor plantar responses) and severe intellectual disability. This subtype represents the severe end of the L1 syndrome spectrum and is associated with poor prognosis.</Contents>
+              <Contents>A congenital, X-linked, clinical subtype of L1 syndrome characterized by severe hydrocephalus often of prenatal onset, adducted thumbs, spasticity (mostly evidenced by brisk tendon reflexes and extensor plantar responses) and moderate to severe intellectual disability. This subtype represents the severe end of the L1 syndrome spectrum and is associated with poor prognosis.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -20479,8 +20478,7 @@
           <Label>Non-rare disease in Europe</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
-        <Synonym lang="en">PAD</Synonym>
+      <SynonymList count="2">
         <Synonym lang="en">Patent ductus arteriosus</Synonym>
         <Synonym lang="en">Persistent patency of the arterial duct</Synonym>
       </SynonymList>
@@ -27316,17 +27314,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="1754" lang="en">
-          <TextSectionList count="1">
-            <TextSection id="80985" lang="en">
-              <TextSectionType id="16907">
-                <Name lang="en">Definition</Name>
-              </TextSectionType>
-              <Contents>Treacher-Collins syndrome is a congenital disorder of craniofacial development characterized by bilateral symmetrical oto-mandibular dysplasia without abnormalities of the extremities, and associated with several head and neck defects.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="294">
@@ -28062,7 +28050,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Holoprosencephaly (HPE) is a complex brain malformation resulting from incomplete cleavage of the prosencephalon, occurring between the 18th and 28th day of gestation, and affecting both the forebrain and face, which results in neurological manifestations and facial anomalies of variable severity.</Contents>
+              <Contents>A rare complex brain malformation characterized by incomplete cleavage of the prosencephalon, and affecting both the forebrain and face and resulting in neurological manifestations and facial anomalies of variable severity.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -35380,7 +35368,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Smith-Magenis syndrome (SMS) is a complex genetic disorder characterized by variable intellectual deficit, sleep disturbance, craniofacial and skeletal anomalies, psychiatric disorders, and speech and motor delay.</Contents>
+              <Contents>A rare, genetic, neurodevelopmental disorder characterized by cognitive impairment of variable severity, behavioral abnormalities, and sleep disturbance. Patients present with distinctive physical features and a wide range of malformations (e.g. cardiac, renal).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -43447,7 +43435,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Freeman-Sheldon syndrome (FSS) is a very rare, multiple congenital contractures syndrome characterized by a microstomia with a whistling appearance of the mouth, distinctive facies, club foot and joint contractures. FSS is the most severe form of distal arthrogryposis.</Contents>
+              <Contents>A rare congenital, distal arthogryposis syndrome characterized by microstomia, whistling-face appearance, Chin with V- or H- shaped creased, and prominent nasolabial folds; most patients present club foot and congenital joint contractures of the hands and feet. It is the most severe form of distal arthrogryposis.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -45709,7 +45697,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Prune belly syndrome is a rare congenital disorder, belonging to the group of fetal lower urinary tract obstructions (LUTO), involving variable dilation of the lower urinary tract in association with partial or complete absence of the lateral and inferior abdominal wall musculature and in males bilateral non-palpable undescended testes.</Contents>
+              <Contents>A rare lower urinary tract obstruction (LUTO) characterized by varying degrees of an enlarged urinary bladder, dilated ureters, hydronephrosis, and poorly contractile and disorganized detrusor and ureteral smooth muscle, in association with hypoplastic or absent midline abdominal skeletal musculature, and bilaterally undescended testes in males.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -54393,7 +54381,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Sotos syndrome is a rare multisystemic genetic disorder characterized by a typical facial appearance, overgrowth of the body in early life with macrocephaly, and mild to severe intellectual disability.</Contents>
+              <Contents>A rare genetic overgrowth syndrome characterized by a typical facial appearance, overgrowth with macrocephaly and variable intellectual impairment.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -64398,7 +64386,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rapidly progressive and generally fulminant rare autoimmune condition characterized by the presence of anti-GBM antibodies, affecting glomerular and/or pulmonary capillaries. It may manifest as an isolated glomerulonephritis (anti-GBM nephritis) or as a pulmonary-renal syndrome with severe lung hemorrhage (Goodpasture's syndrome).</Contents>
+              <Contents>A rare, fulminant small vessel vasculitis that affects the capillary beds of the kidneys and lungs and characterized by the presence of anti-glomerular basement membrane (GBM) and, in its full-blown form, anti-alveolar basement membrane (ABM) antibodies. Consequently, it may manifest as a rapidly progressive, isolated glomerulonephritis (anti-GBM nephritis) or as a pulmonary-renal syndrome with severe lung hemorrhage.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -80554,10 +80542,12 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
+      <SynonymList count="5">
         <Synonym lang="en">CAVC</Synonym>
         <Synonym lang="en">Complete AVSD</Synonym>
+        <Synonym lang="en">Complete atrioventricular canal</Synonym>
         <Synonym lang="en">Complete atrioventricular canal defect</Synonym>
+        <Synonym lang="en">Complete atrioventricular septal defect with atrial and ventricular components</Synonym>
       </SynonymList>
       <DisorderType id="21415">
         <Name lang="en">Morphological anomaly</Name>
@@ -88217,7 +88207,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Simpson-Golabi-Behmel syndrome (SGBS, also referred to as SGBS type 1) is a rare X-linked multiple congenital anomalies syndrome, characterized by pre- and postnatal overgrowth, distinctive craniofacial features, variable congenital malformations, organomegaly and an increased tumor risk.</Contents>
+              <Contents>A rare X-linked multiple congenital anomalies syndrome characterized by pre- and postnatal overgrowth, distinctive craniofacial features, variable congenital malformations, organomegaly and an increased tumor risk.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -92889,7 +92879,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18199">
@@ -96347,7 +96344,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A congenital form of strabismus characterized by limited horizontal eye movement accompanied by globe retraction and palpebral fissure narrowing on attempted adduction. It is an ocular congenital cranial dysinnervation disorder (CCDD) and can lead to amblyopia.</Contents>
+              <Contents>A rare, ocular congenital cranial dysinnervation disorder characterized by limited horizontal eye movement accompanied by globe retraction and palpebral fissure narrowing on attempted adduction.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -131755,7 +131752,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1539">
@@ -132719,7 +132723,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1559">
@@ -141529,7 +141540,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A congenital, X-linked, clinical subtype of L1 syndrome, characterized by variable spastic paraplegia, mild to moderate intellectual deficit, and dysplasia, hypoplasia or aplasia of the corpus callosum. This subtype represents an unusual clinical presentation of the L1 syndrome spectrum in which no hydrocephalus, adducted thumbs, or absent speech are observed.</Contents>
+              <Contents>A congenital, X-linked, clinical subtype of L1 syndrome, characterized by variable spastic paraplegia, mild to moderate intellectual disability, and dysplasia, hypoplasia or aplasia of the corpus callosum. In this subtype hydrocephalus, adducted thumbs, or absent speech are not observed.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -153489,7 +153500,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2014">
@@ -155399,7 +155417,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Congenital muscular dystrophy due to LMNA mutation is a rare congenital muscular dystrophy characterized by prominent axial hypotonia, dropped head syndrome, predominantly proximal muscle weakness in upper limbs/distal in lower limbs (with absent, poor or lost motor development), joint contractures (initially distal, later proximal), spine rigidity, and early respiratory insufficiency, in the presence of moderately elevated serum creatine kinase. Cardiac arrhythmias and sudden death have been also reported.</Contents>
+              <Contents>A rare congenital muscular dystrophy characterized by prominent axial hypotonia, predominantly proximal muscle weakness in upper limbs and distal in lower limbs, joint contractures (initially distal, later proximal), spinal rigidity, and progressive respiratory insufficiency, in the presence of moderately elevated serum creatine kinase. Cardiac arrhythmias and sudden death have also been reported.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -160712,7 +160730,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="en">Mild peroxismal disorder due to PEX10 deficiency</Synonym>
+        <Synonym lang="en">Mild peroxisomal disorder due to PEX10 deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -167447,6 +167465,38 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="19474">
+      <OrphaCode>244275</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=244275</ExpertLink>
+      <Name lang="en">De novo thrombotic microangiopathy after kidney transplantation</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21429">
+        <Name lang="en">Particular clinical situation in a disease or syndrome</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term designates an indication for a pharmacogenetic test.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="2259">
       <OrphaCode>2438</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2438</ExpertLink>
@@ -185920,7 +185970,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Noonan-like syndrome with loose anagen hair (NS/LAH) is a Noonan-related syndrome, characterized by facial anomalies suggestive of Noonan syndrome (see this term); a distinctive hair anomaly described as loose anagen hair syndrome (see this term); frequent congenital heart defects; distinctive skin features with darkly pigmented skin, keratosis pilaris, eczema or occasional neonatal ichtyosis (see this term); and short stature, often associated with a GH deficiency and psychomotor delays.</Contents>
+              <Contents>A Noonan-related syndrome, characterized by facial anomalies suggestive of Noonan syndrome, loose anagen hair, frequent congenital heart defects, distinctive skin features (darkly pigmented skin, keratosis pilaris, eczema or ichtyosis), and short stature that is often associated with a growth hormone deficiency. Psychomotor delay with attention deficit/hyperactivity disorder (ADHD) is frequently observed.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -187978,7 +188028,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Oculocerebrofacial syndrome, Kaufman type is characterized by psychomotor retardation, microcephaly, upslanting palpebral fissures, eye abnormalities (microcornea, strabismus, myopia, optic atrophy), high-arched palate, preauricular skin tags and micrognathia with respiratory distress. It has been described in about 10 cases. Other anomalies can be present: long thin hands and feet, ambiguous genitalia, hypertelorism, &lt;I&gt;etc&lt;/I&gt;. An autosomal recessive mode of inheritance seems most likely.</Contents>
+              <Contents>A rare, genetic, syndromic intellectual disability characterized by severe intellectual disability, distinctive craniofacial features and variable multiple congenital anomalies including ocular, brain, urogenital and skeletal abnormalities.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -207342,7 +207392,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Rare disorders characterized by intrauterine and postnatal growth retardation, microcephaly, facial dysmorphism, skeletal dysplasia, low-birth weight and brain anomalies. Although they were originally described as two separate entities on the basis of radiological criteria (notably small differences in pelvic and long bone structure), later reports confirmed that the two forms represent different modes of expression of the same syndrome.</Contents>
+              <Contents>A rare, severe, primary bone dysplasia characterized by intrauterine and postnatal growth retardation, microcephaly, facial dysmorphism, skeletal dysplasia, low-birth weight and brain anomalies.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -228413,7 +228463,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare congenital malformation characterized by a unilateral, complete or partial, absence of the pectoralis major (and often minor) muscle, ipsilateral breast and nipple anomalies, hypoplasia of the pectoral subcutaneous tissue, absence of pectoral and axillary hair, and possibly accompanied by chest wall and/or upper arm defects.</Contents>
+              <Contents>A rare congenital malformation characterized by a unilateral, complete or partial, absence of the pectoralis major (and often minor) muscle, ipsilateral breast and nipple anomalies, hypoplasia of the pectoral subcutaneous tissue, absence of pectoral and axillary hair, and possibly accompanied by chest wall and/or upper limb defects.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -233791,7 +233841,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>L1 syndrome is a mild to severe congenital X-linked developmental disorder characterized by hydrocephalus of varying degrees of severity, intellectual deficit, spasticity of the legs, and adducted thumbs. The syndrome represents a spectrum of disorders including: X-linked hydrocephalus with stenosis of the aqueduct of Sylvius (HSAS), MASA syndrome, X-linked complicated hereditary spastic paraplegia type 1, and X-linked complicated corpus callosum agenesis (see these terms).</Contents>
+              <Contents>A rare, congenital X-linked developmental disorder characterized by hydrocephalus of varying degrees of severity, intellectual deficit, spasticity of the legs, and adducted thumbs. The syndrome represents a spectrum of disorders including: X-linked hydrocephalus with stenosis of the aqueduct of Sylvius (HSAS), MASA syndrome, X-linked complicated hereditary spastic paraplegia type 1, and X-linked complicated corpus callosum agenesis.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -241427,7 +241477,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Summitt syndrome is an extremely rare disorder originally described in two brothers and characterized by mild to severe craniosynostosis and syndactyly, obesity, and normal intelligence. Acrocephaly, brachydactyly, clinodactyly, mild syndactyly of the hands and feet, genu valgum and marked obesity were later described in another patient. There have been no further descriptions in the literature since 1979. Summitt syndrome could be a variant of Carpenter syndrome.</Contents>
+              <Contents>A rare syndromic trigonocephaly characterized by marked malformations of the head and face (essentially acrocephaly), broad depressed nasal bridge, narrow maxillae, abnormalities of the hands and feet (polydactyly, brachydactyly, syndactyly, clinodactyly, camptodactyly, ulnar deviation), obesity and congenital heart disease. This disease is considered a variant of Carpenter syndrome without intellectual disabaility. There have been no further descriptions in the literature since 1992.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -244396,7 +244446,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Microform holoprosencephaly is a benign form of holoprosencephaly (HPE; see this term) characterized by midline defects without the typical HPE defect in brain cleavage.</Contents>
+              <Contents>A benign form of holoprosencephaly characterized by midline defects without the typical HPE defect in brain cleavage and which can variably manifest with microcephaly, hypotelorism, midline cleft lip and/or flat nose, choanal stenosis, pyriform sinus stenosis, coloboma as well as a single median maxillary incisor.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -251183,8 +251233,7 @@
           <Label>Non-rare disease in Europe</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="11">
-        <Synonym lang="en">CH</Synonym>
+      <SynonymList count="10">
         <Synonym lang="en">Ciliary neuralgia</Synonym>
         <Synonym lang="en">Cluster migraine</Synonym>
         <Synonym lang="en">Erythromelalgia of the head</Synonym>
@@ -278474,7 +278523,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>von Willebrand disease (VWD) is a hereditary bleeding disorder caused by a genetic anomaly leading to quantitative, structural or functional abnormalities of the Willebrand factor (von Willebrand factor; VWF). Two major groups of VWF deficiency have been defined: quantitative and partial (type 1) or total (type 3), and qualitative (type 2) with several subtypes (2A, 2B, 2M, 2N; see these terms).</Contents>
+              <Contents>A rare, inherited bleeding disorder characterized by defective platelet adhesion and secondary coagulation defect that manifests as abnormal bleeding of variable severity occurring either spontaneously or in association with an invasive procedure. Three main subtypes are defined based on the type of von Willebrand factor defect: partial (type 1) or total (type 3) deficiency, and qualitative/functional anomalies (type 2).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -282117,7 +282166,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Opitz G/BBB syndrome (OS) is a multiple congenital anomalies disorder characterized by malformations of the midline including hypertelorism, laryngo-tracheo-esophalgeal defects and hypospadias. There are two clinically indistinguishable genetic subtypes of Opitz G/BBB: X-linked Opitz G/BBB syndrome (XLOS), and autosomal dominant Opitz G/BBB syndrome (ADOS) (see these terms).</Contents>
+              <Contents>A rare X-linked congenital midline malformation syndrome characterized by hypertelorism, laryngo-tracheo-esophageal defects and hypospadias.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -282241,7 +282290,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121767" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="157855" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare hematologic disease characterized by eosinophilia without evidence of clonality persisting for at least six months, for which no underlying cause can be identified. The condition is associated with signs of organ damage and dysfunction. Clinical manifestations are highly variable, depending on the organ systems involved, and include rapidly developing, life-threatening cardiovascular or neurological complications.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18844">
@@ -287823,7 +287882,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Semilobar holoprosencephaly is one of the classical forms of holoprosencephaly (HPE; see this term) in which the left and right frontal and parietal lobes are fused and the interhemispheric fissure is only present posteriorly.</Contents>
+              <Contents>A form of holoprosencephaly characterized by fusion of the left and right frontal and parietal lobes with only a posterior interhemispheric fissure. Craniofacial features variably include ocular hypotelorism, midline cleft lip (complete or partial) and a flat nose.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -293710,7 +293769,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121861" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158256" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare autoimmune polyendocrinopathy characterized by autoimmune activity against an endocrine organ in combination with at least one more endocrine or non-endocrine organ. Typical autoimmune diseases occurring in this type include insulin-requiring diabetes, pernicious anemia, alopecia, vitiligo, or myasthenia gravis, but not Addison disease, thyroid disease, or hypoparathyroidism.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="3762">
@@ -296628,7 +296697,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Pseudoxanthoma elasticum (PXE) is an inherited connective tissue disorder characterized by progressive calcification and fragmentation of elastic fibers in the skin, retina, and arterial walls.</Contents>
+              <Contents>A rare, genetic, metabolic disease with connective tissue and eye involvement, characterized by progressive ectopic mineralization and fragmented elastic fibers in the skin, retina and vascular walls.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -304487,7 +304556,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Narcolepsy with cataplexy is a sleep disorder characterized by excessive day-time sleepiness associated with uncontrollable sleep urges and cataplexy (loss of muscle tone often triggered by pleasant emotions).</Contents>
+              <Contents>A rare neurologic disease characterized by excessive daytime sleepiness associated with uncontrollable sleep urges and cataplexy (sudden loss of muscle tone while awake, often triggered by pleasant emotions).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -319358,7 +319427,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Proteasome disability syndrome describes a group of autosomal recessively inherited autoinflammatory disorders characterized by lipodystrophy and skin eruptions. The disorders belonging to this group include Nakajo-Nishimura syndrome (NNS), JMP syndrome and CANDLE syndrome and all are caused by mutations in the &lt;i&gt;PSMB8&lt;/i&gt; gene (6p21.3).</Contents>
+              <Contents>A rare, autosomal recessive autoinflammatory disorder characterized by early-onset erythematous popular/nodular skin eruptions, recurrent fever, possible joint contractures, lipodystrophy, erythematous inflammatory skin changes, joint and muscle involvement (joint contractures, arthralgia, muscle weakness), and hepatosplenomegaly.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -333932,7 +334001,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, genetic, multiple congenital anomalies/dysmorphic syndrome characterized by varying degrees of intellectual disability, global developmental delay (notably with severe speech and language impairment), muscular hypotonia, and facial dysmorphism (i.e. broad forehead, bitemporal narrowing, upslanting palpebral fissures, low-set ears, flat nasal bridge, bulbous nose and, variably, macroglossia). Highly variable additional features include cardiac defects (including persistent foramen ovale, ventricular septal defects, tetralogy of Fallot), coordination problems, seizures, abnormal growth parameters (including microcephaly, low birth and postnatal weight), and brain morphology anomalies (such as ventriculomegaly and myelination defects).</Contents>
+              <Contents>A rare, genetic syndromic intellectual disability characterized by developmental delay, mild to severe intellectual disability, facial features (bulbous nasal tip, and macroglossia, macrostomia, or open mouth appearance) and a wide spectrum of other nonspecific variable clinical features.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -338836,7 +338905,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>THOC6-related developmental delay-microcephaly-facial dysmorphism syndrome is a rare, genetic, syndromic intellectual disability disorder characterized by global development delay, microcephaly, moderate to severe intellectual disability and facial dysmorphism which includes tall forehead, high anterior hairline, short upslanting palpebral fissures, deep-set eyes and a long nose with a low-hanging columella. Additionally, congenital renal and cardiac malformations (such as horseshoe kidney, unilateral renal agenesis atrioventricular septal defects, patent ductus arteriosus), as well as corpus callosum dysplasia, may be associated.</Contents>
+              <Contents>A rare, autosomal recessive, syndromic intellectual disability disorder characterized by global development delay, mild microcephaly, mild to severe intellectual disability and non-specific facial dysmorphism in association with variable multiple congenital anomalies including congenital heart defects, dental anomalies, cryptorchidism, renal and cerebral malformations. Short stature is frequent.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -352461,7 +352530,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121775" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="157895" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare familial congenital mitral malformation characterized by systolic displacement of one or both mitral leaflets &gt;2 mm beyond the annular plane into the left atrium. Typical histological findings include myxomatous degeneration and degradation of collagen and elastin. Patients may remain asymptomatic or develop complications such as severe mitral regurgitation, endocarditis, and heart failure.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20836">
@@ -354585,8 +354664,7 @@
           <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="4">
-        <Synonym lang="en">APS</Synonym>
+      <SynonymList count="3">
         <Synonym lang="en">Antiphospholipid antibody syndrome</Synonym>
         <Synonym lang="en">Familial lupus anticoagulant</Synonym>
         <Synonym lang="en">Hughes syndrome</Synonym>
@@ -361310,12 +361388,15 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
+        <SummaryInformation id="121862" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158257" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A group of rare endocrine diseases characterized by autoimmune activity against more than one endocrine organ, with possible additional involvement of non-endocrine organs. Autoimmunity is typically directed against different target antigens in different tissues. The two more common autoimmune polyendocrine syndromes (APS), APS type 1 and type 2, have a strong genetic background and have Addison's disease as a major feature. The group furthermore includes APS type 3 and type 4.</Contents>
+            </TextSection>
           </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
-          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -361393,8 +361474,7 @@
           <Label>Non-rare disease in Europe</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">RA</Synonym>
+      <SynonymList count="0">
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -371036,7 +371116,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21345">
@@ -374797,12 +374884,15 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
+        <SummaryInformation id="121840" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158128" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A congenital, X-linked, clinical subtype of L1 syndrome, characterized by spastic paraplegia, mild to moderate intellectual disability and normal brain morphology. This subtype represents the milder end of the L1 syndrome spectrum.</Contents>
+            </TextSection>
           </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is described under  L1 syndrome</Info>
-          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -378854,7 +378944,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare genetic endocrine disease characterized by early-onset (from &lt;12 months to 3 years), rapid and excessive acceleration of linear growth and body size due to pituitary mixed growth hormone- and prolactin-secreting adenomas and/or mixed-cell pituitary hyperplasia. Patients present with gigantism and may have associated acromegalic features (e.g. coarse facial features, frontal bossing, prognathism, increased interdental space) as well as marked enlargement of hands and feet, soft tissue swelling, increased appetite and acanthosis nigricans.</Contents>
+              <Contents>A rare, genetic pituitary disease characterized by infantile-onset, rapid and excessive acceleration of linear growth and body size due to mixed growth hormone (GH)- and prolactin-secreting adenomas and/or pituitary hyperplasia. Patients present with gigantism and may have associated acromegalic features (e.g. coarse facial features, frontal bossing, prognathism, increased interdental space) as well as marked enlargement of hands and feet, soft tissue swelling, increased appetite and acanthosis nigricans.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -399375,7 +399465,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="22715">
@@ -405477,7 +405574,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121776" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="157896" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic disease characterized by co-occurrence of sick sinus syndrome (manifesting as sinus bradycardia, often requiring pacemaker implantation) and chronic intestinal pseudo-obstruction (which may be of myogenic or neurogenic origin and usually requires total parenteral nutrition), with an age of onset within the first four decades of life. Other cardiac features, such as atrial flutter or fibrillation and valve anomalies, may also be present.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23320">
@@ -406592,8 +406699,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
         <Synonym lang="en">FAR1 deficiency</Synonym>
+        <Synonym lang="en">PFCRD</Synonym>
+        <Synonym lang="en">Peroxisomal fatty acyl-CoA reductase 1 disorder</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -416809,7 +416918,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121768" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="157856" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare hematologic disease characterized by clinical and morphological findings indistinguishable from those of acute myeloid leukemia, typically occurring in newborns with Down syndrome. Peripheral blood and bone marrow blasts display features suggestive of megakaryoblasts. In addition to trisomy 21, acquired GATA1 mutations are present in blast cells. Patients may be asymptomatic or present with thrombocytopenia, less commonly other cytopenias, leukocytosis, hepatosplenomegaly, jaundice, ascites, respiratory distress, bleeding, and pericardial or pleural effusions. Most patients undergo spontaneous remission within the first three months of life, although some may develop life-threatening hepatic, renal, or cardiac complications.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23116">
@@ -428122,7 +428241,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121769" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="157857" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A group of esophageal epithelial neoplasms characterized by neuroendocrine differentiation, comprising well-differentiated neuroendocrine tumors (NETs), poorly differentiated neuroendocrine carcinomas (NECs), and mixed neuroendocrine-non-neuroendocrine neoplasms, an umbrella category including mixed adenoneuroendocrine carcinoma. The tumors typically occur in the lower esophagus, often in association with Barrett mucosa. NECs may also arise in other parts of the esophagus. On endoscopy, NETs usually appear as small polypoid or nodular submucosal masses, while NECs are large, infiltrative, and ulcerated. Patients most commonly present with dysphagia, pain, weight loss, and sometimes melena. Metastatic NETs may be associated with carcinoid syndrome.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="26441">
@@ -438764,7 +438893,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Endocrine tumours, also referred to as neuroendocrine tumours (NETs), are defined by a common phenotype which is characterized by the expression of general markers (neuron specific enolase, chromogranin, synaptophysin) and hormone secretion products. These tumours may be localized in any part of the body and are generally discovered in non-specific situations, i.e. not immediately suggestive of NETs (tests for inherited predisposition to tumours or for a clinical syndrome caused by abnormal hormone secretion).</Contents>
+              <Contents>A group of rare tumors characterized by predominantly neuroendocrine differentiation, potentially arising in most organs of the body, including the central nervous system, respiratory tract, larynx, gastrointestinal tract, thyroid, skin, breast, and urogenital system. The gastrointestinal tract and lungs are the most common primary tumor sites. Based on clinical behavior, histology, and proliferation rate, the tumors may be categorized as well differentiated (low grade to intermediate grade) neuroendocrine tumors and poorly differentiated (high grade) neuroendocrine carcinomas. They may or may not be associated with clinical hormone hypersecretion syndromes.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -452301,7 +452430,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Post-transplant lymphoproliferative disease (PTLD) describes a heterogeneous group of lymphoproliferative diseases occurring in the context of post-transplant immunosuppression and ranging from benign polyclonal B-cell proliferation to malignant lymphomas.</Contents>
+              <Contents>A group of rare immunodeficiency-associated lymphoproliferative disorders characterized by lymphoid or plasmacytic proliferations developing in the context of immunosuppression in a recipient of a solid organ or stem cell allograft. The group includes non-destructive post-transplant lymphoproliferative disorders (PTLDs), polymorphic PTLD, monomorphic PTLDs, and classic Hodgkin lymphoma PTLD. Patients may have more than one type of PTLD in a single or in different locations. The most commonly involved sites are lymph nodes, gastrointestinal tract, lungs, and liver, although the disease may occur almost anywhere in the body. In solid organ transplant recipients, PTLD may also involve the allograft.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -456797,7 +456926,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Goodman syndrome is an extremely rare genetic disorder characterized by marked malformations of the head and face (essentially acrocephaly), abnormalities of the hands and feet (polydactyly, syndactyly, clinodactyly, camptodactyly, ulnar deviation), and congenital heart disease. There have been no further descriptions in the literature since 1979. Goodman syndrome could be a variant of Carpenter syndrome.</Contents>
+              <Contents>A rare syndromic trigonocephaly characterized by marked malformations of the head and face (essentially acrocephaly), broad depressed nasal bridge, narrow maxillae, abnormalities of the hands and feet (polydactyly, brachydactyly, syndactyly, clinodactyly, camptodactyly, ulnar deviation), obesity and congenital heart disease. This disease is considered a variant of Carpenter syndrome without intellectual disability. There have been no further descriptions in the literature since 1992.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -456892,7 +457021,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A multi-system disorder characterized by cardiac, hand, facial and neurodevelopmental features that include QT prolongation, webbed fingers and toes, flattened nasal bridge, low-set ears, small upper jaw, thin upper lip, and characteristic features of autism or autistic spectrum disorders.</Contents>
+              <Contents>A rare, multiple congenital anomalies syndrome with cardiac involvement as a major feature characterized by QT prolongation, congenital heart defects, syndactyly, facial dysmorphism and neurodevelopmental features. There three clinical phenotypes recognized, the classical types that present with a prolonged QT interval and either with (TS1) or without (TS2) cutaneous syndactyly of fingers and toes. The atypical form (ATS) causes multi-system health concerns but not necessarily with prolonged QT.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -458013,7 +458142,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Carpenter syndrome is a subtype of a family of genetic disorders known as acrocephalopolysyndactyly (ACPS) disorders.</Contents>
+              <Contents>A rare syndromic craniosynostosis with variable phenotypic expression characterized by craniosynostosis, intellectual disability, distinctive facies, abnormalities of the fingers and toes (brachydactyly, polydactyly and syndactyly), short stature, congenital heart disease, skeletal defects, obesity, genital abnormalities and umbilical hernia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -469254,7 +469383,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11189">
@@ -471562,7 +471698,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122246" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158774" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare form of mast cell leukemia characterized by the presence of at least 20% mast cells in bone marrow aspirate smears but often mature mast cell morphology, low proliferation rate, and absence of organ damage and C findings (cytopenias, hepatomegaly, ascites, portal hypertension, splenomegaly, skeletal lesions, malabsorption). The disease course is less aggressive than in the acute form, although patients may later progress.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11153">
@@ -476554,7 +476700,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Non-24-hour sleep-wake disorder (non-24 disorder), also known as hypernychthemeral syndrome, is a circadian rhythm sleep disorder characterized by non-synchronization to a 24-hour day leading to insomnia and daytime sleepiness with sometimes severe associated manifestations.</Contents>
+              <Contents>A rare neurological disease which is a circadian rhythm sleep disorder characterized by non-synchronization to a 24-hour day leading to insomnia and daytime sleepiness with sometimes severe associated manifestations.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -480810,7 +480956,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121836" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158125" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare parathyroid disease and phosphocalcic metabolism anomaly characterized by hypocalcemia, hyperphosphatemia, hypercalciuria, and low serum parathyroid hormone levels, in the presence of autoantibodies against parathyroid tissue. Clinical signs and symptoms are of variable severity and include paresthesia, seizures, laryngospasm, tetany, cardiac dysrhythmias, calcifications of the basal ganglia, and neuropsychological manifestations such as anxiety, depression, confusion, or hallucination. The condition may occur as an isolated disease or in association with other autoimmune diseases.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10437">
@@ -481453,7 +481609,17 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122243" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158771" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A group of rare acute leukemias of ambiguous lineage characterized by the presence of separate populations of blasts of more than one lineage (bilineal), a single population of blasts coexpressing antigens of more than one lineage (biphenotypic), or a combination thereof. The diagnosis relies on immunophenotyping, the T-cell component being characterized by strong expression of cytoplasmic CD3, usually in the absence of surface CD3, the B-cell component expressing CD19, almost always together with CD10, cCD79a, CD22, or PAX5, while the most specific hallmark of the myeloid component is the presence of myeloperoxidase in the blast cytoplasm.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10465">
@@ -485176,10 +485342,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
+      <SynonymList count="2">
         <Synonym lang="en">F5F8D</Synonym>
         <Synonym lang="en">FV and FVIII combined deficiency</Synonym>
-        <Synonym lang="en">Familial multiple coagulation factor deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -487761,7 +487926,8 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="1">
+        <Synonym lang="en">Idiopathic excessive sleepiness</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -487824,7 +487990,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Idiopathic hypersomnia is a sleep disorder classified in two forms: idiopathic hypersomnia with long sleep time and idiopathic hypersomnia without long sleep time.</Contents>
+              <Contents>A rare neurologic disease characterized by an excessive daytime sleepiness with long and unrefreshing naps, and/or prolonged and undisturbed nocturnal sleep, impaired daytime alertness, and/or sleep inertia (ie, great difficulty in waking up after sleep) and where other causes have been excluded.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -502097,7 +502263,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10523">
@@ -502135,7 +502308,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28123">
@@ -515472,7 +515652,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122241" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158769" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare intestinal disease characterized by immune-mediated injury of the intestinal mucosa, leading to severe, chronic, intractable diarrhea, malabsorption, and severe weight loss or failure to thrive. Characteristic histologic findings in the small intestine include partial or complete blunting of the villi, deep crypt lymphocytosis, increased crypt apoptosis, and minimal surface intraepithelial lymphocytosis. In addition, the stomach, colon, and esophagus may also be involved. Circulating autoantibodies against enterocytes and/or goblet cells are found in many, but not all, patients. The diagnosis requires exclusion of other causes of villous atrophy.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11787">
@@ -517716,7 +517906,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="3">
+        <Synonym lang="en">PKDTS</Synonym>
+        <Synonym lang="en">TSC2/PKD1 contiguous gene syndrome</Synonym>
         <Synonym lang="en">Tuberous sclerosis/polycystic kidney disease contiguous gene syndrome</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -527743,7 +527935,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122242" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158770" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare hematologic disease characterized by symptoms of mast cell activation in the absence of cutaneous findings, as well as absence of diagnostic criteria of systemic mastocytosis with tryptase levels of less than 20 ng/ml and normal to low burden of mast cells. Bone marrow biopsy reveals the presence of monoclonal mast cells carrying the KIT D816V mutation and/or expressing CD25. Patients present with recurrent episodes of flushing, headache, hypotension, abdominal cramping, nausea, diarrhea, cardiac arrhythmias, bronchoconstriction, and bleeding diathesis, among others.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12041">
@@ -549131,9 +549333,8 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
+      <SynonymList count="2">
         <Synonym lang="en">Still disease</Synonym>
-        <Synonym lang="en">Systemic polyarthritis</Synonym>
         <Synonym lang="en">Systemic-onset JIA</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -549210,7 +549411,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Systemic-onset juvenile idiopathic arthritis is marked by the severity of the extra-articular manifestations (fever, cutaneous eruptions) and by an equal sex ratio.</Contents>
+              <Contents>A rare pediatric rheumatological disease characterized by the variable occurrence of chronic arthritis, intermittently high spiking fever, maculopapular rash during fever episodes, hepatomegaly and/or splenomegaly, lymphadenopathy, and serositis.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -552791,7 +552992,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122244" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158772" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare acute myeloid leukemia (AML) characterized by the presence of acute leukemia with at least 20% peripheral blood or bone marrow blasts with morphological features of myelodysplasia, or occurrence in patients with a prior history of a myelodysplastic syndrome (MDS) or myelodysplastic/myeloproliferative neoplasm, with MDS-related cytogenetic abnormalities, in the absence of specific genetic abnormalities characteristic of AML with recurrent genetic abnormalities. Prior cytotoxic or radiation therapy for an unrelated disease must be excluded. The condition occurs mainly in elderly patients and is rare in children. Patients often present with severe pancytopenia. Prognosis is generally poor.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11766">
@@ -556265,7 +556476,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A disorder that is characterized by excessive day-time sleepiness associated with uncontrollable sleep urges and sometimes paralysis at sleep, hypnagogic hallucinations and automatic behavior.</Contents>
+              <Contents>A rare neurologic disease characterized by excessive daytime sleepiness associated with uncontrollable sleep urges and sometimes sleep paralysis, and hypnagogic/hypnopompic hallucinations.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -559342,8 +559553,7 @@
           <Label>Non-rare disease in Europe</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="5">
-        <Synonym lang="en">APA</Synonym>
+      <SynonymList count="4">
         <Synonym lang="en">Aldosterone-secreting adenoma</Synonym>
         <Synonym lang="en">Aldosteronoma</Synonym>
         <Synonym lang="en">Conn adenoma</Synonym>
@@ -581778,7 +581988,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122239" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158767" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare non-amyloid monoclonal immunoglobulin deposition disease characterized by secretion of abnormal light and heavy chains, which are deposited in tissues and cause organ dysfunction, but do not form amyloid beta-pleated sheets or contain an amyloid P component. The condition most frequently occurs in association with multiple myeloma. The kidneys are most commonly affected (clinically manifesting as nephrotic syndrome and renal failure), but liver, heart, peripheral nerves, blood vessels, and joints may also be involved.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12398">
@@ -581817,7 +582037,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122238" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158766" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare non-amyloid monoclonal immunoglobulin deposition disease characterized by production of monoclonal immunoglobulins with truncated heavy chains and no detectable light chains, which are deposited in tissues and cause organ dysfunction, but do not form amyloid beta-pleated sheets or contain an amyloid P component. The condition frequently occurs in association with multiple myeloma. Patients most commonly present with renal involvement (manifesting as hypertension, progressive renal dysfunction, anemia, and nephrotic syndrome with microhematuria), but other organs (such as the liver or skin) may also be affected. Production of IgG1 or IgG3 isotypes results in hypercomplementemia.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12393">
@@ -582137,7 +582367,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122240" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="158768" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare non-amyloid monoclonal immunoglobulin deposition disease characterized by deposition of abnormal immunoglobulin light chains in the kidneys, resulting in nephrotic syndrome and renal failure. Symptomatic extrarenal deposition is uncommon, although hepatic, cardiac, and neural deposits have been reported. The condition frequently occurs in association with multiple myeloma or in patients with M protein and marrow plasma cells at monoclonal gammopathy of undetermined significance levels.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12401">
@@ -583301,7 +583541,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12370">
@@ -584920,7 +585167,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12334">
@@ -586527,8 +586781,7 @@
           <Label>Non-rare disease in Europe</Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
-        <Synonym lang="en">BDD</Synonym>
+      <SynonymList count="0">
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
@@ -588688,7 +588941,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Lobar holoprosencephaly is the mildest classical form of holoprosencephaly (HPE; see this term) characterized by separation of the right and left cerebral hemispheres and lateral ventricules with some continuity across the frontal neocortex, especially rostrally and ventrally.</Contents>
+              <Contents>A form of holoprosencephaly characterized by separation of the right and left cerebral hemispheres and lateral ventricules with some continuity only across the frontal neocortex, especially rostrally and ventrally. Craniofacial features are variable may include ocular hypotelorism, midline cleft lip (complete or partial) and/or flat nose amongst other features.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -588791,7 +589044,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A disorder of the most severe classical form of holoprosencephaly (HPE) characterized by a single brain ventricle and no interhemispheric fissure.</Contents>
+              <Contents>A severe form of holoprosencephaly characterized by a single brain ventricle and no interhemispheric fissure. Severe craniofacial features may manifest as cyclopia, ethmocephaly or cebocephaly.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -591038,7 +591291,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Congenital thrombotic thrombocytopenic purpura is the hereditary form of thrombotic thrombocytopenic purpura (TTP; see this term) characterized by profound peripheral thrombocytopenia, microangiopathic hemolytic anemia (MAHA) and single or multiple organ failure of variable severity.</Contents>
+              <Contents>A hereditary form of thrombotic thrombocytopenic purpura (TTP) characterized by profound peripheral thrombocytopenia, microangiopathic hemolytic anemia (MAHA) and single or multiple organ failure of variable severity.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -597776,8 +598029,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21436">
-        <Name lang="en">Clinical group</Name>
+      <DisorderType id="36561">
+        <Name lang="en">Category</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="en">Group of disorders</Name>
@@ -601882,7 +602135,7 @@
     <Disorder id="29437">
       <OrphaCode>589608</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589608</ExpertLink>
-      <Name lang="en">Linear hypopigmentation and craniofacial asymmetry with acral, ocular, and brain anomalies</Name>
+      <Name lang="en">Linear hypopigmentation and craniofacial asymmetry with acral, ocular and brain anomalies</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -602750,7 +603003,7 @@
     <Disorder id="29430">
       <OrphaCode>589515</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589515</ExpertLink>
-      <Name lang="en">PADDAS syndrome</Name>
+      <Name lang="en">PUM1-associated developmental disability-ataxia-seizure syndrome</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -602758,7 +603011,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="en">PUM1-associated developmental disability, ataxia, and seizure syndrome</Synonym>
+        <Synonym lang="en">PADDAS syndrome</Synonym>
         <Synonym lang="en">SCA47</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -611158,7 +611411,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="121774" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="157894" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare sporadic human prion disease characterized by adult onset of progredient neurodegeneration presenting as a combination of psychiatric, sleep, and oculomotor disturbances, with development of progressive cognitive impairment (the predominantly affected cognitive domains being memory, temporal and/or spatial orientation, language, executive functions, and attention), postural instability, and sometimes additional motor abnormalities and autonomic hyperactivity, in the course of the disease. Bilateral thalamic hypometabolism on FDG-PET imaging and positive prion seeding activity in the cerebrospinal fluid are present in many cases. The disease is fatal within typically two to three years.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="14059">
@@ -622957,7 +623220,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="13327">
