commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml
index 60f94e5..0986113 100644
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:35:57" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:37:21" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10603">
+  <DisorderList count="10617">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -18267,14 +18267,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="167">
@@ -64338,15 +64331,14 @@
     <Disorder id="17996">
       <OrphaCode>177926</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=177926</ExpertLink>
-      <Name lang="en">Bleeding disorder in hemophilia A carriers without FVIII deficiency</Name>
+      <Name lang="en">Bleeding disorder in hemophilia A carriers</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
-        <Synonym lang="en">Bleeding disorder in hemophilia A carriers without Factor VIII deficiency</Synonym>
+      <SynonymList count="1">
         <Synonym lang="en">Symptomatic form of hemophilia A in female carriers</Synonym>
       </SynonymList>
       <DisorderType id="21450">
@@ -64550,15 +64542,14 @@
     <Disorder id="17997">
       <OrphaCode>177929</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=177929</ExpertLink>
-      <Name lang="en">Bleeding disorder in hemophilia B carriers without FIX deficiency</Name>
+      <Name lang="en">Bleeding disorder in hemophilia B carriers</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
-        <Synonym lang="en">Bleeding disorder in hemophilia B carriers without Factor IX deficiency</Synonym>
+      <SynonymList count="1">
         <Synonym lang="en">Symptomatic form of hemophilia B in female carriers</Synonym>
       </SynonymList>
       <DisorderType id="21450">
@@ -81287,12 +81278,20 @@
     <Disorder id="813">
       <OrphaCode>1549</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1549</ExpertLink>
-      <Name lang="en">Cryptosporidiosis</Name>
-      <DisorderFlagList count="1">
+      <Name lang="en">NON RARE IN EUROPE: Cryptosporidiosis</Name>
+      <DisorderFlagList count="3">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
+        <DisorderFlag id="456">
+          <Value>32</Value>
+          <Label>Non-rare disease in Europe</Label>
+        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
@@ -81302,18 +81301,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="5">
-        <ExternalReference id="107110">
-          <Source>MedDRA</Source>
-          <Reference>10011502</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="1">
         <ExternalReference id="107113">
           <Source>ICD-10</Source>
           <Reference>A07.2</Reference>
@@ -81327,53 +81315,10 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="145137">
-          <Source>GARD</Source>
-          <Reference>6219</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="107109">
-          <Source>UMLS</Source>
-          <Reference>C0010418</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="107108">
-          <Source>MeSH</Source>
-          <Reference>D003457</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="3266" lang="en">
-          <TextSectionList count="1">
-            <TextSection id="121223" lang="en">
-              <TextSectionType id="16907">
-                <Name lang="en">Definition</Name>
-              </TextSectionType>
-              <Contents>Cryptosporidiosis is a protozoan disease caused by small coccidia, &lt;i&gt;e.g.&lt;/i&gt; &lt;i&gt;Cryptosporidium spp&lt;/i&gt;, that colonize the intestinal epithelial cells of humans and of a wide variety of animals.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18412">
@@ -139843,7 +139788,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131141" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171340" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare otorhinolaryngological malformation characterized by a unilateral or bilateral fistula located at the corner of the mouth, where the vermillion border of the upper lip meets that of the lower lip. The lesion is lined by labial mucosa. It is potentially susceptible to infection.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="17029">
@@ -152350,7 +152305,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131162" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171359" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare autosomal dominant distal hereditary motor neuropathy disease characterized by muscle weakness and wasting predominantly affecting the hands, in particular the thenar and first dorsal interosseus muscles, and/or marked foot deformity and gait disturbance. Sensation is normal, although reduced response to vibration has been described. The disease is slowly progressive with an age of onset within the first few decades of life.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1749">
@@ -167935,7 +167900,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131176" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171368" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare non-syndromic cerebral malformation due to abnormal neuronal migration characterized by clusters of disorganized neurons in abnormal locations such as periventricular and subcortical. The extent of the lesions ranges from isolated single to bilateral confluent nodules. Pediatric patients typically show variable degrees of developmental delay, intellectual disability, and intractable epilepsy, and concomitant cerebral and/or systemic malformations are frequent. Milder forms may present with onset of seizures in adulthood.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2047">
@@ -188272,7 +188247,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131174" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171366" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic neurological disorder characterized by postnatal onset of severe global developmental delay, profound mental retardation, progressive microcephaly, progressive spasticity evolving into spastic quadriplegia with joint contractures, generalized seizures, and irritability. Severe choreoathetosis and dysmorphic features are absent. Brain imaging shows progressive cerebellar atrophy followed by cerebral atrophy affecting both white and grey matter, but no pontine involvement.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2294">
@@ -199284,7 +199269,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131154" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171358" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare autosomal ichthyosis syndrome with prominent neurologic signs characterized by the association of congenital ichthyosis with severe developmental delay, microcephaly, spastic tetraplegia, sensorineural hearing impairment, athetosis, and myoclonus. Marked epileptic discharges with occurrence of tonic spasms have also been reported. Cerebral MRI shows diffuse cortical atrophy. There have been no further descriptions in the literature since 1995.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19611">
@@ -304392,7 +304387,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131130" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171329" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic glycogen storage disease characterized by either lactate dehydrogenase (LDH) M- or H-subunit deficiency. Main features of LDH M-subunit deficiency are exertional fatigue and muscle pain potentially accompanied by myoglobinuria. Some patients may develop pustular psoriasis-like skin lesions. Complications of pregnancy, such as frequent abdominal pains and increased uterine tone with a risk of dystocia have also been described. LDH H-subunit deficiency manifests with low serum LDH activity of unclear clinical relevance.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="3573">
@@ -340265,7 +340270,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131201" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171372" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare skin disease characterized by a hamartomatous epidermal lesion presenting as a linear array of verrucous, hyperkeratotic papules that often coalesce into plaques and are formed along the lines of Blaschko. The condition is associated with involvement of other organ systems, mainly brain, eye, and skeletal system. It is the result of mosaic post-zygotic mutations and most commonly presents at birth, but may occur anytime during childhood, rarely also in adulthood.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19182">
@@ -355791,7 +355806,7 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="2">
+      <DisorderDisorderAssociationList count="3">
         <DisorderDisorderAssociation>
           <TargetDisorder id="21817" cycle="true"/>
           <RootDisorder id="3393">
@@ -355802,6 +355817,16 @@
             <Name lang="en">Referred to</Name>
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="21817" cycle="true"/>
+          <RootDisorder id="10773">
+            <OrphaCode>57194</OrphaCode>
+            <Name lang="en">OBSOLETE: Aseptic osteitis</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="en">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
         <DisorderDisorderAssociation>
           <TargetDisorder id="21817" cycle="true"/>
           <RootDisorder id="12460">
@@ -364873,12 +364898,15 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
+        <SummaryInformation id="131208" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171380" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare hypereosinophilic syndrome characterized by hypereosinophilia produced by reactive/non-clonal eosinophils secondary to an underlying medical condition and persisting for at least six months. The disorder can derive from non-neoplastic conditions (such as chronic infections and infestations, allergic reactions, intoxications, or autoimmune and chronic inflammatory disorders) or from neoplasms including non-myeloid malignancies, among others. It is associated with signs of organ infiltration, dysfunction, and damage. Clinical manifestations are highly variable, depending on the organ systems involved, and most commonly include dermatologic, pulmonary, cardiac, gastrointestinal, and cerebral manifestations.</Contents>
+            </TextSection>
           </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is described under  Hypereosinophilic syndrome</Info>
-          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -364923,12 +364951,15 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
+        <SummaryInformation id="131198" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171371" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare hypereosinophilic syndrome characterized by hypereosinophilia produced by clonal eosinophils derived from neoplastic stem cells in the absence of any secondary cause of eosinophilia and persisting for at least six months. The condition is associated with signs of organ infiltration, dysfunction, and damage. Clinical manifestations are highly variable, depending on the organ systems involved, and include dermatologic, pulmonary, cardiac, gastrointestinal, and cerebral manifestations, among others.</Contents>
+            </TextSection>
           </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is described under  Hypereosinophilic syndrome</Info>
-          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -381381,7 +381412,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131175" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171367" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare autosomal ichthyosis syndrome with prominent neurologic signs characterized by the association of congenital ichthyosis with global developmental delay, intellectual disability, infantile-onset seizures, and spastic tetraplegia. Brain imaging may show delayed myelination and cerebral atrophy. Marked intrafamilial variability has been reported.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="22056">
@@ -393555,24 +393596,18 @@
     <Disorder id="5517">
       <OrphaCode>80</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=80</ExpertLink>
-      <Name lang="en">NON RARE IN EUROPE: Antiphospholipid syndrome</Name>
-      <DisorderFlagList count="3">
-        <DisorderFlag id="495">
-          <Value>8192</Value>
-          <Label>Inactive</Label>
-        </DisorderFlag>
+      <Name lang="en">Antiphospholipid syndrome</Name>
+      <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
-        <DisorderFlag id="456">
-          <Value>32</Value>
-          <Label>Non-rare disease in Europe</Label>
-        </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
+      <SynonymList count="5">
+        <Synonym lang="en">APLS</Synonym>
         <Synonym lang="en">Antiphospholipid antibody syndrome</Synonym>
-        <Synonym lang="en">Familial lupus anticoagulant</Synonym>
+        <Synonym lang="en">Classic APLS</Synonym>
+        <Synonym lang="en">Classic antiphospholipid syndrome</Synonym>
         <Synonym lang="en">Hughes syndrome</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -393585,14 +393620,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This disease is not rare in Europe. It does not belong to the Orphanet nomenclature of rare diseases.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20670">
@@ -394213,7 +394241,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This disease is described under  Glycogen storage disease due to lactate dehydrogenase deficiency</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20633">
@@ -403245,14 +403280,16 @@
     <Disorder id="21452">
       <OrphaCode>313800</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313800</ExpertLink>
-      <Name lang="en">Optic nerve edema-splenomegaly syndrome</Name>
+      <Name lang="en">Retinal dystrophy-optic nerve edema-splenomegaly-anhidrosis-migraine headache syndrome</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="2">
+        <Synonym lang="en">Optic nerve edema-splenomegaly syndrome</Synonym>
+        <Synonym lang="en">ROSAH syndrome</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -431325,7 +431362,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare constitutional hemolytic anemia characterized in symptomatic forms by mild to severe chronic hemolysis, which is further exacerbated by oxidative stress and may lead to chronic non-shperocytic hemolytic anemia of variable severity. Variation in glucose-6-phosphate dehydrogenase levels accounts for differences in sensitivity to oxidants, with chronic hemolysis occurring in association with very low enzyme levels, while the majority of affected individuals remain asymptomatic. The most common clinical manifestations are neonatal jaundice and signs and symptoms of acute hemolysis (such as fatigue, back pain, anemia, and jaundice).</Contents>
+              <Contents>A rare constitutional hemolytic anemia due to an enzyme disorder characterized by severe glucose-6-phosphate dehydrogenase deficiency (typically &lt;10% residual enzyme activity) associated with chronic non-spherocytic hemolytic anemia of highly variable severity. Patients are at risk of developing neonatal jaundice (potentially leading to kernicterus), gallstones, and reticulocytosis and splenomegaly. They have an increased susceptibility to oxidizing agents provoking episodes of acute hemolysis. Favism, which describes the occurrence of an acute hemolytic reaction in response to the ingestion of fava beans, is more common in infants and young children.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -445989,7 +446026,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131209" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171381" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare osteonecrosis characterized by bone necrosis due to disrupted blood supply in the absence of a known cause. Affected bones include the femoral head, talus, vertebral body, humerus, and scaphoid, among others. Patients may initially be asymptomatic but subsequently present with gradually developing refractory pain, swelling, and reduced range of motion. If left untreated, the condition may progress to bone collapse with secondary degeneration, fragmentation, and pathological fracture, as well as osteoarthritis.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="22759">
@@ -509854,7 +509901,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131142" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171341" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic neurological disorder characterized by early onset of microcephaly, severe global developmental delay and cognitive impairment, dyskinesia and hyperkinetic movements, visual impairment, autistic behavior, stereotypies, sleep disturbance, epilepsy, and cerebral malformations (such as corpus callosum hypogenesis, forebrain anomaly, and delayed myelination). Speech is minimal or absent, and ambulation is not attained. Patients with a larger 14q12 microdeletion show a more severe phenotype than those with intragenic alterations, with the addition of facial dysmorphism and agenesis of the corpus callosum.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10844">
@@ -512626,14 +512683,24 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="21817">
+            <OrphaCode>324964</OrphaCode>
+            <Name lang="en">Chronic nonbacterial osteomyelitis/Chronic recurrent multifocal osteomyelitis</Name>
+          </TargetDisorder>
+          <RootDisorder id="10773" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="en">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="-1" lang="en">
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="en">This entity has been obsoleted from the Orphanet nomenclature of rare diseases.</Info>
+            <Info lang="en">This entity has been obsoleted from the Orphanet nomenclature of rare diseases.&lt;br/&gt;&lt;br/&gt;Instead, consider using  Chronic nonbacterial osteomyelitis/Chronic recurrent multifocal osteomyelitis</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -528480,7 +528547,7 @@
     <Disorder id="11041">
       <OrphaCode>73247</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73247</ExpertLink>
-      <Name lang="en">Eosinophilic esophagitis</Name>
+      <Name lang="en">NON RARE IN EUROPE: Eosinophilic esophagitis</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -528496,89 +528563,11 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="6">
-        <ExternalReference id="114930">
-          <Source>ICD-10</Source>
-          <Reference>K20</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="14890">
-          <Source>OMIM</Source>
-          <Reference>610247</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="45508">
-          <Source>OMIM</Source>
-          <Reference>613412</Reference>
-          <DisorderMappingRelation id="21541">
-            <Name lang="en">BTNT (ORPHA code's Broader Term maps to a Narrower Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114926">
-          <Source>MeSH</Source>
-          <Reference>D057765</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114927">
-          <Source>UMLS</Source>
-          <Reference>C0341106</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114928">
-          <Source>MedDRA</Source>
-          <Reference>10064212</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="en">E (Exact mapping: the two concepts are equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="36750" lang="en">
-          <TextSectionList count="1">
-            <TextSection id="81992" lang="en">
-              <TextSectionType id="16907">
-                <Name lang="en">Definition</Name>
-              </TextSectionType>
-              <Contents>A rare chronic, local immune-mediated disease of the esophagus characterized clinically by symptoms of esophageal dysfunction (including, dysphagia, feeding disorders, food impaction, vomiting and abdominal pain) and histologically by eosinophil-predominant inflammation in esophageal biopsies.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28655">
@@ -528971,7 +528960,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131193" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171369" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic cerebral small vessel disease characterized by isolated marked tortuosity of second-order and third-order retinal arteries with normal first-order arteries and venous system, typically located in the macular and peripapillary area and developing during childhood or early adulthood. The disease may be asymptomatic, although most patients present variable degrees of transient vision loss due to retinal hemorrhage following physical exertion or minor trauma.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11070">
@@ -541273,7 +541272,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131131" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="171330" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare viral disease characterized by fulminant cytomegalovirus infection with multiple organ involvement including the brain, lung, liver, and/or heart, among others, and marked constitutional symptoms in immunocompetent patients. The condition is associated with a high case fatality rate.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10363">
@@ -577758,7 +577767,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Unilateral renal dysplasia is a form of renal dysplasia (RD; see this term), a renal tract malformation in which the development of one kidney is abnormal and incomplete. Unilateral RD can be segmental, and of variable severity, with renal aplasia corresponding to extreme RD.</Contents>
+              <Contents>A form of renal dysplasia (RD) characterized by abnormal or incomplete development of one kidney. Unilateral RD can be segmental, and of variable severity, with renal aplasia corresponding to extreme RD. Patients may be asymptomatic if the contralateral kidney is functional. Even in cases of severe unilateral RD, i.e. renal aplasia, the risk of renal failure in childhood is minimal; however, patients may develop hypertension, proteinuria and renal failure as adults.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -646804,7 +646813,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This disease is described under  FOXG1 syndrome</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12402">
@@ -728688,6 +728704,210 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31348">
+      <OrphaCode>617449</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617449</ExpertLink>
+      <Name lang="en">Congenital aphakia-iris hypoplasia-microphthalmia-microcornea syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31346">
+      <OrphaCode>617408</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617408</ExpertLink>
+      <Name lang="en">Classic eosinophilic pustular folliculitis</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="2">
+        <Synonym lang="en">Classic EPF</Synonym>
+        <Synonym lang="en">Ofuji disease</Synonym>
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31347">
+      <OrphaCode>617440</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617440</ExpertLink>
+      <Name lang="en">Painful legs and moving toes syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">PLMT syndrome</Synonym>
+      </SynonymList>
+      <DisorderType id="21422">
+        <Name lang="en">Clinical syndrome</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31341">
+      <OrphaCode>617307</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617307</ExpertLink>
+      <Name lang="en">Rare disorder related to monochorionic twin pregnancy</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="en">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="en">Group of disorders</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31340">
+      <OrphaCode>617304</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617304</ExpertLink>
+      <Name lang="en">Amniotic fluid embolism</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31343">
+      <OrphaCode>617313</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617313</ExpertLink>
+      <Name lang="en">Rare disorder due to inadequate sharing of the placenta</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="en">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="en">Group of disorders</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31342">
+      <OrphaCode>617310</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617310</ExpertLink>
+      <Name lang="en">Rare disorder due to unbalanced inter-twin blood transfusion</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="en">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="en">Group of disorders</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31337">
+      <OrphaCode>617294</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617294</ExpertLink>
+      <Name lang="en">Twin anemia-polycythemia sequence</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">TAPS</Synonym>
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="31336">
       <OrphaCode>616874</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=616874</ExpertLink>
@@ -728701,11 +728921,11 @@
       <SynonymList count="1">
         <Synonym lang="en">Fully investigated rare disorder without a determined diagnosis</Synonym>
       </SynonymList>
-      <DisorderType id="21464">
-        <Name lang="en">No type</Name>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
       </DisorderType>
-      <DisorderGroup id="38983">
-        <Name lang="en">No type</Name>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
@@ -728718,11 +728938,164 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare disorder for which all reasonable efforts have been done by rare diseases experts to determine a diagnosis according to the state of the art and available diagnostic capabilities, but did not enable to conclude on a clinically known concept. It is recommended to restrict the use of this entity for coding purpose to rare disease experts.</Contents>
+              <Contents>A rare disorder for which all reasonable efforts have been done by rare diseases experts to determine a diagnosis according to the state of the art and available diagnostic capabilities, but did not enable to conclude on a clinically known concept. It is recommended to restrict the use of this entity for coding purposes to rare disease experts.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31339">
+      <OrphaCode>617301</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617301</ExpertLink>
+      <Name lang="en">Selective intrauterine growth restriction</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31338">
+      <OrphaCode>617297</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617297</ExpertLink>
+      <Name lang="en">Twin-reversed arterial perfusion sequence</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">TRAP</Synonym>
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31370">
+      <OrphaCode>617910</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617910</ExpertLink>
+      <Name lang="en">Conjunctival malignant melanoma</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">Conjunctival melanoma</Synonym>
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31371">
+      <OrphaCode>617916</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617916</ExpertLink>
+      <Name lang="en">Diffuse idiopathic pulmonary neuroendocrine cell hyperplasia</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31372">
+      <OrphaCode>617919</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617919</ExpertLink>
+      <Name lang="en">F12-associated cold auto-inflammatory syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31373">
+      <OrphaCode>617930</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617930</ExpertLink>
+      <Name lang="en">Hemophilia B Leyden</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">Factor IX deficiency, Leyden type</Synonym>
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
