commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml
old mode 100755
new mode 100644
index d3cafc2..1b1c2bc
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/en_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:29:12" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:31:41" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10542">
+  <DisorderList count="10543">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -6313,7 +6313,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>An inherited hemoglobinopathy characterized by impaired synthesis of alpha-globin chains leading to a variable clinical picture depending on the number of affected alleles.</Contents>
+              <Contents>A rare inherited hemoglobinopathy characterized by impaired synthesis of two to all four alpha-globin chains leading to a variable clinical picture depending on the number of affected alleles.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -10154,7 +10154,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="97">
@@ -16440,7 +16447,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare, severe form of alpha-thalassemia that is almost always lethal. It is characterized by fetal onset of generalized edema, pleural and pericardial effusions, and severe hypochromic anemia.</Contents>
+              <Contents>A severe form of alpha-thalassemia that is mostly lethal, and associated with severe long-term outcome and lifelong transfusions in survivors. It is characterized by fetal onset of generalized edema, pleural and pericardial effusions, and severe hypochromic anemia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -59470,7 +59477,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Osteogenesis imperfecta (OI) comprises a heterogeneous group of genetic disorders characterized by increased bone fragility, low bone mass, and susceptibility to bone fractures with variable severity.</Contents>
+              <Contents>A rare, genetic, primary bone dysplasias characterized by increased bone fragility, low bone mass, and susceptibility to bone fractures. The clinical severity is heterogeneous.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -71280,7 +71287,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>MEDNIK syndrome, previously known as Erythrokeratodermia Variabilis type 3 (EKV3), is characterized by intellectual deficit, enteropathy, sensorineural hearing loss, peripheral neuropathy, lamellar and erythrodermic ichthyosis, and keratodermia (MEDNIK stands for Mental retardation, Enteropathy, Deafness, peripheral Neuropathy, Ichtyosis, Keratodermia).</Contents>
+              <Contents>A rare disorder of copper metabolism characterized by intellectual deficit, enteropathy, sensorineural hearing loss, peripheral neuropathy, lamellar and erythrodermic ichthyosis, and keratodermia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -74856,9 +74863,9 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="121079">
+        <ExternalReference id="188714">
           <Source>ICD-10</Source>
-          <Reference>F01.1</Reference>
+          <Reference>I67.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
@@ -75045,7 +75052,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Gerstmann-Straussler-Scheinker syndrome (GSSS) is a particular and rare form of human transmissible spongiform encephalopathy (TSE) due to a defective gene encoding the prion protein (&lt;i&gt;PRNP&lt;/i&gt; gene) and marked by particular multicentric amyloid plaques in the brain.</Contents>
+              <Contents>A rare inherited human prion disease characterized by adult onset of slowly progressive cerebellar ataxia, with dementia developing relatively late in the disease course (classic ataxic phenotype). Patients may present with gait disturbances and frequent falls, dysarthria, dysphagia, nystagmus, dysmetry, and eventually pancerebellar syndrome, myoclonus, spasticity, severe dementia, and mutism. The disease is invariably fatal after five years on average. Neuropathological hallmark is the presence of numerous multicentric prion protein plaques in the cerebral and cerebellar cortex.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -90225,7 +90232,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1018">
@@ -91390,9 +91404,9 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="107737">
+        <ExternalReference id="188713">
           <Source>ICD-10</Source>
-          <Reference>F01.1</Reference>
+          <Reference>I67.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
@@ -102845,19 +102859,18 @@
     <Disorder id="1276">
       <OrphaCode>955</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=955</ExpertLink>
-      <Name lang="en">Acroosteolysis dominant type</Name>
+      <Name lang="en">Hajdu-Cheney syndrome</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="5">
-        <Synonym lang="en">Acrodentoosteodysplasia</Synonym>
+      <SynonymList count="4">
+        <Synonym lang="en">Acroosteolysis dominant type</Synonym>
         <Synonym lang="en">Acroosteolysis with osteoporosis and changes in skull and mandible</Synonym>
         <Synonym lang="en">Arthrodentoosteodysplasia</Synonym>
         <Synonym lang="en">Cheney syndrome</Synonym>
-        <Synonym lang="en">Hajdu-Cheney syndrome</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
@@ -103874,7 +103887,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Distal Xq duplications refer to chromosomal disorders resulting from involvement of the long arm of the X chromosome (Xq). Clinical manifestations vary widely depending on the gender of the patient and on the gene content of the duplicated segment. The prevalence of Xq duplications remains unknown.</Contents>
+              <Contents>A rare X-linked genomic disorder associated with interstitial chromosomal duplications at Xq28 encompassing the &lt;i&gt;MECP2&lt;/i&gt; gene. It is characterized in males by infantile onset hypotonia, severe global developmental delay, intellectual disability, progressive spasticity, seizures, gastrointestinal symptoms and recurrent respiratory infections. In females, the phenotype is more variable.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -112133,7 +112146,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>This syndrome is characterized by the association of total alopecia (present at birth), mild intellectual deficit and hypergonadotropic hypogonadism.</Contents>
+              <Contents>A rare syndromic intellectual disability characterized by the association of total, congenital alopecia, mild intellectual deficit and hypergonadotropic hypogonadism. Reported electroencephalography findings were normal.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -117493,7 +117506,7 @@
     <Disorder id="16731">
       <OrphaCode>137862</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137862</ExpertLink>
-      <Name lang="en">MartÝnez-FrÝas syndrome</Name>
+      <Name lang="en">Martínez-Frías syndrome</Name>
       <DisorderFlagList count="3">
         <DisorderFlag id="495">
           <Value>8192</Value>
@@ -159713,7 +159726,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Dystrophic epidermolysis bullosa, nails only is a rare subtype of dystrophic epidermolysis bullosa (DEB, see this term) that shows no blistering and that is characterized by dystrophic or absent nails.</Contents>
+              <Contents>A form of localized dystrophic epidermolysis bullosa characterized by dystrophic nails in the absence of blistering. The nail deformity is often limited to toenails which can appear thickened and shortened, or may be absent. No other cutaneous or extracutaneous symptoms are observed.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -159762,7 +159775,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A very rare dystrophic epidermolysis bullosa (DEB) characterized by blistering confined primarily to the hands and feet.</Contents>
+              <Contents>A form of localized dystrophic epidermolysis bullosa characterized by trauma-induced blistering confined primarily to the hands and feet. Healing of blisters is associated with milia formation, atrophic scarring and dystrophic nails. There is no extracutaneous involvement.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -160219,7 +160232,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125418" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164138" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare subtype of indolent systemic mastocytosis characterized by isolated bone marrow involvement without skin lesions, low burden of neoplastic mast cells, and often normal or near normal serum tryptase levels. The &lt;i&gt;KIT&lt;/i&gt; D816V mutation is present in the majority of cases.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="17241">
@@ -162370,7 +162393,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Multiple endocrine neoplasia 2B (MEN2B) syndrome is a rare aggressive form of MEN2 (see this term) characterized by medullary thyroid carcinoma (MTC, see this term), pheochromocytoma (see this term), mucosal ganglioneuroma, and marfanoid habitus.</Contents>
+              <Contents>A rare form of multiple endocrine neoplasia type 2 (MEN2) syndrome characterized by aggressive medullary thyroid carcinoma in association with other endocrine tumors, notably pheochromocytoma (one or both adrenal glands can be affected). Onset is typically in infancy or childhood and patients often have a typical facies (mucosal neuromas of the lips and tongue, and bumpy lips), ophthalmologic abnormalities (alacrima in infancy, thickened and everted eyelids, mild ptosis, and prominent corneal nerves), skeletal anomalies (marfanoid body habitus, narrow long facies, pes cavus, pectus excavatum, high-arched palate, scoliosis, hyperextensible joints and slipped capital femoral epiphyses), and a generalized ganglioneuromatosis throughout the aerodigestive tract. Chronic constipation, abdominal distension, diarrhea, or megacolon at birth are often the initial manifestations.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -162454,7 +162477,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Multiple endocrine neoplasia 2A (MEN2A) syndrome is a form of MEN2 (see this term) characterized by medullary thyroid carcinoma (MTC; see this term) in combination with pheochromocytoma (see this term) and primary mild hyperparathyroidism resulting from hyperplasia or adenoma of the parathyroid cells.</Contents>
+              <Contents>A form of multiple endocrine neoplasia type 2 (MEN2) syndrome characterized by medullary thyroid carcinoma in association with pheochromocytoma (one or both adrenal glands can be affected) and/or primary hyperparathyroidism (caused by parathyroid adenoma). Onset is typically later than in MEN2B, before 35 years of age. Diarrhea is the most frequent systemic symptom. Patients can develop Hirschsprung disease and, less frequently, cutaneous lichen amyloidosis or excessive production of adrenocorticotropic hormone.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -183176,7 +183199,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Junctional epidermolysis bullosa, localized non-Herlitz-type is a form of non-Herlitz junctional epidermolysis bullosa (JEB-nH, see this term) characterized by localized blistering, and dystrophic or absent nails.</Contents>
+              <Contents>A form of junctional epidermolysis bullosa characterized by neonatal onset of localized blistering, and dystrophic or absent nails. Skin blistering is mainly confined to hands, feet, lower legs and face. Additional findings may include dental enamel hypoplasia and an increased incidence of caries.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -224728,7 +224751,7 @@
         <DisorderDisorderAssociation>
           <TargetDisorder id="1276">
             <OrphaCode>955</OrphaCode>
-            <Name lang="en">Acroosteolysis dominant type</Name>
+            <Name lang="en">Hajdu-Cheney syndrome</Name>
           </TargetDisorder>
           <RootDisorder id="2587" cycle="true"/>
           <DisorderDisorderAssociationType id="21471">
@@ -242589,7 +242612,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Holoprosencephaly-radial heart renal anomalies syndrome is characterised by holoprosencephaly, predominantly radial limb deficiency (absent thumbs, phocomelia), heart defects, kidney malformations and absence of gallbladder.</Contents>
+              <Contents>A rare multiple congenital anomalies syndrome characterised by holoprosencephaly, predominantly radial limb deficiency (absent thumbs, phocomelia), heart defects, kidney malformations and absence of gallbladder. Variable manifestations include vertebral anomalies, cleft lip/palate, microphthalmia, absent nose, dysplastic ears, hearing loss, colobomas of the iris and retina and/or bifid uvula.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -244330,7 +244353,7 @@
         <Synonym lang="en">Ataxia-telangiectasia, variant 1</Synonym>
         <Synonym lang="en">Berlin breakage syndrome</Synonym>
         <Synonym lang="en">Immunodeficiency-microcephaly-chromosomal instability syndrome</Synonym>
-        <Synonym lang="en">Microcephaly-immunodeficiency-lymphoreticuloma syndrome</Synonym>
+        <Synonym lang="en">Microcephaly-immunodeficiency-lymphoid malignancy syndrome</Synonym>
         <Synonym lang="en">NBS</Synonym>
         <Synonym lang="en">Seemanova syndrome type 2</Synonym>
       </SynonymList>
@@ -280727,7 +280750,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Osteogenesis imperfecta type II is a lethal type of osteogenesis imperfecta (OI; see this term), a genetic disorder characterized by increased bone fragility, low bone mass and susceptibility to bone fractures. Patients with type II present multiple rib and long bone fractures at birth, marked deformities, broad long bones, low density on skull X-rays, and dark sclera.</Contents>
+              <Contents>A lethal type of osteogenesis imperfecta (OI) characterized by increased bone fragility, low bone mass and susceptibility to bone fractures and presenting with multiple rib and long bone fractures at birth, marked deformities, broad long bones, low density skull on X-ray, and dark sclera.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -288126,7 +288149,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Pulmonary arteriovenous malformation (PAVM) describes an anatomic communication between a pulmonary artery and a pulmonary vein leading to a right to left extracardiac shunt that can be asymptomatic or can lead to varying manifestations such as dyspnea, hemoptysis, and neurological symptoms.</Contents>
+              <Contents>An aberrant communication between one or more pulmonary arteries and one or more pulmonary veins leading to an anatomic intrapulmonary right-to-left shunt.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -302050,7 +302073,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Frontonasal dysplasia with alopecia and genital anomaly is a new phenotype of frontonasal dysplasia associated with total alopecia and hypogonadism.</Contents>
+              <Contents>A rare syndromic frontonasal dysplasia characterized by frontonasal dysplasia associated with total alopecia, hypogonadism and mild to moderate intellectual disability. The frontonasal dysplasia includes coronal craniosynostosis, large skull defect with aplasia of ethmoid and nasal bones, hypertelorism, severely depressed nasal bridge and bifid nasal tip.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -311865,7 +311888,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Generalized dominant dystrophic epidermolysis bullosa (DDEB-gen) is a subtype of dystrophic epidermolysis bullosa (DEB, see this term), formerly known as DDEB, Pasini and Cockayne-Touraine types, characterized by generalized blistering, milia formation, atrophic scarring, and dystrophic nails.</Contents>
+              <Contents>A rare dystrophic epidermolysis bullosa (DEB) characterized by generalized blistering, milia formation, atrophic scarring, and dystrophic nails.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -314852,7 +314875,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19272">
@@ -314951,7 +314981,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19287">
@@ -348568,7 +348605,7 @@
           <TargetDisorder id="20883" cycle="true"/>
           <RootDisorder id="16731">
             <OrphaCode>137862</OrphaCode>
-            <Name lang="en">MartÝnez-FrÝas syndrome</Name>
+            <Name lang="en">Martínez-Frías syndrome</Name>
           </RootDisorder>
           <DisorderDisorderAssociationType id="21471">
             <Name lang="en">Moved to</Name>
@@ -348797,7 +348834,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare multiple congenital anomalies syndrome characterized by a spectrum of developmental anomalies including cleft lip and/or palate, craniosynostosis, intellectual disability and/or learning disability, radioulnar synostosis,genital and vesicorenal anomalies. Observed facial dysmorphism includes hypertelorism, blepharophimosis, blepharoptosis, high arched eyebrows. Less common features reported include anterior chamber defects, cardiac anomalies (e.g. ventricular septal defect; see this term), caudal appendage, umbilical hernia/omphalocele and diastasis recti.</Contents>
+              <Contents>A rare multiple congenital anomalies syndrome characterized by a spectrum of developmental anomalies including cleft lip and/or palate, craniosynostosis, intellectual disability and/or learning disability, radioulnar synostosis, genital and vesicorenal anomalies. Observed facial dysmorphism includes hypertelorism, blepharophimosis, blepharoptosis, high arched eyebrows. Less common features reported include anterior chamber defects, cardiac anomalies (e.g. ventricular septal defect; see this term), caudal appendage, umbilical hernia/omphalocele and diastasis recti.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -360441,7 +360478,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20523">
@@ -360608,7 +360652,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20532">
@@ -360969,7 +361020,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20538">
@@ -360994,7 +361052,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20539">
@@ -361019,7 +361084,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20536">
@@ -361044,7 +361116,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20537">
@@ -371740,7 +371819,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125417" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164137" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare acquired skeletal muscle disease characterized by infiltration of the epimysium, perimysium, and perifascicular endomysium by macrophages with crystal inclusions composed of aluminum salts at the site of a previous vaccination (most commonly the deltoid muscle). Muscle necrosis is typically absent. Patients may present with myalgias, arthralgias, muscle weakness, chronic fatigue, asthenia, fever, and cognitive dysfunction. Signs and symptoms usually develop slowly over several months.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="6018">
@@ -373395,7 +373484,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21349">
@@ -381541,12 +381637,15 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="en">
-          <TextSectionList count="0">
+        <SummaryInformation id="125678" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164362" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare bone disease characterized by secondary hyperparathyroidism in patients with chronic renal failure, caused by improper treatment in the early stages of the disease with retention of phosphorus, vitamin D deficiency, and disturbed calcium-phosphorus metabolism, which result in increased parathyroid hormone levels. Patients present with short stature, severe changes of the skull and jaws as well as other skeletal deformities, dental anomalies, ''brown tumors'' in the mouth, hearing loss, and neuropsychiatric disorders.</Contents>
+            </TextSection>
           </TextSectionList>
-          <TextAuto>
-            <Info lang="en">This term designates an indication for a pharmacogenetic test.</Info>
-          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -383076,7 +383175,7 @@
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="en">This is a rare condition for which there is a European and/or American orphan designation.This term designates an indication for a pharmacogenetic test.</Info>
+            <Info lang="en">This is a rare condition for which there is a European and/or American orphan designation.</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -422583,7 +422682,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="en">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="en">This term does not characterize a disease but a group of diseases. To learn about the diseases included under this term, you can consult the classifications.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23179">
@@ -424961,17 +425067,17 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179672">
+        <ExternalReference id="188684">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>G11.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -426046,9 +426152,8 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
-        <Synonym lang="en">Noradrenaline deficiency</Synonym>
-        <Synonym lang="en">Norepinephrine deficiency</Synonym>
+      <SynonymList count="1">
+        <Synonym lang="en">DBH deficiency</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
@@ -426113,7 +426218,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>A rare form of primary autonomic failure characterized by autonomic dysregulation leading mainly to orthostatic hypotension.</Contents>
+              <Contents>A very rare primary monoamine neurotransmitter synthesis disorder with norepinephrine and adrenaline deficiency that leads to young-onset severe orthostatic hypotension and eyelid ptosis.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -430959,7 +431064,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="154602">
+        <ExternalReference id="188685">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -430968,8 +431073,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="158701">
@@ -434637,17 +434742,17 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179670">
+        <ExternalReference id="188682">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>Q07.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -437881,7 +437986,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179671">
+        <ExternalReference id="188683">
           <Source>ICD-10</Source>
           <Reference>G71.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -437890,8 +437995,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="156032">
@@ -446409,7 +446514,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Dystrophic epidermolysis bullosa (DEB) is a form of inherited epidermolysis bullosa (EB) characterized by cutaneous and mucosal fragility resulting in blisters and superficial ulcerations that develop below the lamina densa of the cutaneous basement membrane and that heal with significant scarring and milia formation. It comprises ten sub-types with the three most common being generalized dominant DEB (DDEB), severe generalized recessive DEB (RDEB- sev gen) and RDEB generalized-other (see these terms).</Contents>
+              <Contents>A group of inherited epidermolysis bullosa (EB) characterized by cutaneous and mucosal fragility resulting in blisters and superficial ulcerations that develop below the lamina densa of the cutaneous basement membrane and that heal with significant scarring and milia formation. Dystrophic epidermolysis bullosa (DEB) comprises four major and several rare sub-types with the three most common being intermediate dominant DEB, severe recessive DEB and intermediate recessive DEB.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -446468,7 +446573,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Junctional epidermolysis bullosa (JEB) is a form of inherited epidermolysis bullosa (see this term) characterized by involvement of the skin and mucous membranes, and is defined by the formation of blistering lesions between the epidermis and the dermis at the lamina lucida level of the cutaneous basement membrane zone and by healing of lesions with atrophy and/or exuberant granulation tissue formation.</Contents>
+              <Contents>A group of inherited epidermolysis bullosa characterized by involvement of the skin and mucous membranes, and is defined by the formation of blistering lesions between the epidermis and the dermis at the lamina lucida level of the cutaneous basement membrane zone and by healing of lesions with atrophy and/or exuberant granulation tissue formation.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -446652,7 +446757,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Kindler syndrome (KS) is the fourth major type of epidermolysis bullosa (EB), besides simplex, junctional and dystrophic forms, and is characterized by skin fragility and blistering at birth followed by development of photosensitivity and progressive poikilodermatous skin changes.</Contents>
+              <Contents>A rare inherited epidermolysis bullosa (EB) characterized by skin fragility and blistering at birth followed by development of photosensitivity and progressive poikilodermatous skin changes.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -454622,19 +454727,6 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="3">
-        <ExternalReference id="114885">
-          <Source>ICD-10</Source>
-          <Reference>G93.4</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="en">Validated</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
         <ExternalReference id="16869">
           <Source>OMIM</Source>
           <Reference>606777</Reference>
@@ -454657,6 +454749,19 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
+        <ExternalReference id="188712">
+          <Source>ICD-10</Source>
+          <Reference>G40.4</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
@@ -455187,7 +455292,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179634">
+        <ExternalReference id="188705">
           <Source>ICD-10</Source>
           <Reference>K31.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -455196,8 +455301,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -470852,7 +470957,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="178532">
+        <ExternalReference id="188706">
           <Source>ICD-10</Source>
           <Reference>E75.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -470861,8 +470966,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -471000,17 +471105,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179635">
+        <ExternalReference id="188707">
           <Source>ICD-10</Source>
-          <Reference>K59.9</Reference>
+          <Reference>K59.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -473253,7 +473358,20 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188709">
+          <Source>ICD-10</Source>
+          <Reference>E07.8</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
@@ -473370,7 +473488,7 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178095">
+        <ExternalReference id="188708">
           <Source>ICD-10</Source>
           <Reference>E07.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -473379,8 +473497,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="172336">
@@ -487864,7 +487982,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Hypocomplementemic urticarial vasculitis (HUV) is an immune complex-mediated small vessel vasculitis characterized by urticaria and hypocomplementemia (low C1q with or without low C3 and C4), and usually associated with circulating anti-C1q autoantibodies. Arthritis, pulmonary disease, ocular inflammation, and glomerulonephritis are common systemic manifestations.</Contents>
+              <Contents>A rare immune complex-mediated small vessel vasculitis characterized by urticaria and hypocomplementemia (low C3, C4 and/or C1q), and usually associated with circulating anti-C1q autoantibodies. Arthritis, pulmonary disease, ocular inflammation are common systemic manifestations.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -493663,7 +493781,7 @@
         <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179602">
+        <ExternalReference id="188721">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -493672,8 +493790,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -493965,7 +494083,7 @@
         <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179603">
+        <ExternalReference id="188717">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -493974,8 +494092,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -494169,7 +494287,7 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179604">
+        <ExternalReference id="188687">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -494178,8 +494296,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -494289,7 +494407,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179605">
+        <ExternalReference id="188688">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -494298,8 +494416,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -495805,17 +495923,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179607">
+        <ExternalReference id="188690">
           <Source>ICD-10</Source>
-          <Reference>D23.2</Reference>
+          <Reference>D23.9</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163484">
@@ -495876,7 +495994,7 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179606">
+        <ExternalReference id="188689">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -495885,8 +496003,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -496462,7 +496580,20 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
+      <ExternalReferenceList count="2">
+        <ExternalReference id="188710">
+          <Source>ICD-10</Source>
+          <Reference>D82.3</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21590">
+            <Name lang="en">Inclusion term (The ORPHA code is included under a ICD10 category and has not its own code)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
         <ExternalReference id="163478">
           <Source>OMIM</Source>
           <Reference>300635</Reference>
@@ -496511,7 +496642,20 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188711">
+          <Source>ICD-10</Source>
+          <Reference>D89.8</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -496551,7 +496695,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179612">
+        <ExternalReference id="188691">
           <Source>ICD-10</Source>
           <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -496560,8 +496704,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163477">
@@ -498139,17 +498283,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179613">
+        <ExternalReference id="188692">
           <Source>ICD-10</Source>
-          <Reference>D81.1</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163479">
@@ -498271,17 +498415,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179614">
+        <ExternalReference id="188693">
           <Source>ICD-10</Source>
-          <Reference>D81.1</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163481">
@@ -501189,17 +501333,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179615">
+        <ExternalReference id="188694">
           <Source>ICD-10</Source>
-          <Reference>D81.9</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="172005">
@@ -502295,17 +502439,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179618">
+        <ExternalReference id="188695">
           <Source>ICD-10</Source>
-          <Reference>G71.3</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -502879,7 +503023,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179619">
+        <ExternalReference id="188696">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -502888,8 +503032,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -504186,7 +504330,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179622">
+        <ExternalReference id="188699">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -504195,8 +504339,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171455">
@@ -504473,17 +504617,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179621">
+        <ExternalReference id="188698">
           <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
+          <Reference>E72.4</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -504521,17 +504665,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179620">
+        <ExternalReference id="188697">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>Q07.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171424">
@@ -504941,7 +505085,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179623">
+        <ExternalReference id="188700">
           <Source>ICD-10</Source>
           <Reference>G71.2</Reference>
           <DisorderMappingRelation id="21534">
@@ -504950,8 +505094,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171478">
@@ -505244,17 +505388,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179624">
+        <ExternalReference id="188701">
           <Source>ICD-10</Source>
-          <Reference>G31.8</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171468">
@@ -505954,7 +506098,7 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179627">
+        <ExternalReference id="188702">
           <Source>ICD-10</Source>
           <Reference>Q22.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -505963,8 +506107,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -506619,17 +506763,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179632">
+        <ExternalReference id="188704">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="179040">
@@ -506678,17 +506822,17 @@
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179631">
+        <ExternalReference id="188703">
           <Source>ICD-10</Source>
-          <Reference>G11.1</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="en">Not yet validated</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -511461,7 +511605,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Dystrophic epidermolysis bullosa pruriginosa is a rare subtype of dystrophic epidermolysis bullosa (DEB, see this term) characterized by generalized or localized skin lesions associated with severe, if not intractable, pruritus.</Contents>
+              <Contents>A rare dystrophic epidermolysis bullosa (DEB) characterized by generalized or localized skin lesions associated with severe, if not intractable, pruritus.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -511523,7 +511667,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Recessive dystrophic epidermolysis bullosa (RDEB)-generalized other, also known as RDEB non-Hallopeau-Siemens type, is a subtype of DEB (see this term) characterized by generalized cutaneous and mucosal blistering that is not associated with severe deformities.</Contents>
+              <Contents>A rare dystrophic epidermolysis bullosa (DEB) characterized by generalized cutaneous and mucosal blistering that is not associated with severe deformities.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -522007,7 +522151,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Polycystic kidney disease with tuberous sclerosis (PKD-TSC) is characterised by early-onset and severe polycystic kidney disease with various manifestations of tuberous sclerosis (multiple angiomyolipomas, lymphangioleiomyomatosis and periventricular calcifications of the central nervous system).</Contents>
+              <Contents>A rare contiguous gene syndrome involving a partial deletion of chromosome 16 and characterized by early-onset and severe polycystic kidney disease with various manifestations of tuberous sclerosis (multiple angiomyolipomas, lymphangioleiomyomatosis and periventricular calcifications of the central nervous system).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -533848,7 +533992,7 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="117037">
+        <ExternalReference id="188715">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -538035,7 +538179,20 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188719">
+          <Source>ICD-10</Source>
+          <Reference>D84.1</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -538121,7 +538278,20 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188720">
+          <Source>ICD-10</Source>
+          <Reference>D84.1</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="en">NTBT (ORPHA code's Narrower Term maps to a Broader Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="en">Attributed (The ICD10 code is attributed by Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="en">Validated</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -539915,12 +540085,10 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="7">
-        <Synonym lang="en">GABEB</Synonym>
+      <SynonymList count="5">
         <Synonym lang="en">Generalized atrophic benign epidermolysis bullosa</Synonym>
         <Synonym lang="en">Generalized junctional epidermolysis bullosa, non-Herlitz type</Synonym>
         <Synonym lang="en">Intermediate generalized JEB</Synonym>
-        <Synonym lang="en">JEB-nH gen</Synonym>
         <Synonym lang="en">Junctional epidermolysis bullosa generalisata mitis</Synonym>
         <Synonym lang="en">Junctional epidermolysis bullosa, Disentis type</Synonym>
       </SynonymList>
@@ -540047,7 +540215,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Junctional epidermolysis bullosa inversa is a rare severe subtype of junctional epidermolysis bullosa (JEB, see this term) characterized by blistering and erosions confined to intertriginous skin sites, the esophagus, and vagina.</Contents>
+              <Contents>A rare intermediate form of junctional epidermolysis bullosa characterized by congenital blistering and erosions confined to intertriginous skin sites, the esophagus, groin, and perineum. Blistering is usually severe and lesions may heal with atrophic scarring and milia formation. Extracutaneous manifestations include nail dystrophy, enamel hypoplasia and dental caries, oral, esophageal and vaginal blisters and erosions.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -540225,7 +540393,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Late-onset junctional epidermolysis bullosa is a subtype of junctional epidermolysis bullosa (JEB, see this term) occurring in childhood or young adulthood.</Contents>
+              <Contents>A form of junctional epidermolysis bullosa characterized by onset in childhood or young adulthood of blistering that first occurs around nails, accompanied by nail dystrophy and shedding, and then affects the hands and feet and, to a lesser extent, the elbows, and knees. Lesions heal with atrophic scarring. Other manifestations include disappearance of dermatoglyphs and palmoplantar hyperhidrosis. Extracutaneous involvement is restricted to soft tissue abnormalities of the oral cavity and enamel defects with development of caries.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -540234,15 +540402,14 @@
     <Disorder id="11435">
       <OrphaCode>79409</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79409</ExpertLink>
-      <Name lang="en">Dystrophic epidermolysis bullosa inversa</Name>
+      <Name lang="en">Recessive dystrophic epidermolysis bullosa inversa</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
-        <Synonym lang="en">Autosomal recessive dystrophic epidermolysis bullosa inversa</Synonym>
+      <SynonymList count="2">
         <Synonym lang="en">RDEB inversa</Synonym>
         <Synonym lang="en">RDEB-I</Synonym>
       </SynonymList>
@@ -540287,7 +540454,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Recessive dystrophic epidermolysis bullosa inversa (RDEB-I) is rare subtype of dystrophic epidermolysis bullosa (DEB, see this term) characterized by blisters and erosions which are primarily confined to intertriginous skin sites, the base of the neck, the uppermost back, and the lumbosacral area.</Contents>
+              <Contents>A rare subtype of dystrophic epidermolysis bullosa (DEB) characterized by blisters and erosions which from adolescence or early adulthood are primarily confined to flexural skin sites.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -540352,7 +540519,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Severe generalized recessive dystrophic epidermolysis bullosa (RDEB-sev gen) is the most severe subtype of dystrophic epidermolysis bullosa (DEB, see this term), formerly known as the Hallopeau-Siemens type, and is characterized by generalized cutaneous and mucosal blistering and scarring associated with severe deformities and major extracutaneous involvement.</Contents>
+              <Contents>A severe form of dystrophic epidermolysis bullosa (DEB) characterized by generalized cutaneous and mucosal blistering and scarring associated with severe deformities and major extracutaneous involvement.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -540435,7 +540602,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Transient bullous dermolysis of the newborn is a rare subtype of dystrophic epidermolysis bullosa (DEB, see this term) characterized by generalized blistering at birth that usually regresses within the first 6 to 24 months of life.</Contents>
+              <Contents>A rare dystrophic epidermolysis bullosa (DEB) characterized by generalized blistering at birth that usually regresses within the first 6 to 24 months of life.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -540507,7 +540674,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Pretibial dystrophic epidermolysis bullosa is a rare subtype of dystrophic epidermolysis bullosa (DEB, see this term) characterized by the development of blisters, erosions, and lichenoid lesions predominantly in the pretibial region.</Contents>
+              <Contents>A form of localized dystrophic epidermolysis bullosa characterized by the development of blisters, erosions, and lichenoid lesions predominantly in the anterior lower legs (pretibial areas and feet), the hands and nails. Individual lesions, which tend to be papular or plaque-like, are often violaceous. Pruritus is possible. Healing of blisters is associated with hypertrophic scarring and milia formation. Dystrophy of both fingernails and toenails is characteristic.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -559832,8 +559999,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="6">
+      <SynonymList count="7">
         <Synonym lang="en">Infantile spinal muscular atrophy</Synonym>
+        <Synonym lang="en">Infantile-onset spinal muscular atrophy</Synonym>
         <Synonym lang="en">SMA type 1</Synonym>
         <Synonym lang="en">SMA type I</Synonym>
         <Synonym lang="en">SMA-I</Synonym>
@@ -560168,9 +560336,7 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="7">
-        <Synonym lang="en">Chronic infantile spinal muscular atrophy</Synonym>
-        <Synonym lang="en">Chronic spinal muscular atrophy</Synonym>
+      <SynonymList count="5">
         <Synonym lang="en">Intermediate spinal muscular atrophy</Synonym>
         <Synonym lang="en">SMA type 2</Synonym>
         <Synonym lang="en">SMA type II</Synonym>
@@ -570225,7 +570391,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Insulinoma is the most common type of functioning pancreatic neuroendocrine tumor (see this term) characterized most commonly by a solitary, small pancreatic lesion that causes hyperinsulinemic hypoglycemia.</Contents>
+              <Contents>A form of functioning pancreatic neuroendocrine tumor characterized most commonly by a solitary, small pancreatic lesion that causes hyperinsulinemic hypoglycemia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -572777,7 +572943,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>'Renal tubular dysgenesis due to twin-twin transfusion syndrome (TTTS; see this term) is an acquired form of renal tubular dysgenesis (see this term) that develops in donor fetuses due to the uneven shunting of growth factor and nutrients to the kidney of the recipient and is characterized by absent or poorly developed proximal tubules, persistent oligohydramnios and consequently the Potter sequence (facial dysmorphism with large and flat low-set ears, lung hypoplasia, arthrogryposis and limb positioning defects).'</Contents>
+              <Contents>A rare acquired form of renal tubular dysgenesis that develops in donor fetuses due to the shunting of blood flow to the kidney of the recipient and characterized by absent or poorly developed proximal tubules, persistent oligohydramnios and consequently the Potter sequence (facial dysmorphism with large and flat low-set ears, lung hypoplasia, arthrogryposis and limb positioning defects).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -591187,6 +591353,32 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29899">
+      <OrphaCode>599082</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599082</ExpertLink>
+      <Name lang="en">CHD3-related developmental delay-speech delay-intellectual disability-abnormalities of vision-facial dysmorphism syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="1">
+        <Synonym lang="en">Snijders Blok-Campeau syndrome</Synonym>
+      </SynonymList>
+      <DisorderType id="21401">
+        <Name lang="en">Malformation syndrome</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12296">
       <OrphaCode>93367</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93367</ExpertLink>
@@ -592231,7 +592423,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125430" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164159" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A localized form of dystrophic epidermolysis bullosa characterized by blisters confined primarily to the hands and feet (acral form) or to the pretibial region (pretibial form). Nail dystrophy or loss is common and may be an isolated finding (nail only form).</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12526">
@@ -594650,7 +594852,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125420" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164140" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare soft tissue tumor characterized by distinctive perivascular epitheloid cells, often arranged radially around a vascular lumen, as well as spindled cells in variable proportion. Melanocytic and muscle markers are typically positive. The tumors have been reported in the uterus, falciform ligament, and large and small intestine, among others. Depending on their location, they may present as a painful or painless mass, or with vaginal bleeding. Tumors displaying infiltrative growth, marked hypercellularity, nuclear enlargement and hyperchromasia, high mitotic activity, atypical mitotic figures, and/or coagulative necrosis should be regarded as malignant.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12508">
@@ -595470,7 +595682,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Hemoglobin H (HbH) disease is a moderate to severe form of alpha-thalassemia (see this term) characterized by pronounced microcytic hypochromic hemolytic anemia.</Contents>
+              <Contents>An intermediate form of alpha-thalassemia characterized by increased hemolysis and mild to severe anemia with marked microcytosis and hypochromia. Hemoglobin H disease (HbH) disease belongs to the group of nontransfusion-dependent thalassemia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -603556,7 +603768,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125680" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164364" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare multiple congenital anomalies/dysmorphic syndrome characterized by global developmental delay or regression, variable congenital heart defects (such as patent ductus arteriosus, atrial or ventricular septal defects, and double outlet right ventricle, among others), and dysmorphic features (including ptosis, epicanthal folds, abnormally set/dysplastic ears, low hairline or excess nuchal skin, wide-spaced/inverted nipples, umbilical hernia or diastasis recti, and digital anomalies). Additional variable manifestations are hyper- or hypotonia, seizures, hearing loss, cortical blindness, and optic atrophy. Brain imaging may show cerebral and cerebellar atrophy and hydrocephalus.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="14267">
@@ -603804,7 +604026,7 @@
               <TextSectionType id="16907">
                 <Name lang="en">Definition</Name>
               </TextSectionType>
-              <Contents>Leber `plus' disease describes patients with the clinical features of Leber's hereditary optic neuropathy (LHON; see term) in combination with other serious systemic or neurological abnormalities. These abnormalities include: postural tremor, motor disorder, multiple sclerosis-like syndrome, spinal cord disease, skeletal changes, Parkinsonism with dystonia, anarthria, dystonia, motor and sensory peripheral neuropathy, spasticity and mild encephalopathy. It is caused by maternally-inherited mitochondrial DNA (mtDNA) mutations.</Contents>
+              <Contents>A rare inherited mitochondrial disease characterized by the clinical features of Leber hereditary optic neuropathy in combination with other systemic or neurological abnormalities. These abnormalities include: postural tremor, motor disorder, multiple sclerosis-like syndrome, spinal cord disease, skeletal changes, Parkinsonism with dystonia, anarthria, dystonia, motor and sensory peripheral neuropathy, spasticity, mild encephalopathy, and cardiac arrhythmias.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -603833,7 +604055,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125419" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164139" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare nervous system tumor characterized by cells phenotypically representing Schwann cells but containing melanosomes and expressing melanoma markers. The entity occurs as a non-psammomatous (typically affecting spinal nerves) or a psammomatous (also involving nerves of the intestinal tract and heart) variant. About 50% of psammomatous tumors are associated with Carney complex. Slightly over 10% of melanotic schwannomas follow a malignant course.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="14295">
@@ -604234,7 +604466,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125679" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164363" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic multiple congenital anomalies/dysmorphic syndrome characterized by variable developmental delay and intellectual disability, overweight or obesity, behavioral abnormalities (including hyperactivity, aggressive behavior, anxiety, mood disorder, or autistic features), and facial dysmorphism (such as high forehead, full eyebrows and/or synophrys, upturned nose, and fleshy ears, among others). Additional reported manifestations are hypotonia, ocular anomalies, anomalies of the fingers and toes, joint hypermobility, or abnormal pigmentation. Brain imaging may show mild nonspecific abnormalities.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="29446">
@@ -627824,7 +628066,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125426" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164146" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic multiple congenital anomalies/dysmorphic syndrome characterized by the association of developmental delay and mild chondrodysplasia with short stature and abnormal growth plate morphology. Dysmorphic facial features are variable and may include hypertelorism, upslanting palpebral fissures, broad nose with broad nasal tip, and low-set, cup-shaped ears, among others. Autism spectrum disorder and neurologic abnormalities have also been reported.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="13765">
@@ -627899,7 +628151,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125428" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164148" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare ophthalmic disorder characterized by typically bilateral, asymmetric, yellowish, punctate chorioretinal lesions of the posterior pole forming a linear branching pattern and progressing to atrophic scars. Subretinal neovascular membranes occur in many cases. Vitritis is always absent. Patients may present with blurred vision, scotoma, floaters, photopsia, and metamorphopsia. Choroidal neovascular membrane formation and subretinal fibrosis are the major causes of visual loss. The condition predominantly occurs in young myopic females.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="13767">
@@ -628029,7 +628291,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="125681" lang="en">
+          <TextSectionList count="1">
+            <TextSection id="164365" lang="en">
+              <TextSectionType id="16907">
+                <Name lang="en">Definition</Name>
+              </TextSectionType>
+              <Contents>A rare genetic lethal multiple congenital anomalies/dysmorphic syndrome characterized by early intrauterine growth retardation, generalized edema, craniofacial dysmorphism (such as microcephaly, brachycephaly, frontal bossing, hypertelorism, short palpebral fissures, or absent nasal bone), cerebellar hypoplasia, sex reversal in male fetuses, congenital heart defects (including septal and valve defects and cardiomegaly), and late fetal loss.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="13762">
@@ -647392,7 +647664,7 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="119528">
+        <ExternalReference id="188716">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -650049,7 +650321,7 @@
             <Name lang="en">Validated</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="119519">
+        <ExternalReference id="188718">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
