commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml
index d56920f..1f73153 100644
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:37:02" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:38:27" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10603">
+  <DisorderList count="10617">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -5696,7 +5696,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Convulsiones por la ingesta de alimentos</Synonym>
+        <Synonym lang="es">Crisis por la ingesta de alimentos</Synonym>
         <Synonym lang="es">Epilepsia por la ingesta de alimentos</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -8784,8 +8784,14 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="63704" lang="es">
-          <TextSectionList count="0">
+        <SummaryInformation id="131213" lang="es">
+          <TextSectionList count="1">
+            <TextSection id="171384" lang="es">
+              <TextSectionType id="16907">
+                <Name lang="es">Definición</Name>
+              </TextSectionType>
+              <Contents>Es un trastorno multisistémico de origen genético poco frecuente caracterizado por una amplia gama de manifestaciones musculares [debilidad muscular, miotonía, cataratas de inicio temprano (antes de los 50 años)] y manifestaciones sistémicas (cerebral, endocrina, cardíaca, del tubo digestivo, útero, piel y afección inmunológica) que varían en función de la edad de inicio. El amplio espectro clínico va desde presentaciones letales en el periodo de lactancia hasta una enfermedad leve de aparición tardía.</Contents>
+            </TextSection>
           </TextSectionList>
         </SummaryInformation>
       </SummaryInformationList>
@@ -18131,14 +18137,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="es">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="167">
@@ -18771,7 +18770,7 @@
       </DisorderFlagList>
       <SynonymList count="9">
         <Synonym lang="es">AERRPS</Synonym>
-        <Synonym lang="es">Encefalitis aguda con convulsiones parciales repetitivas refractarias</Synonym>
+        <Synonym lang="es">Encefalitis aguda con crisis parciales repetitivas refractarias</Synonym>
         <Synonym lang="es">Encefalitis no herpética aguda con estado epiléptico refractario grave</Synonym>
         <Synonym lang="es">Encefalopatía epiléptica catastrófica en niños en edad escolar</Synonym>
         <Synonym lang="es">Encefalopatía epiléptica catastrófica idiopática</Synonym>
@@ -34084,7 +34083,7 @@
     <Disorder id="373">
       <OrphaCode>2773</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2773</ExpertLink>
-      <Name lang="es">Síndrome de osteogénesis imperfecta-retinopatía-convulsiones-discapacidad intelectual</Name>
+      <Name lang="es">Síndrome de osteogénesis imperfecta-retinopatía-crisis-discapacidad intelectual</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="460">
           <Value>512</Value>
@@ -36367,7 +36366,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>Es un trastorno neuromuscular hereditario caracterizado por la presencia de numerosos núcleos situados en posición central en la biopsia muscular y por signos clínicos de miopatía congénita.</Contents>
+              <Contents>Es una miopatía congénita autosómica recesiva poco frecuente caracterizada por numerosos núcleos situados en posición central en la biopsia muscular y hallazgos clínicos de una miopatía congénita, incluyendo debilidad facial, anomalías oculares (ptosis y oftalmoplejía externa) y debilidad muscular de gravedad variable predominantemente proximal con una posible afectación distal.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -48784,8 +48783,14 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="61241" lang="es">
-          <TextSectionList count="0">
+        <SummaryInformation id="131215" lang="es">
+          <TextSectionList count="1">
+            <TextSection id="171396" lang="es">
+              <TextSectionType id="16907">
+                <Name lang="es">Definición</Name>
+              </TextSectionType>
+              <Contents>Es una distrofia miotónica poco frecuente de inicio juvenil o en la edad adulta caracterizada por miotonía leve y fluctuante, debilidad muscular y, excepcionalmente, trastornos de la conducción cardíaca.</Contents>
+            </TextSection>
           </TextSectionList>
         </SummaryInformation>
       </SummaryInformationList>
@@ -78621,7 +78626,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>La comunicación interauricular es una malformación congénita caracterizada por una comunicación entre las cámaras auriculares del corazón.</Contents>
+              <Contents>Es una malformación cardíaca congénita caracterizada por una comunicación entre las cavidades auriculares del corazón.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -80494,11 +80499,19 @@
       <OrphaCode>1549</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1549</ExpertLink>
       <Name lang="es">Criptosporidiosis</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="3">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
+        <DisorderFlag id="456">
+          <Value>32</Value>
+          <Label>Non-rare disease in Europe</Label>
+        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
@@ -80508,18 +80521,7 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="5">
-        <ExternalReference id="107110">
-          <Source>MedDRA</Source>
-          <Reference>10011502</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="1">
         <ExternalReference id="107113">
           <Source>ICD-10</Source>
           <Reference>A07.2</Reference>
@@ -80533,53 +80535,10 @@
             <Name lang="es">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="145137">
-          <Source>GARD</Source>
-          <Reference>6219</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="107109">
-          <Source>UMLS</Source>
-          <Reference>C0010418</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="107108">
-          <Source>MeSH</Source>
-          <Reference>D003457</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="121243" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="155626" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>La criptosporidiosis es una enfermedad protozoica causada por coccidios pequeños, &lt;i&gt;e.g. Cryptosporidium spp&lt;/i&gt;, que colonizan las células epiteliales del intestino humano y de una gran variedad de animales.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18412">
@@ -85560,7 +85519,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Convulsiones dependientes de vitamina B6</Synonym>
+        <Synonym lang="es">Crisis dependientes de vitamina B6</Synonym>
         <Synonym lang="es">Deficiencia de antiquitina</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -87599,18 +87558,17 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Defecto del septo aurículoventricular completo</Name>
+      <Name lang="es">Canal aurículoventricular completo</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="4">
+      <SynonymList count="3">
         <Synonym lang="es">CAVC</Synonym>
-        <Synonym lang="es">Canal aurículoventricular completo</Synonym>
         <Synonym lang="es">DSAV completo</Synonym>
-        <Synonym lang="es">Defecto del septo auriculoventricular completo con componentes auricular y ventricular</Synonym>
+        <Synonym lang="es">Defecto del septo aurículoventricular completo</Synonym>
       </SynonymList>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
@@ -87679,8 +87637,14 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="36244" lang="es">
-          <TextSectionList count="0">
+        <SummaryInformation id="131143" lang="es">
+          <TextSectionList count="1">
+            <TextSection id="171342" lang="es">
+              <TextSectionType id="16907">
+                <Name lang="es">Definición</Name>
+              </TextSectionType>
+              <Contents>Es una anomalía cardíaca congénita poco frecuente caracterizada por una conexión auriculoventricular común mediante una válvula AV común, una comunicación interauricular justo por encima de la válvula AV común (comunicación ostium primum) y una comunicación interventricular posterior (CIV de entrada), que provoca un cortocircuito a nivel tanto auricular como ventricular. Morfológicamente, la válvula auriculoventricular común presenta un único anillo con 4 ó 5 velos, incluyendo los acabalgados superior e inferior.</Contents>
+            </TextSection>
           </TextSectionList>
         </SummaryInformation>
       </SummaryInformationList>
@@ -100998,7 +100962,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="es">BFNS</Synonym>
-        <Synonym lang="es">Convulsiones neonatales benignas familiares</Synonym>
+        <Synonym lang="es">Crisis neonatales benignas familiares</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -127503,7 +127467,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>Es una ataxia cerebelosa autosómica recesiva (ACAR) poco frecuente caracterizada por ataxia cerebelosa progresiva asociada a apraxia oculomotora, neuropatía grave e hipoalbuminemia.</Contents>
+              <Contents>Es una ataxia cerebelosa autosómica recesiva poco frecuente caracterizada por ataxia cerebelosa progresiva asociada a apraxia oculomotora, neuropatía grave e hipoalbuminemia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -155063,7 +155027,7 @@
     <Disorder id="1720">
       <OrphaCode>1568</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1568</ExpertLink>
-      <Name lang="es">Síndrome de discapacidad intelectual ligada al cromosoma X-malformación de Dandy-Walker-enfermedad de los ganglios basales-convulsiones</Name>
+      <Name lang="es">Síndrome de discapacidad intelectual ligada al cromosoma X-malformación de Dandy-Walker-enfermedad de los ganglios basales-crisis</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -179624,7 +179588,7 @@
         <Synonym lang="es">Síndrome HRD</Synonym>
         <Synonym lang="es">Síndrome de Richardson-Kirk</Synonym>
         <Synonym lang="es">Síndrome de hipoparatiroidismo-discapacidad intelectual-síndrome dismórfico</Synonym>
-        <Synonym lang="es">Síndrome de hipoparatiroidismo-talla baja-discapacidad intelectual-convulsiones</Synonym>
+        <Synonym lang="es">Síndrome de hipoparatiroidismo-talla baja-discapacidad intelectual-crisis</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
@@ -222609,7 +222573,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>El síndrome MOMO es un síndrome genético muy raro de sobrecrecimiento/obesidad (ver este término) caracterizado por macrocefalia, obesidad, retraso mental (intelectual) y anomalías oculares. Otros signos clínicos frecuentes incluyen macrosomia, fisuras palpebrales oblicuas hacía abajo, hipertelorismo, raíz nasal ancha, frente amplia y retraso de la maduración ósea, en asociación con función tiroidea y cariotipo normales.</Contents>
+              <Contents>El síndrome MOMO es un síndrome genético muy poco frecuente de sobrecrecimiento/obesidad caracterizado por macrocefalia, obesidad, discapacidad intelectual y anomalías oculares. Otros signos clínicos frecuentes incluyen macrosomia, fisuras palpebrales descendentes, hipertelorismo, raíz nasal ancha, frente amplia y retraso de la maduración ósea, en asociación con función tiroidea y cariotipo normales.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -229828,7 +229792,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>El síndrome de discapacidad intelectual ligada al X - hipogonadismo - ictiosis - obesidad - talla baja es un síndrome de discapacidad intelectual ligado al X, poco frecuente, caracterizado por una discapacidad intelectual asociada a talla baja, obesidad, hipogonadismo primario y una afección cutánea ictiosiforme. No ha habido más descripciones en la literatura desde 1982.</Contents>
+              <Contents>El síndrome de discapacidad intelectual ligada al cromosoma X - hipogonadismo - ictiosis - obesidad - talla baja, es un síndrome poco frecuente de discapacidad intelectual ligado al cromosoma X caracterizado por discapacidad intelectual asociada a talla baja, obesidad, hipogonadismo primario y una afección cutánea ictiosiforme. No ha habido más descripciones en la literatura desde 1982.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -332646,7 +332610,7 @@
     <Disorder id="3662">
       <OrphaCode>1544</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1544</ExpertLink>
-      <Name lang="es">Convulsiones focales benignas del adolescente</Name>
+      <Name lang="es">Epilepsia focal benigna de la adolescencia</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -332686,7 +332650,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>Es una epilepsia poco frecuente caracterizada típicamente por crisis focales motoras y somatosensitivas aisladas. Con menor frecuencia se han descrito otros tipos de crisis focales, con o sin generalización secundaria. Las crisis suelen ocurrir cuando el paciente está despierto y siguen un curso benigno. El trastorno es transitorio, la exploración interictal es normal y, por lo general, no hay antecedentes familiares de epilepsia.</Contents>
+              <Contents>Es una epilepsia poco frecuente caracterizada típicamente por crisis focales motoras y somatosensitivas aisladas. Con menor frecuencia se ha descrito otros tipos de crisis focales, con o sin generalización secundaria. Las crisis suelen ocurrir cuando el paciente está despierto y siguen un curso benigno. El trastorno es transitorio, la exploración interictal es normal y, por lo general, no hay antecedentes familiares de epilepsia.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -333054,10 +333018,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="4">
+      <SynonymList count="3">
         <Synonym lang="es">BFIE</Synonym>
         <Synonym lang="es">BFIS</Synonym>
-        <Synonym lang="es">Convulsiones benignas del lactante familiares</Synonym>
         <Synonym lang="es">Crisis benignas del lactante familiares</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -348579,8 +348542,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="2">
         <Synonym lang="es">Síndrome asociado a PACS1</Synonym>
+        <Synonym lang="es">Síndrome de Schuurs-Hoeijmakers</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
@@ -350906,7 +350870,7 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="2">
+      <DisorderDisorderAssociationList count="3">
         <DisorderDisorderAssociation>
           <TargetDisorder id="21817" cycle="true"/>
           <RootDisorder id="3393">
@@ -350917,6 +350881,16 @@
             <Name lang="es">Referido a</Name>
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="21817" cycle="true"/>
+          <RootDisorder id="10773">
+            <OrphaCode>57194</OrphaCode>
+            <Name lang="es">OBSOLETO: Osteitis aséptica</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="es">Referido a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
         <DisorderDisorderAssociation>
           <TargetDisorder id="21817" cycle="true"/>
           <RootDisorder id="12460">
@@ -359915,14 +359889,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="es">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="es">Está enfermedad se describe en  Síndromes hipereosinofílicos</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21543">
@@ -359965,14 +359932,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="es">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="es">Está enfermedad se describe en  Síndromes hipereosinofílicos</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21542">
@@ -361026,7 +360986,7 @@
     <Disorder id="21510">
       <OrphaCode>314655</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=314655</ExpertLink>
-      <Name lang="es">Síndrome de hipotonía-convulsiones-encefalopatía neonatal grave por microdeleción 5q31.3</Name>
+      <Name lang="es">Síndrome de hipotonía-crisis-encefalopatía neonatal grave por microdeleción 5q31.3</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -364890,7 +364850,7 @@
     <Disorder id="22449">
       <OrphaCode>369929</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=369929</ExpertLink>
-      <Name lang="es">Síndrome de hiperaldosteronismo primario-convulsiones-anomalías neurológicas</Name>
+      <Name lang="es">Síndrome de hiperaldosteronismo primario-crisis-anomalías neurológicas</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -365222,7 +365182,7 @@
     <Disorder id="22452">
       <OrphaCode>369950</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=369950</ExpertLink>
-      <Name lang="es">Síndrome de discapacidad intelectual-convulsiones-macrocefalia-obesidad</Name>
+      <Name lang="es">Síndrome de discapacidad intelectual-crisis-macrocefalia-obesidad</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -365724,7 +365684,7 @@
     <Disorder id="22433">
       <OrphaCode>369837</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=369837</ExpertLink>
-      <Name lang="es">Síndrome de discapacidad intelectual-convulsiones-hipofosfatasia-anomalías oftalmológicas y esqueléticas</Name>
+      <Name lang="es">Síndrome de discapacidad intelectual-crisis-hipofosfatasia-anomalías oftalmológicas y esqueléticas</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -365734,7 +365694,7 @@
       <SynonymList count="4">
         <Synonym lang="es">MCAHS tipo 3</Synonym>
         <Synonym lang="es">PIGT-CDG</Synonym>
-        <Synonym lang="es">Síndrome de anomalías congénitas múltiples-hipotonía-convulsiones tipo 3</Synonym>
+        <Synonym lang="es">Síndrome de anomalías congénitas múltiples-hipotonía-crisis tipo 3</Synonym>
         <Synonym lang="es">Trastorno congénito de la glicosilación por deficiencia de PIGT</Synonym>
       </SynonymList>
       <DisorderType id="21401">
@@ -386430,7 +386390,7 @@
     <Disorder id="20841">
       <OrphaCode>293181</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293181</ExpertLink>
-      <Name lang="es">Convulsiones focales migratorias malignas del lactante</Name>
+      <Name lang="es">Epilepsia focal migratoria maligna del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -386438,9 +386398,9 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="9">
-        <Synonym lang="es">Convulsiones parciales migratorias del lactante</Synonym>
-        <Synonym lang="es">Convulsiones parciales migratorias malignas del lactante</Synonym>
-        <Synonym lang="es">Epilepsia del lactante con convulsiones focales migratorias</Synonym>
+        <Synonym lang="es">Crisis parciales migratorias del lactante</Synonym>
+        <Synonym lang="es">Crisis parciales migratorias malignas del lactante</Synonym>
+        <Synonym lang="es">Epilepsia del lactante con crisis focales migratorias</Synonym>
         <Synonym lang="es">Epilepsia parcial migratoria del lactante</Synonym>
         <Synonym lang="es">Epilepsia parcial migratoria maligna del lactante</Synonym>
         <Synonym lang="es">MMPEI</Synonym>
@@ -388458,19 +388418,11 @@
       <OrphaCode>80</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=80</ExpertLink>
       <Name lang="es">NO ES ENFERMEDAD RARA EN EUROPA: Síndrome antifosfolípido</Name>
-      <DisorderFlagList count="3">
-        <DisorderFlag id="495">
-          <Value>8192</Value>
-          <Label>Inactive</Label>
-        </DisorderFlag>
+      <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
-        <DisorderFlag id="456">
-          <Value>32</Value>
-          <Label>Non-rare disease in Europe</Label>
-        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
         <Synonym lang="es">Anticoagulante lúpico familiar</Synonym>
@@ -388487,14 +388439,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="es">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="es">Esta enfermedad no es rara en Europa. No pertenece a la nomenclatura de enfermedades raras de Orphanet.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20670">
@@ -389105,7 +389050,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Está enfermedad se describe en  Enfermedad de almacenamiento de glucógeno por deficiencia de lactato deshidrogenasa</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20633">
@@ -407337,7 +407289,7 @@
     <Disorder id="21120">
       <OrphaCode>300496</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300496</ExpertLink>
-      <Name lang="es">Síndrome de anomalías congénitas múltiples-hipotonía-convulsiones tipo 2</Name>
+      <Name lang="es">Síndrome de anomalías congénitas múltiples-hipotonía-crisis tipo 2</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -415636,9 +415588,9 @@
       </DisorderFlagList>
       <SynonymList count="4">
         <Synonym lang="es">Síndrome SPPRS</Synonym>
-        <Synonym lang="es">Síndrome de paraparesia espástica-retraso psicomotor-convulsiones</Synonym>
+        <Synonym lang="es">Síndrome de paraparesia espástica-retraso psicomotor-crisis</Synonym>
         <Synonym lang="es">Síndrome de paraplejía espástica-retraso grave del desarrollo-epilepsia</Synonym>
-        <Synonym lang="es">Síndrome de paraplejía espástica-retraso psicomotor-convulsiones</Synonym>
+        <Synonym lang="es">Síndrome de paraplejía espástica-retraso psicomotor-crisis</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -424786,7 +424738,7 @@
     <Disorder id="24178">
       <OrphaCode>466926</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=466926</ExpertLink>
-      <Name lang="es">Síndrome de convulsiones-escoliosis-macrocefalia</Name>
+      <Name lang="es">Síndrome de crisis-escoliosis-macrocefalia</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -430760,7 +430712,7 @@
     <Disorder id="22918">
       <OrphaCode>404437</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=404437</ExpertLink>
-      <Name lang="es">Síndrome de atrofia cerebral y cerebelosa difusa-convulsiones intratables-microcefalia progresiva</Name>
+      <Name lang="es">Síndrome de atrofia cerebral y cerebelosa difusa-crisis intratables-microcefalia progresiva</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -442952,7 +442904,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Síndrome de rigidez neonatal letal-convulsiones multifocales</Synonym>
+        <Synonym lang="es">Síndrome de rigidez neonatal letal-crisis multifocales</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
@@ -444950,7 +444902,7 @@
     <Disorder id="23370">
       <OrphaCode>438216</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=438216</ExpertLink>
-      <Name lang="es">Síndrome neonatal de hipotonía-convulsiones-encefalopatía grave asociado al gen PURA por una mutación puntual</Name>
+      <Name lang="es">Síndrome neonatal de hipotonía-crisis-encefalopatía grave asociado al gen PURA por una mutación puntual</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -445006,7 +444958,7 @@
     <Disorder id="23369">
       <OrphaCode>438213</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=438213</ExpertLink>
-      <Name lang="es">Síndrome neonatal de hipotonía-convulsiones-encefalopatía grave asociado al gen PURA</Name>
+      <Name lang="es">Síndrome neonatal de hipotonía-crisis-encefalopatía grave asociado al gen PURA</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -465124,7 +465076,7 @@
     <Disorder id="26327">
       <OrphaCode>505237</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=505237</ExpertLink>
-      <Name lang="es">Síndrome de inicio temprano de convulsiones-anomalías de las extremidades distales-dismorfia facial-retraso generalizado del desarrollo</Name>
+      <Name lang="es">Síndrome de crisis de inicio precoz-anomalías de las extremidades distales-dismorfia facial-retraso generalizado del desarrollo</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -481153,7 +481105,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>Es una malformación cardíaca congénita caracterizada por comunicación interauricular (CIA; ostium primum), hendiduras de la válvula mitral y, en ocasiones, de la tricúspide, dos anillos auriculoventriculares (AV) separados y un septo ventricular íntegro. Los síntomas típicos del canal auriculoventricular parcial (CAVP) son una capacidad funcional reducida y disnea de esfuerzo.</Contents>
+              <Contents>Es una malformación cardíaca congénita poco frecuente, consistente en una variante de los defectos del septo aurículoventricular (DSAV), consistente en una comunicación interauricular tipo ostium primum (justo por encima de la válvula auriculoventricular -AV- común), ausencia de una comunicación interventricular por debajo de la válvula auriculoventricular, presencia de una unión auriculoventricular común aunque con orificios valvulares derecho e izquierdo separados, y un componente trivalva en el lado izquierdo de la válvula auriculoventricular común (''hendidura''). El cortocircuito se limita al nivel auricular debido a la fusión de las valvas de la válvula AV común con la cresta del tabique ventricular.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -486376,7 +486328,7 @@
     <Disorder id="9793">
       <OrphaCode>31709</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=31709</ExpertLink>
-      <Name lang="es">Convulsiones del lactante y coreoatetosis</Name>
+      <Name lang="es">Crisis del lactante con coreoatetosis</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -486384,7 +486336,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Discinesia paroxística cinesigénica y convulsiones del lactante</Synonym>
+        <Synonym lang="es">Discinesia paroxística cinesigénica y crisis del lactante</Synonym>
         <Synonym lang="es">Síndrome ICCA</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -487986,7 +487938,7 @@
     <Disorder id="25159">
       <OrphaCode>477814</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477814</ExpertLink>
-      <Name lang="es">Síndrome de microcefalia progresiva-convulsiones-ceguera cortical-retraso del desarrollo</Name>
+      <Name lang="es">Síndrome de microcefalia progresiva-crisis-ceguera cortical-retraso del desarrollo</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -488549,7 +488501,7 @@
     <Disorder id="25168">
       <OrphaCode>478049</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=478049</ExpertLink>
-      <Name lang="es">Síndrome letal de no compactación del ventrículo izquierdo-convulsiones-hipotonía-cataratas-retraso del desarrollo</Name>
+      <Name lang="es">Síndrome letal de no compactación del ventrículo izquierdo-crisis-hipotonía-cataratas-retraso del desarrollo</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -501634,7 +501586,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>Es una ataxia cerebelosa autosómica recesiva (ACAR) poco frecuente, caracterizada por ataxia cerebelosa progresiva asociada a apraxia oculomotora frecuente, neuropatía grave y niveles elevados de alfa-fetoproteína sérica (AFP).</Contents>
+              <Contents>Es una ataxia cerebelosa autosómica recesiva (ACAR) poco frecuente caracterizada por ataxia cerebelosa progresiva asociada a apraxia oculomotora frecuente, neuropatía grave y niveles elevados de alfa-fetoproteína sérica (AFP).</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -502279,7 +502231,7 @@
     <Disorder id="10834">
       <OrphaCode>64545</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=64545</ExpertLink>
-      <Name lang="es">Convulsiones neonatales idiopáticas benignas</Name>
+      <Name lang="es">Epilepsia neonatal idiopática benigna</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -506063,14 +506015,24 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="21817">
+            <OrphaCode>324964</OrphaCode>
+            <Name lang="es">Osteomielitis crónica no bacteriana/Osteomielitis crónica multifocal recurrente</Name>
+          </TargetDisorder>
+          <RootDisorder id="10773" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="es">Referido a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="-1" lang="es">
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="es">Esta entidad ha quedado obsoleta en la nomenclatura de enfermedades raras de Orphanet.</Info>
+            <Info lang="es">Esta entidad ha quedado obsoleta en la nomenclatura de enfermedades raras de Orphanet.&lt;br/&gt;&lt;br/&gt;En su lugar, considere usar  Osteomielitis crónica no bacteriana/Osteomielitis crónica multifocal recurrente</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -511709,7 +511671,7 @@
     <Disorder id="11182">
       <OrphaCode>79156</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79156</ExpertLink>
-      <Name lang="es">Síndrome de convulsiones-discapacidad intelectual por hidroxilsinuria</Name>
+      <Name lang="es">Síndrome de crisis-discapacidad intelectual por hidroxilsinuria</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="460">
           <Value>512</Value>
@@ -514766,7 +514728,7 @@
     <Disorder id="11139">
       <OrphaCode>79097</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79097</ExpertLink>
-      <Name lang="es">Convulsiones sensibles al ácido folínico</Name>
+      <Name lang="es">Epilepsia sensible al ácido folínico</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -514814,7 +514776,7 @@
     <Disorder id="11138">
       <OrphaCode>79096</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79096</ExpertLink>
-      <Name lang="es">Convulsiones sensibles al piridoxal fosfato</Name>
+      <Name lang="es">Epilepsia sensible al piridoxal fosfato</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -517327,7 +517289,7 @@
         <DisorderDisorderAssociation>
           <TargetDisorder id="21120">
             <OrphaCode>300496</OrphaCode>
-            <Name lang="es">Síndrome de anomalías congénitas múltiples-hipotonía-convulsiones tipo 2</Name>
+            <Name lang="es">Síndrome de anomalías congénitas múltiples-hipotonía-crisis tipo 2</Name>
           </TargetDisorder>
           <RootDisorder id="11123" cycle="true"/>
           <DisorderDisorderAssociationType id="21471">
@@ -521587,89 +521549,11 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="6">
-        <ExternalReference id="114930">
-          <Source>ICD-10</Source>
-          <Reference>K20</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="14890">
-          <Source>OMIM</Source>
-          <Reference>610247</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="45508">
-          <Source>OMIM</Source>
-          <Reference>613412</Reference>
-          <DisorderMappingRelation id="21541">
-            <Name lang="es">TGTE (el Término más Genérico del código ORPHA se asigna a un Término más Específico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114926">
-          <Source>MeSH</Source>
-          <Reference>D057765</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114927">
-          <Source>UMLS</Source>
-          <Reference>C0341106</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114928">
-          <Source>MedDRA</Source>
-          <Reference>10064212</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="106025" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="132901" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>Es una enfermedad localizada del esófago, inmunomediada, crónica y poco frecuente, caracterizada clínicamente por síntomas de disfunción esofágica (que incluyen disfagia, trastornos de la alimentación, impactación alimentaria, vómitos y dolor abdominal) e histológicamente por inflamación con predominio de eosinófilos en biopsias esofágicas.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28655">
@@ -524147,8 +524031,14 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="55005" lang="es">
-          <TextSectionList count="0">
+        <SummaryInformation id="131212" lang="es">
+          <TextSectionList count="1">
+            <TextSection id="171383" lang="es">
+              <TextSectionType id="16907">
+                <Name lang="es">Definición</Name>
+              </TextSectionType>
+              <Contents>Es una enfermedad neurológica poco frecuente caracterizada por cataratas bilaterales, malformación de Dandy-Walker y atrofia muscular espinal distal de inicio en la infancia. Los pacientes presentan debilidad muscular distal simétrica de deterioro progresivo y atrofia de las extremidades inferiores (y, en mucho menor grado, de las superiores), así como disminución de los reflejos tendinosos en las extremidades superiores e inferiores.</Contents>
+            </TextSection>
           </TextSectionList>
         </SummaryInformation>
       </SummaryInformationList>
@@ -528825,7 +528715,7 @@
     <Disorder id="10424">
       <OrphaCode>36387</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=36387</ExpertLink>
-      <Name lang="es">Epilepsia generalizada con convulsiones febriles plus</Name>
+      <Name lang="es">Epilepsia generalizada con crisis febriles plus</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -536145,9 +536035,8 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="2">
+      <SynonymList count="1">
         <Synonym lang="es">Deficiencia SLC6A8</Synonym>
-        <Synonym lang="es">Deficiencia de transportador de creatina</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -567935,7 +567824,7 @@
     <Disorder id="12210">
       <OrphaCode>93267</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93267</ExpertLink>
-      <Name lang="es">Síndrome de cráneo en hoja de trébol-anomalías congénitas múltiples</Name>
+      <Name lang="es">Síndrome de cráneo en trébol-anomalías congénitas múltiples</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -568519,7 +568408,7 @@
       </DisorderFlagList>
       <SynonymList count="4">
         <Synonym lang="es">Enanismo tanatofórico tipo 2</Synonym>
-        <Synonym lang="es">Síndrome de cráneo en hoja de trébol-displasia ósea micromélica</Synonym>
+        <Synonym lang="es">Síndrome de cráneo en trébol-displasia ósea micromélica</Synonym>
         <Synonym lang="es">Síndrome de enanismo tanatofórico-cráneo en hoja de trébol</Synonym>
         <Synonym lang="es">TD2</Synonym>
       </SynonymList>
@@ -577943,7 +577832,7 @@
     <Disorder id="27609">
       <OrphaCode>529665</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=529665</ExpertLink>
-      <Name lang="es">Síndrome de retraso del neurodesarrollo-convulsiones-anomalías oftálmicas-osteopenia-atrofia cerebelosa</Name>
+      <Name lang="es">Síndrome de retraso del neurodesarrollo-crisis-anomalías oftálmicas-osteopenia-atrofia cerebelosa</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -605702,7 +605591,7 @@
         <Synonym lang="es">Síndrome de sordera-onico-osteodistrofia-discapacidad intelectual</Synonym>
         <Synonym lang="es">Síndrome de sordera-onicodistrofia autosómica recesiva</Synonym>
         <Synonym lang="es">Síndrome de sordera-onicodistrofia-osteodistrofia-discapacidad intelectual</Synonym>
-        <Synonym lang="es">Síndrome de sordera-onicodistrofia-osteodistrofia-discapacidad intelectual-convulsiones</Synonym>
+        <Synonym lang="es">Síndrome de sordera-onicodistrofia-osteodistrofia-discapacidad intelectual-crisis</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
@@ -610230,7 +610119,7 @@
     <Disorder id="27025">
       <OrphaCode>513456</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=513456</ExpertLink>
-      <Name lang="es">Síndrome de discapacidad intelectual-convulsiones-trastornos de la marcha-dismorfia facial</Name>
+      <Name lang="es">Síndrome de discapacidad intelectual-crisis-trastornos de la marcha-dismorfia facial</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -637329,7 +637218,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Está enfermedad se describe en  Síndrome FOXG1</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12402">
@@ -650809,7 +650705,7 @@
         <DisorderDisorderAssociation>
           <TargetDisorder id="871">
             <OrphaCode>1329</OrphaCode>
-            <Name lang="es">Defecto del septo aurículoventricular completo</Name>
+            <Name lang="es">Canal aurículoventricular completo</Name>
           </TargetDisorder>
           <RootDisorder id="14083" cycle="true"/>
           <DisorderDisorderAssociationType id="27341">
@@ -680414,7 +680310,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="131211" lang="es">
+          <TextSectionList count="1">
+            <TextSection id="171382" lang="es">
+              <TextSectionType id="16907">
+                <Name lang="es">Definición</Name>
+              </TextSectionType>
+              <Contents>Es un trastorno genético poco frecuente del movimiento caracterizado por distonía que afecta inicialmente a una extremidad superior y, con menor frecuencia, a la región de la cabeza y del cuello, antes de extenderse lentamente a otras localizaciones. Tanto el espectro clínico como la edad de aparición son variables, con una distribución focal, segmentaria o generalizada, aunque la afectación craneal con dificultades del habla y la afectación cervical son habituales, mientras que las extremidades inferiores suelen estar preservadas. Con la progresión de la enfermedad, muchos pacientes sufren de distonía generalizada mientras permanecen en su mayoría ambulatorios.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="13822">
@@ -700971,7 +700877,7 @@
     <Disorder id="14689">
       <OrphaCode>100978</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100978</ExpertLink>
-      <Name lang="es">Síndrome de cráneo en hoja de trébol-displasia torácica asfixiante</Name>
+      <Name lang="es">Síndrome de cráneo en trébol-displasia torácica asfixiante</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="460">
           <Value>512</Value>
@@ -701026,7 +700932,7 @@
               <TextSectionType id="16907">
                 <Name lang="es">Definición</Name>
               </TextSectionType>
-              <Contents>Es una craneosinostosis sindrómica poco frecuente caracterizada por la presentación prenatal de cráneo en forma de hoja de trébol, micromelia y displasia torácica asfixiante. Los hallazgos radiológicos incluyen costillas cortas, techo horizontal del acetábulo con prominencia mediana redondeada y espolones laterales, huesos largos deformados con metáfisis amplias y osificación ausente de las falanges terminales. No se han descrito más casos en la literatura desde 1987.</Contents>
+              <Contents>Es una craneosinostosis sindrómica poco frecuente caracterizada por la presentación prenatal de cráneo en trébol, micromelia y displasia torácica asfixiante. Los hallazgos radiológicos incluyen costillas cortas, techo horizontal del acetábulo con prominencia mediana redondeada y espolones laterales, huesos largos deformados con metáfisis amplias y osificación ausente de las falanges terminales. No se han descrito más casos en la literatura desde 1987.</Contents>
             </TextSection>
           </TextSectionList>
         </SummaryInformation>
@@ -718261,6 +718167,206 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31348">
+      <OrphaCode>617449</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617449</ExpertLink>
+      <Name lang="es">Congenital aphakia-iris hypoplasia-microphthalmia-microcornea syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31346">
+      <OrphaCode>617408</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617408</ExpertLink>
+      <Name lang="es">Classic eosinophilic pustular folliculitis</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31347">
+      <OrphaCode>617440</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617440</ExpertLink>
+      <Name lang="es">Painful legs and moving toes syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21422">
+        <Name lang="es">Síndrome clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31341">
+      <OrphaCode>617307</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617307</ExpertLink>
+      <Name lang="es">Rare disorder related to monochorionic twin pregnancy</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="es">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31340">
+      <OrphaCode>617304</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617304</ExpertLink>
+      <Name lang="es">Amniotic Fluid Embolism</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31343">
+      <OrphaCode>617313</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617313</ExpertLink>
+      <Name lang="es">Rare disorder related to Inadequate sharing of the placenta</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="es">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31342">
+      <OrphaCode>617310</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617310</ExpertLink>
+      <Name lang="es">Rare disorder related to Unbalanced inter-twin blood transfusion</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="es">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31337">
+      <OrphaCode>617294</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617294</ExpertLink>
+      <Name lang="es">Twin Anemia Polycythemia Sequence</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="31336">
       <OrphaCode>616874</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=616874</ExpertLink>
@@ -718273,11 +718379,11 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21464">
-        <Name lang="es">Ningún tipo</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="38983">
-        <Name lang="es">No type</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
@@ -718296,5 +718402,155 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31339">
+      <OrphaCode>617301</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617301</ExpertLink>
+      <Name lang="es">Selective Intrauterine Growth Restriction</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31338">
+      <OrphaCode>617297</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617297</ExpertLink>
+      <Name lang="es">Twin Reversed Arterial Perfusion sequence</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31370">
+      <OrphaCode>617910</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617910</ExpertLink>
+      <Name lang="es">Conjunctival malignant melanoma</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31371">
+      <OrphaCode>617916</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617916</ExpertLink>
+      <Name lang="es">Diffuse idiopathic pulmonary neuroendocrine cell hyperplasia</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31372">
+      <OrphaCode>617919</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617919</ExpertLink>
+      <Name lang="es">F12-associated cold auto-inflammatory syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31373">
+      <OrphaCode>617930</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=617930</ExpertLink>
+      <Name lang="es">Hemophilia B Leyden</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
