commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml
old mode 100644
new mode 100755
index f5b4a71..cbd2eb3
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/es_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:28:33" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:29:09" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10512">
+  <DisorderList count="10521">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -974,7 +974,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
-        <Synonym lang="es">Encefalopatía infantil letal con hipoplasia olivopontocerebelosa</Synonym>
+        <Synonym lang="es">Encefalopatía letal del lactante con hipoplasia olivopontocerebelosa</Synonym>
         <Synonym lang="es">Hipoplasia olivopontocerebelosa</Synonym>
         <Synonym lang="es">PCH4</Synonym>
       </SynonymList>
@@ -1158,7 +1158,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Encefalopatía infantil letal con defectos en la cadena respiratoria mitocondrial</Synonym>
+        <Synonym lang="es">Encefalopatía letal del lactante con defectos en la cadena respiratoria mitocondrial</Synonym>
         <Synonym lang="es">PCH6</Synonym>
       </SynonymList>
       <DisorderType id="21401">
@@ -2566,7 +2566,7 @@
     <Disorder id="17619">
       <OrphaCode>166105</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166105</ExpertLink>
-      <Name lang="es">Encefalomiopatía mitocondrial infantil asociada al gen FASTKD2</Name>
+      <Name lang="es">Encefalomiopatía mitocondrial del lactante asociada al gen FASTKD2</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -4386,7 +4386,7 @@
     <Disorder id="17634">
       <OrphaCode>166305</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166305</ExpertLink>
-      <Name lang="es">Crisis infantiles benignas asociadas con gastroenteritis leve</Name>
+      <Name lang="es">Crisis benignas infantiles asociadas con gastroenteritis leve</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -4898,7 +4898,7 @@
     <Disorder id="17636">
       <OrphaCode>166311</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166311</ExpertLink>
-      <Name lang="es">Crisis infantiles parciales benignas</Name>
+      <Name lang="es">Crisis parciales benignas infantiles</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -30030,7 +30030,7 @@
     <Disorder id="17792">
       <OrphaCode>168782</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=168782</ExpertLink>
-      <Name lang="es">Trastorno desintegrativo de la infancia</Name>
+      <Name lang="es">Trastorno desintegrativo infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -31400,7 +31400,7 @@
           <TargetDisorder id="369" cycle="true"/>
           <RootDisorder id="20735">
             <OrphaCode>289527</OrphaCode>
-            <Name lang="es">OBSOLETO: Miocardiopatía hipertrófica infantil letal por deficiencia del complejo I mitocondrial</Name>
+            <Name lang="es">OBSOLETO: Miocardiopatía hipertrófica letal infantil por deficiencia del complejo I mitocondrial</Name>
           </RootDisorder>
           <DisorderDisorderAssociationType id="27341">
             <Name lang="es">Referido a</Name>
@@ -37917,7 +37917,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Hiperostosis cortical infantil</Synonym>
+        <Synonym lang="es">Hiperostosis cortical del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
@@ -39018,8 +39018,8 @@
       <SynonymList count="4">
         <Synonym lang="es">Enfermedad de Jansky-Bielschowsky</Synonym>
         <Synonym lang="es">LINCL</Synonym>
-        <Synonym lang="es">LNC tardía infantil</Synonym>
-        <Synonym lang="es">NCL tardía infantil</Synonym>
+        <Synonym lang="es">LNC infantil tardía</Synonym>
+        <Synonym lang="es">NCL infantil tardía</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -47610,7 +47610,8 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="1">
+        <Synonym lang="es">Encefalopatia necrotizante subaguda infantil</Synonym>
       </SynonymList>
       <DisorderType id="21436">
         <Name lang="es">Grupo de fenomas</Name>
@@ -53544,7 +53545,7 @@
     <Disorder id="589">
       <OrphaCode>3173</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3173</ExpertLink>
-      <Name lang="es">Síndrome de espasmos infantiles-pulgar ancho</Name>
+      <Name lang="es">Síndrome de espasmos-pulgar ancho del lactante</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="460">
           <Value>512</Value>
@@ -54610,7 +54611,7 @@
       <SynonymList count="3">
         <Synonym lang="es">Botulismo con toxemia intestinal infantil</Synonym>
         <Synonym lang="es">Botulismo intestinal infantil</Synonym>
-        <Synonym lang="es">Botulismo intestinal infantil por toxinas</Synonym>
+        <Synonym lang="es">Botulismo intestinal por toxinas infantil</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -57487,7 +57488,7 @@
         <Synonym lang="es">Botulismo con toxemia intestinal de adultos</Synonym>
         <Synonym lang="es">Botulismo intestinal del adulto por toxinas</Synonym>
         <Synonym lang="es">Botulismo por colonización intestinal en el adulto</Synonym>
-        <Synonym lang="es">Botulismo tipo infantil</Synonym>
+        <Synonym lang="es">Botulismo similar al infantil</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -59519,7 +59520,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641" cycle="true"/>
+          <RootDisorder id="17230">
+            <OrphaCode>158661</OrphaCode>
+            <Name lang="es">Epidermólisis ampollosa simple suprabasal</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="es">Referido a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="71674" lang="es">
@@ -77190,7 +77201,7 @@
     <Disorder id="776">
       <OrphaCode>17</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=17</ExpertLink>
-      <Name lang="es">Acidosis láctica infantil letal con aciduria metilmalónica</Name>
+      <Name lang="es">Acidosis láctica letal del lactante con aciduria metilmalónica</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -79226,7 +79237,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Defecto de la comunicación auriculoventricular completo</Name>
+      <Name lang="es">Canal aurículoventricular completo</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -79234,9 +79245,9 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
-        <Synonym lang="es">AVSD completo</Synonym>
         <Synonym lang="es">CAVC</Synonym>
-        <Synonym lang="es">Defecto del canal auriculoventricular completo</Synonym>
+        <Synonym lang="es">DSAV completo</Synonym>
+        <Synonym lang="es">Defecto del septo aurículoventricular completo</Synonym>
       </SynonymList>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
@@ -94866,7 +94877,7 @@
       </DisorderFlagList>
       <SynonymList count="3">
         <Synonym lang="es">CHI</Synonym>
-        <Synonym lang="es">Hipoglucemia hiperinsulinémica persistente del lactante</Synonym>
+        <Synonym lang="es">Hipoglucemia hiperinsulinémica persistente infantil</Synonym>
         <Synonym lang="es">PHHI</Synonym>
       </SynonymList>
       <DisorderType id="21436">
@@ -103521,7 +103532,7 @@
     <Disorder id="16778">
       <OrphaCode>138118</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=138118</ExpertLink>
-      <Name lang="es">OBSOLETO: Trastorno adquirido del comportamiento alimentario del lactante</Name>
+      <Name lang="es">OBSOLETO: Trastorno adquirido del comportamiento alimentario de la infancia</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="455">
           <Value>16</Value>
@@ -108297,9 +108308,9 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=139420</ExpertLink>
       <Name lang="es">Mielitis transversa aguda secundaria</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="461">
+          <Value>1024</Value>
+          <Label>Obsolete with resources</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -108310,34 +108321,11 @@
       <DisorderGroup id="36554">
         <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="119981">
-          <Source>ICD-10</Source>
-          <Reference>G37.3</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="117092" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="149020" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>Es un trastorno poco frecuente caracterizado por la inflamación focal de la médula espinal debido a una causa conocida, normalmente una enfermedad inflamatoria.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1341">
@@ -116157,7 +116145,7 @@
     <Disorder id="1414">
       <OrphaCode>1145</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1145</ExpertLink>
-      <Name lang="es">Atrofia muscular espinal de inicio precoz ligada al cromosoma X</Name>
+      <Name lang="es">Atrofia muscular espinal de inicio en el lactante ligada al cromosoma X</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -118328,7 +118316,7 @@
     <Disorder id="1443">
       <OrphaCode>1186</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1186</ExpertLink>
-      <Name lang="es">Ataxia espinocerebelosa de inicio en la lactancia</Name>
+      <Name lang="es">Ataxia espinocerebelosa de inicio en el lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -127537,7 +127525,7 @@
     <Disorder id="17140">
       <OrphaCode>157716</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=157716</ExpertLink>
-      <Name lang="es">Síndrome CACH de inicio en la infancia tardía</Name>
+      <Name lang="es">Síndrome CACH infantil tardío</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -127772,7 +127760,7 @@
     <Disorder id="17139">
       <OrphaCode>157713</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=157713</ExpertLink>
-      <Name lang="es">Síndrome CACH congénito o de inicio en la primera infancia</Name>
+      <Name lang="es">Síndrome CACH congénito o infantil temprano</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -137155,7 +137143,7 @@
     <Disorder id="17000">
       <OrphaCode>140927</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=140927</ExpertLink>
-      <Name lang="es">Convulsiones neonatales-infantiles benignas familiares</Name>
+      <Name lang="es">Convulsiones benignas del neonato-lactante familiares</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -137164,7 +137152,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="es">BFNIS</Synonym>
-        <Synonym lang="es">Epilepsia neonatal-infantil benigna</Synonym>
+        <Synonym lang="es">Epilepsia benigna del neonato-lactante</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -155843,11 +155831,11 @@
         <Synonym lang="es">Epidermólisis bullosa distrófica dominante, solo uñas</Synonym>
         <Synonym lang="es">Epidermólisis bullosa dominante con distrofia ungueal solo</Synonym>
       </SynonymList>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
         <ExternalReference id="120315">
@@ -155892,11 +155880,11 @@
       <SynonymList count="1">
         <Synonym lang="es">Epidermólisis bullosa distrófica acral</Synonym>
       </SynonymList>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
         <ExternalReference id="120314">
@@ -156915,9 +156903,9 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158661</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa simple suprabasal</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
@@ -156929,33 +156917,27 @@
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="120309">
-          <Source>ICD-10</Source>
-          <Reference>Q81.0</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641">
+            <OrphaCode>304</OrphaCode>
+            <Name lang="es">Epidermólisis ampollosa simple</Name>
+          </TargetDisorder>
+          <RootDisorder id="17230" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="es">Referido a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="104376" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="127178" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>La epidermólisis ampollosa simple suprabasal es un subtipo de epidermólisis ampollosa simple hereditaria que comprende un grupo de afecciones clínicas y genéticamente heterogéneas, con un fenotipo variable de leve a grave caracterizado principalmente por erosiones cutáneas del lactante superficiales, localizadas o generalizadas, debido a la formación de ampollas en las capas epidérmicas intermedias o superiores. Se asocia con frecuencia con características de displasia ectodérmica y, dependiendo del trastorno específico, se puede observar un compromiso extracutáneo variable.</Contents>
-            </TextSection>
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
           </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
+          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -156964,9 +156946,9 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158665</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa simple basal</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="461">
+          <Value>1024</Value>
+          <Label>Obsolete with resources</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
@@ -156978,20 +156960,7 @@
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="120310">
-          <Source>ICD-10</Source>
-          <Reference>Q81.0</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -159600,8 +159569,8 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Enfermedad de Rathburn infantil</Synonym>
-        <Synonym lang="es">Fosfoetanolaminuria infantil</Synonym>
+        <Synonym lang="es">Enfermedad de Rathburn del lactante</Synonym>
+        <Synonym lang="es">Fosfoetanolaminuria del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -178560,11 +178529,11 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
         <ExternalReference id="121961">
@@ -180014,7 +179983,7 @@
     <Disorder id="19624">
       <OrphaCode>251304</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251304</ExpertLink>
-      <Name lang="es">Paniculitis de inicio en la lactancia con uveitis y granulomatosis sistémica</Name>
+      <Name lang="es">Paniculitis de inicio en el lactante con uveitis y granulomatosis sistémica</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -180422,7 +180391,7 @@
     <Disorder id="19790">
       <OrphaCode>254857</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=254857</ExpertLink>
-      <Name lang="es">Miopatía mitocondrial letal infantil</Name>
+      <Name lang="es">Miopatía mitocondrial letal del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -180430,7 +180399,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
-        <Synonym lang="es">Enfermedad mitocondrial letal infantil</Synonym>
+        <Synonym lang="es">Enfermedad mitocondrial letal del lactante</Synonym>
         <Synonym lang="es">LIMD</Synonym>
         <Synonym lang="es">LIMM</Synonym>
       </SynonymList>
@@ -180607,9 +180576,9 @@
       </DisorderFlagList>
       <SynonymList count="6">
         <Synonym lang="es">Deficiencia benigna de COX</Synonym>
-        <Synonym lang="es">Deficiencia infantil reversible de citocromo C oxidasa</Synonym>
-        <Synonym lang="es">Deficiencia infantil reversible de la cadena respiratoria</Synonym>
-        <Synonym lang="es">Miopatía con deficiencia infantil reversible de citocromo C oxidasa</Synonym>
+        <Synonym lang="es">Deficiencia reversible de citocromo C oxidasa del lactante</Synonym>
+        <Synonym lang="es">Deficiencia reversible de la cadena respiratoria del lactante</Synonym>
+        <Synonym lang="es">Miopatía con deficiencia reversible de citocromo C oxidasa del lactante</Synonym>
         <Synonym lang="es">Miopatía mitocondrial con deficiencia reversible de COX</Synonym>
         <Synonym lang="es">Miopatía mitocondrial con deficiencia reversible de complejo IV</Synonym>
       </SynonymList>
@@ -190900,7 +190869,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Osteopetrosis infantil maligna</Synonym>
+        <Synonym lang="es">Osteopetrosis maligna del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
@@ -201391,7 +201360,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19879" cycle="true"/>
+          <RootDisorder id="20606">
+            <OrphaCode>284408</OrphaCode>
+            <Name lang="es">OBSOLETO: Deficiencia de glicerol quinasa, forma infantil</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="es">Referido a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="111799" lang="es">
@@ -237578,7 +237557,7 @@
     <Disorder id="20472">
       <OrphaCode>280553</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280553</ExpertLink>
-      <Name lang="es">Miopatía miofibrilar hipertónica infantil letal</Name>
+      <Name lang="es">Miopatía miofibrilar hipertónica letal del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -261337,7 +261316,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Toxicosis hepática por cobre no Wilson del lactante y de la infancia</Synonym>
+        <Synonym lang="es">Toxicosis hepática por cobre no Wilson infantil</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -261915,7 +261894,7 @@
     <Disorder id="3525">
       <OrphaCode>3286</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3286</ExpertLink>
-      <Name lang="es">Taquicardia ventricular polimórfica catecolaminérgica</Name>
+      <Name lang="es">Taquicardia ventricular catecolaminérgica polimórfica</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -262894,7 +262873,7 @@
     <Disorder id="18700">
       <OrphaCode>210592</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=210592</ExpertLink>
-      <Name lang="es">OBSOLETO: Hemangioma infantil gigante</Name>
+      <Name lang="es">OBSOLETO: Hemangioma gigante infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="455">
           <Value>16</Value>
@@ -276216,11 +276195,11 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="6">
-        <Synonym lang="es">Enfermedad inflamatoria multisistémica de inicio en la lactancia</Synonym>
+        <Synonym lang="es">Enfermedad inflamatoria multisistémica de inicio en el lactante</Synonym>
         <Synonym lang="es">Enfermedad inflamatoria multisistémica de inicio neonatal</Synonym>
         <Synonym lang="es">Síndrome IOMID</Synonym>
         <Synonym lang="es">Síndrome NOMID</Synonym>
-        <Synonym lang="es">Síndrome articular-cutáneo-neurológico infantil crónico</Synonym>
+        <Synonym lang="es">Síndrome crónico articular-cutáneo-neurológico del lactante</Synonym>
         <Synonym lang="es">Síndrome de Prieur-Griscelli</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -276469,7 +276448,7 @@
     <Disorder id="18816">
       <OrphaCode>217052</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217052</ExpertLink>
-      <Name lang="es">OBSOLETO: Catarata no sindrómica de inicio precoz</Name>
+      <Name lang="es">OBSOLETO: Catarata no sindrómica infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="455">
           <Value>16</Value>
@@ -277850,7 +277829,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Neumonitis intersticial celular infantil</Synonym>
+        <Synonym lang="es">Neumonitis intersticial celular del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -285862,7 +285841,7 @@
     <Disorder id="3379">
       <OrphaCode>1561</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1561</ExpertLink>
-      <Name lang="es">Deficiencia infantil fatal de citocromo C oxidasa</Name>
+      <Name lang="es">Deficiencia de citocromo C oxidasa fatal del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -285870,8 +285849,8 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Cardioencefalomiopatía infantil fatal por deficiencia de citocromo c oxidasa</Synonym>
-        <Synonym lang="es">Deficiencia infantil fatal de COX</Synonym>
+        <Synonym lang="es">Cardioencefalomiopatía fatal del lactante por deficiencia de citocromo C oxidasa</Synonym>
+        <Synonym lang="es">Deficiencia de COX fatal del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -286962,7 +286941,7 @@
     <Disorder id="18977">
       <OrphaCode>225147</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=225147</ExpertLink>
-      <Name lang="es">Necrosis estriatal bilateral infantil esporádica</Name>
+      <Name lang="es">Necrosis estriatal bilateral esporádica infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -286971,10 +286950,10 @@
       </DisorderFlagList>
       <SynonymList count="5">
         <Synonym lang="es">ABSN</Synonym>
-        <Synonym lang="es">Degeneración estriatonigral infantil esporádica</Synonym>
+        <Synonym lang="es">Degeneración estriatonigral esporádica infantil</Synonym>
         <Synonym lang="es">IBSN esporádica</Synonym>
         <Synonym lang="es">Necrosis estriatal bilateral aguda</Synonym>
-        <Synonym lang="es">Necrosis estriatonigral infantil esporádica</Synonym>
+        <Synonym lang="es">Necrosis estriatonigral esporádica infantil</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -295376,7 +295355,7 @@
     <Disorder id="19100">
       <OrphaCode>228305</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=228305</ExpertLink>
-      <Name lang="es">Deficiencia de carnitina palmitoiltransferasa II, forma infantil grave</Name>
+      <Name lang="es">Deficiencia de carnitina palmitoiltransferasa II, forma grave infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -295384,13 +295363,13 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="7">
+        <Synonym lang="es">CPT2, forma grave infantil</Synonym>
         <Synonym lang="es">CPT2, forma hepatocardiomuscular</Synonym>
-        <Synonym lang="es">CPT2, forma infantil grave</Synonym>
+        <Synonym lang="es">CPTII, forma grave infantil</Synonym>
         <Synonym lang="es">CPTII, forma hepatocardiomuscular</Synonym>
-        <Synonym lang="es">CPTII, forma infantil grave</Synonym>
         <Synonym lang="es">Deficiencia de carnitina palmitoiltransferasa II, forma hepatocardiomuscular</Synonym>
+        <Synonym lang="es">Deficiencia de carnitina palmitoiltransferasa tipo 2, forma grave infantil</Synonym>
         <Synonym lang="es">Deficiencia de carnitina palmitoiltransferasa tipo 2, forma hepatocardiomuscular</Synonym>
-        <Synonym lang="es">Deficiencia de carnitina palmitoiltransferasa tipo 2, forma infantil grave</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -296729,7 +296708,7 @@
     <Disorder id="3651">
       <OrphaCode>306</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=306</ExpertLink>
-      <Name lang="es">Epilepsia infantil familiar benigna</Name>
+      <Name lang="es">Epilepsia benigna del lactante familiar</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -296739,8 +296718,8 @@
       <SynonymList count="4">
         <Synonym lang="es">BFIE</Synonym>
         <Synonym lang="es">BFIS</Synonym>
-        <Synonym lang="es">Convulsiones infantiles familiares benignas</Synonym>
-        <Synonym lang="es">Crisis infantiles familiares benignas</Synonym>
+        <Synonym lang="es">Convulsiones benignas del lactante familiares</Synonym>
+        <Synonym lang="es">Crisis benignas del lactante familiares</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -298228,8 +298207,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="2">
         <Synonym lang="es">Degeneración estriatonigral infantil</Synonym>
+        <Synonym lang="es">Necrosis estriatonigral infantil</Synonym>
       </SynonymList>
       <DisorderType id="21436">
         <Name lang="es">Grupo de fenomas</Name>
@@ -323654,7 +323634,7 @@
           <TargetDisorder id="22485" cycle="true"/>
           <RootDisorder id="20878">
             <OrphaCode>293838</OrphaCode>
-            <Name lang="es">Síndrome de encefalopatía infantil letal-hipertensión pulmonar</Name>
+            <Name lang="es">Síndrome de encefalopatía letal-hipertensión pulmonar del lactante</Name>
           </RootDisorder>
           <DisorderDisorderAssociationType id="21471">
             <Name lang="es">Desplazado a</Name>
@@ -324544,7 +324524,7 @@
     <Disorder id="22479">
       <OrphaCode>370088</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=370088</ExpertLink>
-      <Name lang="es">Síndrome de insuficiencia hepática aguda infantil-afectación multisistémica</Name>
+      <Name lang="es">Síndrome de insuficiencia hepática aguda-afectación multisistémica del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -324552,7 +324532,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Síndrome de fallo hepático infantil agudo-afectación multisistémica</Synonym>
+        <Synonym lang="es">Síndrome de fallo hepático agudo-afectación multisistémica del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -331024,7 +331004,7 @@
     <Disorder id="22288">
       <OrphaCode>363300</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=363300</ExpertLink>
-      <Name lang="es">Diarrea intratable de la infancia de origen genético</Name>
+      <Name lang="es">Diarrea intratable genético de la infancia</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -331892,7 +331872,7 @@
       <SynonymList count="3">
         <Synonym lang="es">AESD</Synonym>
         <Synonym lang="es">AIEF</Synonym>
-        <Synonym lang="es">Encefalopatía infantil aguda que afecta principalmente a los lóbulos frontales</Synonym>
+        <Synonym lang="es">Encefalopatía aguda infantil con afección principal de los lóbulos frontales</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -339727,7 +339707,7 @@
     <Disorder id="20878">
       <OrphaCode>293838</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293838</ExpertLink>
-      <Name lang="es">Síndrome de encefalopatía infantil letal-hipertensión pulmonar</Name>
+      <Name lang="es">Síndrome de encefalopatía letal-hipertensión pulmonar del lactante</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="459">
           <Value>256</Value>
@@ -344317,7 +344297,7 @@
     <Disorder id="20766">
       <OrphaCode>289857</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289857</ExpertLink>
-      <Name lang="es">Encefalopatía neonatal por glicina</Name>
+      <Name lang="es">Encefalopatía por glicina neonatal</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -344390,7 +344370,7 @@
     <Disorder id="20767">
       <OrphaCode>289860</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289860</ExpertLink>
-      <Name lang="es">Encefalopatía infantil por glicina</Name>
+      <Name lang="es">Encefalopatía por glicina infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -345985,7 +345965,7 @@
     <Disorder id="20839">
       <OrphaCode>293168</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293168</ExpertLink>
-      <Name lang="es">Parálisis espástica hereditaria ascendente de inicio en la lactancia</Name>
+      <Name lang="es">Parálisis espástica ascendente hereditaria de inicio en el lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -349819,7 +349799,7 @@
     <Disorder id="20735">
       <OrphaCode>289527</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289527</ExpertLink>
-      <Name lang="es">OBSOLETO: Miocardiopatía hipertrófica infantil letal por deficiencia del complejo I mitocondrial</Name>
+      <Name lang="es">OBSOLETO: Miocardiopatía hipertrófica letal infantil por deficiencia del complejo I mitocondrial</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="455">
           <Value>16</Value>
@@ -349827,9 +349807,9 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
-        <Synonym lang="es">HCEM infantil letal por deficiencia del complejo I mitocondrial</Synonym>
-        <Synonym lang="es">Miocardiopatía hipertrófica infantil letal por deficiencia de NADH-CoQ reductasa</Synonym>
-        <Synonym lang="es">Miocardiopatía hipertrófica infantil letal por deficiencia de NADH-coenzima Q reductasa</Synonym>
+        <Synonym lang="es">HCEM letal infantil por deficiencia del complejo I mitocondrial</Synonym>
+        <Synonym lang="es">Miocardiopatía hipertrófica letal infantil por deficiencia de NADH-CoQ reductasa</Synonym>
+        <Synonym lang="es">Miocardiopatía hipertrófica letal infantil por deficiencia de NADH-coenzima Q reductasa</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -354098,11 +354078,11 @@
     <Disorder id="20606">
       <OrphaCode>284408</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=284408</ExpertLink>
-      <Name lang="es">Deficiencia de glicerol quinasa infantil</Name>
+      <Name lang="es">OBSOLETO: Deficiencia de glicerol quinasa, forma infantil</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -354115,7 +354095,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19879">
+            <OrphaCode>261476</OrphaCode>
+            <Name lang="es">Síndrome de microdeleción Xp21</Name>
+          </TargetDisorder>
+          <RootDisorder id="20606" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="es">Referido a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -356589,7 +356579,7 @@
     <Disorder id="21458">
       <OrphaCode>313850</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=313850</ExpertLink>
-      <Name lang="es">Degeneración cerebelosa-retiniana infantil</Name>
+      <Name lang="es">Degeneración cerebelosa-retiniana del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -364541,7 +364531,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="es">Deficiencia de hexosaminidasas A y B, forma infantil</Synonym>
-        <Synonym lang="es">Gangliosidosis GM2 infantil variante 0</Synonym>
+        <Synonym lang="es">Gangliosidosis GM2 variante 0 infantil</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -377139,7 +377129,7 @@
           <TargetDisorder id="23753" cycle="true"/>
           <RootDisorder id="18700">
             <OrphaCode>210592</OrphaCode>
-            <Name lang="es">OBSOLETO: Hemangioma infantil gigante</Name>
+            <Name lang="es">OBSOLETO: Hemangioma gigante infantil</Name>
           </RootDisorder>
           <DisorderDisorderAssociationType id="27341">
             <Name lang="es">Referido a</Name>
@@ -377854,7 +377844,7 @@
     <Disorder id="23724">
       <OrphaCode>457375</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=457375</ExpertLink>
-      <Name lang="es">Trastorno neurológico infantil letal con cataratas y afectación cardíaca asociado al gen ITPA</Name>
+      <Name lang="es">Trastorno neurológico con cataratas y afectación cardíaca letal infantil asociado al gen ITPA</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -378771,8 +378761,8 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
+        <Synonym lang="es">Nistagmo congénito idiopático</Synonym>
         <Synonym lang="es">Nistagmo congénito motor</Synonym>
-        <Synonym lang="es">Nistagmo idiopático congénito</Synonym>
         <Synonym lang="es">Síndrome de nistagmo infantil</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -380742,7 +380732,7 @@
     <Disorder id="24170">
       <OrphaCode>466794</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=466794</ExpertLink>
-      <Name lang="es">Síndrome de insuficiencia hepática infantil-ataxia cerebelosa-neuropatía periférica sensitivo-motora</Name>
+      <Name lang="es">Síndrome de insuficiencia hepática-ataxia cerebelosa-neuropatía periférica sensitivo-motora infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -380752,7 +380742,7 @@
       <SynonymList count="3">
         <Synonym lang="es">Ataxia espinocerebelosa autosómica recesiva tipo 21</Synonym>
         <Synonym lang="es">SCAR21</Synonym>
-        <Synonym lang="es">Síndrome de fallo hepático infantil-ataxia cerebelosa-neuropatía periférica sensitivo-motora</Synonym>
+        <Synonym lang="es">Síndrome de fallo hepático-ataxia cerebelosa-neuropatía periférica sensitivo-motora infantil</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -388792,7 +388782,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="es">Síndrome IHPRF</Synonym>
-        <Synonym lang="es">Síndrome infantil de hipotonía-retraso psicomotor-facies característica</Synonym>
+        <Synonym lang="es">Síndrome de hipotonía infantil-retraso psicomotor-facies característica</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -390810,7 +390800,7 @@
     <Disorder id="22603">
       <OrphaCode>391316</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391316</ExpertLink>
-      <Name lang="es">Epilepsia del lóbulo temporal mesial de inicio en la lactancia con regresión cognitiva grave</Name>
+      <Name lang="es">Epilepsia del lóbulo temporal mesial de inicio en el lactante con regresión cognitiva grave</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -397961,7 +397951,7 @@
     <Disorder id="23330">
       <OrphaCode>436166</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436166</ExpertLink>
-      <Name lang="es">Síndrome autoinflamatorio-fiebre periódica-enterocolitis infantil</Name>
+      <Name lang="es">Síndrome autoinflamatorio-fiebre periódica-enterocolitis del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -397973,7 +397963,7 @@
         <Synonym lang="es">Síndrome autoinflamatorio con SAM asociado al gen NLRC4</Synonym>
         <Synonym lang="es">Síndrome autoinflamatorio con síndrome de activación del macrófago asociado al gen NLRC4</Synonym>
         <Synonym lang="es">Síndrome autoinflamatorio con síndrome de activación macrofágica asociado al gen NLRC4</Synonym>
-        <Synonym lang="es">Síndrome autoinflamatorio-enterocolitis infantil asociado al gen NLRC4</Synonym>
+        <Synonym lang="es">Síndrome autoinflamatorio-enterocolitis del lactante asociado al gen NLRC4</Synonym>
         <Synonym lang="es">Síndrome de activación del macrófago asociado al gen NLRC4</Synonym>
         <Synonym lang="es">Síndrome de activación macrofágica asociado al gen NLRC4</Synonym>
       </SynonymList>
@@ -411546,7 +411536,7 @@
     <Disorder id="23209">
       <OrphaCode>425120</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=425120</ExpertLink>
-      <Name lang="es">Vasculopatía asociada al gen STING con inicio en la infancia</Name>
+      <Name lang="es">Vasculopatía con inicio en el lactante asociada al gen STING</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -422170,7 +422160,7 @@
     <Disorder id="25994">
       <OrphaCode>500062</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=500062</ExpertLink>
-      <Name lang="es">Síndrome de fiebre periódica-paniculitis-dermatosis de inicio en la lactancia</Name>
+      <Name lang="es">Síndrome de fiebre periódica-paniculitis-dermatosis de inicio en el lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -430654,16 +430644,17 @@
     <Disorder id="8597">
       <OrphaCode>1330</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1330</ExpertLink>
-      <Name lang="es">Defecto de la comunicación auriculoventricular parcial</Name>
+      <Name lang="es">Canal aurículoventricular parcial</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
-        <Synonym lang="es">AVSD parcial</Synonym>
-        <Synonym lang="es">Defecto del canal auriculoventricular parcial</Synonym>
+      <SynonymList count="4">
+        <Synonym lang="es">CAVP</Synonym>
+        <Synonym lang="es">DSAV parcial</Synonym>
+        <Synonym lang="es">Defecto del septo aurículoventricular parcial</Synonym>
         <Synonym lang="es">PAVC</Synonym>
       </SynonymList>
       <DisorderType id="21415">
@@ -435144,7 +435135,7 @@
     <Disorder id="9793">
       <OrphaCode>31709</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=31709</ExpertLink>
-      <Name lang="es">Convulsiones infantiles y coreoatetosis</Name>
+      <Name lang="es">Convulsiones del lactante y coreoatetosis</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -435152,7 +435143,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Discinesia paroxística cinesigénica y convulsiones infantiles</Synonym>
+        <Synonym lang="es">Discinesia paroxística cinesigénica y convulsiones del lactante</Synonym>
         <Synonym lang="es">Síndrome ICCA</Synonym>
       </SynonymList>
       <DisorderType id="21394">
@@ -437056,7 +437047,7 @@
     <Disorder id="25136">
       <OrphaCode>477661</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477661</ExpertLink>
-      <Name lang="es">Enfermedad inflamatoria intestinal infantil asociada al gen IL21</Name>
+      <Name lang="es">Enfermedad inflamatoria intestinal del lactante asociada al gen IL21</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -437064,7 +437055,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">EII infantil asociada al gen IL21</Synonym>
+        <Synonym lang="es">EII del lactante asociada al gen IL21</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -437143,7 +437134,7 @@
     <Disorder id="25138">
       <OrphaCode>477673</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477673</ExpertLink>
-      <Name lang="es">Síndrome postnatal de microcefalia-hipotonía infantil-diplejía espástica-disartria-discapacidad intelectual</Name>
+      <Name lang="es">Síndrome de microcefalia postnatal-hipotonía infantil-diplejía espástica-disartria-discapacidad intelectual</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -442002,14 +441993,15 @@
     <Disorder id="10958">
       <OrphaCode>70587</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=70587</ExpertLink>
-      <Name lang="es">Síndrome de  distrés respiratorio agudo infantil</Name>
+      <Name lang="es">Síndrome de dificultad respiratoria aguda del lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="1">
+        <Synonym lang="es">Síndrome de dificultad respiratoria aguda del neonato</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -454775,7 +454767,7 @@
     <Disorder id="28448">
       <OrphaCode>565788</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=565788</ExpertLink>
-      <Name lang="es">Enfermedad inflamatoria intestinal infantil con afectación neurológica</Name>
+      <Name lang="es">Enfermedad inflamatoria intestinal del lactante con afectación neurológica</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -465008,7 +465000,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="2">
+        <Synonym lang="es">Enfermedad de Gaucher aguda neuronopática</Synonym>
+        <Synonym lang="es">Enfermedad de Gaucher cerebral infantil</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -468119,7 +468113,7 @@
     <Disorder id="11032">
       <OrphaCode>73014</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73014</ExpertLink>
-      <Name lang="es">Diarrea intratable del lactante</Name>
+      <Name lang="es">Diarrea intratable infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -495230,9 +495224,9 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=89841</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa distrófica centrípeta recesiva</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="461">
+          <Value>1024</Value>
+          <Label>Obsolete with resources</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="5">
@@ -495248,34 +495242,11 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="116609">
-          <Source>ICD-10</Source>
-          <Reference>Q81.2</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="75776" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="84778" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>La epidermólisis ampollosa distrófica centrípeta recesiva (RCEB-Ce) es un subtipo extremadamente poco frecuente de epidermólisis ampollosa distrófica (DEB, consulte este término), que se caracteriza por la formación de ampollas que comienza en las zonas acrales y se extiende progresivamente hacia el tronco.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11905">
@@ -495283,9 +495254,9 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=89840</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa juntural tipo no Herlitz</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="461">
+          <Value>1024</Value>
+          <Label>Obsolete with resources</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -495296,56 +495267,11 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="3">
-        <ExternalReference id="140593">
-          <Source>UMLS</Source>
-          <Reference>C0268374</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="38496">
-          <Source>OMIM</Source>
-          <Reference>226650</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="116608">
-          <Source>ICD-10</Source>
-          <Reference>Q81.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="72017" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="69866" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>La epidermólisis ampollosa juntural no-Herlitz (EAJ-nH) es un subtipo de epidermólisis ampollosa juntural (JEB, consulte este término) caracterizado por la formación de ampollas en la piel y las mucosas, distrofia ungueal o ausencia de uñas, e hipoplasia del esmalte.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11904">
@@ -495353,9 +495279,9 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=89839</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa simple superficial</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="461">
+          <Value>1024</Value>
+          <Label>Obsolete with resources</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
@@ -495368,56 +495294,11 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="3">
-        <ExternalReference id="116607">
-          <Source>ICD-10</Source>
-          <Reference>Q81.0</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="es">TETG (el Término más Específico del código ORPHA se asigna a un Término más Genérico)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="es">Atribuido (Orphanet atribuye el código CIE10)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="45080">
-          <Source>OMIM</Source>
-          <Reference>607600</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="140594">
-          <Source>UMLS</Source>
-          <Reference>C1843477</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="es">correspondencia exacta (los términos y los conceptos son equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="es">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="71671" lang="es">
-          <TextSectionList count="1">
-            <TextSection id="68798" lang="es">
-              <TextSectionType id="16907">
-                <Name lang="es">Definición</Name>
-              </TextSectionType>
-              <Contents>La epidermólisis ampollar simple superficial (EBSS) es un subtipo suprabasal de epidermólisis ampollar simple (EBS, consulte este término) caracterizado por erosiones superficiales generalizadas o acrales con ausencia de ampollas.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11919">
@@ -497648,8 +497529,9 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="1">
+      <SynonymList count="2">
         <Synonym lang="es">Lipodistrofia centrífuga abdominal infantil</Synonym>
+        <Synonym lang="es">Lipodistrophia centrifugalis abdominalis infantilis</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -502088,7 +501970,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Sialidosis infantil dismórfica</Synonym>
+        <Synonym lang="es">Sialidosis dismórfica infantil</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -514984,7 +514866,7 @@
     <Disorder id="12039">
       <OrphaCode>90395</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90395</ExpertLink>
-      <Name lang="es">Mucinosis papular del lactante</Name>
+      <Name lang="es">Mucinosis papular infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -514992,7 +514874,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
-        <Synonym lang="es">Mucinosis cutánea del lactante</Synonym>
+        <Synonym lang="es">Mucinosis cutánea infantil</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -517868,7 +517750,7 @@
           <TargetDisorder id="12152" cycle="true"/>
           <RootDisorder id="18816">
             <OrphaCode>217052</OrphaCode>
-            <Name lang="es">OBSOLETO: Catarata no sindrómica de inicio precoz</Name>
+            <Name lang="es">OBSOLETO: Catarata no sindrómica infantil</Name>
           </RootDisorder>
           <DisorderDisorderAssociationType id="27341">
             <Name lang="es">Referido a</Name>
@@ -522189,11 +522071,11 @@
         <Synonym lang="es">JEB generalizada intermedia</Synonym>
         <Synonym lang="es">JEB-nH gen</Synonym>
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
         <ExternalReference id="50928">
@@ -522713,11 +522595,11 @@
         <Synonym lang="es">DEB-Pt</Synonym>
         <Synonym lang="es">Epidermólisis bullosa distrófica pretibial</Synonym>
       </SynonymList>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <ExternalReferenceList count="3">
         <ExternalReference id="139692">
@@ -525243,14 +525125,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="es">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="es">Está enfermedad se describe en  Síndrome de Griscelli</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11502">
@@ -526033,7 +525908,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="es">Enfermedad de Norman-Landing</Synonym>
-        <Synonym lang="es">Gangliosidosis GM1 infantil</Synonym>
+        <Synonym lang="es">Gangliosidosis GM1 del lactante</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -545971,7 +545846,7 @@
     <Disorder id="11644">
       <OrphaCode>85179</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=85179</ExpertLink>
-      <Name lang="es">Osteopetrosis con displasia neuroaxonal, forma infantil</Name>
+      <Name lang="es">Osteopetrosis del lactante con displasia neuroaxonal</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -549368,7 +549243,7 @@
     <Disorder id="13275">
       <OrphaCode>98258</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98258</ExpertLink>
-      <Name lang="es">Síndrome epiléptico de la infancia</Name>
+      <Name lang="es">Síndrome epiléptico infantil</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -560951,7 +560826,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12600">
@@ -562946,7 +562828,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Diarrea prolongada de la infancia inmunomediada</Synonym>
+        <Synonym lang="es">Diarrea inmunomediada prolongada de la infancia</Synonym>
         <Synonym lang="es">Enteropatía autoinmune</Synonym>
       </SynonymList>
       <DisorderType id="36561">
@@ -572041,6 +571923,31 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29740">
+      <OrphaCode>595356</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595356</ExpertLink>
+      <Name lang="es">Localized dystrophic epidermolysis bullosa</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12526">
       <OrphaCode>93968</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93968</ExpertLink>
@@ -572307,6 +572214,38 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29737">
+      <OrphaCode>595337</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595337</ExpertLink>
+      <Name lang="es">Adrenal hypoplasia congenita</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21436">
+        <Name lang="es">Grupo de fenomas</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12520">
       <OrphaCode>93962</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93962</ExpertLink>
@@ -572380,6 +572319,38 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29739">
+      <OrphaCode>595351</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595351</ExpertLink>
+      <Name lang="es">Epidermolysis bullosa simplex with extracutaneous involvement</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="es">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12522">
       <OrphaCode>93964</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93964</ExpertLink>
@@ -572440,6 +572411,38 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29738">
+      <OrphaCode>595346</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595346</ExpertLink>
+      <Name lang="es">Epidermolysis bullosa simplex without extracutaneous involvement</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="es">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12516">
       <OrphaCode>93958</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93958</ExpertLink>
@@ -572499,6 +572502,38 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29734">
+      <OrphaCode>595216</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595216</ExpertLink>
+      <Name lang="es">Fibrous dysplasia/McCune-Albright syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21436">
+        <Name lang="es">Grupo de fenomas</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="es">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="es">Este término no caracteriza una enfermedad, sino un grupo de enfermedades. Para saber más acerca de las enfermedades incluidas bajo este término, puede consultar las clasificaciones.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12519">
       <OrphaCode>93961</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93961</ExpertLink>
@@ -574169,6 +574204,31 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29724">
+      <OrphaCode>595109</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595109</ExpertLink>
+      <Name lang="es">Atypical Timothy syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12509">
       <OrphaCode>93951</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93951</ExpertLink>
@@ -574204,6 +574264,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29725">
+      <OrphaCode>595133</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595133</ExpertLink>
+      <Name lang="es">Perivascular Epithelioid Cell Neoplasm</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12508">
       <OrphaCode>93950</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93950</ExpertLink>
@@ -574246,6 +574331,31 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29722">
+      <OrphaCode>595098</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595098</ExpertLink>
+      <Name lang="es">Timothy syndrome type 1</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12507">
       <OrphaCode>93947</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93947</ExpertLink>
@@ -574281,6 +574391,31 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29723">
+      <OrphaCode>595105</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=595105</ExpertLink>
+      <Name lang="es">Timothy syndrome type 2</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12506">
       <OrphaCode>93946</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93946</ExpertLink>
@@ -578321,17 +578456,16 @@
     <Disorder id="14085">
       <OrphaCode>99068</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99068</ExpertLink>
-      <Name lang="es">Defecto de la comunicación auriculoventricular completo-tetralogía de Fallot</Name>
+      <Name lang="es">Defecto del septo aurículoventricular completo con Tetralogía de Fallot</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="3">
-        <Synonym lang="es">CACV-tetralogía de Fallot</Synonym>
-        <Synonym lang="es">CAVC completo-tetralogía de Fallot</Synonym>
-        <Synonym lang="es">Defecto del canal auriculoventricular completo-tetralogía de Fallot</Synonym>
+      <SynonymList count="2">
+        <Synonym lang="es">CAVC con Tetralogía de Fallot</Synonym>
+        <Synonym lang="es">Canal aurículoventricular completo con Tetralogía de Fallot</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -578380,7 +578514,7 @@
     <Disorder id="14084">
       <OrphaCode>99067</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99067</ExpertLink>
-      <Name lang="es">Defecto de la comunicación auriculoventricular completo con hipoplasia ventricular</Name>
+      <Name lang="es">Ventrículo único con defecto del septo auriculoventricular completo</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -578388,11 +578522,11 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="5">
-        <Synonym lang="es">AVSD completo con hipoplasia ventricular</Synonym>
-        <Synonym lang="es">CAVC con hipoplasia ventricular</Synonym>
-        <Synonym lang="es">Canal auriculoventricular completo desequilibrado</Synonym>
-        <Synonym lang="es">Defecto del canal auriculoventricular completo con desequilibrio ventricular</Synonym>
-        <Synonym lang="es">Defecto del canal auriculoventricular completo con hipoplasia ventricular</Synonym>
+        <Synonym lang="es">Canal aurículoventricular completo con disbalance ventricular</Synonym>
+        <Synonym lang="es">Canal aurículoventricular completo disbalanceado</Synonym>
+        <Synonym lang="es">Ventrículo único con CAVC</Synonym>
+        <Synonym lang="es">Ventrículo único con DSAV completo</Synonym>
+        <Synonym lang="es">Ventrículo único con canal auriculoventricular completo</Synonym>
       </SynonymList>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
@@ -578675,7 +578809,7 @@
         <DisorderDisorderAssociation>
           <TargetDisorder id="871">
             <OrphaCode>1329</OrphaCode>
-            <Name lang="es">Defecto de la comunicación auriculoventricular completo</Name>
+            <Name lang="es">Canal aurículoventricular completo</Name>
           </TargetDisorder>
           <RootDisorder id="14083" cycle="true"/>
           <DisorderDisorderAssociationType id="27341">
@@ -591021,7 +591155,13 @@
           <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
-      <SynonymList count="0">
+      <SynonymList count="6">
+        <Synonym lang="es">Atrofia muscular espinal autosómica recesiva con dificultad respiratoria</Synonym>
+        <Synonym lang="es">Atrofia muscular espinal diafragmática</Synonym>
+        <Synonym lang="es">Neuropatía axonal grave del lactante con insuficiencia respiratoria tipo 1</Synonym>
+        <Synonym lang="es">Neuropatía motora distal hereditaria tipo 6</Synonym>
+        <Synonym lang="es">dHMN6</Synonym>
+        <Synonym lang="es">dSMA1</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -611128,7 +611268,7 @@
     <Disorder id="28746">
       <OrphaCode>572428</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=572428</ExpertLink>
-      <Name lang="es">Proteinosis alveolar pulmonar-hipogammaglobulinemia de inicio en la infancia</Name>
+      <Name lang="es">Proteinosis alveolar pulmonar-hipogammaglobulinemia de inicio en el lactante</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -611137,7 +611277,7 @@
       </DisorderFlagList>
       <SynonymList count="2">
         <Synonym lang="es">Deficiencia de OAS1</Synonym>
-        <Synonym lang="es">Proteinosis alveolar pulmonar-hipogammaglobulinemia infantil asociada al gen OAS1</Synonym>
+        <Synonym lang="es">Proteinosis alveolar pulmonar-hipogammaglobulinemia en el lactante asociada al gen OAS1</Synonym>
       </SynonymList>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
@@ -635949,7 +636089,7 @@
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="2">
-        <Synonym lang="es">Encefalopatía progresiva con anorexia infantil grave</Synonym>
+        <Synonym lang="es">Encefalopatía progresiva con anorexia grave del lactante</Synonym>
         <Synonym lang="es">Síndrome de la Isla de la Reunión-anorexia-vómitos incontenibles-signos neurológicos</Synonym>
       </SynonymList>
       <DisorderType id="21394">
