commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml
index 297dd39..d4a6766 100644
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:38:07" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:39:33" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10603">
+  <DisorderList count="10617">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -17764,14 +17764,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="it">Questo termine non si riferisce a una malattia ma a un gruppo di malattie. Per conoscere le malattie comprese all'interno di questo termine, è possibile consultare le classificazioni.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="167">
@@ -78927,11 +78920,19 @@
       <OrphaCode>1549</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1549</ExpertLink>
       <Name lang="it">Criptosporidiosi</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="3">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
+        <DisorderFlag id="456">
+          <Value>32</Value>
+          <Label>Non-rare disease in Europe</Label>
+        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
@@ -78941,18 +78942,7 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="5">
-        <ExternalReference id="107110">
-          <Source>MedDRA</Source>
-          <Reference>10011502</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="1">
         <ExternalReference id="107113">
           <Source>ICD-10</Source>
           <Reference>A07.2</Reference>
@@ -78966,47 +78956,10 @@
             <Name lang="it">Confermato</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="145137">
-          <Source>GARD</Source>
-          <Reference>6219</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="107109">
-          <Source>UMLS</Source>
-          <Reference>C0010418</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="107108">
-          <Source>MeSH</Source>
-          <Reference>D003457</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="15504" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18412">
@@ -344921,7 +344874,7 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="2">
+      <DisorderDisorderAssociationList count="3">
         <DisorderDisorderAssociation>
           <TargetDisorder id="21817" cycle="true"/>
           <RootDisorder id="3393">
@@ -344932,6 +344885,16 @@
             <Name lang="it">Riferito a</Name>
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="21817" cycle="true"/>
+          <RootDisorder id="10773">
+            <OrphaCode>57194</OrphaCode>
+            <Name lang="it">OBSOLETO: Osteite asettica</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="it">Riferito a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
         <DisorderDisorderAssociation>
           <TargetDisorder id="21817" cycle="true"/>
           <RootDisorder id="12460">
@@ -353822,14 +353785,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="it">Questa malattia è descritta alla pagina  Sindrome da ipereosinofilia</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21543">
@@ -353872,14 +353828,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="it">Questa malattia è descritta alla pagina  Sindrome da ipereosinofilia</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21542">
@@ -381921,19 +381870,11 @@
       <OrphaCode>80</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=80</ExpertLink>
       <Name lang="it">MALATTIA NON RARA IN EUROPA: Sindrome da antifosfolipidi</Name>
-      <DisorderFlagList count="3">
-        <DisorderFlag id="495">
-          <Value>8192</Value>
-          <Label>Inactive</Label>
-        </DisorderFlag>
+      <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
-        <DisorderFlag id="456">
-          <Value>32</Value>
-          <Label>Non-rare disease in Europe</Label>
-        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
         <Synonym lang="it">Lupus anticoagulante familiare</Synonym>
@@ -381950,14 +381891,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="it">Questa malattia non è considerata rara in Europa. Non è inclusa nella nomenclatura delle malattie rare di Orphanet</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20670">
@@ -382558,7 +382492,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Questa malattia è descritta alla pagina  Malattia da deposito di glicogeno causata dal deficit di lattato deidrogenasi</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20633">
@@ -494874,14 +494815,24 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="21817">
+            <OrphaCode>324964</OrphaCode>
+            <Name lang="it">Osteomielite multifocale ricorrente cronica</Name>
+          </TargetDisorder>
+          <RootDisorder id="10773" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="it">Riferito a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="-1" lang="it">
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="it">Questa malattia è considerata obsoleta secondo la nomenclatura delle malattie rare di Orphanet.</Info>
+            <Info lang="it">Questa malattia è considerata obsoleta secondo la nomenclatura delle malattie rare di Orphanet.&lt;br/&gt;&lt;br/&gt;Al suo posto, si prega di utilizzare  Osteomielite multifocale ricorrente cronica</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -509963,83 +509914,11 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="6">
-        <ExternalReference id="114930">
-          <Source>ICD-10</Source>
-          <Reference>K20</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="it">NTBT (il termine più ristretto dell'ORPHA code corrisponde a un termine più ampio)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="it">Attribuito (il codice ICD10 è attribuito da Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="14890">
-          <Source>OMIM</Source>
-          <Reference>610247</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="45508">
-          <Source>OMIM</Source>
-          <Reference>613412</Reference>
-          <DisorderMappingRelation id="21541">
-            <Name lang="it">BTNT (il termine più ampio dell'ORPHA code corrisponde a un termine più ristretto)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114926">
-          <Source>MeSH</Source>
-          <Reference>D057765</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114927">
-          <Source>UMLS</Source>
-          <Reference>C0341106</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="114928">
-          <Source>MedDRA</Source>
-          <Reference>10064212</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="45168" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28655">
@@ -622968,7 +622847,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Questa malattia è descritta alla pagina  Sindrome FOXG1</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12402">
@@ -703276,6 +703162,206 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31348">
+      <OrphaCode>617449</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617449</ExpertLink>
+      <Name lang="it">Congenital aphakia-iris hypoplasia-microphthalmia-microcornea syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31346">
+      <OrphaCode>617408</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617408</ExpertLink>
+      <Name lang="it">Classic eosinophilic pustular folliculitis</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31347">
+      <OrphaCode>617440</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617440</ExpertLink>
+      <Name lang="it">Painful legs and moving toes syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21422">
+        <Name lang="it">Sindrome clinica</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31341">
+      <OrphaCode>617307</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617307</ExpertLink>
+      <Name lang="it">Rare disorder related to monochorionic twin pregnancy</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="it">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="it">Gruppo di malattie</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31340">
+      <OrphaCode>617304</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617304</ExpertLink>
+      <Name lang="it">Amniotic Fluid Embolism</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31343">
+      <OrphaCode>617313</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617313</ExpertLink>
+      <Name lang="it">Rare disorder related to Inadequate sharing of the placenta</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="it">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="it">Gruppo di malattie</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31342">
+      <OrphaCode>617310</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617310</ExpertLink>
+      <Name lang="it">Rare disorder related to Unbalanced inter-twin blood transfusion</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="36561">
+        <Name lang="it">Category</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="it">Gruppo di malattie</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31337">
+      <OrphaCode>617294</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617294</ExpertLink>
+      <Name lang="it">Twin Anemia Polycythemia Sequence</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="31336">
       <OrphaCode>616874</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=616874</ExpertLink>
@@ -703289,11 +703375,11 @@
       <SynonymList count="1">
         <Synonym lang="it">Malattia rara esaminata a fondo priva di diagnosi</Synonym>
       </SynonymList>
-      <DisorderType id="21464">
-        <Name lang="it">Nessun tipo</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="38983">
-        <Name lang="it">No type</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
@@ -703312,5 +703398,155 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31339">
+      <OrphaCode>617301</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617301</ExpertLink>
+      <Name lang="it">Selective Intrauterine Growth Restriction</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31338">
+      <OrphaCode>617297</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617297</ExpertLink>
+      <Name lang="it">Twin Reversed Arterial Perfusion sequence</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31370">
+      <OrphaCode>617910</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617910</ExpertLink>
+      <Name lang="it">Conjunctival malignant melanoma</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31371">
+      <OrphaCode>617916</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617916</ExpertLink>
+      <Name lang="it">Diffuse idiopathic pulmonary neuroendocrine cell hyperplasia</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31372">
+      <OrphaCode>617919</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617919</ExpertLink>
+      <Name lang="it">F12-associated cold auto-inflammatory syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31373">
+      <OrphaCode>617930</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=617930</ExpertLink>
+      <Name lang="it">Hemophilia B Leyden</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
