commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml
old mode 100644
new mode 100755
index 0a76242..254ea26
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/it_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:29:24" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:29:54" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10501">
+  <DisorderList count="10512">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -17564,7 +17564,7 @@
     <Disorder id="17526">
       <OrphaCode>163696</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=163696</ExpertLink>
-      <Name lang="it">Sindrome mioclono d'azione-insufficienza renale</Name>
+      <Name lang="it">Sindrome da mioclono d'azione e insufficienza renale</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -17608,7 +17608,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122379" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158976" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sindrome che associa il mioclono d'azione e l'insufficienza renale (AMRF) è una forma rara di epilessia, caratterizzata da epilessia mioclonica progressiva e glomerulopatia primitiva. I pazienti presentano sintomi neurologici (tremore, mioclono d'azione, crisi tonico-cloniche e, successivamente, atassia e disartria) che precedono, insorgono contemporaneamente o seguono i sintomi renali (compresa la proteinuria che progredisce verso la sindrome nefrosica e la malattia renale terminale). Alcuni pazienti presentano inoltre una neuropatia sensitivo-motoria periferica, sordità neurosensoriale e cardiomiopatia dilatativa.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="180">
@@ -58271,8 +58281,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21422">
-        <Name lang="it">Sindrome clinica</Name>
+      <DisorderType id="21429">
+        <Name lang="it">Situazione clinica particolare in una malattia o sindrome</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
@@ -58314,7 +58324,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Per questa malattia rara esiste una designazione orfana europea e/o statunitense.Questo termine designa un'indicazione per un test farmacogenetico.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18012">
@@ -59343,7 +59360,7 @@
     <Disorder id="18011">
       <OrphaCode>178311</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=178311</ExpertLink>
-      <Name lang="it">Iperostosi sterno-costo-clavicolare isolata</Name>
+      <Name lang="it">Iperostosi isolata sterno-costo-clavicolare</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -59386,7 +59403,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122392" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159038" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'iperostosi isolata sterno-costo-clavicolare è una malattia reumatologica rara, caratterizzata da infiammazione sterile cronica, per lo più bilaterale, dell'articolazione sterno-costo-clavicolare, associata a sclerosi progressiva e iperostosi, con ossificazione del tessuto molle adiacente, senza interessamento di altre articolazioni. Esordisce con episodi ricorrenti di dolore, edema e/o eritema della regione sterno-clavicolare. Alcuni pazienti presentano pustolosi palmo-plantare.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18010">
@@ -62715,15 +62742,12 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="73432" lang="it">
-          <TextSectionList count="1">
-            <TextSection id="74325" lang="it">
-              <TextSectionType id="16907">
-                <Name lang="it">Definizione</Name>
-              </TextSectionType>
-              <Contents>I</Contents>
-            </TextSection>
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
           </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Questo termine non si riferisce a una malattia ma a un gruppo di malattie. Per conoscere le malattie comprese all'interno di questo termine, è possibile consultare le classificazioni.</Info>
+          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -65836,7 +65860,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122395" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159050" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La miopatia metabolica da difetto del trasportatore del lattato è una miopatia metabolica rara caratterizzata da crampi muscolari e/o rigidità secondari all'esercizio fisico (in particolare da esposizione al caldo), rabdomiolisi e mioglobinuria indotte dall'esercizio fisico e livelli elevati di creatin-chinasi nel siero.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="710">
@@ -95689,7 +95723,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122402" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159057" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La tetrasomia 9p è un'anomalia cromosomica rara, caratterizzata da ritardo della crescita pre- e post-natale, ritardo psicomotorio, disabilità intellettiva lieve o moderata, ipotonia, microcefalia, dismorfismi facciali (ipertelorismo oculare, orecchie malformate e a basso impianto, naso globoso/a becco, microretrognazia, enoftalmo/microftalmo, epicanto, strabismo) labio/palatoschisi, anomalie scheletriche (ipoplasia delle unghie/falangi distali, bassa statura, collo corto, contratture), cardiopatie congenite, malformazioni renali e urogenitali (ipoplasia dei reni e dei genitali, criptorchidismo).</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1225">
@@ -137689,7 +137733,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122365" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158933" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La displasia cranio-telencefalica è un difetto genetico molto raro dello sviluppo dell'embrione, caratterizzato da craniosinostosi con encefalocele frontale e altre anomalie cerebrali (idrocefalo grave, agenesia del corpo calloso, lissencefalia e polimicrogiria, cisti parenchimali, displasia setto-ottica), che esitano in una marcata disfunzione cerebrale, nelle crisi epilettiche e in un ritardo psicomotorio molto marcato. Non sono stati descritti nuovi casi dopi il 1983.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1695">
@@ -167733,7 +167787,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122390" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159020" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il ganglioneuroma è una neoplasia rara benigna del tessuto neuroepiteliale, che si presenta come massa differenziata di derivazione dalla cresta neurale, che origina dal sistema nervoso simpatico ed è composta da cellule ganglionari e stromali di Schwann. Può localizzarsi in ogni distretto, dalla base del cranio fino alla pelvi, anche se di solito predilige le ghiandole surrenali, il retroperitoneo, il mediastino posteriore e la pelvi; localizzazioni meno frequenti sono il sistema nervoso centrale, il cuore, le ossa e l'intestino. I pazienti possono essere asintomatici oppure presentare segni clinici secondari alla presenza della massa. Può associarsi alla neurofibromatosi tipo 1, alla neoplasia endocrina multipla tipo 2B e alla sindrome di Turner.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19709">
@@ -176795,12 +176859,15 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="it">
-          <TextSectionList count="0">
+        <SummaryInformation id="122396" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159051" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La forma encefalomiopatica della sindrome da deplezione del DNA mitocondriale comprende un gruppo di malattie della sindrome da mantenimento del DNA mitocondriale, caratterizzate prevalentemente da sintomi neuromuscolari. Di solito l'esordio avviene nel periodo neonatale con ipotonia, acidosi lattica, ritardo psicomotorio, movimenti ipercinetici-distonici progressivi, oftalmoplegia esterna, sordità neurosensoriale, crisi epilettiche generalizzate e disfunzione tubulare renale di gravità variabile. Possono associarsi altri segni clinici.</Contents>
+            </TextSection>
           </TextSectionList>
-          <TextAuto>
-            <Info lang="it">Questo termine non si riferisce a una malattia ma a un gruppo di malattie. Per conoscere le malattie comprese all'interno di questo termine, è possibile consultare le classificazioni.</Info>
-          </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
@@ -181911,7 +181978,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122398" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159053" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il melanoma primitivo del sistema nervoso centrale è una neoplasia rara delle meningi che origina dai melanociti leptomeningei, in particolare nella regione perimidollare o cervicale superiore, in assenza di altri melanomi al di fuori del SNC. Di solito esordisce sotto forma di massa solida, di colore scuro, spesso contenente aree emorragiche o necrotiche, composte da strati di cellule pleomorfe con nucleoli prominenti, spesso con figure mitotiche e invasione parenchimale. La forma intracranica esordisce con segni clinici secondari all'aumento della pressione intracranica, sintomi neurologici focali correlati alla localizzazione del tumore e crisi epilettiche o emorragia subaracnoidea; la forma spinale esordisce invece con dolore alla schiena, debolezza muscolare, insensibilità, paralisi o incontinenza urinaria.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2503">
@@ -184052,9 +184129,9 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=252175</ExpertLink>
       <Name lang="it">Schwannoma vestibolare</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
@@ -184068,11 +184145,56 @@
       <DisorderGroup id="36554">
         <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="3">
+        <ExternalReference id="186438">
+          <Source>MedDRA</Source>
+          <Reference>10000523</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation/>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="it">Confermato</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="186437">
+          <Source>UMLS</Source>
+          <Reference>C0027859</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation/>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="it">Confermato</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="186440">
+          <Source>ICD-10</Source>
+          <Reference>D33.3</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="it">mappatura corretta (i termini e i concetti sono equivalenti)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21583">
+            <Name lang="it">Codice specifico (l'ORPHA code ha un proprio codice in ICD10)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="it">Confermato</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122481" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159205" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Lo schwannoma vestibolare è un tumore raro della fossa posteriore che origina nelle cellule di Schwann dell'area transizionale vestibolare del nervo vestibolo-cocleare. Può essere benigno, di piccole dimensioni, a progressione lenta e asintomatico, oppure di grandi dimensioni, a progressione veloce, aggressivo e potenzialmente letale, con disturbi dell'udito e dell'equilibrio, vertigini, atassia, cefalea, disfunzione del quinto, sesto o settimo nervo cranico e intorpidimento facciale.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19756">
@@ -195681,7 +195803,7 @@
     <Disorder id="2396">
       <OrphaCode>2617</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2617</ExpertLink>
-      <Name lang="it">Nanismo a testa di uccello, tipo Montreal</Name>
+      <Name lang="it">Nanismo primitivo microcefalico tipo Montreal</Name>
       <DisorderFlagList count="2">
         <DisorderFlag id="460">
           <Value>512</Value>
@@ -195750,7 +195872,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122399" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159054" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il nanismo primitivo microcefalico tipo Montreal è una malattia genetica rara da anomalie/dismorfismi congeniti multipli, caratterizzata da statura notevolmente bassa e dismorfismi craniofacciali (microcefalia, volto stretto con guance appiattite, ptosi, naso prominente con cresta convessa, orecchie a basso impianto con lobi piccoli o assenti, palato ogivale o palatoschisi, micrognazia), associati a incanutimento precoce e perdita dei capelli, cute ridondante, secca e rugosa sul palmo delle mani, invecchiamento precoce e disabilità intellettiva di grado variabile. Possono associarsi criptorchidismo e anomalie scheletriche. Non sono stati descritti nuovi casi dopo il 1970.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19866">
@@ -202482,7 +202614,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122368" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158944" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sindrome radio-renale è un difetto genetico raro dello sviluppo dell'embrione, caratterizzato da difetti in riduzione degli arti superiori di gravità variabile e da anomalie renali. Di solito i pazienti presentano aplasia/ipoplasia delle dita, del radio e/o dell'ulna, bassa statura, malformazioni sfumate dell'orecchio esterno e agenesia o ectopia dei reni. Non sono stati descritti nuovi casi dopo il 1983.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20055">
@@ -209120,7 +209262,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Questo termine non si riferisce a una malattia ma a un gruppo di malattie. Per conoscere le malattie comprese all'interno di questo termine, è possibile consultare le classificazioni.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2614">
@@ -227278,7 +227427,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122381" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158987" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La polimicrogiria bilaterale è una malformazione cerebrale rara, causata da anomalie della migrazione neuronale, che consiste nella presenza di numerose circonvoluzioni di dimensioni molto ridotte a livello della corteccia cerebrale. I pazienti presentano ritardo dello sviluppo, disabilità intellettiva, crisi epilettiche e deficit neurologici variabili. La malformazione può essere isolata oppure rientrare nel quadro di numerose malattie genetiche. Può anche essere secondaria all'infezione perinatale da citomegalovirus.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20227">
@@ -235273,7 +235432,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122378" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158975" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sinostosi lambdoidea familiare è una malformazione genetica rara del cranio, caratterizzata da sinostosi mono- o bi-laterale della sutura lambdoidea in più membri della stessa famiglia. Di solito i pazienti affetti dalla forma monolaterale presentano una protuberanza occipitomastoidea omolaterale, una sporgenza parietale e frontale controlaterale compensatoria, la dislocazione di un orecchio, la deviazione laterale della mascella e una deformazione compensatoria del tratto cervicale della colonna. La forma bilaterale invece, in genere si manifesta con occipite piatto e allargato, dislocazione di entrambe le orecchie e, spesso, aumento della pressione intracranica.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20390">
@@ -235451,14 +235620,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="it">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="it">Questa malattia è descritta alla pagina  Sindrome di Coffin-Lowry</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2920">
@@ -252199,7 +252361,7 @@
     <Disorder id="3080">
       <OrphaCode>1277</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1277</ExpertLink>
-      <Name lang="it">Brachidattilia mesomalica - ritardo mentale - malformazioni cardiache</Name>
+      <Name lang="it">Sindrome da brachidattilia, mesomelia, disabilità intellettiva e cardiopatie</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -252232,7 +252394,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122388" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159010" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sindrome da brachidattilia, mesomelia, disabilità intellettiva e cardiopatie è una malattia genetica rara da anomalie/dismorfismi congeniti multipli, caratterizzata da ritardo dello sviluppo, disabilità intellettiva, habitus esile con spalle strette, accorciamento mesomelico delle braccia, dismorfismi craniofacciali (porzione inferiore del viso allungata, ipoplasia mascellare, naso a becco, columella corta, prognazia, palato ogivale, angolo mandibolare ottuso), brachidattilia (che interessa soprattutto le falangi medie) e anomalie cardiovascolari (dilatazione della radice dell'aorta, prolasso della valvola mitrale).</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18632">
@@ -264381,7 +264553,7 @@
     <Disorder id="18779">
       <OrphaCode>213812</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=213812</ExpertLink>
-      <Name lang="it">Tumore neuroectodermico periferico maligno della cervice uterina</Name>
+      <Name lang="it">Tumore neuroectodermico primitivo della cervice uterina</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -264442,7 +264614,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122374" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158959" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il tumore neuroectodermico primitivo della cervice uterina è una neoplasia rara che origina dalle cellule della cresta neurale. Dal punto di vista istologico, è composto da piccole cellule neoplastiche arrotondate con differenziazione neurale, gliale ed ependimale di grado variabile. Dal punto di vista macroscopico, spesso si presenta come una grossa massa soffice, scarsamente circoscritta, con bordi infiltranti e aree necrotiche. Esordisce con emorragie vaginali disfunzionali o perdite, dolore nella parte inferiore dell'addome e aumento di volume dell'utero.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="3482">
@@ -273865,7 +274047,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3338" cycle="true"/>
+          <RootDisorder id="20880">
+            <OrphaCode>293848</OrphaCode>
+            <Name lang="it">Demenza frontotemporale, variante dell'atrofia temporale destra</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="21471">
+            <Name lang="it">Spostato/a nella pagina</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="16004" lang="it">
@@ -285169,7 +285361,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Questo termine non si riferisce a una malattia ma a un gruppo di malattie. Per conoscere le malattie comprese all'interno di questo termine, è possibile consultare le classificazioni.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19020">
@@ -306417,7 +306616,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122373" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158955" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La palatoschisi e l'anchiloglossia legata all'X è un difetto genetico raro dello sviluppo dell'embrione, caratterizzato dalla combinazione tra palatoschisi completa, parziale o sottomucosa e anchiloglossia. I pazienti possono presentare anomalie dell'ugola (aplastica, bifida, accorciata o deviata lateralmente), frenulo linguale corto e anomalie dei denti (morso incrociato, agenesie/deformità dei denti). Sono stati descritti pazienti con anomalie delle dita (clinodattilia e/o sindattilia).</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21788">
@@ -308492,7 +308701,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122377" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158974" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La paraplegia spastica autosomica recessiva tipo 45 è una forma rara, pura o complessa, di paraplegia spastica ereditaria, che esordisce nel periodo neonatale con spasticità progressiva degli arti inferiori, anomalie dell'andatura, aumento dei riflessi tendinei profondi e risposte plantari in estensione, talvolta associati a disabilità intellettiva. Possono associarsi altri segni clinici, che comprendono le contratture negli arti inferiori, l'amiotrofia, i piedi torti e l'atrofia ottica.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21723">
@@ -313058,7 +313277,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122387" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159009" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'atassia cerebellare non progressiva legata all'X è un'atassia ereditaria rara, caratterizzata da ritardo precoce dello sviluppo motorio, grave ipotonia neonatale, atassia non progressiva e rallentamento dei movimenti oculari, abilità cognitive normali e assenza di segni piramidali. Spesso i pazienti presentano tremore intenzionale, lieve disfagia e disartria. La RMN del cervello evidenzia atrofia cerebellare generalizzata, in assenza di altre malformazioni o degenerazioni del sistema nervoso centrale e periferico.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21545">
@@ -326551,8 +326780,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="it">Sottotipo della malattia</Name>
@@ -327489,8 +327718,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="it">Sottotipo della malattia</Name>
@@ -327545,8 +327774,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="it">Sottotipo della malattia</Name>
@@ -330143,7 +330372,11 @@
       <OrphaCode>293848</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=293848</ExpertLink>
       <Name lang="it">Demenza frontotemporale, variante dell'atrofia temporale destra</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="2">
+        <DisorderFlag id="459">
+          <Value>256</Value>
+          <Label>Deprecated entity</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
@@ -330159,35 +330392,28 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="122990">
-          <Source>ICD-10</Source>
-          <Reference>G31.0</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="it">NTBT (il termine più ristretto dell'ORPHA code corrisponde a un termine più ampio)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="it">Attribuito (il codice ICD10 è attribuito da Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="141351">
-          <Source>OMIM</Source>
-          <Reference>600274</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="it">NTBT (il termine più ristretto dell'ORPHA code corrisponde a un termine più ampio)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3338">
+            <OrphaCode>282</OrphaCode>
+            <Name lang="it">Demenza frontotemporale</Name>
+          </TargetDisorder>
+          <RootDisorder id="20880" cycle="true"/>
+          <DisorderDisorderAssociationType id="21471">
+            <Name lang="it">Spostato/a nella pagina</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="it">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="it">Questa malattia è stata spostata nella pagina  Demenza frontotemporale</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20878">
@@ -344030,9 +344256,9 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=284408</ExpertLink>
       <Name lang="it">Deficit infantile di glicerolo chinasi</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="461">
+          <Value>1024</Value>
+          <Label>Obsolete with resources</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -344043,45 +344269,11 @@
       <DisorderGroup id="36554">
         <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="141326">
-          <Source>OMIM</Source>
-          <Reference>307030</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="it">NTBT (il termine più ristretto dell'ORPHA code corrisponde a un termine più ampio)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="122860">
-          <Source>ICD-10</Source>
-          <Reference>E74.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="it">NTBT (il termine più ristretto dell'ORPHA code corrisponde a un termine più ampio)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="it">Attribuito (il codice ICD10 è attribuito da Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="it">Confermato</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="87615" lang="it">
-          <TextSectionList count="1">
-            <TextSection id="99959" lang="it">
-              <TextSectionType id="16907">
-                <Name lang="it">Definizione</Name>
-              </TextSectionType>
-              <Contents>Il deficit infantile di glicerolo chinasi (GKD) è una forma grave di GKD (si veda questo termine) caratterizzata, dal punto di vista clinico, da scarsa alimentazione, ritardo della crescita, disidratazione con perdita di sali, vomito, pigmentazione addisoniana, ipotonia e turbe della coscienza. Alcuni pazienti presentano il complesso GKD associato all'ipoplasia surrenalica congenita e/o alla distrofia muscolare di Duchenne (DMD) (si vedano questi termini), con deficit cognitivo, dismorfismi facciali, anomalie dei genitali esterni, strabismo, convulsioni e letargia progressiva.</Contents>
-            </TextSection>
-          </TextSectionList>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20607">
@@ -346192,7 +346384,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122397" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159052" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sindrome da microduplicazione 2q23.1 è un'anomalia cromosomica rara, causata dalla duplicazione parziale del braccio lungo del cromosoma 2; è caratterizzata prevalentemente da ritardo dello sviluppo globale, ipotonia, tratti simil-autistici e problemi comportamentali. Spesso i pazienti presentano dismorfismi craniofacciali (sopracciglia arcuate, ipertelorismo, ptosi bilaterale, naso prominente, bocca grande, micro/retrognazia) e personalità socievole. In rari casi, sono presenti lievi anomalie delle dita (clinodattilia del quinto dito e alluci grandi e larghi).</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21465">
@@ -349049,7 +349251,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122401" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159056" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il parkinsonismo da cianuro è un'intossicazione rara che insorge nei soggetti sopravvissuti a un avvelenamento acuto da cianuro o a un'esposizione cronica a piccole dosi. Esordisce diverse settimane dopo l'esposizione acuta con i segni clinici progressivi caratteristici del parkinsonismo, che comprendono la bradicinesia, la rigidità, la distonia, l'ipomimia, la disartria ipocinetica, l'instabilità posturale e la retropulsione, in assenza di tremore posturale o a riposo. La RMN del cervello evidenzia lesioni bilaterali nel globo pallido, nel putamen posteriore, nella substantia nigra, nel nucleo subtalamico, nella corteccia temporale e occipitale e nel cervelletto.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21252">
@@ -372826,7 +373038,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122366" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158934" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sindrome da microdelezione 1p31p32 è un'anomalia cromosomica rara, causata dalla delezione parziale del braccio corto del cromosoma 1. È caratterizzata da ritardo dello sviluppo, agenesia/ipoplasia del corpo calloso e dismorfismi craniofacciali, che comprendono la macrocefalia (secondaria all'idrocefalo o alla ventricolomegalia), le orecchie a basso impianto, le narici anteverse e la micrognazia. Spesso i pazienti presentano difetti delle vie urinarie (reflusso vescico-ureterale, incontinenza urinaria). Altri segni clinici di gravità variabile sono l'ipotonia, il midollo ancorato, la malformazione di Chiari tipo I e le crisi epilettiche.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="22862">
@@ -394947,7 +395169,7 @@
     <Disorder id="23125">
       <OrphaCode>420789</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=420789</ExpertLink>
-      <Name lang="it">Encefalopatia autoimmune con parasonnia e apnea ostruttiva del sonno</Name>
+      <Name lang="it">Encefalopatia autoimmune con parasonnia e apnea ostruttiva durante il sonno</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -394979,7 +395201,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122383" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159005" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'encefalopatia autoimmune con parasonnia e apnea ostruttiva durante il sonno è una malattia neurologica rara caratterizzata da una peculiare parasonnia non-REM e REM, associata a disfunzione respiratoria durante il sonno, instabilità dell'andatura, episodi ripetuti di insufficienza respiratoria e presenza di autoanticorpi diretti contro IgLON5. I pazienti presentano stridore, corea, atassia degli arti, anomalie dei movimenti oculari e sintomi bulbari (disfagia, disartria, episodi di ipoventilazione centrale), con una RMN del cervello normale. Possono associarsi una marcata sonnolenza diurna e il deterioramento cognitivo.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23126">
@@ -398250,7 +398482,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122403" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159058" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il carcinoma a cellule squamose del colon (SCC) è una neoplasia epiteliale rara che origina dalle cellule squamose dell'epitelio del colon-retto, in assenza di tratti fistolosi rivestiti di cellule squamose o di una estensione prossimale di un SCC di origine anale. Spesso i sintomi sono aspecifici e comprendono l'anoressia, la perdita di peso, il dolore addominale, i cambiamenti delle abitudini intestinali, l'ematochezia o la melena. Sono stati descritti pazienti con ipercalcemia sintomatica grave.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23181">
@@ -399203,7 +399445,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122369" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158945" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>Il carcinoma a cellule squamose della cistifellea e delle vie biliari extraepatiche è una neoplasia rara del fegato e delle vie biliari, che origina dalla cistifellea o dall'epitelio che riveste l'albero biliare extraepatico, il dotto cistico e le ghiandole peribiliari. È caratterizzato da una marcata cheratinizzazione con abbondanti perle di cheratoialina, da depositi centrali di materiale cheratinico denso all'interno dei nidi delle cellule infiltranti e da una natura localmente aggressiva. Nelle prime fasi della malattia, i sintomi sono sfumati e aspecifici (dolore addominale, ittero e vomito). Negli stati più avanzati, la malattia può manifestarsi con una grossa massa e con sintomi correlati all'interessamento degli organi vicini.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23204">
@@ -421090,7 +421342,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122400" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159055" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'embrio/fetopatia da propiltiouracile è una malattia teratologica rara, caratterizzata da anomalie congenite di gravità variabile, indotte dal trattamento della madre o dall'esposizione del feto al propiltiouracile. Le anomalie più comuni comprendono le malformazioni dell'orecchio (orecchio soprannumerario, seno/fistola/cisti preauricolare) e dell'apparato urinario (agenesia isolata di un rene, idronefrosi congenita), le anomalie gastrointestinali (bande congenite con malrotazione dell'intestino) e le cardiopatie (destrocardia situs inversus, difetti del tratto di efflusso cardiaco).</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="25305">
@@ -421709,7 +421971,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122372" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158954" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La colangite sclerosante neonatale isolata è una malattia genetica rara delle vie biliari, caratterizzata da colangiopatia grave ad esordio neonatale con dotti biliari pervi e assenza di lesioni cutanee ittiosiformi. I pazienti presentano ittero, feci acoliche, epatosplenomegalia e un'elevata attività della gamma-glutamiltransferasi nel siero. L'esame istologico del fegato evidenzia fibrosi portale, proliferazione duttale, depositi epatocellulari di metallotioneina e accumuli intralobulari di pigmento biliare. Alcuni pazienti presentano una malattia renale.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="25218">
@@ -424371,7 +424643,7 @@
     <Disorder id="25553">
       <OrphaCode>493342</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=493342</ExpertLink>
-      <Name lang="it">Orticaria da vibrazioni</Name>
+      <Name lang="it">Orticaria da vibrazione</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -424401,7 +424673,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122391" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159030" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'orticaria da vibrazione è una forma genetica rara di orticaria, caratterizzata dalla presenza di un rash localizzato, di breve durata (che scompare nel giro di un'ora), pruriginoso, eritematoso ed edematoso, secondario a una stimolazione ripetitiva della cute dovuta allo sfregamento o alle vibrazioni. In alcuni pazienti si associa ad un arrossamento del viso, cefalea o sensazione di sapore metallico. Sono reperti caratteristici la degranulazione localizzata dei mastociti e l'innalzamento dei valori dell'istamina nel siero.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="25555">
@@ -449817,7 +450099,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122371" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158953" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La sindrome angio-osteoipotrofica è un'anomalia vascolare congenita rara, caratterizzata da malformazioni venose o, occasionalmente, arteriose, che esitano nell'ipertrofia dei tessuti molli e nell'ipoplasia dello scheletro. Di solito gli arti interessati sono più corti, marcatamente deformati, doloranti ed edematosi. Si associa a ipotrofia delle ossa e dei muscoli. In genere sono interessate singole porzioni degli arti, o più porzioni di ridotte dimensioni; tuttavia, alcuni pazienti presentano un interessamento più esteso, ad esempio di un intero arto, del cingolo della spalla e dell'ascella.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11087">
@@ -484502,7 +484794,7 @@
     <Disorder id="11822">
       <OrphaCode>88660</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=88660</ExpertLink>
-      <Name lang="it">Pseudoiperaldosteronismo, tipo 2</Name>
+      <Name lang="it">Ipertensione da mutazione con guadagno di funzione nel recettore dei mineralcorticoidi</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -484547,7 +484839,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122385" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159007" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'ipertensione da mutazioni con guadagno di funzione nel recettore dei mineralcorticoidi è una forma genetica rara di ipertensione, caratterizzata da ipertensione familiare grave che esordisce prima dei vent'anni, associata all'assenza di renina nel plasma e a livelli bassi o normali del mineralcorticoide aldosterone. La malattia è fortemente resistente ai farmaci antiipertensivi. Durante la gravidanza, l'ipertensione peggiora significativamente, in associazione con livelli bassi di potassio e non misurabili di aldosterone nel siero, in assenza di segni di preeclampsia che raccomandino un parto pretermine.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="27374">
@@ -538109,7 +538411,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122404" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159059" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La disomia uniparentale materna del cromosoma 4 è una disomia di origine materna che di solito non ha effetti sul fenotipo, ad eccezione dei casi di omozigosi per una mutazione responsabile di una malattia recessiva, di cui solo la madre è portatrice.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12803">
@@ -538148,7 +538460,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122386" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159008" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La disomia uniparentale materna del cromosoma 2 è una disomia di origine materna che di solito non ha effetti sul fenotipo, ad eccezione dei casi di omozigosi per una mutazione responsabile di una malattia recessiva, di cui solo la madre è portatrice.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12802">
@@ -543916,7 +544238,7 @@
     <Disorder id="12610">
       <OrphaCode>95486</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=95486</ExpertLink>
-      <Name lang="it">Chiusura prematura del dotto arterioso pervio</Name>
+      <Name lang="it">Chiusura prematura del dotto arterioso</Name>
       <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
@@ -543949,7 +544271,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122384" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159006" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La chiusura prematura del dotto arterioso è un'anomalia rara caratterizzata da un marcato restringimento o dalla chiusura del dotto arterioso fetale in assenza di altri difetti della struttura cardiaca. I segni patognomonici comprendono l'aumento del postcarico ventricolare destro, il rigurgito tricuspidale e, di conseguenza, la dilatazione dell'atrio destro e l'ipertrofia del ventricolo destro. La gravità dei sintomi correla con il grado e il tasso di restringimento del dotto, che varia dal lieve distress respiratorio subito dopo la nascita, all'insufficienza ventricolare con idrope fetale, alla morte intrauterina o alla grave compromissione cardiopolmonare dopo la nascita. Può essere secondaria all'esposizione prenatale agli inibitori della ciclossigenasi e ai corticosteroidi.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12620">
@@ -561455,7 +561787,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122370" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158946" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'anomalia di Neuhauser è un'anomalia rara della morfologia cardiovascolare, secondaria a un difetto dello sviluppo dell'aorta embrionale, che esita nella destro-posizione dell'arco aortico e nel posizionamento a sinistra del legamento arterioso, che causano i sintomi caratteristici della compressione tracheoesofagea (stridore, dispnea, disfagia, episodi di apnea, infezioni respiratorie ricorrenti). 01</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="14094">
@@ -562487,7 +562829,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122375" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="158964" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>L'agenesia della vena cava superiore (VCS) è un'anomalia congenita rara dei grandi vasi, caratterizzata dall'assenza, monolaterale o bilaterale, della VCS. L'agenesia monolaterale è per lo più asintomatica (di solito viene diagnosticata casualmente) e il resto della struttura cardiaca è normale. L'agenesia bilaterale invece si associa spesso ad altre cardiopatie congenite e/o ad anomalie della conduzione (tetralogia di Fallot, difetto del setto interatriale); i pazienti presentano i sintomi caratteristici della sindrome della VCS.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="14128">
@@ -563516,6 +563868,56 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29576">
+      <OrphaCode>592850</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592850</ExpertLink>
+      <Name lang="it">Neuromyelitis optica spectrum disorder with anti-AQP4 antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="29577">
+      <OrphaCode>592856</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592856</ExpertLink>
+      <Name lang="it">Neuromyelitis optica spectrum disorder with anti-MOG antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="14152">
       <OrphaCode>99135</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99135</ExpertLink>
@@ -563554,6 +563956,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29578">
+      <OrphaCode>592869</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592869</ExpertLink>
+      <Name lang="it">Neuromyelitis optica spectrum disorder without anti-MOG and without anti-AQP4 antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="14155">
       <OrphaCode>99138</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99138</ExpertLink>
@@ -563602,6 +564029,81 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29579">
+      <OrphaCode>592873</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592873</ExpertLink>
+      <Name lang="it">Acute transverse myelitis with anti-MOG antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="29580">
+      <OrphaCode>592885</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592885</ExpertLink>
+      <Name lang="it">Isolated optic neuritis without anti-MOG antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="29581">
+      <OrphaCode>592888</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592888</ExpertLink>
+      <Name lang="it">Optic neuritis with anti-MOG antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="14156">
       <OrphaCode>99139</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99139</ExpertLink>
@@ -563638,6 +564140,56 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29582">
+      <OrphaCode>592894</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592894</ExpertLink>
+      <Name lang="it">Acute disseminated encephalomyelitis with anti-MOG antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="29583">
+      <OrphaCode>592900</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592900</ExpertLink>
+      <Name lang="it">Acute disseminated encephalomyelitis without anti-MOG antibodies</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="14158">
       <OrphaCode>99141</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99141</ExpertLink>
@@ -565384,6 +565936,81 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29566">
+      <OrphaCode>592574</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592574</ExpertLink>
+      <Name lang="it">Menke-Hennekam syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592564</ExpertLink>
+      <Name lang="it">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592570</ExpertLink>
+      <Name lang="it">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="14267">
       <OrphaCode>99694</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99694</ExpertLink>
@@ -609943,7 +610570,17 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="122393" lang="it">
+          <TextSectionList count="1">
+            <TextSection id="159039" lang="it">
+              <TextSectionType id="16907">
+                <Name lang="it">Definizione</Name>
+              </TextSectionType>
+              <Contents>La malattia di Charcot-Marie-Tooth autosomica dominante intermedia tipo C è una neuropatia motoria e sensoriale ereditaria rara, caratterizzata da velocità di conduzione intermedie del nervo mediano (di solito comprese tra 25 e 60 m/s). La malattia esordisce con i segni clinici caratteristici della malattia di Charcot-Marie-Tooth, moderatamente gravi e a progressione lenta (debolezza muscolare e atrofia della porzione distale delle estremità, perdita sensoriale distale, riduzione o assenza dei riflessi tendinei profondi, deformità dei piedi, atrofia dell'estensore breve delle dita). La biopsia dei nervi evidenzia una degenerazione assonale che correla con l'età, una riduzione del numero di fibre larghe demielinizzate, una rimielinizzazione segmentale e l'assenza dei ''bulbi di cipolla''.</Contents>
+            </TextSection>
+          </TextSectionList>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="14619">
