commit c92e9c5f9940abcb2c7950377e1c2245059cad90
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 1 10:54:09 2021 +0100

    MARCH 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/nl_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/nl_product1.xml
old mode 100755
new mode 100644
index 22d6ae8..1e0dce3
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/nl_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/nl_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-02-01 04:31:10" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-03-01 04:31:16" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10521">
+  <DisorderList count="10524">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -802,7 +802,11 @@
       <OrphaCode>166068</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166068</ExpertLink>
       <Name lang="nl">Pontocerebellaire hypoplasie type 5</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="2">
+        <DisorderFlag id="459">
+          <Value>256</Value>
+          <Label>Deprecated entity</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
@@ -818,7 +822,7 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="3">
+      <ExternalReferenceList count="2">
         <ExternalReference id="139338">
           <Source>UMLS</Source>
           <Reference>C1857762</Reference>
@@ -830,19 +834,6 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="120425">
-          <Source>ICD-10</Source>
-          <Reference>Q04.3</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
         <ExternalReference id="38970">
           <Source>OMIM</Source>
           <Reference>610204</Reference>
@@ -855,9 +846,26 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="17608">
+            <OrphaCode>166063</OrphaCode>
+            <Name lang="nl">Pontocerebellaire hypoplasie type 4</Name>
+          </TargetDisorder>
+          <RootDisorder id="17609" cycle="true"/>
+          <DisorderDisorderAssociationType id="21471">
+            <Name lang="nl">Verplaatst naar</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="nl">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="nl">Deze ziekte is verplaatst naar  Pontocerebellaire hypoplasie type 4</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="8">
@@ -1036,7 +1044,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="17608" cycle="true"/>
+          <RootDisorder id="17609">
+            <OrphaCode>166068</OrphaCode>
+            <Name lang="nl">Pontocerebellaire hypoplasie type 5</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="21471">
+            <Name lang="nl">Verplaatst naar</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="105049" lang="nl">
@@ -2424,17 +2442,17 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178497">
+        <ExternalReference id="187716">
           <Source>ICD-10</Source>
-          <Reference>Q77.7</Reference>
+          <Reference>Q87.0</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -7305,8 +7323,8 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="120376">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="187714">
           <Source>ICD-10</Source>
           <Reference>G04.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -7315,19 +7333,6 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="120375">
-          <Source>ICD-10</Source>
-          <Reference>G13.1</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
           <DisorderMappingValidationStatus id="21611">
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
@@ -8457,17 +8462,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="120378">
+        <ExternalReference id="187715">
           <Source>ICD-10</Source>
-          <Reference>A86</Reference>
+          <Reference>G04.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -17113,7 +17118,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="120364">
+        <ExternalReference id="187713">
           <Source>ICD-10</Source>
           <Reference>G40.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -17122,8 +17127,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -36128,7 +36133,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179636">
+        <ExternalReference id="187697">
           <Source>ICD-10</Source>
           <Reference>Q04.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -36137,8 +36142,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -48204,17 +48209,17 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178090">
+        <ExternalReference id="187718">
           <Source>ICD-10</Source>
           <Reference>D24</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
+          <DisorderMappingRelation id="21534">
+            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -57312,7 +57317,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="153085">
+        <ExternalReference id="187717">
           <Source>ICD-10</Source>
           <Reference>C84.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -57321,8 +57326,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -59982,7 +59987,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="1">
+      <DisorderDisorderAssociationList count="3">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641" cycle="true"/>
+          <RootDisorder id="11904">
+            <OrphaCode>89839</OrphaCode>
+            <Name lang="nl">Epidermolysis bullosa simplex superficialis</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
         <DisorderDisorderAssociation>
           <TargetDisorder id="641" cycle="true"/>
           <RootDisorder id="17230">
@@ -59993,6 +60008,16 @@
             <Name lang="nl">Referred to</Name>
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641" cycle="true"/>
+          <RootDisorder id="17231">
+            <OrphaCode>158665</OrphaCode>
+            <Name lang="nl">Basale epidermolysis bullosa simplex</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="72314" lang="nl">
@@ -81662,18 +81687,70 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="5">
-        <ExternalReference id="179637">
+      <ExternalReferenceList count="9">
+        <ExternalReference id="187700">
           <Source>ICD-10</Source>
-          <Reference>A16.9</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
+          <Reference>A17</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="nl">BTNT (Bredere Term van ORPHA-code mapt naar een Nauwere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187699">
+          <Source>ICD-10</Source>
+          <Reference>A16</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="nl">BTNT (Bredere Term van ORPHA-code mapt naar een Nauwere Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="nl">null</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187701">
+          <Source>ICD-10</Source>
+          <Reference>A18</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="nl">BTNT (Bredere Term van ORPHA-code mapt naar een Nauwere Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="nl">null</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187702">
+          <Source>ICD-10</Source>
+          <Reference>A19</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="nl">BTNT (Bredere Term van ORPHA-code mapt naar een Nauwere Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="nl">null</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187698">
+          <Source>ICD-10</Source>
+          <Reference>A15</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="nl">BTNT (Bredere Term van ORPHA-code mapt naar een Nauwere Term)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="nl">null</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="107348">
@@ -108710,7 +108787,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="16890" cycle="true"/>
+          <RootDisorder id="16891">
+            <OrphaCode>139420</OrphaCode>
+            <Name lang="nl">Secundaire acute transversale myelitis</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="117802" lang="nl">
@@ -108833,9 +108920,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139420</ExpertLink>
       <Name lang="nl">Secundaire acute transversale myelitis</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
@@ -108849,7 +108936,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="16890">
+            <OrphaCode>139417</OrphaCode>
+            <Name lang="nl">Acute transversale myelitis</Name>
+          </TargetDisorder>
+          <RootDisorder id="16891" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -113667,7 +113764,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="171777">
+        <ExternalReference id="187712">
           <Source>ICD-10</Source>
           <Reference>P91.6</Reference>
           <DisorderMappingRelation id="21527">
@@ -113676,8 +113773,8 @@
           <DisorderMappingICDRelation id="21583">
             <Name lang="nl">Inclusieterm (De ORPHA-code is opgenomen in een ICD10-categorie en heeft geen eigen code)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="138811">
@@ -157066,9 +157163,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158665</ExpertLink>
       <Name lang="nl">Basale epidermolysis bullosa simplex</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -157081,7 +157178,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641">
+            <OrphaCode>304</OrphaCode>
+            <Name lang="nl">Epidermolysis bullosa simplex</Name>
+          </TargetDisorder>
+          <RootDisorder id="17231" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="-1" lang="nl">
@@ -165763,17 +165870,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179646">
+        <ExternalReference id="187721">
           <Source>ICD-10</Source>
-          <Reference>G11.2</Reference>
+          <Reference>G31.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -167862,7 +167969,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="124984">
+        <ExternalReference id="187723">
           <Source>ICD-10</Source>
           <Reference>C71.9</Reference>
           <DisorderMappingRelation id="21534">
@@ -167871,8 +167978,8 @@
           <DisorderMappingICDRelation id="21597">
             <Name lang="nl">Toegekend (De ICD10-code is toegekend door Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -173679,17 +173786,17 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179638">
+        <ExternalReference id="187703">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="140403">
@@ -179269,7 +179376,18 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="121940">
+        <ExternalReference id="50926">
+          <Source>OMIM</Source>
+          <Reference>604391</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation/>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187722">
           <Source>ICD-10</Source>
           <Reference>G11.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -179278,17 +179396,6 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="50926">
-          <Source>OMIM</Source>
-          <Reference>604391</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
           <DisorderMappingValidationStatus id="21611">
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
@@ -185135,7 +185242,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179640">
+        <ExternalReference id="187705">
           <Source>ICD-10</Source>
           <Reference>Q04.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -185144,8 +185251,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -199047,17 +199154,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="5">
-        <ExternalReference id="179639">
+        <ExternalReference id="187704">
           <Source>ICD-10</Source>
-          <Reference>G11.0</Reference>
+          <Reference>G11.1</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="110119">
@@ -217529,7 +217636,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179641">
+        <ExternalReference id="187706">
           <Source>ICD-10</Source>
           <Reference>Q68.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -217538,8 +217645,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="47190">
@@ -218576,7 +218683,18 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="122447">
+        <ExternalReference id="94944">
+          <Source>OMIM</Source>
+          <Reference>614470</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation/>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187724">
           <Source>ICD-10</Source>
           <Reference>D72.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -218585,17 +218703,6 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="94944">
-          <Source>OMIM</Source>
-          <Reference>614470</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
           <DisorderMappingValidationStatus id="21611">
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
@@ -219338,8 +219445,8 @@
         <Synonym lang="nl">Del(21)(q22.13q22.2)</Synonym>
         <Synonym lang="nl">Monosomie 21q22.13q22.2</Synonym>
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="nl">Subtype van aandoening</Name>
@@ -226639,7 +226746,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179648">
+        <ExternalReference id="187725">
           <Source>ICD-10</Source>
           <Reference>K76.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -226648,8 +226755,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -237527,7 +237634,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179649">
+        <ExternalReference id="187726">
           <Source>ICD-10</Source>
           <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -237536,8 +237643,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -248680,17 +248787,17 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179642">
+        <ExternalReference id="187707">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>Q68.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="95412">
@@ -269149,17 +269256,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="153082">
+        <ExternalReference id="187719">
           <Source>ICD-10</Source>
-          <Reference>C54.9</Reference>
+          <Reference>C54.1</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -276057,7 +276164,27 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="2">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423" cycle="true"/>
+          <RootDisorder id="21237">
+            <OrphaCode>306588</OrphaCode>
+            <Name lang="nl">Autosomaal dominante Opitz G/BBB-syndroom</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423" cycle="true"/>
+          <RootDisorder id="21238">
+            <OrphaCode>306597</OrphaCode>
+            <Name lang="nl">X-gebonden Opitz G/BBB-syndroom</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="86706" lang="nl">
@@ -277069,17 +277196,17 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179643">
+        <ExternalReference id="187708">
           <Source>ICD-10</Source>
-          <Reference>G11.0</Reference>
+          <Reference>G11.1</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="7363">
@@ -278112,17 +278239,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="121474">
+        <ExternalReference id="187720">
           <Source>ICD-10</Source>
-          <Reference>G90.8</Reference>
+          <Reference>G60.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -286935,29 +287062,7 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="69442">
-          <Source>OMIM</Source>
-          <Reference>610042</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="69443">
-          <Source>OMIM</Source>
-          <Reference>614325</Reference>
-          <DisorderMappingRelation id="21541">
-            <Name lang="nl">BTNT (Bredere Term van ORPHA-code mapt naar een Nauwere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -330432,20 +330537,7 @@
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="123904">
-          <Source>ICD-10</Source>
-          <Reference>Q93.2</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -337732,20 +337824,7 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="123000">
-          <Source>ICD-10</Source>
-          <Reference>Q74.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -338267,17 +338346,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179650">
+        <ExternalReference id="187727">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>G96.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="61792">
@@ -355760,11 +355839,11 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
         <ExternalReference id="98695">
@@ -355808,11 +355887,11 @@
         <Synonym lang="nl">Del(12)(p12.1)</Synonym>
         <Synonym lang="nl">Monosomie 12p12.1</Synonym>
       </SynonymList>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
         <ExternalReference id="123514">
@@ -357467,17 +357546,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179651">
+        <ExternalReference id="187729">
           <Source>ICD-10</Source>
-          <Reference>G11.0</Reference>
+          <Reference>G11.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="74934">
@@ -367277,9 +367356,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=306597</ExpertLink>
       <Name lang="nl">X-gebonden Opitz G/BBB-syndroom</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
@@ -367293,53 +367372,21 @@
       <DisorderGroup id="36554">
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="3">
-        <ExternalReference id="140146">
-          <Source>UMLS</Source>
-          <Reference>C2936904</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="123287">
-          <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="71929">
-          <Source>OMIM</Source>
-          <Reference>300000</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423">
+            <OrphaCode>2745</OrphaCode>
+            <Name lang="nl">Opitz G/BBB-syndroom</Name>
+          </TargetDisorder>
+          <RootDisorder id="21238" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="nl">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="nl">Deze ziekte is beschreven op  Opitz G/BBB-syndroom</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21237">
@@ -367347,9 +367394,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=306588</ExpertLink>
       <Name lang="nl">Autosomaal dominante Opitz G/BBB-syndroom</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="3">
@@ -367363,42 +367410,21 @@
       <DisorderGroup id="36554">
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="123286">
-          <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="71927">
-          <Source>OMIM</Source>
-          <Reference>145410</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423">
+            <OrphaCode>2745</OrphaCode>
+            <Name lang="nl">Opitz G/BBB-syndroom</Name>
+          </TargetDisorder>
+          <RootDisorder id="21237" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="nl">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="nl">Deze ziekte is beschreven op  Opitz G/BBB-syndroom</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21236">
@@ -370856,7 +370882,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="153086">
+        <ExternalReference id="187728">
           <Source>ICD-10</Source>
           <Reference>D72.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -370865,8 +370891,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -372193,8 +372219,8 @@
       <SynonymList count="1">
         <Synonym lang="nl">DYRK1A-gerelateerd intellectuele achterstandsyndroom door een puntmutatie</Synonym>
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="nl">Subtype van aandoening</Name>
@@ -372203,7 +372229,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="nl">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="nl">Deze ziekte is beschreven op  DYRK1A-gerelateerde intellectuele achterstand-syndroom</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="24023">
@@ -418669,17 +418702,17 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="155137">
+        <ExternalReference id="187730">
           <Source>ICD-10</Source>
-          <Reference>L51.8</Reference>
+          <Reference>L51.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -442211,20 +442244,7 @@
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="179633">
-          <Source>ICD-10</Source>
-          <Reference>Q43.1</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="nl">null</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -443515,17 +443535,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="7">
-        <ExternalReference id="179645">
+        <ExternalReference id="187710">
           <Source>ICD-10</Source>
-          <Reference>Q82.2</Reference>
+          <Reference>Q82.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="10352">
@@ -449469,11 +449489,7 @@
       <OrphaCode>63269</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63269</ExpertLink>
       <Name lang="nl">Syndroom van Antley-Bixler met genitale anomalie en stoornis van steroïdogenese</Name>
-      <DisorderFlagList count="2">
-        <DisorderFlag id="459">
-          <Value>256</Value>
-          <Label>Deprecated entity</Label>
-        </DisorderFlag>
+      <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
@@ -449503,26 +449519,9 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12654">
-            <OrphaCode>95699</OrphaCode>
-            <Name lang="nl">Congenitale adrenale hyperplasie als gevolg van cytochroom P450-oxidoreductasedeficiëntie</Name>
-          </TargetDisorder>
-          <RootDisorder id="10819" cycle="true"/>
-          <DisorderDisorderAssociationType id="21471">
-            <Name lang="nl">Verplaatst naar</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
+      <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="nl">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="nl">Deze ziekte is verplaatst naar  Congenitale adrenale hyperplasie als gevolg van cytochroom P450-oxidoreductasedeficiëntie</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10816">
@@ -477265,7 +477264,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179599">
+        <ExternalReference id="187736">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -477274,8 +477273,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -477315,7 +477314,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179598">
+        <ExternalReference id="187735">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -477324,8 +477323,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -477378,7 +477377,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179597">
+        <ExternalReference id="187734">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -477387,8 +477386,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -477430,7 +477429,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179601">
+        <ExternalReference id="187738">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -477439,8 +477438,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -477493,7 +477492,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179600">
+        <ExternalReference id="187737">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -477502,8 +477501,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -479559,7 +479558,7 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179608">
+        <ExternalReference id="187739">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -479568,8 +479567,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -480955,7 +480954,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179644">
+        <ExternalReference id="187709">
           <Source>ICD-10</Source>
           <Reference>Q68.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -480964,8 +480963,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="139311">
@@ -481005,7 +481004,7 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179611">
+        <ExternalReference id="187742">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -481014,8 +481013,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -481161,7 +481160,7 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179610">
+        <ExternalReference id="187741">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -481170,8 +481169,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -481209,7 +481208,7 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179609">
+        <ExternalReference id="187740">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -481218,8 +481217,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -485168,17 +485167,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179616">
+        <ExternalReference id="187743">
           <Source>ICD-10</Source>
           <Reference>L92.1</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
+          <DisorderMappingRelation id="21527">
+            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
           </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21597">
-            <Name lang="nl">Toegekend (De ICD10-code is toegekend door Orphanet)</Name>
+          <DisorderMappingICDRelation id="21583">
+            <Name lang="nl">Inclusieterm (De ORPHA-code is opgenomen in een ICD10-categorie en heeft geen eigen code)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -485325,17 +485324,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179617">
+        <ExternalReference id="187744">
           <Source>ICD-10</Source>
           <Reference>L95.0</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
+          <DisorderMappingRelation id="21527">
+            <Name lang="nl">exacte overeenkomst (de termen en concepten zijn equivalent)</Name>
           </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21597">
-            <Name lang="nl">Toegekend (De ICD10-code is toegekend door Orphanet)</Name>
+          <DisorderMappingICDRelation id="21583">
+            <Name lang="nl">Inclusieterm (De ORPHA-code is opgenomen in een ICD10-categorie en heeft geen eigen code)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -488911,17 +488910,17 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179625">
+        <ExternalReference id="187745">
           <Source>ICD-10</Source>
-          <Reference>K76.8</Reference>
+          <Reference>K75.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -489179,17 +489178,17 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179626">
+        <ExternalReference id="187746">
           <Source>ICD-10</Source>
-          <Reference>K76.8</Reference>
+          <Reference>K75.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -489350,17 +489349,17 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179628">
+        <ExternalReference id="187748">
           <Source>ICD-10</Source>
-          <Reference>Q22</Reference>
+          <Reference>Q23.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163329">
@@ -489613,7 +489612,7 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179629">
+        <ExternalReference id="187749">
           <Source>ICD-10</Source>
           <Reference>E27.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -489622,8 +489621,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -489678,7 +489677,7 @@
         <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179630">
+        <ExternalReference id="187750">
           <Source>ICD-10</Source>
           <Reference>E27.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -489687,8 +489686,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -493288,7 +493287,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179171">
+        <ExternalReference id="187747">
           <Source>ICD-10</Source>
           <Reference>K75.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -494441,7 +494440,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11907" cycle="true"/>
+          <RootDisorder id="11906">
+            <OrphaCode>89841</OrphaCode>
+            <Name lang="nl">Centripetalis recessieve dystrofische epidermolysis bullosa</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="104731" lang="nl">
@@ -494521,9 +494530,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89841</ExpertLink>
       <Name lang="nl">Centripetalis recessieve dystrofische epidermolysis bullosa</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="5">
@@ -494541,7 +494550,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11907">
+            <OrphaCode>89842</OrphaCode>
+            <Name lang="nl">Recessieve dystrofische epidermolysis bullosa, gegeneraliseerd intermediair</Name>
+          </TargetDisorder>
+          <RootDisorder id="11906" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -494551,9 +494570,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89840</ExpertLink>
       <Name lang="nl">Junctionele epidermolysis bullosa, niet-Herlitz-type</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
@@ -494567,7 +494586,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11428">
+            <OrphaCode>79402</OrphaCode>
+            <Name lang="nl">Junctionele epidermolysis bullosa, gegeneraliseerd intermediair</Name>
+          </TargetDisorder>
+          <RootDisorder id="11905" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -494577,9 +494606,9 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89839</ExpertLink>
       <Name lang="nl">Epidermolysis bullosa simplex superficialis</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
@@ -494593,7 +494622,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641">
+            <OrphaCode>304</OrphaCode>
+            <Name lang="nl">Epidermolysis bullosa simplex</Name>
+          </TargetDisorder>
+          <RootDisorder id="11904" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -506460,7 +506499,7 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179675">
+        <ExternalReference id="187733">
           <Source>ICD-10</Source>
           <Reference>E75.2</Reference>
           <DisorderMappingRelation id="21534">
@@ -506469,8 +506508,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -508912,7 +508951,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179674">
+        <ExternalReference id="187732">
           <Source>ICD-10</Source>
           <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -508921,8 +508960,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -521482,7 +521521,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11428" cycle="true"/>
+          <RootDisorder id="11905">
+            <OrphaCode>89840</OrphaCode>
+            <Name lang="nl">Junctionele epidermolysis bullosa, niet-Herlitz-type</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="nl">Referred to</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -537467,7 +537516,7 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179673">
+        <ExternalReference id="187731">
           <Source>ICD-10</Source>
           <Reference>G11.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -537476,8 +537525,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -558366,17 +558415,7 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12654" cycle="true"/>
-          <RootDisorder id="10819">
-            <OrphaCode>63269</OrphaCode>
-            <Name lang="nl">Syndroom van Antley-Bixler met genitale anomalie en stoornis van steroïdogenese</Name>
-          </RootDisorder>
-          <DisorderDisorderAssociationType id="21471">
-            <Name lang="nl">Verplaatst naar</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
+      <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -563023,17 +563062,17 @@
             <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="117905">
+        <ExternalReference id="187711">
           <Source>ICD-10</Source>
-          <Reference>Q93.5</Reference>
+          <Reference>Q87.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="nl">null</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="nl">Nog niet gevalideerd</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="nl">Gevalideerd</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -572163,18 +572202,7 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="52010">
-          <Source>OMIM</Source>
-          <Reference>309580</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="nl">NTBT (Nauwere Term van ORPHA-code mapt naar een Bredere Term)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="nl">Gevalideerd</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -575151,6 +575179,38 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29775">
+      <OrphaCode>596426</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=596426</ExpertLink>
+      <Name lang="nl">Syndromes of reduced sensitivity to thyroid hormone</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21436">
+        <Name lang="nl">Group of phenomes</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="nl">Groep van aandoeningen</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="nl">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="nl">Deze term is niet kenmerkend voor één ziekte maar voor een groep van ziekten. Om meer te leren over de ziekten die overeenkomen met deze term, kan u de classificatie raadplegen.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12431">
       <OrphaCode>93598</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93598</ExpertLink>
@@ -575320,6 +575380,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29762">
+      <OrphaCode>596008</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=596008</ExpertLink>
+      <Name lang="nl">Antley-Bixler syndrome without genital anomaly or disorder of steroidogenesis</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12419">
       <OrphaCode>93579</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93579</ExpertLink>
@@ -576324,6 +576409,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29776">
+      <OrphaCode>596448</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=596448</ExpertLink>
+      <Name lang="nl">IgG4-related systemic disease</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12433">
       <OrphaCode>93600</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93600</ExpertLink>
@@ -608650,8 +608760,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+      <DisorderType id="21436">
+        <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -608745,7 +608855,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="nl">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="nl">Deze term is niet kenmerkend voor één ziekte maar voor een groep van ziekten. Om meer te leren over de ziekten die overeenkomen met deze term, kan u de classificatie raadplegen.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28857">
