commit c92e9c5f9940abcb2c7950377e1c2245059cad90
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 1 10:54:09 2021 +0100

    MARCH 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml
old mode 100755
new mode 100644
index 5d8bdc3..a380608
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-02-01 04:31:51" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-03-01 04:31:56" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10521">
+  <DisorderList count="10524">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -722,7 +722,11 @@
       <OrphaCode>166068</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166068</ExpertLink>
       <Name lang="pt">Hipoplasia olivopontocerebelosa, tipo 5</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="2">
+        <DisorderFlag id="459">
+          <Value>256</Value>
+          <Label>Deprecated entity</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
@@ -736,7 +740,7 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="3">
+      <ExternalReferenceList count="2">
         <ExternalReference id="139338">
           <Source>UMLS</Source>
           <Reference>C1857762</Reference>
@@ -748,19 +752,6 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="120425">
-          <Source>ICD-10</Source>
-          <Reference>Q04.3</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
         <ExternalReference id="38970">
           <Source>OMIM</Source>
           <Reference>610204</Reference>
@@ -773,9 +764,26 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="17608">
+            <OrphaCode>166063</OrphaCode>
+            <Name lang="pt">Hipoplasia pontocerebelosa tipo 4</Name>
+          </TargetDisorder>
+          <RootDisorder id="17609" cycle="true"/>
+          <DisorderDisorderAssociationType id="21471">
+            <Name lang="pt">Movido para</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Esta doença foi movida para  Hipoplasia pontocerebelosa tipo 4</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="8">
@@ -950,7 +958,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="17608" cycle="true"/>
+          <RootDisorder id="17609">
+            <OrphaCode>166068</OrphaCode>
+            <Name lang="pt">Hipoplasia olivopontocerebelosa, tipo 5</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="21471">
+            <Name lang="pt">Movido para</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -2306,17 +2324,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178497">
+        <ExternalReference id="187716">
           <Source>ICD-10</Source>
-          <Reference>Q77.7</Reference>
+          <Reference>Q87.0</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -6796,8 +6814,8 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="120376">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="187714">
           <Source>ICD-10</Source>
           <Reference>G04.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -6806,19 +6824,6 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="120375">
-          <Source>ICD-10</Source>
-          <Reference>G13.1</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
           <DisorderMappingValidationStatus id="21611">
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
@@ -7858,17 +7863,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="120378">
+        <ExternalReference id="187715">
           <Source>ICD-10</Source>
-          <Reference>A86</Reference>
+          <Reference>G04.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -16018,7 +16023,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="120364">
+        <ExternalReference id="187713">
           <Source>ICD-10</Source>
           <Reference>G40.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -16027,8 +16032,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -33712,7 +33717,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179636">
+        <ExternalReference id="187697">
           <Source>ICD-10</Source>
           <Reference>Q04.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -33721,8 +33726,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -44877,17 +44882,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178090">
+        <ExternalReference id="187718">
           <Source>ICD-10</Source>
           <Reference>D24</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -53292,7 +53297,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="153085">
+        <ExternalReference id="187717">
           <Source>ICD-10</Source>
           <Reference>C84.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -53301,8 +53306,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -55857,7 +55862,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="1">
+      <DisorderDisorderAssociationList count="3">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641" cycle="true"/>
+          <RootDisorder id="11904">
+            <OrphaCode>89839</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa simples superficial</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
         <DisorderDisorderAssociation>
           <TargetDisorder id="641" cycle="true"/>
           <RootDisorder id="17230">
@@ -55868,6 +55883,16 @@
             <Name lang="pt">Referente a</Name>
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641" cycle="true"/>
+          <RootDisorder id="17231">
+            <OrphaCode>158665</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa simples basal</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="85539" lang="pt">
@@ -76386,18 +76411,70 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="5">
-        <ExternalReference id="179637">
+      <ExternalReferenceList count="9">
+        <ExternalReference id="187700">
           <Source>ICD-10</Source>
-          <Reference>A16.9</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          <Reference>A17</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="pt">BTBT (O Termo Abrangente do código ORPHA relaciona-se com o Termo Detalhado)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187699">
+          <Source>ICD-10</Source>
+          <Reference>A16</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="pt">BTBT (O Termo Abrangente do código ORPHA relaciona-se com o Termo Detalhado)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187701">
+          <Source>ICD-10</Source>
+          <Reference>A18</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="pt">BTBT (O Termo Abrangente do código ORPHA relaciona-se com o Termo Detalhado)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187702">
+          <Source>ICD-10</Source>
+          <Reference>A19</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="pt">BTBT (O Termo Abrangente do código ORPHA relaciona-se com o Termo Detalhado)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187698">
+          <Source>ICD-10</Source>
+          <Reference>A15</Reference>
+          <DisorderMappingRelation id="21541">
+            <Name lang="pt">BTBT (O Termo Abrangente do código ORPHA relaciona-se com o Termo Detalhado)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="107348">
@@ -101753,7 +101830,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="16890" cycle="true"/>
+          <RootDisorder id="16891">
+            <OrphaCode>139420</OrphaCode>
+            <Name lang="pt">Mielite transversa, aguda, secundária</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="65938" lang="pt">
@@ -101865,9 +101952,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=139420</ExpertLink>
       <Name lang="pt">Mielite transversa, aguda, secundária</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -101880,7 +101967,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="16890">
+            <OrphaCode>139417</OrphaCode>
+            <Name lang="pt">Mielite transversa, aguda</Name>
+          </TargetDisorder>
+          <RootDisorder id="16891" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -106400,7 +106497,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="171777">
+        <ExternalReference id="187712">
           <Source>ICD-10</Source>
           <Reference>P91.6</Reference>
           <DisorderMappingRelation id="21527">
@@ -106409,8 +106506,8 @@
           <DisorderMappingICDRelation id="21583">
             <Name lang="pt">Código específico (O código ORPHA tem o seu código próprio na classificação ICD10)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="138811">
@@ -146883,9 +146980,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=158665</ExpertLink>
       <Name lang="pt">Epidermólise bolhosa simples basal</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -146898,7 +146995,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641">
+            <OrphaCode>304</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa simples</Name>
+          </TargetDisorder>
+          <RootDisorder id="17231" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="-1" lang="pt">
@@ -154882,17 +154989,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179646">
+        <ExternalReference id="187721">
           <Source>ICD-10</Source>
-          <Reference>G11.2</Reference>
+          <Reference>G31.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -156835,7 +156942,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="124984">
+        <ExternalReference id="187723">
           <Source>ICD-10</Source>
           <Reference>C71.9</Reference>
           <DisorderMappingRelation id="21534">
@@ -156844,8 +156951,8 @@
           <DisorderMappingICDRelation id="21597">
             <Name lang="pt">Termo index (O código ORPHA está listado no index ICD10)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -162164,17 +162271,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179638">
+        <ExternalReference id="187703">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="140403">
@@ -167388,7 +167495,18 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="121940">
+        <ExternalReference id="50926">
+          <Source>OMIM</Source>
+          <Reference>604391</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation/>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187722">
           <Source>ICD-10</Source>
           <Reference>G11.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -167397,17 +167515,6 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="50926">
-          <Source>OMIM</Source>
-          <Reference>604391</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
           <DisorderMappingValidationStatus id="21611">
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
@@ -172894,7 +173001,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179640">
+        <ExternalReference id="187705">
           <Source>ICD-10</Source>
           <Reference>Q04.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -172903,8 +173010,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -186118,17 +186225,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="5">
-        <ExternalReference id="179639">
+        <ExternalReference id="187704">
           <Source>ICD-10</Source>
-          <Reference>G11.0</Reference>
+          <Reference>G11.1</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="110119">
@@ -203295,7 +203402,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179641">
+        <ExternalReference id="187706">
           <Source>ICD-10</Source>
           <Reference>Q68.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -203304,8 +203411,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="47190">
@@ -204316,7 +204423,18 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="122447">
+        <ExternalReference id="94944">
+          <Source>OMIM</Source>
+          <Reference>614470</Reference>
+          <DisorderMappingRelation id="21527">
+            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation/>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
+        <ExternalReference id="187724">
           <Source>ICD-10</Source>
           <Reference>D72.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -204325,17 +204443,6 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="94944">
-          <Source>OMIM</Source>
-          <Reference>614470</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
           <DisorderMappingValidationStatus id="21611">
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
@@ -204982,8 +205089,8 @@
         <Synonym lang="pt">Monossomia 21q22.13q22.2</Synonym>
         <Synonym lang="pt">Síndrome de microdeleção 21q22.13-q22.2</Synonym>
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="pt">Subtipo clínico</Name>
+      <DisorderType id="21443">
+        <Name lang="pt">Subtipo etiologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="pt">Subtipo de patologia</Name>
@@ -211849,7 +211956,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179648">
+        <ExternalReference id="187725">
           <Source>ICD-10</Source>
           <Reference>K76.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -211858,8 +211965,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -222150,7 +222257,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179649">
+        <ExternalReference id="187726">
           <Source>ICD-10</Source>
           <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -222159,8 +222266,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -232407,17 +232514,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179642">
+        <ExternalReference id="187707">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>Q68.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="95412">
@@ -251696,17 +251803,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="153082">
+        <ExternalReference id="187719">
           <Source>ICD-10</Source>
-          <Reference>C54.9</Reference>
+          <Reference>C54.1</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -258055,7 +258162,27 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="2">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423" cycle="true"/>
+          <RootDisorder id="21237">
+            <OrphaCode>306588</OrphaCode>
+            <Name lang="pt">Síndrome de OpitzG/BBB, autossómica dominante</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423" cycle="true"/>
+          <RootDisorder id="21238">
+            <OrphaCode>306597</OrphaCode>
+            <Name lang="pt">Síndrome de OpitzG /BBB, ligada ao cromossoma X</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -259005,17 +259132,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179643">
+        <ExternalReference id="187708">
           <Source>ICD-10</Source>
-          <Reference>G11.0</Reference>
+          <Reference>G11.1</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="7363">
@@ -259981,17 +260108,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="121474">
+        <ExternalReference id="187720">
           <Source>ICD-10</Source>
-          <Reference>G90.8</Reference>
+          <Reference>G60.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -268299,29 +268426,7 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="69442">
-          <Source>OMIM</Source>
-          <Reference>610042</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="69443">
-          <Source>OMIM</Source>
-          <Reference>614325</Reference>
-          <DisorderMappingRelation id="21541">
-            <Name lang="pt">BTBT (O Termo Abrangente do código ORPHA relaciona-se com o Termo Detalhado)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -307510,20 +307615,7 @@
       <DisorderGroup id="36540">
         <Name lang="pt">Grupo de patologias</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="123904">
-          <Source>ICD-10</Source>
-          <Reference>Q93.2</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -313847,20 +313939,7 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="123000">
-          <Source>ICD-10</Source>
-          <Reference>Q74.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -314291,17 +314370,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179650">
+        <ExternalReference id="187727">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>G96.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="61792">
@@ -330203,11 +330282,11 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
         <ExternalReference id="98695">
@@ -330241,11 +330320,11 @@
         <Synonym lang="pt">Del(12)(p12.1)</Synonym>
         <Synonym lang="pt">Monossomia 12p12.1</Synonym>
       </SynonymList>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
         <ExternalReference id="123514">
@@ -331621,17 +331700,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179651">
+        <ExternalReference id="187729">
           <Source>ICD-10</Source>
-          <Reference>G11.0</Reference>
+          <Reference>G11.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="74934">
@@ -340563,9 +340642,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=306597</ExpertLink>
       <Name lang="pt">Síndrome de OpitzG /BBB, ligada ao cromossoma X</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -340576,53 +340655,21 @@
       <DisorderGroup id="36554">
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="3">
-        <ExternalReference id="140146">
-          <Source>UMLS</Source>
-          <Reference>C2936904</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="123287">
-          <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="71929">
-          <Source>OMIM</Source>
-          <Reference>300000</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423">
+            <OrphaCode>2745</OrphaCode>
+            <Name lang="pt">Síndrome de Opitz BBB/G</Name>
+          </TargetDisorder>
+          <RootDisorder id="21238" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Esta doença está descrita em  Síndrome de Opitz BBB/G</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21237">
@@ -340630,9 +340677,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=306588</ExpertLink>
       <Name lang="pt">Síndrome de OpitzG/BBB, autossómica dominante</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -340643,42 +340690,21 @@
       <DisorderGroup id="36554">
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="2">
-        <ExternalReference id="123286">
-          <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
-        <ExternalReference id="71927">
-          <Source>OMIM</Source>
-          <Reference>145410</Reference>
-          <DisorderMappingRelation id="21527">
-            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="3423">
+            <OrphaCode>2745</OrphaCode>
+            <Name lang="pt">Síndrome de Opitz BBB/G</Name>
+          </TargetDisorder>
+          <RootDisorder id="21237" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Esta doença está descrita em  Síndrome de Opitz BBB/G</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21236">
@@ -344058,7 +344084,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="153086">
+        <ExternalReference id="187728">
           <Source>ICD-10</Source>
           <Reference>D72.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -344067,8 +344093,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -345202,8 +345228,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="21450">
-        <Name lang="pt">Subtipo clínico</Name>
+      <DisorderType id="21443">
+        <Name lang="pt">Subtipo etiologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="pt">Subtipo de patologia</Name>
@@ -345212,7 +345238,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Esta doença está descrita em  Síndrome de défice intelectual relacionada a DYRK1A</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="24023">
@@ -386815,17 +386848,17 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="155137">
+        <ExternalReference id="187730">
           <Source>ICD-10</Source>
-          <Reference>L51.8</Reference>
+          <Reference>L51.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -408700,20 +408733,7 @@
       <DisorderGroup id="36540">
         <Name lang="pt">Grupo de patologias</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="179633">
-          <Source>ICD-10</Source>
-          <Reference>Q43.1</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -409936,17 +409956,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="7">
-        <ExternalReference id="179645">
+        <ExternalReference id="187710">
           <Source>ICD-10</Source>
-          <Reference>Q82.2</Reference>
+          <Reference>Q82.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="10352">
@@ -415430,11 +415450,7 @@
       <OrphaCode>63269</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=63269</ExpertLink>
       <Name lang="pt">Síndrome de Antley-Bixler - ambiguidade genital - anomalias da esteroidogénese</Name>
-      <DisorderFlagList count="2">
-        <DisorderFlag id="459">
-          <Value>256</Value>
-          <Label>Deprecated entity</Label>
-        </DisorderFlag>
+      <DisorderFlagList count="1">
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
@@ -415461,26 +415477,9 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12654">
-            <OrphaCode>95699</OrphaCode>
-            <Name lang="pt">Hiperplasia suprarrenal congénita por deficiência na citocromo P450 oxidorreductase (POR)</Name>
-          </TargetDisorder>
-          <RootDisorder id="10819" cycle="true"/>
-          <DisorderDisorderAssociationType id="21471">
-            <Name lang="pt">Movido para</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
+      <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Esta doença foi movida para  Hiperplasia suprarrenal congénita por deficiência na citocromo P450 oxidorreductase (POR)</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="10816">
@@ -441733,7 +441732,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179599">
+        <ExternalReference id="187736">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -441742,8 +441741,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -441771,7 +441770,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179598">
+        <ExternalReference id="187735">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -441780,8 +441779,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -441820,7 +441819,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179597">
+        <ExternalReference id="187734">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -441829,8 +441828,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -441858,7 +441857,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179601">
+        <ExternalReference id="187738">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -441867,8 +441866,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -441907,7 +441906,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179600">
+        <ExternalReference id="187737">
           <Source>ICD-10</Source>
           <Reference>Q79.6</Reference>
           <DisorderMappingRelation id="21534">
@@ -441916,8 +441915,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -443799,7 +443798,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179608">
+        <ExternalReference id="187739">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -443808,8 +443807,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -445077,7 +445076,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179644">
+        <ExternalReference id="187709">
           <Source>ICD-10</Source>
           <Reference>Q68.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -445086,8 +445085,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="139311">
@@ -445126,7 +445125,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179611">
+        <ExternalReference id="187742">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -445135,8 +445134,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -445260,7 +445259,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179610">
+        <ExternalReference id="187741">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -445269,8 +445268,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -445298,7 +445297,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179609">
+        <ExternalReference id="187740">
           <Source>ICD-10</Source>
           <Reference>L88</Reference>
           <DisorderMappingRelation id="21534">
@@ -445307,8 +445306,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -448891,17 +448890,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179616">
+        <ExternalReference id="187743">
           <Source>ICD-10</Source>
           <Reference>L92.1</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          <DisorderMappingRelation id="21527">
+            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
           </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21597">
-            <Name lang="pt">Termo index (O código ORPHA está listado no index ICD10)</Name>
+          <DisorderMappingICDRelation id="21583">
+            <Name lang="pt">Código específico (O código ORPHA tem o seu código próprio na classificação ICD10)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -449042,17 +449041,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179617">
+        <ExternalReference id="187744">
           <Source>ICD-10</Source>
           <Reference>L95.0</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          <DisorderMappingRelation id="21527">
+            <Name lang="pt">Direção exacta (os termos e os conceitos são equivalentes)</Name>
           </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21597">
-            <Name lang="pt">Termo index (O código ORPHA está listado no index ICD10)</Name>
+          <DisorderMappingICDRelation id="21583">
+            <Name lang="pt">Código específico (O código ORPHA tem o seu código próprio na classificação ICD10)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -452436,17 +452435,17 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179625">
+        <ExternalReference id="187745">
           <Source>ICD-10</Source>
-          <Reference>K76.8</Reference>
+          <Reference>K75.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -452690,17 +452689,17 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179626">
+        <ExternalReference id="187746">
           <Source>ICD-10</Source>
-          <Reference>K76.8</Reference>
+          <Reference>K75.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -452849,17 +452848,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179628">
+        <ExternalReference id="187748">
           <Source>ICD-10</Source>
-          <Reference>Q22</Reference>
+          <Reference>Q23.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163329">
@@ -453100,7 +453099,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179629">
+        <ExternalReference id="187749">
           <Source>ICD-10</Source>
           <Reference>E27.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -453109,8 +453108,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -453163,7 +453162,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179630">
+        <ExternalReference id="187750">
           <Source>ICD-10</Source>
           <Reference>E27.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -453172,8 +453171,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -456659,7 +456658,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179171">
+        <ExternalReference id="187747">
           <Source>ICD-10</Source>
           <Reference>K75.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -457805,7 +457804,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11907" cycle="true"/>
+          <RootDisorder id="11906">
+            <OrphaCode>89841</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa distrófica centrípeta (forma rara)</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -457875,9 +457884,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=89841</ExpertLink>
       <Name lang="pt">Epidermólise bolhosa distrófica centrípeta (forma rara)</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -457890,7 +457899,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11907">
+            <OrphaCode>89842</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa distrófica, autossómica recessiva, tipo não-Hallopeau-Siemens</Name>
+          </TargetDisorder>
+          <RootDisorder id="11906" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -457900,9 +457919,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=89840</ExpertLink>
       <Name lang="pt">Epidermólise bolhosa juncional, tipo não-Herlitz</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -457915,7 +457934,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11428">
+            <OrphaCode>79402</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa benigna com atrofia generalizada</Name>
+          </TargetDisorder>
+          <RootDisorder id="11905" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -457925,9 +457954,9 @@
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=89839</ExpertLink>
       <Name lang="pt">Epidermólise bolhosa simples superficial</Name>
       <DisorderFlagList count="1">
-        <DisorderFlag id="461">
-          <Value>1024</Value>
-          <Label>Obsolete with resources</Label>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="1">
@@ -457941,7 +457970,17 @@
       </DisorderGroup>
       <ExternalReferenceList count="0">
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="641">
+            <OrphaCode>304</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa simples</Name>
+          </TargetDisorder>
+          <RootDisorder id="11904" cycle="true"/>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -469238,7 +469277,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179675">
+        <ExternalReference id="187733">
           <Source>ICD-10</Source>
           <Reference>E75.2</Reference>
           <DisorderMappingRelation id="21534">
@@ -469247,8 +469286,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -471498,7 +471537,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179674">
+        <ExternalReference id="187732">
           <Source>ICD-10</Source>
           <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -471507,8 +471546,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -483512,7 +483551,17 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11428" cycle="true"/>
+          <RootDisorder id="11905">
+            <OrphaCode>89840</OrphaCode>
+            <Name lang="pt">Epidermólise bolhosa juncional, tipo não-Herlitz</Name>
+          </RootDisorder>
+          <DisorderDisorderAssociationType id="27341">
+            <Name lang="pt">Referente a</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="0">
       </SummaryInformationList>
@@ -498364,7 +498413,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179673">
+        <ExternalReference id="187731">
           <Source>ICD-10</Source>
           <Reference>G11.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -498373,8 +498422,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -517837,17 +517886,7 @@
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12654" cycle="true"/>
-          <RootDisorder id="10819">
-            <OrphaCode>63269</OrphaCode>
-            <Name lang="pt">Síndrome de Antley-Bixler - ambiguidade genital - anomalias da esteroidogénese</Name>
-          </RootDisorder>
-          <DisorderDisorderAssociationType id="21471">
-            <Name lang="pt">Movido para</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
+      <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
       <SummaryInformationList count="1">
         <SummaryInformation id="77798" lang="pt">
@@ -522129,17 +522168,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="117905">
+        <ExternalReference id="187711">
           <Source>ICD-10</Source>
-          <Reference>Q93.5</Reference>
+          <Reference>Q87.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -530703,18 +530742,7 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="52010">
-          <Source>OMIM</Source>
-          <Reference>309580</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation/>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -533572,6 +533600,38 @@
         </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29775">
+      <OrphaCode>596426</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=596426</ExpertLink>
+      <Name lang="pt">Syndromes of reduced sensitivity to thyroid hormone</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21436">
+        <Name lang="pt">Grupo de fenótipos</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="pt">Grupo de patologias</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12431">
       <OrphaCode>93598</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93598</ExpertLink>
@@ -533735,6 +533795,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29762">
+      <OrphaCode>596008</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=596008</ExpertLink>
+      <Name lang="pt">Antley-Bixler syndrome without genital anomaly or disorder of steroidogenesis</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12419">
       <OrphaCode>93579</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93579</ExpertLink>
@@ -534637,6 +534722,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29776">
+      <OrphaCode>596448</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=596448</ExpertLink>
+      <Name lang="pt">IgG4-related systemic disease</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12433">
       <OrphaCode>93600</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93600</ExpertLink>
@@ -564882,8 +564992,8 @@
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
-      <DisorderType id="36561">
-        <Name lang="pt">Category</Name>
+      <DisorderType id="21436">
+        <Name lang="pt">Grupo de fenótipos</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="pt">Grupo de patologias</Name>
@@ -564977,7 +565087,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28857">
