commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml
old mode 100755
new mode 100644
index f222b48..db676e6
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:33:07" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:35:37" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10542">
+  <DisorderList count="10543">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -9375,7 +9375,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="97">
@@ -69267,9 +69274,9 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="121079">
+        <ExternalReference id="188714">
           <Source>ICD-10</Source>
-          <Reference>F01.1</Reference>
+          <Reference>I67.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
@@ -83636,7 +83643,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1018">
@@ -84761,9 +84775,9 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="107737">
+        <ExternalReference id="188713">
           <Source>ICD-10</Source>
-          <Reference>F01.1</Reference>
+          <Reference>I67.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
@@ -271239,7 +271253,7 @@
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="pt">Esta é uma doença rara para a qual existe uma designação orfã Europeia e/ou Americana.Este termo designa uma indicação para um teste farmacogenético.</Info>
+            <Info lang="pt">Esta é uma doença rara para a qual existe uma designação orfã Europeia e/ou Americana.</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -287618,7 +287632,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19272">
@@ -287705,7 +287726,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="19287">
@@ -293883,7 +293911,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21811">
@@ -327449,7 +327484,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20523">
@@ -327616,7 +327658,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20532">
@@ -327954,7 +328003,14 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20538">
@@ -327979,7 +328035,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20539">
@@ -328004,7 +328067,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20536">
@@ -328029,7 +328099,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20537">
@@ -339060,7 +339137,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21349">
@@ -346609,14 +346693,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo designa uma indicação para um teste farmacogenético.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21116">
@@ -347877,7 +347954,7 @@
           <TextSectionList count="0">
           </TextSectionList>
           <TextAuto>
-            <Info lang="pt">Esta é uma doença rara para a qual existe uma designação orfã Europeia e/ou Americana.Este termo designa uma indicação para um teste farmacogenético.</Info>
+            <Info lang="pt">Esta é uma doença rara para a qual existe uma designação orfã Europeia e/ou Americana.</Info>
           </TextAuto>
         </SummaryInformation>
       </SummaryInformationList>
@@ -381806,7 +381883,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23179">
@@ -383865,17 +383949,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179672">
+        <ExternalReference id="188684">
           <Source>ICD-10</Source>
-          <Reference>G60.8</Reference>
+          <Reference>G11.2</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -389202,7 +389286,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="154602">
+        <ExternalReference id="188685">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -389211,8 +389295,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="158701">
@@ -392249,17 +392333,17 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179670">
+        <ExternalReference id="188682">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>Q07.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -395086,7 +395170,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179671">
+        <ExternalReference id="188683">
           <Source>ICD-10</Source>
           <Reference>G71.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -395095,8 +395179,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="156032">
@@ -409597,19 +409681,6 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="3">
-        <ExternalReference id="114885">
-          <Source>ICD-10</Source>
-          <Reference>G93.4</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
         <ExternalReference id="16869">
           <Source>OMIM</Source>
           <Reference>606777</Reference>
@@ -409632,6 +409703,19 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
+        <ExternalReference id="188712">
+          <Source>ICD-10</Source>
+          <Reference>G40.4</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
@@ -410116,7 +410200,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179634">
+        <ExternalReference id="188705">
           <Source>ICD-10</Source>
           <Reference>K31.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -410125,8 +410209,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -424169,7 +424253,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="178532">
+        <ExternalReference id="188706">
           <Source>ICD-10</Source>
           <Reference>E75.5</Reference>
           <DisorderMappingRelation id="21534">
@@ -424178,8 +424262,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -424314,17 +424398,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179635">
+        <ExternalReference id="188707">
           <Source>ICD-10</Source>
-          <Reference>K59.9</Reference>
+          <Reference>K59.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -426380,7 +426464,20 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188709">
+          <Source>ICD-10</Source>
+          <Reference>E07.8</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
@@ -426474,7 +426571,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="178095">
+        <ExternalReference id="188708">
           <Source>ICD-10</Source>
           <Reference>E07.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -426483,8 +426580,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="172336">
@@ -444626,7 +444723,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179602">
+        <ExternalReference id="188721">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -444635,8 +444732,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -444904,7 +445001,7 @@
         <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179603">
+        <ExternalReference id="188717">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -444913,8 +445010,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -445096,7 +445193,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179604">
+        <ExternalReference id="188687">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -445105,8 +445202,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -445196,7 +445293,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179605">
+        <ExternalReference id="188688">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -445205,8 +445302,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -446526,17 +446623,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179607">
+        <ExternalReference id="188690">
           <Source>ICD-10</Source>
-          <Reference>D23.2</Reference>
+          <Reference>D23.9</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163484">
@@ -446586,7 +446683,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="179606">
+        <ExternalReference id="188689">
           <Source>ICD-10</Source>
           <Reference>G60.0</Reference>
           <DisorderMappingRelation id="21534">
@@ -446595,8 +446692,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -447108,7 +447205,20 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
+      <ExternalReferenceList count="2">
+        <ExternalReference id="188710">
+          <Source>ICD-10</Source>
+          <Reference>D82.3</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21590">
+            <Name lang="pt">Termo inclusivo (O código ORPHA está incluído numa categoria ICD10 e não tem o seu código próprio)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
         <ExternalReference id="163478">
           <Source>OMIM</Source>
           <Reference>300635</Reference>
@@ -447150,7 +447260,20 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188711">
+          <Source>ICD-10</Source>
+          <Reference>D89.8</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -447176,7 +447299,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179612">
+        <ExternalReference id="188691">
           <Source>ICD-10</Source>
           <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
@@ -447185,8 +447308,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163477">
@@ -448557,17 +448680,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179613">
+        <ExternalReference id="188692">
           <Source>ICD-10</Source>
-          <Reference>D81.1</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163479">
@@ -448667,17 +448790,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179614">
+        <ExternalReference id="188693">
           <Source>ICD-10</Source>
-          <Reference>D81.1</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="163481">
@@ -451206,17 +451329,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179615">
+        <ExternalReference id="188694">
           <Source>ICD-10</Source>
-          <Reference>D81.9</Reference>
+          <Reference>D82.3</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="172005">
@@ -452187,17 +452310,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179618">
+        <ExternalReference id="188695">
           <Source>ICD-10</Source>
-          <Reference>G71.3</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -452692,7 +452815,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179619">
+        <ExternalReference id="188696">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -452701,8 +452824,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -453869,7 +453992,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179622">
+        <ExternalReference id="188699">
           <Source>ICD-10</Source>
           <Reference>G40.4</Reference>
           <DisorderMappingRelation id="21534">
@@ -453878,8 +454001,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171455">
@@ -454122,17 +454245,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179621">
+        <ExternalReference id="188698">
           <Source>ICD-10</Source>
-          <Reference>Q87.8</Reference>
+          <Reference>E72.4</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -454160,17 +454283,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179620">
+        <ExternalReference id="188697">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>Q07.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171424">
@@ -454557,7 +454680,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179623">
+        <ExternalReference id="188700">
           <Source>ICD-10</Source>
           <Reference>G71.2</Reference>
           <DisorderMappingRelation id="21534">
@@ -454566,8 +454689,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171478">
@@ -454836,17 +454959,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179624">
+        <ExternalReference id="188701">
           <Source>ICD-10</Source>
-          <Reference>G31.8</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="171468">
@@ -455503,7 +455626,7 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179627">
+        <ExternalReference id="188702">
           <Source>ICD-10</Source>
           <Reference>Q22.8</Reference>
           <DisorderMappingRelation id="21534">
@@ -455512,8 +455635,8 @@
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -456090,17 +456213,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="2">
-        <ExternalReference id="179632">
+        <ExternalReference id="188704">
           <Source>ICD-10</Source>
-          <Reference>G40.4</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
         <ExternalReference id="179040">
@@ -456139,17 +456262,17 @@
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <ExternalReferenceList count="1">
-        <ExternalReference id="179631">
+        <ExternalReference id="188703">
           <Source>ICD-10</Source>
-          <Reference>G11.1</Reference>
+          <Reference>E88.8</Reference>
           <DisorderMappingRelation id="21534">
             <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
           </DisorderMappingRelation>
           <DisorderMappingICDRelation id="21604">
             <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
           </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21618">
-            <Name lang="pt">Ainda não validado</Name>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
       </ExternalReferenceList>
@@ -480982,7 +481105,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="117037">
+        <ExternalReference id="188715">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -484743,7 +484866,20 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188719">
+          <Source>ICD-10</Source>
+          <Reference>D84.1</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -484813,7 +484949,20 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="0">
+      <ExternalReferenceList count="1">
+        <ExternalReference id="188720">
+          <Source>ICD-10</Source>
+          <Reference>D84.1</Reference>
+          <DisorderMappingRelation id="21534">
+            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
+          </DisorderMappingRelation>
+          <DisorderMappingICDRelation id="21604">
+            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
+          </DisorderMappingICDRelation>
+          <DisorderMappingValidationStatus id="21611">
+            <Name lang="pt">Validado</Name>
+          </DisorderMappingValidationStatus>
+        </ExternalReference>
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -532468,6 +532617,31 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="29899">
+      <OrphaCode>599082</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=599082</ExpertLink>
+      <Name lang="pt">CHD3-related developmental delay-speech delay-intellectual disability-abnormalities of vision-facial dysmorphism syndrome</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21401">
+        <Name lang="pt">Síndrome malformativo</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="12296">
       <OrphaCode>93367</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93367</ExpertLink>
@@ -584881,7 +585055,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="119528">
+        <ExternalReference id="188716">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
@@ -587217,7 +587391,7 @@
             <Name lang="pt">Validado</Name>
           </DisorderMappingValidationStatus>
         </ExternalReference>
-        <ExternalReference id="119519">
+        <ExternalReference id="188718">
           <Source>ICD-10</Source>
           <Reference>D84.1</Reference>
           <DisorderMappingRelation id="21534">
