commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml
index 31b60e0..bde4293 100644
--- a/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml	
+++ b/Rare diseases and classifications/Cross-referencing of rare diseases/XML/pt_product1.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:40:39" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:40:52" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="10617">
+  <DisorderList count="10619">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -59829,14 +59829,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="655">
@@ -122126,14 +122119,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="1423">
@@ -154728,14 +154714,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2036">
@@ -182059,7 +182038,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Esta doença está descrita em  Síndrome SATB2-associada</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="2135">
@@ -265820,14 +265806,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Esta não é uma doença rara na Europa. Não pertence à nomenclatura Orphanet para doenças raras. </Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="18584">
@@ -292458,10 +292437,14 @@
       <OrphaCode>217601</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=217601</ExpertLink>
       <Name lang="pt">Miocardiopatia hipertrófica por exercício físico excessivo</Name>
-      <DisorderFlagList count="1">
-        <DisorderFlag id="475">
-          <Value>1</Value>
-          <Label></Label>
+      <DisorderFlagList count="2">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
+        <DisorderFlag id="455">
+          <Value>16</Value>
+          <Label>Obsolete entity</Label>
         </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
@@ -292472,20 +292455,7 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <ExternalReferenceList count="1">
-        <ExternalReference id="121486">
-          <Source>ICD-10</Source>
-          <Reference>I42.2</Reference>
-          <DisorderMappingRelation id="21534">
-            <Name lang="pt">NTBT (O Termo Detalhado do código ORPHA relaciona-se com o Termo Abrangente)</Name>
-          </DisorderMappingRelation>
-          <DisorderMappingICDRelation id="21604">
-            <Name lang="pt">Atribuido (O código ICD10 é atribuido pela Orphanet)</Name>
-          </DisorderMappingICDRelation>
-          <DisorderMappingValidationStatus id="21611">
-            <Name lang="pt">Validado</Name>
-          </DisorderMappingValidationStatus>
-        </ExternalReference>
+      <ExternalReferenceList count="0">
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
@@ -312544,14 +312514,7 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="3630">
@@ -361768,14 +361731,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Esta não é uma doença rara na Europa. Não pertence à nomenclatura Orphanet para doenças raras. </Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="20721">
@@ -376390,14 +376346,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="21355">
@@ -418740,14 +418689,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Esta não é uma doença rara na Europa. Não pertence à nomenclatura Orphanet para doenças raras. </Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="23124">
@@ -469097,14 +469039,7 @@
           </DisorderDisorderAssociationType>
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="11214">
@@ -479355,11 +479290,19 @@
       <OrphaCode>73247</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=73247</ExpertLink>
       <Name lang="pt">Esofagite eosinfílica</Name>
-      <DisorderFlagList count="1">
+      <DisorderFlagList count="3">
+        <DisorderFlag id="495">
+          <Value>8192</Value>
+          <Label>Inactive</Label>
+        </DisorderFlag>
         <DisorderFlag id="475">
           <Value>1</Value>
           <Label></Label>
         </DisorderFlag>
+        <DisorderFlag id="456">
+          <Value>32</Value>
+          <Label>Non-rare disease in Europe</Label>
+        </DisorderFlag>
       </DisorderFlagList>
       <SynonymList count="0">
       </SynonymList>
@@ -536536,14 +536479,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="27542">
@@ -579373,14 +579309,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12618">
@@ -580773,14 +580702,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="12590">
@@ -625490,14 +625412,7 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="1">
-        <SummaryInformation id="-1" lang="pt">
-          <TextSectionList count="0">
-          </TextSectionList>
-          <TextAuto>
-            <Info lang="pt">Este termo não caracteriza uma doença mas sim um grupo de doenças. Para saber mais sobre as doenças inlcuidas no termo abaixo, consulte por favor as classificações.</Info>
-          </TextAuto>
-        </SummaryInformation>
+      <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
     <Disorder id="13741">
@@ -632848,7 +632763,14 @@
       </ExternalReferenceList>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
-      <SummaryInformationList count="0">
+      <SummaryInformationList count="1">
+        <SummaryInformation id="-1" lang="pt">
+          <TextSectionList count="0">
+          </TextSectionList>
+          <TextAuto>
+            <Info lang="pt">Esta doença está descrita em  Síndrome SATB2-associada</Info>
+          </TextAuto>
+        </SummaryInformation>
       </SummaryInformationList>
     </Disorder>
     <Disorder id="28850">
@@ -663276,6 +663198,56 @@
       <SummaryInformationList count="0">
       </SummaryInformationList>
     </Disorder>
+    <Disorder id="31389">
+      <OrphaCode>618899</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=618899</ExpertLink>
+      <Name lang="pt">Acid sphingomyelinase deficiency</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21436">
+        <Name lang="pt">Grupo de fenótipos</Name>
+      </DisorderType>
+      <DisorderGroup id="36540">
+        <Name lang="pt">Grupo de patologias</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
+    <Disorder id="31388">
+      <OrphaCode>618891</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=618891</ExpertLink>
+      <Name lang="pt">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
+      <DisorderFlagList count="1">
+        <DisorderFlag id="475">
+          <Value>1</Value>
+          <Label></Label>
+        </DisorderFlag>
+      </DisorderFlagList>
+      <SynonymList count="0">
+      </SynonymList>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
+      </DisorderGroup>
+      <ExternalReferenceList count="0">
+      </ExternalReferenceList>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+      <SummaryInformationList count="0">
+      </SummaryInformationList>
+    </Disorder>
     <Disorder id="31370">
       <OrphaCode>617910</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=617910</ExpertLink>
