commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Epidemiological data/Rare disease epidemiology/cs_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/cs_product9_prev.xml
index 92f0328..dbb6c44 100644
--- a/Epidemiological data/Rare disease epidemiology/cs_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/cs_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<JDBOR date="2022-04-01 05:00:38" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:57:44" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6063">
+  <DisorderList count="6058">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=166024</ExpertLink>
@@ -8133,16 +8133,16 @@
           <PrevalenceClass id="23739">
             <Name lang="cs">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.22</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="cs">Evropa</Name>
+          <ValMoy>6.66</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="cs">Celosvětově</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="cs">Dosud neoveřeno</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="cs">Ověřeno</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="293">
-          <Source>9918480[PMID]</Source>
+          <Source>35024668[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="cs">Prevalence při narození</Name>
           </PrevalenceType>
@@ -8152,7 +8152,7 @@
           <PrevalenceClass id="23739">
             <Name lang="cs">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.85</ValMoy>
+          <ValMoy>6.96</ValMoy>
           <PrevalenceGeographic id="23949">
             <Name lang="cs">Austrálie</Name>
           </PrevalenceGeographic>
@@ -74189,7 +74189,7 @@
       <DisorderGroup id="36547">
         <Name lang="cs">Onemocnění</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="2539">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -74198,12 +74198,50 @@
           <PrevalenceQualification id="23704">
             <Name lang="cs">Pouze třída</Name>
           </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="cs">neznámý</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="cs">Celosvětově</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="cs">Ověřeno</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16781">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="cs">Prevalence při narození</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="cs">Pouze třída</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="cs">neznámý</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="cs">Celosvětově</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="cs">Ověřeno</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16782">
+          <Source>20734338[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="cs">Prevalence při narození</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="cs">Hodnota a třída</Name>
+          </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="cs">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="cs">Evropa</Name>
+          <ValMoy>1.75</ValMoy>
+          <PrevalenceGeographic id="23949">
+            <Name lang="cs">Austrálie</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="cs">Ověřeno</Name>
@@ -82623,38 +82661,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=946</ExpertLink>
-      <Name lang="cs">Akrocefalosyndaktylie</Name>
-      <DisorderType id="21436">
-        <Name lang="cs">Klinická skupina</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="cs">Skupina onemocnění</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="2783">
-          <Source>ORPHANET_[EXPERT]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="cs">Bod prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="cs">Pouze třída</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="cs">neznámý</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="cs">Evropa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=957</ExpertLink>
@@ -99304,55 +99310,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=1533</ExpertLink>
-      <Name lang="cs">Syndrom zahrnující kraniosynostózu a aplázii fibuly</Name>
-      <DisorderType id="21401">
-        <Name lang="cs">Malformační syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="cs">Onemocnění</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="3180">
-          <Source>4403018[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="cs">Případy / rodiny</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="cs">Případ</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="cs">Celosvětově</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="3181">
-          <Source>ORPHANET_4403018[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="cs">Bod prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="cs">Pouze třída</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="cs">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="cs">Celosvětově</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=140874</ExpertLink>
@@ -115596,7 +115553,7 @@
     <Disorder id="2078">
       <OrphaCode>2196</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=2196</ExpertLink>
-      <Name lang="cs">Familiární primární hypomagnesémie s hyperkalciurií a nefrokalcinózou s vážným postižením očí</Name>
+      <Name lang="cs">Primární hypomagnesémie s hyperkalciurií a nefrokalcinózou s vážným postižením očí</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -190778,55 +190735,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=329341</ExpertLink>
-      <Name lang="cs">Limbická encefalitida s DPP6 protilátkami</Name>
-      <DisorderType id="21394">
-        <Name lang="cs">Onemocnění</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="cs">Onemocnění</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="10180">
-          <Source>23225603[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="cs">Případy / rodiny</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="cs">Případ</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="cs">Celosvětově</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10181">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="cs">Bod prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="cs">Pouze třída</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="cs">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="cs">Celosvětově</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=329228</ExpertLink>
@@ -218918,7 +218826,7 @@
     <Disorder id="21222">
       <OrphaCode>306516</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=306516</ExpertLink>
-      <Name lang="cs">Familiární primární hypomagnézie s hyperkalciurií a nefrokalcinózou</Name>
+      <Name lang="cs">Primární hypomagnézie s hyperkalciurií a nefrokalcinózou</Name>
       <DisorderType id="21394">
         <Name lang="cs">Onemocnění</Name>
       </DisorderType>
@@ -265668,55 +265576,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=498700</ExpertLink>
-      <Name lang="cs">Limbická encefalitida s protilátkami proti neurexinu 3</Name>
-      <DisorderType id="21394">
-        <Name lang="cs">Onemocnění</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="cs">Onemocnění</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13782">
-          <Source>27170573[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="cs">Případy / rodiny</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="cs">Případ</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="cs">Celosvětově</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13783">
-          <Source>ORPHANET_27170573[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="cs">Bod prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="cs">Pouze třída</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="cs">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="cs">Celosvětově</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="cs">Ověřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=874</ExpertLink>
@@ -272092,7 +271951,7 @@
     <Disorder id="9284">
       <OrphaCode>31043</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=31043</ExpertLink>
-      <Name lang="cs">Familiární primární hypomagnezémie s hyperkalciurií a nefrokalcinózou bez těžkého postižení očí</Name>
+      <Name lang="cs">Primární hypomagnezémie s hyperkalciurií a nefrokalcinózou bez těžkého postižení očí</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -277369,7 +277228,7 @@
     <Disorder id="10882">
       <OrphaCode>66529</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=66529</ExpertLink>
-      <Name lang="cs">Tako-tsubo kardiomyopatie</Name>
+      <Name lang="cs">Tako-Tsubo kardiomyopatie</Name>
       <DisorderType id="21394">
         <Name lang="cs">Onemocnění</Name>
       </DisorderType>
@@ -279297,38 +279156,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=63440</ExpertLink>
-      <Name lang="cs">Izolovaná oxycefalie</Name>
-      <DisorderType id="21415">
-        <Name lang="cs">Morfologická anomálie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="cs">Onemocnění</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="5872">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="cs">Bod prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="cs">Pouze třída</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="cs">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="cs">Evropa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="cs">Dosud neoveřeno</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=63442</ExpertLink>
@@ -355843,7 +355670,7 @@
     <Disorder id="28672">
       <OrphaCode>570371</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=570371</ExpertLink>
-      <Name lang="cs">Tranzientní antenatální Bartterův syndrom</Name>
+      <Name lang="cs">Bartterův syndrom typu 5</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
