commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml
index aa4560d..dba4c19 100644
--- a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:41:54" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:42:37" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6037">
+  <DisorderList count="6026">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166024</ExpertLink>
@@ -26513,7 +26513,7 @@
     <Disorder id="17817">
       <OrphaCode>169085</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=169085</ExpertLink>
-      <Name lang="de">Erhöhte Anfälligkeit für Atemwegsinfektionen durch CD8-Mangel</Name>
+      <Name lang="de">Erhöhte Anfälligkeit für Atemwegsinfektionen durch CD8alpha-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -76214,8 +76214,8 @@
       <OrphaCode>847</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=847</ExpertLink>
       <Name lang="de">Alpha-Thalassämie-X-chromosomale Intelligenzminderung-Syndrom</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21401">
+        <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
@@ -86710,7 +86710,7 @@
     <Disorder id="1339">
       <OrphaCode>1031</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1031</ExpertLink>
-      <Name lang="de">Zahnschmelz-Nieren-Syndrom</Name>
+      <Name lang="de">Zahnschmelz-Niere-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -111825,7 +111825,7 @@
     <Disorder id="2191">
       <OrphaCode>2339</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2339</ExpertLink>
-      <Name lang="de">Keratosis follicularis - Kleinwuchs - zerebrale Atrophie</Name>
+      <Name lang="de">Keratosis follicularis-Kleinwuchs-Hirnatrophie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -117943,7 +117943,7 @@
     <Disorder id="2084">
       <OrphaCode>2201</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2201</ExpertLink>
-      <Name lang="de">Palmoplantarkeratose - spastische Lähmung</Name>
+      <Name lang="de">Palmoplantarkeratose-spastische Paralyse-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -125421,7 +125421,7 @@
     <Disorder id="19728">
       <OrphaCode>252175</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=252175</ExpertLink>
-      <Name lang="de">OBSOLET: Schwannom, vestibuläres</Name>
+      <Name lang="de">Schwannom, vestibuläres</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -128661,7 +128661,7 @@
     <Disorder id="2344">
       <OrphaCode>2536</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2536</ExpertLink>
-      <Name lang="de">Mikrokornea - Glaukom - fehlende Stirnhöhlen</Name>
+      <Name lang="de">Mikrokornea-Glaukom-fehlende Stirnhöhlen-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -129910,7 +129910,7 @@
     <Disorder id="2376">
       <OrphaCode>2576</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2576</ExpertLink>
-      <Name lang="de">MULIBREY-Kleinwuchs</Name>
+      <Name lang="de">Mulibrey-Kleinwuchs</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -132327,55 +132327,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="2746">
-      <OrphaCode>3056</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3056</ExpertLink>
-      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Brooks</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="11998">
-          <Source>16477654[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>9.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="11999">
-          <Source>ORPHANET_16477654[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="2747">
       <OrphaCode>3057</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3057</ExpertLink>
@@ -134375,7 +134326,7 @@
     <Disorder id="2706">
       <OrphaCode>3011</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3011</ExpertLink>
-      <Name lang="de">Spastische Tetraparese - Retinitis pigmentosa - Intelligenzminderung</Name>
+      <Name lang="de">Spastische Tetraplegie-Retinitis pigmentosa-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -146217,7 +146168,7 @@
     <Disorder id="2974">
       <OrphaCode>3355</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3355</ExpertLink>
-      <Name lang="de">Tricho-Odonto-Onycho-Dysplasie</Name>
+      <Name lang="de">Tricho-odonto-onychiale Dysplasie</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -160081,7 +160032,7 @@
     <Disorder id="3130">
       <OrphaCode>3405</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3405</ExpertLink>
-      <Name lang="de">Nabelschnur-Ulzera - Darmatresie</Name>
+      <Name lang="de">Nabelschnur-Ulzera-Darmatresie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -207813,55 +207764,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="20904">
-      <OrphaCode>294049</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=294049</ExpertLink>
-      <Name lang="de">Reunion Insel-Variante des Larsen-Syndroms</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="9797">
-          <Source>21882400[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>30.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="9798">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="20900">
       <OrphaCode>293987</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=293987</ExpertLink>
@@ -209184,7 +209086,7 @@
     <Disorder id="20741">
       <OrphaCode>289560</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=289560</ExpertLink>
-      <Name lang="de">Mitochondrienmembran-Protein-Assoziierte Neurodegeneration</Name>
+      <Name lang="de">Mitochondrienmembran-Protein-assoziierte Neurodegeneration</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -217077,7 +216979,7 @@
     <Disorder id="21283">
       <OrphaCode>307766</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=307766</ExpertLink>
-      <Name lang="de">Kaushaar-akrales Keratoderm-Karies-Syndrom</Name>
+      <Name lang="de">Kraushaar-akrales Keratoderm-Karies-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -258675,7 +258577,7 @@
     <Disorder id="8731">
       <OrphaCode>600</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=600</ExpertLink>
-      <Name lang="de">Distale Myopathie mit Stimmband- und Rachenschwäche</Name>
+      <Name lang="de">Distale Myopathie mit Stimmband- und Pharynxschwäche</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -260544,55 +260446,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="25668">
-      <OrphaCode>494348</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=494348</ExpertLink>
-      <Name lang="de">Nicht-zirrhotische portale Hypertension mit frühem Beginn, familiäre Form</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13746">
-          <Source>26874653[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>3.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13747">
-          <Source>ORPHANET_26874653[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="25683">
       <OrphaCode>494547</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=494547</ExpertLink>
@@ -279217,11 +279070,11 @@
       <OrphaCode>64743</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=64743</ExpertLink>
       <Name lang="de">Hepatoportale Sklerose</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21457">
+        <Name lang="de">Histopathologischer Subtyp</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <PrevalenceList count="1">
         <Prevalence id="5894">
@@ -289741,38 +289594,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="11034">
-      <OrphaCode>73220</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=73220</ExpertLink>
-      <Name lang="de">X-chromosomale Intelligenzminderung-Syndrom-hypotones Gesicht-Syndrom</Name>
-      <DisorderType id="21436">
-        <Name lang="de">Phenom-Gruppe</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="de">Gruppe von Störungen</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="6114">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="de">Unbekannt</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="10462">
       <OrphaCode>40366</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=40366</ExpertLink>
@@ -290906,7 +290727,7 @@
     <Disorder id="10399">
       <OrphaCode>35737</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=35737</ExpertLink>
-      <Name lang="de">Morning-Glory-Syndrom</Name>
+      <Name lang="de">Morning-Glory-Papille</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -299046,7 +298867,7 @@
     <Disorder id="10651">
       <OrphaCode>50812</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=50812</ExpertLink>
-      <Name lang="de">Zellweger-ähnliches Syndrom ohne Anomalien der Peroxysomen</Name>
+      <Name lang="de">Zellweger-ähnliches Syndrom ohne Anomalien der Peroxisomen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -302185,11 +302006,11 @@
       <OrphaCode>48372</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=48372</ExpertLink>
       <Name lang="de">Noduläre regenerative Hyperplasie (NRH) der Leber</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21457">
+        <Name lang="de">Histopathologischer Subtyp</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
         <Prevalence id="5672">
@@ -323609,38 +323430,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="11698">
-      <OrphaCode>85328</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85328</ExpertLink>
-      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Turner</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="6801">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="de">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="11699">
       <OrphaCode>85329</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85329</ExpertLink>
@@ -330471,7 +330260,7 @@
     <Disorder id="11623">
       <OrphaCode>85112</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85112</ExpertLink>
-      <Name lang="de">Palmoplantarkeratose - XX-Geschlechtsumkehr - Prädisposition für squamöse Zellkarzinome</Name>
+      <Name lang="de">Palmoplantarkeratose-XX-Geschlechtsumkehr-Prädisposition für Plattenepithelkarzinom-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -334962,7 +334751,7 @@
     <Disorder id="12593">
       <OrphaCode>95434</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=95434</ExpertLink>
-      <Name lang="de">Autosomal-rezessive zerebelläre Ataxie - einschließende Sakkaden</Name>
+      <Name lang="de">Autosomal-rezessive zerebelläre Ataxie-sakkadische Intrusionen-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -338515,104 +338304,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="12533">
-      <OrphaCode>93975</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93975</ExpertLink>
-      <Name lang="de">Renier-Gabreels-Jasper-Syndrom</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="7271">
-          <Source>7077653[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="7272">
-          <Source>ORPHANET_7077653[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="12532">
-      <OrphaCode>93974</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93974</ExpertLink>
-      <Name lang="de">Smith-Fineman-Myers-Syndrom</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="7269">
-          <Source>[EXPERT]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23725">
-            <Name lang="de">Family</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>11.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="7270">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="12534">
       <OrphaCode>93976</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93976</ExpertLink>
@@ -338911,202 +338602,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="12529">
-      <OrphaCode>93971</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93971</ExpertLink>
-      <Name lang="de">Chudley-Lowry-Hoar-Syndrom</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="7263">
-          <Source>3239563[PMID]_[EXPERT]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>3.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="7264">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="12528">
-      <OrphaCode>93970</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93970</ExpertLink>
-      <Name lang="de">Holmes-Gang-Syndrom</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="7261">
-          <Source>6711605[PMID]_[EXPERT]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>3.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="7262">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="12531">
-      <OrphaCode>93973</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93973</ExpertLink>
-      <Name lang="de">Carpenter-Waziri-Syndrom</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="7267">
-          <Source>10398237[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>6.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="7268">
-          <Source>ORPHANET_10398237[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="12530">
-      <OrphaCode>93972</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93972</ExpertLink>
-      <Name lang="de">Juberg-Marsidi-Syndrom</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="7265">
-          <Source>8533845[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>16.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="7266">
-          <Source>ORPHANET_8533845[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="12490">
       <OrphaCode>93930</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93930</ExpertLink>
