commit 481d14fea17f62fcae7251f3673242ec271b28fa
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Dec 4 20:10:24 2023 +0100

    DEC 2023

diff --git a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml
index 164c512..2df56c9 100644
--- a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2023-06-22 14:17:03" version="1.3.25 / 4.1.7 [2023-04-17] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2023-12-04 08:18:49" version="1.3.29 / 4.1.7 [2023-08-02] (orientdb version)" copyright="Orphanet (c) 2023" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6089">
+  <DisorderList count="6088">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166024</ExpertLink>
@@ -551,7 +551,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="12">
-          <Source>ISBN:9780071363228[OTHER]</Source>
+          <Source>36980153[PMID]_37732042[PMID]_ORPHANET</Source>
           <PrevalenceType id="23697">
             <Name lang="de">Cases/families</Name>
           </PrevalenceType>
@@ -559,7 +559,7 @@
             <Name lang="de">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>50.0</ValMoy>
+          <ValMoy>154.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="de">Weltweit</Name>
           </PrevalenceGeographic>
@@ -568,15 +568,15 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13">
-          <Source>ORPHANET_ISBN:9780071363228[OTHER]</Source>
+          <Source>36980153[PMID]_37732042[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="de">Punktprävalenz</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23704">
             <Name lang="de">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
@@ -1222,9 +1222,9 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <PrevalenceList count="3">
+      <PrevalenceList count="4">
         <Prevalence id="42">
-          <Source>10094192[PMID]</Source>
+          <Source>36082656[PMID]</Source>
           <PrevalenceType id="23697">
             <Name lang="de">Cases/families</Name>
           </PrevalenceType>
@@ -1232,7 +1232,7 @@
             <Name lang="de">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>100.0</ValMoy>
+          <ValMoy>161.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="de">Weltweit</Name>
           </PrevalenceGeographic>
@@ -1241,15 +1241,15 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="44">
-          <Source>ORPHANET_10094192[PMID]</Source>
+          <Source>36082656[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="de">Punktprävalenz</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23704">
             <Name lang="de">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
@@ -1278,6 +1278,25 @@
             <Name lang="de">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="17450">
+          <Source>27706744[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="de">Prävalenz bei Geburt</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="de">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="de">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.63</ValMoy>
+          <PrevalenceGeographic id="24243">
+            <Name lang="de">Cuba</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="17615">
@@ -5558,7 +5577,7 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <PrevalenceList count="2">
+      <PrevalenceList count="3">
         <Prevalence id="228">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
@@ -5597,6 +5616,25 @@
             <Name lang="de">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="17460">
+          <Source>25390741[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="de">Europa</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="49">
@@ -6793,7 +6831,7 @@
     <Disorder id="55">
       <OrphaCode>262</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=262</ExpertLink>
-      <Name lang="de">Duchenne- und Becker-Muskeldystrophien</Name>
+      <Name lang="de">Duchenne- und Becker-Muskeldystrophie</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -12323,7 +12361,7 @@
     <Disorder id="125">
       <OrphaCode>232</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=232</ExpertLink>
-      <Name lang="de">Sichelzellkrankheit</Name>
+      <Name lang="de">Sichelzellanämie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -16148,45 +16186,7 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <PrevalenceList count="5">
-        <Prevalence id="579">
-          <Source>[EXPERT]_ORPHANET</Source>
-          <PrevalenceType id="23683">
-            <Name lang="de">Prävalenz bei Geburt</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="de">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="de">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>10.8</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="580">
-          <Source>EUROCAT European surveillance of congenital anomalies 2010[REG]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="de">Prävalenz bei Geburt</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="de">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="de">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>1.8</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="de">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
+      <PrevalenceList count="2">
         <Prevalence id="581">
           <Source>12088082[PMID]</Source>
           <PrevalenceType id="23683">
@@ -16225,25 +16225,6 @@
             <Name lang="de">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="583">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="de">Unbekannt</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="147">
@@ -16307,7 +16288,7 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <PrevalenceList count="30">
+      <PrevalenceList count="31">
         <Prevalence id="586">
           <Source>32803853[PMID]</Source>
           <PrevalenceType id="23683">
@@ -16878,6 +16859,25 @@
             <Name lang="de">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="17461">
+          <Source>32803853[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="149">
@@ -18685,7 +18685,7 @@
     <Disorder id="17521">
       <OrphaCode>163681</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=163681</ExpertLink>
-      <Name lang="de">Kortikale Dysplasie-fokale Epilepsie-Syndrom</Name>
+      <Name lang="de">CNTNAP2-assoziierte entwicklungsbedingte und epileptische Enzephalopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -21839,7 +21839,7 @@
     <Disorder id="238">
       <OrphaCode>126</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=126</ExpertLink>
-      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom</Name>
+      <Name lang="de">Blepharophimose-Ptosis-Epicanthus inversus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -39950,6 +39950,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="515">
+      <OrphaCode>2300</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2300</ExpertLink>
+      <Name lang="de">Multiple intestinale Atresie</Name>
+      <DisorderType id="21415">
+        <Name lang="de">Morphologische Anomalie</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17448">
+          <Source>29437734[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17449">
+          <Source>29437734[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="de">Prävalenz bei Geburt</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="de">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="de">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>4.05</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="18125">
       <OrphaCode>180226</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=180226</ExpertLink>
@@ -41085,55 +41136,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="543">
-      <OrphaCode>587</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=587</ExpertLink>
-      <Name lang="de">Muir-Torre-Syndrom</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="1317">
-          <Source>16321766[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>205.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="1318">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="de">Unbekannt</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="542">
       <OrphaCode>570</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570</ExpertLink>
@@ -58540,7 +58542,7 @@
     <Disorder id="17934">
       <OrphaCode>171719</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=171719</ExpertLink>
-      <Name lang="de">Cutix laxa-mafanoides Syndrom</Name>
+      <Name lang="de">Cutis laxa-marfanoides Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -66182,7 +66184,7 @@
     <Disorder id="784">
       <OrphaCode>3002</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3002</ExpertLink>
-      <Name lang="de">Immunthrombozytopenie, immune</Name>
+      <Name lang="de">Immunthrombozytopenie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -77875,24 +77877,22 @@
         <Name lang="de">Störung</Name>
       </DisorderGroup>
       <PrevalenceList count="1">
-        <Prevalence id="16539">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
+        <Prevalence id="17459">
+          <Source>[https://www.ncbi.nlm.nih.gov/books/NBK1760/]_ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="de">Prävalenz bei Geburt</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
+          <PrevalenceQualification id="23711">
+            <Name lang="de">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="de">Unbekannt</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="de">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
+          <ValMoy>1.875</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="de">Weltweit</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
+          <PrevalenceValidationStatus/>
         </Prevalence>
       </PrevalenceList>
     </Disorder>
@@ -85952,55 +85952,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1270">
-      <OrphaCode>1795</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1795</ExpertLink>
-      <Name lang="de">Dysostose, periphere</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="14022">
-          <Source>7205902[PMID]_6018472[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>6.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="14023">
-          <Source>ORPHANET_7205902[PMID]_6018472[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1268">
       <OrphaCode>37</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=37</ExpertLink>
@@ -91361,55 +91312,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1486">
-      <OrphaCode>1240</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1240</ExpertLink>
-      <Name lang="de">Akroscyphodysplasie, metaphysäre</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="2997">
-          <Source>6543849[PMID]_1860252[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="2998">
-          <Source>ORPHANET_6543849[PMID]_1860252[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1483">
       <OrphaCode>1237</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1237</ExpertLink>
@@ -97461,7 +97363,7 @@
     <Disorder id="1614">
       <OrphaCode>1416</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1416</ExpertLink>
-      <Name lang="de">Kalzium-Pyrophosphat-Speicherkrankheit, familiäre</Name>
+      <Name lang="de">Familiäre Kalziumpyrophosphat-Ablagerung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -104721,7 +104623,7 @@
     <Disorder id="1673">
       <OrphaCode>1506</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1506</ExpertLink>
-      <Name lang="de">Rippen, dünne - tubuläre Knochen - Dysmorphien</Name>
+      <Name lang="de">Dünne Rippen und Röhrenknochen-Dysmorphie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -109379,7 +109281,7 @@
     <Disorder id="17186">
       <OrphaCode>158061</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=158061</ExpertLink>
-      <Name lang="de">Makrophagen-Aktivierungssyndrom (MAS)</Name>
+      <Name lang="de">Makrophagen-Aktivierungssyndrom</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -114699,7 +114601,7 @@
     <Disorder id="2188">
       <OrphaCode>2337</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2337</ExpertLink>
-      <Name lang="de">Palmoplantarkeratose, nichtepidermolytische</Name>
+      <Name lang="de">Diffuse Palmoplantarkeratose vom Bottnischen Typ</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -118736,7 +118638,7 @@
     <Disorder id="2065">
       <OrphaCode>2172</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2172</ExpertLink>
-      <Name lang="de">Mikrozephalie - Glomerulonephritis - marfanoider Habitus</Name>
+      <Name lang="de">Mikrozephalie-Glomerulonephritis-marfanoider Habitus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -120916,38 +120818,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="19687">
-      <OrphaCode>251883</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=251883</ExpertLink>
-      <Name lang="de">Medulloepitheliom des Zentralnervensystems</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="13673">
-          <Source>RARECARE surveillance of rare cancers in Europe[REG]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="de">Jahresinzidenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="de">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="2087">
       <OrphaCode>2206</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2206</ExpertLink>
@@ -133504,11 +133374,11 @@
       <OrphaCode>261190</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=261190</ExpertLink>
       <Name lang="de">Mikrodeletionssyndrom 15q14</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
+      <DisorderType id="21450">
+        <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="9233">
@@ -137099,7 +136969,7 @@
     <Disorder id="2710">
       <OrphaCode>3015</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3015</ExpertLink>
-      <Name lang="de">Radio-renales Syndrom</Name>
+      <Name lang="de">Radial-renales Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -140518,7 +140388,7 @@
     <Disorder id="20193">
       <OrphaCode>268744</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=268744</ExpertLink>
-      <Name lang="de">Spina bifida cystica</Name>
+      <Name lang="de">Spinale Dysraphie mit posteriorer Meningozele</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -140795,7 +140665,7 @@
     <Disorder id="20203">
       <OrphaCode>268810</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=268810</ExpertLink>
-      <Name lang="de">Meningozele, posteriore</Name>
+      <Name lang="de">Meningozele, isolierte posteriore</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -141656,7 +141526,7 @@
     <Disorder id="20186">
       <OrphaCode>268369</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=268369</ExpertLink>
-      <Name lang="de">Spina bifida aperta</Name>
+      <Name lang="de">Spinale Dysraphie, offene</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -152262,7 +152132,7 @@
     <Disorder id="20244">
       <OrphaCode>269206</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=269206</ExpertLink>
-      <Name lang="de">Isolierte totale Aplasie des Kleinhirnwurmes</Name>
+      <Name lang="de">Isolierte totale Agenesie des Kleinhirnwurmes</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -152586,7 +152456,7 @@
     <Disorder id="20460">
       <OrphaCode>280365</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280365</ExpertLink>
-      <Name lang="de">Laminopathie, lipodystrophe, schwere, autosomal-semidominante</Name>
+      <Name lang="de">Laminopathie, lipodystrophe, schwere, autosomal-dominante</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -152995,7 +152865,7 @@
     <Disorder id="2869">
       <OrphaCode>3214</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3214</ExpertLink>
-      <Name lang="de">Taubheit - Blindheit - Hypopigmentierung</Name>
+      <Name lang="de">Hörverlust-Blindheit-Hypopigmentierung-Syndrom, jemenitischer Typ</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -153370,7 +153240,7 @@
     <Disorder id="2877">
       <OrphaCode>3220</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3220</ExpertLink>
-      <Name lang="de">Schwerhörigkeit - Schmelzhypoplasie - Nageldefekte</Name>
+      <Name lang="de">Schwerhörigkeit-Schmelzhypoplasie-Nageldefekte-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -153566,7 +153436,7 @@
     <Disorder id="2873">
       <OrphaCode>3217</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3217</ExpertLink>
-      <Name lang="de">Schwerhörigkeit - Dündarm-Divertikulose - Neuropathie</Name>
+      <Name lang="de">Schwerhörigkeit-Dündarm-Divertikulose-Neuropathie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -161740,7 +161610,7 @@
     <Disorder id="3155">
       <OrphaCode>2538</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2538</ExpertLink>
-      <Name lang="de">Mikrogastrie - Extremitätenverkürzung</Name>
+      <Name lang="de">Mikrogastrie-Extremitätenreduktionsdefekt-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -165057,7 +164927,7 @@
     <Disorder id="3521">
       <OrphaCode>3240</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3240</ExpertLink>
-      <Name lang="de">ZNS-Kalzifikation-Schwerhörigkeit-tubuläre Azidose-Anämie-Syndrom</Name>
+      <Name lang="de">Früh einsetzende progrediente Leukoenzephalopathie-ZNS-Kalzifikation-Schwerhörigkeit-Sehstörung-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -166333,11 +166203,11 @@
       <OrphaCode>1685</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1685</ExpertLink>
       <Name lang="de">Distomatose</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21436">
+        <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36540">
+        <Name lang="de">Gruppe von Störungen</Name>
       </DisorderGroup>
       <PrevalenceList count="1">
         <Prevalence id="4672">
@@ -167766,7 +167636,7 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <PrevalenceList count="2">
+      <PrevalenceList count="3">
         <Prevalence id="4737">
           <Source>34229749[PMID]</Source>
           <PrevalenceType id="23683">
@@ -167805,6 +167675,25 @@
             <Name lang="de">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="17462">
+          <Source>34229749[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="3566">
@@ -175706,8 +175595,8 @@
       <OrphaCode>2330</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2330</ExpertLink>
       <Name lang="de">Kasabach-Merritt-Syndrom</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21429">
+        <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
@@ -177955,7 +177844,7 @@
     <Disorder id="3425">
       <OrphaCode>980</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=980</ExpertLink>
-      <Name lang="de">Fehlen der Pulomalarterie</Name>
+      <Name lang="de">Fehlen der Pulmonalarterie</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -178965,7 +178854,7 @@
     <Disorder id="18885">
       <OrphaCode>217656</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=217656</ExpertLink>
-      <Name lang="de">Arrhythmogene rechtsventrikuläre Kardiomyopathie, familiäre isolierte</Name>
+      <Name lang="de">Hereditäre isolierte arrhythmogene Kardiomyopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -180363,7 +180252,7 @@
     <Disorder id="3384">
       <OrphaCode>145</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=145</ExpertLink>
-      <Name lang="de">'Brust- und/oder Ovarialkrebssyndrom, hereditäres'</Name>
+      <Name lang="de">Brust- und/oder Ovarialkrebssyndrom, hereditäres</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -181534,7 +181423,7 @@
     <Disorder id="18976">
       <OrphaCode>225123</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=225123</ExpertLink>
-      <Name lang="de">'Hämochromatose, TFR2-Gen-assoziierte'</Name>
+      <Name lang="de">Hämochromatose, TFR2-Gen-assoziierte</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -186848,55 +186737,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="19112">
-      <OrphaCode>228357</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=228357</ExpertLink>
-      <Name lang="de">CLN9-Krankheit</Name>
-      <DisorderType id="21443">
-        <Name lang="de">Ätiologischer Subtyp</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13516">
-          <Source>15349861[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13517">
-          <Source>ORPHANET_15349861[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="19119">
       <OrphaCode>228384</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=228384</ExpertLink>
@@ -187133,38 +186973,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="19105">
-      <OrphaCode>228329</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=228329</ExpertLink>
-      <Name lang="de">CLN1-Krankheit</Name>
-      <DisorderType id="21443">
-        <Name lang="de">Ätiologischer Subtyp</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="13520">
-          <Source>25205113[PMID]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="de">Jahresinzidenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="de">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="de">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="24362">
-            <Name lang="de">Finland</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="19096">
       <OrphaCode>228290</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=228290</ExpertLink>
@@ -190167,38 +189975,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="3611">
-      <OrphaCode>2194</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2194</ExpertLink>
-      <Name lang="de">Anti-HLA-Hyperimmunisierung</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="4785">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="de">Unbekannt</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="19163">
       <OrphaCode>231147</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=231147</ExpertLink>
@@ -191798,7 +191574,7 @@
     <Disorder id="4044">
       <OrphaCode>1899</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1899</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, Arthrochalasie Typ</Name>
+      <Name lang="de">Ehlers-Danlos-Syndrom, Arthrochalasie-Typ</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -197959,8 +197735,8 @@
       <OrphaCode>324307</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=324307</ExpertLink>
       <Name lang="de">Schwere laterale Tibiaverkrümmung-Kleinwuchs-milde Scapula alata-Gesichtsdysmorphie-Syndrom</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21401">
+        <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
@@ -199181,7 +198957,7 @@
     <Disorder id="21679">
       <OrphaCode>319595</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=319595</ExpertLink>
-      <Name lang="de">Suszeptibilität für Mykobakteriosen durch partiellen STAT1-Defekt</Name>
+      <Name lang="de">Suszeptibilität für Mykobakteriosen durch partielle STAT1-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -199671,7 +199447,7 @@
     <Disorder id="21684">
       <OrphaCode>319635</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=319635</ExpertLink>
-      <Name lang="de">Amyloidosis cutis dyschromia</Name>
+      <Name lang="de">Amyloidosis cutis dyschromica</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -201705,55 +201481,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="21505">
-      <OrphaCode>314629</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=314629</ExpertLink>
-      <Name lang="de">CLN11-Krankheit</Name>
-      <DisorderType id="21443">
-        <Name lang="de">Ätiologischer Subtyp</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="9999">
-          <Source>22608501[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10000">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="21504">
       <OrphaCode>314621</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=314621</ExpertLink>
@@ -203132,7 +202859,7 @@
     <Disorder id="22464">
       <OrphaCode>370015</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=370015</ExpertLink>
-      <Name lang="de">'Dysplasie, spondyloepimetaphysäre, Typ Isidor-Toutain'</Name>
+      <Name lang="de">Dysplasie, spondyloepimetaphysäre, Typ Isidor-Toutain</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -205357,7 +205084,7 @@
     <Disorder id="5013">
       <OrphaCode>823</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=823</ExpertLink>
-      <Name lang="de">Spina bifida, isolierte</Name>
+      <Name lang="de">Spina bifida und weitere spinale Dysraphien</Name>
       <DisorderType id="36561">
         <Name lang="de">Kategorie</Name>
       </DisorderType>
@@ -210064,7 +209791,7 @@
     <Disorder id="22110">
       <OrphaCode>352734</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=352734</ExpertLink>
-      <Name lang="de">Albinismus, okulokutaner minimal-pigmentierter, Typ 1</Name>
+      <Name lang="de">Albinismus, okulokutaner , Typ 1, minimal-pigmentierter</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -210113,7 +209840,7 @@
     <Disorder id="22111">
       <OrphaCode>352737</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=352737</ExpertLink>
-      <Name lang="de">Albinismus, okulokutaner, Typ TS</Name>
+      <Name lang="de">Albinismus, okulokutaner, Typ 1, temperaturempfindlicher</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -210159,55 +209886,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="22104">
-      <OrphaCode>352709</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=352709</ExpertLink>
-      <Name lang="de">CLN13-Krankheit</Name>
-      <DisorderType id="21443">
-        <Name lang="de">Ätiologischer Subtyp</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="10280">
-          <Source>23297359[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="de">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="de">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10281">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="22105">
       <OrphaCode>352712</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=352712</ExpertLink>
@@ -213889,7 +213567,7 @@
     <Disorder id="20840">
       <OrphaCode>293173</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=293173</ExpertLink>
-      <Name lang="de">Dermatitis, akute exanthematische generalisierte pustulöse</Name>
+      <Name lang="de">Akute generalisierte exanthematische Pustulose</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -219366,7 +219044,7 @@
     <Disorder id="21502">
       <OrphaCode>314613</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=314613</ExpertLink>
-      <Name lang="de">Growing-Teratom-Syndrom</Name>
+      <Name lang="de">Growing-Teratoma-Syndrom</Name>
       <DisorderType id="21429">
         <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
       </DisorderType>
@@ -220114,7 +219792,7 @@
     <Disorder id="21482">
       <OrphaCode>314404</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=314404</ExpertLink>
-      <Name lang="de">Autosomal-dominante zerebelläre Ataxie mit Schwerhörigkeit und Narkolepsie</Name>
+      <Name lang="de">Syndrom der autosomal-dominanten zerebellären Ataxie mit Schwerhörigkeit und Narkolepsie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -220440,7 +220118,7 @@
     <Disorder id="21474">
       <OrphaCode>314041</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=314041</ExpertLink>
-      <Name lang="de">Marfanoider Habitus - Inguinalhernie - Beschleunigte Knochenalterung</Name>
+      <Name lang="de">Marfanoider Habitus-Inguinalhernie-Beschleunigte Knochenalterung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -222745,7 +222423,7 @@
     <Disorder id="21121">
       <OrphaCode>300501</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=300501</ExpertLink>
-      <Name lang="de">Schmerzhafte orbitale und systemische Neurofibrome - marfanoider Habitus</Name>
+      <Name lang="de">Schmerzhafte orbitale und systemische Neurofibrome-marfanoider Habitus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -224358,7 +224036,7 @@
     <Disorder id="21106">
       <OrphaCode>300298</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=300298</ExpertLink>
-      <Name lang="de">Schwere kongenitale hypochrome Anämie mit beringten Sideroblasten</Name>
+      <Name lang="de">Schwere kongenitale hypochrome Anämie mit Ringsideroblasten</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -225159,7 +224837,7 @@
     <Disorder id="21117">
       <OrphaCode>300382</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=300382</ExpertLink>
-      <Name lang="de">Marfan-Syndrom mit neonataler progeroid-Syndrom ähnlicher Lipodystrophie</Name>
+      <Name lang="de">Lipodystrophie-Syndrom mit marfanoiden und progeroiden Merkmalen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -230807,11 +230485,11 @@
       <OrphaCode>1303</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1303</ExpertLink>
       <Name lang="de">Bronchiolitis obliterans mit obstruktiver Lungenerkrankung</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21436">
+        <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36540">
+        <Name lang="de">Gruppe von Störungen</Name>
       </DisorderGroup>
       <PrevalenceList count="1">
         <Prevalence id="5059">
@@ -231105,38 +230783,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="7037">
-      <OrphaCode>225</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=225</ExpertLink>
-      <Name lang="de">Maternal vererbter Diabetes mit Schwerhörigkeit</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="5069">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="de">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="de">1-9 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.1</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="de">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="24156">
       <OrphaCode>466677</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=466677</ExpertLink>
@@ -237661,7 +237307,7 @@
     <Disorder id="22612">
       <OrphaCode>391372</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=391372</ExpertLink>
-      <Name lang="de">Intelligenzminderung-ausgeprägte Sprachverzögerung-milde Dysmorphien-Syndrom</Name>
+      <Name lang="de">FOXP1-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -240515,7 +240161,7 @@
     <Disorder id="22689">
       <OrphaCode>397618</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=397618</ExpertLink>
-      <Name lang="de">Foveahypoplasie-Sehnervenkreuzung-Dysgenesie des vorderen Augensegmentes-Syndrom</Name>
+      <Name lang="de">Syndrom der Foveahypoplasie mit Anomalie des Chiasma opticum und Dysgenesie des vorderen Augensegmentes</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -240573,7 +240219,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="10970">
-          <Source>24136356[PMID]</Source>
+          <Source>34052541[PMID]</Source>
           <PrevalenceType id="23697">
             <Name lang="de">Cases/families</Name>
           </PrevalenceType>
@@ -240581,7 +240227,7 @@
             <Name lang="de">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>18.0</ValMoy>
+          <ValMoy>250.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="de">Weltweit</Name>
           </PrevalenceGeographic>
@@ -240590,15 +240236,15 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10976">
-          <Source>ORPHANET_24136356[PMID]</Source>
+          <Source>34052541[PMID]_31111319[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="de">Punktprävalenz</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23704">
             <Name lang="de">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
@@ -263511,7 +263157,7 @@
     <Disorder id="26570">
       <OrphaCode>508498</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=508498</ExpertLink>
-      <Name lang="de">Intelligenzminderung-kardiale Anomalien-Kleinwuch-Gelenkinstabilität-Syndrom</Name>
+      <Name lang="de">Intelligenzminderung-kardiale Anomalien-Kleinwuchs-Gelenkinstabilität-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -266059,7 +265705,7 @@
     <Disorder id="25993">
       <OrphaCode>500055</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=500055</ExpertLink>
-      <Name lang="de">Mikrodeletionsyndrom 16p13.2</Name>
+      <Name lang="de">Hao-Fountain-Syndrom durch Mikrodeletion 16p13.2</Name>
       <DisorderType id="21443">
         <Name lang="de">Ätiologischer Subtyp</Name>
       </DisorderType>
@@ -268046,11 +267692,11 @@
       <OrphaCode>679</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=679</ExpertLink>
       <Name lang="de">Papulose, atrophische maligne</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21450">
+        <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="5236">
@@ -271042,7 +270688,7 @@
     <Disorder id="8591">
       <OrphaCode>247</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=247</ExpertLink>
-      <Name lang="de">Arrhythmogene rechtsventrikuläre Kardiomyopathie</Name>
+      <Name lang="de">Hereditäre arrhythmogene Kardiomyopathie</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -275674,7 +275320,7 @@
     <Disorder id="25408">
       <OrphaCode>488618</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=488618</ExpertLink>
-      <Name lang="de">Transketolase Mangel</Name>
+      <Name lang="de">Transketolase-Mangel</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -277447,7 +277093,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="6085">
-          <Source>ORPHANET</Source>
+          <Source>37219052[PMID]</Source>
           <PrevalenceType id="23697">
             <Name lang="de">Cases/families</Name>
           </PrevalenceType>
@@ -277455,7 +277101,7 @@
             <Name lang="de">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>98.0</ValMoy>
+          <ValMoy>558.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="de">Weltweit</Name>
           </PrevalenceGeographic>
@@ -277464,15 +277110,15 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="6086">
-          <Source>ORPHANET</Source>
+          <Source>37219052[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="de">Punktprävalenz</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23704">
             <Name lang="de">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
@@ -278542,7 +278188,7 @@
     <Disorder id="10966">
       <OrphaCode>70595</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=70595</ExpertLink>
-      <Name lang="de">Neuropathie, sensorisch-ataktische - Dysarthrie - Ophthalmoplegie</Name>
+      <Name lang="de">Sensorische ataktische Neuropathie-Dysarthrie-Ophthalmoparese-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -282135,7 +281781,7 @@
     <Disorder id="10881">
       <OrphaCode>66518</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=66518</ExpertLink>
-      <Name lang="de">Insulinresistenz - kurzes 5. Metakarpale</Name>
+      <Name lang="de">Kurzer fünfter Mittelhandknochen-Insulinresistenz-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -282331,7 +281977,7 @@
     <Disorder id="28337">
       <OrphaCode>562639</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562639</ExpertLink>
-      <Name lang="de">Primäre biliäre Cholangitis/primäre sklerosiierende Cholangitis und autoimmune Hepatitis-Übergangsform</Name>
+      <Name lang="de">Primäre biliäre Cholangitis/primäre sklerosierende Cholangitis und autoimmune Hepatitis-Übergangsform</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -283400,57 +283046,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="10862">
-      <OrphaCode>65250</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=65250</ExpertLink>
-      <Name lang="de">Perineuralzyste</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="5903">
-          <Source>[EXPERT]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="de">Jahresinzidenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="de">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="de">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>50.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="de">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="5904">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="de">Unbekannt</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="de">Weltweit</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="28334">
       <OrphaCode>562569</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562569</ExpertLink>
@@ -287327,7 +286922,7 @@
     <Disorder id="11256">
       <OrphaCode>79230</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=79230</ExpertLink>
-      <Name lang="de">'Hämochromatose, HJV oder HAMP-assoziierte '</Name>
+      <Name lang="de">Hämochromatose, HJV oder HAMP-assoziierte</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288946,7 +288541,7 @@
     <Disorder id="28487">
       <OrphaCode>566192</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566192</ExpertLink>
-      <Name lang="de">Kongenitale autosomale-rezessive Thrombozytopenie der kleinen Blutplättchen</Name>
+      <Name lang="de">Kongenitale autosomal-rezessive Thrombozytopenie der kleinen Blutplättchen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -294801,7 +294396,7 @@
     <Disorder id="10440">
       <OrphaCode>37042</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=37042</ExpertLink>
-      <Name lang="de">Immun-Dysregulation-Polyendokrinopathie-Enteropathie-Syndrom, X-chromosomal</Name>
+      <Name lang="de">Immundysregulation-Polyendokrinopathie-Enteropathie-Syndrom, X-chromosomales</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -308556,7 +308151,7 @@
     <Disorder id="11950">
       <OrphaCode>90059</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90059</ExpertLink>
-      <Name lang="de">Akuter sensorineuraler Hörverlust durch akustisches Trauma oder Hörsturz oder operationsbedingten Hörverlust</Name>
+      <Name lang="de">Plötzliche Schallempfindungsschwerhörigkeit</Name>
       <DisorderType id="21429">
         <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
       </DisorderType>
@@ -320133,19 +319728,19 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <PrevalenceList count="3">
+      <PrevalenceList count="2">
         <Prevalence id="6521">
-          <Source>ISBN:801860245[OTHER]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
+          <Source>31920360[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="de">Prävalenz bei Geburt</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="de">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="de">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.042</ValMoy>
+          <ValMoy>0.3</ValMoy>
           <PrevalenceGeographic id="25482">
             <Name lang="de">United States</Name>
           </PrevalenceGeographic>
@@ -320154,38 +319749,19 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="6522">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="de">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="de">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="de">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6523">
-          <Source>9155958[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
+          <Source>31920360[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="de">Prävalenz bei Geburt</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="de">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="de">&lt;1 / 1 000 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="de">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.08</ValMoy>
-          <PrevalenceGeographic id="25475">
-            <Name lang="de">United Kingdom</Name>
+          <ValMoy>1.3</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="de">Europa</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="de">Validated</Name>
@@ -328005,7 +327581,7 @@
     <Disorder id="11691">
       <OrphaCode>85321</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85321</ExpertLink>
-      <Name lang="de">Schwerhörigkeit-Intelligenzminderung-Syndrom vom Typ Martin-Probst</Name>
+      <Name lang="de">Schwerhörigkeit-Intelligenzminderung-Syndrom Typ Martin-Probst</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -332980,7 +332556,7 @@
     <Disorder id="11770">
       <OrphaCode>86880</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=86880</ExpertLink>
-      <Name lang="de">T-Zell-Lymphom vom Enteropathie-Typ</Name>
+      <Name lang="de">Enteropathie-assoziiertes T-Zell-Lymphom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -335239,7 +334815,7 @@
     <Disorder id="11597">
       <OrphaCode>83619</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=83619</ExpertLink>
-      <Name lang="de">Makrostomie - präaurikuläre Anhängsel - externe Ophthalmoplegie</Name>
+      <Name lang="de">Makrostomie-präaurikuläre Anhängsel-externe Ophthalmoplegie-Sndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -336578,7 +336154,7 @@
     <Disorder id="11624">
       <OrphaCode>85128</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85128</ExpertLink>
-      <Name lang="de">Netzhautdystrophie Typ Bothnia</Name>
+      <Name lang="de">Netzhautdystrophie Typ Bottnien</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -337768,6 +337344,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="30477">
+      <OrphaCode>600691</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=600691</ExpertLink>
+      <Name lang="de">Faktor VII und Faktor X, kombinierter Mangel</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17473">
+          <Source>28164683[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>7.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17474">
+          <Source>28164683[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="30478">
       <OrphaCode>600731</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=600731</ExpertLink>
@@ -341042,7 +340667,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="7390">
-          <Source>[EXPERT]</Source>
+          <Source>34237446[PMID]_ORPHANET</Source>
           <PrevalenceType id="23697">
             <Name lang="de">Cases/families</Name>
           </PrevalenceType>
@@ -341050,7 +340675,7 @@
             <Name lang="de">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>50.0</ValMoy>
+          <ValMoy>24.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="de">Weltweit</Name>
           </PrevalenceGeographic>
@@ -341059,7 +340684,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="7391">
-          <Source>ORPHANET</Source>
+          <Source>34237446[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="de">Punktprävalenz</Name>
           </PrevalenceType>
@@ -341326,6 +340951,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="30241">
+      <OrphaCode>600194</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=600194</ExpertLink>
+      <Name lang="de">Gerinnungsstörung, Faktor V-Atlanta</Name>
+      <DisorderType id="21443">
+        <Name lang="de">Ätiologischer Subtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17471">
+          <Source>33773040[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>1.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17472">
+          <Source>33773040[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12942">
       <OrphaCode>97678</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=97678</ExpertLink>
@@ -346193,6 +345867,93 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29929">
+      <OrphaCode>599490</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599490</ExpertLink>
+      <Name lang="de">Faktor V-Mangel, erworbener</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="17451">
+          <Source>27428013[PMID]_36713339[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17452">
+          <Source>27428013[PMID]_36713339[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>200.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17453">
+          <Source>15613918[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="de">Jahresinzidenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="de">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.029</ValMoy>
+          <PrevalenceGeographic id="23949">
+            <Name lang="de">Australia</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17454">
+          <Source>19404538[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="de">Jahresinzidenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="de">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.009</ValMoy>
+          <PrevalenceGeographic id="25244">
+            <Name lang="de">Singapore</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12331">
       <OrphaCode>93409</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93409</ExpertLink>
@@ -346242,6 +346003,251 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29930">
+      <OrphaCode>599495</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599495</ExpertLink>
+      <Name lang="de">Faktor VII-Mangel, erworbener</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17455">
+          <Source>36546749[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>83.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17456">
+          <Source>36546749[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29931">
+      <OrphaCode>599501</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599501</ExpertLink>
+      <Name lang="de">Faktor X-Mangel, erworbener</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17457">
+          <Source>37587057[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>77.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17458">
+          <Source>37587057[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="de">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29932">
+      <OrphaCode>599507</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599507</ExpertLink>
+      <Name lang="de">Faktor XI-Mangel, erworbener</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17463">
+          <Source>18519152[PMID]_29027258[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>15.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17464">
+          <Source>18519152[PMID]_29027258[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29933">
+      <OrphaCode>599513</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599513</ExpertLink>
+      <Name lang="de">Faktor XIII-Mangel, erworbener</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17465">
+          <Source>30446212[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>95.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17466">
+          <Source>30446212[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29934">
+      <OrphaCode>599519</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599519</ExpertLink>
+      <Name lang="de">Gerinnungsstörung, Faktor V kurze-Isoformen-abhängige</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17467">
+          <Source>25634741[PMID]_33773040[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>3.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17468">
+          <Source>25634741[PMID]_33773040[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12334">
       <OrphaCode>93420</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93420</ExpertLink>
@@ -346274,10 +346280,59 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29935">
+      <OrphaCode>599579</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=599579</ExpertLink>
+      <Name lang="de">Gerinnungsstörung, Faktor V-Amsterdam</Name>
+      <DisorderType id="21443">
+        <Name lang="de">Ätiologischer Subtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="17469">
+          <Source>25634741[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="de">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="de">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="17470">
+          <Source>25634741[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="de">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="de">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12291">
       <OrphaCode>93360</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93360</ExpertLink>
-      <Name lang="de">'Dysplasie, spondyloepimetaphysäre, mit multiplen Luxationen, leptodaktyler Typ'</Name>
+      <Name lang="de">Dysplasie, spondyloepimetaphysäre, mit multiplen Luxationen, leptodaktyler Typ</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -349589,7 +349644,7 @@
     <Disorder id="14158">
       <OrphaCode>99141</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=99141</ExpertLink>
-      <Name lang="de">Posteriore Choanalatresie - Lymphödem-Syndrom</Name>
+      <Name lang="de">Lymphödem-posteriore Choanalatresie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -353859,7 +353914,7 @@
     <Disorder id="29430">
       <OrphaCode>589515</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=589515</ExpertLink>
-      <Name lang="de">'PUM1-assoziiertes Syndrom der Entwicklungsstörung, Ataxie und Krampfanfälle'</Name>
+      <Name lang="de">PUM1-assoziiertes Syndrom der Entwicklungsstörung, Ataxie und Krampfanfälle</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -360488,7 +360543,7 @@
     <Disorder id="14030">
       <OrphaCode>99013</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=99013</ExpertLink>
-      <Name lang="de">Spastische Paraplegie, autosomal-rezessive, Typ 7</Name>
+      <Name lang="de">Spastische Paraplegie Typ 7</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
