commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml
old mode 100755
new mode 100644
index ad7a61a..89d72b3
--- a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:43:38" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:46:08" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -267914,7 +267914,7 @@
       <DisorderGroup id="36540">
         <Name lang="de">Gruppe von Störungen</Name>
       </DisorderGroup>
-      <PrevalenceList count="12">
+      <PrevalenceList count="11">
         <Prevalence id="5195">
           <Source>20566927[PMID]</Source>
           <PrevalenceType id="23669">
@@ -267942,10 +267942,10 @@
           <PrevalenceQualification id="23711">
             <Name lang="de">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="de">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="de">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.1</ValMoy>
+          <ValMoy>0.572</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="de">Europa</Name>
           </PrevalenceGeographic>
@@ -268105,27 +268105,8 @@
             <Name lang="de">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="5205">
-          <Source>ISBN:951563203[OTHER]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="de">Punktprävalenz</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="de">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="de">1-9 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.12</ValMoy>
-          <PrevalenceGeographic id="25279">
-            <Name lang="de">South Africa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="de">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
         <Prevalence id="5206">
-          <Source>ISBN:801860245[OTHER]</Source>
+          <Source>PMID: 27463098</Source>
           <PrevalenceType id="23669">
             <Name lang="de">Punktprävalenz</Name>
           </PrevalenceType>
@@ -268135,7 +268116,7 @@
           <PrevalenceClass id="23739">
             <Name lang="de">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.24</ValMoy>
+          <ValMoy>0.14</ValMoy>
           <PrevalenceGeographic id="25482">
             <Name lang="de">United States</Name>
           </PrevalenceGeographic>
@@ -285773,7 +285754,7 @@
     <Disorder id="28590">
       <OrphaCode>567548</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567548</ExpertLink>
-      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom</Name>
+      <Name lang="de">Idiopathisches Steroid-resistentes nephrotisches Syndrom</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
