commit a9304bcd06b429331d3fd652192800e94c21e73c
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml
index 137ae1d..d289547 100644
--- a/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/de_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:37:16" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:40:12" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6042">
+  <DisorderList count="6043">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166024</ExpertLink>
@@ -20836,7 +20836,7 @@
     <Disorder id="238">
       <OrphaCode>126</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=126</ExpertLink>
-      <Name lang="de">Blepharophimose - Epicanthus inversus - Ptosis</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -26513,7 +26513,7 @@
     <Disorder id="17819">
       <OrphaCode>169095</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=169095</ExpertLink>
-      <Name lang="de">Alymphoide zystische Thymus-Dysgenesie</Name>
+      <Name lang="de">Immundefekt, kombinierter schwerer, durch FOXN1-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -62888,7 +62888,7 @@
     <Disorder id="18409">
       <OrphaCode>199354</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=199354</ExpertLink>
-      <Name lang="de">CARASIL</Name>
+      <Name lang="de">Zerebrale Arteriopathie mit subkortikalen Infarkten und Leukoenzephalopathie, autosomal-rezessiv</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -73988,7 +73988,7 @@
     <Disorder id="18313">
       <OrphaCode>183713</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=183713</ExpertLink>
-      <Name lang="de">Pyogene Bakterieninfektion durch MyD88-Mangel</Name>
+      <Name lang="de">Suszeptibilität für Bakterieninfektion durch TLR Signalweg-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -76694,7 +76694,7 @@
     <Disorder id="1001">
       <OrphaCode>136</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=136</ExpertLink>
-      <Name lang="de">CADASIL</Name>
+      <Name lang="de">Zerebrale autosomal-dominante Arteriopathie mit subkortikalen Infarkten und Leukenzephalopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -76898,7 +76898,7 @@
     <Disorder id="1007">
       <OrphaCode>528</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=528</ExpertLink>
-      <Name lang="de">Lipodystrophie Typ Berardinelli</Name>
+      <Name lang="de">Lipodystrophie, kongenitale, Typ Berardinelli</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -99830,7 +99830,7 @@
     <Disorder id="1757">
       <OrphaCode>1671</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1671</ExpertLink>
-      <Name lang="de">Diastematomyelie</Name>
+      <Name lang="de">Split-Cord-Malformation Typ I</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -108610,7 +108610,7 @@
     <Disorder id="17164">
       <OrphaCode>157965</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=157965</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, spondylo-cheiro dysplastische Form</Name>
+      <Name lang="de">SLC39A13-assoziiertes spondylodysplastisches Ehlers-Danlos-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -110855,7 +110855,7 @@
     <Disorder id="2200">
       <OrphaCode>2348</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2348</ExpertLink>
-      <Name lang="de">Lipodystrophie, familiäre partielle, Typ Dunningan</Name>
+      <Name lang="de">Lipodystrophie, familiäre partielle, Typ  Dunnigan</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -114300,6 +114300,38 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="19474">
+      <OrphaCode>244275</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=244275</ExpertLink>
+      <Name lang="de">De-novo thrombotische Mikroangiopathie nach Nierentransplantation</Name>
+      <DisorderType id="21429">
+        <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <PrevalenceList count="1">
+        <Prevalence id="14647">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="de">Punktprävalenz</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="de">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="de">Unbekannt</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="de">Weltweit</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25965">
+            <Name lang="de">Not yet validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19468">
       <OrphaCode>243343</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=243343</ExpertLink>
@@ -118090,7 +118122,7 @@
     <Disorder id="19684">
       <OrphaCode>251870</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=251870</ExpertLink>
-      <Name lang="de">Primitiver neuroektodermaler Tumor des Zentralnervensystems</Name>
+      <Name lang="de">ZNS-Tumor, embryonaler</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -120152,7 +120184,7 @@
     <Disorder id="2121">
       <OrphaCode>2251</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2251</ExpertLink>
-      <Name lang="de">Daumenfehlbildung - Alopezie - Pigmentanomalien</Name>
+      <Name lang="de">Daumenfehlbildung-Alopezie-Pigmentanomalie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -131836,7 +131868,7 @@
     <Disorder id="19882">
       <OrphaCode>261501</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=261501</ExpertLink>
-      <Name lang="de">Norrie-Syndrom, atypisches, durch Monosomie Xp11.3</Name>
+      <Name lang="de">Norrie-Syndrom, atypisches, durch Mikrodeletion Xp11.3</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -154584,7 +154616,7 @@
     <Disorder id="2908">
       <OrphaCode>3253</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3253</ExpertLink>
-      <Name lang="de">Zlotogora-Ogur-Syndrom</Name>
+      <Name lang="de">Lippen-Kiefer-Gaumen-Spalte-ektodermale Dysplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -155176,7 +155208,7 @@
     <Disorder id="3279">
       <OrphaCode>572</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572</ExpertLink>
-      <Name lang="de">Immundefekt durch HLA-Klasse II-Expressionsdefekt</Name>
+      <Name lang="de">Immundefekt durch MHC Klasse II-Expressionsdefekt</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -172615,7 +172647,7 @@
     <Disorder id="18845">
       <OrphaCode>217330</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=217330</ExpertLink>
-      <Name lang="de">Hyperurikämie - Anämie - Nierenversagen</Name>
+      <Name lang="de">Nierenkrankheit, tubulointerstitielle, autosomal-dominante, REN-assoziierte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -175023,7 +175055,7 @@
     <Disorder id="3443">
       <OrphaCode>2041</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2041</ExpertLink>
-      <Name lang="de">Koronararterienfisteln</Name>
+      <Name lang="de">Koronarfistel</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -180482,7 +180514,7 @@
     <Disorder id="3735">
       <OrphaCode>2126</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2126</ExpertLink>
-      <Name lang="de">Tumor, fibröser solitärer</Name>
+      <Name lang="de">Solitärer fibröser Tumor/Hämangioperizytom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -182017,7 +182049,7 @@
     <Disorder id="3713">
       <OrphaCode>2542</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2542</ExpertLink>
-      <Name lang="de">Anophthalmie/Mikrophthalmie-Syndrom, isoliertes</Name>
+      <Name lang="de">Mikrophthalmie-Anophthalmie-Kolobom, isoliert</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -205362,7 +205394,7 @@
     <Disorder id="22195">
       <OrphaCode>357008</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=357008</ExpertLink>
-      <Name lang="de">Hämolytisch-urämisches Syndrom, atypische From mit DGKE-Mangel</Name>
+      <Name lang="de">Hämolytisch-urämisches Syndrom mit DGKE-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -240729,7 +240761,7 @@
     <Disorder id="23367">
       <OrphaCode>438178</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=438178</ExpertLink>
-      <Name lang="de">Schwere Intelligenzminderung-Epilepsie-Katarakt-Syndrom durch Acyl-CoA-Reduktase 1-Mangel</Name>
+      <Name lang="de">Fettsäure Acyl-CoA-Reduktase 1-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -273685,7 +273717,7 @@
     <Disorder id="10999">
       <OrphaCode>71277</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=71277</ExpertLink>
-      <Name lang="de">Enzephalopathie durch GLUT1-Defekt</Name>
+      <Name lang="de">Klassisches Glukosetransporter-Typ-1-Mangel-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -273968,7 +274000,7 @@
     <Disorder id="28179">
       <OrphaCode>558411</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=558411</ExpertLink>
-      <Name lang="de">Idiopathic gastroparesis</Name>
+      <Name lang="de">Gastroparese, idiopathische</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -277921,7 +277953,7 @@
     <Disorder id="28337">
       <OrphaCode>562639</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562639</ExpertLink>
-      <Name lang="de">Primary biliary cholangitis/primary sclerosing cholangitis and autoimmune hepatitis overlap syndrome</Name>
+      <Name lang="de">Primäre biliäre Cholangitis/primäre sklerosiierende Cholangitis und autoimmune Hepatitis-Übergangsform</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -278699,7 +278731,7 @@
     <Disorder id="28329">
       <OrphaCode>562509</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562509</ExpertLink>
-      <Name lang="de">Heme oxygenase-1 deficiency</Name>
+      <Name lang="de">Hämoxygenase 1-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -278863,7 +278895,7 @@
     <Disorder id="28331">
       <OrphaCode>562528</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562528</ExpertLink>
-      <Name lang="de">Congenital limbs-face contractures-hypotonia-developmental delay syndrome</Name>
+      <Name lang="de">Kongenitale Extremitäten- und Gesichtskontrakturen-Hypotonie-Entwicklungsverzögerung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -278944,7 +278976,7 @@
     <Disorder id="28333">
       <OrphaCode>562559</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562559</ExpertLink>
-      <Name lang="de">Anterior maxillary protrusion-strabismus-intellectual disability syndrome</Name>
+      <Name lang="de">Vordere Oberkieferprotrusion-Strabismus-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -279044,7 +279076,7 @@
     <Disorder id="28334">
       <OrphaCode>562569</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562569</ExpertLink>
-      <Name lang="de">TMEM94-associated congenital heart defect-facial dysmorphism-developmental delay syndrome</Name>
+      <Name lang="de">TMEM94-assoziierter kongenitaler Herzdefekt-Gesichtsdysmorphie-Entwicklungsverzögerung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -280549,7 +280581,7 @@
     <Disorder id="28397">
       <OrphaCode>564003</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=564003</ExpertLink>
-      <Name lang="de">Osteochondrosis of the metatarsal bone</Name>
+      <Name lang="de">Osteochondrose des Mittelfußknochens</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -280613,7 +280645,7 @@
     <Disorder id="28396">
       <OrphaCode>563991</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563991</ExpertLink>
-      <Name lang="de">Osteochondrosis of the tarsal bone</Name>
+      <Name lang="de">Osteochondrose des Fußwurzelknochens</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -280677,7 +280709,7 @@
     <Disorder id="28399">
       <OrphaCode>564178</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=564178</ExpertLink>
-      <Name lang="de">Primary hypomagnesemia with refractory seizures and intellectual deficiency</Name>
+      <Name lang="de">Primäre Hypomagnesiämie mit refraktären Krämpfen und Intelligenzminderung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -280937,7 +280969,7 @@
     <Disorder id="28372">
       <OrphaCode>563690</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563690</ExpertLink>
-      <Name lang="de">Furuncular myiasis due to Cordylobia rodhaini</Name>
+      <Name lang="de">Fliegenmadenbefall durch Cordylobia rodhaini</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -281020,7 +281052,7 @@
     <Disorder id="28370">
       <OrphaCode>563684</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563684</ExpertLink>
-      <Name lang="de">Furuncular myiasis due to Dermatobia hominis</Name>
+      <Name lang="de">Furunkuloide Myiasis durch Dermatobia hominis</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -281052,7 +281084,7 @@
     <Disorder id="28371">
       <OrphaCode>563687</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563687</ExpertLink>
-      <Name lang="de">Furuncular myiasis due to Cordylobia anthropophaga</Name>
+      <Name lang="de">Furunkuloide Myiasis durch Cordylobia anthropophaga</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -282180,7 +282212,7 @@
     <Disorder id="28451">
       <OrphaCode>565858</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565858</ExpertLink>
-      <Name lang="de">Craniosynostosis-microretrognathia-several intellectual disability syndrome</Name>
+      <Name lang="de">Kraniosynostose-Mikroretrognathie-schwere Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -282229,7 +282261,7 @@
     <Disorder id="28448">
       <OrphaCode>565788</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565788</ExpertLink>
-      <Name lang="de">Infantile inflammatory bowel disease with neurological involvement</Name>
+      <Name lang="de">Infantile chronisch-entzündliche Darmerkrankung mit neurologischer Beteiligung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -282671,7 +282703,7 @@
     <Disorder id="28452">
       <OrphaCode>565899</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565899</ExpertLink>
-      <Name lang="de">POMGNT2-related limb-girdle muscular dystrophy R24</Name>
+      <Name lang="de">POMGNT2-assoziierte Gliedergürtelmuskeldystrophie R24</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -282720,7 +282752,7 @@
     <Disorder id="28453">
       <OrphaCode>565909</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565909</ExpertLink>
-      <Name lang="de">Calpain-3-related limb-girdle muscular dystrophy D4</Name>
+      <Name lang="de">Calpain 3-assoziierte Gliedergürtelmuskeldystrophie D4</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -282882,7 +282914,7 @@
     <Disorder id="28477">
       <OrphaCode>566067</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566067</ExpertLink>
-      <Name lang="de">CEBPE-associated autoinflammation-immunodeficiency-neutrophil dysfunction syndrome</Name>
+      <Name lang="de">CEBPE-assoziierte Autoinflammation-Immundefekt- neutrophile Funktionsstörung-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -283103,7 +283135,7 @@
     <Disorder id="28443">
       <OrphaCode>565624</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565624</ExpertLink>
-      <Name lang="de">Combined oxidative phosphorylation defect type 39</Name>
+      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung Typ 39</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -283152,7 +283184,7 @@
     <Disorder id="28442">
       <OrphaCode>565612</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565612</ExpertLink>
-      <Name lang="de">Triglyceride deposit cardiomyovasculopathy</Name>
+      <Name lang="de">Triglycerid-Speicher-Kardiomyovaskulopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -283201,7 +283233,7 @@
     <Disorder id="28446">
       <OrphaCode>565782</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565782</ExpertLink>
-      <Name lang="de">Methotrexate toxicity</Name>
+      <Name lang="de">Methotrexat-Toxizität</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -283945,7 +283977,7 @@
     <Disorder id="28493">
       <OrphaCode>566231</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566231</ExpertLink>
-      <Name lang="de">Resistance to thyroid hormone due to a mutation in thyroid hormone receptor alpha</Name>
+      <Name lang="de">Resistenz gegen Schilddrüsenhormone durch Mutation im Thyroidhormonrezeptor alpha</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -284043,7 +284075,7 @@
     <Disorder id="28494">
       <OrphaCode>566243</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566243</ExpertLink>
-      <Name lang="de">Resistance to thyroid hormone due to a mutation in thyroid hormone receptor beta</Name>
+      <Name lang="de">Resistenz gegen Schilddrüsenhormone durch Mutation im Thyroidhormonrezeptor beta</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -284388,7 +284420,7 @@
     <Disorder id="28487">
       <OrphaCode>566192</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566192</ExpertLink>
-      <Name lang="de">Congenital autosmal recessive small-platelet thrombocytopenia</Name>
+      <Name lang="de">Kongenitale autosmale-rezessive Thrombozytopenie der kleinen Blutplättchen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -284567,7 +284599,7 @@
     <Disorder id="28482">
       <OrphaCode>566175</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566175</ExpertLink>
-      <Name lang="de">Complement Hyperactivation - angiopathic thrombosis and Protein-losing enteropathy</Name>
+      <Name lang="de">Komplement-Hyperaktivierung-angiopathische Thrombose-Eiweißverlustenteropathie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -285782,7 +285814,7 @@
     <Disorder id="28585">
       <OrphaCode>567502</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567502</ExpertLink>
-      <Name lang="de">B-cell immunodeficiency-limb abnormalities-urogenital malformations syndrome</Name>
+      <Name lang="de">B-Zell-Immundefekt-Extremitätenanomalien-urogenitale Fehlbildungen-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -285929,7 +285961,7 @@
     <Disorder id="28591">
       <OrphaCode>567550</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567550</ExpertLink>
-      <Name lang="de">Idiopathic multidrug-resistant nephrotic syndrome</Name>
+      <Name lang="de">Idiopathisches MDR-resistentes nephrotisches Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -285961,7 +285993,7 @@
     <Disorder id="28590">
       <OrphaCode>567548</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567548</ExpertLink>
-      <Name lang="de">Idiopathic steroid-resistant nephrotic syndrome</Name>
+      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -285993,7 +286025,7 @@
     <Disorder id="28589">
       <OrphaCode>567546</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567546</ExpertLink>
-      <Name lang="de">Idiopathic steroid-sensitive nephrotic syndrome with secondary steroid resistance</Name>
+      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom mit sekundärer Steroid-Resistenz</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -286025,7 +286057,7 @@
     <Disorder id="28588">
       <OrphaCode>567544</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567544</ExpertLink>
-      <Name lang="de">Idiopathic non-lupus full-house nephropathy</Name>
+      <Name lang="de">Idiopathische non-Lupus Full-House-Nephropathie</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -286387,7 +286419,7 @@
     <Disorder id="28606">
       <OrphaCode>567983</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567983</ExpertLink>
-      <Name lang="de">Parenteral nutrition-associated cholestasis</Name>
+      <Name lang="de">Parenterale Ernährung-assoziierte Cholestase</Name>
       <DisorderType id="21429">
         <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
       </DisorderType>
@@ -286958,7 +286990,7 @@
     <Disorder id="28552">
       <OrphaCode>566841</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566841</ExpertLink>
-      <Name lang="de">Liver adenomatosis</Name>
+      <Name lang="de">Adenomatose der Leber</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -287039,7 +287071,7 @@
     <Disorder id="11082">
       <OrphaCode>75392</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=75392</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, Periodontose Typ</Name>
+      <Name lang="de">Ehlers-Danlos-Syndrom, parodontaler Typ</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -287088,7 +287120,7 @@
     <Disorder id="11083">
       <OrphaCode>75496</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=75496</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, progeroider Typ</Name>
+      <Name lang="de">B4GALT7-assoziiertes spondylodysplastisches Ehlers-Danlos-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -287267,7 +287299,7 @@
     <Disorder id="28560">
       <OrphaCode>566943</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566943</ExpertLink>
-      <Name lang="de">Mueller-Weiss syndrome</Name>
+      <Name lang="de">Mueller-Weiss-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288205,7 +288237,7 @@
     <Disorder id="28655">
       <OrphaCode>569821</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569821</ExpertLink>
-      <Name lang="de">Congenital primary lymphedema of Gordon</Name>
+      <Name lang="de">Kongenitales primäres Lymphödem vom Typ Gordon</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288286,7 +288318,7 @@
     <Disorder id="28654">
       <OrphaCode>569816</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569816</ExpertLink>
-      <Name lang="de">CELSR1-related late-onset primary lymphedema</Name>
+      <Name lang="de">CELSR1-assoziiertes spät-einsetzendes primäres Lymphödem</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288737,7 +288769,7 @@
     <Disorder id="28613">
       <OrphaCode>568065</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568065</ExpertLink>
-      <Name lang="de">EPHB4-related fetal hydrops with generalized lymphatic dysplasia</Name>
+      <Name lang="de">EPHB4-assoziierter fetaler Hydrops mit generalisierter lymphatischer Dysplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288786,7 +288818,7 @@
     <Disorder id="28612">
       <OrphaCode>568062</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568062</ExpertLink>
-      <Name lang="de">PIEZO1-related fetal hydrops with generalized lymphatic dysplasia</Name>
+      <Name lang="de">PIEZO1-assoziierter fetaler Hydrops mit generalisierter lymphatischer Dysplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288899,7 +288931,7 @@
     <Disorder id="28611">
       <OrphaCode>568056</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568056</ExpertLink>
-      <Name lang="de">Warts-immunodeficiency-lymphoedema-anogenital dysplasia syndrome</Name>
+      <Name lang="de">Warzen-Immundefekt-Lymphödem-anogenitale Dysplasie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -288948,7 +288980,7 @@
     <Disorder id="28610">
       <OrphaCode>568051</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568051</ExpertLink>
-      <Name lang="de">GJC2-related congenital primary lymphedema</Name>
+      <Name lang="de">GJC2-assoziiertes spät-beginnendes primäres Lymphödem</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -289178,7 +289210,7 @@
     <Disorder id="28622">
       <OrphaCode>569164</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569164</ExpertLink>
-      <Name lang="de">Angiomatoid fibrous histiocytoma</Name>
+      <Name lang="de">Histiozytom, angiomatöses fibröses</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -289340,7 +289372,7 @@
     <Disorder id="28629">
       <OrphaCode>569274</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569274</ExpertLink>
-      <Name lang="de">Multiple mitochondrial dysfunctions syndrome type 5</Name>
+      <Name lang="de">Mitochondriales Dysfunktions-Syndrom Typ 5</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -289389,7 +289421,7 @@
     <Disorder id="28631">
       <OrphaCode>569290</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569290</ExpertLink>
-      <Name lang="de">Multiple mitochondrial dysfunctions syndrome type 6</Name>
+      <Name lang="de">Mitochondriales Dysfunktions-Syndrom Typ 6</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -289438,7 +289470,7 @@
     <Disorder id="28626">
       <OrphaCode>569248</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569248</ExpertLink>
-      <Name lang="de">Microcystic stromal tumor</Name>
+      <Name lang="de">Mikrozystischer Stromatumor</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -290815,7 +290847,7 @@
     <Disorder id="10395">
       <OrphaCode>35706</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=35706</ExpertLink>
-      <Name lang="de">Glutaryl-CoA-Oxidase-Mangel</Name>
+      <Name lang="de">Glutarazidurie Typ 3</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -295528,7 +295560,7 @@
     <Disorder id="10349">
       <OrphaCode>34592</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=34592</ExpertLink>
-      <Name lang="de">Immundefekt durch HLA-Klasse-I-Expressionsdefekt</Name>
+      <Name lang="de">Immundefekt durch MHC Klasse I-Expressionsdefekt</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -298989,7 +299021,7 @@
     <Disorder id="10649">
       <OrphaCode>50810</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=50810</ExpertLink>
-      <Name lang="de">Mikrolissenzephalie - Mikromelie</Name>
+      <Name lang="de">Mikrolissenzephalie-Mikromelie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -303345,7 +303377,7 @@
     <Disorder id="28139">
       <OrphaCode>556985</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=556985</ExpertLink>
-      <Name lang="de">Early-onset calcifying  leukoencephalopathy-skeletal dysplasia</Name>
+      <Name lang="de">Früh-einsetzende kalzifizierende Leukenzephalopathie mit Skelettdysplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -303394,7 +303426,7 @@
     <Disorder id="28138">
       <OrphaCode>556955</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=556955</ExpertLink>
-      <Name lang="de">Pancreatic agenesis-holoprosencephaly syndrome</Name>
+      <Name lang="de">Pankreasagenesie-Holoprosenzephalie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -303443,7 +303475,7 @@
     <Disorder id="28142">
       <OrphaCode>557064</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=557064</ExpertLink>
-      <Name lang="de">Neonatal epileptic encephalopathy due to glutaminase deficiency</Name>
+      <Name lang="de">Neonatale epileptische Enzephalopathie durch Glutaminase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -303492,7 +303524,7 @@
     <Disorder id="28141">
       <OrphaCode>557056</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=557056</ExpertLink>
-      <Name lang="de">Spastic ataxia-dysarthria due to glutaminase deficiency</Name>
+      <Name lang="de">Spastische Ataxie mit Dysarthrie durch Glutaminase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -304428,7 +304460,7 @@
     <Disorder id="11925">
       <OrphaCode>90023</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90023</ExpertLink>
-      <Name lang="de">Infektionen, rekurrente - Kleinwuchs - Hypopigmentation - grobe Gesichtszüge</Name>
+      <Name lang="de">Primäres Immundefekt-Syndrom durch LAMTOR2-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -304477,7 +304509,7 @@
     <Disorder id="11921">
       <OrphaCode>90003</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90003</ExpertLink>
-      <Name lang="de">Hepatopathie, IgG4-assoziierte</Name>
+      <Name lang="de">Pseudotumor der Leber, inflammatorischer</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -304558,7 +304590,7 @@
     <Disorder id="11935">
       <OrphaCode>90038</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90038</ExpertLink>
-      <Name lang="de">Hämolytisch-urämisches Syndrom, typisches</Name>
+      <Name lang="de">Shiga-ähnliches Toxin-assoziiertes Hämolytisch-urämisches Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -324237,7 +324269,7 @@
     <Disorder id="11709">
       <OrphaCode>85408</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85408</ExpertLink>
-      <Name lang="de">Arthritis, idiopathische juvenile, Rheumafaktor-negative</Name>
+      <Name lang="de">Arthritis, idiopathische juvenile polyartikuläre, Rheumafaktor-negative</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -324484,7 +324516,7 @@
     <Disorder id="11712">
       <OrphaCode>85435</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85435</ExpertLink>
-      <Name lang="de">Arthritis, idiopathische juvenile, Rheumafaktor-positive</Name>
+      <Name lang="de">Arthritis, idiopathische juvenile polyartikuläre, Rheumafaktor-positive</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -332977,7 +333009,7 @@
     <Disorder id="12914">
       <OrphaCode>97363</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=97363</ExpertLink>
-      <Name lang="de">Nierendysplasie, multizystische, unilaterale From</Name>
+      <Name lang="de">Nierendysplasie, multizystische, unilaterale Form</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -334462,7 +334494,7 @@
     <Disorder id="12663">
       <OrphaCode>95708</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=95708</ExpertLink>
-      <Name lang="de">Pubertät, vorzeitige</Name>
+      <Name lang="de">Seltene Form der vorzeitigen Pubertät</Name>
       <DisorderType id="36561">
         <Name lang="de">Category</Name>
       </DisorderType>
@@ -338050,7 +338082,7 @@
     <Disorder id="12290">
       <OrphaCode>93359</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93359</ExpertLink>
-      <Name lang="de">Dysplasie, spondyloepimetaphysäre - Überstreckbarkeit der Gelenke</Name>
+      <Name lang="de">Spondyloepimetaphysäre Dysplasie mit Überstreckbarkeit der Gelenke</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -340221,7 +340253,7 @@
     <Disorder id="12466">
       <OrphaCode>93685</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93685</ExpertLink>
-      <Name lang="de">Castleman-Krankheit, lokalisierte</Name>
+      <Name lang="de">Castleman-Krankheit, unizentrische</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -353709,7 +353741,7 @@
     <Disorder id="28815">
       <OrphaCode>573253</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=573253</ExpertLink>
-      <Name lang="de">Split cord malformation type II</Name>
+      <Name lang="de">Split-Cord-Malformation Typ II</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -353741,7 +353773,7 @@
     <Disorder id="28824">
       <OrphaCode>574957</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=574957</ExpertLink>
-      <Name lang="de">Autosomal recessive mendelian susceptibility to mycobacterial diseases due to partial JAK1 deficiency</Name>
+      <Name lang="de">Autosomal-rezessive Mendelsche Anfälligkeit für Erkrankungen durch Mykobakterien durch partiellen JAK1-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -353773,7 +353805,7 @@
     <Disorder id="28823">
       <OrphaCode>574918</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=574918</ExpertLink>
-      <Name lang="de">Predisposition to severe viral infection due to IRF7 deficiency</Name>
+      <Name lang="de">Prädisposition zur schweren Virusinfektion durch IRF7-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -353911,7 +353943,7 @@
     <Disorder id="28816">
       <OrphaCode>573278</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=573278</ExpertLink>
-      <Name lang="de">Split cord malformation</Name>
+      <Name lang="de">Split-Cord-Malformation</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -353975,7 +354007,7 @@
     <Disorder id="28738">
       <OrphaCode>572385</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572385</ExpertLink>
-      <Name lang="de">Brachydactyly type B1</Name>
+      <Name lang="de">Brachydaktylie Typ B1</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354007,7 +354039,7 @@
     <Disorder id="28736">
       <OrphaCode>572361</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572361</ExpertLink>
-      <Name lang="de">Blepharophimosis-ptosis-epicanthus inversus syndrome type 2</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom Typ 2</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354071,7 +354103,7 @@
     <Disorder id="28746">
       <OrphaCode>572428</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572428</ExpertLink>
-      <Name lang="de">Infantile-onset pulmonary alveolar proteinosis-hypogammaglobulinemia</Name>
+      <Name lang="de">Pulmonale Alveolarproteinose-Hypogammaglobulinämie, infantile Form</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -354169,7 +354201,7 @@
     <Disorder id="28754">
       <OrphaCode>572543</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572543</ExpertLink>
-      <Name lang="de">RFVT2-related riboflavin transporter deficiency</Name>
+      <Name lang="de">RFVT2-assoziierte Riboflavin-Transporter-Defizienz</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354201,7 +354233,7 @@
     <Disorder id="28755">
       <OrphaCode>572550</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572550</ExpertLink>
-      <Name lang="de">RFVT3-related riboflavin transporter deficiency</Name>
+      <Name lang="de">RFVT3-assoziierte Riboflavin-Transporter-Defizienz</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354233,7 +354265,7 @@
     <Disorder id="28766">
       <OrphaCode>572773</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572773</ExpertLink>
-      <Name lang="de">Microcephaly-short stature-limb abnormalities syndrome</Name>
+      <Name lang="de">Mikrozphalie-Kleinwuchs-Extremitätenanomalien-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354282,7 +354314,7 @@
     <Disorder id="28767">
       <OrphaCode>572798</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572798</ExpertLink>
-      <Name lang="de">WARS2-related combined oxidative phosphorylation defect</Name>
+      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung, WARS2-assoziierter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -354331,7 +354363,7 @@
     <Disorder id="28764">
       <OrphaCode>572761</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572761</ExpertLink>
-      <Name lang="de">DONSON-related microcephaly-short stature-limb abnormalities spectrum</Name>
+      <Name lang="de">DONSON-assoziierte Mikrozphalie-Kleinwuchs-Extremitätenanomalien-Spektrum</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -354361,7 +354393,7 @@
     <Disorder id="28765">
       <OrphaCode>572768</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572768</ExpertLink>
-      <Name lang="de">Microcephaly-micromelia syndrome</Name>
+      <Name lang="de">Mikrozephalie-Mikromelie-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354442,7 +354474,7 @@
     <Disorder id="28726">
       <OrphaCode>572013</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572013</ExpertLink>
-      <Name lang="de">Posterior-predominant lissencephaly-broad flat pons and medulla-midline crossing defects syndrome</Name>
+      <Name lang="de">Vorwiegend posteriore Lissenzephalie mit breiter flacher Pons und Medulla-Mittelliniendefekt</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -354491,7 +354523,7 @@
     <Disorder id="28732">
       <OrphaCode>572333</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572333</ExpertLink>
-      <Name lang="de">Blepharophimosis-ptosis-epicanthus inversus syndrome plus</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom plus</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -354523,7 +354555,7 @@
     <Disorder id="28735">
       <OrphaCode>572354</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572354</ExpertLink>
-      <Name lang="de">Blepharophimosis-ptosis-epicanthus inversus syndrome type 1</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom Typ 1</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354555,7 +354587,7 @@
     <Disorder id="28678">
       <OrphaCode>570762</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570762</ExpertLink>
-      <Name lang="de">Infective endocariditis</Name>
+      <Name lang="de">Endokarditis, infektiöse</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -354644,7 +354676,7 @@
     <Disorder id="28677">
       <OrphaCode>570491</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570491</ExpertLink>
-      <Name lang="de">QRSL1-related combined oxidative phosphorylation defect</Name>
+      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung, QRSL1-assoziierter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -354676,7 +354708,7 @@
     <Disorder id="28676">
       <OrphaCode>570470</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570470</ExpertLink>
-      <Name lang="de">Ricin poisoning</Name>
+      <Name lang="de">Rizin-Vergiftung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -354727,7 +354759,7 @@
     <Disorder id="28675">
       <OrphaCode>570438</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570438</ExpertLink>
-      <Name lang="de">HHV-8-associated multicentric Castleman disease</Name>
+      <Name lang="de">Castleman-Krankheit, multizentrische, HHV-8-assoziierte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354759,7 +354791,7 @@
     <Disorder id="28674">
       <OrphaCode>570431</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570431</ExpertLink>
-      <Name lang="de">Multicentric idiopathic Castleman disease</Name>
+      <Name lang="de">Castleman-Krankheit, multizentrische, idiopathische</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -354810,7 +354842,7 @@
     <Disorder id="28673">
       <OrphaCode>570422</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570422</ExpertLink>
-      <Name lang="de">Galactose mutarotase deficiency</Name>
+      <Name lang="de">Galaktose-Mutarotase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -354880,7 +354912,7 @@
     <Disorder id="28672">
       <OrphaCode>570371</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570371</ExpertLink>
-      <Name lang="de">Transient antenatal Bartter syndrome</Name>
+      <Name lang="de">Bartter -Syndrom, antenatales transientes</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
