commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml
old mode 100644
new mode 100755
index 91511ca..9b52a46
--- a/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:40:24" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:41:22" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -31575,7 +31575,7 @@
     <Disorder id="421">
       <OrphaCode>1146</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-      <Name lang="en">Digitotalar dysmorphism</Name>
+      <Name lang="en">Distal arthrogryposis type 1</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -48342,8 +48342,8 @@
       <OrphaCode>178320</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=178320</ExpertLink>
       <Name lang="en">Acute lung injury</Name>
-      <DisorderType id="21422">
-        <Name lang="en">Clinical syndrome</Name>
+      <DisorderType id="21429">
+        <Name lang="en">Particular clinical situation in a disease or syndrome</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
@@ -125499,6 +125499,95 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="19728">
+      <OrphaCode>252175</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=252175</ExpertLink>
+      <Name lang="en">Vestibular schwannoma</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="16415">
+          <Source>20871439[PMID]_24655069[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="en">Annual incidence</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="en">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="en">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="24264">
+            <Name lang="en">Denmark</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="en">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16416">
+          <Source>National Cancer Institute[INST]_23432451[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="en">Annual incidence</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="en">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="en">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>1.2</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="en">United States</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="en">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16417">
+          <Source>21897393[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="en">Annual incidence</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="en">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="en">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.8</ValMoy>
+          <PrevalenceGeographic id="23851">
+            <Name lang="en">Specific population</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="en">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16418">
+          <Source/>
+          <PrevalenceType id="23676">
+            <Name lang="en">Annual incidence</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="en">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="en">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="en">Europe</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25965">
+            <Name lang="en">Not yet validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19756">
       <OrphaCode>254516</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=254516</ExpertLink>
@@ -180514,7 +180603,7 @@
     <Disorder id="3735">
       <OrphaCode>2126</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2126</ExpertLink>
-      <Name lang="en">Solitary fibrous tumour/hemangiopericytoma</Name>
+      <Name lang="en">Solitary fibrous tumor/hemangiopericytoma</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -196362,7 +196451,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-      <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+      <Name lang="en">Alazami syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -203775,7 +203864,7 @@
     <Disorder id="22327">
       <OrphaCode>363611</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-      <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+      <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -207238,8 +207327,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
       <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -208540,55 +208629,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293848</ExpertLink>
-      <Name lang="en">Frontotemporal dementia, right temporal atrophy variant</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="9768">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23697">
-            <Name lang="en">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="en">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>200.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10677">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="en">Point prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="en">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="en">Unknown</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="en">Europe</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="en">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293843</ExpertLink>
@@ -221467,7 +221507,7 @@
     <Disorder id="24042">
       <OrphaCode>464738</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-      <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+      <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -221987,7 +222027,7 @@
     <Disorder id="24022">
       <OrphaCode>464359</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464359</ExpertLink>
-      <Name lang="en">Benign metanephric tumour</Name>
+      <Name lang="en">Benign metanephric tumor</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -261528,7 +261568,7 @@
     <Disorder id="26002">
       <OrphaCode>500163</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=500163</ExpertLink>
-      <Name lang="en">SIN3A-related intellectual disability syndrome</Name>
+      <Name lang="en">Witteveen-Kolk syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -273836,7 +273876,7 @@
     <Disorder id="10986">
       <OrphaCode>71211</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=71211</ExpertLink>
-      <Name lang="en">Neuromyelitis optica</Name>
+      <Name lang="en">Neuromyelitis optica spectrum disorder</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
