commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml
index 11b3403..4c5528a 100644
--- a/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 05:00:38" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:57:44" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6063">
+  <DisorderList count="6058">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -8133,16 +8133,16 @@
           <PrevalenceClass id="23739">
             <Name lang="en">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.22</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="en">Europe</Name>
+          <ValMoy>6.66</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="en">Worldwide</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="en">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="en">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="293">
-          <Source>9918480[PMID]</Source>
+          <Source>35024668[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="en">Prevalence at birth</Name>
           </PrevalenceType>
@@ -8152,7 +8152,7 @@
           <PrevalenceClass id="23739">
             <Name lang="en">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.85</ValMoy>
+          <ValMoy>6.96</ValMoy>
           <PrevalenceGeographic id="23949">
             <Name lang="en">Australia</Name>
           </PrevalenceGeographic>
@@ -21833,7 +21833,7 @@
     <Disorder id="244">
       <OrphaCode>207</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=207</ExpertLink>
-      <Name lang="en">Crouzon disease</Name>
+      <Name lang="en">Crouzon syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -74182,14 +74182,14 @@
     <Disorder id="946">
       <OrphaCode>3027</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3027</ExpertLink>
-      <Name lang="en">Caudal regression sequence</Name>
+      <Name lang="en">Caudal regression syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="2539">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -74198,12 +74198,50 @@
           <PrevalenceQualification id="23704">
             <Name lang="en">Class only</Name>
           </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="en">Unknown</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="en">Worldwide</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="en">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16781">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="en">Prevalence at birth</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="en">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="en">Unknown</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="en">Worldwide</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="en">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16782">
+          <Source>20734338[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="en">Prevalence at birth</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="en">Value and class</Name>
+          </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="en">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="en">Europe</Name>
+          <ValMoy>1.75</ValMoy>
+          <PrevalenceGeographic id="23949">
+            <Name lang="en">Australia</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="en">Validated</Name>
@@ -82623,38 +82661,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=946</ExpertLink>
-      <Name lang="en">Acrocephalosyndactyly</Name>
-      <DisorderType id="21436">
-        <Name lang="en">Clinical group</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="en">Group of disorders</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="2783">
-          <Source>ORPHANET_[EXPERT]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="en">Point prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="en">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="en">Unknown</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="en">Europe</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=957</ExpertLink>
@@ -99304,55 +99310,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1533</ExpertLink>
-      <Name lang="en">Craniosynostosis-fibular aplasia syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="3180">
-          <Source>4403018[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="en">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="en">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="3181">
-          <Source>ORPHANET_4403018[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="en">Point prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="en">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="en">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140874</ExpertLink>
@@ -101183,7 +101140,7 @@
     <Disorder id="1684">
       <OrphaCode>1517</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1517</ExpertLink>
-      <Name lang="en">Hypertrichotic osteochondrodysplasia, Cantu type</Name>
+      <Name lang="en">Cantú syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -123784,7 +123741,7 @@
     <Disorder id="2497">
       <OrphaCode>661</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=661</ExpertLink>
-      <Name lang="en">Ondine syndrome</Name>
+      <Name lang="en">Congenital central hypoventilation syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -170795,7 +170752,7 @@
     <Disorder id="18827">
       <OrphaCode>217253</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217253</ExpertLink>
-      <Name lang="en">Limbic encephalitis with NMDA receptor antibodies</Name>
+      <Name lang="en">NMDA receptor encephalitis</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -190778,55 +190735,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329341</ExpertLink>
-      <Name lang="en">Limbic encephalitis with DPP6 antibodies</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="10180">
-          <Source>23225603[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="en">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="en">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10181">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="en">Point prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="en">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="en">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329228</ExpertLink>
@@ -214948,7 +214856,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314555</ExpertLink>
-      <Name lang="en">Facial dysmorphism-severe myopia-osteopenia-intellectual disability-dental anomalies syndrome</Name>
+      <Name lang="en">Facial dysmorphism-ocular anomalies-osteopenia-intellectual disability-dental anomalies syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -265668,55 +265576,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=498700</ExpertLink>
-      <Name lang="en">Limbic encephalitis with neurexin-3 antibodies</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13782">
-          <Source>27170573[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="en">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="en">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13783">
-          <Source>ORPHANET_27170573[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="en">Point prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="en">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="en">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="en">Worldwide</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="en">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=874</ExpertLink>
@@ -279297,38 +279156,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63440</ExpertLink>
-      <Name lang="en">OBSOLETE: Isolated oxycephaly</Name>
-      <DisorderType id="21415">
-        <Name lang="en">Morphological anomaly</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="5872">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="en">Point prevalence</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="en">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="en">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="en">Europe</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="en">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63442</ExpertLink>
@@ -331832,7 +331659,7 @@
     <Disorder id="12896">
       <OrphaCode>97340</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97340</ExpertLink>
-      <Name lang="en">Hunter-McAlpine craniosynostosis</Name>
+      <Name lang="en">Hunter-McAlpine syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
