commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml
old mode 100644
new mode 100755
index fd73b99..da16e7e
--- a/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/en_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:42:48" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:43:51" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -65806,7 +65806,7 @@
     <Disorder id="891">
       <OrphaCode>1942</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1942</ExpertLink>
-      <Name lang="en">Myoclonic-astastic epilepsy</Name>
+      <Name lang="en">Myoclonic-astatic epilepsy</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -65936,7 +65936,7 @@
     <Disorder id="894">
       <OrphaCode>3451</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3451</ExpertLink>
-      <Name lang="en">West syndrome</Name>
+      <Name lang="en">Infantile spasms syndrome</Name>
       <DisorderType id="21422">
         <Name lang="en">Clinical syndrome</Name>
       </DisorderType>
@@ -77849,7 +77849,7 @@
     <Disorder id="972">
       <OrphaCode>776</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=776</ExpertLink>
-      <Name lang="en">X-linked intellectual disability with marfanoid habitus</Name>
+      <Name lang="en">Lujan-Fryns syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -129663,12 +129663,12 @@
     <Disorder id="19853">
       <OrphaCode>261144</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261144</ExpertLink>
-      <Name lang="en">14q12 microdeletion syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
+      <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="9229">
@@ -228140,7 +228140,7 @@
     <Disorder id="24223">
       <OrphaCode>468631</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468631</ExpertLink>
-      <Name lang="en">Microcephalic primordial dwarfism due to RTTN deficiency</Name>
+      <Name lang="en">Microcephalic cortical malformations-short stature due to RTTN deficiency</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -241909,7 +241909,7 @@
     <Disorder id="23427">
       <OrphaCode>442835</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=442835</ExpertLink>
-      <Name lang="en">Undetermined early-onset epileptic encephalopathy</Name>
+      <Name lang="en">Non-specific early-onset epileptic encephalopathy</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -258956,7 +258956,7 @@
     <Disorder id="26573">
       <OrphaCode>508512</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=508512</ExpertLink>
-      <Name lang="en">Congenital multiple café-au-lait macules-increased sister chromatid exchange syndrome</Name>
+      <Name lang="en">Intrauterine growth restriction-congenital multiple café-au-lait macules-increased sister chromatid exchange syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -342519,11 +342519,11 @@
       <OrphaCode>98813</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98813</ExpertLink>
       <Name lang="en">Hypohidrotic ectodermal dysplasia with immunodeficiency</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="7562">
@@ -351965,8 +351965,8 @@
       <OrphaCode>98791</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98791</ExpertLink>
       <Name lang="en">Alpha-thalassemia-intellectual disability syndrome linked to chromosome 16</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
+      <DisorderType id="21401">
+        <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
