commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml
old mode 100644
new mode 100755
index 20e14b2..6cf1e8d
--- a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:40:35" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:41:33" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -25130,7 +25130,7 @@
     <Disorder id="343">
       <OrphaCode>1727</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1727</ExpertLink>
-      <Name lang="es">Síndrome de microduplicación 22q11.2</Name>
+      <Name lang="es">Síndrome de duplicación 22q11.2</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -48342,8 +48342,8 @@
       <OrphaCode>178320</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=178320</ExpertLink>
       <Name lang="es">Lesión pulmonar aguda</Name>
-      <DisorderType id="21422">
-        <Name lang="es">Síndrome clínico</Name>
+      <DisorderType id="21429">
+        <Name lang="es">Situación clínica particular en una enfermedad o síndrome</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
@@ -63352,7 +63352,7 @@
     <Disorder id="784">
       <OrphaCode>3002</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3002</ExpertLink>
-      <Name lang="es">Púrpura trombocitopénica inmunitaria</Name>
+      <Name lang="es">Trombocitopenia inmune</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -67070,7 +67070,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Canal auriculoventricular completo</Name>
+      <Name lang="es">Defecto de la comunicación auriculoventricular completo</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -69125,7 +69125,7 @@
     <Disorder id="849">
       <OrphaCode>3198</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3198</ExpertLink>
-      <Name lang="es">Síndrome de la persona rígida y síndromes relacionados</Name>
+      <Name lang="es">Trastorno del espectro de la persona rígida</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -85892,7 +85892,7 @@
     <Disorder id="1299">
       <OrphaCode>981</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=981</ExpertLink>
-      <Name lang="es">Agenesia de la carótida interna</Name>
+      <Name lang="es">Ausencia de la carótida interna</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -95796,7 +95796,7 @@
     <Disorder id="17146">
       <OrphaCode>157798</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=157798</ExpertLink>
-      <Name lang="es">Síndrome de poliposis hiperplásica</Name>
+      <Name lang="es">Síndrome de poliposis serrada</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -104139,7 +104139,7 @@
     <Disorder id="1868">
       <OrphaCode>1906</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1906</ExpertLink>
-      <Name lang="es">Síndrome fetal por valproato</Name>
+      <Name lang="es">Trastorno del espectro valproato fetal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -110855,7 +110855,7 @@
     <Disorder id="2200">
       <OrphaCode>2348</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2348</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial tipo Dunnigan</Name>
+      <Name lang="es">Lipodistrofia parcial familiar tipo Dunnigan</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -118582,7 +118582,7 @@
     <Disorder id="19605">
       <OrphaCode>251028</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251028</ExpertLink>
-      <Name lang="es">Síndrome de microdeleción 2q33.1</Name>
+      <Name lang="es">Síndrome asociado al gen SATB2 por un reordenamiento cromosómico</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -125499,6 +125499,95 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="19728">
+      <OrphaCode>252175</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=252175</ExpertLink>
+      <Name lang="es">OBSOLETO: Schwannoma vestibular</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="16415">
+          <Source>20871439[PMID]_24655069[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="es">Incidencia anual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="24264">
+            <Name lang="es">Denmark</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16416">
+          <Source>National Cancer Institute[INST]_23432451[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="es">Incidencia anual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>1.2</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="es">United States</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16417">
+          <Source>21897393[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="es">Incidencia anual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="es">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.8</ValMoy>
+          <PrevalenceGeographic id="23851">
+            <Name lang="es">Población específic</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16418">
+          <Source/>
+          <PrevalenceType id="23676">
+            <Name lang="es">Incidencia anual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="es">Europa</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25965">
+            <Name lang="es">Not yet validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19756">
       <OrphaCode>254516</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=254516</ExpertLink>
@@ -135389,7 +135478,7 @@
     <Disorder id="2814">
       <OrphaCode>3145</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3145</ExpertLink>
-      <Name lang="es">Síndrome de Schofer-Beetz-Bohl</Name>
+      <Name lang="es">Síndrome de diabetes insípida nefrogénica-calcificación intracraneal-dismorfia facial</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -152163,7 +152252,7 @@
     <Disorder id="20389">
       <OrphaCode>276608</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=276608</ExpertLink>
-      <Name lang="es">Hipoglucemia hiperinsulinémica persistente sin insulinoma de inicio en el adulto</Name>
+      <Name lang="es">Encefalomielitis progresiva con rigidez y mioclonías</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -164404,7 +164493,7 @@
     <Disorder id="3563">
       <OrphaCode>359</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=359</ExpertLink>
-      <Name lang="es">Glaucoma hereditario</Name>
+      <Name lang="es">Glaucoma de inicio pediátrico de origen genético</Name>
       <DisorderType id="36561">
         <Name lang="es">Category</Name>
       </DisorderType>
@@ -182314,7 +182403,7 @@
     <Disorder id="3727">
       <OrphaCode>3337</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3337</ExpertLink>
-      <Name lang="es">Síndrome de Fanconi primario</Name>
+      <Name lang="es">Síndrome de Fanconi primario renotubular</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -207238,8 +207327,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=353284</ExpertLink>
       <Name lang="es">Síndrome de Rubinstein-Taybi por haploinsuficiencia EP300</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21443">
+        <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="es">Subtipo de trastorno</Name>
@@ -208540,55 +208629,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293848</ExpertLink>
-      <Name lang="es">Demencia frontotemporal, variante con atrofia temporal derecha</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="9768">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23697">
-            <Name lang="es">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>200.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10677">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="es">Desconocido</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="es">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293843</ExpertLink>
@@ -211220,7 +211260,7 @@
     <Disorder id="20609">
       <OrphaCode>284417</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=284417</ExpertLink>
-      <Name lang="es">Deficiencia de fosfoserina aminotransferasa</Name>
+      <Name lang="es">Deficiencia de fosfoserina aminotransferasa, forma infantil/juvenil</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -233590,7 +233630,7 @@
     <Disorder id="22609">
       <OrphaCode>391348</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391348</ExpertLink>
-      <Name lang="es">Síndrome de retraso en el crecimiento y el desarrollo-hipotonía-trastorno ocular-acidosis láctica</Name>
+      <Name lang="es">Síndrome de retraso del crecimiento y el desarrollo-hipotonía-trastorno ocular-acidosis láctica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -233688,7 +233728,7 @@
     <Disorder id="22611">
       <OrphaCode>391366</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391366</ExpertLink>
-      <Name lang="es">Síndrome de retraso en el crecimiento-retraso del desarrollo leve-hepatitis crónica</Name>
+      <Name lang="es">Síndrome de retraso del crecimiento-retraso del desarrollo leve-hepatitis crónica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -239487,7 +239527,7 @@
     <Disorder id="23310">
       <OrphaCode>435660</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=435660</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen LIPE</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen LIPE</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -239536,7 +239576,7 @@
     <Disorder id="23309">
       <OrphaCode>435651</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=435651</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen CIDEC</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen CIDEC</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -266938,7 +266978,7 @@
     <Disorder id="8597">
       <OrphaCode>1330</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1330</ExpertLink>
-      <Name lang="es">Canal auriculoventricular parcial</Name>
+      <Name lang="es">Defecto de la comunicación auriculoventricular parcial</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -277706,7 +277746,7 @@
     <Disorder id="10882">
       <OrphaCode>66529</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=66529</ExpertLink>
-      <Name lang="es">Miocardiopatía tipo Tako-Tsubo</Name>
+      <Name lang="es">Miocardiopatía de Tako-Tsubo</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -286321,7 +286361,7 @@
     <Disorder id="11129">
       <OrphaCode>79085</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79085</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen AKT2</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen AKT2</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -286370,7 +286410,7 @@
     <Disorder id="11128">
       <OrphaCode>79084</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79084</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial tipo Köbberling</Name>
+      <Name lang="es">Lipodistrofia parcial familiar tipo Köbberling</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -288437,7 +288477,7 @@
     <Disorder id="11048">
       <OrphaCode>73272</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73272</ExpertLink>
-      <Name lang="es">Retraso en el crecimiento por deficiencia del factor de crecimiento similar a la insulina tipo 1</Name>
+      <Name lang="es">Retraso del crecimiento por deficiencia del factor de crecimiento similar a la insulina tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -288486,7 +288526,7 @@
     <Disorder id="11049">
       <OrphaCode>73273</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73273</ExpertLink>
-      <Name lang="es">Retraso en el crecimiento por resistencia al factor de crecimiento similar a la insulina tipo 1</Name>
+      <Name lang="es">Retraso del crecimiento por resistencia al factor de crecimiento similar a la insulina tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -298157,7 +298197,7 @@
     <Disorder id="10738">
       <OrphaCode>53696</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=53696</ExpertLink>
-      <Name lang="es">Síndrome de artrogriposis letal-enfermedad de las astas medulares anteriores</Name>
+      <Name lang="es">Síndrome de artrogriposis-enfermedad de las células del cuerno anterior</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -298874,7 +298914,7 @@
     <Disorder id="10631">
       <OrphaCode>48686</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=48686</ExpertLink>
-      <Name lang="es">Linfoma primario de cavidades</Name>
+      <Name lang="es">Linfoma primario de efusiones</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -321725,7 +321765,7 @@
     <Disorder id="11376">
       <OrphaCode>79350</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79350</ExpertLink>
-      <Name lang="es">Deficiencia de 3-fosfoserina fosfatasa</Name>
+      <Name lang="es">Deficiencia de 3-fosfoserina fosfatasa, forma infantil/juvenil</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -330095,7 +330135,7 @@
     <Disorder id="11600">
       <OrphaCode>83629</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=83629</ExpertLink>
-      <Name lang="es">Síndrome de leucoencefalopatía-displasia espondilometafisaria</Name>
+      <Name lang="es">Leukoencephalopathy-spondyloepimetaphyseal dysplasia syndrome</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -330706,7 +330746,7 @@
     <Disorder id="11618">
       <OrphaCode>84132</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=84132</ExpertLink>
-      <Name lang="es">Miopatía asociada a la desmina con inclusiones tipo cuerpos de Mallory</Name>
+      <Name lang="es">Miopatía asociada a desmina con inclusiones similares a cuerpos de Mallory</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -337308,7 +337348,7 @@
     <Disorder id="12396">
       <OrphaCode>93554</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93554</ExpertLink>
-      <Name lang="es">Crioglobulinemia mixta tipo 2</Name>
+      <Name lang="es">Crioglobulinemia mixta tipo II</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -340304,7 +340344,7 @@
     <Disorder id="12469">
       <OrphaCode>93890</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93890</ExpertLink>
-      <Name lang="es">Defectos del desarrollo embrionario</Name>
+      <Name lang="es">Defectos raros del desarrollo embrionario</Name>
       <DisorderType id="36561">
         <Name lang="es">Category</Name>
       </DisorderType>
@@ -340565,7 +340605,7 @@
     <Disorder id="12423">
       <OrphaCode>93585</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93585</ExpertLink>
-      <Name lang="es">Púrpura trombótica trombocitopénica adquirida</Name>
+      <Name lang="es">Immune-mediated thrombotic thrombocytopenic purpura</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -353709,7 +353749,7 @@
     <Disorder id="13323">
       <OrphaCode>98306</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98306</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial</Name>
+      <Name lang="es">Lipodistrofia parcial familiar</Name>
       <DisorderType id="21436">
         <Name lang="es">Grupo de fenomas</Name>
       </DisorderType>
