commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml
index f16f1b0..34fa1e4 100644
--- a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:52:33" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6044">
+<JDBOR date="2021-12-01 04:57:11" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6052">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -2886,7 +2879,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="11716">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -2896,7 +2889,7 @@
           <PrevalenceClass id="23739">
             <Name lang="es">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.31</ValMoy>
+          <ValMoy>0.28</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="es">Brazil</Name>
           </PrevalenceGeographic>
@@ -4068,7 +4061,7 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <PrevalenceList count="8">
+      <PrevalenceList count="9">
         <Prevalence id="136">
           <Source>European Medicines Agency 2012[INST]</Source>
           <PrevalenceType id="23669">
@@ -4107,6 +4100,25 @@
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16640">
+          <Source>PMID: 31926052</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="es">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.02</ValMoy>
+          <PrevalenceGeographic id="24068">
+            <Name lang="es">Brazil</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="11717">
           <Source>25060283[PMID]</Source>
           <PrevalenceType id="23683">
@@ -14374,7 +14386,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13915">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -14384,7 +14396,7 @@
           <PrevalenceClass id="23739">
             <Name lang="es">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.38</ValMoy>
+          <ValMoy>0.37</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="es">Brazil</Name>
           </PrevalenceGeographic>
@@ -14862,7 +14874,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13910">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -14913,7 +14925,7 @@
       </DisorderGroup>
       <PrevalenceList count="11">
         <Prevalence id="520">
-          <Source>17276780[PMID]</Source>
+          <Source>PMID: 31449670</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -14923,7 +14935,7 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.2</ValMoy>
+          <ValMoy>2.25</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
@@ -14951,7 +14963,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="522">
-          <Source>DOI:10.1016/S0168-8278(85)80013-1[OTHER]_17276780[PMID]</Source>
+          <Source>PMID: 31449670</Source>
           <PrevalenceType id="23669">
             <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
@@ -14961,7 +14973,7 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>3.3</ValMoy>
+          <ValMoy>2.02</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
@@ -16167,7 +16179,7 @@
     <Disorder id="17515">
       <OrphaCode>163654</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=163654</ExpertLink>
-      <Name lang="es">Displasia espondiloepifisaria tipo Cantu</Name>
+      <Name lang="es">Síndrome de displasia espondiloepifisaria-braquidactilia-trastorno del habla</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -24022,7 +24034,7 @@
     <Disorder id="315">
       <OrphaCode>1349</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1349</ExpertLink>
-      <Name lang="es">Miocardiopatía y pérdida auditiva asociada al ADN mitocondrial</Name>
+      <Name lang="es">Miocardiopatía e hipoacusia asociada al ADN mitocondrial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -24296,7 +24308,7 @@
     <Disorder id="294">
       <OrphaCode>308</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=308</ExpertLink>
-      <Name lang="es">Enfermedad de Unverricht-Lundborg</Name>
+      <Name lang="es">Epilepsia mioclónica progresiva tipo 1</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -38594,11 +38606,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2346</ExpertLink>
       <Name lang="es">Síndrome angio-osteo-hipertrófico</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21436">
+        <Name lang="es">Grupo de fenomas</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="1286">
@@ -46160,7 +46172,26 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <PrevalenceList count="25">
+      <PrevalenceList count="26">
+        <Prevalence id="16639">
+          <Source>PMID: 31926052</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="es">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.21</ValMoy>
+          <PrevalenceGeographic id="24068">
+            <Name lang="es">Brazil</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="1671">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
@@ -46883,7 +46914,7 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <PrevalenceList count="8">
+      <PrevalenceList count="6">
         <Prevalence id="1689">
           <Source>PMID: 31769437, 25944380, 24715559</Source>
           <PrevalenceType id="23669">
@@ -46998,40 +47029,6 @@
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="16436">
-          <Source>PMID: 22913777</Source>
-          <PrevalenceType id="23697">
-            <Name lang="es">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>1314.0</ValMoy>
-          <PrevalenceGeographic id="24166">
-            <Name lang="es">China</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="16435">
-          <Source>PMID: 26627451</Source>
-          <PrevalenceType id="23697">
-            <Name lang="es">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>72.0</ValMoy>
-          <PrevalenceGeographic id="25356">
-            <Name lang="es">Taiwan, Province of China</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="17999">
@@ -50097,7 +50094,7 @@
     <Disorder id="745">
       <OrphaCode>183</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183</ExpertLink>
-      <Name lang="es">Granulomatosis eosinofílica con poliangitis</Name>
+      <Name lang="es">Granulomatosis eosinofílica con poliangeítis</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -53690,7 +53687,7 @@
     <Disorder id="753">
       <OrphaCode>727</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=727</ExpertLink>
-      <Name lang="es">Poliangitis microscópica</Name>
+      <Name lang="es">Poliangeitis microscópica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -54102,7 +54099,7 @@
     <Disorder id="759">
       <OrphaCode>900</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=900</ExpertLink>
-      <Name lang="es">Granulomatosis con poliangitis</Name>
+      <Name lang="es">Granulomatosis con poliangeítis</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -58088,7 +58085,7 @@
     <Disorder id="17940">
       <OrphaCode>171848</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=171848</ExpertLink>
-      <Name lang="es">Síndrome de polineuropatía-pérdida auditiva-ataxia-retinosis pigmentaria-cataratas</Name>
+      <Name lang="es">Síndrome de polineuropatía-hipoacusia-ataxia-retinosis pigmentaria-cataratas</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -66384,7 +66381,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Canal aurículoventricular completo</Name>
+      <Name lang="es">Defecto del septo aurículoventricular completo</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -66758,7 +66755,7 @@
           <PrevalenceClass id="23739">
             <Name lang="es">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.11</ValMoy>
+          <ValMoy>0.14</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="es">Brazil</Name>
           </PrevalenceGeographic>
@@ -71890,7 +71887,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183678</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak con neutropenia</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de AP-3</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -73335,7 +73332,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2262">
-          <Source>15054950[PMID]_10817953[PMID]_[EXPERT]</Source>
+          <Source>PMID: 27037234  ; 34220661; ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
@@ -73345,7 +73342,7 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>6.0</ValMoy>
+          <ValMoy>5.26</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
@@ -75630,27 +75627,27 @@
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
-        <Prevalence id="11997">
-          <Source>20301500[PMID]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="es">Prevalencia al nacer</Name>
+        <Prevalence id="16626">
+          <Source>PMID: 31969461</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="es">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="es">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="es">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.125</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
+          <ValMoy>0.47</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="es">Italy</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="12014">
-          <Source>Kroes_Fransen van de Putte_Ravesloot 2007[AUTHOR]</Source>
+        <Prevalence id="11997">
+          <Source>20301500[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -75660,9 +75657,9 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.0</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="es">Netherlands</Name>
+          <ValMoy>1.125</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
@@ -75676,15 +75673,15 @@
           <PrevalenceQualification id="23704">
             <Name lang="es">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="es">1-9 / 100 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="es">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -77919,9 +77916,9 @@
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
       </DisorderGroup>
-      <PrevalenceList count="5">
-        <Prevalence id="2605">
-          <Source>14518075[PMID]</Source>
+      <PrevalenceList count="6">
+        <Prevalence id="16623">
+          <Source>PMID: 26431509</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -77931,16 +77928,16 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="es">Netherlands</Name>
+          <ValMoy>2.24</ValMoy>
+          <PrevalenceGeographic id="24257">
+            <Name lang="es">Czech Republic</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2606">
-          <Source>9130925[PMID]_14518075[PMID]</Source>
+        <Prevalence id="16624">
+          <Source>PMID: 31742894</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -77950,54 +77947,73 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="25482">
-            <Name lang="es">United States</Name>
+          <ValMoy>5.25</ValMoy>
+          <PrevalenceGeographic id="24621">
+            <Name lang="es">Japan</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2607">
-          <Source>21967988[PMID]</Source>
+        <Prevalence id="16625">
+          <Source>PMID :32027664</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="es">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="es">1-5 / 10 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>40.0</ValMoy>
-          <PrevalenceGeographic id="25209">
-            <Name lang="es">Saudi Arabia</Name>
+          <ValMoy>3.52</ValMoy>
+          <PrevalenceGeographic id="25475">
+            <Name lang="es">United Kingdom</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2608">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
+        <Prevalence id="2605">
+          <Source>14518075[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
           </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="24936">
+            <Name lang="es">Netherlands</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="es">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="2607">
+          <Source>21967988[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="es">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>40.0</ValMoy>
+          <PrevalenceGeographic id="25209">
+            <Name lang="es">Saudi Arabia</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10928">
-          <Source>[EXPERT]_ORPHANET</Source>
+          <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -78007,7 +78023,7 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
+          <ValMoy>3.67</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
@@ -81092,6 +81108,38 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="1228">
+      <OrphaCode>3306</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3306</ExpertLink>
+      <Name lang="es">Síndrome de inversión duplicación del cromosoma 15</Name>
+      <DisorderType id="21401">
+        <Name lang="es">Síndrome de malformación</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="1">
+        <Prevalence id="16620">
+          <Source>PMID: 11579431</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>3.33</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="1231">
       <OrphaCode>3375</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3375</ExpertLink>
@@ -86856,7 +86904,7 @@
     <Disorder id="1318">
       <OrphaCode>1005</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1005</ExpertLink>
-      <Name lang="es">Síndrome de alopecia-contracturas-talla baja-discapacidad intelectual</Name>
+      <Name lang="es">Síndrome de alopecia-contracturas-talla baja significativa-discapacidad intelectual</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -93773,7 +93821,7 @@
     <Disorder id="1614">
       <OrphaCode>1416</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1416</ExpertLink>
-      <Name lang="es">Deposición familiar de pirofosfatos de calcio</Name>
+      <Name lang="es">Deposición familiar de pirofosfato cálcico</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -102325,7 +102373,7 @@
     <Disorder id="1899">
       <OrphaCode>1952</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1952</ExpertLink>
-      <Name lang="es">Displasia de Pacman</Name>
+      <Name lang="es">Síndrome de epífisis punteada-hiperplasia osteoclástica</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -109314,7 +109362,7 @@
     <Disorder id="17233">
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158673</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa distrófica localizada, forma acral</Name>
+      <Name lang="es">Epidermólisis ampollosa distrófica localizada forma acral</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -111000,7 +111048,7 @@
     <Disorder id="2191">
       <OrphaCode>2339</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2339</ExpertLink>
-      <Name lang="es">Síndrome de queratosis folicular-talla baja-atrofia cerebral</Name>
+      <Name lang="es">Síndrome de queratosis folicular-talla baja significativa-atrofia cerebral</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -120629,7 +120677,7 @@
     <Disorder id="2147">
       <OrphaCode>2278</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2278</ExpertLink>
-      <Name lang="es">Síndrome de ictiosis-discapacidad intelectual-talla baja-afectación renal</Name>
+      <Name lang="es">Síndrome de ictiosis-discapacidad intelectual-talla baja significativa-afectación renal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -129939,7 +129987,7 @@
     <Disorder id="2396">
       <OrphaCode>2617</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2617</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica tipo Montreal</Name>
+      <Name lang="es">Talla baja significativa primordial microcefálica tipo Montreal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -131260,7 +131308,7 @@
     <Disorder id="2418">
       <OrphaCode>2636</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2636</ExpertLink>
-      <Name lang="es">Talla baja primordial osteodisplásica microcefálica tipos I y III</Name>
+      <Name lang="es">Talla baja significativa primordial osteodisplásica microcefálica tipos I y III</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -131309,7 +131357,7 @@
     <Disorder id="2429">
       <OrphaCode>2658</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2658</ExpertLink>
-      <Name lang="es">Enanismo hiperostótico de Lenz-Majewski</Name>
+      <Name lang="es">Talla baja significativa hiperostótica de Lenz-Majewski</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -142463,7 +142511,7 @@
     <Disorder id="2981">
       <OrphaCode>3363</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3363</ExpertLink>
-      <Name lang="es">Síndrome de tricomegalia-degeneración retiniana pigmentaria-talla baja</Name>
+      <Name lang="es">Síndrome de tricomegalia-degeneración retiniana pigmentaria-talla baja significativa</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -152449,7 +152497,7 @@
     <Disorder id="2882">
       <OrphaCode>3225</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3225</ExpertLink>
-      <Name lang="es">Síndrome de pérdida auditiva-insensibilidad a la aldosterona de la glándula salivar, familiar</Name>
+      <Name lang="es">Síndrome de hipoacusia-insensibilidad a la aldosterona de la glándula salivar, familiar</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -154953,7 +155001,7 @@
     <Disorder id="3298">
       <OrphaCode>832</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=832</ExpertLink>
-      <Name lang="es">Deficiencia de succinil-CoA:3-cetoácido-CoA transferasa</Name>
+      <Name lang="es">Deficiencia de succinil-CoA: 3 oxoácido CoA transferasa</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -158723,7 +158771,7 @@
     <Disorder id="3171">
       <OrphaCode>2619</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2619</ExpertLink>
-      <Name lang="es">Talla baja con braquidactilia tipo Mseleni</Name>
+      <Name lang="es">Talla baja significativa con braquidactilia tipo Mseleni</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -171694,7 +171742,7 @@
     <Disorder id="3423">
       <OrphaCode>2745</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2745</ExpertLink>
-      <Name lang="es">Síndrome de Opitz G/BBB</Name>
+      <Name lang="es">Síndrome de Opitz GBBB</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -173465,7 +173513,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8794">
-          <Source>23625533[PMID]_25665837[PMID]</Source>
+          <Source>PMID: 33365252 ; 33485800</Source>
           <PrevalenceType id="23697">
             <Name lang="es">Cases/families</Name>
           </PrevalenceType>
@@ -173473,7 +173521,7 @@
             <Name lang="es">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>19.0</ValMoy>
+          <ValMoy>32.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
@@ -173482,7 +173530,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="8795">
-          <Source>ORPHANET_23625533[PMID]_25665837[PMID]</Source>
+          <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
@@ -178104,7 +178152,7 @@
     <Disorder id="19064">
       <OrphaCode>228012</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=228012</ExpertLink>
-      <Name lang="es">Síndrome de pérdida auditiva neurosensorial progresiva-miocardiopatía hipertrófica</Name>
+      <Name lang="es">Síndrome de hipoacusia neurosensorial progresiva-miocardiopatía hipertrófica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -183959,7 +184007,7 @@
     <Disorder id="19198">
       <OrphaCode>231531</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231531</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak tipo 7</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de BLOC-1</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -184008,7 +184056,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231512</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak sin fibrosis pulmonar</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -184180,7 +184228,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231500</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak con fibrosis pulmonar</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -189607,7 +189655,7 @@
     <Disorder id="21965">
       <OrphaCode>330054</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=330054</ExpertLink>
-      <Name lang="es">Síndrome de catarata congénita-hipotonía muscular progresiva-pérdida auditiva-retraso del desarrollo</Name>
+      <Name lang="es">Síndrome de catarata congénita-hipotonía muscular progresiva-hipoacusia-retraso del desarrollo</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -190677,7 +190725,7 @@
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=329228</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica por deficiencia de ZNF335</Name>
+      <Name lang="es">Talla baja significativa primordial microcefálica por deficiencia de ZNF335</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -195350,7 +195398,7 @@
     <Disorder id="21691">
       <OrphaCode>319675</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319675</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica tipo Dauber</Name>
+      <Name lang="es">Talla baja significativa primordial microcefálica tipo Dauber</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -206323,7 +206371,7 @@
     <Disorder id="22057">
       <OrphaCode>352333</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=352333</ExpertLink>
-      <Name lang="es">Síndrome de ictions congénita-discapacidad intelectual-cuadriplejía espástica</Name>
+      <Name lang="es">Síndrome de ictiosis congénita-discapacidad intelectual-cuadriplejía espástica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -207041,7 +207089,7 @@
     <Disorder id="20897">
       <OrphaCode>293967</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293967</ExpertLink>
-      <Name lang="es">Síndrome de hipogonadismo hipogonadotrópico-microcefalia grave-pérdida auditiva neurosensorial-dismorfia</Name>
+      <Name lang="es">Síndrome de hipogonadismo hipogonadotrópico-microcefalia grave-hipoacusia neurosensorial-dismorfia</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -214723,7 +214771,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=314555</ExpertLink>
-      <Name lang="es">Síndrome de displasia craneofacial-osteopenia</Name>
+      <Name lang="es">Síndrome de dismorfia facial-miopía severa-osteopenia-discapacidad intelectual-anomalías dentales</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -216336,7 +216384,7 @@
     <Disorder id="6020">
       <OrphaCode>2637</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2637</ExpertLink>
-      <Name lang="es">Enanismo primordial osteodisplásico microcefálico tipo II</Name>
+      <Name lang="es">Talla baja significativa primordial osteodisplásica microcefálica tipo II</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -218595,7 +218643,7 @@
     <Disorder id="21226">
       <OrphaCode>306530</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=306530</ExpertLink>
-      <Name lang="es">Síndrome de parálisis facial hereditaria congénita-pérdida auditiva variable</Name>
+      <Name lang="es">Síndrome de parálisis facial hereditaria congénita-hipoacusia variable</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -219181,7 +219229,7 @@
     <Disorder id="21109">
       <OrphaCode>300313</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300313</ExpertLink>
-      <Name lang="es">Síndrome de cataratas congénitas-pérdida auditiva-retraso grave del desarrollo</Name>
+      <Name lang="es">Síndrome de cataratas congénitas-hipoacusia-retraso grave del desarrollo</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -224590,7 +224638,7 @@
     <Disorder id="23721">
       <OrphaCode>457351</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=457351</ExpertLink>
-      <Name lang="es">Síndrome de microcefalia-discapacidad intelectual-pérdida auditiva neurosensorial-epilepsia-tono muscular anómalo</Name>
+      <Name lang="es">Síndrome de microcefalia-discapacidad intelectual-hipoacusia neurosensorial-epilepsia-tono muscular anómalo</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -227120,7 +227168,7 @@
     <Disorder id="24223">
       <OrphaCode>468631</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=468631</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica-malformaciones cerebrales por deficiencia de RTTN</Name>
+      <Name lang="es">Malformaciones corticales microcefálicas-talla baja por deficiencia de RTTN</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -234895,7 +234943,7 @@
     <Disorder id="22698">
       <OrphaCode>397744</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=397744</ExpertLink>
-      <Name lang="es">Síndrome de neuropatía periférica-miopatía-ronquera-pérdida auditiva</Name>
+      <Name lang="es">Síndrome de neuropatía periférica-miopatía-ronquera-hipoacusia</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -239227,7 +239275,7 @@
     <Disorder id="23333">
       <OrphaCode>436182</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436182</ExpertLink>
-      <Name lang="es">Síndrome de talla baja primordial microcefálica-resistencia a la insulina</Name>
+      <Name lang="es">Síndrome de talla baja significativa primordial microcefálica-resistencia a la insulina</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -239276,7 +239324,7 @@
     <Disorder id="23332">
       <OrphaCode>436174</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436174</ExpertLink>
-      <Name lang="es">Síndrome de catarata-deficiencia de hormona de crecimiento-neuropatía sensitiva-pérdida auditiva neurosensorial-displasia esquelética</Name>
+      <Name lang="es">Síndrome de catarata-deficiencia de hormona de crecimiento-neuropatía sensitiva-hipoacusia neurosensorial-displasia esquelética</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -240563,7 +240611,7 @@
     <Disorder id="23401">
       <OrphaCode>440354</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=440354</ExpertLink>
-      <Name lang="es">Síndrome de miopía autosómica dominante-retrusión del tercio medio facial-pérdida auditiva neurosensorial-displasia rizomélica</Name>
+      <Name lang="es">Síndrome de miopía autosómica dominante-retrusión del tercio medio facial-hipoacusia neurosensorial-displasia rizomélica</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -258061,7 +258109,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=508529</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa simple generalizada basal con miocardiopatía</Name>
+      <Name lang="es">Epidermólisis ampollosa simple intermedia con miocardiopatía</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -258208,7 +258256,7 @@
     <Disorder id="26568">
       <OrphaCode>508476</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=508476</ExpertLink>
-      <Name lang="es">Síndrome de labio leporino y paladar hendido-dismorfia craneofacial-defecto cardíaco congénito-pérdida auditiva</Name>
+      <Name lang="es">Síndrome de labio leporino y paladar hendido-dismorfia craneofacial-defecto cardíaco congénito-hipoacusia</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -259277,7 +259325,7 @@
     <Disorder id="25675">
       <OrphaCode>494439</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=494439</ExpertLink>
-      <Name lang="es">Síndrome de retinosis pigmentaria-pérdida auditiva-envejecimiento prematuro-talla baja-dismorfia facial</Name>
+      <Name lang="es">Síndrome de retinosis pigmentaria-hipoacusia-envejecimiento prematuro-talla baja-dismorfia facial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -260805,7 +260853,7 @@
     <Disorder id="25998">
       <OrphaCode>500144</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=500144</ExpertLink>
-      <Name lang="es">Síndrome de encefalopatía progresiva-pérdida auditiva-hipoplasia pontina-atrofia cerebral de inicio temprano</Name>
+      <Name lang="es">Síndrome de encefalopatía progresiva-hipoacusia-hipoplasia pontina-atrofia cerebral de inicio temprano</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -268205,7 +268253,7 @@
     <Disorder id="25256">
       <OrphaCode>482606</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=482606</ExpertLink>
-      <Name lang="es">Síndrome ligado al cromosoma X de cicatrización queloide-movilidad articular reducida-aumento de la relación copa/disco</Name>
+      <Name lang="es">Síndrome ligado al cromosoma X de cicatrización queloide-movilidad articular reducida-aumento de la relación copa/disco óptico</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -269769,7 +269817,7 @@
     <Disorder id="25161">
       <OrphaCode>477831</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477831</ExpertLink>
-      <Name lang="es">Síndrome de sobrecrecimiento esquelético-dismorfia craneofacial-piel hiperelástica-lesiones en la sustancia blanca</Name>
+      <Name lang="es">Síndrome de sobrecrecimiento de Kosaki</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -276735,7 +276783,7 @@
     <Disorder id="10891">
       <OrphaCode>66633</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=66633</ExpertLink>
-      <Name lang="es">Síndrome de pérdida auditiva neurosensorial-aparición temprana de canas-temblor esencial</Name>
+      <Name lang="es">Síndrome de hipoacusia neurosensorial-aparición temprana de canas-temblor esencial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -276980,7 +277028,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=66627</ExpertLink>
-      <Name lang="es">Sinovitis villonodular pigmentada</Name>
+      <Name lang="es">Tumor tenosinovial de células gigantes</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -282029,7 +282077,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13938">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]_28595941[PMID]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -282039,7 +282087,7 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.04</ValMoy>
+          <ValMoy>1.25</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="es">Brazil</Name>
           </PrevalenceGeographic>
@@ -285476,7 +285524,7 @@
     <Disorder id="11127">
       <OrphaCode>79083</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79083</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen PPARG</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen PPARG</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -287835,7 +287883,7 @@
     <Disorder id="11071">
       <OrphaCode>75326</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=75326</ExpertLink>
-      <Name lang="es">Tortuosidad de las arterias retinianas</Name>
+      <Name lang="es">Tortuosidad arterial retiniana</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -294452,7 +294500,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35062</ExpertLink>
-      <Name lang="es">Infección idiopática diseminada por citomegalovirus</Name>
+      <Name lang="es">Infección diseminada por citomegalovirus grave en pacientes inmunocompetentes</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -299331,7 +299379,7 @@
     <Disorder id="28036">
       <OrphaCode>542301</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=542301</ExpertLink>
-      <Name lang="es">Inmunodeficiencia combinada grave por deficiencia de CARMIL2</Name>
+      <Name lang="es">Inmunodeficiencia combinada por deficiencia de CARMIL2</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -301866,7 +301914,7 @@
     <Disorder id="28081">
       <OrphaCode>544472</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=544472</ExpertLink>
-      <Name lang="es">Síndrome hemolítico urémico atípico por anomalía del gen del complemento</Name>
+      <Name lang="es">Síndrome hemolítico urémico atípico con anomalías en los genes del complemento</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -303420,7 +303468,7 @@
     <Disorder id="11950">
       <OrphaCode>90059</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90059</ExpertLink>
-      <Name lang="es">Pérdida auditiva neurosensorial a causa de trauma acústico grave o sordera repentina, o por trauma acústico inducido por cirugía</Name>
+      <Name lang="es">Hipoacusia neurosensorial a causa de trauma acústico grave o sordera repentina, o por trauma acústico inducido por cirugía</Name>
       <DisorderType id="21429">
         <Name lang="es">Situación clínica particular en una enfermedad o síndrome</Name>
       </DisorderType>
@@ -303999,29 +304047,29 @@
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
-        <Prevalence id="12928">
-          <Source>European Medicines Agency 2014[INST]</Source>
+        <Prevalence id="16645">
+          <Source>PMID: 26543054</Source>
           <PrevalenceType id="23669">
             <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="es">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="es">1-9 / 1 000 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.21</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
+          <ValMoy>1.89</ValMoy>
+          <PrevalenceGeographic id="25006">
+            <Name lang="es">Norway</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="12929">
-          <Source>19095780[PMID]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="es">Prevalencia al nacer</Name>
+        <Prevalence id="16646">
+          <Source>PMID: 31730177</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="es">Value and class</Name>
@@ -304029,16 +304077,16 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.95</ValMoy>
-          <PrevalenceGeographic id="24264">
-            <Name lang="es">Denmark</Name>
+          <ValMoy>1.54</ValMoy>
+          <PrevalenceGeographic id="25475">
+            <Name lang="es">United Kingdom</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="13994">
-          <Source>19095780[PMID]</Source>
+        <Prevalence id="12928">
+          <Source>PMID: 26543054 ; 31730177</Source>
           <PrevalenceType id="23669">
             <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
@@ -304048,9 +304096,9 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.4</ValMoy>
-          <PrevalenceGeographic id="24264">
-            <Name lang="es">Denmark</Name>
+          <ValMoy>1.66</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="es">Europa</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
@@ -304745,7 +304793,7 @@
     <Disorder id="12002">
       <OrphaCode>90291</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90291</ExpertLink>
-      <Name lang="es">Esclerodermia sistémica</Name>
+      <Name lang="es">Esclerosis sistémica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -305044,11 +305092,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90308</ExpertLink>
       <Name lang="es">Síndrome de Klippel-Trénaunay</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="7026">
@@ -308418,7 +308466,7 @@
     <Disorder id="12199">
       <OrphaCode>93256</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93256</ExpertLink>
-      <Name lang="es">Síndrome de ataxia/temblor asociado al X frágil</Name>
+      <Name lang="es">Síndrome de ataxia/temblor asociado al cromosoma X frágil</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -309638,7 +309686,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93111</ExpertLink>
-      <Name lang="es">Nefropatía tubulointersticial autosómica dominante asociada al gen HNF1B</Name>
+      <Name lang="es">Enfermedad renal tubulointersticial autosómica dominante asociada al gen HNF1B</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -313184,38 +313232,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="12110">
-      <OrphaCode>91133</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=91133</ExpertLink>
-      <Name lang="es">Síndrome de osteopenia-miopía-pérdida auditiva-discapacidad intelectual-dismorfia facial</Name>
-      <DisorderType id="21401">
-        <Name lang="es">Síndrome de malformación</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="7086">
-          <Source>ORPHANET_17230486[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="es">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="12096">
       <OrphaCode>90793</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90793</ExpertLink>
@@ -314862,7 +314878,7 @@
     <Disorder id="11436">
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79410</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa distrófica localizada, forma pretibial</Name>
+      <Name lang="es">Epidermólisis ampollosa distrófica localizada forma pretibial</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -321631,7 +321647,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=85279</ExpertLink>
-      <Name lang="es">Discapacidad intelectual ligada al cromosoma X sindrómica por una mutación en el gen JARID1C</Name>
+      <Name lang="es">Discapacidad intelectual sindrómica ligada al cromosoma asociada al gen KDM5C</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -329082,7 +329098,7 @@
     <Disorder id="11600">
       <OrphaCode>83629</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=83629</ExpertLink>
-      <Name lang="es">Síndrome de leucoencefalopatía-displasia espondilometafisaria</Name>
+      <Name lang="es">Síndrome de leucoencefalopatía-displasia espondiloepimetafisaria</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -329278,7 +329294,7 @@
     <Disorder id="11631">
       <OrphaCode>85164</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=85164</ExpertLink>
-      <Name lang="es">Síndrome de camptodactilia-talla alta-escoliosis-pérdida auditiva</Name>
+      <Name lang="es">Síndrome de camptodactilia-talla alta-escoliosis-hipoacusia</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -336768,6 +336784,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=599373</ExpertLink>
+      <Name lang="es">Encefalopatía asociada a STXBP1</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16637">
+          <Source>PMID: 32643187</Source>
+          <PrevalenceType id="23697">
+            <Name lang="es">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="es">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>282.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16638">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93403</ExpertLink>
@@ -340425,6 +340490,104 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=592564</ExpertLink>
+      <Name lang="es">Espectro de trastornos del movimiento-crisis-retraso del desarrollo asociado al gen GNAO1</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16643">
+          <Source>PMID: 26060304 ; 29758257 ; 30682224 ; 34122306</Source>
+          <PrevalenceType id="23697">
+            <Name lang="es">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="es">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>75.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16644">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="es">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=592570</ExpertLink>
+      <Name lang="es">Síndrome de retraso motor y de la adquisición del habla-dismorfia facial-anomalías digitales-defecto cardíaco asociado al gen TRAF7</Name>
+      <DisorderType id="21401">
+        <Name lang="es">Síndrome de malformación</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16641">
+          <Source>PMID: 29961569 ;  32376980 ;  32459067 ;  34088006</Source>
+          <PrevalenceType id="23697">
+            <Name lang="es">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="es">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>55.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16642">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="es">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="14245">
       <OrphaCode>99672</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99672</ExpertLink>
@@ -340589,6 +340752,297 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589821</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio congénito</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="16621">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16622">
+          <Source>PMID: 23415617</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.1</ValMoy>
+          <PrevalenceGeographic id="24124">
+            <Name lang="es">Canada</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16627">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="es">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.94</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="es">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16628">
+          <Source>PMID: 33472919</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="es">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>47.6</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="es">United States</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589827</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio juvenil</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16631">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16632">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>1.64</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="es">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589824</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio en la infancia</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16629">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16630">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>1.29</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="es">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589833</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio tardío</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16635">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16636">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="es">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>8.14</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="es">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589830</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio en el adulto</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16633">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16634">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="es">Prevalencia puntual</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="es">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>12.54</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="es">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589905</ExpertLink>
@@ -343496,7 +343950,7 @@
     <Disorder id="29429">
       <OrphaCode>589442</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589442</ExpertLink>
-      <Name lang="es">Síndrome de talla baja-displasia esquelética-degeneración retiniana-discapacidad intelectual-pérdida auditiva neurosensorial</Name>
+      <Name lang="es">Síndrome de talla baja-displasia esquelética-degeneración retiniana-discapacidad intelectual-hipoacusia neurosensorial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -348072,10 +348526,10 @@
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <PrevalenceList count="15">
-        <Prevalence id="7708">
-          <Source>European Medicines Agency 2018[INST]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
+        <Prevalence id="16647">
+          <Source>PMID: 32503598</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="es">Value and class</Name>
@@ -348083,9 +348537,9 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
+          <ValMoy>9.9</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
@@ -348244,7 +348698,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="11209">
-          <Source>24780148[PMID]</Source>
+          <Source>PMID: 32503598</Source>
           <PrevalenceType id="23669">
             <Name lang="es">Prevalencia puntual</Name>
           </PrevalenceType>
@@ -348254,7 +348708,7 @@
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>4.78</ValMoy>
+          <ValMoy>2.8</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
@@ -355335,7 +355789,7 @@
     <Disorder id="14984">
       <OrphaCode>103908</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=103908</ExpertLink>
-      <Name lang="es">Diarrea congénita sódica</Name>
+      <Name lang="es">Diarrea sódica congénita</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
