commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml
index 732345e..d1be1b6 100644
--- a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:41:47" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:30:11" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="6056">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -6961,7 +6954,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=593</ExpertLink>
       <Name lang="es">Miopatía miofibrilar</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -22292,7 +22285,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2443</ExpertLink>
       <Name lang="es">Trastorno de la fosforilación oxidativa mitocondrial por anomalías del ADN nuclear</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -22984,7 +22977,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99</ExpertLink>
       <Name lang="es">Ataxia cerebelosa autosómica dominante</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -27830,7 +27823,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=352</ExpertLink>
       <Name lang="es">Galactosemia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -32376,7 +32369,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=167714</ExpertLink>
       <Name lang="es">Leucemia mieloide aguda no clasificada</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -41218,7 +41211,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=69</ExpertLink>
       <Name lang="es">Amiloidosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -41434,7 +41427,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166</ExpertLink>
       <Name lang="es">Enfermedad de Charcot-Marie-Tooth/Neuropatía sensitivo-motora hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -48130,7 +48123,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=177107</ExpertLink>
       <Name lang="es">Hipotiroidismo sindrómico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -51289,7 +51282,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=171901</ExpertLink>
       <Name lang="es">Linfoma cutáneo primario de células T</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -51807,7 +51800,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=171915</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin de células B</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -51935,7 +51928,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=171918</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin de células T</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -52016,7 +52009,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=187</ExpertLink>
       <Name lang="es">Citrulinemia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -52097,7 +52090,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=442</ExpertLink>
       <Name lang="es">Hipotiroidismo congénito</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -56441,7 +56434,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=795</ExpertLink>
       <Name lang="es">Forma rara de la salmonelosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -63359,7 +63352,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1531</ExpertLink>
       <Name lang="es">Craneosinostosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -71604,7 +71597,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183669</ExpertLink>
       <Name lang="es">Agammaglobulinemia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -75145,7 +75138,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183518</ExpertLink>
       <Name lang="es">Ataxia hereditaria rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -75645,7 +75638,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183435</ExpertLink>
       <Name lang="es">Ictiosis hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -75947,7 +75940,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183422</ExpertLink>
       <Name lang="es">Síndromes genéticos polimalformativos con aumento del riesgo de desarrollar cáncer</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76049,7 +76042,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=182114</ExpertLink>
       <Name lang="es">Tumor urogenital raro</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76132,7 +76125,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=182130</ExpertLink>
       <Name lang="es">Tumor de las glándulas endocrinas</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76285,7 +76278,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=182090</ExpertLink>
       <Name lang="es">Hipertensión arterial pulmonar</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76526,7 +76519,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=182095</ExpertLink>
       <Name lang="es">Enfermedad pulmonar intersticial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76683,7 +76676,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1047</ExpertLink>
       <Name lang="es">Anemia sideroblástica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -90540,7 +90533,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=138044</ExpertLink>
       <Name lang="es">Enfermedad rara con síndrome de Pierre Robin</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -99334,7 +99327,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=140874</ExpertLink>
       <Name lang="es">Síndrome de Joubert y trastornos relacionados</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -106827,7 +106820,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158032</ExpertLink>
       <Name lang="es">Síndrome hemofagocítico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -111635,7 +111628,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=247582</ExpertLink>
       <Name lang="es">Deficiencia de citrina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -115623,7 +115616,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251852</ExpertLink>
       <Name lang="es">Tumor embrionario del tejido neuroepitelial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -123600,7 +123593,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=252057</ExpertLink>
       <Name lang="es">Tumor de los nervios craneales y espinales</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -123730,7 +123723,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=252028</ExpertLink>
       <Name lang="es">Tumor melanocítico primario del sistema nervioso central</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -123862,7 +123855,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=254367</ExpertLink>
       <Name lang="es">Liquen plano raro</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -134869,7 +134862,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=262941</ExpertLink>
       <Name lang="es">Duplicación parcial del brazo largo del cromosoma 14</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -135817,7 +135810,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=268843</ExpertLink>
       <Name lang="es">Malformación del canal neuroentérico, la médula espinal y la columna</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -137552,7 +137545,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2846</ExpertLink>
       <Name lang="es">Anomalía congénita del pericardio</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -137714,7 +137707,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=268357</ExpertLink>
       <Name lang="es">Defectos del cierre del tubo neural</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -142171,7 +142164,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=275791</ExpertLink>
       <Name lang="es">Hipertensión arterial pulmonar asociada a otra enfermedad</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -144342,7 +144335,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3388</ExpertLink>
       <Name lang="es">Anomalías del tubo neural</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -147281,7 +147274,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=268926</ExpertLink>
       <Name lang="es">Malformación de la linea media cerebral</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -147609,7 +147602,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=269224</ExpertLink>
       <Name lang="es">Malformación global del cerebelo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -161365,7 +161358,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=599</ExpertLink>
       <Name lang="es">Miopatía distal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -162104,7 +162097,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=211062</ExpertLink>
       <Name lang="es">Ataxia episódica hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -162136,7 +162129,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2034</ExpertLink>
       <Name lang="es">Filariasis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -162451,7 +162444,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2415</ExpertLink>
       <Name lang="es">Malformación linfática rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -162958,7 +162951,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=137</ExpertLink>
       <Name lang="es">Trastorno congénito de la glicosilación</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -163645,7 +163638,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=359</ExpertLink>
       <Name lang="es">Glaucoma de inicio pediátrico de origen genético</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -163677,7 +163670,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=213500</ExpertLink>
       <Name lang="es">Cáncer de ovario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -169075,7 +169068,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=216675</ExpertLink>
       <Name lang="es">Transposición de las grandes arterias</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -171199,7 +171192,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217074</ExpertLink>
       <Name lang="es">Carcinoma raro de páncreas</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -171804,8 +171797,8 @@
           <PrevalenceType id="23697">
             <Name lang="es">Cases/families</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
+          <PrevalenceQualification id="23725">
+            <Name lang="es">Family</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
           <ValMoy>35.0</ValMoy>
@@ -172021,7 +172014,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=344</ExpertLink>
       <Name lang="es">Fiebre por arbovirus</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -172053,7 +172046,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=341</ExpertLink>
       <Name lang="es">Fiebre hemorrágica vírica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -173248,7 +173241,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1132</ExpertLink>
       <Name lang="es">Arcos aórticos anómalos</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -174365,7 +174358,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=542</ExpertLink>
       <Name lang="es">Linfoma cutáneo primario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -175559,7 +175552,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=220489</ExpertLink>
       <Name lang="es">Hemocromatosis hereditaria rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -177543,7 +177536,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3398</ExpertLink>
       <Name lang="es">Neoplasia epitelial tímica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -177645,7 +177638,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=547</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -178060,7 +178053,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3399</ExpertLink>
       <Name lang="es">Tumor de células germinales</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -179316,7 +179309,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=226292</ExpertLink>
       <Name lang="es">Hipotiroidismo congénito permanente</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -181747,7 +181740,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1172</ExpertLink>
       <Name lang="es">Ataxia cerebelosa autosómica recesiva</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -184491,7 +184484,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1866</ExpertLink>
       <Name lang="es">Distonía focal, segmentaria o multifocal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -186785,7 +186778,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231637</ExpertLink>
       <Name lang="es">Hiperaldosteronismo primario raro corregible por cirugía</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -188261,7 +188254,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=238547</ExpertLink>
       <Name lang="es">Policitemia secundaria adquirida</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -188342,7 +188335,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=238536</ExpertLink>
       <Name lang="es">Policitemia secundaria congénita</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -194181,7 +194174,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319543</ExpertLink>
       <Name lang="es">Susceptibilidad mendeliana autosómica dominante a enfermedades micobacterianas por deficiencia parcial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -194262,7 +194255,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319535</ExpertLink>
       <Name lang="es">Susceptibilidad mendeliana autosómica recesiva a enfermedades micobacterianas por deficiencia completa</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -194294,7 +194287,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319539</ExpertLink>
       <Name lang="es">Susceptibilidad mendeliana autosómica recesiva a enfermedades micobacterianas por deficiencia parcial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -202365,7 +202358,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=363203</ExpertLink>
       <Name lang="es">Cromosoma en anillo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -202836,7 +202829,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=363579</ExpertLink>
       <Name lang="es">Tumores extragonadales de células germinales</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -203541,7 +203534,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=363472</ExpertLink>
       <Name lang="es">Tumor testicular y paratesticular</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -211419,7 +211412,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280898</ExpertLink>
       <Name lang="es">Panuveítis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -211451,7 +211444,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280892</ExpertLink>
       <Name lang="es">Uveítis posterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -211483,7 +211476,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280886</ExpertLink>
       <Name lang="es">Uveítis anterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -211687,7 +211680,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=281085</ExpertLink>
       <Name lang="es">Ictiosis hereditaria sindrómica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -211719,7 +211712,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=281082</ExpertLink>
       <Name lang="es">Ictiosis hereditaria no sindrómica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -218216,7 +218209,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300755</ExpertLink>
       <Name lang="es">Laminopatía con afectación del músculo estriado</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -218280,7 +218273,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300763</ExpertLink>
       <Name lang="es">Laminopatía con lipodistrofia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -218312,7 +218305,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300758</ExpertLink>
       <Name lang="es">Laminopatía con neuropatía periférica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -218344,7 +218337,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300842</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin de células B indolente</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -218376,7 +218369,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300766</ExpertLink>
       <Name lang="es">Laminopatía con envejecimiento prematuro</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -219128,7 +219121,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=298644</ExpertLink>
       <Name lang="es">Trastorno del metabolismo y transporte de la tiamina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -234227,7 +234220,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398043</ExpertLink>
       <Name lang="es">Tumor maligno del pene</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -236630,7 +236623,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398934</ExpertLink>
       <Name lang="es">Tumor epitelial maligno de ovario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -237131,7 +237124,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398940</ExpertLink>
       <Name lang="es">Tumor no epitelial maligno de ovario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -253036,7 +253029,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=431353</ExpertLink>
       <Name lang="es">Enfermedad veno-oclusiva pulmonar y/o hemangiomatosis capilar pulmonar</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -262196,7 +262189,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=450</ExpertLink>
       <Name lang="es">Heterotaxia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -262279,7 +262272,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=224</ExpertLink>
       <Name lang="es">Diabetes mellitus neonatal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -264434,7 +264427,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=877</ExpertLink>
       <Name lang="es">Tumor neuroendocrino</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -264729,7 +264722,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=467</ExpertLink>
       <Name lang="es">Deficiencia combinada no adquirida de hormonas hipofisarias</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -273445,7 +273438,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=71198</ExpertLink>
       <Name lang="es">Hipertensión pulmonar rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -276294,7 +276287,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=69028</ExpertLink>
       <Name lang="es">Disostosis con braquidactilia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -279339,7 +279332,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=63454</ExpertLink>
       <Name lang="es">Displasia en patrón del epitelio pigmentario de la retina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -279420,7 +279413,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=63443</ExpertLink>
       <Name lang="es">Tumor epitelial raro de estómago</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -280669,7 +280662,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=56970</ExpertLink>
       <Name lang="es">Enfermedad priónica humana</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -281863,7 +281856,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79213</ExpertLink>
       <Name lang="es">Mucopolisacaridosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -283385,7 +283378,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79168</ExpertLink>
       <Name lang="es">Anomalía en la síntesis de ácidos biliares</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -283417,7 +283410,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79158</ExpertLink>
       <Name lang="es">Aciduria orgánica cerebral</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -287167,7 +287160,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=77240</ExpertLink>
       <Name lang="es">Linfedema primario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -290065,7 +290058,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35705</ExpertLink>
       <Name lang="es">Trastorno neurometabólico por deficiencia de serina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -290408,7 +290401,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35698</ExpertLink>
       <Name lang="es">Síndrome de depleción del ADN mitocondrial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -290847,7 +290840,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=36388</ExpertLink>
       <Name lang="es">Síndrome neurológico paraneoplásico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -291305,7 +291298,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35808</ExpertLink>
       <Name lang="es">Tumor maligno del estroma de los cordones sexuales de los ovarios</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -291831,7 +291824,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35807</ExpertLink>
       <Name lang="es">Tumor maligno de ovario de células germinales</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -294787,7 +294780,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=34533</ExpertLink>
       <Name lang="es">Distrofia corneal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -296298,7 +296291,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=52759</ExpertLink>
       <Name lang="es">Vasculitis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -302078,7 +302071,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=68367</ExpertLink>
       <Name lang="es">Errores congénitos raros del metabolismo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -302643,7 +302636,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90061</ExpertLink>
       <Name lang="es">Uveítis posterior no infecciosa</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -308125,7 +308118,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=88991</ExpertLink>
       <Name lang="es">Malformación cardíaca rara congénita no sindrómica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -313431,7 +313424,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90970</ExpertLink>
       <Name lang="es">Lipodistrofias primarias</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -314088,7 +314081,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79373</ExpertLink>
       <Name lang="es">Síndrome de displasia ectodérmica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -320337,7 +320330,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79361</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -324947,7 +324940,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=86846</ExpertLink>
       <Name lang="es">Síndrome mielodisplásico y leucemia mieloide aguda asociados al tratamiento</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330286,7 +330279,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98085</ExpertLink>
       <Name lang="es">Anomalía del desarrollo sexual 46,XY</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330437,7 +330430,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98287</ExpertLink>
       <Name lang="es">Tumor de células histiocíticas y dentríticas</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330577,7 +330570,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98292</ExpertLink>
       <Name lang="es">Mastocitosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330679,7 +330672,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98277</ExpertLink>
       <Name lang="es">Leucemia mieloide aguda con anomalía genética recurrente</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330743,7 +330736,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97245</ExpertLink>
       <Name lang="es">Miopatía congénita</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330775,7 +330768,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97242</ExpertLink>
       <Name lang="es">Distrofia muscular congénita</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -330858,7 +330851,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97253</ExpertLink>
       <Name lang="es">Tumor neuroendocrino del páncreas</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -333313,7 +333306,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97593</ExpertLink>
       <Name lang="es">Pseudohipoparatiroidismo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -333528,7 +333521,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=95708</ExpertLink>
       <Name lang="es">Pubertad precoz rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -333777,7 +333770,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=95714</ExpertLink>
       <Name lang="es">Hipotiroidismo congénito primario sin anomalía del desarrollo del tiroides</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -333809,7 +333802,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=95711</ExpertLink>
       <Name lang="es">Hipotiroidismo congénito por anomalía del desarrollo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -337082,7 +337075,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93420</ExpertLink>
       <Name lang="es">Condrodisplasia asociada al gen FGFR3</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -339155,7 +339148,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93890</ExpertLink>
       <Name lang="es">Defectos raros del desarrollo embrionario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -351680,7 +351673,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98555</ExpertLink>
       <Name lang="es">Microftalmia-anoftalmia-coloboma</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -352680,7 +352673,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98627</ExpertLink>
       <Name lang="es">Distrofia corneal posterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -352712,7 +352705,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98626</ExpertLink>
       <Name lang="es">Distrofia corneal estromal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -352744,7 +352737,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98625</ExpertLink>
       <Name lang="es">Distrofia corneal superficial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -352897,7 +352890,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98715</ExpertLink>
       <Name lang="es">Uveítis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -354783,7 +354776,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98482</ExpertLink>
       <Name lang="es">Miopatía inflamatoria idiopática</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -354879,7 +354872,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98428</ExpertLink>
       <Name lang="es">Policitemia secundaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -355788,7 +355781,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98497</ExpertLink>
       <Name lang="es">Neuropatía periférica genética</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -356568,7 +356561,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=101997</ExpertLink>
       <Name lang="es">Inmunodeficiencia primaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -356847,7 +356840,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=101959</ExpertLink>
       <Name lang="es">Insuficiencia suprarrenal crónica primaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -359262,7 +359255,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100088</ExpertLink>
       <Name lang="es">Carcinoma de tiroides</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -359839,7 +359832,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100087</ExpertLink>
       <Name lang="es">Tumor de tiroides</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
