commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml
index 69474b6..5a2eb71 100644
--- a/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/es_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 05:00:58" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:58:03" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6063">
+  <DisorderList count="6058">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -8133,16 +8133,16 @@
           <PrevalenceClass id="23739">
             <Name lang="es">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.22</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
+          <ValMoy>6.66</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="es">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="293">
-          <Source>9918480[PMID]</Source>
+          <Source>35024668[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="es">Prevalencia al nacer</Name>
           </PrevalenceType>
@@ -8152,7 +8152,7 @@
           <PrevalenceClass id="23739">
             <Name lang="es">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.85</ValMoy>
+          <ValMoy>6.96</ValMoy>
           <PrevalenceGeographic id="23949">
             <Name lang="es">Australia</Name>
           </PrevalenceGeographic>
@@ -23605,7 +23605,7 @@
     <Disorder id="17870">
       <OrphaCode>169796</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=169796</ExpertLink>
-      <Name lang="es">Hemofilia B moderadamente grave</Name>
+      <Name lang="es">Hemofilia B moderada</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -24242,7 +24242,7 @@
     <Disorder id="17915">
       <OrphaCode>171622</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=171622</ExpertLink>
-      <Name lang="es">Paraparesia espástica autosómica dominante tipo 32</Name>
+      <Name lang="es">Paraparesia espástica autosómica recesiva tipo 32</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -46243,7 +46243,7 @@
     <Disorder id="17996">
       <OrphaCode>177926</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=177926</ExpertLink>
-      <Name lang="es">Forma sintomática de hemofilia A en mujeres portadoras</Name>
+      <Name lang="es">Trastorno de sangrado en portadores de hemofilia A</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -46782,7 +46782,7 @@
     <Disorder id="17997">
       <OrphaCode>177929</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=177929</ExpertLink>
-      <Name lang="es">Forma sintomática de hemofilia B en mujeres portadoras</Name>
+      <Name lang="es">Trastorno de sangrado en portadores de hemofilia B</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -74189,7 +74189,7 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="2539">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -74198,12 +74198,50 @@
           <PrevalenceQualification id="23704">
             <Name lang="es">Class only</Name>
           </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16781">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="es">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="es">Desconocido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="es">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="es">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16782">
+          <Source>20734338[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="es">Prevalencia al nacer</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="es">Value and class</Name>
+          </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="es">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
+          <ValMoy>1.75</ValMoy>
+          <PrevalenceGeographic id="23949">
+            <Name lang="es">Australia</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="es">Validated</Name>
@@ -82623,38 +82661,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=946</ExpertLink>
-      <Name lang="es">Acrocefalosindactilia</Name>
-      <DisorderType id="21436">
-        <Name lang="es">Grupo de fenomas</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="es">Grupo de trastornos</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="2783">
-          <Source>ORPHANET_[EXPERT]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="es">Desconocido</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=957</ExpertLink>
@@ -86517,7 +86523,7 @@
     <Disorder id="16886">
       <OrphaCode>139402</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=139402</ExpertLink>
-      <Name lang="es">Erupción con eosinofilia y síntomas sistémicos causada por medicamentos</Name>
+      <Name lang="es">Reacción a fármacos con eosinofilia y síntomas sistémicos</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -99304,55 +99310,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1533</ExpertLink>
-      <Name lang="es">Síndrome de craneosinostosis-aplasia de peroné</Name>
-      <DisorderType id="21401">
-        <Name lang="es">Síndrome de malformación</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="3180">
-          <Source>4403018[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="es">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="3181">
-          <Source>ORPHANET_4403018[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="es">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=140874</ExpertLink>
@@ -114713,7 +114670,7 @@
     <Disorder id="2276">
       <OrphaCode>2463</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2463</ExpertLink>
-      <Name lang="es">Síndrome de hábito marfanoide-discapacidad intelectual autosómica recesiva</Name>
+      <Name lang="es">Síndrome de hábito marfanoide-discapacidad intelectual autosómico recesivo</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -160231,7 +160188,7 @@
     <Disorder id="18667">
       <OrphaCode>209908</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=209908</ExpertLink>
-      <Name lang="es">Apraxia del habla infantil</Name>
+      <Name lang="es">Apraxia del habla infantil aislada</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -173396,7 +173353,7 @@
     <Disorder id="18855">
       <OrphaCode>217399</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217399</ExpertLink>
-      <Name lang="es">Insensibilidad congénita al dolor con hiperhidrosis</Name>
+      <Name lang="es">Insensibilidad congénita al dolor-hiperhidrosis-ausencia de inervación de fibras C</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -190778,55 +190735,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=329341</ExpertLink>
-      <Name lang="es">Encefalitis límbica con anticuerpos anti-DPP6</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="10180">
-          <Source>23225603[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="es">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10181">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="es">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=329228</ExpertLink>
@@ -214279,7 +214187,7 @@
     <Disorder id="21452">
       <OrphaCode>313800</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=313800</ExpertLink>
-      <Name lang="es">Síndrome de edema del nervio óptico-esplenomegalia</Name>
+      <Name lang="es">Síndrome de distrofia retiniana-edema del nervio óptico-esplenomegalia-anhidrosis-cefalea migrañosa</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -233391,7 +233299,7 @@
     <Disorder id="22604">
       <OrphaCode>391320</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391320</ExpertLink>
-      <Name lang="es">Coagulopatía del este de Texas</Name>
+      <Name lang="es">Trastorno de sangrado del este de Texas</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -239354,7 +239262,7 @@
     <Disorder id="23331">
       <OrphaCode>436169</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436169</ExpertLink>
-      <Name lang="es">Trastorno de la coagulación asociado a la trombomodulina</Name>
+      <Name lang="es">Trastorno de sangrado asociado a trombomodulina</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -245806,7 +245714,7 @@
     <Disorder id="23114">
       <OrphaCode>420566</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=420566</ExpertLink>
-      <Name lang="es">Trastorno de la coagulación por deficiencia de CalDAG-GEFI</Name>
+      <Name lang="es">Trastorno de sangrado por deficiencia de CalDAG-GEFI</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -265668,55 +265576,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=498700</ExpertLink>
-      <Name lang="es">Encefalitis límbica por anticuerpos anti-neurexina-3</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13782">
-          <Source>27170573[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="es">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="es">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13783">
-          <Source>ORPHANET_27170573[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="es">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="es">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="es">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=874</ExpertLink>
@@ -269751,7 +269610,7 @@
     <Disorder id="25153">
       <OrphaCode>477787</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477787</ExpertLink>
-      <Name lang="es">Trastorno de la coagulación asociado con una deficiencia de fosfolipasa A2 alfa citosólica</Name>
+      <Name lang="es">Trastorno de sangrado asociado a una deficiencia de fosfolipasa A2 alfa citosólica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -279297,38 +279156,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=63440</ExpertLink>
-      <Name lang="es">Oxicefalia aislada</Name>
-      <DisorderType id="21415">
-        <Name lang="es">Anomalía morfológica</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="5872">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="es">Prevalencia puntual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="es">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="es">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="es">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="es">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=63442</ExpertLink>
@@ -282741,7 +282568,7 @@
     <Disorder id="11215">
       <OrphaCode>79189</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79189</ExpertLink>
-      <Name lang="es">Trastorno de la biogénesis peroxisomal</Name>
+      <Name lang="es">Trastorno de la biogénesis del peroxisoma</Name>
       <DisorderType id="21436">
         <Name lang="es">Grupo de fenomas</Name>
       </DisorderType>
@@ -285185,7 +285012,7 @@
     <Disorder id="11106">
       <OrphaCode>77293</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=77293</ExpertLink>
-      <Name lang="es">Enfermedad de Niemann-Pick tipo B</Name>
+      <Name lang="es">Deficiencia de esfingomielinasa ácida visceral crónica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -285298,7 +285125,7 @@
     <Disorder id="11105">
       <OrphaCode>77292</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=77292</ExpertLink>
-      <Name lang="es">Enfermedad de Niemann-Pick tipo A</Name>
+      <Name lang="es">Deficiencia de esfingomielinasa ácida neurovisceral infantil</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -291136,7 +290963,7 @@
     <Disorder id="10417">
       <OrphaCode>36355</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=36355</ExpertLink>
-      <Name lang="es">Trastorno de la coagulación por defecto en P2Y12</Name>
+      <Name lang="es">Trastorno de sangrado por un defecto de P2Y12</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -304185,7 +304012,7 @@
     <Disorder id="11918">
       <OrphaCode>90000</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90000</ExpertLink>
-      <Name lang="es">Eritema elevado persistente</Name>
+      <Name lang="es">Eritema elevatum diutinum</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -306454,7 +306281,7 @@
     <Disorder id="11818">
       <OrphaCode>88642</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=88642</ExpertLink>
-      <Name lang="es">Insensibilidad congénita al dolor asociada a canalopatía</Name>
+      <Name lang="es">Insensibilidad congénita al dolor-anosmia-artropatía neuropática</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -329723,7 +329550,7 @@
     <Disorder id="11619">
       <OrphaCode>84142</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=84142</ExpertLink>
-      <Name lang="es">Síndrome de Isaac</Name>
+      <Name lang="es">Síndrome de Isaacs</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
