commit bbf94d552a3c459862632b82d2e2166d94fb6b4e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml
index ce55adb..1beb1c6 100644
--- a/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:37:49" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:40:46" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6042">
+  <DisorderList count="6043">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=166024</ExpertLink>
@@ -114300,6 +114300,38 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="19474">
+      <OrphaCode>244275</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=244275</ExpertLink>
+      <Name lang="fr">Microangiopathie thrombotique de novo après transplantation rénale</Name>
+      <DisorderType id="21429">
+        <Name lang="fr">Situation clinique particulière dans une maladie ou un syndrome</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
+      </DisorderGroup>
+      <PrevalenceList count="1">
+        <Prevalence id="14647">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="fr">Prévalence ponctuelle</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="fr">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="fr">Inconnu</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="fr">Mondial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25965">
+            <Name lang="fr">Not yet validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19468">
       <OrphaCode>243343</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=243343</ExpertLink>
