commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml
old mode 100644
new mode 100755
index bcb467e..f02570b
--- a/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:41:45" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:42:23" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6043">
+  <DisorderList count="6039">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=166024</ExpertLink>
@@ -34631,7 +34631,7 @@
     <Disorder id="485">
       <OrphaCode>1880</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=1880</ExpertLink>
-      <Name lang="fr">Malformation d'Ebstein</Name>
+      <Name lang="fr">Malformation d'Ebstein de la valve tricuspide</Name>
       <DisorderType id="21415">
         <Name lang="fr">Anomalie morphologique</Name>
       </DisorderType>
@@ -76898,7 +76898,7 @@
     <Disorder id="1007">
       <OrphaCode>528</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=528</ExpertLink>
-      <Name lang="fr">Lipodystrophie congénitale de Berardinelli-Seip</Name>
+      <Name lang="fr">Lipodystrophie généralisée congénitale</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -86805,38 +86805,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="16891">
-      <OrphaCode>139420</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=139420</ExpertLink>
-      <Name lang="fr">Myélite transverse aiguë secondaire</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="8137">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="fr">Prévalence ponctuelle</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="fr">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="fr">Inconnu</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="fr">Mondial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="fr">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16892">
       <OrphaCode>139423</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=139423</ExpertLink>
@@ -90604,7 +90572,7 @@
     <Disorder id="16705">
       <OrphaCode>137678</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=137678</ExpertLink>
-      <Name lang="fr">Dysplasie tchèque type métatarsique</Name>
+      <Name lang="fr">Dysplasie spondyloépiphysaire avec raccourcissement des métatarsiens</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -92450,7 +92418,7 @@
     <Disorder id="1451">
       <OrphaCode>1200</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=1200</ExpertLink>
-      <Name lang="fr">Syndrome d'atrésie des choanes-surdité-cardiopathie-dysmorphie craniofaciale</Name>
+      <Name lang="fr">Syndrome de Burn-McKeown</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -109939,7 +109907,7 @@
     <Disorder id="17237">
       <OrphaCode>158687</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=158687</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse létale acantholytique</Name>
+      <Name lang="fr">Maladie érosive acantholytique létale</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -110153,11 +110121,11 @@
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=158676</ExpertLink>
       <Name lang="fr">Epidermolyse bulleuse dystrophique dominante isolée des ongles</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8266">
@@ -110202,11 +110170,11 @@
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=158673</ExpertLink>
       <Name lang="fr">Epidermolyse bulleuse dystrophique acrale</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8263">
@@ -110250,7 +110218,7 @@
     <Disorder id="17232">
       <OrphaCode>158668</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=158668</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse simple par déficit en plakophiline</Name>
+      <Name lang="fr">Syndrome de dysplasie ectodermique-fragilité cutanée</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -121095,11 +121063,11 @@
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=251393</ExpertLink>
       <Name lang="fr">Epidermolyse bulleuse jonctionnelle localisée type non-Herlitz</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="9158">
@@ -131324,7 +131292,7 @@
     <Disorder id="19879">
       <OrphaCode>261476</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=261476</ExpertLink>
-      <Name lang="fr">Syndrome de microdélétion Xp21</Name>
+      <Name lang="fr">Syndrome de délétion Xp21</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -149972,7 +149940,7 @@
     <Disorder id="20470">
       <OrphaCode>280406</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=280406</ExpertLink>
-      <Name lang="fr">Syndrome néphrotique familial cortico-résistant avec surdité neurosensorielle</Name>
+      <Name lang="fr">Syndrome néphrotique familial corticorésistant avec surdité neurosensorielle</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -155779,7 +155747,7 @@
     <Disorder id="3298">
       <OrphaCode>832</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=832</ExpertLink>
-      <Name lang="fr">Déficit en succinyl-CoA:3-cétoacide CoA transférase</Name>
+      <Name lang="fr">Déficit en succinyl-CoA:3-oxoacide CoA transférase</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -189720,7 +189688,7 @@
     <Disorder id="19334">
       <OrphaCode>240103</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=240103</ExpertLink>
-      <Name lang="fr">Syndrome de paralysie supranucléaire progressive-syndrome cortico-basal</Name>
+      <Name lang="fr">Syndrome de paralysie supranucléaire progressive-syndrome corticobasal</Name>
       <DisorderType id="21450">
         <Name lang="fr">Sous-type clinique</Name>
       </DisorderType>
@@ -222830,7 +222798,7 @@
     <Disorder id="23668">
       <OrphaCode>454887</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=454887</ExpertLink>
-      <Name lang="fr">Syndrome cortico-basal</Name>
+      <Name lang="fr">Syndrome corticobasal</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -228694,7 +228662,7 @@
     <Disorder id="24234">
       <OrphaCode>468678</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=468678</ExpertLink>
-      <Name lang="fr">Syndrome de déficience intellectuelle-microcéphalie-strabisme-troubles du comportement</Name>
+      <Name lang="fr">Syndrome de White-Sutton</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -259216,7 +259184,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=508529</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse simple basale généralisée avec atrophie cutanée, cicatrices et alopécie</Name>
+      <Name lang="fr">Epidermolyse bulleuse simple intermédiaire avec cardiomyopathie</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -268662,7 +268630,7 @@
     <Disorder id="8620">
       <OrphaCode>2908</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=2908</ExpertLink>
-      <Name lang="fr">Syndrome de Kindler</Name>
+      <Name lang="fr">Epidermolyse bulleuse de Kindler</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -276706,7 +276674,7 @@
     <Disorder id="10910">
       <OrphaCode>69061</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=69061</ExpertLink>
-      <Name lang="fr">Syndrome néphrotique idiopathique cortico-sensible</Name>
+      <Name lang="fr">Syndrome néphrotique idiopathique corticosensible</Name>
       <DisorderType id="21422">
         <Name lang="fr">Syndrome clinique</Name>
       </DisorderType>
@@ -305106,119 +305074,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="11906">
-      <OrphaCode>89841</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=89841</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse dystrophique récessive centripète</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="6950">
-          <Source>Dr Giovanna ZAMBRUNO[EXPERT]_4026337[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="fr">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="fr">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>10.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="fr">Mondial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="fr">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6951">
-          <Source>ORPHANET_4026337[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="fr">Prévalence ponctuelle</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="fr">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="fr">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="fr">Mondial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="fr">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="11905">
-      <OrphaCode>89840</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=89840</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse jonctionnelle type non-Herlitz</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="10622">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="fr">Prévalence ponctuelle</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="fr">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="fr">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="fr">Europe</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="fr">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="11904">
-      <OrphaCode>89839</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=89839</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse simple superficielle</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="6949">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="fr">Prévalence ponctuelle</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="fr">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="fr">Inconnu</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="fr">Mondial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="fr">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="11919">
       <OrphaCode>90001</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=90001</ExpertLink>
@@ -307947,7 +307802,7 @@
     <Disorder id="11805">
       <OrphaCode>88618</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=88618</ExpertLink>
-      <Name lang="fr">Retard psychomoteur par déficit en S-adénosylhomocystéine hydrolase</Name>
+      <Name lang="fr">Déficit en S-adénosylhomocystéine hydrolase</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -315423,7 +315278,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=79401</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse simple type Ogna</Name>
+      <Name lang="fr">Epidermolyse bulleuse simple sans manifestations extracutanées associée à PLEC</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -315554,11 +315409,11 @@
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=79402</ExpertLink>
       <Name lang="fr">Epidermolyse bulleuse jonctionnelle généralisée intermédiaire</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <PrevalenceList count="1">
         <Prevalence id="6509">
@@ -315960,11 +315815,11 @@
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=79410</ExpertLink>
       <Name lang="fr">Epidermolyse bulleuse dystrophique prétibiale</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="6526">
@@ -334617,7 +334472,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=95702</ExpertLink>
-      <Name lang="fr">Hypoplasie surrénalienne congénitale cytomégalique</Name>
+      <Name lang="fr">Hypoplasie congénitale des surrénales liée à l'X due à une mutation ponctuelle de NR0B1</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
