commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml
old mode 100755
new mode 100644
index 6858ffb..fbddddb
--- a/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/fr_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:44:18" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:46:48" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -83398,7 +83398,7 @@
     <Disorder id="1276">
       <OrphaCode>955</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=955</ExpertLink>
-      <Name lang="fr">Acro-ostéolyse autosomique dominante</Name>
+      <Name lang="fr">Syndrome de Hajdu-Cheney</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -267914,7 +267914,7 @@
       <DisorderGroup id="36540">
         <Name lang="fr">Groupe de pathologies</Name>
       </DisorderGroup>
-      <PrevalenceList count="12">
+      <PrevalenceList count="11">
         <Prevalence id="5195">
           <Source>20566927[PMID]</Source>
           <PrevalenceType id="23669">
@@ -267942,10 +267942,10 @@
           <PrevalenceQualification id="23711">
             <Name lang="fr">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="fr">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="fr">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.1</ValMoy>
+          <ValMoy>0.572</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="fr">Europe</Name>
           </PrevalenceGeographic>
@@ -268105,27 +268105,8 @@
             <Name lang="fr">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="5205">
-          <Source>ISBN:951563203[OTHER]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="fr">Prévalence ponctuelle</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="fr">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="fr">1-9 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.12</ValMoy>
-          <PrevalenceGeographic id="25279">
-            <Name lang="fr">South Africa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="fr">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
         <Prevalence id="5206">
-          <Source>ISBN:801860245[OTHER]</Source>
+          <Source>PMID: 27463098</Source>
           <PrevalenceType id="23669">
             <Name lang="fr">Prévalence ponctuelle</Name>
           </PrevalenceType>
@@ -268135,7 +268116,7 @@
           <PrevalenceClass id="23739">
             <Name lang="fr">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.24</ValMoy>
+          <ValMoy>0.14</ValMoy>
           <PrevalenceGeographic id="25482">
             <Name lang="fr">United States</Name>
           </PrevalenceGeographic>
@@ -315386,7 +315367,7 @@
     <Disorder id="11435">
       <OrphaCode>79409</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=79409</ExpertLink>
-      <Name lang="fr">Epidermolyse bulleuse dystrophique inversée</Name>
+      <Name lang="fr">Epidermolyse bulleuse dystrophique inversée autosomique récessive</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
