commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml
index a59a995..1fcd8b3 100644
--- a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:53:08" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6044">
+<JDBOR date="2021-12-01 04:57:46" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6052">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -2886,7 +2879,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="11716">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -2896,7 +2889,7 @@
           <PrevalenceClass id="23739">
             <Name lang="it">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.31</ValMoy>
+          <ValMoy>0.28</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="it">Brazil</Name>
           </PrevalenceGeographic>
@@ -4068,7 +4061,7 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <PrevalenceList count="8">
+      <PrevalenceList count="9">
         <Prevalence id="136">
           <Source>European Medicines Agency 2012[INST]</Source>
           <PrevalenceType id="23669">
@@ -4107,6 +4100,25 @@
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16640">
+          <Source>PMID: 31926052</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="it">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.02</ValMoy>
+          <PrevalenceGeographic id="24068">
+            <Name lang="it">Brazil</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="11717">
           <Source>25060283[PMID]</Source>
           <PrevalenceType id="23683">
@@ -14374,7 +14386,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13915">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -14384,7 +14396,7 @@
           <PrevalenceClass id="23739">
             <Name lang="it">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.38</ValMoy>
+          <ValMoy>0.37</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="it">Brazil</Name>
           </PrevalenceGeographic>
@@ -14862,7 +14874,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13910">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -14913,7 +14925,7 @@
       </DisorderGroup>
       <PrevalenceList count="11">
         <Prevalence id="520">
-          <Source>17276780[PMID]</Source>
+          <Source>PMID: 31449670</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -14923,7 +14935,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.2</ValMoy>
+          <ValMoy>2.25</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
@@ -14951,7 +14963,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="522">
-          <Source>DOI:10.1016/S0168-8278(85)80013-1[OTHER]_17276780[PMID]</Source>
+          <Source>PMID: 31449670</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -14961,7 +14973,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>3.3</ValMoy>
+          <ValMoy>2.02</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
@@ -16167,7 +16179,7 @@
     <Disorder id="17515">
       <OrphaCode>163654</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=163654</ExpertLink>
-      <Name lang="it">Displasia spondiloepifisaria, tipo Cantu</Name>
+      <Name lang="it">Sindrome da displasia spondiloepifisaria-brachidattilia-disturbo del linguaggio</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -20001,7 +20013,7 @@
     <Disorder id="209">
       <OrphaCode>628</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=628</ExpertLink>
-      <Name lang="it">Nanismo diastrofico</Name>
+      <Name lang="it">Displasia diastrofica</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -24296,7 +24308,7 @@
     <Disorder id="294">
       <OrphaCode>308</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=308</ExpertLink>
-      <Name lang="it">Malattia di Unverricht-Lundborg</Name>
+      <Name lang="it">Epilessia mioclonica progressiva tipo 1</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -38594,11 +38606,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2346</ExpertLink>
       <Name lang="it">Sindrome angio-osteoipertrofica</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21436">
+        <Name lang="it">Gruppo di fenomi</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36540">
+        <Name lang="it">Gruppo di malattie</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="1286">
@@ -46160,7 +46172,26 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <PrevalenceList count="25">
+      <PrevalenceList count="26">
+        <Prevalence id="16639">
+          <Source>PMID: 31926052</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="it">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.21</ValMoy>
+          <PrevalenceGeographic id="24068">
+            <Name lang="it">Brazil</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="1671">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
@@ -46883,7 +46914,7 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <PrevalenceList count="8">
+      <PrevalenceList count="6">
         <Prevalence id="1689">
           <Source>PMID: 31769437, 25944380, 24715559</Source>
           <PrevalenceType id="23669">
@@ -46998,40 +47029,6 @@
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="16436">
-          <Source>PMID: 22913777</Source>
-          <PrevalenceType id="23697">
-            <Name lang="it">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="it">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>1314.0</ValMoy>
-          <PrevalenceGeographic id="24166">
-            <Name lang="it">China</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="16435">
-          <Source>PMID: 26627451</Source>
-          <PrevalenceType id="23697">
-            <Name lang="it">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="it">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>72.0</ValMoy>
-          <PrevalenceGeographic id="25356">
-            <Name lang="it">Taiwan, Province of China</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="17999">
@@ -55675,7 +55672,7 @@
     <Disorder id="17947">
       <OrphaCode>171871</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171871</ExpertLink>
-      <Name lang="it">Pseudoipoaldosteronismo renale, tipo 1</Name>
+      <Name lang="it">Pseudoipoaldosteronismo renale tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -56300,7 +56297,7 @@
     <Disorder id="17948">
       <OrphaCode>171876</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171876</ExpertLink>
-      <Name lang="it">Pseudoipoaldosteronismo generalizzato, tipo 1</Name>
+      <Name lang="it">Pseudoipoaldosteronismo generalizzato tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -66758,7 +66755,7 @@
           <PrevalenceClass id="23739">
             <Name lang="it">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.11</ValMoy>
+          <ValMoy>0.14</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="it">Brazil</Name>
           </PrevalenceGeographic>
@@ -71890,7 +71887,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183678</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak con neutropenia</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di AP-3</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -73335,7 +73332,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2262">
-          <Source>15054950[PMID]_10817953[PMID]_[EXPERT]</Source>
+          <Source>PMID: 27037234  ; 34220661; ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -73345,7 +73342,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>6.0</ValMoy>
+          <ValMoy>5.26</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
@@ -75630,27 +75627,27 @@
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
-        <Prevalence id="11997">
-          <Source>20301500[PMID]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="it">Prevalenza alla nascita</Name>
+        <Prevalence id="16626">
+          <Source>PMID: 31969461</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="it">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.125</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
+          <ValMoy>0.47</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="12014">
-          <Source>Kroes_Fransen van de Putte_Ravesloot 2007[AUTHOR]</Source>
+        <Prevalence id="11997">
+          <Source>20301500[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -75660,9 +75657,9 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.0</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="it">Netherlands</Name>
+          <ValMoy>1.125</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
@@ -75676,15 +75673,15 @@
           <PrevalenceQualification id="23704">
             <Name lang="it">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -77919,9 +77916,9 @@
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
       </DisorderGroup>
-      <PrevalenceList count="5">
-        <Prevalence id="2605">
-          <Source>14518075[PMID]</Source>
+      <PrevalenceList count="6">
+        <Prevalence id="16623">
+          <Source>PMID: 26431509</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -77931,16 +77928,16 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="it">Netherlands</Name>
+          <ValMoy>2.24</ValMoy>
+          <PrevalenceGeographic id="24257">
+            <Name lang="it">Czech Republic</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2606">
-          <Source>9130925[PMID]_14518075[PMID]</Source>
+        <Prevalence id="16624">
+          <Source>PMID: 31742894</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -77950,54 +77947,73 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="25482">
-            <Name lang="it">United States</Name>
+          <ValMoy>5.25</ValMoy>
+          <PrevalenceGeographic id="24621">
+            <Name lang="it">Japan</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2607">
-          <Source>21967988[PMID]</Source>
+        <Prevalence id="16625">
+          <Source>PMID :32027664</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>40.0</ValMoy>
-          <PrevalenceGeographic id="25209">
-            <Name lang="it">Saudi Arabia</Name>
+          <ValMoy>3.52</ValMoy>
+          <PrevalenceGeographic id="25475">
+            <Name lang="it">United Kingdom</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2608">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
+        <Prevalence id="2605">
+          <Source>14518075[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
           </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="24936">
+            <Name lang="it">Netherlands</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="2607">
+          <Source>21967988[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="it">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>40.0</ValMoy>
+          <PrevalenceGeographic id="25209">
+            <Name lang="it">Saudi Arabia</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10928">
-          <Source>[EXPERT]_ORPHANET</Source>
+          <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -78007,7 +78023,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
+          <ValMoy>3.67</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
@@ -81092,6 +81108,38 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="1228">
+      <OrphaCode>3306</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3306</ExpertLink>
+      <Name lang="it">Duplicazione/inversione 15q11</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="1">
+        <Prevalence id="16620">
+          <Source>PMID: 11579431</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>3.33</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="1231">
       <OrphaCode>3375</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3375</ExpertLink>
@@ -95582,7 +95630,7 @@
     <Disorder id="1546">
       <OrphaCode>1323</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1323</ExpertLink>
-      <Name lang="it">Camptodattilia - contratture articolari - anomalie facciali e scheletriche</Name>
+      <Name lang="it">Sindrome da camptodattilia-contratture articolari-anomalie facciali e scheletriche</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -102325,7 +102373,7 @@
     <Disorder id="1899">
       <OrphaCode>1952</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1952</ExpertLink>
-      <Name lang="it">Displasia di Pacman</Name>
+      <Name lang="it">Sindrome delle epifisi punteggiate-iperplasia osteoclastica</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -127069,7 +127117,7 @@
     <Disorder id="2314">
       <OrphaCode>2502</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2502</ExpertLink>
-      <Name lang="it">Disostosi metafisaria - ritardo mentale - sordità</Name>
+      <Name lang="it">Sindrome da disostosi metafisaria-disabilità intellettiva-sordità di conduzione</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -154239,7 +154287,7 @@
     <Disorder id="3264">
       <OrphaCode>412</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=412</ExpertLink>
-      <Name lang="it">Iperlipoproteinemia, tipo 3</Name>
+      <Name lang="it">Disbetalipoproteinemia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -159635,7 +159683,7 @@
     <Disorder id="3134">
       <OrphaCode>2582</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2582</ExpertLink>
-      <Name lang="it">Sindrome da mialgia ed eosinofilia correlaa al triptofano</Name>
+      <Name lang="it">Sindrome da mialgia ed eosinofilia correlata al triptofano</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -173465,7 +173513,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8794">
-          <Source>23625533[PMID]_25665837[PMID]</Source>
+          <Source>PMID: 33365252 ; 33485800</Source>
           <PrevalenceType id="23697">
             <Name lang="it">Cases/families</Name>
           </PrevalenceType>
@@ -173473,7 +173521,7 @@
             <Name lang="it">Case</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>19.0</ValMoy>
+          <ValMoy>32.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
@@ -173482,7 +173530,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="8795">
-          <Source>ORPHANET_23625533[PMID]_25665837[PMID]</Source>
+          <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -183959,7 +184007,7 @@
     <Disorder id="19198">
       <OrphaCode>231531</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231531</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak, tipo 7</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di BLOC-1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -184008,7 +184056,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231512</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak senza fibrosi polmonare</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -184180,7 +184228,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231500</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak con fibrosi polmonare</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -206323,7 +206371,7 @@
     <Disorder id="22057">
       <OrphaCode>352333</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=352333</ExpertLink>
-      <Name lang="it">Sindrome ittiosi congenita-deficit cognitivo-quadriplegia spastica</Name>
+      <Name lang="it">Sindrome da ittiosi congenita-disabilità intellettiva-quadriplegia spastica</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -214723,7 +214771,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=314555</ExpertLink>
-      <Name lang="it">Sindrome da displasia craniofacciale ed osteopenia</Name>
+      <Name lang="it">Sindrome da dismorfismi facciali-miopia grave-osteopenia-disabilità intellettiva-anomalie dei denti</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -222562,7 +222610,7 @@
     <Disorder id="23623">
       <OrphaCode>449432</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=449432</ExpertLink>
-      <Name lang="it">Malattia della ghiandola sottomandibolare IgG4-correlata</Name>
+      <Name lang="it">Malattia della ghiandola sottomandibolare correlata alle IgG4</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -254650,7 +254698,7 @@
     <Disorder id="8739">
       <OrphaCode>756</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=756</ExpertLink>
-      <Name lang="it">Pseudoipoaldosteronismo, tipo 1</Name>
+      <Name lang="it">Pseudoipoaldosteronismo tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -269769,7 +269817,7 @@
     <Disorder id="25161">
       <OrphaCode>477831</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=477831</ExpertLink>
-      <Name lang="it">Sindrome iperaccrescimento scheletrico-dismorfismi craniofacciali - iperelasticità della cute - lesioni della sostanza bianca</Name>
+      <Name lang="it">Sindrome da iperaccrescimento di Kosaki</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -276980,7 +277028,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=66627</ExpertLink>
-      <Name lang="it">Sinovite villonodulare pigmentata</Name>
+      <Name lang="it">Tumore tenosinoviale a cellule giganti</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -282029,7 +282077,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13938">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]_28595941[PMID]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -282039,7 +282087,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.04</ValMoy>
+          <ValMoy>1.25</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="it">Brazil</Name>
           </PrevalenceGeographic>
@@ -282350,7 +282398,7 @@
     <Disorder id="11260">
       <OrphaCode>79234</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79234</ExpertLink>
-      <Name lang="it">Sindrome di Crigler-Najjar, tipo 1</Name>
+      <Name lang="it">Sindrome di Crigler-Najjar tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -282382,7 +282430,7 @@
     <Disorder id="11261">
       <OrphaCode>79235</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79235</ExpertLink>
-      <Name lang="it">Sindrome di Crigler-Najjar, tipo 2</Name>
+      <Name lang="it">Sindrome di Crigler-Najjar tipo 2</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -285525,7 +285573,7 @@
     <Disorder id="11126">
       <OrphaCode>79078</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79078</ExpertLink>
-      <Name lang="it">Dacrioadenite e sialoadenite IgG4-correlate</Name>
+      <Name lang="it">Dacrioadenite e scialoadenite correlate alle IgG4</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -287835,7 +287883,7 @@
     <Disorder id="11071">
       <OrphaCode>75326</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=75326</ExpertLink>
-      <Name lang="it">Tortuosità delle arteriole retiniche</Name>
+      <Name lang="it">Tortuosità delle arterie retiniche</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -293045,7 +293093,7 @@
     <Disorder id="10332">
       <OrphaCode>34149</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=34149</ExpertLink>
-      <Name lang="it">Malattia cistica della midollare renale, autosomica dominante, con o senza iperuricemia</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante </Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -294452,7 +294500,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35062</ExpertLink>
-      <Name lang="it">Infezione idiopatica disseminata da citomegalovirus</Name>
+      <Name lang="it">Infezione grave disseminata da citomegalovirus nei pazienti immunocompetenti</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -303999,29 +304047,29 @@
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
-        <Prevalence id="12928">
-          <Source>European Medicines Agency 2014[INST]</Source>
+        <Prevalence id="16645">
+          <Source>PMID: 26543054</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="it">1-9 / 1 000 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.21</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
+          <ValMoy>1.89</ValMoy>
+          <PrevalenceGeographic id="25006">
+            <Name lang="it">Norway</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="12929">
-          <Source>19095780[PMID]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="it">Prevalenza alla nascita</Name>
+        <Prevalence id="16646">
+          <Source>PMID: 31730177</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
@@ -304029,16 +304077,16 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.95</ValMoy>
-          <PrevalenceGeographic id="24264">
-            <Name lang="it">Denmark</Name>
+          <ValMoy>1.54</ValMoy>
+          <PrevalenceGeographic id="25475">
+            <Name lang="it">United Kingdom</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="13994">
-          <Source>19095780[PMID]</Source>
+        <Prevalence id="12928">
+          <Source>PMID: 26543054 ; 31730177</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -304048,9 +304096,9 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.4</ValMoy>
-          <PrevalenceGeographic id="24264">
-            <Name lang="it">Denmark</Name>
+          <ValMoy>1.66</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="it">Europa</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
@@ -305044,11 +305092,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90308</ExpertLink>
       <Name lang="it">Sindrome di Klippel-Trénaunay</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="7026">
@@ -308235,7 +308283,7 @@
     <Disorder id="11882">
       <OrphaCode>88950</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=88950</ExpertLink>
-      <Name lang="it">Malattia cistica della midollare renale, autosomica dominante, con iperuricemia</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante correlata a UMOD</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -309638,7 +309686,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93111</ExpertLink>
-      <Name lang="it">Sindrome da cisti renali e diabete</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante correlata a HNF1B</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -313184,38 +313232,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="12110">
-      <OrphaCode>91133</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=91133</ExpertLink>
-      <Name lang="it">Osteopenia - miopia - perdità dell'udito - deficit cognitivo - dismorfismi facciali</Name>
-      <DisorderType id="21401">
-        <Name lang="it">Sindrome malformativa</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="7086">
-          <Source>ORPHANET_17230486[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="it">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="12096">
       <OrphaCode>90793</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90793</ExpertLink>
@@ -321631,7 +321647,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=85279</ExpertLink>
-      <Name lang="it">Ritardo mentale sindromico legato all'X da mutazioni di JARID1C</Name>
+      <Name lang="it">Disabilità intellettiva sindromica legata all’X correlata a KDM5C</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -333117,7 +333133,7 @@
     <Disorder id="12856">
       <OrphaCode>97229</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97229</ExpertLink>
-      <Name lang="it">Deficit del trasportatore di riboflavina</Name>
+      <Name lang="it">Deficit del trasportatore della riboflavina</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -336768,6 +336784,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=599373</ExpertLink>
+      <Name lang="it">Encefalopatia correlata a STXBP1</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16637">
+          <Source>PMID: 32643187</Source>
+          <PrevalenceType id="23697">
+            <Name lang="it">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="it">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>282.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16638">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93403</ExpertLink>
@@ -338940,7 +339005,7 @@
     <Disorder id="12453">
       <OrphaCode>93622</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93622</ExpertLink>
-      <Name lang="it">Malattia di Dent, tipo 1</Name>
+      <Name lang="it">Malattia di Dent tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -338972,7 +339037,7 @@
     <Disorder id="12454">
       <OrphaCode>93623</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93623</ExpertLink>
-      <Name lang="it">Malattia di Dent, tipo 2</Name>
+      <Name lang="it">Malattia di Dent tipo 2</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -340425,6 +340490,104 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592564</ExpertLink>
+      <Name lang="it">Ritardo dello sviluppo, convulsioni, disturbi del movimento correlati a GNAO1</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16643">
+          <Source>PMID: 26060304 ; 29758257 ; 30682224 ; 34122306</Source>
+          <PrevalenceType id="23697">
+            <Name lang="it">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="it">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>75.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16644">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="it">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592570</ExpertLink>
+      <Name lang="it">Sindrome da cardiopatia, anomalie delle dita, dismorfismi facciali, ritardo motorio e del linguaggio correlata a TRAF7</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16641">
+          <Source>PMID: 29961569 ;  32376980 ;  32459067 ;  34088006</Source>
+          <PrevalenceType id="23697">
+            <Name lang="it">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="it">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>55.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16642">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="it">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="14245">
       <OrphaCode>99672</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99672</ExpertLink>
@@ -340589,6 +340752,297 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589821</ExpertLink>
+      <Name lang="it">Distrofia miotonica congenita di Steinert</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="16621">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16622">
+          <Source>PMID: 23415617</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.1</ValMoy>
+          <PrevalenceGeographic id="24124">
+            <Name lang="it">Canada</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16627">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="it">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.94</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16628">
+          <Source>PMID: 33472919</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="it">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>47.6</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="it">United States</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589827</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio giovanile</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16631">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16632">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>1.64</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589824</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio infantile</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16629">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16630">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>1.29</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589833</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio tardivo</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16635">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16636">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>8.14</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589830</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio nell'età adulta</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16633">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16634">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="it">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>12.54</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589905</ExpertLink>
@@ -348072,10 +348526,10 @@
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="15">
-        <Prevalence id="7708">
-          <Source>European Medicines Agency 2018[INST]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
+        <Prevalence id="16647">
+          <Source>PMID: 32503598</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
@@ -348083,9 +348537,9 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
+          <ValMoy>9.9</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="it">Validated</Name>
@@ -348244,7 +348698,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="11209">
-          <Source>24780148[PMID]</Source>
+          <Source>PMID: 32503598</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -348254,7 +348708,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>4.78</ValMoy>
+          <ValMoy>2.8</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
@@ -362253,7 +362707,7 @@
     <Disorder id="14511">
       <OrphaCode>99938</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99938</ExpertLink>
-      <Name lang="it">Malattia di Charcot-Marie-Tooth autosomica dominante, tipo 2D</Name>
+      <Name lang="it">Malattia di Charcot-Marie-Tooth autosomica dominante tipo 2D</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
