commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml
index 792c96e..ae8fb29 100644
--- a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:42:25" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 17:46:58" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="6056">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -6961,7 +6954,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=593</ExpertLink>
       <Name lang="it">Miopatia miofibrillare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -22292,7 +22285,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2443</ExpertLink>
       <Name lang="it">Difetto della fosforilazione ossidativa mitocondriale da anomalie del DNA nucleare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -22984,7 +22977,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99</ExpertLink>
       <Name lang="it">Atassia cerebellare, autosomica dominante</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -27830,7 +27823,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=352</ExpertLink>
       <Name lang="it">Galattosemia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -32376,7 +32369,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=167714</ExpertLink>
       <Name lang="it">Leucemia mieloide acuta non classificata</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -41218,7 +41211,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=69</ExpertLink>
       <Name lang="it">Amiloidosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -41434,7 +41427,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166</ExpertLink>
       <Name lang="it">Malattia di Charcot-Marie-Tooth</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -48130,7 +48123,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=177107</ExpertLink>
       <Name lang="it">Ipotiroidismo sindromico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -51289,7 +51282,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171901</ExpertLink>
       <Name lang="it">Linfoma cutaneo primitivo a cellule T</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -51807,7 +51800,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171915</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin a cellule B</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -51935,7 +51928,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171918</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin a cellule T</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -52016,7 +52009,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=187</ExpertLink>
       <Name lang="it">Citrullinemia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -52097,7 +52090,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=442</ExpertLink>
       <Name lang="it">Ipotiroidismo congenito</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -56441,7 +56434,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=795</ExpertLink>
       <Name lang="it">Forma rara di salmonellosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -63359,7 +63352,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1531</ExpertLink>
       <Name lang="it">Craniosinostosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -71604,7 +71597,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183669</ExpertLink>
       <Name lang="it">Agammaglobulinemia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -75145,7 +75138,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183518</ExpertLink>
       <Name lang="it">Atassia ereditaria rara</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -75645,7 +75638,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183435</ExpertLink>
       <Name lang="it">Ittiosi ereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -75947,7 +75940,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183422</ExpertLink>
       <Name lang="it">Malattia genetica da malformazioni multiple associata a un rischio aumentato di tumore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76049,7 +76042,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=182114</ExpertLink>
       <Name lang="it">Tumore raro dell'apparato urogenitale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76132,7 +76125,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=182130</ExpertLink>
       <Name lang="it">Tumore delle ghiandole endocrine</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76285,7 +76278,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=182090</ExpertLink>
       <Name lang="it">Ipertensione arteriosa polmonare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76526,7 +76519,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=182095</ExpertLink>
       <Name lang="it">Pneumopatia interstiziale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76683,7 +76676,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1047</ExpertLink>
       <Name lang="it">Anemia sideroblastica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -84640,7 +84633,7 @@
     <Disorder id="1399">
       <OrphaCode>1118</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1118</ExpertLink>
-      <Name lang="it">Aplasia del perone - ectrodattilia</Name>
+      <Name lang="it">Sindrome da aplasia del perone-ectrodattilia</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -90540,7 +90533,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=138044</ExpertLink>
       <Name lang="it">Malattia rara con sindrome di Pierre Robin</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -99334,7 +99327,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=140874</ExpertLink>
       <Name lang="it">Sindrome di Joubert e sindromi correlate</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -106827,7 +106820,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=158032</ExpertLink>
       <Name lang="it">Sindrome emofagocitica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -111014,7 +111007,7 @@
     <Disorder id="2191">
       <OrphaCode>2339</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2339</ExpertLink>
-      <Name lang="it">Cheratosi follicolare - nanismo - atrofia cerebrale</Name>
+      <Name lang="it">Sindrome da cheratosi follicolare-nanismo-atrofia cerebrale</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -111635,7 +111628,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=247582</ExpertLink>
       <Name lang="it">Deficit di citrina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -115623,7 +115616,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=251852</ExpertLink>
       <Name lang="it">Tumore embrionale del tessuto neuroepiteliale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -123600,7 +123593,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=252057</ExpertLink>
       <Name lang="it">Tumore dei nervi cranici e spinali</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -123730,7 +123723,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=252028</ExpertLink>
       <Name lang="it">Tumore melanocitico primitivo del sistema nervoso centrale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -123862,7 +123855,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=254367</ExpertLink>
       <Name lang="it">Lichen planus raro</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -125920,7 +125913,7 @@
     <Disorder id="2556">
       <OrphaCode>2815</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2815</ExpertLink>
-      <Name lang="it">Paraparesi spastica - sordità</Name>
+      <Name lang="it">Sindrome da paraparesi spastica-sordità</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -127886,7 +127879,7 @@
     <Disorder id="2344">
       <OrphaCode>2536</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2536</ExpertLink>
-      <Name lang="it">Microcornea - glaucoma - assenza del seno frontale</Name>
+      <Name lang="it">Sindrome da microcornea-glaucoma-assenza dei seni frontali</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -134869,7 +134862,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=262941</ExpertLink>
       <Name lang="it">Duplicazione parziale del braccio lungo del cromosoma 14</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -135817,7 +135810,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=268843</ExpertLink>
       <Name lang="it">Malformazione del canale neuroenterico, del midollo spinale e della colonna</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -137552,7 +137545,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2846</ExpertLink>
       <Name lang="it">Anomalia congenita del pericardio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -137714,7 +137707,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=268357</ExpertLink>
       <Name lang="it">Difetto di chiusura del tubo neurale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -142171,7 +142164,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=275791</ExpertLink>
       <Name lang="it">Ipertensione arteriosa polmonare associata ad altra malattia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -144342,7 +144335,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3388</ExpertLink>
       <Name lang="it">Anomalie di chiusura del tubo neurale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -147281,7 +147274,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=268926</ExpertLink>
       <Name lang="it">Malformazione cerebrale della linea mediana</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -147609,7 +147602,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=269224</ExpertLink>
       <Name lang="it">Malformazione cerebellare globale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -160942,7 +160935,7 @@
     <Disorder id="3521">
       <OrphaCode>3240</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3240</ExpertLink>
-      <Name lang="it">Calcificazione del sistema nervoso centrale - sordità - acidosi tubolare - anemia</Name>
+      <Name lang="it">Sindrome da calcificazione del sistema nervoso centrale-sordità-acidosi tubolare-anemia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -161365,7 +161358,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=599</ExpertLink>
       <Name lang="it">Miopatia distale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -162104,7 +162097,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=211062</ExpertLink>
       <Name lang="it">Atassia episodica ereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -162136,7 +162129,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2034</ExpertLink>
       <Name lang="it">Filariosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -162451,7 +162444,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2415</ExpertLink>
       <Name lang="it">Malformazione linfatica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -162958,7 +162951,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=137</ExpertLink>
       <Name lang="it">Disturbo congenito della glicosilazione</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -163645,7 +163638,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=359</ExpertLink>
       <Name lang="it">Glaucoma ereditario</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -163677,7 +163670,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=213500</ExpertLink>
       <Name lang="it">Tumore ovarico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -169075,7 +169068,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=216675</ExpertLink>
       <Name lang="it">Trasposizione delle grandi arterie</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -171199,7 +171192,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=217074</ExpertLink>
       <Name lang="it">Carcinoma raro del pancreas</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -171804,8 +171797,8 @@
           <PrevalenceType id="23697">
             <Name lang="it">Cases/families</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="it">Case</Name>
+          <PrevalenceQualification id="23725">
+            <Name lang="it">Family</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
           <ValMoy>35.0</ValMoy>
@@ -172021,7 +172014,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=344</ExpertLink>
       <Name lang="it">Febbre da arbovirus</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -172053,7 +172046,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=341</ExpertLink>
       <Name lang="it">Febbre emorragica virale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -173248,7 +173241,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1132</ExpertLink>
       <Name lang="it">Anomalie dell'arco aortico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -174365,7 +174358,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=542</ExpertLink>
       <Name lang="it">Linfoma cutaneo primitivo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -175559,7 +175552,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=220489</ExpertLink>
       <Name lang="it">Emocromatosi ereditaria rara</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -177543,7 +177536,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3398</ExpertLink>
       <Name lang="it">Neoplasia epiteliale del timo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -177645,7 +177638,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=547</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -178060,7 +178053,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3399</ExpertLink>
       <Name lang="it">Tumore delle cellule germinali</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -179316,7 +179309,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=226292</ExpertLink>
       <Name lang="it">Ipotiroidismo congenito permanente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -181747,7 +181740,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1172</ExpertLink>
       <Name lang="it">Atassia cerebellare, autosomica recessiva</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -184491,7 +184484,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1866</ExpertLink>
       <Name lang="it">Distonia focale, segmentale o multifocale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -186785,7 +186778,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231637</ExpertLink>
       <Name lang="it">Iperaldosteronismo primitivo raro corregibile chirurgicamente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -188261,7 +188254,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=238547</ExpertLink>
       <Name lang="it">Policitemia secondaria acquisita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -188342,7 +188335,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=238536</ExpertLink>
       <Name lang="it">Policitemia secondaria congenita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -194181,7 +194174,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=319543</ExpertLink>
       <Name lang="it">Suscettibilità mendeliana autosomica dominante a malattie micobatteriche da deficit parziale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -194262,7 +194255,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=319535</ExpertLink>
       <Name lang="it">Suscettibilità mendeliana autosomica recessiva a malattie micobatteriche da deficit completo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -194294,7 +194287,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=319539</ExpertLink>
       <Name lang="it">Suscettibilità mendeliana autosomica recessiva a malattie micobatteriche da deficit parziale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -197243,7 +197236,7 @@
     <Disorder id="21506">
       <OrphaCode>314632</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=314632</ExpertLink>
-      <Name lang="it">Parkinsonimo da deficit di ATP13A2</Name>
+      <Name lang="it">Ceroidolipofuscinosi neuronale correlata a ATP13A2</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -202365,7 +202358,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=363203</ExpertLink>
       <Name lang="it">Cromosoma ad anello</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -202836,7 +202829,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=363579</ExpertLink>
       <Name lang="it">Tumore extragonadico delle cellule germinali</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -203541,7 +203534,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=363472</ExpertLink>
       <Name lang="it">Tumore testicolare e paratesticolare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -211419,7 +211412,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280898</ExpertLink>
       <Name lang="it">Panuveite</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -211451,7 +211444,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280892</ExpertLink>
       <Name lang="it">Uveite posteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -211483,7 +211476,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280886</ExpertLink>
       <Name lang="it">Uveite anteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -211687,7 +211680,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=281085</ExpertLink>
       <Name lang="it">Ittiosi ereditaria, forma sindromica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -211719,7 +211712,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=281082</ExpertLink>
       <Name lang="it">Ittiosi ereditaria, forma non sindromica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -218216,7 +218209,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300755</ExpertLink>
       <Name lang="it">Laminopatia con coinvolgimento dei muscoli striati</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -218280,7 +218273,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300763</ExpertLink>
       <Name lang="it">Laminopatia con lipodistrofia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -218312,7 +218305,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300758</ExpertLink>
       <Name lang="it">Laminopatia con neuropatia periferica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -218344,7 +218337,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300842</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin B indolente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -218376,7 +218369,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300766</ExpertLink>
       <Name lang="it">Laminopatia con invecchiamento precoce</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -219128,7 +219121,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=298644</ExpertLink>
       <Name lang="it">Difetto del metabolismo e del trasporto della tiamina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -224273,7 +224266,7 @@
     <Disorder id="23717">
       <OrphaCode>457260</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=457260</ExpertLink>
-      <Name lang="it">Sindrome disabilità intellettiva - ipotonia - anomalie del comportamento legata all'X</Name>
+      <Name lang="it">Sindrome da disabilità intellettiva-ipotonia-disturbi del movimento, legata all'X</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -224420,7 +224413,7 @@
     <Disorder id="23727">
       <OrphaCode>457406</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=457406</ExpertLink>
-      <Name lang="it">Sindrome letale da disfunzione mitocondriale multipla tipo 4</Name>
+      <Name lang="it">Sindrome da disfunzioni mitocondriali multiple tipo 4</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -234227,7 +234220,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398043</ExpertLink>
       <Name lang="it">Tumore maligno del pene</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -236630,7 +236623,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398934</ExpertLink>
       <Name lang="it">Tumore epiteliale maligno dell'ovaio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -237131,7 +237124,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398940</ExpertLink>
       <Name lang="it">Tumore non epiteliale maligno dell'ovaio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -253036,7 +253029,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=431353</ExpertLink>
       <Name lang="it">Malattia veno-occlusiva polmonare e/o emangiomatosi capillare polmonare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -261440,7 +261433,7 @@
     <Disorder id="8560">
       <OrphaCode>517</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=517</ExpertLink>
-      <Name lang="it">Leucemia acuta mielomonocitica</Name>
+      <Name lang="it">Leucemia mielomonocitica acuta</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -262196,7 +262189,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=450</ExpertLink>
       <Name lang="it">Eterotassia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -262279,7 +262272,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=224</ExpertLink>
       <Name lang="it">Diabete mellito neonatale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -264434,7 +264427,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=877</ExpertLink>
       <Name lang="it">Tumore endocrino</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -264729,7 +264722,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=467</ExpertLink>
       <Name lang="it">Deficit combinato non acquisito dell'ormone ipofisario</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -273445,7 +273438,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=71198</ExpertLink>
       <Name lang="it">Ipertensione polmonare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -276294,7 +276287,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=69028</ExpertLink>
       <Name lang="it">Sindrome con brachidattilia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -279339,7 +279332,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=63454</ExpertLink>
       <Name lang="it">Distrofia dell'epitelio pigmentato della retina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -279420,7 +279413,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=63443</ExpertLink>
       <Name lang="it">Tumore epiteliale raro dello stomaco</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -280669,7 +280662,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=56970</ExpertLink>
       <Name lang="it">Malattia umana da prioni</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -281863,7 +281856,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79213</ExpertLink>
       <Name lang="it">Mucopolisaccaridosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -282914,7 +282907,7 @@
     <Disorder id="11181">
       <OrphaCode>79155</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79155</ExpertLink>
-      <Name lang="it">Encefalopatia da idrossichinureninuria</Name>
+      <Name lang="it">Idrossichinureninuria</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -283385,7 +283378,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79168</ExpertLink>
       <Name lang="it">Malattia della sintesi degli acidi biliari</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -283417,7 +283410,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79158</ExpertLink>
       <Name lang="it">Aciduria organica cerebrale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -287167,7 +287160,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=77240</ExpertLink>
       <Name lang="it">Linfedema primitivo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -290065,7 +290058,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35705</ExpertLink>
       <Name lang="it">Malattia neurometabolica da deficit di serina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -290408,7 +290401,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35698</ExpertLink>
       <Name lang="it">Sindrome da deplezione del DNA mitocondriale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -290847,7 +290840,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=36388</ExpertLink>
       <Name lang="it">Sindrome neurologica paraneoplastica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -291305,7 +291298,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35808</ExpertLink>
       <Name lang="it">Tumore maligno dello strome e dei cordoni sessuali dell'ovaio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -291831,7 +291824,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35807</ExpertLink>
       <Name lang="it">Tumore maligno dell'ovaio a cellule germinali</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -294787,7 +294780,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=34533</ExpertLink>
       <Name lang="it">Distrofia corneale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -296298,7 +296291,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=52759</ExpertLink>
       <Name lang="it">Vasculite</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -302078,7 +302071,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=68367</ExpertLink>
       <Name lang="it">Difetti congeniti rari del metabolismo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -302643,7 +302636,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90061</ExpertLink>
       <Name lang="it">Uveite posteriore non infettiva</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -308125,7 +308118,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=88991</ExpertLink>
       <Name lang="it">Malformazione cardiaca rara</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -313431,7 +313424,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90970</ExpertLink>
       <Name lang="it">Lipodistrofie primitive</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -314088,7 +314081,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79373</ExpertLink>
       <Name lang="it">Sindrome da displasia ectodermica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -320337,7 +320330,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79361</ExpertLink>
       <Name lang="it">Epidermolisi bollosa ereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -324947,7 +324940,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=86846</ExpertLink>
       <Name lang="it">Leucemia mieloide acuta e sindrome mielodisplastica, terapia-dipendenti</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330286,7 +330279,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98085</ExpertLink>
       <Name lang="it">Disturbo dello sviluppo sessuale 46,XY</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330437,7 +330430,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98287</ExpertLink>
       <Name lang="it">Tumore delle cellule dendritiche e istiocitiche</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330577,7 +330570,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98292</ExpertLink>
       <Name lang="it">Mastocitosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330679,7 +330672,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98277</ExpertLink>
       <Name lang="it">Leucemia mieloide acuta con anomalia genetica ricorrente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330743,7 +330736,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97245</ExpertLink>
       <Name lang="it">Miopatia congenita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330775,7 +330768,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97242</ExpertLink>
       <Name lang="it">Distrofia muscolare congenita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -330858,7 +330851,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97253</ExpertLink>
       <Name lang="it">Tumore endocrino pancreatico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -333313,7 +333306,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97593</ExpertLink>
       <Name lang="it">Pseudoipoparatiroidismo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -333528,7 +333521,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=95708</ExpertLink>
       <Name lang="it">Pubertà precoce</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -333777,7 +333770,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=95714</ExpertLink>
       <Name lang="it">Ipotiroidismo congenito primitivo senza anomalia dello sviluppo della tiroide</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -333809,7 +333802,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=95711</ExpertLink>
       <Name lang="it">Ipotiroidismo congenito da anomalia dello sviluppo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -337082,7 +337075,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93420</ExpertLink>
       <Name lang="it">Condroplasia FGFR3-correlata</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -337670,7 +337663,7 @@
     <Disorder id="12316">
       <OrphaCode>93394</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93394</ExpertLink>
-      <Name lang="it">Brachidattilia, tipo A4</Name>
+      <Name lang="it">Brachidattilia tipo A4</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -339155,7 +339148,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93890</ExpertLink>
       <Name lang="it">Anomalie dello sviluppo durante l'embriogenesi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -343086,7 +343079,7 @@
     <Disorder id="13846">
       <OrphaCode>98829</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98829</ExpertLink>
-      <Name lang="it">'Leucemia mieloide acuta con eosinofili midollari anomali, associata a inv(16)(p13q22) o t(16;16)(p13;q22)'</Name>
+      <Name lang="it">Leucemia mieloide acuta con eosinofili midollari anomali, associata a inv(16)(p13q22) o t(16;16)(p13;q22)</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -351680,7 +351673,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98555</ExpertLink>
       <Name lang="it">Anoftalmia - microftalmia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -352680,7 +352673,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98627</ExpertLink>
       <Name lang="it">Distrofia corneale posteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -352712,7 +352705,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98626</ExpertLink>
       <Name lang="it">Distrofia corneale stromale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -352744,7 +352737,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98625</ExpertLink>
       <Name lang="it">Distrofia corneale superficiale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -352897,7 +352890,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98715</ExpertLink>
       <Name lang="it">Uveite</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -354783,7 +354776,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98482</ExpertLink>
       <Name lang="it">Miopatia infiammatoria idiopatica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -354879,7 +354872,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98428</ExpertLink>
       <Name lang="it">Policitemia secondaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -355788,7 +355781,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98497</ExpertLink>
       <Name lang="it">Malattia genetica dei nervi periferici</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -356568,7 +356561,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=101997</ExpertLink>
       <Name lang="it">Immunodeficienza primitiva</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -356847,7 +356840,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=101959</ExpertLink>
       <Name lang="it">Insufficienza surrenalica primitiva cronica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -359262,7 +359255,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100088</ExpertLink>
       <Name lang="it">Carcinoma tiroideo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -359839,7 +359832,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100087</ExpertLink>
       <Name lang="it">Tumore tiroideo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
