commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml
index 49035c0..35cb961 100644
--- a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:56:23" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:56:42" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6064">
+  <DisorderList count="6069">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -77673,6 +77673,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="1090">
+      <OrphaCode>1587</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1587</ExpertLink>
+      <Name lang="it">Monosomia 13q14</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16758">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16759">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="1115">
       <OrphaCode>1621</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1621</ExpertLink>
@@ -82881,6 +82932,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="1160">
+      <OrphaCode>1713</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1713</ExpertLink>
+      <Name lang="it">Sindrome da microduplicazione 17p11.2</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16762">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23697">
+            <Name lang="it">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="it">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>170.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16763">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="1174">
       <OrphaCode>1738</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1738</ExpertLink>
@@ -83751,7 +83851,7 @@
     <Disorder id="1365">
       <OrphaCode>1071</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1071</ExpertLink>
-      <Name lang="it">Anchiloblefaron - anomalie ectodermiche - labiopalatoschisi</Name>
+      <Name lang="it">Sindrome da anchiloblefaron, difetti ectodermici e labio/palatoschisi</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -85448,7 +85548,7 @@
     <Disorder id="1290">
       <OrphaCode>970</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=970</ExpertLink>
-      <Name lang="it">Neuropatia autonomica e sensoriale ereditaria, tipo 2</Name>
+      <Name lang="it">Neuropatia sensoriale e autonoma ereditaria tipo 2</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -109825,7 +109925,7 @@
     <Disorder id="17241">
       <OrphaCode>158775</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=158775</ExpertLink>
-      <Name lang="it">Mastocitosi sistemica "smouldering"</Name>
+      <Name lang="it">Mastocitosi sistemica intermedia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -145587,6 +145687,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="2945">
+      <OrphaCode>3314</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3314</ExpertLink>
+      <Name lang="it">Malattia di Thiemann, forma familiare</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16766">
+          <Source>31248428[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="it">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="it">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>33.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16767">
+          <Source>ORPHANET_31248428[PMID]</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="it">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="2974">
       <OrphaCode>3355</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3355</ExpertLink>
@@ -208398,7 +208547,7 @@
     <Disorder id="20741">
       <OrphaCode>289560</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=289560</ExpertLink>
-      <Name lang="it">Neurodegenerazione associata a proteine della membrana mitocondriale</Name>
+      <Name lang="it">Neurodegenerazione associata alle proteine della membrana mitocondriale</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -209434,7 +209583,7 @@
     <Disorder id="5546">
       <OrphaCode>393</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=393</ExpertLink>
-      <Name lang="it">Sindrome del maschio XX</Name>
+      <Name lang="it">Malattia testicolare dello sviluppo sessuale 46,XX</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -254126,11 +254275,11 @@
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -264376,7 +264525,7 @@
     <Disorder id="8691">
       <OrphaCode>465</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=465</ExpertLink>
-      <Name lang="it">Deficit congenito dell'inibitore dell'attivatore del plasminogeno, tipo 1</Name>
+      <Name lang="it">Deficit congenito dell'inibitore dell'attivatore del plasminogeno tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -313920,7 +314069,7 @@
     <Disorder id="12113">
       <OrphaCode>91136</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=91136</ExpertLink>
-      <Name lang="it">Sindrome di Fanconi associata alle catene leggere delle Ig monoclonali</Name>
+      <Name lang="it">Sindrome di Fanconi associata alla acquisizione di catene leggere Ig monoclonali</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -335824,6 +335973,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="12792">
+      <OrphaCode>96168</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=96168</ExpertLink>
+      <Name lang="it">Monosomia 13q34</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16760">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16761">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12791">
       <OrphaCode>96167</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=96167</ExpertLink>
@@ -343519,7 +343719,7 @@
     <Disorder id="13866">
       <OrphaCode>98849</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98849</ExpertLink>
-      <Name lang="it">Mastocitosi sistemica associata a emopatia clonale non mastocitaria</Name>
+      <Name lang="it">Mastocitosi sistemica associata a neoplasia ematologica</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -354685,6 +354885,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="28850">
+      <OrphaCode>576278</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=576278</ExpertLink>
+      <Name lang="it">Sindrome correlata a SATB2</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16764">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23697">
+            <Name lang="it">Cases/families</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="it">Case</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>171.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16765">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="28815">
       <OrphaCode>573253</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=573253</ExpertLink>
