commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml
old mode 100644
new mode 100755
index c6674f9..7b5e1ba
--- a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:41:56" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:42:35" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6043">
+  <DisorderList count="6039">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -42561,7 +42561,7 @@
     <Disorder id="589">
       <OrphaCode>3173</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3173</ExpertLink>
-      <Name lang="it">Spasmi infantili - pollici larghi</Name>
+      <Name lang="it">Sindrome spasmi infantili - pollici larghi</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -86805,38 +86805,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="16891">
-      <OrphaCode>139420</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=139420</ExpertLink>
-      <Name lang="it">Mielite trasversa acuta secondaria</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="8137">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="it">Sconosciuto</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16892">
       <OrphaCode>139423</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=139423</ExpertLink>
@@ -106622,7 +106590,7 @@
     <Disorder id="2021">
       <OrphaCode>2110</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2110</ExpertLink>
-      <Name lang="it">Alluce varo - polisindattilia preassiale</Name>
+      <Name lang="it">Sindrome da alluce varo e polisindattilia preassiale</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -107353,7 +107321,7 @@
     <Disorder id="17179">
       <OrphaCode>158025</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=158025</ExpertLink>
-      <Name lang="it">Istiocitosi progressiva ereditaria delle mucose</Name>
+      <Name lang="it">Istiocitosi mucinosa ereditaria progressiva</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -110153,11 +110121,11 @@
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=158676</ExpertLink>
       <Name lang="it">Epidermolisi bollosa distrofica esclusivamente ungueale</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8266">
@@ -110202,11 +110170,11 @@
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=158673</ExpertLink>
       <Name lang="it">Epidermolisi bollosa distrofica acrale</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8263">
@@ -114663,7 +114631,7 @@
     <Disorder id="2300">
       <OrphaCode>2487</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2487</ExpertLink>
-      <Name lang="it">Anomalie degli arti inferiori - ipospadia</Name>
+      <Name lang="it">Sindrome da malformazioni degli arti inferiori e ipospadia</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -120282,7 +120250,7 @@
     <Disorder id="19595">
       <OrphaCode>250972</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=250972</ExpertLink>
-      <Name lang="it">Polimicrogiria con ipoplasia dei nervi ottici</Name>
+      <Name lang="it">Polimicrogiria con ipoplasia del nervo ottico</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -121095,11 +121063,11 @@
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=251393</ExpertLink>
       <Name lang="it">Epidermolisi bollosa giunzionale localizzata, tipo non-Herlitz</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="9158">
@@ -136115,7 +136083,7 @@
     <Disorder id="2778">
       <OrphaCode>3104</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3104</ExpertLink>
-      <Name lang="it">Sequenza di Pierre Robin - oligodattilia</Name>
+      <Name lang="it">Sindrome da sequenza di Pierre Robin e oligodattilia</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -138553,7 +138521,7 @@
     <Disorder id="2583">
       <OrphaCode>2848</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2848</ExpertLink>
-      <Name lang="it">Sindrome camptodattilia-artropatia-ginocchio varo-pericardite</Name>
+      <Name lang="it">Sindrome camptodattilia-artropatia-coxa-vara-pericardite</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -160081,7 +160049,7 @@
     <Disorder id="3087">
       <OrphaCode>3074</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3074</ExpertLink>
-      <Name lang="it">Ritardo mentale - bassa statura - ipertelorismo</Name>
+      <Name lang="it">Sindrome da disabilità intellettiva, bassa statura e ipertelorismo</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -161502,7 +161470,7 @@
     <Disorder id="18656">
       <OrphaCode>209370</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=209370</ExpertLink>
-      <Name lang="it">Encefalopatia grave a esordio neonatale con microcefalia</Name>
+      <Name lang="it">Encefalopatia grave ad esordio neonatale con microcefalia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -189816,7 +189784,7 @@
     <Disorder id="19345">
       <OrphaCode>240760</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=240760</ExpertLink>
-      <Name lang="it">Disturbo sindromico da rottura cromosomica simil-Nijmegen</Name>
+      <Name lang="it">Malattia simile alla sindrome da rotture cromosomiche di Nijmegen</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -200385,7 +200353,7 @@
     <Disorder id="22452">
       <OrphaCode>369950</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=369950</ExpertLink>
-      <Name lang="it">Sindrome deficit cognitivo-convulsioni-macrocefalia-obesità</Name>
+      <Name lang="it">Sindrome da disabilità intellettiva crisi epilettiche, macrocefalia e obesità</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -200743,7 +200711,7 @@
     <Disorder id="22433">
       <OrphaCode>369837</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=369837</ExpertLink>
-      <Name lang="it">Sindrome deficit cognitivo-convulsioni-ipotonia-anomalie oftalmologiche e scheletriche</Name>
+      <Name lang="it">Sindrome da disabilità intellettiva, crisi epilettiche, ipotonia e anomalie oculari e scheletriche</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -215449,7 +215417,7 @@
     <Disorder id="21449">
       <OrphaCode>313772</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=313772</ExpertLink>
-      <Name lang="it">Sindrome atassia spastica-neuropatia, a esordio precoce</Name>
+      <Name lang="it">Sindrome da atassia spastica, epilessia mioclonica e neuropatia ad esordio precoce</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -215824,7 +215792,7 @@
     <Disorder id="21492">
       <OrphaCode>314485</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=314485</ExpertLink>
-      <Name lang="it">Neuropatia motoria ereditaria distale a esordio nel giovane adulto</Name>
+      <Name lang="it">Neuropatia motoria ereditaria distale ad esordio nella prima età adulta</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -221654,7 +221622,7 @@
     <Disorder id="24056">
       <OrphaCode>465508</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=465508</ExpertLink>
-      <Name lang="it">Emocromatosi tipo 1, forma sintomatica</Name>
+      <Name lang="it">Forma sintomatica dell'emocromatosi tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -221686,7 +221654,7 @@
     <Disorder id="24010">
       <OrphaCode>464282</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=464282</ExpertLink>
-      <Name lang="it">Sindrome paraplegia spastica - ritardo psicomotorio - crisi epilettiche</Name>
+      <Name lang="it">Sindrome da paraplegia spastica, grave ritardo dello sviluppo ed epilessia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -227780,7 +227748,7 @@
     <Disorder id="24171">
       <OrphaCode>466801</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=466801</ExpertLink>
-      <Name lang="it">Distrofia dei cingoli autosomica recessiva tipo 2W</Name>
+      <Name lang="it">Distrofia muscolare dei cingoli automica recessiva tipo 2W</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -228449,7 +228417,7 @@
     <Disorder id="24203">
       <OrphaCode>467176</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=467176</ExpertLink>
-      <Name lang="it">Sindrome ipotonia grave-ritardo dello sviluppo psicomotorio - paralisi del nervo abducente - difetto del setto cardiaco autosomica recessiva</Name>
+      <Name lang="it">Sindrome da ipotonia grave, ritardo dello sviluppo psicomotorio, strabismo e difetto del setto cardiaco</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -235636,7 +235604,7 @@
     <Disorder id="22711">
       <OrphaCode>397951</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=397951</ExpertLink>
-      <Name lang="it">Sindrome microcefalia-corpo calloso sottile-deficit cognitivo</Name>
+      <Name lang="it">Sindrome da microcefalia, corpo calloso sottile e disabilità intellettiva</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -251961,7 +251929,7 @@
     <Disorder id="23231">
       <OrphaCode>431140</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=431140</ExpertLink>
-      <Name lang="it">Sindrome legata all'X microftalmia colobomatosa-microcefalia-disabilità cognitiva-bassa statura</Name>
+      <Name lang="it">Sindrome legata all'X da microftalmia colobomatosa, microcefalia, disabilità intellettiva e bassa statura</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -254378,7 +254346,7 @@
     <Disorder id="23235">
       <OrphaCode>431166</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=431166</ExpertLink>
-      <Name lang="it">Immunodeficienza primitiva da deficit di STAT2</Name>
+      <Name lang="it">Immunodeficienza primitiva con infezione virale post-vaccinazione anti morbillo-parotite-rosolia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -270506,7 +270474,7 @@
     <Disorder id="25213">
       <OrphaCode>480528</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=480528</ExpertLink>
-      <Name lang="it">Sindrome letale idranencefalia-ernia diaframmatica</Name>
+      <Name lang="it">Sindrome letale da idranencefalia ed ernia diaframmatica</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -270864,7 +270832,7 @@
     <Disorder id="25164">
       <OrphaCode>477993</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=477993</ExpertLink>
-      <Name lang="it">Sindrome anomalie del palato - denti iperdistanziati - dismorfismi facciali - ritardo dello sviluppo</Name>
+      <Name lang="it">Sindrome da anomalie del palato, denti iperdistanziati, dismorfismi facciali e ritardo dello sviluppo</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -273269,7 +273237,7 @@
     <Disorder id="25086">
       <OrphaCode>476096</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=476096</ExpertLink>
-      <Name lang="it">Sindrome eritrocheratodermia - cardiomiopatia</Name>
+      <Name lang="it">Eritrocheratodermia con cardiomiopatia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -305106,119 +305074,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="11906">
-      <OrphaCode>89841</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=89841</ExpertLink>
-      <Name lang="it">Epidermolisi bollosa distrofica recessiva centripeta</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="6950">
-          <Source>Dr Giovanna ZAMBRUNO[EXPERT]_4026337[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="it">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="it">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>10.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6951">
-          <Source>ORPHANET_4026337[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="it">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="11905">
-      <OrphaCode>89840</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=89840</ExpertLink>
-      <Name lang="it">Epidermolisi bollosa giunzionale, tipo non-Herlitz</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="10622">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="it">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
-    <Disorder id="11904">
-      <OrphaCode>89839</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=89839</ExpertLink>
-      <Name lang="it">Epidermolisi bollosa semplice superficiale</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="6949">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="it">Sconosciuto</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="11919">
       <OrphaCode>90001</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90001</ExpertLink>
@@ -315554,11 +315409,11 @@
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79402</ExpertLink>
       <Name lang="it">Epidermolisi bollosa giunzionale generalizzata, tipo non-Herlitz</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="1">
         <Prevalence id="6509">
@@ -315960,11 +315815,11 @@
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79410</ExpertLink>
       <Name lang="it">Epidermolisi bollosa distrofica pretibiale</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="6526">
@@ -333623,7 +333478,7 @@
     <Disorder id="12808">
       <OrphaCode>96184</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=96184</ExpertLink>
-      <Name lang="it">Disomia uniparentale materna del cromosoma 14</Name>
+      <Name lang="it">SIndrome di Temple da disomia uniparentale materna del cromosoma 14</Name>
       <DisorderType id="21443">
         <Name lang="it">Sottotipo eziologico</Name>
       </DisorderType>
