commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml
index 1ffeaae..49035c0 100644
--- a/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/it_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:54:36" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:56:23" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6052">
+  <DisorderList count="6064">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -57186,7 +57186,7 @@
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
       </DisorderGroup>
-      <PrevalenceList count="7">
+      <PrevalenceList count="8">
         <Prevalence id="1844">
           <Source>17181917[PMID]</Source>
           <PrevalenceType id="23676">
@@ -57301,6 +57301,25 @@
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16724">
+          <Source>31150159[PMID]</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23753">
+            <Name lang="it">6-9 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>74.6</ValMoy>
+          <PrevalenceGeographic id="23851">
+            <Name lang="it">Popolazione specifica</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="16513">
           <Source>24210707[PMID]</Source>
           <PrevalenceType id="23676">
@@ -59660,9 +59679,9 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <PrevalenceList count="16">
+      <PrevalenceList count="19">
         <Prevalence id="2178">
-          <Source>EUROCAT European surveillance of congenital anomalies 2012[REG]</Source>
+          <Source>31270117[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -59672,7 +59691,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>4.2</ValMoy>
+          <ValMoy>5.5625</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="it">Europa</Name>
           </PrevalenceGeographic>
@@ -59719,7 +59738,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2181">
-          <Source>EUROCAT European surveillance of congenital anomalies 2012[REG]</Source>
+          <Source>31270117[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -59729,7 +59748,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.8</ValMoy>
+          <ValMoy>9.0</ValMoy>
           <PrevalenceGeographic id="24369">
             <Name lang="it">France</Name>
           </PrevalenceGeographic>
@@ -59738,7 +59757,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2182">
-          <Source>EUROCAT European surveillance of congenital anomalies 2010[REG]</Source>
+          <Source>31270117[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -59748,7 +59767,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>9.8</ValMoy>
+          <ValMoy>3.0</ValMoy>
           <PrevalenceGeographic id="24418">
             <Name lang="it">Germany</Name>
           </PrevalenceGeographic>
@@ -59776,7 +59795,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2184">
-          <Source>EUROCAT European surveillance of congenital anomalies 2012[REG]</Source>
+          <Source>31270117[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -59786,7 +59805,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>4.3</ValMoy>
+          <ValMoy>6.5</ValMoy>
           <PrevalenceGeographic id="24607">
             <Name lang="it">Italy</Name>
           </PrevalenceGeographic>
@@ -59795,7 +59814,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2185">
-          <Source>EUROCAT European surveillance of congenital anomalies 2012[REG]</Source>
+          <Source>31270117[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -59805,7 +59824,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>5.7</ValMoy>
+          <ValMoy>4.0</ValMoy>
           <PrevalenceGeographic id="24936">
             <Name lang="it">Netherlands</Name>
           </PrevalenceGeographic>
@@ -59909,7 +59928,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2191">
-          <Source>EUROCAT European surveillance of congenital anomalies 2012[REG]</Source>
+          <Source>31270117[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
@@ -59919,7 +59938,7 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>6.2</ValMoy>
+          <ValMoy>5.0</ValMoy>
           <PrevalenceGeographic id="25475">
             <Name lang="it">United Kingdom</Name>
           </PrevalenceGeographic>
@@ -59965,6 +59984,63 @@
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16752">
+          <Source>31270117[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>8.0</ValMoy>
+          <PrevalenceGeographic id="24803">
+            <Name lang="it">Malta</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16753">
+          <Source>31270117[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>6.0</ValMoy>
+          <PrevalenceGeographic id="24257">
+            <Name lang="it">Czech Republic</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16754">
+          <Source>31270117[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>3.0</ValMoy>
+          <PrevalenceGeographic id="25251">
+            <Name lang="it">Slovakia</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="816">
@@ -61518,76 +61594,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="813">
-      <OrphaCode>1549</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1549</ExpertLink>
-      <Name lang="it">Criptosporidiosi</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="3">
-        <Prevalence id="2173">
-          <Source>Center for Diseases Control and Prevention 2010[INST]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>2.9</ValMoy>
-          <PrevalenceGeographic id="25482">
-            <Name lang="it">United States</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="2174">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="it">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="it">Sconosciuto</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="11949">
-          <Source>European Centre for Disease prevention and Control 2011[INST]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>1.96</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="815">
       <OrphaCode>549</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=549</ExpertLink>
@@ -185114,6 +185120,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="3625">
+      <OrphaCode>2197</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2197</ExpertLink>
+      <Name lang="it">Ipercalciuria idiopatica</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16731">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16732">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19175">
       <OrphaCode>231237</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231237</ExpertLink>
@@ -209061,7 +209118,7 @@
       <DisorderGroup id="36554">
         <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="2">
         <Prevalence id="5012">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -209081,6 +209138,25 @@
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16735">
+          <Source>32386559[PMID]</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>3.9</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="it">United States</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="20841">
@@ -287644,209 +287720,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="11041">
-      <OrphaCode>73247</OrphaCode>
-      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=73247</ExpertLink>
-      <Name lang="it">Esofagite eosinofila</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="10">
-        <Prevalence id="6127">
-          <Source>22994460[PMID]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>11.0</ValMoy>
-          <PrevalenceGeographic id="24124">
-            <Name lang="it">Canada</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6128">
-          <Source>21750884[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>17.1</ValMoy>
-          <PrevalenceGeographic id="24621">
-            <Name lang="it">Japan</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6129">
-          <Source>23075697[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>44.6</ValMoy>
-          <PrevalenceGeographic id="25293">
-            <Name lang="it">Spain</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6130">
-          <Source>30887555[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>29.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6131">
-          <Source>15696105[PMID]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>9.45</ValMoy>
-          <PrevalenceGeographic id="25482">
-            <Name lang="it">United States</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6132">
-          <Source>19577011[PMID]_24035773[PMID]_ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>55.85</ValMoy>
-          <PrevalenceGeographic id="25482">
-            <Name lang="it">United States</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6133">
-          <Source>22963642[PMID]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>1.31</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="it">Netherlands</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6134">
-          <Source>22019091[PMID]</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
-          </PrevalenceClass>
-          <ValMoy>7.4</ValMoy>
-          <PrevalenceGeographic id="25342">
-            <Name lang="it">Switzerland</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="6135">
-          <Source>22019091[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>42.8</ValMoy>
-          <PrevalenceGeographic id="25342">
-            <Name lang="it">Switzerland</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="11611">
-          <Source>30887555[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="it">Prevalenza puntuale</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="it">1-5 / 10 000</Name>
-          </PrevalenceClass>
-          <ValMoy>34.4</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="it">Mondiale</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="it">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="28655">
       <OrphaCode>569821</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=569821</ExpertLink>
@@ -326847,26 +326720,26 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="6597">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
+          <Source>1483045[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="it">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.1</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
+          <ValMoy>0.6623</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="it">Italy</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10617">
-          <Source>ORPHANET</Source>
+          <Source>28676062[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -326876,12 +326749,12 @@
           <PrevalenceClass id="23739">
             <Name lang="it">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
+          <ValMoy>0.61</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="it">Europa</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -326902,18 +326775,18 @@
           <PrevalenceType id="23676">
             <Name lang="it">Incidenza annuale</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="it">Value and class</Name>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="it">1-9 / 1 000 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
           </PrevalenceClass>
-          <ValMoy>0.32</ValMoy>
+          <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="it">Europa</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10618">
@@ -326924,15 +326797,15 @@
           <PrevalenceQualification id="23704">
             <Name lang="it">Class only</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="it">1-9 / 1 000 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="it">Europa</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -326949,9 +326822,9 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="6596">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23676">
-            <Name lang="it">Incidenza annuale</Name>
+          <Source>10677857[PMID]_1483045[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="it">Prevalenza alla nascita</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
@@ -326959,16 +326832,16 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.23</ValMoy>
+          <ValMoy>2.0</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="it">Europa</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10616">
-          <Source>ORPHANET</Source>
+          <Source>28676062[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
@@ -326978,12 +326851,12 @@
           <PrevalenceClass id="23746">
             <Name lang="it">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="it">Europa</Name>
+          <ValMoy>2.0322</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="it">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -332452,6 +332325,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="12927">
+      <OrphaCode>97560</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97560</ExpertLink>
+      <Name lang="it">Glomerulonefrite membranosa primitiva</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16725">
+          <Source>21068142[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="it">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.8103</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16726">
+          <Source>21068142[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="it">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.9194</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="it">Europa</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12805">
       <OrphaCode>96181</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=96181</ExpertLink>
@@ -333129,6 +333053,76 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="12850">
+      <OrphaCode>96369</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=96369</ExpertLink>
+      <Name lang="it">Schizofrenia a esordio precoce</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="3">
+        <Prevalence id="16755">
+          <Source>22906158[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="it">Europa</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16756">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="it">Europa</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16757">
+          <Source>22906158[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>3.17</ValMoy>
+          <PrevalenceGeographic id="24264">
+            <Name lang="it">Denmark</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12860">
       <OrphaCode>97234</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97234</ExpertLink>
@@ -343795,6 +343789,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29436">
+      <OrphaCode>589595</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589595</ExpertLink>
+      <Name lang="it">'Leucemia acuta a fenotipo misto con t(v;11q23.3)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16748">
+          <Source>WHO Classification of Tumour_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16749">
+          <Source>WHO Classification of Tumour_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="13885">
       <OrphaCode>98868</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98868</ExpertLink>
@@ -343976,6 +344021,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29433">
+      <OrphaCode>589534</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589534</ExpertLink>
+      <Name lang="it">'Leucemia acuta a fenotipo misto con t(9;22)(q34.1;q11.2)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16746">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16747">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="13880">
       <OrphaCode>98863</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98863</ExpertLink>
@@ -344144,6 +344240,57 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29434">
+      <OrphaCode>589542</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589542</ExpertLink>
+      <Name lang="it">Neoplasia mieloide/linfoide associata a riarrangiamenti in JAK2</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16750">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16751">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="29429">
       <OrphaCode>589442</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589442</ExpertLink>
@@ -344414,7 +344561,7 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <PrevalenceList count="100">
+      <PrevalenceList count="101">
         <Prevalence id="7689">
           <Source>21649801[PMID]_European Medicines Agency 2018[INST]</Source>
           <PrevalenceType id="23669">
@@ -346315,6 +346462,25 @@
             <Name lang="it">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16723">
+          <Source>https://www.francecoag.org/SiteWebPublic/public/stats/stats_page.jsp?stat3=on</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="it">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="it">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.44</ValMoy>
+          <PrevalenceGeographic id="24369">
+            <Name lang="it">France</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="13890">
@@ -346399,17 +346565,17 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="7581">
-          <Source>19845775[PMID]</Source>
+          <Source>https://www.francecoag.org/SiteWebPublic/public/stats/stats_page.jsp?stat2=on</Source>
           <PrevalenceType id="23669">
             <Name lang="it">Prevalenza puntuale</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="it">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="it">1-9 / 100 000</Name>
+          <PrevalenceClass id="23732">
+            <Name lang="it">1-5 / 10 000</Name>
           </PrevalenceClass>
-          <ValMoy>6.1</ValMoy>
+          <ValMoy>10.5</ValMoy>
           <PrevalenceGeographic id="24369">
             <Name lang="it">France</Name>
           </PrevalenceGeographic>
@@ -351063,6 +351229,312 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29205">
+      <OrphaCode>585877</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585877</ExpertLink>
+      <Name lang="it">Leucemia linfoblastica a cellule B/linfoma con anomalie genetiche ricorrenti</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16727">
+          <Source>ORPHANET_WHO Classification of Tumours</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16728">
+          <Source>ORPHANET_WHO Classification of Tumours</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29211">
+      <OrphaCode>585909</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585909</ExpertLink>
+      <Name lang="it">'Leucemia linfoblastica a cellule B/linfoma con t(9;22)(q34.1;q11.2)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16729">
+          <Source>ORPHANET_WHO Classification of Tumours</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16730">
+          <Source>ORPHANET_WHO Classification of Tumours</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29214">
+      <OrphaCode>585936</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585936</ExpertLink>
+      <Name lang="it">Leucemia linfoblastica a cellule B/linfoma con iperdiploidia</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16738">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16739">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29215">
+      <OrphaCode>585942</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585942</ExpertLink>
+      <Name lang="it">Leucemia linfoblastica a cellule B/linfoma con ipodiploidia</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16740">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16741">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29212">
+      <OrphaCode>585918</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585918</ExpertLink>
+      <Name lang="it">'Leucemia linfoblastica a cellule B/linfoma con t(v;11q23.3)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16733">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16734">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29213">
+      <OrphaCode>585929</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585929</ExpertLink>
+      <Name lang="it">'Leucemia linfoblastica a cellule B/linfoma con t(12;21)(p13.2;q22.1)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16736">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16737">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="14044">
       <OrphaCode>99027</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99027</ExpertLink>
@@ -351112,6 +351584,108 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29217">
+      <OrphaCode>585956</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585956</ExpertLink>
+      <Name lang="it">'Leucemia linfoblastica a cellule B/linfoma con t(1;19)(q23;p13.3)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16744">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16745">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29216">
+      <OrphaCode>585948</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=585948</ExpertLink>
+      <Name lang="it">'Leucemia linfoblastica a cellule B/linfoma con t(5;14)(q31.1;q32.3)'</Name>
+      <DisorderType id="21443">
+        <Name lang="it">Sottotipo eziologico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16742">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="it">Incidenza annuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23837">
+            <Name lang="it">Asia occidentale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16743">
+          <Source>WHO Classification of Tumours_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="it">Prevalenza puntuale</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="it">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="it">Sconosciuto</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="it">Mondiale</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="it">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="29226">
       <OrphaCode>586130</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=586130</ExpertLink>
