commit a9304bcd06b429331d3fd652192800e94c21e73c
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml
index 5fa1bff..ee65e5d 100644
--- a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:38:12" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:41:08" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6042">
+  <DisorderList count="6043">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -2546,7 +2546,7 @@
     <Disorder id="24">
       <OrphaCode>583</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=583</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 6</Name>
+      <Name lang="nl">Mucopolysacharidose type 6</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -4167,7 +4167,7 @@
     <Disorder id="40">
       <OrphaCode>584</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=584</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 7</Name>
+      <Name lang="nl">Mucopolysacharidose type 7</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -6697,7 +6697,7 @@
     <Disorder id="62">
       <OrphaCode>269</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=269</ExpertLink>
-      <Name lang="nl">Facio-scapulo-humerale dystrofie</Name>
+      <Name lang="nl">Facioscapulohumerale dystrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -13996,7 +13996,7 @@
     <Disorder id="131">
       <OrphaCode>580</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=580</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 2</Name>
+      <Name lang="nl">Mucopolysacharidose type 2</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -14503,7 +14503,7 @@
     <Disorder id="132">
       <OrphaCode>579</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=579</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 1</Name>
+      <Name lang="nl">Mucopolysacharidose type 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -25130,7 +25130,7 @@
     <Disorder id="343">
       <OrphaCode>1727</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1727</ExpertLink>
-      <Name lang="nl">22q11.2 microduplicatiesyndroom</Name>
+      <Name lang="nl">22q11.2-duplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -26675,7 +26675,7 @@
     <Disorder id="324">
       <OrphaCode>753</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=753</ExpertLink>
-      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis als gevolg van 5-alfa-reductase 2-deficiëntie</Name>
+      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis door 5-alfa-reductase 2-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -29774,7 +29774,7 @@
     <Disorder id="442">
       <OrphaCode>1442</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1442</ExpertLink>
-      <Name lang="nl">Ringchromosoom 18 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 18-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -45467,7 +45467,7 @@
     <Disorder id="653">
       <OrphaCode>581</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=581</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 3</Name>
+      <Name lang="nl">Mucopolysacharidose type 3</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -63352,7 +63352,7 @@
     <Disorder id="784">
       <OrphaCode>3002</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3002</ExpertLink>
-      <Name lang="nl">Immuungemedieerde trombocytopenische purpura</Name>
+      <Name lang="nl">Immuuntrombocytopenie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -66689,7 +66689,7 @@
     <Disorder id="866">
       <OrphaCode>770</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=770</ExpertLink>
-      <Name lang="nl">Hondsdolheid</Name>
+      <Name lang="nl">Rabiës</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -67070,7 +67070,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1329</ExpertLink>
-      <Name lang="nl">Compleet atrioventriculair kanaal</Name>
+      <Name lang="nl">Compleet atrioventriculair septumdefect</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -67121,7 +67121,7 @@
     <Disorder id="872">
       <OrphaCode>582</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=582</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 4</Name>
+      <Name lang="nl">Mucopolysacharidose type 4</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -69125,7 +69125,7 @@
     <Disorder id="849">
       <OrphaCode>3198</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3198</ExpertLink>
-      <Name lang="nl">Stiff person-syndroom en verwante aandoeningen</Name>
+      <Name lang="nl">Stiff person-spectrumstoornis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -74069,7 +74069,7 @@
     <Disorder id="846">
       <OrphaCode>683</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=683</ExpertLink>
-      <Name lang="nl">Progressieve supranucleaire verlamming</Name>
+      <Name lang="nl">Progressieve supranucleaire paralyse</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -74880,7 +74880,7 @@
     <Disorder id="946">
       <OrphaCode>3027</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3027</ExpertLink>
-      <Name lang="nl">Caudale regressiesyndroom</Name>
+      <Name lang="nl">Caudale regressie-sequentie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -76311,7 +76311,7 @@
     <Disorder id="896">
       <OrphaCode>1446</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1446</ExpertLink>
-      <Name lang="nl">Ringchromosoom 22 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 22-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81545,7 +81545,7 @@
     <Disorder id="1081">
       <OrphaCode>1450</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1450</ExpertLink>
-      <Name lang="nl">Ringchromosoom 8 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 8-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81594,7 +81594,7 @@
     <Disorder id="1080">
       <OrphaCode>1448</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1448</ExpertLink>
-      <Name lang="nl">Ringchromosoom 6 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 6-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81675,7 +81675,7 @@
     <Disorder id="1075">
       <OrphaCode>1437</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1437</ExpertLink>
-      <Name lang="nl">Ringchromosoom 1 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 1-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81756,7 +81756,7 @@
     <Disorder id="1079">
       <OrphaCode>1447</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1447</ExpertLink>
-      <Name lang="nl">Ringchromosoom 4 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 4-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81805,7 +81805,7 @@
     <Disorder id="1078">
       <OrphaCode>1444</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1444</ExpertLink>
-      <Name lang="nl">Ringchromosoom 20 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 20-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81854,7 +81854,7 @@
     <Disorder id="1077">
       <OrphaCode>1439</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1439</ExpertLink>
-      <Name lang="nl">Ringchromosoom 12 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 12-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -81903,7 +81903,7 @@
     <Disorder id="1076">
       <OrphaCode>1438</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1438</ExpertLink>
-      <Name lang="nl">Ringchromosoom 10 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 10-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -82065,7 +82065,7 @@
     <Disorder id="1226">
       <OrphaCode>3176</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3176</ExpertLink>
-      <Name lang="nl">Spina bifida - hypospadieën-syndroom</Name>
+      <Name lang="nl">Spina bifida - hypospadie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -82457,7 +82457,7 @@
     <Disorder id="1241">
       <OrphaCode>1445</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1445</ExpertLink>
-      <Name lang="nl">Ringchromosoom 21 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 21-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -85790,7 +85790,7 @@
     <Disorder id="1310">
       <OrphaCode>994</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=994</ExpertLink>
-      <Name lang="nl">Foetale akinesie-deformatiesequentie</Name>
+      <Name lang="nl">Foetale akinesie - deformatie-sequentie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -85892,7 +85892,7 @@
     <Disorder id="1299">
       <OrphaCode>981</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=981</ExpertLink>
-      <Name lang="nl">Binnenste halsslagaderagenesie</Name>
+      <Name lang="nl">Afwezigheid van arteria carotis interna</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -93165,7 +93165,7 @@
     <Disorder id="1455">
       <OrphaCode>1208</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1208</ExpertLink>
-      <Name lang="nl">Pulmonale atresie - intact ventriculair septum-syndroom</Name>
+      <Name lang="nl">Pulmonale atresie - intact ventrikelseptum-syndroom</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -93216,7 +93216,7 @@
     <Disorder id="1642">
       <OrphaCode>1449</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1449</ExpertLink>
-      <Name lang="nl">Ringchromosoom 7 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 7-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -93314,7 +93314,7 @@
     <Disorder id="1640">
       <OrphaCode>1440</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1440</ExpertLink>
-      <Name lang="nl">Ringchromosoom 14 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 14-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -93363,7 +93363,7 @@
     <Disorder id="1641">
       <OrphaCode>1443</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1443</ExpertLink>
-      <Name lang="nl">Ringchromosoom 19 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 19-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -95796,7 +95796,7 @@
     <Disorder id="17146">
       <OrphaCode>157798</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=157798</ExpertLink>
-      <Name lang="nl">Hyperplastische polyposis-syndroom</Name>
+      <Name lang="nl">Gekartelde polyposis-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -96735,7 +96735,7 @@
     <Disorder id="1536">
       <OrphaCode>1305</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1305</ExpertLink>
-      <Name lang="nl">Feingoldsyndroom</Name>
+      <Name lang="nl">Syndroom van Feingold</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -104139,7 +104139,7 @@
     <Disorder id="1868">
       <OrphaCode>1906</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1906</ExpertLink>
-      <Name lang="nl">Foetaal valproaatsyndroom</Name>
+      <Name lang="nl">Foetaal valproaat-spectrumstoornis</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -111000,7 +111000,7 @@
     <Disorder id="2202">
       <OrphaCode>2353</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2353</ExpertLink>
-      <Name lang="nl">Schilbach-Rottsyndroom</Name>
+      <Name lang="nl">Syndroom van Schilbach-Rott</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -114300,6 +114300,38 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="19474">
+      <OrphaCode>244275</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=244275</ExpertLink>
+      <Name lang="nl">De novo trombotische microangiopathie na niertransplantatie</Name>
+      <DisorderType id="21429">
+        <Name lang="nl">Particulaire klinische situatie binnen een ziekte of syndroom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Disorder</Name>
+      </DisorderGroup>
+      <PrevalenceList count="1">
+        <Prevalence id="14647">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="nl">Puntprevalentie</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="nl">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="nl">Unknown</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="nl">Wereldwijd</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25965">
+            <Name lang="nl">Not yet validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19468">
       <OrphaCode>243343</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=243343</ExpertLink>
@@ -114631,7 +114663,7 @@
     <Disorder id="2300">
       <OrphaCode>2487</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2487</ExpertLink>
-      <Name lang="nl">Malformatie van de onderste ledematen - hypospadieën-syndroom</Name>
+      <Name lang="nl">Malformatie van onderste ledematen - hypospadie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -118273,7 +118305,7 @@
     <Disorder id="2089">
       <OrphaCode>2211</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2211</ExpertLink>
-      <Name lang="nl">Hypertelorisme - hypospadieën - polysyndactylie-syndroom</Name>
+      <Name lang="nl">Hypertelorisme - hypospadie - polysyndactylie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -118712,7 +118744,7 @@
     <Disorder id="2129">
       <OrphaCode>2261</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2261</ExpertLink>
-      <Name lang="nl">Hypospadieën - intellectuele achterstand, Goldblatt-type-syndroom</Name>
+      <Name lang="nl">Hypospadie - intellectuele achterstand, Goldblatt-type-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -119264,7 +119296,7 @@
     <Disorder id="19608">
       <OrphaCode>251043</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251043</ExpertLink>
-      <Name lang="nl">Ringchromosoom 5 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 5-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -120201,7 +120233,7 @@
     <Disorder id="2122">
       <OrphaCode>2252</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2252</ExpertLink>
-      <Name lang="nl">Radiale hypoplasie - trifalangeale duimen - hypospadieën - maxillair diasteem</Name>
+      <Name lang="nl">Radiale hypoplasie - trifalangeale duimen - hypospadie - maxillair diasteem-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -131608,7 +131640,7 @@
     <Disorder id="19886">
       <OrphaCode>261529</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261529</ExpertLink>
-      <Name lang="nl">Ringchromosoom Y syndroom</Name>
+      <Name lang="nl">Ringchromosoom Y-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -135357,7 +135389,7 @@
     <Disorder id="2814">
       <OrphaCode>3145</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3145</ExpertLink>
-      <Name lang="nl">Nefrogene diabetes insipidus - intracraniële calcificatie-syndroom</Name>
+      <Name lang="nl">Nefrogene diabetes insipidus - intracraniële calcificatie - faciale dysmorfie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -154633,7 +154665,7 @@
     <Disorder id="20381">
       <OrphaCode>276435</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=276435</ExpertLink>
-      <Name lang="nl">Lager motorneuronsyndroom met aanvang in de late volwassenheid</Name>
+      <Name lang="nl">Lager motorneuron-syndroom met aanvang in de late volwassenheid</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -155834,7 +155866,7 @@
     <Disorder id="3296">
       <OrphaCode>20</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=20</ExpertLink>
-      <Name lang="nl">3-hydroxy-3-methylglutaarzuuracidurie</Name>
+      <Name lang="nl">3-hydroxy-3-methylglutaaracidurie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -164372,7 +164404,7 @@
     <Disorder id="3563">
       <OrphaCode>359</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=359</ExpertLink>
-      <Name lang="nl">Erfelijk glaucoom</Name>
+      <Name lang="nl">Glaucoom met genetische oorsprong en pediatrische aanvang</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -170877,7 +170909,7 @@
     <Disorder id="3505">
       <OrphaCode>1441</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1441</ExpertLink>
-      <Name lang="nl">Ringchromosoom 17 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 17-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -171836,7 +171868,7 @@
     <Disorder id="18825">
       <OrphaCode>217093</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217093</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 2, geattenueerde vorm</Name>
+      <Name lang="nl">Mucopolysacharidose type 2, geattenueerde vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -171938,7 +171970,7 @@
     <Disorder id="18824">
       <OrphaCode>217085</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217085</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 2, ernstige vorm</Name>
+      <Name lang="nl">Mucopolysacharidose type 2, ernstige vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -179364,7 +179396,7 @@
     <Disorder id="3759">
       <OrphaCode>319</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319</ExpertLink>
-      <Name lang="nl">Skeletaal Ewingsarcoom</Name>
+      <Name lang="nl">Skeletaal Ewing-sarcoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -192002,7 +192034,7 @@
     <Disorder id="21917">
       <OrphaCode>329252</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329252</ExpertLink>
-      <Name lang="nl">Spondylocostale dysostose - hypospadieën - intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Spondylocostale dysostose - hypospadie - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -192487,7 +192519,7 @@
     <Disorder id="21819">
       <OrphaCode>324977</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=324977</ExpertLink>
-      <Name lang="nl">Proteasoomdisfunctiesyndroom</Name>
+      <Name lang="nl">Proteasoom-geassocieerd auto-inflammatoir syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -198757,7 +198789,7 @@
     <Disorder id="22490">
       <OrphaCode>370334</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=370334</ExpertLink>
-      <Name lang="nl">Extraskeletaal Ewingsarcoom</Name>
+      <Name lang="nl">Extraskeletaal Ewing-sarcoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -211188,7 +211220,7 @@
     <Disorder id="20609">
       <OrphaCode>284417</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284417</ExpertLink>
-      <Name lang="nl">Fosfoserineaminotransferasedeficiëntie</Name>
+      <Name lang="nl">Fosfoserine-aminotransferasedeficiëntie, infantiele/juveniele vorm</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -211736,7 +211768,7 @@
     <Disorder id="20728">
       <OrphaCode>289494</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=289494</ExpertLink>
-      <Name lang="nl">Hypomyeliniserende leukodystrofie met of zonder oligondontie en/of hypogonadisme</Name>
+      <Name lang="nl">4H-leukodystrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -218045,7 +218077,7 @@
     <Disorder id="21369">
       <OrphaCode>309297</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=309297</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose type 4A</Name>
+      <Name lang="nl">Mucopolysacharidose type 4A</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -258748,7 +258780,7 @@
     <Disorder id="26330">
       <OrphaCode>505248</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=505248</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose-achtig syndroom met congenitale hartdefecten en hematopoëtische stoornissen</Name>
+      <Name lang="nl">Mucopolysacharidose-achtig syndroom met congenitale hartdefecten en hematopoëtische stoornissen</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -266906,7 +266938,7 @@
     <Disorder id="8597">
       <OrphaCode>1330</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1330</ExpertLink>
-      <Name lang="nl">Partieel atrioventriculair kanaal</Name>
+      <Name lang="nl">Partieel atrioventriculair septumdefect</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -270988,7 +271020,7 @@
     <Disorder id="25168">
       <OrphaCode>478049</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=478049</ExpertLink>
-      <Name lang="nl">Letaal non-compactie van het linkerventrikel - insulten - hypotonie - cataract - ontwikkelingsachterstand-syndroom</Name>
+      <Name lang="nl">Letaal non-compactie van linkerventrikel - insulten - hypotonie - cataract - ontwikkelingsachterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -282278,7 +282310,7 @@
     <Disorder id="11239">
       <OrphaCode>79213</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79213</ExpertLink>
-      <Name lang="nl">Mucopolysaccharidose</Name>
+      <Name lang="nl">Mucopolysacharidose</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -291484,7 +291516,7 @@
     <Disorder id="10373">
       <OrphaCode>35122</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35122</ExpertLink>
-      <Name lang="nl">Congenitale sucrase-isomaltasedeficiëntie</Name>
+      <Name lang="nl">Congenitale deficiëntie van sucrase-isomaltase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -298125,7 +298157,7 @@
     <Disorder id="10738">
       <OrphaCode>53696</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=53696</ExpertLink>
-      <Name lang="nl">Letale artrogrypose - anterieure hoorncelziekte-syndroom</Name>
+      <Name lang="nl">Artrogrypose - voorhoornaandoening-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -310678,7 +310710,7 @@
     <Disorder id="12177">
       <OrphaCode>93172</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93172</ExpertLink>
-      <Name lang="nl">Unilaterale renale dysplasie</Name>
+      <Name lang="nl">Renale dysplasie, unilateraal</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -310729,7 +310761,7 @@
     <Disorder id="12178">
       <OrphaCode>93173</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93173</ExpertLink>
-      <Name lang="nl">Bilaterale renale dysplasie</Name>
+      <Name lang="nl">Renale dysplasie, bilateraal</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -321693,7 +321725,7 @@
     <Disorder id="11376">
       <OrphaCode>79350</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79350</ExpertLink>
-      <Name lang="nl">3-fosfoserinefosfatasedeficiëntie</Name>
+      <Name lang="nl">3-fosfoserinefosfatasedeficiëntie, infantiele/juveniele vorm</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -330674,7 +330706,7 @@
     <Disorder id="11618">
       <OrphaCode>84132</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=84132</ExpertLink>
-      <Name lang="nl">Desminegerelateerde myopathie met Mallorylichaampjesachtige insluitsels</Name>
+      <Name lang="nl">Desminegerelateerde myopathie met Mallory-lichaampjes-achtige inclusies</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -330723,7 +330755,7 @@
     <Disorder id="11619">
       <OrphaCode>84142</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=84142</ExpertLink>
-      <Name lang="nl">Isaacsyndroom</Name>
+      <Name lang="nl">Syndroom van Isaac</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -333274,7 +333306,7 @@
     <Disorder id="12802">
       <OrphaCode>96178</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96178</ExpertLink>
-      <Name lang="nl">Ringchromosoom 16 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 16-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -333323,7 +333355,7 @@
     <Disorder id="12801">
       <OrphaCode>96177</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96177</ExpertLink>
-      <Name lang="nl">Ringchromosoom 15 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 15-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -333372,7 +333404,7 @@
     <Disorder id="12800">
       <OrphaCode>96176</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96176</ExpertLink>
-      <Name lang="nl">Ringchromosoom 13 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 13-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -334494,7 +334526,7 @@
     <Disorder id="12661">
       <OrphaCode>95706</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95706</ExpertLink>
-      <Name lang="nl">Posterieure hypospadieën</Name>
+      <Name lang="nl">Posterieure hypospadie</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -336326,7 +336358,7 @@
     <Disorder id="12799">
       <OrphaCode>96175</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96175</ExpertLink>
-      <Name lang="nl">Ringchromosoom 11 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 11-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -336375,7 +336407,7 @@
     <Disorder id="12797">
       <OrphaCode>96173</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96173</ExpertLink>
-      <Name lang="nl">Ringchromosoom 9 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 9-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -336424,7 +336456,7 @@
     <Disorder id="12796">
       <OrphaCode>96172</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96172</ExpertLink>
-      <Name lang="nl">Ringchromosoom 3 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 3-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -336473,7 +336505,7 @@
     <Disorder id="12795">
       <OrphaCode>96171</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96171</ExpertLink>
-      <Name lang="nl">Ringchromosoom 2 syndroom</Name>
+      <Name lang="nl">Ringchromosoom 2-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -337276,7 +337308,7 @@
     <Disorder id="12396">
       <OrphaCode>93554</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93554</ExpertLink>
-      <Name lang="nl">Type II gemengde cryoglobulinemie</Name>
+      <Name lang="nl">Gemengde cryoglobulinemie type II</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -340533,7 +340565,7 @@
     <Disorder id="12423">
       <OrphaCode>93585</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93585</ExpertLink>
-      <Name lang="nl">Verworven trombotische trombocytopenische purpura</Name>
+      <Name lang="nl">Immuungemedieerde trombotische trombocytopenische purpura</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
