commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml
index cec570f..2d816af 100644
--- a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:42:44" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:31:48" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="6056">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -6961,7 +6954,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=593</ExpertLink>
       <Name lang="nl">Myofibrillaire myopathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -22292,7 +22285,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2443</ExpertLink>
       <Name lang="nl">Stoornis van de mitochondriale oxidatieve fosforylatie door afwijkingen in het nucleaire DNA</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -22984,7 +22977,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99</ExpertLink>
       <Name lang="nl">Autosomale dominante cerebellaire ataxie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -27830,7 +27823,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352</ExpertLink>
       <Name lang="nl">Galactosemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -32376,7 +32369,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=167714</ExpertLink>
       <Name lang="nl">Niet-geclassificeerde acute myeloïde leukemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -41218,7 +41211,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=69</ExpertLink>
       <Name lang="nl">Amyloïdose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -41434,7 +41427,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166</ExpertLink>
       <Name lang="nl">Ziekte van Charcot-Marie-Tooth/Erfelijke motorische en sensorische neuropathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -48130,7 +48123,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=177107</ExpertLink>
       <Name lang="nl">Syndromale hypothyreoïdie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -51289,7 +51282,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=171901</ExpertLink>
       <Name lang="nl">Primair cutaan T-cellymfoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -51807,7 +51800,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=171915</ExpertLink>
       <Name lang="nl">B-cel non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -51935,7 +51928,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=171918</ExpertLink>
       <Name lang="nl">T-cel non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -52016,7 +52009,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=187</ExpertLink>
       <Name lang="nl">Citrullinemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -52097,7 +52090,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=442</ExpertLink>
       <Name lang="nl">Congenitale hypothyreoïdie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -56441,7 +56434,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=795</ExpertLink>
       <Name lang="nl">Zeldzame vorm van salmonellose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -63359,7 +63352,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1531</ExpertLink>
       <Name lang="nl">Craniosynostose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -71604,7 +71597,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=183669</ExpertLink>
       <Name lang="nl">Agammaglobulinemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -75145,7 +75138,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=183518</ExpertLink>
       <Name lang="nl">Zeldzame genetische ataxie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -75645,7 +75638,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=183435</ExpertLink>
       <Name lang="nl">Overgeërfde ichthyosis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -75947,7 +75940,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=183422</ExpertLink>
       <Name lang="nl">Polymalformatief genetisch syndroom met verhoogd risico op het ontwikkelen van kanker</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -76049,7 +76042,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=182114</ExpertLink>
       <Name lang="nl">Zeldzame urogenitale tumor</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -76132,7 +76125,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=182130</ExpertLink>
       <Name lang="nl">Tumor van endocriene klieren</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -76285,7 +76278,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=182090</ExpertLink>
       <Name lang="nl">Pulmonale arteriële hypertensie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -76526,7 +76519,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=182095</ExpertLink>
       <Name lang="nl">Interstitiële longziekte</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -76683,7 +76676,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1047</ExpertLink>
       <Name lang="nl">Sideroblastische anemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -90540,7 +90533,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=138044</ExpertLink>
       <Name lang="nl">Zeldzame ziekte met syndroom van Pierre Robin</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -99334,7 +99327,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=140874</ExpertLink>
       <Name lang="nl">Syndroom van Joubert en aanverwante aandoeningen</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -106827,7 +106820,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158032</ExpertLink>
       <Name lang="nl">Hemofagocytair syndroom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -111635,7 +111628,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=247582</ExpertLink>
       <Name lang="nl">Citrinedeficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -115623,7 +115616,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251852</ExpertLink>
       <Name lang="nl">Embryonale tumor van het neuro-epitheliale weefsel</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -123600,7 +123593,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=252057</ExpertLink>
       <Name lang="nl">Tumor van de craniale en spinale zenuwen</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -123730,7 +123723,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=252028</ExpertLink>
       <Name lang="nl">Primaire melanocytaire tumor van het centrale zenuwstelsel</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -123862,7 +123855,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254367</ExpertLink>
       <Name lang="nl">Zeldzame lichen planus</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -134869,7 +134862,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=262941</ExpertLink>
       <Name lang="nl">Partiële trisomie van de lange arm van chromosoom 14</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -135817,7 +135810,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=268843</ExpertLink>
       <Name lang="nl">Malformatie van het neurenterische kanaal, ruggenmerg en wervelkolom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -137552,7 +137545,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2846</ExpertLink>
       <Name lang="nl">Congenitale pericardanomalie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -137714,7 +137707,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=268357</ExpertLink>
       <Name lang="nl">Sluitingsdefect van neurale buis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -142171,7 +142164,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=275791</ExpertLink>
       <Name lang="nl">Pulmonale arteriële hypertensie geassocieerd met andere ziekte</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -144342,7 +144335,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3388</ExpertLink>
       <Name lang="nl">Neuralebuisdefect</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -147281,7 +147274,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=268926</ExpertLink>
       <Name lang="nl">Middellijn cerebrale malformatie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -147609,7 +147602,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=269224</ExpertLink>
       <Name lang="nl">Algemene cerebellaire malformatie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -161365,7 +161358,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=599</ExpertLink>
       <Name lang="nl">Distale myopathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -162104,7 +162097,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=211062</ExpertLink>
       <Name lang="nl">Erfelijke episodische ataxie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -162136,7 +162129,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2034</ExpertLink>
       <Name lang="nl">Filariasis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -162451,7 +162444,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2415</ExpertLink>
       <Name lang="nl">Zeldzame lymfatische malformatie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -162958,7 +162951,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=137</ExpertLink>
       <Name lang="nl">Congenitale glycosylatiestoornis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -163645,7 +163638,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=359</ExpertLink>
       <Name lang="nl">Glaucoom met genetische oorsprong en pediatrische aanvang</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -163677,7 +163670,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=213500</ExpertLink>
       <Name lang="nl">Ovariumkanker</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -169075,7 +169068,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=216675</ExpertLink>
       <Name lang="nl">Transpositie van de grote slagaders</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -171199,7 +171192,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217074</ExpertLink>
       <Name lang="nl">Zeldzaam carcinoom van pancreas</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -171804,8 +171797,8 @@
           <PrevalenceType id="23697">
             <Name lang="nl">Cases/families</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="nl">Case</Name>
+          <PrevalenceQualification id="23725">
+            <Name lang="nl">Family</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
           <ValMoy>35.0</ValMoy>
@@ -172021,7 +172014,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=344</ExpertLink>
       <Name lang="nl">Koorts door arbovirus</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -172053,7 +172046,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=341</ExpertLink>
       <Name lang="nl">Virale hemorragische koorts</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -173248,7 +173241,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1132</ExpertLink>
       <Name lang="nl">Aortaboogdefecten</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -174365,7 +174358,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=542</ExpertLink>
       <Name lang="nl">Primair cutaan lymfoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -175559,7 +175552,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=220489</ExpertLink>
       <Name lang="nl">Zeldzame erfelijke hemochromatose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -177543,7 +177536,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3398</ExpertLink>
       <Name lang="nl">Epitheliale neoplasmata van de thymus</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -177645,7 +177638,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=547</ExpertLink>
       <Name lang="nl">Non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -178060,7 +178053,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3399</ExpertLink>
       <Name lang="nl">Kiemceltumor</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -179316,7 +179309,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=226292</ExpertLink>
       <Name lang="nl">Permanente congenitale hypothyreoïdie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -181747,7 +181740,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1172</ExpertLink>
       <Name lang="nl">Autosomaal recessieve cerebellaire ataxie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -184491,7 +184484,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1866</ExpertLink>
       <Name lang="nl">Focale, segmentale of multifocale dystonie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -186785,7 +186778,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231637</ExpertLink>
       <Name lang="nl">Zeldzame chirurgisch corrigeerbare vorm van primair aldosteronisme</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -188261,7 +188254,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238547</ExpertLink>
       <Name lang="nl">Verworven secundaire polycytemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -188342,7 +188335,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238536</ExpertLink>
       <Name lang="nl">Congenitale secundaire polycytemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -194181,7 +194174,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319543</ExpertLink>
       <Name lang="nl">Autosomaal dominante mendeliaanse gevoeligheid voor mycobacteriële ziekten door een partiële deficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -194262,7 +194255,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319535</ExpertLink>
       <Name lang="nl">Autosomaal recessieve mendeliaanse gevoeligheid voor mycobacteriële ziekten door een volledige deficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -194294,7 +194287,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319539</ExpertLink>
       <Name lang="nl">Autosomaal recessieve mendeliaanse gevoeligheid voor mycobacteriële ziekten door een partiële deficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -202365,7 +202358,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363203</ExpertLink>
       <Name lang="nl">Ringchromosoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -202836,7 +202829,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363579</ExpertLink>
       <Name lang="nl">Extragonadale kiemceltumor</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -203541,7 +203534,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363472</ExpertLink>
       <Name lang="nl">Testiculaire en paratesticulaire tumor</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -211419,7 +211412,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=280898</ExpertLink>
       <Name lang="nl">Panuveïtis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -211451,7 +211444,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=280892</ExpertLink>
       <Name lang="nl">Posterieure uveïtis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -211483,7 +211476,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=280886</ExpertLink>
       <Name lang="nl">Anterieure uveïtis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -211687,7 +211680,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=281085</ExpertLink>
       <Name lang="nl">Overgeërfde ichthyosis syndromale vorm</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -211719,7 +211712,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=281082</ExpertLink>
       <Name lang="nl">Overgeërfde ichthyosis niet-syndromale vorm</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -218216,7 +218209,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300755</ExpertLink>
       <Name lang="nl">Laminopathie met betrokkenheid van gestreepte spieren</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -218280,7 +218273,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300763</ExpertLink>
       <Name lang="nl">Laminopathie met lipodystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -218312,7 +218305,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300758</ExpertLink>
       <Name lang="nl">Laminopathie met perifere neuropathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -218344,7 +218337,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300842</ExpertLink>
       <Name lang="nl">Indolente B-cel non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -218376,7 +218369,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300766</ExpertLink>
       <Name lang="nl">Laminopathie met vroegtijdige veroudering</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -219128,7 +219121,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=298644</ExpertLink>
       <Name lang="nl">Stoornis van het thiaminemetabolisme en -transport</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -234227,7 +234220,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=398043</ExpertLink>
       <Name lang="nl">Maligne tumor van penis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -236630,7 +236623,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=398934</ExpertLink>
       <Name lang="nl">Maligne epitheliale tumor van ovarium</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -237131,7 +237124,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=398940</ExpertLink>
       <Name lang="nl">Maligne niet-epitheliale tumor van het ovarium</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -253036,7 +253029,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=431353</ExpertLink>
       <Name lang="nl">Pulmonale veno-occlusieve ziekte en/of pulmonale capillaire hemangiomatose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -262196,7 +262189,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=450</ExpertLink>
       <Name lang="nl">Heterotaxie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -262279,7 +262272,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=224</ExpertLink>
       <Name lang="nl">Neonatale diabetes mellitus</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -264434,7 +264427,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=877</ExpertLink>
       <Name lang="nl">Neuro-endocrien neoplasma</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -264729,7 +264722,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=467</ExpertLink>
       <Name lang="nl">Niet-verworven gecombineerde hypofysehormoondeficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -273445,7 +273438,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=71198</ExpertLink>
       <Name lang="nl">Zeldzame pulmonale hypertensie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -276294,7 +276287,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=69028</ExpertLink>
       <Name lang="nl">Dysostose met brachydactylie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -279339,7 +279332,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63454</ExpertLink>
       <Name lang="nl">Patroonvormige dystrofie van het retinale pigmentepitheel</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -279420,7 +279413,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63443</ExpertLink>
       <Name lang="nl">Zeldzame epitheliale tumor van maag</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -280669,7 +280662,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=56970</ExpertLink>
       <Name lang="nl">Humane prionziekte</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -281863,7 +281856,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79213</ExpertLink>
       <Name lang="nl">Mucopolysacharidose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -283385,7 +283378,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79168</ExpertLink>
       <Name lang="nl">Aandoening van galzuursynthese</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -283417,7 +283410,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79158</ExpertLink>
       <Name lang="nl">Cerebrale organische acidurie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -287167,7 +287160,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=77240</ExpertLink>
       <Name lang="nl">Primair lymfoedeem</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -290065,7 +290058,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35705</ExpertLink>
       <Name lang="nl">Neurometabole stoornis door deficiëntie van serine</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -290408,7 +290401,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35698</ExpertLink>
       <Name lang="nl">Mitochondriaal DNA-depletiesyndroom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -290847,7 +290840,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=36388</ExpertLink>
       <Name lang="nl">Paraneoplastisch neurologisch syndroom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -291305,7 +291298,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35808</ExpertLink>
       <Name lang="nl">Maligne sex cord-stromale tumor van ovarium</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -291831,7 +291824,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35807</ExpertLink>
       <Name lang="nl">Maligne kiemceltumor van ovarium</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -294787,7 +294780,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=34533</ExpertLink>
       <Name lang="nl">Corneadystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -296298,7 +296291,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=52759</ExpertLink>
       <Name lang="nl">Vasculitis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -302078,7 +302071,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=68367</ExpertLink>
       <Name lang="nl">Zeldzame aangeboren stofwisselingsziekte</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -302643,7 +302636,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90061</ExpertLink>
       <Name lang="nl">Niet-infectieuze posterieure uveïtis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -308125,7 +308118,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=88991</ExpertLink>
       <Name lang="nl">Zeldzame congenitale niet-syndromale hartmalformatie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -313431,7 +313424,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90970</ExpertLink>
       <Name lang="nl">Primaire lipodystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -314088,7 +314081,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79373</ExpertLink>
       <Name lang="nl">Ectodermale dysplasiesyndroom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -320337,7 +320330,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79361</ExpertLink>
       <Name lang="nl">Overgeërfde epidermolysis bullosa</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -324947,7 +324940,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=86846</ExpertLink>
       <Name lang="nl">Therapiegerelateerde acute myeloïde leukemie en myelodysplastisch syndroom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330286,7 +330279,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98085</ExpertLink>
       <Name lang="nl">46,XY-geslachtsontwikkelingsstoornis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330437,7 +330430,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98287</ExpertLink>
       <Name lang="nl">Histiocytische en dendritische celtumor</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330577,7 +330570,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98292</ExpertLink>
       <Name lang="nl">Mastocytose</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330679,7 +330672,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98277</ExpertLink>
       <Name lang="nl">Acute myeloïde leukemie met terugkerende genetische anomalie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330743,7 +330736,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97245</ExpertLink>
       <Name lang="nl">Congenitale myopathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330775,7 +330768,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97242</ExpertLink>
       <Name lang="nl">Congenitale spierdystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -330858,7 +330851,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97253</ExpertLink>
       <Name lang="nl">Neuro-endocriene tumor van pancreas</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -333313,7 +333306,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97593</ExpertLink>
       <Name lang="nl">Pseudohypoparathyreoïdie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -333528,7 +333521,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95708</ExpertLink>
       <Name lang="nl">Zeldzame vroegtijdige puberteit</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -333777,7 +333770,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95714</ExpertLink>
       <Name lang="nl">Pimaire congenitale hypothyreoïdie zonder schildklierontwikkelingsanomalie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -333809,7 +333802,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95711</ExpertLink>
       <Name lang="nl">Congenitale hypothyreoïdie als gevolg van ontwikkelingsanomalie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -337082,7 +337075,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93420</ExpertLink>
       <Name lang="nl">FGFR3-gerelateerde chondrodysplasie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -339155,7 +339148,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93890</ExpertLink>
       <Name lang="nl">Zeldzame ontwikkelingsstoornis tijdens embryogenese</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -351680,7 +351673,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98555</ExpertLink>
       <Name lang="nl">Anoftalmie - microftalmie-syndroom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -352680,7 +352673,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98627</ExpertLink>
       <Name lang="nl">Posterieure corneadystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -352712,7 +352705,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98626</ExpertLink>
       <Name lang="nl">Stromale corneadystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -352744,7 +352737,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98625</ExpertLink>
       <Name lang="nl">Oppervlakkige corneadystrofie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -352897,7 +352890,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98715</ExpertLink>
       <Name lang="nl">Uveïtis</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -354783,7 +354776,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98482</ExpertLink>
       <Name lang="nl">Idiopathische inflammatoire myopathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -354879,7 +354872,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98428</ExpertLink>
       <Name lang="nl">Secundaire polycytemie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -355788,7 +355781,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98497</ExpertLink>
       <Name lang="nl">Genetische perifere neuropathie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -356568,7 +356561,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=101997</ExpertLink>
       <Name lang="nl">Primaire immuundeficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -356847,7 +356840,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=101959</ExpertLink>
       <Name lang="nl">Chronische primaire adrenale insufficiëntie</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -359262,7 +359255,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100088</ExpertLink>
       <Name lang="nl">Schildkliercarcinoom</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
@@ -359839,7 +359832,7 @@
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100087</ExpertLink>
       <Name lang="nl">Schildkliertumor</Name>
       <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
+        <Name lang="nl">Categorie</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
