commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml
old mode 100755
new mode 100644
index a0e0f26..88d22e6
--- a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:44:44" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:47:14" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -17183,7 +17183,7 @@
     <Disorder id="164">
       <OrphaCode>2182</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2182</ExpertLink>
-      <Name lang="nl">Hydrocefalus met stenose van het aquaduct van Sylvius</Name>
+      <Name lang="nl">Hydrocefalus met stenose van aquaduct van Sylvius</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -90523,7 +90523,7 @@
     <Disorder id="16705">
       <OrphaCode>137678</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=137678</ExpertLink>
-      <Name lang="nl">Tsjechische dysplasie, metatarsaal type</Name>
+      <Name lang="nl">Spondylo-epifysaire dysplasie met abnormaal korte metatarsalen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -94560,7 +94560,7 @@
     <Disorder id="1614">
       <OrphaCode>1416</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1416</ExpertLink>
-      <Name lang="nl">Familiale calciumpyrofosfaatafzetting</Name>
+      <Name lang="nl">Familiale depositie van calciumpyrofosfaat</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -101559,7 +101559,7 @@
     <Disorder id="1668">
       <OrphaCode>1497</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1497</ExpertLink>
-      <Name lang="nl">X-gebonden gecompliceerde dysgenesie van het corpus callosum</Name>
+      <Name lang="nl">X-gebonden gecompliceerde dysgenesie van corpus callosum</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -109858,7 +109858,7 @@
     <Disorder id="17237">
       <OrphaCode>158687</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158687</ExpertLink>
-      <Name lang="nl">Letale acantholytische epidermolysis bullosa</Name>
+      <Name lang="nl">Letale acantholytische erosieve aandoening</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -110039,7 +110039,7 @@
     <Disorder id="17235">
       <OrphaCode>158681</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158681</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex met circinaat migrerend erytheem</Name>
+      <Name lang="nl">Epidermolysis bullosa simplex met circinair migrerend erytheem</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -110071,7 +110071,7 @@
     <Disorder id="17234">
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158676</ExpertLink>
-      <Name lang="nl">Dominante dystrofische epidermolysis bullosa, enkel nagels</Name>
+      <Name lang="nl">Gelokaliseerde dystrofische epidermolysis bullosa van uitsluitend nagels</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -110120,7 +110120,7 @@
     <Disorder id="17233">
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158673</ExpertLink>
-      <Name lang="nl">Acrale dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Gelokaliseerde dystrofische epidermolysis bullosa, acrale vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -110169,7 +110169,7 @@
     <Disorder id="17232">
       <OrphaCode>158668</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158668</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex als gevolg van plakophilinedeficiëntie</Name>
+      <Name lang="nl">Ectodermale dysplasie - fragiliteit van huid-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -114814,7 +114814,7 @@
     <Disorder id="2296">
       <OrphaCode>2483</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2483</ExpertLink>
-      <Name lang="nl">Melkersson-Rosenthalsyndroom</Name>
+      <Name lang="nl">Syndroom van Melkersson-Rosenthal</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -119590,7 +119590,7 @@
     <Disorder id="2119">
       <OrphaCode>2249</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2249</ExpertLink>
-      <Name lang="nl">Ulna-hypoplasie - intellectuele achterstand</Name>
+      <Name lang="nl">Hypoplasie van ulna - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -121013,7 +121013,7 @@
     <Disorder id="19640">
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251393</ExpertLink>
-      <Name lang="nl">Gelokaliseerde junctionele epidermolysis bullosa, niet-Herlitz-type</Name>
+      <Name lang="nl">Gelokaliseerde junctionele epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -125421,7 +125421,7 @@
     <Disorder id="19728">
       <OrphaCode>252175</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=252175</ExpertLink>
-      <Name lang="nl">OBSOLEET: Vestibulair schwannoom</Name>
+      <Name lang="nl">Vestibulair schwannoom</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -126260,7 +126260,7 @@
     <Disorder id="2533">
       <OrphaCode>1306</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1306</ExpertLink>
-      <Name lang="nl">Buschke-Ollendorffsyndroom</Name>
+      <Name lang="nl">Syndroom van Buschke-Ollendorff</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -131243,7 +131243,7 @@
     <Disorder id="19879">
       <OrphaCode>261476</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261476</ExpertLink>
-      <Name lang="nl">Xp21 microdeletiesyndroom</Name>
+      <Name lang="nl">Xp21 deletiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -141255,7 +141255,7 @@
     <Disorder id="2670">
       <OrphaCode>740</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=740</ExpertLink>
-      <Name lang="nl">Hutchinson-Gilford progeriasyndroom</Name>
+      <Name lang="nl">Progeriasyndroom van Hutchinson-Gilford</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -147506,7 +147506,7 @@
     <Disorder id="3057">
       <OrphaCode>3243</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3243</ExpertLink>
-      <Name lang="nl">Sweetsyndroom</Name>
+      <Name lang="nl">Syndroom van Sweet</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -161519,7 +161519,7 @@
     <Disorder id="3525">
       <OrphaCode>3286</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3286</ExpertLink>
-      <Name lang="nl">Catecholaminerge polymorfe ventrikeltachycardie</Name>
+      <Name lang="nl">Catecholaminerge polymorfe ventriculaire tachycardie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -172510,7 +172510,7 @@
     <Disorder id="3423">
       <OrphaCode>2745</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2745</ExpertLink>
-      <Name lang="nl">Opitz G/BBB-syndroom</Name>
+      <Name lang="nl">Opitz GBBB-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -187583,7 +187583,7 @@
     <Disorder id="19201">
       <OrphaCode>231568</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231568</ExpertLink>
-      <Name lang="nl">Gegeneraliseerde dominante dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Autosomaal dominante gegeneraliseerde dystrofische epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -196321,7 +196321,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319671</ExpertLink>
-      <Name lang="nl">Microcefale primordiale dwerggroei, Alazami-type</Name>
+      <Name lang="nl">Syndroom van Alazami</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -203734,7 +203734,7 @@
     <Disorder id="22327">
       <OrphaCode>363611</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363611</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand - voedingsproblemen - ontwikkelingsachterstand - microcefalie-syndroom</Name>
+      <Name lang="nl">CTCF-gerelateerde neurologische ontwikkelingsstoonis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -219843,7 +219843,7 @@
     <Disorder id="21219">
       <OrphaCode>306504</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=306504</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa met respiratoire en renale betrokkenheid</Name>
+      <Name lang="nl">Interstitiële longziekte - nefrotisch syndroom - epidermolysis bullosa-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -221198,7 +221198,7 @@
     <Disorder id="21112">
       <OrphaCode>300333</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300333</ExpertLink>
-      <Name lang="nl">Nefrotisch syndroom - doofheid - pretibiale epidermolysis bullosa-syndroom</Name>
+      <Name lang="nl">Nefrotisch syndroom - epidermolysis bullosa - sensorineurale doofheid-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -221296,7 +221296,7 @@
     <Disorder id="24042">
       <OrphaCode>464738</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=464738</ExpertLink>
-      <Name lang="nl">Congenitaal cataract - microcefalie - nevus flammeus simplex - ernstige intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Syndroom van Basel-Vanagaite-Smirin-Yosef</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -228483,7 +228483,7 @@
     <Disorder id="24234">
       <OrphaCode>468678</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=468678</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand - microcefalie - strabisme - gedragsstoornissen-syndroom</Name>
+      <Name lang="nl">Syndroom van White-Sutton</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -232125,7 +232125,7 @@
     <Disorder id="23023">
       <OrphaCode>411629</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=411629</ExpertLink>
-      <Name lang="nl">Nefropathische infantiele cystinose</Name>
+      <Name lang="nl">Infantiele nefropathische cystinose</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -233181,7 +233181,7 @@
     <Disorder id="22634">
       <OrphaCode>391641</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=391641</ExpertLink>
-      <Name lang="nl">Feingoldsyndroom type 1</Name>
+      <Name lang="nl">Syndroom van Feingold type 1</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -240590,7 +240590,7 @@
     <Disorder id="23367">
       <OrphaCode>438178</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=438178</ExpertLink>
-      <Name lang="nl">Acyl-CoA-reductase 1-deficiëntie</Name>
+      <Name lang="nl">Deficiëntie van vetzuur-acyl-CoA-reductase 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -259005,7 +259005,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=508529</ExpertLink>
-      <Name lang="nl">Gegeneraliseerde basale epidermolysis bullosa simplex met huidatrofie, littekenvorming en haarverlies</Name>
+      <Name lang="nl">Intermediaire epidermolysis bullosa simplex met cardiomyopathie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -261308,7 +261308,7 @@
     <Disorder id="26002">
       <OrphaCode>500163</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=500163</ExpertLink>
-      <Name lang="nl">SIN3A-gerelateerde intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Syndroom van Witteveen-Kolk</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -263966,7 +263966,7 @@
     <Disorder id="8662">
       <OrphaCode>901</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=901</ExpertLink>
-      <Name lang="nl">Wellssyndroom</Name>
+      <Name lang="nl">Syndroom van Wells</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -267914,7 +267914,7 @@
       <DisorderGroup id="36540">
         <Name lang="nl">Groep van aandoeningen</Name>
       </DisorderGroup>
-      <PrevalenceList count="12">
+      <PrevalenceList count="11">
         <Prevalence id="5195">
           <Source>20566927[PMID]</Source>
           <PrevalenceType id="23669">
@@ -267942,10 +267942,10 @@
           <PrevalenceQualification id="23711">
             <Name lang="nl">Value and class</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="nl">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="nl">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.1</ValMoy>
+          <ValMoy>0.572</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="nl">Europa</Name>
           </PrevalenceGeographic>
@@ -268105,27 +268105,8 @@
             <Name lang="nl">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="5205">
-          <Source>ISBN:951563203[OTHER]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="nl">Puntprevalentie</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23711">
-            <Name lang="nl">Value and class</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="nl">1-9 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.12</ValMoy>
-          <PrevalenceGeographic id="25279">
-            <Name lang="nl">South Africa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="nl">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
         <Prevalence id="5206">
-          <Source>ISBN:801860245[OTHER]</Source>
+          <Source>PMID: 27463098</Source>
           <PrevalenceType id="23669">
             <Name lang="nl">Puntprevalentie</Name>
           </PrevalenceType>
@@ -268135,7 +268116,7 @@
           <PrevalenceClass id="23739">
             <Name lang="nl">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.24</ValMoy>
+          <ValMoy>0.14</ValMoy>
           <PrevalenceGeographic id="25482">
             <Name lang="nl">United States</Name>
           </PrevalenceGeographic>
@@ -268402,7 +268383,7 @@
     <Disorder id="8620">
       <OrphaCode>2908</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2908</ExpertLink>
-      <Name lang="nl">Syndroom van Kindler</Name>
+      <Name lang="nl">Epidermolysis bullosa van Kindler</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -269317,7 +269298,7 @@
     <Disorder id="25224">
       <OrphaCode>480864</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=480864</ExpertLink>
-      <Name lang="nl">Recurrente metabole encefalomyopathische crises - rhabdomyolyse - hartritmestoornis - intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Recidiverende metabole encefalomyopathische crises - rhabdomyolyse - hartritmestoornis - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -283273,7 +283254,7 @@
     <Disorder id="11178">
       <OrphaCode>79152</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79152</ExpertLink>
-      <Name lang="nl">Verspreide oppervlakkige actinische porokeratose</Name>
+      <Name lang="nl">Gedissemineerde oppervlakkige actinische porokeratose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -286851,7 +286832,7 @@
     <Disorder id="11082">
       <OrphaCode>75392</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=75392</ExpertLink>
-      <Name lang="nl">Periodontaal syndroom van Ehlers-Danlos</Name>
+      <Name lang="nl">Parodontaal syndroom van Ehlers-Danlos</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -290759,7 +290740,7 @@
     <Disorder id="10397">
       <OrphaCode>35708</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35708</ExpertLink>
-      <Name lang="nl">Aromatisch L-aminozuur-decarboxylasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van aromatisch L-aminozuur-decarboxylase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -298671,7 +298652,7 @@
     <Disorder id="10630">
       <OrphaCode>48652</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=48652</ExpertLink>
-      <Name lang="nl">Monosomie 22q13</Name>
+      <Name lang="nl">Monosomie 22q13.3</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -302509,7 +302490,7 @@
     <Disorder id="28081">
       <OrphaCode>544472</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=544472</ExpertLink>
-      <Name lang="nl">Atypisch hemolytisch-uremisch syndroom met afwijking in een gen van het complementsysteem</Name>
+      <Name lang="nl">Atypisch hemolytisch-uremisch syndroom met afwijking van gen van complementsysteem</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -304785,7 +304766,7 @@
     <Disorder id="11907">
       <OrphaCode>89842</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89842</ExpertLink>
-      <Name lang="nl">Recessieve dystrofische epidermolysis bullosa, gegeneraliseerd intermediair</Name>
+      <Name lang="nl">Autosomaal recessieve gegeneraliseerde dystrofische epidermolysis bullosa, intermediaire vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -307542,7 +307523,7 @@
     <Disorder id="11805">
       <OrphaCode>88618</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=88618</ExpertLink>
-      <Name lang="nl">Psychomotorische retardatie als gevolg van S-adenosylhomocysteïnehydrolasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van S-adenosylhomocysteïnehydrolase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -308748,7 +308729,7 @@
     <Disorder id="11903">
       <OrphaCode>89838</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89838</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, autosomaal recessieve K14</Name>
+      <Name lang="nl">Autosomaal recessieve gegeneraliseerde epidermolysis bullosa simplex</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -309061,7 +309042,7 @@
     <Disorder id="12199">
       <OrphaCode>93256</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93256</ExpertLink>
-      <Name lang="nl">Fragiele X-geassocieerde tremor/ataxiesyndroom</Name>
+      <Name lang="nl">Fragiele X-geassocieerd tremor/ataxie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -310492,7 +310473,7 @@
     <Disorder id="12257">
       <OrphaCode>93320</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93320</ExpertLink>
-      <Name lang="nl">Ulnaire hemimelie</Name>
+      <Name lang="nl">Hemimelie van ulna</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -314790,7 +314771,7 @@
     <Disorder id="11422">
       <OrphaCode>79396</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79396</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, gegeneraliseerd ernstig</Name>
+      <Name lang="nl">Autosomaal dominante gegeneraliseerde epidermolysis bullosa simplex, ernstige vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -314986,7 +314967,7 @@
     <Disorder id="11425">
       <OrphaCode>79399</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79399</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, gegeneraliseerd intermediair</Name>
+      <Name lang="nl">Autosomaal dominante gegeneraliseerde epidermolysis bullosa simplex, intermediaire vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315018,7 +314999,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79401</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, Ogna-type</Name>
+      <Name lang="nl">PLEC-gerelateerde intermediaire epidermolysis bullosa simplex zonder extracutane betrokkenheid</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315099,7 +315080,7 @@
     <Disorder id="11429">
       <OrphaCode>79403</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79403</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa - pylorusatresie</Name>
+      <Name lang="nl">Junctionele epidermolysis bullosa met pylorusatresie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315148,7 +315129,7 @@
     <Disorder id="11428">
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79402</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa, gegeneraliseerd intermediair</Name>
+      <Name lang="nl">Intermediaire gegeneraliseerde junctionele epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315229,7 +315210,7 @@
     <Disorder id="11430">
       <OrphaCode>79404</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79404</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa, gegeneraliseerd ernstig</Name>
+      <Name lang="nl">Ernstige gegeneraliseerde junctionele epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315435,7 +315416,7 @@
     <Disorder id="11434">
       <OrphaCode>79408</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79408</ExpertLink>
-      <Name lang="nl">Ernstige gegeneraliseerde recessieve dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Autosomaal recessieve gegeneraliseerde dystrofische epidermolysis bullosa, ernstige vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315505,7 +315486,7 @@
     <Disorder id="11437">
       <OrphaCode>79411</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79411</ExpertLink>
-      <Name lang="nl">Voorbijgaande bulleuze dermolyse van de neonaat</Name>
+      <Name lang="nl">Spontaan genezende dystrofische epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -315554,7 +315535,7 @@
     <Disorder id="11436">
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79410</ExpertLink>
-      <Name lang="nl">Pretibiale dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Gelokaliseerde dystrofische epidermolysis bullosa, pretibiale vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -334180,7 +334161,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95702</ExpertLink>
-      <Name lang="nl">Cytomegalische congenitale adrenale hypoplasie</Name>
+      <Name lang="nl">X-gebonden congenitale bijnierhypoplasie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -353461,7 +353442,7 @@
     <Disorder id="13451">
       <OrphaCode>98434</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98434</ExpertLink>
-      <Name lang="nl">Erfelijke gecombineerde deficiëntie van vitamine K-afhankelijke stollingsfactoren</Name>
+      <Name lang="nl">Hereditaire gecombineerde deficiëntie van vitamine K-afhankelijke stollingsfactoren</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -354017,7 +353998,7 @@
     <Disorder id="28676">
       <OrphaCode>570470</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=570470</ExpertLink>
-      <Name lang="nl">Ricin poisoning</Name>
+      <Name lang="nl">Intoxicatie door ricine</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
