commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml
old mode 100644
new mode 100755
index ee65e5d..f20348a
--- a/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/nl_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:41:08" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:42:08" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -1952,7 +1952,7 @@
     <Disorder id="18">
       <OrphaCode>369</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=369</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte type 6</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door hepatische glycogeenfosforylasedeficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -12994,7 +12994,7 @@
     <Disorder id="120">
       <OrphaCode>908</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=908</ExpertLink>
-      <Name lang="nl">Fragiele X-syndroom</Name>
+      <Name lang="nl">Fragiele-X-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -19631,7 +19631,7 @@
     <Disorder id="192">
       <OrphaCode>640</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=640</ExpertLink>
-      <Name lang="nl">Erfelijke neuropathie met vatbaarheid voor drukverlammingen</Name>
+      <Name lang="nl">Erfelijke drukneuropathie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -48342,8 +48342,8 @@
       <OrphaCode>178320</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=178320</ExpertLink>
       <Name lang="nl">Acuut longletsel</Name>
-      <DisorderType id="21422">
-        <Name lang="nl">Klinisch syndroom</Name>
+      <DisorderType id="21429">
+        <Name lang="nl">Particulaire klinische situatie binnen een ziekte of syndroom</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="nl">Disorder</Name>
@@ -61103,7 +61103,7 @@
     <Disorder id="18404">
       <OrphaCode>199337</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=199337</ExpertLink>
-      <Name lang="nl">Pancreasinsufficiëntie - anemie - hyperostose</Name>
+      <Name lang="nl">Pancreasinsufficiëntie - anemie - hyperostose-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -95896,7 +95896,7 @@
     <Disorder id="17147">
       <OrphaCode>157801</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=157801</ExpertLink>
-      <Name lang="nl">Meso-axiale synostotische syndactylie met falangeale reductie</Name>
+      <Name lang="nl">Mesoaxiale synostotische syndactylie met falangeale reductie</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -113260,7 +113260,7 @@
     <Disorder id="2216">
       <OrphaCode>2379</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2379</ExpertLink>
-      <Name lang="nl">Ziekte van Parkinson met vroege aanvang - intellectuele achterstand</Name>
+      <Name lang="nl">Ziekte van Parkinson met vroege aanvang - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115219,7 +115219,7 @@
     <Disorder id="2289">
       <OrphaCode>2476</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2476</ExpertLink>
-      <Name lang="nl">Dysrafie - gespleten lip/verhemelte - ledemaatreductiedefecten-syndroom</Name>
+      <Name lang="nl">Dysrafie - gespleten lip/verhemelte - reductiedefect van ledemaat-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -125499,6 +125499,95 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="19728">
+      <OrphaCode>252175</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=252175</ExpertLink>
+      <Name lang="nl">OBSOLEET: Vestibulair schwannoom</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype of disorder</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="16415">
+          <Source>20871439[PMID]_24655069[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="nl">Jaarlijkse incidentie</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="nl">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="nl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="24264">
+            <Name lang="nl">Denmark</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="nl">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16416">
+          <Source>National Cancer Institute[INST]_23432451[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="nl">Jaarlijkse incidentie</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="nl">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="nl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>1.2</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="nl">United States</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="nl">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16417">
+          <Source>21897393[PMID]</Source>
+          <PrevalenceType id="23676">
+            <Name lang="nl">Jaarlijkse incidentie</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="nl">Value and class</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="nl">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.8</ValMoy>
+          <PrevalenceGeographic id="23851">
+            <Name lang="nl">Specifieke populatie</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="nl">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16418">
+          <Source/>
+          <PrevalenceType id="23676">
+            <Name lang="nl">Jaarlijkse incidentie</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="nl">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="nl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="nl">Europa</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25965">
+            <Name lang="nl">Not yet validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="19756">
       <OrphaCode>254516</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254516</ExpertLink>
@@ -127032,7 +127121,7 @@
     <Disorder id="2549">
       <OrphaCode>675</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=675</ExpertLink>
-      <Name lang="nl">Ringvormige pancreas</Name>
+      <Name lang="nl">Annulaire pancreas</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -138675,7 +138764,7 @@
     <Disorder id="2579">
       <OrphaCode>2840</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2840</ExpertLink>
-      <Name lang="nl">Bekkendysplasie - artrogrypose van de onderste ledematen-syndroom</Name>
+      <Name lang="nl">Bekkendysplasie - artrogrypose van onderste ledematen-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -138967,7 +139056,7 @@
     <Disorder id="2590">
       <OrphaCode>708</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=708</ExpertLink>
-      <Name lang="nl">Petersanomalie</Name>
+      <Name lang="nl">Anomalie van Peters</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -140664,7 +140753,7 @@
     <Disorder id="2656">
       <OrphaCode>1848</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1848</ExpertLink>
-      <Name lang="nl">Bilaterale renale agenesie</Name>
+      <Name lang="nl">Agenesie van nier, bilateraal</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -142073,7 +142162,7 @@
     <Disorder id="2630">
       <OrphaCode>2911</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2911</ExpertLink>
-      <Name lang="nl">Polandsyndroom</Name>
+      <Name lang="nl">Syndroom van Poland</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -142162,7 +142251,7 @@
     <Disorder id="20103">
       <OrphaCode>264580</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=264580</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte type 9A</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door hepatische fosforylasekinasedeficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -154927,7 +155016,7 @@
     <Disorder id="3271">
       <OrphaCode>2089</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2089</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte als gevolg van hepatische glycogeensynthasedeficiëntie</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door hepatische glycogeensynthasedeficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -158520,7 +158609,7 @@
     <Disorder id="3155">
       <OrphaCode>2538</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2538</ExpertLink>
-      <Name lang="nl">Microgastrie - ledemaatreductiedefect</Name>
+      <Name lang="nl">Microgastrie - reductiedefect van ledemaat-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -163159,7 +163248,7 @@
     <Disorder id="3549">
       <OrphaCode>655</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=655</ExpertLink>
-      <Name lang="nl">Nefronoftisis</Name>
+      <Name lang="nl">Nefronoftise</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -171550,7 +171639,7 @@
     <Disorder id="3402">
       <OrphaCode>292</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=292</ExpertLink>
-      <Name lang="nl">Congenitale enterovirusinfectie</Name>
+      <Name lang="nl">Congenitale infectie door enterovirus</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -172053,7 +172142,7 @@
     <Disorder id="18822">
       <OrphaCode>217074</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217074</ExpertLink>
-      <Name lang="nl">Zeldzaam carcinoom van de pancreas</Name>
+      <Name lang="nl">Zeldzaam carcinoom van pancreas</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -182924,7 +183013,7 @@
     <Disorder id="3708">
       <OrphaCode>1333</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1333</ExpertLink>
-      <Name lang="nl">Familiaal pancreascarcinoom</Name>
+      <Name lang="nl">Familiaal carcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -191233,7 +191322,7 @@
     <Disorder id="21927">
       <OrphaCode>329324</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329324</ExpertLink>
-      <Name lang="nl">Invers Klippel-Trenaunaysyndroom</Name>
+      <Name lang="nl">Invers syndroom van Klippel-Trenaunay</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -207238,8 +207327,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=353284</ExpertLink>
       <Name lang="nl">Rubinstein-Taybisyndroom door EP300-haplo-insufficiëntie</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="nl">Subtype of disorder</Name>
@@ -208540,55 +208629,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=293848</ExpertLink>
-      <Name lang="nl">Frontotemporale dementie, rechter temporale atrofie variant</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Disorder</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="9768">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23697">
-            <Name lang="nl">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="nl">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>200.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="nl">Wereldwijd</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="nl">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10677">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="nl">Puntprevalentie</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="nl">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="nl">Unknown</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="nl">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="nl">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=293843</ExpertLink>
@@ -214608,7 +214648,7 @@
     <Disorder id="21470">
       <OrphaCode>314022</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314022</ExpertLink>
-      <Name lang="nl">Gastrisch adenocarcinoom en proximale polyposis van de maag</Name>
+      <Name lang="nl">Gastrisch adenocarcinoom en proximale polyposis van maag</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -219134,7 +219174,7 @@
     <Disorder id="21149">
       <OrphaCode>300878</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300878</ExpertLink>
-      <Name lang="nl">Haarcelleukemie variant</Name>
+      <Name lang="nl">Variant van haarcelleukemie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -221565,7 +221605,7 @@
     <Disorder id="24048">
       <OrphaCode>464756</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=464756</ExpertLink>
-      <Name lang="nl">Familiale neuro-endocriene tumor van de maag type 1</Name>
+      <Name lang="nl">Familiale neuro-endocriene tumor van maag type 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -232119,7 +232159,7 @@
     <Disorder id="23016">
       <OrphaCode>411527</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=411527</ExpertLink>
-      <Name lang="nl">Centrale retinale aderocclusie</Name>
+      <Name lang="nl">Veneuze stamocclusie van retina</Name>
       <DisorderType id="21429">
         <Name lang="nl">Particulaire klinische situatie binnen een ziekte of syndroom</Name>
       </DisorderType>
@@ -240467,7 +240507,7 @@
     <Disorder id="23332">
       <OrphaCode>436174</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=436174</ExpertLink>
-      <Name lang="nl">Cataract - groeihormoondeficiëntie - sensorische neuropathie - sensorineuraal gehoorverlies - skeletdysplasie-syndroom</Name>
+      <Name lang="nl">Cataract - deficiëntie van groeihormoon - sensorische neuropathie - sensorineuraal gehoorverlies - skeletdysplasie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -247137,7 +247177,7 @@
     <Disorder id="23193">
       <OrphaCode>424065</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424065</ExpertLink>
-      <Name lang="nl">Solide pseudopapillair carcinoom van de pancreas</Name>
+      <Name lang="nl">Solide pseudopapillair carcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -247378,7 +247418,7 @@
     <Disorder id="23192">
       <OrphaCode>424058</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424058</ExpertLink>
-      <Name lang="nl">Intraductaal papillair mucineus carcinoom van de pancreas</Name>
+      <Name lang="nl">Intraductaal papillair mucineus carcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -249007,7 +249047,7 @@
     <Disorder id="23189">
       <OrphaCode>424039</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424039</ExpertLink>
-      <Name lang="nl">Plaveiselcelcarcinoom van de pancreas</Name>
+      <Name lang="nl">Plaveiselcelcarcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -249438,7 +249478,7 @@
     <Disorder id="23191">
       <OrphaCode>424053</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424053</ExpertLink>
-      <Name lang="nl">Mucineus cystadenocarcinoom van de pancreas</Name>
+      <Name lang="nl">Mucineus cystadenocarcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -249698,7 +249738,7 @@
     <Disorder id="23190">
       <OrphaCode>424046</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424046</ExpertLink>
-      <Name lang="nl">Acinaire celcarcinoom van de pancreas</Name>
+      <Name lang="nl">Acinuscelcarcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -252019,7 +252059,7 @@
     <Disorder id="23202">
       <OrphaCode>424943</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424943</ExpertLink>
-      <Name lang="nl">Adenocarcinoom van de lever en de intrahepatische galwegen</Name>
+      <Name lang="nl">Adenocarcinoom van lever en intrahepatische galwegen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -252469,7 +252509,7 @@
     <Disorder id="23203">
       <OrphaCode>424970</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424970</ExpertLink>
-      <Name lang="nl">Ongedifferentieerd carcinoom van de lever en intrahepatische galwegen</Name>
+      <Name lang="nl">Ongedifferentieerd carcinoom van lever en intrahepatische galwegen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -257422,7 +257462,7 @@
     <Disorder id="8758">
       <OrphaCode>26106</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=26106</ExpertLink>
-      <Name lang="nl">Erfelijke diffuse maagkanker</Name>
+      <Name lang="nl">Hereditaire diffuse maagkanker</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -257828,7 +257868,7 @@
     <Disorder id="8707">
       <OrphaCode>674</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=674</ExpertLink>
-      <Name lang="nl">Bijkomstige pancreas</Name>
+      <Name lang="nl">Accessoire pancreas</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -272880,7 +272920,7 @@
     <Disorder id="9291">
       <OrphaCode>31202</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=31202</ExpertLink>
-      <Name lang="nl">Melioidose</Name>
+      <Name lang="nl">Melioïdose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -279716,7 +279756,7 @@
     <Disorder id="10821">
       <OrphaCode>63275</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63275</ExpertLink>
-      <Name lang="nl">Pemfigoïde gestationis</Name>
+      <Name lang="nl">Pemphigoid gestationis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -279886,7 +279926,7 @@
     <Disorder id="10824">
       <OrphaCode>63443</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63443</ExpertLink>
-      <Name lang="nl">Zeldzame epitheliale tumor van de maag</Name>
+      <Name lang="nl">Zeldzame epitheliale tumor van maag</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -280145,7 +280185,7 @@
     <Disorder id="10804">
       <OrphaCode>60039</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=60039</ExpertLink>
-      <Name lang="nl">Pudendale neuralgie</Name>
+      <Name lang="nl">Pudendusneuralgie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -286990,7 +287030,7 @@
     <Disorder id="28552">
       <OrphaCode>566841</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=566841</ExpertLink>
-      <Name lang="nl">Adenomatose van de lever</Name>
+      <Name lang="nl">Adenomatose van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -292004,7 +292044,7 @@
     <Disorder id="10415">
       <OrphaCode>36273</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=36273</ExpertLink>
-      <Name lang="nl">Gastrische linitis plastica</Name>
+      <Name lang="nl">Linitis plastica van maag</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -302257,7 +302297,7 @@
     <Disorder id="10615">
       <OrphaCode>48372</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=48372</ExpertLink>
-      <Name lang="nl">Nodulaire regeneratieve hyperplasie van de lever</Name>
+      <Name lang="nl">Nodulaire regeneratieve hyperplasie van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -303426,7 +303466,7 @@
     <Disorder id="28138">
       <OrphaCode>556955</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=556955</ExpertLink>
-      <Name lang="nl">Agenesie van de pancreas - holoprosencefalie-syndroom</Name>
+      <Name lang="nl">Agenesie van pancreas - holoprosencefalie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -304509,7 +304549,7 @@
     <Disorder id="11921">
       <OrphaCode>90003</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90003</ExpertLink>
-      <Name lang="nl">Inflammatoire pseudotumor van de lever</Name>
+      <Name lang="nl">Inflammatoire pseudotumor van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -310182,7 +310222,7 @@
     <Disorder id="12164">
       <OrphaCode>93101</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93101</ExpertLink>
-      <Name lang="nl">Renale hypoplasie</Name>
+      <Name lang="nl">Hypoplasie van nier</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -310414,7 +310454,7 @@
     <Disorder id="12163">
       <OrphaCode>93100</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93100</ExpertLink>
-      <Name lang="nl">Unilaterale renale agenesie</Name>
+      <Name lang="nl">Agenesie van nier, unilateraal</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -314726,7 +314766,7 @@
     <Disorder id="12123">
       <OrphaCode>91349</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91349</ExpertLink>
-      <Name lang="nl">Niet-functionerend hypofyseadenoom</Name>
+      <Name lang="nl">Niet-functioneel hypofyseadenoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -330193,7 +330233,7 @@
     <Disorder id="11602">
       <OrphaCode>83642</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=83642</ExpertLink>
-      <Name lang="nl">Microcytaire anemie met ijzerstapeling in de lever</Name>
+      <Name lang="nl">Microcytaire anemie met ijzerstapeling in lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -331945,7 +331985,7 @@
     <Disorder id="12870">
       <OrphaCode>97253</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97253</ExpertLink>
-      <Name lang="nl">Neuro-endocriene tumor van de pancreas</Name>
+      <Name lang="nl">Neuro-endocriene tumor van pancreas</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -358978,7 +359018,7 @@
     <Disorder id="14657">
       <OrphaCode>100085</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100085</ExpertLink>
-      <Name lang="nl">Primair neuro-endocrien carcinoom van de lever</Name>
+      <Name lang="nl">Primair neuro-endocrien carcinoom van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -359442,7 +359482,7 @@
     <Disorder id="14647">
       <OrphaCode>100075</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100075</ExpertLink>
-      <Name lang="nl">Neuro-endocriene tumor van de maag</Name>
+      <Name lang="nl">Neuro-endocriene tumor van maag</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
