commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare disease epidemiology/pl_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/pl_product9_prev.xml
index 0dc85e0..91c0ed8 100644
--- a/Epidemiological data/Rare disease epidemiology/pl_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/pl_product9_prev.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<JDBOR date="2021-11-01 04:52:16" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6044">
+<JDBOR date="2021-12-01 04:56:53" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6052">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=166024</ExpertLink>
@@ -2886,7 +2879,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="11716">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -2896,7 +2889,7 @@
           <PrevalenceClass id="23739">
             <Name lang="pl">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.31</ValMoy>
+          <ValMoy>0.28</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="pl">BRAZYLIA</Name>
           </PrevalenceGeographic>
@@ -3550,7 +3543,7 @@
     <Disorder id="17635">
       <OrphaCode>166308</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=166308</ExpertLink>
-      <Name lang="pl">Łagodna dziecięca padaczka ogniskowa</Name>
+      <Name lang="pl">Łagodna dziecięca padaczka ogniskowa z iglicami i falami w linii pośrodkowej podczas snu</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -4068,7 +4061,7 @@
       <DisorderGroup id="36547">
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
-      <PrevalenceList count="8">
+      <PrevalenceList count="9">
         <Prevalence id="136">
           <Source>European Medicines Agency 2012[INST]</Source>
           <PrevalenceType id="23669">
@@ -4107,6 +4100,25 @@
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16640">
+          <Source>PMID: 31926052</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="pl">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.02</ValMoy>
+          <PrevalenceGeographic id="24068">
+            <Name lang="pl">BRAZYLIA</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="11717">
           <Source>25060283[PMID]</Source>
           <PrevalenceType id="23683">
@@ -14374,7 +14386,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13915">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -14384,7 +14396,7 @@
           <PrevalenceClass id="23739">
             <Name lang="pl">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.38</ValMoy>
+          <ValMoy>0.37</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="pl">BRAZYLIA</Name>
           </PrevalenceGeographic>
@@ -14862,7 +14874,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13910">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -14913,7 +14925,7 @@
       </DisorderGroup>
       <PrevalenceList count="11">
         <Prevalence id="520">
-          <Source>17276780[PMID]</Source>
+          <Source>PMID: 31449670</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -14923,7 +14935,7 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.2</ValMoy>
+          <ValMoy>2.25</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
@@ -14951,7 +14963,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="522">
-          <Source>DOI:10.1016/S0168-8278(85)80013-1[OTHER]_17276780[PMID]</Source>
+          <Source>PMID: 31449670</Source>
           <PrevalenceType id="23669">
             <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
@@ -14961,7 +14973,7 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>3.3</ValMoy>
+          <ValMoy>2.02</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
@@ -22762,7 +22774,7 @@
     <Disorder id="17863">
       <OrphaCode>169464</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=169464</ExpertLink>
-      <Name lang="pl">Pierwotny Niedobór CD59</Name>
+      <Name lang="pl">Pierwotny niedobór CD59</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -30589,7 +30601,7 @@
     <Disorder id="17770">
       <OrphaCode>168555</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=168555</ExpertLink>
-      <Name lang="pl">Dysplazja kręgowo-przynasadowa sprzężona, typ  A4</Name>
+      <Name lang="pl">Dysplazja kręgowo-przynasadowa typu A4</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -38594,11 +38606,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2346</ExpertLink>
       <Name lang="pl">Zespół przerostu naczyń i kości</Name>
-      <DisorderType id="21394">
-        <Name lang="pl">Choroba</Name>
+      <DisorderType id="21436">
+        <Name lang="pl">Grupa fenomenów</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pl">Zaburzenie</Name>
+      <DisorderGroup id="36540">
+        <Name lang="pl">Grupa kliniczna</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="1286">
@@ -46160,7 +46172,26 @@
       <DisorderGroup id="36547">
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
-      <PrevalenceList count="25">
+      <PrevalenceList count="26">
+        <Prevalence id="16639">
+          <Source>PMID: 31926052</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="pl">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.21</ValMoy>
+          <PrevalenceGeographic id="24068">
+            <Name lang="pl">BRAZYLIA</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="1671">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
@@ -46883,7 +46914,7 @@
       <DisorderGroup id="36547">
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
-      <PrevalenceList count="8">
+      <PrevalenceList count="6">
         <Prevalence id="1689">
           <Source>PMID: 31769437, 25944380, 24715559</Source>
           <PrevalenceType id="23669">
@@ -46998,40 +47029,6 @@
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="16436">
-          <Source>PMID: 22913777</Source>
-          <PrevalenceType id="23697">
-            <Name lang="pl">Przypadki/rodziny</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="pl">Przypadek</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>1314.0</ValMoy>
-          <PrevalenceGeographic id="24166">
-            <Name lang="pl">CHINY</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pl">Zwalidowany</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="16435">
-          <Source>PMID: 26627451</Source>
-          <PrevalenceType id="23697">
-            <Name lang="pl">Przypadki/rodziny</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="pl">Przypadek</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>72.0</ValMoy>
-          <PrevalenceGeographic id="25356">
-            <Name lang="pl">TAJWAN</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pl">Zwalidowany</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="17999">
@@ -66758,7 +66755,7 @@
           <PrevalenceClass id="23739">
             <Name lang="pl">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.11</ValMoy>
+          <ValMoy>0.14</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="pl">BRAZYLIA</Name>
           </PrevalenceGeographic>
@@ -73335,7 +73332,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="2262">
-          <Source>15054950[PMID]_10817953[PMID]_[EXPERT]</Source>
+          <Source>PMID: 27037234  ; 34220661; ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
@@ -73345,7 +73342,7 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>6.0</ValMoy>
+          <ValMoy>5.26</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
@@ -75630,27 +75627,27 @@
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
-        <Prevalence id="11997">
-          <Source>20301500[PMID]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+        <Prevalence id="16626">
+          <Source>PMID: 31969461</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="pl">Wartość i klasa</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="pl">1-9 / 100 000</Name>
+          <PrevalenceClass id="23739">
+            <Name lang="pl">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.125</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pl">Cały świat</Name>
+          <ValMoy>0.47</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="pl">WŁOCHY</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="12014">
-          <Source>Kroes_Fransen van de Putte_Ravesloot 2007[AUTHOR]</Source>
+        <Prevalence id="11997">
+          <Source>20301500[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -75660,9 +75657,9 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.0</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="pl">HOLANDIA</Name>
+          <ValMoy>1.125</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
@@ -75676,15 +75673,15 @@
           <PrevalenceQualification id="23704">
             <Name lang="pl">Tylko klasa</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23746">
-            <Name lang="pl">1-9 / 100 000</Name>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
           </PrevalenceClass>
           <ValMoy>0.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="pl">Jeszcze niezwalidowane</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -77919,9 +77916,9 @@
       <DisorderGroup id="36540">
         <Name lang="pl">Grupa kliniczna</Name>
       </DisorderGroup>
-      <PrevalenceList count="5">
-        <Prevalence id="2605">
-          <Source>14518075[PMID]</Source>
+      <PrevalenceList count="6">
+        <Prevalence id="16623">
+          <Source>PMID: 26431509</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -77931,16 +77928,16 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="24936">
-            <Name lang="pl">HOLANDIA</Name>
+          <ValMoy>2.24</ValMoy>
+          <PrevalenceGeographic id="24257">
+            <Name lang="pl">CZECHY</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2606">
-          <Source>9130925[PMID]_14518075[PMID]</Source>
+        <Prevalence id="16624">
+          <Source>PMID: 31742894</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -77950,54 +77947,73 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="25482">
-            <Name lang="pl">STANY ZJEDNOCZONE AMERYKI</Name>
+          <ValMoy>5.25</ValMoy>
+          <PrevalenceGeographic id="24621">
+            <Name lang="pl">JAPONIA</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2607">
-          <Source>21967988[PMID]</Source>
+        <Prevalence id="16625">
+          <Source>PMID :32027664</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="pl">Wartość i klasa</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23732">
-            <Name lang="pl">1-5 / 10 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>40.0</ValMoy>
-          <PrevalenceGeographic id="25209">
-            <Name lang="pl">ARABIA SAUDYJSKA</Name>
+          <ValMoy>3.52</ValMoy>
+          <PrevalenceGeographic id="25475">
+            <Name lang="pl">ZJEDNOCZONE KRÓLESTWO</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="2608">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pl">Częstość występowania punktowa</Name>
+        <Prevalence id="2605">
+          <Source>14518075[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pl">Tylko klasa</Name>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
           </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pl">Europa</Name>
+          <ValMoy>2.0</ValMoy>
+          <PrevalenceGeographic id="24936">
+            <Name lang="pl">HOLANDIA</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="pl">Jeszcze niezwalidowane</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="2607">
+          <Source>21967988[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="pl">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>40.0</ValMoy>
+          <PrevalenceGeographic id="25209">
+            <Name lang="pl">ARABIA SAUDYJSKA</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="10928">
-          <Source>[EXPERT]_ORPHANET</Source>
+          <Source>ORPHANET</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -78007,7 +78023,7 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.0</ValMoy>
+          <ValMoy>3.67</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
@@ -81092,6 +81108,38 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="1228">
+      <OrphaCode>3306</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=3306</ExpertLink>
+      <Name lang="pl">Duplikacja/inwersja 15q11</Name>
+      <DisorderType id="21401">
+        <Name lang="pl">Zespół wad wrodzonych</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
+      </DisorderGroup>
+      <PrevalenceList count="1">
+        <Prevalence id="16620">
+          <Source>PMID: 11579431</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="pl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>3.33</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="1231">
       <OrphaCode>3375</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=3375</ExpertLink>
@@ -85480,7 +85528,7 @@
     <Disorder id="1293">
       <OrphaCode>973</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=973</ExpertLink>
-      <Name lang="pl">Wrodzony brak/niedorozwój palców poza kciukiem, jednostronny</Name>
+      <Name lang="pl">Wrodzony jednostronny brak/niedorozwój palców z wyjątkiem kciuka</Name>
       <DisorderType id="21415">
         <Name lang="pl">Wada morfologiczna</Name>
       </DisorderType>
@@ -85774,7 +85822,7 @@
     <Disorder id="16889">
       <OrphaCode>139414</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=139414</ExpertLink>
-      <Name lang="pl">Congenital panfollicular nevus</Name>
+      <Name lang="pl">Znamię wrodzone mieszków włosa</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -86023,7 +86071,7 @@
     <Disorder id="16893">
       <OrphaCode>139426</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=139426</ExpertLink>
-      <Name lang="pl">Mioklonia okołoustna z nieświadomością</Name>
+      <Name lang="pl">Mioklonia okołoustna z napadami nieświadomości</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -101852,7 +101900,7 @@
     <Disorder id="1913">
       <OrphaCode>1969</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=1969</ExpertLink>
-      <Name lang="pl">Zespół FACES</Name>
+      <Name lang="pl">Zespół dysmorfii twarzy, anoreksji, kacheksji, wad oczu i skóry</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -104120,7 +104168,7 @@
     <Disorder id="1838">
       <OrphaCode>1861</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=1861</ExpertLink>
-      <Name lang="pl">Dysplazja piersiowa - zespół wodogłowia</Name>
+      <Name lang="pl">Zespół dysplazji klatki piersiowej i wodogłowia</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -113757,7 +113805,7 @@
     <Disorder id="2300">
       <OrphaCode>2487</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2487</ExpertLink>
-      <Name lang="pl">Skrócenie kończyny dolnej - spodziectwo</Name>
+      <Name lang="pl">Zniekształcenie kończyny dolnej - spodziectwo</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -116434,7 +116482,7 @@
     <Disorder id="2097">
       <OrphaCode>2220</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2220</ExpertLink>
-      <Name lang="pl">Hipertrichoza łokciowa - niski wzrost</Name>
+      <Name lang="pl">Hipertrichoza łokciowa</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -123616,7 +123664,7 @@
     <Disorder id="2503">
       <OrphaCode>2743</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2743</ExpertLink>
-      <Name lang="pl">Oftalmoplegia - niepełnosprawnoś intelektualna - język mosznowy</Name>
+      <Name lang="pl">Oftalmoplegia - niepełnosprawność intelektualna - język pobrużdżony</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -124040,7 +124088,7 @@
     <Disorder id="19740">
       <OrphaCode>254351</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=254351</ExpertLink>
-      <Name lang="pl">Dystalny zespół mikrodelecji 7q11.23</Name>
+      <Name lang="pl">Zespół dystalnej mikrodelecji 7q11.23</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -124319,7 +124367,7 @@
     <Disorder id="19736">
       <OrphaCode>254334</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=254334</ExpertLink>
-      <Name lang="pl">Autosomalna recesywna pośrednia choroba Charcota, Mariego i Tootha typu B</Name>
+      <Name lang="pl">Autosomalna recesywna choroba Charcota, Mariego i Tootha typu B</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -129939,7 +129987,7 @@
     <Disorder id="2396">
       <OrphaCode>2617</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2617</ExpertLink>
-      <Name lang="pl">Niskorosłość ptasiogłowia typu Montreal</Name>
+      <Name lang="pl">Karłowatość pierwotna z małogłowiem typu Montreal</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -130401,7 +130449,7 @@
     <Disorder id="19879">
       <OrphaCode>261476</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=261476</ExpertLink>
-      <Name lang="pl">Zespół delecji Xp21</Name>
+      <Name lang="pl">Zespół mikrodelecji Xp21</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -135471,7 +135519,7 @@
     <Disorder id="2774">
       <OrphaCode>3098</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=3098</ExpertLink>
-      <Name lang="pl">Zespół rizomeliczny</Name>
+      <Name lang="pl">Zespół rizomeliczny, typu Urbacha</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -137743,7 +137791,7 @@
     <Disorder id="2577">
       <OrphaCode>2838</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2838</ExpertLink>
-      <Name lang="pl">Torbiel nerkowa miedniczkopochodna - głuchota</Name>
+      <Name lang="pl">Zespół uchyłków kielicha nerkowego i głuchoty</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -140675,7 +140723,7 @@
     <Disorder id="2643">
       <OrphaCode>2926</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2926</ExpertLink>
-      <Name lang="pl">Polineuropatia - wada ręki</Name>
+      <Name lang="pl">Aplazja mięśni prostowników palców - polineuropatia</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -149162,7 +149210,7 @@
     <Disorder id="20465">
       <OrphaCode>280384</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=280384</ExpertLink>
-      <Name lang="pl">Recesywna niepełnosprawność intelektualna - dysfunkcja ruchowa - mnogie przykurcze stawowe</Name>
+      <Name lang="pl">Recesywny zespół z niepełnosprawnością intelektualną, dysfunkcją ruchową i mnogimi przykurczami stawowymi</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -150942,7 +150990,7 @@
     <Disorder id="20433">
       <OrphaCode>280183</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=280183</ExpertLink>
-      <Name lang="pl">Acyduria metylomalonowa z powodu defektu receptora transkobalaminy</Name>
+      <Name lang="pl">Kwasica metylomalonowa z powodu defektu receptora transkobalaminy</Name>
       <DisorderType id="21408">
         <Name lang="pl">Wada biologiczna</Name>
       </DisorderType>
@@ -152839,7 +152887,7 @@
     <Disorder id="2891">
       <OrphaCode>3231</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=3231</ExpertLink>
-      <Name lang="pl">Zespół  dominującej głuchoty i onychodystrofii</Name>
+      <Name lang="pl">Zespół niedosłuchu i onychodystrofii</Name>
       <DisorderType id="21436">
         <Name lang="pl">Grupa fenomenów</Name>
       </DisorderType>
@@ -158625,7 +158673,7 @@
     <Disorder id="3175">
       <OrphaCode>2204</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=2204</ExpertLink>
-      <Name lang="pl">Hiperostoza korowa dysplastyczna</Name>
+      <Name lang="pl">Dysplastyczna hiperostoza korowa</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -161071,7 +161119,7 @@
     <Disorder id="18689">
       <OrphaCode>210144</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=210144</ExpertLink>
-      <Name lang="pl">Śmiertelny/Letalny zespół mnogich wad wrodzonych, typ Boissela</Name>
+      <Name lang="pl">Letalny zespół mnogich wad wrodzonych, typ Boissela</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -173465,7 +173513,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="8794">
-          <Source>23625533[PMID]_25665837[PMID]</Source>
+          <Source>PMID: 33365252 ; 33485800</Source>
           <PrevalenceType id="23697">
             <Name lang="pl">Przypadki/rodziny</Name>
           </PrevalenceType>
@@ -173473,7 +173521,7 @@
             <Name lang="pl">Przypadek</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>19.0</ValMoy>
+          <ValMoy>32.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
@@ -173482,7 +173530,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="8795">
-          <Source>ORPHANET_23625533[PMID]_25665837[PMID]</Source>
+          <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
@@ -176822,7 +176870,7 @@
     <Disorder id="18932">
       <OrphaCode>221043</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=221043</ExpertLink>
-      <Name lang="pl">Dziedziczna poikilodermia stwardniająca z zaangażowaniem ścięgien i płuc</Name>
+      <Name lang="pl">Dziedziczna poikilodermia stwardniająca z przykurczami ścięgien, miopatią i zwłóknieniem płuc</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -186601,7 +186649,7 @@
     <Disorder id="19200">
       <OrphaCode>231556</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=231556</ExpertLink>
-      <Name lang="pl">Zespół pęcherzowego oddzielania się naskórka zlokalizowanego łączącego o późnym początku i niepełnosprawności intelektualnej</Name>
+      <Name lang="pl">Zespół pęcherzowego oddzielania się naskórka o typie granicznym zlokalizowanym i późnym początku oraz niepełnosprawności intelektualnej</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -187057,7 +187105,7 @@
     <Disorder id="19213">
       <OrphaCode>231720</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=231720</ExpertLink>
-      <Name lang="pl">Nienabyty złożony Niedobór hormonów przysadki z nieprawidłowościami kręgosłupa</Name>
+      <Name lang="pl">Wrodzony złożony niedobór hormonów przysadki mózgowej z niedosłuchem czuciowo-nerwowym i nieprawidłowościami kręgosłupa</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -188802,7 +188850,7 @@
     <Disorder id="22001">
       <OrphaCode>331226</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=331226</ExpertLink>
-      <Name lang="pl">Autosomalny recesywny zespół  hiper IgE z powodu niedoboru TYK2</Name>
+      <Name lang="pl">Podatność na infekcje z powodu niedoboru TYK2</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -189754,7 +189802,7 @@
     <Disorder id="21943">
       <OrphaCode>329883</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=329883</ExpertLink>
-      <Name lang="pl">Niehipoproteinemiczna gastropatia przerostowa</Name>
+      <Name lang="pl">Gastropatia przerostowa bez hipoproteinemii</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -189933,7 +189981,7 @@
     <Disorder id="21938">
       <OrphaCode>329478</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=329478</ExpertLink>
-      <Name lang="pl">Miopatia dystalna z powodu mutacji VCP o początku w wieku dorosłym</Name>
+      <Name lang="pl">Miopatia dystalna dorosłych wskutek mutacji VCP</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -190221,7 +190269,7 @@
     <Disorder id="21926">
       <OrphaCode>329319</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=329319</ExpertLink>
-      <Name lang="pl">Nadpłytkowość dziedziczna z odwróconą wadą kończyn</Name>
+      <Name lang="pl">Nadpłytkowość z wadami dystalnymi kończyn</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -192527,7 +192575,7 @@
     <Disorder id="21789">
       <OrphaCode>324585</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=324585</ExpertLink>
-      <Name lang="pl">Autosomalna dominująca pośrednia choroba Charcota, Mariego i Tootha z bólem neuropatycznym</Name>
+      <Name lang="pl">Autosomalna dominująca choroba Charcota, Mariego i Tootha z bólem neuropatycznym</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -193588,7 +193636,7 @@
     <Disorder id="21713">
       <OrphaCode>320360</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=320360</ExpertLink>
-      <Name lang="pl">paraplegia spastyczna odziedziczona od matki</Name>
+      <Name lang="pl">Paraplegia spastyczna mitochondrialna zależna od MT-ATP6</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -195399,7 +195447,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=319671</ExpertLink>
-      <Name lang="pl">Zespół Alazamiego</Name>
+      <Name lang="pl">Pierwotna karłowatość mikrocefaliczna typu Alazamiego</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -197142,7 +197190,7 @@
     <Disorder id="21504">
       <OrphaCode>314621</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=314621</ExpertLink>
-      <Name lang="pl">Zdwojenie przysadki</Name>
+      <Name lang="pl">Zdwojenie przysadki mózgowej</Name>
       <DisorderType id="21415">
         <Name lang="pl">Wada morfologiczna</Name>
       </DisorderType>
@@ -197191,7 +197239,7 @@
     <Disorder id="21507">
       <OrphaCode>314637</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=314637</ExpertLink>
-      <Name lang="pl">Przerostowa kardiomiopatia mitochondrialna z kwasicą mleczanową z powodu niedoboru MTO1</Name>
+      <Name lang="pl">Mitochondrialna kardiomiopatia przerostowa z kwasicą mleczanową spowodowana niedoborem  MTO1</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -198517,7 +198565,7 @@
     <Disorder id="22464">
       <OrphaCode>370015</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=370015</ExpertLink>
-      <Name lang="pl">Dysplazja kręgowo-nasadowo-przynasadowa, typu Isidora</Name>
+      <Name lang="pl">Dysplazja kręgowo-nasadowo-przynasadowa typu Isidora</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -198566,7 +198614,7 @@
     <Disorder id="22465">
       <OrphaCode>370019</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=370019</ExpertLink>
-      <Name lang="pl">Dysplazja kręgowo-nasadowa, typu Czarny-Ratajczak</Name>
+      <Name lang="pl">Dysplazja kręgowo-nasadowa typu Czarny-Ratajczak</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -198615,7 +198663,7 @@
     <Disorder id="22479">
       <OrphaCode>370088</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=370088</ExpertLink>
-      <Name lang="pl">Zespół ostra dziecięca niewydolność wątroby-zaburzenia wieloukładowe</Name>
+      <Name lang="pl">Zespół ostrej dziecięcej niewydolności wątroby i zaburzeń wieloukładowych</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -199382,7 +199430,7 @@
     <Disorder id="22455">
       <OrphaCode>369970</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=369970</ExpertLink>
-      <Name lang="pl">Zespół mała rogówka-miopiczna atrofia nazyniówki i siatkówki-telekantus</Name>
+      <Name lang="pl">Zespół mała rogówka - atrofia naczyniówki i siatkówki z krótkowzrocznością - telecanthus</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -199610,7 +199658,7 @@
     <Disorder id="22460">
       <OrphaCode>369999</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=369999</ExpertLink>
-      <Name lang="pl">Rozlany rogowiec dłoniowo-podeszwowy z bolesnymi szczelinami</Name>
+      <Name lang="pl">Rozlany rogowiec dłoniowo-podeszwowy z bolesnymi pęknięciami</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -199691,7 +199739,7 @@
     <Disorder id="22433">
       <OrphaCode>369837</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=369837</ExpertLink>
-      <Name lang="pl">Zespół niepełnosprawności intelektualnej-drgawek-hipofosfatazji-wad oka i układu szkieletowego</Name>
+      <Name lang="pl">Zespół niepełnosprawność intelektualna-napady padaczkowe-hipotonia-wady oczu-wady kostne</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -199838,7 +199886,7 @@
     <Disorder id="22437">
       <OrphaCode>369852</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=369852</ExpertLink>
-      <Name lang="pl">Zespół nawracajace infekcje-mielofibroza-nefromegalia</Name>
+      <Name lang="pl">Zespół wrodzonej neutropenii, zwłóknienia szpiku i nefromegalii</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -202569,7 +202617,7 @@
     <Disorder id="22334">
       <OrphaCode>363665</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=363665</ExpertLink>
-      <Name lang="pl">Zespół osteoliza kończyn-zmiany przypominające keloid-przedwczesne starzenie</Name>
+      <Name lang="pl">Zespół osteolizy końcowych paliczków, zmian przypominających keloid i przedwczesnego starzenia</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -204056,7 +204104,7 @@
     <Disorder id="22212">
       <OrphaCode>357175</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=357175</ExpertLink>
-      <Name lang="pl">Krótka kość łokciowa - dysmorfizm - hipotonia - niepełnosprawność intelektualna</Name>
+      <Name lang="pl">Zespół krótka kość łokciowa - dysmorfia - hipotonia - niepełnosprawność intelektualna</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -204203,7 +204251,7 @@
     <Disorder id="22219">
       <OrphaCode>357237</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=357237</ExpertLink>
-      <Name lang="pl">Ciężki złożony Niedobór odporności z powodu deficytu CARD11</Name>
+      <Name lang="pl">Ciężki złożony niedobór odporności z powodu deficytu CARD11</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -207041,7 +207089,7 @@
     <Disorder id="20897">
       <OrphaCode>293967</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=293967</ExpertLink>
-      <Name lang="pl">Zespół hipogonadyzm hipogonadotropowy-znaczna mikrocefalia-głuchota zmysłowo-nerwowa-dysmorfizm</Name>
+      <Name lang="pl">Zespół hipogonadyzmu hipogonadotropowego, znacznego małogłowia, niedosłuchu zmysłowo-nerwowego i dysmorfii</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -207433,7 +207481,7 @@
     <Disorder id="20883">
       <OrphaCode>293864</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=293864</ExpertLink>
-      <Name lang="pl">Zespół hipoplastyczna trzustka-atrezja jelit-hipoplastyczny pęcherzyk żółciowy</Name>
+      <Name lang="pl">Zespół niedorozwoju trzustki, pęcherzyka żółciowego i atrezji jelit</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -212000,7 +212048,7 @@
     <Disorder id="20480">
       <OrphaCode>280598</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=280598</ExpertLink>
-      <Name lang="pl">Dziedziczna neuropatia czuciowo-ruchowa z hiperelastycznością skóry</Name>
+      <Name lang="pl">Dziedziczna neuropatia czuciowo-ruchowa z nadmiernie elastyczną skórą</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -214250,7 +214298,7 @@
     <Disorder id="21449">
       <OrphaCode>313772</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=313772</ExpertLink>
-      <Name lang="pl">Zespół ataksja spastyczna o wczesnym początku-neuropatia</Name>
+      <Name lang="pl">Zespół ataksja spastyczna o wczesnym początku – padaczka miokloniczna - neuropatia</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -217617,7 +217665,7 @@
     <Disorder id="21121">
       <OrphaCode>300501</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=300501</ExpertLink>
-      <Name lang="pl">Zespół bolących nerwiakowłókniaków oczodołu i uogólnionych - sylwetka marfanoidalna</Name>
+      <Name lang="pl">Zespół bolesnych nerwiakowłókniaków oczodołu i układowych z sylwetką  marfanoidalną</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -218953,7 +219001,7 @@
     <Disorder id="21232">
       <OrphaCode>306558</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=306558</ExpertLink>
-      <Name lang="pl">Zespół małogłowie pierwotne-padaczka-trwała cukrzyca noworodków</Name>
+      <Name lang="pl">Zespół małogłowie pierwotne - padaczka - utrwalona cukrzyca noworodków</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -219181,7 +219229,7 @@
     <Disorder id="21109">
       <OrphaCode>300313</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=300313</ExpertLink>
-      <Name lang="pl">Zespół zaćma wrodzona-utrata słuchu-znaczne opóźnienie rozwoju</Name>
+      <Name lang="pl">Zespół zaćma wrodzona - niedosłuch - ciężkie opóźnienie rozwoju</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -228703,7 +228751,7 @@
     <Disorder id="22889">
       <OrphaCode>402823</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=402823</ExpertLink>
-      <Name lang="pl">Zapalenie wątroby delta</Name>
+      <Name lang="pl">Zapalenie wątroby typu delta</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -229411,7 +229459,7 @@
     <Disorder id="22859">
       <OrphaCode>401959</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=401959</ExpertLink>
-      <Name lang="pl">Zespół częściowa agnezja ciała modzelowatego-hipoplazja robaka móżdżku z torbielami tylnego dołu czaszki</Name>
+      <Name lang="pl">Zespół częściowej agenezji ciała modzelowatego, hipoplazji robaka móżdżku z torbielami tylnego dołu czaszki</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -230129,7 +230177,7 @@
     <Disorder id="22927">
       <OrphaCode>404473</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=404473</ExpertLink>
-      <Name lang="pl">Zespół znaczna niepełnosprawność intelektualna-postępująca diplegia spastyczna</Name>
+      <Name lang="pl">Zespół ciężkiej niepełnosprawności intelektualnej i postępującej diplegii spastycznej</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -230276,7 +230324,7 @@
     <Disorder id="22922">
       <OrphaCode>404451</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=404451</ExpertLink>
-      <Name lang="pl">Zespół opóźnienie  rozwoju związane z FBLN1-wada centralnego układu nerwowego-syndaktylia</Name>
+      <Name lang="pl">Zespół opóźnienie rozwoju związane z FBLN1 - wady ośrodkowego układu nerwowego - syndaktylia</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -230455,7 +230503,7 @@
     <Disorder id="22918">
       <OrphaCode>404437</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=404437</ExpertLink>
-      <Name lang="pl">Zespół rozlany zanik mózgu i móżdżku-nieuleczalne napady padaczkowe-postępująca mikrocefalia</Name>
+      <Name lang="pl">Zespół rozlanego zaniku mózgu i móżdżku, lekoopornych napadów padaczkowych i postępującego małogłowia</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -230879,7 +230927,7 @@
     <Disorder id="22928">
       <OrphaCode>404476</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=404476</ExpertLink>
-      <Name lang="pl">Zespół całościowe opóźnienie w rozwoju-torbiele płucne-przerost-guz Wilmsa</Name>
+      <Name lang="pl">Zespół całościowego opóźnienia w rozwoju, torbieli płucnych, przerostu ciała i guza Wilmsa</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -231450,7 +231498,7 @@
     <Disorder id="23038">
       <OrphaCode>412022</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=412022</ExpertLink>
-      <Name lang="pl">Zespół dysmorfia twarzy-przemieszczenie soczewki-zaburzenia przedniego segmentu-spontaniczne pęcherzyki filtracyjne</Name>
+      <Name lang="pl">Zespół dysmorfia twarzy-przemieszczenie soczewki-zaburzenia przedniego segmentu-spontaniczne pęcherzyki filtracyjne spojówek</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -233215,7 +233263,7 @@
     <Disorder id="22606">
       <OrphaCode>391330</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=391330</ExpertLink>
-      <Name lang="pl">Osteoporoza sprzężona z chromosomem X ze złamaniami</Name>
+      <Name lang="pl">Osteoporoza ze złamaniami sprzężona z chromosomem X</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -234209,7 +234257,7 @@
     <Disorder id="22715">
       <OrphaCode>397973</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397973</ExpertLink>
-      <Name lang="pl">Zespół niepełnosprawność intelektualna-otyłość-prognatyzm-anomalie oczu i skóry</Name>
+      <Name lang="pl">Zespół niepełnosprawność intelektualna - otyłość - prognatyzm - wady oczu i skóry</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -234307,7 +234355,7 @@
     <Disorder id="22713">
       <OrphaCode>397964</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397964</ExpertLink>
-      <Name lang="pl">Złożony Niedobór odporności z powodu deficytu MALT1</Name>
+      <Name lang="pl">Złożony niedobór odporności z powodu deficytu MALT1</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -234552,7 +234600,7 @@
     <Disorder id="22708">
       <OrphaCode>397937</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397937</ExpertLink>
-      <Name lang="pl">Miopatia z ciałami poliglukozanowymi body myopathy</Name>
+      <Name lang="pl">Miopatia z ciałkami poliglukozanowymi typu 1</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -234601,7 +234649,7 @@
     <Disorder id="22707">
       <OrphaCode>397933</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397933</ExpertLink>
-      <Name lang="pl">Zespół  znaczna niepełnosprawność intelektualna-postępujące małogłowie postnatalne-pośrodkowe stereotypowe ruchy ręki</Name>
+      <Name lang="pl">Zespół ciężkiej niepełnosprawności intelektualnej, postępującego małogłowia postnatalnego i stereotypowych ruchów dłoni w linii pośrodkowej ciała</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -234797,7 +234845,7 @@
     <Disorder id="22700">
       <OrphaCode>397755</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397755</ExpertLink>
-      <Name lang="pl">Porażenie okresowe z przejściowym zespołem podobnym do zespołu ciasnoty</Name>
+      <Name lang="pl">Porażenie okresowe z przemijającym zespołem podobnym do zespołu ciasnoty</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -235270,7 +235318,7 @@
     <Disorder id="22690">
       <OrphaCode>397623</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397623</ExpertLink>
-      <Name lang="pl">Zespół niski wzrost-atrezja kanału słuchowego-hipoplazja żuchwy-anomalie szkieletowe</Name>
+      <Name lang="pl">Zespół niski wzrost - atrezja kanału słuchowego - hipoplazja żuchwy - wady układu kostnego</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -235400,7 +235448,7 @@
     <Disorder id="22689">
       <OrphaCode>397618</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=397618</ExpertLink>
-      <Name lang="pl">Zespół hipoplazja dołka-defekt skrzyżowania nerwów wzrokowych-dysgenezja segmentu przedniego</Name>
+      <Name lang="pl">Zespół hipoplazja dołka-defekt skrzyżowania nerwów wzrokowych-dysgenezja segmentu przedniego oka</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -243370,7 +243418,7 @@
     <Disorder id="23042">
       <OrphaCode>412069</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=412069</ExpertLink>
-      <Name lang="pl">Zespół niepełnosprawność intelektualna-obturacyjny bezdech senny-łagodna dysmorfia spowodowana mutacjami AHDC1</Name>
+      <Name lang="pl">Zespół niepełnosprawność intelektualna-obturacyjny bezdech senny-łagodna dysmorfia spowodowany mutacjami AHDC1</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -253221,7 +253269,7 @@
     <Disorder id="23235">
       <OrphaCode>431166</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=431166</ExpertLink>
-      <Name lang="pl">Pierwotny niedobór odporności z rozsianym szczepem odry szczepionkowej</Name>
+      <Name lang="pl">Pierwotny niedobór odporności z infekcją wirusową po szczepionce przeciw śwince-odrze-różyczce (MMR)</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -269019,7 +269067,7 @@
     <Disorder id="25232">
       <OrphaCode>481152</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=481152</ExpertLink>
-      <Name lang="pl">Małogłowie - leukoencefalopatia postępująca zależne od PYCR2</Name>
+      <Name lang="pl">Małogłowie - postępująca leukoencefalopatia zależne od PYCR2</Name>
       <DisorderType id="21401">
         <Name lang="pl">Zespół wad wrodzonych</Name>
       </DisorderType>
@@ -282029,7 +282077,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="13938">
-          <Source>DOI: https://doi.org/10.1016/j.ymgme.2014.12.082[OTHER]_28595941[PMID]</Source>
+          <Source>PMID: 31926052</Source>
           <PrevalenceType id="23683">
             <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
@@ -282039,7 +282087,7 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.04</ValMoy>
+          <ValMoy>1.25</ValMoy>
           <PrevalenceGeographic id="24068">
             <Name lang="pl">BRAZYLIA</Name>
           </PrevalenceGeographic>
@@ -301198,7 +301246,7 @@
     <Disorder id="10593">
       <OrphaCode>46135</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=46135</ExpertLink>
-      <Name lang="pl">Pierwotny chłoniak centralnego układu nerwowego</Name>
+      <Name lang="pl">Pierwotny chłoniak ośrodkowego układu nerwowego</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
@@ -303999,29 +304047,29 @@
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
       <PrevalenceList count="3">
-        <Prevalence id="12928">
-          <Source>European Medicines Agency 2014[INST]</Source>
+        <Prevalence id="16645">
+          <Source>PMID: 26543054</Source>
           <PrevalenceType id="23669">
             <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="pl">Wartość i klasa</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23739">
-            <Name lang="pl">1-9 / 1 000 000</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.21</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pl">Europa</Name>
+          <ValMoy>1.89</ValMoy>
+          <PrevalenceGeographic id="25006">
+            <Name lang="pl">NORWEGIA</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="12929">
-          <Source>19095780[PMID]</Source>
-          <PrevalenceType id="23683">
-            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+        <Prevalence id="16646">
+          <Source>PMID: 31730177</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="pl">Wartość i klasa</Name>
@@ -304029,16 +304077,16 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>1.95</ValMoy>
-          <PrevalenceGeographic id="24264">
-            <Name lang="pl">DANIA</Name>
+          <ValMoy>1.54</ValMoy>
+          <PrevalenceGeographic id="25475">
+            <Name lang="pl">ZJEDNOCZONE KRÓLESTWO</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
-        <Prevalence id="13994">
-          <Source>19095780[PMID]</Source>
+        <Prevalence id="12928">
+          <Source>PMID: 26543054 ; 31730177</Source>
           <PrevalenceType id="23669">
             <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
@@ -304048,9 +304096,9 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>2.4</ValMoy>
-          <PrevalenceGeographic id="24264">
-            <Name lang="pl">DANIA</Name>
+          <ValMoy>1.66</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="pl">Europa</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
@@ -305044,11 +305092,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=90308</ExpertLink>
       <Name lang="pl">Zespół Klippla i Trénaunaya</Name>
-      <DisorderType id="21450">
-        <Name lang="pl">Podtyp kliniczny</Name>
+      <DisorderType id="21394">
+        <Name lang="pl">Choroba</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="pl">Podtyp zaburzenia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="7026">
@@ -313184,38 +313232,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="12110">
-      <OrphaCode>91133</OrphaCode>
-      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=91133</ExpertLink>
-      <Name lang="pl">Osteopenia - miopia - utrata słuchu - niepełnosprawność  intelektualna - dysmorfia twarzy</Name>
-      <DisorderType id="21401">
-        <Name lang="pl">Zespół wad wrodzonych</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pl">Zaburzenie</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="7086">
-          <Source>ORPHANET_17230486[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pl">Częstość występowania punktowa</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pl">Tylko klasa</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="pl">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pl">Cały świat</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pl">Zwalidowany</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="12096">
       <OrphaCode>90793</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=90793</ExpertLink>
@@ -336768,6 +336784,55 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=599373</ExpertLink>
+      <Name lang="pl">Encefalopatia związana z STXBP1</Name>
+      <DisorderType id="21394">
+        <Name lang="pl">Choroba</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16637">
+          <Source>PMID: 32643187</Source>
+          <PrevalenceType id="23697">
+            <Name lang="pl">Przypadki/rodziny</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="pl">Przypadek</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>282.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16638">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=93403</ExpertLink>
@@ -340425,6 +340490,104 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=592564</ExpertLink>
+      <Name lang="pl">Spektrum zaburzeń związanych z GNAO1 obejmujących opóźnienie rozwoju, drgawki i ruchy mimowolne</Name>
+      <DisorderType id="21394">
+        <Name lang="pl">Choroba</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16643">
+          <Source>PMID: 26060304 ; 29758257 ; 30682224 ; 34122306</Source>
+          <PrevalenceType id="23697">
+            <Name lang="pl">Przypadki/rodziny</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="pl">Przypadek</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>75.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16644">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="pl">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=592570</ExpertLink>
+      <Name lang="pl">Zespół zaburzeń związanych z TRAF7</Name>
+      <DisorderType id="21401">
+        <Name lang="pl">Zespół wad wrodzonych</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16641">
+          <Source>PMID: 29961569 ;  32376980 ;  32459067 ;  34088006</Source>
+          <PrevalenceType id="23697">
+            <Name lang="pl">Przypadki/rodziny</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="pl">Przypadek</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>55.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16642">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23760">
+            <Name lang="pl">&lt;1 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="14245">
       <OrphaCode>99672</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=99672</ExpertLink>
@@ -340589,6 +340752,297 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589821</ExpertLink>
+      <Name lang="pl">Wrodzona dystrofia miotoniczna Steinerta</Name>
+      <DisorderType id="21450">
+        <Name lang="pl">Podtyp kliniczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="4">
+        <Prevalence id="16621">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16622">
+          <Source>PMID: 23415617</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="pl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.1</ValMoy>
+          <PrevalenceGeographic id="24124">
+            <Name lang="pl">KANADA</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16627">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="pl">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.94</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="pl">WŁOCHY</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16628">
+          <Source>PMID: 33472919</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="pl">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>47.6</ValMoy>
+          <PrevalenceGeographic id="25482">
+            <Name lang="pl">STANY ZJEDNOCZONE AMERYKI</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589827</ExpertLink>
+      <Name lang="pl">Dystrofia miotoniczna Steinerta młodocianych</Name>
+      <DisorderType id="21450">
+        <Name lang="pl">Podtyp kliniczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16631">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16632">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="pl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>1.64</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="pl">WŁOCHY</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589824</ExpertLink>
+      <Name lang="pl">Dystrofia miotoniczna Steinerta o początku w dzieciństwie</Name>
+      <DisorderType id="21450">
+        <Name lang="pl">Podtyp kliniczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16629">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16630">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>1.29</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="pl">WŁOCHY</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589833</ExpertLink>
+      <Name lang="pl">Miotonia dystroficzna Steinerta o późnym początku</Name>
+      <DisorderType id="21450">
+        <Name lang="pl">Podtyp kliniczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16635">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16636">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="pl">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>8.14</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="pl">WŁOCHY</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589830</ExpertLink>
+      <Name lang="pl">Dystrofia miotoniczna Steinerta wieku dorosłego</Name>
+      <DisorderType id="21450">
+        <Name lang="pl">Podtyp kliniczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <PrevalenceList count="2">
+        <Prevalence id="16633">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pl">Tylko klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pl">nieznany</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16634">
+          <Source>PMID: 26882032</Source>
+          <PrevalenceType id="23669">
+            <Name lang="pl">Częstość występowania punktowa</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pl">Wartość i klasa</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23732">
+            <Name lang="pl">1-5 / 10 000</Name>
+          </PrevalenceClass>
+          <ValMoy>12.54</ValMoy>
+          <PrevalenceGeographic id="24607">
+            <Name lang="pl">WŁOCHY</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pl">Zwalidowany</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+      </PrevalenceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589905</ExpertLink>
@@ -348072,10 +348526,10 @@
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
       <PrevalenceList count="15">
-        <Prevalence id="7708">
-          <Source>European Medicines Agency 2018[INST]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pl">Częstość występowania punktowa</Name>
+        <Prevalence id="16647">
+          <Source>PMID: 32503598</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pl">Częstość występowania przy narodzinach</Name>
           </PrevalenceType>
           <PrevalenceQualification id="23711">
             <Name lang="pl">Wartość i klasa</Name>
@@ -348083,9 +348537,9 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pl">Europa</Name>
+          <ValMoy>9.9</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pl">Zwalidowany</Name>
@@ -348244,7 +348698,7 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="11209">
-          <Source>24780148[PMID]</Source>
+          <Source>PMID: 32503598</Source>
           <PrevalenceType id="23669">
             <Name lang="pl">Częstość występowania punktowa</Name>
           </PrevalenceType>
@@ -348254,7 +348708,7 @@
           <PrevalenceClass id="23746">
             <Name lang="pl">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>4.78</ValMoy>
+          <ValMoy>2.8</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="pl">Cały świat</Name>
           </PrevalenceGeographic>
@@ -359726,7 +360180,7 @@
     <Disorder id="14619">
       <OrphaCode>100046</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=100046</ExpertLink>
-      <Name lang="pl">Autosomalna dominująca pośrednia choroba Charcota, Mariego i Tootha typu D</Name>
+      <Name lang="pl">Autosomalna dominująca choroba Charcota, Mariego i Tootha typu D</Name>
       <DisorderType id="21394">
         <Name lang="pl">Choroba</Name>
       </DisorderType>
