commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Epidemiological data/Rare disease epidemiology/pt_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/pt_product9_prev.xml
index c807def..ef9b0d9 100644
--- a/Epidemiological data/Rare disease epidemiology/pt_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/pt_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 05:02:15" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:59:20" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6063">
+  <DisorderList count="6058">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -8133,16 +8133,16 @@
           <PrevalenceClass id="23739">
             <Name lang="pt">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.22</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pt">Europa</Name>
+          <ValMoy>6.66</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pt">Mundial</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="pt">Not yet validated</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pt">Validated</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="293">
-          <Source>9918480[PMID]</Source>
+          <Source>35024668[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="pt">Prevalência ao nascimento</Name>
           </PrevalenceType>
@@ -8152,7 +8152,7 @@
           <PrevalenceClass id="23739">
             <Name lang="pt">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.85</ValMoy>
+          <ValMoy>6.96</ValMoy>
           <PrevalenceGeographic id="23949">
             <Name lang="pt">Australia</Name>
           </PrevalenceGeographic>
@@ -21347,7 +21347,7 @@
     <Disorder id="225">
       <OrphaCode>912</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=912</ExpertLink>
-      <Name lang="pt">Síndrome de Zellweger</Name>
+      <Name lang="pt">Síndrome Zellweger</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -46243,7 +46243,7 @@
     <Disorder id="17996">
       <OrphaCode>177926</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=177926</ExpertLink>
-      <Name lang="pt">Forma sintomática de Hemofilia A em portadoras</Name>
+      <Name lang="pt">Doença hemorrágica em portadores de hemofilia A</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -46782,7 +46782,7 @@
     <Disorder id="17997">
       <OrphaCode>177929</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=177929</ExpertLink>
-      <Name lang="pt">Forma sintomática de Hemofilia B em portadoras</Name>
+      <Name lang="pt">Doença hemorrágica em portadores de hemofilia B</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -74189,7 +74189,7 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="2539">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -74198,12 +74198,50 @@
           <PrevalenceQualification id="23704">
             <Name lang="pt">Class only</Name>
           </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pt">Desconhecido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pt">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pt">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16781">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pt">Prevalência ao nascimento</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="pt">Class only</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="pt">Desconhecido</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="pt">Mundial</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="pt">Validated</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16782">
+          <Source>20734338[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="pt">Prevalência ao nascimento</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="pt">Value and class</Name>
+          </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="pt">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pt">Europa</Name>
+          <ValMoy>1.75</ValMoy>
+          <PrevalenceGeographic id="23949">
+            <Name lang="pt">Australia</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="pt">Validated</Name>
@@ -79378,7 +79416,7 @@
     <Disorder id="1045">
       <OrphaCode>358</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=358</ExpertLink>
-      <Name lang="pt">Síndrome de Gitelman</Name>
+      <Name lang="pt">Síndrome Gitelman</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -82623,38 +82661,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=946</ExpertLink>
-      <Name lang="pt">Acrocefalossindactilia</Name>
-      <DisorderType id="21436">
-        <Name lang="pt">Grupo de fenótipos</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="pt">Grupo de patologias</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="2783">
-          <Source>ORPHANET_[EXPERT]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pt">Prevalência pontual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pt">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="pt">Desconhecido</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pt">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=957</ExpertLink>
@@ -86517,7 +86523,7 @@
     <Disorder id="16886">
       <OrphaCode>139402</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=139402</ExpertLink>
-      <Name lang="pt">Síndrome DRESS</Name>
+      <Name lang="pt">Reação medicamentosa com eosinofilia e sintomas sistémicos</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -99304,55 +99310,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1533</ExpertLink>
-      <Name lang="pt">Craniossinostose - aplasia peronial</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="3180">
-          <Source>4403018[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="pt">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="pt">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pt">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="3181">
-          <Source>ORPHANET_4403018[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pt">Prevalência pontual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pt">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="pt">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pt">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=140874</ExpertLink>
@@ -160231,7 +160188,7 @@
     <Disorder id="18667">
       <OrphaCode>209908</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=209908</ExpertLink>
-      <Name lang="pt">Dificuldades de linguagem tipo 1</Name>
+      <Name lang="pt">Apraxia do discurso isolada na adolescência</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -190778,55 +190735,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=329341</ExpertLink>
-      <Name lang="pt">Encefalite límbica com anticorpos DPP6</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="10180">
-          <Source>23225603[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="pt">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="pt">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pt">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10181">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pt">Prevalência pontual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pt">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="pt">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pt">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=329228</ExpertLink>
@@ -200462,7 +200370,7 @@
     <Disorder id="5016">
       <OrphaCode>772</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=772</ExpertLink>
-      <Name lang="pt">Doença de Refsum do lactente</Name>
+      <Name lang="pt">Doença Refsum infantil</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -265668,55 +265576,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=498700</ExpertLink>
-      <Name lang="pt">Encefalite límbica com anticorpos neurexina-3</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13782">
-          <Source>27170573[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="pt">Cases/families</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="pt">Case</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pt">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13783">
-          <Source>ORPHANET_27170573[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pt">Prevalência pontual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pt">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="pt">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="pt">Mundial</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="pt">Validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=874</ExpertLink>
@@ -268006,7 +267865,7 @@
     <Disorder id="8614">
       <OrphaCode>838</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=838</ExpertLink>
-      <Name lang="pt">Síndrome de Susac</Name>
+      <Name lang="pt">Síndrome Susac</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -279297,38 +279156,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=63440</ExpertLink>
-      <Name lang="pt">Oxicefalia</Name>
-      <DisorderType id="21415">
-        <Name lang="pt">Anomalia morfológica</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="5872">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="pt">Prevalência pontual</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="pt">Class only</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="pt">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="pt">Europa</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="pt">Not yet validated</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=63442</ExpertLink>
@@ -282741,7 +282568,7 @@
     <Disorder id="11215">
       <OrphaCode>79189</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=79189</ExpertLink>
-      <Name lang="pt">Doença da biogénese do peroxissoma</Name>
+      <Name lang="pt">Doença da biogénese dos peroxissomas</Name>
       <DisorderType id="21436">
         <Name lang="pt">Grupo de fenótipos</Name>
       </DisorderType>
@@ -285185,7 +285012,7 @@
     <Disorder id="11106">
       <OrphaCode>77293</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=77293</ExpertLink>
-      <Name lang="pt">Doença de Niemann-Pick tipo B</Name>
+      <Name lang="pt">Deficiência de esfingomielinase ácida visceral crónica</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -285298,7 +285125,7 @@
     <Disorder id="11105">
       <OrphaCode>77292</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=77292</ExpertLink>
-      <Name lang="pt">Doença de Niemann-Pick tipo A</Name>
+      <Name lang="pt">Deficiência de esfingomielinase ácida neurovisceral infantil</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -304185,7 +304012,7 @@
     <Disorder id="11918">
       <OrphaCode>90000</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=90000</ExpertLink>
-      <Name lang="pt">Eritema elevatum diutinum</Name>
+      <Name lang="pt">Eritema elevado diutino</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -306454,7 +306281,7 @@
     <Disorder id="11818">
       <OrphaCode>88642</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=88642</ExpertLink>
-      <Name lang="pt">Indiferença congénita à dor</Name>
+      <Name lang="pt">Insensibilidade congénita à dor-anosmia-artropatia neuropática</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -327112,7 +326939,7 @@
     <Disorder id="11577">
       <OrphaCode>83467</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=83467</ExpertLink>
-      <Name lang="pt">Síndrome de Morvan</Name>
+      <Name lang="pt">Síndrome Morvan</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -329723,7 +329550,7 @@
     <Disorder id="11619">
       <OrphaCode>84142</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=84142</ExpertLink>
-      <Name lang="pt">Síndrome de Isaac</Name>
+      <Name lang="pt">Síndrome Isaac</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
