commit 73e5239676f0390ef5e7117e4a353a4b9ec5030d
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 7 11:31:35 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare disease epidemiology/ru_product9_prev.xml b/Epidemiological data/Rare disease epidemiology/ru_product9_prev.xml
index 369e9df..c29e4da 100644
--- a/Epidemiological data/Rare disease epidemiology/ru_product9_prev.xml	
+++ b/Epidemiological data/Rare disease epidemiology/ru_product9_prev.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<JDBOR date="2022-04-01 05:00:38" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-06-07 04:41:28" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6063">
+  <DisorderList count="6056">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=166024</ExpertLink>
@@ -1731,7 +1731,7 @@
       </DisorderGroup>
       <PrevalenceList count="13">
         <Prevalence id="8340">
-          <Source>PMID: 30306084 &amp; ORPHANET</Source>
+          <Source>30306084[PMID]_33780098[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="ru">Точечное распространение</Name>
           </PrevalenceType>
@@ -1741,7 +1741,7 @@
           <PrevalenceClass id="23739">
             <Name lang="ru">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.2038</ValMoy>
+          <ValMoy>0.1865</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="ru">Весь мир</Name>
           </PrevalenceGeographic>
@@ -8133,16 +8133,16 @@
           <PrevalenceClass id="23739">
             <Name lang="ru">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.22</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="ru">Европа</Name>
+          <ValMoy>6.66</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="293">
-          <Source>9918480[PMID]</Source>
+          <Source>35024668[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="ru">Распространенность при рождении</Name>
           </PrevalenceType>
@@ -8152,7 +8152,7 @@
           <PrevalenceClass id="23739">
             <Name lang="ru">1-9 / 1 000 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.85</ValMoy>
+          <ValMoy>6.96</ValMoy>
           <PrevalenceGeographic id="23949">
             <Name lang="ru">Австралия</Name>
           </PrevalenceGeographic>
@@ -20974,22 +20974,22 @@
           </PrevalenceValidationStatus>
         </Prevalence>
         <Prevalence id="787">
-          <Source>ORPHANET</Source>
+          <Source>7468659[PMID]_ORPHANET</Source>
           <PrevalenceType id="23669">
             <Name lang="ru">Точечное распространение</Name>
           </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
+          <PrevalenceQualification id="23711">
+            <Name lang="ru">Значение и класс</Name>
           </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="ru">неизвестно</Name>
+          <PrevalenceClass id="23746">
+            <Name lang="ru">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
+          <ValMoy>2.5</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="ru">Весь мир</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -74189,7 +74189,7 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="2539">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -74198,12 +74198,50 @@
           <PrevalenceQualification id="23704">
             <Name lang="ru">Только класс</Name>
           </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16781">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23683">
+            <Name lang="ru">Распространенность при рождении</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16782">
+          <Source>20734338[PMID]</Source>
+          <PrevalenceType id="23683">
+            <Name lang="ru">Распространенность при рождении</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="ru">Значение и класс</Name>
+          </PrevalenceQualification>
           <PrevalenceClass id="23746">
             <Name lang="ru">1-9 / 100 000</Name>
           </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="ru">Европа</Name>
+          <ValMoy>1.75</ValMoy>
+          <PrevalenceGeographic id="23949">
+            <Name lang="ru">Австралия</Name>
           </PrevalenceGeographic>
           <PrevalenceValidationStatus id="25958">
             <Name lang="ru">Подтверждено</Name>
@@ -78034,7 +78072,7 @@
       <DisorderGroup id="36540">
         <Name lang="ru">Группа заболеваний</Name>
       </DisorderGroup>
-      <PrevalenceList count="6">
+      <PrevalenceList count="7">
         <Prevalence id="16623">
           <Source>PMID: 26431509</Source>
           <PrevalenceType id="23683">
@@ -78149,6 +78187,25 @@
             <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16784">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="ru">Точечное распространение</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="1030">
@@ -82623,38 +82680,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=946</ExpertLink>
-      <Name lang="ru">Акроцефалосиндактилия</Name>
-      <DisorderType id="21436">
-        <Name lang="ru">Клиническая группа</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="ru">Группа заболеваний</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="2783">
-          <Source>ORPHANET_[EXPERT]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="ru">неизвестно</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="ru">Европа</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=957</ExpertLink>
@@ -99304,55 +99329,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=1533</ExpertLink>
-      <Name lang="ru">Синдром краниосиностоза-аплазии малоберцовой коти</Name>
-      <DisorderType id="21401">
-        <Name lang="ru">Синдром врожденных пороков развития</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="3180">
-          <Source>4403018[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="ru">Случаи/семьи</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="ru">Случай</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>2.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="3181">
-          <Source>ORPHANET_4403018[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="ru">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=140874</ExpertLink>
@@ -126196,7 +126172,7 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="9213">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -126212,8 +126188,44 @@
           <PrevalenceGeographic id="23844">
             <Name lang="ru">Весь мир</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16792">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="ru">Ежегодные показатели заболеваемости</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23795">
+            <Name lang="ru">Европа</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16793">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23690">
+            <Name lang="ru">Распространённость в течение всего срока жизни</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -140154,55 +140166,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="2662">
-      <OrphaCode>2950</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=2950</ExpertLink>
-      <Name lang="ru">Синдром трехфалангового большого пальца-полисиндактилии</Name>
-      <DisorderType id="21401">
-        <Name lang="ru">Синдром врожденных пороков развития</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="3965">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23697">
-            <Name lang="ru">Случаи/семьи</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23725">
-            <Name lang="ru">Семья</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>15.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="3966">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="ru">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="2663">
       <OrphaCode>2951</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=2951</ExpertLink>
@@ -144986,38 +144949,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="2956">
-      <OrphaCode>3332</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=3332</ExpertLink>
-      <Name lang="ru">Синдром гипоплазии большеберцовой кости-постаксиальной полидактилии</Name>
-      <DisorderType id="21401">
-        <Name lang="ru">Синдром врожденных пороков развития</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="4158">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23774">
-            <Name lang="ru">неизвестно</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="2954">
       <OrphaCode>3329</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=3329</ExpertLink>
@@ -171877,7 +171808,7 @@
             <Name lang="ru">Случай</Name>
           </PrevalenceQualification>
           <PrevalenceClass/>
-          <ValMoy>21.0</ValMoy>
+          <ValMoy>35.0</ValMoy>
           <PrevalenceGeographic id="23844">
             <Name lang="ru">Весь мир</Name>
           </PrevalenceGeographic>
@@ -175454,7 +175385,7 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="2">
         <Prevalence id="4476">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -175470,8 +175401,25 @@
           <PrevalenceGeographic id="23795">
             <Name lang="ru">Европа</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16783">
+          <Source>33036647[PMID]_33231058[PMID]</Source>
+          <PrevalenceType id="23697">
+            <Name lang="ru">Случаи/семьи</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23718">
+            <Name lang="ru">Случай</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass/>
+          <ValMoy>99.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -190778,55 +190726,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=329341</ExpertLink>
-      <Name lang="ru">Лимбический энцефалит с антителами к DPP6</Name>
-      <DisorderType id="21394">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="10180">
-          <Source>23225603[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="ru">Случаи/семьи</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="ru">Случай</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>4.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="10181">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="ru">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=329228</ExpertLink>
@@ -265668,55 +265567,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=498700</ExpertLink>
-      <Name lang="ru">Лимбический энцефалит с антителами к нейрексину-3</Name>
-      <DisorderType id="21394">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderGroup>
-      <PrevalenceList count="2">
-        <Prevalence id="13782">
-          <Source>27170573[PMID]</Source>
-          <PrevalenceType id="23697">
-            <Name lang="ru">Случаи/семьи</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23718">
-            <Name lang="ru">Случай</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass/>
-          <ValMoy>5.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-        <Prevalence id="13783">
-          <Source>ORPHANET_27170573[PMID]</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="ru">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23844">
-            <Name lang="ru">Весь мир</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25958">
-            <Name lang="ru">Подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=874</ExpertLink>
@@ -279297,38 +279147,6 @@
         </Prevalence>
       </PrevalenceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=63440</ExpertLink>
-      <Name lang="ru">Изолированная оксицефалия</Name>
-      <DisorderType id="21415">
-        <Name lang="ru">Морфологическая аномалия</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="ru">Заболевание</Name>
-      </DisorderGroup>
-      <PrevalenceList count="1">
-        <Prevalence id="5872">
-          <Source>ORPHANET</Source>
-          <PrevalenceType id="23669">
-            <Name lang="ru">Точечное распространение</Name>
-          </PrevalenceType>
-          <PrevalenceQualification id="23704">
-            <Name lang="ru">Только класс</Name>
-          </PrevalenceQualification>
-          <PrevalenceClass id="23760">
-            <Name lang="ru">&lt;1 / 1 000 000</Name>
-          </PrevalenceClass>
-          <ValMoy>0.0</ValMoy>
-          <PrevalenceGeographic id="23795">
-            <Name lang="ru">Европа</Name>
-          </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
-          </PrevalenceValidationStatus>
-        </Prevalence>
-      </PrevalenceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="ru">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=ru&amp;Expert=63442</ExpertLink>
@@ -280138,7 +279956,7 @@
       <DisorderGroup id="36540">
         <Name lang="ru">Группа заболеваний</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="2">
         <Prevalence id="5847">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -280158,6 +279976,25 @@
             <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16786">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="ru">Ежегодные показатели заболеваемости</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="10791">
@@ -285714,7 +285551,26 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="2">
+        <Prevalence id="16785">
+          <Source>30552564[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="ru">Ежегодные показатели заболеваемости</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="ru">Значение и класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23739">
+            <Name lang="ru">1-9 / 1 000 000</Name>
+          </PrevalenceClass>
+          <ValMoy>0.2582</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
         <Prevalence id="16361">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -303960,7 +303816,7 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="3">
+      <PrevalenceList count="5">
         <Prevalence id="16645">
           <Source>PMID: 26543054</Source>
           <PrevalenceType id="23669">
@@ -304018,6 +303874,44 @@
             <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16788">
+          <Source>34009447[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="ru">Ежегодные показатели заболеваемости</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="ru">Значение и класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="ru">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>4.5</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16789">
+          <Source>34009447[PMID]_ORPHANET</Source>
+          <PrevalenceType id="23669">
+            <Name lang="ru">Точечное распространение</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23711">
+            <Name lang="ru">Значение и класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23746">
+            <Name lang="ru">1-9 / 100 000</Name>
+          </PrevalenceClass>
+          <ValMoy>2.14</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="11909">
@@ -333673,7 +333567,7 @@
       </DisorderGroup>
       <PrevalenceList count="2">
         <Prevalence id="14711">
-          <Source>[EXPERT]_ORPHANET_25712311[PMID]</Source>
+          <Source>25712311[PMID]_26810252[PMID]</Source>
           <PrevalenceType id="23683">
             <Name lang="ru">Распространенность при рождении</Name>
           </PrevalenceType>
@@ -333683,7 +333577,7 @@
           <PrevalenceClass id="23732">
             <Name lang="ru">1-5 / 10 000</Name>
           </PrevalenceClass>
-          <ValMoy>20.0</ValMoy>
+          <ValMoy>19.25</ValMoy>
           <PrevalenceGeographic id="23795">
             <Name lang="ru">Европа</Name>
           </PrevalenceGeographic>
@@ -363531,7 +363425,7 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="3">
         <Prevalence id="7864">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -363547,8 +363441,46 @@
           <PrevalenceGeographic id="23844">
             <Name lang="ru">Весь мир</Name>
           </PrevalenceGeographic>
-          <PrevalenceValidationStatus id="25965">
-            <Name lang="ru">Еще не подтверждено</Name>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16790">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="ru">Ежегодные показатели заболеваемости</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
+        <Prevalence id="16791">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23690">
+            <Name lang="ru">Распространённость в течение всего срока жизни</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
       </PrevalenceList>
@@ -363563,7 +363495,7 @@
       <DisorderGroup id="36547">
         <Name lang="ru">Заболевание</Name>
       </DisorderGroup>
-      <PrevalenceList count="1">
+      <PrevalenceList count="2">
         <Prevalence id="7865">
           <Source>ORPHANET</Source>
           <PrevalenceType id="23669">
@@ -363583,6 +363515,25 @@
             <Name lang="ru">Подтверждено</Name>
           </PrevalenceValidationStatus>
         </Prevalence>
+        <Prevalence id="16787">
+          <Source>ORPHANET</Source>
+          <PrevalenceType id="23676">
+            <Name lang="ru">Ежегодные показатели заболеваемости</Name>
+          </PrevalenceType>
+          <PrevalenceQualification id="23704">
+            <Name lang="ru">Только класс</Name>
+          </PrevalenceQualification>
+          <PrevalenceClass id="23774">
+            <Name lang="ru">неизвестно</Name>
+          </PrevalenceClass>
+          <ValMoy>0.0</ValMoy>
+          <PrevalenceGeographic id="23844">
+            <Name lang="ru">Весь мир</Name>
+          </PrevalenceGeographic>
+          <PrevalenceValidationStatus id="25958">
+            <Name lang="ru">Подтверждено</Name>
+          </PrevalenceValidationStatus>
+        </Prevalence>
       </PrevalenceList>
     </Disorder>
     <Disorder id="14501">
