commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
old mode 100644
new mode 100755
index 04fcbbb..f98bfe8
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:39:15" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:40:12" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="7459">
+  <DisorderList count="7470">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -6626,7 +6626,7 @@
     <Disorder id="421">
       <OrphaCode>1146</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-      <Name lang="en">Digitotalar dysmorphism</Name>
+      <Name lang="en">Distal arthrogryposis type 1</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -28751,7 +28751,17 @@
       <OrphaCode>244275</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=244275</ExpertLink>
       <Name lang="en">De novo thrombotic microangiopathy after kidney transplantation</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="19474" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="2259">
@@ -34101,6 +34111,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="19728">
+      <OrphaCode>252175</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=252175</ExpertLink>
+      <Name lang="en">Vestibular schwannoma</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19592">
+            <OrphaCode>250908</OrphaCode>
+            <Name lang="en">Rare neoplastic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="19728" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="19756">
       <OrphaCode>254516</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=254516</ExpertLink>
@@ -52602,7 +52629,7 @@
     <Disorder id="3735">
       <OrphaCode>2126</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2126</ExpertLink>
-      <Name lang="en">Solitary fibrous tumour/hemangiopericytoma</Name>
+      <Name lang="en">Solitary fibrous tumor/hemangiopericytoma</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="19592">
@@ -58604,7 +58631,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-      <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+      <Name lang="en">Alazami syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12333">
@@ -61723,7 +61750,7 @@
     <Disorder id="22327">
       <OrphaCode>363611</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-      <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+      <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -66485,23 +66512,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="20606">
-      <OrphaCode>284408</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284408</ExpertLink>
-      <Name lang="en">Glycerol kinase deficiency, infantile form</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="10507">
-            <OrphaCode>68367</OrphaCode>
-            <Name lang="en">Rare inborn errors of metabolism</Name>
-          </TargetDisorder>
-          <RootDisorder id="20606" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="20607">
       <OrphaCode>284411</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284411</ExpertLink>
@@ -69763,7 +69773,7 @@
     <Disorder id="24042">
       <OrphaCode>464738</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-      <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+      <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -69967,7 +69977,7 @@
     <Disorder id="24022">
       <OrphaCode>464359</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464359</ExpertLink>
-      <Name lang="en">Benign metanephric tumour</Name>
+      <Name lang="en">Benign metanephric tumor</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -81744,7 +81754,7 @@
     <Disorder id="26002">
       <OrphaCode>500163</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=500163</ExpertLink>
-      <Name lang="en">SIN3A-related intellectual disability syndrome</Name>
+      <Name lang="en">Witteveen-Kolk syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -84376,9 +84386,9 @@
       <Name lang="en">Antisynthetase syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="8611" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -86532,7 +86542,7 @@
     <Disorder id="10986">
       <OrphaCode>71211</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=71211</ExpertLink>
-      <Name lang="en">Neuromyelitis optica</Name>
+      <Name lang="en">Neuromyelitis optica spectrum disorder</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -107418,9 +107428,9 @@
       <Name lang="en">Acute disseminated encephalomyelitis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="10557">
-            <OrphaCode>68416</OrphaCode>
-            <Name lang="en">Rare infectious disease</Name>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="11590" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -114004,6 +114014,40 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29576">
+      <OrphaCode>592850</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592850</ExpertLink>
+      <Name lang="en">Neuromyelitis optica spectrum disorder with anti-AQP4 antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29576" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29577">
+      <OrphaCode>592856</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592856</ExpertLink>
+      <Name lang="en">Neuromyelitis optica spectrum disorder with anti-MOG antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29577" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="14152">
       <OrphaCode>99135</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99135</ExpertLink>
@@ -114021,6 +114065,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29578">
+      <OrphaCode>592869</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592869</ExpertLink>
+      <Name lang="en">Neuromyelitis optica spectrum disorder without anti-MOG and without anti-AQP4 antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29578" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="14155">
       <OrphaCode>99138</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99138</ExpertLink>
@@ -114038,6 +114099,57 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29579">
+      <OrphaCode>592873</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592873</ExpertLink>
+      <Name lang="en">Acute transverse myelitis with anti-MOG antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29579" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29580">
+      <OrphaCode>592885</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592885</ExpertLink>
+      <Name lang="en">Isolated optic neuritis without anti-MOG antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12984">
+            <OrphaCode>97966</OrphaCode>
+            <Name lang="en">Rare ophthalmic disorder</Name>
+          </TargetDisorder>
+          <RootDisorder id="29580" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29581">
+      <OrphaCode>592888</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592888</ExpertLink>
+      <Name lang="en">Optic neuritis with anti-MOG antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12984">
+            <OrphaCode>97966</OrphaCode>
+            <Name lang="en">Rare ophthalmic disorder</Name>
+          </TargetDisorder>
+          <RootDisorder id="29581" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="14156">
       <OrphaCode>99139</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99139</ExpertLink>
@@ -114055,6 +114167,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29582">
+      <OrphaCode>592894</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592894</ExpertLink>
+      <Name lang="en">Acute disseminated encephalomyelitis with anti-MOG antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29582" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="14159">
       <OrphaCode>99142</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99142</ExpertLink>
@@ -114062,6 +114191,23 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29583">
+      <OrphaCode>592900</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592900</ExpertLink>
+      <Name lang="en">Acute disseminated encephalomyelitis without anti-MOG antibodies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29583" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="14158">
       <OrphaCode>99141</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99141</ExpertLink>
@@ -114484,6 +114630,57 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29566">
+      <OrphaCode>592574</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592574</ExpertLink>
+      <Name lang="en">Menke-Hennekam syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29566" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592564</ExpertLink>
+      <Name lang="en">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29564" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=592570</ExpertLink>
+      <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29565" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="14267">
       <OrphaCode>99694</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99694</ExpertLink>
@@ -114536,7 +114733,17 @@
       <OrphaCode>590539</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=590539</ExpertLink>
       <Name lang="en">Isolated melanotic schwannoma</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19592">
+            <OrphaCode>250908</OrphaCode>
+            <Name lang="en">Rare neoplastic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29460" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="14298">
