commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
index 9fdbceb..15b51c5 100644
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-01-01 04:52:09" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:53:54" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="7208">
+  <DisorderList count="7219">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -9565,7 +9565,7 @@
     <Disorder id="17996">
       <OrphaCode>177926</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=177926</ExpertLink>
-      <Name lang="en">Bleeding disorder in hemophilia A carriers without FVIII deficiency</Name>
+      <Name lang="en">Bleeding disorder in hemophilia A carriers</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13010">
@@ -9599,7 +9599,7 @@
     <Disorder id="17997">
       <OrphaCode>177929</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=177929</ExpertLink>
-      <Name lang="en">Bleeding disorder in hemophilia B carriers without FIX deficiency</Name>
+      <Name lang="en">Bleeding disorder in hemophilia B carriers</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13010">
@@ -11687,23 +11687,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="813">
-      <OrphaCode>1549</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1549</ExpertLink>
-      <Name lang="en">Cryptosporidiosis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="10557">
-            <OrphaCode>68416</OrphaCode>
-            <Name lang="en">Rare infectious disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="813" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="18415">
       <OrphaCode>199642</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=199642</ExpertLink>
@@ -63537,6 +63520,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="5517">
+      <OrphaCode>80</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=80</ExpertLink>
+      <Name lang="en">Antiphospholipid syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="5517" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="20612">
       <OrphaCode>284448</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284448</ExpertLink>
@@ -65308,7 +65308,7 @@
     <Disorder id="21452">
       <OrphaCode>313800</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313800</ExpertLink>
-      <Name lang="en">Optic nerve edema-splenomegaly syndrome</Name>
+      <Name lang="en">Retinal dystrophy-optic nerve edema-splenomegaly-anhidrosis-migraine headache syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12984">
@@ -71788,9 +71788,9 @@
       <Name lang="en">Familial bicuspid aortic valve</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="12980">
-            <OrphaCode>97962</OrphaCode>
-            <Name lang="en">Rare surgical thoracic disease</Name>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
           </TargetDisorder>
           <RootDisorder id="22877" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -73505,9 +73505,9 @@
       <Name lang="en">Joubert syndrome with Jeune asphyxiating thoracic dystrophy</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="12980">
-            <OrphaCode>97962</OrphaCode>
-            <Name lang="en">Rare surgical thoracic disease</Name>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
           </TargetDisorder>
           <RootDisorder id="22695" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -89924,18 +89924,8 @@
     <Disorder id="11041">
       <OrphaCode>73247</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73247</ExpertLink>
-      <Name lang="en">Eosinophilic esophagitis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12954">
-            <OrphaCode>97935</OrphaCode>
-            <Name lang="en">Rare gastroenterologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="11041" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
+      <Name lang="en">NON RARE IN EUROPE: Eosinophilic esophagitis</Name>
+      <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="28655">
@@ -95826,9 +95816,9 @@
       <Name lang="en">Erythema elevatum diutinum</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="13041">
-            <OrphaCode>98023</OrphaCode>
-            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          <TargetDisorder id="11896">
+            <OrphaCode>89826</OrphaCode>
+            <Name lang="en">Rare skin disease</Name>
           </TargetDisorder>
           <RootDisorder id="11918" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -108542,9 +108532,9 @@
       <Name lang="en">Twin to twin transfusion syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="13022">
-            <OrphaCode>98004</OrphaCode>
-            <Name lang="en">Rare immune disease</Name>
+          <TargetDisorder id="12845">
+            <OrphaCode>96344</OrphaCode>
+            <Name lang="en">Rare gynecologic or obstetric disease</Name>
           </TargetDisorder>
           <RootDisorder id="12590" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -122527,6 +122517,91 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="31348">
+      <OrphaCode>617449</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617449</ExpertLink>
+      <Name lang="en">Congenital aphakia-iris hypoplasia-microphthalmia-microcornea syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12984">
+            <OrphaCode>97966</OrphaCode>
+            <Name lang="en">Rare ophthalmic disorder</Name>
+          </TargetDisorder>
+          <RootDisorder id="31348" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31346">
+      <OrphaCode>617408</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617408</ExpertLink>
+      <Name lang="en">Classic eosinophilic pustular folliculitis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11896">
+            <OrphaCode>89826</OrphaCode>
+            <Name lang="en">Rare skin disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31346" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31347">
+      <OrphaCode>617440</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617440</ExpertLink>
+      <Name lang="en">Painful legs and moving toes syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31347" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31340">
+      <OrphaCode>617304</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617304</ExpertLink>
+      <Name lang="en">Amniotic fluid embolism</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12845">
+            <OrphaCode>96344</OrphaCode>
+            <Name lang="en">Rare gynecologic or obstetric disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31340" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31337">
+      <OrphaCode>617294</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617294</ExpertLink>
+      <Name lang="en">Twin anemia-polycythemia sequence</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12845">
+            <OrphaCode>96344</OrphaCode>
+            <Name lang="en">Rare gynecologic or obstetric disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31337" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="31336">
       <OrphaCode>616874</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=616874</ExpertLink>
@@ -122534,5 +122609,67 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="31339">
+      <OrphaCode>617301</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617301</ExpertLink>
+      <Name lang="en">Selective intrauterine growth restriction</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12845">
+            <OrphaCode>96344</OrphaCode>
+            <Name lang="en">Rare gynecologic or obstetric disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31339" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31338">
+      <OrphaCode>617297</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617297</ExpertLink>
+      <Name lang="en">Twin-reversed arterial perfusion sequence</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12845">
+            <OrphaCode>96344</OrphaCode>
+            <Name lang="en">Rare gynecologic or obstetric disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31338" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31370">
+      <OrphaCode>617910</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617910</ExpertLink>
+      <Name lang="en">Conjunctival malignant melanoma</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31371">
+      <OrphaCode>617916</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617916</ExpertLink>
+      <Name lang="en">Diffuse idiopathic pulmonary neuroendocrine cell hyperplasia</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31372">
+      <OrphaCode>617919</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617919</ExpertLink>
+      <Name lang="en">F12-associated cold auto-inflammatory syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31373">
+      <OrphaCode>617930</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617930</ExpertLink>
+      <Name lang="en">Hemophilia B Leyden</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
