commit 76201feb251c2058792f9cf480d515ad9cf39106
Author: David Lagorce <david.lagorce@inserm.fr>
Date:   Wed Jul 7 12:35:15 2021 +0200

    JULY 2021

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
old mode 100644
new mode 100755
index 3c31938..ed7f707
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-06-01 04:45:10" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-07-01 04:46:32" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="7490">
+  <DisorderList count="7513">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -2943,7 +2943,7 @@
     <Disorder id="17521">
       <OrphaCode>163681</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=163681</ExpertLink>
-      <Name lang="en">Cortical dysplasia-focal epilepsy syndrome</Name>
+      <Name lang="en">CNTNAP2-related developmental and epileptic encephalopathy</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -4528,23 +4528,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="17901">
-      <OrphaCode>171201</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=171201</ExpertLink>
-      <Name lang="en">High isolated anorectal malformation</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="17901" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="301">
       <OrphaCode>2162</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2162</ExpertLink>
@@ -4579,23 +4562,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="17903">
-      <OrphaCode>171215</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=171215</ExpertLink>
-      <Name lang="en">Low isolated anorectal malformation</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="17903" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="303">
       <OrphaCode>998</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=998</ExpertLink>
@@ -4613,23 +4579,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="17902">
-      <OrphaCode>171208</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=171208</ExpertLink>
-      <Name lang="en">Intermediate isolated anorectal malformation</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="17902" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="343">
       <OrphaCode>1727</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1727</ExpertLink>
@@ -11956,7 +11905,7 @@
     <Disorder id="815">
       <OrphaCode>549</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=549</ExpertLink>
-      <Name lang="en">Legionellosis</Name>
+      <Name lang="en">Legionnaires disease</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="10557">
@@ -16034,7 +15983,7 @@
     <Disorder id="1190">
       <OrphaCode>1762</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1762</ExpertLink>
-      <Name lang="en">Trisomy Xq28</Name>
+      <Name lang="en">Proximal Xq28 duplication syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -18410,7 +18359,7 @@
     <Disorder id="16683">
       <OrphaCode>137602</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137602</ExpertLink>
-      <Name lang="en">Endotheliitis</Name>
+      <Name lang="en">Corneal endotheliitis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12984">
@@ -35713,9 +35662,9 @@
       <Name lang="en">FOXG1 syndrome due to 14q12 microdeletion</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="19853" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -52147,23 +52096,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="18950">
-      <OrphaCode>221150</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=221150</ExpertLink>
-      <Name lang="en">Pitt-Hopkins-like syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="18950" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="19062">
       <OrphaCode>228003</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=228003</ExpertLink>
@@ -58662,7 +58594,7 @@
     <Disorder id="21638">
       <OrphaCode>319269</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319269</ExpertLink>
-      <Name lang="en">Susceptibility/resistance to HIV infection</Name>
+      <Name lang="en">Selection of therapeutic option in AIDS</Name>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
@@ -67359,9 +67291,9 @@
       <Name lang="en">Xq12-q13.3 duplication syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="21479" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -70593,7 +70525,7 @@
     <Disorder id="23617">
       <OrphaCode>449306</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=449306</ExpertLink>
-      <Name lang="en">Susceptibility to infection in immunocompromised patient</Name>
+      <Name lang="en">Dose selection of antibiotic therapy</Name>
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
@@ -80891,12 +80823,12 @@
     <Disorder id="26390">
       <OrphaCode>505652</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=505652</ExpertLink>
-      <Name lang="en">CDKL5-related epileptic encephalopathy</Name>
+      <Name lang="en">CDKL5-deficiency disorder</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="26390" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -81530,9 +81462,9 @@
       <Name lang="en">Infantile-onset generalized dyskinesia with orofacial involvement</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="19592">
-            <OrphaCode>250908</OrphaCode>
-            <Name lang="en">Rare neoplastic disease</Name>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="25681" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -94179,23 +94111,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="27912">
-      <OrphaCode>537891</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=537891</ExpertLink>
-      <Name lang="en">ANGPT1-related hereditary angioedema with normal C1Inh</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13041">
-            <OrphaCode>98023</OrphaCode>
-            <Name lang="en">Rare systemic or rheumatologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="27912" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="10697">
       <OrphaCode>52429</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=52429</ExpertLink>
@@ -94709,9 +94624,9 @@
       <Name lang="en">Corticosteroid-sensitive aseptic abscess syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="11896">
-            <OrphaCode>89826</OrphaCode>
-            <Name lang="en">Rare skin disease</Name>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
           </TargetDisorder>
           <RootDisorder id="10749" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -95723,7 +95638,7 @@
     <Disorder id="28036">
       <OrphaCode>542301</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=542301</ExpertLink>
-      <Name lang="en">Severe combined immunodeficiency due to CARMIL2 deficiency</Name>
+      <Name lang="en">Combined immunodeficiency due to CARMIL2 deficiency</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13022">
@@ -102806,23 +102721,6 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="11485">
-      <OrphaCode>79459</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79459</ExpertLink>
-      <Name lang="en">Follicular atrophoderma-basal cell carcinoma</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="11896">
-            <OrphaCode>89826</OrphaCode>
-            <Name lang="en">Rare skin disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="11485" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="11482">
       <OrphaCode>79456</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79456</ExpertLink>
@@ -104871,13 +104769,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="11671">
-      <OrphaCode>85281</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85281</ExpertLink>
-      <Name lang="en">MECP2 duplication syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="11672">
       <OrphaCode>85282</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85282</ExpertLink>
@@ -108025,6 +107916,54 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="30475">
+      <OrphaCode>600668</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600668</ExpertLink>
+      <Name lang="en">CCNK-related neurodevelopmental disorder-severe intellectual disability-facial dysmorphism syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30474">
+      <OrphaCode>600663</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600663</ExpertLink>
+      <Name lang="en">NRXN1-related severe neurodevelopmental disorder-motor stereotypies-chronic constipation-sleep-wake cycle disturbance</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="30474" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30477">
+      <OrphaCode>600691</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600691</ExpertLink>
+      <Name lang="en">Combined deficiency of factor VII and factor X</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30478">
+      <OrphaCode>600731</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600731</ExpertLink>
+      <Name lang="en">Clark-Baraitser syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30478" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="13284">
       <OrphaCode>98267</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98267</ExpertLink>
@@ -108042,6 +107981,244 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="30552">
+      <OrphaCode>601028</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601028</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectovaginal fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30552" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30553">
+      <OrphaCode>601033</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601033</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with H-type fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30553" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30548">
+      <OrphaCode>601008</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601008</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with anal stenosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30548" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30549">
+      <OrphaCode>601013</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601013</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with pouch colon</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30549" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30550">
+      <OrphaCode>601018</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601018</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectal atresia</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30550" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30551">
+      <OrphaCode>601023</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601023</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectal stenosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30551" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30544">
+      <OrphaCode>600984</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600984</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectovesical fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30544" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30545">
+      <OrphaCode>600993</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600993</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with vestibular fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30545" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30546">
+      <OrphaCode>600998</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600998</ExpertLink>
+      <Name lang="en">Non-syndromic cloacal malformation</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30546" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30547">
+      <OrphaCode>601002</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=601002</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation without fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30547" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30541">
+      <OrphaCode>600961</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600961</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectourethral fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30541" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30540">
+      <OrphaCode>600952</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600952</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with perineal fistula</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30540" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30543">
+      <OrphaCode>600975</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600975</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectourethral fistula, prostatic type</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30543" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="30542">
+      <OrphaCode>600966</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600966</ExpertLink>
+      <Name lang="en">Non-syndromic anorectal malformation with rectourethral fistula, bulbar type</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="30542" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12866">
       <OrphaCode>97244</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97244</ExpertLink>
@@ -109389,6 +109566,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="30241">
+      <OrphaCode>600194</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=600194</ExpertLink>
+      <Name lang="en">Factor V Atlanta bleeding disorder</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12942">
       <OrphaCode>97678</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97678</ExpertLink>
@@ -109885,7 +110069,7 @@
     <Disorder id="12593">
       <OrphaCode>95434</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95434</ExpertLink>
-      <Name lang="en">Autosomal recessive cerebellar ataxia-saccadic intrusion syndrome</Name>
+      <Name lang="en">Autosomal recessive cerebellar ataxia-movement disorder syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -111422,29 +111606,69 @@
     <Disorder id="29860">
       <OrphaCode>597623</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597623</ExpertLink>
-      <Name lang="en">IRF2BPL- related regressive neurodevelopmental disorder-dystonia-seizures syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">IRF2BPL-related regressive neurodevelopmental disorder-dystonia-seizures syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29860" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29862">
       <OrphaCode>597733</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597733</ExpertLink>
       <Name lang="en">Oculocutaneous albinism type 8</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="11896">
+            <OrphaCode>89826</OrphaCode>
+            <Name lang="en">Rare skin disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29862" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29863">
       <OrphaCode>597738</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597738</ExpertLink>
       <Name lang="en">Luscan-Lumish syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29863" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29868">
       <OrphaCode>597887</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597887</ExpertLink>
       <Name lang="en">ALPI-related inflammatory bowel disease</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12954">
+            <OrphaCode>97935</OrphaCode>
+            <Name lang="en">Rare gastroenterologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29868" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12397">
@@ -111501,8 +111725,18 @@
     <Disorder id="29871">
       <OrphaCode>597939</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597939</ExpertLink>
-      <Name lang="en">Euthyroid dysprealbuinemic hyperthyroxinemia</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">Euthyroid dysprealbuminemic hyperthyroxinemia</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12996">
+            <OrphaCode>97978</OrphaCode>
+            <Name lang="en">Rare endocrine disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29871" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12398">
@@ -111526,14 +111760,34 @@
       <OrphaCode>597743</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597743</ExpertLink>
       <Name lang="en">SETD2-related microcephaly-severe intellectual disability-multiple congenital anomalies syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29864" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29865">
       <OrphaCode>597746</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597746</ExpertLink>
-      <Name lang="en">Blepharophimosis-intellectual disability syndrome, SBBYS type/genitopatellar overlap syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">Blepharophimosis-intellectual disability syndrome/genitopatellar overlap syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29865" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12394">
@@ -111556,8 +111810,18 @@
     <Disorder id="29867">
       <OrphaCode>597874</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597874</ExpertLink>
-      <Name lang="en">MTHFS-related developmental delay-microcephaly-short stature- epilepsy syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">MTHFS-related developmental delay-microcephaly-short stature-epilepsy syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29867" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12404">
@@ -111598,14 +111862,34 @@
       <OrphaCode>598603</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598603</ExpertLink>
       <Name lang="en">Facial dysmorphism-hypertrichosis-epilepsy-intellectual disability/developmental delay-gingival overgrowth syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29879" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29878">
       <OrphaCode>598363</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=598363</ExpertLink>
       <Name lang="en">Multisystem inflammatory syndrome in children and adults</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29878" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12400">
@@ -111738,7 +112022,17 @@
       <OrphaCode>597201</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=597201</ExpertLink>
       <Name lang="en">TRIM22-related inflammatory bowel disease</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12954">
+            <OrphaCode>97935</OrphaCode>
+            <Name lang="en">Rare gastroenterologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29831" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12382">
@@ -111843,6 +112137,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599373</ExpertLink>
+      <Name lang="en">STXBP1-related encephalopathy</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29924" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93403</ExpertLink>
@@ -111860,6 +112171,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29925">
+      <OrphaCode>599376</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599376</ExpertLink>
+      <Name lang="en">Hypomyelination of early myelinating structures</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29925" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12324">
       <OrphaCode>93402</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93402</ExpertLink>
@@ -111894,6 +112222,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29926">
+      <OrphaCode>599418</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599418</ExpertLink>
+      <Name lang="en">Hereditary angioedema with normal C1Inh not related to F12 or PLG variant</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29926" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12326">
       <OrphaCode>93404</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93404</ExpertLink>
@@ -111911,6 +112256,20 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29927">
+      <OrphaCode>599480</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599480</ExpertLink>
+      <Name lang="en">Acquired hemophilia A</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29928">
+      <OrphaCode>599485</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599485</ExpertLink>
+      <Name lang="en">Acquired hemophilia B</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12328">
       <OrphaCode>93406</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93406</ExpertLink>
@@ -111928,6 +112287,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29929">
+      <OrphaCode>599490</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599490</ExpertLink>
+      <Name lang="en">Acquired factor V deficiency</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12331">
       <OrphaCode>93409</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93409</ExpertLink>
@@ -111945,6 +112311,48 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29930">
+      <OrphaCode>599495</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599495</ExpertLink>
+      <Name lang="en">Acquired factor VII deficiency</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29931">
+      <OrphaCode>599501</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599501</ExpertLink>
+      <Name lang="en">Acquired factor X deficiency</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29932">
+      <OrphaCode>599507</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599507</ExpertLink>
+      <Name lang="en">Acquired factor XI deficiency</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29933">
+      <OrphaCode>599513</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599513</ExpertLink>
+      <Name lang="en">Acquired factor XIII deficiency</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29934">
+      <OrphaCode>599519</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599519</ExpertLink>
+      <Name lang="en">Factor V short isoforms-related bleeding disorder</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29935">
+      <OrphaCode>599579</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599579</ExpertLink>
+      <Name lang="en">Factor V Amsterdam bleeding disorder</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12291">
       <OrphaCode>93360</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93360</ExpertLink>
@@ -112017,7 +112425,17 @@
       <OrphaCode>599082</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=599082</ExpertLink>
       <Name lang="en">CHD3-related developmental delay-speech delay-intellectual disability-abnormalities of vision-facial dysmorphism syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29899" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12300">
@@ -112539,7 +112957,17 @@
       <OrphaCode>595109</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595109</ExpertLink>
       <Name lang="en">Atypical Timothy syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12948">
+            <OrphaCode>97929</OrphaCode>
+            <Name lang="en">Rare cardiac disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29724" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29725">
@@ -112580,7 +113008,17 @@
       <OrphaCode>595098</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595098</ExpertLink>
       <Name lang="en">Timothy syndrome type 1</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12948">
+            <OrphaCode>97929</OrphaCode>
+            <Name lang="en">Rare cardiac disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29722" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12507">
@@ -112604,7 +113042,17 @@
       <OrphaCode>595105</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595105</ExpertLink>
       <Name lang="en">Timothy syndrome type 2</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12948">
+            <OrphaCode>97929</OrphaCode>
+            <Name lang="en">Rare cardiac disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29723" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12506">
@@ -112880,7 +113328,17 @@
       <OrphaCode>596008</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596008</ExpertLink>
       <Name lang="en">Antley-Bixler syndrome without genital anomaly or disorder of steroidogenesis</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29762" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12422">
@@ -112961,8 +113419,18 @@
     <Disorder id="29784">
       <OrphaCode>596759</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596759</ExpertLink>
-      <Name lang="en">Combined immunodeficiency due to Rela haploinsufficiency</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">Combined immunodeficiency due to RELA haploinsufficiency</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13022">
+            <OrphaCode>98004</OrphaCode>
+            <Name lang="en">Rare immune disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29784" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12441">
@@ -113112,7 +113580,17 @@
       <OrphaCode>596448</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596448</ExpertLink>
       <Name lang="en">IgG4-related systemic disease</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29776" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12433">
@@ -113177,7 +113655,17 @@
       <OrphaCode>596753</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596753</ExpertLink>
       <Name lang="en">VEXAS syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29783" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="12438">
@@ -113544,23 +114032,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="14086">
-      <OrphaCode>99069</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99069</ExpertLink>
-      <Name lang="en">Univentricular heart with single atrio-ventricular valve</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="14086" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="14081">
       <OrphaCode>99064</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99064</ExpertLink>
@@ -123222,7 +123693,7 @@
     <Disorder id="14458">
       <OrphaCode>99885</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99885</ExpertLink>
-      <Name lang="en">Permanent neonatal diabetes mellitus</Name>
+      <Name lang="en">Isolated permanent neonatal diabetes mellitus</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12996">
