commit c92e9c5f9940abcb2c7950377e1c2245059cad90
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 1 10:54:09 2021 +0100

    MARCH 2021

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
old mode 100755
new mode 100644
index 0851f63..4ba22d0
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-02-01 04:40:51" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-03-01 04:40:56" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -182,17 +182,7 @@
       <OrphaCode>166068</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166068</ExpertLink>
       <Name lang="en">Pontocerebellar hypoplasia type 5</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="17609" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
+      <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="8">
@@ -23738,7 +23728,7 @@
     <Disorder id="1686">
       <OrphaCode>1519</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1519</ExpertLink>
-      <Name lang="en">Hypertelorism, Teebi type</Name>
+      <Name lang="en">SPECC1L-related hypertelorism syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -26937,7 +26927,7 @@
     <Disorder id="17234">
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158676</ExpertLink>
-      <Name lang="en">Dominant dystrophic epidermolysis bullosa, nails only</Name>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa, nails only</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -26954,7 +26944,7 @@
     <Disorder id="17233">
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158673</ExpertLink>
-      <Name lang="en">Acral dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa, acral form</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -28124,7 +28114,7 @@
     <Disorder id="2233">
       <OrphaCode>2407</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-      <Name lang="en">LOC syndrome</Name>
+      <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -31964,7 +31954,7 @@
     <Disorder id="19640">
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251393</ExpertLink>
-      <Name lang="en">Localized junctional epidermolysis bullosa, non-Herlitz type</Name>
+      <Name lang="en">Localized junctional epidermolysis bullosa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -50194,7 +50184,7 @@
     <Disorder id="3423">
       <OrphaCode>2745</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2745</ExpertLink>
-      <Name lang="en">Opitz G/BBB syndrome</Name>
+      <Name lang="en">Opitz GBBB syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -55139,7 +55129,7 @@
     <Disorder id="19201">
       <OrphaCode>231568</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231568</ExpertLink>
-      <Name lang="en">Generalized dominant dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Autosomal dominant generalized dystrophic epidermolysis bullosa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -68974,7 +68964,7 @@
     <Disorder id="21219">
       <OrphaCode>306504</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306504</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa with respiratory and renal involvement</Name>
+      <Name lang="en">Interstitial lung disease-nephrotic syndrome-epidermolysis bullosa syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -69056,40 +69046,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="21238">
-      <OrphaCode>306597</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-      <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="21238" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
-    <Disorder id="21237">
-      <OrphaCode>306588</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-      <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="21237" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="21236">
       <OrphaCode>306577</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306577</ExpertLink>
@@ -69722,7 +69678,7 @@
     <Disorder id="21112">
       <OrphaCode>300333</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=300333</ExpertLink>
-      <Name lang="en">Nephrotic syndrome-deafness-pretibial epidermolysis bullosa syndrome</Name>
+      <Name lang="en">Nephrotic syndrome-epidermolysis bullosa-sensorineural deafness syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -97732,7 +97688,7 @@
     <Disorder id="11907">
       <OrphaCode>89842</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89842</ExpertLink>
-      <Name lang="en">Recessive dystrophic epidermolysis bullosa, generalized intermediate</Name>
+      <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, intermediate form</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -99372,7 +99328,7 @@
     <Disorder id="11903">
       <OrphaCode>89838</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89838</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, autosomal recessive K14</Name>
+      <Name lang="en">Autosomal recessive generalized epidermolysis bullosa simplex</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102515,7 +102471,7 @@
     <Disorder id="11422">
       <OrphaCode>79396</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79396</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, generalized severe</Name>
+      <Name lang="en">Autosomal dominant generalized epidermolysis bullosa simplex, severe form</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102600,7 +102556,7 @@
     <Disorder id="11425">
       <OrphaCode>79399</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79399</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, generalized intermediate</Name>
+      <Name lang="en">Autosomal dominant generalized epidermolysis bullosa simplex, intermediate form</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102617,7 +102573,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79401</ExpertLink>
-      <Name lang="en">PLEC-related epidermolysis bullosa simplex without extracutaneous involvement</Name>
+      <Name lang="en">PLEC-related intermediate epidermolysis bullosa simplex without extracutaneous involvement</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102651,7 +102607,7 @@
     <Disorder id="11429">
       <OrphaCode>79403</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79403</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa-pyloric atresia syndrome</Name>
+      <Name lang="en">Junctional epidermolysis bullosa with pyloric atresia</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102668,7 +102624,7 @@
     <Disorder id="11428">
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79402</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa, generalized intermediate</Name>
+      <Name lang="en">Intermediate generalized junctional epidermolysis bullosa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102702,7 +102658,7 @@
     <Disorder id="11430">
       <OrphaCode>79404</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79404</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa, generalized severe</Name>
+      <Name lang="en">Severe generalized junctional epidermolysis bullosa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102743,7 +102699,7 @@
     <Disorder id="11435">
       <OrphaCode>79409</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79409</ExpertLink>
-      <Name lang="en">Recessive dystrophic epidermolysis bullosa inversa</Name>
+      <Name lang="en">Dystrophic epidermolysis bullosa inversa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102760,7 +102716,7 @@
     <Disorder id="11434">
       <OrphaCode>79408</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79408</ExpertLink>
-      <Name lang="en">Severe generalized recessive dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102777,7 +102733,7 @@
     <Disorder id="11437">
       <OrphaCode>79411</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79411</ExpertLink>
-      <Name lang="en">Transient bullous dermolysis of the newborn</Name>
+      <Name lang="en">Self-improving dystrophic epidermolysis bullosa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -102794,7 +102750,7 @@
     <Disorder id="11436">
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79410</ExpertLink>
-      <Name lang="en">Pretibial dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa, pretibial form</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -112872,6 +112828,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29762">
+      <OrphaCode>596008</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596008</ExpertLink>
+      <Name lang="en">Antley-Bixler syndrome without genital anomaly or disorder of steroidogenesis</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12422">
       <OrphaCode>93583</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93583</ExpertLink>
@@ -113056,6 +113019,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29776">
+      <OrphaCode>596448</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=596448</ExpertLink>
+      <Name lang="en">IgG4-related systemic disease</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12433">
       <OrphaCode>93600</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93600</ExpertLink>
@@ -116166,7 +116136,7 @@
     <Disorder id="13889">
       <OrphaCode>98872</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=98872</ExpertLink>
-      <Name lang="en">Adult pure red cell aplasia</Name>
+      <Name lang="en">Primary acquired pure red cell aplasia</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13010">
