commit bbf94d552a3c459862632b82d2e2166d94fb6b4e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
index ebced64..04fcbbb 100644
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:36:19" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:39:15" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -114560,14 +114560,34 @@
       <OrphaCode>589821</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589821</ExpertLink>
       <Name lang="en">Congenital-onset Steinert myotonic dystrophy</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29441" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29443">
       <OrphaCode>589827</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589827</ExpertLink>
       <Name lang="en">Juvenile-onset Steinert myotonic dystrophy</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29443" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="14274">
@@ -114591,21 +114611,51 @@
       <OrphaCode>589824</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589824</ExpertLink>
       <Name lang="en">Childhood-onset Steinert myotonic dystrophy</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29442" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29445">
       <OrphaCode>589833</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589833</ExpertLink>
       <Name lang="en">Late-onset Steinert myotonic dystrophy</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29445" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29444">
       <OrphaCode>589830</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589830</ExpertLink>
       <Name lang="en">Adult-onset Steinert myotonic dystrophy</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29444" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="14277">
@@ -114619,14 +114669,34 @@
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589905</ExpertLink>
       <Name lang="en">PHIP-related behavioral problems-intellectual disability-obesity-dysmorphic features syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12333">
+            <OrphaCode>93419</OrphaCode>
+            <Name lang="en">Rare bone disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29447" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29446">
       <OrphaCode>589856</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589856</ExpertLink>
       <Name lang="en">Choanal atresia-athelia-hypothyroidism-delayed puberty-short stature syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29446" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="14283">
@@ -115503,8 +115573,18 @@
     <Disorder id="29437">
       <OrphaCode>589608</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589608</ExpertLink>
-      <Name lang="en">Linear hypopigmentation and craniofacial asymmetry with acral, ocular, and brain anomalies</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">Linear hypopigmentation and craniofacial asymmetry with acral, ocular and brain anomalies</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29437" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13884">
@@ -115518,7 +115598,17 @@
       <OrphaCode>589595</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589595</ExpertLink>
       <Name lang="en">Mixed phenotype acute leukemia with t(v;11q23.3)</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19592">
+            <OrphaCode>250908</OrphaCode>
+            <Name lang="en">Rare neoplastic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29436" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13885">
@@ -115559,7 +115649,17 @@
       <OrphaCode>589618</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589618</ExpertLink>
       <Name lang="en">Dystonia 28</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29438" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13887">
@@ -115583,7 +115683,17 @@
       <OrphaCode>589534</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589534</ExpertLink>
       <Name lang="en">Mixed phenotype acute leukemia with t(9;22)(q34.1;q11.2)</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19592">
+            <OrphaCode>250908</OrphaCode>
+            <Name lang="en">Rare neoplastic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29433" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13880">
@@ -115607,7 +115717,17 @@
       <OrphaCode>589527</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589527</ExpertLink>
       <Name lang="en">Spinocerebellar ataxia type 45</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29432" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13881">
@@ -115621,7 +115741,17 @@
       <OrphaCode>589547</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589547</ExpertLink>
       <Name lang="en">GRIN2B-related developmental delay, intellectual disability and autism spectrum disorder</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29435" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13882">
@@ -115635,14 +115765,34 @@
       <OrphaCode>589542</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589542</ExpertLink>
       <Name lang="en">Myeloid/lymphoid neoplasm associated with JAK2 rearrangement</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="19592">
+            <OrphaCode>250908</OrphaCode>
+            <Name lang="en">Rare neoplastic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29434" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29429">
       <OrphaCode>589442</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589442</ExpertLink>
       <Name lang="en">Short stature-skeletal dysplasia-retinal degeneration-intellectual disability-sensorineural hearing loss syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="29429" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="29428">
@@ -115666,7 +115816,17 @@
       <OrphaCode>589522</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589522</ExpertLink>
       <Name lang="en">Spinocerebellar ataxia type 46</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29431" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13878">
@@ -115679,8 +115839,18 @@
     <Disorder id="29430">
       <OrphaCode>589515</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589515</ExpertLink>
-      <Name lang="en">PADDAS syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">PUM1-associated developmental disability-ataxia-seizure syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13024">
+            <OrphaCode>98006</OrphaCode>
+            <Name lang="en">Rare neurologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="29430" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="13872">
