commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
index 258fc4a..5ed39d3 100644
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:54:14" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:58:46" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="7218">
+  <DisorderList count="7232">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -7845,23 +7845,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="521">
-      <OrphaCode>2343</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2343</ExpertLink>
-      <Name lang="en">Isolated cloverleaf skull syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="521" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="18133">
       <OrphaCode>180247</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=180247</ExpertLink>
@@ -11160,23 +11143,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="727">
-      <OrphaCode>1428</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1428</ExpertLink>
-      <Name lang="en">Familial chondromalacia patellae</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="727" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="820">
       <OrphaCode>3303</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3303</ExpertLink>
@@ -23301,7 +23267,7 @@
     <Disorder id="1683">
       <OrphaCode>1516</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1516</ExpertLink>
-      <Name lang="en">Craniofacial dyssynostosis</Name>
+      <Name lang="en">Non-syndromic bilambdoid and sagittal craniosynostosis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -28945,7 +28911,7 @@
     <Disorder id="2078">
       <OrphaCode>2196</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2196</ExpertLink>
-      <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
+      <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis with severe ocular involvement</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -37272,23 +37238,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="2804">
-      <OrphaCode>3133</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3133</ExpertLink>
-      <Name lang="en">Say-Field-Coldwell syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="2804" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="2805">
       <OrphaCode>3134</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3134</ExpertLink>
@@ -39927,7 +39876,7 @@
     <Disorder id="2983">
       <OrphaCode>3366</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3366</ExpertLink>
-      <Name lang="en">Isolated trigonocephaly</Name>
+      <Name lang="en">Non-syndromic metopic craniosynostosis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -42933,23 +42882,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="2918">
-      <OrphaCode>3267</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3267</ExpertLink>
-      <Name lang="en">Familial lambdoid synostosis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="2918" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="20390">
       <OrphaCode>276621</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276621</ExpertLink>
@@ -46336,7 +46268,7 @@
     <Disorder id="18667">
       <OrphaCode>209908</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=209908</ExpertLink>
-      <Name lang="en">Childhood apraxia of speech</Name>
+      <Name lang="en">Isolated childhood apraxia of speech</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -55683,23 +55615,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="21917">
-      <OrphaCode>329252</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329252</ExpertLink>
-      <Name lang="en">Spondylocostal dysostosis-hypospadias-intellectual disability syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="21917" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="21916">
       <OrphaCode>329249</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329249</ExpertLink>
@@ -56315,7 +56230,7 @@
     <Disorder id="21781">
       <OrphaCode>324525</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=324525</ExpertLink>
-      <Name lang="en">Hypertrophic cardiomyopathy and renal tubular disease due to mitochondrial DNA mutation</Name>
+      <Name lang="en">Hypertrophic cardiomyopathy with kidney anomalies due to mitochondrial DNA mutation</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="10507">
@@ -67362,6 +67277,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="21222">
+      <OrphaCode>306516</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306516</ExpertLink>
+      <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="21222" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="21221">
       <OrphaCode>306511</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306511</ExpertLink>
@@ -84416,7 +84348,7 @@
     <Disorder id="9284">
       <OrphaCode>31043</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=31043</ExpertLink>
-      <Name lang="en">Familial primary hypomagnesemia with hypercalciuria and nephrocalcinosis without severe ocular involvement</Name>
+      <Name lang="en">Primary hypomagnesemia with hypercalciuria and nephrocalcinosis without severe ocular involvement</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -86708,23 +86640,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63440</ExpertLink>
-      <Name lang="en">Isolated oxycephaly</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="10822" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=63442</ExpertLink>
@@ -87102,7 +87017,7 @@
     <Disorder id="28399">
       <OrphaCode>564178</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=564178</ExpertLink>
-      <Name lang="en">Primary hypomagnesemia with refractory seizures and intellectual disability</Name>
+      <Name lang="en">Primary hypomagnesemia-refractory seizures-intellectual disability syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -90400,7 +90315,7 @@
     <Disorder id="11036">
       <OrphaCode>73224</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=73224</ExpertLink>
-      <Name lang="en">Tubular renal disease-cardiomyopathy syndrome</Name>
+      <Name lang="en">Kidney tubulopathy-dilated cardiomyopathy syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -91114,7 +91029,7 @@
     <Disorder id="10369">
       <OrphaCode>35099</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35099</ExpertLink>
-      <Name lang="en">Isolated brachycephaly</Name>
+      <Name lang="en">Non-syndromic bicoronal craniosynostosis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -91128,23 +91043,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="10368">
-      <OrphaCode>35098</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35098</ExpertLink>
-      <Name lang="en">Isolated plagiocephaly</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="10368" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="10375">
       <OrphaCode>35125</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35125</ExpertLink>
@@ -91981,7 +91879,7 @@
     <Disorder id="10367">
       <OrphaCode>35093</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=35093</ExpertLink>
-      <Name lang="en">Isolated scaphocephaly</Name>
+      <Name lang="en">Non-syndromic sagittal craniosynostosis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -92148,23 +92046,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="10345">
-      <OrphaCode>34527</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34527</ExpertLink>
-      <Name lang="en">Familial primary hypomagnesemia with normocalciuria and normocalcemia</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12456">
-            <OrphaCode>93626</OrphaCode>
-            <Name lang="en">Rare renal disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="10345" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="10346">
       <OrphaCode>34528</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34528</ExpertLink>
@@ -95806,7 +95687,7 @@
     <Disorder id="11913">
       <OrphaCode>89938</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89938</ExpertLink>
-      <Name lang="en">Infantile Bartter syndrome with sensorineural deafness</Name>
+      <Name lang="en">Bartter syndrome type 4</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -105547,7 +105428,7 @@
     <Disorder id="11619">
       <OrphaCode>84142</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=84142</ExpertLink>
-      <Name lang="en">Isaac syndrome</Name>
+      <Name lang="en">Isaacs syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -108587,23 +108468,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="12562">
-      <OrphaCode>94095</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=94095</ExpertLink>
-      <Name lang="en">Spondylocostal dysostosis-anal atresia-genitourinary malformation syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="12562" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="12563">
       <OrphaCode>94122</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=94122</ExpertLink>
@@ -109046,23 +108910,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="12776">
-      <OrphaCode>96152</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96152</ExpertLink>
-      <Name lang="en">Distal monosomy 20q</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="12776" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="12774">
       <OrphaCode>96150</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=96150</ExpertLink>
@@ -111837,7 +111684,7 @@
     <Disorder id="12438">
       <OrphaCode>93605</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93605</ExpertLink>
-      <Name lang="en">Classic Bartter syndrome</Name>
+      <Name lang="en">Bartter syndrome type 3</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -111851,23 +111698,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="12437">
-      <OrphaCode>93604</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93604</ExpertLink>
-      <Name lang="en">Antenatal Bartter syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12456">
-            <OrphaCode>93626</OrphaCode>
-            <Name lang="en">Rare renal disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="12437" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="14109">
       <OrphaCode>99092</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99092</ExpertLink>
@@ -118042,7 +117872,7 @@
     <Disorder id="28672">
       <OrphaCode>570371</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=570371</ExpertLink>
-      <Name lang="en">Transient antenatal Bartter syndrome</Name>
+      <Name lang="en">Bartter syndrome type 5</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12456">
@@ -122619,11 +122449,293 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="31442">
+      <OrphaCode>620371</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620371</ExpertLink>
+      <Name lang="en">Gitelman-like kidney tubulopathy due to mitochondrial DNA mutation</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31442" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31441">
+      <OrphaCode>620368</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620368</ExpertLink>
+      <Name lang="en">EGF-related primary hypomagnesemia with intellectual disability</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31441" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31440">
+      <OrphaCode>620363</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620363</ExpertLink>
+      <Name lang="en">Primary hypomagnesemia-generalized seizures-intellectual disability-obesity syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31440" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31438">
+      <OrphaCode>620217</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620217</ExpertLink>
+      <Name lang="en">Bartter syndrome type 1</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31438" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31439">
+      <OrphaCode>620220</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620220</ExpertLink>
+      <Name lang="en">Bartter syndrome type 2</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12456">
+            <OrphaCode>93626</OrphaCode>
+            <Name lang="en">Rare renal disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31439" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31436">
+      <OrphaCode>620205</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620205</ExpertLink>
+      <Name lang="en">Non-syndromic bicoronal and sagittal craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31436" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31437">
+      <OrphaCode>620212</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620212</ExpertLink>
+      <Name lang="en">Non-syndromic pansynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31437" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31434">
+      <OrphaCode>620192</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620192</ExpertLink>
+      <Name lang="en">Non-syndromic metopic and sagittal craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31434" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31435">
+      <OrphaCode>620198</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620198</ExpertLink>
+      <Name lang="en">Non-syndromic bicoronal and metopic craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31435" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31432">
+      <OrphaCode>620178</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620178</ExpertLink>
+      <Name lang="en">Non-syndromic bilambdoid craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31432" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31433">
+      <OrphaCode>620186</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620186</ExpertLink>
+      <Name lang="en">Non-syndromic unicoronal and sagittal craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31433" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31431">
+      <OrphaCode>620158</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620158</ExpertLink>
+      <Name lang="en">Non-syndromic non-specific multisutural craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31431" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31428">
+      <OrphaCode>620139</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620139</ExpertLink>
+      <Name lang="en">Non-syndromic unifrontosphenoidal craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31428" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31429">
+      <OrphaCode>620146</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620146</ExpertLink>
+      <Name lang="en">Non-syndromic unisquamosal craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31429" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31426">
+      <OrphaCode>620102</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620102</ExpertLink>
+      <Name lang="en">Non-syndromic unicoronal craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31426" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31427">
+      <OrphaCode>620113</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620113</ExpertLink>
+      <Name lang="en">Non-syndromic unilambdoid craniosynostosis</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31427" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="31388">
       <OrphaCode>618891</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
       <Name lang="en">Chronic neurovisceral acid sphingomyelinase deficiency</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="10507">
+            <OrphaCode>68367</OrphaCode>
+            <Name lang="en">Rare inborn errors of metabolism</Name>
+          </TargetDisorder>
+          <RootDisorder id="31388" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31370">
@@ -122694,5 +122806,61 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="31417">
+      <OrphaCode>619941</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619941</ExpertLink>
+      <Name lang="en">Immune deficiency due to impaired neutrophil phagocytosis and migration</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31416">
+      <OrphaCode>619367</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619367</ExpertLink>
+      <Name lang="en">SAMD9L-associated auto-inflammatory syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31419">
+      <OrphaCode>619953</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619953</ExpertLink>
+      <Name lang="en">Familial hyperinflammatory lymphoproliferative immunodeficiency</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31418">
+      <OrphaCode>619948</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619948</ExpertLink>
+      <Name lang="en">Early-onset autoimmunity-autoinflammation-immunodeficiency syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31423">
+      <OrphaCode>619979</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619979</ExpertLink>
+      <Name lang="en">Developmental delay-immunodeficiency-leukoencephalopathy-hypohomocysteinemia syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31422">
+      <OrphaCode>619972</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619972</ExpertLink>
+      <Name lang="en">CADINS disease</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31415">
+      <OrphaCode>619363</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619363</ExpertLink>
+      <Name lang="en">Neonatal onset-severe multisystem auto-inflammatory disease-increased IL-18</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31405">
+      <OrphaCode>619233</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619233</ExpertLink>
+      <Name lang="en">Hereditary persistence of fetal hemoglobin-intellectual disability syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
