commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
old mode 100644
new mode 100755
index f98bfe8..0851f63
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:40:12" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:40:51" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="7470">
+  <DisorderList count="7471">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -7344,7 +7344,7 @@
     <Disorder id="485">
       <OrphaCode>1880</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1880</ExpertLink>
-      <Name lang="en">Ebstein malformation</Name>
+      <Name lang="en">Ebstein malformation of the tricuspid valve</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -14336,7 +14336,7 @@
     <Disorder id="1007">
       <OrphaCode>528</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=528</ExpertLink>
-      <Name lang="en">Berardinelli-Seip congenital lipodystrophy</Name>
+      <Name lang="en">Congenital generalized lipodystrophy</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12996">
@@ -17222,23 +17222,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="16891">
-      <OrphaCode>139420</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139420</ExpertLink>
-      <Name lang="en">Secondary acute transverse myelitis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="16891" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="16892">
       <OrphaCode>139423</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139423</ExpertLink>
@@ -18818,7 +18801,7 @@
     <Disorder id="16705">
       <OrphaCode>137678</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137678</ExpertLink>
-      <Name lang="en">Czech dysplasia, metatarsal type</Name>
+      <Name lang="en">Spondyloepiphyseal dysplasia with metatarsal shortening</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12333">
@@ -19420,7 +19403,7 @@
     <Disorder id="1451">
       <OrphaCode>1200</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1200</ExpertLink>
-      <Name lang="en">Choanal atresia-hearing loss-cardiac defects-craniofacial dysmorphism syndrome</Name>
+      <Name lang="en">Burn-McKeown syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -26869,7 +26852,7 @@
     <Disorder id="17237">
       <OrphaCode>158687</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158687</ExpertLink>
-      <Name lang="en">Lethal acantholytic epidermolysis bullosa</Name>
+      <Name lang="en">Lethal acantholytic erosive disorder</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -26988,7 +26971,7 @@
     <Disorder id="17232">
       <OrphaCode>158668</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158668</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex due to plakophilin deficiency</Name>
+      <Name lang="en">Ectodermal dysplasia-skin fragility syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -28022,7 +28005,7 @@
     <Disorder id="2180">
       <OrphaCode>2325</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2325</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex with deafness, alopecia, anodontia/hypodontia and nail dystrophy</Name>
+      <Name lang="en">Epidermolysis bullosa simplex with anodontia/hypodontia</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -36315,7 +36298,7 @@
     <Disorder id="19879">
       <OrphaCode>261476</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=261476</ExpertLink>
-      <Name lang="en">Xp21 microdeletion syndrome</Name>
+      <Name lang="en">Xp21 deletion syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -45390,7 +45373,7 @@
     <Disorder id="3298">
       <OrphaCode>832</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=832</ExpertLink>
-      <Name lang="en">Succinyl-CoA:3-ketoacid CoA transferase deficiency</Name>
+      <Name lang="en">Succinyl-CoA:3-oxoacid CoA transferase deficiency</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="10507">
@@ -72541,7 +72524,7 @@
     <Disorder id="24234">
       <OrphaCode>468678</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468678</ExpertLink>
-      <Name lang="en">Intellectual disability-microcephaly-strabismus-behavioral abnormalities syndrome</Name>
+      <Name lang="en">White-Sutton syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -80870,7 +80853,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=508529</ExpertLink>
-      <Name lang="en">Generalized basal epidermolysis bullosa simplex with skin atrophy, scarring and hair loss</Name>
+      <Name lang="en">Intermediate epidermolysis bullosa simplex with cardiomyopathy</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -84366,7 +84349,7 @@
     <Disorder id="8620">
       <OrphaCode>2908</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2908</ExpertLink>
-      <Name lang="en">Kindler syndrome</Name>
+      <Name lang="en">Kindler epidermolysis bullosa</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -97763,57 +97746,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="11906">
-      <OrphaCode>89841</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89841</ExpertLink>
-      <Name lang="en">Centripetalis recessive dystrophic epidermolysis bullosa</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="11896">
-            <OrphaCode>89826</OrphaCode>
-            <Name lang="en">Rare skin disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="11906" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
-    <Disorder id="11905">
-      <OrphaCode>89840</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89840</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa, non-Herlitz type</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="11896">
-            <OrphaCode>89826</OrphaCode>
-            <Name lang="en">Rare skin disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="11905" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
-    <Disorder id="11904">
-      <OrphaCode>89839</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89839</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex superficialis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="11896">
-            <OrphaCode>89826</OrphaCode>
-            <Name lang="en">Rare skin disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="11904" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="11919">
       <OrphaCode>90001</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90001</ExpertLink>
@@ -98838,7 +98770,7 @@
     <Disorder id="11805">
       <OrphaCode>88618</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=88618</ExpertLink>
-      <Name lang="en">Psychomotor delay due to S-adenosylhomocysteine hydrolase deficiency</Name>
+      <Name lang="en">S-adenosylhomocysteine hydrolase deficiency</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -102685,7 +102617,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79401</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, Ogna type</Name>
+      <Name lang="en">PLEC-related epidermolysis bullosa simplex without extracutaneous involvement</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="11896">
@@ -109680,7 +109612,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95702</ExpertLink>
-      <Name lang="en">Cytomegalic congenital adrenal hypoplasia</Name>
+      <Name lang="en">X-linked adrenal hypoplasia congenita due to a NR0B1 point mutation</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12996">
@@ -112204,6 +112136,13 @@
       <DisorderDisorderAssociationList count="0">
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29740">
+      <OrphaCode>595356</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595356</ExpertLink>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12526">
       <OrphaCode>93968</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93968</ExpertLink>
@@ -112602,6 +112541,20 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29724">
+      <OrphaCode>595109</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595109</ExpertLink>
+      <Name lang="en">Atypical Timothy syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="29725">
+      <OrphaCode>595133</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595133</ExpertLink>
+      <Name lang="en">Perivascular Epithelioid Cell Neoplasm</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12508">
       <OrphaCode>93950</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93950</ExpertLink>
@@ -112619,6 +112572,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29722">
+      <OrphaCode>595098</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595098</ExpertLink>
+      <Name lang="en">Timothy syndrome type 1</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12507">
       <OrphaCode>93947</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93947</ExpertLink>
@@ -112636,6 +112596,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="29723">
+      <OrphaCode>595105</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=595105</ExpertLink>
+      <Name lang="en">Timothy syndrome type 2</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="12506">
       <OrphaCode>93946</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93946</ExpertLink>
@@ -119587,7 +119554,7 @@
     <Disorder id="15005">
       <OrphaCode>104075</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=104075</ExpertLink>
-      <Name lang="en">Adenocarcinoma of the small instestine</Name>
+      <Name lang="en">Adenocarcinoma of the small intestine</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="19592">
