commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml
index 5ed39d3..9f0fcb4 100644
--- a/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
+++ b/Rare diseases and classifications/Linearisation of disorders/en_product7.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:58:46" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:55:54" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="7232">
+  <DisorderList count="7227">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -1266,40 +1266,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="17541">
-      <OrphaCode>163898</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=163898</ExpertLink>
-      <Name lang="en">Classic paraneoplastic limbic encephalitis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="17541" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
-    <Disorder id="17543">
-      <OrphaCode>163908</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=163908</ExpertLink>
-      <Name lang="en">Limbic encephalitis with LGI1 antibodies</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="17543" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="64">
       <OrphaCode>551</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=551</ExpertLink>
@@ -1453,23 +1419,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="17547">
-      <OrphaCode>163924</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=163924</ExpertLink>
-      <Name lang="en">Non-herpetic acute limbic encephalitis</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="17547" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="75">
       <OrphaCode>614</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=614</ExpertLink>
@@ -3649,7 +3598,7 @@
     <Disorder id="244">
       <OrphaCode>207</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=207</ExpertLink>
-      <Name lang="en">Crouzon disease</Name>
+      <Name lang="en">Crouzon syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -13441,7 +13390,7 @@
     <Disorder id="946">
       <OrphaCode>3027</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3027</ExpertLink>
-      <Name lang="en">Caudal regression sequence</Name>
+      <Name lang="en">Caudal regression syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -16719,40 +16668,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="1284">
-      <OrphaCode>965</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=965</ExpertLink>
-      <Name lang="en">Acromegaloid facial appearance syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="1284" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
-    <Disorder id="1285">
-      <OrphaCode>966</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=966</ExpertLink>
-      <Name lang="en">Hypertrichosis-acromegaloid facial appearance syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="1285" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="16888">
       <OrphaCode>139411</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139411</ExpertLink>
@@ -20955,7 +20870,7 @@
     <Disorder id="1544">
       <OrphaCode>1321</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1321</ExpertLink>
-      <Name lang="en">Camptodactyly-fibrous tissue hyperplasia-skeletal dysplasia syndrome</Name>
+      <Name lang="en">Camptodactyly-fibrous tissue hyperplasia-skeletal anomalies syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -22397,23 +22312,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1533</ExpertLink>
-      <Name lang="en">Craniosynostosis-fibular aplasia syndrome</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="12469">
-            <OrphaCode>93890</OrphaCode>
-            <Name lang="en">Rare developmental defect during embryogenesis</Name>
-          </TargetDisorder>
-          <RootDisorder id="1697" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="16993">
       <OrphaCode>140896</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140896</ExpertLink>
@@ -23148,7 +23046,7 @@
     <Disorder id="1684">
       <OrphaCode>1517</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1517</ExpertLink>
-      <Name lang="en">Hypertrichotic osteochondrodysplasia, Cantu type</Name>
+      <Name lang="en">Cantú syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -32906,7 +32804,7 @@
     <Disorder id="2497">
       <OrphaCode>661</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=661</ExpertLink>
-      <Name lang="en">Ondine syndrome</Name>
+      <Name lang="en">Congenital central hypoventilation syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -43681,23 +43579,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="20372">
-      <OrphaCode>276402</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=276402</ExpertLink>
-      <Name lang="en">Limbic encephalitis with caspr2 antibodies</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="20372" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="2900">
       <OrphaCode>3242</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3242</ExpertLink>
@@ -47577,7 +47458,7 @@
     <Disorder id="18763">
       <OrphaCode>213726</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=213726</ExpertLink>
-      <Name lang="en">Papillary carcinoma of the corpus uteri</Name>
+      <Name lang="en">Serous carcinoma of the corpus uteri</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="19592">
@@ -48648,7 +48529,7 @@
     <Disorder id="18827">
       <OrphaCode>217253</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=217253</ExpertLink>
-      <Name lang="en">Limbic encephalitis with NMDA receptor antibodies</Name>
+      <Name lang="en">NMDA receptor encephalitis</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13024">
@@ -55462,23 +55343,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329341</ExpertLink>
-      <Name lang="en">Limbic encephalitis with DPP6 antibodies</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="21931" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=329228</ExpertLink>
@@ -65444,12 +65308,12 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314555</ExpertLink>
-      <Name lang="en">Facial dysmorphism-severe myopia-osteopenia-intellectual disability-dental anomalies syndrome</Name>
+      <Name lang="en">Facial dysmorphism-ocular anomalies-osteopenia-intellectual disability-dental anomalies syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
-          <TargetDisorder id="12333">
-            <OrphaCode>93419</OrphaCode>
-            <Name lang="en">Rare bone disease</Name>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
           </TargetDisorder>
           <RootDisorder id="21494" cycle="true"/>
           <DisorderDisorderAssociationType id="21485">
@@ -69334,23 +69198,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="23782">
-      <OrphaCode>459353</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=459353</ExpertLink>
-      <Name lang="en">C1 inhibitor deficiency</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13022">
-            <OrphaCode>98004</OrphaCode>
-            <Name lang="en">Rare immune disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="23782" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="23767">
       <OrphaCode>459033</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=459033</ExpertLink>
@@ -80727,7 +80574,7 @@
     <Disorder id="26069">
       <OrphaCode>502305</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=502305</ExpertLink>
-      <Name lang="en">Cochleovestibular dysplasia</Name>
+      <Name lang="en">Cochleovestibular malformation</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="13054">
@@ -81829,23 +81676,6 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=498700</ExpertLink>
-      <Name lang="en">Limbic encephalitis with neurexin-3 antibodies</Name>
-      <DisorderDisorderAssociationList count="1">
-        <DisorderDisorderAssociation>
-          <TargetDisorder id="13024">
-            <OrphaCode>98006</OrphaCode>
-            <Name lang="en">Rare neurologic disease</Name>
-          </TargetDisorder>
-          <RootDisorder id="25936" cycle="true"/>
-          <DisorderDisorderAssociationType id="21485">
-            <Name lang="en">Preferential parent</Name>
-          </DisorderDisorderAssociationType>
-        </DisorderDisorderAssociation>
-      </DisorderDisorderAssociationList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=874</ExpertLink>
@@ -106737,7 +106567,7 @@
     <Disorder id="12896">
       <OrphaCode>97340</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=97340</ExpertLink>
-      <Name lang="en">Hunter-McAlpine craniosynostosis</Name>
+      <Name lang="en">Hunter-McAlpine syndrome</Name>
       <DisorderDisorderAssociationList count="1">
         <DisorderDisorderAssociation>
           <TargetDisorder id="12469">
@@ -113330,6 +113160,13 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="14277">
+      <OrphaCode>99704</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=99704</ExpertLink>
+      <Name lang="en">Early-onset obesity-hyperphagia-severe developmental delay syndrome</Name>
+      <DisorderDisorderAssociationList count="0">
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=589905</ExpertLink>
@@ -122449,6 +122286,23 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="31449">
+      <OrphaCode>621758</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=621758</ExpertLink>
+      <Name lang="en">Fibrosis-neurodegeneration-cerebral angiomatosis syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12974">
+            <OrphaCode>97955</OrphaCode>
+            <Name lang="en">Rare respiratory disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31449" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="31442">
       <OrphaCode>620371</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620371</ExpertLink>
@@ -122721,6 +122575,57 @@
         </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
+    <Disorder id="31460">
+      <OrphaCode>622925</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=622925</ExpertLink>
+      <Name lang="en">X-linked severe syndromic thoracic aortic aneurysm and dissection</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31460" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31461">
+      <OrphaCode>622934</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=622934</ExpertLink>
+      <Name lang="en">SBDS-related severe neonatal spondylometaphyseal dysplasia</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12333">
+            <OrphaCode>93419</OrphaCode>
+            <Name lang="en">Rare bone disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31461" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
+    <Disorder id="31457">
+      <OrphaCode>622099</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=622099</ExpertLink>
+      <Name lang="en">Superior mesenteric artery syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12954">
+            <OrphaCode>97935</OrphaCode>
+            <Name lang="en">Rare gastroenterologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31457" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
+      </DisorderDisorderAssociationList>
+    </Disorder>
     <Disorder id="31388">
       <OrphaCode>618891</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=618891</ExpertLink>
@@ -122810,56 +122715,136 @@
       <OrphaCode>619941</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619941</ExpertLink>
       <Name lang="en">Immune deficiency due to impaired neutrophil phagocytosis and migration</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13022">
+            <OrphaCode>98004</OrphaCode>
+            <Name lang="en">Rare immune disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31417" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31416">
       <OrphaCode>619367</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619367</ExpertLink>
-      <Name lang="en">SAMD9L-associated auto-inflammatory syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">SAMD9L-associated autoinflammatory syndrome</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31416" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31419">
       <OrphaCode>619953</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619953</ExpertLink>
       <Name lang="en">Familial hyperinflammatory lymphoproliferative immunodeficiency</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13022">
+            <OrphaCode>98004</OrphaCode>
+            <Name lang="en">Rare immune disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31419" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31418">
       <OrphaCode>619948</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619948</ExpertLink>
       <Name lang="en">Early-onset autoimmunity-autoinflammation-immunodeficiency syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13022">
+            <OrphaCode>98004</OrphaCode>
+            <Name lang="en">Rare immune disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31418" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31423">
       <OrphaCode>619979</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619979</ExpertLink>
       <Name lang="en">Developmental delay-immunodeficiency-leukoencephalopathy-hypohomocysteinemia syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13022">
+            <OrphaCode>98004</OrphaCode>
+            <Name lang="en">Rare immune disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31423" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31422">
       <OrphaCode>619972</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619972</ExpertLink>
       <Name lang="en">CADINS disease</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13022">
+            <OrphaCode>98004</OrphaCode>
+            <Name lang="en">Rare immune disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31422" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31415">
       <OrphaCode>619363</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619363</ExpertLink>
-      <Name lang="en">Neonatal onset-severe multisystem auto-inflammatory disease-increased IL-18</Name>
-      <DisorderDisorderAssociationList count="0">
+      <Name lang="en">Neonatal-onset severe multisystemic autoinflammatory disease with increased IL18</Name>
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="13041">
+            <OrphaCode>98023</OrphaCode>
+            <Name lang="en">Rare systemic or rheumatologic disease</Name>
+          </TargetDisorder>
+          <RootDisorder id="31415" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
     <Disorder id="31405">
       <OrphaCode>619233</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619233</ExpertLink>
       <Name lang="en">Hereditary persistence of fetal hemoglobin-intellectual disability syndrome</Name>
-      <DisorderDisorderAssociationList count="0">
+      <DisorderDisorderAssociationList count="1">
+        <DisorderDisorderAssociation>
+          <TargetDisorder id="12469">
+            <OrphaCode>93890</OrphaCode>
+            <Name lang="en">Rare developmental defect during embryogenesis</Name>
+          </TargetDisorder>
+          <RootDisorder id="31405" cycle="true"/>
+          <DisorderDisorderAssociationType id="21485">
+            <Name lang="en">Preferential parent</Name>
+          </DisorderDisorderAssociationType>
+        </DisorderDisorderAssociation>
       </DisorderDisorderAssociationList>
     </Disorder>
   </DisorderList>
