commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare diseases natural history/cs_product9_ages.xml b/Epidemiological data/Rare diseases natural history/cs_product9_ages.xml
index 4e7996d..85f27c5 100644
--- a/Epidemiological data/Rare diseases natural history/cs_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/cs_product9_ages.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<JDBOR date="2021-11-01 04:51:06" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6633">
+<JDBOR date="2021-12-01 04:55:31" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=166024</ExpertLink>
@@ -12687,11 +12680,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=2346</ExpertLink>
       <Name lang="cs">Angioosteohypertrofický syndrom</Name>
-      <DisorderType id="21394">
-        <Name lang="cs">Onemocnění</Name>
+      <DisorderType id="21436">
+        <Name lang="cs">Klinická skupina</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="cs">Onemocnění</Name>
+      <DisorderGroup id="36540">
+        <Name lang="cs">Skupina onemocnění</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="3">
         <AverageAgeOfOnset id="23536">
@@ -20778,7 +20771,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=183678</ExpertLink>
-      <Name lang="cs">Heřmanského-Pudlákův syndrom s neutropenií</Name>
+      <Name lang="cs">Heřmanského-Pudlákův syndrom způsobený deficitem AP-3</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -21023,10 +21016,13 @@
       <DisorderGroup id="36547">
         <Name lang="cs">Onemocnění</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="cs">Dospělý</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="cs">Starší</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23634">
@@ -80048,7 +80044,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=231512</ExpertLink>
-      <Name lang="cs">Heřmanského-Pudlákův syndrom bez plicní fibrózy</Name>
+      <Name lang="cs">Heřmanského-Pudlákův syndrom způsobený deficitem BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -80135,7 +80131,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=231500</ExpertLink>
-      <Name lang="cs">Heřmanského-Pudlákův syndrom s plicní fibrózou</Name>
+      <Name lang="cs">Heřmanského-Pudlákův syndrom způsobený deficitem BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -98233,7 +98229,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=314555</ExpertLink>
-      <Name lang="cs">Syndrom zahrnující kraniofaciální dysplázii a osteopenii</Name>
+      <Name lang="cs">Syndrom zahrnující obličejový dysmorfismus, těžkou myopii, osteopenii, mentální retardaci a zubní anomálie</Name>
       <DisorderType id="21401">
         <Name lang="cs">Malformační syndrom</Name>
       </DisorderType>
@@ -137809,7 +137805,7 @@
     <Disorder id="10332">
       <OrphaCode>34149</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=34149</ExpertLink>
-      <Name lang="cs">Autozomálně dominantně dědičná medulární cystická choroba ledvin s nebo bez hyperurikémie</Name>
+      <Name lang="cs">Autozomálně dominantně tubulointersticiální onemocnění ledvin</Name>
       <DisorderType id="21394">
         <Name lang="cs">Onemocnění</Name>
       </DisorderType>
@@ -138365,7 +138361,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=35062</ExpertLink>
-      <Name lang="cs">Idiopatická diseminovaná cytomegalovirová infekce</Name>
+      <Name lang="cs">Těžká diseminovaná cytomegalovirová infekce u imunokompetentních pacientů</Name>
       <DisorderType id="21394">
         <Name lang="cs">Onemocnění</Name>
       </DisorderType>
@@ -144302,11 +144298,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=90308</ExpertLink>
       <Name lang="cs">Klippelova-Trenaunayova nemoc</Name>
-      <DisorderType id="21450">
-        <Name lang="cs">Klinický podtyp</Name>
+      <DisorderType id="21394">
+        <Name lang="cs">Onemocnění</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="cs">Podtyp onemocnění</Name>
+      <DisorderGroup id="36547">
+        <Name lang="cs">Onemocnění</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -144331,11 +144327,11 @@
       <OrphaCode>90307</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=90307</ExpertLink>
       <Name lang="cs">Parkes Weberův syndrom</Name>
-      <DisorderType id="21450">
-        <Name lang="cs">Klinický podtyp</Name>
+      <DisorderType id="21394">
+        <Name lang="cs">Onemocnění</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="cs">Podtyp onemocnění</Name>
+      <DisorderGroup id="36547">
+        <Name lang="cs">Onemocnění</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -146108,7 +146104,7 @@
     <Disorder id="11881">
       <OrphaCode>88949</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=88949</ExpertLink>
-      <Name lang="cs">Autozomálně dominantní dřeňová cystická nemoc ledvin bez hyperurikémie</Name>
+      <Name lang="cs">Autozomálně dominantní tubulointersticiální onemocnění ledvin související s MUC1</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -146128,7 +146124,7 @@
     <Disorder id="11882">
       <OrphaCode>88950</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=88950</ExpertLink>
-      <Name lang="cs">Autozomálně dominantní dřeňová cystická nemoc ledvin s hyperurikémií</Name>
+      <Name lang="cs">Autozomálně dominantní tubulointersticiální onemocnění ledvin související s UMOD</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -147121,7 +147117,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=93111</ExpertLink>
-      <Name lang="cs">Syndrom renálních cyst a diabetes</Name>
+      <Name lang="cs">Autozomálně dominantní tubulointersticiální onemocnění ledvin související s HNF1B</Name>
       <DisorderType id="21450">
         <Name lang="cs">Klinický podtyp</Name>
       </DisorderType>
@@ -153827,7 +153823,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=85279</ExpertLink>
-      <Name lang="cs">Syndromická mentální retardace způsobená mutací JARID1C</Name>
+      <Name lang="cs">Syndromická X-vázaná mentální retardace související s KDM5C</Name>
       <DisorderType id="21401">
         <Name lang="cs">Malformační syndrom</Name>
       </DisorderType>
@@ -158556,7 +158552,7 @@
     <Disorder id="30683">
       <OrphaCode>610569</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=610569</ExpertLink>
-      <Name lang="en">KIAA1109-related early lethal congenital brain mlaformations-arthrogryposis syndrome</Name>
+      <Name lang="en">KIAA1109-related early lethal congenital brain malformations-arthrogryposis syndrome</Name>
       <DisorderType id="21394">
         <Name lang="cs">Onemocnění</Name>
       </DisorderType>
@@ -163189,6 +163185,29 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=599373</ExpertLink>
+      <Name lang="en">STXBP1-related encephalopathy</Name>
+      <DisorderType id="21394">
+        <Name lang="cs">Onemocnění</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="cs">Onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23515">
+          <Name lang="cs">Novorozenecké</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="cs">Data nejsou k dispozici</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=93403</ExpertLink>
@@ -165734,6 +165753,61 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=592564</ExpertLink>
+      <Name lang="en">GNAO1-related developmental delay-seizures-movement disorder spectrum</Name>
+      <DisorderType id="21394">
+        <Name lang="cs">Onemocnění</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="cs">Onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="cs">Dětství</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="cs">Rané dětství</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="cs">Novorozenecké</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="cs">Data nejsou k dispozici</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=592570</ExpertLink>
+      <Name lang="en">TRAF7-associated heart defect-digital anomalies-facial dysmorphism-motor and speech delay syndrome</Name>
+      <DisorderType id="21401">
+        <Name lang="cs">Malformační syndrom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="cs">Onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="cs">Prenatální</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="cs">Novorozenecké</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="cs">Data nejsou k dispozici</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14291">
       <OrphaCode>99718</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=99718</ExpertLink>
@@ -165836,6 +165910,172 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=589821</ExpertLink>
+      <Name lang="en">Congenital-onset Steinert myotonic dystrophy</Name>
+      <DisorderType id="21450">
+        <Name lang="cs">Klinický podtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="cs">Podtyp onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="cs">Prenatální</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="cs">Novorozenecké</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="cs">Adolescent</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="cs">Dospělý</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="cs">rané dětství</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="cs">dětský</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="cs">pozdní dětství</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="cs">Dospívající</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=589827</ExpertLink>
+      <Name lang="en">Juvenile-onset Steinert myotonic dystrophy</Name>
+      <DisorderType id="21450">
+        <Name lang="cs">Klinický podtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="cs">Podtyp onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="cs">Dětství</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="cs">Adolescent</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="cs">Dospělý</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="cs">rané dětství</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="cs">pozdní dětství</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="cs">Dospívající</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=589824</ExpertLink>
+      <Name lang="en">Childhood-onset Steinert myotonic dystrophy</Name>
+      <DisorderType id="21450">
+        <Name lang="cs">Klinický podtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="cs">Podtyp onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="cs">Dětství</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="cs">Rané dětství</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="cs">Adolescent</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="cs">Dospělý</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="cs">rané dětství</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="cs">pozdní dětství</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="cs">Dospívající</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=589833</ExpertLink>
+      <Name lang="en">Late-onset Steinert myotonic dystrophy</Name>
+      <DisorderType id="21450">
+        <Name lang="cs">Klinický podtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="cs">Podtyp onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23550">
+          <Name lang="cs">Starší</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="cs">Starší</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=589830</ExpertLink>
+      <Name lang="en">Adult-onset Steinert myotonic dystrophy</Name>
+      <DisorderType id="21450">
+        <Name lang="cs">Klinický podtyp</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="cs">Podtyp onemocnění</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="cs">Dospělý</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="cs">Dospělý</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="cs">Starší</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="cs">Dospívající</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=589905</ExpertLink>
@@ -169317,7 +169557,7 @@
     <Disorder id="14067">
       <OrphaCode>99050</OrphaCode>
       <ExpertLink lang="cs">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=cs&amp;Expert=99050</ExpertLink>
-      <Name lang="cs">Plicnice vycházející z aorty</Name>
+      <Name lang="cs">Plicní tepna vycházející z aorty</Name>
       <DisorderType id="21415">
         <Name lang="cs">Morfologická anomálie</Name>
       </DisorderType>
