commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Epidemiological data/Rare diseases natural history/de_product9_ages.xml b/Epidemiological data/Rare diseases natural history/de_product9_ages.xml
old mode 100644
new mode 100755
index 8fde4a1..5080494
--- a/Epidemiological data/Rare diseases natural history/de_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/de_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:41:49" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:42:50" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -13718,7 +13718,7 @@
     <Disorder id="595">
       <OrphaCode>3320</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3320</ExpertLink>
-      <Name lang="de">Thrombozytopenie - Radiusaplasie</Name>
+      <Name lang="de">Thrombozytopenie-Radiusaplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -29282,7 +29282,7 @@
     <Disorder id="1470">
       <OrphaCode>1226</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1226</ExpertLink>
-      <Name lang="de">Bamforth-Syndrom</Name>
+      <Name lang="de">Bamforth-Lazarus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -32905,7 +32905,7 @@
     <Disorder id="1773">
       <OrphaCode>1757</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1757</ExpertLink>
-      <Name lang="de">Fibuläre Dimelie - Diplopodie</Name>
+      <Name lang="de">Fibuläre Dimelie-Diplopodie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -48666,7 +48666,7 @@
     <Disorder id="2436">
       <OrphaCode>2669</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2669</ExpertLink>
-      <Name lang="de">Nephrose - Schwerhörigkeit - Harnwegsanomalien - Fingerfehlbildungen</Name>
+      <Name lang="de">Nephrose-Schwerhörigkeit-Harnwegsanomalien-Fingerfehlbildungen-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -51531,7 +51531,7 @@
     <Disorder id="2556">
       <OrphaCode>2815</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2815</ExpertLink>
-      <Name lang="de">Spastische Paraparese - Schwerhörigkeit</Name>
+      <Name lang="de">Spastische Paraparese-Schwerhörigkeit-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -53238,11 +53238,11 @@
       <OrphaCode>261144</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=261144</ExpertLink>
       <Name lang="de">Mikrodeletionssyndrom 14q12</Name>
-      <DisorderType id="21401">
-        <Name lang="de">Fehlbildungs-Syndrom</Name>
+      <DisorderType id="21450">
+        <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36554">
+        <Name lang="de">Subtyp der Störung</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -57338,7 +57338,7 @@
     <Disorder id="2599">
       <OrphaCode>2866</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2866</ExpertLink>
-      <Name lang="de">Kleinwuchs - Schwerhörigkeit - neutrophile Funktionsstörung - Dysmorphien</Name>
+      <Name lang="de">Kleinwuchs-Schwerhörigkeit-neutrophile Funktionsstörung-Dysmorphien-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -60388,7 +60388,7 @@
     <Disorder id="2985">
       <OrphaCode>3368</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3368</ExpertLink>
-      <Name lang="de">Trigonozephalie - bifide Nase - akrale Anomalien</Name>
+      <Name lang="de">Trigonozephalie-bifide Nase-akrale Anomalien-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -61978,7 +61978,7 @@
     <Disorder id="3053">
       <OrphaCode>3319</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3319</ExpertLink>
-      <Name lang="de">Thrombozytopenie, amegakaryozytische, kongenitale</Name>
+      <Name lang="de">Angeborenen amegakaryozytäre Thrombozytopenie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -62734,11 +62734,11 @@
       <OrphaCode>280315</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280315</ExpertLink>
       <Name lang="de">Autoimmun-Pankreatitis Typ 2</Name>
-      <DisorderType id="21450">
-        <Name lang="de">Klinischer Subtyp</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -68008,7 +68008,7 @@
     <Disorder id="3154">
       <OrphaCode>1192</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1192</ExpertLink>
-      <Name lang="de">Atherosklerose - Schwerhörigkeit - Diabetes - Epilepsie - Nephropathie</Name>
+      <Name lang="de">Atherosklerose-Schwerhörigkeit-Diabetes-Epilepsie-Nephropathie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68612,7 +68612,7 @@
     <Disorder id="3183">
       <OrphaCode>3051</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3051</ExpertLink>
-      <Name lang="de">Intelligenzminderung-spärliches Haar-Brachydaktylie-Syndrom</Name>
+      <Name lang="de">Nicolaides-Baraitser-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -69733,7 +69733,7 @@
     <Disorder id="3118">
       <OrphaCode>2666</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2666</ExpertLink>
-      <Name lang="de">Nephronophthise, adulte familiäre - spastische Tetraparese</Name>
+      <Name lang="de">Adulte familiäre Nephronophthise-spastische Tetraparese-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -121097,7 +121097,7 @@
     <Disorder id="8691">
       <OrphaCode>465</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=465</ExpertLink>
-      <Name lang="de">Plasminogen-Aktivator-Inhibitor-1-Mangel, kongenitaler</Name>
+      <Name lang="de">Plasminogenaktivator-Inhibitor Typ 1-Mangel, kongenitaler</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -163546,11 +163546,11 @@
       <OrphaCode>99361</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=99361</ExpertLink>
       <Name lang="de">Schilddrüsenkarzinom, medulläres, familiäres</Name>
-      <DisorderType id="21450">
-        <Name lang="de">Klinischer Subtyp</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
@@ -164164,11 +164164,11 @@
       <OrphaCode>98813</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=98813</ExpertLink>
       <Name lang="de">Hypohidrotische ektodermale Dysplasie mit Immundefekt</Name>
-      <DisorderType id="21450">
-        <Name lang="de">Klinischer Subtyp</Name>
+      <DisorderType id="21394">
+        <Name lang="de">Krankheit</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -167343,8 +167343,8 @@
       <OrphaCode>98791</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=98791</ExpertLink>
       <Name lang="de">Alpha-Thalassämie-Intelligenzminderung-Syndrom, gekoppelt an Chr. 16</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21401">
+        <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
@@ -168831,11 +168831,11 @@
       <OrphaCode>103919</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=103919</ExpertLink>
       <Name lang="de">Autoimmun-Pankreatitis</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
+      <DisorderType id="21436">
+        <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
+      <DisorderGroup id="36540">
+        <Name lang="de">Gruppe von Störungen</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
