commit bbf94d552a3c459862632b82d2e2166d94fb6b4e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Dec 1 15:14:38 2020 +0100

    DEC 2020

diff --git a/Epidemiological data/Rare diseases natural history/de_product9_ages.xml b/Epidemiological data/Rare diseases natural history/de_product9_ages.xml
index c75b6c1..e6928fd 100644
--- a/Epidemiological data/Rare diseases natural history/de_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/de_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-11-01 04:36:29" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2020-12-01 04:39:26" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6585">
+  <DisorderList count="6586">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166024</ExpertLink>
@@ -784,7 +784,7 @@
     <Disorder id="17618">
       <OrphaCode>166100</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166100</ExpertLink>
-      <Name lang="de">Stickler-Syndrom Typ 3</Name>
+      <Name lang="de">Oto-spondylo-megaepiphysäre Dysplasie, autosomal-dominante</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -5411,7 +5411,7 @@
     <Disorder id="238">
       <OrphaCode>126</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=126</ExpertLink>
-      <Name lang="de">Blepharophimose - Epicanthus inversus - Ptosis</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -7565,7 +7565,7 @@
     <Disorder id="17819">
       <OrphaCode>169095</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=169095</ExpertLink>
-      <Name lang="de">Alymphoide zystische Thymus-Dysgenesie</Name>
+      <Name lang="de">Immundefekt, kombinierter schwerer, durch FOXN1-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -8303,7 +8303,7 @@
     <Disorder id="17831">
       <OrphaCode>169160</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=169160</ExpertLink>
-      <Name lang="de">Immundefekt, kombinierter schwerer, T- B+, durch CD3delta/CD3epsilon/CD3dzeta-Defekt</Name>
+      <Name lang="de">Immundefekt, kombinierter schwerer, T- B+, durch CD3delta/CD3epsilon/CD3zeta-Defekt</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -11956,14 +11956,23 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="de">Vorgeburtlich</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="de">Neugeborenenzeit</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="de">Alle Alter</Name>
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="de">Erwachsene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="de">Ältere Erwachsene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="de">junges Erwachsenenalter</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="3">
@@ -13617,7 +13626,10 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="de">Vorgeburtlich</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="de">Neugeborenenzeit</Name>
         </AverageAgeOfOnset>
@@ -13627,16 +13639,13 @@
           <Name lang="de">Normale Lebenserwartung</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="3">
+      <TypeOfInheritanceList count="2">
         <TypeOfInheritance id="23410">
           <Name lang="de">Autosomal-dominant</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="de">Autosomal-rezessiv</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23480">
-          <Name lang="de">Unbekannt</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="589">
@@ -15688,13 +15697,16 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="3">
         <AverageAgeOfOnset id="23536">
           <Name lang="de">Jugendalter</Name>
         </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23543">
           <Name lang="de">Erwachsenenalter</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="de">Ältere Erwachsene</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
@@ -18242,7 +18254,7 @@
     <Disorder id="18409">
       <OrphaCode>199354</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=199354</ExpertLink>
-      <Name lang="de">CARASIL</Name>
+      <Name lang="de">Zerebrale Arteriopathie mit subkortikalen Infarkten und Leukoenzephalopathie, autosomal-rezessiv</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -20712,7 +20724,7 @@
     <Disorder id="18313">
       <OrphaCode>183713</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=183713</ExpertLink>
-      <Name lang="de">Pyogene Bakterieninfektion durch MyD88-Mangel</Name>
+      <Name lang="de">Suszeptibilität für Bakterieninfektion durch TLR Signalweg-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -21961,7 +21973,7 @@
     <Disorder id="1001">
       <OrphaCode>136</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=136</ExpertLink>
-      <Name lang="de">CADASIL</Name>
+      <Name lang="de">Zerebrale autosomal-dominante Arteriopathie mit subkortikalen Infarkten und Leukenzephalopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -22019,7 +22031,7 @@
     <Disorder id="1007">
       <OrphaCode>528</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=528</ExpertLink>
-      <Name lang="de">Lipodystrophie Typ Berardinelli</Name>
+      <Name lang="de">Lipodystrophie, kongenitale, Typ Berardinelli</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -34438,7 +34450,7 @@
     <Disorder id="1757">
       <OrphaCode>1671</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1671</ExpertLink>
-      <Name lang="de">Diastematomyelie</Name>
+      <Name lang="de">Split-Cord-Malformation Typ I</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -39586,7 +39598,7 @@
     <Disorder id="17164">
       <OrphaCode>157965</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=157965</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, spondylo-cheiro dysplastische Form</Name>
+      <Name lang="de">SLC39A13-assoziiertes spondylodysplastisches Ehlers-Danlos-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -40902,7 +40914,7 @@
     <Disorder id="2200">
       <OrphaCode>2348</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2348</ExpertLink>
-      <Name lang="de">Lipodystrophie, familiäre partielle, Typ Dunningan</Name>
+      <Name lang="de">Lipodystrophie, familiäre partielle, Typ  Dunnigan</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -43218,6 +43230,29 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="19474">
+      <OrphaCode>244275</OrphaCode>
+      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=244275</ExpertLink>
+      <Name lang="de">De-novo thrombotische Mikroangiopathie nach Nierentransplantation</Name>
+      <DisorderType id="21429">
+        <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="de">Störung</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23557">
+          <Name lang="de">Alle Altersgruppen</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23648">
+          <Name lang="de">Alle Alter</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="2259">
       <OrphaCode>2438</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2438</ExpertLink>
@@ -45778,7 +45813,7 @@
     <Disorder id="19684">
       <OrphaCode>251870</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=251870</ExpertLink>
-      <Name lang="de">Primitiver neuroektodermaler Tumor des Zentralnervensystems</Name>
+      <Name lang="de">ZNS-Tumor, embryonaler</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -47140,7 +47175,7 @@
     <Disorder id="2121">
       <OrphaCode>2251</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2251</ExpertLink>
-      <Name lang="de">Daumenfehlbildung - Alopezie - Pigmentanomalien</Name>
+      <Name lang="de">Daumenfehlbildung-Alopezie-Pigmentanomalie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -54424,7 +54459,7 @@
     <Disorder id="19882">
       <OrphaCode>261501</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=261501</ExpertLink>
-      <Name lang="de">Norrie-Syndrom, atypisches, durch Monosomie Xp11.3</Name>
+      <Name lang="de">Norrie-Syndrom, atypisches, durch Mikrodeletion Xp11.3</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -66169,7 +66204,7 @@
     <Disorder id="2908">
       <OrphaCode>3253</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3253</ExpertLink>
-      <Name lang="de">Zlotogora-Ogur-Syndrom</Name>
+      <Name lang="de">Lippen-Kiefer-Gaumen-Spalte-ektodermale Dysplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -66513,7 +66548,7 @@
     <Disorder id="3279">
       <OrphaCode>572</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572</ExpertLink>
-      <Name lang="de">Immundefekt durch HLA-Klasse II-Expressionsdefekt</Name>
+      <Name lang="de">Immundefekt durch MHC Klasse II-Expressionsdefekt</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -73574,7 +73609,7 @@
     <Disorder id="18845">
       <OrphaCode>217330</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=217330</ExpertLink>
-      <Name lang="de">Hyperurikämie - Anämie - Nierenversagen</Name>
+      <Name lang="de">Nierenkrankheit, tubulointerstitielle, autosomal-dominante, REN-assoziierte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -74655,7 +74690,7 @@
     <Disorder id="3443">
       <OrphaCode>2041</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2041</ExpertLink>
-      <Name lang="de">Koronararterienfisteln</Name>
+      <Name lang="de">Koronarfistel</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -77267,7 +77302,7 @@
     <Disorder id="3735">
       <OrphaCode>2126</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2126</ExpertLink>
-      <Name lang="de">Tumor, fibröser solitärer</Name>
+      <Name lang="de">Solitärer fibröser Tumor/Hämangioperizytom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -77556,7 +77591,7 @@
     <Disorder id="3713">
       <OrphaCode>2542</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2542</ExpertLink>
-      <Name lang="de">Anophthalmie/Mikrophthalmie-Syndrom, isoliertes</Name>
+      <Name lang="de">Mikrophthalmie-Anophthalmie-Kolobom, isoliert</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -90916,7 +90951,7 @@
     <Disorder id="22195">
       <OrphaCode>357008</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=357008</ExpertLink>
-      <Name lang="de">Hämolytisch-urämisches Syndrom, atypische From mit DGKE-Mangel</Name>
+      <Name lang="de">Hämolytisch-urämisches Syndrom mit DGKE-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -100539,10 +100574,16 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="de">Vorgeburtlich</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23522">
           <Name lang="de">Kleinkindalter</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="de">Neugeborenenzeit</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23655">
@@ -110646,7 +110687,7 @@
     <Disorder id="23367">
       <OrphaCode>438178</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=438178</ExpertLink>
-      <Name lang="de">Schwere Intelligenzminderung-Epilepsie-Katarakt-Syndrom durch Acyl-CoA-Reduktase 1-Mangel</Name>
+      <Name lang="de">Fettsäure Acyl-CoA-Reduktase 1-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -119212,7 +119253,10 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="de">Vorgeburtlich</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="de">Neugeborenenzeit</Name>
         </AverageAgeOfOnset>
@@ -125657,7 +125701,7 @@
     <Disorder id="10999">
       <OrphaCode>71277</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=71277</ExpertLink>
-      <Name lang="de">Enzephalopathie durch GLUT1-Defekt</Name>
+      <Name lang="de">Klassisches Glukosetransporter-Typ-1-Mangel-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -125796,7 +125840,7 @@
     <Disorder id="28179">
       <OrphaCode>558411</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=558411</ExpertLink>
-      <Name lang="de">Idiopathic gastroparesis</Name>
+      <Name lang="de">Gastroparese, idiopathische</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -127689,7 +127733,16 @@
       <DisorderGroup id="36547">
         <Name lang="de">Störung</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="de">Vorgeburtlich</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="de">Kleinkindalter</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="de">Neugeborenenzeit</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
@@ -127737,7 +127790,7 @@
     <Disorder id="28337">
       <OrphaCode>562639</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562639</ExpertLink>
-      <Name lang="de">Primary biliary cholangitis/primary sclerosing cholangitis and autoimmune hepatitis overlap syndrome</Name>
+      <Name lang="de">Primäre biliäre Cholangitis/primäre sklerosiierende Cholangitis und autoimmune Hepatitis-Übergangsform</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -128303,7 +128356,7 @@
     <Disorder id="28329">
       <OrphaCode>562509</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562509</ExpertLink>
-      <Name lang="de">Heme oxygenase-1 deficiency</Name>
+      <Name lang="de">Hämoxygenase 1-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -128419,7 +128472,7 @@
     <Disorder id="28331">
       <OrphaCode>562528</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562528</ExpertLink>
-      <Name lang="de">Congenital limbs-face contractures-hypotonia-developmental delay syndrome</Name>
+      <Name lang="de">Kongenitale Extremitäten- und Gesichtskontrakturen-Hypotonie-Entwicklungsverzögerung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -128468,7 +128521,7 @@
     <Disorder id="28333">
       <OrphaCode>562559</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562559</ExpertLink>
-      <Name lang="de">Anterior maxillary protrusion-strabismus-intellectual disability syndrome</Name>
+      <Name lang="de">Vordere Oberkieferprotrusion-Strabismus-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -128497,7 +128550,7 @@
     <Disorder id="28332">
       <OrphaCode>562538</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562538</ExpertLink>
-      <Name lang="de">Autosomal recessive extra-oral halitosis</Name>
+      <Name lang="de">Extraorale Halitosis, autosomal-rezessive</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -128543,7 +128596,7 @@
     <Disorder id="28334">
       <OrphaCode>562569</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=562569</ExpertLink>
-      <Name lang="de">TMEM94-associated congenital heart defect-facial dysmorphism-developmental delay syndrome</Name>
+      <Name lang="de">TMEM94-assoziierter kongenitaler Herzdefekt-Gesichtsdysmorphie-Entwicklungsverzögerung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -128846,7 +128899,7 @@
     <Disorder id="28316">
       <OrphaCode>561854</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=561854</ExpertLink>
-      <Name lang="de">FOXG1 syndrome</Name>
+      <Name lang="de">FOXG1-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -129441,7 +129494,7 @@
     <Disorder id="28397">
       <OrphaCode>564003</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=564003</ExpertLink>
-      <Name lang="de">Osteochondrosis of the metatarsal bone</Name>
+      <Name lang="de">Osteochondrose des Mittelfußknochens</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -129493,7 +129546,7 @@
     <Disorder id="28396">
       <OrphaCode>563991</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563991</ExpertLink>
-      <Name lang="de">Osteochondrosis of the tarsal bone</Name>
+      <Name lang="de">Osteochondrose des Fußwurzelknochens</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -129519,7 +129572,7 @@
     <Disorder id="28399">
       <OrphaCode>564178</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=564178</ExpertLink>
-      <Name lang="de">Primary hypomagnesemia with refractory seizures and intellectual deficiency</Name>
+      <Name lang="de">Primäre Hypomagnesiämie mit refraktären Krämpfen und Intelligenzminderung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -129693,7 +129746,7 @@
     <Disorder id="28372">
       <OrphaCode>563690</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563690</ExpertLink>
-      <Name lang="de">Furuncular myiasis due to Cordylobia rodhaini</Name>
+      <Name lang="de">Fliegenmadenbefall durch Cordylobia rodhaini</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -129716,7 +129769,7 @@
     <Disorder id="28373">
       <OrphaCode>563708</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563708</ExpertLink>
-      <Name lang="de">Syndromic congenital sodium diarrhea</Name>
+      <Name lang="de">Syndromale kongenitale Natriumdiarrhöe</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -129762,7 +129815,7 @@
     <Disorder id="28370">
       <OrphaCode>563684</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563684</ExpertLink>
-      <Name lang="de">Furuncular myiasis due to Dermatobia hominis</Name>
+      <Name lang="de">Furunkuloide Myiasis durch Dermatobia hominis</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -129785,7 +129838,7 @@
     <Disorder id="28371">
       <OrphaCode>563687</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563687</ExpertLink>
-      <Name lang="de">Furuncular myiasis due to Cordylobia anthropophaga</Name>
+      <Name lang="de">Furunkuloide Myiasis durch Cordylobia anthropophaga</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -129828,7 +129881,7 @@
     <Disorder id="28368">
       <OrphaCode>563671</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563671</ExpertLink>
-      <Name lang="de">Mucinous cystadenoma of childhood</Name>
+      <Name lang="de">Muzinöses Zystadenom der Kindheit</Name>
       <DisorderType id="21457">
         <Name lang="de">Histopathologischer Subtyp</Name>
       </DisorderType>
@@ -129874,7 +129927,7 @@
     <Disorder id="28369">
       <OrphaCode>563676</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563676</ExpertLink>
-      <Name lang="de">Seromucinous cystadenoma of childhood</Name>
+      <Name lang="de">Seromuzinöses Zystadenom der Kindheit</Name>
       <DisorderType id="21457">
         <Name lang="de">Histopathologischer Subtyp</Name>
       </DisorderType>
@@ -130174,7 +130227,7 @@
     <Disorder id="28367">
       <OrphaCode>563666</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563666</ExpertLink>
-      <Name lang="de">Serous cystadenoma of childhood</Name>
+      <Name lang="de">Seröses Zystadenom der Kindheit</Name>
       <DisorderType id="21457">
         <Name lang="de">Histopathologischer Subtyp</Name>
       </DisorderType>
@@ -130226,7 +130279,7 @@
     <Disorder id="28366">
       <OrphaCode>563612</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563612</ExpertLink>
-      <Name lang="de">Isolated exencephaly</Name>
+      <Name lang="de">Exenzephalie, isolierte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -130246,7 +130299,7 @@
     <Disorder id="28365">
       <OrphaCode>563609</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563609</ExpertLink>
-      <Name lang="de">Isolated anencephaly</Name>
+      <Name lang="de">Anenzephalie, isolierte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -130295,7 +130348,7 @@
     <Disorder id="28364">
       <OrphaCode>563589</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563589</ExpertLink>
-      <Name lang="de">Seronegative autoimmune hepatitis</Name>
+      <Name lang="de">Hepatitis, autoimmune seronegative</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -130347,7 +130400,7 @@
     <Disorder id="28363">
       <OrphaCode>563581</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563581</ExpertLink>
-      <Name lang="de">Autoimmune hepatitis type 2</Name>
+      <Name lang="de">Hepatitis, autoimmune, Typ 2</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -130405,7 +130458,7 @@
     <Disorder id="28362">
       <OrphaCode>563576</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563576</ExpertLink>
-      <Name lang="de">Autoimmune hepatitis type 1</Name>
+      <Name lang="de">Hepatitis, autoimmune, Typ 1</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -130454,7 +130507,7 @@
     <Disorder id="28451">
       <OrphaCode>565858</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565858</ExpertLink>
-      <Name lang="de">Craniosynostosis-microretrognathia-several intellectual disability syndrome</Name>
+      <Name lang="de">Kraniosynostose-Mikroretrognathie-schwere Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -130483,7 +130536,7 @@
     <Disorder id="28448">
       <OrphaCode>565788</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565788</ExpertLink>
-      <Name lang="de">Infantile inflammatory bowel disease with neurological involvement</Name>
+      <Name lang="de">Infantile chronisch-entzündliche Darmerkrankung mit neurologischer Beteiligung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -130509,7 +130562,7 @@
     <Disorder id="28452">
       <OrphaCode>565899</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565899</ExpertLink>
-      <Name lang="de">POMGNT2-related limb-girdle muscular dystrophy R24</Name>
+      <Name lang="de">POMGNT2-assoziierte Gliedergürtelmuskeldystrophie R24</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -130535,7 +130588,7 @@
     <Disorder id="28453">
       <OrphaCode>565909</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565909</ExpertLink>
-      <Name lang="de">Calpain-3-related limb-girdle muscular dystrophy D4</Name>
+      <Name lang="de">Calpain 3-assoziierte Gliedergürtelmuskeldystrophie D4</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -130636,7 +130689,7 @@
     <Disorder id="28477">
       <OrphaCode>566067</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566067</ExpertLink>
-      <Name lang="de">CEBPE-associated autoinflammation-immunodeficiency-neutrophil dysfunction syndrome</Name>
+      <Name lang="de">CEBPE-assoziierte Autoinflammation-Immundefekt- neutrophile Funktionsstörung-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -130746,7 +130799,7 @@
     <Disorder id="28443">
       <OrphaCode>565624</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565624</ExpertLink>
-      <Name lang="de">Combined oxidative phosphorylation defect type 39</Name>
+      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung Typ 39</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -130778,7 +130831,7 @@
     <Disorder id="28442">
       <OrphaCode>565612</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=565612</ExpertLink>
-      <Name lang="de">Triglyceride deposit cardiomyovasculopathy</Name>
+      <Name lang="de">Triglycerid-Speicher-Kardiomyovaskulopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -131339,7 +131392,7 @@
     <Disorder id="28493">
       <OrphaCode>566231</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566231</ExpertLink>
-      <Name lang="de">Resistance to thyroid hormone due to a mutation in thyroid hormone receptor alpha</Name>
+      <Name lang="de">Resistenz gegen Schilddrüsenhormone durch Mutation im Thyroidhormonrezeptor alpha</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -131388,7 +131441,7 @@
     <Disorder id="28494">
       <OrphaCode>566243</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566243</ExpertLink>
-      <Name lang="de">Resistance to thyroid hormone due to a mutation in thyroid hormone receptor beta</Name>
+      <Name lang="de">Resistenz gegen Schilddrüsenhormone durch Mutation im Thyroidhormonrezeptor beta</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -131587,7 +131640,7 @@
     <Disorder id="28487">
       <OrphaCode>566192</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566192</ExpertLink>
-      <Name lang="de">Congenital autosmal recessive small-platelet thrombocytopenia</Name>
+      <Name lang="de">Kongenitale autosmale-rezessive Thrombozytopenie der kleinen Blutplättchen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -131691,7 +131744,7 @@
     <Disorder id="28482">
       <OrphaCode>566175</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566175</ExpertLink>
-      <Name lang="de">Complement Hyperactivation - angiopathic thrombosis and Protein-losing enteropathy</Name>
+      <Name lang="de">Komplement-Hyperaktivierung-angiopathische Thrombose-Eiweißverlustenteropathie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -132374,7 +132427,7 @@
     <Disorder id="28585">
       <OrphaCode>567502</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567502</ExpertLink>
-      <Name lang="de">B-cell immunodeficiency-limb abnormalities-urogenital malformations syndrome</Name>
+      <Name lang="de">B-Zell-Immundefekt-Extremitätenanomalien-urogenitale Fehlbildungen-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -132458,7 +132511,7 @@
     <Disorder id="28591">
       <OrphaCode>567550</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567550</ExpertLink>
-      <Name lang="de">Idiopathic multidrug-resistant nephrotic syndrome</Name>
+      <Name lang="de">Idiopathisches MDR-resistentes nephrotisches Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -132481,7 +132534,7 @@
     <Disorder id="28590">
       <OrphaCode>567548</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567548</ExpertLink>
-      <Name lang="de">Idiopathic steroid-resistant nephrotic syndrome</Name>
+      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -132504,7 +132557,7 @@
     <Disorder id="28589">
       <OrphaCode>567546</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567546</ExpertLink>
-      <Name lang="de">Idiopathic steroid-sensitive nephrotic syndrome with secondary steroid resistance</Name>
+      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom mit sekundärer Steroid-Resistenz</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -132527,7 +132580,7 @@
     <Disorder id="28588">
       <OrphaCode>567544</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567544</ExpertLink>
-      <Name lang="de">Idiopathic non-lupus full-house nephropathy</Name>
+      <Name lang="de">Idiopathische non-Lupus Full-House-Nephropathie</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -132550,7 +132603,7 @@
     <Disorder id="28592">
       <OrphaCode>567552</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567552</ExpertLink>
-      <Name lang="de">Idiopathic steroid-resistant nephrotic syndrome with sensitivity to second-line immunosuppressive therapy</Name>
+      <Name lang="de">Idiopathisches Steroid-sensitives nephrotisches Syndrom mit Sensitivität zu immunsuppressiver Zweitlinien-Therapie</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -132573,7 +132626,7 @@
     <Disorder id="28598">
       <OrphaCode>567564</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567564</ExpertLink>
-      <Name lang="de">Nephrotic syndrome without extrarenal manifestations</Name>
+      <Name lang="de">Nephrotisches Syndrom ohne extrarenale Manifestationen</Name>
       <DisorderType id="36561">
         <Name lang="de">Category</Name>
       </DisorderType>
@@ -132778,7 +132831,7 @@
     <Disorder id="28606">
       <OrphaCode>567983</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=567983</ExpertLink>
-      <Name lang="de">Parenteral nutrition-associated cholestasis</Name>
+      <Name lang="de">Parenterale Ernährung-assoziierte Cholestase</Name>
       <DisorderType id="21429">
         <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
       </DisorderType>
@@ -133137,7 +133190,7 @@
     <Disorder id="28552">
       <OrphaCode>566841</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566841</ExpertLink>
-      <Name lang="de">Liver adenomatosis</Name>
+      <Name lang="de">Adenomatose der Leber</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -133192,7 +133245,7 @@
     <Disorder id="11082">
       <OrphaCode>75392</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=75392</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, Periodontose Typ</Name>
+      <Name lang="de">Ehlers-Danlos-Syndrom, parodontaler Typ</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -133218,7 +133271,7 @@
     <Disorder id="11083">
       <OrphaCode>75496</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=75496</ExpertLink>
-      <Name lang="de">Ehlers-Danlos-Syndrom, progeroider Typ</Name>
+      <Name lang="de">B4GALT7-assoziiertes spondylodysplastisches Ehlers-Danlos-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -133322,7 +133375,7 @@
     <Disorder id="28560">
       <OrphaCode>566943</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=566943</ExpertLink>
-      <Name lang="de">Mueller-Weiss syndrome</Name>
+      <Name lang="de">Mueller-Weiss-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -133828,7 +133881,7 @@
     <Disorder id="28655">
       <OrphaCode>569821</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569821</ExpertLink>
-      <Name lang="de">Congenital primary lymphedema of Gordon</Name>
+      <Name lang="de">Kongenitales primäres Lymphödem vom Typ Gordon</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -133889,7 +133942,7 @@
     <Disorder id="28654">
       <OrphaCode>569816</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569816</ExpertLink>
-      <Name lang="de">CELSR1-related late-onset primary lymphedema</Name>
+      <Name lang="de">CELSR1-assoziiertes spät-einsetzendes primäres Lymphödem</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134156,7 +134209,7 @@
     <Disorder id="28613">
       <OrphaCode>568065</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568065</ExpertLink>
-      <Name lang="de">EPHB4-related fetal hydrops with generalized lymphatic dysplasia</Name>
+      <Name lang="de">EPHB4-assoziierter fetaler Hydrops mit generalisierter lymphatischer Dysplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134199,7 +134252,7 @@
     <Disorder id="28612">
       <OrphaCode>568062</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568062</ExpertLink>
-      <Name lang="de">PIEZO1-related fetal hydrops with generalized lymphatic dysplasia</Name>
+      <Name lang="de">PIEZO1-assoziierter fetaler Hydrops mit generalisierter lymphatischer Dysplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134280,7 +134333,7 @@
     <Disorder id="28611">
       <OrphaCode>568056</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568056</ExpertLink>
-      <Name lang="de">Warts-immunodeficiency-lymphoedema-anogenital dysplasia syndrome</Name>
+      <Name lang="de">Warzen-Immundefekt-Lymphödem-anogenitale Dysplasie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134306,7 +134359,7 @@
     <Disorder id="28610">
       <OrphaCode>568051</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=568051</ExpertLink>
-      <Name lang="de">GJC2-related congenital primary lymphedema</Name>
+      <Name lang="de">GJC2-assoziiertes spät-beginnendes primäres Lymphödem</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134448,7 +134501,7 @@
     <Disorder id="28622">
       <OrphaCode>569164</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569164</ExpertLink>
-      <Name lang="de">Angiomatoid fibrous histiocytoma</Name>
+      <Name lang="de">Histiozytom, angiomatöses fibröses</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134585,7 +134638,7 @@
     <Disorder id="28629">
       <OrphaCode>569274</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569274</ExpertLink>
-      <Name lang="de">Multiple mitochondrial dysfunctions syndrome type 5</Name>
+      <Name lang="de">Mitochondriales Dysfunktions-Syndrom Typ 5</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134614,7 +134667,7 @@
     <Disorder id="28631">
       <OrphaCode>569290</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569290</ExpertLink>
-      <Name lang="de">Multiple mitochondrial dysfunctions syndrome type 6</Name>
+      <Name lang="de">Mitochondriales Dysfunktions-Syndrom Typ 6</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134637,7 +134690,7 @@
     <Disorder id="28626">
       <OrphaCode>569248</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=569248</ExpertLink>
-      <Name lang="de">Microcystic stromal tumor</Name>
+      <Name lang="de">Mikrozystischer Stromatumor</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -135469,7 +135522,7 @@
     <Disorder id="10395">
       <OrphaCode>35706</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=35706</ExpertLink>
-      <Name lang="de">Glutaryl-CoA-Oxidase-Mangel</Name>
+      <Name lang="de">Glutarazidurie Typ 3</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -137601,7 +137654,7 @@
     <Disorder id="10349">
       <OrphaCode>34592</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=34592</ExpertLink>
-      <Name lang="de">Immundefekt durch HLA-Klasse-I-Expressionsdefekt</Name>
+      <Name lang="de">Immundefekt durch MHC Klasse I-Expressionsdefekt</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -139596,7 +139649,7 @@
     <Disorder id="10649">
       <OrphaCode>50810</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=50810</ExpertLink>
-      <Name lang="de">Mikrolissenzephalie - Mikromelie</Name>
+      <Name lang="de">Mikrolissenzephalie-Mikromelie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -141993,7 +142046,7 @@
     <Disorder id="28139">
       <OrphaCode>556985</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=556985</ExpertLink>
-      <Name lang="de">Early-onset calcifying  leukoencephalopathy-skeletal dysplasia</Name>
+      <Name lang="de">Früh-einsetzende kalzifizierende Leukenzephalopathie mit Skelettdysplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -142025,7 +142078,7 @@
     <Disorder id="28138">
       <OrphaCode>556955</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=556955</ExpertLink>
-      <Name lang="de">Pancreatic agenesis-holoprosencephaly syndrome</Name>
+      <Name lang="de">Pankreasagenesie-Holoprosenzephalie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -142048,7 +142101,7 @@
     <Disorder id="28142">
       <OrphaCode>557064</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=557064</ExpertLink>
-      <Name lang="de">Neonatal epileptic encephalopathy due to glutaminase deficiency</Name>
+      <Name lang="de">Neonatale epileptische Enzephalopathie durch Glutaminase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -142071,7 +142124,7 @@
     <Disorder id="28141">
       <OrphaCode>557056</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=557056</ExpertLink>
-      <Name lang="de">Spastic ataxia-dysarthria due to glutaminase deficiency</Name>
+      <Name lang="de">Spastische Ataxie mit Dysarthrie durch Glutaminase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -142440,7 +142493,7 @@
     <Disorder id="11925">
       <OrphaCode>90023</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90023</ExpertLink>
-      <Name lang="de">Infektionen, rekurrente - Kleinwuchs - Hypopigmentation - grobe Gesichtszüge</Name>
+      <Name lang="de">Primäres Immundefekt-Syndrom durch LAMTOR2-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -142466,7 +142519,7 @@
     <Disorder id="11921">
       <OrphaCode>90003</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90003</ExpertLink>
-      <Name lang="de">Hepatopathie, IgG4-assoziierte</Name>
+      <Name lang="de">Pseudotumor der Leber, inflammatorischer</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -142515,7 +142568,7 @@
     <Disorder id="11935">
       <OrphaCode>90038</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90038</ExpertLink>
-      <Name lang="de">Hämolytisch-urämisches Syndrom, typisches</Name>
+      <Name lang="de">Shiga-ähnliches Toxin-assoziiertes Hämolytisch-urämisches Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -153547,7 +153600,7 @@
     <Disorder id="11709">
       <OrphaCode>85408</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85408</ExpertLink>
-      <Name lang="de">Arthritis, idiopathische juvenile, Rheumafaktor-negative</Name>
+      <Name lang="de">Arthritis, idiopathische juvenile polyartikuläre, Rheumafaktor-negative</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -153709,7 +153762,7 @@
     <Disorder id="11712">
       <OrphaCode>85435</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85435</ExpertLink>
-      <Name lang="de">Arthritis, idiopathische juvenile, Rheumafaktor-positive</Name>
+      <Name lang="de">Arthritis, idiopathische juvenile polyartikuläre, Rheumafaktor-positive</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -158138,7 +158191,7 @@
     <Disorder id="12914">
       <OrphaCode>97363</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=97363</ExpertLink>
-      <Name lang="de">Nierendysplasie, multizystische, unilaterale From</Name>
+      <Name lang="de">Nierendysplasie, multizystische, unilaterale Form</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -159178,7 +159231,7 @@
     <Disorder id="12663">
       <OrphaCode>95708</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=95708</ExpertLink>
-      <Name lang="de">Pubertät, vorzeitige</Name>
+      <Name lang="de">Seltene Form der vorzeitigen Pubertät</Name>
       <DisorderType id="36561">
         <Name lang="de">Category</Name>
       </DisorderType>
@@ -162019,7 +162072,7 @@
     <Disorder id="12290">
       <OrphaCode>93359</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93359</ExpertLink>
-      <Name lang="de">Dysplasie, spondyloepimetaphysäre - Überstreckbarkeit der Gelenke</Name>
+      <Name lang="de">Spondyloepimetaphysäre Dysplasie mit Überstreckbarkeit der Gelenke</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -163210,7 +163263,7 @@
     <Disorder id="12466">
       <OrphaCode>93685</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93685</ExpertLink>
-      <Name lang="de">Castleman-Krankheit, lokalisierte</Name>
+      <Name lang="de">Castleman-Krankheit, unizentrische</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168522,7 +168575,7 @@
     <Disorder id="28832">
       <OrphaCode>575553</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=575553</ExpertLink>
-      <Name lang="de">Cathepsin A-related arteriopathy-strokes-leukoencephalopathy</Name>
+      <Name lang="de">Cathepsin A-assoziierte Arteriopathie-Schlaganfall-Leukoenzephalopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168545,7 +168598,7 @@
     <Disorder id="28815">
       <OrphaCode>573253</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=573253</ExpertLink>
-      <Name lang="de">Split cord malformation type II</Name>
+      <Name lang="de">Split-Cord-Malformation Typ II</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168577,7 +168630,7 @@
     <Disorder id="28824">
       <OrphaCode>574957</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=574957</ExpertLink>
-      <Name lang="de">Autosomal recessive mendelian susceptibility to mycobacterial diseases due to partial JAK1 deficiency</Name>
+      <Name lang="de">Autosomal-rezessive Mendelsche Anfälligkeit für Erkrankungen durch Mykobakterien durch partiellen JAK1-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168603,7 +168656,7 @@
     <Disorder id="28823">
       <OrphaCode>574918</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=574918</ExpertLink>
-      <Name lang="de">Predisposition to severe viral infection due to IRF7 deficiency</Name>
+      <Name lang="de">Prädisposition zur schweren Virusinfektion durch IRF7-Defizienz</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168655,7 +168708,7 @@
     <Disorder id="28816">
       <OrphaCode>573278</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=573278</ExpertLink>
-      <Name lang="de">Split cord malformation</Name>
+      <Name lang="de">Split-Cord-Malformation</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -168687,7 +168740,7 @@
     <Disorder id="28738">
       <OrphaCode>572385</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572385</ExpertLink>
-      <Name lang="de">Brachydactyly type B1</Name>
+      <Name lang="de">Brachydaktylie Typ B1</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168710,7 +168763,7 @@
     <Disorder id="28736">
       <OrphaCode>572361</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572361</ExpertLink>
-      <Name lang="de">Blepharophimosis-ptosis-epicanthus inversus syndrome type 2</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom Typ 2</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168762,7 +168815,7 @@
     <Disorder id="28746">
       <OrphaCode>572428</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572428</ExpertLink>
-      <Name lang="de">Infantile-onset pulmonary alveolar proteinosis-hypogammaglobulinemia</Name>
+      <Name lang="de">Pulmonale Alveolarproteinose-Hypogammaglobulinämie, infantile Form</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168814,7 +168867,7 @@
     <Disorder id="28754">
       <OrphaCode>572543</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572543</ExpertLink>
-      <Name lang="de">RFVT2-related riboflavin transporter deficiency</Name>
+      <Name lang="de">RFVT2-assoziierte Riboflavin-Transporter-Defizienz</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168843,7 +168896,7 @@
     <Disorder id="28755">
       <OrphaCode>572550</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572550</ExpertLink>
-      <Name lang="de">RFVT3-related riboflavin transporter deficiency</Name>
+      <Name lang="de">RFVT3-assoziierte Riboflavin-Transporter-Defizienz</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168878,7 +168931,7 @@
     <Disorder id="28766">
       <OrphaCode>572773</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572773</ExpertLink>
-      <Name lang="de">Microcephaly-short stature-limb abnormalities syndrome</Name>
+      <Name lang="de">Mikrozphalie-Kleinwuchs-Extremitätenanomalien-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -168901,7 +168954,7 @@
     <Disorder id="28767">
       <OrphaCode>572798</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572798</ExpertLink>
-      <Name lang="de">WARS2-related combined oxidative phosphorylation defect</Name>
+      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung, WARS2-assoziierter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168933,7 +168986,7 @@
     <Disorder id="28764">
       <OrphaCode>572761</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572761</ExpertLink>
-      <Name lang="de">DONSON-related microcephaly-short stature-limb abnormalities spectrum</Name>
+      <Name lang="de">DONSON-assoziierte Mikrozphalie-Kleinwuchs-Extremitätenanomalien-Spektrum</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -168959,7 +169012,7 @@
     <Disorder id="28765">
       <OrphaCode>572768</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572768</ExpertLink>
-      <Name lang="de">Microcephaly-micromelia syndrome</Name>
+      <Name lang="de">Mikrozephalie-Mikromelie-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -169014,7 +169067,7 @@
     <Disorder id="28726">
       <OrphaCode>572013</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572013</ExpertLink>
-      <Name lang="de">Posterior-predominant lissencephaly-broad flat pons and medulla-midline crossing defects syndrome</Name>
+      <Name lang="de">Vorwiegend posteriore Lissenzephalie mit breiter flacher Pons und Medulla-Mittelliniendefekt</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -169040,7 +169093,7 @@
     <Disorder id="28732">
       <OrphaCode>572333</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572333</ExpertLink>
-      <Name lang="de">Blepharophimosis-ptosis-epicanthus inversus syndrome plus</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom plus</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -169063,7 +169116,7 @@
     <Disorder id="28735">
       <OrphaCode>572354</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=572354</ExpertLink>
-      <Name lang="de">Blepharophimosis-ptosis-epicanthus inversus syndrome type 1</Name>
+      <Name lang="de">Blepharophimose-Epicanthus inversus-Ptosis-Syndrom Typ 1</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -169086,7 +169139,7 @@
     <Disorder id="28678">
       <OrphaCode>570762</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570762</ExpertLink>
-      <Name lang="de">Infective endocariditis</Name>
+      <Name lang="de">Endokarditis, infektiöse</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -169109,7 +169162,7 @@
     <Disorder id="28677">
       <OrphaCode>570491</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570491</ExpertLink>
-      <Name lang="de">QRSL1-related combined oxidative phosphorylation defect</Name>
+      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung, QRSL1-assoziierter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -169138,7 +169191,7 @@
     <Disorder id="28676">
       <OrphaCode>570470</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570470</ExpertLink>
-      <Name lang="de">Ricin poisoning</Name>
+      <Name lang="de">Rizin-Vergiftung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -169164,7 +169217,7 @@
     <Disorder id="28675">
       <OrphaCode>570438</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570438</ExpertLink>
-      <Name lang="de">HHV-8-associated multicentric Castleman disease</Name>
+      <Name lang="de">Castleman-Krankheit, multizentrische, HHV-8-assoziierte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -169184,7 +169237,7 @@
     <Disorder id="28674">
       <OrphaCode>570431</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570431</ExpertLink>
-      <Name lang="de">Multicentric idiopathic Castleman disease</Name>
+      <Name lang="de">Castleman-Krankheit, multizentrische, idiopathische</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -169204,7 +169257,7 @@
     <Disorder id="28673">
       <OrphaCode>570422</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570422</ExpertLink>
-      <Name lang="de">Galactose mutarotase deficiency</Name>
+      <Name lang="de">Galaktose-Mutarotase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -169230,7 +169283,7 @@
     <Disorder id="28672">
       <OrphaCode>570371</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=570371</ExpertLink>
-      <Name lang="de">Transient antenatal Bartter syndrome</Name>
+      <Name lang="de">Bartter -Syndrom, antenatales transientes</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
