commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare diseases natural history/de_product9_ages.xml b/Epidemiological data/Rare diseases natural history/de_product9_ages.xml
old mode 100644
new mode 100755
index e6928fd..87d233a
--- a/Epidemiological data/Rare diseases natural history/de_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/de_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:39:26" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:40:23" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6586">
+  <DisorderList count="6584">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=166024</ExpertLink>
@@ -2427,7 +2427,7 @@
     <Disorder id="17554">
       <OrphaCode>163956</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=163956</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale, Typ Nascimento</Name>
+      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Nascimento</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -7244,7 +7244,7 @@
     <Disorder id="343">
       <OrphaCode>1727</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1727</ExpertLink>
-      <Name lang="de">Mikroduplikationssyndrom 22q11</Name>
+      <Name lang="de">Mikroduplikationssyndrom 22q11.2</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -8808,7 +8808,7 @@
     <Disorder id="413">
       <OrphaCode>1046</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1046</ExpertLink>
-      <Name lang="de">Anämie, hämolytische letale - Genitalfehlbildungen</Name>
+      <Name lang="de">Letale hämolytischeAnämie-Genitalfehlbildungen-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -9396,7 +9396,7 @@
     <Disorder id="446">
       <OrphaCode>1488</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1488</ExpertLink>
-      <Name lang="de">Cooper-Wang-Jabs-Syndrom</Name>
+      <Name lang="de">Cooper-Jabs-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -9801,7 +9801,7 @@
     <Disorder id="17769">
       <OrphaCode>168552</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=168552</ExpertLink>
-      <Name lang="de">Dysplasie, spondyloepimetaphysäre - gebogene Unterarme - Gesichtsdysmorphien</Name>
+      <Name lang="de">Spondyloepimetaphysäre Dysplasie-gebogene Unterarme-Gesichtsdysmorphien-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -10143,7 +10143,7 @@
     <Disorder id="17761">
       <OrphaCode>168443</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=168443</ExpertLink>
-      <Name lang="de">Dysplasie, spondyloepimetaphysäre - Hypotrichose</Name>
+      <Name lang="de">Spondyloepimetaphysäre Dysplasie-Hypotrichose-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -10204,7 +10204,7 @@
     <Disorder id="17763">
       <OrphaCode>168451</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=168451</ExpertLink>
-      <Name lang="de">Dysplasie, spondyloepimetaphysäre - gestörte Zahnentwicklung</Name>
+      <Name lang="de">Spondyloepimetaphysäre Dysplasie mit gestörter Zahnentwicklung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -11103,7 +11103,7 @@
     <Disorder id="510">
       <OrphaCode>2233</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2233</ExpertLink>
-      <Name lang="de">Hypogonadismus - Mitralklappenprolaps - geistige Retardierung</Name>
+      <Name lang="de">Hypogonadismus-Mitralklappenprolaps-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -14218,7 +14218,7 @@
     <Disorder id="678">
       <OrphaCode>62</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=62</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive,  Typ 2D</Name>
+      <Name lang="de">Alpha-Sarkoglykan assoziierte Gliedergürtelmuskeldystrophie R3</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -17149,7 +17149,7 @@
     <Disorder id="17939">
       <OrphaCode>171844</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=171844</ExpertLink>
-      <Name lang="de">Blindheit - Skoliose - Arachnodaktylie</Name>
+      <Name lang="de">Blindheit-Skoliose-Arachnodaktylie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -18520,7 +18520,7 @@
     <Disorder id="784">
       <OrphaCode>3002</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3002</ExpertLink>
-      <Name lang="de">Purpura, immune thrombozytopenische</Name>
+      <Name lang="de">Immunthrombozytopenie, immune</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -19841,7 +19841,7 @@
     <Disorder id="870">
       <OrphaCode>267</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=267</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2A</Name>
+      <Name lang="de">Calpain-3-assoziierte Gliedergürtelmuskeldystrophie R1</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -19873,7 +19873,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1329</ExpertLink>
-      <Name lang="de">Atrioventrikulärer (AV-) Kanal, kompletter</Name>
+      <Name lang="de">Atrioventrikulärer Septumdefekt, kompletter</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -20134,7 +20134,7 @@
     <Disorder id="849">
       <OrphaCode>3198</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3198</ExpertLink>
-      <Name lang="de">Stiff-person-Syndrom und verwandte Krankheiten</Name>
+      <Name lang="de">Stiff-person-Spektrum-Erkrankungen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -22260,7 +22260,7 @@
     <Disorder id="988">
       <OrphaCode>1473</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1473</ExpertLink>
-      <Name lang="de">Uvea-Kolobom - Lippen-Gaumenspalte - geistige Retardierung</Name>
+      <Name lang="de">Uvea-Kolobom mit Lippen-Gaumenspalte und Intelligenzminderung</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -23556,7 +23556,7 @@
     <Disorder id="1228">
       <OrphaCode>3306</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3306</ExpertLink>
-      <Name lang="de">Duplikation/Inversion 15q11</Name>
+      <Name lang="de">Inversion/Duplikation Chromosom 15</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -24804,7 +24804,7 @@
     <Disorder id="1195">
       <OrphaCode>1878</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1878</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2H</Name>
+      <Name lang="de">TRIM32-assoziierte Gliedergürtelmuskeldystrophie R8</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -24862,7 +24862,7 @@
     <Disorder id="1199">
       <OrphaCode>1948</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1948</ExpertLink>
-      <Name lang="de">Epilepsie - Mikrozephalie - Skelettdysplasie</Name>
+      <Name lang="de">Epilepsie-Mikrozephalie-Skelettdysplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -25100,7 +25100,7 @@
     <Disorder id="1372">
       <OrphaCode>1078</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1078</ExpertLink>
-      <Name lang="de">Steifer Daumen - Brachydaktylie - geistige Retardierung</Name>
+      <Name lang="de">Steifer Daumen-Brachydaktylie-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -25207,7 +25207,7 @@
     <Disorder id="1365">
       <OrphaCode>1071</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1071</ExpertLink>
-      <Name lang="de">Ankyloblepharon - ektodermale Defekte - Lippen-Kiefer-Gaumenspalte</Name>
+      <Name lang="de">Ankyloblepharon-ektodermale Defekte-Lippen-Kiefer-Gaumenspalte-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -25262,7 +25262,7 @@
     <Disorder id="1363">
       <OrphaCode>1068</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1068</ExpertLink>
-      <Name lang="de">Aniridie - geistige Retardierung</Name>
+      <Name lang="de">Aniridie-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -25314,7 +25314,7 @@
     <Disorder id="1360">
       <OrphaCode>1064</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1064</ExpertLink>
-      <Name lang="de">Aniridie - Nierenagenesie - psychomotorische Retardierung</Name>
+      <Name lang="de">Aniridie-Nierenagenesie-psychomotorische Retardierung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -25719,7 +25719,7 @@
     <Disorder id="1398">
       <OrphaCode>1117</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1117</ExpertLink>
-      <Name lang="de">Aplasia cutis congenita - Myopie</Name>
+      <Name lang="de">Aplasia cutis congenita-Myopie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -25803,7 +25803,7 @@
     <Disorder id="1394">
       <OrphaCode>1112</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1112</ExpertLink>
-      <Name lang="de">Aphalangie - Hemivertebrae - urogenital-intestinale Dysgenesie</Name>
+      <Name lang="de">Aphalangie-Hemivertebrae-urogenital-intestinale Dysgenesie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -25959,7 +25959,7 @@
     <Disorder id="1305">
       <OrphaCode>990</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=990</ExpertLink>
-      <Name lang="de">Agnathie - Holoprosenzephalie - Situs inversus</Name>
+      <Name lang="de">Agnathie-Holoprosenzephalie-Situs inversus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -26090,7 +26090,7 @@
     <Disorder id="1299">
       <OrphaCode>981</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=981</ExpertLink>
-      <Name lang="de">Agenesie der Arteria carotis interna</Name>
+      <Name lang="de">Fehlen der Arteria carotis interna</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -27997,7 +27997,7 @@
     <Disorder id="1528">
       <OrphaCode>1295</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1295</ExpertLink>
-      <Name lang="de">Brachytelephalangie - Dysmorphien - Kallmann-Syndrom</Name>
+      <Name lang="de">Brachytelephalangie mit Dysmorphien und Kallmann-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -28341,7 +28341,7 @@
     <Disorder id="16682">
       <OrphaCode>137599</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=137599</ExpertLink>
-      <Name lang="de">Stromakeratitis</Name>
+      <Name lang="de">Herpes simplex-Stromakeratitis</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -28647,7 +28647,7 @@
     <Disorder id="1438">
       <OrphaCode>1180</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1180</ExpertLink>
-      <Name lang="de">Ataxie - Hypogonadismus - chorioretinale Dystrophie</Name>
+      <Name lang="de">Ataxie-Hypogonadismus-chorioretinale Dystrophie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -28977,7 +28977,7 @@
     <Disorder id="1413">
       <OrphaCode>1144</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1144</ExpertLink>
-      <Name lang="de">Arthrogrypose-ähnliche Handanomalie - sensorineurale Schwerhörigkeit</Name>
+      <Name lang="de">Arthrogrypose-ähnliche Handanomalie mit sensorineuraler Schwerhörigkeit</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -29032,7 +29032,7 @@
     <Disorder id="1417">
       <OrphaCode>1150</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1150</ExpertLink>
-      <Name lang="de">Arthrogryposis multiplex congenita - Whistling-face-Syndrom</Name>
+      <Name lang="de">Arthrogryposis multiplex congenita-Whistling-face-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -29374,7 +29374,7 @@
     <Disorder id="1441">
       <OrphaCode>1184</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1184</ExpertLink>
-      <Name lang="de">Ataxie - Lichtempfindlichkeit - Kleinwuchs</Name>
+      <Name lang="de">Ataxie-Lichtempfindlichkeit-Kleinwuchs-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -29707,7 +29707,7 @@
     <Disorder id="16743">
       <OrphaCode>137911</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=137911</ExpertLink>
-      <Name lang="de">Autismus - fazialer Portwein-Nävus</Name>
+      <Name lang="de">Autismus-fazialer Portwein-Nävus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -30469,7 +30469,7 @@
     <Disorder id="1639">
       <OrphaCode>1436</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1436</ExpertLink>
-      <Name lang="de">Skelettdysplasie-Intelligenzminderung-Syndrom, X-chromosomales</Name>
+      <Name lang="de">X-chromosomale Skelettdysplasie-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -31252,7 +31252,7 @@
     <Disorder id="1606">
       <OrphaCode>1399</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1399</ExpertLink>
-      <Name lang="de">Ketoazidose - geistige Retardierung - Ataxie - Taubheit</Name>
+      <Name lang="de">Richards-Rundle-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -31924,7 +31924,7 @@
     <Disorder id="1595">
       <OrphaCode>1389</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1389</ExpertLink>
-      <Name lang="de">Kortikale Blindheit - geistige Retardierung - Polydaktylie</Name>
+      <Name lang="de">Kortikale Blindheit-Intelligenzminderung-Polydaktylie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -31956,7 +31956,7 @@
     <Disorder id="17146">
       <OrphaCode>157798</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=157798</ExpertLink>
-      <Name lang="de">Polyposis-Syndrom, hyperplastisches</Name>
+      <Name lang="de">Serratiertes Polyposissyndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -32164,7 +32164,7 @@
     <Disorder id="1588">
       <OrphaCode>1381</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1381</ExpertLink>
-      <Name lang="de">Katarakt - geistige Retardierung - Analatresie - Uropathie</Name>
+      <Name lang="de">Katarakt-Intelligenzminderung-Analatresie-Uropathie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -32626,7 +32626,7 @@
     <Disorder id="1539">
       <OrphaCode>1313</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1313</ExpertLink>
-      <Name lang="de">Plexus choroideus-Kalzifizierung - geistige Retardierung, infantile Form</Name>
+      <Name lang="de">Infantile Plexus choroideus-Kalzifikation-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -33686,7 +33686,7 @@
     <Disorder id="1738">
       <OrphaCode>1606</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1606</ExpertLink>
-      <Name lang="de">1p36-Deletions-Syndrom</Name>
+      <Name lang="de">1p36-Deletionssyndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -33978,7 +33978,7 @@
     <Disorder id="1746">
       <OrphaCode>1658</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1658</ExpertLink>
-      <Name lang="de">Dermatoglyphen, fehlende - kongenitale Milien</Name>
+      <Name lang="de">Fehlende Dermatoglyphen-kongenitale Milien-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -34036,7 +34036,7 @@
     <Disorder id="1747">
       <OrphaCode>1659</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1659</ExpertLink>
-      <Name lang="de">Dermato-Leukodystrophie</Name>
+      <Name lang="de">Dermatoleukodystrophie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -34557,7 +34557,7 @@
     <Disorder id="16999">
       <OrphaCode>140922</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=140922</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2J</Name>
+      <Name lang="de">Titin-assoziierte Gliedergürtelmuskeldystrophie R10</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -35137,7 +35137,7 @@
     <Disorder id="1715">
       <OrphaCode>1562</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1562</ExpertLink>
-      <Name lang="de">Dakryozystitis - Osteopoikilose</Name>
+      <Name lang="de">Dakryozystitis-Osteopoikilose-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -35163,7 +35163,7 @@
     <Disorder id="17010">
       <OrphaCode>140963</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=140963</ExpertLink>
-      <Name lang="de">Bilaterale Mikrotie - Schwerhörigkeit - Gaumenspalte</Name>
+      <Name lang="de">Bilaterale Mikrotie-Schwerhörigkeit-Gaumenspalte-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -36215,7 +36215,7 @@
     <Disorder id="1908">
       <OrphaCode>1964</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1964</ExpertLink>
-      <Name lang="de">Extrasystolen - Kleinwuchs - Hyperpigmentierung - Mikrozephalie</Name>
+      <Name lang="de">Extrasystolen-Kleinwuchs-Hyperpigmentierung-Mikrozephalie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -36903,7 +36903,7 @@
     <Disorder id="1868">
       <OrphaCode>1906</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1906</ExpertLink>
-      <Name lang="de">Valproat-Embryopathie</Name>
+      <Name lang="de">Fetale Valproat-Spektrumstörung</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -36990,7 +36990,7 @@
     <Disorder id="1861">
       <OrphaCode>1892</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1892</ExpertLink>
-      <Name lang="de">Ektrodaktylie - Polydaktylie</Name>
+      <Name lang="de">Ektrodaktylie-Polydaktylie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -37650,7 +37650,7 @@
     <Disorder id="1811">
       <OrphaCode>1825</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1825</ExpertLink>
-      <Name lang="de">Dysplasie, epiphysäre - Hörverlust - Dysmorphien</Name>
+      <Name lang="de">Epiphysäre Dysplasie mit Hörverlust und Dysmorphien</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -37731,7 +37731,7 @@
     <Disorder id="1805">
       <OrphaCode>1812</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1812</ExpertLink>
-      <Name lang="de">Dysplasie, ektodermale - geistige Retardierung - Fehlbildungen des Zentralnervensystems</Name>
+      <Name lang="de">Ektodermale Dysplasie-Intelligenzminderung-ZNS-Fehlbildungen-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -39017,7 +39017,7 @@
     <Disorder id="2015">
       <OrphaCode>2104</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2104</ExpertLink>
-      <Name lang="de">Dysmorphien - Pectus carinatum - schlaffe Gelenke</Name>
+      <Name lang="de">Dysmorphien-Pectus carinatum-schlaffe Gelenke-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -39861,7 +39861,7 @@
     <Disorder id="1978">
       <OrphaCode>2057</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2057</ExpertLink>
-      <Name lang="de">Blepharophimose - Ptosis - Esotropie - Syndaktylie - Kleinwuchs</Name>
+      <Name lang="de">Blepharophimose-Ptosis-Esotropie-Syndaktylie-Kleinwuchs-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -40124,7 +40124,7 @@
     <Disorder id="1965">
       <OrphaCode>2824</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2824</ExpertLink>
-      <Name lang="de">Paraplegie - geistige Retardierung - Hyperkeratosen</Name>
+      <Name lang="de">Paraplegie-Intelligenzminderung-Hyperkeratose-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -41117,7 +41117,7 @@
     <Disorder id="19548">
       <OrphaCode>247794</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=247794</ExpertLink>
-      <Name lang="de">Juvenile Katarakt - Mikrokornea - renale Glukosurie</Name>
+      <Name lang="de">Juvenile Katarakt-Mikrokornea-renale Glukosurie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -42152,7 +42152,7 @@
     <Disorder id="2180">
       <OrphaCode>2325</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2325</ExpertLink>
-      <Name lang="de">Epidermolysis bullosa simplex mit  Anodontie/Hypodontie</Name>
+      <Name lang="de">Epidermolysis bullosa simplex mit Taubheit, Alopezie, Anodontie/Hypodontie und Nageldystrophie</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -42932,7 +42932,7 @@
     <Disorder id="2213">
       <OrphaCode>2375</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2375</ExpertLink>
-      <Name lang="de">Abduktorenlähmung, laryngeale - geistige Retardierung</Name>
+      <Name lang="de">Laryngeale Abduktorenlähmung-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -43493,7 +43493,7 @@
     <Disorder id="2300">
       <OrphaCode>2487</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2487</ExpertLink>
-      <Name lang="de">Anomalien der unteren Extremitäten - Hypospadie</Name>
+      <Name lang="de">Anomalien der unteren Extremitäten-Hypospadie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -43888,7 +43888,7 @@
     <Disorder id="2289">
       <OrphaCode>2476</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2476</ExpertLink>
-      <Name lang="de">Dysraphie - Lippen-Kiefer-Gaumen-Spalte - Reduktionsdefekt der Extremitäten</Name>
+      <Name lang="de">Dysraphie mit Lippen-Kiefer-Gaumen-Spalte und Reduktionsdefekt der Extremitäten</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -45435,7 +45435,7 @@
     <Disorder id="2108">
       <OrphaCode>2234</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2234</ExpertLink>
-      <Name lang="de">Hypogonadismus, männlicher - geistige Retardierung - Skelettanomalien</Name>
+      <Name lang="de">Männlicher Hypogonadismus-Intelligenzminderung-Skelettanomalien-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -46073,7 +46073,7 @@
     <Disorder id="2133">
       <OrphaCode>2266</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2266</ExpertLink>
-      <Name lang="de">Hypotrichose - geistige Retardierung, Typ Lopes</Name>
+      <Name lang="de">Hypotrichose mit Intelligenzminderung Typ Lopes</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -46102,7 +46102,7 @@
     <Disorder id="19605">
       <OrphaCode>251028</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=251028</ExpertLink>
-      <Name lang="de">Mikrodeletionssyndrom 2q33.1</Name>
+      <Name lang="de">SATB2-assoziiertes-Syndrom durch chromosomales Rearrangement</Name>
       <DisorderType id="21443">
         <Name lang="de">Ätiologischer Subtyp</Name>
       </DisorderType>
@@ -46856,7 +46856,7 @@
     <Disorder id="2119">
       <OrphaCode>2249</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2249</ExpertLink>
-      <Name lang="de">Ulnahypoplasie - geistige Retardierung</Name>
+      <Name lang="de">Ulnahypoplasie-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -48048,7 +48048,7 @@
     <Disorder id="2147">
       <OrphaCode>2278</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2278</ExpertLink>
-      <Name lang="de">Ichthyose - geistige Retardierung - Kleinwuchs - Niereninsuffizienz</Name>
+      <Name lang="de">Ichthyose-Intelligenzminderung-Kleinwuchs-Niereninsuffizienz-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -48475,7 +48475,7 @@
     <Disorder id="19791">
       <OrphaCode>254864</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=254864</ExpertLink>
-      <Name lang="de">Myopathie, mitochondriale, mit reversiblem Cytochrom-C-Oxidase-Mangel</Name>
+      <Name lang="de">Myopathie, mitochondriale, mit reversiblem Cytochrom-c-Oxidase-Mangel</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -48790,7 +48790,7 @@
     <Disorder id="2459">
       <OrphaCode>2697</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2697</ExpertLink>
-      <Name lang="de">Arthrogrypose - Nierenfunktionsstörung - Cholestase</Name>
+      <Name lang="de">Arthrogrypose-Nierenfunktionsstörung-Cholestase-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -49163,7 +49163,7 @@
     <Disorder id="19799">
       <OrphaCode>254905</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=254905</ExpertLink>
-      <Name lang="de">Cytochrom-C-Oxidase-Mangel, isolierter</Name>
+      <Name lang="de">Cytochrom-c-Oxidase-Mangel, isolierter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -49374,7 +49374,7 @@
     <Disorder id="2476">
       <OrphaCode>2715</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2715</ExpertLink>
-      <Name lang="de">Okulo-reno-zerebelläres Syndrom</Name>
+      <Name lang="de">Okulo-reno-zerebelläres Syndrom, schweres</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -50203,7 +50203,7 @@
     <Disorder id="2503">
       <OrphaCode>2743</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2743</ExpertLink>
-      <Name lang="de">Ophthalmoplegie - geistige Retardierung - Lingua scrotalis</Name>
+      <Name lang="de">Ophthalmoplegie-Intelligenzminderung-Lingua scrotalis-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -50411,7 +50411,7 @@
     <Disorder id="19742">
       <OrphaCode>254361</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=254361</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2Q</Name>
+      <Name lang="de">Plectin-assoziierte Gliedergürtelmuskeldystrophie R17</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -50998,7 +50998,7 @@
     <Disorder id="2543">
       <OrphaCode>2798</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2798</ExpertLink>
-      <Name lang="de">Pachygyrie mit mentaler Retardierung und Epilepsie</Name>
+      <Name lang="de">Pachygyrie-Intelligenzminderung-Epilepsie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -52189,7 +52189,7 @@
     <Disorder id="2314">
       <OrphaCode>2502</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2502</ExpertLink>
-      <Name lang="de">Metaphysäre Dysostose - geistige Retardierung - Schallleitungsschwerhörigkeit</Name>
+      <Name lang="de">Metaphysäre Dysostose-Intelligenzminderung-Schallleitungsschwerhörigkeit-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -54188,7 +54188,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="de">Autosomal-dominant</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="19874">
@@ -54661,7 +54664,7 @@
     <Disorder id="2746">
       <OrphaCode>3056</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3056</ExpertLink>
-      <Name lang="de">X-chromosomale Intelligenzminderung Typ Brooks</Name>
+      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Brooks</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -54739,7 +54742,7 @@
     <Disorder id="2741">
       <OrphaCode>3052</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3052</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale - Krämpfe - Psoriasis</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Krämpfe-Psoriasis-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -54794,7 +54797,7 @@
     <Disorder id="2737">
       <OrphaCode>3044</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3044</ExpertLink>
-      <Name lang="de">Geistige Retardierung - Dysmorphien - Hypogonadismus - Diabetes mellitus</Name>
+      <Name lang="de">Intelligenzminderung-Dysmorphien-Hypogonadismus-Diabetes mellitus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -54820,7 +54823,7 @@
     <Disorder id="2735">
       <OrphaCode>3042</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3042</ExpertLink>
-      <Name lang="de">Geistige Retardierung - Katarakt - kalzifizierte Ohrknorpel - Myopathie</Name>
+      <Name lang="de">Intelligenzminderung-Katarakt-kalzifizierte Ohrknorpel-Myopathie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -54846,7 +54849,7 @@
     <Disorder id="2734">
       <OrphaCode>3041</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3041</ExpertLink>
-      <Name lang="de">Geistige Retardierung - Glatzenbildung - Patellaluxation - Akromikrie</Name>
+      <Name lang="de">Intelligenzminderung-Glatzenbildung-Patellaluxation-Akromikrie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -54950,7 +54953,7 @@
     <Disorder id="2729">
       <OrphaCode>3035</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3035</ExpertLink>
-      <Name lang="de">Wachstumsretardierung - Hydrozephalus - Lungenhypoplasie</Name>
+      <Name lang="de">Wachstumsretardierung-Hydrozephalus-Lungenhypoplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -56328,7 +56331,7 @@
     <Disorder id="2814">
       <OrphaCode>3145</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3145</ExpertLink>
-      <Name lang="de">Nephrogener Diabetes insipidus-intrakranielle Kalzifikation-Syndrom</Name>
+      <Name lang="de">Nephrogener Diabetes insipidus-intrakranielle Kalzifikation-Gesichtsdysmorphie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -56892,7 +56895,7 @@
     <Disorder id="2765">
       <OrphaCode>3078</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3078</ExpertLink>
-      <Name lang="de">Geistige Retardierung, schwere X-chromosomale, Typ Gustavson</Name>
+      <Name lang="de">Schwere X-chromosomale Intelligenzminderung Typ Gustavson</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -56921,7 +56924,7 @@
     <Disorder id="2764">
       <OrphaCode>3077</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3077</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale - Psychose - Makroorchidie</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Psychose-Makroorchidie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -56990,7 +56993,7 @@
     <Disorder id="2752">
       <OrphaCode>3063</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3063</ExpertLink>
-      <Name lang="de">X-chromosomale Intelligenzminderung Typ Snyder</Name>
+      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Snyder</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -57022,7 +57025,7 @@
     <Disorder id="2757">
       <OrphaCode>3068</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3068</ExpertLink>
-      <Name lang="de">Geistige Retardierung - Myopathie - Kleinwuchs - endokrine Störung</Name>
+      <Name lang="de">Intelligenzminderung-Myopathie-Kleinwuchs-endokrine Störung-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -57809,7 +57812,7 @@
     <Disorder id="2603">
       <OrphaCode>2872</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2872</ExpertLink>
-      <Name lang="de">Kraniosynostose - kongenitale Herzerkrankung - Intelligenzminderung</Name>
+      <Name lang="de">Kardiokraniales Syndrom Typ Pfeiffer</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -59328,7 +59331,7 @@
     <Disorder id="2662">
       <OrphaCode>2950</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2950</ExpertLink>
-      <Name lang="de">Daumen, triphalangeale - Polysyndaktylie</Name>
+      <Name lang="de">Triphalangeale Daumen-Polysyndaktylie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -59455,7 +59458,7 @@
     <Disorder id="2664">
       <OrphaCode>2952</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2952</ExpertLink>
-      <Name lang="de">Daumen, adduzierte - Arthrogrypose, Typ Christian</Name>
+      <Name lang="de">Adduzierte Daumen-Arthrogrypose-Syndrom Typ Christian</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -59562,7 +59565,7 @@
     <Disorder id="2669">
       <OrphaCode>2958</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2958</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale - Dysmorphie - zerebrale Atrophie</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Dysmorphie-zerebrale Atrophie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -59637,7 +59640,7 @@
     <Disorder id="2645">
       <OrphaCode>2928</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2928</ExpertLink>
-      <Name lang="de">Polyneuropathie - geistige Retardierung - Akromikrie - vorzeitige Menopause</Name>
+      <Name lang="de">Polyneuropathie-Intelligenzminderung-Akromikrie-vorzeitige Menopause-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -59840,7 +59843,7 @@
     <Disorder id="2625">
       <OrphaCode>2899</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2899</ExpertLink>
-      <Name lang="de">Brachyolmie - Amelogenesis imperfecta</Name>
+      <Name lang="de">Brachyolmie-Amelogenesis imperfecta-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -60162,7 +60165,7 @@
     <Disorder id="2638">
       <OrphaCode>2920</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2920</ExpertLink>
-      <Name lang="de">Polydaktylie, postaxiale - geistige Retardierung</Name>
+      <Name lang="de">Oliver-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -60211,7 +60214,7 @@
     <Disorder id="2639">
       <OrphaCode>2921</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2921</ExpertLink>
-      <Name lang="de">Präaxiale Polydaktylie - Kolobome - geistige Retardierung</Name>
+      <Name lang="de">Präaxiale Polydaktylie-Kolobom-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -61301,7 +61304,7 @@
     <Disorder id="2956">
       <OrphaCode>3332</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3332</ExpertLink>
-      <Name lang="de">Hypoplastische Tibia - postaxiale Polydaktylie</Name>
+      <Name lang="de">Hypoplastische Tibia-postaxiale Polydaktylie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -61530,7 +61533,7 @@
     <Disorder id="2945">
       <OrphaCode>3314</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3314</ExpertLink>
-      <Name lang="de">Thiemann-Krankheit</Name>
+      <Name lang="de">Thiemann-Krankheit, familiäre From</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -62362,7 +62365,7 @@
     <Disorder id="3063">
       <OrphaCode>2076</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2076</ExpertLink>
-      <Name lang="de">Geistige Retardierung - Epilepsie, X-chromosomal</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Epilepsie-Syndrom</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -62764,7 +62767,7 @@
     <Disorder id="2852">
       <OrphaCode>1856</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1856</ExpertLink>
-      <Name lang="de">Dysplasie, spondylometaphysäre periphere mit kurzer Ulna</Name>
+      <Name lang="de">Spondyloperiphere Dysplasie mit kurzer Ulna</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -62839,7 +62842,7 @@
     <Disorder id="20455">
       <OrphaCode>280333</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280333</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2P</Name>
+      <Name lang="de">Alpha-Dystroglycan-assoziierte Gliedergürtelmuskeldystrophie R16</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -64661,7 +64664,7 @@
     <Disorder id="20389">
       <OrphaCode>276608</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=276608</ExpertLink>
-      <Name lang="de">Hypoglykämie, hyperinsulinämische, ohne Insulinom, persistierende Form des Erwachsenen</Name>
+      <Name lang="de">Non-Insulinoma pankreatogenes Hypoglykämie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -65222,7 +65225,7 @@
     <Disorder id="2940">
       <OrphaCode>3304</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3304</ExpertLink>
-      <Name lang="de">Fallot-Tetralogie - Intelligenzminderung - Wachstumsverzögerungen</Name>
+      <Name lang="de">Fallot-Tetralogie mit Intelligenzminderung und Wachstumsverzögerungen</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -66901,7 +66904,7 @@
     <Disorder id="3221">
       <OrphaCode>1129</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1129</ExpertLink>
-      <Name lang="de">Arachnodaktylie - Ossifikationsstörungen - geistige Retardierung</Name>
+      <Name lang="de">Arachnodaktylie-Ossifikationsstörungen-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -67005,7 +67008,7 @@
     <Disorder id="18521">
       <OrphaCode>206554</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=206554</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2M</Name>
+      <Name lang="de">Fukutin-assoziierte Gliedergürtelmuskeldystrophie R13</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -67051,7 +67054,7 @@
     <Disorder id="18520">
       <OrphaCode>206549</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=206549</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2L</Name>
+      <Name lang="de">Anoctamin-5-assoziierte Gliedergürtelmuskeldystrophie R12</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -67080,7 +67083,7 @@
     <Disorder id="3226">
       <OrphaCode>1123</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1123</ExpertLink>
-      <Name lang="de">Appendix, kaudaler - Schwerhörigkeit</Name>
+      <Name lang="de">Kaudaler Appendix-Taubheit-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -67103,7 +67106,7 @@
     <Disorder id="18523">
       <OrphaCode>206564</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=206564</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2O</Name>
+      <Name lang="de">POMGNT1-assoziierte Gliedergürtelmuskeldystrophie R15</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -67132,7 +67135,7 @@
     <Disorder id="18522">
       <OrphaCode>206559</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=206559</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2N</Name>
+      <Name lang="de">POMT2-assoziierte Gliedergürtelmuskeldystrophie R14</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -67296,7 +67299,7 @@
     <Disorder id="18499">
       <OrphaCode>206470</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=206470</ExpertLink>
-      <Name lang="de">Zystadenom, seröses oder muzinöses, in der Kindheit</Name>
+      <Name lang="de">Zystadenom im Kindesalter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -67316,7 +67319,7 @@
     <Disorder id="3202">
       <OrphaCode>1217</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1217</ExpertLink>
-      <Name lang="de">Atrophie, spinale - Ophthalmoplegie - Pyramidenbahn-Symptomatik</Name>
+      <Name lang="de">Spinale Atrophie-Ophthalmoplegie-Pyramidenbahn-Symptomatik-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -67967,7 +67970,7 @@
     <Disorder id="3161">
       <OrphaCode>1130</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1130</ExpertLink>
-      <Name lang="de">Arachnodaktylie - geistige Retardierung - Dysmorphien</Name>
+      <Name lang="de">Arachnodaktylie-Intelligenzminderung-Dysmorphie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -67996,7 +67999,7 @@
     <Disorder id="3157">
       <OrphaCode>3207</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3207</ExpertLink>
-      <Name lang="de">Hypoplasie der weißen Gehirnsubstanz - Corpus-callosum-Agenesie - geistige Retardierung</Name>
+      <Name lang="de">Hypoplasie der weißen Gehirnsubstanz-Corpus-callosum-Agenesie-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68175,7 +68178,7 @@
     <Disorder id="3144">
       <OrphaCode>2898</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2898</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale - Plagiozephalie</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Plagiozephalie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68449,7 +68452,7 @@
     <Disorder id="3191">
       <OrphaCode>1101</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1101</ExpertLink>
-      <Name lang="de">Anophthalmie - Megalokornea - Kardiopathie - Skelettanomalien</Name>
+      <Name lang="de">Anophthalmie-Megalokornea-Kardiopathie-Skelettanomalien-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68501,7 +68504,7 @@
     <Disorder id="3188">
       <OrphaCode>1779</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1779</ExpertLink>
-      <Name lang="de">Dysmorphien - Gaumenspalte - lose Haut</Name>
+      <Name lang="de">Dysmorphien-Gaumenspalte-lose Haut-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68568,7 +68571,7 @@
     <Disorder id="3187">
       <OrphaCode>1272</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1272</ExpertLink>
-      <Name lang="de">Fine-Lubinsky-Syndrom</Name>
+      <Name lang="de">Aymé-Gripp-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68672,7 +68675,7 @@
     <Disorder id="3183">
       <OrphaCode>3051</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3051</ExpertLink>
-      <Name lang="de">Geistige Retardierung - spärliches Haar - Brachydaktylie</Name>
+      <Name lang="de">Intelligenzminderung-spärliches Haar-Brachydaktylie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -68874,7 +68877,7 @@
     <Disorder id="3096">
       <OrphaCode>1415</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1415</ExpertLink>
-      <Name lang="de">Cholestase - Pigment-Retinopathie - Gaumenspalte</Name>
+      <Name lang="de">Cholestase- Pigment-Retinopathie-Gaumenspalte-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -69114,7 +69117,7 @@
     <Disorder id="3087">
       <OrphaCode>3074</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3074</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale - Kleinwuchs - Hypertelorismus</Name>
+      <Name lang="de">Intelligenzminderung-Kleinwuchs-Hypertelorismus-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -69238,7 +69241,7 @@
     <Disorder id="3129">
       <OrphaCode>1884</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1884</ExpertLink>
-      <Name lang="de">Ectopia lentis - chorioretinale Dystrophie - Myopie</Name>
+      <Name lang="de">Ectopia lentis-chorioretinale Dystrophie-Myopie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -69938,7 +69941,7 @@
     <Disorder id="3106">
       <OrphaCode>2519</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2519</ExpertLink>
-      <Name lang="de">Mikrozephalie - Krämpfe - geistige Retardierung - Herzfehler</Name>
+      <Name lang="de">Mikrozephalie-Krämpfe-Intelligenzminderung-Kardiopathie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -70718,7 +70721,7 @@
     <Disorder id="3551">
       <OrphaCode>656</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=656</ExpertLink>
-      <Name lang="de">Nephrotisches Syndrom, idiopathisches steroid-resistentes, familiäre Form</Name>
+      <Name lang="de">Nephrotisches Syndrom, steroid-resistentes, genetisch bedingtes</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -71241,7 +71244,7 @@
     <Disorder id="3563">
       <OrphaCode>359</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=359</ExpertLink>
-      <Name lang="de">Glaukom, hereditäres</Name>
+      <Name lang="de">Glaukom, genetisch bedingtes, mit Beginn im Kindesalter</Name>
       <DisorderType id="36561">
         <Name lang="de">Category</Name>
       </DisorderType>
@@ -71339,7 +71342,7 @@
     <Disorder id="18742">
       <OrphaCode>213504</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=213504</ExpertLink>
-      <Name lang="de">Adenokarzinom, ovarielles</Name>
+      <Name lang="de">Adenokarzinom des Ovars</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -72134,7 +72137,7 @@
     <Disorder id="3484">
       <OrphaCode>3082</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3082</ExpertLink>
-      <Name lang="de">Geistige Retardierung - Polydaktylie - unkämmbare Haare</Name>
+      <Name lang="de">Intelligenzminderung-Polydaktylie-unkämmbare Haare-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -72935,7 +72938,7 @@
     <Disorder id="3513">
       <OrphaCode>2088</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=2088</ExpertLink>
-      <Name lang="de">Glykogenose Typ 11</Name>
+      <Name lang="de">Fanconi-Bickel-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -74334,7 +74337,7 @@
     <Disorder id="3425">
       <OrphaCode>980</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=980</ExpertLink>
-      <Name lang="de">Pulmonalarterienagenesie</Name>
+      <Name lang="de">Fehlen der Pulomalarterie</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -76028,7 +76031,7 @@
     <Disorder id="3379">
       <OrphaCode>1561</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1561</ExpertLink>
-      <Name lang="de">Cytochrom-C-Oxidase-Mangel, infantile fatale Form</Name>
+      <Name lang="de">Cytochrom-c-Oxidase-Mangel, infantile fatale Form</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -76386,7 +76389,7 @@
     <Disorder id="18946">
       <OrphaCode>221126</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=221126</ExpertLink>
-      <Name lang="de">Fowler-Syndrom</Name>
+      <Name lang="de">Fowler-Vaskulopathie</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -77054,7 +77057,7 @@
     <Disorder id="3759">
       <OrphaCode>319</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=319</ExpertLink>
-      <Name lang="de">Ewing-Sarkom</Name>
+      <Name lang="de">Ewing-Sarkom, skelettales</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -77687,7 +77690,7 @@
     <Disorder id="3727">
       <OrphaCode>3337</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=3337</ExpertLink>
-      <Name lang="de">Fanconi-Syndrom, primäres</Name>
+      <Name lang="de">Renotubuläres Fanconi-Syndrom, primäres</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -79946,7 +79949,7 @@
     <Disorder id="19182">
       <OrphaCode>231393</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=231393</ExpertLink>
-      <Name lang="de">Beta-Thalassämie - X-chromosomale Thrombozytopenie</Name>
+      <Name lang="de">Beta-Thalassämie-X-chromosomale Thrombozytopenie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -81548,7 +81551,7 @@
     <Disorder id="19215">
       <OrphaCode>231742</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=231742</ExpertLink>
-      <Name lang="de">Epibulbäre Lipodermoid - präaurikuläre Anhänge - Polythelie</Name>
+      <Name lang="de">Epibulbäre Lipodermoid-präaurikuläre Anhänge-Polythelie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -84278,7 +84281,7 @@
     <Disorder id="21819">
       <OrphaCode>324977</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=324977</ExpertLink>
-      <Name lang="de">Störung der Proteasom-Funktionalität</Name>
+      <Name lang="de">Proteasom-assoziiertes autoinflammatorisches Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -86167,7 +86170,7 @@
     <Disorder id="21692">
       <OrphaCode>319678</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=319678</ExpertLink>
-      <Name lang="de">Enzephalopathie - hypertrophe Kardiomyopathie - renal-tubuläre Störung</Name>
+      <Name lang="de">Enzephalopathie mit hypertropher Kardiomyopathie und renal-tubulärer Störung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -88931,7 +88934,7 @@
     <Disorder id="22434">
       <OrphaCode>369840</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=369840</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2S</Name>
+      <Name lang="de">TRAPPC11-assoziierte Gliedergürtelmuskeldystrophie R18</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -90310,7 +90313,7 @@
     <Disorder id="22329">
       <OrphaCode>363623</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=363623</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2T</Name>
+      <Name lang="de">GMPPB-assoziierte Gliedergürtelmuskeldystrophie R19</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -92089,8 +92092,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=353284</ExpertLink>
       <Name lang="de">Rubinstein-Taybi-Syndrom durch EP300-Haploinsuffizienz</Name>
-      <DisorderType id="21450">
-        <Name lang="de">Klinischer Subtyp</Name>
+      <DisorderType id="21443">
+        <Name lang="de">Ätiologischer Subtyp</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="de">Subtyp der Störung</Name>
@@ -92281,7 +92284,7 @@
     <Disorder id="22066">
       <OrphaCode>352479</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=352479</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie durch ISPD-Mangel</Name>
+      <Name lang="de">ISPD-assoziierte Gliedergürtelmuskeldystrophie R20</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -92819,32 +92822,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=293848</ExpertLink>
-      <Name lang="de">Frontotemporale Demenz, rechtstemporale Variante</Name>
-      <DisorderType id="21394">
-        <Name lang="de">Krankheit</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="de">Störung</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="de">Erwachsenenalter</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23634">
-          <Name lang="de">Ältere Erwachsene</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23480">
-          <Name lang="de">Unbekannt</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=293843</ExpertLink>
@@ -94566,7 +94543,7 @@
     <Disorder id="20609">
       <OrphaCode>284417</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=284417</ExpertLink>
-      <Name lang="de">Phosphoserin-Aminotransferase-Mangel</Name>
+      <Name lang="de">Phosphoserin-Aminotransferase-Mangel, infantile/juvenile Form</Name>
       <DisorderType id="21443">
         <Name lang="de">Ätiologischer Subtyp</Name>
       </DisorderType>
@@ -96012,7 +95989,7 @@
     <Disorder id="20508">
       <OrphaCode>280840</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280840</ExpertLink>
-      <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 2</Name>
+      <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 2</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -96038,7 +96015,7 @@
     <Disorder id="20509">
       <OrphaCode>280847</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280847</ExpertLink>
-      <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 3</Name>
+      <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 3</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -96064,7 +96041,7 @@
     <Disorder id="20510">
       <OrphaCode>280854</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280854</ExpertLink>
-      <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 4</Name>
+      <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 4</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -96090,7 +96067,7 @@
     <Disorder id="20506">
       <OrphaCode>280827</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=280827</ExpertLink>
-      <Name lang="de">Atemwegsfehlbildung, kongenitale, Typ 0</Name>
+      <Name lang="de">Kongenitale Fehlbildung der Atemwege Typ 0</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -96595,26 +96572,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20606">
-      <OrphaCode>284408</OrphaCode>
-      <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=284408</ExpertLink>
-      <Name lang="de">Glycerol-Kinase-Mangel, infantiler</Name>
-      <DisorderType id="21450">
-        <Name lang="de">Klinischer Subtyp</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="de">Subtyp der Störung</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="de">Kleinkindalter</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20607">
       <OrphaCode>284411</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=284411</ExpertLink>
@@ -100260,7 +100217,7 @@
     <Disorder id="21116">
       <OrphaCode>300373</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=300373</ExpertLink>
-      <Name lang="de">Gigantismus, infantiler, familiäre Form</Name>
+      <Name lang="de">Akrogigantismus, X-chromosomaler</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -102544,7 +102501,7 @@
     <Disorder id="23767">
       <OrphaCode>459033</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=459033</ExpertLink>
-      <Name lang="de">Ataxie - okulomotorische Apraxie, Typ 4</Name>
+      <Name lang="de">Ataxie mit okulomotorischer Apraxie Typ 4</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -103170,7 +103127,7 @@
     <Disorder id="23724">
       <OrphaCode>457375</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=457375</ExpertLink>
-      <Name lang="de">ITPA-abhängige Enzephalopathie</Name>
+      <Name lang="de">ITPA-assoziierte letale infantile neurologische Störung mit Katarakt und kardialer Beteiligung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -104066,7 +104023,7 @@
     <Disorder id="7037">
       <OrphaCode>225</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=225</ExpertLink>
-      <Name lang="de">Diabetes - Schwerhörigkeit, maternal vererbt</Name>
+      <Name lang="de">Maternal vererbter Diabetes mit Schwerhörigkeit</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -104138,7 +104095,7 @@
     <Disorder id="24158">
       <OrphaCode>466688</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=466688</ExpertLink>
-      <Name lang="de">Schwere geistige Retardierung - Corpus callosum-Agenesie - Gesichtsdysmorphien -zerebelläre Ataxie</Name>
+      <Name lang="de">Schwere Intelligenzminderung-Corpus callosum-Agenesie-faziale Dysmorphien-zerebelläre Ataxie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -112421,7 +112378,7 @@
     <Disorder id="8029">
       <OrphaCode>854</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=854</ExpertLink>
-      <Name lang="de">Pfortaderthrombose</Name>
+      <Name lang="de">Pfortaderthrombose, primäre</Name>
       <DisorderType id="21422">
         <Name lang="de">Klinisches Syndrom</Name>
       </DisorderType>
@@ -112745,7 +112702,7 @@
     <Disorder id="23470">
       <OrphaCode>443950</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=443950</ExpertLink>
-      <Name lang="de">Charcot-Marie-Tooth-Krankheit, Typ 2, DNAJB2-assoziierte</Name>
+      <Name lang="de">Charcot-Marie-Tooth-Krankheit, autosomal-rezessive, Typ 2, DNAJB2-assoziierte</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -117073,7 +117030,7 @@
     <Disorder id="8716">
       <OrphaCode>353</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=353</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2C</Name>
+      <Name lang="de">Gamma-Sarkoglykan-assoziierte Gliedergürtelmuskeldystrophie R5</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -117099,7 +117056,7 @@
     <Disorder id="8717">
       <OrphaCode>219</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=219</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2F</Name>
+      <Name lang="de">Delta-Sarkoglykan-assoziierte Gliedergürtelmuskeldystrophie R6</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -117157,7 +117114,7 @@
     <Disorder id="8715">
       <OrphaCode>119</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=119</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2E</Name>
+      <Name lang="de">Beta-Sarkoglykan-assoziierte Gliedergürtelmuskeldystrophie R4</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -117385,7 +117342,7 @@
     <Disorder id="8720">
       <OrphaCode>268</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=268</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-dominante, Typ 2B</Name>
+      <Name lang="de">Dysferlin-assoziierte Gliedergürtelmuskeldystrophie R2</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -118689,7 +118646,7 @@
     <Disorder id="25682">
       <OrphaCode>494541</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=494541</ExpertLink>
-      <Name lang="de">Benigne Chorea-striatale Beteiligung-Syndrom mit Beginn im Kindesalter</Name>
+      <Name lang="de">Im Kindesalter beginnende benigne Chorea mit striataler Beteiligung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -120822,7 +120779,7 @@
     <Disorder id="25864">
       <OrphaCode>497623</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=497623</ExpertLink>
-      <Name lang="de">Kombinierter Defekt der oxidativen Phosphorylierung, C12ORF65-assoziierter</Name>
+      <Name lang="de">C12ORF65-assoziierter kombinierter Defekt der oxidativen Phosphorylierung</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -121631,7 +121588,7 @@
     <Disorder id="8672">
       <OrphaCode>786</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=786</ExpertLink>
-      <Name lang="de">Glukokortikoid-Resistenz</Name>
+      <Name lang="de">Glukokortikoid-Resistenz, generalisierte</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -121833,7 +121790,7 @@
     <Disorder id="8597">
       <OrphaCode>1330</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1330</ExpertLink>
-      <Name lang="de">Atrioventrikulärer Kanal, partieller</Name>
+      <Name lang="de">Atrioventrikulärer Septumdefekt, partieller</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -121856,7 +121813,7 @@
     <Disorder id="8599">
       <OrphaCode>1677</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=1677</ExpertLink>
-      <Name lang="de">Dilatation des rechten Atrium (Vorhof), familiär</Name>
+      <Name lang="de">Familiäre idiopathische rechtsseitige Vorhofdilatation</Name>
       <DisorderType id="21415">
         <Name lang="de">Morphologische Anomalie</Name>
       </DisorderType>
@@ -122035,7 +121992,7 @@
     <Disorder id="25934">
       <OrphaCode>498693</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=498693</ExpertLink>
-      <Name lang="de">Arthrogryposis multiplex congenita-Syndrom, autosomal-rezessives, nicht-tödliches, MYBPC1-assoziiertes</Name>
+      <Name lang="de">Arthrogryposis multiplex congenita-Syndrom, autosomal-rezessives, nicht-letales, MYBPC1-assoziiertes</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -123619,7 +123576,7 @@
     <Disorder id="25220">
       <OrphaCode>480682</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=480682</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2Z</Name>
+      <Name lang="de">POGLUT1-assoziierte Gliedergürtelmuskeldystrophie R21</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -125788,7 +125745,7 @@
     <Disorder id="10989">
       <OrphaCode>71267</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=71267</ExpertLink>
-      <Name lang="de">Dentinogenesis imperfecta - Kleinwuchs - Hörverlust - geistige Retardierung</Name>
+      <Name lang="de">Dentinogenesis imperfecta-Kleinwuchs-Hörverlust-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -126870,7 +126827,7 @@
     <Disorder id="10912">
       <OrphaCode>69076</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=69076</ExpertLink>
-      <Name lang="de">Renale Glukosurie, hereditäre Form</Name>
+      <Name lang="de">Renale Glukosurie, familiäre</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -127119,7 +127076,7 @@
     <Disorder id="10905">
       <OrphaCode>67045</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=67045</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale mit isoliertem Wachstumshormonmangel</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung mit isoliertem Wachstumshormonmangel</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -127174,7 +127131,7 @@
     <Disorder id="10903">
       <OrphaCode>67043</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=67043</ExpertLink>
-      <Name lang="de">Acanthamoeba-Keratitis</Name>
+      <Name lang="de">Amöbenkeratitis</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -128009,7 +127966,7 @@
     <Disorder id="10875">
       <OrphaCode>65720</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=65720</ExpertLink>
-      <Name lang="de">Arthrogrypose - schwere Skoliose</Name>
+      <Name lang="de">Arthrogrypose-schwere Skoliose-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -129746,7 +129703,7 @@
     <Disorder id="28372">
       <OrphaCode>563690</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=563690</ExpertLink>
-      <Name lang="de">Fliegenmadenbefall durch Cordylobia rodhaini</Name>
+      <Name lang="de">Furunkuloide Myiasis durch Cordylobia rodhaini</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -130039,7 +129996,7 @@
     <Disorder id="10758">
       <OrphaCode>55595</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=55595</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-dominante, Typ 1F</Name>
+      <Name lang="de">TNP03-assoziierte Gliedergürtelmuskeldystrophie D2</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -130074,7 +130031,7 @@
     <Disorder id="10759">
       <OrphaCode>55596</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=55596</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-dominante, Typ 1G</Name>
+      <Name lang="de">HNRNPDL-assoziierte Gliedergürtelmuskeldystrophie D3</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -131366,7 +131323,7 @@
     <Disorder id="11149">
       <OrphaCode>79107</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=79107</ExpertLink>
-      <Name lang="de">Entwicklungsdefekte - Schwerhörigkeit - Dystonie</Name>
+      <Name lang="de">Entwicklungsdefekte-Schwerhörigkeit-Dystonie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -132022,7 +131979,7 @@
     <Disorder id="11163">
       <OrphaCode>79137</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=79137</ExpertLink>
-      <Name lang="de">Epilepsie, generalisierte - paroxysmale Dyskinesie</Name>
+      <Name lang="de">Generalisierte Epilepsie-paroxysmale Dyskinesie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -134869,7 +134826,7 @@
     <Disorder id="11035">
       <OrphaCode>73223</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=73223</ExpertLink>
-      <Name lang="de">Allgemeine Entwicklungsverzögerung - Osteopenie - ektodermaler Defekt</Name>
+      <Name lang="de">Allgemeine Entwicklungsverzögerung-Osteopenie-ektodermaler Defekt-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -134895,7 +134852,7 @@
     <Disorder id="11034">
       <OrphaCode>73220</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=73220</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale - hypotones Gesicht</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Syndrom-hypotones Gesicht-Syndrom</Name>
       <DisorderType id="21436">
         <Name lang="de">Phenom-Gruppe</Name>
       </DisorderType>
@@ -137535,7 +137492,7 @@
     <Disorder id="10336">
       <OrphaCode>34514</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=34514</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2G</Name>
+      <Name lang="de">Telethonin-assoziierte Gliedergürtelmuskeldystrophie R7</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -137567,7 +137524,7 @@
     <Disorder id="10337">
       <OrphaCode>34515</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=34515</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2I</Name>
+      <Name lang="de">FKRP-assoziierte Gliedergürtelmuskeldystrophie R9</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -137602,7 +137559,7 @@
     <Disorder id="10338">
       <OrphaCode>34516</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=34516</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-dominante, Typ 1D</Name>
+      <Name lang="de">DNAJB6-assoziierte Gliedergürtelmuskeldystrophie D1</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -137680,7 +137637,7 @@
     <Disorder id="10344">
       <OrphaCode>34526</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=34526</ExpertLink>
-      <Name lang="de">Hypomagnesiämie, familiäre primäre</Name>
+      <Name lang="de">Hypomagnesiämie, primäre, genetisch bedingte</Name>
       <DisorderType id="36561">
         <Name lang="de">Category</Name>
       </DisorderType>
@@ -139105,7 +139062,7 @@
     <Disorder id="10738">
       <OrphaCode>53696</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=53696</ExpertLink>
-      <Name lang="de">Arthrogrypose, letale - Verlust von Vorderhornzellen</Name>
+      <Name lang="de">Arthrogrypose-anteriore Hornzellkrankheit-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -142447,7 +142404,7 @@
     <Disorder id="11944">
       <OrphaCode>90053</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90053</ExpertLink>
-      <Name lang="de">Hämatopoetische Stammzell-Transplantation</Name>
+      <Name lang="de">Komplikationen nach hämatopoetischer Stammzell-Transplantation</Name>
       <DisorderType id="21429">
         <Name lang="de">Besondere klinische Situation bei einer Krankheit oder einem Syndrom</Name>
       </DisorderType>
@@ -143542,7 +143499,7 @@
     <Disorder id="11973">
       <OrphaCode>90103</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90103</ExpertLink>
-      <Name lang="de">Charcot-Marie-Tooth-Krankheit - Schwerhörigkeit - Intelligenzminderung</Name>
+      <Name lang="de">Charcot-Marie-Tooth-Krankheit-Schwerhörigkeit-Intelligenzminderung-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -147550,7 +147507,7 @@
     <Disorder id="12081">
       <OrphaCode>90695</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90695</ExpertLink>
-      <Name lang="de">Panhypopituitarismus</Name>
+      <Name lang="de">Panhypopituitarismus, genetisch bedingter</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -147865,7 +147822,7 @@
     <Disorder id="12034">
       <OrphaCode>90390</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=90390</ExpertLink>
-      <Name lang="de">Anonychie - Onychodystrophie</Name>
+      <Name lang="de">Anonychie-Onychodystrophie-Syndrom</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -152065,7 +152022,7 @@
     <Disorder id="11376">
       <OrphaCode>79350</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=79350</ExpertLink>
-      <Name lang="de">3-Phosphoserin-Phosphatase-Mangel</Name>
+      <Name lang="de">3-Phosphoserin-Phosphatase-Mangel, infantile/juvenile Form</Name>
       <DisorderType id="21443">
         <Name lang="de">Ätiologischer Subtyp</Name>
       </DisorderType>
@@ -152950,7 +152907,7 @@
     <Disorder id="11683">
       <OrphaCode>85293</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85293</ExpertLink>
-      <Name lang="de">Cabezas-Syndrom</Name>
+      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Cabezas</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -153100,7 +153057,7 @@
     <Disorder id="11684">
       <OrphaCode>85294</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85294</ExpertLink>
-      <Name lang="de">Epilepsie - Lernstörungen - Verhaltensauffälligkeiten, X-chromosomal</Name>
+      <Name lang="de">X-chromosomale Epilepsie mit Lernstörungen und Verhaltensauffälligkeiten</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -153123,7 +153080,7 @@
     <Disorder id="11691">
       <OrphaCode>85321</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85321</ExpertLink>
-      <Name lang="de">Schwerhörigkeit-geistige Retardierung, Typ Martin-Probst</Name>
+      <Name lang="de">Schwerhörigkeit-Intelligenzminderung-Syndrom vom Typ Martin-Probst</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -153577,7 +153534,7 @@
     <Disorder id="11708">
       <OrphaCode>85338</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85338</ExpertLink>
-      <Name lang="de">Ataxie - Apraxie - geistige Retardierung, X-chromosomal</Name>
+      <Name lang="de">X-chromosomale Intelligenzminderung-Ataxie-Apraxie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -154022,7 +153979,7 @@
     <Disorder id="11732">
       <OrphaCode>86812</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=86812</ExpertLink>
-      <Name lang="de">Gliedergürtelmuskeldystrophie, autosomal-rezessive, Typ 2K</Name>
+      <Name lang="de">POMT1-assoziierte Gliedergürtelmuskeldystrophie R11</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -154299,7 +154256,7 @@
     <Disorder id="11738">
       <OrphaCode>86818</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=86818</ExpertLink>
-      <Name lang="de">Alport-Syndrom - geistige Retardierung - Mittelgesichtshypoplasie - Elliptozytose</Name>
+      <Name lang="de">Alport-Syndrom-Intelligenzminderung-Mittelgesichtshypoplasie-Elliptozytose-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -156607,7 +156564,7 @@
     <Disorder id="11600">
       <OrphaCode>83629</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=83629</ExpertLink>
-      <Name lang="de">Leukoenzephalopathie - metaphysäre Chondrodysplasie</Name>
+      <Name lang="de">Leukoenzephalopathie-spondylometaphysäre Dysplasie-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -156937,7 +156894,7 @@
     <Disorder id="11618">
       <OrphaCode>84132</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=84132</ExpertLink>
-      <Name lang="de">Myopathie, desmin-abhängige, mit Mallory Körperchen-ähnlichen Einschlüssen</Name>
+      <Name lang="de">Desmin-abhängige Myopathie mit Mallory Körperchen-ähnlichen Einschlüssen</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -157018,7 +156975,7 @@
     <Disorder id="11647">
       <OrphaCode>85186</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85186</ExpertLink>
-      <Name lang="de">Endosteale Sklerose - zerebelläre Hypoplasie</Name>
+      <Name lang="de">Endosteale Sklerose-zerebelläre Hypoplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -157264,7 +157221,7 @@
     <Disorder id="11636">
       <OrphaCode>85169</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85169</ExpertLink>
-      <Name lang="de">Arthropathie, digitale - Brachydaktylie, familiäre Form</Name>
+      <Name lang="de">Familiäre digitale Arthropathie mit Brachydaktylie</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -157316,7 +157273,7 @@
     <Disorder id="11634">
       <OrphaCode>85167</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85167</ExpertLink>
-      <Name lang="de">Dysplasie, spondylometaphysäre - Zapfen-Stäbchendystrophie</Name>
+      <Name lang="de">Spondylometaphysäre Dysplasie mit Zapfen-Stäbchendystrophie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -157371,7 +157328,7 @@
     <Disorder id="11632">
       <OrphaCode>85165</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=85165</ExpertLink>
-      <Name lang="de">Achondroplasie, schwere - Entwicklungsverzögerung - Acanthosis nigricans</Name>
+      <Name lang="de">Schwere Achondroplasie-Entwicklungsverzögerung-Acanthosis nigricans-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -158303,7 +158260,7 @@
     <Disorder id="12922">
       <OrphaCode>97548</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=97548</ExpertLink>
-      <Name lang="de">Ivemark-Syndrom</Name>
+      <Name lang="de">Rechtsatriale Isomerie</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -162110,7 +162067,7 @@
     <Disorder id="12289">
       <OrphaCode>93358</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93358</ExpertLink>
-      <Name lang="de">Dysplasie, spondyloepimetaphysäre - kurze Extremitäten - anormale Kalzifizierung</Name>
+      <Name lang="de">Spondyloepimetaphysäre Dysplasie mit kurzen Extremitäten und anormaler Kalzifizierung</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -162963,7 +162920,7 @@
     <Disorder id="12508">
       <OrphaCode>93950</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93950</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale, Typ Sutherland-Haan</Name>
+      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Sutherland-Haan</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -163041,7 +162998,7 @@
     <Disorder id="12505">
       <OrphaCode>93945</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93945</ExpertLink>
-      <Name lang="de">Geistige Retardierung, X-chromosomale, Typ Porteous</Name>
+      <Name lang="de">Intelligenzminderung, X-chromosomale, Typ Porteous</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -163416,7 +163373,7 @@
     <Disorder id="12423">
       <OrphaCode>93585</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=93585</ExpertLink>
-      <Name lang="de">Thrombotisch-thrombozytopenische Purpura, erworbene Form</Name>
+      <Name lang="de">Thrombotisch-thrombozytopenische Purpura, immun-vermittelte</Name>
       <DisorderType id="21450">
         <Name lang="de">Klinischer Subtyp</Name>
       </DisorderType>
@@ -165133,7 +165090,7 @@
     <Disorder id="13867">
       <OrphaCode>98850</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=98850</ExpertLink>
-      <Name lang="de">Mastozytose, systemische, agressive Form</Name>
+      <Name lang="de">Mastzell-Leukämie, agressive</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -165168,7 +165125,7 @@
     <Disorder id="13866">
       <OrphaCode>98849</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=98849</ExpertLink>
-      <Name lang="de">Mastozytose, systemische, assoziiert mit klonaler hämatologischer nicht-Mastzellen-Krankheit</Name>
+      <Name lang="de">Mastozytose, systemische, mit assoziierter hämatologischer Neoplasie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168315,7 +168272,7 @@
     <Disorder id="28932">
       <OrphaCode>580940</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=580940</ExpertLink>
-      <Name lang="de">QRICH1-related intellectual disability-chondrodysplasia syndrome</Name>
+      <Name lang="de">QRICH1-assoziierte Intelligenzminderung-Chondrodysplasie-Syndrom</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -168341,7 +168298,7 @@
     <Disorder id="28934">
       <OrphaCode>580951</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=580951</ExpertLink>
-      <Name lang="de">Punctate inner choroidopathy</Name>
+      <Name lang="de">Punktierte innere Choroidopathie</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -168361,7 +168318,7 @@
     <Disorder id="28931">
       <OrphaCode>580933</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=580933</ExpertLink>
-      <Name lang="de">Lethal brain and heart developmental defects</Name>
+      <Name lang="de">Gehirn- und Herzentwicklungsdefekte, letale</Name>
       <DisorderType id="21401">
         <Name lang="de">Fehlbildungs-Syndrom</Name>
       </DisorderType>
@@ -168491,7 +168448,7 @@
     <Disorder id="28936">
       <OrphaCode>581271</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=581271</ExpertLink>
-      <Name lang="de">Cramp-fasciculation syndrome</Name>
+      <Name lang="de">Crampus-Faszikulations-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
@@ -174395,7 +174352,7 @@
     <Disorder id="14365">
       <OrphaCode>99792</OrphaCode>
       <ExpertLink lang="de">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=de&amp;Expert=99792</ExpertLink>
-      <Name lang="de">Dentindysplasie - Knochensklerose</Name>
+      <Name lang="de">Dentindysplasie-Knochensklerose-Syndrom</Name>
       <DisorderType id="21394">
         <Name lang="de">Krankheit</Name>
       </DisorderType>
