commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare diseases natural history/en_product9_ages.xml b/Epidemiological data/Rare diseases natural history/en_product9_ages.xml
old mode 100644
new mode 100755
index 48fc58a..ee165fa
--- a/Epidemiological data/Rare diseases natural history/en_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/en_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:39:33" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:40:30" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6586">
+  <DisorderList count="6584">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -10282,7 +10282,7 @@
     <Disorder id="421">
       <OrphaCode>1146</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-      <Name lang="en">Digitotalar dysmorphism</Name>
+      <Name lang="en">Distal arthrogryposis type 1</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -54188,7 +54188,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="en">Autosomal dominant</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="19874">
@@ -77302,7 +77305,7 @@
     <Disorder id="3735">
       <OrphaCode>2126</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2126</ExpertLink>
-      <Name lang="en">Solitary fibrous tumour/hemangiopericytoma</Name>
+      <Name lang="en">Solitary fibrous tumor/hemangiopericytoma</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -86216,7 +86219,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-      <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+      <Name lang="en">Alazami syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -90503,7 +90506,7 @@
     <Disorder id="22327">
       <OrphaCode>363611</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-      <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+      <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -92089,8 +92092,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
       <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -92819,32 +92822,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293848</ExpertLink>
-      <Name lang="en">Frontotemporal dementia, right temporal atrophy variant</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="en">Adult</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23634">
-          <Name lang="en">elderly</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23480">
-          <Name lang="en">Unknown</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=293843</ExpertLink>
@@ -96595,26 +96572,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20606">
-      <OrphaCode>284408</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284408</ExpertLink>
-      <Name lang="en">Glycerol kinase deficiency, infantile form</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="en">Infancy</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20607">
       <OrphaCode>284411</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284411</ExpertLink>
@@ -100431,7 +100388,7 @@
     <Disorder id="24042">
       <OrphaCode>464738</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-      <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+      <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -100770,7 +100727,7 @@
     <Disorder id="24022">
       <OrphaCode>464359</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464359</ExpertLink>
-      <Name lang="en">Benign metanephric tumour</Name>
+      <Name lang="en">Benign metanephric tumor</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -118923,7 +118880,7 @@
     <Disorder id="26002">
       <OrphaCode>500163</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=500163</ExpertLink>
-      <Name lang="en">SIN3A-related intellectual disability syndrome</Name>
+      <Name lang="en">Witteveen-Kolk syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -125762,7 +125719,7 @@
     <Disorder id="10986">
       <OrphaCode>71211</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=71211</ExpertLink>
-      <Name lang="en">Neuromyelitis optica</Name>
+      <Name lang="en">Neuromyelitis optica spectrum disorder</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
