commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Epidemiological data/Rare diseases natural history/en_product9_ages.xml b/Epidemiological data/Rare diseases natural history/en_product9_ages.xml
index 2553a5c..344e873 100644
--- a/Epidemiological data/Rare diseases natural history/en_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/en_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:54:18" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:54:39" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6653">
+  <DisorderList count="6654">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -27179,7 +27179,7 @@
     <Disorder id="16886">
       <OrphaCode>139402</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=139402</ExpertLink>
-      <Name lang="en">Drug rash with eosinophilia and systemic symptoms</Name>
+      <Name lang="en">Drug reaction with eosinophilia and systemic symptoms</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -61984,7 +61984,10 @@
           <Name lang="en">Adolescent</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23641">
+          <Name lang="en">normal life expectancy</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -108493,7 +108496,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="en">Autosomal dominant</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="en">Autosomal recessive</Name>
         </TypeOfInheritance>
@@ -117007,9 +117013,15 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="en">All ages</Name>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="en">Adolescent</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="en">Adult</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="en">Elderly</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
@@ -133180,7 +133192,7 @@
     <Disorder id="11106">
       <OrphaCode>77293</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77293</ExpertLink>
-      <Name lang="en">Niemann-Pick disease type B</Name>
+      <Name lang="en">Chronic visceral acid sphingomyelinase deficiency</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -133255,7 +133267,7 @@
     <Disorder id="11105">
       <OrphaCode>77292</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=77292</ExpertLink>
-      <Name lang="en">Niemann-Pick disease type A</Name>
+      <Name lang="en">Infantile neurovisceral acid sphingomyelinase deficiency</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -171386,6 +171398,32 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="28850">
+      <OrphaCode>576278</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=576278</ExpertLink>
+      <Name lang="en">SATB2-associated syndrome</Name>
+      <DisorderType id="21401">
+        <Name lang="en">Malformation syndrome</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23522">
+          <Name lang="en">Infancy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="en">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="en">No data available</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="28815">
       <OrphaCode>573253</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=573253</ExpertLink>
@@ -177367,7 +177405,7 @@
     <Disorder id="31372">
       <OrphaCode>617919</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=617919</ExpertLink>
-      <Name lang="en">F12-associated cold auto-inflammatory syndrome</Name>
+      <Name lang="en">F12-associated cold autoinflammatory syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
