commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml
index 4d6b4b7..a82b4ee 100644
--- a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:40:14" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:29:57" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="6636">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -1964,7 +1957,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=593</ExpertLink>
       <Name lang="es">Miopatía miofibrilar</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -6156,7 +6149,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2443</ExpertLink>
       <Name lang="es">Trastorno de la fosforilación oxidativa mitocondrial por anomalías del ADN nuclear</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -6472,7 +6465,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99</ExpertLink>
       <Name lang="es">Ataxia cerebelosa autosómica dominante</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -8534,7 +8527,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=352</ExpertLink>
       <Name lang="es">Galactosemia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -13438,7 +13431,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=69</ExpertLink>
       <Name lang="es">Amiloidosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -13504,7 +13497,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166</ExpertLink>
       <Name lang="es">Enfermedad de Charcot-Marie-Tooth/Neuropatía sensitivo-motora hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -15333,7 +15326,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=174590</ExpertLink>
       <Name lang="es">Hipogonadismo hipogonadotrópico congénito</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -16201,7 +16194,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=187</ExpertLink>
       <Name lang="es">Citrulinemia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -16265,7 +16258,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=442</ExpertLink>
       <Name lang="es">Hipotiroidismo congénito</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -17191,7 +17184,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=795</ExpertLink>
       <Name lang="es">Forma rara de la salmonelosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -19019,7 +19012,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1531</ExpertLink>
       <Name lang="es">Craneosinostosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -22136,7 +22129,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183422</ExpertLink>
       <Name lang="es">Síndromes genéticos polimalformativos con aumento del riesgo de desarrollar cáncer</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -22272,7 +22265,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=182090</ExpertLink>
       <Name lang="es">Hipertensión arterial pulmonar</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -22333,7 +22326,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1047</ExpertLink>
       <Name lang="es">Anemia sideroblástica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -34847,7 +34840,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=140874</ExpertLink>
       <Name lang="es">Síndrome de Joubert y trastornos relacionados</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -39229,7 +39222,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158032</ExpertLink>
       <Name lang="es">Síndrome hemofagocítico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -41615,7 +41608,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=247765</ExpertLink>
       <Name lang="es">Ataxia cerebelosa ligada al cromosoma X</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -42410,7 +42403,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=247582</ExpertLink>
       <Name lang="es">Deficiencia de citrina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -43777,7 +43770,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=247239</ExpertLink>
       <Name lang="es">Ataxia degenerativa no hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -47722,7 +47715,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251355</ExpertLink>
       <Name lang="es">Anemia falciforme asociada a otra anomalía de la hemoglobina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -50534,7 +50527,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=254367</ExpertLink>
       <Name lang="es">Liquen plano raro</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -52963,7 +52956,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=261911</ExpertLink>
       <Name lang="es">Deleción parcial del brazo corto del cromosoma 7</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -58142,7 +58135,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2846</ExpertLink>
       <Name lang="es">Anomalía congénita del pericardio</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -60787,7 +60780,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=275791</ExpertLink>
       <Name lang="es">Hipertensión arterial pulmonar asociada a otra enfermedad</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -68463,7 +68456,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=206997</ExpertLink>
       <Name lang="es">Miositis parasítica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -70719,7 +70712,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=599</ExpertLink>
       <Name lang="es">Miopatía distal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -70872,7 +70865,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2034</ExpertLink>
       <Name lang="es">Filariasis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -71095,7 +71088,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2415</ExpertLink>
       <Name lang="es">Malformación linfática rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -71307,7 +71300,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=137</ExpertLink>
       <Name lang="es">Trastorno congénito de la glicosilación</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -71551,7 +71544,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=359</ExpertLink>
       <Name lang="es">Glaucoma de inicio pediátrico de origen genético</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -72609,7 +72602,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=216675</ExpertLink>
       <Name lang="es">Transposición de las grandes arterias</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -73633,7 +73626,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217074</ExpertLink>
       <Name lang="es">Carcinoma raro de páncreas</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -74159,7 +74152,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=344</ExpertLink>
       <Name lang="es">Fiebre por arbovirus</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -74185,7 +74178,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=341</ExpertLink>
       <Name lang="es">Fiebre hemorrágica vírica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -74549,7 +74542,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1132</ExpertLink>
       <Name lang="es">Arcos aórticos anómalos</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -74893,7 +74886,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217604</ExpertLink>
       <Name lang="es">Miocardiopatía dilatada</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -75919,7 +75912,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=220489</ExpertLink>
       <Name lang="es">Hemocromatosis hereditaria rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76934,7 +76927,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3398</ExpertLink>
       <Name lang="es">Neoplasia epitelial tímica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -76986,7 +76979,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=547</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -77610,7 +77603,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=226292</ExpertLink>
       <Name lang="es">Hipotiroidismo congénito permanente</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -78305,7 +78298,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1172</ExpertLink>
       <Name lang="es">Ataxia cerebelosa autosómica recesiva</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -80306,7 +80299,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231413</ExpertLink>
       <Name lang="es">Variante del síndrome de Guillain-Barré</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -80335,7 +80328,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1866</ExpertLink>
       <Name lang="es">Distonía focal, segmentaria o multifocal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -80540,7 +80533,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231230</ExpertLink>
       <Name lang="es">Beta-talasemia asociada a otra anomalía de la hemoglobina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -81790,7 +81783,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231637</ExpertLink>
       <Name lang="es">Hiperaldosteronismo primario raro corregible por cirugía</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -81946,7 +81939,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231641</ExpertLink>
       <Name lang="es">Hiperaldosteronismo primario raro no corregible por cirugía</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -82803,7 +82796,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=238547</ExpertLink>
       <Name lang="es">Policitemia secundaria adquirida</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -82855,7 +82848,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=238536</ExpertLink>
       <Name lang="es">Policitemia secundaria congénita</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -86112,7 +86105,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319543</ExpertLink>
       <Name lang="es">Susceptibilidad mendeliana autosómica dominante a enfermedades micobacterianas por deficiencia parcial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -86164,7 +86157,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319535</ExpertLink>
       <Name lang="es">Susceptibilidad mendeliana autosómica recesiva a enfermedades micobacterianas por deficiencia completa</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -86187,7 +86180,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319539</ExpertLink>
       <Name lang="es">Susceptibilidad mendeliana autosómica recesiva a enfermedades micobacterianas por deficiencia parcial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -93401,7 +93394,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293830</ExpertLink>
       <Name lang="es">Anemia diseritropoyética constitucional</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -93473,7 +93466,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293815</ExpertLink>
       <Name lang="es">Dermatosis tóxica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -95800,7 +95793,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280926</ExpertLink>
       <Name lang="es">Enfermedades sistémicas con uveítis anterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -95899,7 +95892,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280898</ExpertLink>
       <Name lang="es">Panuveítis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -95925,7 +95918,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280892</ExpertLink>
       <Name lang="es">Uveítis posterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -95951,7 +95944,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280886</ExpertLink>
       <Name lang="es">Uveítis anterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -99526,7 +99519,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300579</ExpertLink>
       <Name lang="es">Toxemia estafilocócica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -99725,7 +99718,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300846</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin de células B agresivo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -99919,7 +99912,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300842</ExpertLink>
       <Name lang="es">Linfoma no Hodgkin de células B indolente</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -100478,7 +100471,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=298644</ExpertLink>
       <Name lang="es">Trastorno del metabolismo y transporte de la tiamina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -106139,7 +106132,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=401854</ExpertLink>
       <Name lang="es">Defecto de la biosíntesis de ácido lipoico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -108896,7 +108889,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398043</ExpertLink>
       <Name lang="es">Tumor maligno del pene</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -109975,7 +109968,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398934</ExpertLink>
       <Name lang="es">Tumor epitelial maligno de ovario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -110015,7 +110008,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398940</ExpertLink>
       <Name lang="es">Tumor no epitelial maligno de ovario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -110320,7 +110313,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=398091</ExpertLink>
       <Name lang="es">Enfermedad autoinmune neonatal secundaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -110429,7 +110422,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=435743</ExpertLink>
       <Name lang="es">Anomalía congénita uracal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -116154,7 +116147,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=431353</ExpertLink>
       <Name lang="es">Enfermedad veno-oclusiva pulmonar y/o hemangiomatosis capilar pulmonar</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -116507,7 +116500,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=434809</ExpertLink>
       <Name lang="es">Síndrome con cabello lanoso</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -118556,7 +118549,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=506060</ExpertLink>
       <Name lang="es">Tumor neuroendocrino de páncreas funcionante</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -120620,7 +120613,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=450</ExpertLink>
       <Name lang="es">Heterotaxia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -120678,7 +120671,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=224</ExpertLink>
       <Name lang="es">Diabetes mellitus neonatal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -122034,7 +122027,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=877</ExpertLink>
       <Name lang="es">Tumor neuroendocrino</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -122147,7 +122140,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=467</ExpertLink>
       <Name lang="es">Deficiencia combinada no adquirida de hormonas hipofisarias</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -127660,7 +127653,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=69028</ExpertLink>
       <Name lang="es">Disostosis con braquidactilia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -129745,7 +129738,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=63454</ExpertLink>
       <Name lang="es">Displasia en patrón del epitelio pigmentario de la retina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -129797,7 +129790,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=63443</ExpertLink>
       <Name lang="es">Tumor epitelial raro de estómago</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -130531,7 +130524,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=56970</ExpertLink>
       <Name lang="es">Enfermedad priónica humana</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -131553,7 +131546,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79196</ExpertLink>
       <Name lang="es">Enfermedad del ciclo de la gamma-glutamil</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -132013,7 +132006,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79168</ExpertLink>
       <Name lang="es">Anomalía en la síntesis de ácidos biliares</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -133322,7 +133315,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=567564</ExpertLink>
       <Name lang="es">Síndrome nefrótico sin manifestaciones extrarrenales</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -134259,7 +134252,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=77240</ExpertLink>
       <Name lang="es">Linfedema primario</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -135484,7 +135477,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73014</ExpertLink>
       <Name lang="es">Diarrea intratable infantil</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -136114,7 +136107,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35705</ExpertLink>
       <Name lang="es">Trastorno neurometabólico por deficiencia de serina</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -136322,7 +136315,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35698</ExpertLink>
       <Name lang="es">Síndrome de depleción del ADN mitocondrial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -136693,7 +136686,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=36388</ExpertLink>
       <Name lang="es">Síndrome neurológico paraneoplásico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -137082,7 +137075,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35808</ExpertLink>
       <Name lang="es">Tumor maligno del estroma de los cordones sexuales de los ovarios</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -137108,7 +137101,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35807</ExpertLink>
       <Name lang="es">Tumor maligno de ovario de células germinales</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -138307,7 +138300,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=34533</ExpertLink>
       <Name lang="es">Distrofia corneal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -140027,7 +140020,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=48471</ExpertLink>
       <Name lang="es">Lisencefalia</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -144403,7 +144396,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=525677</ExpertLink>
       <Name lang="es">Malformación congénita genética del ojo con glaucoma como característica principal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -144423,7 +144416,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=523000</ExpertLink>
       <Name lang="es">Glaucoma de inicio pediátrico</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -144681,7 +144674,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=88632</ExpertLink>
       <Name lang="es">Anomalía del desarrollo del segmento anterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -149633,7 +149626,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90970</ExpertLink>
       <Name lang="es">Lipodistrofias primarias</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -150037,7 +150030,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79373</ExpertLink>
       <Name lang="es">Síndrome de displasia ectodérmica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -152824,7 +152817,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79361</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa hereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -155383,7 +155376,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=86851</ExpertLink>
       <Name lang="es">Leucemia aguda de línea ambigua</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -155429,7 +155422,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=86846</ExpertLink>
       <Name lang="es">Síndrome mielodisplásico y leucemia mieloide aguda asociados al tratamiento</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -158456,7 +158449,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98292</ExpertLink>
       <Name lang="es">Mastocitosis</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -158476,7 +158469,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98291</ExpertLink>
       <Name lang="es">Enfermedad linfoproliferativa asociada con la inmunodeficiencia primaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -158562,7 +158555,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97242</ExpertLink>
       <Name lang="es">Distrofia muscular congénita</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -158614,7 +158607,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97253</ExpertLink>
       <Name lang="es">Tumor neuroendocrino del páncreas</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -160099,7 +160092,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97593</ExpertLink>
       <Name lang="es">Pseudohipoparatiroidismo</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -160279,7 +160272,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=95708</ExpertLink>
       <Name lang="es">Pubertad precoz rara</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -166052,7 +166045,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99739</ExpertLink>
       <Name lang="es">Trastorno familiar raro con miocardiopatía hipertrófica</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -169747,7 +169740,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98555</ExpertLink>
       <Name lang="es">Microftalmia-anoftalmia-coloboma</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -169869,7 +169862,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98627</ExpertLink>
       <Name lang="es">Distrofia corneal posterior</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -169898,7 +169891,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98626</ExpertLink>
       <Name lang="es">Distrofia corneal estromal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -169924,7 +169917,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98625</ExpertLink>
       <Name lang="es">Distrofia corneal superficial</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -171181,7 +171174,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98428</ExpertLink>
       <Name lang="es">Policitemia secundaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -171883,7 +171876,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=102006</ExpertLink>
       <Name lang="es">Malformación neurovascular</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -171958,7 +171951,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=101959</ExpertLink>
       <Name lang="es">Insuficiencia suprarrenal crónica primaria</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -173519,7 +173512,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100088</ExpertLink>
       <Name lang="es">Carcinoma de tiroides</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
@@ -173934,7 +173927,7 @@
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=100077</ExpertLink>
       <Name lang="es">Tumor neuroendocrino yeyunal</Name>
       <DisorderType id="36561">
-        <Name lang="es">Category</Name>
+        <Name lang="es">Categoría</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="es">Grupo de trastornos</Name>
