commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml
old mode 100644
new mode 100755
index b4d31e0..9e30abe
--- a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:42:03" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:43:04" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -1108,7 +1108,7 @@
     <Disorder id="17626">
       <OrphaCode>166272</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166272</ExpertLink>
-      <Name lang="es">Síndrome de Goldblatt</Name>
+      <Name lang="es">Odontocondrodisplasia</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -50750,7 +50750,7 @@
     <Disorder id="19731">
       <OrphaCode>252202</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=252202</ExpertLink>
-      <Name lang="es">Síndrome de deficiencia constitucional de la reparación de errores de apareamiento</Name>
+      <Name lang="es">Síndrome constitutivo de deficiencia de reparación de emparejamientos erróneos</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -53238,11 +53238,11 @@
       <OrphaCode>261144</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=261144</ExpertLink>
       <Name lang="es">Síndrome de microdeleción 14q12</Name>
-      <DisorderType id="21401">
-        <Name lang="es">Síndrome de malformación</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -62734,11 +62734,11 @@
       <OrphaCode>280315</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280315</ExpertLink>
       <Name lang="es">Pancreatitis autoinmune tipo 2</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -67982,7 +67982,7 @@
     <Disorder id="3155">
       <OrphaCode>2538</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2538</ExpertLink>
-      <Name lang="es">Síndrome de microgastria-anomalía de las extremidades</Name>
+      <Name lang="es">Síndrome de microgastria-anomalía de reducción de las extremidades</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -73043,7 +73043,7 @@
     <Disorder id="18827">
       <OrphaCode>217253</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=217253</ExpertLink>
-      <Name lang="es">Encefalitis límbica con anticuerpos anti-receptor NMDA</Name>
+      <Name lang="es">Encefalitis límbica por anticuerpos anti-receptor NMDA</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -103395,7 +103395,7 @@
     <Disorder id="23707">
       <OrphaCode>457205</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=457205</ExpertLink>
-      <Name lang="es">Síndrome neurodegenerativo de neuropatía axonal sensitivo-motora-atrofia óptica de inicio en la lactancia</Name>
+      <Name lang="es">Síndrome neurodegenerativo de neuropatía axonal sensitivo-motora-atrofia óptica de inicio en la infancia</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -123490,7 +123490,7 @@
     <Disorder id="25247">
       <OrphaCode>482077</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=482077</ExpertLink>
-      <Name lang="es">Enfermedad cerebral de pequeños vasos asociada al gen HTRA1 autosómica dominante</Name>
+      <Name lang="es">Enfermedad cerebral de pequeño vaso autosómica dominante asociada al gen HTRA1</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -124731,7 +124731,7 @@
     <Disorder id="25407">
       <OrphaCode>488613</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=488613</ExpertLink>
-      <Name lang="es">Síndrome de retraso global del desarrollo-anomalías neuro-oftalmológicas-convulsiones-discapacidad intelectual</Name>
+      <Name lang="es">Síndrome de retraso global del desarrollo-anomalías neuro-oftalmológicas-crisis-discapacidad intelectual</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -140836,7 +140836,7 @@
     <Disorder id="28070">
       <OrphaCode>544254</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=544254</ExpertLink>
-      <Name lang="es">Encefalopatía del desarrollo y epiléptica asociada al gen SYNGAP1</Name>
+      <Name lang="es">Encefalopatía epiléptica y del desarrollo asociada al gen SYNGAP1</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -144584,7 +144584,7 @@
     <Disorder id="27322">
       <OrphaCode>521305</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=521305</ExpertLink>
-      <Name lang="es">Miopatía proximal con depleción focal de la mitocondria</Name>
+      <Name lang="es">Miopatía proximal con depleción mitocondrial focal</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -163546,11 +163546,11 @@
       <OrphaCode>99361</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99361</ExpertLink>
       <Name lang="es">Carcinoma medular de tiroides familiar</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
@@ -164164,11 +164164,11 @@
       <OrphaCode>98813</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98813</ExpertLink>
       <Name lang="es">Displasia ectodérmica hipohidrótica con inmunodeficiencia</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -167343,8 +167343,8 @@
       <OrphaCode>98791</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98791</ExpertLink>
       <Name lang="es">Síndrome de alfa talasemia-discapacidad intelectual ligado al cromosoma 16</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21401">
+        <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
@@ -168831,11 +168831,11 @@
       <OrphaCode>103919</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=103919</ExpertLink>
       <Name lang="es">Pancreatitis autoinmune</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21436">
+        <Name lang="es">Grupo de fenomas</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
