commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml
old mode 100644
new mode 100755
index 7e15727..b20d963
--- a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:40:37" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:41:15" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6584">
+  <DisorderList count="6580">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -845,7 +845,7 @@
     <Disorder id="17619">
       <OrphaCode>166105</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166105</ExpertLink>
-      <Name lang="es">Encefalomiopatía mitocondrial infantil asociada al gen FASTKD2</Name>
+      <Name lang="es">Encefalomiopatía mitocondrial del lactante asociada al gen FASTKD2</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -1391,7 +1391,7 @@
     <Disorder id="17634">
       <OrphaCode>166305</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166305</ExpertLink>
-      <Name lang="es">Crisis infantiles benignas asociadas con gastroenteritis leve</Name>
+      <Name lang="es">Crisis benignas infantiles asociadas con gastroenteritis leve</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -7878,7 +7878,7 @@
     <Disorder id="17792">
       <OrphaCode>168782</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=168782</ExpertLink>
-      <Name lang="es">Trastorno desintegrativo de la infancia</Name>
+      <Name lang="es">Trastorno desintegrativo infantil</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -13651,7 +13651,7 @@
     <Disorder id="589">
       <OrphaCode>3173</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3173</ExpertLink>
-      <Name lang="es">Síndrome de espasmos infantiles-pulgar ancho</Name>
+      <Name lang="es">Síndrome de espasmos-pulgar ancho del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -19343,7 +19343,7 @@
     <Disorder id="776">
       <OrphaCode>17</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=17</ExpertLink>
-      <Name lang="es">Acidosis láctica infantil letal con aciduria metilmalónica</Name>
+      <Name lang="es">Acidosis láctica letal del lactante con aciduria metilmalónica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -19873,7 +19873,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Defecto de la comunicación auriculoventricular completo</Name>
+      <Name lang="es">Canal aurículoventricular completo</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -21534,10 +21534,16 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="4">
         <AverageAgeOfOnset id="23508">
           <Name lang="es">Prenatal</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="es">Neonatal</Name>
         </AverageAgeOfOnset>
@@ -26595,32 +26601,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="16891">
-      <OrphaCode>139420</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=139420</ExpertLink>
-      <Name lang="es">Mielitis transversa aguda secundaria</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="es">Cualquier edad</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="es">Cualquier edad</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="es">No aplicable</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="16892">
       <OrphaCode>139423</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=139423</ExpertLink>
@@ -28922,7 +28902,7 @@
     <Disorder id="1414">
       <OrphaCode>1145</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1145</ExpertLink>
-      <Name lang="es">Atrofia muscular espinal de inicio precoz ligada al cromosoma X</Name>
+      <Name lang="es">Atrofia muscular espinal de inicio en el lactante ligada al cromosoma X</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -29487,7 +29467,7 @@
     <Disorder id="1443">
       <OrphaCode>1186</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1186</ExpertLink>
-      <Name lang="es">Ataxia espinocerebelosa de inicio en la lactancia</Name>
+      <Name lang="es">Ataxia espinocerebelosa de inicio en el lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -32141,7 +32121,7 @@
     <Disorder id="17140">
       <OrphaCode>157716</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=157716</ExpertLink>
-      <Name lang="es">Síndrome CACH de inicio en la infancia tardía</Name>
+      <Name lang="es">Síndrome CACH infantil tardío</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -32242,7 +32222,7 @@
     <Disorder id="17139">
       <OrphaCode>157713</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=157713</ExpertLink>
-      <Name lang="es">Síndrome CACH congénito o de inicio en la primera infancia</Name>
+      <Name lang="es">Síndrome CACH congénito o infantil temprano</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -34935,7 +34915,7 @@
     <Disorder id="17000">
       <OrphaCode>140927</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=140927</ExpertLink>
-      <Name lang="es">Convulsiones neonatales-infantiles benignas familiares</Name>
+      <Name lang="es">Convulsiones benignas del neonato-lactante familiares</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -40478,11 +40458,11 @@
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158676</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa dominante con distrofia ungueal solo</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -40507,11 +40487,11 @@
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158673</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa distrófica acral</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -47803,11 +47783,11 @@
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251393</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa juntural tipo no Herlitz localizada</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -48287,7 +48267,7 @@
     <Disorder id="19624">
       <OrphaCode>251304</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251304</ExpertLink>
-      <Name lang="es">Paniculitis de inicio en la lactancia con uveitis y granulomatosis sistémica</Name>
+      <Name lang="es">Paniculitis de inicio en el lactante con uveitis y granulomatosis sistémica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -48420,7 +48400,7 @@
     <Disorder id="19790">
       <OrphaCode>254857</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=254857</ExpertLink>
-      <Name lang="es">Miopatía mitocondrial letal infantil</Name>
+      <Name lang="es">Miopatía mitocondrial letal del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -63559,7 +63539,7 @@
     <Disorder id="20472">
       <OrphaCode>280553</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=280553</ExpertLink>
-      <Name lang="es">Miopatía miofibrilar hipertónica infantil letal</Name>
+      <Name lang="es">Miopatía miofibrilar hipertónica letal del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -70051,7 +70031,7 @@
     <Disorder id="3525">
       <OrphaCode>3286</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3286</ExpertLink>
-      <Name lang="es">Taquicardia ventricular polimórfica catecolaminérgica</Name>
+      <Name lang="es">Taquicardia ventricular catecolaminérgica polimórfica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -76031,7 +76011,7 @@
     <Disorder id="3379">
       <OrphaCode>1561</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1561</ExpertLink>
-      <Name lang="es">Deficiencia infantil fatal de citocromo C oxidasa</Name>
+      <Name lang="es">Deficiencia de citocromo C oxidasa fatal del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -76343,7 +76323,7 @@
     <Disorder id="18977">
       <OrphaCode>225147</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=225147</ExpertLink>
-      <Name lang="es">Necrosis estriatal bilateral infantil esporádica</Name>
+      <Name lang="es">Necrosis estriatal bilateral esporádica infantil</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -78705,7 +78685,7 @@
     <Disorder id="19100">
       <OrphaCode>228305</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=228305</ExpertLink>
-      <Name lang="es">Deficiencia de carnitina palmitoiltransferasa II, forma infantil grave</Name>
+      <Name lang="es">Deficiencia de carnitina palmitoiltransferasa II, forma grave infantil</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -79225,7 +79205,7 @@
     <Disorder id="3651">
       <OrphaCode>306</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=306</ExpertLink>
-      <Name lang="es">Epilepsia infantil familiar benigna</Name>
+      <Name lang="es">Epilepsia benigna del lactante familiar</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -88218,7 +88198,7 @@
     <Disorder id="22479">
       <OrphaCode>370088</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=370088</ExpertLink>
-      <Name lang="es">Síndrome de insuficiencia hepática aguda infantil-afectación multisistémica</Name>
+      <Name lang="es">Síndrome de insuficiencia hepática aguda-afectación multisistémica del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -93232,7 +93212,7 @@
     <Disorder id="20766">
       <OrphaCode>289857</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289857</ExpertLink>
-      <Name lang="es">Encefalopatía neonatal por glicina</Name>
+      <Name lang="es">Encefalopatía por glicina neonatal</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -93258,7 +93238,7 @@
     <Disorder id="20767">
       <OrphaCode>289860</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=289860</ExpertLink>
-      <Name lang="es">Encefalopatía infantil por glicina</Name>
+      <Name lang="es">Encefalopatía por glicina infantil</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -93724,7 +93704,7 @@
     <Disorder id="20839">
       <OrphaCode>293168</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293168</ExpertLink>
-      <Name lang="es">Parálisis espástica hereditaria ascendente de inicio en la lactancia</Name>
+      <Name lang="es">Parálisis espástica ascendente hereditaria de inicio en el lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -97000,7 +96980,7 @@
     <Disorder id="21458">
       <OrphaCode>313850</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=313850</ExpertLink>
-      <Name lang="es">Degeneración cerebelosa-retiniana infantil</Name>
+      <Name lang="es">Degeneración cerebelosa-retiniana del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -103127,7 +103107,7 @@
     <Disorder id="23724">
       <OrphaCode>457375</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=457375</ExpertLink>
-      <Name lang="es">Trastorno neurológico infantil letal con cataratas y afectación cardíaca asociado al gen ITPA</Name>
+      <Name lang="es">Trastorno neurológico con cataratas y afectación cardíaca letal infantil asociado al gen ITPA</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -104422,7 +104402,7 @@
     <Disorder id="24170">
       <OrphaCode>466794</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=466794</ExpertLink>
-      <Name lang="es">Síndrome de insuficiencia hepática infantil-ataxia cerebelosa-neuropatía periférica sensitivo-motora</Name>
+      <Name lang="es">Síndrome de insuficiencia hepática-ataxia cerebelosa-neuropatía periférica sensitivo-motora infantil</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -108119,7 +108099,7 @@
     <Disorder id="22603">
       <OrphaCode>391316</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391316</ExpertLink>
-      <Name lang="es">Epilepsia del lóbulo temporal mesial de inicio en la lactancia con regresión cognitiva grave</Name>
+      <Name lang="es">Epilepsia del lóbulo temporal mesial de inicio en el lactante con regresión cognitiva grave</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -110363,7 +110343,7 @@
     <Disorder id="23330">
       <OrphaCode>436166</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436166</ExpertLink>
-      <Name lang="es">Síndrome autoinflamatorio-fiebre periódica-enterocolitis infantil</Name>
+      <Name lang="es">Síndrome autoinflamatorio-fiebre periódica-enterocolitis del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -115382,7 +115362,7 @@
     <Disorder id="23209">
       <OrphaCode>425120</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=425120</ExpertLink>
-      <Name lang="es">Vasculopatía asociada al gen STING con inicio en la infancia</Name>
+      <Name lang="es">Vasculopatía con inicio en el lactante asociada al gen STING</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -119232,7 +119212,7 @@
     <Disorder id="25994">
       <OrphaCode>500062</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=500062</ExpertLink>
-      <Name lang="es">Síndrome de fiebre periódica-paniculitis-dermatosis de inicio en la lactancia</Name>
+      <Name lang="es">Síndrome de fiebre periódica-paniculitis-dermatosis de inicio en el lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -121790,7 +121770,7 @@
     <Disorder id="8597">
       <OrphaCode>1330</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1330</ExpertLink>
-      <Name lang="es">Defecto de la comunicación auriculoventricular parcial</Name>
+      <Name lang="es">Canal aurículoventricular parcial</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -123446,7 +123426,7 @@
     <Disorder id="9793">
       <OrphaCode>31709</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=31709</ExpertLink>
-      <Name lang="es">Convulsiones infantiles y coreoatetosis</Name>
+      <Name lang="es">Convulsiones del lactante y coreoatetosis</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -124190,7 +124170,7 @@
     <Disorder id="25136">
       <OrphaCode>477661</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477661</ExpertLink>
-      <Name lang="es">Enfermedad inflamatoria intestinal infantil asociada al gen IL21</Name>
+      <Name lang="es">Enfermedad inflamatoria intestinal del lactante asociada al gen IL21</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -124245,7 +124225,7 @@
     <Disorder id="25138">
       <OrphaCode>477673</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477673</ExpertLink>
-      <Name lang="es">Síndrome postnatal de microcefalia-hipotonía infantil-diplejía espástica-disartria-discapacidad intelectual</Name>
+      <Name lang="es">Síndrome de microcefalia postnatal-hipotonía infantil-diplejía espástica-disartria-discapacidad intelectual</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -126141,7 +126121,7 @@
     <Disorder id="10958">
       <OrphaCode>70587</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=70587</ExpertLink>
-      <Name lang="es">Síndrome de  distrés respiratorio agudo infantil</Name>
+      <Name lang="es">Síndrome de dificultad respiratoria aguda del lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -130493,7 +130473,7 @@
     <Disorder id="28448">
       <OrphaCode>565788</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=565788</ExpertLink>
-      <Name lang="es">Enfermedad inflamatoria intestinal infantil con afectación neurológica</Name>
+      <Name lang="es">Enfermedad inflamatoria intestinal del lactante con afectación neurológica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -134803,7 +134783,7 @@
     <Disorder id="11032">
       <OrphaCode>73014</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73014</ExpertLink>
-      <Name lang="es">Diarrea intratable del lactante</Name>
+      <Name lang="es">Diarrea intratable infantil</Name>
       <DisorderType id="36561">
         <Name lang="es">Category</Name>
       </DisorderType>
@@ -142799,90 +142779,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="11906">
-      <OrphaCode>89841</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=89841</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa distrófica centrípeta recesiva</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="es">Infancia</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="es">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="es">Autosómico recesivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="11905">
-      <OrphaCode>89840</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=89840</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa juntural tipo no Herlitz</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="es">Infancia</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="es">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="2">
-        <AverageAgeOfDeath id="23613">
-          <Name lang="es">Adolescencia</Name>
-        </AverageAgeOfDeath>
-        <AverageAgeOfDeath id="23606">
-          <Name lang="es">Infancia tardía</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="es">Autosómico recesivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="11904">
-      <OrphaCode>89839</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=89839</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa simple superficial</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="es">Infancia</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="es">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="es">Autosómico dominate</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="11919">
       <OrphaCode>90001</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90001</ExpertLink>
@@ -149363,11 +149259,11 @@
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79402</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa juntural generalizada intermedia</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -149557,11 +149453,11 @@
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79410</ExpertLink>
       <Name lang="es">Epidermólisis ampollosa distrófica pretibial</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -157047,7 +156943,7 @@
     <Disorder id="11644">
       <OrphaCode>85179</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=85179</ExpertLink>
-      <Name lang="es">Osteopetrosis con displasia neuroaxonal, forma infantil</Name>
+      <Name lang="es">Osteopetrosis del lactante con displasia neuroaxonal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -168772,7 +168668,7 @@
     <Disorder id="28746">
       <OrphaCode>572428</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=572428</ExpertLink>
-      <Name lang="es">Proteinosis alveolar pulmonar-hipogammaglobulinemia de inicio en la infancia</Name>
+      <Name lang="es">Proteinosis alveolar pulmonar-hipogammaglobulinemia de inicio en el lactante</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
