commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml
old mode 100755
new mode 100644
index 9e30abe..3f8e492
--- a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:43:04" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:45:34" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6566">
+  <DisorderList count="6568">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -3716,12 +3716,12 @@
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23424">
+          <Name lang="es">Multigénico / multifactorial</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23494">
           <Name lang="es">No aplicable</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23480">
-          <Name lang="es">Desconocido</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="17490">
@@ -14854,13 +14854,16 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="3">
         <TypeOfInheritance id="23410">
           <Name lang="es">Autosómico dominate</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="es">Autosómico recesivo</Name>
         </TypeOfInheritance>
+        <TypeOfInheritance id="23431">
+          <Name lang="es">Recesivo ligado al X</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="17999">
@@ -67435,13 +67438,16 @@
           <Name lang="es">Expectativa de vida normal</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="3">
+      <TypeOfInheritanceList count="4">
         <TypeOfInheritance id="23410">
           <Name lang="es">Autosómico dominate</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="es">Autosómico recesivo</Name>
         </TypeOfInheritance>
+        <TypeOfInheritance id="23424">
+          <Name lang="es">Multigénico / multifactorial</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23431">
           <Name lang="es">Recesivo ligado al X</Name>
         </TypeOfInheritance>
@@ -68629,13 +68635,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23410">
           <Name lang="es">Autosómico dominate</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="es">Autosómico recesivo</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="18604">
@@ -74621,9 +74624,12 @@
           <Name lang="es">Expectativa de vida normal</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="es">No aplicable</Name>
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23424">
+          <Name lang="es">Multigénico / multifactorial</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23480">
+          <Name lang="es">Desconocido</Name>
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
@@ -77963,7 +77969,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="es">Autosómico dominate</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23424">
           <Name lang="es">Multigénico / multifactorial</Name>
         </TypeOfInheritance>
@@ -79975,6 +79984,26 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="3625">
+      <OrphaCode>2197</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2197</ExpertLink>
+      <Name lang="es">Hipercalciuria idiopática</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="es">Autosómico dominate</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="19175">
       <OrphaCode>231237</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231237</ExpertLink>
@@ -82463,6 +82492,38 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="19292">
+      <OrphaCode>238666</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=238666</ExpertLink>
+      <Name lang="es">Hipogonadismo hipogonadotrópico congénito aislado</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="5">
+        <TypeOfInheritance id="23410">
+          <Name lang="es">Autosómico dominate</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23417">
+          <Name lang="es">Autosómico recesivo</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23459">
+          <Name lang="es">Oligogénico</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23480">
+          <Name lang="es">Desconocido</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23431">
+          <Name lang="es">Recesivo ligado al X</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="19291">
       <OrphaCode>238654</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=238654</ExpertLink>
@@ -86815,13 +86876,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23410">
           <Name lang="es">Autosómico dominate</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23431">
-          <Name lang="es">Recesivo ligado al X</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="21581">
@@ -91393,7 +91451,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="es">Autosómico dominate</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23494">
           <Name lang="es">No aplicable</Name>
         </TypeOfInheritance>
@@ -93730,13 +93791,16 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="3">
         <TypeOfInheritance id="23410">
           <Name lang="es">Autosómico dominate</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="es">Autosómico recesivo</Name>
         </TypeOfInheritance>
+        <TypeOfInheritance id="23431">
+          <Name lang="es">Recesivo ligado al X</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="5544">
@@ -118684,7 +118748,10 @@
           <Name lang="es">No se dispone de datos</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="es">Autosómico dominate</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="26001">
@@ -122161,8 +122228,8 @@
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="es">No aplicable</Name>
+        <TypeOfInheritance id="23480">
+          <Name lang="es">Desconocido</Name>
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
@@ -124390,7 +124457,10 @@
           <Name lang="es">No se dispone de datos</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23417">
+          <Name lang="es">Autosómico recesivo</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="25413">
@@ -135947,8 +136017,8 @@
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23424">
-          <Name lang="es">Multigénico / multifactorial</Name>
+        <TypeOfInheritance id="23417">
+          <Name lang="es">Autosómico recesivo</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23494">
           <Name lang="es">No aplicable</Name>
@@ -152465,10 +152535,7 @@
           <Name lang="es">Infantil</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23438">
-          <Name lang="es">Herencia mitocondrial</Name>
-        </TypeOfInheritance>
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23431">
           <Name lang="es">Recesivo ligado al X</Name>
         </TypeOfInheritance>
@@ -157560,7 +157627,7 @@
     <Disorder id="12885">
       <OrphaCode>97292</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=97292</ExpertLink>
-      <Name lang="es">Choque cardiogénico</Name>
+      <Name lang="es">Cardiogenic shock</Name>
       <DisorderType id="21429">
         <Name lang="es">Situación clínica particular en una enfermedad o síndrome</Name>
       </DisorderType>
