commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml
old mode 100644
new mode 100755
index 1ccfd88..7e15727
--- a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:39:39" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:40:37" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6586">
+  <DisorderList count="6584">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -7244,7 +7244,7 @@
     <Disorder id="343">
       <OrphaCode>1727</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1727</ExpertLink>
-      <Name lang="es">Síndrome de microduplicación 22q11.2</Name>
+      <Name lang="es">Síndrome de duplicación 22q11.2</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -18520,7 +18520,7 @@
     <Disorder id="784">
       <OrphaCode>3002</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3002</ExpertLink>
-      <Name lang="es">Púrpura trombocitopénica inmunitaria</Name>
+      <Name lang="es">Trombocitopenia inmune</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -19873,7 +19873,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Canal auriculoventricular completo</Name>
+      <Name lang="es">Defecto de la comunicación auriculoventricular completo</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -20134,7 +20134,7 @@
     <Disorder id="849">
       <OrphaCode>3198</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3198</ExpertLink>
-      <Name lang="es">Síndrome de la persona rígida y síndromes relacionados</Name>
+      <Name lang="es">Trastorno del espectro de la persona rígida</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -23556,7 +23556,7 @@
     <Disorder id="1228">
       <OrphaCode>3306</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3306</ExpertLink>
-      <Name lang="es">Duplicación/inversión 15q11</Name>
+      <Name lang="es">Síndrome de inversión duplicación del cromosoma 15</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -26090,7 +26090,7 @@
     <Disorder id="1299">
       <OrphaCode>981</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=981</ExpertLink>
-      <Name lang="es">Agenesia de la carótida interna</Name>
+      <Name lang="es">Ausencia de la carótida interna</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -31956,7 +31956,7 @@
     <Disorder id="17146">
       <OrphaCode>157798</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=157798</ExpertLink>
-      <Name lang="es">Síndrome de poliposis hiperplásica</Name>
+      <Name lang="es">Síndrome de poliposis serrada</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -36903,7 +36903,7 @@
     <Disorder id="1868">
       <OrphaCode>1906</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1906</ExpertLink>
-      <Name lang="es">Síndrome fetal por valproato</Name>
+      <Name lang="es">Trastorno del espectro valproato fetal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -40914,7 +40914,7 @@
     <Disorder id="2200">
       <OrphaCode>2348</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2348</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial tipo Dunnigan</Name>
+      <Name lang="es">Lipodistrofia parcial familiar tipo Dunnigan</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -42152,7 +42152,7 @@
     <Disorder id="2180">
       <OrphaCode>2325</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2325</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa simple con anodoncia/hipodoncia</Name>
+      <Name lang="es">Epidermólisis ampollosa simple con sordera, alopecia, anodoncia/hipodoncia y distrofia ungueal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -46102,7 +46102,7 @@
     <Disorder id="19605">
       <OrphaCode>251028</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=251028</ExpertLink>
-      <Name lang="es">Síndrome de microdeleción 2q33.1</Name>
+      <Name lang="es">Síndrome asociado al gen SATB2 por un reordenamiento cromosómico</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -54188,7 +54188,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="es">Autosómico dominate</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="19874">
@@ -56328,7 +56331,7 @@
     <Disorder id="2814">
       <OrphaCode>3145</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3145</ExpertLink>
-      <Name lang="es">Síndrome de Schofer-Beetz-Bohl</Name>
+      <Name lang="es">Síndrome de diabetes insípida nefrogénica-calcificación intracraneal-dismorfia facial</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -64661,7 +64664,7 @@
     <Disorder id="20389">
       <OrphaCode>276608</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=276608</ExpertLink>
-      <Name lang="es">Hipoglucemia hiperinsulinémica persistente sin insulinoma de inicio en el adulto</Name>
+      <Name lang="es">Encefalomielitis progresiva con rigidez y mioclonías</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -71241,7 +71244,7 @@
     <Disorder id="3563">
       <OrphaCode>359</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=359</ExpertLink>
-      <Name lang="es">Glaucoma hereditario</Name>
+      <Name lang="es">Glaucoma de inicio pediátrico de origen genético</Name>
       <DisorderType id="36561">
         <Name lang="es">Category</Name>
       </DisorderType>
@@ -77687,7 +77690,7 @@
     <Disorder id="3727">
       <OrphaCode>3337</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3337</ExpertLink>
-      <Name lang="es">Síndrome de Fanconi primario</Name>
+      <Name lang="es">Síndrome de Fanconi primario renotubular</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -92089,8 +92092,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=353284</ExpertLink>
       <Name lang="es">Síndrome de Rubinstein-Taybi por haploinsuficiencia EP300</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21443">
+        <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="es">Subtipo de trastorno</Name>
@@ -92819,32 +92822,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293848</ExpertLink>
-      <Name lang="es">Demencia frontotemporal, variante con atrofia temporal derecha</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="es">La edad adulta</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23634">
-          <Name lang="es">Senectud</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23480">
-          <Name lang="es">Desconocido</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293843</ExpertLink>
@@ -94566,7 +94543,7 @@
     <Disorder id="20609">
       <OrphaCode>284417</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=284417</ExpertLink>
-      <Name lang="es">Deficiencia de fosfoserina aminotransferasa</Name>
+      <Name lang="es">Deficiencia de fosfoserina aminotransferasa, forma infantil/juvenil</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -96595,26 +96572,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20606">
-      <OrphaCode>284408</OrphaCode>
-      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=284408</ExpertLink>
-      <Name lang="es">Deficiencia de glicerol quinasa infantil</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="es">Infancia</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20607">
       <OrphaCode>284411</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=284411</ExpertLink>
@@ -107760,7 +107717,7 @@
     <Disorder id="22609">
       <OrphaCode>391348</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391348</ExpertLink>
-      <Name lang="es">Síndrome de retraso en el crecimiento y el desarrollo-hipotonía-trastorno ocular-acidosis láctica</Name>
+      <Name lang="es">Síndrome de retraso del crecimiento y el desarrollo-hipotonía-trastorno ocular-acidosis láctica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -107809,7 +107766,7 @@
     <Disorder id="22611">
       <OrphaCode>391366</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=391366</ExpertLink>
-      <Name lang="es">Síndrome de retraso en el crecimiento-retraso del desarrollo leve-hepatitis crónica</Name>
+      <Name lang="es">Síndrome de retraso del crecimiento-retraso del desarrollo leve-hepatitis crónica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -109902,7 +109859,7 @@
     <Disorder id="23310">
       <OrphaCode>435660</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=435660</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen LIPE</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen LIPE</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -109928,7 +109885,7 @@
     <Disorder id="23309">
       <OrphaCode>435651</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=435651</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen CIDEC</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen CIDEC</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -120393,7 +120350,7 @@
     <Disorder id="8668">
       <OrphaCode>432</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=432</ExpertLink>
-      <Name lang="es">Hipogonadismo hipogonadotropo normosómico congénito</Name>
+      <Name lang="es">Hipogonadismo hipogonadotrópico normosómico congénito</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -121833,7 +121790,7 @@
     <Disorder id="8597">
       <OrphaCode>1330</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1330</ExpertLink>
-      <Name lang="es">Canal auriculoventricular parcial</Name>
+      <Name lang="es">Defecto de la comunicación auriculoventricular parcial</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -127648,7 +127605,7 @@
     <Disorder id="10882">
       <OrphaCode>66529</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=66529</ExpertLink>
-      <Name lang="es">Miocardiopatía tipo Tako-Tsubo</Name>
+      <Name lang="es">Miocardiopatía de Tako-Tsubo</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -132785,7 +132742,7 @@
     <Disorder id="11129">
       <OrphaCode>79085</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79085</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen AKT2</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen AKT2</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -132808,7 +132765,7 @@
     <Disorder id="11128">
       <OrphaCode>79084</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79084</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial tipo Köbberling</Name>
+      <Name lang="es">Lipodistrofia parcial familiar tipo Köbberling</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -134000,7 +133957,7 @@
     <Disorder id="11048">
       <OrphaCode>73272</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73272</ExpertLink>
-      <Name lang="es">Retraso en el crecimiento por deficiencia del factor de crecimiento similar a la insulina tipo 1</Name>
+      <Name lang="es">Retraso del crecimiento por deficiencia del factor de crecimiento similar a la insulina tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -134029,7 +133986,7 @@
     <Disorder id="11049">
       <OrphaCode>73273</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=73273</ExpertLink>
-      <Name lang="es">Retraso en el crecimiento por resistencia al factor de crecimiento similar a la insulina tipo 1</Name>
+      <Name lang="es">Retraso del crecimiento por resistencia al factor de crecimiento similar a la insulina tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -139105,7 +139062,7 @@
     <Disorder id="10738">
       <OrphaCode>53696</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=53696</ExpertLink>
-      <Name lang="es">Síndrome de artrogriposis letal-enfermedad de las astas medulares anteriores</Name>
+      <Name lang="es">Síndrome de artrogriposis-enfermedad de las células del cuerno anterior</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -139571,7 +139528,7 @@
     <Disorder id="10631">
       <OrphaCode>48686</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=48686</ExpertLink>
-      <Name lang="es">Linfoma primario de cavidades</Name>
+      <Name lang="es">Linfoma primario de efusiones</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -147550,7 +147507,7 @@
     <Disorder id="12081">
       <OrphaCode>90695</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90695</ExpertLink>
-      <Name lang="es">Panhipopituitarismo</Name>
+      <Name lang="es">Panhipopituitarismo no adquirido</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -152065,7 +152022,7 @@
     <Disorder id="11376">
       <OrphaCode>79350</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79350</ExpertLink>
-      <Name lang="es">Deficiencia de 3-fosfoserina fosfatasa</Name>
+      <Name lang="es">Deficiencia de 3-fosfoserina fosfatasa, forma infantil/juvenil</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -156607,7 +156564,7 @@
     <Disorder id="11600">
       <OrphaCode>83629</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=83629</ExpertLink>
-      <Name lang="es">Síndrome de leucoencefalopatía-displasia espondilometafisaria</Name>
+      <Name lang="es">Leukoencephalopathy-spondyloepimetaphyseal dysplasia syndrome</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -156937,7 +156894,7 @@
     <Disorder id="11618">
       <OrphaCode>84132</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=84132</ExpertLink>
-      <Name lang="es">Miopatía asociada a la desmina con inclusiones tipo cuerpos de Mallory</Name>
+      <Name lang="es">Miopatía asociada a desmina con inclusiones similares a cuerpos de Mallory</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -161567,7 +161524,7 @@
     <Disorder id="12396">
       <OrphaCode>93554</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93554</ExpertLink>
-      <Name lang="es">Crioglobulinemia mixta tipo 2</Name>
+      <Name lang="es">Crioglobulinemia mixta tipo II</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -163416,7 +163373,7 @@
     <Disorder id="12423">
       <OrphaCode>93585</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93585</ExpertLink>
-      <Name lang="es">Púrpura trombótica trombocitopénica adquirida</Name>
+      <Name lang="es">Immune-mediated thrombotic thrombocytopenic purpura</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -168315,7 +168272,7 @@
     <Disorder id="28932">
       <OrphaCode>580940</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=580940</ExpertLink>
-      <Name lang="es">QRICH1-related intellectual disability-chondrodysplasia syndrome</Name>
+      <Name lang="es">Síndrome de discapacidad intelectual-condrodisplasia asociado al gen QRICH1</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -168341,7 +168298,7 @@
     <Disorder id="28934">
       <OrphaCode>580951</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=580951</ExpertLink>
-      <Name lang="es">Punctate inner choroidopathy</Name>
+      <Name lang="es">Coroidopatía punteada interna</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -168361,7 +168318,7 @@
     <Disorder id="28931">
       <OrphaCode>580933</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=580933</ExpertLink>
-      <Name lang="es">Trastornos letales del desarrollo cerebral y cardiaco</Name>
+      <Name lang="es">Defectos del desarrollo cerebral y cardíaco letales</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -168491,7 +168448,7 @@
     <Disorder id="28936">
       <OrphaCode>581271</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=581271</ExpertLink>
-      <Name lang="es">Cramp-fasciculation syndrome</Name>
+      <Name lang="es">Síndrome de calambres-fasciculaciones</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -168537,7 +168494,7 @@
     <Disorder id="13323">
       <OrphaCode>98306</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=98306</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial</Name>
+      <Name lang="es">Lipodistrofia parcial familiar</Name>
       <DisorderType id="21436">
         <Name lang="es">Grupo de fenomas</Name>
       </DisorderType>
