commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml
index bebd551..c5e845b 100644
--- a/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/es_product9_ages.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:51:14" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6633">
+<JDBOR date="2021-12-01 04:55:42" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=166024</ExpertLink>
@@ -3958,7 +3951,7 @@
     <Disorder id="17515">
       <OrphaCode>163654</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=163654</ExpertLink>
-      <Name lang="es">Displasia espondiloepifisaria tipo Cantu</Name>
+      <Name lang="es">Síndrome de displasia espondiloepifisaria-braquidactilia-trastorno del habla</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -7022,7 +7015,7 @@
     <Disorder id="315">
       <OrphaCode>1349</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1349</ExpertLink>
-      <Name lang="es">Miocardiopatía y pérdida auditiva asociada al ADN mitocondrial</Name>
+      <Name lang="es">Miocardiopatía e hipoacusia asociada al ADN mitocondrial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -7138,7 +7131,7 @@
     <Disorder id="294">
       <OrphaCode>308</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=308</ExpertLink>
-      <Name lang="es">Enfermedad de Unverricht-Lundborg</Name>
+      <Name lang="es">Epilepsia mioclónica progresiva tipo 1</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -12687,11 +12680,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2346</ExpertLink>
       <Name lang="es">Síndrome angio-osteo-hipertrófico</Name>
-      <DisorderType id="21394">
-        <Name lang="es">Enfermedad</Name>
+      <DisorderType id="21436">
+        <Name lang="es">Grupo de fenomas</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="es">Trastorno</Name>
+      <DisorderGroup id="36540">
+        <Name lang="es">Grupo de trastornos</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="3">
         <AverageAgeOfOnset id="23536">
@@ -15897,7 +15890,7 @@
     <Disorder id="745">
       <OrphaCode>183</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183</ExpertLink>
-      <Name lang="es">Granulomatosis eosinofílica con poliangitis</Name>
+      <Name lang="es">Granulomatosis eosinofílica con poliangeítis</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -16462,7 +16455,7 @@
     <Disorder id="753">
       <OrphaCode>727</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=727</ExpertLink>
-      <Name lang="es">Poliangitis microscópica</Name>
+      <Name lang="es">Poliangeitis microscópica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -16491,7 +16484,7 @@
     <Disorder id="759">
       <OrphaCode>900</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=900</ExpertLink>
-      <Name lang="es">Granulomatosis con poliangitis</Name>
+      <Name lang="es">Granulomatosis con poliangeítis</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -17448,7 +17441,7 @@
     <Disorder id="17940">
       <OrphaCode>171848</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=171848</ExpertLink>
-      <Name lang="es">Síndrome de polineuropatía-pérdida auditiva-ataxia-retinosis pigmentaria-cataratas</Name>
+      <Name lang="es">Síndrome de polineuropatía-hipoacusia-ataxia-retinosis pigmentaria-cataratas</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -20063,7 +20056,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1329</ExpertLink>
-      <Name lang="es">Canal aurículoventricular completo</Name>
+      <Name lang="es">Defecto del septo aurículoventricular completo</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -20778,7 +20771,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=183678</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak con neutropenia</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de AP-3</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -21023,10 +21016,13 @@
       <DisorderGroup id="36547">
         <Name lang="es">Trastorno</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="es">La edad adulta</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="es">Senectud</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23634">
@@ -27418,7 +27414,7 @@
     <Disorder id="1318">
       <OrphaCode>1005</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1005</ExpertLink>
-      <Name lang="es">Síndrome de alopecia-contracturas-talla baja-discapacidad intelectual</Name>
+      <Name lang="es">Síndrome de alopecia-contracturas-talla baja significativa-discapacidad intelectual</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -30417,7 +30413,7 @@
     <Disorder id="17071">
       <OrphaCode>141261</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=141261</ExpertLink>
-      <Name lang="es">Hendidura facial número 5 de Tessier</Name>
+      <Name lang="es">Tessier number 5 facial cleft</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -31411,7 +31407,7 @@
     <Disorder id="1614">
       <OrphaCode>1416</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1416</ExpertLink>
-      <Name lang="es">Deposición familiar de pirofosfatos de calcio</Name>
+      <Name lang="es">Deposición familiar de pirofosfato cálcico</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -36634,7 +36630,7 @@
     <Disorder id="1899">
       <OrphaCode>1952</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=1952</ExpertLink>
-      <Name lang="es">Displasia de Pacman</Name>
+      <Name lang="es">Síndrome de epífisis punteada-hiperplasia osteoclástica</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -40853,7 +40849,7 @@
     <Disorder id="17233">
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=158673</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa distrófica localizada, forma acral</Name>
+      <Name lang="es">Epidermólisis ampollosa distrófica localizada forma acral</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -42175,7 +42171,7 @@
     <Disorder id="2191">
       <OrphaCode>2339</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2339</ExpertLink>
-      <Name lang="es">Síndrome de queratosis folicular-talla baja-atrofia cerebral</Name>
+      <Name lang="es">Síndrome de queratosis folicular-talla baja significativa-atrofia cerebral</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -48398,7 +48394,7 @@
     <Disorder id="2147">
       <OrphaCode>2278</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2278</ExpertLink>
-      <Name lang="es">Síndrome de ictiosis-discapacidad intelectual-talla baja-afectación renal</Name>
+      <Name lang="es">Síndrome de ictiosis-discapacidad intelectual-talla baja significativa-afectación renal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -54253,7 +54249,7 @@
     <Disorder id="2396">
       <OrphaCode>2617</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2617</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica tipo Montreal</Name>
+      <Name lang="es">Talla baja significativa primordial microcefálica tipo Montreal</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -54951,7 +54947,7 @@
     <Disorder id="2418">
       <OrphaCode>2636</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2636</ExpertLink>
-      <Name lang="es">Talla baja primordial osteodisplásica microcefálica tipos I y III</Name>
+      <Name lang="es">Talla baja significativa primordial osteodisplásica microcefálica tipos I y III</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -54992,7 +54988,7 @@
     <Disorder id="2429">
       <OrphaCode>2658</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2658</ExpertLink>
-      <Name lang="es">Enanismo hiperostótico de Lenz-Majewski</Name>
+      <Name lang="es">Talla baja significativa hiperostótica de Lenz-Majewski</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -61040,7 +61036,7 @@
     <Disorder id="2981">
       <OrphaCode>3363</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3363</ExpertLink>
-      <Name lang="es">Síndrome de tricomegalia-degeneración retiniana pigmentaria-talla baja</Name>
+      <Name lang="es">Síndrome de tricomegalia-degeneración retiniana pigmentaria-talla baja significativa</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -65800,7 +65796,7 @@
     <Disorder id="2882">
       <OrphaCode>3225</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=3225</ExpertLink>
-      <Name lang="es">Síndrome de pérdida auditiva-insensibilidad a la aldosterona de la glándula salivar, familiar</Name>
+      <Name lang="es">Síndrome de hipoacusia-insensibilidad a la aldosterona de la glándula salivar, familiar</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -67187,7 +67183,7 @@
     <Disorder id="3298">
       <OrphaCode>832</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=832</ExpertLink>
-      <Name lang="es">Deficiencia de succinil-CoA:3-cetoácido-CoA transferasa</Name>
+      <Name lang="es">Deficiencia de succinil-CoA: 3 oxoácido CoA transferasa</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -69288,7 +69284,7 @@
     <Disorder id="3171">
       <OrphaCode>2619</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2619</ExpertLink>
-      <Name lang="es">Talla baja con braquidactilia tipo Mseleni</Name>
+      <Name lang="es">Talla baja significativa con braquidactilia tipo Mseleni</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -74105,7 +74101,7 @@
     <Disorder id="3423">
       <OrphaCode>2745</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2745</ExpertLink>
-      <Name lang="es">Síndrome de Opitz G/BBB</Name>
+      <Name lang="es">Síndrome de Opitz GBBB</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -77498,7 +77494,7 @@
     <Disorder id="19064">
       <OrphaCode>228012</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=228012</ExpertLink>
-      <Name lang="es">Síndrome de pérdida auditiva neurosensorial progresiva-miocardiopatía hipertrófica</Name>
+      <Name lang="es">Síndrome de hipoacusia neurosensorial progresiva-miocardiopatía hipertrófica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -80048,7 +80044,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231512</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak sin fibrosis pulmonar</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -80135,7 +80131,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=231500</ExpertLink>
-      <Name lang="es">Síndrome de Hermansky-Pudlak con fibrosis pulmonar</Name>
+      <Name lang="es">Síndrome de Hermansky-Pudlak por deficiencia de BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -83964,7 +83960,7 @@
     <Disorder id="21965">
       <OrphaCode>330054</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=330054</ExpertLink>
-      <Name lang="es">Síndrome de catarata congénita-hipotonía muscular progresiva-pérdida auditiva-retraso del desarrollo</Name>
+      <Name lang="es">Síndrome de catarata congénita-hipotonía muscular progresiva-hipoacusia-retraso del desarrollo</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -84613,7 +84609,7 @@
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=329228</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica por deficiencia de ZNF335</Name>
+      <Name lang="es">Talla baja significativa primordial microcefálica por deficiencia de ZNF335</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -86973,7 +86969,7 @@
     <Disorder id="21691">
       <OrphaCode>319675</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=319675</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica tipo Dauber</Name>
+      <Name lang="es">Talla baja significativa primordial microcefálica tipo Dauber</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -92927,7 +92923,7 @@
     <Disorder id="22057">
       <OrphaCode>352333</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=352333</ExpertLink>
-      <Name lang="es">Síndrome de ictions congénita-discapacidad intelectual-cuadriplejía espástica</Name>
+      <Name lang="es">Síndrome de ictiosis congénita-discapacidad intelectual-cuadriplejía espástica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -93343,7 +93339,7 @@
     <Disorder id="20897">
       <OrphaCode>293967</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=293967</ExpertLink>
-      <Name lang="es">Síndrome de hipogonadismo hipogonadotrópico-microcefalia grave-pérdida auditiva neurosensorial-dismorfia</Name>
+      <Name lang="es">Síndrome de hipogonadismo hipogonadotrópico-microcefalia grave-hipoacusia neurosensorial-dismorfia</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -98233,7 +98229,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=314555</ExpertLink>
-      <Name lang="es">Síndrome de displasia craneofacial-osteopenia</Name>
+      <Name lang="es">Síndrome de dismorfia facial-miopía severa-osteopenia-discapacidad intelectual-anomalías dentales</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -99188,7 +99184,7 @@
     <Disorder id="6020">
       <OrphaCode>2637</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=2637</ExpertLink>
-      <Name lang="es">Enanismo primordial osteodisplásico microcefálico tipo II</Name>
+      <Name lang="es">Talla baja significativa primordial osteodisplásica microcefálica tipo II</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -100329,7 +100325,7 @@
     <Disorder id="21226">
       <OrphaCode>306530</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=306530</ExpertLink>
-      <Name lang="es">Síndrome de parálisis facial hereditaria congénita-pérdida auditiva variable</Name>
+      <Name lang="es">Síndrome de parálisis facial hereditaria congénita-hipoacusia variable</Name>
       <DisorderType id="21415">
         <Name lang="es">Anomalía morfológica</Name>
       </DisorderType>
@@ -100802,7 +100798,7 @@
     <Disorder id="21109">
       <OrphaCode>300313</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=300313</ExpertLink>
-      <Name lang="es">Síndrome de cataratas congénitas-pérdida auditiva-retraso grave del desarrollo</Name>
+      <Name lang="es">Síndrome de cataratas congénitas-hipoacusia-retraso grave del desarrollo</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -104051,7 +104047,7 @@
     <Disorder id="23721">
       <OrphaCode>457351</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=457351</ExpertLink>
-      <Name lang="es">Síndrome de microcefalia-discapacidad intelectual-pérdida auditiva neurosensorial-epilepsia-tono muscular anómalo</Name>
+      <Name lang="es">Síndrome de microcefalia-discapacidad intelectual-hipoacusia neurosensorial-epilepsia-tono muscular anómalo</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -105558,7 +105554,7 @@
     <Disorder id="24223">
       <OrphaCode>468631</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=468631</ExpertLink>
-      <Name lang="es">Talla baja primordial microcefálica-malformaciones cerebrales por deficiencia de RTTN</Name>
+      <Name lang="es">Malformaciones corticales microcefálicas-talla baja por deficiencia de RTTN</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -109505,7 +109501,7 @@
     <Disorder id="22698">
       <OrphaCode>397744</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=397744</ExpertLink>
-      <Name lang="es">Síndrome de neuropatía periférica-miopatía-ronquera-pérdida auditiva</Name>
+      <Name lang="es">Síndrome de neuropatía periférica-miopatía-ronquera-hipoacusia</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -111210,7 +111206,7 @@
     <Disorder id="23333">
       <OrphaCode>436182</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436182</ExpertLink>
-      <Name lang="es">Síndrome de talla baja primordial microcefálica-resistencia a la insulina</Name>
+      <Name lang="es">Síndrome de talla baja significativa primordial microcefálica-resistencia a la insulina</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -111242,7 +111238,7 @@
     <Disorder id="23332">
       <OrphaCode>436174</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=436174</ExpertLink>
-      <Name lang="es">Síndrome de catarata-deficiencia de hormona de crecimiento-neuropatía sensitiva-pérdida auditiva neurosensorial-displasia esquelética</Name>
+      <Name lang="es">Síndrome de catarata-deficiencia de hormona de crecimiento-neuropatía sensitiva-hipoacusia neurosensorial-displasia esquelética</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -112352,7 +112348,7 @@
     <Disorder id="23401">
       <OrphaCode>440354</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=440354</ExpertLink>
-      <Name lang="es">Síndrome de miopía autosómica dominante-retrusión del tercio medio facial-pérdida auditiva neurosensorial-displasia rizomélica</Name>
+      <Name lang="es">Síndrome de miopía autosómica dominante-retrusión del tercio medio facial-hipoacusia neurosensorial-displasia rizomélica</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -118518,7 +118514,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=508529</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa simple generalizada basal con miocardiopatía</Name>
+      <Name lang="es">Epidermólisis ampollosa simple intermedia con miocardiopatía</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -118596,7 +118592,7 @@
     <Disorder id="26568">
       <OrphaCode>508476</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=508476</ExpertLink>
-      <Name lang="es">Síndrome de labio leporino y paladar hendido-dismorfia craneofacial-defecto cardíaco congénito-pérdida auditiva</Name>
+      <Name lang="es">Síndrome de labio leporino y paladar hendido-dismorfia craneofacial-defecto cardíaco congénito-hipoacusia</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -119200,7 +119196,7 @@
     <Disorder id="25675">
       <OrphaCode>494439</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=494439</ExpertLink>
-      <Name lang="es">Síndrome de retinosis pigmentaria-pérdida auditiva-envejecimiento prematuro-talla baja-dismorfia facial</Name>
+      <Name lang="es">Síndrome de retinosis pigmentaria-hipoacusia-envejecimiento prematuro-talla baja-dismorfia facial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -120003,7 +119999,7 @@
     <Disorder id="25998">
       <OrphaCode>500144</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=500144</ExpertLink>
-      <Name lang="es">Síndrome de encefalopatía progresiva-pérdida auditiva-hipoplasia pontina-atrofia cerebral de inicio temprano</Name>
+      <Name lang="es">Síndrome de encefalopatía progresiva-hipoacusia-hipoplasia pontina-atrofia cerebral de inicio temprano</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -124056,7 +124052,7 @@
     <Disorder id="25256">
       <OrphaCode>482606</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=482606</ExpertLink>
-      <Name lang="es">Síndrome ligado al cromosoma X de cicatrización queloide-movilidad articular reducida-aumento de la relación copa/disco</Name>
+      <Name lang="es">Síndrome ligado al cromosoma X de cicatrización queloide-movilidad articular reducida-aumento de la relación copa/disco óptico</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -124968,7 +124964,7 @@
     <Disorder id="25161">
       <OrphaCode>477831</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=477831</ExpertLink>
-      <Name lang="es">Síndrome de sobrecrecimiento esquelético-dismorfia craneofacial-piel hiperelástica-lesiones en la sustancia blanca</Name>
+      <Name lang="es">Síndrome de sobrecrecimiento de Kosaki</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -128297,7 +128293,7 @@
     <Disorder id="10891">
       <OrphaCode>66633</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=66633</ExpertLink>
-      <Name lang="es">Síndrome de pérdida auditiva neurosensorial-aparición temprana de canas-temblor esencial</Name>
+      <Name lang="es">Síndrome de hipoacusia neurosensorial-aparición temprana de canas-temblor esencial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -128439,7 +128435,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=66627</ExpertLink>
-      <Name lang="es">Sinovitis villonodular pigmentada</Name>
+      <Name lang="es">Tumor tenosinovial de células gigantes</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -133565,7 +133561,7 @@
     <Disorder id="11127">
       <OrphaCode>79083</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79083</ExpertLink>
-      <Name lang="es">Lipodistrofia familiar parcial asociada al gen PPARG</Name>
+      <Name lang="es">Lipodistrofia parcial familiar asociada al gen PPARG</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -134948,7 +134944,7 @@
     <Disorder id="11071">
       <OrphaCode>75326</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=75326</ExpertLink>
-      <Name lang="es">Tortuosidad de las arterias retinianas</Name>
+      <Name lang="es">Tortuosidad arterial retiniana</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -138365,7 +138361,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=35062</ExpertLink>
-      <Name lang="es">Infección idiopática diseminada por citomegalovirus</Name>
+      <Name lang="es">Infección diseminada por citomegalovirus grave en pacientes inmunocompetentes</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -139425,7 +139421,7 @@
     <Disorder id="10734">
       <OrphaCode>53689</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=53689</ExpertLink>
-      <Name lang="es">Diarrea congénita de cloruros</Name>
+      <Name lang="es">Diarrea clorada congénita</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -141258,7 +141254,7 @@
     <Disorder id="28036">
       <OrphaCode>542301</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=542301</ExpertLink>
-      <Name lang="es">Inmunodeficiencia combinada grave por deficiencia de CARMIL2</Name>
+      <Name lang="es">Inmunodeficiencia combinada por deficiencia de CARMIL2</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -142503,7 +142499,7 @@
     <Disorder id="28081">
       <OrphaCode>544472</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=544472</ExpertLink>
-      <Name lang="es">Síndrome hemolítico urémico atípico por anomalía del gen del complemento</Name>
+      <Name lang="es">Síndrome hemolítico urémico atípico con anomalías en los genes del complemento</Name>
       <DisorderType id="21443">
         <Name lang="es">Subtipo etiológico</Name>
       </DisorderType>
@@ -143481,7 +143477,7 @@
     <Disorder id="11950">
       <OrphaCode>90059</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90059</ExpertLink>
-      <Name lang="es">Pérdida auditiva neurosensorial a causa de trauma acústico grave o sordera repentina, o por trauma acústico inducido por cirugía</Name>
+      <Name lang="es">Hipoacusia neurosensorial a causa de trauma acústico grave o sordera repentina, o por trauma acústico inducido por cirugía</Name>
       <DisorderType id="21429">
         <Name lang="es">Situación clínica particular en una enfermedad o síndrome</Name>
       </DisorderType>
@@ -144272,7 +144268,7 @@
     <Disorder id="12002">
       <OrphaCode>90291</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90291</ExpertLink>
-      <Name lang="es">Esclerodermia sistémica</Name>
+      <Name lang="es">Esclerosis sistémica</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -144302,11 +144298,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90308</ExpertLink>
       <Name lang="es">Síndrome de Klippel-Trénaunay</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -144331,11 +144327,11 @@
       <OrphaCode>90307</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=90307</ExpertLink>
       <Name lang="es">Síndrome de Parkes Weber</Name>
-      <DisorderType id="21450">
-        <Name lang="es">Subtipo clínico</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="es">Subtipo de trastorno</Name>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -146240,7 +146236,7 @@
     <Disorder id="12199">
       <OrphaCode>93256</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93256</ExpertLink>
-      <Name lang="es">Síndrome de ataxia/temblor asociado al X frágil</Name>
+      <Name lang="es">Síndrome de ataxia/temblor asociado al cromosoma X frágil</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -147121,7 +147117,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93111</ExpertLink>
-      <Name lang="es">Nefropatía tubulointersticial autosómica dominante asociada al gen HNF1B</Name>
+      <Name lang="es">Enfermedad renal tubulointersticial autosómica dominante asociada al gen HNF1B</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -150702,7 +150698,7 @@
     <Disorder id="11436">
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=79410</ExpertLink>
-      <Name lang="es">Epidermólisis ampollosa distrófica localizada, forma pretibial</Name>
+      <Name lang="es">Epidermólisis ampollosa distrófica localizada forma pretibial</Name>
       <DisorderType id="21450">
         <Name lang="es">Subtipo clínico</Name>
       </DisorderType>
@@ -153827,7 +153823,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=85279</ExpertLink>
-      <Name lang="es">Discapacidad intelectual ligada al cromosoma X sindrómica por una mutación en el gen JARID1C</Name>
+      <Name lang="es">Discapacidad intelectual sindrómica ligada al cromosoma asociada al gen KDM5C</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -157737,7 +157733,7 @@
     <Disorder id="11600">
       <OrphaCode>83629</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=83629</ExpertLink>
-      <Name lang="es">Síndrome de leucoencefalopatía-displasia espondilometafisaria</Name>
+      <Name lang="es">Síndrome de leucoencefalopatía-displasia espondiloepimetafisaria</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -157850,7 +157846,7 @@
     <Disorder id="11631">
       <OrphaCode>85164</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=85164</ExpertLink>
-      <Name lang="es">Síndrome de camptodactilia-talla alta-escoliosis-pérdida auditiva</Name>
+      <Name lang="es">Síndrome de camptodactilia-talla alta-escoliosis-hipoacusia</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -158536,7 +158532,7 @@
     <Disorder id="30684">
       <OrphaCode>610573</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=610573</ExpertLink>
-      <Name lang="es">CLCN6-related childhood onset progressive neurodegeneration-peripheral neuropathy-syndrome</Name>
+      <Name lang="es">Síndrome de neuropatía periférica-neurodegeneración progresiva de inicio en la infancia asociado al gen CLCN6</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -158556,7 +158552,7 @@
     <Disorder id="30683">
       <OrphaCode>610569</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=610569</ExpertLink>
-      <Name lang="es">KIAA1109-related early lethal-congenital brain mlaformations-arthrogryposis-syndrome</Name>
+      <Name lang="es">Síndrome de malformaciones cerebrales letales tempranas congénitas-artrogriposis asociado al gen KIAA1109</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
@@ -163189,6 +163185,29 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=599373</ExpertLink>
+      <Name lang="es">Encefalopatía asociada a STXBP1</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23515">
+          <Name lang="es">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="es">No se dispone de datos</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=93403</ExpertLink>
@@ -165734,6 +165753,61 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=592564</ExpertLink>
+      <Name lang="es">Espectro de trastornos del movimiento-crisis-retraso del desarrollo asociado al gen GNAO1</Name>
+      <DisorderType id="21394">
+        <Name lang="es">Enfermedad</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="es">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="es">No se dispone de datos</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=592570</ExpertLink>
+      <Name lang="es">Síndrome de retraso motor y de la adquisición del habla-dismorfia facial-anomalías digitales-defecto cardíaco asociado al gen TRAF7</Name>
+      <DisorderType id="21401">
+        <Name lang="es">Síndrome de malformación</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="es">Trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="es">Prenatal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="es">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="es">No se dispone de datos</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14291">
       <OrphaCode>99718</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=99718</ExpertLink>
@@ -165836,6 +165910,172 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589821</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio congénito</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="es">Prenatal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="es">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="es">Adolescencia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="es">Adultos</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="es">Infancia temprana</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="es">Infantil</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="es">Infancia tardía</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="es">Edad adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589827</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio juvenil</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="es">Adolescencia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="es">Adultos</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="es">Infancia temprana</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="es">Infancia tardía</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="es">Edad adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589824</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio en la infancia</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="es">Infancia</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="es">Adolescencia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="es">Adultos</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="es">Infancia temprana</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="es">Infancia tardía</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="es">Edad adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589833</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio tardío</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23550">
+          <Name lang="es">Senectud</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="es">Senectud</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589830</ExpertLink>
+      <Name lang="es">Distrofia miotónica de Steinert de inicio en el adulto</Name>
+      <DisorderType id="21450">
+        <Name lang="es">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="es">Subtipo de trastorno</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="es">La edad adulta</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="es">Adultos</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="es">Senectud</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="es">Edad adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589905</ExpertLink>
@@ -167127,7 +167367,7 @@
     <Disorder id="29429">
       <OrphaCode>589442</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=589442</ExpertLink>
-      <Name lang="es">Síndrome de talla baja-displasia esquelética-degeneración retiniana-discapacidad intelectual-pérdida auditiva neurosensorial</Name>
+      <Name lang="es">Síndrome de talla baja-displasia esquelética-degeneración retiniana-discapacidad intelectual-hipoacusia neurosensorial</Name>
       <DisorderType id="21401">
         <Name lang="es">Síndrome de malformación</Name>
       </DisorderType>
@@ -171401,7 +171641,7 @@
     <Disorder id="14984">
       <OrphaCode>103908</OrphaCode>
       <ExpertLink lang="es">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=es&amp;Expert=103908</ExpertLink>
-      <Name lang="es">Diarrea congénita sódica</Name>
+      <Name lang="es">Diarrea sódica congénita</Name>
       <DisorderType id="21394">
         <Name lang="es">Enfermedad</Name>
       </DisorderType>
