commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml
index e5b3976..20900fb 100644
--- a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:41:27" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:42:10" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6577">
+  <DisorderList count="6566">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=166024</ExpertLink>
@@ -2154,10 +2154,22 @@
       <DisorderGroup id="36547">
         <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="6">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="fr">Adolescence</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23543">
           <Name lang="fr">Adulte</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23508">
+          <Name lang="fr">Prénatal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="fr">Enfance</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="fr">Petite enfance</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="fr">Néonatal</Name>
         </AverageAgeOfOnset>
@@ -21699,8 +21711,8 @@
       <OrphaCode>847</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=847</ExpertLink>
       <Name lang="fr">Alpha-thalassémie-déficience intellectuelle liée à l'X</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21401">
+        <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="fr">Pathologie</Name>
@@ -48371,7 +48383,7 @@
     <Disorder id="19790">
       <OrphaCode>254857</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=254857</ExpertLink>
-      <Name lang="fr">Myopathy mitochondriale létale infantile</Name>
+      <Name lang="fr">Myopathie mitochondriale létale infantile</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -54612,32 +54624,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="2746">
-      <OrphaCode>3056</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=3056</ExpertLink>
-      <Name lang="fr">Déficience intellectuelle liée à l'X type Brooks</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="fr">Non documenté</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="2747">
       <OrphaCode>3057</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=3057</ExpertLink>
@@ -68626,7 +68612,7 @@
     <Disorder id="3183">
       <OrphaCode>3051</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=3051</ExpertLink>
-      <Name lang="fr">Syndrome de déficience intellectuelle-hypotrichose-brachydactylie</Name>
+      <Name lang="fr">Syndrome de Nicolaides-Baraitser</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -91390,7 +91376,7 @@
     <Disorder id="22081">
       <OrphaCode>352577</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=352577</ExpertLink>
-      <Name lang="fr">Syndrome de troubles sévères de l'alimentation-retard staturo-pondéral-microcéphalie par déficit en ASXL3</Name>
+      <Name lang="fr">Syndrome de Bainbridge-Ropers</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -92359,35 +92345,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20904">
-      <OrphaCode>294049</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=294049</ExpertLink>
-      <Name lang="fr">Syndrome Larsen-like de la Réunion</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="fr">Non documenté</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="fr">Autosomique récessive</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20900">
       <OrphaCode>293987</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=293987</ExpertLink>
@@ -106282,7 +106239,7 @@
     <Disorder id="22920">
       <OrphaCode>404443</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=404443</ExpertLink>
-      <Name lang="fr">Syndrome de grande taille-déficience intellectuelle-dysmorphie faciale</Name>
+      <Name lang="fr">Syndrome de Tatton-Brown-Rahman</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -118502,32 +118459,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="25668">
-      <OrphaCode>494348</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=494348</ExpertLink>
-      <Name lang="fr">Hypertension portale non cirrhotique familiale à début précoce</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23529">
-          <Name lang="fr">Enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="fr">Non documenté</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="25683">
       <OrphaCode>494547</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=494547</ExpertLink>
@@ -128686,12 +128617,12 @@
     <Disorder id="10847">
       <OrphaCode>64743</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=64743</ExpertLink>
-      <Name lang="fr">Veinopathie portale oblitérative</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <Name lang="fr">Veinopathie portale oblitérante</Name>
+      <DisorderType id="21457">
+        <Name lang="fr">Sous-type histo-pathologique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23557">
@@ -134757,32 +134688,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="11034">
-      <OrphaCode>73220</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=73220</ExpertLink>
-      <Name lang="fr">Syndrome de déficience intellectuelle liée à l'X-faciès hypotonique</Name>
-      <DisorderType id="21436">
-        <Name lang="fr">Groupe clinique</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="fr">Groupe de pathologies</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10462">
       <OrphaCode>40366</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=40366</ExpertLink>
@@ -141238,11 +141143,11 @@
       <OrphaCode>48372</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=48372</ExpertLink>
       <Name lang="fr">Hyperplasie nodulaire régénérative du foie</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21457">
+        <Name lang="fr">Sous-type histo-pathologique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -153095,26 +153000,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="11698">
-      <OrphaCode>85328</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=85328</ExpertLink>
-      <Name lang="fr">Déficience intellectuelle liée à l'X type Turner</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23445">
-          <Name lang="fr">Dominante liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="11699">
       <OrphaCode>85329</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=85329</ExpertLink>
@@ -159076,7 +158961,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=95702</ExpertLink>
-      <Name lang="fr">Hypoplasie congénitale des surrénales liée à l'X due à une mutation ponctuelle de NR0B1</Name>
+      <Name lang="fr">Hypoplasie congénitale des surrénales liée à l'X</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -162310,58 +162195,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12533">
-      <OrphaCode>93975</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93975</ExpertLink>
-      <Name lang="fr">Syndrome de Renier-Gabreels-Jasper</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12532">
-      <OrphaCode>93974</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93974</ExpertLink>
-      <Name lang="fr">Syndrome de Smith-Fineman-Myers</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12534">
       <OrphaCode>93976</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93976</ExpertLink>
@@ -162388,113 +162221,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12529">
-      <OrphaCode>93971</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93971</ExpertLink>
-      <Name lang="fr">Syndrome de Chudley-Lowry-Hoar</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12528">
-      <OrphaCode>93970</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93970</ExpertLink>
-      <Name lang="fr">Syndrome de Holmes-Gang</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23599">
-          <Name lang="fr">petite enfance</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12531">
-      <OrphaCode>93973</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93973</ExpertLink>
-      <Name lang="fr">Syndrome de Carpenter-Waziri</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12530">
-      <OrphaCode>93972</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93972</ExpertLink>
-      <Name lang="fr">Syndrome de Juberg-Marsidi</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="fr">Récessive liée à l'X</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12492">
       <OrphaCode>93932</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93932</ExpertLink>
