commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml
old mode 100644
new mode 100755
index 20900fb..a3a31ca
--- a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:42:10" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:43:11" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -19691,7 +19691,7 @@
     <Disorder id="894">
       <OrphaCode>3451</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=3451</ExpertLink>
-      <Name lang="fr">Syndrome de West</Name>
+      <Name lang="fr">Syndrome des spasmes infantiles</Name>
       <DisorderType id="21422">
         <Name lang="fr">Syndrome clinique</Name>
       </DisorderType>
@@ -22388,7 +22388,7 @@
     <Disorder id="972">
       <OrphaCode>776</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=776</ExpertLink>
-      <Name lang="fr">Syndrome marfanoïde avec déficience intellectuelle liée à l'X</Name>
+      <Name lang="fr">Syndrome de Lujan-Fryns</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -53237,12 +53237,12 @@
     <Disorder id="19853">
       <OrphaCode>261144</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=261144</ExpertLink>
-      <Name lang="fr">Syndrome de microdélétion 14q12</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
+      <Name lang="fr">Syndrome FOXG1 dû à une microdélétion 14q12</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -62734,11 +62734,11 @@
       <OrphaCode>280315</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=280315</ExpertLink>
       <Name lang="fr">Pancréatite auto-immune type 2</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -82596,7 +82596,7 @@
     <Disorder id="19335">
       <OrphaCode>240112</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=240112</ExpertLink>
-      <Name lang="fr">Syndrome de paralysie supranucléaire progressive-aphasie progressive non-fluente</Name>
+      <Name lang="fr">Syndrome de paralysie supranucléaire progressive-aphasie progressive non fluente</Name>
       <DisorderType id="21450">
         <Name lang="fr">Sous-type clinique</Name>
       </DisorderType>
@@ -104623,7 +104623,7 @@
     <Disorder id="24223">
       <OrphaCode>468631</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=468631</ExpertLink>
-      <Name lang="fr">Nanisme microcéphalique primordial par déficit en RTTN</Name>
+      <Name lang="fr">Malformations corticales microcéphaliques-petite taille par déficit en RTTN</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -111869,7 +111869,7 @@
     <Disorder id="23427">
       <OrphaCode>442835</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=442835</ExpertLink>
-      <Name lang="fr">Encéphalopathie épileptique précoce indéterminée</Name>
+      <Name lang="fr">Encéphalopathie épileptique à début précoce non spécifique</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -117514,7 +117514,7 @@
     <Disorder id="26573">
       <OrphaCode>508512</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=508512</ExpertLink>
-      <Name lang="fr">Syndrome de taches café-au-lait-augmentation du taux d'échanges entre chromatides soeurs</Name>
+      <Name lang="fr">Syndrome de retard de croissance intra-utérin-taches café-au-lait-augmentation du taux d'échanges entre chromatides soeurs</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -163546,11 +163546,11 @@
       <OrphaCode>99361</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=99361</ExpertLink>
       <Name lang="fr">Cancer médullaire de la thyroïde familial</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
@@ -164164,11 +164164,11 @@
       <OrphaCode>98813</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=98813</ExpertLink>
       <Name lang="fr">Dysplasie ectodermique hypohidrotique avec déficit immunitaire</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -167343,8 +167343,8 @@
       <OrphaCode>98791</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=98791</ExpertLink>
       <Name lang="fr">Syndrome d'alpha-thalassémie-déficience intellectuelle associée au chromosome 16</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21401">
+        <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="fr">Pathologie</Name>
@@ -168831,11 +168831,11 @@
       <OrphaCode>103919</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=103919</ExpertLink>
       <Name lang="fr">Pancréatite auto-immune</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21436">
+        <Name lang="fr">Groupe clinique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36540">
+        <Name lang="fr">Groupe de pathologies</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
