commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml
index f4a8a18..4fca3b9 100644
--- a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:59:36" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:56:42" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6641">
+  <DisorderList count="6636">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=166024</ExpertLink>
@@ -2475,15 +2475,24 @@
       <DisorderGroup id="36547">
         <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="fr">Adolescence</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="fr">Adulte</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23529">
           <Name lang="fr">Enfance</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
+      <AverageAgeOfDeathList count="2">
         <AverageAgeOfDeath id="23627">
           <Name lang="fr">Adulte</Name>
         </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="fr">Troisième age</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
         <TypeOfInheritance id="23445">
@@ -5823,7 +5832,7 @@
     <Disorder id="244">
       <OrphaCode>207</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=207</ExpertLink>
-      <Name lang="fr">Maladie de Crouzon</Name>
+      <Name lang="fr">Syndrome de Crouzon</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -21127,7 +21136,7 @@
     <Disorder id="946">
       <OrphaCode>3027</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=3027</ExpertLink>
-      <Name lang="fr">Séquence de régression caudale</Name>
+      <Name lang="fr">Syndrome de régression caudale</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -21135,14 +21144,26 @@
         <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="fr">Petite enfance</Name>
+        <AverageAgeOfOnset id="23508">
+          <Name lang="fr">Prénatal</Name>
         </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="fr">Néonatal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="4">
+        <AverageAgeOfDeath id="23648">
+          <Name lang="fr">Tout âge</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23578">
+          <Name lang="fr">embryofétale</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="fr">infantile</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23585">
+          <Name lang="fr">mortinatalité</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
         <TypeOfInheritance id="23424">
@@ -21675,7 +21696,10 @@
           <Name lang="fr">Espérance de vie normale</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="fr">Autosomique dominante</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="fr">Autosomique récessive</Name>
         </TypeOfInheritance>
@@ -24594,38 +24618,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=946</ExpertLink>
-      <Name lang="fr">Acro-céphalo-syndactylie</Name>
-      <DisorderType id="21436">
-        <Name lang="fr">Groupe clinique</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="fr">Groupe de pathologies</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23508">
-          <Name lang="fr">Prénatal</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="fr">Néonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="fr">Tout âge</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="fr">Autosomique dominante</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="fr">Non applicable</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=957</ExpertLink>
@@ -34850,32 +34842,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=1533</ExpertLink>
-      <Name lang="fr">Syndrome de craniosynostose-aplasie du péroné</Name>
-      <DisorderType id="21401">
-        <Name lang="fr">Syndrome  malformatif</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23529">
-          <Name lang="fr">Enfance</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="fr">Non documenté</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="fr">Autosomique récessive</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=140874</ExpertLink>
@@ -35887,7 +35853,7 @@
     <Disorder id="1684">
       <OrphaCode>1517</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=1517</ExpertLink>
-      <Name lang="fr">Ostéochondrodysplasie hypertrichotique type Cantu</Name>
+      <Name lang="fr">Syndrome de Cantú</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -64957,10 +64923,7 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="fr">Autosomique dominante</Name>
-        </TypeOfInheritance>
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23417">
           <Name lang="fr">Autosomique récessive</Name>
         </TypeOfInheritance>
@@ -73518,7 +73481,7 @@
     <Disorder id="18827">
       <OrphaCode>217253</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=217253</ExpertLink>
-      <Name lang="fr">Encéphalite limbique avec anticorps anti-récepteur NMDA</Name>
+      <Name lang="fr">Encéphalite anti-NMDAr</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -84477,26 +84440,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=329341</ExpertLink>
-      <Name lang="fr">Encéphalite limbique avec anticorps anti-DPP6</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="fr">Adulte</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=329228</ExpertLink>
@@ -98103,7 +98046,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=314555</ExpertLink>
-      <Name lang="fr">Syndrome de dysmorphie faciale-myopie sévère-ostéopénie-déficience intellectuelle-anomalies dentaires</Name>
+      <Name lang="fr">Syndrome de dysmorphie faciale-anomalies oculaires-ostéopénie-déficience intellectuelle-anomalies dentaires</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -103127,26 +103070,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="23782">
-      <OrphaCode>459353</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=459353</ExpertLink>
-      <Name lang="fr">Déficit de l'inhibiteur C1</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="fr">Autosomique dominante</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="23767">
       <OrphaCode>459033</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=459033</ExpertLink>
@@ -120863,7 +120786,7 @@
     <Disorder id="26069">
       <OrphaCode>502305</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=502305</ExpertLink>
-      <Name lang="fr">Dysplasie cochléovestibulaire</Name>
+      <Name lang="fr">Malformation cochléovestibulaire</Name>
       <DisorderType id="21415">
         <Name lang="fr">Anomalie morphologique</Name>
       </DisorderType>
@@ -122485,26 +122408,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=498700</ExpertLink>
-      <Name lang="fr">Encéphalite limbique avec anticorps neurexine-3</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="fr">Adulte</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=874</ExpertLink>
@@ -177089,7 +176992,7 @@
     <Disorder id="31415">
       <OrphaCode>619363</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=619363</ExpertLink>
-      <Name lang="fr">Maladie auto-inflammatoire multisystémique sévère-apparition néonatale-augmentation de IL-18</Name>
+      <Name lang="fr">Maladie auto-inflammatoire multisystémique sévère néonatale avec augmentation de l'IL18</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
