commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml
index 236d9ee..1a00395 100644
--- a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:40:25" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:30:34" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="6636">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -14341,7 +14334,7 @@
     <Disorder id="678">
       <OrphaCode>62</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=62</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à l'alpha-sarcoglycane R3</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à l'alpha-sarcoglycane R3</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -34741,7 +34734,7 @@
     <Disorder id="16999">
       <OrphaCode>140922</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=140922</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à la titine R10</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à la titine R10</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -50610,7 +50603,7 @@
     <Disorder id="19742">
       <OrphaCode>254361</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=254361</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à la plectine R17</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à la plectine R17</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -63072,7 +63065,7 @@
     <Disorder id="20455">
       <OrphaCode>280333</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=280333</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à l'alpha dystroglycane R16</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à l'alpha dystroglycane R16</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -67250,7 +67243,7 @@
     <Disorder id="18521">
       <OrphaCode>206554</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=206554</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à la fukutine R13</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à la fukutine R13</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -67296,7 +67289,7 @@
     <Disorder id="18520">
       <OrphaCode>206549</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=206549</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à l'anoctamine-5 R12</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à l'anoctamine-5 R12</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -117669,7 +117662,7 @@
     <Disorder id="8717">
       <OrphaCode>219</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=219</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée au delta-sarcoglycane R6</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée au delta-sarcoglycane R6</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -117727,7 +117720,7 @@
     <Disorder id="8715">
       <OrphaCode>119</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=119</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée au bêta-sarcoglycane R4</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée au bêta-sarcoglycane R4</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -138134,7 +138127,7 @@
     <Disorder id="10336">
       <OrphaCode>34514</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=34514</ExpertLink>
-      <Name lang="fr">Dystrophie musculaire des ceintures  liée à la téléthonine R7</Name>
+      <Name lang="fr">Dystrophie musculaire des ceintures liée à la téléthonine R7</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
