commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml
index 4889c07..e8d4a4b 100644
--- a/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/fr_product9_ages.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:51:23" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6633">
+<JDBOR date="2021-12-01 04:55:53" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=166024</ExpertLink>
@@ -7752,7 +7745,7 @@
     <Disorder id="324">
       <OrphaCode>753</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=753</ExpertLink>
-      <Name lang="fr">Anomalie du développement sexuel 46,XY par déficit en 5-alpha-réductase</Name>
+      <Name lang="fr">Anomalie du développement sexuel 46,XY par déficit en 5-alpha-réductase de type 2</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -12687,11 +12680,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=2346</ExpertLink>
       <Name lang="fr">Syndrome angio-ostéo-hypertrophique</Name>
-      <DisorderType id="21394">
-        <Name lang="fr">Maladie</Name>
+      <DisorderType id="21436">
+        <Name lang="fr">Groupe clinique</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="fr">Pathologie</Name>
+      <DisorderGroup id="36540">
+        <Name lang="fr">Groupe de pathologies</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="3">
         <AverageAgeOfOnset id="23536">
@@ -21023,10 +21016,13 @@
       <DisorderGroup id="36547">
         <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="fr">Adulte</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="fr">Troisième age</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23634">
@@ -74784,7 +74780,7 @@
     <Disorder id="18855">
       <OrphaCode>217399</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=217399</ExpertLink>
-      <Name lang="fr">Insensibilité congénitale à la douleur avec hyperhidrose</Name>
+      <Name lang="fr">Insensibilité congénitale à la douleur-hyperhidrose-absence d'innervation des fibres C</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -98233,7 +98229,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=314555</ExpertLink>
-      <Name lang="fr">Syndrome de dysplasie craniofaciale et ostéopénie</Name>
+      <Name lang="fr">Syndrome de dysmorphie faciale-myopie sévère-ostéopénie-déficience intellectuelle-anomalies dentaires</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -128439,7 +128435,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=66627</ExpertLink>
-      <Name lang="fr">Synovite villonodulaire pigmentée</Name>
+      <Name lang="fr">Tumeur ténosynoviale à cellules géantes</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -144302,11 +144298,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=90308</ExpertLink>
       <Name lang="fr">Syndrome de Klippel-Trénaunay</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -144331,11 +144327,11 @@
       <OrphaCode>90307</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=90307</ExpertLink>
       <Name lang="fr">Syndrome de Parkes Weber</Name>
-      <DisorderType id="21450">
-        <Name lang="fr">Sous-type clinique</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="fr">Sous-type d'une pathologie</Name>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -144980,7 +144976,7 @@
     <Disorder id="11818">
       <OrphaCode>88642</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=88642</ExpertLink>
-      <Name lang="fr">Canalopathie associée à une insensibilité congénitale à la douleur</Name>
+      <Name lang="fr">Insensibilité congénitale à la douleur-anosmie-arthropathie neuropathique</Name>
       <DisorderType id="21394">
         <Name lang="fr">Maladie</Name>
       </DisorderType>
@@ -153827,7 +153823,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=85279</ExpertLink>
-      <Name lang="fr">Déficience intellectuelle liée à l'X syndromique par mutation de JARID1C</Name>
+      <Name lang="fr">Déficience intellectuelle liée à l'X syndromique associée à KDM5C</Name>
       <DisorderType id="21401">
         <Name lang="fr">Syndrome  malformatif</Name>
       </DisorderType>
@@ -163189,6 +163185,29 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=599373</ExpertLink>
+      <Name lang="fr">Encéphalopathie liée à STXBP1</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23515">
+          <Name lang="fr">Néonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="fr">Non documenté</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=93403</ExpertLink>
@@ -165734,6 +165753,61 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=592564</ExpertLink>
+      <Name lang="fr">Spectre de retard de développement-épilepsies-anomalies du mouvement lié à GNAO1</Name>
+      <DisorderType id="21394">
+        <Name lang="fr">Maladie</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="fr">Enfance</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="fr">Petite enfance</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="fr">Néonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="fr">Non documenté</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=592570</ExpertLink>
+      <Name lang="fr">Syndrome d'anomalies cardiaques-anomalies digitales-dysmorphie faciale-retard moteur et verbal associé à TRAF7</Name>
+      <DisorderType id="21401">
+        <Name lang="fr">Syndrome  malformatif</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="fr">Pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="fr">Prénatal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="fr">Néonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="fr">Non documenté</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14291">
       <OrphaCode>99718</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=99718</ExpertLink>
@@ -165836,6 +165910,172 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=589821</ExpertLink>
+      <Name lang="fr">Dystrophie myotonique de Steinert, forme congénitale</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="fr">Prénatal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="fr">Néonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="fr">adolescent</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="fr">Adulte</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="fr">petite enfance</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="fr">infantile</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="fr">enfance</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="fr">adulte jeune</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=589827</ExpertLink>
+      <Name lang="fr">Dystrophie myotonique de Steinert à début juvénile</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="fr">Enfance</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="fr">adolescent</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="fr">Adulte</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="fr">petite enfance</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="fr">enfance</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="fr">adulte jeune</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=589824</ExpertLink>
+      <Name lang="fr">Dystrophie myotonique de Steinert, forme infantile</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="fr">Enfance</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="fr">Petite enfance</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="fr">adolescent</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="fr">Adulte</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="fr">petite enfance</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="fr">enfance</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="fr">adulte jeune</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=589833</ExpertLink>
+      <Name lang="fr">Dystrophie myotonique de Steinert à début tardif</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23550">
+          <Name lang="fr">Troisième age</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="fr">Troisième age</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=589830</ExpertLink>
+      <Name lang="fr">Dystrophie myotonique de Steinert de l'adulte</Name>
+      <DisorderType id="21450">
+        <Name lang="fr">Sous-type clinique</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="fr">Sous-type d'une pathologie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="fr">Adulte</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="fr">Adulte</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="fr">Troisième age</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="fr">adulte jeune</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="fr">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=fr&amp;Expert=589905</ExpertLink>
