commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Epidemiological data/Rare diseases natural history/it_product9_ages.xml b/Epidemiological data/Rare diseases natural history/it_product9_ages.xml
index bc12cfb..deec323 100644
--- a/Epidemiological data/Rare diseases natural history/it_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/it_product9_ages.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:40:36" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 17:46:44" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="6636">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -1964,7 +1957,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=593</ExpertLink>
       <Name lang="it">Miopatia miofibrillare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -6156,7 +6149,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2443</ExpertLink>
       <Name lang="it">Difetto della fosforilazione ossidativa mitocondriale da anomalie del DNA nucleare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -6472,7 +6465,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99</ExpertLink>
       <Name lang="it">Atassia cerebellare, autosomica dominante</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -8534,7 +8527,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=352</ExpertLink>
       <Name lang="it">Galattosemia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -13438,7 +13431,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=69</ExpertLink>
       <Name lang="it">Amiloidosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -13504,7 +13497,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166</ExpertLink>
       <Name lang="it">Malattia di Charcot-Marie-Tooth</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -15333,7 +15326,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=174590</ExpertLink>
       <Name lang="it">Ipogonadismo ipogonadotropo congenito</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -16201,7 +16194,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=187</ExpertLink>
       <Name lang="it">Citrullinemia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -16265,7 +16258,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=442</ExpertLink>
       <Name lang="it">Ipotiroidismo congenito</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -17191,7 +17184,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=795</ExpertLink>
       <Name lang="it">Forma rara di salmonellosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -19019,7 +19012,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1531</ExpertLink>
       <Name lang="it">Craniosinostosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -22136,7 +22129,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183422</ExpertLink>
       <Name lang="it">Malattia genetica da malformazioni multiple associata a un rischio aumentato di tumore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -22272,7 +22265,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=182090</ExpertLink>
       <Name lang="it">Ipertensione arteriosa polmonare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -22333,7 +22326,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1047</ExpertLink>
       <Name lang="it">Anemia sideroblastica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -25872,7 +25865,7 @@
     <Disorder id="1399">
       <OrphaCode>1118</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1118</ExpertLink>
-      <Name lang="it">Aplasia del perone - ectrodattilia</Name>
+      <Name lang="it">Sindrome da aplasia del perone-ectrodattilia</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -34847,7 +34840,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=140874</ExpertLink>
       <Name lang="it">Sindrome di Joubert e sindromi correlate</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -39229,7 +39222,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=158032</ExpertLink>
       <Name lang="it">Sindrome emofagocitica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -41615,7 +41608,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=247765</ExpertLink>
       <Name lang="it">Atassia cerebellare legata all'X</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -42015,7 +42008,7 @@
     <Disorder id="2191">
       <OrphaCode>2339</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2339</ExpertLink>
-      <Name lang="it">Cheratosi follicolare - nanismo - atrofia cerebrale</Name>
+      <Name lang="it">Sindrome da cheratosi follicolare-nanismo-atrofia cerebrale</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -42410,7 +42403,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=247582</ExpertLink>
       <Name lang="it">Deficit di citrina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -43777,7 +43770,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=247239</ExpertLink>
       <Name lang="it">Atassia degenerativa non ereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -47722,7 +47715,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=251355</ExpertLink>
       <Name lang="it">Drepanocitosi associata ad altro difetto dell'emoglobina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -50534,7 +50527,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=254367</ExpertLink>
       <Name lang="it">Lichen planus raro</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -51767,7 +51760,7 @@
     <Disorder id="2556">
       <OrphaCode>2815</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2815</ExpertLink>
-      <Name lang="it">Paraparesi spastica - sordità</Name>
+      <Name lang="it">Sindrome da paraparesi spastica-sordità</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -52932,7 +52925,7 @@
     <Disorder id="2344">
       <OrphaCode>2536</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2536</ExpertLink>
-      <Name lang="it">Microcornea - glaucoma - assenza del seno frontale</Name>
+      <Name lang="it">Sindrome da microcornea-glaucoma-assenza dei seni frontali</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -52963,7 +52956,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=261911</ExpertLink>
       <Name lang="it">Delezione parziale del braccio corto del cromosoma 7</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -58142,7 +58135,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2846</ExpertLink>
       <Name lang="it">Anomalia congenita del pericardio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -60787,7 +60780,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=275791</ExpertLink>
       <Name lang="it">Ipertensione arteriosa polmonare associata ad altra malattia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -68463,7 +68456,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=206997</ExpertLink>
       <Name lang="it">Miosite parassitaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -70488,7 +70481,7 @@
     <Disorder id="3521">
       <OrphaCode>3240</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3240</ExpertLink>
-      <Name lang="it">Calcificazione del sistema nervoso centrale - sordità - acidosi tubolare - anemia</Name>
+      <Name lang="it">Sindrome da calcificazione del sistema nervoso centrale-sordità-acidosi tubolare-anemia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -70719,7 +70712,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=599</ExpertLink>
       <Name lang="it">Miopatia distale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -70872,7 +70865,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2034</ExpertLink>
       <Name lang="it">Filariosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -71095,7 +71088,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2415</ExpertLink>
       <Name lang="it">Malformazione linfatica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -71307,7 +71300,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=137</ExpertLink>
       <Name lang="it">Disturbo congenito della glicosilazione</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -71551,7 +71544,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=359</ExpertLink>
       <Name lang="it">Glaucoma ereditario</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -72609,7 +72602,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=216675</ExpertLink>
       <Name lang="it">Trasposizione delle grandi arterie</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -73633,7 +73626,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=217074</ExpertLink>
       <Name lang="it">Carcinoma raro del pancreas</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -74159,7 +74152,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=344</ExpertLink>
       <Name lang="it">Febbre da arbovirus</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -74185,7 +74178,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=341</ExpertLink>
       <Name lang="it">Febbre emorragica virale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -74549,7 +74542,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1132</ExpertLink>
       <Name lang="it">Anomalie dell'arco aortico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -74893,7 +74886,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=217604</ExpertLink>
       <Name lang="it">Cardiomiopatia dilatativa</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -75919,7 +75912,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=220489</ExpertLink>
       <Name lang="it">Emocromatosi ereditaria rara</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76934,7 +76927,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=3398</ExpertLink>
       <Name lang="it">Neoplasia epiteliale del timo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -76986,7 +76979,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=547</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -77610,7 +77603,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=226292</ExpertLink>
       <Name lang="it">Ipotiroidismo congenito permanente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -78305,7 +78298,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1172</ExpertLink>
       <Name lang="it">Atassia cerebellare, autosomica recessiva</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -80306,7 +80299,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231413</ExpertLink>
       <Name lang="it">Variante della sindrome di Guillain-Barré</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -80335,7 +80328,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1866</ExpertLink>
       <Name lang="it">Distonia focale, segmentale o multifocale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -80540,7 +80533,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231230</ExpertLink>
       <Name lang="it">Beta talassemia associata ad altro difetto dell'emoglobina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -81790,7 +81783,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231637</ExpertLink>
       <Name lang="it">Iperaldosteronismo primitivo raro corregibile chirurgicamente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -81946,7 +81939,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231641</ExpertLink>
       <Name lang="it">Iperaldosteronismo primitivo raro non corregibile chirugicamente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -82803,7 +82796,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=238547</ExpertLink>
       <Name lang="it">Policitemia secondaria acquisita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -82855,7 +82848,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=238536</ExpertLink>
       <Name lang="it">Policitemia secondaria congenita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -86112,7 +86105,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=319543</ExpertLink>
       <Name lang="it">Suscettibilità mendeliana autosomica dominante a malattie micobatteriche da deficit parziale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -86164,7 +86157,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=319535</ExpertLink>
       <Name lang="it">Suscettibilità mendeliana autosomica recessiva a malattie micobatteriche da deficit completo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -86187,7 +86180,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=319539</ExpertLink>
       <Name lang="it">Suscettibilità mendeliana autosomica recessiva a malattie micobatteriche da deficit parziale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -88147,7 +88140,7 @@
     <Disorder id="21506">
       <OrphaCode>314632</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=314632</ExpertLink>
-      <Name lang="it">Parkinsonimo da deficit di ATP13A2</Name>
+      <Name lang="it">Ceroidolipofuscinosi neuronale correlata a ATP13A2</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -93401,7 +93394,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=293830</ExpertLink>
       <Name lang="it">Anemia diseritropoietica costituzionale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -93473,7 +93466,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=293815</ExpertLink>
       <Name lang="it">Dermatosi tossica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -95800,7 +95793,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280926</ExpertLink>
       <Name lang="it">Malattie sistemiche con uveite anteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -95899,7 +95892,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280898</ExpertLink>
       <Name lang="it">Panuveite</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -95925,7 +95918,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280892</ExpertLink>
       <Name lang="it">Uveite posteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -95951,7 +95944,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280886</ExpertLink>
       <Name lang="it">Uveite anteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -99526,7 +99519,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300579</ExpertLink>
       <Name lang="it">Tossiemia da Stafilococco</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -99725,7 +99718,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300846</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin B aggressivo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -99919,7 +99912,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=300842</ExpertLink>
       <Name lang="it">Linfoma non-Hodgkin B indolente</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -100478,7 +100471,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=298644</ExpertLink>
       <Name lang="it">Difetto del metabolismo e del trasporto della tiamina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -103522,7 +103515,7 @@
     <Disorder id="23717">
       <OrphaCode>457260</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=457260</ExpertLink>
-      <Name lang="it">Sindrome disabilità intellettiva - ipotonia - anomalie del comportamento legata all'X</Name>
+      <Name lang="it">Sindrome da disabilità intellettiva-ipotonia-disturbi del movimento, legata all'X</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -103644,7 +103637,7 @@
     <Disorder id="23727">
       <OrphaCode>457406</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=457406</ExpertLink>
-      <Name lang="it">Sindrome letale da disfunzione mitocondriale multipla tipo 4</Name>
+      <Name lang="it">Sindrome da disfunzioni mitocondriali multiple tipo 4</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -106139,7 +106132,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=401854</ExpertLink>
       <Name lang="it">Difetto della biosintesi dell'acido lipoico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -108896,7 +108889,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398043</ExpertLink>
       <Name lang="it">Tumore maligno del pene</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -109975,7 +109968,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398934</ExpertLink>
       <Name lang="it">Tumore epiteliale maligno dell'ovaio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -110015,7 +110008,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398940</ExpertLink>
       <Name lang="it">Tumore non epiteliale maligno dell'ovaio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -110320,7 +110313,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=398091</ExpertLink>
       <Name lang="it">Malattia autoimmune neonatale secondaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -110429,7 +110422,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=435743</ExpertLink>
       <Name lang="it">Anomalia congenita dell'uraco</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -116154,7 +116147,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=431353</ExpertLink>
       <Name lang="it">Malattia veno-occlusiva polmonare e/o emangiomatosi capillare polmonare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -116507,7 +116500,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=434809</ExpertLink>
       <Name lang="it">Sindrome con capelli lanosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -118556,7 +118549,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=506060</ExpertLink>
       <Name lang="it">Tumore neuroendocrino pancreatico funzionante</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -120089,7 +120082,7 @@
     <Disorder id="8560">
       <OrphaCode>517</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=517</ExpertLink>
-      <Name lang="it">Leucemia acuta mielomonocitica</Name>
+      <Name lang="it">Leucemia mielomonocitica acuta</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -120620,7 +120613,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=450</ExpertLink>
       <Name lang="it">Eterotassia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -120678,7 +120671,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=224</ExpertLink>
       <Name lang="it">Diabete mellito neonatale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -122034,7 +122027,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=877</ExpertLink>
       <Name lang="it">Tumore endocrino</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -122147,7 +122140,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=467</ExpertLink>
       <Name lang="it">Deficit combinato non acquisito dell'ormone ipofisario</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -127660,7 +127653,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=69028</ExpertLink>
       <Name lang="it">Sindrome con brachidattilia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -129745,7 +129738,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=63454</ExpertLink>
       <Name lang="it">Distrofia dell'epitelio pigmentato della retina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -129797,7 +129790,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=63443</ExpertLink>
       <Name lang="it">Tumore epiteliale raro dello stomaco</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -130531,7 +130524,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=56970</ExpertLink>
       <Name lang="it">Malattia umana da prioni</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -131553,7 +131546,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79196</ExpertLink>
       <Name lang="it">Malattia del ciclo del gamma-glutamile</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -131635,7 +131628,7 @@
     <Disorder id="11181">
       <OrphaCode>79155</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79155</ExpertLink>
-      <Name lang="it">Encefalopatia da idrossichinureninuria</Name>
+      <Name lang="it">Idrossichinureninuria</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -132013,7 +132006,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79168</ExpertLink>
       <Name lang="it">Malattia della sintesi degli acidi biliari</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -133322,7 +133315,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=567564</ExpertLink>
       <Name lang="it">Sindrome nefrosica senza sintomi extrarenali</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -134259,7 +134252,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=77240</ExpertLink>
       <Name lang="it">Linfedema primitivo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -135484,7 +135477,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=73014</ExpertLink>
       <Name lang="it">Diarrea intrattabile dell'infanzia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -136114,7 +136107,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35705</ExpertLink>
       <Name lang="it">Malattia neurometabolica da deficit di serina</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -136322,7 +136315,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35698</ExpertLink>
       <Name lang="it">Sindrome da deplezione del DNA mitocondriale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -136693,7 +136686,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=36388</ExpertLink>
       <Name lang="it">Sindrome neurologica paraneoplastica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -137082,7 +137075,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35808</ExpertLink>
       <Name lang="it">Tumore maligno dello strome e dei cordoni sessuali dell'ovaio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -137108,7 +137101,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35807</ExpertLink>
       <Name lang="it">Tumore maligno dell'ovaio a cellule germinali</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -138307,7 +138300,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=34533</ExpertLink>
       <Name lang="it">Distrofia corneale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -140027,7 +140020,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=48471</ExpertLink>
       <Name lang="it">Lissencefalia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -144403,7 +144396,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=525677</ExpertLink>
       <Name lang="it">Malformazione congenita dell'occhio di origine genetica con glaucoma come segno maggiore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -144423,7 +144416,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=523000</ExpertLink>
       <Name lang="it">Glaucoma a esordio pediatrico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -144681,7 +144674,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=88632</ExpertLink>
       <Name lang="it">Disgenesia mesenchimale familiare del segmento anteriore dell'occhio</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -149633,7 +149626,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90970</ExpertLink>
       <Name lang="it">Lipodistrofie primitive</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -150037,7 +150030,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79373</ExpertLink>
       <Name lang="it">Sindrome da displasia ectodermica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -152824,7 +152817,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79361</ExpertLink>
       <Name lang="it">Epidermolisi bollosa ereditaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -155383,7 +155376,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=86851</ExpertLink>
       <Name lang="it">Leucemia acuta con lineage ambiguo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -155429,7 +155422,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=86846</ExpertLink>
       <Name lang="it">Leucemia mieloide acuta e sindrome mielodisplastica, terapia-dipendenti</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -158456,7 +158449,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98292</ExpertLink>
       <Name lang="it">Mastocitosi</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -158476,7 +158469,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98291</ExpertLink>
       <Name lang="it">Malattia linfoproliferativa associata a malattia immune primaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -158562,7 +158555,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97242</ExpertLink>
       <Name lang="it">Distrofia muscolare congenita</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -158614,7 +158607,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97253</ExpertLink>
       <Name lang="it">Tumore endocrino pancreatico</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -160099,7 +160092,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97593</ExpertLink>
       <Name lang="it">Pseudoipoparatiroidismo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -160279,7 +160272,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=95708</ExpertLink>
       <Name lang="it">Pubertà precoce</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -163472,7 +163465,7 @@
     <Disorder id="12316">
       <OrphaCode>93394</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93394</ExpertLink>
-      <Name lang="it">Brachidattilia, tipo A4</Name>
+      <Name lang="it">Brachidattilia tipo A4</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -166052,7 +166045,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99739</ExpertLink>
       <Name lang="it">Malattia familiare rara con cardiomiopatia ipertrofica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -169747,7 +169740,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98555</ExpertLink>
       <Name lang="it">Anoftalmia - microftalmia</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -169869,7 +169862,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98627</ExpertLink>
       <Name lang="it">Distrofia corneale posteriore</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -169898,7 +169891,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98626</ExpertLink>
       <Name lang="it">Distrofia corneale stromale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -169924,7 +169917,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98625</ExpertLink>
       <Name lang="it">Distrofia corneale superficiale</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -171181,7 +171174,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98428</ExpertLink>
       <Name lang="it">Policitemia secondaria</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -171883,7 +171876,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=102006</ExpertLink>
       <Name lang="it">Malformazione neurovascolare</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -171958,7 +171951,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=101959</ExpertLink>
       <Name lang="it">Insufficienza surrenalica primitiva cronica</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -173519,7 +173512,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100088</ExpertLink>
       <Name lang="it">Carcinoma tiroideo</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
@@ -173934,7 +173927,7 @@
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=100077</ExpertLink>
       <Name lang="it">Tumore neuroendocrino del digiuno</Name>
       <DisorderType id="36561">
-        <Name lang="it">Category</Name>
+        <Name lang="it">Categoria</Name>
       </DisorderType>
       <DisorderGroup id="36540">
         <Name lang="it">Gruppo di malattie</Name>
