commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Epidemiological data/Rare diseases natural history/it_product9_ages.xml b/Epidemiological data/Rare diseases natural history/it_product9_ages.xml
old mode 100644
new mode 100755
index 5885ed3..1ce0610
--- a/Epidemiological data/Rare diseases natural history/it_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/it_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:42:16" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:43:18" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -53238,11 +53238,11 @@
       <OrphaCode>261144</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=261144</ExpertLink>
       <Name lang="it">Sindrome da microdelezione 14q12</Name>
-      <DisorderType id="21401">
-        <Name lang="it">Sindrome malformativa</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -62734,11 +62734,11 @@
       <OrphaCode>280315</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=280315</ExpertLink>
       <Name lang="it">Pancreatite autoimmune, tipo 2</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -163546,11 +163546,11 @@
       <OrphaCode>99361</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99361</ExpertLink>
       <Name lang="it">Carcinoma midollare familiare della tiroide</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
@@ -164164,11 +164164,11 @@
       <OrphaCode>98813</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98813</ExpertLink>
       <Name lang="it">Displasia ectodermica ipoidrotica con immunodeficienza</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -167343,8 +167343,8 @@
       <OrphaCode>98791</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=98791</ExpertLink>
       <Name lang="it">Sindrome alfa talassemia-disabilità cognitiva, legata al cromosoma 16</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
@@ -168831,11 +168831,11 @@
       <OrphaCode>103919</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=103919</ExpertLink>
       <Name lang="it">Pancreatite autoimmune</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21436">
+        <Name lang="it">Gruppo di fenomi</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36540">
+        <Name lang="it">Gruppo di malattie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
