commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare diseases natural history/it_product9_ages.xml b/Epidemiological data/Rare diseases natural history/it_product9_ages.xml
index ad4d175..1be04df 100644
--- a/Epidemiological data/Rare diseases natural history/it_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/it_product9_ages.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:51:32" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6633">
+<JDBOR date="2021-12-01 04:56:04" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=166024</ExpertLink>
@@ -3958,7 +3951,7 @@
     <Disorder id="17515">
       <OrphaCode>163654</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=163654</ExpertLink>
-      <Name lang="it">Displasia spondiloepifisaria, tipo Cantu</Name>
+      <Name lang="it">Sindrome da displasia spondiloepifisaria-brachidattilia-disturbo del linguaggio</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -5351,7 +5344,7 @@
     <Disorder id="209">
       <OrphaCode>628</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=628</ExpertLink>
-      <Name lang="it">Nanismo diastrofico</Name>
+      <Name lang="it">Displasia diastrofica</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -7138,7 +7131,7 @@
     <Disorder id="294">
       <OrphaCode>308</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=308</ExpertLink>
-      <Name lang="it">Malattia di Unverricht-Lundborg</Name>
+      <Name lang="it">Epilessia mioclonica progressiva tipo 1</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -12687,11 +12680,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2346</ExpertLink>
       <Name lang="it">Sindrome angio-osteoipertrofica</Name>
-      <DisorderType id="21394">
-        <Name lang="it">Malattia</Name>
+      <DisorderType id="21436">
+        <Name lang="it">Gruppo di fenomi</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="it">Malattia</Name>
+      <DisorderGroup id="36540">
+        <Name lang="it">Gruppo di malattie</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="3">
         <AverageAgeOfOnset id="23536">
@@ -17006,7 +16999,7 @@
     <Disorder id="17947">
       <OrphaCode>171871</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171871</ExpertLink>
-      <Name lang="it">Pseudoipoaldosteronismo renale, tipo 1</Name>
+      <Name lang="it">Pseudoipoaldosteronismo renale tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -17171,7 +17164,7 @@
     <Disorder id="17948">
       <OrphaCode>171876</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=171876</ExpertLink>
-      <Name lang="it">Pseudoipoaldosteronismo generalizzato, tipo 1</Name>
+      <Name lang="it">Pseudoipoaldosteronismo generalizzato tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -20778,7 +20771,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=183678</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak con neutropenia</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di AP-3</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -21023,10 +21016,13 @@
       <DisorderGroup id="36547">
         <Name lang="it">Malattia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="it">Età adulta</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="it">Anziani</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23634">
@@ -32785,7 +32781,7 @@
     <Disorder id="1546">
       <OrphaCode>1323</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1323</ExpertLink>
-      <Name lang="it">Camptodattilia - contratture articolari - anomalie facciali e scheletriche</Name>
+      <Name lang="it">Sindrome da camptodattilia-contratture articolari-anomalie facciali e scheletriche</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -36634,7 +36630,7 @@
     <Disorder id="1899">
       <OrphaCode>1952</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=1952</ExpertLink>
-      <Name lang="it">Displasia di Pacman</Name>
+      <Name lang="it">Sindrome delle epifisi punteggiate-iperplasia osteoclastica</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -52548,7 +52544,7 @@
     <Disorder id="2314">
       <OrphaCode>2502</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2502</ExpertLink>
-      <Name lang="it">Disostosi metafisaria - ritardo mentale - sordità</Name>
+      <Name lang="it">Sindrome da disostosi metafisaria-disabilità intellettiva-sordità di conduzione</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -66860,7 +66856,7 @@
     <Disorder id="3264">
       <OrphaCode>412</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=412</ExpertLink>
-      <Name lang="it">Iperlipoproteinemia, tipo 3</Name>
+      <Name lang="it">Disbetalipoproteinemia</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -69863,7 +69859,7 @@
     <Disorder id="3134">
       <OrphaCode>2582</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=2582</ExpertLink>
-      <Name lang="it">Sindrome da mialgia ed eosinofilia correlaa al triptofano</Name>
+      <Name lang="it">Sindrome da mialgia ed eosinofilia correlata al triptofano</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -80048,7 +80044,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231512</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak senza fibrosi polmonare</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -80135,7 +80131,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=231500</ExpertLink>
-      <Name lang="it">Sindrome di Hermansky-Pudlak con fibrosi polmonare</Name>
+      <Name lang="it">Sindrome di Hermansky-Pudlak da deficit di BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -92927,7 +92923,7 @@
     <Disorder id="22057">
       <OrphaCode>352333</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=352333</ExpertLink>
-      <Name lang="it">Sindrome ittiosi congenita-deficit cognitivo-quadriplegia spastica</Name>
+      <Name lang="it">Sindrome da ittiosi congenita-disabilità intellettiva-quadriplegia spastica</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -98233,7 +98229,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=314555</ExpertLink>
-      <Name lang="it">Sindrome da displasia craniofacciale ed osteopenia</Name>
+      <Name lang="it">Sindrome da dismorfismi facciali-miopia grave-osteopenia-disabilità intellettiva-anomalie dei denti</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -102736,7 +102732,7 @@
     <Disorder id="23623">
       <OrphaCode>449432</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=449432</ExpertLink>
-      <Name lang="it">Malattia della ghiandola sottomandibolare IgG4-correlata</Name>
+      <Name lang="it">Malattia della ghiandola sottomandibolare correlata alle IgG4</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -117245,7 +117241,7 @@
     <Disorder id="8739">
       <OrphaCode>756</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=756</ExpertLink>
-      <Name lang="it">Pseudoipoaldosteronismo, tipo 1</Name>
+      <Name lang="it">Pseudoipoaldosteronismo tipo 1</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -124968,7 +124964,7 @@
     <Disorder id="25161">
       <OrphaCode>477831</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=477831</ExpertLink>
-      <Name lang="it">Sindrome iperaccrescimento scheletrico-dismorfismi craniofacciali - iperelasticità della cute - lesioni della sostanza bianca</Name>
+      <Name lang="it">Sindrome da iperaccrescimento di Kosaki</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -128439,7 +128435,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=66627</ExpertLink>
-      <Name lang="it">Sinovite villonodulare pigmentata</Name>
+      <Name lang="it">Tumore tenosinoviale a cellule giganti</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -131611,7 +131607,7 @@
     <Disorder id="11260">
       <OrphaCode>79234</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79234</ExpertLink>
-      <Name lang="it">Sindrome di Crigler-Najjar, tipo 1</Name>
+      <Name lang="it">Sindrome di Crigler-Najjar tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -131637,7 +131633,7 @@
     <Disorder id="11261">
       <OrphaCode>79235</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79235</ExpertLink>
-      <Name lang="it">Sindrome di Crigler-Najjar, tipo 2</Name>
+      <Name lang="it">Sindrome di Crigler-Najjar tipo 2</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -133588,7 +133584,7 @@
     <Disorder id="11126">
       <OrphaCode>79078</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=79078</ExpertLink>
-      <Name lang="it">Dacrioadenite e sialoadenite IgG4-correlate</Name>
+      <Name lang="it">Dacrioadenite e scialoadenite correlate alle IgG4</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -134948,7 +134944,7 @@
     <Disorder id="11071">
       <OrphaCode>75326</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=75326</ExpertLink>
-      <Name lang="it">Tortuosità delle arteriole retiniche</Name>
+      <Name lang="it">Tortuosità delle arterie retiniche</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -137809,7 +137805,7 @@
     <Disorder id="10332">
       <OrphaCode>34149</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=34149</ExpertLink>
-      <Name lang="it">Malattia cistica della midollare renale, autosomica dominante, con o senza iperuricemia</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante </Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -138365,7 +138361,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=35062</ExpertLink>
-      <Name lang="it">Infezione idiopatica disseminata da citomegalovirus</Name>
+      <Name lang="it">Infezione grave disseminata da citomegalovirus nei pazienti immunocompetenti</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -144302,11 +144298,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90308</ExpertLink>
       <Name lang="it">Sindrome di Klippel-Trénaunay</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -144331,11 +144327,11 @@
       <OrphaCode>90307</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=90307</ExpertLink>
       <Name lang="it">Sindrome di Parkes Weber</Name>
-      <DisorderType id="21450">
-        <Name lang="it">Sottotipo clinico</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="it">Sottotipo della malattia</Name>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -146108,7 +146104,7 @@
     <Disorder id="11881">
       <OrphaCode>88949</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=88949</ExpertLink>
-      <Name lang="it">Malattia cistica della midollare renale, autosomica dominante, senza iperuricemia</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante correlata a MUC1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -146128,7 +146124,7 @@
     <Disorder id="11882">
       <OrphaCode>88950</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=88950</ExpertLink>
-      <Name lang="it">Malattia cistica della midollare renale, autosomica dominante, con iperuricemia</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante correlata a UMOD</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -147121,7 +147117,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93111</ExpertLink>
-      <Name lang="it">Sindrome da cisti renali e diabete</Name>
+      <Name lang="it">Nefropatia tubulo-interstiziale autosomica dominante correlata a HNF1B</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -153827,7 +153823,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=85279</ExpertLink>
-      <Name lang="it">Ritardo mentale sindromico legato all'X da mutazioni di JARID1C</Name>
+      <Name lang="it">Disabilità intellettiva sindromica legata all’X correlata a KDM5C</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -158536,7 +158532,7 @@
     <Disorder id="30684">
       <OrphaCode>610573</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=610573</ExpertLink>
-      <Name lang="it">CLCN6-related childhood onset progressive neurodegeneration-peripheral neuropathy-syndrome</Name>
+      <Name lang="it">Sindrome da degenerazione neurologica progressiva e neuropatia periferica ad esordio infantile correlata a CLCN6</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -158556,7 +158552,7 @@
     <Disorder id="30683">
       <OrphaCode>610569</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=610569</ExpertLink>
-      <Name lang="it">KIAA1109-related early lethal-congenital brain mlaformations-arthrogryposis-syndrome</Name>
+      <Name lang="it">Sindrome da malformazioni cerebrali congenite precocemente letali con artrogriposi correlata a KIAA1109</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
@@ -160254,7 +160250,7 @@
     <Disorder id="12856">
       <OrphaCode>97229</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=97229</ExpertLink>
-      <Name lang="it">Deficit del trasportatore di riboflavina</Name>
+      <Name lang="it">Deficit del trasportatore della riboflavina</Name>
       <DisorderType id="21401">
         <Name lang="it">Sindrome malformativa</Name>
       </DisorderType>
@@ -163189,6 +163185,29 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=599373</ExpertLink>
+      <Name lang="it">Encefalopatia correlata a STXBP1</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23515">
+          <Name lang="it">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="it">Nessun dato disponibile</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93403</ExpertLink>
@@ -164560,7 +164579,7 @@
     <Disorder id="12453">
       <OrphaCode>93622</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93622</ExpertLink>
-      <Name lang="it">Malattia di Dent, tipo 1</Name>
+      <Name lang="it">Malattia di Dent tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -164586,7 +164605,7 @@
     <Disorder id="12454">
       <OrphaCode>93623</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=93623</ExpertLink>
-      <Name lang="it">Malattia di Dent, tipo 2</Name>
+      <Name lang="it">Malattia di Dent tipo 2</Name>
       <DisorderType id="21450">
         <Name lang="it">Sottotipo clinico</Name>
       </DisorderType>
@@ -165734,6 +165753,61 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592564</ExpertLink>
+      <Name lang="it">Ritardo dello sviluppo, convulsioni, disturbi del movimento correlati a GNAO1</Name>
+      <DisorderType id="21394">
+        <Name lang="it">Malattia</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="it">Infanzia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="it">Infanzia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="it">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="it">Nessun dato disponibile</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=592570</ExpertLink>
+      <Name lang="it">Sindrome da cardiopatia, anomalie delle dita, dismorfismi facciali, ritardo motorio e del linguaggio correlata a TRAF7</Name>
+      <DisorderType id="21401">
+        <Name lang="it">Sindrome malformativa</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="it">Malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="it">Prenatale</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="it">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="it">Nessun dato disponibile</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14291">
       <OrphaCode>99718</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99718</ExpertLink>
@@ -165836,6 +165910,172 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589821</ExpertLink>
+      <Name lang="it">Distrofia miotonica congenita di Steinert</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="it">Prenatale</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="it">Neonatal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="it">adolescenti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="it">Adulti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="it">prima infanzia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="it">infantile</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="it">tarda infanzia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="it">Giovane età adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589827</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio giovanile</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="it">Infanzia</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="it">adolescenti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="it">Adulti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="it">prima infanzia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="it">tarda infanzia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="it">Giovane età adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589824</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio infantile</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="it">Infanzia</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="it">Infanzia</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="it">adolescenti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="it">Adulti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="it">prima infanzia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="it">tarda infanzia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="it">Giovane età adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589833</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio tardivo</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23550">
+          <Name lang="it">Anziani</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="it">Anziani</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589830</ExpertLink>
+      <Name lang="it">Distrofia miotonica di Steinert ad esordio nell'età adulta</Name>
+      <DisorderType id="21450">
+        <Name lang="it">Sottotipo clinico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="it">Sottotipo della malattia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="it">Età adulta</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="it">Adulti</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="it">Anziani</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="it">Giovane età adulta</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=589905</ExpertLink>
@@ -175029,7 +175269,7 @@
     <Disorder id="14511">
       <OrphaCode>99938</OrphaCode>
       <ExpertLink lang="it">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=it&amp;Expert=99938</ExpertLink>
-      <Name lang="it">Malattia di Charcot-Marie-Tooth autosomica dominante, tipo 2D</Name>
+      <Name lang="it">Malattia di Charcot-Marie-Tooth autosomica dominante tipo 2D</Name>
       <DisorderType id="21394">
         <Name lang="it">Malattia</Name>
       </DisorderType>
