commit 3d5c9a9207e09e0708e502abbd0aaed3e628b8b8
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 10 21:53:34 2021 +0200

    MAY 2021

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
old mode 100644
new mode 100755
index a9046ac..07418d4
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-04-01 04:42:23" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-05-01 04:43:24" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -10265,7 +10265,7 @@
     <Disorder id="421">
       <OrphaCode>1146</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1146</ExpertLink>
-      <Name lang="nl">Digito-talaire dysmorfie</Name>
+      <Name lang="nl">Distale artrogrypose type 1</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -10427,7 +10427,7 @@
     <Disorder id="478">
       <OrphaCode>246</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=246</ExpertLink>
-      <Name lang="nl">Post-axiale acrofaciale dysostose</Name>
+      <Name lang="nl">Postaxiale acrofaciale dysostose</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -11439,7 +11439,7 @@
     <Disorder id="485">
       <OrphaCode>1880</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1880</ExpertLink>
-      <Name lang="nl">Malformatie van Ebstein</Name>
+      <Name lang="nl">Ebstein-malformatie van tricuspidalisklep</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -19856,7 +19856,7 @@
     <Disorder id="871">
       <OrphaCode>1329</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1329</ExpertLink>
-      <Name lang="nl">Compleet atrioventriculair septumdefect</Name>
+      <Name lang="nl">Totaal atrioventriculair septumdefect</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -21246,7 +21246,7 @@
     <Disorder id="927">
       <OrphaCode>526</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=526</ExpertLink>
-      <Name lang="nl">Liddlesyndroom</Name>
+      <Name lang="nl">Syndroom van Liddle</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -22020,7 +22020,7 @@
     <Disorder id="1007">
       <OrphaCode>528</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=528</ExpertLink>
-      <Name lang="nl">Congenitale lipodystrofie van Berardinelli-Seip</Name>
+      <Name lang="nl">Congenitale gegeneraliseerde lipodystrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -29774,7 +29774,7 @@
     <Disorder id="1451">
       <OrphaCode>1200</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1200</ExpertLink>
-      <Name lang="nl">Choane atresie - gehoorverlies - hartdefecten - craniofaciale dysmorfie-syndroom</Name>
+      <Name lang="nl">Syndroom van Burn-McKeown</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -35658,7 +35658,7 @@
     <Disorder id="1686">
       <OrphaCode>1519</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1519</ExpertLink>
-      <Name lang="nl">Hypertelorisme, Teebi-type</Name>
+      <Name lang="nl">SPECC1L-gerelateerd hypertelorisme-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -41872,7 +41872,7 @@
     <Disorder id="2177">
       <OrphaCode>2322</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2322</ExpertLink>
-      <Name lang="nl">Kabukisyndroom</Name>
+      <Name lang="nl">Kabuki-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -41933,7 +41933,7 @@
     <Disorder id="2176">
       <OrphaCode>2321</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2321</ExpertLink>
-      <Name lang="nl">Jung-Wolff-Back-Stahlsyndroom</Name>
+      <Name lang="nl">Syndroom van Jung</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -42115,7 +42115,7 @@
     <Disorder id="2180">
       <OrphaCode>2325</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2325</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex met doofheid, alopecie, anodontie/hypodontie en nageldystrofie</Name>
+      <Name lang="nl">Epidermolysis bullosa simplex met anodontie/hypodontie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -42317,7 +42317,7 @@
     <Disorder id="2233">
       <OrphaCode>2407</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2407</ExpertLink>
-      <Name lang="nl">LOC-syndroom</Name>
+      <Name lang="nl">Laryngo-onycho-cutaan syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -44039,7 +44039,7 @@
     <Disorder id="2276">
       <OrphaCode>2463</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2463</ExpertLink>
-      <Name lang="nl">Marfanoid habitus-autosomal recessive intellectual disability syndrome</Name>
+      <Name lang="nl">Marfanoïde habitus - autosomaal recessieve intellectuele achterstand-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -45617,7 +45617,7 @@
     <Disorder id="19687">
       <OrphaCode>251883</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251883</ExpertLink>
-      <Name lang="nl">Medullo-epithelioom van het centrale zenuwstelsel</Name>
+      <Name lang="nl">Medullo-epithelioom van centraal zenuwstelsel</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -53238,11 +53238,11 @@
       <OrphaCode>261144</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261144</ExpertLink>
       <Name lang="nl">14q12 microdeletiesyndroom</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -62734,11 +62734,11 @@
       <OrphaCode>280315</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=280315</ExpertLink>
       <Name lang="nl">Auto-immune pancreatitis type 2</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype van aandoening</Name>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -66676,7 +66676,7 @@
     <Disorder id="3298">
       <OrphaCode>832</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=832</ExpertLink>
-      <Name lang="nl">Succinyl-CoA:3-ketozuur-CoA-transferasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van succinyl-CoA:3-oxozuur-CoA-transferase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -67560,7 +67560,7 @@
     <Disorder id="3257">
       <OrphaCode>766</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=766</ExpertLink>
-      <Name lang="nl">Hemolytische anemie als gevolg van pyruvaatkinasedeficiëntie van de rode bloedcellen</Name>
+      <Name lang="nl">Hemolytische anemie door deficiëntie van pyruvaatkinase in rode bloedcellen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -67716,7 +67716,7 @@
     <Disorder id="3243">
       <OrphaCode>65</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=65</ExpertLink>
-      <Name lang="nl">Leber congenital amaurosis</Name>
+      <Name lang="nl">Congenitale amaurose van Leber</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -67959,7 +67959,7 @@
     <Disorder id="3152">
       <OrphaCode>2058</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2058</ExpertLink>
-      <Name lang="nl">Fryns-Smeets-Thirysyndroom</Name>
+      <Name lang="nl">Syndroom van Fryns-Smeets-Thiry</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -73699,7 +73699,7 @@
     <Disorder id="3413">
       <OrphaCode>344</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=344</ExpertLink>
-      <Name lang="nl">Arbovirus fever</Name>
+      <Name lang="nl">Koorts door arbovirus</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -92028,7 +92028,7 @@
     <Disorder id="22129">
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=353284</ExpertLink>
-      <Name lang="nl">Rubinstein-Taybisyndroom door EP300-haplo-insufficiëntie</Name>
+      <Name lang="nl">Syndroom van Rubinstein-Taybi door haplo-insufficiëntie van EP300</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -102016,7 +102016,7 @@
     <Disorder id="23567">
       <OrphaCode>448270</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=448270</ExpertLink>
-      <Name lang="nl">Ectopia Cordis</Name>
+      <Name lang="nl">Ectopia cordis</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -106314,7 +106314,7 @@
     <Disorder id="22919">
       <OrphaCode>404440</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=404440</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand - faciale dysmorfie-syndroom door SETD5-haploinsufficiëntie</Name>
+      <Name lang="nl">Intellectuele achterstand - faciale dysmorfie-syndroom door haplo-insufficiëntie van SETD5</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -110141,7 +110141,7 @@
     <Disorder id="23329">
       <OrphaCode>436159</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=436159</ExpertLink>
-      <Name lang="nl">Auto-immuun lymfoproliferatief syndroom door CTLA4-haploinsufficiëntie</Name>
+      <Name lang="nl">Auto-immuun lymfoproliferatief syndroom door haplo-insufficiëntie van CTLA4</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -111082,7 +111082,7 @@
     <Disorder id="23391">
       <OrphaCode>439822</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=439822</ExpertLink>
-      <Name lang="nl">PDE4D-haploinsufficiëntie-syndroom</Name>
+      <Name lang="nl">Syndroom door haplo-insufficiëntie van PDE4D</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -122273,7 +122273,7 @@
     <Disorder id="8627">
       <OrphaCode>122</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=122</ExpertLink>
-      <Name lang="nl">Birt-Hogg-Dubesyndroom</Name>
+      <Name lang="nl">Syndroom van Birt-Hogg-Dubé</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -134155,7 +134155,7 @@
     <Disorder id="28610">
       <OrphaCode>568051</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=568051</ExpertLink>
-      <Name lang="nl">GJC2-related late-onset primary lymphedema</Name>
+      <Name lang="nl">GJC2-gerelateerd primair lymfoedeem met late aanvang</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -140262,7 +140262,7 @@
     <Disorder id="10575">
       <OrphaCode>42665</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=42665</ExpertLink>
-      <Name lang="nl">Tietzsyndroom</Name>
+      <Name lang="nl">Syndroom van Tietz</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -140340,7 +140340,7 @@
     <Disorder id="28052">
       <OrphaCode>542585</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=542585</ExpertLink>
-      <Name lang="nl">Auditory neuropathy-optic atrophy syndrome</Name>
+      <Name lang="nl">Auditieve neuropathie - opticusatrofie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -147164,7 +147164,7 @@
     <Disorder id="27636">
       <OrphaCode>529980</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=529980</ExpertLink>
-      <Name lang="nl">Inflammatoire darmziekte - recurrente sinopulmonale infecties-syndroom</Name>
+      <Name lang="nl">Inflammatoire darmziekte - recidiverende sinopulmonale infecties-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -163546,11 +163546,11 @@
       <OrphaCode>99361</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99361</ExpertLink>
       <Name lang="nl">Familiaal medullair schildkliercarcinoom</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype van aandoening</Name>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
@@ -164164,11 +164164,11 @@
       <OrphaCode>98813</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98813</ExpertLink>
       <Name lang="nl">Hypohidrotische ectodermale dysplasie met immuundeficiëntie</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype van aandoening</Name>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -166771,7 +166771,7 @@
     <Disorder id="14030">
       <OrphaCode>99013</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99013</ExpertLink>
-      <Name lang="nl">Autosomaal recessieve spastische paraplegie type 7</Name>
+      <Name lang="nl">Spastische paraplegie type 7</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -167343,8 +167343,8 @@
       <OrphaCode>98791</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98791</ExpertLink>
       <Name lang="nl">Alfa-thalassemie - intellectuele achterstand-syndroom gekoppeld aan chromosoom 16</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
+      <DisorderType id="21401">
+        <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
@@ -168831,11 +168831,11 @@
       <OrphaCode>103919</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=103919</ExpertLink>
       <Name lang="nl">Auto-immune pancreatitis</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
+      <DisorderType id="21436">
+        <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36540">
+        <Name lang="nl">Groep van aandoeningen</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
