commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
index cafd2f3..823c2ca 100644
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:59:57" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:57:03" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6641">
+  <DisorderList count="6636">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -2475,15 +2475,24 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="nl">Volwassenheid</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23529">
           <Name lang="nl">Kindertijd</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
+      <AverageAgeOfDeathList count="2">
         <AverageAgeOfDeath id="23627">
           <Name lang="nl">Volwassene</Name>
         </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
         <TypeOfInheritance id="23445">
@@ -21127,7 +21136,7 @@
     <Disorder id="946">
       <OrphaCode>3027</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3027</ExpertLink>
-      <Name lang="nl">Caudale regressie-sequentie</Name>
+      <Name lang="nl">Caudale regressie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -21135,14 +21144,26 @@
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
+        <AverageAgeOfOnset id="23508">
+          <Name lang="nl">Antenataal</Name>
         </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="nl">Neonataal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="4">
+        <AverageAgeOfDeath id="23648">
+          <Name lang="nl">Elke leeftijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23578">
+          <Name lang="nl">Embyrofoetaal</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="nl">Infantiel</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23585">
+          <Name lang="nl">Doodgeboorte</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
         <TypeOfInheritance id="23424">
@@ -21675,7 +21696,10 @@
           <Name lang="nl">Normale levensverwachting</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
@@ -24594,38 +24618,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1265">
-      <OrphaCode>946</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=946</ExpertLink>
-      <Name lang="nl">Acrocefalosyndactylie</Name>
-      <DisorderType id="21436">
-        <Name lang="nl">Group of phenomes</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="nl">Groep van aandoeningen</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23508">
-          <Name lang="nl">Antenataal</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="nl">Elke leeftijd</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="nl">Niet toepasbaar</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="1278">
       <OrphaCode>957</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=957</ExpertLink>
@@ -27006,7 +26998,7 @@
     <Disorder id="16882">
       <OrphaCode>139390</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139390</ExpertLink>
-      <Name lang="nl">Geïsoleerde craniosynostose</Name>
+      <Name lang="nl">Niet-syndromale craniosynostose</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -34850,32 +34842,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1697">
-      <OrphaCode>1533</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1533</ExpertLink>
-      <Name lang="nl">Craniosynostose - fibulaire aplasie-syndroom</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23529">
-          <Name lang="nl">Kindertijd</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="nl">No data available</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="16992">
       <OrphaCode>140874</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=140874</ExpertLink>
@@ -35887,7 +35853,7 @@
     <Disorder id="1684">
       <OrphaCode>1517</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1517</ExpertLink>
-      <Name lang="nl">Hypertrichotische osteochondrodysplasie, Cantu-type</Name>
+      <Name lang="nl">Syndroom van Cantú</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -36075,7 +36041,7 @@
     <Disorder id="1683">
       <OrphaCode>1516</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1516</ExpertLink>
-      <Name lang="nl">Craniofaciale dyssynostose</Name>
+      <Name lang="nl">Niet-syndromale bilambdoïde en sagittale craniosynostose</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -44806,7 +44772,7 @@
     <Disorder id="2078">
       <OrphaCode>2196</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2196</ExpertLink>
-      <Name lang="nl">Familiale primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose met ernstige oogbetrokkenheid</Name>
+      <Name lang="nl">Primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose met ernstige oculaire betrokkenheid</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -50502,7 +50468,7 @@
     <Disorder id="2497">
       <OrphaCode>661</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=661</ExpertLink>
-      <Name lang="nl">Syndroom van Ondine</Name>
+      <Name lang="nl">Congenitale centrale hypoventilatie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -60955,7 +60921,7 @@
     <Disorder id="2983">
       <OrphaCode>3366</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3366</ExpertLink>
-      <Name lang="nl">Geïsoleerde trigonocefalie</Name>
+      <Name lang="nl">Niet-syndromale metopische craniosynostose</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -64957,10 +64923,7 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23417">
           <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
@@ -67789,7 +67752,7 @@
     <Disorder id="3249">
       <OrphaCode>478</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=478</ExpertLink>
-      <Name lang="nl">Kallmannsyndroom</Name>
+      <Name lang="nl">Syndroom van Kallmann</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -71964,7 +71927,7 @@
     <Disorder id="18746">
       <OrphaCode>213528</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=213528</ExpertLink>
-      <Name lang="nl">Zeldzaam adenocarcinoom van de borst</Name>
+      <Name lang="nl">Zeldzaam adenocarcinoom van borst</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -83297,7 +83260,7 @@
     <Disorder id="19334">
       <OrphaCode>240103</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=240103</ExpertLink>
-      <Name lang="nl">Progressieve supranucleaire paralyse - corticobasaal parkinsonisme</Name>
+      <Name lang="nl">Progressieve supranucleaire paralyse - corticobasaal syndroom</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -83323,7 +83286,7 @@
     <Disorder id="19333">
       <OrphaCode>240094</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=240094</ExpertLink>
-      <Name lang="nl">Progressieve supranucleaire paralyse - zuivere akinesie met bevroren gang-syndroom</Name>
+      <Name lang="nl">Progressieve supranucleaire paralyse - zuivere akinesie met gangblokkade-syndroom</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -84477,26 +84440,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="21931">
-      <OrphaCode>329341</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329341</ExpertLink>
-      <Name lang="nl">Limbische encefalitis met DPP6-antilichamen</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="nl">Volwassenheid</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="21911">
       <OrphaCode>329228</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329228</ExpertLink>
@@ -85479,7 +85422,7 @@
     <Disorder id="21781">
       <OrphaCode>324525</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=324525</ExpertLink>
-      <Name lang="nl">Hypertrofische cardiomyopathie en renale tubulaire ziekte door mitochondriale DNA-mutatie</Name>
+      <Name lang="nl">Hypertrofische cardiomyopathie met nieranomalieën door mutatie van mitochondriaal DNA</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -94981,7 +94924,7 @@
     <Disorder id="5533">
       <OrphaCode>890</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=890</ExpertLink>
-      <Name lang="nl">Hepatische veno-occlusieve ziekte</Name>
+      <Name lang="nl">Veno-occlusieve ziekte van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -98103,7 +98046,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314555</ExpertLink>
-      <Name lang="nl">Faciale dysmorfie - ernstige myopie - osteopenie - intellectuele achterstand - dentale anomalieën-syndroom</Name>
+      <Name lang="nl">Faciale dysmorfie - oculaire anomalieën - osteopenie - intellectuele achterstand - dentale anomalieën-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -100260,7 +100203,7 @@
     <Disorder id="21222">
       <OrphaCode>306516</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=306516</ExpertLink>
-      <Name lang="nl">Familiale primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose</Name>
+      <Name lang="nl">Primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -103127,26 +103070,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="23782">
-      <OrphaCode>459353</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=459353</ExpertLink>
-      <Name lang="nl">C1-inhibitor-deficiëntie</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="23767">
       <OrphaCode>459033</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=459033</ExpertLink>
@@ -104330,7 +104253,7 @@
     <Disorder id="7024">
       <OrphaCode>629</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=629</ExpertLink>
-      <Name lang="nl">Kleine gestalte als gevolg van groeihormoonkwaliteitanomalie</Name>
+      <Name lang="nl">Kleine gestalte door kwalitatieve anomalie van groeihormoon</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -122485,26 +122408,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="25936">
-      <OrphaCode>498700</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=498700</ExpertLink>
-      <Name lang="nl">Limbische encefalitis met neurexine-3 antilichamen</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="nl">Volwassenheid</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="8594">
       <OrphaCode>874</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=874</ExpertLink>
@@ -126018,7 +125921,7 @@
     <Disorder id="9284">
       <OrphaCode>31043</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=31043</ExpertLink>
-      <Name lang="nl">Familiale primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose zonder ernstige oculaire betrokkenheid</Name>
+      <Name lang="nl">Primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose zonder ernstige oculaire betrokkenheid</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -126296,7 +126199,7 @@
     <Disorder id="10993">
       <OrphaCode>71271</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=71271</ExpertLink>
-      <Name lang="nl">Split hand-split foot-deafness syndrome</Name>
+      <Name lang="nl">Gespleten hand - gespleten voet - doofheid-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -130354,7 +130257,7 @@
     <Disorder id="28399">
       <OrphaCode>564178</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=564178</ExpertLink>
-      <Name lang="nl">Primaire hypomagnesiëmie met refractaire insulten en intellectuele achterstand</Name>
+      <Name lang="nl">Primaire hypomagnesiëmie - refractaire insulten - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -131926,7 +131829,7 @@
     <Disorder id="11175">
       <OrphaCode>79149</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79149</ExpertLink>
-      <Name lang="nl">Dermo-chondro-corneale dystrofie</Name>
+      <Name lang="nl">Dermochondrocorneale dystrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -132151,7 +132054,7 @@
     <Disorder id="11185">
       <OrphaCode>79159</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79159</ExpertLink>
-      <Name lang="nl">Isobutyryl-CoA-dehydrogenasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van isobutyryl-CoA-dehydrogenase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -132697,7 +132600,7 @@
     <Disorder id="11166">
       <OrphaCode>79140</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79140</ExpertLink>
-      <Name lang="nl">Cutaan neuroendocrien carcinoom</Name>
+      <Name lang="nl">Cutaan neuro-endocrien carcinoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -133509,7 +133412,7 @@
     <Disorder id="11125">
       <OrphaCode>79076</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79076</ExpertLink>
-      <Name lang="nl">Juveniele polyposis bij jonge kinderen</Name>
+      <Name lang="nl">Juveniele polyposis beginnend in de zuigelingentijd</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -135495,7 +135398,7 @@
     <Disorder id="11036">
       <OrphaCode>73224</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=73224</ExpertLink>
-      <Name lang="nl">Tubulaire nefropathie - cardiomyopathie-syndroom</Name>
+      <Name lang="nl">Tubulopathie van nier - gedilateerde cardiomyopathie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -136666,7 +136569,7 @@
     <Disorder id="10369">
       <OrphaCode>35099</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35099</ExpertLink>
-      <Name lang="nl">Geïsoleerde brachycefalie</Name>
+      <Name lang="nl">Niet-syndromale bicoronale craniosynostose</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -138144,7 +138047,7 @@
     <Disorder id="10367">
       <OrphaCode>35093</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35093</ExpertLink>
-      <Name lang="nl">Geïsoleerde scafocefalie</Name>
+      <Name lang="nl">Niet-syndromale sagittale craniosynostose</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -143732,7 +143635,7 @@
     <Disorder id="11913">
       <OrphaCode>89938</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89938</ExpertLink>
-      <Name lang="nl">Infantiel syndroom van Bartter met sensorineurale doofheid</Name>
+      <Name lang="nl">Syndroom van Bartter type 4</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -144718,7 +144621,7 @@
     <Disorder id="11817">
       <OrphaCode>88639</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=88639</ExpertLink>
-      <Name lang="nl">Neurodegeneratie als gevolg van 3-hydroxyisobutyryl-CoA-hydrolasedeficiëntie</Name>
+      <Name lang="nl">Neurodegeneratie door deficiëntie van 3-hydroxyisobutyryl-CoA-hydrolase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -147924,7 +147827,7 @@
     <Disorder id="12252">
       <OrphaCode>93315</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93315</ExpertLink>
-      <Name lang="nl">Spondylometafysaire dysplasie, 'hoekfractuur'-type</Name>
+      <Name lang="nl">Spondylometafysaire dysplasie, hoekfractuur-type</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -148239,7 +148142,7 @@
     <Disorder id="12095">
       <OrphaCode>90791</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90791</ExpertLink>
-      <Name lang="nl">Congenitale adrenale hyperplasie door 3-bèta-hydroxysteroïdedehydrogenasedeficiëntie</Name>
+      <Name lang="nl">Congenitale adrenale hyperplasie door deficiëntie van 3-bèta-hydroxysteroïdedehydrogenase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149201,7 +149104,7 @@
     <Disorder id="12153">
       <OrphaCode>91494</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91494</ExpertLink>
-      <Name lang="nl">Maculacoloboom - gespleten verhemelte - hallux valgus-syndroom</Name>
+      <Name lang="nl">Maculacoloboom - gespleten gehemelte - hallux valgus-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -149673,7 +149576,7 @@
     <Disorder id="12098">
       <OrphaCode>90795</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90795</ExpertLink>
-      <Name lang="nl">Congenitale adrenale hyperplasie door 11-bèta-hydroxylasedeficiëntie</Name>
+      <Name lang="nl">Congenitale adrenale hyperplasie door deficiëntie van 11-bèta-hydroxylase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -153107,7 +153010,7 @@
     <Disorder id="11649">
       <OrphaCode>85191</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85191</ExpertLink>
-      <Name lang="nl">Singleton-Mertendysplasie</Name>
+      <Name lang="nl">Dysplasie, Singleton-Merten-type</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -153662,7 +153565,7 @@
     <Disorder id="11670">
       <OrphaCode>85280</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85280</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand, X-gebonden - cubitus valgus - dysmorfie</Name>
+      <Name lang="nl">X-gebonden intellectuele achterstand - cubitus valgus - dysmorfie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -155107,7 +155010,7 @@
     <Disorder id="11741">
       <OrphaCode>86821</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=86821</ExpertLink>
-      <Name lang="nl">Lissencefalie type 3 - familiale foetale akinesiesequentie-syndroom</Name>
+      <Name lang="nl">Lissencefalie type 3 - familiale foetale akinesie sequentie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -155133,7 +155036,7 @@
     <Disorder id="11742">
       <OrphaCode>86822</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=86822</ExpertLink>
-      <Name lang="nl">Lissencefalie type 3 - metacarpaalbotdysplasie</Name>
+      <Name lang="nl">Lissencefalie type 3 - dysplasie van metacarpus-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -157732,7 +157635,7 @@
     <Disorder id="11629">
       <OrphaCode>85162</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85162</ExpertLink>
-      <Name lang="nl">Sensorische en motorische neuronopathie met faciale aanvang</Name>
+      <Name lang="nl">Sensorische en motorische neuropathie met faciale aanvang</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -157932,7 +157835,7 @@
     <Disorder id="11619">
       <OrphaCode>84142</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=84142</ExpertLink>
-      <Name lang="nl">Syndroom van Isaac</Name>
+      <Name lang="nl">Syndroom van Isaacs</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -158337,7 +158240,7 @@
     <Disorder id="11632">
       <OrphaCode>85165</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85165</ExpertLink>
-      <Name lang="nl">Ernstige achondroplasie - ontwikkelingsachterstand - acanthosis nigricans</Name>
+      <Name lang="nl">Ernstige achondroplasie - ontwikkelingsachterstand - acanthosis nigricans-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -160608,7 +160511,7 @@
     <Disorder id="12654">
       <OrphaCode>95699</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95699</ExpertLink>
-      <Name lang="nl">Congenitale hyperplasie van bijnier door deficiëntie van cytochroom P450-oxidoreductase</Name>
+      <Name lang="nl">Congenitale adrenale hyperplasie door deficiëntie van cytochroom P450-oxidoreductase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -164924,7 +164827,7 @@
     <Disorder id="12438">
       <OrphaCode>93605</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93605</ExpertLink>
-      <Name lang="nl">Klassiek syndroom van Bartter</Name>
+      <Name lang="nl">Syndroom van Bartter type 3</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -171803,7 +171706,7 @@
     <Disorder id="28672">
       <OrphaCode>570371</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=570371</ExpertLink>
-      <Name lang="nl">Transiënt prenataal syndroom van Bartter</Name>
+      <Name lang="nl">Syndroom van Bartter type 5</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -177089,7 +176992,7 @@
     <Disorder id="31415">
       <OrphaCode>619363</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=619363</ExpertLink>
-      <Name lang="nl">Neonatal onset-severe multisystem auto-inflammatory disease-increased IL-18</Name>
+      <Name lang="nl">Neonatale ernstige multisystemische auto-inflammatoire aandoening met verhoogd IL18</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -177109,7 +177012,7 @@
     <Disorder id="31405">
       <OrphaCode>619233</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=619233</ExpertLink>
-      <Name lang="nl">Hereditary persistence of fetal hemoglobin-intellectual disability syndrome</Name>
+      <Name lang="nl">Hereditaire persistentie van foetaal hemoglobine - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
