commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
old mode 100644
new mode 100755
index ee74cff..0e12554
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:39:59" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:40:57" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6586">
+  <DisorderList count="6584">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -877,7 +877,7 @@
     <Disorder id="18">
       <OrphaCode>369</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=369</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte type 6</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door hepatische glycogeenfosforylasedeficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -3328,7 +3328,7 @@
     <Disorder id="120">
       <OrphaCode>908</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=908</ExpertLink>
-      <Name lang="nl">Fragiele X-syndroom</Name>
+      <Name lang="nl">Fragiele-X-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -5110,7 +5110,7 @@
     <Disorder id="192">
       <OrphaCode>640</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=640</ExpertLink>
-      <Name lang="nl">Erfelijke neuropathie met vatbaarheid voor drukverlammingen</Name>
+      <Name lang="nl">Erfelijke drukneuropathie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -17814,7 +17814,7 @@
     <Disorder id="18404">
       <OrphaCode>199337</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=199337</ExpertLink>
-      <Name lang="nl">Pancreasinsufficiëntie - anemie - hyperostose</Name>
+      <Name lang="nl">Pancreasinsufficiëntie - anemie - hyperostose-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -32014,7 +32014,7 @@
     <Disorder id="17147">
       <OrphaCode>157801</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=157801</ExpertLink>
-      <Name lang="nl">Meso-axiale synostotische syndactylie met falangeale reductie</Name>
+      <Name lang="nl">Mesoaxiale synostotische syndactylie met falangeale reductie</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -42666,7 +42666,7 @@
     <Disorder id="2216">
       <OrphaCode>2379</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2379</ExpertLink>
-      <Name lang="nl">Ziekte van Parkinson met vroege aanvang - intellectuele achterstand</Name>
+      <Name lang="nl">Ziekte van Parkinson met vroege aanvang - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -43888,7 +43888,7 @@
     <Disorder id="2289">
       <OrphaCode>2476</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2476</ExpertLink>
-      <Name lang="nl">Dysrafie - gespleten lip/verhemelte - ledemaatreductiedefecten-syndroom</Name>
+      <Name lang="nl">Dysrafie - gespleten lip/verhemelte - reductiedefect van ledemaat-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -51787,7 +51787,7 @@
     <Disorder id="2549">
       <OrphaCode>675</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=675</ExpertLink>
-      <Name lang="nl">Ringvormige pancreas</Name>
+      <Name lang="nl">Annulaire pancreas</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -54188,7 +54188,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="19874">
@@ -58067,7 +58070,7 @@
     <Disorder id="2579">
       <OrphaCode>2840</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2840</ExpertLink>
-      <Name lang="nl">Bekkendysplasie - artrogrypose van de onderste ledematen-syndroom</Name>
+      <Name lang="nl">Bekkendysplasie - artrogrypose van onderste ledematen-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -58281,7 +58284,7 @@
     <Disorder id="2590">
       <OrphaCode>708</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=708</ExpertLink>
-      <Name lang="nl">Petersanomalie</Name>
+      <Name lang="nl">Anomalie van Peters</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -59299,7 +59302,7 @@
     <Disorder id="2656">
       <OrphaCode>1848</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1848</ExpertLink>
-      <Name lang="nl">Bilaterale renale agenesie</Name>
+      <Name lang="nl">Agenesie van nier, bilateraal</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -59976,7 +59979,7 @@
     <Disorder id="2630">
       <OrphaCode>2911</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2911</ExpertLink>
-      <Name lang="nl">Polandsyndroom</Name>
+      <Name lang="nl">Syndroom van Poland</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -60014,7 +60017,7 @@
     <Disorder id="20103">
       <OrphaCode>264580</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=264580</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte type 9A</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door hepatische fosforylasekinasedeficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -66386,7 +66389,7 @@
     <Disorder id="3271">
       <OrphaCode>2089</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2089</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte als gevolg van hepatische glycogeensynthasedeficiëntie</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door hepatische glycogeensynthasedeficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -68042,7 +68045,7 @@
     <Disorder id="3155">
       <OrphaCode>2538</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2538</ExpertLink>
-      <Name lang="nl">Microgastrie - ledemaatreductiedefect</Name>
+      <Name lang="nl">Microgastrie - reductiedefect van ledemaat-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -70337,7 +70340,7 @@
     <Disorder id="18698">
       <OrphaCode>210584</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=210584</ExpertLink>
-      <Name lang="nl">Spindelcelhemangioom</Name>
+      <Name lang="nl">Spoelcelhemangioom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -70744,7 +70747,7 @@
     <Disorder id="3549">
       <OrphaCode>655</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=655</ExpertLink>
-      <Name lang="nl">Nefronoftisis</Name>
+      <Name lang="nl">Nefronoftise</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -73068,7 +73071,7 @@
     <Disorder id="3402">
       <OrphaCode>292</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=292</ExpertLink>
-      <Name lang="nl">Congenitale enterovirusinfectie</Name>
+      <Name lang="nl">Congenitale infectie door enterovirus</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -73259,7 +73262,7 @@
     <Disorder id="18822">
       <OrphaCode>217074</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217074</ExpertLink>
-      <Name lang="nl">Zeldzaam carcinoom van de pancreas</Name>
+      <Name lang="nl">Zeldzaam carcinoom van pancreas</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -78009,7 +78012,7 @@
     <Disorder id="3708">
       <OrphaCode>1333</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1333</ExpertLink>
-      <Name lang="nl">Familiaal pancreascarcinoom</Name>
+      <Name lang="nl">Familiaal carcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -83614,7 +83617,7 @@
     <Disorder id="21927">
       <OrphaCode>329324</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329324</ExpertLink>
-      <Name lang="nl">Invers Klippel-Trenaunaysyndroom</Name>
+      <Name lang="nl">Invers syndroom van Klippel-Trenaunay</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92089,8 +92092,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=353284</ExpertLink>
       <Name lang="nl">Rubinstein-Taybisyndroom door EP300-haplo-insufficiëntie</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="nl">Subtype of disorder</Name>
@@ -92819,32 +92822,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20880">
-      <OrphaCode>293848</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=293848</ExpertLink>
-      <Name lang="nl">Frontotemporale dementie, rechter temporale atrofie variant</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Disorder</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23543">
-          <Name lang="nl">Volwassenheid</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23634">
-          <Name lang="nl">Op leeftijd</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23480">
-          <Name lang="nl">Unknown</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20879">
       <OrphaCode>293843</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=293843</ExpertLink>
@@ -96595,26 +96572,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20606">
-      <OrphaCode>284408</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284408</ExpertLink>
-      <Name lang="nl">Glycerolkinasedeficiëntie, infantiele vorm</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype of disorder</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20607">
       <OrphaCode>284411</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284411</ExpertLink>
@@ -96771,7 +96728,7 @@
     <Disorder id="21470">
       <OrphaCode>314022</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314022</ExpertLink>
-      <Name lang="nl">Gastrisch adenocarcinoom en proximale polyposis van de maag</Name>
+      <Name lang="nl">Gastrisch adenocarcinoom en proximale polyposis van maag</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -99264,7 +99221,7 @@
     <Disorder id="21149">
       <OrphaCode>300878</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300878</ExpertLink>
-      <Name lang="nl">Haarcelleukemie variant</Name>
+      <Name lang="nl">Variant van haarcelleukemie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -100489,7 +100446,7 @@
     <Disorder id="24048">
       <OrphaCode>464756</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=464756</ExpertLink>
-      <Name lang="nl">Familiale neuro-endocriene tumor van de maag type 1</Name>
+      <Name lang="nl">Familiale neuro-endocriene tumor van maag type 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -106769,7 +106726,7 @@
     <Disorder id="23016">
       <OrphaCode>411527</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=411527</ExpertLink>
-      <Name lang="nl">Centrale retinale aderocclusie</Name>
+      <Name lang="nl">Veneuze stamocclusie van retina</Name>
       <DisorderType id="21429">
         <Name lang="nl">Particulaire klinische situatie binnen een ziekte of syndroom</Name>
       </DisorderType>
@@ -110464,7 +110421,7 @@
     <Disorder id="23332">
       <OrphaCode>436174</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=436174</ExpertLink>
-      <Name lang="nl">Cataract - groeihormoondeficiëntie - sensorische neuropathie - sensorineuraal gehoorverlies - skeletdysplasie-syndroom</Name>
+      <Name lang="nl">Cataract - deficiëntie van groeihormoon - sensorische neuropathie - sensorineuraal gehoorverlies - skeletdysplasie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -114940,7 +114897,7 @@
     <Disorder id="23193">
       <OrphaCode>424065</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424065</ExpertLink>
-      <Name lang="nl">Solide pseudopapillair carcinoom van de pancreas</Name>
+      <Name lang="nl">Solide pseudopapillair carcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -114963,7 +114920,7 @@
     <Disorder id="23192">
       <OrphaCode>424058</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424058</ExpertLink>
-      <Name lang="nl">Intraductaal papillair mucineus carcinoom van de pancreas</Name>
+      <Name lang="nl">Intraductaal papillair mucineus carcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115194,7 +115151,7 @@
     <Disorder id="23189">
       <OrphaCode>424039</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424039</ExpertLink>
-      <Name lang="nl">Plaveiselcelcarcinoom van de pancreas</Name>
+      <Name lang="nl">Plaveiselcelcarcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115217,7 +115174,7 @@
     <Disorder id="23191">
       <OrphaCode>424053</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424053</ExpertLink>
-      <Name lang="nl">Mucineus cystadenocarcinoom van de pancreas</Name>
+      <Name lang="nl">Mucineus cystadenocarcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115240,7 +115197,7 @@
     <Disorder id="23190">
       <OrphaCode>424046</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424046</ExpertLink>
-      <Name lang="nl">Acinaire celcarcinoom van de pancreas</Name>
+      <Name lang="nl">Acinuscelcarcinoom van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115451,7 +115408,7 @@
     <Disorder id="23202">
       <OrphaCode>424943</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424943</ExpertLink>
-      <Name lang="nl">Adenocarcinoom van de lever en de intrahepatische galwegen</Name>
+      <Name lang="nl">Adenocarcinoom van lever en intrahepatische galwegen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115474,7 +115431,7 @@
     <Disorder id="23203">
       <OrphaCode>424970</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424970</ExpertLink>
-      <Name lang="nl">Ongedifferentieerd carcinoom van de lever en intrahepatische galwegen</Name>
+      <Name lang="nl">Ongedifferentieerd carcinoom van lever en intrahepatische galwegen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -116767,7 +116724,7 @@
     <Disorder id="8758">
       <OrphaCode>26106</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=26106</ExpertLink>
-      <Name lang="nl">Erfelijke diffuse maagkanker</Name>
+      <Name lang="nl">Hereditaire diffuse maagkanker</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -117018,7 +116975,7 @@
     <Disorder id="8707">
       <OrphaCode>674</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=674</ExpertLink>
-      <Name lang="nl">Bijkomstige pancreas</Name>
+      <Name lang="nl">Accessoire pancreas</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -117981,7 +117938,7 @@
     <Disorder id="26434">
       <OrphaCode>506060</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=506060</ExpertLink>
-      <Name lang="nl">Functionerende neuro-endocriene tumor van de pancreas</Name>
+      <Name lang="nl">Functionele neuro-endocriene tumor van pancreas</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -118004,7 +117961,7 @@
     <Disorder id="26435">
       <OrphaCode>506075</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=506075</ExpertLink>
-      <Name lang="nl">Non-functioning neuroendocrine tumor of pancreas</Name>
+      <Name lang="nl">Niet-functionele neuro-endocriene tumor van pancreas</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -125206,7 +125163,7 @@
     <Disorder id="9291">
       <OrphaCode>31202</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=31202</ExpertLink>
-      <Name lang="nl">Melioidose</Name>
+      <Name lang="nl">Melioïdose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -129064,7 +129021,7 @@
     <Disorder id="10821">
       <OrphaCode>63275</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63275</ExpertLink>
-      <Name lang="nl">Pemfigoïde gestationis</Name>
+      <Name lang="nl">Pemphigoid gestationis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -129139,7 +129096,7 @@
     <Disorder id="10824">
       <OrphaCode>63443</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63443</ExpertLink>
-      <Name lang="nl">Zeldzame epitheliale tumor van de maag</Name>
+      <Name lang="nl">Zeldzame epitheliale tumor van maag</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -129272,7 +129229,7 @@
     <Disorder id="10804">
       <OrphaCode>60039</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=60039</ExpertLink>
-      <Name lang="nl">Pudendale neuralgie</Name>
+      <Name lang="nl">Pudendusneuralgie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -133190,7 +133147,7 @@
     <Disorder id="28552">
       <OrphaCode>566841</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=566841</ExpertLink>
-      <Name lang="nl">Adenomatose van de lever</Name>
+      <Name lang="nl">Adenomatose van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -136441,7 +136398,7 @@
     <Disorder id="10415">
       <OrphaCode>36273</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=36273</ExpertLink>
-      <Name lang="nl">Gastrische linitis plastica</Name>
+      <Name lang="nl">Linitis plastica van maag</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -141372,7 +141329,7 @@
     <Disorder id="10615">
       <OrphaCode>48372</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=48372</ExpertLink>
-      <Name lang="nl">Nodulaire regeneratieve hyperplasie van de lever</Name>
+      <Name lang="nl">Nodulaire regeneratieve hyperplasie van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -142078,7 +142035,7 @@
     <Disorder id="28138">
       <OrphaCode>556955</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=556955</ExpertLink>
-      <Name lang="nl">Agenesie van de pancreas - holoprosencefalie-syndroom</Name>
+      <Name lang="nl">Agenesie van pancreas - holoprosencefalie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -142519,7 +142476,7 @@
     <Disorder id="11921">
       <OrphaCode>90003</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90003</ExpertLink>
-      <Name lang="nl">Inflammatoire pseudotumor van de lever</Name>
+      <Name lang="nl">Inflammatoire pseudotumor van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -145772,7 +145729,7 @@
     <Disorder id="12164">
       <OrphaCode>93101</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93101</ExpertLink>
-      <Name lang="nl">Renale hypoplasie</Name>
+      <Name lang="nl">Hypoplasie van nier</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -145902,7 +145859,7 @@
     <Disorder id="12163">
       <OrphaCode>93100</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93100</ExpertLink>
-      <Name lang="nl">Unilaterale renale agenesie</Name>
+      <Name lang="nl">Agenesie van nier, unilateraal</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -148939,7 +148896,7 @@
     <Disorder id="12123">
       <OrphaCode>91349</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91349</ExpertLink>
-      <Name lang="nl">Niet-functionerend hypofyseadenoom</Name>
+      <Name lang="nl">Niet-functioneel hypofyseadenoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -148988,7 +148945,7 @@
     <Disorder id="12126">
       <OrphaCode>91352</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91352</ExpertLink>
-      <Name lang="nl">Germinoom van het centrale zenuwstelsel</Name>
+      <Name lang="nl">Germinoom van centraal zenuwstelsel</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -156665,7 +156622,7 @@
     <Disorder id="11602">
       <OrphaCode>83642</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=83642</ExpertLink>
-      <Name lang="nl">Microcytaire anemie met ijzerstapeling in de lever</Name>
+      <Name lang="nl">Microcytaire anemie met ijzerstapeling in lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -157634,7 +157591,7 @@
     <Disorder id="12870">
       <OrphaCode>97253</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97253</ExpertLink>
-      <Name lang="nl">Neuro-endocriene tumor van de pancreas</Name>
+      <Name lang="nl">Neuro-endocriene tumor van pancreas</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -157946,7 +157903,7 @@
     <Disorder id="12894">
       <OrphaCode>97338</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97338</ExpertLink>
-      <Name lang="nl">Melanoom van de weke delen</Name>
+      <Name lang="nl">Melanoom van weke delen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -171094,7 +171051,7 @@
     <Disorder id="14657">
       <OrphaCode>100085</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100085</ExpertLink>
-      <Name lang="nl">Primair neuro-endocrien carcinoom van de lever</Name>
+      <Name lang="nl">Primair neuro-endocrien carcinoom van lever</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -171414,7 +171371,7 @@
     <Disorder id="14647">
       <OrphaCode>100075</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100075</ExpertLink>
-      <Name lang="nl">Neuro-endocriene tumor van de maag</Name>
+      <Name lang="nl">Neuro-endocriene tumor van maag</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
