commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
index 05ac003..cafd2f3 100644
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:55:14" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 04:59:57" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6654">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -5001,7 +5001,7 @@
     <Disorder id="203">
       <OrphaCode>752</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=752</ExpertLink>
-      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis als gevolg van 17-bèta-hydroxysteroïde dehydrogenase 3-deficiëntie</Name>
+      <Name lang="nl">46,XY-geslachtsontwikkelingsstoornis door deficiëntie van 17-bèta-hydroxysteroïde-dehydrogenase 3</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -6061,7 +6061,7 @@
     <Disorder id="17873">
       <OrphaCode>169805</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=169805</ExpertLink>
-      <Name lang="nl">Matig ernstige hemofilie A</Name>
+      <Name lang="nl">Matige hemofilie A</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -6635,7 +6635,7 @@
     <Disorder id="17870">
       <OrphaCode>169796</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=169796</ExpertLink>
-      <Name lang="nl">Matig ernstige hemofilie B</Name>
+      <Name lang="nl">Matige hemofilie B</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -7761,7 +7761,7 @@
     <Disorder id="324">
       <OrphaCode>753</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=753</ExpertLink>
-      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis door 5-alfa-reductase 2-deficiëntie</Name>
+      <Name lang="nl">46,XY-geslachtsontwikkelingsstoornis door deficiëntie van 5-alfa-reductase 2</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -9975,7 +9975,7 @@
     <Disorder id="17771">
       <OrphaCode>168558</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=168558</ExpertLink>
-      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis - adrenale insufficiëntie door CYP11A1-deficiëntie</Name>
+      <Name lang="nl">46,XY-geslachtsontwikkelingsstoornis - adrenale insufficiëntie door CYP11A1-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -12666,32 +12666,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="521">
-      <OrphaCode>2343</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2343</ExpertLink>
-      <Name lang="nl">Geïsoleerd klaverbladschedelsyndroom</Name>
-      <DisorderType id="21415">
-        <Name lang="nl">Morfologische anomalie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23599">
-          <Name lang="nl">Vroege kindertijd</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="nl">Niet toepasbaar</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="523">
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2346</ExpertLink>
@@ -17529,32 +17503,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="727">
-      <OrphaCode>1428</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1428</ExpertLink>
-      <Name lang="nl">Familiale chondromalacia patellae</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23536">
-          <Name lang="nl">Puber</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="nl">Elke leeftijd</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23473">
-          <Name lang="nl">Y-gelinkt</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="820">
       <OrphaCode>3303</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3303</ExpertLink>
@@ -25814,7 +25762,7 @@
     <Disorder id="1404">
       <OrphaCode>1125</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1125</ExpertLink>
-      <Name lang="nl">Oculaire motorische apraxie, Cogan-type</Name>
+      <Name lang="nl">Oculomotorische apraxie, Cogan-type</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -26705,55 +26653,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1284">
-      <OrphaCode>965</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=965</ExpertLink>
-      <Name lang="nl">Acromegaloïd aangezichtssyndroom</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="1285">
-      <OrphaCode>966</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=966</ExpertLink>
-      <Name lang="nl">Hypertrichose - acromegaloïd aangezicht-syndroom</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="16888">
       <OrphaCode>139411</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139411</ExpertLink>
@@ -27179,7 +27078,7 @@
     <Disorder id="16886">
       <OrphaCode>139402</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139402</ExpertLink>
-      <Name lang="nl">Geneesmiddelexantheem met eosinofilie en systemische symptomen</Name>
+      <Name lang="nl">Reactie op medicatie met eosinofilie en systemische symptomen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -28636,7 +28535,7 @@
     <Disorder id="1426">
       <OrphaCode>1166</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1166</ExpertLink>
-      <Name lang="nl">Congenitale unilaterale hypoplasie van de depressor anguli oris</Name>
+      <Name lang="nl">Congenitale unilaterale hypoplasie van depressor anguli oris</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -29785,7 +29684,7 @@
     <Disorder id="1443">
       <OrphaCode>1186</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1186</ExpertLink>
-      <Name lang="nl">Spinocerebellaire ataxie met infantiele aanvang</Name>
+      <Name lang="nl">Infantiele spinocerebellaire ataxie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -32431,7 +32330,7 @@
     <Disorder id="17145">
       <OrphaCode>157794</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=157794</ExpertLink>
-      <Name lang="nl">Erfelijke gemengde polyposis-syndroom</Name>
+      <Name lang="nl">Erfelijk gemengde polyposis-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -32783,32 +32682,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1544">
-      <OrphaCode>1321</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1321</ExpertLink>
-      <Name lang="nl">Camptodactylie - hyperplasie van fibreus weefsel - skeletdysplasie-syndroom</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="nl">No data available</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="1546">
       <OrphaCode>1323</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1323</ExpertLink>
@@ -33046,32 +32919,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1561">
-      <OrphaCode>1342</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1342</ExpertLink>
-      <Name lang="nl">Hart-handsyndroom type 3</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="nl">No data available</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="1563">
       <OrphaCode>1345</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1345</ExpertLink>
@@ -44960,11 +44807,11 @@
       <OrphaCode>2196</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2196</ExpertLink>
       <Name lang="nl">Familiale primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose met ernstige oogbetrokkenheid</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23529">
@@ -47016,7 +46863,7 @@
     <Disorder id="2137">
       <OrphaCode>165</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=165</ExpertLink>
-      <Name lang="nl">Neutrale vetopslagziekte</Name>
+      <Name lang="nl">Neutrale vetstapelingsziekte</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -52656,7 +52503,7 @@
     <Disorder id="2316">
       <OrphaCode>2504</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2504</ExpertLink>
-      <Name lang="nl">Metafysaire dysplasie - maxillaire hypoplasie - brachydactie-syndroom</Name>
+      <Name lang="nl">Metafysaire dysplasie - maxillaire hypoplasie - brachydactylie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -53015,7 +52862,7 @@
     <Disorder id="2332">
       <OrphaCode>2522</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2522</ExpertLink>
-      <Name lang="nl">Microcefalie - cervicale wervelfusieanomalieën</Name>
+      <Name lang="nl">Microcefalie - fusie van cervicale wervelkolom-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -56918,29 +56765,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="2804">
-      <OrphaCode>3133</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3133</ExpertLink>
-      <Name lang="nl">Syndroom van Say-Field-Coldwell</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="nl">Normale levensverwachting</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="2805">
       <OrphaCode>3134</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3134</ExpertLink>
@@ -60308,7 +60132,7 @@
     <Disorder id="2626">
       <OrphaCode>2900</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2900</ExpertLink>
-      <Name lang="nl">Pleonosteose van Léri</Name>
+      <Name lang="nl">Pleonosteose van Leri</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -62538,7 +62362,7 @@
     <Disorder id="3041">
       <OrphaCode>3459</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3459</ExpertLink>
-      <Name lang="nl">Wilson-Turnersyndroom</Name>
+      <Name lang="nl">Syndroom van Wilson-Turner</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -62671,7 +62495,7 @@
     <Disorder id="3066">
       <OrphaCode>2995</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2995</ExpertLink>
-      <Name lang="nl">Cerebro-fronto-faciaal syndroom van Baraitser-Winter</Name>
+      <Name lang="nl">Cerebrofrontofaciaal syndroom van Baraitser-Winter</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -64072,8 +63896,8 @@
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
+        <TypeOfInheritance id="23417">
+          <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
@@ -66283,7 +66107,7 @@
     <Disorder id="20371">
       <OrphaCode>276399</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=276399</ExpertLink>
-      <Name lang="nl">Familiaal multinodulair kropgezwel</Name>
+      <Name lang="nl">Familiaal multinodulair struma</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -70223,7 +70047,7 @@
     <Disorder id="18667">
       <OrphaCode>209908</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=209908</ExpertLink>
-      <Name lang="nl">Spraakapraxie tijdens de kindertijd</Name>
+      <Name lang="nl">Geïsoleerde spraakapraxie op kinderleeftijd</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -70881,7 +70705,10 @@
           <Name lang="nl">Op leeftijd</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23641">
+          <Name lang="nl">Normale levensverwachting</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -71814,7 +71641,7 @@
     <Disorder id="3572">
       <OrphaCode>2364</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2364</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte als gevolg van lactaatdehydrogenasedeficiëntie</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door deficiëntie van lactaatdehydrogenase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -72262,7 +72089,10 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="nl">Kindertijd</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23522">
           <Name lang="nl">Kindsheid</Name>
         </AverageAgeOfOnset>
@@ -74832,7 +74662,7 @@
     <Disorder id="18855">
       <OrphaCode>217399</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217399</ExpertLink>
-      <Name lang="nl">Congenitale ongevoeligheid voor pijn met hyperhidrose</Name>
+      <Name lang="nl">Congenitale ongevoeligheid voor pijn - hyperhidrose - afwezigheid van innervatie door C-vezels</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -78814,7 +78644,7 @@
     <Disorder id="3700">
       <OrphaCode>2781</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2781</ExpertLink>
-      <Name lang="nl">Osteopetrose</Name>
+      <Name lang="nl">Osteopetrose en gerelateerde aandoeningen</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -81366,7 +81196,7 @@
     <Disorder id="19152">
       <OrphaCode>231080</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231080</ExpertLink>
-      <Name lang="nl">Hoge graad dysplasie bij patiënten met Barrett-oesofagus</Name>
+      <Name lang="nl">Hooggradige dysplasie bij patiënten met Barrett-oesofagus</Name>
       <DisorderType id="21429">
         <Name lang="nl">Particulaire klinische situatie binnen een ziekte of syndroom</Name>
       </DisorderType>
@@ -84866,32 +84696,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="21917">
-      <OrphaCode>329252</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329252</ExpertLink>
-      <Name lang="nl">Spondylocostale dysostose - hypospadie - intellectuele achterstand-syndroom</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="21916">
       <OrphaCode>329249</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329249</ExpertLink>
@@ -85952,7 +85756,7 @@
     <Disorder id="21762">
       <OrphaCode>324299</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=324299</ExpertLink>
-      <Name lang="nl">Multipele paragangliomen geassocieerd met polycythemie</Name>
+      <Name lang="nl">Multipele paragangliomen geassocieerd met polycytemie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -90458,7 +90262,10 @@
           <Name lang="nl">Volwassenheid</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="nl">No data available</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -94037,12 +93844,18 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="nl">Volwassenheid</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23550">
           <Name lang="nl">Op leeftijd</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -97512,7 +97325,7 @@
     <Disorder id="21405">
       <OrphaCode>309854</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=309854</ExpertLink>
-      <Name lang="nl">Cirrose - dystonie - polycythemie - hypermanganemie-syndroom</Name>
+      <Name lang="nl">Cirrose - dystonie - polycytemie - hypermanganemie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -97911,7 +97724,7 @@
     <Disorder id="21452">
       <OrphaCode>313800</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313800</ExpertLink>
-      <Name lang="nl">Oedeem van nervus opticus - splenomegalie-syndroom</Name>
+      <Name lang="nl">Retinadystrofie - oedeem van nervus opticus - splenomegalie - anhidrose - migrainehoofdpijn-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -100448,11 +100261,11 @@
       <OrphaCode>306516</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=306516</ExpertLink>
       <Name lang="nl">Familiale primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose</Name>
-      <DisorderType id="21436">
-        <Name lang="nl">Group of phenomes</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
       </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="nl">Groep van aandoeningen</Name>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23529">
@@ -105563,7 +105376,7 @@
     <Disorder id="24102">
       <OrphaCode>466026</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=466026</ExpertLink>
-      <Name lang="nl">Glucose-6-fosfaatdehydrogenasedeficiëntie klasse 1</Name>
+      <Name lang="nl">Deficiëntie van glucose-6-fosfaatdehydrogenase klasse 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -113343,7 +113156,7 @@
     <Disorder id="23440">
       <OrphaCode>443087</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=443087</ExpertLink>
-      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis als gevolg van testiculaire 17,20-desmolasedeficiëntie</Name>
+      <Name lang="nl">46,XY-geslachtsontwikkelingsstoornis door testiculaire deficiëntie van 17,20-desmolase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -117783,7 +117596,7 @@
     <Disorder id="8762">
       <OrphaCode>26349</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=26349</ExpertLink>
-      <Name lang="nl">Proteïne-S verworven deficiëntie</Name>
+      <Name lang="nl">Verworven proteïne S-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -120310,12 +120123,39 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23543">
           <Name lang="nl">Volwassenheid</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="nl">Kindertijd</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="nl">Vroege kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="nl">Late kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="nl">Jonge volwassenheid</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -126179,11 +126019,11 @@
       <OrphaCode>31043</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=31043</ExpertLink>
       <Name lang="nl">Familiale primaire hypomagnesiëmie met hypercalciurie en nefrocalcinose zonder ernstige oculaire betrokkenheid</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23529">
@@ -126456,7 +126296,7 @@
     <Disorder id="10993">
       <OrphaCode>71271</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=71271</ExpertLink>
-      <Name lang="nl">Gespleten hand - gespleten voet - doofheid-syndroom</Name>
+      <Name lang="nl">Split hand-split foot-deafness syndrome</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -127269,7 +127109,7 @@
     <Disorder id="10948">
       <OrphaCode>70475</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=70475</ExpertLink>
-      <Name lang="nl">Proctitis door straling</Name>
+      <Name lang="nl">Radiatieproctitis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -128800,7 +128640,7 @@
     <Disorder id="10869">
       <OrphaCode>65287</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=65287</ExpertLink>
-      <Name lang="nl">Bèta-ureïdopropionasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van bèta-ureïdopropionase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -128878,7 +128718,7 @@
     <Disorder id="10870">
       <OrphaCode>65288</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=65288</ExpertLink>
-      <Name lang="nl">Permanente neonatale diabetes mellitus - pancreatische en cerebellaire agenesie-syndroom</Name>
+      <Name lang="nl">Permanente neonatale diabetes mellitus - agenesie van pancreas en cerebellum-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -129665,7 +129505,25 @@
           <Name lang="nl">Op leeftijd</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="nl">Vroege kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="nl">Late kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="nl">Jonge volwassenheid</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -129924,32 +129782,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63440</ExpertLink>
-      <Name lang="nl">Geïsoleerde oxycefalie</Name>
-      <DisorderType id="21415">
-        <Name lang="nl">Morfologische anomalie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23529">
-          <Name lang="nl">Kindertijd</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="nl">Niet toepasbaar</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=63442</ExpertLink>
@@ -131732,7 +131564,7 @@
     <Disorder id="11215">
       <OrphaCode>79189</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79189</ExpertLink>
-      <Name lang="nl">Stoornis van peroxisoombiogenese</Name>
+      <Name lang="nl">Peroxisomale biogenesestoornis</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -133192,7 +133024,7 @@
     <Disorder id="11106">
       <OrphaCode>77293</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=77293</ExpertLink>
-      <Name lang="nl">Ziekte van Niemann-Pick type B</Name>
+      <Name lang="nl">Chronische viscerale deficiëntie van zure sfingomyelinase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -133267,7 +133099,7 @@
     <Disorder id="11105">
       <OrphaCode>77292</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=77292</ExpertLink>
-      <Name lang="nl">Ziekte van Niemann-Pick type A</Name>
+      <Name lang="nl">Infantiele neuroviscerale deficiëntie van zure sfingomyelinase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -136863,38 +136695,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="10368">
-      <OrphaCode>35098</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35098</ExpertLink>
-      <Name lang="nl">Geïsoleerde plagiocefalie</Name>
-      <DisorderType id="21415">
-        <Name lang="nl">Morfologische anomalie</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="nl">Normale levensverwachting</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="nl">Niet toepasbaar</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10375">
       <OrphaCode>35125</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35125</ExpertLink>
@@ -138596,76 +138396,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="10344">
-      <OrphaCode>34526</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=34526</ExpertLink>
-      <Name lang="nl">Genetische primaire hypomagnesiëmie</Name>
-      <DisorderType id="36561">
-        <Name lang="nl">Category</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="nl">Groep van aandoeningen</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="3">
-        <AverageAgeOfOnset id="23536">
-          <Name lang="nl">Puber</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23543">
-          <Name lang="nl">Volwassenheid</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23529">
-          <Name lang="nl">Kindertijd</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="nl">Elke leeftijd</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="10345">
-      <OrphaCode>34527</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=34527</ExpertLink>
-      <Name lang="nl">Familiale primaire hypomagnesiëmie met normocalciurie en normocalcïemie</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="3">
-        <AverageAgeOfOnset id="23536">
-          <Name lang="nl">Puber</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23529">
-          <Name lang="nl">Kindertijd</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="nl">Elke leeftijd</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="nl">Autosomaal dominant</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10346">
       <OrphaCode>34528</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=34528</ExpertLink>
@@ -143452,7 +143182,7 @@
     <Disorder id="11938">
       <OrphaCode>90042</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90042</ExpertLink>
-      <Name lang="nl">Primaire familiale polycythemie</Name>
+      <Name lang="nl">Primaire familiale polycytemie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -145017,7 +144747,7 @@
     <Disorder id="11818">
       <OrphaCode>88642</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=88642</ExpertLink>
-      <Name lang="nl">Kanalopathie-geässocieerde congenitale ongevoelighied voor pijn</Name>
+      <Name lang="nl">Congenitale ongevoelighied voor pijn - anosmie - neuropathische artropathie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -145700,7 +145430,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23424">
+          <Name lang="nl">Multigeen/multifactorieel</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="27333">
@@ -149731,7 +149464,7 @@
     <Disorder id="12148">
       <OrphaCode>91483</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91483</ExpertLink>
-      <Name lang="nl">Riegeranomalie</Name>
+      <Name lang="nl">Rieger-anomalie</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -149855,7 +149588,7 @@
     <Disorder id="12096">
       <OrphaCode>90793</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90793</ExpertLink>
-      <Name lang="nl">Congenitale adrenale hyperplasie door 17-alfa-hydroxylasedeficiëntie</Name>
+      <Name lang="nl">Congenitale adrenale hyperplasie door deficiëntie van 17-alfa-hydroxylase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149966,7 +149699,7 @@
     <Disorder id="12099">
       <OrphaCode>90796</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90796</ExpertLink>
-      <Name lang="nl">46,XY geslachtsontwikkelingsstoornis door geïsoleerde 17,20-lyasedeficiëntie</Name>
+      <Name lang="nl">46,XY-geslachtsontwikkelingsstoornis door geïsoleerde deficiëntie van 17,20-lyase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -155939,12 +155672,21 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="nl">Volwassenheid</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="2">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -156321,7 +156063,7 @@
     <Disorder id="11533">
       <OrphaCode>79507</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79507</ExpertLink>
-      <Name lang="nl">Hypotonie - groeifalen - microcefalie-syndroom</Name>
+      <Name lang="nl">Hypotonie - niet-gedijen - microcefalie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -158132,7 +157874,7 @@
     <Disorder id="11623">
       <OrphaCode>85112</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85112</ExpertLink>
-      <Name lang="nl">Familiale encefalopathie met neuroserpine-inclusielichamen</Name>
+      <Name lang="nl">Palmoplantaire keratodermie - XX-geslachtsomkering - predispositie voor plaveiselcelcarcinoom-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -160084,7 +159826,7 @@
     <Disorder id="12833">
       <OrphaCode>96264</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96264</ExpertLink>
-      <Name lang="nl">49,XXXXY syndroom</Name>
+      <Name lang="nl">49,XXXXY-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -160113,7 +159855,7 @@
     <Disorder id="12832">
       <OrphaCode>96263</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96263</ExpertLink>
-      <Name lang="nl">48,XXXY syndroom</Name>
+      <Name lang="nl">48,XXXY-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -161357,32 +161099,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12562">
-      <OrphaCode>94095</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94095</ExpertLink>
-      <Name lang="nl">Spondylocostale dysostose - anusatresie - urogenitale malformatie-syndroom</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23508">
-          <Name lang="nl">Antenataal</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="nl">No data available</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12563">
       <OrphaCode>94122</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94122</ExpertLink>
@@ -162016,26 +161732,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12776">
-      <OrphaCode>96152</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96152</ExpertLink>
-      <Name lang="nl">Distale monosomie 20q</Name>
-      <DisorderType id="21401">
-        <Name lang="nl">Malformatiesyndroom</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12774">
       <OrphaCode>96150</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96150</ExpertLink>
@@ -165257,38 +164953,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12437">
-      <OrphaCode>93604</OrphaCode>
-      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93604</ExpertLink>
-      <Name lang="nl">Prenataal syndroom van Bartter</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype van aandoening</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="3">
-        <AverageAgeOfOnset id="23508">
-          <Name lang="nl">Antenataal</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23522">
-          <Name lang="nl">Kindsheid</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="nl">Neonataal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23431">
-          <Name lang="nl">X-gebonden recessief</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="14111">
       <OrphaCode>99094</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99094</ExpertLink>
@@ -169151,7 +168815,7 @@
     <Disorder id="13995">
       <OrphaCode>98978</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98978</ExpertLink>
-      <Name lang="nl">Anomalie van Axenfeld</Name>
+      <Name lang="nl">Axenfeld-anomalie</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -176599,7 +176263,7 @@
     <Disorder id="14422">
       <OrphaCode>99849</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99849</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte door bèta-enolasedeficiëntie in de spier</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte door deficiëntie van bèta-enolase in spier</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -177405,7 +177069,47 @@
     <Disorder id="31372">
       <OrphaCode>617919</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=617919</ExpertLink>
-      <Name lang="nl">F12-associated cold auto-inflammatory syndrome</Name>
+      <Name lang="nl">F12-geassocieerd auto-inflammatoir syndroom door koude</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="31415">
+      <OrphaCode>619363</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=619363</ExpertLink>
+      <Name lang="nl">Neonatal onset-severe multisystem auto-inflammatory disease-increased IL-18</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="31405">
+      <OrphaCode>619233</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=619233</ExpertLink>
+      <Name lang="nl">Hereditary persistence of fetal hemoglobin-intellectual disability syndrome</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
