commit 655dce35bd7c9138100d7097b3f9ac34d1cb8241
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 7 11:48:42 2022 +0100

    MAR 2022

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
index e93bb3d..05ac003 100644
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-02-01 04:54:54" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-03-01 04:55:14" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6653">
+  <DisorderList count="6654">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -9816,7 +9816,7 @@
     <Disorder id="17780">
       <OrphaCode>168598</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=168598</ExpertLink>
-      <Name lang="nl">Hersendemyelinisatie door methionine-adenosyltransferasedeficiëntie</Name>
+      <Name lang="nl">Demyelinisatie van hersenen door deficiëntie van methionine-adenosyltransferase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -14110,7 +14110,7 @@
     <Disorder id="18030">
       <OrphaCode>178478</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=178478</ExpertLink>
-      <Name lang="nl">Botulisme bij kinderen</Name>
+      <Name lang="nl">Botulisme bij zuigeling</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -14882,7 +14882,7 @@
     <Disorder id="18032">
       <OrphaCode>178487</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=178487</ExpertLink>
-      <Name lang="nl">Intestinaal botulisme bij volwassenen</Name>
+      <Name lang="nl">Intestinaal botulisme bij volwassene</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -14905,7 +14905,7 @@
     <Disorder id="17996">
       <OrphaCode>177926</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=177926</ExpertLink>
-      <Name lang="nl">Symptomatische vorm van hemofilie A bij vrouwelijke dragers</Name>
+      <Name lang="nl">Stollingsstoornis bij dragers van hemofilie A</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -14948,7 +14948,7 @@
     <Disorder id="17997">
       <OrphaCode>177929</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=177929</ExpertLink>
-      <Name lang="nl">Symptomatische vorm van hemofilie B bij vrouwelijke dragers</Name>
+      <Name lang="nl">Stollingsstoornis bij dragers van hemofilie B</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -17044,7 +17044,7 @@
     <Disorder id="730">
       <OrphaCode>322</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=322</ExpertLink>
-      <Name lang="nl">Exstrofie - epispadieën-complex</Name>
+      <Name lang="nl">Exstrofie - epispadie-complex</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -17287,7 +17287,7 @@
     <Disorder id="17936">
       <OrphaCode>171829</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=171829</ExpertLink>
-      <Name lang="nl">6q16 microdeletiesyndroom</Name>
+      <Name lang="nl">6q16-microdeletiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -18797,7 +18797,7 @@
     <Disorder id="18399">
       <OrphaCode>199318</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=199318</ExpertLink>
-      <Name lang="nl">15q13.3 microdeletiesyndroom</Name>
+      <Name lang="nl">15q13.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -22776,7 +22776,7 @@
     <Disorder id="1115">
       <OrphaCode>1621</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1621</ExpertLink>
-      <Name lang="nl">3q13 microdeletiesyndroom</Name>
+      <Name lang="nl">3q13-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -22822,7 +22822,7 @@
     <Disorder id="1132">
       <OrphaCode>1643</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1643</ExpertLink>
-      <Name lang="nl">Xp22.3 microdeletiesyndroom</Name>
+      <Name lang="nl">Xp22.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -24866,7 +24866,7 @@
     <Disorder id="1160">
       <OrphaCode>1713</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1713</ExpertLink>
-      <Name lang="nl">17p11.2 microduplicatiesyndroom</Name>
+      <Name lang="nl">17p11.2-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -27361,7 +27361,7 @@
     <Disorder id="1313">
       <OrphaCode>1001</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1001</ExpertLink>
-      <Name lang="nl">2q37 microdeletiesyndroom</Name>
+      <Name lang="nl">2q37-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -28064,7 +28064,7 @@
     <Disorder id="16695">
       <OrphaCode>137628</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=137628</ExpertLink>
-      <Name lang="nl">Hartanomalieën - heterotaxie</Name>
+      <Name lang="nl">Hartanomalieën - heterotaxie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -29194,7 +29194,7 @@
     <Disorder id="16711">
       <OrphaCode>137754</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=137754</ExpertLink>
-      <Name lang="nl">Neurologische aandoeningen geassocieerd met aminoacylase 1-deficiëntie</Name>
+      <Name lang="nl">Neurologische aandoeningen geassocieerd met deficiëntie van aminoacylase 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -29411,7 +29411,7 @@
     <Disorder id="1423">
       <OrphaCode>1159</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1159</ExpertLink>
-      <Name lang="nl">Progressieve pseudoreumatoïde artropathie in de kinderjaren</Name>
+      <Name lang="nl">Progressieve pseudoreumatoïde artropathie op kinderleeftijd</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -30744,7 +30744,7 @@
     <Disorder id="1638">
       <OrphaCode>1435</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1435</ExpertLink>
-      <Name lang="nl">Xq21 microdeletiesyndroom</Name>
+      <Name lang="nl">Xq21-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -30871,7 +30871,7 @@
     <Disorder id="1659">
       <OrphaCode>1484</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1484</ExpertLink>
-      <Name lang="nl">Contracturen - ectodermale dysplasie - gespleten lip/verhemelte</Name>
+      <Name lang="nl">Contracturen - ectodermale dysplasie - gespleten lip/gehemelte-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -33878,7 +33878,7 @@
     <Disorder id="1732">
       <OrphaCode>1574</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1574</ExpertLink>
-      <Name lang="nl">Retinadegeneratie - nanoftalmos - glaucoom-syndroom</Name>
+      <Name lang="nl">Retinadegeneratie - nanoftalmie - glaucoom-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -33950,7 +33950,7 @@
     <Disorder id="16904">
       <OrphaCode>139471</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139471</ExpertLink>
-      <Name lang="nl">Microftalmie met hersen- en vingeranomalieën</Name>
+      <Name lang="nl">Microftalmie met anomalieën van hersenen en digitus</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -33973,7 +33973,7 @@
     <Disorder id="16905">
       <OrphaCode>139474</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139474</ExpertLink>
-      <Name lang="nl">17q11.2 microduplicatiesyndroom</Name>
+      <Name lang="nl">17q11.2-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -33999,7 +33999,7 @@
     <Disorder id="1739">
       <OrphaCode>1617</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1617</ExpertLink>
-      <Name lang="nl">2q24 microdeletiesyndroom</Name>
+      <Name lang="nl">2q24-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -34028,7 +34028,7 @@
     <Disorder id="1738">
       <OrphaCode>1606</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1606</ExpertLink>
-      <Name lang="nl">1p36 deletiesyndroom</Name>
+      <Name lang="nl">1p36-deletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -36907,7 +36907,7 @@
     <Disorder id="1880">
       <OrphaCode>1917</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1917</ExpertLink>
-      <Name lang="nl">Foetaal methylkwiksyndroom</Name>
+      <Name lang="nl">Foetaal syndroom door methylkwik</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -40993,7 +40993,7 @@
     <Disorder id="1949">
       <OrphaCode>2013</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2013</ExpertLink>
-      <Name lang="nl">Gespleten verhemelte - grote oren - klein hoofd</Name>
+      <Name lang="nl">Gespleten gehemelte - grote oren - klein hoofd-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -41071,7 +41071,7 @@
     <Disorder id="1945">
       <OrphaCode>2008</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2008</ExpertLink>
-      <Name lang="nl">Acro-cardio-faciaal syndroom</Name>
+      <Name lang="nl">Acrocardiofaciaal syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -42858,7 +42858,7 @@
     <Disorder id="2227">
       <OrphaCode>2400</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2400</ExpertLink>
-      <Name lang="nl">Perifere motorische neuropathie - dysautonomie</Name>
+      <Name lang="nl">Perifere motorische neuropathie - dysautonomie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -43737,7 +43737,7 @@
     <Disorder id="2249">
       <OrphaCode>2429</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2429</ExpertLink>
-      <Name lang="nl">Macrocefalie - spastische paraplegie - dysmorfie</Name>
+      <Name lang="nl">Macrocefalie - spastische paraplegie - dysmorfie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -45008,7 +45008,7 @@
     <Disorder id="2050">
       <OrphaCode>2150</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2150</ExpertLink>
-      <Name lang="nl">Ziekte van Hirschsprung - type D-brachydactylie-syndroom</Name>
+      <Name lang="nl">Ziekte van Hirschsprung - brachydactylie type D-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46421,7 +46421,7 @@
     <Disorder id="19604">
       <OrphaCode>251019</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251019</ExpertLink>
-      <Name lang="nl">2q32q33 microdeletiesyndroom</Name>
+      <Name lang="nl">2q32q33-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46537,7 +46537,7 @@
     <Disorder id="19607">
       <OrphaCode>251038</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251038</ExpertLink>
-      <Name lang="nl">3q29 microduplicatiesyndroom</Name>
+      <Name lang="nl">3q29-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46566,7 +46566,7 @@
     <Disorder id="19600">
       <OrphaCode>250999</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=250999</ExpertLink>
-      <Name lang="nl">1q41q42 microdeletiesyndroom</Name>
+      <Name lang="nl">1q41q42-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46717,7 +46717,7 @@
     <Disorder id="19603">
       <OrphaCode>251014</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251014</ExpertLink>
-      <Name lang="nl">2q31.1 microdeletiesyndroom</Name>
+      <Name lang="nl">2q31.1-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46775,7 +46775,7 @@
     <Disorder id="19612">
       <OrphaCode>251061</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251061</ExpertLink>
-      <Name lang="nl">7q31 microdeletiesyndroom</Name>
+      <Name lang="nl">7q31-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46804,7 +46804,7 @@
     <Disorder id="19613">
       <OrphaCode>251066</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251066</ExpertLink>
-      <Name lang="nl">8p11.2 deletiesyndroom</Name>
+      <Name lang="nl">8p11.2-deletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46862,7 +46862,7 @@
     <Disorder id="19614">
       <OrphaCode>251071</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251071</ExpertLink>
-      <Name lang="nl">8p23.1 microdeletiesyndroom</Name>
+      <Name lang="nl">8p23.1-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46923,7 +46923,7 @@
     <Disorder id="19615">
       <OrphaCode>251076</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251076</ExpertLink>
-      <Name lang="nl">8p23.1 duplicatiesyndroom</Name>
+      <Name lang="nl">8p23.1-duplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -47042,7 +47042,7 @@
     <Disorder id="19609">
       <OrphaCode>251046</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251046</ExpertLink>
-      <Name lang="nl">6p22 microdeletiesyndroom</Name>
+      <Name lang="nl">6p22-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -47158,7 +47158,7 @@
     <Disorder id="19611">
       <OrphaCode>251056</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251056</ExpertLink>
-      <Name lang="nl">6q25 microdeletiesyndroom</Name>
+      <Name lang="nl">6q25-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -47433,7 +47433,7 @@
     <Disorder id="19599">
       <OrphaCode>250994</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=250994</ExpertLink>
-      <Name lang="nl">1q21.1 microduplicatiesyndroom</Name>
+      <Name lang="nl">1q21.1-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -47462,7 +47462,7 @@
     <Disorder id="19598">
       <OrphaCode>250989</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=250989</ExpertLink>
-      <Name lang="nl">1q21.1 microdeletiesyndroom</Name>
+      <Name lang="nl">1q21.1-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -50199,7 +50199,7 @@
     <Disorder id="2480">
       <OrphaCode>2719</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2719</ExpertLink>
-      <Name lang="nl">Oculocerebrale hypopigmentatiesyndroom, Cross-type</Name>
+      <Name lang="nl">Oculocerebrale hypopigmentatie-syndroom, Cross-type</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -50820,7 +50820,7 @@
     <Disorder id="2524">
       <OrphaCode>2774</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2774</ExpertLink>
-      <Name lang="nl">Multicentrische carpo-tarsale osteolyse met of zonder nefropathie</Name>
+      <Name lang="nl">Multicentrische carpotarsale osteolyse met of zonder nefropathie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -50843,7 +50843,7 @@
     <Disorder id="19740">
       <OrphaCode>254351</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254351</ExpertLink>
-      <Name lang="nl">Distale 7q11.23 microdeletiesyndroom</Name>
+      <Name lang="nl">Distale 7q11.23-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -50895,7 +50895,7 @@
     <Disorder id="19739">
       <OrphaCode>254346</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254346</ExpertLink>
-      <Name lang="nl">19q13.12 microdeletiesyndroom</Name>
+      <Name lang="nl">19q13.12-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -51355,7 +51355,7 @@
     <Disorder id="19758">
       <OrphaCode>254525</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254525</ExpertLink>
-      <Name lang="nl">Syndroom van Temple door paternale 14q32.2 microdeletie</Name>
+      <Name lang="nl">Syndroom van Temple door paternale 14q32.2-microdeletie</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -51433,7 +51433,7 @@
     <Disorder id="19759">
       <OrphaCode>254528</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=254528</ExpertLink>
-      <Name lang="nl">Syndroom van Kagami-Ogata door maternale 14q32.2 microdeletie</Name>
+      <Name lang="nl">Syndroom van Kagami-Ogata door maternale 14q32.2-microdeletie</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -53692,7 +53692,7 @@
     <Disorder id="19851">
       <OrphaCode>261120</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261120</ExpertLink>
-      <Name lang="nl">14q11.2 microdeletiesyndroom</Name>
+      <Name lang="nl">14q11.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -53805,7 +53805,7 @@
     <Disorder id="19849">
       <OrphaCode>261102</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261102</ExpertLink>
-      <Name lang="nl">Distale 7q11.23 microduplicatiesyndroom</Name>
+      <Name lang="nl">Distale 7q11.23-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -53860,7 +53860,7 @@
     <Disorder id="19862">
       <OrphaCode>261236</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261236</ExpertLink>
-      <Name lang="nl">16p13.11 microdeletiesyndroom</Name>
+      <Name lang="nl">16p13.11-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -53889,7 +53889,7 @@
     <Disorder id="19863">
       <OrphaCode>261243</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261243</ExpertLink>
-      <Name lang="nl">16p13.11 microduplicatiesyndroom</Name>
+      <Name lang="nl">16p13.11-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54028,7 +54028,7 @@
     <Disorder id="19859">
       <OrphaCode>261211</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261211</ExpertLink>
-      <Name lang="nl">16p11.2p12.2 microdeletiesyndroom</Name>
+      <Name lang="nl">16p11.2p12.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54083,7 +54083,7 @@
     <Disorder id="19856">
       <OrphaCode>261190</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261190</ExpertLink>
-      <Name lang="nl">15q14 microdeletiesyndroom</Name>
+      <Name lang="nl">15q14-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54141,7 +54141,7 @@
     <Disorder id="19857">
       <OrphaCode>261197</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261197</ExpertLink>
-      <Name lang="nl">Proximale 16p11.2 microdeletiesyndroom</Name>
+      <Name lang="nl">Proximale 16p11.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54170,7 +54170,7 @@
     <Disorder id="19870">
       <OrphaCode>261295</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261295</ExpertLink>
-      <Name lang="nl">20p12.3 microdeletiesyndroom</Name>
+      <Name lang="nl">20p12.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54199,7 +54199,7 @@
     <Disorder id="19871">
       <OrphaCode>261304</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261304</ExpertLink>
-      <Name lang="nl">Paternaal 20q13.2q13.3-microdeletiesyndroom</Name>
+      <Name lang="nl">Paternale 20q13.2q13.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54225,7 +54225,7 @@
     <Disorder id="19868">
       <OrphaCode>261279</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261279</ExpertLink>
-      <Name lang="nl">17q23.1q23.2 microdeletiesyndroom</Name>
+      <Name lang="nl">17q23.1q23.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54309,7 +54309,7 @@
     <Disorder id="19864">
       <OrphaCode>261250</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261250</ExpertLink>
-      <Name lang="nl">16q24.3 microdeletiesyndroom</Name>
+      <Name lang="nl">16q24.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54416,7 +54416,7 @@
     <Disorder id="19876">
       <OrphaCode>261337</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261337</ExpertLink>
-      <Name lang="nl">Distale 22q11.2 microduplicatiesyndroom</Name>
+      <Name lang="nl">Distale 22q11.2-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54448,7 +54448,7 @@
     <Disorder id="19879">
       <OrphaCode>261476</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261476</ExpertLink>
-      <Name lang="nl">Xp21 deletiesyndroom</Name>
+      <Name lang="nl">Xp21-deletiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -54477,7 +54477,7 @@
     <Disorder id="19878">
       <OrphaCode>261349</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261349</ExpertLink>
-      <Name lang="nl">2p15p16.1 microdeletiesyndroom</Name>
+      <Name lang="nl">2p15p16.1-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54529,7 +54529,7 @@
     <Disorder id="19872">
       <OrphaCode>261311</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261311</ExpertLink>
-      <Name lang="nl">20q13.33 microdeletiesyndroom</Name>
+      <Name lang="nl">20q13.33-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54575,7 +54575,7 @@
     <Disorder id="19875">
       <OrphaCode>261330</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261330</ExpertLink>
-      <Name lang="nl">Distale 22q11.2 microdeletiesyndroom</Name>
+      <Name lang="nl">Distale 22q11.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54598,7 +54598,7 @@
     <Disorder id="19874">
       <OrphaCode>261323</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261323</ExpertLink>
-      <Name lang="nl">21q22.11q22.12 microdeletiesyndroom</Name>
+      <Name lang="nl">21q22.11q22.12-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -55048,7 +55048,7 @@
     <Disorder id="20095">
       <OrphaCode>264200</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=264200</ExpertLink>
-      <Name lang="nl">14q22q23 microdeletiesyndroom</Name>
+      <Name lang="nl">14q22q23-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -55233,7 +55233,7 @@
     <Disorder id="2734">
       <OrphaCode>3041</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3041</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand - kaalheid - patellaluxatie - acromicrie-syndroom</Name>
+      <Name lang="nl">Intellectuele achterstand - kaalheid - luxatie van patella - acromicrie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -55262,7 +55262,7 @@
     <Disorder id="2732">
       <OrphaCode>3038</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3038</ExpertLink>
-      <Name lang="nl">Uitgestelde spraak - faciale asymmetrie - strabisme - oorlelplooien-syndroom</Name>
+      <Name lang="nl">Spraakachterstand - faciale asymmetrie - strabisme - oorlelplooien-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -58515,7 +58515,7 @@
     <Disorder id="2578">
       <OrphaCode>2839</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2839</ExpertLink>
-      <Name lang="nl">Bekken-schouderdysplasie</Name>
+      <Name lang="nl">Dysplasie van bekken-schouder</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -59050,7 +59050,7 @@
     <Disorder id="2561">
       <OrphaCode>2820</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2820</ExpertLink>
-      <Name lang="nl">Spastische paraplegie - nefritis - doofheid</Name>
+      <Name lang="nl">Spastische paraplegie - nefritis - doofheid-syndroom</Name>
       <DisorderType id="21422">
         <Name lang="nl">Klinisch syndroom</Name>
       </DisorderType>
@@ -59160,7 +59160,7 @@
     <Disorder id="20173">
       <OrphaCode>268261</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=268261</ExpertLink>
-      <Name lang="nl">DYRK1A-gerelateerd intellectuele achterstandsyndroom door microdeletie van 21q22.13q22.2</Name>
+      <Name lang="nl">DYRK1A-gerelateerd intellectuele achterstand-syndroom door 21q22.13q22.2-microdeletie</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -61018,7 +61018,7 @@
     <Disorder id="2986">
       <OrphaCode>3369</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3369</ExpertLink>
-      <Name lang="nl">Trigonocefalie - kleine gestalte - ontwikkelingsachterstand</Name>
+      <Name lang="nl">Trigonocefalie - kleine gestalte - ontwikkelingsachterstand-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -61356,7 +61356,7 @@
     <Disorder id="3007">
       <OrphaCode>3412</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3412</ExpertLink>
-      <Name lang="nl">VACTERL met hydrocefalus</Name>
+      <Name lang="nl">VACTERL met hydrocefalie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -61518,7 +61518,7 @@
     <Disorder id="2996">
       <OrphaCode>3383</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3383</ExpertLink>
-      <Name lang="nl">Aplasie van de trochlea humerus</Name>
+      <Name lang="nl">Aplasie van trochlea humeri</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -61847,7 +61847,7 @@
     <Disorder id="2951">
       <OrphaCode>3326</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3326</ExpertLink>
-      <Name lang="nl">Thymische-renale-anale-longdysplasie</Name>
+      <Name lang="nl">Dysplasie van thymus-nier-anus-long</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -61914,7 +61914,7 @@
     <Disorder id="2947">
       <OrphaCode>3317</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3317</ExpertLink>
-      <Name lang="nl">Thoracolaryngopelvische dysplasie</Name>
+      <Name lang="nl">Thoracolaryngopelviene dysplasie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -61984,7 +61984,10 @@
           <Name lang="nl">Puber</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23641">
+          <Name lang="nl">Normale levensverwachting</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -66188,7 +66191,7 @@
     <Disorder id="2892">
       <OrphaCode>3232</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3232</ExpertLink>
-      <Name lang="nl">Doofheid - oormisvorming - facialisparalyse-syndroom</Name>
+      <Name lang="nl">Doofheid - malformatie van oor - aangezichtsverlamming-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -66404,7 +66407,7 @@
     <Disorder id="20375">
       <OrphaCode>276413</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=276413</ExpertLink>
-      <Name lang="nl">10q22.3q23.3 microdeletiesyndroom</Name>
+      <Name lang="nl">10q22.3q23.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -66543,7 +66546,7 @@
     <Disorder id="20376">
       <OrphaCode>276422</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=276422</ExpertLink>
-      <Name lang="nl">10q22.3q23.3 microduplicatiesyndroom</Name>
+      <Name lang="nl">10q22.3q23.3-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -69396,7 +69399,7 @@
     <Disorder id="3101">
       <OrphaCode>2151</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2151</ExpertLink>
-      <Name lang="nl">Ziekte van Hirschsprung - ganglioneuroblastoom</Name>
+      <Name lang="nl">Ziekte van Hirschsprung - ganglioneuroblastoom-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -70808,7 +70811,7 @@
     <Disorder id="3535">
       <OrphaCode>1546</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1546</ExpertLink>
-      <Name lang="nl">Cryptococcosis</Name>
+      <Name lang="nl">Cryptokokkose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -74185,7 +74188,7 @@
     <Disorder id="18847">
       <OrphaCode>217340</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217340</ExpertLink>
-      <Name lang="nl">17q21.31 microduplicatiesyndroom</Name>
+      <Name lang="nl">17q21.31-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -74647,7 +74650,7 @@
     <Disorder id="18856">
       <OrphaCode>217407</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217407</ExpertLink>
-      <Name lang="nl">Erfelijke hypotrichose met terugkerende huidblaasjes</Name>
+      <Name lang="nl">Hereditaire hypotrichose met recidiverende huidblaasjes</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -74800,7 +74803,7 @@
     <Disorder id="18852">
       <OrphaCode>217385</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217385</ExpertLink>
-      <Name lang="nl">17p13.3 microduplicatiesyndroom</Name>
+      <Name lang="nl">17p13.3-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -74939,7 +74942,7 @@
     <Disorder id="18848">
       <OrphaCode>217346</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=217346</ExpertLink>
-      <Name lang="nl">19q13.11 microdeletiesyndroom</Name>
+      <Name lang="nl">19q13.11-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -75474,7 +75477,7 @@
     <Disorder id="3335">
       <OrphaCode>79</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79</ExpertLink>
-      <Name lang="nl">Congenitale alfa2-antiplasminedeficiëntie</Name>
+      <Name lang="nl">Congenitale deficiëntie van alfa2-antiplasmine</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -76737,7 +76740,7 @@
     <Disorder id="3381">
       <OrphaCode>851</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=851</ExpertLink>
-      <Name lang="nl">Paris-Trousseautrombocytopenie</Name>
+      <Name lang="nl">Trombocytopenie, Paris-Trousseau-type</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -78478,7 +78481,7 @@
     <Disorder id="19128">
       <OrphaCode>228415</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228415</ExpertLink>
-      <Name lang="nl">5q35 microduplicatiesyndroom</Name>
+      <Name lang="nl">5q35-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -78675,7 +78678,7 @@
     <Disorder id="19123">
       <OrphaCode>228396</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228396</ExpertLink>
-      <Name lang="nl">Ptosis - bovenste oculaire bewegingsbeperking - agenesie van punctum lacrimale-syndroom</Name>
+      <Name lang="nl">Ptosis - beperkte opwaartse oogbeweging - agenesie van punctum lacrimale-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -78782,7 +78785,7 @@
     <Disorder id="19124">
       <OrphaCode>228399</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228399</ExpertLink>
-      <Name lang="nl">8q12 microduplicatiesyndroom</Name>
+      <Name lang="nl">8q12-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -78840,7 +78843,7 @@
     <Disorder id="19125">
       <OrphaCode>228402</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228402</ExpertLink>
-      <Name lang="nl">2q23.1 microdeletiesyndroom</Name>
+      <Name lang="nl">2q23.1-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -78967,7 +78970,7 @@
     <Disorder id="19119">
       <OrphaCode>228384</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=228384</ExpertLink>
-      <Name lang="nl">5q14.3 microdeletiesyndroom</Name>
+      <Name lang="nl">5q14.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -80895,7 +80898,7 @@
     <Disorder id="19171">
       <OrphaCode>231214</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231214</ExpertLink>
-      <Name lang="nl">Bèta-thalassemie major</Name>
+      <Name lang="nl">Beta-thalassemia major</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -81836,7 +81839,7 @@
     <Disorder id="19263">
       <OrphaCode>238446</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238446</ExpertLink>
-      <Name lang="nl">15q11q13 microduplicatiesyndroom</Name>
+      <Name lang="nl">15q11q13-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -82275,7 +82278,7 @@
     <Disorder id="19215">
       <OrphaCode>231742</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231742</ExpertLink>
-      <Name lang="nl">Epibulbaire lipodermoïde - preauriculair aanhangsel - polythelie</Name>
+      <Name lang="nl">Epibulbair lipodermoïd - preauriculair aanhangsel - polythelie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -82576,7 +82579,7 @@
     <Disorder id="19300">
       <OrphaCode>238750</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238750</ExpertLink>
-      <Name lang="nl">4q21 microdeletiesyndroom</Name>
+      <Name lang="nl">4q21-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -82721,7 +82724,7 @@
     <Disorder id="19305">
       <OrphaCode>238769</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238769</ExpertLink>
-      <Name lang="nl">1q44 microdeletiesyndroom</Name>
+      <Name lang="nl">1q44-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -84070,7 +84073,7 @@
     <Disorder id="21940">
       <OrphaCode>329802</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329802</ExpertLink>
-      <Name lang="nl">5p13 microduplicatiesyndroom</Name>
+      <Name lang="nl">5p13-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -85897,7 +85900,7 @@
     <Disorder id="21764">
       <OrphaCode>324313</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=324313</ExpertLink>
-      <Name lang="nl">9p13 microdeletiesyndroom</Name>
+      <Name lang="nl">9p13-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -87881,7 +87884,7 @@
     <Disorder id="21549">
       <OrphaCode>315311</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=315311</ExpertLink>
-      <Name lang="nl">Klassieke congenitale adrenale hyperplasie door 21-hydroxylase-deficiëntie, eenvoudige viriliserende vorm</Name>
+      <Name lang="nl">Klassieke congenitale adrenale hyperplasie door deficiëntie van 21-hydroxylase, eenvoudige viriliserende vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -87907,7 +87910,7 @@
     <Disorder id="21548">
       <OrphaCode>315306</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=315306</ExpertLink>
-      <Name lang="nl">Klassieke congenitale adrenale hyperplasie door 21-hydroxylasedeficiëntie, zoutafvoerende vorm</Name>
+      <Name lang="nl">Klassieke congenitale adrenale hyperplasie door deficiëntie van 21-hydroxylase, vorm met zoutverlies</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -88322,7 +88325,7 @@
     <Disorder id="21510">
       <OrphaCode>314655</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314655</ExpertLink>
-      <Name lang="nl">Ernstige neonatale hypotonie - insulten - encefalopathie-syndroom als gevolg van 5q31.3 microdeletie</Name>
+      <Name lang="nl">Ernstige neonatale hypotonie - insulten - encefalopathie-syndroom door 5q31.3-microdeletie</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -89113,7 +89116,7 @@
     <Disorder id="22477">
       <OrphaCode>370079</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=370079</ExpertLink>
-      <Name lang="nl">Proximale 16p11.2 microduplicatiesyndroom</Name>
+      <Name lang="nl">Proximale 16p11.2-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -90570,7 +90573,7 @@
     <Disorder id="22350">
       <OrphaCode>363958</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363958</ExpertLink>
-      <Name lang="nl">17q21.31 microdeletiesyndroom</Name>
+      <Name lang="nl">17q21.31-microdeletiesyndroom</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -90833,7 +90836,7 @@
     <Disorder id="22338">
       <OrphaCode>363680</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363680</ExpertLink>
-      <Name lang="nl">2p13.2 microdeletiesyndroom</Name>
+      <Name lang="nl">2p13.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -91772,7 +91775,7 @@
     <Disorder id="22188">
       <OrphaCode>356947</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=356947</ExpertLink>
-      <Name lang="nl">3q26q27 microdeletiesyndroom</Name>
+      <Name lang="nl">3q26q27-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -91850,7 +91853,7 @@
     <Disorder id="22194">
       <OrphaCode>357001</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=357001</ExpertLink>
-      <Name lang="nl">19p13.13 microdeletiesyndroom</Name>
+      <Name lang="nl">19p13.13-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -92274,7 +92277,7 @@
     <Disorder id="22087">
       <OrphaCode>352629</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352629</ExpertLink>
-      <Name lang="nl">16q24.1 microdeletiesyndroom</Name>
+      <Name lang="nl">16q24.1-microdeletiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92563,7 +92566,7 @@
     <Disorder id="22096">
       <OrphaCode>352665</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352665</ExpertLink>
-      <Name lang="nl">Neurologische ontwikkelingsstoornis - craniofaciale dysmorfie - hartdefect - skeletanomalieën-syndroom door 9q21.3 microdeletiesyndroom</Name>
+      <Name lang="nl">Neurologische ontwikkelingsstoornis - craniofaciale dysmorfie - hartdefect - skeletanomalieën-syndroom door 9q21.3-microdeletie</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -93498,7 +93501,7 @@
     <Disorder id="20893">
       <OrphaCode>293948</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=293948</ExpertLink>
-      <Name lang="nl">1p21.3 microdeletiesyndroom</Name>
+      <Name lang="nl">1p21.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -95367,7 +95370,7 @@
     <Disorder id="20609">
       <OrphaCode>284417</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284417</ExpertLink>
-      <Name lang="nl">Fosfoserine-aminotransferasedeficiëntie, infantiele/juveniele vorm</Name>
+      <Name lang="nl">Deficiëntie van fosfoserine-aminotransferase, infantiele/juveniele vorm</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -95869,7 +95872,7 @@
     <Disorder id="20733">
       <OrphaCode>289513</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=289513</ExpertLink>
-      <Name lang="nl">12q15q21.1 microdeletiesyndroom</Name>
+      <Name lang="nl">12q15q21.1-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -97004,7 +97007,7 @@
     <Disorder id="20583">
       <OrphaCode>284180</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284180</ExpertLink>
-      <Name lang="nl">Xp22.13p22.2 duplicatiesyndroom</Name>
+      <Name lang="nl">Xp22.13p22.2-duplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -97033,7 +97036,7 @@
     <Disorder id="20582">
       <OrphaCode>284169</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284169</ExpertLink>
-      <Name lang="nl">Faciale dysmorfie - ontwikkelingsachterstand - gedragsstoornissen-syndroom door microdeletie van 10p11.21p12.31</Name>
+      <Name lang="nl">Faciale dysmorfie - ontwikkelingsachterstand - gedragsstoornissen-syndroom door 10p11.21p12.31-microdeletie</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -97062,7 +97065,7 @@
     <Disorder id="20581">
       <OrphaCode>284160</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=284160</ExpertLink>
-      <Name lang="nl">8q21.11 microdeletiesyndroom</Name>
+      <Name lang="nl">8q21.11-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -97633,7 +97636,7 @@
     <Disorder id="21466">
       <OrphaCode>313947</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313947</ExpertLink>
-      <Name lang="nl">2q23.1 microduplicatiesyndroom</Name>
+      <Name lang="nl">2q23.1-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -97766,7 +97769,7 @@
     <Disorder id="21460">
       <OrphaCode>313884</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313884</ExpertLink>
-      <Name lang="nl">12p12.1 microdeletiesyndroom</Name>
+      <Name lang="nl">12p12.1-microdeletiesyndroom</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -97957,7 +97960,7 @@
     <Disorder id="21450">
       <OrphaCode>313781</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313781</ExpertLink>
-      <Name lang="nl">20p13 microdeletiesyndroom</Name>
+      <Name lang="nl">20p13-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -98656,7 +98659,7 @@
     <Disorder id="21479">
       <OrphaCode>314389</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314389</ExpertLink>
-      <Name lang="nl">Xq12-q13.3 duplicatiesyndroom</Name>
+      <Name lang="nl">Xq12-q13.3-duplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -98682,7 +98685,7 @@
     <Disorder id="21472">
       <OrphaCode>314034</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314034</ExpertLink>
-      <Name lang="nl">7p22.1 microduplicatiesyndroom</Name>
+      <Name lang="nl">7p22.1-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -100917,7 +100920,7 @@
     <Disorder id="21107">
       <OrphaCode>300305</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300305</ExpertLink>
-      <Name lang="nl">11p15.4 microduplicatiesyndroom</Name>
+      <Name lang="nl">11p15.4-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -101687,7 +101690,7 @@
     <Disorder id="24025">
       <OrphaCode>464440</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=464440</ExpertLink>
-      <Name lang="nl">Primaire dystonie, DYT27 type</Name>
+      <Name lang="nl">Primaire dystonie, DYT27-type</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -101875,7 +101878,7 @@
     <Disorder id="23675">
       <OrphaCode>456298</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=456298</ExpertLink>
-      <Name lang="nl">1p35.2 microdeletiesyndroom</Name>
+      <Name lang="nl">1p35.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -103201,7 +103204,7 @@
     <Disorder id="23558">
       <OrphaCode>447980</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=447980</ExpertLink>
-      <Name lang="nl">19p13.3 microduplicatiesyndroom</Name>
+      <Name lang="nl">19p13.3-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -103444,7 +103447,7 @@
     <Disorder id="23773">
       <OrphaCode>459074</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=459074</ExpertLink>
-      <Name lang="nl">Agenesie van het corpus callosum - macrocefalie - hypertelorisme-syndroom</Name>
+      <Name lang="nl">Agenesie van corpus callosum - macrocefalie - hypertelorisme-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -106623,7 +106626,7 @@
     <Disorder id="22854">
       <OrphaCode>401935</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=401935</ExpertLink>
-      <Name lang="nl">14q24.1q24.3 microdeletiesyndroom</Name>
+      <Name lang="nl">14q24.1q24.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -106646,7 +106649,7 @@
     <Disorder id="22852">
       <OrphaCode>401923</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=401923</ExpertLink>
-      <Name lang="nl">9q31.1q31.3 microdeletiesyndroom</Name>
+      <Name lang="nl">9q31.1q31.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -106770,7 +106773,7 @@
     <Disorder id="22863">
       <OrphaCode>401986</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=401986</ExpertLink>
-      <Name lang="nl">1p31p32 microdeletiesyndroom</Name>
+      <Name lang="nl">1p31p32-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -107964,7 +107967,7 @@
     <Disorder id="23039">
       <OrphaCode>412035</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=412035</ExpertLink>
-      <Name lang="nl">13q12.3 microdeletiesyndroom</Name>
+      <Name lang="nl">13q12.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -108493,7 +108496,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
@@ -109729,7 +109735,7 @@
     <Disorder id="22693">
       <OrphaCode>397695</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=397695</ExpertLink>
-      <Name lang="nl">3q27.3 microdeletiesyndroom</Name>
+      <Name lang="nl">3q27.3-microdeletiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -110786,7 +110792,7 @@
     <Disorder id="23308">
       <OrphaCode>435638</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=435638</ExpertLink>
-      <Name lang="nl">3p25.3 microdeletiesyndroom</Name>
+      <Name lang="nl">3p25.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -111122,7 +111128,7 @@
     <Disorder id="23325">
       <OrphaCode>436003</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=436003</ExpertLink>
-      <Name lang="nl">Contracturen - ontwikkelingsachterstand - syndroom van Pierre Robin</Name>
+      <Name lang="nl">Contracturen - ontwikkelingsachterstand - Pierre Robin-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -113716,7 +113722,7 @@
     <Disorder id="23473">
       <OrphaCode>444002</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=444002</ExpertLink>
-      <Name lang="nl">11q22.2q22.3 microdeletiesyndroom</Name>
+      <Name lang="nl">11q22.2q22.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -113861,7 +113867,7 @@
     <Disorder id="23476">
       <OrphaCode>444051</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=444051</ExpertLink>
-      <Name lang="nl">20q11.2 microdeletiesyndroom</Name>
+      <Name lang="nl">20q11.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -115111,7 +115117,7 @@
     <Disorder id="23109">
       <OrphaCode>420492</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=420492</ExpertLink>
-      <Name lang="nl">Cervicale dystonie met aanvang op volwassen leeftijd, type DYT23</Name>
+      <Name lang="nl">Cervicale dystonie met aanvang op volwassen leeftijd, DYT23-type</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -117007,9 +117013,15 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="nl">Elke leeftijd</Name>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="nl">Volwassenheid</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="nl">Op leeftijd</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
@@ -118624,7 +118636,7 @@
     <Disorder id="26569">
       <OrphaCode>508488</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=508488</ExpertLink>
-      <Name lang="nl">8q24.3 microdeletiesyndroom</Name>
+      <Name lang="nl">8q24.3-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -118927,7 +118939,7 @@
     <Disorder id="25800">
       <OrphaCode>495818</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=495818</ExpertLink>
-      <Name lang="nl">9q33.3q34.11 microdeletiesyndroom</Name>
+      <Name lang="nl">9q33.3q34.11-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -120080,7 +120092,7 @@
     <Disorder id="25993">
       <OrphaCode>500055</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=500055</ExpertLink>
-      <Name lang="nl">16p13.2 microdeletiesyndroom</Name>
+      <Name lang="nl">16p13.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -120638,7 +120650,7 @@
     <Disorder id="8568">
       <OrphaCode>345</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=345</ExpertLink>
-      <Name lang="nl">Dissecting cellulitis van de hoofdhuid</Name>
+      <Name lang="nl">Dissecerende cellulitis van hoofdhuid</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -121190,7 +121202,7 @@
     <Disorder id="26079">
       <OrphaCode>502437</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=502437</ExpertLink>
-      <Name lang="nl">Proximale 4q25 deletiesyndroom</Name>
+      <Name lang="nl">Proximale 4q25-deletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -125762,7 +125774,7 @@
     <Disorder id="25397">
       <OrphaCode>488280</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=488280</ExpertLink>
-      <Name lang="nl">14q32 duplicatiesyndroom</Name>
+      <Name lang="nl">14q32-duplicatiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -129187,7 +129199,7 @@
     <Disorder id="10851">
       <OrphaCode>64747</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=64747</ExpertLink>
-      <Name lang="nl">Ziekte van Charcot-Marie-Tooth, X-gebonden</Name>
+      <Name lang="nl">X-gebonden ziekte van Charcot-Marie-Tooth</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -132746,7 +132758,7 @@
     <Disorder id="11139">
       <OrphaCode>79097</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79097</ExpertLink>
-      <Name lang="nl">Folinezuurresponsieve epileptische aanvallen</Name>
+      <Name lang="nl">Folinezuurresponsieve insulten</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -132772,7 +132784,7 @@
     <Disorder id="11138">
       <OrphaCode>79096</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79096</ExpertLink>
-      <Name lang="nl">Pyridoxaalfosfaatafhankelijke epilepsie</Name>
+      <Name lang="nl">Pyridoxaalfosfaatresponsieve insulten</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -134475,7 +134487,7 @@
     <Disorder id="11095">
       <OrphaCode>75857</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=75857</ExpertLink>
-      <Name lang="nl">6q terminale deletiesyndroom</Name>
+      <Name lang="nl">Terminale 6q-deletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -136293,7 +136305,7 @@
     <Disorder id="27690">
       <OrphaCode>531151</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=531151</ExpertLink>
-      <Name lang="nl">9q21.13 microdeletiesyndroom</Name>
+      <Name lang="nl">9q21.13-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -136345,7 +136357,7 @@
     <Disorder id="10464">
       <OrphaCode>41751</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=41751</ExpertLink>
-      <Name lang="nl">Bietti kristallijne dystrofie</Name>
+      <Name lang="nl">Kristallijne dystrofie van Bietti</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -136388,7 +136400,7 @@
     <Disorder id="10394">
       <OrphaCode>35705</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35705</ExpertLink>
-      <Name lang="nl">Neurometabole stoornis als gevolg van serinedeficiëntie</Name>
+      <Name lang="nl">Neurometabole stoornis door deficiëntie van serine</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -137918,7 +137930,7 @@
     <Disorder id="10328">
       <OrphaCode>33573</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=33573</ExpertLink>
-      <Name lang="nl">Gamma-glutamyltranspeptidasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van gamma-glutamyltranspeptidase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -139430,7 +139442,7 @@
     <Disorder id="10735">
       <OrphaCode>53690</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=53690</ExpertLink>
-      <Name lang="nl">Congenitale lactasedeficiëntie</Name>
+      <Name lang="nl">Congenitale deficiëntie van lactase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -141683,7 +141695,7 @@
     <Disorder id="10579">
       <OrphaCode>43115</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=43115</ExpertLink>
-      <Name lang="nl">Erfelijke myopathie met melkzuuracidose door ISCU-deficiëntie</Name>
+      <Name lang="nl">Erfelijke myopathie met lactaatacidose door ISCU-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -145942,7 +145954,7 @@
     <Disorder id="27320">
       <OrphaCode>521258</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=521258</ExpertLink>
-      <Name lang="nl">Xq25 microduplicatiesyndroom</Name>
+      <Name lang="nl">Xq25-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -146654,7 +146666,7 @@
     <Disorder id="12222">
       <OrphaCode>93282</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93282</ExpertLink>
-      <Name lang="nl">Spondylo-epimetafysaire dysplasie, Pakistani-type</Name>
+      <Name lang="nl">Spondylo-epimetafysaire dysplasie, PAPSS2-type</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -148263,7 +148275,7 @@
     <Disorder id="27631">
       <OrphaCode>529962</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=529962</ExpertLink>
-      <Name lang="nl">17q24.2 microdeletiesyndroom</Name>
+      <Name lang="nl">17q24.2-microdeletiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -149980,7 +149992,7 @@
     <Disorder id="12100">
       <OrphaCode>90797</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90797</ExpertLink>
-      <Name lang="nl">Partieel androgeenongevoeligheidssyndroom</Name>
+      <Name lang="nl">Partieel androgeeninsensitiviteitssyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -161643,7 +161655,7 @@
     <Disorder id="12547">
       <OrphaCode>94066</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94066</ExpertLink>
-      <Name lang="nl">Ernstige intellectuele achterstand - epilepsie - anale anomalieën - distale falangeale hypoplasie</Name>
+      <Name lang="nl">Ernstige intellectuele achterstand - epilepsie - anale anomalieën - distale falangeale hypoplasie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -161672,7 +161684,7 @@
     <Disorder id="12546">
       <OrphaCode>94065</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=94065</ExpertLink>
-      <Name lang="nl">15q24 microdeletiesyndroom</Name>
+      <Name lang="nl">15q24-microdeletiesyndroom</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -162217,7 +162229,7 @@
     <Disorder id="12745">
       <OrphaCode>96121</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96121</ExpertLink>
-      <Name lang="nl">7q11.23 microduplicatiesyndroom</Name>
+      <Name lang="nl">7q11.23-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -162562,7 +162574,7 @@
     <Disorder id="12719">
       <OrphaCode>96095</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96095</ExpertLink>
-      <Name lang="nl">3q26 microduplicatiesyndroom</Name>
+      <Name lang="nl">3q26-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -162588,7 +162600,7 @@
     <Disorder id="12716">
       <OrphaCode>96092</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96092</ExpertLink>
-      <Name lang="nl">8p inversie duplicatie/deletie-syndroom</Name>
+      <Name lang="nl">8p geïnverteerde duplicatie/deletie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -162620,7 +162632,7 @@
     <Disorder id="12696">
       <OrphaCode>96072</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96072</ExpertLink>
-      <Name lang="nl">4p16.3 microduplicatiesyndroom</Name>
+      <Name lang="nl">4p16.3-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -162669,7 +162681,7 @@
     <Disorder id="12702">
       <OrphaCode>96078</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=96078</ExpertLink>
-      <Name lang="nl">16p13.3 microduplicatiesyndroom</Name>
+      <Name lang="nl">16p13.3-microduplicatiesyndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -164127,7 +164139,7 @@
     <Disorder id="12488">
       <OrphaCode>93928</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93928</ExpertLink>
-      <Name lang="nl">Geïsoleerde epispadieën</Name>
+      <Name lang="nl">Geïsoleerde epispadie</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -165712,7 +165724,7 @@
     <Disorder id="14212">
       <OrphaCode>99429</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99429</ExpertLink>
-      <Name lang="nl">Compleet androgeenongevoeligheidssyndroom</Name>
+      <Name lang="nl">Compleet androgeeninsensitiviteitssyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -168475,7 +168487,7 @@
     <Disorder id="13936">
       <OrphaCode>98919</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98919</ExpertLink>
-      <Name lang="nl">Syndroom van Miller Fisher</Name>
+      <Name lang="nl">Syndroom van Miller-Fisher</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -170149,7 +170161,7 @@
     <Disorder id="13623">
       <OrphaCode>98606</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98606</ExpertLink>
-      <Name lang="nl">Syndromale oogkasrandhypoplasie</Name>
+      <Name lang="nl">Syndromale hypoplasie van oogkasrand</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -170456,7 +170468,7 @@
     <Disorder id="13740">
       <OrphaCode>98723</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98723</ExpertLink>
-      <Name lang="nl">Hypoplastisch rechterhartsyndroom</Name>
+      <Name lang="nl">Hypoplastisch rechterhart-syndroom</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -171386,6 +171398,32 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="28850">
+      <OrphaCode>576278</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=576278</ExpertLink>
+      <Name lang="nl">SATB2-geassocieerd syndroom</Name>
+      <DisorderType id="21401">
+        <Name lang="nl">Malformatiesyndroom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23522">
+          <Name lang="nl">Kindsheid</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="nl">Neonataal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="nl">No data available</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="28815">
       <OrphaCode>573253</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=573253</ExpertLink>
@@ -171911,7 +171949,7 @@
     <Disorder id="28732">
       <OrphaCode>572333</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=572333</ExpertLink>
-      <Name lang="nl">Blefarofimose - ptose - epicanthus inversus-syndroom plus</Name>
+      <Name lang="nl">Blefarofimose - ptosis - epicanthus inversus-syndroom plus</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -173987,7 +174025,7 @@
     <Disorder id="14678">
       <OrphaCode>100924</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=100924</ExpertLink>
-      <Name lang="nl">Porfyrie door ALA-dehydratasedeficiëntie</Name>
+      <Name lang="nl">Porfyrie door deficiëntie van ALA-dehydratase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -175181,7 +175219,7 @@
     <Disorder id="14540">
       <OrphaCode>99967</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99967</ExpertLink>
-      <Name lang="nl">Myxoïd/ronde cel liposarcoom</Name>
+      <Name lang="nl">Myxoïd/rondcellig liposarcoom</Name>
       <DisorderType id="21457">
         <Name lang="nl">Histopathologisch subtype</Name>
       </DisorderType>
