commit 713764cbe665f619367fa836fae8e630672b2e3e
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Wed Dec 1 06:28:16 2021 +0100

    DECEMBER 2021

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
index 80f57e7..d01dd7e 100644
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-11-01 04:51:41" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
-  <DisorderList count="6633">
+<JDBOR date="2021-12-01 04:56:14" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -63,7 +56,7 @@
     <Disorder id="17603">
       <OrphaCode>166032</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166032</ExpertLink>
-      <Name lang="nl">Multipele epifysaire dysplasie, met mini-epifysen</Name>
+      <Name lang="nl">Multipele epifysaire dysplasie met mini-epifysen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -121,7 +114,7 @@
     <Disorder id="17602">
       <OrphaCode>166029</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166029</ExpertLink>
-      <Name lang="nl">Multipele epifysaire dysplasie, met ernstige proximale femorale dysplasie</Name>
+      <Name lang="nl">Multipele epifysaire dysplasie met ernstige proximale femorale dysplasie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -2008,7 +2001,7 @@
     <Disorder id="65">
       <OrphaCode>597</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=597</ExpertLink>
-      <Name lang="nl">Central core myopathie</Name>
+      <Name lang="nl">Central core-ziekte</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -3958,7 +3951,7 @@
     <Disorder id="17515">
       <OrphaCode>163654</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=163654</ExpertLink>
-      <Name lang="nl">Spondylo-epifysaire dysplasie, Cantu-type</Name>
+      <Name lang="nl">Spondylo-epifysaire dysplasie - brachydactylie - spraakstoornis-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -4589,7 +4582,7 @@
     <Disorder id="17521">
       <OrphaCode>163681</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=163681</ExpertLink>
-      <Name lang="nl">Corticale dysplasie - focale epilepsie-syndroom</Name>
+      <Name lang="nl">CNTNAP2-gerelateerde ontwikkelingsachterstand en epileptische encefalopathie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -5351,7 +5344,7 @@
     <Disorder id="209">
       <OrphaCode>628</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=628</ExpertLink>
-      <Name lang="nl">Diastrofische dwerggroei</Name>
+      <Name lang="nl">Diastrofische dysplasie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -6397,7 +6390,7 @@
     <Disorder id="257">
       <OrphaCode>1646</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1646</ExpertLink>
-      <Name lang="nl">Partiële chromosoom Y-deletie</Name>
+      <Name lang="nl">Partiële deletie van chromosoom Y</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -7138,7 +7131,7 @@
     <Disorder id="294">
       <OrphaCode>308</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=308</ExpertLink>
-      <Name lang="nl">Ziekte van Unverricht-Lundborg</Name>
+      <Name lang="nl">Progressieve myoclonische epilepsie type 1</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -11034,7 +11027,7 @@
     <Disorder id="457">
       <OrphaCode>226</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=226</ExpertLink>
-      <Name lang="nl">Dihydropteridinereductasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van dihydrobiopteridinereductase</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -12687,11 +12680,11 @@
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2346</ExpertLink>
       <Name lang="nl">Angio-osteohypertrofisch syndroom</Name>
-      <DisorderType id="21394">
-        <Name lang="nl">Ziekte</Name>
+      <DisorderType id="21436">
+        <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="nl">Aandoening</Name>
+      <DisorderGroup id="36540">
+        <Name lang="nl">Groep van aandoeningen</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="3">
         <AverageAgeOfOnset id="23536">
@@ -13985,7 +13978,7 @@
     <Disorder id="602">
       <OrphaCode>291</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=291</ExpertLink>
-      <Name lang="nl">Foetaal varicellasyndroom</Name>
+      <Name lang="nl">Congenitaal varicellasyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -19295,7 +19288,7 @@
     <Disorder id="771">
       <OrphaCode>13</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=13</ExpertLink>
-      <Name lang="nl">6-pyruvoyl-tetrahydropterinesynthasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van 6-pyruvoyl-tetrahydropterine-synthase</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -20778,7 +20771,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=183678</ExpertLink>
-      <Name lang="nl">Syndroom van Hermansky-Pudlak met neutropenie</Name>
+      <Name lang="nl">Syndroom van Hermansky-Pudlak door deficiëntie van AP-3</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -21023,10 +21016,13 @@
       <DisorderGroup id="36547">
         <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="nl">Volwassenheid</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23634">
@@ -22680,7 +22676,7 @@
     <Disorder id="967">
       <OrphaCode>888</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=888</ExpertLink>
-      <Name lang="nl">Syndroom van Van Der Woude</Name>
+      <Name lang="nl">Syndroom van Van der Woude</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -25347,7 +25343,7 @@
     <Disorder id="1372">
       <OrphaCode>1078</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1078</ExpertLink>
-      <Name lang="nl">Duimstijfheid - brachydactylie - intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Stijve duim - brachydactylie - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -28814,7 +28810,7 @@
     <Disorder id="1429">
       <OrphaCode>1170</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1170</ExpertLink>
-      <Name lang="nl">Autosomaal recessieve cerebelloparenchymale stoornis type 3</Name>
+      <Name lang="nl">Autosomaal recessieve cerebelloparenchymale aandoening type 3</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -29503,7 +29499,7 @@
     <Disorder id="1462">
       <OrphaCode>1216</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1216</ExpertLink>
-      <Name lang="nl">Autosomaal dominante congenitale goedaardige spinale musculaire atrofie</Name>
+      <Name lang="nl">Autosomaal dominante congenitale goedaardige spinale spieratrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -29677,7 +29673,7 @@
     <Disorder id="1440">
       <OrphaCode>1182</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1182</ExpertLink>
-      <Name lang="nl">Spastische ataxie met congenitale miosis</Name>
+      <Name lang="nl">Spastische ataxie met congenitale miose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -31460,7 +31456,7 @@
     <Disorder id="1600">
       <OrphaCode>1394</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1394</ExpertLink>
-      <Name lang="nl">Cerebro-facio-thoracale dysplasie</Name>
+      <Name lang="nl">Cerebrofaciothoracale dysplasie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -34412,7 +34408,7 @@
     <Disorder id="16916">
       <OrphaCode>139536</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139536</ExpertLink>
-      <Name lang="nl">Distale erfelijke motorische neuropathie type 5</Name>
+      <Name lang="nl">Distale hereditaire motorische neuropathie type 5</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -34589,7 +34585,7 @@
     <Disorder id="16921">
       <OrphaCode>139573</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=139573</ExpertLink>
-      <Name lang="nl">Erfelijke sensorische en autonome neuropathie met doofheid en algehele achterstand</Name>
+      <Name lang="nl">Erfelijke sensorische en autonome neuropathie met doofheid en algehele ontwikkelingsachterstand</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -35692,7 +35688,7 @@
     <Disorder id="1720">
       <OrphaCode>1568</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1568</ExpertLink>
-      <Name lang="nl">X-gebonden intellectuele achterstand - Dandy-Walker malformatie - basale gangliaziekte - insulten-syndroom</Name>
+      <Name lang="nl">X-gebonden intellectuele achterstand - Dandy-Walker malformatie - aandoening van basale ganglia - insulten-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -36634,7 +36630,7 @@
     <Disorder id="1899">
       <OrphaCode>1952</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1952</ExpertLink>
-      <Name lang="nl">Pacmandysplasie</Name>
+      <Name lang="nl">Epifysaire stippeling - osteoclastische hyperplasie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -43958,7 +43954,7 @@
     <Disorder id="19513">
       <OrphaCode>247234</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=247234</ExpertLink>
-      <Name lang="nl">Sporadische ataxie met aanvang bij volwassenen met onbekende oorzaak</Name>
+      <Name lang="nl">Sporadische ataxie met aanvang op volwassen leeftijd met onbekende etiologie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -47551,7 +47547,7 @@
     <Disorder id="2122">
       <OrphaCode>2252</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2252</ExpertLink>
-      <Name lang="nl">Radiale hypoplasie - trifalangeale duimen - hypospadie - maxillair diasteem-syndroom</Name>
+      <Name lang="nl">Hypoplasie van radius - trifalangeale duimen - hypospadie - maxillair diasteem-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -48291,7 +48287,7 @@
     <Disorder id="19621">
       <OrphaCode>251287</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251287</ExpertLink>
-      <Name lang="nl">Goedaardige concentrische ringvormige maculadystrofie</Name>
+      <Name lang="nl">Goedaardige concentrische annulaire maculadystrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -48614,7 +48610,7 @@
     <Disorder id="19625">
       <OrphaCode>251307</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251307</ExpertLink>
-      <Name lang="nl">Idiopathische recurrente pericarditis</Name>
+      <Name lang="nl">Idiopathische recidiverende pericarditis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -48715,7 +48711,7 @@
     <Disorder id="2441">
       <OrphaCode>2673</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2673</ExpertLink>
-      <Name lang="nl">Neuro-facio-digito-renaal syndroom</Name>
+      <Name lang="nl">Neurofaciodigitorenaal syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54827,7 +54823,7 @@
     <Disorder id="19882">
       <OrphaCode>261501</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261501</ExpertLink>
-      <Name lang="nl">Atypische ziekte van Norrie door microdeletie van Xp11.3</Name>
+      <Name lang="nl">Atypische ziekte van Norrie door Xp11.3-microdeletie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -56543,7 +56539,7 @@
     <Disorder id="2692">
       <OrphaCode>2990</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2990</ExpertLink>
-      <Name lang="nl">Autosomaal recessief meervoudig pterygium-syndroom</Name>
+      <Name lang="nl">Autosomaal recessief multipele pterygium-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -56572,7 +56568,7 @@
     <Disorder id="20039">
       <OrphaCode>263297</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=263297</ExpertLink>
-      <Name lang="nl">Glycogeenstapelingsziekte type 15</Name>
+      <Name lang="nl">Glycogeenstapelingsziekte met ernstige cardiomyopathie door deficiëntie van glycogenine</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -56627,7 +56623,7 @@
     <Disorder id="2688">
       <OrphaCode>2985</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2985</ExpertLink>
-      <Name lang="nl">Pseudoprogeriasyndroom</Name>
+      <Name lang="nl">Pseudoprogeria-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -56763,7 +56759,7 @@
     <Disorder id="2813">
       <OrphaCode>3144</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3144</ExpertLink>
-      <Name lang="nl">Schneckenbeckendysplasie</Name>
+      <Name lang="nl">Schneckenbecken-dysplasie</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -59535,7 +59531,7 @@
     <Disorder id="2678">
       <OrphaCode>2972</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2972</ExpertLink>
-      <Name lang="nl">Niet doorkomen van tanden - maxillaire hypoplasie - genu valgum</Name>
+      <Name lang="nl">Niet doorkomen van tanden - hypoplasie van maxilla - genu valgum-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -61739,7 +61735,7 @@
     <Disorder id="2954">
       <OrphaCode>3329</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3329</ExpertLink>
-      <Name lang="nl">Tibiale aplasie - ectrodactylie</Name>
+      <Name lang="nl">Aplasie van tibia - ectrodactylie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -62049,7 +62045,7 @@
     <Disorder id="2972">
       <OrphaCode>3353</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3353</ExpertLink>
-      <Name lang="nl">Trichodermodysplasie - dentale veranderingen</Name>
+      <Name lang="nl">Trichodermodysplasie - dentale veranderingen-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -62147,7 +62143,7 @@
     <Disorder id="20315">
       <OrphaCode>275517</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=275517</ExpertLink>
-      <Name lang="nl">Auto-immuun lymfoproliferatief syndroom met terugkerende virale infecties</Name>
+      <Name lang="nl">Auto-immuun lymfoproliferatief syndroom met recidiverende virale infecties</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -63905,7 +63901,7 @@
     <Disorder id="2879">
       <OrphaCode>3222</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3222</ExpertLink>
-      <Name lang="nl">Fosforibosylpyrofosfaatsynthetase-superactiviteit</Name>
+      <Name lang="nl">Fosforibosylpyrofosfaat-synthetase-superactiviteit</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -64687,7 +64683,7 @@
     <Disorder id="2836">
       <OrphaCode>3177</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3177</ExpertLink>
-      <Name lang="nl">Spinocerebellaire degeneratie - hoornvliesdystrofie-syndroom</Name>
+      <Name lang="nl">Spinocerebellaire degeneratie - corneadystrofie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -65439,7 +65435,7 @@
     <Disorder id="2932">
       <OrphaCode>3292</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3292</ExpertLink>
-      <Name lang="nl">Tel Hashomer camptodactyliesyndroom</Name>
+      <Name lang="nl">Camptodactylie-syndroom van Tel Hashomer</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -66788,7 +66784,7 @@
     <Disorder id="3295">
       <OrphaCode>943</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=943</ExpertLink>
-      <Name lang="nl">Malonzuuracidurie</Name>
+      <Name lang="nl">Malonacidurie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -70439,7 +70435,7 @@
     <Disorder id="18656">
       <OrphaCode>209370</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=209370</ExpertLink>
-      <Name lang="nl">Ernstige encefalopathie met microcefalie met neonatale aanvang</Name>
+      <Name lang="nl">Ernstige neonataal optredende encefalopathie met microcefalie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -72518,7 +72514,7 @@
     <Disorder id="18772">
       <OrphaCode>213777</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=213777</ExpertLink>
-      <Name lang="nl">Hooggradig neuro-endocrien carcinoom van de baarmoederhals</Name>
+      <Name lang="nl">Hooggradig neuro-endocrien carcinoom van cervix uteri</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -72541,7 +72537,7 @@
     <Disorder id="18779">
       <OrphaCode>213812</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=213812</ExpertLink>
-      <Name lang="nl">Maligne perifere neuro-ectodermale tumor van de baarmoederhals</Name>
+      <Name lang="nl">Primitieve neuro-ectodermale tumor van cervix uteri</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -75845,7 +75841,7 @@
     <Disorder id="3351">
       <OrphaCode>2170</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2170</ExpertLink>
-      <Name lang="nl">Methylcobalaminedeficiëntie type cblG</Name>
+      <Name lang="nl">Deficiëntie van methylcobalamine type cblG</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -76614,7 +76610,7 @@
     <Disorder id="3379">
       <OrphaCode>1561</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1561</ExpertLink>
-      <Name lang="nl">Fatale infantiele cytochroom C-oxidasedeficiëntie</Name>
+      <Name lang="nl">Fatale infantiele deficiëntie van cytochroom C-oxidase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -77169,7 +77165,7 @@
     <Disorder id="3765">
       <OrphaCode>547</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=547</ExpertLink>
-      <Name lang="nl">Non-Hodgkinlymfoom</Name>
+      <Name lang="nl">Non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -77611,7 +77607,7 @@
     <Disorder id="3747">
       <OrphaCode>543</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=543</ExpertLink>
-      <Name lang="nl">Burkittlymfoom</Name>
+      <Name lang="nl">Burkitt-lymfoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -78079,7 +78075,7 @@
     <Disorder id="3739">
       <OrphaCode>391</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=391</ExpertLink>
-      <Name lang="nl">Klassiek Hodgkinlymfoom</Name>
+      <Name lang="nl">Klassiek Hodgkin-lymfoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -80048,7 +80044,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231512</ExpertLink>
-      <Name lang="nl">Syndroom van Hermansky-Pudlak zonder longfibrose</Name>
+      <Name lang="nl">Syndroom van Hermansky-Pudlak door deficiëntie van BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -80135,7 +80131,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231500</ExpertLink>
-      <Name lang="nl">Syndroom van Hermansky-Pudlak met longfibrose</Name>
+      <Name lang="nl">Syndroom van Hermansky-Pudlak door deficiëntie van BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -82328,7 +82324,7 @@
     <Disorder id="19213">
       <OrphaCode>231720</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231720</ExpertLink>
-      <Name lang="nl">Niet-verworven gecombineerde hypofysehormoondeficiëntie - sensorineuraal gehoorverlies - wervelkolomafwijkingen-syndroom</Name>
+      <Name lang="nl">Niet-verworven gecombineerde deficiëntie van hypofysehormonen - sensorineuraal gehoorverlies - afwijkingen van wervelkolom-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -82951,7 +82947,7 @@
     <Disorder id="19277">
       <OrphaCode>238569</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238569</ExpertLink>
-      <Name lang="nl">Immuundisregulatie - inflammatoire darmziekte - artritis - recurrente infecties-syndroom</Name>
+      <Name lang="nl">Immuundisregulatie - inflammatoire darmziekte - artritis - recidiverende infecties-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -83678,7 +83674,7 @@
     <Disorder id="21955">
       <OrphaCode>330001</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=330001</ExpertLink>
-      <Name lang="nl">Wild type ATTR-amyloïdose</Name>
+      <Name lang="nl">Wildtype ATTR-amyloïdose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -84273,7 +84269,7 @@
     <Disorder id="21947">
       <OrphaCode>329918</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=329918</ExpertLink>
-      <Name lang="nl">Niet-immunoglobuline-gemedieerde membranoproliferatieve glomerulonefritis</Name>
+      <Name lang="nl">C3-glomerulopathie</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -85197,7 +85193,7 @@
     <Disorder id="21797">
       <OrphaCode>324648</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=324648</ExpertLink>
-      <Name lang="nl">Invasieve niet-tyfeuze salmonellose</Name>
+      <Name lang="nl">Invasieve niet-tyfoïde salmonellose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -86537,7 +86533,7 @@
     <Disorder id="21678">
       <OrphaCode>319589</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319589</ExpertLink>
-      <Name lang="nl">Autosomaal dominante Mendeliaanse gevoeligheid voor mycobacteriële ziekten door partiële IFNgammaR2-deficiëntie</Name>
+      <Name lang="nl">Autosomaal dominante Mendeliaanse susceptibiliteit voor mycobacteriële ziekten door partiële IFNgammaR2-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -86580,7 +86576,7 @@
     <Disorder id="21676">
       <OrphaCode>319574</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319574</ExpertLink>
-      <Name lang="nl">Autosomaal recessieve Mendeliaanse gevoeligheid voor mycobacteriële ziekten door partiële IFNgammaR2-deficiëntie</Name>
+      <Name lang="nl">Autosomaal recessieve Mendeliaanse susceptibiliteit voor mycobacteriële ziekten door partiële IFNgammaR2-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -87275,7 +87271,7 @@
     <Disorder id="21643">
       <OrphaCode>319308</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319308</ExpertLink>
-      <Name lang="nl">Niercelcarcinoom met translocatie van de MiT-familie</Name>
+      <Name lang="nl">Niercelcarcinoom met translocatie in MiT-familie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -88733,7 +88729,7 @@
     <Disorder id="22495">
       <OrphaCode>370396</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=370396</ExpertLink>
-      <Name lang="nl">Kleincellig ovariumcarcinoom</Name>
+      <Name lang="nl">Kleincellig carcinoom van ovarium</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -90701,7 +90697,7 @@
     <Disorder id="22342">
       <OrphaCode>363705</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363705</ExpertLink>
-      <Name lang="nl">Cranio-facio-fronto-digitaal syndroom</Name>
+      <Name lang="nl">Craniofaciofrontodigitaal syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92087,7 +92083,7 @@
     <Disorder id="22094">
       <OrphaCode>352657</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352657</ExpertLink>
-      <Name lang="nl">Erfelijke goedaardige intra-epitheliale dyskeratose</Name>
+      <Name lang="nl">Hereditaire benigne intra-epitheliale dyskeratose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92113,7 +92109,7 @@
     <Disorder id="22089">
       <OrphaCode>352641</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352641</ExpertLink>
-      <Name lang="nl">Autosomaal recessieve cerebellaire ataxie met spasticiteit met late aanvang</Name>
+      <Name lang="nl">Autosomaal recessieve cerebellaire ataxie met laat optredende spasticiteit</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92463,7 +92459,7 @@
     <Disorder id="22106">
       <OrphaCode>352718</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352718</ExpertLink>
-      <Name lang="nl">Progressieve retinale dystrofie door een defect van het retinoltransport</Name>
+      <Name lang="nl">Progressieve retinadystrofie door defect van retinoltransport</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92656,7 +92652,7 @@
     <Disorder id="22119">
       <OrphaCode>353217</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=353217</ExpertLink>
-      <Name lang="nl">Epileptische encefalopathie met globale cerebrale demyelinisatie</Name>
+      <Name lang="nl">Epileptische encefalopathie met gegeneraliseerde cerebrale demyelinisatie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -92904,7 +92900,7 @@
     <Disorder id="22058">
       <OrphaCode>352403</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=352403</ExpertLink>
-      <Name lang="nl">Spectrinegeassocieerde autosomaal recessieve cerebellaire ataxie</Name>
+      <Name lang="nl">Spectrine-geassocieerde autosomaal recessieve cerebellaire ataxie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -95538,7 +95534,7 @@
     <Disorder id="20713">
       <OrphaCode>289356</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=289356</ExpertLink>
-      <Name lang="nl">Primair niet-gestationeel choriocarcinoom van het ovarium</Name>
+      <Name lang="nl">Primair niet-gestationeel choriocarcinoom van ovarium</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -95613,7 +95609,7 @@
     <Disorder id="20707">
       <OrphaCode>289290</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=289290</ExpertLink>
-      <Name lang="nl">Hypermethioninemie-encefalopathie door adenosinekinasedeficiëntie</Name>
+      <Name lang="nl">Encefalopathie met hypermethioninemie door deficiëntie van adenosinekinase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -97961,7 +97957,7 @@
     <Disorder id="21449">
       <OrphaCode>313772</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=313772</ExpertLink>
-      <Name lang="nl">Spastisch ataxie - myoclonische epilepsie - neuropathie-syndroom met vroege aanvang</Name>
+      <Name lang="nl">Vroeg optredende spastisch ataxie - myoclonische epilepsie - neuropathie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -98184,7 +98180,7 @@
     <Disorder id="21492">
       <OrphaCode>314485</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314485</ExpertLink>
-      <Name lang="nl">Distale erfelijke motorneuropathie met aanvang in de vroege volwassenheid</Name>
+      <Name lang="nl">Distale hereditaire motorische neuropathie met aanvang in de vroege volwassenheid</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -98233,7 +98229,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314555</ExpertLink>
-      <Name lang="nl">Craniofaciale dysplasie - osteopenie-syndroom</Name>
+      <Name lang="nl">Faciale dysmorfie - ernstige myopie - osteopenie - intellectuele achterstand - dentale anomalieën-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -98550,7 +98546,7 @@
     <Disorder id="21477">
       <OrphaCode>314376</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=314376</ExpertLink>
-      <Name lang="nl">Intestinale obstructie bij de pasgeborene door gyanylaatcyclase 2C-deficiëntie</Name>
+      <Name lang="nl">Intestinale obstructie bij neonaat door gyanylaatcyclase 2C-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -99038,7 +99034,7 @@
     <Disorder id="21340">
       <OrphaCode>309031</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=309031</ExpertLink>
-      <Name lang="nl">Pancreatische triacylglycerollipasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van pancreatisch triacylglycerollipase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -99936,7 +99932,7 @@
     <Disorder id="21144">
       <OrphaCode>300846</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300846</ExpertLink>
-      <Name lang="nl">Agressieve B-cel non-Hodgkinlymfoom</Name>
+      <Name lang="nl">Agressieve B-cel non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -100130,7 +100126,7 @@
     <Disorder id="21143">
       <OrphaCode>300842</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300842</ExpertLink>
-      <Name lang="nl">Indolente B-cel non-Hodgkinlymfoom</Name>
+      <Name lang="nl">Indolente B-cel non-Hodgkin-lymfoom</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -100779,7 +100775,7 @@
     <Disorder id="21111">
       <OrphaCode>300324</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300324</ExpertLink>
-      <Name lang="nl">Persistente polyklonale B-cellymfocytose</Name>
+      <Name lang="nl">Persisterende polyklonale B-cellymfocytose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -100892,7 +100888,7 @@
     <Disorder id="21104">
       <OrphaCode>300284</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300284</ExpertLink>
-      <Name lang="nl">Bindweefselstoornis door lysylhydroxylase 3-deficiëntie</Name>
+      <Name lang="nl">Aandoening van bindweefsel door lysylhydroxylase 3-deficiëntie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -100970,7 +100966,7 @@
     <Disorder id="21119">
       <OrphaCode>300493</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300493</ExpertLink>
-      <Name lang="nl">Saglikersyndroom</Name>
+      <Name lang="nl">Syndroom van Sagliker</Name>
       <DisorderType id="21429">
         <Name lang="nl">Particulaire klinische situatie binnen een ziekte of syndroom</Name>
       </DisorderType>
@@ -102646,7 +102642,7 @@
     <Disorder id="23616">
       <OrphaCode>449291</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=449291</ExpertLink>
-      <Name lang="nl">Symptomatische vorm van fragiele X-syndroom in een vrouwelijke drager</Name>
+      <Name lang="nl">Symptomatische vorm van fragiele-X-syndroom bij vrouwelijke dragers</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -102736,7 +102732,7 @@
     <Disorder id="23623">
       <OrphaCode>449432</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=449432</ExpertLink>
-      <Name lang="nl">IgG4-gerelateerde submandibulaire klierziekte</Name>
+      <Name lang="nl">IgG4-gerelateerde aandoening van submandibulaire klier</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -107555,7 +107551,7 @@
     <Disorder id="23018">
       <OrphaCode>411536</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=411536</ExpertLink>
-      <Name lang="nl">Milde fosforibosylpyrofosfaatsynthetase-superactiviteit</Name>
+      <Name lang="nl">Milde fosforibosylpyrofosfaat-synthetase-superactiviteit</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -107581,7 +107577,7 @@
     <Disorder id="23019">
       <OrphaCode>411543</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=411543</ExpertLink>
-      <Name lang="nl">Ernstige fosforibosylpyrofosfaatsynthetase-superactiviteit</Name>
+      <Name lang="nl">Ernstige fosforibosylpyrofosfaat-synthetase-superactiviteit</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -109482,7 +109478,7 @@
     <Disorder id="22701">
       <OrphaCode>397758</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=397758</ExpertLink>
-      <Name lang="nl">Retinale dystrofie met disfunctie van inwendige retina en ganglioncelanomalieën</Name>
+      <Name lang="nl">Retinadystrofie met disfunctie van inwendige retina en anomalieën van ganglioncellen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -115548,7 +115544,7 @@
     <Disorder id="23164">
       <OrphaCode>423693</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=423693</ExpertLink>
-      <Name lang="nl">Rechterventrikel met dubbele uitgang met subaortaal of doubly-committed ventrikelseptumdefect</Name>
+      <Name lang="nl">Rechterventrikel met dubbele uitstroom met subaortaal of doubly-committed ventrikelseptumdefect</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -115568,7 +115564,7 @@
     <Disorder id="23165">
       <OrphaCode>423712</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=423712</ExpertLink>
-      <Name lang="nl">Rechterventrikel met dubbele uitgang met atrioventriculair septumdefect, pulmonalisklepstenose, heterotaxie</Name>
+      <Name lang="nl">Rechterventrikel met dubbele uitstroom met atrioventriculair septumdefect, pulmonalisklepstenose, heterotaxie</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -116281,7 +116277,7 @@
     <Disorder id="23207">
       <OrphaCode>424996</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=424996</ExpertLink>
-      <Name lang="nl">Plaveiselcelcarcinoom van de galblaas en extrahepatische galwegen</Name>
+      <Name lang="nl">Plaveiselcelcarcinoom van galblaas en extrahepatische galwegen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -117535,7 +117531,7 @@
     <Disorder id="8759">
       <OrphaCode>26137</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=26137</ExpertLink>
-      <Name lang="nl">Juveniele temporale arteriitis</Name>
+      <Name lang="nl">Juveniele arteriitis temporalis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -124968,7 +124964,7 @@
     <Disorder id="25161">
       <OrphaCode>477831</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=477831</ExpertLink>
-      <Name lang="nl">Skeletale overgroei - craniofaciale dysmorfie - hyperelastische huid - wittestofletsels-syndroom</Name>
+      <Name lang="nl">Overgroeisyndroom van Kosaki</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -126526,7 +126522,7 @@
     <Disorder id="10998">
       <OrphaCode>71276</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=71276</ExpertLink>
-      <Name lang="nl">Silent sinussyndroom</Name>
+      <Name lang="nl">Silent sinus-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -127394,7 +127390,7 @@
     <Disorder id="10932">
       <OrphaCode>69663</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=69663</ExpertLink>
-      <Name lang="nl">Laag fosfolipidegeassocieerde cholelithiasis</Name>
+      <Name lang="nl">Cholelithiase geassocieerd met laag gehalte aan fosfolipiden</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -128352,7 +128348,7 @@
     <Disorder id="10888">
       <OrphaCode>66630</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=66630</ExpertLink>
-      <Name lang="nl">Congenitale pseudoartrose van de clavicula</Name>
+      <Name lang="nl">Congenitale pseudoartrose van clavicula</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -128439,7 +128435,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=66627</ExpertLink>
-      <Name lang="nl">Gepigmenteerde villonodulaire synovitis</Name>
+      <Name lang="nl">Tenosynoviale reusceltumor</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -128858,7 +128854,7 @@
     <Disorder id="10872">
       <OrphaCode>65682</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=65682</ExpertLink>
-      <Name lang="nl">Goedaardige terugkerende intrahepatische cholestase</Name>
+      <Name lang="nl">Goedaardige recidiverende intrahepatische cholestase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -129211,7 +129207,7 @@
     <Disorder id="10855">
       <OrphaCode>64751</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=64751</ExpertLink>
-      <Name lang="nl">Erfelijke motorische en sensorische neuropathie type 5</Name>
+      <Name lang="nl">Hereditaire motorische en sensorische neuropathie type 5</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -129630,7 +129626,7 @@
     <Disorder id="10836">
       <OrphaCode>64692</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=64692</ExpertLink>
-      <Name lang="nl">Oroyakoorts</Name>
+      <Name lang="nl">Oroya-koorts</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -131666,7 +131662,7 @@
     <Disorder id="11215">
       <OrphaCode>79189</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79189</ExpertLink>
-      <Name lang="nl">Stoornis van de peroxisoombiogenese</Name>
+      <Name lang="nl">Stoornis van peroxisoombiogenese</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -132533,7 +132529,7 @@
     <Disorder id="11143">
       <OrphaCode>79101</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79101</ExpertLink>
-      <Name lang="nl">Hyperprolinemie type II</Name>
+      <Name lang="nl">Hyperprolinemie type 2</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -132747,7 +132743,7 @@
     <Disorder id="11164">
       <OrphaCode>79138</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79138</ExpertLink>
-      <Name lang="nl">Bickerstaff hersenstamencefalitis</Name>
+      <Name lang="nl">Bickerstaff-hersenstamencefalitis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -137583,7 +137579,7 @@
     <Disorder id="10327">
       <OrphaCode>33572</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=33572</ExpertLink>
-      <Name lang="nl">5-oxoprolinasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van 5-oxoprolinase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -137867,7 +137863,7 @@
     <Disorder id="10329">
       <OrphaCode>33574</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=33574</ExpertLink>
-      <Name lang="nl">Glutamaat-cysteïneligasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van glutamaat-cysteïneligase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -137974,7 +137970,7 @@
     <Disorder id="10308">
       <OrphaCode>33108</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=33108</ExpertLink>
-      <Name lang="nl">Letaal meervoudig pterygium-syndroom</Name>
+      <Name lang="nl">Letaal multipele pterygium-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -138365,7 +138361,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35062</ExpertLink>
-      <Name lang="nl">Idiopathische gedissemineerde cytomegalovirusinfectie</Name>
+      <Name lang="nl">Ernstige gedissemineerde infectie door cytomegalovirus bij immuuncompetente patiënten</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -138391,7 +138387,7 @@
     <Disorder id="10342">
       <OrphaCode>34520</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=34520</ExpertLink>
-      <Name lang="nl">Congenitale spierdystrofie met integrine alfa-7-deficiëntie</Name>
+      <Name lang="nl">Congenitale spierdystrofie met deficiëntie van integrine alfa-7</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -140884,7 +140880,7 @@
     <Disorder id="10662">
       <OrphaCode>50942</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=50942</ExpertLink>
-      <Name lang="nl">Gegroefde palmoplantaire keratodermie</Name>
+      <Name lang="nl">Keratosis palmoplantaris striata</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -141765,7 +141761,7 @@
     <Disorder id="10590">
       <OrphaCode>45452</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=45452</ExpertLink>
-      <Name lang="nl">Idiopathische neonatale atriale flutter</Name>
+      <Name lang="nl">Idiopathische neonatale atriumflutter</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -144302,11 +144298,11 @@
       <OrphaCode>90308</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90308</ExpertLink>
       <Name lang="nl">Syndroom van Klippel-Trénaunay</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype van aandoening</Name>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -144331,11 +144327,11 @@
       <OrphaCode>90307</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=90307</ExpertLink>
       <Name lang="nl">Syndroom van Parkes Weber</Name>
-      <DisorderType id="21450">
-        <Name lang="nl">Klinisch subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="nl">Subtype van aandoening</Name>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -147121,7 +147117,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93111</ExpertLink>
-      <Name lang="nl">Renale cysten en diabetes-syndroom</Name>
+      <Name lang="nl">HNF1B-gerelateerde autosomaal dominante tubulo-interstitiële nierziekte</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -149390,7 +149386,7 @@
     <Disorder id="12154">
       <OrphaCode>91495</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=91495</ExpertLink>
-      <Name lang="nl">Persistent hyperplastisch primair glasachtig lichaam</Name>
+      <Name lang="nl">Persisterend hyperplastisch primair glasvocht</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -150757,7 +150753,7 @@
     <Disorder id="11473">
       <OrphaCode>79447</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79447</ExpertLink>
-      <Name lang="nl">X-gebonden letaal meervoudig pterygium-syndroom</Name>
+      <Name lang="nl">X-gebonden letaal multipele pterygium-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -153827,7 +153823,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85279</ExpertLink>
-      <Name lang="nl">Syndromale X-gebonden intellectuele achterstand als gevolg van JARID1C-mutaties</Name>
+      <Name lang="nl">KDM5C-gerelateerde syndromale X-gebonden intellectuele achterstand</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -155991,7 +155987,7 @@
     <Disorder id="11775">
       <OrphaCode>86893</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=86893</ExpertLink>
-      <Name lang="nl">Nodulair lymfocyt-predominant Hodgkinlymfoom</Name>
+      <Name lang="nl">Nodulair lymfocyt-predominant Hodgkin-lymfoom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -156167,7 +156163,7 @@
     <Disorder id="11528">
       <OrphaCode>79502</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79502</ExpertLink>
-      <Name lang="nl">Puntvormige palmoplantaire keratodermie type 2</Name>
+      <Name lang="nl">Keratoderma palmoplantaris punctata type 2</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -157708,7 +157704,7 @@
     <Disorder id="11601">
       <OrphaCode>83639</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=83639</ExpertLink>
-      <Name lang="nl">Hypercoagulabiliteitssyndroom als gevolg van glycosylfosfatidylinositoldeficiëntie</Name>
+      <Name lang="nl">Hypercoagulabiliteit-syndroom door deficiëntie van glycosylfosfatidylinositol</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -158154,7 +158150,7 @@
     <Disorder id="11647">
       <OrphaCode>85186</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=85186</ExpertLink>
-      <Name lang="nl">Endosteale sclerose - cerebellaire hypoplasie</Name>
+      <Name lang="nl">Endosteale sclerose - cerebellaire hypoplasie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -158536,7 +158532,7 @@
     <Disorder id="30684">
       <OrphaCode>610573</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=610573</ExpertLink>
-      <Name lang="nl">CLCN6-related childhood onset progressive neurodegeneration-peripheral neuropathy-syndrome</Name>
+      <Name lang="nl">CLCN6-gerelateerd in de kindertijd optredend progressieve neurodegeneratie - perifere neuropathie-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -158556,7 +158552,7 @@
     <Disorder id="30683">
       <OrphaCode>610569</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=610569</ExpertLink>
-      <Name lang="nl">KIAA1109-related early lethal-congenital brain mlaformations-arthrogryposis-syndrome</Name>
+      <Name lang="nl">KIAA1109-gerelateerd vroeg letaal congenitale hersenmalformaties - artrogrypose-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -158689,7 +158685,7 @@
     <Disorder id="13310">
       <OrphaCode>98293</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98293</ExpertLink>
-      <Name lang="nl">Hodgkinlymfoom</Name>
+      <Name lang="nl">Hodgkin-lymfoom</Name>
       <DisorderType id="21436">
         <Name lang="nl">Group of phenomes</Name>
       </DisorderType>
@@ -160254,7 +160250,7 @@
     <Disorder id="12856">
       <OrphaCode>97229</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97229</ExpertLink>
-      <Name lang="nl">Riboflavinetransporterdeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van riboflavinetransporter</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -160338,7 +160334,7 @@
     <Disorder id="12937">
       <OrphaCode>97598</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=97598</ExpertLink>
-      <Name lang="nl">Congenitale renale arteriestenose</Name>
+      <Name lang="nl">Congenitale nierarteriestenose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -163189,6 +163185,29 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29924">
+      <OrphaCode>599373</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=599373</ExpertLink>
+      <Name lang="nl">STXBP1-gerelateerde encefalopathie</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23515">
+          <Name lang="nl">Neonataal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="nl">No data available</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="12325">
       <OrphaCode>93403</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93403</ExpertLink>
@@ -165734,6 +165753,61 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29564">
+      <OrphaCode>592564</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=592564</ExpertLink>
+      <Name lang="nl">GNAO1-gerelateerd ontwikkelingsachterstand - insulten - bewegingsstoornis-spectrum</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="nl">Kindertijd</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="nl">Kindsheid</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="nl">Neonataal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="nl">No data available</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29565">
+      <OrphaCode>592570</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=592570</ExpertLink>
+      <Name lang="nl">TRAF7-geassocieerd hartdefect - digitale anomalieën - faciale dysmorfie - motorische achterstand en spraakachterstand-syndroom</Name>
+      <DisorderType id="21401">
+        <Name lang="nl">Malformatiesyndroom</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="nl">Antenataal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="nl">Neonataal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="nl">No data available</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14291">
       <OrphaCode>99718</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99718</ExpertLink>
@@ -165836,6 +165910,172 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29441">
+      <OrphaCode>589821</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=589821</ExpertLink>
+      <Name lang="nl">Myotone dystrofie van Steinert met congenitale aanvang</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="nl">Antenataal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="nl">Neonataal</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="nl">Vroege kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="nl">Infantiel</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="nl">Late kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="nl">Jonge volwassenheid</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29443">
+      <OrphaCode>589827</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=589827</ExpertLink>
+      <Name lang="nl">Myotone dystrofie van Steinert met juveniele aanvang</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="nl">Kindertijd</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="nl">Vroege kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="nl">Late kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="nl">Jonge volwassenheid</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29442">
+      <OrphaCode>589824</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=589824</ExpertLink>
+      <Name lang="nl">Myotone dystrofie van Steinert met aanvang in de kindertijd</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="nl">Kindertijd</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="nl">Kindsheid</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="nl">Puber</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="nl">Vroege kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="nl">Late kindertijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="nl">Jonge volwassenheid</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29445">
+      <OrphaCode>589833</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=589833</ExpertLink>
+      <Name lang="nl">Myotone dystrofie van Steinert met late aanvang</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23550">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29444">
+      <OrphaCode>589830</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=589830</ExpertLink>
+      <Name lang="nl">Myotone dystrofie van Steinert met aanvang op volwassen leeftijd</Name>
+      <DisorderType id="21450">
+        <Name lang="nl">Klinisch subtype</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="nl">Subtype van aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="1">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="nl">Volwassenheid</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="nl">Volwassene</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="nl">Op leeftijd</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="nl">Jonge volwassenheid</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29447">
       <OrphaCode>589905</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=589905</ExpertLink>
@@ -166767,7 +167007,7 @@
     <Disorder id="13861">
       <OrphaCode>98844</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98844</ExpertLink>
-      <Name lang="nl">Klassiek Hodgkinlymfoom, type gemengde cellulariteit</Name>
+      <Name lang="nl">Klassiek Hodgkin-lymfoom, type gemengde cellulariteit</Name>
       <DisorderType id="21457">
         <Name lang="nl">Histopathologisch subtype</Name>
       </DisorderType>
@@ -166787,7 +167027,7 @@
     <Disorder id="13860">
       <OrphaCode>98843</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98843</ExpertLink>
-      <Name lang="nl">Klassiek Hodgkinlymfoom, type nodulaire sclerose</Name>
+      <Name lang="nl">Klassiek Hodgkin-lymfoom, type nodulaire sclerose</Name>
       <DisorderType id="21457">
         <Name lang="nl">Histopathologisch subtype</Name>
       </DisorderType>
@@ -166807,7 +167047,7 @@
     <Disorder id="13863">
       <OrphaCode>98846</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98846</ExpertLink>
-      <Name lang="nl">Klassiek Hodgkinlymfoom, lymfocytarm type</Name>
+      <Name lang="nl">Klassiek Hodgkin-lymfoom, lymfocytarm type</Name>
       <DisorderType id="21457">
         <Name lang="nl">Histopathologisch subtype</Name>
       </DisorderType>
@@ -166827,7 +167067,7 @@
     <Disorder id="13862">
       <OrphaCode>98845</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98845</ExpertLink>
-      <Name lang="nl">Klassiek Hodgkinlymfoom, lymfocytrijk type</Name>
+      <Name lang="nl">Klassiek Hodgkin-lymfoom, lymfocytrijk type</Name>
       <DisorderType id="21457">
         <Name lang="nl">Histopathologisch subtype</Name>
       </DisorderType>
@@ -167427,7 +167667,7 @@
     <Disorder id="13888">
       <OrphaCode>98871</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98871</ExpertLink>
-      <Name lang="nl">Tijdelijke erytroblastopenie in de kinderjaren</Name>
+      <Name lang="nl">Transiënte erytroblastopenie in de kindertijd</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -168005,7 +168245,7 @@
     <Disorder id="13937">
       <OrphaCode>98920</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98920</ExpertLink>
-      <Name lang="nl">Spinale musculaire atrofie met respiratoire insufficiëntie type 1</Name>
+      <Name lang="nl">Spinale spieratrofie met respiratoire insufficiëntie type 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -168170,7 +168410,7 @@
     <Disorder id="13973">
       <OrphaCode>98956</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98956</ExpertLink>
-      <Name lang="nl">Epitheliale basale membraan dystrofie</Name>
+      <Name lang="nl">Epitheliale basale membraan-dystrofie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -169257,7 +169497,7 @@
     <Disorder id="14060">
       <OrphaCode>99043</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99043</ExpertLink>
-      <Name lang="nl">Rechterventrikel met dubbele uitgang met subaortaal of doubly-committed ventrikelseptumdefect met pulmonalisklepstenose</Name>
+      <Name lang="nl">Rechterventrikel met dubbele uitstroom met subaortaal of doubly-committed ventrikelseptumdefect met pulmonalisklepstenose</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -169277,7 +169517,7 @@
     <Disorder id="14062">
       <OrphaCode>99045</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99045</ExpertLink>
-      <Name lang="nl">Rechterventrikel met dubbele uitgang met subpulmonaal ventrikelseptumdefect</Name>
+      <Name lang="nl">Rechterventrikel met dubbele uitstroom met subpulmonaal ventrikelseptumdefect</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -169297,7 +169537,7 @@
     <Disorder id="14063">
       <OrphaCode>99046</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=99046</ExpertLink>
-      <Name lang="nl">Rechterventrikel met dubbele uitgang met non-committed subpulmonaal ventrikelseptumdefect</Name>
+      <Name lang="nl">Rechterventrikel met dubbele uitstroom met non-committed subpulmonaal ventrikelseptumdefect</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
