commit 3ebab16f9c1ea026b2455fead790a4163a9f32d1
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 1 11:23:35 2021 +0200

    JUNE 2021

diff --git a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml
old mode 100755
new mode 100644
index 07418d4..5801c8f
--- a/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/nl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-05-01 04:43:24" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-06-01 04:45:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6566">
+  <DisorderList count="6568">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=166024</ExpertLink>
@@ -3716,12 +3716,12 @@
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23424">
+          <Name lang="nl">Multigeen/multifactorieel</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23494">
           <Name lang="nl">Niet toepasbaar</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23480">
-          <Name lang="nl">Unknown</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="17490">
@@ -4303,7 +4303,7 @@
     <Disorder id="164">
       <OrphaCode>2182</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2182</ExpertLink>
-      <Name lang="nl">Hydrocefalus met stenose van het aquaduct van Sylvius</Name>
+      <Name lang="nl">Hydrocefalus met stenose van aquaduct van Sylvius</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -14854,13 +14854,16 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="3">
         <TypeOfInheritance id="23410">
           <Name lang="nl">Autosomaal dominant</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
+        <TypeOfInheritance id="23431">
+          <Name lang="nl">X-gebonden recessief</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="17999">
@@ -28833,7 +28836,7 @@
     <Disorder id="16705">
       <OrphaCode>137678</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=137678</ExpertLink>
-      <Name lang="nl">Tsjechische dysplasie, metatarsaal type</Name>
+      <Name lang="nl">Spondylo-epifysaire dysplasie met abnormaal korte metatarsalen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -31094,7 +31097,7 @@
     <Disorder id="1614">
       <OrphaCode>1416</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1416</ExpertLink>
-      <Name lang="nl">Familiale calciumpyrofosfaatafzetting</Name>
+      <Name lang="nl">Familiale depositie van calciumpyrofosfaat</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -35366,7 +35369,7 @@
     <Disorder id="1668">
       <OrphaCode>1497</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1497</ExpertLink>
-      <Name lang="nl">X-gebonden gecompliceerde dysgenesie van het corpus callosum</Name>
+      <Name lang="nl">X-gebonden gecompliceerde dysgenesie van corpus callosum</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -40301,7 +40304,7 @@
     <Disorder id="17237">
       <OrphaCode>158687</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158687</ExpertLink>
-      <Name lang="nl">Letale acantholytische epidermolysis bullosa</Name>
+      <Name lang="nl">Letale acantholytische erosieve aandoening</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -40414,7 +40417,7 @@
     <Disorder id="17235">
       <OrphaCode>158681</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158681</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex met circinaat migrerend erytheem</Name>
+      <Name lang="nl">Epidermolysis bullosa simplex met circinair migrerend erytheem</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -40440,7 +40443,7 @@
     <Disorder id="17234">
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158676</ExpertLink>
-      <Name lang="nl">Dominante dystrofische epidermolysis bullosa, enkel nagels</Name>
+      <Name lang="nl">Gelokaliseerde dystrofische epidermolysis bullosa van uitsluitend nagels</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -40469,7 +40472,7 @@
     <Disorder id="17233">
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158673</ExpertLink>
-      <Name lang="nl">Acrale dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Gelokaliseerde dystrofische epidermolysis bullosa, acrale vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -40498,7 +40501,7 @@
     <Disorder id="17232">
       <OrphaCode>158668</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=158668</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex als gevolg van plakophilinedeficiëntie</Name>
+      <Name lang="nl">Ectodermale dysplasie - fragiliteit van huid-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -43606,7 +43609,7 @@
     <Disorder id="2296">
       <OrphaCode>2483</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2483</ExpertLink>
-      <Name lang="nl">Melkersson-Rosenthalsyndroom</Name>
+      <Name lang="nl">Syndroom van Melkersson-Rosenthal</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -46819,7 +46822,7 @@
     <Disorder id="2119">
       <OrphaCode>2249</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2249</ExpertLink>
-      <Name lang="nl">Ulna-hypoplasie - intellectuele achterstand</Name>
+      <Name lang="nl">Hypoplasie van ulna - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -47765,7 +47768,7 @@
     <Disorder id="19640">
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=251393</ExpertLink>
-      <Name lang="nl">Gelokaliseerde junctionele epidermolysis bullosa, niet-Herlitz-type</Name>
+      <Name lang="nl">Gelokaliseerde junctionele epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -51243,7 +51246,7 @@
     <Disorder id="2533">
       <OrphaCode>1306</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=1306</ExpertLink>
-      <Name lang="nl">Buschke-Ollendorffsyndroom</Name>
+      <Name lang="nl">Syndroom van Buschke-Ollendorff</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -54010,7 +54013,7 @@
     <Disorder id="19879">
       <OrphaCode>261476</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=261476</ExpertLink>
-      <Name lang="nl">Xp21 microdeletiesyndroom</Name>
+      <Name lang="nl">Xp21 deletiesyndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -59421,7 +59424,7 @@
     <Disorder id="2670">
       <OrphaCode>740</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=740</ExpertLink>
-      <Name lang="nl">Hutchinson-Gilford progeriasyndroom</Name>
+      <Name lang="nl">Progeriasyndroom van Hutchinson-Gilford</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -62247,7 +62250,7 @@
     <Disorder id="3057">
       <OrphaCode>3243</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3243</ExpertLink>
-      <Name lang="nl">Sweetsyndroom</Name>
+      <Name lang="nl">Syndroom van Sweet</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -67435,13 +67438,16 @@
           <Name lang="nl">Normale levensverwachting</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="3">
+      <TypeOfInheritanceList count="4">
         <TypeOfInheritance id="23410">
           <Name lang="nl">Autosomaal dominant</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
+        <TypeOfInheritance id="23424">
+          <Name lang="nl">Multigeen/multifactorieel</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23431">
           <Name lang="nl">X-gebonden recessief</Name>
         </TypeOfInheritance>
@@ -68629,13 +68635,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23410">
           <Name lang="nl">Autosomaal dominant</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="nl">Autosomaal recessief</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="18604">
@@ -69988,7 +69991,7 @@
     <Disorder id="3525">
       <OrphaCode>3286</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=3286</ExpertLink>
-      <Name lang="nl">Catecholaminerge polymorfe ventrikeltachycardie</Name>
+      <Name lang="nl">Catecholaminerge polymorfe ventriculaire tachycardie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -73468,7 +73471,7 @@
     <Disorder id="3423">
       <OrphaCode>2745</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2745</ExpertLink>
-      <Name lang="nl">Opitz G/BBB-syndroom</Name>
+      <Name lang="nl">Opitz GBBB-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -74621,9 +74624,12 @@
           <Name lang="nl">Normale levensverwachting</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="nl">Niet toepasbaar</Name>
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23424">
+          <Name lang="nl">Multigeen/multifactorieel</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23480">
+          <Name lang="nl">Unknown</Name>
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
@@ -77963,7 +77969,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23424">
           <Name lang="nl">Multigeen/multifactorieel</Name>
         </TypeOfInheritance>
@@ -79975,6 +79984,26 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="3625">
+      <OrphaCode>2197</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2197</ExpertLink>
+      <Name lang="nl">Idiopathische hypercalciurie</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="19175">
       <OrphaCode>231237</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231237</ExpertLink>
@@ -81179,7 +81208,7 @@
     <Disorder id="19201">
       <OrphaCode>231568</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=231568</ExpertLink>
-      <Name lang="nl">Gegeneraliseerde dominante dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Autosomaal dominante gegeneraliseerde dystrofische epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -82463,6 +82492,38 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="19292">
+      <OrphaCode>238666</OrphaCode>
+      <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238666</ExpertLink>
+      <Name lang="nl">Geïsoleerd congenitaal hypogonadotroop hypogonadisme</Name>
+      <DisorderType id="21394">
+        <Name lang="nl">Ziekte</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="nl">Aandoening</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="5">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23417">
+          <Name lang="nl">Autosomaal recessief</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23459">
+          <Name lang="nl">Oligogeen</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23480">
+          <Name lang="nl">Unknown</Name>
+        </TypeOfInheritance>
+        <TypeOfInheritance id="23431">
+          <Name lang="nl">X-gebonden recessief</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="19291">
       <OrphaCode>238654</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=238654</ExpertLink>
@@ -86156,7 +86217,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=319671</ExpertLink>
-      <Name lang="nl">Microcefale primordiale dwerggroei, Alazami-type</Name>
+      <Name lang="nl">Syndroom van Alazami</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -86815,13 +86876,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23410">
           <Name lang="nl">Autosomaal dominant</Name>
         </TypeOfInheritance>
-        <TypeOfInheritance id="23431">
-          <Name lang="nl">X-gebonden recessief</Name>
-        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="21581">
@@ -90443,7 +90501,7 @@
     <Disorder id="22327">
       <OrphaCode>363611</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=363611</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand - voedingsproblemen - ontwikkelingsachterstand - microcefalie-syndroom</Name>
+      <Name lang="nl">CTCF-gerelateerde neurologische ontwikkelingsstoonis</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -91393,7 +91451,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23494">
           <Name lang="nl">Niet toepasbaar</Name>
         </TypeOfInheritance>
@@ -93730,13 +93791,16 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
+      <TypeOfInheritanceList count="3">
         <TypeOfInheritance id="23410">
           <Name lang="nl">Autosomaal dominant</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
+        <TypeOfInheritance id="23431">
+          <Name lang="nl">X-gebonden recessief</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="5544">
@@ -99580,7 +99644,7 @@
     <Disorder id="21219">
       <OrphaCode>306504</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=306504</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa met respiratoire en renale betrokkenheid</Name>
+      <Name lang="nl">Interstitiële longziekte - nefrotisch syndroom - epidermolysis bullosa-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -100201,7 +100265,7 @@
     <Disorder id="21112">
       <OrphaCode>300333</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=300333</ExpertLink>
-      <Name lang="nl">Nefrotisch syndroom - doofheid - pretibiale epidermolysis bullosa-syndroom</Name>
+      <Name lang="nl">Nefrotisch syndroom - epidermolysis bullosa - sensorineurale doofheid-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -100253,7 +100317,7 @@
     <Disorder id="24042">
       <OrphaCode>464738</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=464738</ExpertLink>
-      <Name lang="nl">Congenitaal cataract - microcefalie - nevus flammeus simplex - ernstige intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Syndroom van Basel-Vanagaite-Smirin-Yosef</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -104823,7 +104887,7 @@
     <Disorder id="24234">
       <OrphaCode>468678</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=468678</ExpertLink>
-      <Name lang="nl">Intellectuele achterstand - microcefalie - strabisme - gedragsstoornissen-syndroom</Name>
+      <Name lang="nl">Syndroom van White-Sutton</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -106750,7 +106814,7 @@
     <Disorder id="23023">
       <OrphaCode>411629</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=411629</ExpertLink>
-      <Name lang="nl">Nefropathische infantiele cystinose</Name>
+      <Name lang="nl">Infantiele nefropathische cystinose</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -107452,7 +107516,7 @@
     <Disorder id="22634">
       <OrphaCode>391641</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=391641</ExpertLink>
-      <Name lang="nl">Feingoldsyndroom type 1</Name>
+      <Name lang="nl">Syndroom van Feingold type 1</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -110509,7 +110573,7 @@
     <Disorder id="23367">
       <OrphaCode>438178</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=438178</ExpertLink>
-      <Name lang="nl">Acyl-CoA-reductase 1-deficiëntie</Name>
+      <Name lang="nl">Deficiëntie van vetzuur-acyl-CoA-reductase 1</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -117543,7 +117607,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=508529</ExpertLink>
-      <Name lang="nl">Gegeneraliseerde basale epidermolysis bullosa simplex met huidatrofie, littekenvorming en haarverlies</Name>
+      <Name lang="nl">Intermediaire epidermolysis bullosa simplex met cardiomyopathie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -118684,7 +118748,10 @@
           <Name lang="nl">No data available</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="nl">Autosomaal dominant</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="26001">
@@ -118719,7 +118786,7 @@
     <Disorder id="26002">
       <OrphaCode>500163</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=500163</ExpertLink>
-      <Name lang="nl">SIN3A-gerelateerde intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Syndroom van Witteveen-Kolk</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -120360,7 +120427,7 @@
     <Disorder id="8662">
       <OrphaCode>901</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=901</ExpertLink>
-      <Name lang="nl">Wellssyndroom</Name>
+      <Name lang="nl">Syndroom van Wells</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -121025,7 +121092,7 @@
     <Disorder id="8693">
       <OrphaCode>483</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=483</ExpertLink>
-      <Name lang="nl">Congenitale hoogmoleculaire kininogeendeficiëntie</Name>
+      <Name lang="nl">Congenitale deficiëntie van hoogmoleculair kininogeen</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -122161,8 +122228,8 @@
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="nl">Niet toepasbaar</Name>
+        <TypeOfInheritance id="23480">
+          <Name lang="nl">Unknown</Name>
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
@@ -122629,7 +122696,7 @@
     <Disorder id="8620">
       <OrphaCode>2908</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=2908</ExpertLink>
-      <Name lang="nl">Syndroom van Kindler</Name>
+      <Name lang="nl">Epidermolysis bullosa van Kindler</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -123094,7 +123161,7 @@
     <Disorder id="25224">
       <OrphaCode>480864</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=480864</ExpertLink>
-      <Name lang="nl">Recurrente metabole encefalomyopathische crises - rhabdomyolyse - hartritmestoornis - intellectuele achterstand-syndroom</Name>
+      <Name lang="nl">Recidiverende metabole encefalomyopathische crises - rhabdomyolyse - hartritmestoornis - intellectuele achterstand-syndroom</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -124390,7 +124457,10 @@
           <Name lang="nl">No data available</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23417">
+          <Name lang="nl">Autosomaal recessief</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="25413">
@@ -130821,7 +130891,7 @@
     <Disorder id="11178">
       <OrphaCode>79152</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79152</ExpertLink>
-      <Name lang="nl">Verspreide oppervlakkige actinische porokeratose</Name>
+      <Name lang="nl">Gedissemineerde oppervlakkige actinische porokeratose</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -133041,7 +133111,7 @@
     <Disorder id="11082">
       <OrphaCode>75392</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=75392</ExpertLink>
-      <Name lang="nl">Periodontaal syndroom van Ehlers-Danlos</Name>
+      <Name lang="nl">Parodontaal syndroom van Ehlers-Danlos</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -135393,7 +135463,7 @@
     <Disorder id="10397">
       <OrphaCode>35708</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=35708</ExpertLink>
-      <Name lang="nl">Aromatisch L-aminozuur-decarboxylasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van aromatisch L-aminozuur-decarboxylase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -135947,8 +136017,8 @@
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23424">
-          <Name lang="nl">Multigeen/multifactorieel</Name>
+        <TypeOfInheritance id="23417">
+          <Name lang="nl">Autosomaal recessief</Name>
         </TypeOfInheritance>
         <TypeOfInheritance id="23494">
           <Name lang="nl">Niet toepasbaar</Name>
@@ -139364,7 +139434,7 @@
     <Disorder id="10630">
       <OrphaCode>48652</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=48652</ExpertLink>
-      <Name lang="nl">Monosomie 22q13</Name>
+      <Name lang="nl">Monosomie 22q13.3</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -141411,7 +141481,7 @@
     <Disorder id="28081">
       <OrphaCode>544472</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=544472</ExpertLink>
-      <Name lang="nl">Atypisch hemolytisch-uremisch syndroom met afwijking in een gen van het complementsysteem</Name>
+      <Name lang="nl">Atypisch hemolytisch-uremisch syndroom met afwijking van gen van complementsysteem</Name>
       <DisorderType id="21443">
         <Name lang="nl">Etiologisch subtype</Name>
       </DisorderType>
@@ -142589,7 +142659,7 @@
     <Disorder id="11907">
       <OrphaCode>89842</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89842</ExpertLink>
-      <Name lang="nl">Recessieve dystrofische epidermolysis bullosa, gegeneraliseerd intermediair</Name>
+      <Name lang="nl">Autosomaal recessieve gegeneraliseerde dystrofische epidermolysis bullosa, intermediaire vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -143685,7 +143755,7 @@
     <Disorder id="11812">
       <OrphaCode>88632</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=88632</ExpertLink>
-      <Name lang="nl">Dysgenesie van het voorste oogsegment</Name>
+      <Name lang="nl">Ontwikkelingsanomalie van voorste oogsegment</Name>
       <DisorderType id="36561">
         <Name lang="nl">Category</Name>
       </DisorderType>
@@ -143902,7 +143972,7 @@
     <Disorder id="11805">
       <OrphaCode>88618</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=88618</ExpertLink>
-      <Name lang="nl">Psychomotorische retardatie als gevolg van S-adenosylhomocysteïnehydrolasedeficiëntie</Name>
+      <Name lang="nl">Deficiëntie van S-adenosylhomocysteïnehydrolase</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -144691,7 +144761,7 @@
     <Disorder id="11903">
       <OrphaCode>89838</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=89838</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, autosomaal recessieve K14</Name>
+      <Name lang="nl">Autosomaal recessieve gegeneraliseerde epidermolysis bullosa simplex</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -144881,7 +144951,7 @@
     <Disorder id="12199">
       <OrphaCode>93256</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93256</ExpertLink>
-      <Name lang="nl">Fragiele X-geassocieerde tremor/ataxiesyndroom</Name>
+      <Name lang="nl">Fragiele X-geassocieerd tremor/ataxie-syndroom</Name>
       <DisorderType id="21401">
         <Name lang="nl">Malformatiesyndroom</Name>
       </DisorderType>
@@ -145957,7 +146027,7 @@
     <Disorder id="12257">
       <OrphaCode>93320</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=93320</ExpertLink>
-      <Name lang="nl">Ulnaire hemimelie</Name>
+      <Name lang="nl">Hemimelie van ulna</Name>
       <DisorderType id="21415">
         <Name lang="nl">Morfologische anomalie</Name>
       </DisorderType>
@@ -148810,7 +148880,7 @@
     <Disorder id="11422">
       <OrphaCode>79396</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79396</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, gegeneraliseerd ernstig</Name>
+      <Name lang="nl">Autosomaal dominante gegeneraliseerde epidermolysis bullosa simplex, ernstige vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -148975,7 +149045,7 @@
     <Disorder id="11425">
       <OrphaCode>79399</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79399</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, gegeneraliseerd intermediair</Name>
+      <Name lang="nl">Autosomaal dominante gegeneraliseerde epidermolysis bullosa simplex, intermediaire vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149007,7 +149077,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79401</ExpertLink>
-      <Name lang="nl">Epidermolysis bullosa simplex, Ogna-type</Name>
+      <Name lang="nl">PLEC-gerelateerde intermediaire epidermolysis bullosa simplex zonder extracutane betrokkenheid</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149062,7 +149132,7 @@
     <Disorder id="11429">
       <OrphaCode>79403</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79403</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa - pylorusatresie</Name>
+      <Name lang="nl">Junctionele epidermolysis bullosa met pylorusatresie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149091,7 +149161,7 @@
     <Disorder id="11428">
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79402</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa, gegeneraliseerd intermediair</Name>
+      <Name lang="nl">Intermediaire gegeneraliseerde junctionele epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149146,7 +149216,7 @@
     <Disorder id="11430">
       <OrphaCode>79404</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79404</ExpertLink>
-      <Name lang="nl">Junctionele epidermolysis bullosa, gegeneraliseerd ernstig</Name>
+      <Name lang="nl">Ernstige gegeneraliseerde junctionele epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149230,7 +149300,7 @@
     <Disorder id="11434">
       <OrphaCode>79408</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79408</ExpertLink>
-      <Name lang="nl">Ernstige gegeneraliseerde recessieve dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Autosomaal recessieve gegeneraliseerde dystrofische epidermolysis bullosa, ernstige vorm</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149256,7 +149326,7 @@
     <Disorder id="11437">
       <OrphaCode>79411</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79411</ExpertLink>
-      <Name lang="nl">Voorbijgaande bulleuze dermolyse van de neonaat</Name>
+      <Name lang="nl">Spontaan genezende dystrofische epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -149285,7 +149355,7 @@
     <Disorder id="11436">
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=79410</ExpertLink>
-      <Name lang="nl">Pretibiale dystrofische epidermolysis bullosa</Name>
+      <Name lang="nl">Gelokaliseerde dystrofische epidermolysis bullosa, pretibiale vorm</Name>
       <DisorderType id="21450">
         <Name lang="nl">Klinisch subtype</Name>
       </DisorderType>
@@ -152465,10 +152535,7 @@
           <Name lang="nl">Infantiel</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23438">
-          <Name lang="nl">Mitochondriale overerving</Name>
-        </TypeOfInheritance>
+      <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23431">
           <Name lang="nl">X-gebonden recessief</Name>
         </TypeOfInheritance>
@@ -158961,7 +159028,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=95702</ExpertLink>
-      <Name lang="nl">Cytomegalische congenitale adrenale hypoplasie</Name>
+      <Name lang="nl">X-gebonden congenitale bijnierhypoplasie</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -168348,7 +168415,7 @@
     <Disorder id="13451">
       <OrphaCode>98434</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=98434</ExpertLink>
-      <Name lang="nl">Erfelijke gecombineerde deficiëntie van vitamine K-afhankelijke stollingsfactoren</Name>
+      <Name lang="nl">Hereditaire gecombineerde deficiëntie van vitamine K-afhankelijke stollingsfactoren</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
@@ -168698,7 +168765,7 @@
     <Disorder id="28676">
       <OrphaCode>570470</OrphaCode>
       <ExpertLink lang="nl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=nl&amp;Expert=570470</ExpertLink>
-      <Name lang="nl">Ricin poisoning</Name>
+      <Name lang="nl">Intoxicatie door ricine</Name>
       <DisorderType id="21394">
         <Name lang="nl">Ziekte</Name>
       </DisorderType>
