commit f65bd9395f4ed3e473ab300a88d98e05768a8ad2
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Feb 1 09:55:47 2022 +0100

    FEB 2022

diff --git a/Epidemiological data/Rare diseases natural history/pl_product9_ages.xml b/Epidemiological data/Rare diseases natural history/pl_product9_ages.xml
index 7f7d8a2..b33e61b 100644
--- a/Epidemiological data/Rare diseases natural history/pl_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/pl_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<JDBOR date="2022-01-01 04:52:33" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-02-01 04:54:18" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6643">
+  <DisorderList count="6653">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=166024</ExpertLink>
@@ -15273,10 +15273,16 @@
       <DisorderGroup id="36540">
         <Name lang="pl">Grupa kliniczna</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="pl">Prenatalny</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23522">
           <Name lang="pl">Okres niemowlęcy</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="pl">Okres noworodkowy</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
@@ -18343,32 +18349,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="813">
-      <OrphaCode>1549</OrphaCode>
-      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=1549</ExpertLink>
-      <Name lang="pl">Kryptosporydioza</Name>
-      <DisorderType id="21394">
-        <Name lang="pl">Choroba</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pl">Zaburzenie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="pl">W każdym wieku</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="pl">normalna oczekiwana długość życia</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="pl">Nie dotyczy</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="18415">
       <OrphaCode>199642</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=199642</ExpertLink>
@@ -32687,7 +32667,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
+      <TypeOfInheritanceList count="2">
+        <TypeOfInheritance id="23410">
+          <Name lang="pl">Autosomalny dominujący</Name>
+        </TypeOfInheritance>
         <TypeOfInheritance id="23417">
           <Name lang="pl">Autosomalny recesywny</Name>
         </TypeOfInheritance>
@@ -71268,9 +71251,24 @@
       <DisorderGroup id="36547">
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="pl">W każdym wieku</Name>
+      <AverageAgeOfOnsetList count="6">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23508">
+          <Name lang="pl">Prenatalny</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pl">Okres niemowlęcy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23515">
+          <Name lang="pl">Okres noworodkowy</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
@@ -80722,7 +80720,16 @@
       <DisorderGroup id="36547">
         <Name lang="pl">Zaburzenie</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
@@ -134809,35 +134816,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="11041">
-      <OrphaCode>73247</OrphaCode>
-      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=73247</ExpertLink>
-      <Name lang="pl">Eozynofilowe zapalenie przełyku</Name>
-      <DisorderType id="21394">
-        <Name lang="pl">Choroba</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pl">Zaburzenie</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="pl">W każdym wieku</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="pl">normalna oczekiwana długość życia</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23424">
-          <Name lang="pl">Wielogenowy/wieloczynnikowy</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="pl">Nie dotyczy</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="28655">
       <OrphaCode>569821</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=569821</ExpertLink>
@@ -156658,10 +156636,19 @@
       <DisorderGroup id="36554">
         <Name lang="pl">Podtyp zaburzenia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23529">
           <Name lang="pl">Dzieciństwo</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pl">Okres niemowlęcy</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="1">
         <AverageAgeOfDeath id="23641">
@@ -156710,15 +156697,15 @@
       <DisorderGroup id="36554">
         <Name lang="pl">Podtyp zaburzenia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23522">
           <Name lang="pl">Okres niemowlęcy</Name>
         </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pl">Okres noworodkowy</Name>
-        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23417">
@@ -160239,7 +160226,16 @@
           <Name lang="pl">Dzieciństwo</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -167304,6 +167300,50 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29436">
+      <OrphaCode>589595</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589595</ExpertLink>
+      <Name lang="en">Mixed phenotype acute leukemia with t(v;11q23.3)</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pl">Okres niemowlęcy</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pl">wczesne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="13885">
       <OrphaCode>98868</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=98868</ExpertLink>
@@ -167417,6 +167457,53 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29433">
+      <OrphaCode>589534</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589534</ExpertLink>
+      <Name lang="en">Mixed phenotype acute leukemia with t(9;22)(q34.1;q11.2)</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pl">Wiek podeszły</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pl">wczesne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pl">osoba starsza</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="13880">
       <OrphaCode>98863</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=98863</ExpertLink>
@@ -167492,6 +167579,50 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29434">
+      <OrphaCode>589542</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589542</ExpertLink>
+      <Name lang="pl">Nowotwór szpikowy/limfoidalny związany z rearanżacją JAK2</Name>
+      <DisorderType id="21394">
+        <Name lang="pl">Choroba</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pl">Wiek podeszły</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pl">osoba starsza</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29429">
       <OrphaCode>589442</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=589442</ExpertLink>
@@ -169541,6 +169672,255 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29205">
+      <OrphaCode>585877</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585877</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z powtarzalnymi zmianami genetycznymi</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="5">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pl">Wiek podeszły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pl">Okres niemowlęcy</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pl">wczesne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pl">osoba starsza</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29211">
+      <OrphaCode>585909</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585909</ExpertLink>
+      <Name lang="en">B-lymphoblastic leukemia/lymphoma with t(9;22)(q34.1;q11.2)</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="5">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pl">Wiek podeszły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pl">Okres niemowlęcy</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="7">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pl">wczesne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pl">osoba starsza</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="pl">niemowlę</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29214">
+      <OrphaCode>585936</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585936</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z hiperdiploidią</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pl">brak dostępnych danych</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29215">
+      <OrphaCode>585942</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585942</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z hipodiploidią</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="5">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pl">wczesne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29212">
+      <OrphaCode>585918</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585918</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z t(v;11q23.3)</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pl">Okres niemowlęcy</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pl">nastolatek</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pl">dorosły</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pl">wczesne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="pl">niemowlę</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pl">późne dzieciństwo</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pl">młody dorosły</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29213">
+      <OrphaCode>585929</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585929</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z t(12;21)(p13.2;q22.1) </Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pl">brak dostępnych danych</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14044">
       <OrphaCode>99027</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=99027</ExpertLink>
@@ -169564,6 +169944,64 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29217">
+      <OrphaCode>585956</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585956</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z t(1;19)(q23;p13.3)</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pl">brak dostępnych danych</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="29216">
+      <OrphaCode>585948</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=585948</ExpertLink>
+      <Name lang="pl">Białaczka/chłoniak limfoblastyczny z komórek B z t(5;14)(q31.1;q32.3)</Name>
+      <DisorderType id="21443">
+        <Name lang="pl">Podtyp etiologiczny</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pl">Podtyp zaburzenia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pl">Wiek młodzieńczy</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pl">Dorosły</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pl">Dzieciństwo</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pl">brak dostępnych danych</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="29226">
       <OrphaCode>586130</OrphaCode>
       <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=586130</ExpertLink>
@@ -176926,5 +177364,25 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="31372">
+      <OrphaCode>617919</OrphaCode>
+      <ExpertLink lang="pl">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pl&amp;Expert=617919</ExpertLink>
+      <Name lang="en">F12-associated cold auto-inflammatory syndrome</Name>
+      <DisorderType id="21394">
+        <Name lang="pl">Choroba</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pl">Zaburzenie</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="pl">Autosomalny dominujący</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
