commit 57fdf392dcec7eacab11e5cf66bf35fafd0ff752
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sun Jan 2 10:18:09 2022 +0100

    JAN 2021

diff --git a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml
index 1f1e911..197add4 100644
--- a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-12-01 04:56:25" version="1.3.14 / 4.1.7 [2021-06-23] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <DisorderList count="6641">
+<JDBOR date="2022-01-01 04:53:18" version="1.3.15 / 4.1.7 [2021-11-25] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+  <Availability> 
+    <Licence>
+      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
+      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
+      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
+    </Licence>
+  </Availability>
+  <DisorderList count="6643">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -494,7 +501,10 @@
           <Name lang="pt">Qualquer idade</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pt">Não há dados disponíveis</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
         <TypeOfInheritance id="23410">
@@ -708,7 +718,10 @@
           <Name lang="pt">Neonatal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pt">Não há dados disponíveis</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23417">
@@ -1620,9 +1633,12 @@
           <Name lang="pt">Neonatal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="pt">esperança de vida normal</Name>
+      <AverageAgeOfDeathList count="2">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pt">Adulto</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pt">Idoso</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="2">
@@ -3951,7 +3967,7 @@
     <Disorder id="17515">
       <OrphaCode>163654</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=163654</ExpertLink>
-      <Name lang="pt">Displasia espondiloepifisária, tipo Cantu</Name>
+      <Name lang="pt">Síndrome de displasia espondiloepifisária-braquidactilia-perturbação da linguagem</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -5344,7 +5360,7 @@
     <Disorder id="209">
       <OrphaCode>628</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=628</ExpertLink>
-      <Name lang="pt">Nanismo diastrófico</Name>
+      <Name lang="pt">Displasia diastrófica</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -5564,7 +5580,7 @@
     <Disorder id="233">
       <OrphaCode>2869</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2869</ExpertLink>
-      <Name lang="pt">Síndrome de Peutz-Jeghers</Name>
+      <Name lang="pt">Síndrome Peutz-Jeghers</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -5836,7 +5852,7 @@
     <Disorder id="242">
       <OrphaCode>205</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=205</ExpertLink>
-      <Name lang="pt">Síndrome de Crigler-Najjar</Name>
+      <Name lang="pt">Síndrome Crigler-Najjar</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -7131,7 +7147,7 @@
     <Disorder id="294">
       <OrphaCode>308</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=308</ExpertLink>
-      <Name lang="pt">Doença de Unverricht-Lundborg</Name>
+      <Name lang="pt">Epilepsia mioclónica progressiva tipo 1</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -12401,7 +12417,7 @@
     <Disorder id="518">
       <OrphaCode>2308</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2308</ExpertLink>
-      <Name lang="pt">Síndrome de Jacobsen</Name>
+      <Name lang="pt">Síndrome Jacobsen</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -15115,15 +15131,9 @@
       <DisorderGroup id="36540">
         <Name lang="pt">Grupo de patologias</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23557">
-          <Name lang="pt">Qualquer idade</Name>
-        </AverageAgeOfOnset>
+      <AverageAgeOfOnsetList count="0">
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="pt">esperança de vida normal</Name>
-        </AverageAgeOfDeath>
+      <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23417">
@@ -15215,7 +15225,7 @@
     <Disorder id="647">
       <OrphaCode>388</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=388</ExpertLink>
-      <Name lang="pt">Doença de Hirschsprung</Name>
+      <Name lang="pt">Doença Hirschsprung</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -16999,7 +17009,7 @@
     <Disorder id="17947">
       <OrphaCode>171871</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=171871</ExpertLink>
-      <Name lang="pt">Pseudo-hipoaldosteronismo renal, tipo 1</Name>
+      <Name lang="pt">Pseudo-hipoaldosteronismo renal tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -17164,7 +17174,7 @@
     <Disorder id="17948">
       <OrphaCode>171876</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=171876</ExpertLink>
-      <Name lang="pt">Pseudo-hipoaldosteronismo tipo 1, generalizado</Name>
+      <Name lang="pt">Pseudo-hipoaldosteronismo tipo 1 generalizado</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -19909,7 +19919,10 @@
           <Name lang="pt">Neonatal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pt">Não há dados disponíveis</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="3">
         <TypeOfInheritance id="23410">
@@ -20771,7 +20784,7 @@
     <Disorder id="18309">
       <OrphaCode>183678</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=183678</ExpertLink>
-      <Name lang="pt">Síndrome de Hermansky-Pudlak tipo 2</Name>
+      <Name lang="pt">Síndrome Hermansky-Pudlak devido a deficiência AP-3</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -26830,6 +26843,41 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="16890">
+      <OrphaCode>139417</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=139417</ExpertLink>
+      <Name lang="pt">Mielite transversa, aguda</Name>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="5">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pt">Adolescente</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pt">Idade adulta</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23641">
+          <Name lang="pt">esperança de vida normal</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="1339">
       <OrphaCode>1031</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1031</ExpertLink>
@@ -32781,7 +32829,7 @@
     <Disorder id="1546">
       <OrphaCode>1323</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1323</ExpertLink>
-      <Name lang="pt">Camptodactilia - artrogripose - defeitos esqueléticos faciais</Name>
+      <Name lang="pt">Síndrome de camptodactilia-artrogripose-defeitos esqueléticos faciais</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -34408,7 +34456,7 @@
     <Disorder id="16916">
       <OrphaCode>139536</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=139536</ExpertLink>
-      <Name lang="pt">Neuropatia, motora, distal, hereditária, tipo 5</Name>
+      <Name lang="pt">Neuropatia motora distal hereditária tipo 5</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -36630,7 +36678,7 @@
     <Disorder id="1899">
       <OrphaCode>1952</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1952</ExpertLink>
-      <Name lang="pt">Epífises pontuadasSíndrome - hiperplasia osteoclástica</Name>
+      <Name lang="pt">Síndrome de epífises ponteadas-hiperplasia osteoclástica</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -52499,7 +52547,10 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23508">
+          <Name lang="pt">Pré-natal</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23522">
           <Name lang="pt">Infância</Name>
         </AverageAgeOfOnset>
@@ -52544,7 +52595,7 @@
     <Disorder id="2314">
       <OrphaCode>2502</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2502</ExpertLink>
-      <Name lang="pt">Disostose metafisária - atraso mental - surdez de condução</Name>
+      <Name lang="pt">Síndrome de disostose metafisária-perturbação do desenvolvimento intelectual-surdez de condução</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -65435,7 +65486,7 @@
     <Disorder id="2932">
       <OrphaCode>3292</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3292</ExpertLink>
-      <Name lang="pt">Síndrome camptodactilia de Tel Hashomer</Name>
+      <Name lang="pt">Síndrome de camptodactilia Tel Hashomer</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -72557,7 +72608,7 @@
     <Disorder id="3482">
       <OrphaCode>747</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=747</ExpertLink>
-      <Name lang="pt">Proteinose alveolar pulmonar</Name>
+      <Name lang="pt">Proteinose alveolar pulmonar autoimune</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -80044,7 +80095,7 @@
     <Disorder id="19196">
       <OrphaCode>231512</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=231512</ExpertLink>
-      <Name lang="pt">Síndrome de Hermansky-Pudlak sem fibrose pulmonar</Name>
+      <Name lang="pt">Síndrome Hermansky-Pudlak devido a deficiência BLOC-2</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -80131,7 +80182,7 @@
     <Disorder id="19195">
       <OrphaCode>231500</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=231500</ExpertLink>
-      <Name lang="pt">Síndrome de Hermansky-Pudlak com fibrose pulmonar</Name>
+      <Name lang="pt">Síndrome Hermansky-Pudlak devido a deficiência BLOC-3</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -83674,7 +83725,7 @@
     <Disorder id="21955">
       <OrphaCode>330001</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=330001</ExpertLink>
-      <Name lang="pt">Amiloidose sistémica, forma senil</Name>
+      <Name lang="pt">Amiloidose ATTR tipo selvagem</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -92923,7 +92974,7 @@
     <Disorder id="22057">
       <OrphaCode>352333</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=352333</ExpertLink>
-      <Name lang="pt">Ictiose congénita - perturbação do desenvolvimento inteletual - tetraplegia espástica</Name>
+      <Name lang="pt">Síndrome de ictiose congénita-perturbação do desenvolvimento inteletual-tetraplegia espástica</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -98229,7 +98280,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=314555</ExpertLink>
-      <Name lang="pt">Síndrome de displasia craniofacial-osteopenia</Name>
+      <Name lang="pt">Síndrome de dismorfia facial-miopia grave-osteopenia-perturbação do desenvolvimento intelectual-anomalias dentárias</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -102642,7 +102693,7 @@
     <Disorder id="23616">
       <OrphaCode>449291</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=449291</ExpertLink>
-      <Name lang="pt">Síndrome de X-Frágil em mulheres portadoras, forma sintomática</Name>
+      <Name lang="pt">Síndrome de X-Frágil sintomático em mulheres portadoras</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -102732,7 +102783,7 @@
     <Disorder id="23623">
       <OrphaCode>449432</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=449432</ExpertLink>
-      <Name lang="pt">Doença da glândula submandibular relacionada comIgG4</Name>
+      <Name lang="pt">Doença da glândula submandibular IgG4-relacionada</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -124964,7 +125015,7 @@
     <Disorder id="25161">
       <OrphaCode>477831</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=477831</ExpertLink>
-      <Name lang="pt">Síndrome de sobrecrescimento esqulético-dismorfismo craniofacial-pele híper-elástica-lesões da substância branca</Name>
+      <Name lang="pt">Síndrome de sobrecrescimento Kosaki</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -128294,8 +128345,8 @@
       <OrphaCode>66633</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=66633</ExpertLink>
       <Name lang="pt">Surdez neuro-senssorial - embranquecimento precoce do cabelo - tremor idiopático</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
@@ -128435,7 +128486,7 @@
     <Disorder id="10885">
       <OrphaCode>66627</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=66627</ExpertLink>
-      <Name lang="pt">Sinovite vilonodular pigmentada</Name>
+      <Name lang="pt">Tumor tenosinovial de células gigantes</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -131607,7 +131658,7 @@
     <Disorder id="11260">
       <OrphaCode>79234</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=79234</ExpertLink>
-      <Name lang="pt">Síndrome de Crigler-Najjar tipo 1</Name>
+      <Name lang="pt">Síndrome Crigler-Najjar tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -131633,7 +131684,7 @@
     <Disorder id="11261">
       <OrphaCode>79235</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=79235</ExpertLink>
-      <Name lang="pt">Síndrome de Crigler-Najjar tipo 2 (síndrome de Arias)</Name>
+      <Name lang="pt">Síndrome Crigler-Najjar tipo 2</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -133584,7 +133635,7 @@
     <Disorder id="11126">
       <OrphaCode>79078</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=79078</ExpertLink>
-      <Name lang="pt">Doença de Mickulicz</Name>
+      <Name lang="pt">Sialadenite e dacrioadenite IgG4-relacionada</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -134944,7 +134995,7 @@
     <Disorder id="11071">
       <OrphaCode>75326</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=75326</ExpertLink>
-      <Name lang="pt">Tortuosidade das artérias da retina</Name>
+      <Name lang="pt">Tortuosidade arterial da retina</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -137805,7 +137856,7 @@
     <Disorder id="10332">
       <OrphaCode>34149</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=34149</ExpertLink>
-      <Name lang="pt">Doença renal medular quística autossómica dominante</Name>
+      <Name lang="pt">Doença renal tubulointersticial autossómica dominante</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -138361,7 +138412,7 @@
     <Disorder id="10360">
       <OrphaCode>35062</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=35062</ExpertLink>
-      <Name lang="pt">Infeção disseminada por citomegalovirus idiopática</Name>
+      <Name lang="pt">Infeção disseminada grave por citomegalovirus em doentes sem imunodeficiência</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -146104,7 +146155,7 @@
     <Disorder id="11881">
       <OrphaCode>88949</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=88949</ExpertLink>
-      <Name lang="pt">Doença renal medular quística, autossómica dominante, sem hiperuricemia</Name>
+      <Name lang="pt">Doença tubulo-intersticial renal autossómica dominante MUC1-relacionada</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -146124,7 +146175,7 @@
     <Disorder id="11882">
       <OrphaCode>88950</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=88950</ExpertLink>
-      <Name lang="pt">Doença renal medular quística, autossómica dominante, com hiperuricemia</Name>
+      <Name lang="pt">Doença tubulo-intersticial renal autossómica dominante UMOD-relacionada</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -147117,7 +147168,7 @@
     <Disorder id="12168">
       <OrphaCode>93111</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93111</ExpertLink>
-      <Name lang="pt">Síndrome renal - diabetes MODY5</Name>
+      <Name lang="pt">Doença tubulo-intersticial renal autossómica dominante HNF1B-relacionada</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -148498,7 +148549,10 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23522">
           <Name lang="pt">Infância</Name>
         </AverageAgeOfOnset>
@@ -148506,7 +148560,16 @@
           <Name lang="pt">Neonatal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="3">
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pt">primeira infância</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="pt">Infantil</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23641">
+          <Name lang="pt">esperança de vida normal</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23417">
@@ -149835,17 +149898,47 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="6">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pt">Adolescente</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pt">Idade adulta</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23508">
           <Name lang="pt">Pré-natal</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="pt">Neonatal</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="pt">Qualquer idade</Name>
+      <AverageAgeOfDeathList count="7">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pt">adolescente</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pt">Adulto</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pt">primeira infância</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23592">
+          <Name lang="pt">Infantil</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pt">infância tardia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pt">Adulto jovem</Name>
         </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
@@ -153823,7 +153916,7 @@
     <Disorder id="11669">
       <OrphaCode>85279</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=85279</ExpertLink>
-      <Name lang="pt">Atraso mental sindrómico ligado ao X por mutação no gene JARID1C</Name>
+      <Name lang="pt">Perturbação do desenvolvimento intelectual ligada ao X sindromática KDM5C-relacionada</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -160250,7 +160343,7 @@
     <Disorder id="12856">
       <OrphaCode>97229</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=97229</ExpertLink>
-      <Name lang="pt">Síndrome de Brown-Vialetto-van Laere</Name>
+      <Name lang="pt">Deficiência do transportador de riboflavina</Name>
       <DisorderType id="21401">
         <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
@@ -164579,7 +164672,7 @@
     <Disorder id="12453">
       <OrphaCode>93622</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93622</ExpertLink>
-      <Name lang="pt">Doença de Dent tipo 1</Name>
+      <Name lang="pt">Doença Dent tipo 1</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -164605,7 +164698,7 @@
     <Disorder id="12454">
       <OrphaCode>93623</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93623</ExpertLink>
-      <Name lang="pt">Doença de Dent tipo 2</Name>
+      <Name lang="pt">Doença Dent tipo 2</Name>
       <DisorderType id="21450">
         <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
@@ -165381,6 +165474,41 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="29579">
+      <OrphaCode>592873</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=592873</ExpertLink>
+      <Name lang="pt">Mielite transversa aguda com anticorpos anti-MOG</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
+      </DisorderType>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="5">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pt">Adolescente</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pt">Idade adulta</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23641">
+          <Name lang="pt">esperança de vida normal</Name>
+        </AverageAgeOfDeath>
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="0">
+      </TypeOfInheritanceList>
+    </Disorder>
     <Disorder id="14158">
       <OrphaCode>99141</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=99141</ExpertLink>
@@ -175269,7 +175397,7 @@
     <Disorder id="14511">
       <OrphaCode>99938</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=99938</ExpertLink>
-      <Name lang="pt">Doença de Charcot-Marie-Tooth tipo 2D, autossómica dominante</Name>
+      <Name lang="pt">Doença de Charcot-Marie-Tooth autossómica dominante tipo 2D</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
