commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml
index ca7a6bb..ea53a6a 100644
--- a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:41:47" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:42:30" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6577">
+  <DisorderList count="6566">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -2154,10 +2154,22 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="6">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pt">Adolescente</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23543">
           <Name lang="pt">Idade adulta</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23508">
+          <Name lang="pt">Pré-natal</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23522">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23515">
           <Name lang="pt">Neonatal</Name>
         </AverageAgeOfOnset>
@@ -6065,7 +6077,7 @@
     <Disorder id="277">
       <OrphaCode>2443</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2443</ExpertLink>
-      <Name lang="pt">Doenças mitocondriais de origem nuclear (termo genérico)</Name>
+      <Name lang="pt">Doenças mitocondriais de origem nuclear</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -13249,7 +13261,7 @@
     <Disorder id="633">
       <OrphaCode>70</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=70</ExpertLink>
-      <Name lang="pt">Atrofia muscular espinhal proximal (termo genérico)</Name>
+      <Name lang="pt">Atrofia muscular espinhal proximal</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -15036,7 +15048,7 @@
     <Disorder id="644">
       <OrphaCode>355</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=355</ExpertLink>
-      <Name lang="pt">Doença de Gaucher (termo genérico)</Name>
+      <Name lang="pt">Doença de Gaucher</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -15507,7 +15519,7 @@
     <Disorder id="663">
       <OrphaCode>3440</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3440</ExpertLink>
-      <Name lang="pt">Síndrome de Waardenburg (termo genérico)</Name>
+      <Name lang="pt">Síndrome de Waardenburg</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -18858,7 +18870,7 @@
     <Disorder id="792">
       <OrphaCode>1531</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1531</ExpertLink>
-      <Name lang="pt">Craniossinostose (termo genérico)</Name>
+      <Name lang="pt">Craniossinostose</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -21699,8 +21711,8 @@
       <OrphaCode>847</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=847</ExpertLink>
       <Name lang="pt">Alfa talassemia - atraso mental, ligada ao X</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21401">
+        <Name lang="pt">Síndrome malformativo</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
@@ -22118,7 +22130,7 @@
     <Disorder id="999">
       <OrphaCode>1047</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1047</ExpertLink>
-      <Name lang="pt">Anemia sideroblástica (termo genérico)</Name>
+      <Name lang="pt">Anemia sideroblástica</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -24393,7 +24405,7 @@
     <Disorder id="1265">
       <OrphaCode>946</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=946</ExpertLink>
-      <Name lang="pt">Acrocefalossindactilia (termo genérico)</Name>
+      <Name lang="pt">Acrocefalossindactilia</Name>
       <DisorderType id="21436">
         <Name lang="pt">Grupo de fenótipos</Name>
       </DisorderType>
@@ -30368,7 +30380,7 @@
     <Disorder id="1633">
       <OrphaCode>1427</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1427</ExpertLink>
-      <Name lang="pt">'Displasia oto-espondilo-megaepifísária; OSMED'</Name>
+      <Name lang="pt">Displasia oto-espondilo-megaepifísária; OSMED</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -54612,32 +54624,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="2746">
-      <OrphaCode>3056</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3056</ExpertLink>
-      <Name lang="pt">Atraso mental ligado ao X tipo Brooks</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="pt">Não há dados disponíveis</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="2747">
       <OrphaCode>3057</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3057</ExpertLink>
@@ -70381,7 +70367,7 @@
     <Disorder id="3528">
       <OrphaCode>599</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=599</ExpertLink>
-      <Name lang="pt">Miopatia distal (termo genérico)</Name>
+      <Name lang="pt">Miopatia distal</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -70531,7 +70517,7 @@
     <Disorder id="3538">
       <OrphaCode>2034</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2034</ExpertLink>
-      <Name lang="pt">Filariose (termo genérico)</Name>
+      <Name lang="pt">Filariose</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -70951,7 +70937,7 @@
     <Disorder id="3553">
       <OrphaCode>137</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=137</ExpertLink>
-      <Name lang="pt">Síndrome das CDG (termo genérico)</Name>
+      <Name lang="pt">Síndrome das CDG</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -76563,7 +76549,7 @@
     <Disorder id="3765">
       <OrphaCode>547</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=547</ExpertLink>
-      <Name lang="pt">Linfoma maligno não-Hodgkin (termo genérico)</Name>
+      <Name lang="pt">Linfoma maligno não-Hodgkin</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -78122,7 +78108,7 @@
     <Disorder id="3700">
       <OrphaCode>2781</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2781</ExpertLink>
-      <Name lang="pt">Osteopetrose (termo genérico)</Name>
+      <Name lang="pt">Osteopetrose</Name>
       <DisorderType id="21436">
         <Name lang="pt">Grupo de fenótipos</Name>
       </DisorderType>
@@ -80997,7 +80983,7 @@
     <Disorder id="3586">
       <OrphaCode>519</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=519</ExpertLink>
-      <Name lang="pt">Leucemia aguda não-linfoblástica (termo genérico)</Name>
+      <Name lang="pt">Leucemia aguda não-linfoblástica</Name>
       <DisorderType id="21436">
         <Name lang="pt">Grupo de fenótipos</Name>
       </DisorderType>
@@ -92359,35 +92345,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="20904">
-      <OrphaCode>294049</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=294049</ExpertLink>
-      <Name lang="pt">Síndrome de Larsen da ilha da Reunião</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="pt">Não há dados disponíveis</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="20900">
       <OrphaCode>293987</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=293987</ExpertLink>
@@ -117279,7 +117236,7 @@
     <Disorder id="8734">
       <OrphaCode>263</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=263</ExpertLink>
-      <Name lang="pt">Distrofia muscular das cinturas (termo genérico)</Name>
+      <Name lang="pt">Distrofia muscular das cinturas</Name>
       <DisorderType id="21436">
         <Name lang="pt">Grupo de fenótipos</Name>
       </DisorderType>
@@ -118502,32 +118459,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="25668">
-      <OrphaCode>494348</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=494348</ExpertLink>
-      <Name lang="pt">Hipertensão portal não-cirrótica familiar de início precoce</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23529">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="pt">Não há dados disponíveis</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="25683">
       <OrphaCode>494547</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=494547</ExpertLink>
@@ -128687,11 +128618,11 @@
       <OrphaCode>64743</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=64743</ExpertLink>
       <Name lang="pt">Esclerose hepato-portal</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21457">
+        <Name lang="pt">Subtipo histopatológico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23557">
@@ -129726,7 +129657,7 @@
     <Disorder id="10770">
       <OrphaCode>56970</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=56970</ExpertLink>
-      <Name lang="pt">Encefalopatias espongiformes transmissíveis (termo genérico)</Name>
+      <Name lang="pt">Encefalopatias espongiformes transmissíveis</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -133431,7 +133362,7 @@
     <Disorder id="11097">
       <OrphaCode>77240</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=77240</ExpertLink>
-      <Name lang="pt">Linfedema primário (termo genérico)</Name>
+      <Name lang="pt">Linfedema primário</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -134711,7 +134642,7 @@
     <Disorder id="11032">
       <OrphaCode>73014</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=73014</ExpertLink>
-      <Name lang="pt">Diarreia intratável da infância (termo genérico)</Name>
+      <Name lang="pt">Diarreia intratável da infância</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -134757,32 +134688,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="11034">
-      <OrphaCode>73220</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=73220</ExpertLink>
-      <Name lang="pt">Atraso mental ligado ao X - hipotonia da face</Name>
-      <DisorderType id="21436">
-        <Name lang="pt">Grupo de fenótipos</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="pt">Grupo de patologias</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10462">
       <OrphaCode>40366</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=40366</ExpertLink>
@@ -137125,7 +137030,7 @@
     <Disorder id="10319">
       <OrphaCode>33364</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=33364</ExpertLink>
-      <Name lang="pt">Tricotiodistrofia (termo genérico)</Name>
+      <Name lang="pt">Tricotiodistrofia</Name>
       <DisorderType id="21394">
         <Name lang="pt">Doença</Name>
       </DisorderType>
@@ -137545,7 +137450,7 @@
     <Disorder id="10344">
       <OrphaCode>34526</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=34526</ExpertLink>
-      <Name lang="pt">Hipomagnesemia primária hereditária (termo genérico)</Name>
+      <Name lang="pt">Hipomagnesemia primária hereditária</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -139387,7 +139292,7 @@
     <Disorder id="10627">
       <OrphaCode>48471</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=48471</ExpertLink>
-      <Name lang="pt">Lisencefalia (termo genérico)</Name>
+      <Name lang="pt">Lisencefalia</Name>
       <DisorderType id="36561">
         <Name lang="pt">Category</Name>
       </DisorderType>
@@ -141238,11 +141143,11 @@
       <OrphaCode>48372</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=48372</ExpertLink>
       <Name lang="pt">Hiperplasia regenerativa nodular</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21457">
+        <Name lang="pt">Subtipo histopatológico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23543">
@@ -153095,26 +153000,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="11698">
-      <OrphaCode>85328</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=85328</ExpertLink>
-      <Name lang="pt">Atraso mental ligado ao X tipo Turner</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23445">
-          <Name lang="pt">Ligado ao X dominante</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="11699">
       <OrphaCode>85329</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=85329</ExpertLink>
@@ -162310,58 +162195,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12533">
-      <OrphaCode>93975</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93975</ExpertLink>
-      <Name lang="pt">Síndrome de Renier-Gabreels-Jasper</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12532">
-      <OrphaCode>93974</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93974</ExpertLink>
-      <Name lang="pt">Síndrome de Smith-Fineman-Myers</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12534">
       <OrphaCode>93976</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93976</ExpertLink>
@@ -162388,113 +162221,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12529">
-      <OrphaCode>93971</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93971</ExpertLink>
-      <Name lang="pt">Síndrome de Chudley-Lowry-Hoar</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12528">
-      <OrphaCode>93970</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93970</ExpertLink>
-      <Name lang="pt">Síndrome de Holmes-Gang</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23599">
-          <Name lang="pt">primeira infância</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12531">
-      <OrphaCode>93973</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93973</ExpertLink>
-      <Name lang="pt">Síndrome de Carpenter-Waziri</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="12530">
-      <OrphaCode>93972</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93972</ExpertLink>
-      <Name lang="pt">Síndrome de Juberg-Marsidi</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12492">
       <OrphaCode>93932</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93932</ExpertLink>
