commit c92e9c5f9940abcb2c7950377e1c2245059cad90
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 1 10:54:09 2021 +0100

    MARCH 2021

diff --git a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml
old mode 100755
new mode 100644
index 2f27b52..ca7a6bb
--- a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-02-01 04:41:42" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-03-01 04:41:47" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6580">
+  <DisorderList count="6577">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -277,35 +277,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="17609">
-      <OrphaCode>166068</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166068</ExpertLink>
-      <Name lang="pt">Hipoplasia olivopontocerebelosa, tipo 5</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23599">
-          <Name lang="pt">primeira infância</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="8">
       <OrphaCode>141</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=141</ExpertLink>
@@ -58722,8 +58693,8 @@
       <OrphaCode>268261</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=268261</ExpertLink>
       <Name lang="pt">Síndrome de microdeleção 21q22.13q22.2</Name>
-      <DisorderType id="21450">
-        <Name lang="pt">Subtipo clínico</Name>
+      <DisorderType id="21443">
+        <Name lang="pt">Subtipo etiologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="pt">Subtipo de patologia</Name>
@@ -96885,11 +96856,11 @@
       <OrphaCode>313892</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=313892</ExpertLink>
       <Name lang="pt">Atraso do desenvolvimento e da linguagem devido à deficiência SOX5</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -96917,11 +96888,11 @@
       <OrphaCode>313884</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=313884</ExpertLink>
       <Name lang="pt">Síndrome da microdeleção 12p12.1</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23522">
@@ -99741,49 +99712,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="21238">
-      <OrphaCode>306597</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=306597</ExpertLink>
-      <Name lang="pt">Síndrome de OpitzG /BBB, ligada ao cromossoma X</Name>
-      <DisorderType id="21443">
-        <Name lang="pt">Subtipo etiologico</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="pt">Subtipo de patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="21237">
-      <OrphaCode>306588</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=306588</ExpertLink>
-      <Name lang="pt">Síndrome de OpitzG/BBB, autossómica dominante</Name>
-      <DisorderType id="21443">
-        <Name lang="pt">Subtipo etiologico</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="pt">Subtipo de patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="0">
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="21236">
       <OrphaCode>306577</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=306577</ExpertLink>
@@ -100656,8 +100584,8 @@
       <OrphaCode>464311</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=464311</ExpertLink>
       <Name lang="pt">Síndrome de défice intelectual devida a mutação pontual doDYRK1A</Name>
-      <DisorderType id="21450">
-        <Name lang="pt">Subtipo clínico</Name>
+      <DisorderType id="21443">
+        <Name lang="pt">Subtipo etiologico</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="pt">Subtipo de patologia</Name>
