commit 45f1b105cb2ab1f31cd982124355df258b02ec5a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Fri Apr 1 09:56:06 2022 +0200

    APR 2022

diff --git a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml
index ec7e7e1..fab6c0a 100644
--- a/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
+++ b/Epidemiological data/Rare diseases natural history/pt_product9_ages.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-03-01 04:55:23" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-04-01 05:00:08" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="6654">
+  <DisorderList count="6641">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=166024</ExpertLink>
@@ -12666,32 +12666,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="521">
-      <OrphaCode>2343</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2343</ExpertLink>
-      <Name lang="pt">Síndrome crânio em trevo</Name>
-      <DisorderType id="21415">
-        <Name lang="pt">Anomalia morfológica</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23599">
-          <Name lang="pt">primeira infância</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23494">
-          <Name lang="pt">Não aplicável</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="523">
       <OrphaCode>2346</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2346</ExpertLink>
@@ -17529,32 +17503,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="727">
-      <OrphaCode>1428</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1428</ExpertLink>
-      <Name lang="pt">Condromalácia da rótula</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23536">
-          <Name lang="pt">Adolescente</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="pt">Qualquer idade</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23473">
-          <Name lang="pt">Ligado ao Y</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="820">
       <OrphaCode>3303</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3303</ExpertLink>
@@ -26705,55 +26653,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1284">
-      <OrphaCode>965</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=965</ExpertLink>
-      <Name lang="pt">Síndrome face acromegalóide</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="1285">
-      <OrphaCode>966</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=966</ExpertLink>
-      <Name lang="pt">Hipertricose - face acromegalóide</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="16888">
       <OrphaCode>139411</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=139411</ExpertLink>
@@ -32783,32 +32682,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1544">
-      <OrphaCode>1321</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1321</ExpertLink>
-      <Name lang="pt">Camptodactilia - hiperplasia do tecido fibroso - displasia esquelética</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="pt">Não há dados disponíveis</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="1546">
       <OrphaCode>1323</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1323</ExpertLink>
@@ -33046,32 +32919,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="1561">
-      <OrphaCode>1342</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1342</ExpertLink>
-      <Name lang="pt">Síndrome coração - mão tipo espanhol</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="pt">Não há dados disponíveis</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="1563">
       <OrphaCode>1345</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=1345</ExpertLink>
@@ -44960,11 +44807,11 @@
       <OrphaCode>2196</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=2196</ExpertLink>
       <Name lang="pt">Hipercalciúria - coloboma da mácula</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23529">
@@ -56918,29 +56765,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="2804">
-      <OrphaCode>3133</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3133</ExpertLink>
-      <Name lang="pt">Síndrome de Say-Field-Coldwell</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="pt">esperança de vida normal</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="2805">
       <OrphaCode>3134</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=3134</ExpertLink>
@@ -64072,8 +63896,8 @@
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
+        <TypeOfInheritance id="23417">
+          <Name lang="pt">Autossómica recessiva</Name>
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
@@ -70881,7 +70705,10 @@
           <Name lang="pt">Idoso</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23641">
+          <Name lang="pt">esperança de vida normal</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -72262,7 +72089,10 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
+      <AverageAgeOfOnsetList count="3">
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23522">
           <Name lang="pt">Infância</Name>
         </AverageAgeOfOnset>
@@ -84866,32 +84696,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="21917">
-      <OrphaCode>329252</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=329252</ExpertLink>
-      <Name lang="pt">Disostose espondilocostal – hipospádias – défice intelectual</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="1">
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="21916">
       <OrphaCode>329249</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=329249</ExpertLink>
@@ -90458,7 +90262,10 @@
           <Name lang="pt">Idade adulta</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23655">
+          <Name lang="pt">Não há dados disponíveis</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -94037,12 +93844,18 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
+        <AverageAgeOfOnset id="23543">
+          <Name lang="pt">Idade adulta</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23550">
           <Name lang="pt">Idoso</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="1">
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -100448,11 +100261,11 @@
       <OrphaCode>306516</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=306516</ExpertLink>
       <Name lang="pt">Hipomagnesemia primária familiar com hipercalcúria</Name>
-      <DisorderType id="21436">
-        <Name lang="pt">Grupo de fenótipos</Name>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
       </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="pt">Grupo de patologias</Name>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23529">
@@ -120310,12 +120123,39 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="4">
+        <AverageAgeOfOnset id="23536">
+          <Name lang="pt">Adolescente</Name>
+        </AverageAgeOfOnset>
         <AverageAgeOfOnset id="23543">
           <Name lang="pt">Idade adulta</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23529">
+          <Name lang="pt">Infância</Name>
+        </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pt">adolescente</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pt">Adulto</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pt">primeira infância</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pt">infância tardia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pt">Adulto jovem</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -126179,11 +126019,11 @@
       <OrphaCode>31043</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=31043</ExpertLink>
       <Name lang="pt">Hipomagnesemia renal - hipercalciúria - nefrocalcinose</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
+      <DisorderType id="21450">
+        <Name lang="pt">Subtipo clínico</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
+      <DisorderGroup id="36554">
+        <Name lang="pt">Subtipo de patologia</Name>
       </DisorderGroup>
       <AverageAgeOfOnsetList count="1">
         <AverageAgeOfOnset id="23529">
@@ -129665,7 +129505,25 @@
           <Name lang="pt">Idoso</Name>
         </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="6">
+        <AverageAgeOfDeath id="23613">
+          <Name lang="pt">adolescente</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pt">Adulto</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23599">
+          <Name lang="pt">primeira infância</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23606">
+          <Name lang="pt">infância tardia</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23620">
+          <Name lang="pt">Adulto jovem</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -129924,32 +129782,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="10822">
-      <OrphaCode>63440</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=63440</ExpertLink>
-      <Name lang="pt">Oxicefalia</Name>
-      <DisorderType id="21415">
-        <Name lang="pt">Anomalia morfológica</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23529">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="pt">Não aplicável</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10823">
       <OrphaCode>63442</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=63442</ExpertLink>
@@ -136863,38 +136695,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="10368">
-      <OrphaCode>35098</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=35098</ExpertLink>
-      <Name lang="pt">Plagiocefalia</Name>
-      <DisorderType id="21415">
-        <Name lang="pt">Anomalia morfológica</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23641">
-          <Name lang="pt">esperança de vida normal</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23494">
-          <Name lang="pt">Não aplicável</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10375">
       <OrphaCode>35125</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=35125</ExpertLink>
@@ -138596,76 +138396,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="10344">
-      <OrphaCode>34526</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=34526</ExpertLink>
-      <Name lang="pt">Hipomagnesemia primária hereditária</Name>
-      <DisorderType id="36561">
-        <Name lang="pt">Category</Name>
-      </DisorderType>
-      <DisorderGroup id="36540">
-        <Name lang="pt">Grupo de patologias</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="3">
-        <AverageAgeOfOnset id="23536">
-          <Name lang="pt">Adolescente</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23543">
-          <Name lang="pt">Idade adulta</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23529">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="pt">Qualquer idade</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
-    <Disorder id="10345">
-      <OrphaCode>34527</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=34527</ExpertLink>
-      <Name lang="pt">Hipomagnesemia com normocalciúria</Name>
-      <DisorderType id="21394">
-        <Name lang="pt">Doença</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="3">
-        <AverageAgeOfOnset id="23536">
-          <Name lang="pt">Adolescente</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23529">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23648">
-          <Name lang="pt">Qualquer idade</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23410">
-          <Name lang="pt">Autossómica dominante</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="10346">
       <OrphaCode>34528</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=34528</ExpertLink>
@@ -145700,7 +145430,10 @@
       </AverageAgeOfOnsetList>
       <AverageAgeOfDeathList count="0">
       </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23424">
+          <Name lang="pt">Multigénico/Multifactorial</Name>
+        </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
     <Disorder id="27333">
@@ -155939,12 +155672,21 @@
       <DisorderGroup id="36547">
         <Name lang="pt">Patologia</Name>
       </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
+      <AverageAgeOfOnsetList count="2">
         <AverageAgeOfOnset id="23543">
           <Name lang="pt">Idade adulta</Name>
         </AverageAgeOfOnset>
+        <AverageAgeOfOnset id="23550">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfOnset>
       </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
+      <AverageAgeOfDeathList count="2">
+        <AverageAgeOfDeath id="23627">
+          <Name lang="pt">Adulto</Name>
+        </AverageAgeOfDeath>
+        <AverageAgeOfDeath id="23634">
+          <Name lang="pt">Idoso</Name>
+        </AverageAgeOfDeath>
       </AverageAgeOfDeathList>
       <TypeOfInheritanceList count="1">
         <TypeOfInheritance id="23494">
@@ -161357,32 +161099,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12562">
-      <OrphaCode>94095</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=94095</ExpertLink>
-      <Name lang="pt">Disostose espondilocostal - malformações anais e genito-urinárias</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="2">
-        <AverageAgeOfOnset id="23508">
-          <Name lang="pt">Pré-natal</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="1">
-        <AverageAgeOfDeath id="23655">
-          <Name lang="pt">Não há dados disponíveis</Name>
-        </AverageAgeOfDeath>
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12563">
       <OrphaCode>94122</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=94122</ExpertLink>
@@ -162016,26 +161732,6 @@
       <TypeOfInheritanceList count="0">
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12776">
-      <OrphaCode>96152</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=96152</ExpertLink>
-      <Name lang="pt">Monossomia distal 20q</Name>
-      <DisorderType id="21401">
-        <Name lang="pt">Síndrome malformativo</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="pt">Patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="1">
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="0">
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="12774">
       <OrphaCode>96150</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=96150</ExpertLink>
@@ -165257,38 +164953,6 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
-    <Disorder id="12437">
-      <OrphaCode>93604</OrphaCode>
-      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=93604</ExpertLink>
-      <Name lang="pt">Síndrome de Bartter, forma pré-natal ou infantil</Name>
-      <DisorderType id="21450">
-        <Name lang="pt">Subtipo clínico</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="pt">Subtipo de patologia</Name>
-      </DisorderGroup>
-      <AverageAgeOfOnsetList count="3">
-        <AverageAgeOfOnset id="23508">
-          <Name lang="pt">Pré-natal</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23522">
-          <Name lang="pt">Infância</Name>
-        </AverageAgeOfOnset>
-        <AverageAgeOfOnset id="23515">
-          <Name lang="pt">Neonatal</Name>
-        </AverageAgeOfOnset>
-      </AverageAgeOfOnsetList>
-      <AverageAgeOfDeathList count="0">
-      </AverageAgeOfDeathList>
-      <TypeOfInheritanceList count="2">
-        <TypeOfInheritance id="23417">
-          <Name lang="pt">Autossómica recessiva</Name>
-        </TypeOfInheritance>
-        <TypeOfInheritance id="23431">
-          <Name lang="pt">Ligado ao X recessivo</Name>
-        </TypeOfInheritance>
-      </TypeOfInheritanceList>
-    </Disorder>
     <Disorder id="14111">
       <OrphaCode>99094</OrphaCode>
       <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=99094</ExpertLink>
@@ -177422,5 +177086,45 @@
         </TypeOfInheritance>
       </TypeOfInheritanceList>
     </Disorder>
+    <Disorder id="31415">
+      <OrphaCode>619363</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=619363</ExpertLink>
+      <Name lang="pt">Neonatal onset-severe multisystem auto-inflammatory disease-increased IL-18</Name>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="pt">Autossómica dominante</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
+    <Disorder id="31405">
+      <OrphaCode>619233</OrphaCode>
+      <ExpertLink lang="pt">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=pt&amp;Expert=619233</ExpertLink>
+      <Name lang="pt">Hereditary persistence of fetal hemoglobin-intellectual disability syndrome</Name>
+      <DisorderType id="21394">
+        <Name lang="pt">Doença</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="pt">Patologia</Name>
+      </DisorderGroup>
+      <AverageAgeOfOnsetList count="0">
+      </AverageAgeOfOnsetList>
+      <AverageAgeOfDeathList count="0">
+      </AverageAgeOfDeathList>
+      <TypeOfInheritanceList count="1">
+        <TypeOfInheritance id="23410">
+          <Name lang="pt">Autossómica dominante</Name>
+        </TypeOfInheritance>
+      </TypeOfInheritanceList>
+    </Disorder>
   </DisorderList>
 </JDBOR>
