commit 39c0a07f6c7cd2d512f3042c2ffe7b1329ca806a
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Tue Jun 14 17:59:25 2022 +0200

    JUNE 2022

diff --git a/Genes associated with rare diseases/en_product6.xml b/Genes associated with rare diseases/en_product6.xml
index af75d6a..aca6af8 100644
--- a/Genes associated with rare diseases/en_product6.xml	
+++ b/Genes associated with rare diseases/en_product6.xml	
@@ -1,12 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-06-07 04:39:05" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
-  <Availability> 
-    <Licence>
-      <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
-      <ShortIdentifier>CC-BY-4.0</ShortIdentifier>
-      <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
-    </Licence>
-  </Availability>
+<JDBOR date="2022-06-14 16:27:47" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <DisorderList count="3886">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
@@ -60389,7 +60382,7 @@
     <Disorder id="678">
       <OrphaCode>62</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=62</ExpertLink>
-      <Name lang="en">Alpha-sarcoglycan-related  limb-girdle muscular dystrophy R3</Name>
+      <Name lang="en">Alpha-sarcoglycan-related limb-girdle muscular dystrophy R3</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -84720,7 +84713,7 @@
     <Disorder id="870">
       <OrphaCode>267</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=267</ExpertLink>
-      <Name lang="en">Calpain-3-related  limb-girdle muscular dystrophy R1</Name>
+      <Name lang="en">Calpain-3-related limb-girdle muscular dystrophy R1</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -102003,7 +101996,7 @@
     <Disorder id="1195">
       <OrphaCode>1878</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1878</ExpertLink>
-      <Name lang="en">TRIM32-related  limb-girdle muscular dystrophy R8</Name>
+      <Name lang="en">TRIM32-related limb-girdle muscular dystrophy R8</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -118670,7 +118663,7 @@
     <Disorder id="16999">
       <OrphaCode>140922</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=140922</ExpertLink>
-      <Name lang="en">Titin-related  limb-girdle muscular dystrophy R10</Name>
+      <Name lang="en">Titin-related limb-girdle muscular dystrophy R10</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -152009,7 +152002,7 @@
     <Disorder id="19742">
       <OrphaCode>254361</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=254361</ExpertLink>
-      <Name lang="en">Plectin-related  limb-girdle muscular dystrophy R17</Name>
+      <Name lang="en">Plectin-related limb-girdle muscular dystrophy R17</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -178439,7 +178432,7 @@
     <Disorder id="20455">
       <OrphaCode>280333</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=280333</ExpertLink>
-      <Name lang="en">Alpha-dystroglycan-related  limb-girdle muscular dystrophy R16</Name>
+      <Name lang="en">Alpha-dystroglycan-related limb-girdle muscular dystrophy R16</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -189773,7 +189766,7 @@
     <Disorder id="18521">
       <OrphaCode>206554</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206554</ExpertLink>
-      <Name lang="en">Fukutin-related  limb-girdle muscular dystrophy R13</Name>
+      <Name lang="en">Fukutin-related limb-girdle muscular dystrophy R13</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -189833,7 +189826,7 @@
     <Disorder id="18520">
       <OrphaCode>206549</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206549</ExpertLink>
-      <Name lang="en">Anoctamin-5-related  limb-girdle muscular dystrophy R12</Name>
+      <Name lang="en">Anoctamin-5-related limb-girdle muscular dystrophy R12</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -189897,7 +189890,7 @@
     <Disorder id="18523">
       <OrphaCode>206564</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206564</ExpertLink>
-      <Name lang="en">POMGNT1-related  limb-girdle muscular dystrophy R15</Name>
+      <Name lang="en">POMGNT1-related limb-girdle muscular dystrophy R15</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -189964,7 +189957,7 @@
     <Disorder id="18522">
       <OrphaCode>206559</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=206559</ExpertLink>
-      <Name lang="en">POMT2-related  limb-girdle muscular dystrophy R14</Name>
+      <Name lang="en">POMT2-related limb-girdle muscular dystrophy R14</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -251465,7 +251458,7 @@
     <Disorder id="22434">
       <OrphaCode>369840</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=369840</ExpertLink>
-      <Name lang="en">TRAPPC11-related  limb-girdle muscular dystrophy R18</Name>
+      <Name lang="en">TRAPPC11-related limb-girdle muscular dystrophy R18</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -255102,7 +255095,7 @@
     <Disorder id="22329">
       <OrphaCode>363623</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363623</ExpertLink>
-      <Name lang="en">GMPPB-related  limb-girdle muscular dystrophy R19</Name>
+      <Name lang="en">GMPPB-related limb-girdle muscular dystrophy R19</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -259977,7 +259970,7 @@
     <Disorder id="22066">
       <OrphaCode>352479</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352479</ExpertLink>
-      <Name lang="en">ISPD-related  limb-girdle muscular dystrophy R20</Name>
+      <Name lang="en">ISPD-related limb-girdle muscular dystrophy R20</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -321429,7 +321422,7 @@
     <Disorder id="8716">
       <OrphaCode>353</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353</ExpertLink>
-      <Name lang="en">Gamma-sarcoglycan-related  limb-girdle muscular dystrophy R5</Name>
+      <Name lang="en">Gamma-sarcoglycan-related limb-girdle muscular dystrophy R5</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -321503,7 +321496,7 @@
     <Disorder id="8717">
       <OrphaCode>219</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=219</ExpertLink>
-      <Name lang="en">Delta-sarcoglycan-related  limb-girdle muscular dystrophy R6</Name>
+      <Name lang="en">Delta-sarcoglycan-related limb-girdle muscular dystrophy R6</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -321565,7 +321558,7 @@
     <Disorder id="8715">
       <OrphaCode>119</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=119</ExpertLink>
-      <Name lang="en">Beta-sarcoglycan-related  limb-girdle muscular dystrophy R4</Name>
+      <Name lang="en">Beta-sarcoglycan-related limb-girdle muscular dystrophy R4</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -323283,7 +323276,7 @@
     <Disorder id="8720">
       <OrphaCode>268</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268</ExpertLink>
-      <Name lang="en">Dysferlin-related  limb-girdle muscular dystrophy R2</Name>
+      <Name lang="en">Dysferlin-related limb-girdle muscular dystrophy R2</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -337199,7 +337192,7 @@
     <Disorder id="25220">
       <OrphaCode>480682</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=480682</ExpertLink>
-      <Name lang="en">POGLUT1-related  limb-girdle muscular dystrophy R21</Name>
+      <Name lang="en">POGLUT1-related limb-girdle muscular dystrophy R21</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -350229,7 +350222,7 @@
     <Disorder id="10758">
       <OrphaCode>55595</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=55595</ExpertLink>
-      <Name lang="en">TNP03-related  limb-girdle muscular dystrophy D2</Name>
+      <Name lang="en">TNP03-related limb-girdle muscular dystrophy D2</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -350293,7 +350286,7 @@
     <Disorder id="10759">
       <OrphaCode>55596</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=55596</ExpertLink>
-      <Name lang="en">HNRNPDL-related  limb-girdle muscular dystrophy D3</Name>
+      <Name lang="en">HNRNPDL-related limb-girdle muscular dystrophy D3</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -367321,7 +367314,7 @@
     <Disorder id="10336">
       <OrphaCode>34514</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34514</ExpertLink>
-      <Name lang="en">Telethonin-related  limb-girdle muscular dystrophy R7</Name>
+      <Name lang="en">Telethonin-related limb-girdle muscular dystrophy R7</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -367390,7 +367383,7 @@
     <Disorder id="10337">
       <OrphaCode>34515</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34515</ExpertLink>
-      <Name lang="en">FKRP-related  limb-girdle muscular dystrophy R9</Name>
+      <Name lang="en">FKRP-related limb-girdle muscular dystrophy R9</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -367452,7 +367445,7 @@
     <Disorder id="10338">
       <OrphaCode>34516</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=34516</ExpertLink>
-      <Name lang="en">DNAJB6-related  limb-girdle muscular dystrophy D1</Name>
+      <Name lang="en">DNAJB6-related limb-girdle muscular dystrophy D1</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -415580,7 +415573,7 @@
     <Disorder id="11732">
       <OrphaCode>86812</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=86812</ExpertLink>
-      <Name lang="en">POMT1-related  limb-girdle muscular dystrophy R11</Name>
+      <Name lang="en">POMT1-related limb-girdle muscular dystrophy R11</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
