commit f6243403226f6a943b8a6925758d3dfb41a1e668
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Sat Jan 2 20:37:59 2021 +0100

    JAN 2021

diff --git a/Genes associated with rare diseases/en_product6.xml b/Genes associated with rare diseases/en_product6.xml
old mode 100644
new mode 100755
index 9ba331e..2838df6
--- a/Genes associated with rare diseases/en_product6.xml	
+++ b/Genes associated with rare diseases/en_product6.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2020-12-01 04:38:58" version="1.3.6 / 4.1.7 [2020-04-17] (orientdb version)" copyright="Orphanet (c) 2020" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-01-01 04:39:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="3859">
+  <DisorderList count="3858">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -41706,7 +41706,7 @@
     <Disorder id="421">
       <OrphaCode>1146</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1146</ExpertLink>
-      <Name lang="en">Digitotalar dysmorphism</Name>
+      <Name lang="en">Distal arthrogryposis type 1</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -214779,7 +214779,7 @@
     <Disorder id="3735">
       <OrphaCode>2126</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2126</ExpertLink>
-      <Name lang="en">Solitary fibrous tumour/hemangiopericytoma</Name>
+      <Name lang="en">Solitary fibrous tumor/hemangiopericytoma</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -236490,7 +236490,7 @@
     <Disorder id="21690">
       <OrphaCode>319671</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=319671</ExpertLink>
-      <Name lang="en">Microcephalic primordial dwarfism, Alazami type</Name>
+      <Name lang="en">Alazami syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -247511,7 +247511,7 @@
     <Disorder id="22327">
       <OrphaCode>363611</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=363611</ExpertLink>
-      <Name lang="en">Intellectual disability-feeding difficulties-developmental delay-microcephaly syndrome</Name>
+      <Name lang="en">CTCF-related neurodevelopmental disorder</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -250213,8 +250213,8 @@
       <OrphaCode>353277</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353277</ExpertLink>
       <Name lang="en">Rubinstein-Taybi syndrome due to CREBBP mutations</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -251492,8 +251492,8 @@
       <OrphaCode>353281</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353281</ExpertLink>
       <Name lang="en">Rubinstein-Taybi syndrome due to 16p13.3 microdeletion</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -251562,8 +251562,8 @@
       <OrphaCode>353284</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=353284</ExpertLink>
       <Name lang="en">Rubinstein-Taybi syndrome due to EP300 haploinsufficiency</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -262482,71 +262482,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="20606">
-      <OrphaCode>284408</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284408</ExpertLink>
-      <Name lang="en">Glycerol kinase deficiency, infantile form</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>8651297[PMID]</SourceOfValidation>
-          <Gene id="16134">
-            <Name lang="en">glycerol kinase</Name>
-            <Symbol>GK</Symbol>
-            <SynonymList count="2">
-              <Synonym lang="en">GK1</Synonym>
-              <Synonym lang="en">GKD</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="33149">
-                <Source>SwissProt</Source>
-                <Reference>P32189</Reference>
-              </ExternalReference>
-              <ExternalReference id="57310">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000198814</Reference>
-              </ExternalReference>
-              <ExternalReference id="29870">
-                <Source>Genatlas</Source>
-                <Reference>GK</Reference>
-              </ExternalReference>
-              <ExternalReference id="29872">
-                <Source>HGNC</Source>
-                <Reference>4289</Reference>
-              </ExternalReference>
-              <ExternalReference id="29871">
-                <Source>OMIM</Source>
-                <Reference>300474</Reference>
-              </ExternalReference>
-              <ExternalReference id="57311">
-                <Source>Reactome</Source>
-                <Reference>P32189</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="21443">
-                <GeneLocus>Xp21.2</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="20607">
       <OrphaCode>284411</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=284411</ExpertLink>
@@ -271307,7 +271242,7 @@
     <Disorder id="24042">
       <OrphaCode>464738</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464738</ExpertLink>
-      <Name lang="en">Congenital cataract-microcephaly-nevus flammeus simplex-severe intellectual disability syndrome</Name>
+      <Name lang="en">Basel-Vanagaite-Smirin-Yosef syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
