commit 46569fa389bb2350b5d923f7ab0e9d09817eb4bc
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon May 2 10:13:18 2022 +0200

    MAY 2022

diff --git a/Genes associated with rare diseases/en_product6.xml b/Genes associated with rare diseases/en_product6.xml
index 28dff8e..da84787 100644
--- a/Genes associated with rare diseases/en_product6.xml	
+++ b/Genes associated with rare diseases/en_product6.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2022-04-01 04:58:12" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2022-05-01 04:55:23" version="1.3.16 / 4.1.7 [2022-01-26] (orientdb version)" copyright="Orphanet (c) 2022" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="3887">
+  <DisorderList count="3886">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -5670,121 +5670,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="17543">
-      <OrphaCode>163908</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=163908</ExpertLink>
-      <Name lang="en">Limbic encephalitis with LGI1 antibodies</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="2">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>28026029_28026046[PMID]</SourceOfValidation>
-          <Gene id="16202">
-            <Name lang="en">major histocompatibility complex, class II, DR beta 1</Name>
-            <Symbol>HLA-DRB1</Symbol>
-            <SynonymList count="0">
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="56818">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000196126</Reference>
-              </ExternalReference>
-              <ExternalReference id="37478">
-                <Source>Genatlas</Source>
-                <Reference>HLA-DRB1</Reference>
-              </ExternalReference>
-              <ExternalReference id="30196">
-                <Source>HGNC</Source>
-                <Reference>4948</Reference>
-              </ExternalReference>
-              <ExternalReference id="30195">
-                <Source>OMIM</Source>
-                <Reference>142857</Reference>
-              </ExternalReference>
-              <ExternalReference id="56819">
-                <Source>Reactome</Source>
-                <Reference>P04229</Reference>
-              </ExternalReference>
-              <ExternalReference id="189394">
-                <Source>SwissProt</Source>
-                <Reference>P01911</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="58635">
-                <GeneLocus>6p21.32</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17961">
-            <Name lang="en">Major susceptibility factor in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-        <DisorderGeneAssociation>
-          <SourceOfValidation>28026029_28026046[PMID]</SourceOfValidation>
-          <Gene id="18703">
-            <Name lang="en">major histocompatibility complex, class II, DQ beta 1</Name>
-            <Symbol>HLA-DQB1</Symbol>
-            <SynonymList count="2">
-              <Synonym lang="en">CELIAC1</Synonym>
-              <Synonym lang="en">IDDM1</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="56816">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000179344</Reference>
-              </ExternalReference>
-              <ExternalReference id="43209">
-                <Source>Genatlas</Source>
-                <Reference>HLA-DQB1</Reference>
-              </ExternalReference>
-              <ExternalReference id="43210">
-                <Source>HGNC</Source>
-                <Reference>4944</Reference>
-              </ExternalReference>
-              <ExternalReference id="43211">
-                <Source>OMIM</Source>
-                <Reference>604305</Reference>
-              </ExternalReference>
-              <ExternalReference id="82670">
-                <Source>Reactome</Source>
-                <Reference>P01920</Reference>
-              </ExternalReference>
-              <ExternalReference id="82625">
-                <Source>SwissProt</Source>
-                <Reference>P01920</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="24817">
-                <GeneLocus>6p21.32</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17961">
-            <Name lang="en">Major susceptibility factor in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="64">
       <OrphaCode>551</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=551</ExpertLink>
@@ -13726,7 +13611,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation>20301558[PMID]</SourceOfValidation>
           <Gene id="16063">
-            <Name lang="en">FMRP translational regulator 1</Name>
+            <Name lang="en">fragile X messenger ribonucleoprotein 1</Name>
             <Symbol>FMR1</Symbol>
             <SynonymList count="3">
               <Synonym lang="en">FMRP</Synonym>
@@ -19372,14 +19257,26 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="1">
+            <ExternalReferenceList count="4">
               <ExternalReference id="205335">
                 <Source>HGNC</Source>
                 <Reference>17722</Reference>
               </ExternalReference>
+              <ExternalReference id="205732">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000198369</Reference>
+              </ExternalReference>
+              <ExternalReference id="205733">
+                <Source>OMIM</Source>
+                <Reference>609292</Reference>
+              </ExternalReference>
+              <ExternalReference id="205734">
+                <Source>SwissProt</Source>
+                <Reference>Q7Z698</Reference>
+              </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="88051">
+              <Locus id="88171">
                 <GeneLocus>2p14</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -23627,7 +23524,7 @@
     <Disorder id="244">
       <OrphaCode>207</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=207</ExpertLink>
-      <Name lang="en">Crouzon disease</Name>
+      <Name lang="en">Crouzon syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -31242,14 +31139,26 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="1">
+            <ExternalReferenceList count="4">
               <ExternalReference id="205331">
                 <Source>HGNC</Source>
                 <Reference>6396</Reference>
               </ExternalReference>
+              <ExternalReference id="205738">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000102753</Reference>
+              </ExternalReference>
+              <ExternalReference id="205739">
+                <Source>OMIM</Source>
+                <Reference>601892</Reference>
+              </ExternalReference>
+              <ExternalReference id="205740">
+                <Source>SwissProt</Source>
+                <Reference>O00505</Reference>
+              </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="88043">
+              <Locus id="88183">
                 <GeneLocus>13q14.2</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -32805,7 +32714,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation/>
           <Gene id="18441">
-            <Name lang="en">CD3g molecule</Name>
+            <Name lang="en">CD3 gamma subunit of T-cell receptor complex</Name>
             <Symbol>CD3G</Symbol>
             <SynonymList count="0">
             </SynonymList>
@@ -36444,7 +36353,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation/>
           <Gene id="17285">
-            <Name lang="en">CD3d molecule</Name>
+            <Name lang="en">CD3 delta subunit of T-cell receptor complex</Name>
             <Symbol>CD3D</Symbol>
             <SynonymList count="0">
             </SynonymList>
@@ -36494,7 +36403,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation/>
           <Gene id="17286">
-            <Name lang="en">CD3e molecule</Name>
+            <Name lang="en">CD3 epsilon subunit of T-cell receptor complex</Name>
             <Symbol>CD3E</Symbol>
             <SynonymList count="0">
             </SynonymList>
@@ -51435,7 +51344,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="46">
+      <DisorderGeneAssociationList count="47">
         <DisorderGeneAssociation>
           <SourceOfValidation>28472305[PMID]</SourceOfValidation>
           <Gene id="25572">
@@ -53881,6 +53790,36 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>35044787[PMID]</SourceOfValidation>
+          <Gene id="31454">
+            <Name lang="en">BAG cochaperone 5</Name>
+            <Symbol>BAG5</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="1">
+              <ExternalReference id="205798">
+                <Source>HGNC</Source>
+                <Reference>941</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="88291">
+                <GeneLocus>14q32.33</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="25972">
+            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
     <Disorder id="634">
@@ -57758,7 +57697,66 @@
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="40">
+      <DisorderGeneAssociationList count="42">
+        <DisorderGeneAssociation>
+          <SourceOfValidation>33754465[PMID]</SourceOfValidation>
+          <Gene id="16296">
+            <Name lang="en">potassium voltage-gated channel subfamily Q member 2</Name>
+            <Symbol>KCNQ2</Symbol>
+            <SynonymList count="5">
+              <Synonym lang="en">BFNC</Synonym>
+              <Synonym lang="en">ENB1</Synonym>
+              <Synonym lang="en">HNSPC</Synonym>
+              <Synonym lang="en">KCNA11</Synonym>
+              <Synonym lang="en">Kv7.2</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="7">
+              <ExternalReference id="57761">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000075043</Reference>
+              </ExternalReference>
+              <ExternalReference id="30645">
+                <Source>Genatlas</Source>
+                <Reference>KCNQ2</Reference>
+              </ExternalReference>
+              <ExternalReference id="30643">
+                <Source>HGNC</Source>
+                <Reference>6296</Reference>
+              </ExternalReference>
+              <ExternalReference id="82971">
+                <Source>IUPHAR</Source>
+                <Reference>561</Reference>
+              </ExternalReference>
+              <ExternalReference id="30642">
+                <Source>OMIM</Source>
+                <Reference>602235</Reference>
+              </ExternalReference>
+              <ExternalReference id="57762">
+                <Source>Reactome</Source>
+                <Reference>O43526</Reference>
+              </ExternalReference>
+              <ExternalReference id="33361">
+                <Source>SwissProt</Source>
+                <Reference>O43526</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="9009">
+                <GeneLocus>20q13.33</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>33603162[PMID]</SourceOfValidation>
           <Gene id="22474">
@@ -58022,6 +58020,58 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34707297[PMID]</SourceOfValidation>
+          <Gene id="28565">
+            <Name lang="en">intersectin 1</Name>
+            <Symbol>ITSN1</Symbol>
+            <SynonymList count="6">
+              <Synonym lang="en">MGC134948</Synonym>
+              <Synonym lang="en">MGC134949</Synonym>
+              <Synonym lang="en">SH3 domain protein-1A</Synonym>
+              <Synonym lang="en">SH3P17</Synonym>
+              <Synonym lang="en">Src homology 3 domain-containing protein</Synonym>
+              <Synonym lang="en">human intersectin-SH3 domain-containing protein SH3P17</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="5">
+              <ExternalReference id="178468">
+                <Source>HGNC</Source>
+                <Reference>6183</Reference>
+              </ExternalReference>
+              <ExternalReference id="178469">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000205726</Reference>
+              </ExternalReference>
+              <ExternalReference id="178470">
+                <Source>SwissProt</Source>
+                <Reference>Q15811</Reference>
+              </ExternalReference>
+              <ExternalReference id="178471">
+                <Source>Reactome</Source>
+                <Reference>Q15811</Reference>
+              </ExternalReference>
+              <ExternalReference id="178472">
+                <Source>OMIM</Source>
+                <Reference>602442</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="53369">
+                <GeneLocus>21q22.11</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>34092786[PMID]</SourceOfValidation>
           <Gene id="22910">
@@ -76595,7 +76645,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="6">
+      <DisorderGeneAssociationList count="7">
         <DisorderGeneAssociation>
           <SourceOfValidation>27259053[PMID]</SourceOfValidation>
           <Gene id="23911">
@@ -76913,6 +76963,58 @@
             <Name lang="en">Not yet assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34890546[PMID]</SourceOfValidation>
+          <Gene id="21155">
+            <Name lang="en">intraflagellar transport 140</Name>
+            <Symbol>IFT140</Symbol>
+            <SynonymList count="2">
+              <Synonym lang="en">KIAA0590</Synonym>
+              <Synonym lang="en">gs114</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="6">
+              <ExternalReference id="83376">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000187535</Reference>
+              </ExternalReference>
+              <ExternalReference id="69514">
+                <Source>Genatlas</Source>
+                <Reference>IFT140</Reference>
+              </ExternalReference>
+              <ExternalReference id="69512">
+                <Source>HGNC</Source>
+                <Reference>29077</Reference>
+              </ExternalReference>
+              <ExternalReference id="69513">
+                <Source>OMIM</Source>
+                <Reference>614620</Reference>
+              </ExternalReference>
+              <ExternalReference id="97325">
+                <Source>Reactome</Source>
+                <Reference>Q96RY7</Reference>
+              </ExternalReference>
+              <ExternalReference id="69515">
+                <Source>SwissProt</Source>
+                <Reference>Q96RY7</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="16897">
+                <GeneLocus>16p13.3</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="25972">
+            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
     <Disorder id="18422">
@@ -76995,7 +77097,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="4">
+      <DisorderGeneAssociationList count="5">
         <DisorderGeneAssociation>
           <SourceOfValidation>24753205[PMID]</SourceOfValidation>
           <Gene id="15593">
@@ -77174,6 +77276,58 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34115842[PMID]</SourceOfValidation>
+          <Gene id="23613">
+            <Name lang="en">caseinolytic mitochondrial matrix peptidase chaperone subunit B</Name>
+            <Symbol>CLPB</Symbol>
+            <SynonymList count="6">
+              <Synonym lang="en">ANKCLB</Synonym>
+              <Synonym lang="en">FLJ13152</Synonym>
+              <Synonym lang="en">HSP78</Synonym>
+              <Synonym lang="en">SKD3</Synonym>
+              <Synonym lang="en">ankyrin-repeat containing bacterial clp fusion</Synonym>
+              <Synonym lang="en">suppressor of potassium transport defect 3</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="5">
+              <ExternalReference id="98650">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000162129</Reference>
+              </ExternalReference>
+              <ExternalReference id="98648">
+                <Source>Genatlas</Source>
+                <Reference>CLPB</Reference>
+              </ExternalReference>
+              <ExternalReference id="98646">
+                <Source>HGNC</Source>
+                <Reference>30664</Reference>
+              </ExternalReference>
+              <ExternalReference id="98647">
+                <Source>OMIM</Source>
+                <Reference>616254</Reference>
+              </ExternalReference>
+              <ExternalReference id="98649">
+                <Source>SwissProt</Source>
+                <Reference>Q9H078</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="18393">
+                <GeneLocus>11q13.4</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="25972">
+            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>29914977[PMID]</SourceOfValidation>
           <Gene id="26096">
@@ -82290,7 +82444,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="16">
+      <DisorderGeneAssociationList count="17">
         <DisorderGeneAssociation>
           <SourceOfValidation>31688942[PMID]</SourceOfValidation>
           <Gene id="23734">
@@ -82341,6 +82495,61 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34611970[PMID]</SourceOfValidation>
+          <Gene id="20558">
+            <Name lang="en">glutamate ionotropic receptor NMDA type subunit 1</Name>
+            <Symbol>GRIN1</Symbol>
+            <SynonymList count="1">
+              <Synonym lang="en">GluN1</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="7">
+              <ExternalReference id="60250">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000176884</Reference>
+              </ExternalReference>
+              <ExternalReference id="54465">
+                <Source>Genatlas</Source>
+                <Reference>GRIN1</Reference>
+              </ExternalReference>
+              <ExternalReference id="54463">
+                <Source>HGNC</Source>
+                <Reference>4584</Reference>
+              </ExternalReference>
+              <ExternalReference id="83220">
+                <Source>IUPHAR</Source>
+                <Reference>455</Reference>
+              </ExternalReference>
+              <ExternalReference id="54464">
+                <Source>OMIM</Source>
+                <Reference>138249</Reference>
+              </ExternalReference>
+              <ExternalReference id="60251">
+                <Source>Reactome</Source>
+                <Reference>Q05586</Reference>
+              </ExternalReference>
+              <ExternalReference id="54466">
+                <Source>SwissProt</Source>
+                <Reference>Q05586</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="11641">
+                <GeneLocus>9q34.3</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="25972">
+            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>30323019[PMID]</SourceOfValidation>
           <Gene id="28125">
@@ -87618,7 +87827,7 @@
     <Disorder id="946">
       <OrphaCode>3027</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3027</ExpertLink>
-      <Name lang="en">Caudal regression sequence</Name>
+      <Name lang="en">Caudal regression syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -90486,69 +90695,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="915">
-      <OrphaCode>3315</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3315</ExpertLink>
-      <Name lang="en">NON RARE IN EUROPE: Thiopurine S-methyltransferase deficiency</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation/>
-          <Gene id="15650">
-            <Name lang="en">thiopurine S-methyltransferase</Name>
-            <Symbol>TPMT</Symbol>
-            <SynonymList count="0">
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="57816">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000137364</Reference>
-              </ExternalReference>
-              <ExternalReference id="27548">
-                <Source>Genatlas</Source>
-                <Reference>TPMT</Reference>
-              </ExternalReference>
-              <ExternalReference id="27546">
-                <Source>HGNC</Source>
-                <Reference>12014</Reference>
-              </ExternalReference>
-              <ExternalReference id="27545">
-                <Source>OMIM</Source>
-                <Reference>187680</Reference>
-              </ExternalReference>
-              <ExternalReference id="57817">
-                <Source>Reactome</Source>
-                <Reference>P51580</Reference>
-              </ExternalReference>
-              <ExternalReference id="32622">
-                <Source>SwissProt</Source>
-                <Reference>P51580</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="7803">
-                <GeneLocus>6p22.3</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="29686">
-            <Name lang="en">Biomarker tested in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="912">
       <OrphaCode>373</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=373</ExpertLink>
@@ -94773,7 +94919,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="9">
+      <DisorderGeneAssociationList count="10">
         <DisorderGeneAssociation>
           <SourceOfValidation/>
           <Gene id="17744">
@@ -95147,6 +95293,36 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34402903[PMID]</SourceOfValidation>
+          <Gene id="31453">
+            <Name lang="en">zinc finger SWIM-type containing 7</Name>
+            <Symbol>ZSWIM7</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="1">
+              <ExternalReference id="205796">
+                <Source>HGNC</Source>
+                <Reference>26993</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="88287">
+                <GeneLocus>17p12</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>30830215[PMID]</SourceOfValidation>
           <Gene id="28619">
@@ -95160,7 +95336,11 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="4">
+            <ExternalReferenceList count="5">
+              <ExternalReference id="205716">
+                <Source>OMIM</Source>
+                <Reference>619801</Reference>
+              </ExternalReference>
               <ExternalReference id="178833">
                 <Source>HGNC</Source>
                 <Reference>30349</Reference>
@@ -95179,7 +95359,7 @@
               </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="53541">
+              <Locus id="88135">
                 <GeneLocus>22q13.2</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -108300,7 +108480,7 @@
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="3">
+      <DisorderGeneAssociationList count="4">
         <DisorderGeneAssociation>
           <SourceOfValidation>17435757[PMID]_21733974[PMID]</SourceOfValidation>
           <Gene id="17443">
@@ -108357,6 +108537,36 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34932939[PMID]</SourceOfValidation>
+          <Gene id="31455">
+            <Name lang="en">dynein heavy chain domain 1</Name>
+            <Symbol>DNHD1</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="1">
+              <ExternalReference id="205800">
+                <Source>HGNC</Source>
+                <Reference>26532</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="88295">
+                <GeneLocus>11p15.4</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>33472045[PMID]</SourceOfValidation>
           <Gene id="30247">
@@ -120326,7 +120536,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation>23222957[PMID]</SourceOfValidation>
           <Gene id="21753">
-            <Name lang="en">ectopic P-granules autophagy protein 5 homolog</Name>
+            <Name lang="en">ectopic P-granules 5 autophagy tethering factor</Name>
             <Symbol>EPG5</Symbol>
             <SynonymList count="1">
               <Synonym lang="en">hEPG5</Synonym>
@@ -120549,7 +120759,7 @@
     <Disorder id="1684">
       <OrphaCode>1517</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1517</ExpertLink>
-      <Name lang="en">Hypertrichotic osteochondrodysplasia, Cantu type</Name>
+      <Name lang="en">Cantú syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -151316,7 +151526,7 @@
     <Disorder id="2497">
       <OrphaCode>661</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=661</ExpertLink>
-      <Name lang="en">Ondine syndrome</Name>
+      <Name lang="en">Congenital central hypoventilation syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -155144,7 +155354,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation>21188402[PMID]_24157186[PMID]</SourceOfValidation>
           <Gene id="16063">
-            <Name lang="en">FMRP translational regulator 1</Name>
+            <Name lang="en">fragile X messenger ribonucleoprotein 1</Name>
             <Symbol>FMR1</Symbol>
             <SynonymList count="3">
               <Synonym lang="en">FMRP</Synonym>
@@ -160414,7 +160624,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation>19844254[PMID]</SourceOfValidation>
           <Gene id="16063">
-            <Name lang="en">FMRP translational regulator 1</Name>
+            <Name lang="en">fragile X messenger ribonucleoprotein 1</Name>
             <Symbol>FMR1</Symbol>
             <SynonymList count="3">
               <Synonym lang="en">FMRP</Synonym>
@@ -171526,7 +171736,7 @@
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="5">
+      <DisorderGeneAssociationList count="6">
         <DisorderGeneAssociation>
           <SourceOfValidation>34145744[PMID]</SourceOfValidation>
           <Gene id="31365">
@@ -171537,14 +171747,26 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="1">
+            <ExternalReferenceList count="4">
               <ExternalReference id="205327">
                 <Source>HGNC</Source>
                 <Reference>12779</Reference>
               </ExternalReference>
+              <ExternalReference id="205744">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000158955</Reference>
+              </ExternalReference>
+              <ExternalReference id="205745">
+                <Source>OMIM</Source>
+                <Reference>602864</Reference>
+              </ExternalReference>
+              <ExternalReference id="205746">
+                <Source>SwissProt</Source>
+                <Reference>O14905</Reference>
+              </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="88035">
+              <Locus id="88195">
                 <GeneLocus>17q21.32</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -171770,6 +171992,36 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34737117[PMID]</SourceOfValidation>
+          <Gene id="31452">
+            <Name lang="en">GDNF family receptor alpha 1</Name>
+            <Symbol>GFRA1</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="1">
+              <ExternalReference id="205794">
+                <Source>HGNC</Source>
+                <Reference>4243</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="88283">
+                <GeneLocus>10q25.3</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
     <Disorder id="2662">
@@ -272006,7 +272258,7 @@
     <Disorder id="21494">
       <OrphaCode>314555</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=314555</ExpertLink>
-      <Name lang="en">Facial dysmorphism-severe myopia-osteopenia-intellectual disability-dental anomalies syndrome</Name>
+      <Name lang="en">Facial dysmorphism-ocular anomalies-osteopenia-intellectual disability-dental anomalies syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -281104,7 +281356,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation>20301558[PMID]</SourceOfValidation>
           <Gene id="16063">
-            <Name lang="en">FMRP translational regulator 1</Name>
+            <Name lang="en">fragile X messenger ribonucleoprotein 1</Name>
             <Symbol>FMR1</Symbol>
             <SynonymList count="3">
               <Synonym lang="en">FMRP</Synonym>
@@ -282061,77 +282313,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="23782">
-      <OrphaCode>459353</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=459353</ExpertLink>
-      <Name lang="en">C1 inhibitor deficiency</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>7883978[PMID]</SourceOfValidation>
-          <Gene id="15274">
-            <Name lang="en">serpin family G member 1</Name>
-            <Symbol>SERPING1</Symbol>
-            <SynonymList count="8">
-              <Synonym lang="en">C1-INH</Synonym>
-              <Synonym lang="en">C1-inhibitor</Synonym>
-              <Synonym lang="en">C1IN</Synonym>
-              <Synonym lang="en">C1INH</Synonym>
-              <Synonym lang="en">HAE1</Synonym>
-              <Synonym lang="en">HAE2</Synonym>
-              <Synonym lang="en">angioedema, hereditary</Synonym>
-              <Synonym lang="en">plasma protease C1 inhibitor</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="59985">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000149131</Reference>
-              </ExternalReference>
-              <ExternalReference id="25732">
-                <Source>Genatlas</Source>
-                <Reference>SERPING1</Reference>
-              </ExternalReference>
-              <ExternalReference id="25730">
-                <Source>HGNC</Source>
-                <Reference>1228</Reference>
-              </ExternalReference>
-              <ExternalReference id="25729">
-                <Source>OMIM</Source>
-                <Reference>606860</Reference>
-              </ExternalReference>
-              <ExternalReference id="59986">
-                <Source>Reactome</Source>
-                <Reference>P05155</Reference>
-              </ExternalReference>
-              <ExternalReference id="33832">
-                <Source>SwissProt</Source>
-                <Reference>P05155</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="7109">
-                <GeneLocus>11q12.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="23767">
       <OrphaCode>459033</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=459033</ExpertLink>
@@ -379490,6 +379671,70 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
+    <Disorder id="11993">
+      <OrphaCode>90186</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90186</ExpertLink>
+      <Name lang="en">Meige disease</Name>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <DisorderGeneAssociationList count="1">
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34231312[PMID]</SourceOfValidation>
+          <Gene id="26623">
+            <Name lang="en">EPH receptor B4</Name>
+            <Symbol>EPHB4</Symbol>
+            <SynonymList count="1">
+              <Synonym lang="en">Tyro11</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="6">
+              <ExternalReference id="200506">
+                <Source>SwissProt</Source>
+                <Reference>P54760</Reference>
+              </ExternalReference>
+              <ExternalReference id="156486">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000196411</Reference>
+              </ExternalReference>
+              <ExternalReference id="156487">
+                <Source>IUPHAR</Source>
+                <Reference>1833</Reference>
+              </ExternalReference>
+              <ExternalReference id="190647">
+                <Source>OMIM</Source>
+                <Reference>600011</Reference>
+              </ExternalReference>
+              <ExternalReference id="156315">
+                <Source>HGNC</Source>
+                <Reference>3395</Reference>
+              </ExternalReference>
+              <ExternalReference id="156893">
+                <Source>Genatlas</Source>
+                <Reference>EPHB4</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="79987">
+                <GeneLocus>7q22.1</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
+      </DisorderGeneAssociationList>
+    </Disorder>
     <Disorder id="11979">
       <OrphaCode>90118</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90118</ExpertLink>
@@ -383163,14 +383408,26 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="1">
+            <ExternalReferenceList count="4">
               <ExternalReference id="205325">
                 <Source>HGNC</Source>
                 <Reference>15908</Reference>
               </ExternalReference>
+              <ExternalReference id="205741">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000173418</Reference>
+              </ExternalReference>
+              <ExternalReference id="205742">
+                <Source>OMIM</Source>
+                <Reference>610833</Reference>
+              </ExternalReference>
+              <ExternalReference id="205743">
+                <Source>SwissProt</Source>
+                <Reference>P61599</Reference>
+              </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="88031">
+              <Locus id="88189">
                 <GeneLocus>20p11.23</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -385373,7 +385630,7 @@
         <DisorderGeneAssociation>
           <SourceOfValidation/>
           <Gene id="16063">
-            <Name lang="en">FMRP translational regulator 1</Name>
+            <Name lang="en">fragile X messenger ribonucleoprotein 1</Name>
             <Symbol>FMR1</Symbol>
             <SynonymList count="3">
               <Synonym lang="en">FMRP</Synonym>
@@ -385565,7 +385822,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="59">
+      <DisorderGeneAssociationList count="61">
         <DisorderGeneAssociation>
           <SourceOfValidation>30639322[PMID]</SourceOfValidation>
           <Gene id="28272">
@@ -385706,6 +385963,110 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>35045343[PMID]</SourceOfValidation>
+          <Gene id="29142">
+            <Name lang="en">mannosidase alpha class 2C member 1</Name>
+            <Symbol>MAN2C1</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="5">
+              <ExternalReference id="183603">
+                <Source>HGNC</Source>
+                <Reference>6827</Reference>
+              </ExternalReference>
+              <ExternalReference id="183604">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000140400</Reference>
+              </ExternalReference>
+              <ExternalReference id="183605">
+                <Source>SwissProt</Source>
+                <Reference>Q9NTJ4</Reference>
+              </ExternalReference>
+              <ExternalReference id="183606">
+                <Source>Reactome</Source>
+                <Reference>Q9NTJ4</Reference>
+              </ExternalReference>
+              <ExternalReference id="183607">
+                <Source>OMIM</Source>
+                <Reference>154580</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="55053">
+                <GeneLocus>15q24.2</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>35051358[PMID]</SourceOfValidation>
+          <Gene id="20738">
+            <Name lang="en">BRCA1 associated protein 1</Name>
+            <Symbol>BAP1</Symbol>
+            <SynonymList count="4">
+              <Synonym lang="en">KIAA0272</Synonym>
+              <Synonym lang="en">UCHL2</Synonym>
+              <Synonym lang="en">hucep-6</Synonym>
+              <Synonym lang="en">ubiquitin carboxy-terminal hydrolase</Synonym>
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="7">
+              <ExternalReference id="126405">
+                <Source>Reactome</Source>
+                <Reference>Q92560</Reference>
+              </ExternalReference>
+              <ExternalReference id="60620">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000163930</Reference>
+              </ExternalReference>
+              <ExternalReference id="55818">
+                <Source>Genatlas</Source>
+                <Reference>BAP1</Reference>
+              </ExternalReference>
+              <ExternalReference id="55816">
+                <Source>HGNC</Source>
+                <Reference>950</Reference>
+              </ExternalReference>
+              <ExternalReference id="83235">
+                <Source>IUPHAR</Source>
+                <Reference>2332</Reference>
+              </ExternalReference>
+              <ExternalReference id="55817">
+                <Source>OMIM</Source>
+                <Reference>603089</Reference>
+              </ExternalReference>
+              <ExternalReference id="55819">
+                <Source>SwissProt</Source>
+                <Reference>Q92560</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="26795">
+                <GeneLocus>3p21.1</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
         <DisorderGeneAssociation>
           <SourceOfValidation>28251352[PMID]</SourceOfValidation>
           <Gene id="25002">
@@ -388125,14 +388486,30 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="1">
+            <ExternalReferenceList count="5">
               <ExternalReference id="205333">
                 <Source>HGNC</Source>
                 <Reference>2021</Reference>
               </ExternalReference>
+              <ExternalReference id="205728">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000109572</Reference>
+              </ExternalReference>
+              <ExternalReference id="205729">
+                <Source>OMIM</Source>
+                <Reference>600580</Reference>
+              </ExternalReference>
+              <ExternalReference id="205730">
+                <Source>IUPHAR</Source>
+                <Reference>702</Reference>
+              </ExternalReference>
+              <ExternalReference id="205731">
+                <Source>SwissProt</Source>
+                <Reference>P51790</Reference>
+              </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="88047">
+              <Locus id="88165">
                 <GeneLocus>4q33</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -394650,14 +395027,26 @@
             <GeneType id="25993">
               <Name lang="en">gene with protein product</Name>
             </GeneType>
-            <ExternalReferenceList count="1">
+            <ExternalReferenceList count="4">
               <ExternalReference id="205329">
                 <Source>HGNC</Source>
                 <Reference>18533</Reference>
               </ExternalReference>
+              <ExternalReference id="205735">
+                <Source>Ensembl</Source>
+                <Reference>ENSG00000090686</Reference>
+              </ExternalReference>
+              <ExternalReference id="205736">
+                <Source>OMIM</Source>
+                <Reference>617445</Reference>
+              </ExternalReference>
+              <ExternalReference id="205737">
+                <Source>SwissProt</Source>
+                <Reference>Q86UV5</Reference>
+              </ExternalReference>
             </ExternalReferenceList>
             <LocusList count="1">
-              <Locus id="88039">
+              <Locus id="88177">
                 <GeneLocus>1p36.12</GeneLocus>
                 <LocusKey>1</LocusKey>
               </Locus>
@@ -440295,7 +440684,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
+      <DisorderGeneAssociationList count="2">
         <DisorderGeneAssociation>
           <SourceOfValidation/>
           <Gene id="16109">
@@ -440350,6 +440739,36 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
+        <DisorderGeneAssociation>
+          <SourceOfValidation>34305140[PMID]</SourceOfValidation>
+          <Gene id="31451">
+            <Name lang="en">inosine monophosphate dehydrogenase 2</Name>
+            <Symbol>IMPDH2</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="1">
+              <ExternalReference id="205792">
+                <Source>HGNC</Source>
+                <Reference>6053</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="88279">
+                <GeneLocus>3p21.31</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="25972">
+            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
     <Disorder id="13824">
@@ -479751,6 +480170,49 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
+    <Disorder id="31449">
+      <OrphaCode>621758</OrphaCode>
+      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=621758</ExpertLink>
+      <Name lang="en">Fibrosis-neurodegeneration-cerebral angiomatosis syndrome</Name>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
+      </DisorderType>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
+      </DisorderGroup>
+      <DisorderGeneAssociationList count="1">
+        <DisorderGeneAssociation>
+          <SourceOfValidation>30138417[PMID]_30239752[PMID]</SourceOfValidation>
+          <Gene id="31450">
+            <Name lang="en">NHL repeat containing 2</Name>
+            <Symbol>NHLRC2</Symbol>
+            <SynonymList count="0">
+            </SynonymList>
+            <GeneType id="25993">
+              <Name lang="en">gene with protein product</Name>
+            </GeneType>
+            <ExternalReferenceList count="1">
+              <ExternalReference id="205787">
+                <Source>HGNC</Source>
+                <Reference>24731</Reference>
+              </ExternalReference>
+            </ExternalReferenceList>
+            <LocusList count="1">
+              <Locus id="88275">
+                <GeneLocus>10q25.3</GeneLocus>
+                <LocusKey>1</LocusKey>
+              </Locus>
+            </LocusList>
+          </Gene>
+          <DisorderGeneAssociationType id="17949">
+            <Name lang="en">Disease-causing germline mutation(s) in</Name>
+          </DisorderGeneAssociationType>
+          <DisorderGeneAssociationStatus id="17991">
+            <Name lang="en">Assessed</Name>
+          </DisorderGeneAssociationStatus>
+        </DisorderGeneAssociation>
+      </DisorderGeneAssociationList>
+    </Disorder>
     <Disorder id="31441">
       <OrphaCode>620368</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=620368</ExpertLink>
@@ -480018,7 +480480,7 @@
     <Disorder id="31416">
       <OrphaCode>619367</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619367</ExpertLink>
-      <Name lang="en">SAMD9L-associated auto-inflammatory syndrome</Name>
+      <Name lang="en">SAMD9L-associated autoinflammatory syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -480079,7 +480541,7 @@
     <Disorder id="31415">
       <OrphaCode>619363</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=619363</ExpertLink>
-      <Name lang="en">Neonatal onset-severe multisystem auto-inflammatory disease-increased IL-18</Name>
+      <Name lang="en">Neonatal-onset severe multisystemic autoinflammatory disease with increased IL18</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
