commit c92e9c5f9940abcb2c7950377e1c2245059cad90
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Mar 1 10:54:09 2021 +0100

    MARCH 2021

diff --git a/Genes associated with rare diseases/en_product6.xml b/Genes associated with rare diseases/en_product6.xml
old mode 100755
new mode 100644
index ef1fd89..1163500
--- a/Genes associated with rare diseases/en_product6.xml	
+++ b/Genes associated with rare diseases/en_product6.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-02-01 04:40:33" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-03-01 04:40:37" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="3856">
+  <DisorderList count="3853">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -304,71 +304,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="17609">
-      <OrphaCode>166068</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166068</ExpertLink>
-      <Name lang="en">Pontocerebellar hypoplasia type 5</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>21368912[PMID]</SourceOfValidation>
-          <Gene id="17741">
-            <Name lang="en">tRNA splicing endonuclease subunit 54</Name>
-            <Symbol>TSEN54</Symbol>
-            <SynonymList count="2">
-              <Synonym lang="en">SEN54</Synonym>
-              <Synonym lang="en">SEN54L</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="58222">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000182173</Reference>
-              </ExternalReference>
-              <ExternalReference id="39384">
-                <Source>Genatlas</Source>
-                <Reference>TSEN54</Reference>
-              </ExternalReference>
-              <ExternalReference id="39385">
-                <Source>HGNC</Source>
-                <Reference>27561</Reference>
-              </ExternalReference>
-              <ExternalReference id="39386">
-                <Source>OMIM</Source>
-                <Reference>608755</Reference>
-              </ExternalReference>
-              <ExternalReference id="97270">
-                <Source>Reactome</Source>
-                <Reference>Q7Z6J9</Reference>
-              </ExternalReference>
-              <ExternalReference id="39387">
-                <Source>SwissProt</Source>
-                <Reference>Q7Z6J9</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="16787">
-                <GeneLocus>17q25.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="17608">
       <OrphaCode>166063</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166063</ExpertLink>
@@ -115621,7 +115556,7 @@
     <Disorder id="1686">
       <OrphaCode>1519</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1519</ExpertLink>
-      <Name lang="en">Hypertelorism, Teebi type</Name>
+      <Name lang="en">SPECC1L-related hypertelorism syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -122751,7 +122686,7 @@
     <Disorder id="17234">
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158676</ExpertLink>
-      <Name lang="en">Dominant dystrophic epidermolysis bullosa, nails only</Name>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa, nails only</Name>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
       </DisorderType>
@@ -122816,7 +122751,7 @@
     <Disorder id="17233">
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158673</ExpertLink>
-      <Name lang="en">Acral dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa, acral form</Name>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
       </DisorderType>
@@ -126330,7 +126265,7 @@
     <Disorder id="2233">
       <OrphaCode>2407</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2407</ExpertLink>
-      <Name lang="en">LOC syndrome</Name>
+      <Name lang="en">Laryngo-onycho-cutaneous syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -138186,7 +138121,7 @@
     <Disorder id="19640">
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251393</ExpertLink>
-      <Name lang="en">Localized junctional epidermolysis bullosa, non-Herlitz type</Name>
+      <Name lang="en">Localized junctional epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -163823,8 +163758,8 @@
       <OrphaCode>268261</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=268261</ExpertLink>
       <Name lang="en">DYRK1A-related intellectual disability syndrome due to 21q22.13q22.2 microdeletion</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -225199,7 +225134,7 @@
     <Disorder id="19201">
       <OrphaCode>231568</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=231568</ExpertLink>
-      <Name lang="en">Generalized dominant dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Autosomal dominant generalized dystrophic epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -263193,11 +263128,11 @@
       <OrphaCode>313892</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313892</ExpertLink>
       <Name lang="en">Developmental and speech delay due to SOX5 deficiency</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="1">
         <DisorderGeneAssociation>
@@ -263258,11 +263193,11 @@
       <OrphaCode>313884</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=313884</ExpertLink>
       <Name lang="en">12p12.1 microdeletion syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="1">
         <DisorderGeneAssociation>
@@ -268967,7 +268902,7 @@
     <Disorder id="21219">
       <OrphaCode>306504</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306504</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa with respiratory and renal involvement</Name>
+      <Name lang="en">Interstitial lung disease-nephrotic syndrome-epidermolysis bullosa syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -269326,140 +269261,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="21238">
-      <OrphaCode>306597</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306597</ExpertLink>
-      <Name lang="en">X-linked Opitz G/BBB syndrome</Name>
-      <DisorderType id="21443">
-        <Name lang="en">Etiological subtype</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation/>
-          <Gene id="16398">
-            <Name lang="en">midline 1</Name>
-            <Symbol>MID1</Symbol>
-            <SynonymList count="5">
-              <Synonym lang="en">FXY</Synonym>
-              <Synonym lang="en">OS</Synonym>
-              <Synonym lang="en">Opitz/BBB syndrome</Synonym>
-              <Synonym lang="en">RNF59</Synonym>
-              <Synonym lang="en">TRIM18</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="58607">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000101871</Reference>
-              </ExternalReference>
-              <ExternalReference id="31126">
-                <Source>Genatlas</Source>
-                <Reference>MID1</Reference>
-              </ExternalReference>
-              <ExternalReference id="31124">
-                <Source>HGNC</Source>
-                <Reference>7095</Reference>
-              </ExternalReference>
-              <ExternalReference id="31123">
-                <Source>OMIM</Source>
-                <Reference>300552</Reference>
-              </ExternalReference>
-              <ExternalReference id="97232">
-                <Source>Reactome</Source>
-                <Reference>O15344</Reference>
-              </ExternalReference>
-              <ExternalReference id="33462">
-                <Source>SwissProt</Source>
-                <Reference>O15344</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="16711">
-                <GeneLocus>Xp22</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="25972">
-            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
-    <Disorder id="21237">
-      <OrphaCode>306588</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306588</ExpertLink>
-      <Name lang="en">Autosomal dominant Opitz G/BBB syndrome</Name>
-      <DisorderType id="21443">
-        <Name lang="en">Etiological subtype</Name>
-      </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>25412741[PMID]</SourceOfValidation>
-          <Gene id="20299">
-            <Name lang="en">sperm antigen with calponin homology and coiled-coil domains 1 like</Name>
-            <Symbol>SPECC1L</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">CYTSA</Synonym>
-              <Synonym lang="en">KIAA0376</Synonym>
-              <Synonym lang="en">cytokinesis and spindle organization A</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="142949">
-                <Source>Reactome</Source>
-                <Reference>Q69YQ0</Reference>
-              </ExternalReference>
-              <ExternalReference id="60093">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000100014</Reference>
-              </ExternalReference>
-              <ExternalReference id="52316">
-                <Source>Genatlas</Source>
-                <Reference>SPECC1L</Reference>
-              </ExternalReference>
-              <ExternalReference id="52314">
-                <Source>HGNC</Source>
-                <Reference>29022</Reference>
-              </ExternalReference>
-              <ExternalReference id="52315">
-                <Source>OMIM</Source>
-                <Reference>614140</Reference>
-              </ExternalReference>
-              <ExternalReference id="52317">
-                <Source>SwissProt</Source>
-                <Reference>Q69YQ0</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="37619">
-                <GeneLocus>22q11.23</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="21236">
       <OrphaCode>306577</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=306577</ExpertLink>
@@ -271071,7 +270872,7 @@
     <Disorder id="21112">
       <OrphaCode>300333</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=300333</ExpertLink>
-      <Name lang="en">Nephrotic syndrome-deafness-pretibial epidermolysis bullosa syndrome</Name>
+      <Name lang="en">Nephrotic syndrome-epidermolysis bullosa-sensorineural deafness syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -271832,8 +271633,8 @@
       <OrphaCode>464311</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=464311</ExpertLink>
       <Name lang="en">Intellectual disability syndrome due to a DYRK1A point mutation</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21443">
+        <Name lang="en">Etiological subtype</Name>
       </DisorderType>
       <DisorderGroup id="36554">
         <Name lang="en">Subtype of disorder</Name>
@@ -368032,7 +367833,7 @@
     <Disorder id="11907">
       <OrphaCode>89842</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89842</ExpertLink>
-      <Name lang="en">Recessive dystrophic epidermolysis bullosa, generalized intermediate</Name>
+      <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, intermediate form</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -374233,7 +374034,7 @@
     <Disorder id="11903">
       <OrphaCode>89838</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89838</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, autosomal recessive K14</Name>
+      <Name lang="en">Autosomal recessive generalized epidermolysis bullosa simplex</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -392038,7 +391839,7 @@
     <Disorder id="11422">
       <OrphaCode>79396</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79396</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, generalized severe</Name>
+      <Name lang="en">Autosomal dominant generalized epidermolysis bullosa simplex, severe form</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -392773,7 +392574,7 @@
     <Disorder id="11425">
       <OrphaCode>79399</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79399</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, generalized intermediate</Name>
+      <Name lang="en">Autosomal dominant generalized epidermolysis bullosa simplex, intermediate form</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -392888,7 +392689,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79401</ExpertLink>
-      <Name lang="en">PLEC-related epidermolysis bullosa simplex without extracutaneous involvement</Name>
+      <Name lang="en">PLEC-related intermediate epidermolysis bullosa simplex without extracutaneous involvement</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393068,7 +392869,7 @@
     <Disorder id="11429">
       <OrphaCode>79403</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79403</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa-pyloric atresia syndrome</Name>
+      <Name lang="en">Junctional epidermolysis bullosa with pyloric atresia</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393183,7 +392984,7 @@
     <Disorder id="11428">
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79402</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa, generalized intermediate</Name>
+      <Name lang="en">Intermediate generalized junctional epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393524,7 +393325,7 @@
     <Disorder id="11430">
       <OrphaCode>79404</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79404</ExpertLink>
-      <Name lang="en">Junctional epidermolysis bullosa, generalized severe</Name>
+      <Name lang="en">Severe generalized junctional epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393761,7 +393562,7 @@
     <Disorder id="11435">
       <OrphaCode>79409</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79409</ExpertLink>
-      <Name lang="en">Recessive dystrophic epidermolysis bullosa inversa</Name>
+      <Name lang="en">Dystrophic epidermolysis bullosa inversa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393826,7 +393627,7 @@
     <Disorder id="11434">
       <OrphaCode>79408</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79408</ExpertLink>
-      <Name lang="en">Severe generalized recessive dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Autosomal recessive generalized dystrophic epidermolysis bullosa, severe form</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393946,7 +393747,7 @@
     <Disorder id="11437">
       <OrphaCode>79411</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79411</ExpertLink>
-      <Name lang="en">Transient bullous dermolysis of the newborn</Name>
+      <Name lang="en">Self-improving dystrophic epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -394011,7 +393812,7 @@
     <Disorder id="11436">
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79410</ExpertLink>
-      <Name lang="en">Pretibial dystrophic epidermolysis bullosa</Name>
+      <Name lang="en">Localized dystrophic epidermolysis bullosa, pretibial form</Name>
       <DisorderType id="21450">
         <Name lang="en">Clinical subtype</Name>
       </DisorderType>
@@ -413903,7 +413704,7 @@
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
       </DisorderGroup>
-      <DisorderGeneAssociationList count="2">
+      <DisorderGeneAssociationList count="1">
         <DisorderGeneAssociation>
           <SourceOfValidation>26669662[PMID]</SourceOfValidation>
           <Gene id="16647">
@@ -413952,62 +413753,6 @@
             <Name lang="en">Assessed</Name>
           </DisorderGeneAssociationStatus>
         </DisorderGeneAssociation>
-        <DisorderGeneAssociation>
-          <SourceOfValidation>26669662[PMID]</SourceOfValidation>
-          <Gene id="21196">
-            <Name lang="en">solute carrier family 52 member 2</Name>
-            <Symbol>SLC52A2</Symbol>
-            <SynonymList count="6">
-              <Synonym lang="en">D15Ertd747e</Synonym>
-              <Synonym lang="en">FLJ11856</Synonym>
-              <Synonym lang="en">GPCR41</Synonym>
-              <Synonym lang="en">PAR1</Synonym>
-              <Synonym lang="en">RFVT2</Synonym>
-              <Synonym lang="en">hRFT3</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="126409">
-                <Source>Reactome</Source>
-                <Reference>Q9HAB3</Reference>
-              </ExternalReference>
-              <ExternalReference id="89570">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000185803</Reference>
-              </ExternalReference>
-              <ExternalReference id="100008">
-                <Source>Genatlas</Source>
-                <Reference>SLC52A2</Reference>
-              </ExternalReference>
-              <ExternalReference id="70273">
-                <Source>HGNC</Source>
-                <Reference>30224</Reference>
-              </ExternalReference>
-              <ExternalReference id="70274">
-                <Source>OMIM</Source>
-                <Reference>607882</Reference>
-              </ExternalReference>
-              <ExternalReference id="70276">
-                <Source>SwissProt</Source>
-                <Reference>Q9HAB3</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="26803">
-                <GeneLocus>8q24.3</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
     <Disorder id="12582">
