commit b116a922e5dbc6d1e710e6df2ff6dd8b181e3236
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Mon Feb 1 13:09:28 2021 +0100

    FEB 2021

diff --git a/Genes associated with rare diseases/en_product6.xml b/Genes associated with rare diseases/en_product6.xml
old mode 100644
new mode 100755
index 2838df6..ef1fd89
--- a/Genes associated with rare diseases/en_product6.xml	
+++ b/Genes associated with rare diseases/en_product6.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-01-01 04:39:55" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-02-01 04:40:33" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="3858">
+  <DisorderList count="3856">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -45091,7 +45091,7 @@
     <Disorder id="485">
       <OrphaCode>1880</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1880</ExpertLink>
-      <Name lang="en">Ebstein malformation</Name>
+      <Name lang="en">Ebstein malformation of the tricuspid valve</Name>
       <DisorderType id="21415">
         <Name lang="en">Morphological anomaly</Name>
       </DisorderType>
@@ -91242,7 +91242,7 @@
     <Disorder id="1007">
       <OrphaCode>528</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=528</ExpertLink>
-      <Name lang="en">Berardinelli-Seip congenital lipodystrophy</Name>
+      <Name lang="en">Congenital generalized lipodystrophy</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -103179,7 +103179,7 @@
     <Disorder id="16705">
       <OrphaCode>137678</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=137678</ExpertLink>
-      <Name lang="en">Czech dysplasia, metatarsal type</Name>
+      <Name lang="en">Spondyloepiphyseal dysplasia with metatarsal shortening</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -104596,7 +104596,7 @@
     <Disorder id="1451">
       <OrphaCode>1200</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=1200</ExpertLink>
-      <Name lang="en">Choanal atresia-hearing loss-cardiac defects-craniofacial dysmorphism syndrome</Name>
+      <Name lang="en">Burn-McKeown syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -122450,7 +122450,7 @@
     <Disorder id="17237">
       <OrphaCode>158687</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158687</ExpertLink>
-      <Name lang="en">Lethal acantholytic epidermolysis bullosa</Name>
+      <Name lang="en">Lethal acantholytic erosive disorder</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -122752,11 +122752,11 @@
       <OrphaCode>158676</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158676</ExpertLink>
       <Name lang="en">Dominant dystrophic epidermolysis bullosa, nails only</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="1">
         <DisorderGeneAssociation>
@@ -122817,11 +122817,11 @@
       <OrphaCode>158673</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158673</ExpertLink>
       <Name lang="en">Acral dystrophic epidermolysis bullosa</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="1">
         <DisorderGeneAssociation>
@@ -122881,7 +122881,7 @@
     <Disorder id="17232">
       <OrphaCode>158668</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=158668</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex due to plakophilin deficiency</Name>
+      <Name lang="en">Ectodermal dysplasia-skin fragility syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -138187,11 +138187,11 @@
       <OrphaCode>251393</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=251393</ExpertLink>
       <Name lang="en">Localized junctional epidermolysis bullosa, non-Herlitz type</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="2">
         <DisorderGeneAssociation>
@@ -182291,7 +182291,7 @@
     <Disorder id="3298">
       <OrphaCode>832</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=832</ExpertLink>
-      <Name lang="en">Succinyl-CoA:3-ketoacid CoA transferase deficiency</Name>
+      <Name lang="en">Succinyl-CoA:3-oxoacid CoA transferase deficiency</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -280617,7 +280617,7 @@
     <Disorder id="24234">
       <OrphaCode>468678</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=468678</ExpertLink>
-      <Name lang="en">Intellectual disability-microcephaly-strabismus-behavioral abnormalities syndrome</Name>
+      <Name lang="en">White-Sutton syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -314643,7 +314643,7 @@
     <Disorder id="26575">
       <OrphaCode>508529</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=508529</ExpertLink>
-      <Name lang="en">Generalized basal epidermolysis bullosa simplex with skin atrophy, scarring and hair loss</Name>
+      <Name lang="en">Intermediate epidermolysis bullosa simplex with cardiomyopathy</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -326265,7 +326265,7 @@
     <Disorder id="8620">
       <OrphaCode>2908</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=2908</ExpertLink>
-      <Name lang="en">Kindler syndrome</Name>
+      <Name lang="en">Kindler epidermolysis bullosa</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -368094,136 +368094,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="11906">
-      <OrphaCode>89841</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89841</ExpertLink>
-      <Name lang="en">Centripetalis recessive dystrophic epidermolysis bullosa</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation/>
-          <Gene id="15781">
-            <Name lang="en">collagen type VII alpha 1 chain</Name>
-            <Symbol>COL7A1</Symbol>
-            <SynonymList count="2">
-              <Synonym lang="en">LC collagen</Synonym>
-              <Synonym lang="en">collagen VII, alpha-1 polypeptide</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="59408">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000114270</Reference>
-              </ExternalReference>
-              <ExternalReference id="28159">
-                <Source>Genatlas</Source>
-                <Reference>COL7A1</Reference>
-              </ExternalReference>
-              <ExternalReference id="28157">
-                <Source>HGNC</Source>
-                <Reference>2214</Reference>
-              </ExternalReference>
-              <ExternalReference id="28156">
-                <Source>OMIM</Source>
-                <Reference>120120</Reference>
-              </ExternalReference>
-              <ExternalReference id="82866">
-                <Source>Reactome</Source>
-                <Reference>Q02388</Reference>
-              </ExternalReference>
-              <ExternalReference id="32753">
-                <Source>SwissProt</Source>
-                <Reference>Q02388</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="24001">
-                <GeneLocus>3p21.31</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="25972">
-            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
-    <Disorder id="11904">
-      <OrphaCode>89839</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=89839</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex superficialis</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>11874498[PMID]</SourceOfValidation>
-          <Gene id="15781">
-            <Name lang="en">collagen type VII alpha 1 chain</Name>
-            <Symbol>COL7A1</Symbol>
-            <SynonymList count="2">
-              <Synonym lang="en">LC collagen</Synonym>
-              <Synonym lang="en">collagen VII, alpha-1 polypeptide</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="59408">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000114270</Reference>
-              </ExternalReference>
-              <ExternalReference id="28159">
-                <Source>Genatlas</Source>
-                <Reference>COL7A1</Reference>
-              </ExternalReference>
-              <ExternalReference id="28157">
-                <Source>HGNC</Source>
-                <Reference>2214</Reference>
-              </ExternalReference>
-              <ExternalReference id="28156">
-                <Source>OMIM</Source>
-                <Reference>120120</Reference>
-              </ExternalReference>
-              <ExternalReference id="82866">
-                <Source>Reactome</Source>
-                <Reference>Q02388</Reference>
-              </ExternalReference>
-              <ExternalReference id="32753">
-                <Source>SwissProt</Source>
-                <Reference>Q02388</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="24001">
-                <GeneLocus>3p21.31</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="11919">
       <OrphaCode>90001</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=90001</ExpertLink>
@@ -371176,7 +371046,7 @@
     <Disorder id="11805">
       <OrphaCode>88618</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=88618</ExpertLink>
-      <Name lang="en">Psychomotor delay due to S-adenosylhomocysteine hydrolase deficiency</Name>
+      <Name lang="en">S-adenosylhomocysteine hydrolase deficiency</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393018,7 +392888,7 @@
     <Disorder id="11427">
       <OrphaCode>79401</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79401</ExpertLink>
-      <Name lang="en">Epidermolysis bullosa simplex, Ogna type</Name>
+      <Name lang="en">PLEC-related epidermolysis bullosa simplex without extracutaneous involvement</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -393314,11 +393184,11 @@
       <OrphaCode>79402</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79402</ExpertLink>
       <Name lang="en">Junctional epidermolysis bullosa, generalized intermediate</Name>
-      <DisorderType id="21450">
-        <Name lang="en">Clinical subtype</Name>
+      <DisorderType id="21394">
+        <Name lang="en">Disease</Name>
       </DisorderType>
-      <DisorderGroup id="36554">
-        <Name lang="en">Subtype of disorder</Name>
+      <DisorderGroup id="36547">
+        <Name lang="en">Disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="5">
         <DisorderGeneAssociation>
@@ -394142,11 +394012,11 @@
       <OrphaCode>79410</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=79410</ExpertLink>
       <Name lang="en">Pretibial dystrophic epidermolysis bullosa</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
+      <DisorderType id="21450">
+        <Name lang="en">Clinical subtype</Name>
       </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
+      <DisorderGroup id="36554">
+        <Name lang="en">Subtype of disorder</Name>
       </DisorderGroup>
       <DisorderGeneAssociationList count="1">
         <DisorderGeneAssociation>
@@ -412298,7 +412168,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95702</ExpertLink>
-      <Name lang="en">Cytomegalic congenital adrenal hypoplasia</Name>
+      <Name lang="en">X-linked adrenal hypoplasia congenita due to a NR0B1 point mutation</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
