commit bf321f9bd09e8ca26476600ab0f29df16ef01085
Author: davidlagorce <david.lagorce@inserm.fr>
Date:   Thu Apr 1 11:20:48 2021 +0200

    APRIL 2021

diff --git a/Genes associated with rare diseases/en_product6.xml b/Genes associated with rare diseases/en_product6.xml
index 1163500..8dbdb5c 100644
--- a/Genes associated with rare diseases/en_product6.xml	
+++ b/Genes associated with rare diseases/en_product6.xml	
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
-<JDBOR date="2021-03-01 04:40:37" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
+<JDBOR date="2021-04-01 04:41:19" version="1.3.7 / 4.1.7 [2020-12-03] (orientdb version)" copyright="Orphanet (c) 2021" dbserver="jdbc:sybase:Tds:canard.orpha.net:2020">
   <Availability> 
     <Licence>
       <FullName lang="en">Creative Commons Attribution 4.0 International</FullName>
@@ -7,7 +7,7 @@
       <LegalCode>https://creativecommons.org/licenses/by/4.0/legalcode</LegalCode>
     </Licence>
   </Availability>
-  <DisorderList count="3853">
+  <DisorderList count="3845">
     <Disorder id="17601">
       <OrphaCode>166024</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=166024</ExpertLink>
@@ -89058,8 +89058,8 @@
       <OrphaCode>847</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=847</ExpertLink>
       <Name lang="en">Alpha-thalassemia-X-linked intellectual disability syndrome</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
+      <DisorderType id="21401">
+        <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
       <DisorderGroup id="36547">
         <Name lang="en">Disorder</Name>
@@ -191719,7 +191719,7 @@
     <Disorder id="3183">
       <OrphaCode>3051</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=3051</ExpertLink>
-      <Name lang="en">Intellectual disability-sparse hair-brachydactyly syndrome</Name>
+      <Name lang="en">Nicolaides-Baraitser syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -249241,7 +249241,7 @@
     <Disorder id="22081">
       <OrphaCode>352577</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=352577</ExpertLink>
-      <Name lang="en">Severe feeding difficulties-failure to thrive-microcephaly due to ASXL3 deficiency syndrome</Name>
+      <Name lang="en">Bainbridge-Ropers syndrome</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -252274,72 +252274,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="20904">
-      <OrphaCode>294049</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294049</ExpertLink>
-      <Name lang="en">Reunion Island Larsen-like syndrome</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>24755949[PMID]</SourceOfValidation>
-          <Gene id="15350">
-            <Name lang="en">beta-1,4-galactosyltransferase 7</Name>
-            <Symbol>B4GALT7</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">XGALT-1</Synonym>
-              <Synonym lang="en">beta4Gal-T7</Synonym>
-              <Synonym lang="en">galactosyltransferase I</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="59261">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000027847</Reference>
-              </ExternalReference>
-              <ExternalReference id="26100">
-                <Source>Genatlas</Source>
-                <Reference>B4GALT7</Reference>
-              </ExternalReference>
-              <ExternalReference id="26098">
-                <Source>HGNC</Source>
-                <Reference>930</Reference>
-              </ExternalReference>
-              <ExternalReference id="26097">
-                <Source>OMIM</Source>
-                <Reference>604327</Reference>
-              </ExternalReference>
-              <ExternalReference id="82785">
-                <Source>Reactome</Source>
-                <Reference>Q9UBV7</Reference>
-              </ExternalReference>
-              <ExternalReference id="33907">
-                <Source>SwissProt</Source>
-                <Reference>Q9UBV7</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="23089">
-                <GeneLocus>5q35.3</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="20901">
       <OrphaCode>294016</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=294016</ExpertLink>
@@ -284554,7 +284488,7 @@
     <Disorder id="22920">
       <OrphaCode>404443</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=404443</ExpertLink>
-      <Name lang="en">Tall stature-intellectual disability-facial dysmorphism syndrome</Name>
+      <Name lang="en">Tatton-Brown-Rahman syndrome</Name>
       <DisorderType id="21401">
         <Name lang="en">Malformation syndrome</Name>
       </DisorderType>
@@ -316067,70 +316001,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="25668">
-      <OrphaCode>494348</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=494348</ExpertLink>
-      <Name lang="en">Early-onset familial noncirrhotic portal hypertension</Name>
-      <DisorderType id="21394">
-        <Name lang="en">Disease</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>26874653[PMID]</SourceOfValidation>
-          <Gene id="15860">
-            <Name lang="en">deoxyguanosine kinase</Name>
-            <Symbol>DGUOK</Symbol>
-            <SynonymList count="1">
-              <Synonym lang="en">dGK</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="60542">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000114956</Reference>
-              </ExternalReference>
-              <ExternalReference id="28535">
-                <Source>Genatlas</Source>
-                <Reference>DGUOK</Reference>
-              </ExternalReference>
-              <ExternalReference id="28533">
-                <Source>HGNC</Source>
-                <Reference>2858</Reference>
-              </ExternalReference>
-              <ExternalReference id="28532">
-                <Source>OMIM</Source>
-                <Reference>601465</Reference>
-              </ExternalReference>
-              <ExternalReference id="60543">
-                <Source>Reactome</Source>
-                <Reference>Q16854</Reference>
-              </ExternalReference>
-              <ExternalReference id="32871">
-                <Source>SwissProt</Source>
-                <Reference>Q16854</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="22425">
-                <GeneLocus>2p13.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="25972">
-            <Name lang="en">Disease-causing germline mutation(s) (loss of function) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="25683">
       <OrphaCode>494547</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=494547</ExpertLink>
@@ -400872,72 +400742,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="11698">
-      <OrphaCode>85328</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85328</ExpertLink>
-      <Name lang="en">X-linked intellectual disability, Turner type</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>18252223[PMID]</SourceOfValidation>
-          <Gene id="18651">
-            <Name lang="en">HECT, UBA and WWE domain containing E3 ubiquitin protein ligase 1</Name>
-            <Symbol>HUWE1</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">Ib772</Synonym>
-              <Synonym lang="en">KIAA0312</Synonym>
-              <Synonym lang="en">UREB1</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="59460">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000086758</Reference>
-              </ExternalReference>
-              <ExternalReference id="43060">
-                <Source>Genatlas</Source>
-                <Reference>HUWE1</Reference>
-              </ExternalReference>
-              <ExternalReference id="43061">
-                <Source>HGNC</Source>
-                <Reference>30892</Reference>
-              </ExternalReference>
-              <ExternalReference id="43062">
-                <Source>OMIM</Source>
-                <Reference>300697</Reference>
-              </ExternalReference>
-              <ExternalReference id="59461">
-                <Source>Reactome</Source>
-                <Reference>Q7Z6Z7</Reference>
-              </ExternalReference>
-              <ExternalReference id="43063">
-                <Source>SwissProt</Source>
-                <Reference>Q7Z6Z7</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="10833">
-                <GeneLocus>Xp11.22</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="11699">
       <OrphaCode>85329</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=85329</ExpertLink>
@@ -411969,7 +411773,7 @@
     <Disorder id="12657">
       <OrphaCode>95702</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=95702</ExpertLink>
-      <Name lang="en">X-linked adrenal hypoplasia congenita due to a NR0B1 point mutation</Name>
+      <Name lang="en">X-linked adrenal hypoplasia congenita</Name>
       <DisorderType id="21394">
         <Name lang="en">Disease</Name>
       </DisorderType>
@@ -417340,336 +417144,6 @@
         </DisorderGeneAssociation>
       </DisorderGeneAssociationList>
     </Disorder>
-    <Disorder id="12532">
-      <OrphaCode>93974</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93974</ExpertLink>
-      <Name lang="en">Smith-Fineman-Myers syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>10751095[PMID]</SourceOfValidation>
-          <Gene id="15335">
-            <Name lang="en">ATRX chromatin remodeler</Name>
-            <Symbol>ATRX</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">RAD54 homolog (S. cerevisiae)</Synonym>
-              <Synonym lang="en">XH2</Synonym>
-              <Synonym lang="en">XNP</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="57778">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000085224</Reference>
-              </ExternalReference>
-              <ExternalReference id="26027">
-                <Source>Genatlas</Source>
-                <Reference>ATRX</Reference>
-              </ExternalReference>
-              <ExternalReference id="26025">
-                <Source>HGNC</Source>
-                <Reference>886</Reference>
-              </ExternalReference>
-              <ExternalReference id="26024">
-                <Source>OMIM</Source>
-                <Reference>300032</Reference>
-              </ExternalReference>
-              <ExternalReference id="33892">
-                <Source>SwissProt</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-              <ExternalReference id="143930">
-                <Source>Reactome</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="39327">
-                <GeneLocus>Xq21.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
-    <Disorder id="12529">
-      <OrphaCode>93971</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93971</ExpertLink>
-      <Name lang="en">Chudley-Lowry-Hoar syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>15508018[PMID]</SourceOfValidation>
-          <Gene id="15335">
-            <Name lang="en">ATRX chromatin remodeler</Name>
-            <Symbol>ATRX</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">RAD54 homolog (S. cerevisiae)</Synonym>
-              <Synonym lang="en">XH2</Synonym>
-              <Synonym lang="en">XNP</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="57778">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000085224</Reference>
-              </ExternalReference>
-              <ExternalReference id="26027">
-                <Source>Genatlas</Source>
-                <Reference>ATRX</Reference>
-              </ExternalReference>
-              <ExternalReference id="26025">
-                <Source>HGNC</Source>
-                <Reference>886</Reference>
-              </ExternalReference>
-              <ExternalReference id="26024">
-                <Source>OMIM</Source>
-                <Reference>300032</Reference>
-              </ExternalReference>
-              <ExternalReference id="33892">
-                <Source>SwissProt</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-              <ExternalReference id="143930">
-                <Source>Reactome</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="39327">
-                <GeneLocus>Xq21.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
-    <Disorder id="12528">
-      <OrphaCode>93970</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93970</ExpertLink>
-      <Name lang="en">Holmes-Gang syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>11050622[PMID]</SourceOfValidation>
-          <Gene id="15335">
-            <Name lang="en">ATRX chromatin remodeler</Name>
-            <Symbol>ATRX</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">RAD54 homolog (S. cerevisiae)</Synonym>
-              <Synonym lang="en">XH2</Synonym>
-              <Synonym lang="en">XNP</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="57778">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000085224</Reference>
-              </ExternalReference>
-              <ExternalReference id="26027">
-                <Source>Genatlas</Source>
-                <Reference>ATRX</Reference>
-              </ExternalReference>
-              <ExternalReference id="26025">
-                <Source>HGNC</Source>
-                <Reference>886</Reference>
-              </ExternalReference>
-              <ExternalReference id="26024">
-                <Source>OMIM</Source>
-                <Reference>300032</Reference>
-              </ExternalReference>
-              <ExternalReference id="33892">
-                <Source>SwissProt</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-              <ExternalReference id="143930">
-                <Source>Reactome</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="39327">
-                <GeneLocus>Xq21.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
-    <Disorder id="12531">
-      <OrphaCode>93973</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93973</ExpertLink>
-      <Name lang="en">Carpenter-Waziri syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>10398237[PMID]</SourceOfValidation>
-          <Gene id="15335">
-            <Name lang="en">ATRX chromatin remodeler</Name>
-            <Symbol>ATRX</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">RAD54 homolog (S. cerevisiae)</Synonym>
-              <Synonym lang="en">XH2</Synonym>
-              <Synonym lang="en">XNP</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="57778">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000085224</Reference>
-              </ExternalReference>
-              <ExternalReference id="26027">
-                <Source>Genatlas</Source>
-                <Reference>ATRX</Reference>
-              </ExternalReference>
-              <ExternalReference id="26025">
-                <Source>HGNC</Source>
-                <Reference>886</Reference>
-              </ExternalReference>
-              <ExternalReference id="26024">
-                <Source>OMIM</Source>
-                <Reference>300032</Reference>
-              </ExternalReference>
-              <ExternalReference id="33892">
-                <Source>SwissProt</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-              <ExternalReference id="143930">
-                <Source>Reactome</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="39327">
-                <GeneLocus>Xq21.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
-    <Disorder id="12530">
-      <OrphaCode>93972</OrphaCode>
-      <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93972</ExpertLink>
-      <Name lang="en">Juberg-Marsidi syndrome</Name>
-      <DisorderType id="21401">
-        <Name lang="en">Malformation syndrome</Name>
-      </DisorderType>
-      <DisorderGroup id="36547">
-        <Name lang="en">Disorder</Name>
-      </DisorderGroup>
-      <DisorderGeneAssociationList count="1">
-        <DisorderGeneAssociation>
-          <SourceOfValidation>8630485[PMID]</SourceOfValidation>
-          <Gene id="15335">
-            <Name lang="en">ATRX chromatin remodeler</Name>
-            <Symbol>ATRX</Symbol>
-            <SynonymList count="3">
-              <Synonym lang="en">RAD54 homolog (S. cerevisiae)</Synonym>
-              <Synonym lang="en">XH2</Synonym>
-              <Synonym lang="en">XNP</Synonym>
-            </SynonymList>
-            <GeneType id="25993">
-              <Name lang="en">gene with protein product</Name>
-            </GeneType>
-            <ExternalReferenceList count="6">
-              <ExternalReference id="57778">
-                <Source>Ensembl</Source>
-                <Reference>ENSG00000085224</Reference>
-              </ExternalReference>
-              <ExternalReference id="26027">
-                <Source>Genatlas</Source>
-                <Reference>ATRX</Reference>
-              </ExternalReference>
-              <ExternalReference id="26025">
-                <Source>HGNC</Source>
-                <Reference>886</Reference>
-              </ExternalReference>
-              <ExternalReference id="26024">
-                <Source>OMIM</Source>
-                <Reference>300032</Reference>
-              </ExternalReference>
-              <ExternalReference id="33892">
-                <Source>SwissProt</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-              <ExternalReference id="143930">
-                <Source>Reactome</Source>
-                <Reference>P46100</Reference>
-              </ExternalReference>
-            </ExternalReferenceList>
-            <LocusList count="1">
-              <Locus id="39327">
-                <GeneLocus>Xq21.1</GeneLocus>
-                <LocusKey>1</LocusKey>
-              </Locus>
-            </LocusList>
-          </Gene>
-          <DisorderGeneAssociationType id="17949">
-            <Name lang="en">Disease-causing germline mutation(s) in</Name>
-          </DisorderGeneAssociationType>
-          <DisorderGeneAssociationStatus id="17991">
-            <Name lang="en">Assessed</Name>
-          </DisorderGeneAssociationStatus>
-        </DisorderGeneAssociation>
-      </DisorderGeneAssociationList>
-    </Disorder>
     <Disorder id="12492">
       <OrphaCode>93932</OrphaCode>
       <ExpertLink lang="en">http://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&amp;Expert=93932</ExpertLink>
